
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: natural history of coronavirus and shedding of it from an infected person</h2><br /></div><br /><br /><br /><div class="card"><h3>Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, a novel coronavirus (2019-nCoV) caused an outbreak in Wuhan, China, and soon spread to other parts of the world. It was believed that 2019-nCoV was transmitted through respiratory tract and then induced pneumonia, thus molecular diagnosis based on oral swabs was used for confirmation of this disease. Likewise, patient will be released upon two times of negative detection from oral swabs. However, many coronaviruses can also be transmitted through oral-fecal route by infecting intestines. Whether 2019-nCoV infected patients also carry virus in other organs like intestine need to be tested. We conducted investigation on patients in a local hospital who were infected with this virus. We found the presence of 2019-nCoV in anal swabs and blood as well, and more anal swab positives than oral swab positives in a later stage of infection, suggesting shedding and thereby transmitted through oral-fecal route. We also showed serology test can improve detection positive rate thus should be used in future epidemiology. Our report provides a cautionary warning that 2019-nCoV may be shed through multiple routes.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Coronavirus Strain 692 by Immune Electron Microscopy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Utilization of the technique of immune electron microscopy has enabled the detection of a coronavirus in organ culture harvests derived from a washing from an adult with an acute upper respiratory tract illness; convalescent serum was the source of specific antibody.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>The human enteric coronaviruses.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A coronarirus was seen in the faeces from 15 (4.2%) of 355 adults with diarrhoea and from 5 (5.2%) of 96 adults without diarrhoea. Similar particles were seen in the faeces from 5 (2.2%) of 227 children aged 1--14 years with gastroenteritis, but in none of those from 230 infants under one year of age with gastroenteritis. There was no evidence that the coronavirus was responsible for any of 34 outbreaks of gastroenteritis, although it possibly caused diarrhoea in patients admitted to a psycho-geriatric unit. Excretion of the virus often continued for many months. One strain was propagated in human embryo kidney monolayers and human embryo intestinal organ cultures, although serial passage could not be accomplished.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-28</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</jats:p></p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization and Complete Genome Sequence of a Novel Coronavirus, Coronavirus HKU1, from Patients with Pneumonia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Despite extensive laboratory investigations in patients with respiratory tract infections, no microbiological cause can be identified in a significant proportion of patients. In the past 3 years, several novel respiratory viruses, including human metapneumovirus, severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV), and human coronavirus NL63, were discovered. Here we report the discovery of another novel coronavirus, coronavirus HKU1 (CoV-HKU1), from a 71-year-old man with pneumonia who had just returned from Shenzhen, China. Quantitative reverse transcription-PCR showed that the amount of CoV-HKU1 RNA was 8.5 to 9.6 × 10(6) copies per ml in his nasopharyngeal aspirates (NPAs) during the first week of the illness and dropped progressively to undetectable levels in subsequent weeks. He developed increasing serum levels of specific antibodies against the recombinant nucleocapsid protein of CoV-HKU1, with immunoglobulin M (IgM) titers of 1:20, 1:40, and 1:80 and IgG titers of <1:1,000, 1:2,000, and 1:8,000 in the first, second and fourth weeks of the illness, respectively. Isolation of the virus by using various cell lines, mixed neuron-glia culture, and intracerebral inoculation of suckling mice was unsuccessful. The complete genome sequence of CoV-HKU1 is a 29,926-nucleotide, polyadenylated RNA, with G+C content of 32%, the lowest among all known coronaviruses with available genome sequence. Phylogenetic analysis reveals that CoV-HKU1 is a new group 2 coronavirus. Screening of 400 NPAs, negative for SARS-CoV, from patients with respiratory illness during the SARS period identified the presence of CoV-HKU1 RNA in an additional specimen, with a viral load of 1.13 × 10(6) copies per ml, from a 35-year-old woman with pneumonia. Our data support the existence of a novel group 2 coronavirus associated with pneumonia in humans.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Excretion and detection of SARS coronavirus and its nucleic acid from digestive system</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">AIM: To study whether severe acute respiratory syndrome coronavirus (SARS-CoV) could be excreted from digestive system. METHODS: Cell culture and semi-nested RT-PCR were used to detect SARS-CoV and its RNA from 21 stool and urine samples, and a kind of electropositive filter media particles was used to concentrate the virus in 10 sewage samples from two hospitals receiving SARS patients in Beijing in China. RESULTS: It was demonstrated that there was no live SARS-CoV in all samples collected, but the RNA of SARS-CoV could be detected in seven stool samples from SARS patients with any one of the symptoms of fever, malaise, cough, or dyspnea, in 10 sewage samples before disinfection and 3 samples after disinfection from the two hospitals. The RNA could not be detected in urine and stool samples from patients recovered from SARS. CONCLUSION: Nucleic acid of SARS-CoV can be excreted through the stool of patients into sewage system, and the possibility of SARS-CoV transmitting through digestive system cannot be excluded.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although human coronavirus OC43-OC43 (HCoV-OC43) is the coronavirus most commonly associated with human infections, little is known about its molecular epidemiology and evolution. We conducted a molecular epidemiology study to investigate different genotypes and potential recombination in HCoV-OC43. Twenty-nine HCoV-OC43 strains from nasopharyngeal aspirates, collected from 2004 to 2011, were subjected to RNA-dependent RNA polymerase (RdRp), spike, and nucleocapsid gene analysis. Phylogenetic analysis showed at least three distinct clusters of HCoV-OC43, although 10 unusual strains displayed incongruent phylogenetic positions between RdRp and spike genes. This suggested the presence of four HCoV-OC43 genotypes (A to D), with genotype D most likely arising from recombination. The complete genome sequencing of two genotype C and D strains and bootscan analysis showed recombination events between genotypes B and C in the generation of genotype D. Of the 29 strains, none belonged to the more ancient genotype A, 5 from 2004 belonged to genotype B, 15 from 2004 to 2006 belonged to genotype C, and 1 from 2004 and all 8 from 2008 to 2011 belonged to the recombinant genotype D. Molecular clock analysis using spike and nucleocapsid genes dated the most recent common ancestor of all genotypes to the 1950s, genotype B and C to the 1980s, genotype B to the 1990s, and genotype C to the late 1990s to early 2000s, while the recombinant genotype D strains were detected as early as 2004. This represents the first study to describe natural recombination in HCoV-OC43 and the evolution of different genotypes over time, leading to the emergence of novel genotype D, which is associated with pneumonia in our elderly population.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2&ndash;14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3&ndash;4 days without truncation and at 5&ndash;9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Human coronaviruses and other respiratory infections in young adults on a university campus: Prevalence, symptoms, and shedding</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Sep 24</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The prevalence, symptom course, and shedding in persons infected with the 4 most common human coronaviruses (HCoV)‐229E, HKU1, NL63, and OC43 are poorly described. OBJECTIVES: We estimate their prevalence and associated symptoms among college students identified via a social network study design. PATIENTS/METHODS: We collected 1‐3 samples (n = 250 specimens) from 176 participants between October 2012 and January 17, 2013: participants with acute respiratory infection (ARI; cough and body aches or chills or fever/feverishness) and their social contacts. Virus was detected using RT‐PCR. RESULTS: 30.4% (76/250) of specimens tested positive for any virus tested, and 4.8% (12/250) were positive for 2 or more viruses. Human coronaviruses (HCoVs [22.0%; 55/250]), rhinovirus (7.6%; 19/250), and influenza A (6.4%; 16/250) were most prevalent. Symptoms changed significantly over time among ARI participants with HCoV: the prevalence of cough and chills decreased over 6 days (P = .04, and P = .01, respectively), while runny nose increased over the same period (P = .02). HCoV‐NL63 was the most frequent virus detected 6 days following symptom onset (8.9%), followed by rhinovirus (6.7%). CONCLUSIONS: During a 3‐month period covering a single season, HCoVs were common, even among social contacts without respiratory symptoms; specific symptoms may change over the course of HCoV‐associated illness and were similar to symptoms from influenza and rhinovirus.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory Diagnosis of SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">The virologic test results of 415 patients with severe acute respiratory syndrome (SARS) were examined. The peak detection rate for SARS-associated coronavirus occurred at week 2 after illness onset for respiratory specimens, at weeks 2 to 3 for stool or rectal swab specimens, and at week 4 for urine specimens. The latest stool sample that was positive by reverse transcription–polymerase chain reaction (RT-PCR) was collected on day 75 while the patient was receiving intensive care. Tracheal aspirate and stool samples had a higher diagnostic yield (RT-PCR average positive rate for first 2 weeks: 66.7% and 56.5%, respectively). Pooled throat and nasal swabs, rectal swab, nasal swab, throat swab, and nasopharyngeal aspirate specimens provided a moderate yield (29.7%–40.0%), whereas throat washing and urine specimens showed a lower yield (17.3% and 4.5%). The collection procedures for stool and pooled nasal and throat swab specimens were the least likely to transmit infection, and the combination gave the highest yield for coronavirus detection by RT-PCR. Positive virologic test results in patient groups were associated with mechanical ventilation or death (p < 0.001), suggesting a correlation between viral load and disease severity.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus-specific RNA and Antibody from Convalescent-phase SARS Patients Discharged from Hospital</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Oct</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) is caused by a novel coronavirus (SARS-CoV). In a longitudinal cross-sectional study, we determined the prevalence of virus in bodily excretions and time of seroconversion in discharged patients with SARS. Conjunctival, throat, stool, and urine specimens were collected weekly from 64 patients and tested for SARS-CoV RNA by real-time polymerase chain reaction; serum samples were collected weekly and tested for SARS-CoV antibody with indirect enzyme immunoassay and immunofluorescence assay. In total, 126 conjunctival, 124 throat swab, 116 stool, and 124 urine specimens were analyzed. Five patients had positive stool samples, collected in weeks 5–9. Two patients seroconverted in weeks 7 and 8; the others were seropositive at the first serum sample collection. In this study, 5 (7.8%) of 64 patients continued to shed viral RNA in stool samples only, for up to week 8 of illness. Most seroconversions occurred by week 6 of illness.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Isolation from the First Patient with SARS-CoV-2 in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Novel coronavirus (SARS-CoV-2) is found to cause a large outbreak started from Wuhan since December 2019 in China and SARS-CoV-2 infections have been reported with epidemiological linkage to China in 25 countries until now. We isolated SARS-CoV-2 from the oropharyngeal sample obtained from the patient with the first laboratory-confirmed SARS- CoV-2 infection in Korea. Cytopathic effects of SARS-CoV-2 in the Vero cell cultures were confluent 3 days after the first blind passage of the sample. Coronavirus was confirmed with spherical particle having a fringe reminiscent of crown on transmission electron microscopy.Phylogenetic analyses of whole genome sequences showed that it clustered with other SARS- CoV-2 reported from Wuhan.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Seroprevalence of SARS coronavirus antibody in household contacts.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Between March and July 2003, 671 cases of severe acute respiratory syndrome (SARS) were diagnosed in Taiwan with a total of 84 fatalities. After the epidemic, a serological survey was conducted involving the asymptomatic household contacts. Household contacts of 13 index patients were enrolled in the study. Contact history and clinical symptoms of the household contacts were recorded by standardized questionnaires. Blood samples of patients and household contacts were collected at least 28 days after symptom onset in the index patients or household exposure in the contacts for SARS-associated coronavirus (SARS-CoV) IgG testing. On the basis of this investigation, 29 persons (25 adults and 4 children) were identified as having had unprotected exposure to the index cases before infection-control practices were implemented. Laboratory evaluation of clinical specimens showed no evidence of transmission of SARS-CoV infection to any contacts. This investigation demonstrated that subclinical transmission among household contacts was low in the described setting.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Get Real</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since this is going to be a post about the coronavirus, let’s start off with this PSA: wash your hands. ... OK, either tomorrow or Friday I hope to do a post on all the things that are going on in the biopharma industry for a possible coronavirus treatment. ... It was clearly related to the virus from the first case (reported on January 19 in the same county in Washington state), descended from it in a way that makes it almost certain that the coronavirus has been spreading undetected among that population for weeks.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Isolation from the First Patient with SARS-CoV-2 in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">Novel coronavirus (SARS-CoV-2) is found to cause a large outbreak started from Wuhan since December 2019 in China and SARS-CoV-2 infections have been reported with epidemiological linkage to China in 25 countries until now. We isolated SARS-CoV-2 from the oropharyngeal sample obtained from the patient with the first laboratory-confirmed SARS-CoV-2 infection in Korea. Cytopathic effects of SARS-CoV-2 in the Vero cell cultures were confluent 3 days after the first blind passage of the sample. Coronavirus was confirmed with spherical particle having a fringe reminiscent of crown on transmission electron microscopy. Phylogenetic analyses of whole genome sequences showed that it clustered with other SARS-CoV-2 reported from Wuhan.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct Measurement of Rates of Asymptomatic Infection and Clinical Care-Seeking for Seasonal Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The pandemic potential of the novel coronavirus (nCoV) that emerged in Wuhan, China, during December 2019 is strongly tied to the number and contagiousness of undocumented human infections. Here we present findings from a proactive longitudinal sampling study of acute viral respiratory infections that documents rates of asymptomatic infection and clinical care seeking for seasonal coronavirus.  We find that the majority of infections are asymptomatic by most symptom definitions and that only 4% of individuals experiencing a seasonal coronavirus infection episode sought medical care for their symptoms.  These numbers indicate that a very high percentage of seasonal coronavirus infections are undocumented and provide a reference for understanding the spread of the emergent nCoV.</jats:p></p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation of Coronavirus NL63 from Blood from Children in Rural Haiti: Phylogenetic Similarities with Recent Isolates from Malaysia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus (HCoV) NL63 is recognized as a common cause of upper respiratory infections and influenza-like illness. In screening children with acute undifferentiated febrile illness in a school cohort in rural Haiti, we identified HCoV-NL63 in blood samples from four children. Cases clustered over an 11-day period; children did not have respiratory symptoms, but two had gastrointestinal complaints. On phylogenetic analysis, the Haitian HCoV-NL63 strains cluster together in a highly supported monophyletic clade linked most closely with recently reported strains from Malaysia; two respiratory HCoV-NL63 strains identified in north Florida in the same general period form a separate clade, albeit again with close linkages with the Malaysian strains. Our data highlight the variety of presentations that may be seen with HCoV-NL63, and underscore the apparent ease with which CoV strains move among countries, with our data consistent with recurrent introduction of strains into the Caribbean (Haiti and Florida) from Asia.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Moderate mutation rate in the SARS coronavirus genome and its implications</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Moderate mutation rate in the SARS coronavirus genome and its implications</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Moderate mutation rate in the SARS coronavirus genome and its implications</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Long-term SARS Coronavirus Excretion from Patient Cohort, China</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Oct</span></i><p style="text-align: justify; text-justify: inter-word;">This study investigated the long-term excretion of severe acute respiratory syndrome–associated coronavirus in sputum and stool specimens from 56 infected patients. The median (range) duration of virus excretion in sputa and stools was 21 (14–52) and 27 (16–126) days, respectively. Coexisting illness or conditions were associated with longer viral excretion in stools.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Two antigenic groups of human coronaviruses detected by using enzyme-linked immunosorbent assay.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Paired sera from volunteers inoculated with one of the five recently isolated strains of human coronavirus (HCV), AD, GI, HO, PA, and RO, none of which has been grown in tissue culture, or with strain OC38 were tested against coronavirus antigens by enzyme-linked immunosorbent assay. When HCV strains OC43, 229E, or the 229E-related tissue culture-adapted strains PR and TO were used as antigens, it was shown that all strains fell into one of two antigenic groups. The HCV OC43 group was comprised of strains OC43, GI, HO, and RO, and the HCV 229E group contained strains AD and PA as well as the tissue culture-adapted strains PR, TO, and KI. Enzyme-linked immunosorbent assay of the paired sera with the coronavirus mouse hepatitis virus strain 3 as antigen confirmed the relationship of this virus to the HCV OC43 group but not to the HCV 229E group.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human coronavirus NL63 (HCoV-NL63) is a globally endemic pathogen causing mild and severe respiratory tract infections with reinfections occurring repeatedly throughout a lifetime. METHODS: Nasal samples were collected in coastal Kenya through community-based and hospital-based surveillance. HCoV-NL63 was detected with multiplex real-time reverse transcription PCR, and positive samples were targeted for nucleotide sequencing of the spike (S) protein. Additionally, paired samples from 25 individuals with evidence of repeat HCoV-NL63 infection were selected for whole-genome virus sequencing. RESULTS: HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching. CONCLUSIONS: In this coastal Kenya setting, HCoV-NL63 exhibited low prevalence in hospital pediatric pneumonia admissions. Clade persistence with low genetic diversity suggest limited immune selection, and absence of detectable clade switching in reinfections indicates initial exposure was insufficient to elicit a protective immune response.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human coronavirus NL63 (HCoV-NL63) is a globally endemic pathogen causing mild and severe respiratory tract infections with reinfections occurring repeatedly throughout a lifetime. METHODS: Nasal samples were collected in coastal Kenya through community-based and hospital-based surveillance. HCoV-NL63 was detected with multiplex real-time reverse transcription PCR, and positive samples were targeted for nucleotide sequencing of the spike (S) protein. Additionally, paired samples from 25 individuals with evidence of repeat HCoV-NL63 infection were selected for whole-genome virus sequencing. RESULTS: HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching. CONCLUSIONS: In this coastal Kenya setting, HCoV-NL63 exhibited low prevalence in hospital pediatric pneumonia admissions. Clade persistence with low genetic diversity suggest limited immune selection, and absence of detectable clade switching in reinfections indicates initial exposure was insufficient to elicit a protective immune response.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human coronavirus NL63 (HCoV-NL63) is a globally endemic pathogen causing mild and severe respiratory tract infections with reinfections occurring repeatedly throughout a lifetime. METHODS: Nasal samples were collected in coastal Kenya through community-based and hospital-based surveillance. HCoV-NL63 was detected with multiplex real-time reverse transcription PCR, and positive samples were targeted for nucleotide sequencing of the spike (S) protein. Additionally, paired samples from 25 individuals with evidence of repeat HCoV-NL63 infection were selected for whole-genome virus sequencing. RESULTS: HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching. CONCLUSIONS: In this coastal Kenya setting, HCoV-NL63 exhibited low prevalence in hospital pediatric pneumonia admissions. Clade persistence with low genetic diversity suggest limited immune selection, and absence of detectable clade switching in reinfections indicates initial exposure was insufficient to elicit a protective immune response.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-associated Coronavirus Transmission, United States</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">To better assess the risk for transmission of the severe acute respiratory syndrome–associated coronavirus (SARS-CoV), we obtained serial specimens and clinical and exposure data from seven confirmed U.S. SARS patients and their 10 household contacts. SARS-CoV was detected in a day-14 sputum specimen from one case-patient and in five stool specimens from two case-patients. In one case-patient, SARS-CoV persisted in stool for at least 26 days after symptom onset. The highest amounts of virus were in the day-14 sputum sample and a day-14 stool sample. Residual respiratory symptoms were still present in recovered SARS case-patients 2 months after illness onset. Possible transmission of SARS-CoV occurred in one household contact, but this person had also traveled to a SARS-affected area. The data suggest that SARS-CoV is not always transmitted efficiently. Laboratory diagnosis of SARS-CoV infection is difficult; thus, sputum and stool specimens should be included in the diagnostic work-up for SARS-CoV infection.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation, propagation, genome analysis and epidemiology of HKU1 betacoronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">From 1 January 2009 to 31 May 2013, 15 287 respiratory specimens submitted to the Clinical Virology Laboratory at the Children’s Hospital Colorado were tested for human coronavirus RNA by reverse transcription-PCR. Human coronaviruses HKU1, OC43, 229E and NL63 co-circulated during each of the respiratory seasons but with significant year-to-year variability, and cumulatively accounted for 7.4–15.6 % of all samples tested during the months of peak activity. A total of 79 (0.5 % prevalence) specimens were positive for human betacoronavirus HKU1 RNA. Genotypes HKU1 A and B were both isolated from clinical specimens and propagated on primary human tracheal–bronchial epithelial cells cultured at the air–liquid interface and were neutralized in vitro by human intravenous immunoglobulin and by polyclonal rabbit antibodies to the spike glycoprotein of HKU1. Phylogenetic analysis of the deduced amino acid sequences of seven full-length genomes of Colorado HKU1 viruses and the spike glycoproteins from four additional HKU1 viruses from Colorado and three from Brazil demonstrated remarkable conservation of these sequences with genotypes circulating in Hong Kong and France. Within genotype A, all but one of the Colorado HKU1 sequences formed a unique subclade defined by three amino acid substitutions (W197F, F613Y and S752F) in the spike glycoprotein and exhibited a unique signature in the acidic tandem repeat in the N-terminal region of the nsp3 subdomain. Elucidating the function of and mechanisms responsible for the formation of these varying tandem repeats will increase our understanding of the replication process and pathogenicity of HKU1 and potentially of other coronaviruses.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU1 Infection in the United States</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 May</span></i><p style="text-align: justify; text-justify: inter-word;">In 2005, a new human coronavirus, HCoV-HKU1, was identified in Hong Kong. We screened respiratory specimens collected from December 16, 2001, to December 15, 2002, from children <5 years of age who tested negative for respiratory syncytial virus, parainfluenza viruses, influenza virus, and adenovirus for HCoV-HKU1 by reverse transcription–polymerase chain reaction. Overall, 1,048 respiratory specimens from 851 children were tested, and 9 HCoV-HKU1–positive children (1%) were identified, 2 of whom had 2 positive specimens. Children who had HCoV-HKU1 infection had evidence of either upper or lower respiratory tract infection or both. Two patients had disease beyond the respiratory tract. HCoV-HKU1 was identified from December 2001 to February 2002. Sequence analyses suggest that a single strain was circulating. HCoV-HKU1 is therefore likely circulating in the United States and is associated with upper and lower respiratory tract disease.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>The time course of the immune response to experimental coronavirus infection of man.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">After preliminary trials, the detailed changes in the concentration of specific circulating and local antibodies were followed in 15 volunteers inoculated with coronavirus 229E. Ten of them, who had significantly lower concentrations of pre-existing antibody than the rest, became infected and eight of these developed colds. A limited investigation of circulating lymphocyte populations showed some lymphocytopenia in infected volunteers. In this group, antibody concentrations started to increase 1 week after inoculation and reached a maximum about 1 week later. Thereafter antibody titres slowly declined. Although concentrations were still slightly raised 1 year later, this did not always prevent reinfection when volunteers were then challenged with the homologous virus. However, the period of virus shedding was shorter than before and none developed a cold. All of the uninfected group were infected on re-challenge although they also appeared to show some resistance to disease and in the extent of infection. These results are discussed with reference to natural infections with coronavirus and with other infections, such as rhinovirus infections.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>A new coronavirus associated with human respiratory disease in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging infectious diseases, such as SARS and Zika, present a major threat to public health(1-3). Despite intense research efforts, how, when and where new diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city Wuhan, Hubei province, China. Up to 25th of January 2020, at least 1,975 cases have been reported since the first patient was hospitalized on the 12th of December 2019. Epidemiological investigation suggested that the outbreak was associated with a seafood market in Wuhan. We studied one patient who was a worker at the market, and who was admitted to Wuhan Central Hospital on 26th of December 2019 experiencing a severe respiratory syndrome including fever, dizziness and cough. Metagenomic RNA sequencing(4) of a bronchoalveolar lavage fluid sample identified a novel RNA virus from the family Coronaviridae, designed here as WH-Human-1 coronavirus. Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Human Coronavirus HCoV-NL63</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 25</span></i><p style="text-align: justify; text-justify: inter-word;">Even though coronavirus infection of humans is not normally associated with severe diseases, the identification of the coronavirus responsible for the outbreak of severe acute respiratory syndrome showed that highly pathogenic coronaviruses can enter the human population. Shortly thereafter, in Holland in 2004, another novel human coronavirus (HCoV-NL63) was isolated from a seven-month old infant suffering from respiratory symptoms. This virus has subsequently been identified in various countries, indicating a worldwide distribution. HCoV-NL63 has been shown to infect mainly children and the immunocommpromised, who presented with either mild upper respiratory symptoms (cough, fever and rhinorrhoea) or more serious lower respiratory tract involvement such as bronchiolitis and croup, which was observed mainly in younger children. In fact, HCoV-NL63 is the aetiological agent for up to 10% of all respiratory diseases. This review summarizes recent findings of human coronavirus HCoV-NL63 infections, including isolation and identification, phylogeny and taxonomy, genome structure and transcriptional regulation, transmission and pathogenesis, and detection and diagnosis.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of SARS-associated Coronavirus in Throat Wash and Saliva in Early Diagnosis</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome–associated coronavirus (SARS-CoV) is thought to be transmitted primarily through dispersal of droplets, but little is known about the load of SARS-CoV in oral droplets. We examined oral specimens, including throat wash and saliva, and found large amounts of SARS-CoV RNA in both throat wash (9.58 x 10(2) to 5.93 x 10(6) copies/mL) and saliva (7.08 x 10(3) to 6.38 x 10(8) copies/mL) from all specimens of 17 consecutive probable SARS case-patients, supporting the possibility of transmission through oral droplets. Immunofluorescence study showed replication of SARS-CoV in the cells derived from throat wash, demonstrating the possibility of developing a convenient antigen detection assay. This finding, with the high detection rate a median of 4 days after disease onset and before the development of lung lesions in four patients, suggests that throat wash and saliva should be included in sample collection guidelines for SARS diagnosis.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>The Effects of Temperature and Relative Humidity on the Viability of the SARS Coronavirus</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Oct 1</span></i><p style="text-align: justify; text-justify: inter-word;">The main route of transmission of SARS CoV infection is presumed to be respiratory droplets. However the virus is also detectable in other body fluids and excreta. The stability of the virus at different temperatures and relative humidity on smooth surfaces were studied. The dried virus on smooth surfaces retained its viability for over 5 days at temperatures of 22–25°C and relative humidity of 40–50%, that is, typical air-conditioned environments. However, virus viability was rapidly lost (>3 log(10)) at higher temperatures and higher relative humidity (e.g., 38°C, and relative humidity of >95%). The better stability of SARS coronavirus at low temperature and low humidity environment may facilitate its transmission in community in subtropical area (such as Hong Kong) during the spring and in air-conditioned environments. It may also explain why some Asian countries in tropical area (such as Malaysia, Indonesia or Thailand) with high temperature and high relative humidity environment did not have major community outbreaks of SARS.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic diversity of SARS-CoV-2 in Coronavirus Disease 2019 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) has infected more than 75,000 individuals and spread to over 20 countries. It is still unclear how fast the virus evolved and how the virus interacts with other microorganisms in the lung.We have conducted metatranscriptome sequencing for the bronchoalveolar lavage fluid of eight SARS-CoV-2 patients, 25 community-acquired pneumonia (CAP) patients, and 20 healthy controls.The median number of intra-host variants was 1-4 in SARS-CoV-2 infected patients, which ranged between 0 and 51 in different samples. The distribution of variants on genes was similar to those observed in the population data (110 sequences). However, very few intra-host variants were observed in the population as polymorphism, implying either a bottleneck or purifying selection involved in the transmission of the virus, or a consequence of the limited diversity represented in the current polymorphism data. Although current evidence did not support the transmission of intra-host variants in a person-to-person spread, the risk should not be overlooked. The microbiota in SARS-CoV-2 infected patients was similar to those in CAP, either dominated by the pathogens or with elevated levels of oral and upper respiratory commensal bacteria.SARS-CoV-2 evolves in vivo after infection, which may affect its virulence, infectivity, and transmissibility. Although how the intra-host variant spreads in the population is still elusive, it is necessary to strengthen the surveillance of the viral evolution in the population and associated clinical changes.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus Outbreak: What We Know and What We Don&#x2019;t</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Phylogenetic analyses reveal that SARS-CoV-2 is closely related to a group of SARS-like coronaviruses. However, it remains unclear where the virus comes from and how it was transmitted to humans in the first place. Unlike with other zoonotic agents such as hantavirus and arenavirus, thus far we haven’t found a SARS virus in animals that is the same as that in humans. Fortunately, SARS virus has not appeared in humans since 2004. In contrast, this new virus seems to have stronger transmission capabilities among people. Compared to the primary virus in humans, we still know less about whether, what, and how the virus has changed and the effect of the changes for their epidemics in humans. Control and prevention of the disease is especially difficult in China and elsewhere if there are infected individuals with no clinical signs.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the case fatality ratio of the COVID-19 epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Corona Virus Disease 2019 (COVID-19) due to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan city and rapidly spread throughout China since late December 2019. Crude case fatality ratio (CFR) with dividing the number of known deaths by the number of confirmed cases does not represent the true CFR and might be off by orders of magnitude. We aim to provide a precise estimate of the CFR of COVID-19 using statistical models at the early stage of the epidemic. Methods: We extracted data from the daily released epidemic report published by the National Health Commission P. R. China from 20 Jan 2020, to 1 March 2020. Competing risk model was used to obtain the cumulative hazards for death, cure, and cure-death hazard ratio. Then the CFR was estimated based on the slope of the last piece in joinpoint regression model, which reflected the most recent trend of the epidemic.  Results: As of 1 March 2020, totally 80,369 cases were diagnosed as COVID-19 in China. The CFR of COVID-19 were estimated to be 70.9% (95% CI: 66.8%-75.6%) during Jan 20-Feb 2, 20.2% (18.6%-22.1%) during Feb 3-14, 6.9% (6.4%-7.4%) during Feb 15-23, 1.5% (1.4%-1.6%) during Feb 24-March 1 in Hubei province, and 20.3% (17.0%-25.3%) during Jan 20-28, 1.9% (1.8%-2.1%) during Jan 29-Feb 12, 0.9% (0.8%-1.1%) during Feb 13-18, 0.4% (0.4%-0.5%) during Feb 19-March 1 in other areas of China, respectively.  Conclusions: Based on analyses of public data, we found that the CFR in Hubei was much higher than that of other regions in China, over 3 times in all estimation. The CFR would follow a downwards trend based on our estimation from recently released data. Nevertheless, at early stage of outbreak, CFR estimates should be viewed cautiously because of limited data source on true onset and recovery time.</jats:p></p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of the Coronavirus Associated with Severe Acute Respiratory Syndrome: A Review of Data from The Chinese University of Hong Kong</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The epidemic of the severe acute respiratory syndrome (SARS) has swept through the globe with more than 8000 reported probable cases. In Hong Kong, the hardest hit areas included our teaching hospital and the Amoy Gardens apartment complex. A novel coronavirus, SARS-coronavirus (SARS-CoV), with a single-stranded plus sense RNA genome, was promptly implicated as the causative agent and subsequently fulfilled Koch's postulates. To aid the understanding of SARS-CoV, groups of investigators rapidly sequenced viral isolates around the world. We were the third group in the world to release the complete SARS-CoV genome sequence (isolate CUHK-W1) on the world-wide web. With other isolates from patients of distinct epidemiological backgrounds, we additionally sequenced four complete (CUHK-Su10, CUHK-AG01, CUHK-AG02, CUHK-AG03) and two partial SARS-CoV genomes. The reviewed data obtained from representative patients from the hospital and community outbreaks has documented the evolution of the virus in this epidemic. Their sequence variations also revealed a remarkable epidemiological correlation. We demonstrate that sequence variations in the SARS-CoV genome can be applied as a powerful molecular tool in tracing the route of transmission, when used adjunctively with standard epidemiology.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>First Complete Genome Sequence of Human Coronavirus HKU1 from a Nonill Bat Guano Miner in Thailand</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 7</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus HKU1 (HCoV-HKU1) was first detected in a patient with viral pneumonia from Hong Kong in 2004. Here, we report the first complete genome sequence of HCoV-HKU1 from Thailand, obtained from a nonill person who worked in a bat cave. Phylogenetic tree analysis revealed it as a group B HCoV-HKU1.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Variation of SARS Coronavirus in Beijing Hospital</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">To characterize genetic variation of severe acute respiratory syndrome–associated coronavirus (SARS-CoV) transmitted in the Beijing area during the epidemic outbreak of 2003, we sequenced 29 full-length S genes of SARS-CoV from 20 hospitalized SARS patients on our unit, the Beijing 302 Hospital. Viral RNA templates for the S-gene amplification were directly extracted from raw clinical samples, including plasma, throat swab, sputum, and stool, during the course of the epidemic in the Beijing area. We used a TA-cloning assay with direct analysis of nested reverse transcription–polymerase chain reaction products in sequence. One hundred thirteen sequence variations with nine recurrent variant sites were identified in analyzed S-gene sequences compared with the BJ01 strain of SARS-CoV. Among them, eight variant sites were, we think, the first documented. Our findings demonstrate the coexistence of S-gene sequences with and without substitutions (referred to BJ01) in samples analyzed from some patients.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>The New Coronavirus, the Current King of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are respiratory viruses whose appearance on electron microscopy looks like tiny crowns (Figure 1). The first coronaviruses discovered were the cause of pediatric and adult respiratory infections that were not particularly dangerous. One study from Vanderbilt found coronavirus in about 5% of samples from upper respiratory illness and 8% from lower respiratory illness [1]. Most clinically significant coronavirus infections have been found in children &lt; 2 years of age, as reviewed in this journal in 2009 [2], although adults also might suffer severe infections [3, 4]. This was the picture as late as 2018: a group of respiratory viruses that could cause serious illness in the very young, but with negligible mortality [5, 6].</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization and Complete Genome Sequence of Human Coronavirus NL63 Isolated in China</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus NL63 (HCoV-NL63) was first discovered in Amsterdam in 2004 and was identified as a new human respiratory coronavirus. We here report the first complete genome sequence of HCoV-NL63 strain CBJ 037 isolated in 2008 from a patient with bronchitis in Beijing, China.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviridae and SARS-associated Coronavirus Strain HSR1</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">During the recent severe acute respiratory (SARS) outbreak, the etiologic agent was identified as a new coronavirus (CoV). We have isolated a SARS-associated CoV (SARS-CoV) strain by injecting Vero cells with a sputum specimen from an Italian patient affected by a severe pneumonia; the patient traveled from Vietnam to Italy in March 2003. Ultrastructural analysis of infected Vero cells showed the virions within cell vesicles and around the cell membrane. The full-length viral genome sequence was similar to those derived from the Hong-Kong Hotel M isolate. By using both real-time reverse transcription–polymerase chain reaction TaqMan assay and an infectivity plaque assay, we determined that approximately 360 viral genomes were required to generate a PFU. In addition, heparin (100 μg/mL) inhibited infection of Vero cells by 50%. Overall, the molecular and biologic characteristics of the strain HSR1 provide evidence that SARS-CoV forms a fourth genetic coronavirus group with distinct genomic and biologic features.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objectives Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. Methods Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. Results The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. Conclusion SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus antibody titres in sera of healthy adults and experimentally infected volunteers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Six coronaviruses isolated in the U.S.A. have been inoculated into volunteers and all produced colds. Between 10 and 20% of infected volunteers developed heterologous antibody responses after these and other experimental infections with coronaviruses. The haemagglutination-inhibition test with the OC43 virus strain was found to detect antibody rises after infection with a variety of strains. Studies on normal adult sera taken between 1965 and 1970 revealed a high frequency of neutralizing antibody to one strain (229 E) and a frequency of HI antibody to strain OC43 which fluctuated from year to year. Complement-fixing antibodies to these two viruses were also found, revealing an apparent increase in the activity of coronaviruses in the general population of the U.K., during the winter of 1968-9.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Human coronaviruses are uncommon in patients with gastrointestinal illness</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronaviruses infect numerous animal species causing a variety of illnesses including respiratory, neurologic and enteric disease. Human coronaviruses (HCoV) are mainly associated with respiratory tract disease but have been implicated in enteric disease. OBJECTIVES: To investigate the frequency of coronaviruses in stool samples from children and adults with gastrointestinal illness by RT-PCR. STUDY DESIGN: Clinical samples submitted for infectious diarrhea testing were collected from December 2007 through March 2008. RNA extraction and RT-PCR was performed for stools negative for Clostridium difficile using primer sets against HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1. Clinical data from samples positive for coronaviruses were reviewed and recorded. RESULTS: Samples from 479 patients were collected including 151 pediatric (<= 18yrs), and 328 adults (> 18yrs). Of these samples, 4 patients (1.3%, 2 adult; 2 pediatric) screened positive for the presence of a coronavirus. All detected coronaviruses were identified as HCoV-HKU1. No stools screened positive for either HCoV-229E, HCoV-NL63 or HCoV-OC43. All HCoV-HKU1 positive samples occurred between mid January to mid February. Clinical manifestations from HCoV-HKU1 positive patients included diarrhea, emesis and respiratory complaints. Three (75%) patients were admitted to the hospital with a median length of stay of 6 days. CONCLUSIONS: Coronaviruses as a group are not commonly identified in stool samples of patients presenting with gastrointestinal illness. HCoV-HKU1 can be identified in stool samples from children and adults with gastrointestinal disease, with most individuals having respiratory findings as well. No stool samples screened positive for HCoV-NL63, HCoV-229E, or HCoV-OC43.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Severe Acute Respiratory Syndrome (SARS)-Coronavirus Accessory Proteins in Virus Pathogenesis</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Nov 7</span></i><p style="text-align: justify; text-justify: inter-word;">A respiratory disease caused by a novel coronavirus, termed the severe acute respiratory syndrome coronavirus (SARS-CoV), was first reported in China in late 2002. The subsequent efficient human-to-human transmission of this virus eventually affected more than 30 countries worldwide, resulting in a mortality rate of ~10% of infected individuals. The spread of the virus was ultimately controlled by isolation of infected individuals and there has been no infections reported since April 2004. However, the natural reservoir of the virus was never identified and it is not known if this virus will re-emerge and, therefore, research on this virus continues. The SARS-CoV genome is about 30 kb in length and is predicted to contain 14 functional open reading frames (ORFs). The genome encodes for proteins that are homologous to known coronavirus proteins, such as the replicase proteins (ORFs 1a and 1b) and the four major structural proteins: nucleocapsid (N), spike (S), membrane (M) and envelope (E). SARS-CoV also encodes for eight unique proteins, called accessory proteins, with no known homologues. This review will summarize the current knowledge on SARS-CoV accessory proteins and will include: (i) expression and processing; (ii) the effects on cellular processes; and (iii) functional studies.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63, France</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">The human coronavirus NL63 (HCoV-NL63) was first identified in the Netherlands, and its circulation in France has not been investigated. We studied HCoV-NL63 infection in hospitalized children diagnosed with respiratory tract infections. From November 2002 to April 2003, we evaluated 300 respiratory specimens for HCoV-NL63. Of the 300 samples, 28 (9.3%) were positive for HCoV-NL63. The highest prevalence was found in February (18%). The main symptoms were fever (61%), rhinitis (39%), bronchiolitis (39%), digestive problems (33%), otitis (28%), pharyngitis (22%), and conjunctivitis (17%). A fragment of the spike protein gene was sequenced to determine the variety of circulating HCoV-NL63. Phylogenetic analysis indicated that strains with different genetic markers cocirculate in France.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence for gastrointestinal infection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The new coronavirus (SARS-CoV-2) outbreak originating from Wuhan, China, poses a threat to global health. While it's evident that the virus invades respiratory tract and transmits from human to human through airway, other viral tropisms and transmission routes remain unknown. We tested viral RNA in stool from 73 SARS-CoV-2-infected hospitalized patients using rRT-PCR. 53.42% of the patients tested positive in stool. 23.29% of the patients remained positive in feces even after the viral RNA decreased to undetectable level in respiratory tract. The viral RNA was also detected in gastrointestinal tissues. Furthermore, gastric, duodenal and rectal epithelia showed positive immunofluorescent staining of viral host receptor ACE2 and viral nucleocapsid protein in a case of SARS-CoV-2 infection. Our results provide evidence for gastrointestinal infection of SARS-CoV-2, highlighting its potential fecal-oral transmission route.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Persistence of coronaviruses on inanimate surfaces and their inactivation with biocidal agents</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Currently, the emergence of a novel human coronavirus, SARS-CoV-2, has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described with incubation times between 2-10 days, facilitating its spread via droplets, contaminated hands or surfaces. We therefore reviewed the literature on all available information about the persistence of human and veterinary coronaviruses on inanimate surfaces as well as inactivation strategies with biocidal agents used for chemical disinfection, e.g. in healthcare facilities. The analysis of 22 studies reveals that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days, but can be efficiently inactivated by surface disinfection procedures with 62–71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite within 1 minute. Other biocidal agents such as 0.05–0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. As no specific therapies are available for SARS-CoV-2, early containment and prevention of further spread will be crucial to stop the ongoing outbreak and to control this novel infectious thread.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequence of Human Coronavirus NL63 CN0601/14, First Isolated in South Korea</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Mar 22</span></i><p style="text-align: justify; text-justify: inter-word;">We report here the complete genome sequence of the human coronavirus NL63 CN0601/14 strain, first isolated from South Korea. It contains 18-nucleotide discontinuous deletions of the open reading frame 1a (ORF1a) and spike regions. This study will aid in our understanding of the complete genome sequences of isolated coronaviruses in South Korea.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of SARS Coronavirus in Patients with Suspected SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Cases of severe acute respiratory syndrome (SARS) were investigated for SARS coronavirus (SARS-CoV) through RNA tests, serologic response, and viral culture. Of 537 specimens from patients in whom SARS was clinically diagnosed, 332 (60%) had SARS-CoV RNA in one or more clinical specimens, compared with 1 (0.3%) of 332 samples from controls. Of 417 patients with clinical SARS from whom paired serum samples were available, 92% had an antibody response. Rates of viral RNA positivity increased progressively and peaked at day 11 after onset of illness. Although viral RNA remained detectable in respiratory secretions and stool and urine specimens for >30 days in some patients, virus could not be cultured after week 3 of illness. Nasopharyngeal aspirates, throat swabs, or sputum samples were the most useful clinical specimens in the first 5 days of illness, but later in the illness viral RNA could be detected more readily in stool specimens.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Stability of bovine coronavirus on lettuce surfaces under household refrigeration conditions</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Fecal suspensions with an aerosol route of transmission were responsible for a cluster of severe acute respiratory syndrome (SARS) cases in 2003 in Hong Kong. Based on that event, the World Health Organization recommended that research be implemented to define modes of transmission of SARS coronavirus through sewage, feces, food and water. Environmental studies have shown that animal coronaviruses remain infectious in water and sewage for up to a year depending on the temperature and humidity. In this study, we examined coronavirus stability on lettuce surfaces. A cell culture adapted bovine coronavirus, diluted in growth media or in bovine fecal suspensions to simulate fecal contamination was used to spike romaine lettuce. qRT-PCR detected viral RNA copy number ranging from 6.6 × 10(4) to 1.7 × 10(6) throughout the experimental period of 30 days. Whereas infectious viruses were detected for at least 14 days, the amount of infectious virus varied, depending upon the diluent used for spiking the lettuce. UV and confocal microscopic observation indicated attachment of residual labeled virions to the lettuce surface after the elution procedure, suggesting that rates of inactivation or detection of the virus may be underestimated. Thus, it is possible that contaminated vegetables may be potential vehicles for coronavirus zoonotic transmission to humans.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Microbiologic Characteristics, Serologic Responses, and Clinical Manifestations in Severe Acute Respiratory Syndrome, Taiwan</h3>
<i class="tooltip">2003<span class="tooltiptext">2003 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">The genome of one Taiwanese severe acute respiratory syndrome-associated coronavirus (SARS-CoV) strain (TW1) was 29,729 nt in length. Viral RNA may persist for some time in patients who seroconvert, and some patients may lack an antibody response (immunoglobulin G) to SARS-CoV >21 days after illness onset. An upsurge of antibody response was associated with the aggravation of respiratory failure.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Aerosol and surface stability of HCoV-19 (SARS-CoV-2) compared to SARS-CoV-1</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel human coronavirus, now named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, referred to as HCoV-19 here) that emerged in Wuhan, China in late 2019 is now causing a pandemic.  Here, we analyze the aerosol and surface stability of HCoV-19 and compare it with SARS-CoV-1, the most closely related human coronavirus.2 We evaluated the stability of HCoV-19 and SARS-CoV-1 in aerosols and on different surfaces and estimated their decay rates using a Bayesian regression model</jats:p></p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-related Virus Predating SARS Outbreak, Hong Kong</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Using immunofluorescence and neutralization assays, we detected antibodies to human severe acute respiratory syndrome–associated coronavirus (SARS-CoV) and/or animal SARS-CoV–like virus in 17 (1.8%) of 938 adults recruited in 2001. This finding suggests that a small proportion of healthy persons in Hong Kong had been exposed to SARS-related viruses at least 2 years before the recent SARS outbreak.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1-3. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).  Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of traveller screening to prevent international spread of 2019 novel coronavirus (2019-nCoV)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-30</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Traveller screening is being used to limit further global spread of 2019 novel coronavirus (nCoV) following its recent emergence. Here, we analyze the expected impact of different travel screening programs given remaining uncertainty around the values of key nCoV life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss around half of infected travellers. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. These findings emphasize the need for measures to track travellers who become ill after being missed by a travel screening program. We make our model available for interactive use so stakeholders can explore scenarios of interest using the most up-to-date information. We hope these findings contribute to evidence-based policy to combat the spread of nCoV, and to prospective planning to mitigate future emerging pathogens.</jats:p></p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Case of the Index Patient Who Caused Tertiary Transmission of Coronavirus Disease 2019 in Korea: the Application of Lopinavir/Ritonavir for the Treatment of COVID-19 Pneumonia Monitored by Quantitative RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since mid-December of 2019, coronavirus disease 2019 (COVID-19) has been spreading from Wuhan, China. The confirmed COVID-19 patients in South Korea are those who came from or visited China. As secondary transmissions have occurred and the speed of transmission is accelerating, there are rising concerns about community infections. The 54-year old male is the third patient diagnosed with COVID-19 in Korea. He is a worker for a clothing business and had mild respiratory symptoms and intermittent fever in the beginning of hospitalization, and pneumonia symptoms on chest computerized tomography scan on day 6 of admission. This patient caused one case of secondary transmission and three cases of tertiary transmission. Hereby, we report the clinical findings of the index patient who was the first to cause tertiary transmission outside China. Interestingly, after lopinavir/ritonavir (Kaletra, AbbVie) was administered, beta-coronavirus viral loads significantly decreased and no or little coronavirus titers were observed.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL-63 Infections in Children: a 1-Year Study</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus NL63 (HCoV-NL63), a newly discovered coronavirus, has been associated with acute respiratory tract infections (ARI). Important questions pertaining to the contribution of HCoV-NL63 to ARI and its impact on public health remain. We reviewed HCoV-NL63 in specimens collected from 13 November 2002 to 31 December 2003 from the Stollery Children's Hospital on patients of <17 years of age to assess the role of this virus in ARI in children. Twenty-six of 1,240 specimens (2.1%) from seven outpatients and 19 inpatients aged 7 days to 9.5 years tested positive for HCoV-NL63 by reverse transcription-PCR. The majority of outpatients (86%) had upper respiratory tract infections, while the majority of inpatients (58%) had bronchiolitis. Peak HCoV-NL63 activity occurred in March. These results provide further evidence of the importance of HCoV-NL63 in ARI in children.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Air, Surface Environmental, and Personal Protective Equipment Contamination by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) From a Symptomatic Patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses have been implicated in nosocomial outbreaks with environmental contamination as a route of transmission. Similarly, nosocomial transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been reported. However, the mode of transmission and extent of environmental contamination are unknown.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating the infected population and spread trend of 2019-nCov under different policy by EIR model</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported official data, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared.</jats:p></p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and genetic diversity analysis of human coronaviruses among cross-border children</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: More than a decade after the outbreak of human coronaviruses (HCoVs) SARS in Guangdong province and Hong Kong SAR of China in 2002, there is still no reoccurrence, but the evolution and recombination of the coronaviruses in this region are still unknown. Therefore, surveillance on the prevalence and the virus variation of HCoVs circulation in this region is conducted. METHODS: A total of 3298 nasopharyngeal swabs samples were collected from cross-border children (<6 years, crossing border between Southern China and Hong Kong SAR) showing symptoms of respiratory tract infection, such as fever (body temperature > 37.5 °C), from 2014 May to 2015 Dec. Viral nucleic acids were analyzed and sequenced to study the prevalence and genetic diversity of the four human coronaviruses. The statistical significance of the data was evaluated with Fisher chi-square test. RESULTS: 78 (2.37%; 95%CI 1.8-2.8%) out of 3298 nasopharyngeal swabs specimens were found to be positive for OC43 (36;1.09%), HKU1 (34; 1.03%), NL63 (6; 0.18%) and 229E (2;0.01%). None of SARS or MERS was detected. The HCoVs predominant circulating season was in transition of winter to spring, especially January and February and NL63 detected only in summer and fall. Complex population with an abundant genetic diversity of coronaviruses was circulating and they shared homology with the published strains (99-100%). Besides, phylogenetic evolutionary analysis indicated that OC43 coronaviruses were clustered into three clades (B,D,E), HKU1 clustered into two clades(A,B) and NL63 clustered into two clades(A,B). Moreover, several novel mutations including nucleotides substitution and the insertion of spike of the glycoprotein on the viral surface were discovered. CONCLUSIONS: The detection rate and epidemic trend of coronaviruses were stable and no obvious fluctuations were found. The detected coronaviruses shared a conserved gene sequences in S and RdRp. However, mutants of the epidemic strains were detected, suggesting continuous monitoring of the human coronaviruses is in need among cross-border children, who are more likely to get infected and transmit the viruses across the border easily, in addition to the general public.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Tracing the SARS-coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Four coronaviruses (HCoV-229E, HCoV-OC43, HCoV-NL63, HCoV-HKU1) are endemic in humans and mainly associated with mild respiratory illnesses; whereas the other two coronaviruses [Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV)] present as emerging infections causing severe respiratory syndrome. Coronaviruses evolve by accumulation of point mutations and recombination of genomes among different strains or species. Mammalian coronaviruses including those infect humans are evolved from bat coronaviruses. While SARS-CoV and MERS-CoV are genetically closely related to bat coronaviruses, intermediate host(s) is (are) likely to be involved in the emergence and cross-species transmission of these novel human viruses. High prevalence of SARS-like coronaviruses have been found from masked palm civet cats and raccoon dogs collected from markets around the time of outbreaks in humans, but these animals are likely to be a transient accidental host rather than a persisting reservoir. More research is needed to elucidate the ecology of coronaviruses. Vigilance and surveillance should be maintained to promptly identify newly emerged coronaviruses.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of coronavirus in tears and conjunctival secretions of patients with SARS-CoV-2 infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE: This study aimed to assess the presence of novel coronavirus in tears and conjunctival secretions of SARS-CoV-2 infected patients. METHODS: A prospective interventional case series study was performed, and 30 confirmed novel coronavirus pneumonia (NCP) patients were selected at the First Affiliated Hospital of Zhejiang University from January 26, 2020 to February 9, 2020. At an interval of 2-3 days, tear and conjunctival secretions were collected twice with disposable sampling swabs for reverse transcription polymerase chain reaction (RT-PCR) assay. RESULTS: 21 common type and 9 severe type NCP patients were enrolled. Two samples of tear and conjunctival secretions were obtained from the only one patient with conjunctivitis yielded positive RT-PCR results. 58 samples from other patents were all negative. CONCLUSION: We speculate that SARS-CoV-2 may be detected in the tears and conjunctival secretions in NCP patients with conjunctivitis. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Sequential Emergence and Wide Spread of Neutralization Escape Middle East Respiratory Syndrome Coronavirus Mutants, South Korea, 2015</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">The unexpectedly large outbreak of Middle East respiratory syndrome in South Korea in 2015 was initiated by an infected traveler and amplified by several “superspreading” events. Previously, we reported the emergence and spread of mutant Middle East respiratory syndrome coronavirus bearing spike mutations (I529T or D510G) with reduced affinity to human receptor CD26 during the outbreak. To assess the potential association of spike mutations with superspreading events, we collected virus genetic information reported during the outbreak and systemically analyzed the relationship of spike sequences and epidemiology. We found sequential emergence of the spike mutations in 2 superspreaders. In vivo virulence of the mutant viruses seems to decline in human patients, as assessed by fever duration in affected persons. In addition, neutralizing activity against these 2 mutant viruses in serum samples from mice immunized with wild-type spike antigen were gradually reduced, suggesting emergence and wide spread of neutralization escapers during the outbreak.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Sequential Emergence and Wide Spread of Neutralization Escape Middle East Respiratory Syndrome Coronavirus Mutants, South Korea, 2015</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">The unexpectedly large outbreak of Middle East respiratory syndrome in South Korea in 2015 was initiated by an infected traveler and amplified by several “superspreading” events. Previously, we reported the emergence and spread of mutant Middle East respiratory syndrome coronavirus bearing spike mutations (I529T or D510G) with reduced affinity to human receptor CD26 during the outbreak. To assess the potential association of spike mutations with superspreading events, we collected virus genetic information reported during the outbreak and systemically analyzed the relationship of spike sequences and epidemiology. We found sequential emergence of the spike mutations in 2 superspreaders. In vivo virulence of the mutant viruses seems to decline in human patients, as assessed by fever duration in affected persons. In addition, neutralizing activity against these 2 mutant viruses in serum samples from mice immunized with wild-type spike antigen were gradually reduced, suggesting emergence and wide spread of neutralization escapers during the outbreak.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Respiratory Coronavirus OC43: Genetic Stability and Neuroinvasion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The complete genome sequences of the human coronavirus OC43 (HCoV-OC43) laboratory strain from the American Type Culture Collection (ATCC), and a HCoV-OC43 clinical isolate, designated Paris, were obtained. Both genomes are 30,713 nucleotides long, excluding the poly(A) tail, and only differ by 6 nucleotides. These six mutations are scattered throughout the genome and give rise to only two amino acid substitutions: one in the spike protein gene (I958F) and the other in the nucleocapsid protein gene (V81A). Furthermore, the two variants were shown to reach the central nervous system (CNS) after intranasal inoculation in BALB/c mice, demonstrating neuroinvasive properties. Even though the ATCC strain could penetrate the CNS more effectively than the Paris 2001 isolate, these results suggest that intrinsic neuroinvasive properties already existed for the HCoV-OC43 ATCC human respiratory isolate from the 1960s before it was propagated in newborn mouse brains. It also demonstrates that the molecular structure of HCoV-OC43 is very stable in the environment (the two variants were isolated ca. 40 years apart) despite virus shedding and chances of persistence in the host. The genomes of the two HCoV-OC43 variants display 71, 53.1, and 51.2% identity with those of mouse hepatitis virus A59, severe acute respiratory syndrome human coronavirus Tor2 strain (SARS-HCoV Tor2), and human coronavirus 229E (HCoV-229E), respectively. HCoV-OC43 also possesses well-conserved motifs with regard to the genome sequence of the SARS-HCoV Tor2, especially in open reading frame 1b. These results suggest that HCoV-OC43 and SARS-HCoV may share several important functional properties and that HCoV-OC43 may be used as a model to study the biology of SARS-HCoV without the need for level three biological facilities.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. METHODS: Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. RESULTS: The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. CONCLUSION: SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects of Air Temperature and Relative Humidity on Coronavirus Survival on Surfaces</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Assessment of the risks posed by severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) on surfaces requires data on survival of this virus on environmental surfaces and on how survival is affected by environmental variables, such as air temperature (AT) and relative humidity (RH). The use of surrogate viruses has the potential to overcome the challenges of working with SARS-CoV and to increase the available data on coronavirus survival on surfaces. Two potential surrogates were evaluated in this study; transmissible gastroenteritis virus (TGEV) and mouse hepatitis virus (MHV) were used to determine effects of AT and RH on the survival of coronaviruses on stainless steel. At 4°C, infectious virus persisted for as long as 28 days, and the lowest level of inactivation occurred at 20% RH. Inactivation was more rapid at 20°C than at 4°C at all humidity levels; the viruses persisted for 5 to 28 days, and the slowest inactivation occurred at low RH. Both viruses were inactivated more rapidly at 40°C than at 20°C. The relationship between inactivation and RH was not monotonic, and there was greater survival or a greater protective effect at low RH (20%) and high RH (80%) than at moderate RH (50%). There was also evidence of an interaction between AT and RH. The results show that when high numbers of viruses are deposited, TGEV and MHV may survive for days on surfaces at ATs and RHs typical of indoor environments. TGEV and MHV could serve as conservative surrogates for modeling exposure, the risk of transmission, and control measures for pathogenic enveloped viruses, such as SARS-CoV and influenza virus, on health care surfaces.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Singapore's experience of SARS</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus that causes severe acute respiratory syndrome (SARS) is transmitted mainly via respiratory droplets. Typical presenting symptoms are akin to those of ordinary pneumonia. Young patients start with fever, chills, malaise, headache, or myalgia; cough and dyspnoea follow. Older persons and those taking corticosteroids may have neither fever nor respiratory symptoms. Exceptional suspicion is needed to identify SARS early in the illness. During an outbreak, even patients with low suspicion of SARS should be promptly isolated, and all contacts quarantined. Health workers need training in the use of appropriate barriers against droplets and other body fluids. Any fever cluster in patients or carers requires immediate action: discharges, visits, and transfers between wards and hospitals should be stopped. Halting hospital admissions and ten-day quarantine of suspected cases create wide buffer zones. To counter a possible resurgence of SARS, a system of prepared isolation and quarantine facilities is important.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS transmission in Vietnam outside of the health-care setting</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">To evaluate the risk of transmission of SARS coronavirus outside of the health-care setting, close household and community contacts of laboratory-confirmed SARS cases were identified and followed up for clinical and laboratory evidence of SARS infection. Individual- and household-level risk factors for transmission were investigated. Nine persons with serological evidence of SARS infection were identified amongst 212 close contacts of 45 laboratory- confirmed SARS cases (secondary attack rate 4·2%, 95% CI 1·5–7). In this cohort, the average number of secondary infections caused by a single infectious case was 0·2. Two community contacts with laboratory evidence of SARS coronavirus infection had mild or sub-clinical infection, representing 3% (2/65) of Vietnamese SARS cases. There was no evidence of transmission of infection before symptom onset. Physically caring for a symptomatic laboratory-confirmed SARS case was the only independent risk factor for SARS transmission (OR 5·78, 95% CI 1·23–24·24).</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Lack of SARS Transmission among Public Hospital Workers, Vietnam</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome (SARS) outbreak in Vietnam was amplified by nosocomial spread within hospital A, but no transmission was reported in hospital B, the second of two designated SARS hospitals. Our study documents lack of SARS-associated coronavirus transmission to hospital B workers, despite variable infection control measures and the use of personal protective equipment.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Case of the Index Patient Who Caused Tertiary Transmission of Coronavirus Disease 2019 in Korea: the Application of Lopinavir/Ritonavir for the Treatment of COVID-19 Pneumonia Monitored by Quantitative RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 14</span></i><p style="text-align: justify; text-justify: inter-word;">Since mid-December of 2019, coronavirus disease 2019 (COVID-19) has been spreading from Wuhan, China. The confirmed COVID-19 patients in South Korea are those who came from or visited China. As secondary transmissions have occurred and the speed of transmission is accelerating, there are rising concerns about community infections. The 54-year old male is the third patient diagnosed with COVID-19 in Korea. He is a worker for a clothing business and had mild respiratory symptoms and intermittent fever in the beginning of hospitalization, and pneumonia symptoms on chest computerized tomography scan on day 6 of admission. This patient caused one case of secondary transmission and three cases of tertiary transmission. Hereby, we report the clinical findings of the index patient who was the first to cause tertiary transmission outside China. Interestingly, after lopinavir/ritonavir (Kaletra, AbbVie) was administered, β-coronavirus viral loads significantly decreased and no or little coronavirus titers were observed.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Extremely Low Exposure of a Community to Severe Acute Respiratory Syndrome Coronavirus: False Seropositivity due to Use of Bacterially Derived Antigens</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Estimates of seropositivity to a new infectious agent in a community are useful to public health. For severe acute respiratory syndrome (SARS), the figures are conflicting. Herein, we screened 12,000 people in a community stricken by SARS 10 months previously and found 53 individuals (0.44%) who had immunoglobulin G antibodies to the SARS coronavirus (SARS-CoV) nucleocapsid (N) produced in bacteria. However, only seven of these (group 1) had sera which also reacted with the native N antigen expressed in SARS-CoV-infected Vero cells, N-transfected 293T cells, and tissues of infected SARS patients. Of these, six individuals had had SARS previously. The remaining person, as well as the 46 other individuals (group 2), were healthy and had no history of SARS. Group 1 antibodies recognized epitopes located slightly differently in N from those of group 2 antibodies, and a mouse hybridoma antibody resembling the former type was generated. Unusually, group 2 antibodies appeared to recognize cross-reactive bacterial epitopes that presumably were posttranslationally modified in eukaryotes and hence were probably not induced by SARS-CoV or related coronaviruses but rather by bacteria. The N antigen is thus highly unique. The extremely low rate (0.008%) of asymptomatic SARS infection found attests to the high virulence of the SARS-CoV virus.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultrastructural Characterization of SARS Coronavirus</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) was first described during a 2002–2003 global outbreak of severe pneumonia associated with human deaths and person-to-person disease transmission. The etiologic agent was initially identified as a coronavirus by thin-section electron microscopic examination of a virus isolate. Virions were spherical, 78 nm in mean diameter, and composed of a helical nucleocapsid within an envelope with surface projections. Herein, we show that infection with the SARS-associated coronavirus resulted in distinct ultrastructural features: double-membrane vesicles, nucleocapsid inclusions, and large granular areas of cytoplasm. These three structures and the coronavirus particles were shown to be positive for viral proteins and RNA by using ultrastructural immunogold and in situ hybridization assays. In addition, ultrastructural examination of a bronchiolar lavage specimen from a SARS patient showed numerous coronavirus-infected cells with features similar to those in infected culture cells. Electron microscopic studies were critical in identifying the etiologic agent of the SARS outbreak and in guiding subsequent laboratory and epidemiologic investigations.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Deep Sequencing and Phylogenetic Analysis of Novel Human Betacoronavirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 May</span></i><p style="text-align: justify; text-justify: inter-word;">A novel betacoronavirus associated with lethal respiratory and renal complications was recently identified in patients from several countries in the Middle East. We report the deep genome sequencing of the virus directly from a patient’s sputum sample. Our high-throughput sequencing yielded a substantial depth of genome sequence assembly and showed the minority viral variants in the specimen. Detailed phylogenetic analysis of the virus genome (England/Qatar/2012) revealed its close relationship to European bat coronaviruses circulating among the bat species of the Vespertilionidae family. Molecular clock analysis showed that the 2 human infections of this betacoronavirus in June 2012 (EMC/2012) and September 2012 (England/Qatar/2012) share a common virus ancestor most likely considerably before early 2012, suggesting the human diversity is the result of multiple zoonotic events.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Deep Sequencing and Phylogenetic Analysis of Novel Human Betacoronavirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 May</span></i><p style="text-align: justify; text-justify: inter-word;">A novel betacoronavirus associated with lethal respiratory and renal complications was recently identified in patients from several countries in the Middle East. We report the deep genome sequencing of the virus directly from a patient’s sputum sample. Our high-throughput sequencing yielded a substantial depth of genome sequence assembly and showed the minority viral variants in the specimen. Detailed phylogenetic analysis of the virus genome (England/Qatar/2012) revealed its close relationship to European bat coronaviruses circulating among the bat species of the Vespertilionidae family. Molecular clock analysis showed that the 2 human infections of this betacoronavirus in June 2012 (EMC/2012) and September 2012 (England/Qatar/2012) share a common virus ancestor most likely considerably before early 2012, suggesting the human diversity is the result of multiple zoonotic events.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Seroepidemiological Study on SARS-CoV IgG Antibodies of Different Populations from Several Areas</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE: In order to investigate the clinical and epidemiological rules of severe acute respiratory syndrome (SARS), rates and levels SARS coronavirus (SARS-CoV) IgG antibodies of the patients and community populations from several areas were detected. METHODS: Indirect immunofluorescent assay (IFA) and double-antigen sandwich enzyme-linked immunosorbent assay (ELISA) were used to detect the SARS coronavirus-specific IgG antibodies in sera of 1700, including 1453 general populations from Hongkong, Marco, Guangzhou and Peking and 257 SARS patients from Guangzhou and Peking. The dynamics of the serum antibodies of SARS patients were observed from 3 to 360 days after onset of symptoms. RESULTS: 90% of 257 patient serum specimens after 20 days of disease onset showed positive SARS-CoV IgG either using ELISA or IFA. 257 SARS patients, antibodies titers increased steadily in early 4 to 6 months after onset of SARS. The titers of most cases came to the peak in the 6th month. then antibodies titers declined rapidly in some cases. However, all specimens still were positive for SARS-CoV IgG in the 48th month. CONCLUSIONS: This study suggest that few inapparent infectious patients exist during SARS epidemic. Serum IgG antibodies has diagnostic value for SARS in the late course of disease and the antibodies present more than 48 months.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequences of Five Human Coronavirus NL63 Strains Causing Respiratory Illness in Hospitalized Children in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">We report the complete genome sequences of five human coronavirus NL63 (HCoV-NL63) strains obtained using next-generation sequencing. The five HCoV-NL63 strains were obtained from hospitalized children with severe acute respiratory infection detected in Guangdong, China. This study provides several complete genomes of HCoV-NL63 and improves our understanding of HCoV-NL63 evolution in China.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-22</span></i><p style="text-align: justify; text-justify: inter-word;">Detailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Seroepidemiology of group I human coronaviruses in children</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Recently, several new human coronaviruses have been identified. OBJECTIVES: To define the seroepidemiology of group I human coronaviruses STUDY DESIGN: A recombinant protein enzyme linked immunosorbent assay (ELISA) based on portions of the nucleocapsid protein of group I human coronaviruses was developed and was used to screen serum from 243 children and young adults. RESULTS: For HCoV-229E, the percentages of seropositive individuals were 57.1% for infants < 2 months old; 38.9% for infants 2–3 months old; 4.7% for infants 4–5 months old; 42.9–50.0% for infants 6–12 months old; and 34.8–62.5% for individuals 1–20 years old. For HCoV-NL63, the percentages of seropositive individuals were 45.2% for infants < 2 months old; 11.1% for infants 2–3 months old; 4.7% for infants 4–5 months old; 28.6–40.0% for infants 6–12 months old; and 25.0–70.3% for individuals 1–20 years old. CONCLUSIONS: Infection with these viruses is common in childhood though the prevalence of these viruses may vary from year to year.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>A mathematical model for estimating the age-specific transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: A novel coronavirus named as "SARS-CoV-2" has spread widely in many countries since December 2019, especially in China. This study aimed to quantify the age-specific transmissibility by using a mathematical model. Methods: An age-specific susceptible - exposed - symptomatic - asymptomatic - recovered - seafood market (SEIARW) model was developed based on two suspected transmission routes (from market to person and person to person). The susceptible people from Wuhan City were divided into different age groups. We used the subscript i and j to represent age group 1 to 4 (1: &lt;= 14 years; 2: 15-44 years; 3: 45-64 years; 4: &gt;= 65 years) and 1 to 5 (1: &lt;= 5 years; 2: 6-14 years; 3: 15-24 years; 4: 25-59 years; 4: &gt;= 60 years), respectively. Data of reported COVID-19 cases were collected from one published literature from 26 November to 22 December, 2019 in Wuhan City, China. The age-specific transmissibility of the virus was estimated accordingly secondary attack rate (SAR).  Results: The age-specific SEIARW model fitted with the reported data well by dividing the population into four age groups (χ2 = 4.99 × 10-6, P &gt; 0.999), and five age groups (χ2 = 4.85 × 10-6, P &gt; 0.999). Based on the four-age-group SEIARW model, the highest transmissibility occurred from age group 2 to 3 (SAR23 = 17.56 per 10 million persons), followed by from age group 3 to 2 (SAR32 = 10.17 per 10 million persons). The lowest transmissibility occurred from age group 1 to 2 (SAR12 = 0.002 per 10 million persons). Based on the five-age-group SEIARW model, the highest transmissibility occurred from age group 4 to 5 (SAR45 = 12.40 per 10 million persons), followed by from age group 5 to 4 (SAR54 = 6.61 per 10 million persons). The lowest transmissibility occurred from age group 3 to 4 (SAR34 = 0.0002 per 10 million persons). Conclusions: SARS-CoV-2 has high transmissibility among adults and elder people but low transmissibility among children and young people.</jats:p></p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Topographic Changes in SARS Coronavirus–infected Cells at Late Stages of Infection</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Scanning electron and atomic force microscopy was used for the first time to view the maturation of the severe acute respiratory syndrome–associated coronavirus at the cell surface. The surface form of the cells at advanced infection displayed prolific pseudopodia that, in addition to the rest of the plasma membrane, were also active sites of virus release. High magnification of the maturing virus particles showed a rosette appearance with short knoblike spikes under both the scanning electron and atomic force microscopes. The final expulsion step of the maturing virus particles seemed to result in some disruptions to the plasma membrane. The cytoskeletal network along the edge of the infected cells was enhanced and could be involved in transporting and expelling the progeny virus particles. Thickening of the actin filaments at the cell edge provided the bending force to extrude the virus particles.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Morphogenesis of Coronavirus HCoV-NL63 in Cell Culture: A Transmission Electron Microscopic Study</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">NL63 (HCoV-NL63) is a recently discovered human coronavirus that causes respiratory disease in infants and young children. NL63 productively infects LLCMK2 cells and ciliated epithelial cells of human airway cell cultures. Transmission electron microscopic (TEM) studies of NL63 infected LLCMK2 cells revealed that virions are spherical, spiked, and range from 75 to 115 nm in diameter. Virus replication predominantly occurs on the rough endoplasmic reticulum (RER), both perinuclear and cytoplasmic, and the Golgi. Plasma membrane budding was occasionally observed. As virus production increased, aberrant viral forms appeared with greater frequency. Unusual inclusions were present in infected cells including tubular and laminated structures. Pleomorphic double membrane-bound vesicles (DMV), measuring roughly 140 to 210 nm in diameter, were observed. The virus was released via exocytosis and cell lysis. In summary, we report the key morphologic characteristics of NL63 infection observed by TEM analysis.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Human coronavirus OC43 causes influenza-like illness in residents and staff of aged-care facilities in Melbourne, Australia.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Three outbreaks of respiratory illness associated with human coronavirus HCoV-OC43 infection occurred in geographically unrelated aged-care facilities in Melbourne, Australia during August and September 2002. On clinical and epidemiological grounds the outbreaks were first thought to be caused by influenza virus. HCoV-OC43 was detected by RT-PCR in 16 out of 27 (59%) specimens and was the only virus detected at the time of sampling. Common clinical manifestations were cough (74%), rhinorrhoea (59%) and sore throat (53%). Attack rates and symptoms were similar in residents and staff across the facilities. HCoV-OC43 was also detected in surveillance and diagnostic respiratory samples in the same months. These outbreaks establish this virus as a cause of morbidity in aged-care facilities and add to increasing evidence of the significance of coronavirus infections.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 and 229E Seroconversion in Children</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In 2004, the novel respiratory human coronavirus NL63 (HCoV-NL63) was identified, and subsequent research revealed that the virus has spread worldwide. HCoV-229E is a close relative of HCoV-NL63, and infection with either virus can lead to the hospitalization of young children, immunocompromised persons, and the elderly. Children infected with HCoV-NL63 often develop croup, with obstruction of the airway. In this study we investigated at which age children are confronted for the first time with an HCoV-NL63 infection and, thus, at which age they seroconvert to HCoV-NL63 positivity. We designed a recombinant HCoV-229E and a recombinant HCoV-NL63 nucleocapsid protein enzyme-linked immunosorbent assay and performed a seroepidemiology survey on longitudinal and cross-sectional serum samples. The longitudinal serum samples were collected from 13 newborns, and data for those newborns were available from multiple time points spanning a period of at least 18 months. For the cross-sectional survey we tested serum samples of 139 children, including newborns to children 16 years of age. In examinations of the longitudinal serum samples we observed that all of the children had maternal anti-NL63 and anti-229E antibodies at birth that disappeared within 3 months. Seven of the 13 children became HCoV-NL63 seropositive during follow-up, whereas only 2 became HCoV-229E seropositive. The serology data of the cross-sectional serum samples revealed that 75% and 65% of the children in the age group 2.5 to 3.5 years were HCoV-NL63 and HCoV-229E seropositive, respectively. We conclude that on average, HCoV-NL63 and HCoV-229E seroconversion occurs before children reach the age of 3.5 years.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</jats:p></p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe Acute Respiratory Syndrome–associated Coronavirus Infection</h3>
<i class="tooltip">2003<span class="tooltiptext">2003 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Whether severe acute respiratory syndrome–associated coronavirus (SARS-CoV) infection can be asymptomatic is unclear. We examined the seroprevalence of SARS-CoV among 674 healthcare workers from a hospital in which a SARS outbreak had occurred. A total of 353 (52%) experienced mild self-limiting illnesses, and 321 (48%) were asymptomatic throughout the course of these observations. None of these healthcare workers had antibody to SARS CoV, indicating that subclinical or mild infection attributable to SARS CoV in adults is rare.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: death toll passes 2,000</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Updates on the respiratory illness that has infected tens of thousands of people. Scientists are concerned about a new virus that has infected tens of thousands of people and killed more than 2,000. The virus, which emerged in the Chinese city of Wuhan in December, is a coronavirus and belongs to the same family as the pathogen that causes severe acute respiratory syndrome, or SARS. It causes a respiratory illness called COVID-19, which can spread from person to person.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Serial interval of novel coronavirus (COVID-19) infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n = 28) and a subset of pairs with highest certainty in reporting (n = 18). In addition, we adjust for right truncation of the data as the epidemic is still in its growth phase. Results Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions The serial interval of COVID-19 is close to or shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Case of the Index Patient Who Caused Tertiary Transmission of COVID-19 Infection in Korea: the Application of Lopinavir/Ritonavir for the Treatment of COVID-19 Infected Pneumonia Monitored by Quantitative RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since mid-December of 2019, coronavirus disease 2019 (COVID-19) infection has been spreading from Wuhan, China. The confirmed COVID-19 patients in South Korea are those who came from or visited China. As secondary transmissions have occurred and the speed of transmission is accelerating, there are rising concerns about community infections. The 54-year old male is the third patient diagnosed with COVID-19 infection in Korea. He is a worker for a clothing business and had mild respiratory symptoms and intermittent fever in the beginning of hospitalization, and pneumonia symptoms on chest computerized tomography scan on day 6 of admission. This patient caused one case of secondary transmission and three cases of tertiary transmission. Hereby, we report the clinical findings of the index patient who was the first to cause tertiary transmission outside China. Interestingly, after lopinavir/ritonavir (Kaletra, AbbVie) was administered, ß-coronavirus viral loads significantly decreased and no or little coronavirus titers were observed.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>A mathematical model for simulating the phase-based transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus. Methods In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from Huanan Seafood Wholesale Market (reservoir) to people, we simplified the model as Reservoir-People (RP) transmission network model. The next generation matrix approach was adopted to calculate the basic reproduction number (R 0) from the RP model to assess the transmissibility of the SARS-CoV-2. Results The value of R 0 was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.58. Conclusions Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS coronavirus to novel animal and human coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In 2003, severe acute respiratory syndrome coronavirus (SARS-CoV) caused one of the most devastating epidemics known to the developed world. There were two important lessons from this epidemic. Firstly, coronaviruses, in addition to influenza viruses, can cause severe and rapidly spreading human infections. Secondly, bats can serve as the origin and natural animal reservoir of deadly human viruses. Since then, researchers around the world, especially those in Asia where SARS-CoV was first identified, have turned their focus to find novel coronaviruses infecting humans, bats, and other animals. Two human coronaviruses, HCoV-HKU1 and HCoV-NL63, were identified shortly after the SARS-CoV epidemic as common causes of human respiratory tract infections. In 2012, a novel human coronavirus, now called Middle East respiratory syndrome coronavirus (MERS-CoV), has emerged in the Middle East to cause fatal human infections in three continents. MERS-CoV human infection is similar to SARS-CoV in having a high fatality rate and the ability to spread from person to person which resulted in secondary cases among close contacts including healthcare workers without travel history to the Middle East. Both viruses also have close relationships with bat coronaviruses. New cases of MERS-CoV infection in humans continue to occur with the origins of the virus still unknown in many cases. A multifaceted approach is necessary to control this evolving MERS-CoV outbreak. Source identification requires detailed epidemiological studies of the infected patients and enhanced surveillance of MERS-CoV or similar coronaviruses in humans and animals. Early diagnosis of infected patients and appropriate infection control measures will limit the spread in hospitals, while social distancing strategies may be necessary to control the outbreak in communities if it remained uncontrolled as in the SARS epidemic.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Three Emerging Coronaviruses in Two Decades</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In the past two decades, the world has seen three coronaviruses emerge and cause outbreaks that have caused considerable global health consternation. Coronaviruses are enveloped, nonsegmented, single-stranded, positive-sense RNA viruses that have a characteristic appearance on electron microscopy negative staining Image 1. As a matter of fact, the characteristic electron microscopy appearance was the clue to amplify and sequence nucleic acids from Dr Urbani’s (one of the health care providers who died of severe acute respiratory syndrome [SARS] in 2003) respiratory specimen using a consensus coronavirus primer.1 The sequence of the virus was significantly different from other coronaviruses known to cause human disease at the time. The virus was ultimately named SARS-CoV, as febrile patients had severe acute respiratory syndrome and could present with pneumonia and lower respiratory symptoms such as cough and dyspnea.2 The SARS-CoV outbreak started in Guangdong, China, and spread to many countries in Southeast Asia, North America, Europe, and South Africa. Transmission was primarily person to person through droplets that occurred during coughing or sneezing, through personal contact (shaking hands), or by touching contaminated surfaces. Of note, health professionals were particularly at risk of acquiring the disease, as transmission also occurred if isolation precautions were not followed and during certain procedures. The last case of SARS-CoV occurred in September 2003, after having infected over 8,000 persons and causing 774 deaths with a case fatality rate calculated at 9.5%.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of Contact Tracing for the Containment of the 2019 Novel Coronavirus (COVID-19).</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Contact tracing is a central public health response to infectious disease outbreaks, especially in the early stages of an outbreak when specific treatments are limited. Importation of novel Coronavirus (COVID-19) from China and elsewhere into the United Kingdom highlights the need to understand the impact of contact tracing as a control measure. Using detailed survey information on social encounters coupled to predictive models, we investigate the likely efficacy of the current UK definition of a close contact (within 2 meters for 15 minutes or more) and the distribution of secondary cases that may go untraced. Taking recent estimates for COVID-19 transmission, we show that less than 1 in 5 cases will generate any subsequent untraced cases, although this comes at a high logistical burden with an average of 36.1 individuals (95th percentiles 0-182) traced per case. Changes to the definition of a close contact can reduce this burden, but with increased risk of untraced cases; we estimate that any definition where close contact requires more than 4 hours of contact is likely to lead to uncontrolled spread.</jats:p></p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2.  Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</jats:p></p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential role of inanimate surfaces for the spread of coronaviruses and their inactivation with disinfectant agents</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary The novel human coronavirus 2019-nCoV has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described, probably via droplets but possibly also via contaminated hands or surfaces. In a recent review on the persistence of human and veterinary coronaviruses on inanimate surfaces it was shown that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days. Some disinfectant agents effectively reduce coronavirus infectivity within 1 minute such 62% - 71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite. Other compounds such as 0.05% - 0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. An effective surface disinfection may help to ensure an early containment and prevention of further viral spread.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /></body>