
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: new platforms and technology to improve response times COVID-19 and future diseases</h2><br /></div><br /><br /><br /><div class="card"><h3>How to fight an infodemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">WHO's newly launched platform aims to combat misinformation around COVID-19. John Zarocostas reports from WHO is leading the effort to slow the spread of the 2019 coronavirus disease (COVID-19) outbreak. But a global epidemic of misinformation—spreading rapidly through social media platforms and other outlets—poses a serious problem for public health. “We’re not just fighting an epidemic; we’re fighting an infodemic”, said WHO Director-General Tedros Adhanom Ghebreyesus at the Munich Security Conference on Feb 15. Immediately after COVID-19 was declared a Public Health Emergency of International Concern, WHO's risk communication team launched a new information platform called WHO Information Network for Epidemics (EPI-WIN), with the aim of using a series of amplifiers to share tailored information with specific target groups. Sylvie Briand, director of Infectious Hazards Management at WHO's Health Emergencies Programme and architect of WHO's strategy to counter the infodemic risk, told The Lancet, “We know that every outbreak will be accompanied by a kind of tsunami of information, but also within this information you always have misinformation, rumours, etc. We know that even in the Middle Ages there was this phenomenon”. “But the difference now with social media is that this phenomenon is amplified, it goes faster and further, like the viruses that travel with people and go faster and further. So it is a new challenge, and the challenge is the [timing] because you need to be faster if you want to fill the void…What is at stake during an outbreak is making sure people will do the right thing to control the disease or to mitigate its impact. So it is not only information to make sure people are informed; it is also making sure people are informed to act appropriately.” About 20 staff and some consultants are involved in WHO's communications teams globally, at any given time. This includes social media personnel at each of WHO's six regional offices, risk communications consultants, and WHO communications officers.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Emerging infectious diseases: A proactive approach</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases are now emerging or reemerging almost every year. This trend will continue because a number of factors, including the increased global population, aging, travel, urbanization, and climate change, favor the emergence, evolution, and spread of new pathogens. The approach used so far for emerging infectious diseases (EIDs) does not work from the technical point of view, and it is not sustainable. However, the advent of platform technologies offers vaccine manufacturers an opportunity to develop new vaccines faster and to reduce the investment to build manufacturing facilities, in addition to allowing for the possible streamlining of regulatory processes. The new technologies also make possible the rapid development of human monoclonal antibodies that could become a potent immediate response to an emergency. So far, several proposals to approach EIDs have been made independently by scientists, the private sector, national governments, and international organizations such as the World Health Organization (WHO). While each of them has merit, there is a need for a global governance that is capable of taking a strong leadership role and making it attractive to all partners to come to the same table and to coordinate the global approach.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>‘A completely new culture of doing research.’ Coronavirus outbreak changes how scientists communicate | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 22 January, Dave O’Connor and Tom Friedrich invited several dozen colleagues around the United States to join a new workspace on the instant messaging platform Slack. The scientists, both at the Wisconsin National Primate Research Center, had seen news about a new disease emerging in China and realized researchers would need a primate model if they were going to answer some important questions about its biology. “We put out a call to a bunch of investigators and basically said: ‘Hey, let’s talk,’” O’Connor says. The idea is to coordinate research and make sure results are comparable, Friedrich adds. (They named the Slack workspace the Wu-han Clan, a play on the hip-hop group Wu-Tang Clan.) The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn’t even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>The GenMark ePlex(®): another weapon in the syndromic arsenal for infection diagnosis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As one of the most recent additions to the syndromic testing landscape, the ePlex(®) platform by GenMark Diagnostics is a system that combines the manufacturer's signature electrochemical detection technology with updated microfluidics, providing a new option for multiplex testing that is both rapid and requires minimal hands-on steps. In this review, we detail the ePlex platform and its current/future syndromic panels, with a particular focus on the respiratory pathogen panel – the platform's first assay to undergo clinical trials and receive regulatory approval in the USA. By keeping informed of these ever-expanding laboratory options, clinicians and microbiologists can stay positioned at the forefront of infectious disease diagnosis.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-04-28</span></i><p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live-attenuated coronavirus vaccines.\n\nSignificanceEmergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations like the aged and immune-compromised. This study illustrates that despite success in young models, the NSP16 mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the NSP16 mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for NSP16 attenuation as a major component of future live-attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination Attenuation Offers Strategy for Live Attenuated Coronavirus Vaccines</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2′O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the severe acute respiratory syndrome-CoV mouse model, an NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2′O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2′O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live attenuated coronavirus vaccines. IMPORTANCE Emergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations such as the aged and immunocompromised. This study illustrates that despite success in young models, the 2′O methyltransferase mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the 2′O methyltransferase mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for 2′O methyltransferase attenuation as a major component of future live attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>From Isolation to Coordination: How Can Telemedicine Help Combat the COVID-19 Outbreak?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The rapid spread of Coronavirus disease 2019 (COVID-19) presents China with a critical challenge. As normal capacity of the Chinese hospitals is exceeded, healthcare professionals struggling to manage this unprecedented crisis face the difficult question of how best to coordinate the medical resources used in highly separated locations. Responding rapidly to this crisis, the National Telemedicine Center of China (NTCC), located in Zhengzhou, Henan Province, has established the Emergency Telemedicine Consultation System (ETCS), a telemedicine-enabled outbreak alert and response network. ETCS is built upon a doctor-to-doctor (D2D) approach, in which health services can be accessed remotely through terminals across hospitals. The system architecture of ETCS comprises three major architectural layers: (1) telemedicine service platform layer, (2) telemedicine cloud layer, and (3) telemedicine service application layer. Our ETCS has demonstrated substantial benefits in terms of the effectiveness of consultations and remote patient monitoring, multidisciplinary care, and prevention education and training.</jats:p></p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in ‘lockdown’ directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation’s economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and advances how smart city networks should work towards enhancing standardization protocols for increased data sharing in the event of outbreaks or disasters, leading to better global understanding and management of the same.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple Platform for the Rapid Development of Antimicrobials</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 14</span></i><p style="text-align: justify; text-justify: inter-word;">Recent infectious outbreaks highlight the need for platform technologies that can be quickly deployed to develop therapeutics needed to contain the outbreak. We present a simple concept for rapid development of new antimicrobials. The goal was to produce in as little as one week thousands of doses of an intervention for a new pathogen. We tested the feasibility of a system based on antimicrobial synbodies. The system involves creating an array of 100 peptides that have been selected for broad capability to bind and/or kill viruses and bacteria. The peptides are pre-screened for low cell toxicity prior to large scale synthesis. Any pathogen is then assayed on the chip to find peptides that bind or kill it. Peptides are combined in pairs as synbodies and further screened for activity and toxicity. The lead synbody can be quickly produced in large scale, with completion of the entire process in one week.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple Platform for the Rapid Development of Antimicrobials</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 14</span></i><p style="text-align: justify; text-justify: inter-word;">Recent infectious outbreaks highlight the need for platform technologies that can be quickly deployed to develop therapeutics needed to contain the outbreak. We present a simple concept for rapid development of new antimicrobials. The goal was to produce in as little as one week thousands of doses of an intervention for a new pathogen. We tested the feasibility of a system based on antimicrobial synbodies. The system involves creating an array of 100 peptides that have been selected for broad capability to bind and/or kill viruses and bacteria. The peptides are pre-screened for low cell toxicity prior to large scale synthesis. Any pathogen is then assayed on the chip to find peptides that bind or kill it. Peptides are combined in pairs as synbodies and further screened for activity and toxicity. The lead synbody can be quickly produced in large scale, with completion of the entire process in one week.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>The ongoing crises in China illustrate that the assessment of epidemics in isolation is no longer sufficient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary The interplay of simultaneous COVID-19, African swine fever, and avian influenza emergencies on global health and industries is constantly evolving and difficult to predict, and therefore warrants further scrutiny. The need for a health network of global scope for the rapid and open exchange of information needs to be strengthened in order to address ongoing and future epidemics under competing resources.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>New technologies for reporting real-time emergent infections</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Novel technologies have prompted a new paradigm in disease surveillance. Advances in computation, communications and materials enable new technologies such as mobile phones and microfluidic chips. In this paper we illustrate examples of new technologies that can augment disease detection. We describe technologies harnessing the internet, mobile phones, point of care diagnostic tools and methods that facilitate detection from passively collected unstructured data. We demonstrate how these can all assist in quicker detection, investigation and response to emerging infectious events. Novel technologies enable collection and dissemination of epidemic intelligence data to both public health practitioners and the general public, enabling finer temporal and spatial resolution of disease monitoring than through traditional public health processes.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Straining the System: Novel Coronavirus (COVID-19) and Preparedness for Concomitant Disasters</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Just a few weeks before the first confirmed case of novel coronavirus (COVID-19) was reported in the United States, the US Centers for Disease Control and Prevention (CDC) issued a bold promise to the nation: the agency will use its scientific expertise to bring a new level of preparedness in the United States and global health security against current and growing threats, finally eliminate certain diseases, and bring an end to the devastation of epidemics.(1) The current outbreak of COVID-19 reminds us how urgent this promise is and just how critical it is to continue to sustain and strengthen our nation's public health infrastructure. The unprecedented pace of the public health response to COVID-19 has only been possible because of prior investments in public health preparedness. To accelerate our pace and meet the challenges of current and future health threats, we must advance our world-class data and analytics capabilities; maintain and expand our state-of-the-art public health laboratory capacity; continue building a workforce of trusted, expert, public health professionals; sustain our capacity to rapidly respond to outbreaks at their source; and assure a strong global and domestic preparedness capacity. (Am J Public Health. Published online ahead of print February 13, 2020: e1-e2. doi:10.2105/AJPH.2020.305618).</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Preprints bring ‘firehose’ of outbreak data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn't even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Broadly Accessible Venezuelan Equine Encephalitis Virus Replicon Particle Vaccine Platform</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 May 14</span></i><p style="text-align: justify; text-justify: inter-word;">Zoonotic viruses circulate as swarms in animal reservoirs and can emerge into human populations, causing epidemics that adversely affect public health. Portable, safe, and effective vaccine platforms are needed in the context of these outbreak and emergence situations. In this work, we report the generation and characterization of an alphavirus replicon vaccine platform based on a non-select agent, attenuated Venezuelan equine encephalitis (VEE) virus vaccine, strain 3526 (VRP 3526). Using both noroviruses and coronaviruses as model systems, we demonstrate the utility of the VRP 3526 platform in the generation of recombinant proteins, production of virus-like particles, and in vivo efficacy as a vaccine against emergent viruses. Importantly, packaging under biosafety level 2 (BSL2) conditions distinguishes VRP 3526 from previously reported alphavirus platforms and makes this approach accessible to the majority of laboratories around the world. In addition, improved outcomes in the vulnerable aged models as well as against heterologous challenge suggest improved efficacy compared to that of previously attenuated VRP approaches. Taking these results together, the VRP 3526 platform represents a safe and highly portable system that can be rapidly deployed under BSL2 conditions for generation of candidate vaccines against emerging microbial pathogens. IMPORTANCE While VEE virus replicon particles provide a robust, established platform for antigen expression and vaccination, its utility has been limited by the requirement for high-containment-level facilities for production and packaging. In this work, we utilize an attenuated vaccine strain capable of use at lower biocontainment level but retaining the capacity of the wild-type replicon particle. Importantly, the new replicon platform provides equal protection for aged mice and following heterologous challenge, which distinguishes it from other attenuated replicon platforms. Together, the new system represents a highly portable, safe system for use in the context of disease emergence.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Zika virus using reverse-transcription LAMP coupled with reverse dot blot analysis in saliva</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 5</span></i><p style="text-align: justify; text-justify: inter-word;">In recent years, there have been increasing numbers of infectious disease outbreaks that spread rapidly to population centers resulting from global travel, population vulnerabilities, environmental factors, and ecological disasters such as floods and earthquakes. Some examples of the recent outbreaks are the Ebola epidemic in West Africa, Middle East respiratory syndrome coronavirus (MERS-Co) in the Middle East, and the Zika outbreak through the Americas. We have created a generic protocol for detection of pathogen RNA and/or DNA using loop-mediated isothermal amplification (LAMP) and reverse dot-blot for detection (RDB) and processed automatically in a microfluidic device. In particular, we describe how a microfluidic assay to detect HIV viral RNA was converted to detect Zika virus (ZIKV) RNA. We first optimized the RT-LAMP assay to detect ZIKV RNA using a benchtop isothermal amplification device. Then we implemented the assay in a microfluidic device that will allow analyzing 24 samples simultaneously and automatically from sample introduction to detection by RDB technique. Preliminary data using saliva samples spiked with ZIKV showed that our diagnostic system detects ZIKV RNA in saliva. These results will be validated in further experiments with well-characterized ZIKV human specimens of saliva. The described strategy and methodology to convert the HIV diagnostic assay and platform to a ZIKV RNA detection assay provides a model that can be readily utilized for detection of the next emerging or re-emerging infectious disease.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Pre- and postexposure efficacy of fully human antibodies against Spike protein in a novel humanized mouse model of MERS-CoV infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traditional approaches to antimicrobial drug development are poorly suited to combatting the emergence of novel pathogens. Additionally, the lack of small animal models for these infections hinders the in vivo testing of potential therapeutics. Here we demonstrate the use of the VelocImmune technology (a mouse that expresses human antibody-variable heavy chains and κ light chains) alongside the VelociGene technology (which allows for rapid engineering of the mouse genome) to quickly develop and evaluate antibodies against an emerging viral disease. Specifically, we show the rapid generation of fully human neutralizing antibodies against the recently emerged Middle East Respiratory Syndrome coronavirus (MERS-CoV) and development of a humanized mouse model for MERS-CoV infection, which was used to demonstrate the therapeutic efficacy of the isolated antibodies. The VelocImmune and VelociGene technologies are powerful platforms that can be used to rapidly respond to emerging epidemics.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of the EpiCore outbreak verification system</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 May 1</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE: To describe a crowdsourced disease surveillance project (EpiCore) and evaluate its usefulness in obtaining information regarding potential disease outbreaks. METHODS: Volunteer human, animal and environmental health professionals from around the world were recruited to EpiCore and trained to provide early verification of health threat alerts in their geographical region via a secure, easy-to-use, online platform. Experts in the area of emerging infectious diseases sent requests for information on unverified health threats to these volunteers, who used local knowledge and expertise to respond to requests. Experts reviewed and summarized the responses and rapidly disseminated important information to the global health community through the existing event-based disease surveillance network, ProMED. FINDINGS: From March 2016 to September 2017, 2068 EpiCore volunteers from 142 countries were trained in methods of informal disease surveillance and use of the EpiCore online platform. These volunteers provided 790 individual responses to 759 requests for information addressing unverified health threats in 112 countries; 361 (45%) responses were considered to be useful. Most responses were received within hours of the requests. The responses led to 194 ProMED posts, of which 99 (51%) supported verification of an outbreak, were published on ProMED and sent to over 87 000 subscribers. CONCLUSION: There is widespread willingness among health professionals around the world to voluntarily assist efforts to verify and provide supporting information on unconfirmed health threats in their region. By linking this member network of health experts through a secure online reporting platform, EpiCore enables faster global outbreak detection and reporting.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>FluMob: Enabling Surveillance of Acute Respiratory Infections in Health-care Workers via Mobile Phones</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 17</span></i><p style="text-align: justify; text-justify: inter-word;">Singapore is a hotspot for emerging infectious diseases and faces a constant risk of pandemic outbreaks as a major travel and health hub for Southeast Asia. With an increasing penetration of smart phone usage in this region, Singapore’s pandemic preparedness framework can be strengthened by applying a mobile-based approach to health surveillance and control, and improving upon existing ideas by addressing gaps, such as a lack of health communication. FluMob is a digitally integrated syndromic surveillance system designed to assist health authorities in obtaining real-time epidemiological and surveillance data from health-care workers (HCWs) within Singapore, by allowing them to report influenza incidence using smartphones. The system, integrating a fully responsive web-based interface and a mobile interface, is made available to HCW using various types of mobile devices and web browsers. Real-time data generated from FluMob will be complementary to current health-care- and laboratory-based systems. This paper describes the development of FluMob, as well as challenges faced in the creation of the system.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Patient-Centred Coordinated Care in Times of Emerging Diseases and Epidemics: Contribution of the IMIA Working Group on Patient Safety</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: In this paper the researchers describe how existing health information technologies (HIT) can be repurposed and new technologies can be innovated to provide patient-centered care to individuals affected by new and emerging diseases. METHODS: The researchers conducted a focused review of the published literature describing how HIT can be used to support safe, patient-centred, coordinated care to patients who are affected by Ebola (an emerging disease). RESULTS: New and emerging diseases present opportunities for repurposing existing technologies and for stimulating the development of new HIT innovation. Innovative technologies may be developed such as new software used for tracking patients during new or emerging disease outbreaks or by repurposing and extending existing technologies so they can be used to support patients, families and health professionals who may have been exposed to a disease. The paper describes the development of new technologies and the repurposing and extension of existing ones (such as electronic health records) using the most recent outbreak of Ebola as an example.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS, MERS and novel coronavirus (COVID-19) epidemics, the newest and biggest global health threats: what lessons have we learned?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China.Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made.Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally.We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Field-deployable, quantitative, rapid identification of active Ebola virus infection in unprocessed blood</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 1</span></i><p style="text-align: justify; text-justify: inter-word;">The West African Ebola virus outbreak underlined the importance of delivering mass diagnostic capability outside the clinical or primary care setting in effectively containing public health emergencies caused by infectious disease. Yet, to date, there is no solution for reliably deploying at the point of need the gold standard diagnostic method, real time quantitative reverse transcription polymerase chain reaction (RT-qPCR), in a laboratory infrastructure-free manner. In this proof of principle work, we demonstrate direct performance of RT-qPCR on fresh blood using far-red fluorophores to resolve fluorogenic signal inhibition and controlled, rapid freeze/thawing to achieve viral genome extraction in a single reaction chamber assay. The resulting process is entirely free of manual or automated sample pre-processing, requires no microfluidics or magnetic/mechanical sample handling and thus utilizes low cost consumables. This enables a fast, laboratory infrastructure-free, minimal risk and simple standard operating procedure suited to frontline, field use. Developing this novel approach on recombinant bacteriophage and recombinant human immunodeficiency virus (HIV; Lentivirus), we demonstrate clinical utility in symptomatic EBOV patient screening using live, infectious Filoviruses and surrogate patient samples. Moreover, we evidence assay co-linearity independent of viral particle structure that may enable viral load quantification through pre-calibration, with no loss of specificity across an 8 log-linear maximum dynamic range. The resulting quantitative rapid identification (QuRapID) molecular diagnostic platform, openly accessible for assay development, meets the requirements of resource-limited countries and provides a fast response solution for mass public health screening against emerging biosecurity threats.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Field-deployable, quantitative, rapid identification of active Ebola virus infection in unprocessed blood</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 1</span></i><p style="text-align: justify; text-justify: inter-word;">The West African Ebola virus outbreak underlined the importance of delivering mass diagnostic capability outside the clinical or primary care setting in effectively containing public health emergencies caused by infectious disease. Yet, to date, there is no solution for reliably deploying at the point of need the gold standard diagnostic method, real time quantitative reverse transcription polymerase chain reaction (RT-qPCR), in a laboratory infrastructure-free manner. In this proof of principle work, we demonstrate direct performance of RT-qPCR on fresh blood using far-red fluorophores to resolve fluorogenic signal inhibition and controlled, rapid freeze/thawing to achieve viral genome extraction in a single reaction chamber assay. The resulting process is entirely free of manual or automated sample pre-processing, requires no microfluidics or magnetic/mechanical sample handling and thus utilizes low cost consumables. This enables a fast, laboratory infrastructure-free, minimal risk and simple standard operating procedure suited to frontline, field use. Developing this novel approach on recombinant bacteriophage and recombinant human immunodeficiency virus (HIV; Lentivirus), we demonstrate clinical utility in symptomatic EBOV patient screening using live, infectious Filoviruses and surrogate patient samples. Moreover, we evidence assay co-linearity independent of viral particle structure that may enable viral load quantification through pre-calibration, with no loss of specificity across an 8 log-linear maximum dynamic range. The resulting quantitative rapid identification (QuRapID) molecular diagnostic platform, openly accessible for assay development, meets the requirements of resource-limited countries and provides a fast response solution for mass public health screening against emerging biosecurity threats.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the burden of United States workers exposed to infection or disease: a key factor in containing risk of COVID-19 infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Introduction: With the global spread of COVID-19, there is a compelling public health interest in quantifying who is at increased risk of disease. Occupational characteristics, such as interfacing with the public and being in close quarters with other workers, not only put workers at high risk for disease, but also make them a nexus of disease transmission to the community. This can further be exacerbated through presenteeism, the term used to describe the act of coming to work despite being symptomatic for disease. Understanding which occupational groups are exposed to infection and disease in the workplace can help to inform public health risk response and management for COVID-19, and subsequent infectious disease outbreaks.  Methods: To estimate the burden of United States workers exposed to infection and disease in the workplace, national employment data (by Standard Occupational Classification) maintained by the Bureau of Labor Statistics (BLS) was merged with BLS O*NET survey data, which ranks occupations with particular physical, ergonomic, and structural exposures. For this analysis, occupations reporting exposure to infection or disease more than once a month was the focus.    Results: Based on our analyses, approximately 10% (14.4 M) of United States workers are employed in occupations where exposure to disease or infection occurs at least once per week. Approximately 18.4% (26.7 M) of all United States workers are employed in occupations where exposure to disease or infection occurs at least once per month. While the majority of exposed workers are employed in healthcare sectors, other occupational sectors also have high proportions of exposed workers. These include protective service occupations (e.g. police officers, correctional officers, firefighters), office and administrative support occupations (e.g. couriers and messengers, patient service representatives), education occupations (e.g. preschool and daycare teachers), community and social services occupations (community health workers, social workers, counselors), and even construction and extraction occupations (e.g. plumbers, septic tank installers, elevator repair).   Conclusions: The large number of persons employed in a wide variety of occupations with frequent exposure to infection and disease underscore the importance of all workplaces developing risk response plans for COVID-19. This work also serves as an important reminder that the workplace is a key locus for public health interventions, which could protect both workers and the communities they serve.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Recent advances in lab-on-a-chip technologies for viral diagnosis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The global risk of viral disease outbreaks emphasizes the need for rapid, accurate, and sensitive detection techniques to speed up diagnostics allowing early intervention. An emerging field of microfluidics also known as the lab-on-a-chip (LOC) or micro total analysis system includes a wide range of diagnostic devices. This review briefly covers both conventional and microfluidics-based techniques for rapid viral detection. We first describe conventional detection methods such as cell culturing, immunofluorescence or enzyme-linked immunosorbent assay (ELISA), or reverse transcription polymerase chain reaction (RT-PCR). These methods often have limited speed, sensitivity, or specificity and are performed with typically bulky equipment. Here, we discuss some of the LOC technologies that can overcome these demerits, highlighting the latest advances in LOC devices for viral disease diagnosis. We also discuss the fabrication of LOC systems to produce devices for performing either individual steps or virus detection in samples with the sample to answer method. The complete system consists of sample preparation, and ELISA and RT-PCR for viral-antibody and nucleic acid detection, respectively. Finally, we formulate our opinions on these areas for the future development of LOC systems for viral diagnostics.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19's graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control.  Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation.  Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Clinical-Stage Human Monoclonal Antibodies That Treat Advanced Ebola Virus Disease in Nonhuman Primates</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 15</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: For most classes of drugs, rapid development of therapeutics to treat emerging infections is challenged by the timelines needed to identify compounds with the desired efficacy, safety, and pharmacokinetic profiles. Fully human monoclonal antibodies (mAbs) provide an attractive method to overcome many of these hurdles to rapidly produce therapeutics for emerging diseases. METHODS: In this study, we deployed a platform to generate, test, and develop fully human antibodies to Zaire ebolavirus. We obtained specific anti-Ebola virus (EBOV) antibodies by immunizing VelocImmune mice that use human immunoglobulin variable regions in their humoral responses. RESULTS: Of the antibody clones isolated, 3 were selected as best at neutralizing EBOV and triggering FcγRIIIa. Binding studies and negative-stain electron microscopy revealed that the 3 selected antibodies bind to non-overlapping epitopes, including a potentially new protective epitope not targeted by other antibody-based treatments. When combined, a single dose of a cocktail of the 3 antibodies protected nonhuman primates (NHPs) from EBOV disease even after disease symptoms were apparent. CONCLUSIONS: This antibody cocktail provides complementary mechanisms of actions, incorporates novel specificities, and demonstrates high-level postexposure protection from lethal EBOV disease in NHPs. It is now undergoing testing in normal healthy volunteers in preparation for potential use in future Ebola epidemics.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevent and predict</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the COVID-19 outbreak continues, the next pandemic could be prevented by ending the wildlife trade and reinvesting in the monitoring of potential zoonoses.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus outbreak: the role of companies in preparedness and responses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As in previous health crises, the coronavirus disease 2019 (COVID-19) outbreak has raised questions about preparedness and emergency responses in many countries. In this crisis, what role can companies play? Public and private companies must continue to produce or provide their services, but with consideration of the health context. Many companies are involved with the COVID-19 outbreak because they are established in or work with China (client or supplier), and most have already activated their business continuity planning or equivalent. During an infectious disease outbreak like COVID-19, most large companies around the world have a major part to play, especially in terms of preparedness and emergency response. Indeed, companies should be integrated into the governmental health contingency plan developed in many countries, and by WHO and the International Labor Organization. Helped by their occupational practitioners, healthcare advisers, and safety professionals, companies that have a financial capacity and responsibilities (including governmental, federal, or state administrations) will thus have to prepare their business continuity planning for when cases of infected patients occur in the company. They also must be prepared for the potential psychosocial and psychological effects of outbreaks. All health professionals should be involved in the development and implementation of recommendations for companies and their environments</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Programmable low-cost DNA-based platform for viral RNA detection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-16</span></i><p style="text-align: justify; text-justify: inter-word;">Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with [~]24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using 'Feidian', Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Making vaccines “on demand”: A potential solution for emerging pathogens and biodefense?</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Sep 1</span></i><p style="text-align: justify; text-justify: inter-word;">The integrated US Public Health Emergency Medical Countermeasures Enterprise (PHEMCE) has made great strides in strategic preparedness and response capabilities. There have been numerous advances in planning, biothreat countermeasure development, licensure, manufacturing, stockpiling and deployment. Increased biodefense surveillance capability has dramatically improved, while new tools and increased awareness have fostered rapid identification of new potential public health pathogens. Unfortunately, structural delays in vaccine design, development, manufacture, clinical testing and licensure processes remain significant obstacles to an effective national biodefense rapid response capability. This is particularly true for the very real threat of “novel pathogens” such as the avian-origin influenzas H7N9 and H5N1, and new coronaviruses such as hCoV-EMC. Conventional approaches to vaccine development, production, clinical testing and licensure are incompatible with the prompt deployment needed for an effective public health response. An alternative approach, proposed here, is to apply computational vaccine design tools and rapid production technologies that now make it possible to engineer vaccines for novel emerging pathogen and WMD biowarfare agent countermeasures in record time. These new tools have the potential to significantly reduce the time needed to design string-of-epitope vaccines for previously unknown pathogens. The design process—from genome to gene sequence, ready to insert in a DNA plasmid—can now be accomplished in less than 24 h. While these vaccines are by no means “standard,” the need for innovation in the vaccine design and production process is great. Should such vaccines be developed, their 60-d start-to-finish timeline would represent a 2-fold faster response than the current standard.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Improving vaccine trials in infectious disease emergencies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Unprecedented global effort is underway to facilitate testing of countermeasures in infectious disease emergencies. Better understanding of the various options for trial design, as well as preliminary global agreement on the most suitable designs for the various scenarios, are needed now—in advance of outbreaks. What would enhance, then, the speed, validity, and ethics of clinical studies of such countermeasures? Focusing on studies of vaccine efficacy and effectiveness in emergencies, we highlight three needs: for formal randomized trials—even in most emergencies; for individually-randomized trials—even in many emergencies; and for six areas of innovation in trial methodology. These needs should inform current updates of protocols and roadmaps.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>The Special Pathogens Research Network: Enabling Research Readiness</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The 2013-2016 epidemic of Ebola virus disease (EVD) that originated in West Africa underscored many of the challenges to conducting clinical research during an ongoing infectious disease epidemic, both in the most affected countries of Guinea, Liberia, and Sierra Leone, as well as in the United States and Europe, where a total of 27 patients with EVD received care in biocontainment units. The Special Pathogens Research Network (SPRN) was established in the United States in November 2016 to provide an organizational structure to leverage the expertise of the 10 Regional Ebola and Other Special Pathogen Treatment Centers (RESPTCs); it was intended to develop and support infrastructure to improve readiness to conduct clinical research in the United States. The network enables the rapid activation and coordination of clinical research in the event of an epidemic and facilitates opportunities for multicenter research when the RESPTCs are actively caring for patients requiring a biocontainment unit. Here we provide an overview of opportunities identified in the clinical research infrastructure during the West Africa EVD epidemic and the SPRN activities to meet the ongoing challenges in the context of Ebola virus and other special pathogens.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-21</span></i><p style="text-align: justify; text-justify: inter-word;">Reverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability1-3. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-24 in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Tracking virus outbreaks in the 21st century</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging viruses have the potential to impose substantial mortality, morbidity and economic burdens on human populations. Tracking the spread of infectious diseases to assist in their control has traditionally relied on the analysis of case data gathered as the outbreak proceeds. Here, we describe how many of the key questions in infectious disease epidemiology, from the initial detection and characterization of outbreak viruses, to transmission chain tracking and outbreak mapping, can now be much more accurately addressed using recent advances in virus sequencing and phylogenetics. We highlight the utility of this approach with the hypothetical outbreak of an unknown pathogen, ‘Disease X’, suggested by the World Health Organization to be a potential cause of a future major epidemic. We also outline the requirements and challenges, including the need for flexible platforms that generate sequence data in real-time, and for these data to be shared as widely and openly as possible.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>The United States badly bungled coronavirus testing—but things may soon improve | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? The World Health Organization (WHO) has shipped testing kits to 57 countries. China had five commercial tests on the market 1 month ago and can now do up to 1.6 million tests a week; South Korea has tested 65,000 people so far. The U. S. Centers for Disease Control and Prevention (CDC), in contrast, has done only 459 tests since the epidemic began. The rollout of a CDC-designed test kit to state and local labs has become a fiasco because it contained a faulty reagent. Labs around the country eager to test more suspected cases—and test them faster—have been unable to do so. No commercial or state labs have the approval to use their own tests. In what is already an infamous snafu, CDC initially refused a request to test a patient in Northern California who turned out to be the first probable COVID19 case without known links to an infected person.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Wide Identification of SARS-CoV Susceptibility Loci Using the Collaborative Cross</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Oct 9</span></i><p style="text-align: justify; text-justify: inter-word;">New systems genetics approaches are needed to rapidly identify host genes and genetic networks that regulate complex disease outcomes. Using genetically diverse animals from incipient lines of the Collaborative Cross mouse panel, we demonstrate a greatly expanded range of phenotypes relative to classical mouse models of SARS-CoV infection including lung pathology, weight loss and viral titer. Genetic mapping revealed several loci contributing to differential disease responses, including an 8.5Mb locus associated with vascular cuffing on chromosome 3 that contained 23 genes and 13 noncoding RNAs. Integrating phenotypic and genetic data narrowed this region to a single gene, Trim55, an E3 ubiquitin ligase with a role in muscle fiber maintenance. Lung pathology and transcriptomic data from mice genetically deficient in Trim55 were used to validate its role in SARS-CoV-induced vascular cuffing and inflammation. These data establish the Collaborative Cross platform as a powerful genetic resource for uncovering genetic contributions of complex traits in microbial disease severity, inflammation and virus replication in models of outbred populations.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>A DNA biochip for on-the-spot multiplexed pathogen identification</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Oct 25</span></i><p style="text-align: justify; text-justify: inter-word;">Miniaturized integrated DNA analysis systems have largely been based on a multi-chamber design with microfluidic control to process the sample sequentially from one module to another. This microchip design in connection with optics involved hinders the deployment of this technology for point-of-care applications. In this work, we demonstrate the implementation of sample preparation, DNA amplification, and electrochemical detection in a single silicon and glass-based microchamber and its application for the multiplexed detection of Escherichia coli and Bacillus subtilis cells. The microdevice has a thin-film heater and temperature sensor patterned on the silicon substrate. An array of indium tin oxide (ITO) electrodes was constructed within the microchamber as the transduction element. Oligonucleotide probes specific to the target amplicons are individually positioned at each ITO surface by electrochemical copolymerization of pyrrole and pyrrole−probe conjugate. These immobilized probes were stable to the thermal cycling process and were highly selective. The DNA-based identification of the two model pathogens involved a number of steps including a thermal lysis step, magnetic particle-based isolation of the target genomes, asymmetric PCR, and electrochemical sequence-specific detection using silver-enhanced gold nanoparticles. The microchamber platform described here offers a cost-effective and sample-to-answer technology for on-site monitoring of multiple pathogens.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>The Lab Without Walls: A Deployable Approach to Tropical Infectious Diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Laboratory Without Walls is a modular field application of molecular biology that provides clinical laboratory support in resource-limited, remote locations. The current repertoire arose from early attempts to deliver clinical pathology and public health investigative services in remote parts of tropical Australia, to address the shortcomings of conventional methods when faced with emerging infectious diseases. Advances in equipment platforms and reagent chemistry have enabling rapid progress, but also ensure the Laboratory Without Walls is subject to continual improvement. Although new molecular biology methods may lead to more easily deployable clinical laboratory capability, logistic and technical governance issues continue to act as important constraints on wider implementation.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Disruptive Innovation Can Prevent the Next Pandemic</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Sep 23</span></i><p style="text-align: justify; text-justify: inter-word;">Public health surveillance (PHS) is at a tipping point, where the application of novel processes, technologies, and tools promise to vastly improve efficiency and effectiveness. Yet twentieth century, entrenched ideology and lack of training results in slow uptake and resistance to change. The term disruptive innovation – used to describe advances in technology and processes that change existing markets – is useful to describe the transformation of PHS. Past disruptive innovations used in PHS, such as distance learning, the smart phone, and field-based laboratory testing have outpaced older services, practices, and technologies used in the traditional classroom, governmental offices, and personal communication, respectively. Arguably, the greatest of these is the Internet – an infrastructural innovation that continues to enable exponential benefits in seemingly limitless ways. Considering the Global Health Security Agenda and facing emerging and reemerging infectious disease threats, evolving environmental and behavioral risks, and ever changing epidemiologic trends, PHS must transform. Embracing disruptive innovation in the structures and processes of PHS can be unpredictable. However, it is necessary to strengthen and unlock the potential to prevent, detect, and respond.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Response to COVID-19 in Taiwan: Big Data Analytics, New Technology, and Proactive Testing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Taiwan is 81 miles off the coast of mainland China and was expected to have the second highest number of cases of coronavirus disease 2019 (COVID-19) due to its proximity to and number of flights between China.1 The country has 23 million citizens of which 850 000 reside in and 404 000 work in China.2,3 In 2019, 2.71 million visitors from the mainland traveled to Taiwan.4 As such, Taiwan has been on constant alert and ready to act on epidemics arising from China ever since the severe acute respiratory syndrome (SARS) epidemic in 2003. Given the continual spread of COVID-19 around the world, understanding the action items that were implemented quickly in Taiwan and assessing the effectiveness of these actions in preventing a large-scale epidemic may be instructive for other countries. COVID-19 occurred just before the Lunar New Year during which time millions of Chinese and Taiwanese were expected to travel for the holidays. Taiwan quickly mobilized and instituted specific approaches for case identification, containment, and resource allocation to protect the public health. Taiwan leveraged its national health insurance database and integrated it with its immigration and customs database to begin the creation of big data for analytics; it generated real-time alerts during a clinical visit based on travel history and clinical symptoms to aid case identification. It also used new technology, including QR code scanning and online reporting of travel history and health symptoms to classify travelers’ infectious risks based on flight origin and travel history in the past 14 days. Persons with low risk (no travel to level 3 alert areas) were sent a health declaration border pass via SMS (short message service) messaging to their phones for faster immigration clearance; those with higher risk (recent travel to level 3 alert areas) were quarantined at home and tracked through their mobile phone to ensure that they remained at home during the incubation period. Moreover, Taiwan enhanced COVID-19 case finding by proactively seeking out patients with severe respiratory symptoms (based on information from the National Health Insurance [NHI] database) who had tested negative for influenza and retested them for COVID-19; 1 was found of 113 cases. The toll-free number 1922 served as a hotline for citizens to report suspicious symptoms or cases in themselves or others; as the disease progressed, this hotline has reached full capacity, so each major city was asked to create its own hotline as an alternative. It is not known how often this hotline has been used. The government addressed the issue of disease stigma and compassion for those affected by providing food, frequent health checks, and encouragement for those under quarantine. This rapid response included hundreds of action items (eTable in the Supplement).</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Augmented Intelligence (AI) in Detecting and Preventing the Spread of Novel Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019-nCov will not be the last epidemic to challenge public health experts. The growth of AI-driven techniques to identify epidemiologic risks early will be key to our improvement of prediction, prevention, and detection of future global health risks. The devastating situation in Wuhan, China and future epidemics will also find value in ongoing research in 2019-nCov case detection, spread prediction, treatment effectiveness, and containment. The wide variety, velocity, and veracity of data now available in crises yield data sets that many researchers will now need to incorporate into evermore complex models. This requires expansion of talent within AI for healthcare applications. AI is no longer a niche research area nor is it a tool for the most advanced healthcare systems only, its global impact on healthcare is real and its potential to save lives in this epidemic as well as future epidemics should not be underestimated. It is critical to the global health of all humankind for the scientific community to embrace AI and leverage its power in securing our collective future.ER -</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Recommended psychological crisis intervention response to the 2019 novel coronavirus pneumonia outbreak in China: a model of West China Hospital</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus pneumonia (COVID-19)epidemic has brought serious social psychological impact to the Chinese people, especially those quarantined and thus with limited access to face-to-face communication and traditional social psychological interventions. To better deal with the urgent psychological problems of people involved in the COVID-19 epidemic, we developed a new psychological crisis intervention model by utilizing internet technology. This new model, one of West China Hospital, integrates physicians, psychiatrists, psychologists and social workers into Internet platforms to carry out psychological intervention to patients, their families and medical staff. We hope this model will make a sound basis for developing a more comprehensive psychological crisis intervention response system that is applicable for urgent social and psychological problems.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Self-Assembled Combinatorial Encoding Nanoarrays for Multiplexed Biosensing</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Multiplexed and sensitive detection of nucleic acids, proteins, or other molecules from a single solution and a small amount of sample is of great demand in biomarker profiling and disease diagnostics. Here we describe a new concept using combinatorial self-assembly of DNA nanotiles into micrometer-sized two-dimensional arrays that carry nucleic acid probes and barcoded fluorescent dyes to achieve multiplexed detection. We demonstrated the specificity and sensitivity of the arrays by detecting multiple DNA sequences and aptamer binding molecules. This DNA tile-array-based sensor platform can be constructed through DNA self-assembly. The attachment of different molecular probes can be achieved by simple DNA hybridization so bioconjugation is not necessary for the labeling. Accurate control of the interprobe distances and solution-based binding reactions ensures fast target binding kinetics.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of an Adjustable Epidemiologic Information System</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Jan 27</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In order to facilitate public health response and to achieve early control of infectious disease epidemics, an adjustable epidemiologic information system (AEIS) was established in the Taiwan public health network in February 2006. METHODOLOGY/PRINCIPAL FINDINGS: The performance of AEIS for the period 2006 through 2008 was evaluated based on a number of response times (RT) and the public health impact. After implementation of the system, the apparent overall shortened RT was mainly due to the shortening of personnel response time (PRT) and the time needed to draft a new questionnaire that incurred as personnel-system interface (PSI); PRT dropped from a fluctuating range of 9.8 ∼28.8 days in the first four months to <10 days in the following months and remained low till 2008 (0.88±1.52 days). The PSIs for newly emerged infectious diseases were 2.6 and 3.4 person-hours for H5N1 in 2007 and chikungunya in 2008, respectively, a much improvement from 1142.5 person-hours for SARS in 2003. The duration of each rubella epidemic cluster was evaluated as public health impact and showed a shortening trend (p = 0.019) that concurred with the shortening of PRT from 64.8±47.3 to 25.2±38.2 hours per cluster (p<0.0001). CONCLUSIONS/SIGNIFICANCE: The first evaluation of the novel instrument AEIS that had been used to assist Taiwan's multi-level government for infectious diseases control demonstrated that it was well integrated into the existing public health infrastructure. It provided flexible tools and computer algorithms with friendly interface for timely data collection, integration, and analysis; as a result, it shortened RTs, filled in gaps of personnel lacking sufficient experiences, created a more efficient flow of response, and identified asymptomatic/mild cases early to minimize further spreading. With further development, AEIS is anticipated to be useful in the application of other acute public health events needing immediate orchestrated data collection and public health actions.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of an Adjustable Epidemiologic Information System</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Jan 27</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In order to facilitate public health response and to achieve early control of infectious disease epidemics, an adjustable epidemiologic information system (AEIS) was established in the Taiwan public health network in February 2006. METHODOLOGY/PRINCIPAL FINDINGS: The performance of AEIS for the period 2006 through 2008 was evaluated based on a number of response times (RT) and the public health impact. After implementation of the system, the apparent overall shortened RT was mainly due to the shortening of personnel response time (PRT) and the time needed to draft a new questionnaire that incurred as personnel-system interface (PSI); PRT dropped from a fluctuating range of 9.8 ∼28.8 days in the first four months to <10 days in the following months and remained low till 2008 (0.88±1.52 days). The PSIs for newly emerged infectious diseases were 2.6 and 3.4 person-hours for H5N1 in 2007 and chikungunya in 2008, respectively, a much improvement from 1142.5 person-hours for SARS in 2003. The duration of each rubella epidemic cluster was evaluated as public health impact and showed a shortening trend (p = 0.019) that concurred with the shortening of PRT from 64.8±47.3 to 25.2±38.2 hours per cluster (p<0.0001). CONCLUSIONS/SIGNIFICANCE: The first evaluation of the novel instrument AEIS that had been used to assist Taiwan's multi-level government for infectious diseases control demonstrated that it was well integrated into the existing public health infrastructure. It provided flexible tools and computer algorithms with friendly interface for timely data collection, integration, and analysis; as a result, it shortened RTs, filled in gaps of personnel lacking sufficient experiences, created a more efficient flow of response, and identified asymptomatic/mild cases early to minimize further spreading. With further development, AEIS is anticipated to be useful in the application of other acute public health events needing immediate orchestrated data collection and public health actions.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of the clinical characteristics of suspected or confirmed cases of COVID-19 during home care with isolation: A new retrospective analysis based on O2O</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Summary Background The recent outbreak of the novel coronavirus in December 2019 (COVID-19) has activated top-level response nationwide. We developed a new treatment model based on the online-to-offline (O2O) model for the home isolated patients, because in the early stages the medical staff were insufficient to cope with so many patients. Methods In this single-centered, retrospective study, we enrolled 48 confirmed/suspected COVID-19 patients who underwent home isolation in Wuhan between January 6 and January 31, 2020. By WeChat and online document editing all patients were treated with medical observation scale. The clinical indications such as Fever, Muscle soreness, Dyspnea and Lack of strength were collected with this system led by medical staff in management, medicine, nursing, rehabilitation and psychology.  Findings The mean(SD) age of 48 patients was 39.08(13.88) years, 35(72.9%) were women. Compared with non-hospitalized patients, inpatients were older(≥8805;70years, 2.4% vs 33.3%, P&lt;0.04). All inpatients had fever, 50% inpatients had coughs and showed infiltration in both lungs at the time of diagnosis. 33.3% inpatients exhibited negative changes in their CT results at initial diagnosis. The body temperature of non-hospitalized patients with mild symptoms returned to normal by day 4-5. While dyspnea peaked on day 6 for non-hospitalized patients with mild symptoms, it persisted in hospitalized patients and exacerbated over time. The lack of strength and muscle soreness were both back to normal by day 4 for non-hospitalized patients. Interpretation Monitoring the trends of symptoms is more important for identifying severe cases. Excessive laboratory data and physical examination are not necessary for the evaluation of patients with mild symptoms. The system we developed is the first to convert the subjective symptoms of patients into objective scores. This type of O2O, subjective-to-objective strategy may be used in regions with similar highly infectious diseases to minimize the possibility of infection among medical staff.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Perceptions of the Adult US Population regarding the Novel Coronavirus Outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: COVID-19 outbreak is spreading globally. Although the risk of infection in the US is currently low, it is important to understand the public perception of risk and trust in sources of information to better inform public health messaging. In this study, we surveyed the adult US population to understand their risk perceptions about the COVID-19 outbreak.  Methods and Findings: We used an online platform to survey 718 adults in the US in early February 2020 using a questionnaire that we developed. Our sample was fairly similar to the general adult US population in terms of age, gender, race, ethnicity and education. We found that 69% of the respondents wanted the scientific/public health leadership (either the CDC Director or NIH Director) to lead the US response to COVID-19 outbreak as compared to 14% who wanted the political leadership (either the president or the Congress) to lead the response. Risk perception was low (median score of 5 out of 10) with the respondents trusting health professionals and health officials for information on COVID-19. Majority of the respondents were in favor of strict infection prevention policies to control the outbreak.  Conclusion: Given our results, the public health/scientific leadership should be at the forefront of the COVID-19 response to promote trust.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>FilmArray, an Automated Nested Multiplex PCR System for Multi-Pathogen Detection: Development and Application to Respiratory Tract Infection</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Oct 19</span></i><p style="text-align: justify; text-justify: inter-word;">The ideal clinical diagnostic system should deliver rapid, sensitive, specific and reproducible results while minimizing the requirements for specialized laboratory facilities and skilled technicians. We describe an integrated diagnostic platform, the “FilmArray”, which fully automates the detection and identification of multiple organisms from a single sample in about one hour. An unprocessed biologic/clinical sample is subjected to nucleic acid purification, reverse transcription, a high-order nested multiplex polymerase chain reaction and amplicon melt curve analysis. Biochemical reactions are enclosed in a disposable pouch, minimizing the PCR contamination risk. FilmArray has the potential to detect greater than 100 different nucleic acid targets at one time. These features make the system well-suited for molecular detection of infectious agents. Validation of the FilmArray technology was achieved through development of a panel of assays capable of identifying 21 common viral and bacterial respiratory pathogens. Initial testing of the system using both cultured organisms and clinical nasal aspirates obtained from children demonstrated an analytical and clinical sensitivity and specificity comparable to existing diagnostic platforms. We demonstrate that automated identification of pathogens from their corresponding target amplicon(s) can be accomplished by analysis of the DNA melting curve of the amplicon.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Initial Results of a Low Cost, Disposable, Point-of-Care Testing Device for Pathogen Detection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Development of small footprint, disposable, fast, and inexpensive devices for pathogen detection in the field and clinic would benefit human and veterinary medicine by allowing evidence-based responses to future out breaks. We designed and tested an integrated nucleic acid extraction and amplification device employing a loop-mediated isothermal amplification (LAMP) or reverse transcriptase-LAMP assay. Our system provides a screening tool with polymerase-chain-reaction-level sensitivity and specificity for outbreak detection, response, and recovery. Time to result is ~90 min. The device utilizes a swab that collects sample and then transfers it to a disc of cellulose-based nucleic acid binding paper. The disc is positioned within a disposable containment tube with a manual loading port. In order to test for the presence of target pathogens, LAMP reagents are loaded through the tube’s port into contact with the sample containing cellulose disc. The reagents then are isothermally heated to 63°C for ~1 h to achieve sequence-specific target nucleic acid amplification. Due to the presence of a colorimetric dye, amplification induces visible color change in the reagents from purple to blue. As initial demonstrations, we detected methicillin resistant Staphylococcus aureus genomic DNA, as well as recombinant and live foot-and-mouth disease virus.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<i class="tooltip">2017<span class="tooltiptext">2017-09-22</span></i><p style="text-align: justify; text-justify: inter-word;">Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray(R) Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Developing Universal Influenza Vaccines: Hitting the Nail, Not Just on the Head</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Mar 26</span></i><p style="text-align: justify; text-justify: inter-word;">Influenza viruses have a huge impact on public health. Current influenza vaccines need to be updated annually and protect poorly against antigenic drift variants or novel emerging subtypes. Vaccination against influenza can be improved in two important ways, either by inducing more broadly protective immune responses or by decreasing the time of vaccine production, which is relevant especially during a pandemic outbreak. In this review, we outline the current efforts to develop so-called “universal influenza vaccines”, describing antigens that may induce broadly protective immunity and novel vaccine production platforms that facilitate timely availability of vaccines.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>With $115 million, more than 80 Boston researchers will collaborate to tackle COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A $115 million collaboration to tackle the rapidly spreading viral disease COVID-19, led by heavy hitters of Boston science and funded by a Chinese property development company, kicked off today as the group’s leaders pledged to take on the virus on many fronts. The project brings together researchers at many of the city’s top academic institutions, along with local biotechnology companies such as Moderna. Those leading it hope they can quickly funnel money into studies that will build off a new repository of samples from infected people and community surveillance, materials that can be rapidly shared among scientists. The project, they anticipate, should answer critical questions about how COVID-19 is spreading and how best to prevent and treat infections.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Dynamics Simulations Indicate the COVID-19 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed molecular dynamics simulations of the main protease (Mpro). We compared and contrasted the Mpro for COVID-19 with a highly similar SARS protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the pockets time-dependence indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus mutability will pose a further challenge to the rational design of small-molecule inhibitors.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid development of vaccines against emerging pathogens: The replication-deficient simian adenovirus platform technology</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 16</span></i><p style="text-align: justify; text-justify: inter-word;">Despite the fact that there had been multiple small outbreaks of Ebola Virus Disease, when a large outbreak occurred in 2014 there were no vaccines or drugs available for use. Clinical development of multiple candidate vaccines was then initiated in parallel with attempts to contain the outbreak but only one vaccine was eventually tested in a phase III trial. In order to be better prepared for future outbreaks of known human pathogens, platform technologies to accelerate vaccine development should be employed, allowing vaccine developers to take advantage of detailed knowledge of the vaccine platform and facilitating rapid progress to clinical trials and eventually to vaccine stockpiles. This review gives an example of one such vaccine platform, replication-deficient simian adenoviruses, and describes progress in human and livestock vaccine development for three outbreak pathogens, Ebola virus, Rift Valley Fever Virus and Middle East Respiratory Syndrome Coronavirus.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">1The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Genome Sequencing of RNA Viruses</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">We developed a system for rapid determination of viral RNA sequences whereby genomic sequence is obtained from cultured virus isolates without subcloning into plasmid vectors. This method affords new opportunities to address the challenges of unknown or untypeable emerging viruses.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanorobot Hardware Architecture for Medical Defense</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">This work presents a new approach with details on the integrated platform and hardware architecture for nanorobots application in epidemic control, which should enable real time in vivo prognosis of biohazard infection. The recent developments in the field of nanoelectronics, with transducers progressively shrinking down to smaller sizes through nanotechnology and carbon nanotubes, are expected to result in innovative biomedical instrumentation possibilities, with new therapies and efficient diagnosis methodologies. The use of integrated systems, smart biosensors, and programmable nanodevices are advancing nanoelectronics, enabling the progressive research and development of molecular machines. It should provide high precision pervasive biomedical monitoring with real time data transmission. The use of nanobioelectronics as embedded systems is the natural pathway towards manufacturing methodology to achieve nanorobot applications out of laboratories sooner as possible. To demonstrate the practical application of medical nanorobotics, a 3D simulation based on clinical data addresses how to integrate communication with nanorobots using RFID, mobile phones, and satellites, applied to long distance ubiquitous surveillance and health monitoring for troops in conflict zones. Therefore, the current model can also be used to prevent and save a population against the case of some targeted epidemic disease.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodias first case of COVID-2019.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Effectiveness of control strategies for Coronavirus Disease 2019: a SEIR dynamic modeling study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Since its first cases occurrence in Wuhan, China, the Coronavirus Disease 2019 (COVID-19) has been spreading rapidly to other provinces and neighboring countries. A series of intervention strategies have been implemented, but didn't stop its spread. Methods: Two mathematical models have been developed to simulate the current epidemic situation in the city of Wuhan and in other parts of China. Special considerations were given to the mobility of people for the estimation and forecast the number of asymptomatic infections, symptomatic infections, and the infections of super-spreading events (Isse).  Findings: The basic reproductive number (R0) was calculated for the period between 18 January 2020 and 16 February 2020: R0 declined from 5.75 to 1.69 in Wuhan and from 6.22 to 1.67 in the entire country (not including the Wuhan area). At the same time, Wuhan is estimated to observe a peak in the number of confirmed cases around 6 February 2020. The number of infected individuals in the entire country (not including the Wuhan area) peaked around February 3. The results also show that the peak of new asymptomatic cases per day in Wuhan occurred on February 6, and the peak of new symptomatic infections have occurred on February 3. Concurrently, while the number of confirmed cases nationwide would continue to decline, the number of real-time COVID-19 inpatients in Wuhan has reached a peak of 13,030 on February 14 before it decreases. The model further shows that the COVID-19 cases will gradually wane by the end of April 2020, both in Wuhan and the other parts of China. The number of confirmed cases would reach the single digit on March 27 in Wuhan and March 19 in the entire country. The five cities with top risk index in China with the exclusion of Wuhan are: Huanggang, Xiaogan, Jingzhou, Chongqing, and Xiangyang city. Interpretations: Although the national peak time has been reached, a significant proportion of asymptomatic patients and the infections of super-spreading events (Isse) still exist in the population, indicating the potential difficulty for the prevention and control of the disease. As the Return-to-Work tide is approaching and upgrading, further measures (e.g., escalatory quarantine, mask wearing when going out, and sit apart when taking vehicles) will be particularly crucial to stop the COVID-19 in other cities outside of Wuhan.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>A Sweet Spot for Molecular Diagnostics: Coupling Isothermal Amplification and Strand Exchange Circuits to Glucometers</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jun 8</span></i><p style="text-align: justify; text-justify: inter-word;">Strand exchange nucleic acid circuitry can be used to transduce isothermal nucleic acid amplification products into signals that can be readable on an off-the-shelf glucometer. Loop-mediated isothermal amplification (LAMP) is limited by the accumulation of non-specific products, but nucleic acid circuitry can be used to probe and distinguish specific amplicons. By combining this high temperature isothermal amplification method with a thermostable invertase, we can directly transduce Middle-East respiratory syndrome coronavirus and Zaire Ebolavirus templates into glucose signals, with a sensitivity as low as 20–100 copies/μl, equating to atto-molar (or low zepto-mole). Virus from cell lysates and synthetic templates could be readily amplified and detected even in sputum or saliva. An OR gate that coordinately triggered on viral amplicons further guaranteed fail-safe virus detection. The method describes has potential for accelerating point-of-care applications, in that biological samples could be applied to a transducer that would then directly interface with an off-the-shelf, approved medical device.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>A Sweet Spot for Molecular Diagnostics: Coupling Isothermal Amplification and Strand Exchange Circuits to Glucometers</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jun 8</span></i><p style="text-align: justify; text-justify: inter-word;">Strand exchange nucleic acid circuitry can be used to transduce isothermal nucleic acid amplification products into signals that can be readable on an off-the-shelf glucometer. Loop-mediated isothermal amplification (LAMP) is limited by the accumulation of non-specific products, but nucleic acid circuitry can be used to probe and distinguish specific amplicons. By combining this high temperature isothermal amplification method with a thermostable invertase, we can directly transduce Middle-East respiratory syndrome coronavirus and Zaire Ebolavirus templates into glucose signals, with a sensitivity as low as 20–100 copies/μl, equating to atto-molar (or low zepto-mole). Virus from cell lysates and synthetic templates could be readily amplified and detected even in sputum or saliva. An OR gate that coordinately triggered on viral amplicons further guaranteed fail-safe virus detection. The method describes has potential for accelerating point-of-care applications, in that biological samples could be applied to a transducer that would then directly interface with an off-the-shelf, approved medical device.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>A comparison of smartphones to paper-based questionnaires for routine influenza sentinel surveillance, Kenya, 2011–2012</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Dec 24</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: For disease surveillance, manual data collection using paper-based questionnaires can be time consuming and prone to errors. We introduced smartphone data collection to replace paper-based data collection for an influenza sentinel surveillance system in four hospitals in Kenya. We compared the quality, cost and timeliness of data collection between the smartphone data collection system and the paper-based system. METHODS: Since 2006, the Kenya Ministry of Health (MoH) with technical support from the Kenya Medical Research Institute/Centers for Disease Control and Prevention (KEMRI/CDC) conducted hospital-based sentinel surveillance for influenza in Kenya. In May 2011, the MOH replaced paper-based collection with an electronic data collection system using Field Adapted Survey Toolkit (FAST) on HTC Touch Pro2 smartphones at four sentinel sites. We compared 880 paper-based questionnaires dated Jan 2010-Jun 2011 and 880 smartphone questionnaires dated May 2011-Jun 2012 from the four surveillance sites. For each site, we compared the quality, cost and timeliness of each data collection system. RESULTS: Incomplete records were more likely seen in data collected using pen-and-paper compared to data collected using smartphones (adjusted incidence rate ratio (aIRR) 7, 95% CI: 4.4-10.3). Errors and inconsistent answers were also more likely to be seen in data collected using pen-and-paper compared to data collected using smartphones (aIRR: 25, 95% CI: 12.5-51.8). Smartphone data was uploaded into the database in a median time of 7 days while paper-based data took a median of 21 days to be entered (p < 0.01). It cost USD 1,501 (9.4%) more to establish the smartphone data collection system ($17,500) than the pen-and-paper system (USD $15,999). During two years, however, the smartphone data collection system was $3,801 (7%) less expensive to operate ($50,200) when compared to pen-and-paper system ($54,001). CONCLUSIONS: Compared to paper-based data collection, an electronic data collection system produced fewer incomplete data, fewer errors and inconsistent responses and delivered data faster. Although start-up costs were higher, the overall costs of establishing and running the electronic data collection system were lower compared to paper-based data collection system. Electronic data collection using smartphones has potential to improve timeliness, data integrity and reduce costs.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>A molecular beacon, bead-based assay for the detection of nucleic acids by flow cytometry</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Jan 19</span></i><p style="text-align: justify; text-justify: inter-word;">Molecular beacons are dual-labelled probes that are typically used in real-time PCR assays, but have also been conjugated with solid matrices for use in microarrays or biosensors. We have developed a fluid array system using microsphere-conjugated molecular beacons and the flow cytometer for the specific, multiplexed detection of unlabelled nucleic acids in solution. For this array system, molecular beacons were conjugated with microspheres using a biotin-streptavidin linkage. A bridged conjugation method using streptavidin increased the signal-to-noise ratio, allowing for further discrimination of target quantitation. Using beads of different sizes and molecular beacons in two fluorophore colours, synthetic nucleic acid control sequences were specifically detected for three respiratory pathogens, including the SARS coronavirus in proof-of-concept experiments. Considering that routine flow cytometers are able to detect up to four fluorescent channels, this novel assay may allow for the specific multiplex detection of a nucleic acid panel in a single tube.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>A molecular beacon, bead-based assay for the detection of nucleic acids by flow cytometry</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Jan 19</span></i><p style="text-align: justify; text-justify: inter-word;">Molecular beacons are dual-labelled probes that are typically used in real-time PCR assays, but have also been conjugated with solid matrices for use in microarrays or biosensors. We have developed a fluid array system using microsphere-conjugated molecular beacons and the flow cytometer for the specific, multiplexed detection of unlabelled nucleic acids in solution. For this array system, molecular beacons were conjugated with microspheres using a biotin-streptavidin linkage. A bridged conjugation method using streptavidin increased the signal-to-noise ratio, allowing for further discrimination of target quantitation. Using beads of different sizes and molecular beacons in two fluorophore colours, synthetic nucleic acid control sequences were specifically detected for three respiratory pathogens, including the SARS coronavirus in proof-of-concept experiments. Considering that routine flow cytometers are able to detect up to four fluorescent channels, this novel assay may allow for the specific multiplex detection of a nucleic acid panel in a single tube.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Mining Social Media and Web Searches For Disease Detection</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 May 31</span></i><p style="text-align: justify; text-justify: inter-word;">Web-based social media is increasingly being used across different settings in the health care industry. The increased frequency in the use of the Internet via computer or mobile devices provides an opportunity for social media to be the medium through which people can be provided with valuable health information quickly and directly. While traditional methods of detection relied predominately on hierarchical or bureaucratic lines of communication, these often failed to yield timely and accurate epidemiological intelligence. New web-based platforms promise increased opportunities for a more timely and accurate spreading of information and analysis. This article aims to provide an overview and discussion of the availability of timely and accurate information. It is especially useful for the rapid identification of an outbreak of an infectious disease that is necessary to promptly and effectively develop public health responses. These web-based platforms include search queries, data mining of web and social media, process and analysis of blogs containing epidemic key words, text mining, and geographical information system data analyses. These new sources of analysis and information are intended to complement traditional sources of epidemic intelligence. Despite the attractiveness of these new approaches, further study is needed to determine the accuracy of blogger statements, as increases in public participation may not necessarily mean the information provided is more accurate.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Sensitive one-step isothermal detection of pathogen-derived RNAs</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreaks of Ebola, Zika, MERS, and SARS-CoV-2 (2019-nCoV) require fast, simple, and sensitive onsite nucleic acid diagnostics that can be developed rapidly to prevent the spread of diseases. We have developed a SENsitive Splint-based one-step isothermal RNA detection (SENSR) method for rapid and straightforward onsite detection of pathogen RNAs with high sensitivity and specificity. SENSR consists of two simple enzymatic reactions: a ligation reaction by SplintR ligase and subsequent transcription by T7 RNA polymerase. The resulting transcript forms an RNA aptamer that induces fluorescence. Here, we demonstrate that SENSR is an effective and highly sensitive method for the detection of the current epidemic pathogen, <jats:italic>severe acute respiratory syndrome-related coronavirus 2</jats:italic> (SARS-CoV-2). We also show that the platform can be extended to the detection of five other pathogens. Overall, SENSR is a molecular diagnostic method that can be developed rapidly for onsite uses requiring high sensitivity, specificity, and short assaying times.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccines, inspiring innovation in health()</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Nov 19</span></i><p style="text-align: justify; text-justify: inter-word;">This report covers the topics of pandemics, epidemics and partnerships, including regulatory convergence initiatives, new technologies and novel vaccines, discussed by leading public and private sector stakeholders at the 18th Annual General Meeting (AGM) of the Developing Countries Vaccine Manufacturers’ Network (DCVMN). Contributions of Gavi and the vaccine industry from emerging countries to the growing global vaccine market, by improving the supply base from manufacturers in developing countries and contributing to 58% of doses, were highlighted. The Coalition for Epidemic Preparedness Innovations (CEPI), the International Vaccine Institute (IVI) and others reported on new strategies to ensure speedy progress in preclinical and clinical development of innovative vaccines for future MERS, Zika or other outbreak response. Priorities for vaccine stockpiling, to assure readiness during emergencies and to prevent outbreaks due to re-emerging diseases such as yellow fever, cholera and poliomyelitis, were outlined. The role of partnerships in improving global vaccine access, procurement and immunization coverage, and shared concerns were reviewed. The World Health Organization (WHO) and other international collaborating partners provided updates on the Product, Price and Procurement database, the prequalification of vaccines, the control of neglected tropical diseases, particularly the new rabies elimination initiative, and regulatory convergence proposals to accelerate vaccine registration in developing countries. Updates on supply chain innovations and novel vaccine platforms were presented. The discussions enabled members and partners to reflect on efficiency of research & development, supply chain tools and trends in packaging technologies improving delivery of existing vaccines, and allowing a deeper understanding of the current public-health objectives, industry financing, and global policies, required to ensure optimal investments, alignment and stability of vaccine supply in developing countries.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearmans rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Looming threat of COVID-19 infection in Africa: act collectively, and fast</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Models that enable the continent to better allocate scarce resources to better prepare and respond to the COVID-19 epidemic are crucial. The modelling study by Marius Gilbert and colleagues in The Lancet identifies each African country's risk of importation of COVID-19 from China, using data on the volume of air travel from three airports in provinces in China to African countries. Gilbert and colleagues use two indicators to determine the capacity of countries to detect and respond to cases: preparedness, using the WHO International Health Regulations Monitoring and Evaluation Framework; and vulnerability, using the Infectious Disease Vulnerability Index. Based on their analysis, Egypt, Algeria, and South Africa had the highest importation risk, and a moderate to high capacity to respond to outbreaks. Nigeria, Ethiopia, Sudan, Angola, Tanzania, Ghana, and Kenya had moderate risk with variable capacity and high vulnerability. In the model, the risk mainly originates from Guangdong, Fujian, and Beijing. The study provides a valuable tool that can help countries in Africa prioritise and allocate resources as they prepare to respond to the potential introduction and spread of COVID-19.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanoplasmonic Quantitative Detection of Intact Viruses from Unprocessed Whole Blood</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases such as HIV and Hepatitis B infection pose an omnipresent threat to global health. Reliable, fast, accurate and sensitive platforms that can be deployed at the point-of-care (POC) in multiple settings, such as airports and offices for detection of infectious pathogens are essential for the management of epidemics and possible biological attacks. To the best of our knowledge, no viral load technology adaptable to the POC settings exists today due to critical technical and biological challenges. Here, we present for the first time a broadly applicable technology for quantitative, nanoplasmonic-based intact virus detection at clinically relevant concentrations. The sensing platform is based on unique nanoplasmonic properties of nanoparticles utilizing immobilized antibodies to selectively capture rapidly evolving viral subtypes. We demonstrate the capture, detection and quantification of multiple HIV subtypes (A, B, C, D, E, G, and subtype panel) with high repeatability, sensitivity and specificity down to 98 ± 39 copies/mL (i.e., subtype D) using spiked whole blood samples and clinical discarded HIV-infected patient whole blood samples validated by the gold standard, i.e., RT-qPCR. This platform technology offers an assay time of 1 hour and 10 minutes (1 hour for capture, 10 minutes for detection and data analysis). The presented platform is also able to capture intact viruses at high efficiency using immuno-surface chemistry approaches directly from whole blood samples without any sample preprocessing steps such as spin-down or sorting. Evidence is presented showing the system to be accurate, repeatable and reliable. Additionally, the presented platform technology can be broadly adapted to detect other pathogens having reasonably well-described biomarkers by adapting the surface chemistry. Thus, this broadly applicable detection platform holds great promise to be implemented potentially at POC settings, hospital and primary care settings.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Evaluation of a Multiplexed Immunoassay for Simultaneous Detection of Serum IgG Antibodies to Six Human Coronaviruses</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 4</span></i><p style="text-align: justify; text-justify: inter-word;">Known human coronaviruses (hCoV) usually cause mild to moderate upper-respiratory tract illnesses, except SARS-CoV and MERS-CoV, which, in addition to mild illness can also be associated with severe respiratory diseases and high mortality rates. Well-characterized multiplexed serologic assays are needed to aid in rapid detection and surveillance of hCoVs. The present study describes development and evaluation of a multiplexed magnetic microsphere immunoassay (MMIA) to simultaneously detect immunoglobulin G (IgG) antibodies specific for recombinant nucleocapsid proteins (recN) from hCoVs 229E, NL63, OC43, HKU1, SARS-CoV, and MERS-CoV. We used paired human sera to screen for IgG with reactivity against six hCoVs to determine assay sensitivity, specificity and reproducibility. We found no signal interference between monoplex and multiplex assay formats (R(2) range = 0.87–0.97). Screening of paired human sera using MMIA, resulted in 92 of 106 (sensitivity: 86%) as positive and 68 of 80 (specificity: 84%) as negative. This study serves as a proof of concept that it is feasible to develop and use a multiplexed microsphere immunoassay as a next generation screening tool for use in large scale seroprevalence studies of hCoVs.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: Establishing a social contact data sharing initiative and an online tool to assess mitigation strategies for COVID-19. Results: Using our online tool and the available social contact data, we illustrate that social distancing could have a considerable impact on reducing transmission for COVID-19. The effect itself depends on assumptions made about disease-specific characteristics and the choice of intervention(s).  Keywords: Social contact data, Data sharing initiative, User interface.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Geospatial Science and Point-of-Care Testing: Creating Solutions for Population Access, Emergencies, Outbreaks, and Disasters</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 26</span></i><p style="text-align: justify; text-justify: inter-word;">Objectives: (a) To understand how to integrate geospatial concepts when implementing point-of-care testing (POCT); (b) to facilitate emergency, outbreak, and disaster preparedness and emergency management in healthcare small-world networks; (c) to enhance community resilience by using POCT in tandem with geographic information systems (GISs) and other geospatial tools; and (d) to advance crisis standards of care at points of need, adaptable and scalable for public health practice in limited-resource countries and other global settings. Content: Visual logistics help integrate and synthesize POCT and geospatial concepts. The resulting geospatial solutions presented here comprise: (1) small-world networks and regional topography; (2) space-time transformation, hubs, and asset mapping; (3) spatial and geospatial care paths™; (4) GIS-POCT; (5) isolation laboratories, diagnostics isolators, and mobile laboratories for highly infectious diseases; (6) alternate care facilities; (7) roaming POCT—airborne, ambulances, space, and wearables; (8) connected and wireless POCT outside hospitals; (9) unmanned aerial vehicles; (10) geospatial practice—demographic care unit resource scoring, geographic risk assessment, and national POCT policy and guidelines; (11) the hybrid laboratory; and (12) point-of-careology. Value: Small-world networks and their connectivity facilitate efficient and effective placement of POCT for optimal response, rescue, diagnosis, and treatment. Spatial care paths™ speed transport from primary encounters to referral centers bypassing topographic bottlenecks, process gaps, and time-consuming interruptions. Regional GISs position POCT close to where patients live to facilitate rapid triage, decrease therapeutic turnaround time, and conserve economic resources. Geospatial care paths™ encompass demographic and population access features. Timeliness creates value during acute illness, complex crises, and unexpected disasters. Isolation laboratories equipped with POCT help stop outbreaks and safely support critically ill patients with highly infectious diseases. POCT-enabled spatial grids can map sentinel cases and establish geographic limits of epidemics for ring vaccination. Impact: Geospatial solutions generate inherently optimal and logical placement of POCT conceptually, physically, and temporally as a means to improve crisis response and spatial resilience. If public health professionals, geospatial scientists, and POCT specialists join forces, new collaborative teamwork can create faster response and higher impact during disasters, complex crises, outbreaks, and epidemics, as well as more efficient primary, urgent, and emergency community care.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Initial Performance Evaluation of a Spotted Array Mobile Analysis Platform (MAP) for the Detection of Influenza A/B, RSV and MERS Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Clinical samples were evaluated with the Mobile Analysis Platform (MAP) to determine platform performance for detecting respiratory viruses in samples previously characterized using clinical RT-PCR assays. The percent agreement between MAP and clinical results was 97% for influenza A (73/75), 100% (21/21) for influenza B, 100% (6/6) for RSV, and 80% (4/5) for negative specimens. The approximate LOD of the MAP was 30 copies/assay for RSV and 1500 copies/assay for MERS Coronavirus.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Application of TaqMan Low-Density Arrays for Simultaneous Detection of Multiple Respiratory Pathogens</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The large and growing number of viral and bacterial pathogens responsible for respiratory infections poses a challenge for laboratories seeking to provide rapid and comprehensive pathogen identification. We evaluated a novel application of the TaqMan low-density array (TLDA) cards for real-time PCR detection of 21 respiratory-pathogen targets. The performance of the TLDA was compared to that of individual real-time PCR (IRTP) assays with the same primers and probes using (i) nucleic acids extracted from the 21 pathogen strains and 66 closely related viruses and bacteria and (ii) 292 clinical respiratory specimens. With spiked samples, TLDA cards were about 10-fold less sensitive than IRTP assays. By using 292 clinical specimens to generate 2,238 paired individual assays, the TLDA card exhibited 89% sensitivity (95% confidence interval [CI], 86 to 92%; range per target, 47 to 100%) and 98% specificity (95% CI, 97 to 99%; range per target, 85 to 100%) overall compared to IRTP assays as the gold standard with a threshold cycle (C(T)) cutoff of 43. The TLDA card approach offers promise for rapid and simultaneous identification of multiple respiratory pathogens for outbreak investigations and disease surveillance.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The Global Outbreak Alert and Response Network</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Oct 21</span></i><p style="text-align: justify; text-justify: inter-word;">The Global Outbreak Alert and Response Network (GOARN) was established in 2000 as a network of technical institutions, research institutes, universities, international health organisations and technical networks willing to contribute and participate in internationally coordinated responses to infectious disease outbreaks. It reflected a recognition of the need to strengthen and coordinate rapid mobilisation of experts in responding to international outbreaks and to overcome the sometimes chaotic and fragmented operations characterising previous responses. The network partners agreed that the World Health Organization would coordinate the network and provide a secretariat, which would also function as the operational support team. The network has evolved to comprise 153 institutions/technical partners and 37 additional networks, the latter encompassing a further 355 members and has been directly involved in 137 missions to 79 countries, territories or areas. Future challenges will include supporting countries to achieve the capacity to detect and respond to outbreaks of international concern, as required by the International Health Regulations (2005). GOARN's increasing regional focus and expanding geographic composition will be central to meeting these challenges. The paper summarises some of network's achievements over the past 13 years and presents some of the future challenges.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Updated rapid risk assessment from ECDC on the outbreak of COVID-19: increased transmission globally</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The European Centre for Disease Prevention and Control (ECDC) provides regularly updated information on coronavirus disease-2019 (COVID-19) relevant to Europe on a dedicated webpage. Besides general information including Q&As, daily case counts, and maps with disease distribution, examples of latest updates comprise: Resource estimation for contact tracing, quarantine and monitoring activities for COVID-19 cases in the EU/EEA, Guidance for wearing and removing personal protective equipment in healthcare settings for the care of patients with suspected or confirmed COVID-19 and Checklist for hospitals preparing for the reception and care of coronavirus 2019 (COVID-19) patients. ECDC also publishes regular risk assessments and the Box below contains the summary from the fifth update published on 2 March 2020.%U https://eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.9.2003051</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>A Chimeric Virus-Mouse Model System for Evaluating the Function and Inhibition of Papain-Like Proteases of Emerging Coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">To combat emerging coronaviruses, developing safe and efficient platforms to evaluate viral protease activities and the efficacy of protease inhibitors is a high priority. Here, we exploit a biosafety level 2 (BSL-2) chimeric Sindbis virus system to evaluate protease activities and the efficacy of inhibitors directed against the papain-like protease (PLpro) of severe acute respiratory syndrome coronavirus (SARS-CoV), a biosafety level 3 (BSL-3) pathogen. We engineered Sindbis virus to coexpress PLpro and a substrate, murine interferon-stimulated gene 15 (ISG15), and found that PLpro mediates removal of ISG15 (deISGylation) from cellular proteins. Mutation of the catalytic cysteine residue of PLpro or addition of a PLpro inhibitor blocked deISGylation in virus-infected cells. Thus, deISGylation is a marker of PLpro activity. Infection of alpha/beta interferon receptor knockout (IFNAR(−/−)) mice with these chimeric viruses revealed that PLpro deISGylation activity removed ISG15-mediated protection during viral infection. Importantly, administration of a PLpro inhibitor protected these mice from lethal infection, demonstrating the efficacy of a coronavirus protease inhibitor in a mouse model. However, this PLpro inhibitor was not sufficient to protect the mice from lethal infection with SARS-CoV MA15, suggesting that further optimization of the delivery and stability of PLpro inhibitors is needed. We extended the chimeric-virus platform to evaluate the papain-like protease/deISGylating activity of Middle East respiratory syndrome coronavirus (MERS-CoV) to provide a small-animal model to evaluate PLpro inhibitors of this recently emerged pathogen. This platform has the potential to be universally adaptable to other viral and cellular enzymes that have deISGylating activities. IMPORTANCE Evaluating viral protease inhibitors in a small-animal model is a critical step in the path toward antiviral drug development. We modified a biosafety level 2 chimeric virus system to facilitate evaluation of inhibitors directed against highly pathogenic coronaviruses. We used this system to demonstrate the in vivo efficacy of an inhibitor of the papain-like protease of severe acute respiratory syndrome coronavirus. Furthermore, we demonstrate that the chimeric-virus system can be adapted to study the proteases of emerging human pathogens, such as Middle East respiratory syndrome coronavirus. This system provides an important tool to rapidly assess the efficacy of protease inhibitors targeting existing and emerging human pathogens, as well as other enzymes capable of removing ISG15 from cellular proteins.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>ProMED-mail: 22 years of digital surveillance of emerging infectious diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">ProMED-mail (ProMED) was launched in 1994 as an email service to identify unusual health events related to emerging and re-emerging infectious diseases and toxins affecting humans, animals and plants. It is used daily by public health leaders, government officials at all levels, physicians, veterinarians and other healthcare workers, researchers, private companies, journalists and the general public. Reports are produced and commentary provided by a global team of subject matter experts in a variety of fields including virology, parasitology, epidemiology, entomology, veterinary and plant disease specialists. ProMED operates 24 hours a day, 7 days a week and has over 83 000 subscribers, representing every country in the world. Additionally, ProMED disseminates information via its website and through social media channels such as Twitter and Facebook as well as through RSS feeds. Over the last 22 years, it has been the first to report on numerous major and minor disease outbreaks including SARS, MERS, Ebola and the early spread of Zika. ProMED is transparent, apolitical, open to all and free of charge, making it an important and longstanding contributor to global health surveillance.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic and Clinical Characteristics of 91 Hospitalized Patients with COVID-19 in Zhejiang, China: A retrospective, multi-centre case series</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background Recent studies have focused initial clinical and Epidemiologic characteristics on the COVID-19, mainly revealing situation in Wuhan, Hubei.  Aim To reveal more data on the epidemiologic and clinical characteristics of COVID-19 patients outside of Wuhan, in Zhejiang, China.  Design  Retrospective case series.  Methods 88 cases of laboratory-confirmed and 3 cases of clinical-confirmed COVID-19 were admitted to five hospitals in Zhejiang province, China. Data were collected from 20 January 2020 to 11 February 2020.  Results  Of all 91 patients, 88 (96.70%) were laboratory-confirmed COVID-19 with throat swab samples that tested positive for SARS-Cov-2 while 3 (3.30%) were clinical-diagnosed COVID-19 cases. The median age of the patients was 50 (36.5-57) years, and female accounted for 59.34%. In this sample 40 (43.96%) patients had contracted the diseases from local cases, 31 (34.07%) patients had been to Wuhan/Hubei, 8 (8.79%) cases had contacted with people from Wuhan, 11 (12.09%) cases were confirmed aircraft transmission. In particular within the city of Ningbo, 60.52% cases can be traced back to an event held in a temple. The most common symptoms were fever (71.43%), cough (60.44%) and fatigue (43.96%). The median of incubation period was 6 (IQR, 3-8) days and the median time from first visit to a doctor to confirmed diagnosis was 1 (1-2) days. According to the Chest computed tomography scans, 67.03% cases had bilateral pneumonia.  Conclusions Social activity cluster, family cluster and travel by airplane were how COVID-19 patients get transmitted and could be rapidly diagnosed COVID-19 in Zhejiang.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved Detection of Respiratory Pathogens by Use of High-Quality Sputum with TaqMan Array Card Technology</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">New diagnostic platforms often use nasopharyngeal or oropharyngeal (NP/OP) swabs for pathogen detection for patients hospitalized with community-acquired pneumonia (CAP). We applied multipathogen testing to high-quality sputum specimens to determine if more pathogens can be identified relative to NP/OP swabs. Children (<18 years old) and adults hospitalized with CAP were enrolled over 2.5 years through the Etiology of Pneumonia in the Community (EPIC) study. NP/OP specimens with matching high-quality sputum (defined as ≤10 epithelial cells/low-power field [lpf] and ≥25 white blood cells/lpf or a quality score [q-score] definition of 2+) were tested by TaqMan array card (TAC), a multipathogen real-time PCR detection platform. Among 236 patients with matched specimens, a higher proportion of sputum specimens had ≥1 pathogen detected compared with NP/OP specimens in children (93% versus 68%; P < 0.0001) and adults (88% versus 61%; P < 0.0001); for each pathogen targeted, crossing threshold (C(T)) values were earlier in sputum. Both bacterial (361 versus 294) and viral detections (245 versus 140) were more common in sputum versus NP/OP specimens, respectively, in both children and adults. When available, high-quality sputum may be useful for testing in hospitalized CAP patients.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Differentiating Microbial Forensic qPCR Target and Control Products by Electrospray Ionization Mass Spectrometry</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Molecular bioforensic research is dependent on rapid and sensitive methods such as real-time PCR (qPCR) for the identification of microorganisms. The use of synthetic positive control templates containing small modifications outside the primer and probe regions is essential to ensure all aspects of the assay are functioning properly, including the primers and probes. However, a typical qPCR or reverse transcriptase qPCR (qRT-PCR) assay is limited in differentiating products generated from positive controls and biological samples because the fluorescent probe signals generated from each type of amplicon are indistinguishable. Additional methods used to differentiate amplicons, including melt curves, secondary probes, and amplicon sequencing, require significant time to implement and validate and present technical challenges that limit their use for microbial forensic applications. To solve this problem, we have developed a novel application of electrospray ionization mass spectrometry (ESI-MS) to rapidly differentiate qPCR amplicons generated with positive biological samples from those generated with synthetic positive controls. The method has sensitivity equivalent to qPCR and supports the confident and timely determination of the presence of a biothreat agent that is crucial for policymakers and law enforcement. Additionally, it eliminates the need for time-consuming methods to confirm qPCR results, including development and validation of secondary probes or sequencing of small amplicons. In this study, we demonstrate the effectiveness of this approach with microbial forensic qPCR assays targeting multiple biodefense agents (bacterial, viral, and toxin) for the ability to rapidly discriminate between a positive control and a positive sample.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Strengthening laboratory capacity for detection of respiratory viral pathogens through the Global Health Security Agenda (GHSA) framework</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Endemic and emerging respiratory viruses are a threat to public health, and a robust public health laboratory system is essential to ensure global health security. OBJECTIVE: This program sought to expand molecular laboratory testing capacity to detect a broad range of respiratory pathogens in clinical respiratory specimens collected during disease surveillance and outbreak investigations. METHODS: As a part of the Global Health Security Agenda (GHSA), the United States Centers for Disease Control and Prevention utilised the equipment and training infrastructure already in place at the World Health Organization National Influenza Centers to expand testing capacity for respiratory viruses in laboratories in GHSA partner countries. This was done through the provision of quality assured reagents, including multiplex platforms and technical guidance for laboratory staff, as well as the assessment of laboratory testing accuracy. CONCLUSION: Early findings illustrated that GHSA laboratories have been able to expand testing capacity using specimens from routine surveillance, as well as from outbreak situations.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Surveillance of Acute Respiratory Infections Using Community-Submitted Symptoms and Specimens for Molecular Diagnostic Testing</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 May 27</span></i><p style="text-align: justify; text-justify: inter-word;">Participatory systems for surveillance of acute respiratory infection give real-time information about infections circulating in the community, yet to-date are limited to self-reported syndromic information only and lacking methods of linking symptom reports to infection types. We developed the GoViral platform to evaluate whether a cohort of lay volunteers could, and would find it useful to, contribute self-reported symptoms online and to compare specimen types for self-collected diagnostic information of sufficient quality for respiratory infection surveillance. Volunteers were recruited, given a kit (collection materials and customized instructions), instructed to report their symptoms weekly, and when sick with cold or flu-like symptoms, requested to collect specimens (saliva and nasal swab). We compared specimen types for respiratory virus detection sensitivity (via polymerase-chain-reaction) and ease of collection. Participants were surveyed to determine receptivity to participating when sick, to receiving information on the type of pathogen causing their infection and types circulating near them. Between December 1 2013 and March 1 2014, 295 participants enrolled in the study and received a kit. Of those who reported symptoms, half (71) collected and sent specimens for analysis. Participants submitted kits on average 2.30 days (95 CI: 1.65 to 2.96) after symptoms began. We found good concordance between nasal and saliva specimens for multiple pathogens, with few discrepancies. Individuals report that saliva collection is easiest and report that receiving information about what pathogen they, and those near them, have is valued and can shape public health behaviors. Community-submitted specimens can be used for the detection of acute respiratory infection with individuals showing receptivity for participating and interest in a real-time picture of respiratory pathogens near them.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>[The importance of strengthening the ability of fundamental disease prevention and control system from the perspective of the epidemic situation of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 has been in epidemic for nearly two months. The prevention and control measures have achieved remarkable results. From the response and disposal process of this epidemic, it is exposed that fundamental disease prevention and control system are insufficient in human resources and ability of laboratory testing. It is suggested that the disease control institutions should strengthen the construction of these aspects in future.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Top stories: Bungled coronavirus testing, disarming ‘atomic bomb’ cells, and jet stream blocking</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? One answer is that a test kit designed by the U.S. Centers for Disease Control and Prevention (CDC) and rolled out to state and local labs late last month contained a faulty reagent. The problems have led many to doubt the accuracy of confirmed tallies of the disease. But the situation may soon improve, as the Food and Drug Administration comes up with a workaround for the faulty CDC kit.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Déjà Vu or Jamais Vu? How the Severe Acute Respiratory Syndrome Experience Influenced a Singapore Radiology Department's Response to the Coronavirus Disease (COVID-19) Epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE. This article shares the ground operational perspective of how a tertiary hospital radiology department in Singapore is responding to the coronavirus disease (COVID-19) epidemic. This same department was also deeply impacted by the severe acute respiratory syndrome (SARS) outbreak in 2003. CONCLUSION. Though similar to SARS, the COVID-19 outbreak has several differences. We share how lessons from 2003 are applied and modified in our ongoing operational response to this evolving novel pathogen.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>New frontiers in applied veterinary point‐of‐capture diagnostics: Toward early detection and control of zoonotic influenza</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 23</span></i><p style="text-align: justify; text-justify: inter-word;">Among the chief limitations in achieving early detection and control of animal‐origin influenza of pandemic potential in high‐risk livestock populations is the existing lag time between sample collection and diagnostic result. Advances in molecular diagnostics are permitting deployment of affordable, rapid, highly sensitive, and specific point‐of‐capture assays, providing opportunities for targeted surveillance driving containment strategies with potentially compelling returns on investment. Interrupting disease transmission at source holds promise of disrupting cycles of animal‐origin influenza incursion to endemicity and limiting impact on animal production, food security, and public health. Adoption of new point‐of‐capture diagnostics should be undertaken in the context of promoting robust veterinary services systems and parallel support for operationalizing pre‐authorized plans and communication strategies that will ensure that the full potential of these new platforms is realized.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Screening for Entry Inhibitors of Highly Pathogenic Viruses under Low-Level Biocontainment</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging viruses including Nipah, Hendra, Lujo, and Junin viruses have enormous potential to spread rapidly. Nipah virus, after emerging as a zoonosis, has also evolved the capacity for human-to-human transmission. Most of the diseases caused by these pathogens are untreatable and require high biocontainment conditions. Universal methods for rapidly identifying and screening candidate antivirals are urgently needed. We have developed a modular antiviral platform strategy that relies on simple bioinformatic and genetic information about each pathogen. Central to this platform is the use of envelope glycoprotein cDNAs to establish multi-cycle replication systems under BSL2 conditions for viral pathogens that normally require BSL3 and BSL4 facilities. We generated monoclonal antibodies against Nipah G by cDNA immunization in rats, and we showed that these antibodies neutralize both Nipah and Hendra live viruses. We then used these effective Henipavirus inhibitors to validate our screening strategy. Our proposed strategy should contribute to the response capability for emerging infectious diseases, providing a way to initiate antiviral development immediately upon identifying novel viruses.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>An original design of remote robot-assisted intubation system</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Sep 7</span></i><p style="text-align: justify; text-justify: inter-word;">The success rate of pre-hospital endotracheal intubation (ETI) by paramedics is lower than physicians. We aimed to establish a remote robot-assisted intubation system (RRAIS) and expected it to improve success rate of pre-hospital ETI. To test the robot’s feasibility, 20 pigs were intubated by direct laryngoscope or the robot system. Intubation time, success rate, airway complications were recorded during the experiment. The animal experiment showed that participants achieved a higher success rate in absolute numbers by the robot system. In summary, we have successfully developed a remote robot-assisted intubation system. It is promising for RRAIS to improve the success rate of pre-hospital ETI and change the current rescue model.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Generation of antibodies against COVID-19 virus for development of diagnostic tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Miniaturized PCR chips for nucleic acid amplification and analysis: latest advances and future trends</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">The possibility of performing fast and small-volume nucleic acid amplification and analysis on a single chip has attracted great interest. Devices based on this idea, referred to as micro total analysis, microfluidic analysis, or simply ‘Lab on a chip’ systems, have witnessed steady advances over the last several years. Here, we summarize recent research on chip substrates, surface treatments, PCR reaction volume and speed, architecture, approaches to eliminating cross-contamination and control and measurement of temperature and liquid flow. We also discuss product-detection methods, integration of functional components, biological samples used in PCR chips, potential applications and other practical issues related to implementation of lab-on-a-chip technologies.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /></body>