
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: coupling genomics and COVID-19 diagnostic testing on a large scale</h2><br /></div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 30</span></i><p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Can an Infectious Disease Genomics Project Predict and Prevent the Next Pandemic?</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Oct 26</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases need a globally coordinated genomic-based movement linking sequencing efforts to development of response tools to mitigate the impact of existing and emerging threats.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Microbial sequencing to improve individual and population health</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 19</span></i><p style="text-align: justify; text-justify: inter-word;">Recent advances in sequencing technologies are changing the face of infectious disease investigation and control. Personalized anti-infective therapies and surveillance of emergent pathogen outbreaks are just two examples of the potential benefits of merging the fields of genomics and infectious diseases.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic data and pathogen genome sequences: a powerful synergy for public health</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 18</span></i><p style="text-align: justify; text-justify: inter-word;">Epidemiologists aim to inform the design of public health interventions with evidence on the evolution, emergence and spread of infectious diseases. Sequencing of pathogen genomes, together with date, location, clinical manifestation and other relevant data about sample origins, can contribute to describing nearly every aspect of transmission dynamics, including local transmission and global spread. The analyses of these data have implications for all levels of clinical and public health practice, from institutional infection control to policies for surveillance, prevention and treatment. This review highlights the range of epidemiological questions that can be addressed from the combination of genome sequence and traditional ‘line lists’ (tables of epidemiological data where each line includes demographic and clinical features of infected individuals). We identify opportunities for these data to inform interventions that reduce disease incidence and prevalence. By considering current limitations of, and challenges to, interpreting these data, we aim to outline a research agenda to accelerate the genomics-driven transformation in public health microbiology.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Analyses of Merging Clinical and Viral Genetic Data for Influenza Surveillance</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The annual influenza vaccine is one of the most common public health interventions and is universally recommended for all individuals older than six months. Vaccine composition depends on viruses circulating over the past flu season and are estimated to be the most prevalent and representative strains in the current season. Here, we use clinical data outfitted with viral genetics to characterize confirmed influenza cases from the past two flu seasons and genetically compare them to the strains that they were vaccinated against that year. We show that case similarities to vaccine strains differ by geographic region and that the vaccines appear to have different levels of effectiveness by region. This study demonstrates the value of merging viral genetics with clinical data. Further research is needed to formally evaluate whether this improves biosurveillance efforts and enhances efficacy of influenza vaccines.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Building International Genomics Collaboration for Global Health Security</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 7</span></i><p style="text-align: justify; text-justify: inter-word;">Genome science and technologies are transforming life sciences globally in many ways and becoming a highly desirable area for international collaboration to strengthen global health. The Genome Science Program at the Los Alamos National Laboratory is leveraging a long history of expertise in genomics research to assist multiple partner nations in advancing their genomics and bioinformatics capabilities. The capability development objectives focus on providing a molecular genomics-based scientific approach for pathogen detection, characterization, and biosurveillance applications. The general approaches include introduction of basic principles in genomics technologies, training on laboratory methodologies and bioinformatic analysis of resulting data, procurement, and installation of next-generation sequencing instruments, establishing bioinformatics software capabilities, and exploring collaborative applications of the genomics capabilities in public health. Genome centers have been established with public health and research institutions in the Republic of Georgia, Kingdom of Jordan, Uganda, and Gabon; broader collaborations in genomics applications have also been developed with research institutions in many other countries.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP).  We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS.  We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene.  Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Genomics in the Identification, Prediction, and Prevention of Biological Threats</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Oct 26</span></i><p style="text-align: justify; text-justify: inter-word;">In all likelihood, it is only a matter of time before our public health system will face a major biological threat, whether intentionally dispersed or originating from a known or newly emerging infectious disease. It is necessary not only to increase our reactive “biodefense,” but also to be proactive and increase our preparedness. To achieve this goal, it is essential that the scientific and public health communities fully embrace the genomic revolution, and that novel bioinformatic and computing tools necessary to make great strides in our understanding of these novel and emerging threats be developed. Genomics has graduated from a specialized field of science to a research tool that soon will be routine in research laboratories and clinical settings. Because the technology is becoming more affordable, genomics can and should be used proactively to build our preparedness and responsiveness to biological threats. All pieces, including major continued funding, advances in next-generation sequencing technologies, bioinformatics infrastructures, and open access to data and metadata, are being set in place for genomics to play a central role in our public health system.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19's graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control.  Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation.  Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.</jats:p></p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Top stories: Bungled coronavirus testing, disarming ‘atomic bomb’ cells, and jet stream blocking</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? One answer is that a test kit designed by the U.S. Centers for Disease Control and Prevention (CDC) and rolled out to state and local labs late last month contained a faulty reagent. The problems have led many to doubt the accuracy of confirmed tallies of the disease. But the situation may soon improve, as the Food and Drug Administration comes up with a workaround for the faulty CDC kit.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>To Test Or Not To Test?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The use of genetic tests to improve diagnosis and to screen out potential non-responders to costly biologic therapies would seem like a prudent investment, but the practice is far from mainstream. To payers, the issue is more than black and white.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrating Genome-based Informatics to Modernize Global Disease Monitoring, Information Sharing, and Response</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid advancement of genome technologies holds great promise for improving the quality and speed of clinical and public health laboratory investigations and for decreasing their cost. The latest generation of genome DNA sequencers can provide highly detailed and robust information on disease-causing microbes, and in the near future these technologies will be suitable for routine use in national, regional, and global public health laboratories. With additional improvements in instrumentation, these next- or third-generation sequencers are likely to replace conventional culture-based and molecular typing methods to provide point-of-care clinical diagnosis and other essential information for quicker and better treatment of patients. Provided there is free-sharing of information by all clinical and public health laboratories, these genomic tools could spawn a global system of linked databases of pathogen genomes that would ensure more efficient detection, prevention, and control of endemic, emerging, and other infectious disease outbreaks worldwide.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time digital pathogen surveillance — the time is now</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul 30</span></i><p style="text-align: justify; text-justify: inter-word;">It is time to shake up public health surveillance. New technologies for sequencing, aided by friction-free approaches to data sharing, could have an impact on public health efforts.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic analysis of respiratory syncytial virus infections in households and utility in inferring who infects the infant</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 11</span></i><p style="text-align: justify; text-justify: inter-word;">Infants (under 1-year-old) are at most risk of life threatening respiratory syncytial virus (RSV) disease. RSV epidemiological data alone has been insufficient in defining who acquires infection from whom (WAIFW) within households. We investigated RSV genomic variation within and between infected individuals and assessed its potential utility in tracking transmission in households. Over an entire single RSV season in coastal Kenya, nasal swabs were collected from members of 20 households every 3–4 days regardless of symptom status and screened for RSV nucleic acid. Next generation sequencing was used to generate >90% RSV full-length genomes for 51.1% of positive samples (191/374). Single nucleotide polymorphisms (SNPs) observed during household infection outbreaks ranged from 0–21 (median: 3) while SNPs observed during single-host infection episodes ranged from 0–17 (median: 1). Using the viral genomic data alone there was insufficient resolution to fully reconstruct within-household transmission chains. For households with clear index cases, the most likely source of infant infection was via a toddler (aged 1 to <3 years-old) or school-aged (aged 6 to <12 years-old) co-occupant. However, for best resolution of WAIFW within households, we suggest an integrated analysis of RSV genomic and epidemiological data.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic analysis of respiratory syncytial virus infections in households and utility in inferring who infects the infant</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 11</span></i><p style="text-align: justify; text-justify: inter-word;">Infants (under 1-year-old) are at most risk of life threatening respiratory syncytial virus (RSV) disease. RSV epidemiological data alone has been insufficient in defining who acquires infection from whom (WAIFW) within households. We investigated RSV genomic variation within and between infected individuals and assessed its potential utility in tracking transmission in households. Over an entire single RSV season in coastal Kenya, nasal swabs were collected from members of 20 households every 3–4 days regardless of symptom status and screened for RSV nucleic acid. Next generation sequencing was used to generate >90% RSV full-length genomes for 51.1% of positive samples (191/374). Single nucleotide polymorphisms (SNPs) observed during household infection outbreaks ranged from 0–21 (median: 3) while SNPs observed during single-host infection episodes ranged from 0–17 (median: 1). Using the viral genomic data alone there was insufficient resolution to fully reconstruct within-household transmission chains. For households with clear index cases, the most likely source of infant infection was via a toddler (aged 1 to <3 years-old) or school-aged (aged 6 to <12 years-old) co-occupant. However, for best resolution of WAIFW within households, we suggest an integrated analysis of RSV genomic and epidemiological data.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Preprints bring ‘firehose’ of outbreak data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn't even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Precision Surveillance for Viral Respiratory Pathogens: Virome Capture Sequencing for the Detection and Genomic Characterization of Severe Acute Respiratory Infection in Uganda</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Precision public health is a novel set of methods to target disease prevention and mitigation interventions to high-risk subpopulations. We applied a precision public health strategy to syndromic surveillance for severe acute respiratory infection (SARI) in Uganda by combining spatiotemporal analytics with genomic sequencing to detect and characterize viral respiratory pathogens with epidemic potential. METHODS: Using a national surveillance network we identified patients with unexplained, influenza-negative SARI from 2010 to 2015. Spatiotemporal analyses were performed retrospectively to identify clusters of unexplained SARI. Within clusters, respiratory viruses were detected and characterized in naso- and oropharyngeal swab samples using a novel oligonucleotide probe capture (VirCapSeq-VERT) and high-throughput sequencing platform. Linkage to conventional epidemiologic strategies further characterized transmission dynamics of identified pathogens. RESULTS: Among 2901 unexplained SARI cases, 9 clusters were detected, accounting for 301 (10.4%) cases. Clusters were more likely to occur in urban areas and during biannual rainy seasons. Within detected clusters, we identified an unrecognized outbreak of measles-associated SARI; sequence analysis implicated cocirculation of endemic genotype B3 and genotype D4 likely imported from England. We also detected a likely nosocomial SARI cluster associated with a novel picobirnavirus most closely related to swine and dromedary viruses. CONCLUSIONS: Using a precision approach to public health surveillance, we detected and characterized the genomics of vaccine-preventable and zoonotic respiratory viruses associated with clusters of severe respiratory infections in Uganda. Future studies are needed to assess the feasibility, scalability, and impact of applying similar approaches during real-time public health surveillance in low-income settings.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearmans rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>A rigorous approach to facilitate and guarantee the correctness of the genetic testing management in human genome information systems</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Dec 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Recent medical and biological technology advances have stimulated the development of new testing systems that have been providing huge, varied amounts of molecular and clinical data. Growing data volumes pose significant challenges for information processing systems in research centers. Additionally, the routines of genomics laboratory are typically characterized by high parallelism in testing and constant procedure changes. RESULTS: This paper describes a formal approach to address this challenge through the implementation of a genetic testing management system applied to human genome laboratory. We introduced the Human Genome Research Center Information System (CEGH) in Brazil, a system that is able to support constant changes in human genome testing and can provide patients updated results based on the most recent and validated genetic knowledge. Our approach uses a common repository for process planning to ensure reusability, specification, instantiation, monitoring, and execution of processes, which are defined using a relational database and rigorous control flow specifications based on process algebra (ACP). The main difference between our approach and related works is that we were able to join two important aspects: 1) process scalability achieved through relational database implementation, and 2) correctness of processes using process algebra. Furthermore, the software allows end users to define genetic testing without requiring any knowledge about business process notation or process algebra. CONCLUSIONS: This paper presents the CEGH information system that is a Laboratory Information Management System (LIMS) based on a formal framework to support genetic testing management for Mendelian disorder studies. We have proved the feasibility and showed usability benefits of a rigorous approach that is able to specify, validate, and perform genetic testing using easy end user interfaces.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Human genomics and preparedness for infectious threats</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Dec 29</span></i><p style="text-align: justify; text-justify: inter-word;">Public health preparedness requires effective surveillance of and rapid response to infectious disease outbreaks. Inclusion of research activities within the outbreak setting provides important opportunities to maximize limited resources, to enhance gains in scientific knowledge, and ultimately to increase levels of preparedness. With rapid advances in laboratory technologies, banking and analysis of human genomic specimens can be conducted as part of public health investigations, enabling valuable research well into the future.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>The Era of Genomic Epidemiology</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recent revolution in genomics is already having a profound impact on the practice of epidemiology. The purpose of this commentary is to demonstrate how genomics and epidemiology will continue to rely heavily on each other, now and in the future, by illustrating a number of interaction points between these 2 disciplines: (1) the use of genomics to estimate disease heritability; (2) the impact of genomics on analytical study design; (3) how genome-wide data can be employed to effectively overcome residual population stratification arising from selection bias; (4) the importance of genomics as a tool in epidemiological investigation; (5) the importance of epidemiology in the collection of adequately phenotyped samples for genomics studies, and (6) for unraveling the clinical and therapeutic relevance of genetic variants once they are discovered.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Use of linked electronic health records to assess mortality and length of stay associated with pandemic influenza A(H1N1)pdm09 at a UK teaching hospital</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Apr 14</span></i><p style="text-align: justify; text-justify: inter-word;">Effective use of data linkage is becoming an increasingly important focus in the new healthcare system in England. We linked data from the results of a multiplex PCR assay for respiratory viruses for a population of 230 inpatients at a UK teaching hospital with their patient administrative system records in order to compare the mortality and length of stay of patients who tested positive for influenza A(H1N1)pdm09 with those positive for another influenza A virus. The results indicated a reduced risk of death among influenza A(H1N1)pdm09 patients compared to other influenza A strains, with an adjusted risk ratio of 0·25 (95% confidence interval 0·08–0·75, P = 0·01), while no significant differences were found between the lengths of stay in the hospital for these two groups. Further development of such methods to link hospital data in a routine fashion could provide a rapid means of gaining epidemiological insights into emerging infectious diseases.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrating Advanced Molecular Technologies into Public Health</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Advances in laboratory and information technologies are transforming public health microbiology. High-throughput genome sequencing and bioinformatics are enhancing our ability to investigate and control outbreaks, detect emerging infectious diseases, develop vaccines, and combat antimicrobial resistance, all with increased accuracy, timeliness, and efficiency. The Advanced Molecular Detection (AMD) initiative has allowed the Centers for Disease Control and Prevention (CDC) to provide leadership and coordination in integrating new technologies into routine practice throughout the U.S. public health laboratory system. Collaboration and partnerships are the key to navigating this transition and to leveraging the next generation of methods and tools most effectively for public health.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomics and infectious disease: a call to identify the ethical, legal and social implications for public health and clinical practice</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 18</span></i><p style="text-align: justify; text-justify: inter-word;">Advances in genomics are contributing to the development of more effective, personalized approaches to the prevention and treatment of infectious diseases. Genetic sequencing technologies are furthering our understanding of how human and pathogen genomic factors - and their interactions - contribute to individual differences in immunologic responses to vaccines, infections and drug therapies. Such understanding will influence future policies and procedures for infectious disease management. With the potential for tailored interventions for particular individuals, populations or subpopulations, ethical, legal and social implications (ELSIs) may arise for public health and clinical practice. Potential considerations include balancing health-related benefits and harms between individuals and the larger community, minimizing threats to individual privacy and autonomy, and ensuring just distribution of scarce resources. In this Opinion, we consider the potential application of pathogen and host genomic information to particular viral infections that have large-scale public health consequences but differ in ELSI-relevant characteristics such as ease of transmission, chronicity, severity, preventability and treatability. We argue for the importance of anticipating these ELSI issues in advance of new scientific discoveries, and call for the development of strategies for identifying and exploring ethical questions that should be considered as clinical, public health and policy decisions are made.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Pathogen surveillance in the informal settlement, Kibera, Kenya, using a metagenomics approach</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Oct 10</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Worldwide, the number of emerging and re-emerging infectious diseases is increasing, highlighting the importance of global disease pathogen surveillance. Traditional population-based methods may fail to capture important events, particularly in settings with limited access to health care, such as urban informal settlements. In such environments, a mixture of surface water runoff and human feces containing pathogenic microorganisms could be used as a surveillance surrogate. METHOD: We conducted a temporal metagenomic analysis of urban sewage from Kibera, an urban informal settlement in Nairobi, Kenya, to detect and quantify bacterial and associated antimicrobial resistance (AMR) determinants, viral and parasitic pathogens. Data were examined in conjunction with data from ongoing clinical infectious disease surveillance. RESULTS: A large variation of read abundances related to bacteria, viruses, and parasites of medical importance, as well as bacterial associated antimicrobial resistance genes over time were detected. Significant increased abundances were observed for a number of bacterial pathogens coinciding with higher abundances of AMR genes. Vibrio cholerae as well as rotavirus A, among other virus peaked in several weeks during the study period whereas Cryptosporidium spp. and Giardia spp, varied more over time. CONCLUSION: The metagenomic surveillance approach for monitoring circulating pathogens in sewage was able to detect putative pathogen and resistance loads in an urban informal settlement. Thus, valuable if generated in real time to serve as a comprehensive infectious disease agent surveillance system with the potential to guide disease prevention and treatment. The approach may lead to a paradigm shift in conducting real-time global genomics-based surveillance in settings with limited access to health care.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.</jats:p></p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>From Theory to Practice: Translating Whole-Genome Sequencing (WGS) into the Clinic</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">Hospitals worldwide are facing an increasing incidence of hard-to-treat infections. Limiting infections and providing patients with optimal drug regimens require timely strain identification as well as virulence and drug-resistance profiling. Additionally, prophylactic interventions based on the identification of environmental sources of recurrent infections (e.g., contaminated sinks) and reconstruction of transmission chains (i.e., who infected whom) could help to reduce the incidence of nosocomial infections. WGS could hold the key to solving these issues. However, uptake in the clinic has been slow. Some major scientific and logistical challenges need to be solved before WGS fulfils its potential in clinical microbial diagnostics. In this review we identify major bottlenecks that need to be resolved for WGS to routinely inform clinical intervention and discuss possible solutions.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>New frontiers in applied veterinary point‐of‐capture diagnostics: Toward early detection and control of zoonotic influenza</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 23</span></i><p style="text-align: justify; text-justify: inter-word;">Among the chief limitations in achieving early detection and control of animal‐origin influenza of pandemic potential in high‐risk livestock populations is the existing lag time between sample collection and diagnostic result. Advances in molecular diagnostics are permitting deployment of affordable, rapid, highly sensitive, and specific point‐of‐capture assays, providing opportunities for targeted surveillance driving containment strategies with potentially compelling returns on investment. Interrupting disease transmission at source holds promise of disrupting cycles of animal‐origin influenza incursion to endemicity and limiting impact on animal production, food security, and public health. Adoption of new point‐of‐capture diagnostics should be undertaken in the context of promoting robust veterinary services systems and parallel support for operationalizing pre‐authorized plans and communication strategies that will ensure that the full potential of these new platforms is realized.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Networking of Public Health Microbiology Laboratories Bolsters Europe’s Defenses against Infectious Diseases</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 26</span></i><p style="text-align: justify; text-justify: inter-word;">In an era of global health threats caused by epidemics of infectious diseases and rising multidrug resistance, microbiology laboratories provide essential scientific evidence for risk assessment, prevention, and control. Microbiology has been at the core of European infectious disease surveillance networks for decades. Since 2010, these networks have been coordinated by the European Centre for Disease Prevention and Control (ECDC). Activities delivered in these networks include harmonization of laboratory diagnostic, antimicrobial susceptibility and molecular typing methods, multicentre method validation, technical capacity mapping, training of laboratory staff, and continuing quality assessment of laboratory testing. Cooperation among the European laboratory networks in the past 7 years has proved successful in strengthening epidemic preparedness by enabling adaptive capabilities for rapid detection of emerging pathogens across Europe. In partnership with food safety authorities, international public health agencies and learned societies, ECDC-supported laboratory networks have also progressed harmonization of routinely used antimicrobial susceptibility and molecular typing methods, thereby significantly advancing the quality, comparability and precision of microbiological information gathered by ECDC for surveillance for zoonotic diseases and multidrug-resistant pathogens in Europe. ECDC continues to act as a catalyst for sustaining continuous practice improvements and strengthening wider access to laboratory capacity across the European Union. Key priorities include optimization and broader use of rapid diagnostics, further integration of whole-genome sequencing in surveillance and electronic linkage of laboratory and public health systems. This article highlights some of the network contributions to public health in Europe and the role that ECDC plays managing these networks.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomics and High-Consequence Infectious Diseases: A Scoping Review of Emerging Science and Potential Ethical Issues</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Host genomic research on high-consequence infectious diseases is a growing area, but the ethical, legal, and social implications of such findings related to potential applications of the research have not yet been identified. While there is a robust ethical debate about the ethical, legal, and social implications of research during an emergency, there has been less consideration of issues facing research conducted outside of the scope of emergency response. Addressing the implications of research at an early stage (anticipatory ethics) helps define the issue space, facilitates preparedness, and promotes ethically and socially responsible practices. To lay the groundwork for more comprehensive anticipatory ethics work, this article provides a preliminary assessment of the state of the field with a scoping review of host genomic research on a subset of high-consequence infectious diseases of relevance to high-level isolation units, focusing on its ethically relevant features and identifying several ethical, legal, and social implications raised by the literature. We discuss the challenges of genomic studies of low-frequency, high-risk events and applications of the science, including identifying targets to guide the development of new therapeutics, improving vaccine development, finding biomarkers to predict disease outcome, and guiding decisions about repurposing existing drugs and genetic screening. Some ethical, legal, and social implications identified in the literature included the rise of systems biology and paradigm shifts in medical countermeasure development; controversies over repurposing of existing drugs; genetic privacy and discrimination; and benefit-sharing and global inequity as part of the broader ecosystem surrounding high-level isolation units. Future anticipatory ethics work should forecast the science and its applications; identify a more comprehensive list of ethical, legal, and social implications; and facilitate evaluation by multiple stakeholders to inform the integration of ethical concerns into high-level isolation unit policy and practice.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodias first case of COVID-2019.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease. AVAILABILITY: https://www.genomedetective.com/app/typingtool/cov. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>The ongoing crises in China illustrate that the assessment of epidemics in isolation is no longer sufficient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary The interplay of simultaneous COVID-19, African swine fever, and avian influenza emergencies on global health and industries is constantly evolving and difficult to predict, and therefore warrants further scrutiny. The need for a health network of global scope for the rapid and open exchange of information needs to be strengthened in order to address ongoing and future epidemics under competing resources.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Are high-performing health systems resilient against the COVID-19 epidemic?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As of March 5, 2020, there has been sustained local transmission of coronavirus disease 2019 (COVID-19) in Hong Kong, Singapore, and Japan.1 Containment strategies seem to have prevented smaller transmission chains from amplifying into widespread community transmission. The health systems in these locations have generally been able to adapt,2 , 3 but their resilience could be affected if the COVID-19 epidemic continues for many more months and increasing numbers of people require services. We outline some of the core dimensions of these resilient health systems4 and their responses to the COVID-19 epidemic. First, after variable periods of adaptation, the three locations took actions to manage the outbreak of a new pathogen. Surveillance systems were readjusted to identify potential cases while public health staff identified their contacts. National laboratory networks developed diagnostic tests once the COVID-19 genetic sequences were published5 and laboratory testing capacity was increased in all three locations, although expansion of the diagnostic capacity to university and large private laboratories in Japan is still ongoing. In Hong Kong, initially, only pneumonia patients without a microbiological diagnosis were tested, but surveillance has been broadened to include all inpatients with pneumonia and a purposively sampled proportion of outpatients and emergency attendees totalling about 1500 per day (Leung GM, unpublished). Japan's testing strategy has also evolved with diagnostic tests now offered to all suspected cases irrespective of their travel history; however, there are reports of cases that should have been tested but were not.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Cyberbiosecurity Challenges of Pathogen Genome Databases</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 15</span></i><p style="text-align: justify; text-justify: inter-word;">Pathogen detection, identification, and tracking is shifting from non-molecular methods, DNA fingerprinting methods, and single gene methods to methods relying on whole genomes. Viral Ebola and influenza genome data are being used for real-time tracking, while food-borne bacterial pathogen outbreaks and hospital outbreaks are investigated using whole genomes in the UK, Canada, the USA and the other countries. Also, plant pathogen genomes are starting to be used to investigate plant disease epidemics such as the wheat blast outbreak in Bangladesh. While these genome-based approaches provide never-seen advantages over all previous approaches with regard to public health and biosecurity, they also come with new vulnerabilities and risks with regard to cybersecurity. The more we rely on genome databases, the more likely these databases will become targets for cyber-attacks to interfere with public health and biosecurity systems by compromising their integrity, taking them hostage, or manipulating the data they contain. Also, while there is the potential to collect pathogen genomic data from infected individuals or agricultural and food products during disease outbreaks to improve disease modeling and forecast, how to protect the privacy of individuals, growers, and retailers is another major cyberbiosecurity challenge. As data become linkable to other data sources, individuals and groups become identifiable and potential malicious activities targeting those identified become feasible. Here, we define a number of potential cybersecurity weaknesses in today's pathogen genome databases to raise awareness, and we provide potential solutions to strengthen cyberbiosecurity during the development of the next generation of pathogen genome databases.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomics and outbreak investigation: from sequence to consequence</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Outbreaks of infection can be devastating for individuals and societies. In this review, we examine the applications of new high-throughput sequencing approaches to the identification and characterization of outbreaks, focusing on the application of whole-genome sequencing (WGS) to outbreaks of bacterial infection. We describe traditional epidemiological analysis and show how WGS can be informative at multiple steps in outbreak investigation, as evidenced by many recent studies. We conclude that high-throughput sequencing approaches can make a significant contribution to the investigation of outbreaks of bacterial infection and that the integration of WGS with epidemiological investigation, diagnostic assays and antimicrobial susceptibility testing will precipitate radical changes in clinical microbiology and infectious disease epidemiology in the near future. However, several challenges remain before WGS can be routinely used in outbreak investigation and clinical practice.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 23</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. AIM: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. METHODS: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. RESULTS: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive – Global (EVAg), a European Union infrastructure project. CONCLUSION: The present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 23</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. AIM: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. METHODS: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. RESULTS: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive – Global (EVAg), a European Union infrastructure project. CONCLUSION: The present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.</jats:p></p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Genomic Loci Important in Common Infectious Diseases: Role of High-Throughput Sequencing and Genome-Wide Association Studies</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Mar 20</span></i><p style="text-align: justify; text-justify: inter-word;">HIV/AIDS, tuberculosis (TB), and malaria are 3 major global public health threats that undermine development in many resource-poor settings. Recently, the notion that positive selection during epidemics or longer periods of exposure to common infectious diseases may have had a major effect in modifying the constitution of the human genome is being interrogated at a large scale in many populations around the world. This positive selection from infectious diseases increases power to detect associations in genome-wide association studies (GWASs). High-throughput sequencing (HTS) has transformed both the management of infectious diseases and continues to enable large-scale functional characterization of host resistance/susceptibility alleles and loci; a paradigm shift from single candidate gene studies. Application of genome sequencing technologies and genomics has enabled us to interrogate the host-pathogen interface for improving human health. Human populations are constantly locked in evolutionary arms races with pathogens; therefore, identification of common infectious disease-associated genomic variants/markers is important in therapeutic, vaccine development, and screening susceptible individuals in a population. This review describes a range of host-pathogen genomic loci that have been associated with disease susceptibility and resistant patterns in the era of HTS. We further highlight potential opportunities for these genetic markers.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple laboratory parameter facilitates early identification of COVID-19 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.</jats:p></p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Diseases and Molecular Diagnostics: A Step Closer to Precision Medicine</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The current advent of molecular technologies together with a multidisciplinary interplay of several fields led to the development of genomics, which concentrates on the detection of pathogenic events at the genome level. The structural and functional genomics approaches have now pinpointed the technical challenge in the exploration of disease-related genes and the recognition of their structural alterations or elucidation of gene function. Various promising technologies and diagnostic applications of structural genomics are currently preparing a large database of disease-genes, genetic alterations etc., by mutation scanning and DNA chip technology. Further the functional genomics also exploring the expression genetics (hybridization-, PCR- and sequence-based technologies), two-hybrid technology, next generation sequencing with Bioinformatics and computational biology. Advances in microarray “chip” technology as microarrays have allowed the parallel analysis of gene expression patterns of thousands of genes simultaneously. Sequence information collected from the genomes of many individuals is leading to the rapid discovery of single nucleotide polymorphisms or SNPs. Further advances of genetic engineering have also revolutionized immunoassay biotechnology via engineering of antibody-encoding genes and the phage display technology. The Biotechnology plays an important role in the development of diagnostic assays in response to an outbreak or critical disease response need. However, there is also need to pinpoint various obstacles and issues related to the commercialization and widespread dispersal of genetic knowledge derived from the exploitation of the biotechnology industry and the development and marketing of diagnostic services. Implementation of genetic criteria for patient selection and individual assessment of the risks and benefits of treatment emerges as a major challenge to the pharmaceutical industry. Thus this field is revolutionizing current era and further it may open new vistas in the field of disease management.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated Real-Time Collection of Pathogen-Specific Diagnostic Data: Syndromic Infectious Disease Epidemiology</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Health care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are comprehensive and specific with respect to the pathogen but are rapid in reporting the data. It has proven difficult to implement these requirements on a large scale while maintaining patient privacy. OBJECTIVE: The aim of this study was to demonstrate the automated export, aggregation, and analysis of infectious disease diagnostic test results from clinical laboratories across the United States in a manner that protects patient confidentiality. We hypothesized that such a system could aid in monitoring the seasonal occurrence of respiratory pathogens and may have advantages with regard to scope and ease of reporting compared with existing surveillance systems. METHODS: We describe a system, BioFire Syndromic Trends, for rapid disease reporting that is syndrome-based but pathogen-specific. Deidentified patient test results from the BioFire FilmArray multiplex molecular diagnostic system are sent directly to a cloud database. Summaries of these data are displayed in near real time on the Syndromic Trends public website. We studied this dataset for the prevalence, seasonality, and coinfections of the 20 respiratory pathogens detected in over 362,000 patient samples acquired as a standard-of-care testing over the last 4 years from 20 clinical laboratories in the United States. RESULTS: The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks, and adenovirus and the bacterial pathogens show constant detection over the year. The dataset can also be considered in an ecological framework; the viruses and bacteria detected by this test are parasites of a host (the human patient). Interestingly, the rate of pathogen codetections, on average 7.94% (28,741/362,101), matches predictions based on the relative abundance of organisms present. CONCLUSIONS: Syndromic Trends preserves patient privacy by removing or obfuscating patient identifiers while still collecting much useful information about the bacterial and viral pathogens that they harbor. Test results are uploaded to the database within a few hours of completion compared with delays of up to 10 days for other diagnostic-based reporting systems. This work shows that the barriers to establishing epidemiology systems are no longer scientific and technical but rather administrative, involving questions of patient privacy and data ownership. We have demonstrated here that these barriers can be overcome. This first look at the resulting data stream suggests that Syndromic Trends will be able to provide high-resolution analysis of circulating respiratory pathogens and may aid in the detection of new outbreaks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Generation of antibodies against COVID-19 virus for development of diagnostic tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.</jats:p></p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur.AimWe aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available.MethodsHere we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology.ResultsThe workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive - Global (EVAg), a European Union infrastructure project.ConclusionThe present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel analytic tools for the study of porcine reproductive and respiratory syndrome virus (PRRSv) in endemic settings: lessons learned in the U.S.</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 21</span></i><p style="text-align: justify; text-justify: inter-word;">Since its emergence in the late 1980’s, the porcine reproductive and respiratory syndrome virus (PRRSv) has posed a significant challenge to the pig industry worldwide. Since then, a number of epidemiological tools have been created to support control and eventual elimination of the disease at the farm and regional levels. Still, many aspects of the disease dynamics are yet-to-be elucidated, such as what are the economically optimal control strategies at the farm and regional level, what is the role that the voluntary regional control programs may play, how to optimize the use of molecular tools for surveillance and monitoring in infected settings, what is the full impact of the disease in a farm, or what is the relative contribution of alternative transmission routes on the occurrence of PRRSv outbreaks. Here, we summarize a number of projects demonstrating the use of novel analytical tools in the assessment of PRRSv epidemiology in the United States. Results presented demonstrate how quantitative analysis of routinely collected data may help in understanding regional epidemiology of PRRSv and to quantify its full impact, and how the integration of phylodynamic methods as a standard tool for molecular surveillance of PRRSv might help to inform control and prevention strategies in high-risk epidemiological situations. Ultimately, these tools will help to support PRRSv control at farm and regional levels in endemically infected settings.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen’s evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen’s evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen’s evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen’s evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen’s evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>A clearing house for diagnostic testing: the solution to ensure access to and use of patented genetic inventions?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In genetic diagnostics, the emergence of a so-called "patent thicket" is imminent. Such an overlapping set of patent rights may have restrictive effects on further research and development of diagnostic tests, and the provision of clinical diagnostic services. Currently, two models that may facilitate access to and use of patented genetic inventions are attracting much debate in various national and international fora: patent pools and clearing houses. In this article, we explore the concept of clearing houses. Several types of clearing houses are identified. First, we describe and discuss two types that would provide access to information on the patented inventions: the information clearing house and the technology exchange clearing house. Second, three types of clearing houses are analysed that not only offer access to information but also provide an instrument to facilitate the use of the patented inventions: the open access clearing house, the standardized licences clearing house and the royalty collection clearing house. A royalty collection clearing house for genetic diagnostic testing would be the most comprehensive as it would serve several functions: identifying patents and patent claims essential to diagnostic testing, matching licensees with licensors, developing and supplying standardized licences, collecting royalties, monitoring whether users respect licensing conditions, and providing dispute resolution services such as mediation and arbitration. In this way, it might function as an effective model for users to facilitate access to and use of the patented inventions. However, it remains to be seen whether patent holders with a strong patent portfolio will be convinced by the advantages of the royalty collection clearing house and be willing to participate.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Control of Infectious Diseases in the Era of European Clinical Microbiology Laboratory Consolidation: New Challenges and Opportunities for the Patient and for Public Health Surveillance</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 2</span></i><p style="text-align: justify; text-justify: inter-word;">Many new innovative diagnostic approaches have been made available during the last 10 years with major impact on patient care and public health surveillance. In parallel, to enhance the cost-effectiveness of the clinical microbiology laboratories (CMLs), European laboratory professionals have streamlined their organization leading to amalgamation of activities and restructuring of their professional relationships with clinicians and public health specialists. Through this consolidation process, an operational model has emerged that combines large centralized clinical laboratories performing most tests on one high-throughput analytical platform connected to several distal laboratories dealing locally with urgent analyses at near point of care. The centralization of diagnostic services over a large geographical region has given rise to the concept of regional-scale “microbiology laboratories network.” Although the volume-driven cost savings associated with such laboratory networks seem self-evident, the consequence(s) for the quality of patient care and infectious disease surveillance and control remain less obvious. In this article, we describe the range of opportunities that the changing landscape of CMLs in Europe can contribute toward improving the quality of patient care but also the early detection and enhanced surveillance of public health threats caused by infectious diseases. The success of this transformation of health services is reliant on the appropriate preparation in terms of staff, skills, and processes that would be inclusive of stakeholders. In addition, rigorous metrics are needed to set out more concrete laboratory service performance objectives and assess the expected benefits to society in terms of saving lives and preventing diseases.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>QTL-mapping and genomic prediction for bovine respiratory disease in U.S. Holsteins using sequence imputation and feature selection</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 5</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: National genetic evaluations for disease resistance do not exist, precluding the genetic improvement of cattle for these traits. We imputed BovineHD genotypes to whole genome sequence for 2703 Holsteins that were cases or controls for Bovine Respiratory Disease and sampled from either California or New Mexico to construct and compare genomic prediction models. The sequence variation reference dataset comprised variants called for 1578 animals from Run 5 of the 1000 Bull Genomes Project, including 450 Holsteins and 29 animals sequenced from this study population. Genotypes for 9,282,726 variants with minor allele frequencies ≥5% were imputed and used to obtain genomic predictions in GEMMA using a Bayesian Sparse Linear Mixed Model. RESULTS: Variation explained by markers increased from 13.6% using BovineHD data to 14.4% using imputed whole genome sequence data and the resolution of genomic regions detected as harbouring QTL substantially increased. Explained variation in the analysis of the combined California and New Mexico data was less than when data for each state were separately analysed and the estimated genetic correlation between risk of Bovine Respiratory Disease in California and New Mexico Holsteins was − 0.36. Consequently, genomic predictions trained using the data from one state did not accurately predict disease risk in the other state. To determine if a prediction model could be developed with utility in both states, we selected variants within genomic regions harbouring: 1) genes involved in the normal immune response to infection by pathogens responsible for Bovine Respiratory Disease detected by RNA-Seq analysis, and/or 2) QTL identified in the association analysis of the imputed sequence variants. The model based on QTL selected variants is biased but when trained in one state generated BRD risk predictions with positive accuracies in the other state. CONCLUSIONS: We demonstrate the utility of sequence-based and biology-driven model development for genomic selection. Disease phenotypes cannot be routinely recorded in most livestock species and the observed phenotypes may vary in their genomic architecture due to variation in the pathogen composition across environments. Elucidation of trait biology and genetic architecture may guide the development of prediction models with utility across breeds and environments. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1186/s12864-019-5941-5) contains supplementary material, which is available to authorized users.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Summary Background The novel coronavirus SARS-CoV-2 is a newly emerging virus. The antibody response in infected patient remains largely unknown, and the clinical values of antibody testing have not been fully demonstrated. Methods A total of 173 patients with confirmed SARS-CoV-2 infection were enrolled. Their serial plasma samples (n = 535) collected during the hospitalization period were tested for total antibodies (Ab), IgM and IgG against SARS-CoV-2 using immunoassays. The dynamics of antibodies with the progress and severity of disease was analyzed. Findings Among 173 patients, the seroconversion rate for Ab, IgM and IgG was 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively. Twelve patients who had not seroconverted were those only blood samples at the early stage of illness were collected. The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. The presence of antibodies was &lt; 40% among patients in the first 7 days of illness, and then rapidly increased to 100.0%, 94.3% and 79.8% for Ab, IgM and IgG respectively since day 15 after onset. In contrast, the positive rate of RNA decreased from 66.7% (58/87) in samples collected before day 7 to 45.5% (25/55) during days 15 to 39. Combining RNA and antibody detections significantly improved the sensitivity of pathogenic diagnosis for COVID-19 patients (p &lt; 0.001), even in early phase of 1-week since onset (p = 0.007). Moreover, a higher titer of Ab was independently associated with a worse clinical classification (p = 0.006). Interpretation The antibody detection offers vital clinical information during the course of SARS-CoV-2 infection. The findings provide strong empirical support for the routine application of serological testing in the diagnosis and management of COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Correlation of Chest CT and RT-PCR Testing in Coronavirus Disease 2019 (COVID-19) in China: A Report of 1014 Cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background Chest CT is used for diagnosis of 2019 novel coronavirus disease (COVID-19), as an important complement to the reverse-transcription polymerase chain reaction (RT-PCR) tests. Purpose To investigate the diagnostic value and consistency of chest CT as compared with comparison to RT-PCR assay in COVID-19. Methods From January 6 to February 6, 2020, 1014 patients in Wuhan, China who underwent both chest CT and RT-PCR tests were included. With RT-PCR as reference standard, the performance of chest CT in diagnosing COVID-19 was assessed. Besides, for patients with multiple RT-PCR assays, the dynamic conversion of RT-PCR results (negative to positive, positive to negative, respectively) was analyzed as compared with serial chest CT scans for those with time-interval of 4 days or more. Results Of 1014 patients, 59% (601/1014) had positive RT-PCR results, and 88% (888/1014) had positive chest CT scans. The sensitivity of chest CT in suggesting COVID-19 was 97% (95%CI, 95-98%, 580/601 patients) based on positive RT-PCR results. In patients with negative RT-PCR results, 75% (308/413) had positive chest CT findings; of 308, 48% were considered as highly likely cases, with 33% as probable cases. By analysis of serial RT-PCR assays and CT scans, the mean interval time between the initial negative to positive RT-PCR results was 5.1 ± 1.5 days; the initial positive to subsequent negative RT-PCR result was 6.9 ± 2.3 days). 60% to 93% of cases had initial positive CT consistent with COVID-19 prior (or parallel) to the initial positive RT-PCR results. 42% (24/57) cases showed improvement in follow-up chest CT scans before the RT-PCR results turning negative. Conclusion Chest CT has a high sensitivity for diagnosis of COVID-19. Chest CT may be considered as a primary tool for the current COVID-19 detection in epidemic areas.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>The Return of Lombroso? Ethical Aspects of (Visions of) Preventive Forensic Screening</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Nov 28</span></i><p style="text-align: justify; text-justify: inter-word;">The vision of legendary criminologist Cesare Lombroso to use scientific theories of individual causes of crime as a basis for screening and prevention programmes targeting individuals at risk for future criminal behaviour has resurfaced, following advances in genetics, neuroscience and psychiatric epidemiology. This article analyses this idea and maps its ethical implications from a public health ethical standpoint. Twenty-seven variants of the new Lombrosian vision of forensic screening and prevention are distinguished, and some scientific and technical limitations are noted. Some lures, biases and structural factors, making the application of the Lombrosian idea likely in spite of weak evidence are pointed out and noted as a specific type of ethical aspect. Many classic and complex ethical challenges for health screening programmes are shown to apply to the identified variants and the choice between them, albeit with peculiar and often provoking variations. These variations are shown to actualize an underlying theoretical conundrum in need of further study, pertaining to the relationship between public health ethics and the ethics and values of criminal law policy.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>The Evolution of Ebola virus: Insights from the 2013–2016 Epidemic</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The 2013–2016 epidemic of Ebola virus disease in West Africa was of unprecedented magnitude and changed our perspective on this lethal but sporadically emerging virus. This outbreak also marked the beginning of large-scale real-time molecular epidemiology. Herein, we show how evolutionary analyses of Ebola virus genome sequences provided key insights into virus origins, evolution, and spread during the epidemic. We provide basic scientists, epidemiologists, medical practitioners, and other outbreak responders with an enhanced understanding of the utility and limitations of pathogen genomic sequencing. This will be crucially important in our attempts to track and control future infectious disease outbreaks.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Bayesian inference of transmission chains using timing of symptoms, pathogen genomes and contact data</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 29</span></i><p style="text-align: justify; text-justify: inter-word;">There exists significant interest in developing statistical and computational tools for inferring ‘who infected whom’ in an infectious disease outbreak from densely sampled case data, with most recent studies focusing on the analysis of whole genome sequence data. However, genomic data can be poorly informative of transmission events if mutations accumulate too slowly to resolve individual transmission pairs or if there exist multiple pathogens lineages within-host, and there has been little focus on incorporating other types of outbreak data. We present here a methodology that uses contact data for the inference of transmission trees in a statistically rigorous manner, alongside genomic data and temporal data. Contact data is frequently collected in outbreaks of pathogens spread by close contact, including Ebola virus (EBOV), severe acute respiratory syndrome coronavirus (SARS-CoV) and Mycobacterium tuberculosis (TB), and routinely used to reconstruct transmission chains. As an improvement over previous, ad-hoc approaches, we developed a probabilistic model that relates a set of contact data to an underlying transmission tree and integrated this in the outbreaker2 inference framework. By analyzing simulated outbreaks under various contact tracing scenarios, we demonstrate that contact data significantly improves our ability to reconstruct transmission trees, even under realistic limitations on the coverage of the contact tracing effort and the amount of non-infectious mixing between cases. Indeed, contact data is equally or more informative than fully sampled whole genome sequence data in certain scenarios. We then use our method to analyze the early stages of the 2003 SARS outbreak in Singapore and describe the range of transmission scenarios consistent with contact data and genetic sequence in a probabilistic manner for the first time. This simple yet flexible model can easily be incorporated into existing tools for outbreak reconstruction and should permit a better integration of genomic and epidemiological data for inferring transmission chains.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>The European Classical Swine Fever Virus Database: Blueprint for a Pathogen-Specific Sequence Database with Integrated Sequence Analysis Tools</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Nov 7</span></i><p style="text-align: justify; text-justify: inter-word;">Molecular epidemiology has become an indispensable tool in the diagnosis of diseases and in tracing the infection routes of pathogens. Due to advances in conventional sequencing and the development of high throughput technologies, the field of sequence determination is in the process of being revolutionized. Platforms for sharing sequence information and providing standardized tools for phylogenetic analyses are becoming increasingly important. The database (DB) of the European Union (EU) and World Organisation for Animal Health (OIE) Reference Laboratory for classical swine fever offers one of the world’s largest semi-public virus-specific sequence collections combined with a module for phylogenetic analysis. The classical swine fever (CSF) DB (CSF-DB) became a valuable tool for supporting diagnosis and epidemiological investigations of this highly contagious disease in pigs with high socio-economic impacts worldwide. The DB has been re-designed and now allows for the storage and analysis of traditionally used, well established genomic regions and of larger genomic regions including complete viral genomes. We present an application example for the analysis of highly similar viral sequences obtained in an endemic disease situation and introduce the new geographic “CSF Maps” tool. The concept of this standardized and easy-to-use DB with an integrated genetic typing module is suited to serve as a blueprint for similar platforms for other human or animal viruses.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>What we know so far: COVID-19 current clinical knowledge and research</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, health authorities in Wuhan, China, identified a cluster of pneumonia cases of unknown aetiology linked to the city's South China Seafood Market. Subsequent investigations revealed a novel coronavirus, SARS-CoV-2, as the causative agent now at the heart of a major outbreak. The rising case numbers have been accompanied by unprecedented public health action, including the wholesale isolation of Wuhan. Alongside this has been a robust scientific response, including early publication of the pathogen genome, and rapid development of highly specific diagnostics. This article will review the new knowledge of SARS-CoV-2 COVID-19 acute respiratory disease, and summarise its clinical features.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Rationale and Design of the Genomic Research in Alpha-1 Antitrypsin Deficiency and Sarcoidosis (GRADS) Study. Sarcoidosis Protocol</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Sarcoidosis is a systemic disease characterized by noncaseating granulomatous inflammation with tremendous clinical heterogeneity and uncertain pathobiology and lacking in clinically useful biomarkers. The Genomic Research in Alpha-1 Antitrypsin Deficiency and Sarcoidosis (GRADS) study is an observational cohort study designed to explore the role of the lung microbiome and genome in these two diseases. This article describes the design and rationale for the GRADS study sarcoidosis protocol. The study addresses the hypothesis that distinct patterns in the lung microbiome are characteristic of sarcoidosis phenotypes and are reflected in changes in systemic inflammatory responses as measured by peripheral blood changes in gene transcription. The goal is to enroll 400 participants, with a minimum of 35 in each of 9 clinical phenotype subgroups prioritized by their clinical relevance to understanding of the pathobiology and clinical heterogeneity of sarcoidosis. Participants with a confirmed diagnosis of sarcoidosis undergo a baseline visit with self-administered questionnaires, chest computed tomography, pulmonary function tests, and blood and urine testing. A research or clinical bronchoscopy with a research bronchoalveolar lavage will be performed to obtain samples for genomic and microbiome analyses. Comparisons will be made by blood genomic analysis and with clinical phenotypic variables. A 6-month follow-up visit is planned to assess each participant’s clinical course. By the use of an integrative approach to the analysis of the microbiome and genome in selected clinical phenotypes, the GRADS study is powerfully positioned to inform and direct studies on the pathobiology of sarcoidosis, identify diagnostic or prognostic biomarkers, and provide novel molecular phenotypes that could lead to improved personalized approaches to therapy for sarcoidosis.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Corona Virus Disease 2019 (COVID-19) is a recently emerged life-threatening disease caused by SARS-CoV-2. Real- time fluorescent PCR (RT-PCR) is the clinical standard for SARS-CoV-2 nucleic acid detection. To detect SARS-CoV-2 early and control the disease spreading on time, a faster and more convenient method for SARS-CoV-2 nucleic acid detecting, RT-LAMP method (reverse transcription loop-mediated isothermal amplification) was developed. RNA reverse transcription and nucleic acid amplification were performed in one step at 63 ℃ isothermal conditions, and the results can be obtained within 30 minutes. ORF1ab gene, E gene and N gene were detected at the same time. ORF1ab gene was very specific and N gene was very sensitivity, so they can guarantee both sensitivity and specificity for SARS-CoV-2. The sensitivity of RT-LAMP assay is similar to RT-PCR, and specificity was 99% as detecting 208 clinical specimens. The RT-LAMP assay reported here has the advantages of rapid amplification, simple operation, and easy detection, which is useful for the rapid and reliable clinical diagnosis of SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 31st December 2019, the World Health Organization was informed of a cluster of cases of pneumonia of unknown etiology in Wuhan, China. Subsequent investigations identified a novel coronavirus, now named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), from the affected patients. Highly sensitive and specific laboratory diagnostics are important for controlling the rapidly evolving SARS-CoV-2-associated Coronavirus Disease 2019 (COVID-19) epidemic. In this study, we developed and compared the performance of three novel real-time RT-PCR assays targeting the RNA-dependent RNA polymerase (RdRp)/helicase (Hel), spike (S), and nucleocapsid (N) genes of SARS-CoV-2 with that of the reported RdRp-P2 assay which is used in >30 European laboratories. Among the three novel assays, the COVID-19-RdRp/Hel assay had the lowest limit of detection in vitro (1.8 TCID50/ml with genomic RNA and 11.2 RNA copies/reaction with in vitro RNA transcripts). Among 273 specimens from 15 patients with laboratory-confirmed COVID-19 in Hong Kong, 77 (28.2%) were positive by both the COVID-19-RdRp/Hel and RdRp-P2 assays. The COVID-19-RdRp/Hel assay was positive for an additional 42 RdRd-P2-negative specimens [119/273 (43.6%) vs 77/273 (28.2%), P<0.001], including 29/120 (24.2%) respiratory tract specimens and 13/153 (8.5%) non-respiratory tract specimens. The mean viral load of these specimens was 3.21*104 RNA copies/ml (range, 2.21*102 to 4.71*105 RNA copies/ml). The COVID-19-RdRp/Hel assay did not cross-react with other human-pathogenic coronaviruses and respiratory pathogens in cell culture and clinical specimens, whereas the RdRp-P2 assay cross-reacted with SARS-CoV in cell culture. The highly sensitive and specific COVID-19-RdRp/Hel assay may help to improve the laboratory diagnosis of COVID-19.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic resistance - an alternative for controlling PRRS?</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Nov 16</span></i><p style="text-align: justify; text-justify: inter-word;">PRRS is one of the most challenging diseases for world-wide pig production. Attempts for a sustainable control of this scourge by vaccination have not yet fully satisfied. With an increasing knowledge and methodology in disease resistance, a new world-wide endeavour has been started to support the combat of animal diseases, based on the existence of valuable gene variants with regard to any host-pathogen interaction. Several groups have produced a wealth of evidence for natural variability in resistance/susceptibility to PRRS in our commercial breeding lines. However, up to now, exploiting existing variation has failed because of the difficulty to detect the carriers of favourable and unfavourable alleles, especially with regard to such complex polygenic traits like resistance to PRRS. New hope comes from new genomic tools like next generation sequencing which have become extremely fast and low priced. Thus, research is booming world-wide and the jigsaw puzzle is filling up – slowly but steadily. On the other hand, knowledge from virological and biomedical basic research has opened the way for an “intervening way”, i.e. the modification of identified key genes that occupy key positions in PRRS pathogenesis, like CD163. CD163 was identified as the striking receptor in PRRSV entry and its knockout from the genome by gene editing has led to the production of pigs that were completely resistant to PRRSV – a milestone in modern pig breeding. However, at this early step, concerns remain about the acceptance of societies for gene edited products and regulation still awaits upgrading to the new technology. Further questions arise with regard to upcoming patents from an ethical and legal point of view. Eventually, the importance of CD163 for homeostasis, defence and immunity demands for more insight before its complete or partial silencing can be answered. Whatever path will be followed, even a partial abolishment of PRRSV replication will lead to a significant improvement of the disastrous herd situation, with a significant impact on welfare, performance, antimicrobial consumption and consumer protection. Genetics will be part of a future solution.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>The Lab Without Walls: A Deployable Approach to Tropical Infectious Diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Laboratory Without Walls is a modular field application of molecular biology that provides clinical laboratory support in resource-limited, remote locations. The current repertoire arose from early attempts to deliver clinical pathology and public health investigative services in remote parts of tropical Australia, to address the shortcomings of conventional methods when faced with emerging infectious diseases. Advances in equipment platforms and reagent chemistry have enabling rapid progress, but also ensure the Laboratory Without Walls is subject to continual improvement. Although new molecular biology methods may lead to more easily deployable clinical laboratory capability, logistic and technical governance issues continue to act as important constraints on wider implementation.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrated Multilevel Surveillance of the World's Infecting Microbes and Their Resistance to Antimicrobial Agents</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Summary: Microbial surveillance systems have varied in their source of support; type of laboratory reporting (patient care or reference); inclusiveness of reports filed; extent of microbial typing; whether single hospital, multihospital, or multicountry; proportion of total medical centers participating; and types, levels, integration across levels, and automation of analyses performed. These surveillance systems variably support the diagnosis and treatment of patients, local or regional infection control, local or national policies and guidelines, laboratory capacity building, sentinel surveillance, and patient safety. Overall, however, only a small fraction of available data are under any surveillance, and very few data are fully integrated and analyzed. Advancing informatics and genomics can make microbial surveillance far more efficient and effective at preventing infections and improving their outcomes. The world's microbiology laboratories should upload their reports each day to programs that detect events, trends, and epidemics in communities, hospitals, countries, and the world.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Using population-wide administrative and laboratory data to estimate type- and subtype-specific influenza vaccine effectiveness: a surveillance protocol</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Sep 30</span></i><p style="text-align: justify; text-justify: inter-word;">INTRODUCTION: The appropriateness of using routinely collected laboratory data combined with administrative data for estimating influenza vaccine effectiveness (VE) is still being explored. This paper outlines a protocol to estimate influenza VE using linked laboratory and administrative data which could act as a companion to estimates derived from other methods. METHODS AND ANALYSIS: We will use the test-negative design to estimate VE for each influenza type/subtype and season. Province-wide individual-level records of positive and negative influenza tests at the Provincial Laboratory for Public Health in Alberta will be linked, by unique personal health numbers, to administrative databases and vaccination records held at the Ministry of Health in Alberta to determine covariates and influenza vaccination status, respectively. Covariates of interests include age, sex, immunocompromising chronic conditions and healthcare setting. Cases will be defined based on an individual’s first positive influenza test during the season, and potential controls will be defined based on an individual’s first negative influenza test during the season. One control for each case will be randomly selected based on the week the specimen was collected. We will estimate VE using multivariable logistic regression. ETHICS AND DISSEMINATION: Ethics approval was obtained from the University of Alberta’s Health Research Ethics Board—Health Panel under study ID Pro00075997. Results will be disseminated by public health officials in Alberta.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Using population-wide administrative and laboratory data to estimate type- and subtype-specific influenza vaccine effectiveness: a surveillance protocol</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Sep 30</span></i><p style="text-align: justify; text-justify: inter-word;">INTRODUCTION: The appropriateness of using routinely collected laboratory data combined with administrative data for estimating influenza vaccine effectiveness (VE) is still being explored. This paper outlines a protocol to estimate influenza VE using linked laboratory and administrative data which could act as a companion to estimates derived from other methods. METHODS AND ANALYSIS: We will use the test-negative design to estimate VE for each influenza type/subtype and season. Province-wide individual-level records of positive and negative influenza tests at the Provincial Laboratory for Public Health in Alberta will be linked, by unique personal health numbers, to administrative databases and vaccination records held at the Ministry of Health in Alberta to determine covariates and influenza vaccination status, respectively. Covariates of interests include age, sex, immunocompromising chronic conditions and healthcare setting. Cases will be defined based on an individual’s first positive influenza test during the season, and potential controls will be defined based on an individual’s first negative influenza test during the season. One control for each case will be randomly selected based on the week the specimen was collected. We will estimate VE using multivariable logistic regression. ETHICS AND DISSEMINATION: Ethics approval was obtained from the University of Alberta’s Health Research Ethics Board—Health Panel under study ID Pro00075997. Results will be disseminated by public health officials in Alberta.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Using population-wide administrative and laboratory data to estimate type- and subtype-specific influenza vaccine effectiveness: a surveillance protocol</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Sep 30</span></i><p style="text-align: justify; text-justify: inter-word;">INTRODUCTION: The appropriateness of using routinely collected laboratory data combined with administrative data for estimating influenza vaccine effectiveness (VE) is still being explored. This paper outlines a protocol to estimate influenza VE using linked laboratory and administrative data which could act as a companion to estimates derived from other methods. METHODS AND ANALYSIS: We will use the test-negative design to estimate VE for each influenza type/subtype and season. Province-wide individual-level records of positive and negative influenza tests at the Provincial Laboratory for Public Health in Alberta will be linked, by unique personal health numbers, to administrative databases and vaccination records held at the Ministry of Health in Alberta to determine covariates and influenza vaccination status, respectively. Covariates of interests include age, sex, immunocompromising chronic conditions and healthcare setting. Cases will be defined based on an individual’s first positive influenza test during the season, and potential controls will be defined based on an individual’s first negative influenza test during the season. One control for each case will be randomly selected based on the week the specimen was collected. We will estimate VE using multivariable logistic regression. ETHICS AND DISSEMINATION: Ethics approval was obtained from the University of Alberta’s Health Research Ethics Board—Health Panel under study ID Pro00075997. Results will be disseminated by public health officials in Alberta.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Using population-wide administrative and laboratory data to estimate type- and subtype-specific influenza vaccine effectiveness: a surveillance protocol</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Sep 30</span></i><p style="text-align: justify; text-justify: inter-word;">INTRODUCTION: The appropriateness of using routinely collected laboratory data combined with administrative data for estimating influenza vaccine effectiveness (VE) is still being explored. This paper outlines a protocol to estimate influenza VE using linked laboratory and administrative data which could act as a companion to estimates derived from other methods. METHODS AND ANALYSIS: We will use the test-negative design to estimate VE for each influenza type/subtype and season. Province-wide individual-level records of positive and negative influenza tests at the Provincial Laboratory for Public Health in Alberta will be linked, by unique personal health numbers, to administrative databases and vaccination records held at the Ministry of Health in Alberta to determine covariates and influenza vaccination status, respectively. Covariates of interests include age, sex, immunocompromising chronic conditions and healthcare setting. Cases will be defined based on an individual’s first positive influenza test during the season, and potential controls will be defined based on an individual’s first negative influenza test during the season. One control for each case will be randomly selected based on the week the specimen was collected. We will estimate VE using multivariable logistic regression. ETHICS AND DISSEMINATION: Ethics approval was obtained from the University of Alberta’s Health Research Ethics Board—Health Panel under study ID Pro00075997. Results will be disseminated by public health officials in Alberta.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">1The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Assessing the impact of a symptom-based mass screening and testing intervention during a novel infectious disease outbreak: The case of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A symptom-based mass screening and testing intervention (MSTI) can identify a large fraction of infected individuals during an infectious disease outbreak. China is currently using this strategy for the COVID-19 outbreak. However, MSTI might lead to increased transmission if not properly implemented. We investigate under which conditions MSTI is beneficial.</jats:p></p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>[Potential false-positive rate among the 'asymptomatic infected individuals' in close contacts of COVID-19 patients]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: As the prevention and control of COVID-19continues to advance, the active nucleic acid test screening in the close contacts of the patients has been carrying out in many parts of China. However, the false-positive rate of positive results in the screening has not been reported up to now. But to clearify the false-positive rate during screening is important in COVID-19 control and prevention. Methods: Point values and reasonable ranges of the indicators which impact the false-positive rate of positive results were estimated based on the information available to us at present. The false-positive rate of positive results in the active screening was deduced, and univariate and multivariate-probabilistic sensitivity analyses were performed to understand the robustness of the findings. Results: When the infection rate of the close contacts and the sensitivity and specificity of reported results were taken as the point estimates, the positive predictive value of the active screening was only 19.67%, in contrast, the false-positive rate of positive results was 80.33%. The multivariate-probabilistic sensitivity analysis results supported the base-case findings, with a 75% probability for the false-positive rate of positive results over 47%. Conclusions: In the close contacts of COVID-19 patients, nearly half or even more of the 'asymptomatic infected individuals' reported in the active nucleic acid test screening might be false positives.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>VADR: validation and annotation of virus sequence submissions to GenBank</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-11-22</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundGenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions.  ResultsWe developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of "alerts" that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBanks submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (https://github.com/nawrockie/vadr) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally.  ConclusionVADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allows researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19.  Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<i class="tooltip">2017<span class="tooltiptext">2017-09-22</span></i><p style="text-align: justify; text-justify: inter-word;">Health-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray(R) Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Geospatial Science and Point-of-Care Testing: Creating Solutions for Population Access, Emergencies, Outbreaks, and Disasters</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 26</span></i><p style="text-align: justify; text-justify: inter-word;">Objectives: (a) To understand how to integrate geospatial concepts when implementing point-of-care testing (POCT); (b) to facilitate emergency, outbreak, and disaster preparedness and emergency management in healthcare small-world networks; (c) to enhance community resilience by using POCT in tandem with geographic information systems (GISs) and other geospatial tools; and (d) to advance crisis standards of care at points of need, adaptable and scalable for public health practice in limited-resource countries and other global settings. Content: Visual logistics help integrate and synthesize POCT and geospatial concepts. The resulting geospatial solutions presented here comprise: (1) small-world networks and regional topography; (2) space-time transformation, hubs, and asset mapping; (3) spatial and geospatial care paths™; (4) GIS-POCT; (5) isolation laboratories, diagnostics isolators, and mobile laboratories for highly infectious diseases; (6) alternate care facilities; (7) roaming POCT—airborne, ambulances, space, and wearables; (8) connected and wireless POCT outside hospitals; (9) unmanned aerial vehicles; (10) geospatial practice—demographic care unit resource scoring, geographic risk assessment, and national POCT policy and guidelines; (11) the hybrid laboratory; and (12) point-of-careology. Value: Small-world networks and their connectivity facilitate efficient and effective placement of POCT for optimal response, rescue, diagnosis, and treatment. Spatial care paths™ speed transport from primary encounters to referral centers bypassing topographic bottlenecks, process gaps, and time-consuming interruptions. Regional GISs position POCT close to where patients live to facilitate rapid triage, decrease therapeutic turnaround time, and conserve economic resources. Geospatial care paths™ encompass demographic and population access features. Timeliness creates value during acute illness, complex crises, and unexpected disasters. Isolation laboratories equipped with POCT help stop outbreaks and safely support critically ill patients with highly infectious diseases. POCT-enabled spatial grids can map sentinel cases and establish geographic limits of epidemics for ring vaccination. Impact: Geospatial solutions generate inherently optimal and logical placement of POCT conceptually, physically, and temporally as a means to improve crisis response and spatial resilience. If public health professionals, geospatial scientists, and POCT specialists join forces, new collaborative teamwork can create faster response and higher impact during disasters, complex crises, outbreaks, and epidemics, as well as more efficient primary, urgent, and emergency community care.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Personal genomes, quantitative dynamic omics and personalized medicine</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid technological developments following the Human Genome Project have made possible the availability of personalized genomes. As the focus now shifts from characterizing genomes to making personalized disease associations, in combination with the availability of other omics technologies, the next big push will be not only to obtain a personalized genome, but to quantitatively follow other omics. This will include transcriptomes, proteomes, metabolomes, antibodyomes, and new emerging technologies, enabling the profiling of thousands of molecular components in individuals. Furthermore, omics profiling performed longitudinally can probe the temporal patterns associated with both molecular changes and associated physiological health and disease states. Such data necessitates the development of computational methodology to not only handle and descriptively assess such data, but also construct quantitative biological models. Here we describe the availability of personal genomes and developing omics technologies that can be brought together for personalized implementations and how these novel integrated approaches may effectively provide a precise personalized medicine that focuses on not only characterization and treatment but ultimately the prevention of disease.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>GENOMIC ANALYSIS OF VIRAL OUTBREAKS</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Genomic analysis is a powerful tool for understanding viral disease outbreaks. Sequencing of viral samples is now easier and cheaper than ever before, and can supplement epidemiological methods by providing nucleotide-level resolution of outbreak-causing pathogens. In this review, we describe methods used to answer crucial questions about outbreaks, such as how they began and how a disease is transmitted. More specifically, we explain current techniques for viral sequencing, phylogenetic analysis, transmission reconstruction, and evolutionary investigation of viral pathogens. By detailing the ways in which genomic data can help understand viral disease outbreaks, we aim to provide a resource that will facilitate the response to future outbreaks.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Building laboratory capacity to detect and characterize pathogens of public and global health security concern in Kenya</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 10</span></i><p style="text-align: justify; text-justify: inter-word;">Since 1979, multiple CDC Kenya programs have supported the development of diagnostic expertise and laboratory capacity in Kenya. In 2004, CDC’s Global Disease Detection (GDD) program within the Division of Global Health Protection in Kenya (DGHP-Kenya) initiated close collaboration with Kenya Medical Research Institute (KEMRI) and developed a laboratory partnership called the Diagnostic and Laboratory Systems Program (DLSP). DLSP built onto previous efforts by malaria, human immunodeficiency virus (HIV) and tuberculosis (TB) programs and supported the expansion of the diagnostic expertise and capacity in KEMRI and the Ministry of Health. First, DLSP developed laboratory capacity for surveillance of diarrheal, respiratory, zoonotic and febrile illnesses to understand the etiology burden of these common illnesses and support evidenced-based decisions on vaccine introductions and recommendations in Kenya. Second, we have evaluated and implemented new diagnostic technologies such as TaqMan Array Cards (TAC) to detect emerging or reemerging pathogens and have recently added a next generation sequencer (NGS). Third, DLSP provided rapid laboratory diagnostic support for outbreak investigation to Kenya and regional countries. Fourth, DLSP has been assisting the Kenya National Public Health laboratory-National Influenza Center and microbiology reference laboratory to obtain World Health Organization (WHO) certification and ISO15189 accreditation respectively. Fifth, we have supported biosafety and biosecurity curriculum development to help Kenyan laboratories safely and appropriately manage infectious pathogens. These achievements, highlight how in collaboration with existing CDC programs working on HIV, tuberculosis and malaria, the Global Health Security Agenda can have significantly improve public health in Kenya and the region. Moreover, Kenya provides an example as to how laboratory science can help countries detect and control of infectious disease outbreaks and other public health threats more rapidly, thus enhancing global health security.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Establishment and Application of a Universal Coronavirus Screening Method Using MALDI-TOF Mass Spectrometry</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 9</span></i><p style="text-align: justify; text-justify: inter-word;">There are four human coronaviruses (HCoVs), distributed worldwide, that are associated with a range of respiratory symptoms. The discovery of severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV shows that HCoVs pose a significant threat to human health. Our work aims to develop a sensitive method (mCoV-MS) which can not only identify known HCoVs accurately, but also have the ability to provide clues for the emerging HCoVs. The method was performed using a MassARRAY matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS) system. We developed a 17-plex analysis to detect six HCoVs in Panel A and another 17-plex analysis to detect Alphacoronavirus and Betacoronavirus in Panel B. All tested primers and probes for the mCoV-MS method were effective, with no cross-reactivity observed with other common respiratory viruses. To confirm the usefulness of the mCoV-MS method we screened 384 pharyngeal and/or anal swab samples collected from bats/rodents, and 131 nasal and throat swabs from human patients. The results showed good concordance with the results of metagenomic analysis or PCR-sequencing. The validation test showed mCoV-MS method can detect potentially pathogenic CoVs in Alphacoronavirus and Betacoronavirus and provide convincingly phylogenetic evidences about unknown CoVs. The mCoV-MS method is a sensitive assay that is relatively simple to carry out. We propose that this method be used to complement next generation sequencing technology for large-scale screening studies.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of Bat Coronaviruses through Surveillance and Probe Capture-Based Next-Generation Sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) of bat origin have caused two pandemics in this century. Severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV both originated from bats, and it is highly likely that bat coronaviruses will cause future outbreaks. Active surveillance is both urgent and essential to predict and mitigate the emergence of these viruses in humans. Next-generation sequencing (NGS) is currently the preferred methodology for virus discovery to ensure unbiased sequencing of bat CoVs, considering their high genetic diversity. However, unbiased NGS is an expensive methodology and is prone to missing low-abundance CoV sequences due to the high background level of nonviral sequences present in surveillance field samples. Here, we employ a capture-based NGS approach using baits targeting most of the CoV species. Using this technology, we effectively reduced sequencing costs by increasing the sensitivity of detection. We discovered nine full genomes of bat CoVs in this study and revealed great genetic diversity for eight of them. IMPORTANCE Active surveillance is both urgent and essential to predict and mitigate the emergence of bat-origin CoV in humans and livestock. However, great genetic diversity increases the chance of homologous recombination among CoVs. Performing targeted PCR, a common practice for many surveillance studies, would not reflect this diversity. NGS, on the other hand, is an expensive methodology and is prone to missing low-abundance CoV sequences. Here, we employ a capture-based NGS approach using baits targeting all CoVs. Our work demonstrates that targeted, cost-effective, large-scale, genome-level surveillance of bat CoVs is now highly feasible.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /></body>