
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: bioinformatics for a target regions of SARS-CoV-2 genome</h2><br /></div><br /><br /><br /><div class="card"><h3>A proposal of an alternative primer for the ARTIC Network's multiplex PCR to improve coverage of SARS-CoV-2 genome sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole genome analysis of the novel corona virus, SARS-CoV-2, soon after the epidemics of this pathogen was revealed. The primer set seems to have been adapted already by many researchers worldwide and contributed for the high-quality and prompt genome epidemiology of this potential pandemic virus. We have also seen the great performance of their primer set and protocol; the primer set was able to amplify all desired PCR products with fairy small amplification bias from clinical samples with relatively high viral load. However, we observed acute drop of reads derived from two particular PCR products, 18 and 76, out of the 98 designated products as sample's viral load decreases. We suspected the reason for this low coverage issue was due to dimer formation between primers used to amplify those two PCR products. Here, we propose replacing just one of those primers, nCoV-2019_76_RIGHT(-), to a newly designed primer. The result of the replacement of primer showed improvement in coverage at both regions targeted by the products, 18 and 76. We expect this simple modification will extend the limit for whole SARS-CoV-2 genome analysis to samples with lower viral load and enhance genomic epidemiology of this pathogen.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;">A new coronavirus SARS-CoV-2, recently discovered in Wuhan, China, has caused over 74000 infection cases and 2000 deaths. Due to the rapidly growing cases and the unavailability of specific therapy, there is a desperate need for vaccines to combat the epidemic of SARS-CoV-2. In the present study, we performed an in silico approach based on the available virus genome to identify the antigenic B-cell epitopes and human-leukocyte-antigen (HLA) restricted T-cell epitopes. A total of 61 B-cell epitopes were initially identified, 19 of which with higher potential immunogenicity were used for vaccine design. 499 T-cell epitopes were predicted that showed affinity with the 34 most popular HLA alleles in Chinese population. Based on these epitopes, 30 vaccine candidates were designed and inspected against safety risks, including potential toxicity, human homologous, pharmaceutical peptides and bioactive peptides. Majority of vaccine peptides contained both B-cell and T-cell epitopes, which may interact with the most prevalent HLA alleles accounting for ~99% of Chinese population. Docking analysis showed stable hydrogen bonds of epitopes with their corresponding HLA alleles. In conclusion, these putative antigenic peptides may elicit the resistance response to the viral infection. In vitro and in vivo experiments are required to validate the effectiveness of these peptide vaccine.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3-to-5 exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 has caused tens of thousands of infections and more than one thousand deaths. There are currently no registered therapies for treating coronavirus infections. Because of time consuming process of new drug development, drug repositioning may be the only solution to the epidemic of sudden infectious diseases. We systematically analyzed all the proteins encoded by SARS-CoV-2 genes, compared them with proteins from other coronaviruses, predicted their structures, and built 19 structures that could be done by homology modeling. By performing target-based virtual ligand screening, a total of 21 targets (including two human targets) were screened against compound libraries including ZINC drug database and our own database of natural products. Structure and screening results of important targets such as 3-chymotrypsin-like protease (3CLpro), Spike, RNA-dependent RNA polymerase (RdRp), and papain like protease (PLpro) were discussed in detail. In addition, a database of 78 commonly used anti-viral drugs including those currently on the market and undergoing clinical trials for SARS-CoV-2 was constructed. Possible targets of these compounds and potential drugs acting on a certain target were predicted. This study will provide new lead compounds and targets for further in vitro and in vivo studies of SARS-CoV-2, new insights for those drugs currently ongoing clinical studies, and also possible new strategies for drug repositioning to treat SARS-CoV-2 infections.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.</jats:p></p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the {beta}-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;">Wuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.</jats:p></p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;">Severe Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 - a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Sensitive one-step isothermal detection of pathogen-derived RNAs</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreaks of Ebola, Zika, MERS, and SARS-CoV-2 (2019-nCoV) require fast, simple, and sensitive onsite nucleic acid diagnostics that can be developed rapidly to prevent the spread of diseases. We have developed a SENsitive Splint-based one-step isothermal RNA detection (SENSR) method for rapid and straightforward onsite detection of pathogen RNAs with high sensitivity and specificity. SENSR consists of two simple enzymatic reactions: a ligation reaction by SplintR ligase and subsequent transcription by T7 RNA polymerase. The resulting transcript forms an RNA aptamer that induces fluorescence. Here, we demonstrate that SENSR is an effective and highly sensitive method for the detection of the current epidemic pathogen, <jats:italic>severe acute respiratory syndrome-related coronavirus 2</jats:italic> (SARS-CoV-2). We also show that the platform can be extended to the detection of five other pathogens. Overall, SENSR is a molecular diagnostic method that can be developed rapidly for onsite uses requiring high sensitivity, specificity, and short assaying times.</jats:p></p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the  2019-nCoV_N2, N3 of USA and the  ORF1ab of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 Vaccine Pipeline: an Overview</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19).</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>The outbreak of SARS-CoV-2 pneumonia calls for viral vaccines</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of 2019-novel coronavirus disease (COVID-19) that is caused by SARS-CoV-2 has spread rapidly in China, and has developed to be a Public Health Emergency of International Concern. However, no specific antiviral treatments or vaccines are available yet. This work aims to share strategies and candidate antigens to develop safe and effective vaccines against SARS-CoV-2. An outbreak of 2019-novel coronavirus (SARS-CoV-2) that causes atypical pneumonia (COVID-19) has raged in China since mid-December 2019 and has spread to 26 countries (February 20, 2020). The epidemic was identified by the first four cases confirmed on December 29, 2019 and was traced to the Huanan Seafood Wholesale Market, Wuhan city, Hubei Province, China1. A total of 75,465 cases with SARS-CoV-2 infections have been confirmed up to date (February 20, 2020), and 2,236 people have died in China2. COVID-19 spreads rapidly by human-to-human transmission with a median incubation period of 3.0 days (range, 0 to 24.0), and the time from symptom onset to developing pneumonia is 4.0 days (range, 2.0 to 7.0)3. Respiratory droplets and direct contact are conventional transmission routes for SARS-CoV-2, and fecal-to-oral transmission might also have a role3. Fever, dry cough, and fatigue are common symptoms at onset of COVID-194. Most patients have lymphopenia and bilateral ground-glass opacity changes on chest CT scans4,5. No specific antiviral treatments or vaccines are available because it is a new emerging viral disease. Development of SARS-CoV-2-based vaccines is urgently required. The entire virus particle-based preparation of vaccines, including inactivated and attenuated virus vaccines is advisable, because it is based on previous studies about the prevention and control of seasonal influenza vaccines6. The first SARS-CoV-2 (Wuhan-Hu-1) was successfully sequenced and its genomic sequence submitted to GenBank on January 5, 2020 (Accession no. MN908947.3)7. Subsequently large-scale culture of SARS-CoV-2 was quickly performed, and an inactivated virus vaccine could be prepared through the employment of established physical and chemical methods such as UV light, formaldehyde, and β-propiolactone8. The development of attenuated-virus vaccines is also possible by carefully screening the serially propagated SARS-CoV-2 with reduced pathogenesis such as induced minimal lung injury, diminished limited neutrophil influx, and increased anti-inflammatory cytokine expressions compared with the wild-type virus9. Both inactivated and attenuated virus vaccines have their own disadvantages and side effects (Table 1). Alternatively, new vaccine designs based on the putative protective antigen/peptides derived from SARS-CoV-2 should be considered</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 sensitive to type I interferon pretreatment.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B coronaviruses share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel coronavirus is much more sensitive to type I interferon pretreatment. We subsequently examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN response between SARS-CoV and SARS-CoV-2 that could help inform disease progression, treatment options, and animal model development.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">1The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available for the cure of COVID-19. Therefore, an effective vaccine against SARS-COV-2 is needed to be design. This study was conducted to design an effective multi-epitope vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as a target and epitopes (B cell, IFN-{gamma} and T cell) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected T cell epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the worlds population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and twelve MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the vaccine. The allergenicity, physiochemical properties, antigenicity and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic and antigenic. Molecular docking demonstrated a stable and strong binding affinity of MEV with TLR3 and TLR8. Codon optimization and in silico cloning ensured increased expression in the Escherichia coli K-12 system. However, to ensure its safety and immunogenic profile, the proposed vaccine needs to be experimentally validated.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic variations of SARS-CoV-2 suggest multiple outbreak sources of transmission</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We examined 169 genomes of SARS-CoV-2 and found that they can be classified into two major genotypes, Type I and Type II. Type I can be further divided into Type IA and IB. Our phylogenetic analysis showed that the Type IA resembles the ancestral SARS-CoV-2 most. Type II was likely evolved from Type I and predominant in the infections. Our results suggest that Type II SARS-CoV-2 was the source of the outbreak in the Wuhan Huanan market and it was likely originated from a super-spreader. The outbreak caused by the Type I virus should have occurred somewhere else, because the patients had no direct link to the market. Furthermore, by analyzing three genomic sites that distinguish Type I and Type II strains, we found that synonymous changes at two of the three sites confer higher protein translational efficiencies in Type II strains than in Type I strains, which might explain why Type II strains are predominant, implying that Type II is more contagious (transmissible) than Type I. These findings could be valuable for the current epidemic prevention and control.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;">The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of novel coronavirus’s spike protein solved</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">n a piece of research accomplished with astonishing speed, scientists from the University of Texas at Austin and the National Institutes of Health have released a cryo-electron microscopy (cryo-EM) structure of part of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the novel coronavirus that has infected tens of thousands of people and killed more than 2,000 since the end of December (Science 2020, DOI: 10.1126/science.abb2507). The part of the virus imaged, called the spike protein, helps the virus attach to and infect human cells, and the determination of its structure comes just weeks after the virus’s genome sequence was published. The breakthrough is a huge step toward developing a vaccine against the virus as well as treatments for COVID-19, the disease that it causes, the researchers say. UT Austin’s Jason McLellan and his colleagues, who have spent many years studying other coronaviruses, had already figured out how to use select</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Antiviral drugs for managing infections with human coronaviruses are not yet approved, posing a serious challenge to current global efforts aimed at containing the outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Remdesivir (RDV) is an investigational compound with a broad spectrum of antiviral activities against RNA viruses, including SARS-CoV and Middle East respiratory syndrome (MERS-CoV). RDV is a nucleotide analog inhibitor of RNA-dependent RNA polymerases (RdRps). Here, we co-expressed the MERS-CoV nonstructural proteins nsp5, nsp7, nsp8, and nsp12 (RdRp) in insect cells as a part a polyprotein to study the mechanism of inhibition of MERS-CoV RdRp by RDV. We initially demonstrated that nsp8 and nsp12 form an active complex. The triphosphate form of the inhibitor (RDV-TP) competes with its natural counterpart ATP. Of note, the selectivity value for RDV-TP obtained here with a steady-state approach suggests that it is more efficiently incorporated than ATP and two other nucleotide analogues. Once incorporated at position i, the inhibitor caused RNA synthesis arrest at position i+3. Hence, the likely mechanism of action is delayed RNA chain termination. The additional three nucleotides may protect the inhibitor from excision by the viral 3'-5' exonuclease activity. Together, these results help to explain the high potency of RDV against RNA viruses in cell-based assays.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-16</span></i><p style="text-align: justify; text-justify: inter-word;">At the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China1, even more spread than SARS-CoV infection2. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion3,4. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear5,6. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world.With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world. With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Corona Virus Disease 2019 (COVID-19) is a recently emerged life-threatening disease caused by SARS-CoV-2. Real- time fluorescent PCR (RT-PCR) is the clinical standard for SARS-CoV-2 nucleic acid detection. To detect SARS-CoV-2 early and control the disease spreading on time, a faster and more convenient method for SARS-CoV-2 nucleic acid detecting, RT-LAMP method (reverse transcription loop-mediated isothermal amplification) was developed. RNA reverse transcription and nucleic acid amplification were performed in one step at 63 ℃ isothermal conditions, and the results can be obtained within 30 minutes. ORF1ab gene, E gene and N gene were detected at the same time. ORF1ab gene was very specific and N gene was very sensitivity, so they can guarantee both sensitivity and specificity for SARS-CoV-2. The sensitivity of RT-LAMP assay is similar to RT-PCR, and specificity was 99% as detecting 208 clinical specimens. The RT-LAMP assay reported here has the advantages of rapid amplification, simple operation, and easy detection, which is useful for the rapid and reliable clinical diagnosis of SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic diversity of SARS-CoV-2 in Coronavirus Disease 2019 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) has infected more than 75,000 individuals and spread to over 20 countries. It is still unclear how fast the virus evolved and how the virus interacts with other microorganisms in the lung.We have conducted metatranscriptome sequencing for the bronchoalveolar lavage fluid of eight SARS-CoV-2 patients, 25 community-acquired pneumonia (CAP) patients, and 20 healthy controls.The median number of intra-host variants was 1-4 in SARS-CoV-2 infected patients, which ranged between 0 and 51 in different samples. The distribution of variants on genes was similar to those observed in the population data (110 sequences). However, very few intra-host variants were observed in the population as polymorphism, implying either a bottleneck or purifying selection involved in the transmission of the virus, or a consequence of the limited diversity represented in the current polymorphism data. Although current evidence did not support the transmission of intra-host variants in a person-to-person spread, the risk should not be overlooked. The microbiota in SARS-CoV-2 infected patients was similar to those in CAP, either dominated by the pathogens or with elevated levels of oral and upper respiratory commensal bacteria.SARS-CoV-2 evolves in vivo after infection, which may affect its virulence, infectivity, and transmissibility. Although how the intra-host variant spreads in the population is still elusive, it is necessary to strengthen the surveillance of the viral evolution in the population and associated clinical changes.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>A Peptide-based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Corona Virus Disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A respiratory illness has been spreading rapidly in China, since its outbreak in Wuhan city, Hubei province in December 2019. The illness was caused by a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Clinical manifestations related to SARS-CoV-2 infection ranged from no symptom to fatal pneumonia. World Health Organization (WHO) named the diseases associated with SARS-CoV-2 infection as COVID-19. Real time RT-PCR is the only laboratory test available till now to confirm the infection. However, the accuracy of real time RT-PCR depends on many factors, including sampling location and of methods, quality of RNA extraction and training of operators etc. Variations in these factors might significantly lower the sensitivity of the detection. We developed a peptide-based luminescent immunoassay to detect IgG and IgM. Cut-off value of this assay was determined by the detection of 200 healthy sera and 167 sera from patients infected with other pathogens than SARS-CoV-2. To evaluate the performance of this assay, we detected IgG and IgM in the 276 sera from confirmed patients. The positive rate of IgG and IgM were 71.4% (197/276) and 57.2% (158/276) respectively. By combining with real time RT-PCR detection, this assay might help to enhance the accuracy of diagnosis of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular immune pathogenesis and diagnosis of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is a kind of viral pneumonia with an unusual outbreak in Wuhan, China, in December 2019, which is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The emergence of SARS-CoV-2 has been marked as the third introduction of a highly pathogenic coronavirus into the human population after the severe acute respiratory syndrome coronavirus (SARS-CoV) and the Middle East respiratory syndrome coronavirus (MERS-CoV) in the twenty-first century. In this minireview, we provide a brief introduction of the general features of SARS-CoV-2 and discuss current knowledge of molecular immune pathogenesis, diagnosis and treatment of COVID-19 on the base of the present understanding of SARS-CoV and MERS-CoV infections, which may be helpful in offering novel insights and potential therapeutic targets for combating the SARS-CoV-2 infection.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>The within-host viral kinetics of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus  broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Multivariate Analyses of Codon Usage of SARS-CoV-2 and other betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus which also includes severe acute respiratory syndrome related coronavirus (SARSr-CoV) and Middle East respiratory syndrome related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses can help identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, global correspondence analysis (CA), within-group correspondence analysis (WCA) and between-group correspondence analysis (BCA) were performed among different genes in coronavirus viral sequences. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP).  We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS.  We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene.  Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of coronavirus disease (COVID-19) in China caused by SARS-CoV-2 virus continually lead to worldwide human infections and deaths. It is currently no specific viral protein targeted therapeutics yet. Viral nucleocapsid protein is a potential antiviral drug target, serving multiple critical functions during the viral life cycle. However, the structural information of SARS-CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the crystal structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. Although overall structure is similar with other reported coronavirus nucleocapsid protein N-terminal domain, the surface electrostatic potential characteristics between them are distinct. Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique potential RNA binding pocket alongside the beta-sheet core. Complemented by in vitro binding studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to SARS-CoV-2.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Mucin 4 Protects Female Mice from Coronavirus Pathogenesis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">Using incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4-/- mice and found that female, but not male, Muc4-/- mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4-/- mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4s impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis.  ImportanceGiven the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the world is witnessing the epidemic of COVID-19, a disease caused by a novel coronavirus, SARS-CoV-2, emerging genetics and clinical evidences suggest a similar path to those of SARS and MERS. The rapid genomic sequencing and open access data, together with advanced vaccine technology, are expected to give us more knowledge on the pathogen itself, including the host immune response as well as the plan for therapeutic vaccines in the near future. This review aims to provide a comparative view among SARS-CoV, MERS-CoV and the newly epidemic SARS-CoV-2, in the hope to gain a better understanding of the host-pathogen interaction, host immune responses, and the pathogen immune evasion strategies. This predictive view may help in designing an immune intervention or preventive vaccine for COVID-19 in the near future.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosis of Acute Respiratory Syndrome Coronavirus 2 Infection by Detection of Nucleocapsid Protein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND Nucleic acid test and antibody assay have been employed in the diagnosis for SARS-CoV-2 infection, but the use of viral antigen for diagnosis has not been successfully developed.  Theoretically, viral antigen is the specific marker of the virus and precedes antibody appearance within the infected population. There is a clear need of detection of viral antigen for rapid and early diagnosis. METHODS We included a cohort of 239 participants with suspected SARS-CoV-2 infection from 7 centers for the study. We measured nucleocapsid protein in nasopharyngeal swab samples in parallel with the nucleic acid test. Nucleic acid test was taken as the reference standard, and statistical evaluation was taken in blind. We detected nucleocapsid protein in 20 urine samples in another center, employing nasopharyngeal swab nucleic acid test as reference standard. RESULTS We developed a fluorescence immunochromatographic assay for detecting nucleocapsid protein of SARS-CoV-2 in nasopharyngeal swab sample and urine within 10 minutes.  100% of nucleocapsid protein positive and negative participants accord with nucleic acid test for same samples. Further, earliest participant after 3 days of fever can be identified by the method.  In an additional preliminary study, we detected nucleocapsid protein in urine in 73.6% of diagnosed COVID-19 patients.  CONCLUSIONS Those findings indicate that nucleocapsid protein assay is an accurate, rapid, early and simple method for diagnosis of COVID-19. Appearance of nucleocapsid protein in urine coincides our finding of the SARS-CoV-2 invading kidney and might be of diagnostic value.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 31st December 2019, the World Health Organization was informed of a cluster of cases of pneumonia of unknown etiology in Wuhan, China. Subsequent investigations identified a novel coronavirus, now named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), from the affected patients. Highly sensitive and specific laboratory diagnostics are important for controlling the rapidly evolving SARS-CoV-2-associated Coronavirus Disease 2019 (COVID-19) epidemic. In this study, we developed and compared the performance of three novel real-time RT-PCR assays targeting the RNA-dependent RNA polymerase (RdRp)/helicase (Hel), spike (S), and nucleocapsid (N) genes of SARS-CoV-2 with that of the reported RdRp-P2 assay which is used in >30 European laboratories. Among the three novel assays, the COVID-19-RdRp/Hel assay had the lowest limit of detection in vitro (1.8 TCID50/ml with genomic RNA and 11.2 RNA copies/reaction with in vitro RNA transcripts). Among 273 specimens from 15 patients with laboratory-confirmed COVID-19 in Hong Kong, 77 (28.2%) were positive by both the COVID-19-RdRp/Hel and RdRp-P2 assays. The COVID-19-RdRp/Hel assay was positive for an additional 42 RdRd-P2-negative specimens [119/273 (43.6%) vs 77/273 (28.2%), P<0.001], including 29/120 (24.2%) respiratory tract specimens and 13/153 (8.5%) non-respiratory tract specimens. The mean viral load of these specimens was 3.21*104 RNA copies/ml (range, 2.21*102 to 4.71*105 RNA copies/ml). The COVID-19-RdRp/Hel assay did not cross-react with other human-pathogenic coronaviruses and respiratory pathogens in cell culture and clinical specimens, whereas the RdRp-P2 assay cross-reacted with SARS-CoV in cell culture. The highly sensitive and specific COVID-19-RdRp/Hel assay may help to improve the laboratory diagnosis of COVID-19.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical characteristics of 82 death cases with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center.  Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms.  Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of &gt;5 (94.5%), high systemic immune-inflammation index of &gt;500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (&gt;10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed.  Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Machine learning-based CT radiomics model for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection: A multicenter study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Objectives To develop and test machine learning-based CT radiomics models for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection. Design Cross-sectional Setting Multicenter Participants A total of 52 patients with laboratory-confirmed SARS-CoV-2 infection and their initial CT images were enrolled from 5 designated hospitals in Ankang, Lishui, Zhenjiang, Lanzhou, and Linxia between January 23, 2020 and February 8, 2020. As of February 20, patients remained in hospital or with non-findings in CT were excluded. Therefore, 31 patients with 72 lesion segments were included in the final analysis.  Intervention CT radiomics models based on logistic regression (LR) and random forest (RF) were developed on features extracted from pneumonia lesions in training and inter-validation datasets. The predictive performance was further evaluated in test dataset on lung lobe- and patients-level. Main outcomes Short-term hospital stay (≤10 days) and long-term hospital stay (&gt;10 days). Results The CT radiomics models based on 6 second-order features were effective in discriminating short- and long-term hospital stay in patients with pneumonia associated with SARS-CoV-2 infection, with areas under the curves of 0.97 (95%CI 0.83-1.0) and 0.92 (95%CI 0.67-1.0) by LR and RF, respectively, in the test dataset. The LR model showed a sensitivity and specificity of 1.0 and 0.89, and the RF model showed similar performance with sensitivity and specificity of 0.75 and 1.0 in test dataset.  Conclusions  The machine learning-based CT radiomics models showed feasibility and accuracy for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the Recent 2019 Novel Coronavirus (SARS-CoV-2) in Light of Past Human Coronavirus Outbreaks</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are RNA viruses that have become a major public health concern since the Severe Acute Respiratory Syndrome-CoV (SARS-CoV) outbreak in 2002. The continuous evolution of coronaviruses was further highlighted with the emergence of the Middle East Respiratory Syndrome-CoV (MERS-CoV) outbreak in 2012. Currently, the world is concerned about the 2019 novel CoV (SARS-CoV-2) that was initially identified in the city of Wuhan, China in December 2019. Patients presented with severe viral pneumonia and respiratory illness. The number of cases has been mounting since then. As of late February 2020, tens of thousands of cases and several thousand deaths have been reported in China alone, in addition to thousands of cases in other countries. Although the fatality rate of SARS-CoV-2 is currently lower than SARS-CoV, the virus seems to be highly contagious based on the number of infected cases to date. In this review, we discuss structure, genome organization, entry of CoVs into target cells, and provide insights into past and present outbreaks. The future of human CoV outbreaks will not only depend on how the viruses will evolve, but will also depend on how we develop efficient prevention and treatment strategies to deal with this continuous threat.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>siVirus: web-based antiviral siRNA design software for highly divergent viral sequences</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Jul 1</span></i><p style="text-align: justify; text-justify: inter-word;">siVirus () is a web-based online software system that provides efficient short interfering RNA (siRNA) design for antiviral RNA interference (RNAi). siVirus searches for functional, off-target minimized siRNAs targeting highly conserved regions of divergent viral sequences. These siRNAs are expected to resist viral mutational escape, since their highly conserved targets likely contain structurally/functionally constrained elements. siVirus will be a useful tool for designing optimal siRNAs targeting highly divergent pathogens, including human immunodeficiency virus (HIV), hepatitis C virus (HCV), influenza virus and SARS coronavirus, all of which pose enormous threats to global human health.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease. AVAILABILITY: https://www.genomedetective.com/app/typingtool/cov. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple laboratory parameter facilitates early identification of COVID-19 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Inhibition, Escape, and Attenuated Growth of Severe Acute Respiratory Syndrome Coronavirus Treated with Antisense Morpholino Oligomers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recently emerged severe acute respiratory syndrome coronavirus (SARS-CoV) is a potent pathogen of humans and is capable of rapid global spread. Peptide-conjugated antisense morpholino oligomers (P-PMO) were designed to bind by base pairing to specific sequences in the SARS-CoV (Tor2 strain) genome. The P-PMO were tested for their capacity to inhibit production of infectious virus as well as to probe the function of conserved viral RNA motifs and secondary structures. Several virus-targeted P-PMO and a random-sequence control P-PMO showed low inhibitory activity against SARS coronavirus. Certain other virus-targeted P-PMO reduced virus-induced cytopathology and cell-to-cell spread as a consequence of decreasing viral amplification. Active P-PMO were effective when administered at any time prior to peak viral synthesis and exerted sustained antiviral effects while present in culture medium. P-PMO showed low nonspecific inhibitory activity against translation of nontargeted RNA or growth of the arenavirus lymphocytic choriomeningitis virus. Two P-PMO targeting the viral transcription-regulatory sequence (TRS) region in the 5′ untranslated region were the most effective inhibitors tested. After several viral passages in the presence of a TRS-targeted P-PMO, partially drug-resistant SARS-CoV mutants arose which contained three contiguous base point mutations at the binding site of a TRS-targeted P-PMO. Those partially resistant viruses grew more slowly and formed smaller plaques than wild-type SARS-CoV. These results suggest PMO compounds have powerful therapeutic and investigative potential toward coronavirus infection.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">hina continues to fight the outbreak of a novel coronavirus, called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which has infected tens of thousands and killed more than 2,100 people since the end of 2019. The outbreak’s hot spot is in the central province of Hubei, in particular, its capital city Wuhan, where there have been more than 62,000 infections and 2,029 deaths as of last week, according to the World Health Organization (WHO). There, doctors and scientists are desperately struggling to find reliable ways to diagnose infected patients to help treat them and control the spread of the virus. The struggle has involved two diagnostic options—computed tomography (CT) scans of patients’ lungs and a nucleic acid lab test—each with advantages and disadvantages. With “a suddenly rampant new virus, it is normal to have multiple test approaches,” says a virologist at the Chinese Center for Disease Control and Prevention</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Immunoglobulin Domain Proteins Provide Insights into Evolution and Pathogenesis Mechanisms of SARS-Related Coronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) is the causative agent of an emergent severe respiratory disease (COVID-19) in humans that is threatening to result in a global health crisis. By using genomic, sequence, structural and evolutionary analysis, we show that Alpha- and Beta-CoVs possess several novel families of immunoglobulin (Ig) domain proteins, including ORF8 and ORF7a from SARS-related coronaviruses and two protein groups from certain Alpha-CoVs. Among them, ORF8 is distinguished in being rapidly evolving, possessing a unique insert and a hypervariable position among SARS-CoV-2 genomes in its predicted ligand-binding groove. We also uncover many Ig proteins from several metazoan viruses which are distinct in sequence and structure but share an architecture comparable to that of CoV Ig domain proteins. Hence, we propose that deployment of Ig domain proteins is a widely-used strategy by viruses, and SARS-CoV-2 ORF8 is a potential pathogenicity factor which evolves rapidly to counter the immune response and facilitate the transmission between hosts.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a quantitative RT-PCR assay for the detection of the emerging coronavirus SARS-CoV-2 using a high throughput system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ability to quickly confirm or clear suspected cases is crucial during global outbreak scenarios, especially when clinical manifestations are difficult to distinguish from other respiratory infections such as influenza, molecular diagnostics is key for detection of the emerging virus. A variety of suitable assays were made available early on during the course of the outbreak, notably by Corman et al. and others [4,5]. However, their implementation in the diagnostics laboratory usually relies on manual PCR setups requiring a high degree of human interaction for execution and interpretation, thus limiting their capacity to be scaled up for handling large numbers of samples. In this study we report the analytical evaluation of a laboratory-developed test for the detection of SARS-CoV-2 using the open channel (utility channel) of the cobas 6800 system.%U https://eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.9.2000152</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19.  Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Highly ACE2 Expression in Pancreas May Cause Pancreas Damage After SARS-CoV-2 Infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) started in the end of 2019 in China has triggered a global public health crisis. Previous studies have shown that SARS-CoV-2 infects cells by binding angiotensin-converting enzyme 2 (ACE2), which is the same as SARS-CoV. The expression and distribution of ACE2 in the pancreas are unknown. At the same time, the injury of pancreas after SARS-CoV-2 infection has not been concerned. Here, we collected public datasets (bulk RNA-seq and single-cell RNA-seq) to indicate the expression and the distribution of ACE2 in pancreas (in both exocrine glands and islets). And further, clinical data including mild and severe patients with COVID-19 demonstrated there existed mild pancreatitis. In the 67 severe cases, 11 patients (16.41%) showed elevated levels of both amylase and lipase, and 5 patients (7.46%) showed imaging alterations. Only one patient (1.85%) showed elevated levels of both amylase and lipase in 54 mild cases, without imaging changes. Our study revealed the phenomenon and possible cause of mild pancreatic injury in patients with COVID-19. This suggests that pancreatitis after SARS-CoV-2 infection should also be paid attention in clinical work.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Background: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Diagnostic test based on real-time reverse transcription polymerase chain reaction assay (qRT-PCR) was the main means of confirmation, and sample collection was mostly throat swabs, which was easy to miss the diagnosis. It is necessary to seek specimen types with higher detection efficiency and accuracy. Methods: Paired specimens of throat swabs and sputum were obtained from 54 cases, and RNA was extracted and tested for 2019-nCoV (equated with SARS-CoV-2) by qRT-PCR assay. Results: The positive rates of 2019-nCoV from sputum specimens and throat swabs were 76.9% and 44.2%, respectively. Sputum specimens showed a significantly higher positive rate than throat swabs in detecting viral nucleic acid using qRT-PCR assay (P=0.001). Conclusions: The detection rates of 2019-nCoV from sputum specimens are significantly higher than throat swabs. We suggest that sputum would benefit for the detection of 2019-nCoV in patients who produce sputum. The results can facilitate the selection of specimens and increase the accuracy of diagnosis.</jats:p></p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved Algorithmic Complexity for the 3SEQ Recombination Detection Algorithm</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 3</span></i><p style="text-align: justify; text-justify: inter-word;">Identifying recombinant sequences in an era of large genomic databases is challenging as it requires an efficient algorithm to identify candidate recombinants and parents, as well as appropriate statistical methods to correct for the large number of comparisons performed. In 2007, a computation was introduced for an exact nonparametric mosaicism statistic that gave high-precision P values for putative recombinants. This exact computation meant that multiple-comparisons corrected P values also had high precision, which is crucial when performing millions or billions of tests in large databases. Here, we introduce an improvement to the algorithmic complexity of this computation from O(mn(3)) to O(mn(2)), where m and n are the numbers of recombination-informative sites in the candidate recombinant. This new computation allows for recombination analysis to be performed in alignments with thousands of polymorphic sites. Benchmark runs are presented on viral genome sequence alignments, new features are introduced, and applications outside recombination analysis are discussed.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-10-05</span></i><p style="text-align: justify; text-justify: inter-word;">The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19 A mathematical model for simulating the phase-based transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125?nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is >?96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1].ID - Fisher2020 As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus and the race to distribute reliable diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">International teams worked at speed to make tests for the virus available in record time. The medical community is rallying to develop a set of rapid and reliable molecular diagnostic tests for the new human coronavirus that appeared in China — now dubbed sudden acute respiratory syndrome coronavirus-2 (SARS-CoV-2).</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome-wide data inferring the evolution and population demography of the novel pneumonia coronavirus (SARS-CoV-2)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, coronavirus disease 2019 (COVID-19) emerged in Wuhan, Central China and rapidly spread throughout China. Up to March 3, 2020, SARS-CoV-2 has infected more than 89,000 people in China and other 66 countries across six continents. In this study, we used 10 new sequenced genomes of SARS-CoV-2 and combined 136 genomes from GISAID database to investigate the genetic variation and population demography through different analysis approaches (e.g. Network, EBSP, Mismatch, and neutrality tests) in the previous three months. The results showed that eighty haplotypes had 183 substitution sites, including 27 parsimony-informative and 156 singletons. Sliding window analyses of genetic diversity suggested a certain range in mutations and variation in genomic abundance of SARS-CoV-2, which could be explaining the existing widespread and high adaptation of the deadly virus. Phylogenetic analysis showed that pangolin may not be an intermediate host. The network indicated that, in the original haplotype (H14), one patient sample lived near the Huanan seafood market (approximately 2 km), which indicating high possibility of the patient having a history of unconscious contact with this market. However, based on this clue, we cannot accurately conclude that whether this market was the origin center of SARS-CoV-2. Additionally, 16 genomes, collected from this market, assigned to 10 haplotypes, indicated a circulating infection within the market in a short term, which would have led to the outbreak of SARS-CoV-2 in Wuhan and other areas. The EBSP results showed that the first estimated expansion date began from 7 December 2019, which indicated that the transmission of SARS-CoV-2 could have begun from person to person in mid to late November.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Summary Background The novel coronavirus SARS-CoV-2 is a newly emerging virus. The antibody response in infected patient remains largely unknown, and the clinical values of antibody testing have not been fully demonstrated. Methods A total of 173 patients with confirmed SARS-CoV-2 infection were enrolled. Their serial plasma samples (n = 535) collected during the hospitalization period were tested for total antibodies (Ab), IgM and IgG against SARS-CoV-2 using immunoassays. The dynamics of antibodies with the progress and severity of disease was analyzed. Findings Among 173 patients, the seroconversion rate for Ab, IgM and IgG was 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively. Twelve patients who had not seroconverted were those only blood samples at the early stage of illness were collected. The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. The presence of antibodies was &lt; 40% among patients in the first 7 days of illness, and then rapidly increased to 100.0%, 94.3% and 79.8% for Ab, IgM and IgG respectively since day 15 after onset. In contrast, the positive rate of RNA decreased from 66.7% (58/87) in samples collected before day 7 to 45.5% (25/55) during days 15 to 39. Combining RNA and antibody detections significantly improved the sensitivity of pathogenic diagnosis for COVID-19 patients (p &lt; 0.001), even in early phase of 1-week since onset (p = 0.007). Moreover, a higher titer of Ab was independently associated with a worse clinical classification (p = 0.006). Interpretation The antibody detection offers vital clinical information during the course of SARS-CoV-2 infection. The findings provide strong empirical support for the routine application of serological testing in the diagnosis and management of COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico study of the spike protein from SARS-CoV-2 interaction with ACE2: similarity with SARS-CoV, hot-spot analysis and effect of the receptor polymorphism</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The spread of the COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor via which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. We used homology-based protein-protein docking, binding energy estimation, and decomposition, aiming to predict both qualitative and quantitative aspects of the interaction. Using in silico structural modelling, we brought additional evidence that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than mutation accumulation. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to hACE2 and maintaining the stability of the interface. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Although the SARS-CoV-2 viral load detection of respiratory specimen has been widely used for novel coronavirus disease (COVID-19) diagnosis, it is undeniable that serum SARS-CoV-2 nucleic acid (RNAaemia) could be detected  in  a fraction of the COVID-19 patients. However, it is not clear that if the incidence of RNAaemia could be correlated with the occurrence of cytokine storm or with the specific class of patients.  Methods: This study enrolled 48 patients with COVID-19 admitted to the General Hospital of Central Theater Command, PLA, a designated hospital in Wuhan, China. The patients were divided into three groups according to the Diagnosis and Treatment of New Coronavirus Pneumonia (version 6) published by the National Health Commission of China. The clinical and laboratory data were collected. The serum viral load detection and serum IL-6 levels were determined. Except for routine statistical analysis, Generalized Linear Models (GLMs) analysis was used to establish a patient status prediction model based on real-time RT-PCR Ct value. Findings: The Result showed that cases with RNAaemia were exclusively confirmed in critically ill patients group and appeared to reflect the illness severity. Further more, the inflammatory cytokine IL-6 levels were significantly elevated in critically ill patients, which is almost 10-folds higher than those in other patients. More importantly, the extremely high IL-6 level was closely correlated with the incidence of RNAaemia (R=0.902) and the vital signs of COVID-19 patients (R= -0.682).  Interpretation:    Serum SARS-CoV-2  viral load (RNAaemia) is strongly associated with cytokine storm and can be used to  predict the  poor prognosis of COVID-19 patients. Moreover, our results strongly suggest that cytokine IL-6 should be considered as a therapeutic target in critically ill patients with excessive inflammatory response.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico Design of novel Multi-epitope recombinant Vaccine based on Coronavirus surface glycoprotein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">It is of special significance to find a safe and effective vaccine against coronavirus disease 2019 (COVID-19) that can induce T cell and B cell -mediated immune responses. There is currently no vaccine to prevent COVID-19. In this project, a novel multi-epitope vaccine for COVID-19 virus based on surface glycoprotein was designed through application of bioinformatics methods. At the first, seventeen potent linear B-cell and T-cell binding epitopes from surface glycoprotein were predicted in silico, then the epitopes were joined together via different linkers. The immunogenicity of these epitopes was identified using IFN-{gamma} ELIspot assays. The IFN-{gamma} producing T cell variation ranged from 11.1 {+/-}1.2 SFU to 38.2 {+/-} 2.1 SFU per 10 6 PBMCs. One final vaccine was constructed which composed of 398 amino acids and attached to 50S ribosomal protein L7/L12 as adjuvant. Physicochemical properties, as well as antigenicity in the proposed vaccines, were checked for defining the vaccine stability and its ability to induce cell-mediated immune responses. Three-dimensional structure of the mentioned vaccine was subjected to the molecular docking studies with MHC-I and MHC-II molecules. The results proposed that the multi-epitope vaccine with 50S ribosomal protein L7/L12 was very stable with high aliphatic content and high antigenicity.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Computers and viral diseases. Preliminary bioinformatics studies on the design of a synthetic vaccine and a preventative peptidomimetic antagonist against the SARS-CoV-2 (2019-nCoV, COVID-19) coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The outbreak of Coronavirus Disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, December 2019, and continuously poses a serious threat to public health. Our previous study has shown that cytokine storm occurred during SARS-CoV-2 infection, while the detailed role of cytokines in the disease severity and progression remained unclear due to the limited case number. In this study, we examined 48 cytokines in the plasma samples from 53 COVID-19 cases, among whom 34 were severe cases, and the others moderate. Results showed that 14 cytokines were significantly elevated upon admission in COVID-19 cases. Moreover, IP-10, MCP-3, and IL-1ra were significantly higher in severe cases, and highly associated with the PaO2/FaO2 and Murray score. Furthermore, the three cytokines were independent predictors for the progression of COVID-19, and the combination of IP-10, MCP-3 and IL-1ra showed the biggest area under the curve (AUC) of the receiver-operating characteristics (ROC) calculations. Serial detection of IP-10, MCP-3 and IL-1ra in 14 severe cases showed that the continuous high levels of these cytokines were associated with disease deterioration and fatal outcome. In conclusion, we report three cytokines that closely associated with disease severity and outcome of COVID-19. These findings add to our understanding of the immunopathologic mechanisms of SARS-CoV-2 infection, which suggested novel therapeutic targets and strategy.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovering drugs to treat coronavirus disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 virus emerged in December 2019 and then spread rapidly worldwide, particularly to China, Japan, and South Korea. Scientists are endeavoring to find antivirals specific to the virus. Several drugs such as chloroquine, arbidol, remdesivir, and favipiravir are currently undergoing clinical studies to test their efficacy and safety in the treatment of coronavirus disease 2019 (COVID-19) in China; some promising results have been achieved thus far. This article summarizes agents with potential efficacy against SARS-CoV-2.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The outbreak of the recently emerged novel corona virus disease 2019 (COVID-19) poses a challenge for public health laboratories. We aimed to evaluate the diagnostic value of serological assay for SARS-CoV-2.  Methods A newly-developed ELISA assay for IgM and IgG antibodies against N protein of SARS-CoV-2 were used to screen the serums of 238 admitted hospital patients with confirmed or suspected SARS-CoV-2 infection from February 6 to February 14, 2020. SARS-CoV-2 RNA was detected by real time RT-PCR on pharyngeal swab specimens. Findings Of the 238 patients, 194 (81.5%) were detected to be antibody (IgM and/or IgG) positive, which was significantly higher than the positive rate of viral RNA (64.3%). There was no difference in the positive rate of antibody between the confirmed patients (83.0%, 127/153) and the suspected patients (78.8%, 67/85) whose nucleic acid tests were negative. After the patients were defined to the different stages of disease based on the day when the test samples were collected, the analysis results showed that the antibody positive rates were very low in the first five days after initial onset of symptoms, and then rapidly increased as the disease progressed. After 10 days, the antibody positive rates jumped to above 80% from less than 50%. On the contrary, the positive rates of viral RNA kept above 60% in the first 11 days after initial onset of symptoms, and then rapidly decreased. In addition, half of the suspected patients with symptoms for 6-10 days were detected to be antibody positive. Interpretation The suspected patients were most likely infected by SARS-CoV-2. Before the 11th day after initial onset of symptoms, nucleic acid test is important for confirmation of viral infection. The combination of serological assay can greatly improve the diagnostic efficacy. After that, the diagnosis for viral infection should be majorly dependent on serological assay. Keywords. SARS-CoV-2; diagnosis; serological assay; nucleic acid test</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Construction of murine coronavirus mutants containing interspecies chimeric nucleocapsid proteins.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Targeted RNA recombination was used to construct mouse hepatitis virus (MHV) mutants containing chimeric nucleocapsid (N) protein genes in which segments of the bovine coronavirus N gene were substituted in place of their corresponding MHV sequences. This defined portions of the two N proteins that, despite evolutionary divergence, have remained functionally equivalent. These regions included most of the centrally located RNA-binding domain and two putative spacers that link the three domains of the N protein. By contrast, the amino terminus of N, the acidic carboxy-terminal domain, and a serine- and arginine-rich segment of the central domain could not be transferred from bovine coronavirus to MHV, presumably because these parts of the molecule participate in protein-protein interactions that are specific for each virus (or, possibly, each host). Our results demonstrate that targeted recombination can be used to make extensive substitutions in the coronavirus genome and can generate recombinants that could not otherwise be made between two viruses separated by a species barrier. The implications of these findings for N protein structure and function as well as for coronavirus RNA recombination are discussed.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical outcomes of 402 patients with COVID-2019 from a single center in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The SARS-CoV-2 outbreak is causing widespread infections and significant mortality. Previous studies describing clinical characteristics of the disease contained small cohorts from individual centers or larger series consisting of mixed cases from different hospitals. We report analyses of mortality and disease severity among 402 patients from a single hospital. The cohort included 297 patients with confirmed and 105 with suspected diagnosis. The latter group met the criteria for clinical diagnosis but nucleic acid tests results were initially interpreted as suspicious. Data were compared between genders and among different age groups. The overall case fatality is 5.2%. However, patients 70 years of age or older suffered a significantly higher mortality (17.8%), associated with more patients having severe or critical illness (57.5%). Patients 50 years of age or older had a mortality of 8.0%, and those younger than 50 years, 1.2%. Male patients had a mortality of 7.6% versus 2.9% in females.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus disease (COVID-19) and neonate: What neonatologist need to know</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About 3 neonates and more than 230 children cases are reported. The disease condition of mainly children was mild. There is currently no evidence that SARS-CoV-2can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID-19) is based on adult experience. Thus far, no deaths have been reported in the paediatric age group. This review describes the current understanding of COVID-19 infection in newborns and children. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiplex Degenerate Primer Design for Targeted Whole Genome Amplification of Many Viral Genomes</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 3</span></i><p style="text-align: justify; text-justify: inter-word;">Background. Targeted enrichment improves coverage of highly mutable viruses at low concentration in complex samples. Degenerate primers that anneal to conserved regions can facilitate amplification of divergent, low concentration variants, even when the strain present is unknown. Results. A tool for designing multiplex sets of degenerate sequencing primers to tile overlapping amplicons across multiple whole genomes is described. The new script, run_tiled_primers, is part of the PriMux software. Primers were designed for each segment of South American hemorrhagic fever viruses, tick-borne encephalitis, Henipaviruses, Arenaviruses, Filoviruses, Crimean-Congo hemorrhagic fever virus, Rift Valley fever virus, and Japanese encephalitis virus. Each group is highly diverse with as little as 5% genome consensus. Primer sets were computationally checked for nontarget cross reactions against the NCBI nucleotide sequence database. Primers for murine hepatitis virus were demonstrated in the lab to specifically amplify selected genes from a laboratory cultured strain that had undergone extensive passage in vitro and in vivo. Conclusions. This software should help researchers design multiplex sets of primers for targeted whole genome enrichment prior to sequencing to obtain better coverage of low titer, divergent viruses. Applications include viral discovery from a complex background and improved sensitivity and coverage of rapidly evolving strains or variants in a gene family.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Clinical Application of A Rapid IgM-IgG Combined Antibody Test for SARS-CoV-2 Infection Diagnosis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of the novel coronavirus disease (COVID-19) quickly spread all over China and to more than 20 other countries. Although the virus (SARS-Cov-2) nucleic acid RT-PCR test has become the standard method for diagnosis of SARS-CoV-2 infection, these real-time PCR test kits have many limitations. In addition, high false negative rates were reported. There is an urgent need for an accurate and rapid test method to quickly identify large number of infected patients and asymptomatic carriers to prevent virus transmission and assure timely treatment of patients. We have developed a rapid and simple point-of-care lateral flow immunoassay which can detect IgM and IgG antibodies simultaneously against SARS-CoV-2 virus in human blood within 15 minutes which can detect patients at different infection stages. With this test kit, we carried out clinical studies to validate its clinical efficacy uses. The clinical detection sensitivity and specificity of this test were measured using blood samples collected from 397 PCR confirmed COVID-19 patients and 128 negative patients at 8 different clinical sites. The overall testing sensitivity was 88.66% and specificity was 90.63%. In addition, we evaluated clinical diagnosis results obtained from different types of venous and fingerstick blood samples. The results indicated great detection consistency among samples from fingerstick blood, serum and plasma of venous blood. The IgM-IgG combined assay has better utility and sensitivity compared with a single IgM or IgG test. It can be used for the rapid screening of SARS-CoV-2 carriers, symptomatic or asymptomatic, in hospitals, clinics, and test laboratories. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS coronavirus replicase proteins in pathogenesis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Much progress has been made in understanding the role of structural and accessory proteins in the pathogenesis of severe acute respiratory syndrome coronavirus (SARS-CoV) infections. The SARS epidemic also brought new attention to the proteins translated from ORF1a and ORF1b of the input genome RNA, also known as the replicase/transcriptase gene. Evidence for change within the ORF1ab coding sequence during the SARS epidemic, as well as evidence from studies with other coronaviruses, indicates that it is likely that the ORF1ab proteins play roles in virus pathogenesis distinct from or in addition to functions directly involved in viral replication. Recent reverse genetic studies have confirmed that proteins of ORF1ab may be involved in cellular signaling and modification of cellular gene expression, as well as virulence by mechanisms yet to be determined. Thus, the evolution of the ORF1ab proteins may be determined as much by issues of host range and virulence as they are by specific requirements for intracellular replication.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulate_PCR for amplicon prediction and annotation from multiplex, degenerate primers and probes</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jul 9</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Pairing up primers to amplify desired targets and avoid undesired cross reactions can be a combinatorial challenge. Effective prediction of specificity and inclusivity from multiplexed primers and TaqMan®/Luminex® probes is a critical step in PCR design. RESULTS: Code is described to identify all primer and probe combinations from a list of unpaired, unordered candidates that should produce a product. It predicts and extracts all amplicon sequences in a large sequence database from a list of primers and probes, allowing degenerate bases and user-specified levels of primer-target mismatch tolerance. Amplicons hit by TaqMan®/Luminex® probes are indicated, and products may be annotated with gene information from NCBI. Fragment length distributions are calculated to predict electrophoretic gel banding patterns. CONCLUSIONS: Simulate_PCR is the only freely available software that can be run from the command line for high throughput applications which can calculate all products from large lists of primers and probes compared to a large sequence database such as nt. It requires no prior knowledge of how primers should be paired. Degenerate bases are allowed and entire amplicon sequences are extracted and annotated with gene information. Examples are provided for sets of TaqMan®/Luminex® PCR signatures predicted to amplify all HIV-1 genomes, all Coronaviridae genomes, and a group of antibiotic resistance genes. The software is a command line perl script freely available as open source.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparisons of substitution, insertion and deletion probes for resequencing and mutational analysis using oligonucleotide microarrays</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">Although oligonucleotide probes complementary to single nucleotide substitutions are commonly used in microarray-based screens for genetic variation, little is known about the hybridization properties of probes complementary to small insertions and deletions. It is necessary to define the hybridization properties of these latter probes in order to improve the specificity and sensitivity of oligonucleotide microarray-based mutational analysis of disease-related genes. Here, we compare and contrast the hybridization properties of oligonucleotide microarrays consisting of 25mer probes complementary to all possible single nucleotide substitutions and insertions, and one and two base deletions in the 9168 bp coding region of the ATM (ataxia telangiectasia mutated) gene. Over 68 different dye-labeled single-stranded nucleic acid targets representing all ATM coding exons were applied to these microarrays. We assess hybridization specificity by comparing the relative hybridization signals from probes perfectly matched to ATM sequences to those containing mismatches. Probes complementary to two base substitutions displayed the highest average specificity followed by those complementary to single base substitutions, single base deletions and single base insertions. In all the cases, hybridization specificity was strongly influenced by sequence context and possible intra- and intermolecular probe and/or target structure. Furthermore, single nucleotide substitution probes displayed the most consistent hybridization specificity data followed by single base deletions, two base deletions and single nucleotide insertions. Overall, these studies provide valuable empirical data that can be used to more accurately model the hybridization properties of insertion and deletion probes and improve the design and interpretation of oligonucleotide microarray-based resequencing and mutational analysis.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Characterization of a Highly Specific and Sensitive SYBR Green Reverse Transcriptase PCR Assay for Detection of the 2009 Pandemic H1N1 Influenza Virus on the Basis of Sequence Signatures</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and rapid spread of the 2009 H1N1 pandemic influenza virus showed that many diagnostic tests were unsuitable for detecting the novel virus isolates. In most countries the probe-based TaqMan assay developed by the U.S. Centers for Disease Control and Prevention was used for diagnostic purposes. The substantial sequence data that became available during the course of the pandemic created the opportunity to utilize bioinformatics tools to evaluate the unique sequence properties of this virus for the development of diagnostic tests. We used a comprehensive computational approach to examine conserved 2009 H1N1 sequence signatures that are at least 20 nucleotides long and contain at least two mismatches compared to any other known H1N1 genome. We found that the hemagglutinin (HA) and neuraminidase (NA) genes contained sequence signatures that are highly conserved among 2009 H1N1 isolates. Based on the NA gene signatures, we used Visual-OMP to design primers with optimal hybridization affinity and we used ThermoBLAST to minimize amplification artifacts. This procedure resulted in a highly sensitive and discriminatory 2009 H1N1 detection assay. Importantly, we found that the primer set can be used reliably in both a conventional TaqMan and a SYBR green reverse transcriptase (RT)-PCR assay with no loss of specificity or sensitivity. We validated the diagnostic accuracy of the NA SYBR green assay with 125 clinical specimens obtained between May and August 2009 in Chile, and we showed diagnostic efficacy comparable to the CDC assay. Our approach highlights the use of systematic computational approaches to develop robust diagnostic tests during a viral pandemic.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Dual priming oligonucleotide system for the multiplex detection of respiratory viruses and SNP genotyping of CYP2C19 gene</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Mar 7</span></i><p style="text-align: justify; text-justify: inter-word;">Successful PCR starts with proper priming between an oligonucleotide primer and the template DNA. However, the inevitable risk of mismatched priming cannot be avoided in the currently used primer system, even though considerable time and effort are devoted to primer design and optimization of reaction conditions. Here, we report a novel dual priming oligonucleotide (DPO) which contains two separate priming regions joined by a polydeoxyinosine linker. The linker assumes a bubble-like structure which itself is not involved in priming, but rather delineates the boundary between the two parts of the primer. This structure results in two primer segments with distinct annealing properties: a longer 5′-segment that initiates stable priming, and a short 3′-segment that determines target-specific extension. This DPO-based system is a fundamental tool for blocking extension of non-specifically primed templates, and thereby generates consistently high PCR specificity even under less than optimal PCR conditions. The strength and utility of the DPO system are demonstrated here using multiplex PCR and SNP genotyping PCR.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-28</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the Corona Virus Disease 2019 (COVID-19) cases in China has become a public health emergency of international concern (PHEIC). Based on angiotensin converting enzyme 2 (ACE2) as cell entry receptor of SARS-CoV, we used the hACE2 transgenic mice infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant lymphocytes and monocytes in alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, alveolar macrophages and alveolar epithelia. The phenomenon was not found in wild type mice with SARS-CoV-2 infection. The pathogenicity of SARS-CoV-2 in hACE2 mice was clarified and the Kochs postulates were fulfilled as well, and the mouse model may facilitate the development of therapeutics and vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Cooperative translocation enhances the unwinding of duplex DNA by SARS coronavirus helicase nsP13</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Nov 29</span></i><p style="text-align: justify; text-justify: inter-word;">SARS coronavirus encodes non-structural protein 13 (nsP13), a nucleic acid helicase/NTPase belonging to superfamily 1 helicase, which efficiently unwinds both partial-duplex RNA and DNA. In this study, unwinding of DNA substrates that had different duplex lengths and 5′-overhangs was examined under single-turnover reaction conditions in the presence of excess enzyme. The amount of DNA unwound decreased significantly as the length of the duplex increased, indicating a poor in vitro processivity. However, the quantity of duplex DNA unwound increased as the length of the single-stranded 5′-tail increased for the 50-bp duplex. This enhanced processivity was also observed for duplex DNA that had a longer single-stranded gap in between. These results demonstrate that nsP13 requires the presence of a long 5′-overhang to unwind longer DNA duplexes. In addition, enhanced DNA unwinding was observed for gapped DNA substrates that had a 5′-overhang, indicating that the translocated nsP13 molecules pile up and the preceding helicase facilitate DNA unwinding. Together with the propensity of oligomer formation of nsP13 molecules, we propose that the cooperative translocation by the functionally interacting oligomers of the helicase molecules loaded onto the 5′-overhang account for the observed enhanced processivity of DNA unwinding.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultra-deep mutant spectrum profiling: improving sequencing accuracy using overlapping read pairs</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Feb 12</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGOUND: High throughput sequencing is beginning to make a transformative impact in the area of viral evolution. Deep sequencing has the potential to reveal the mutant spectrum within a viral sample at high resolution, thus enabling the close examination of viral mutational dynamics both within- and between-hosts. The challenge however, is to accurately model the errors in the sequencing data and differentiate real viral mutations, particularly those that exist at low frequencies, from sequencing errors. RESULTS: We demonstrate that overlapping read pairs (ORP) -- generated by combining short fragment sequencing libraries and longer sequencing reads -- significantly reduce sequencing error rates and improve rare variant detection accuracy. Using this sequencing protocol and an error model optimized for variant detection, we are able to capture a large number of genetic mutations present within a viral population at ultra-low frequency levels (<0.05%). CONCLUSIONS: Our rare variant detection strategies have important implications beyond viral evolution and can be applied to any basic and clinical research area that requires the identification of rare mutations.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultra-deep mutant spectrum profiling: improving sequencing accuracy using overlapping read pairs</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Feb 12</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGOUND: High throughput sequencing is beginning to make a transformative impact in the area of viral evolution. Deep sequencing has the potential to reveal the mutant spectrum within a viral sample at high resolution, thus enabling the close examination of viral mutational dynamics both within- and between-hosts. The challenge however, is to accurately model the errors in the sequencing data and differentiate real viral mutations, particularly those that exist at low frequencies, from sequencing errors. RESULTS: We demonstrate that overlapping read pairs (ORP) -- generated by combining short fragment sequencing libraries and longer sequencing reads -- significantly reduce sequencing error rates and improve rare variant detection accuracy. Using this sequencing protocol and an error model optimized for variant detection, we are able to capture a large number of genetic mutations present within a viral population at ultra-low frequency levels (<0.05%). CONCLUSIONS: Our rare variant detection strategies have important implications beyond viral evolution and can be applied to any basic and clinical research area that requires the identification of rare mutations.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus SARS-CoV-2, etiological agent of recently named Coronavirus infected disease (COVID-19) by WHO, has caused more than 2, 000 deaths worldwide since its emergency in Wuhan City, Hubei province, China, in December, 2019. The symptoms of COVID-19 varied from modest, mild to acute respiratory distress syndrome (ARDS), and the latter of which is generally associated with deregulated immune cytokine production; however, we currently know little as to the interplay between the extent of clinical symptoms and the compositions of lung immune microenvironment. Here, we comprehensively characterized the lung immune microenvironment with the bronchoalveolar lavage fluid (BALF) from 3 severe and 3 mild COVID-19 patients and 8 previously reported healthy lung controls through single-cell RNA sequence (scRNA-seq) combined with TCR-seq. Our data shows that monocyte-derived FCN1+ macrophages, whereas notFABP4+ alveolar macrophages that represent a predominant macrophage subset in BALF from patients with mild diseases, overwhelm in the severely damaged lungs from patients with ARDS. These cells are highly inflammatory and enormous chemokine producers implicated in cytokine storm. Furthermore, the formation of tissue resident, highly expanded clonal CD8+ T cells in the lung microenvironment of mild symptom patients suggests a robust adaptive immune response connected to a better control of COVID-19. This study first reported the cellular atlas of lung bronchoalveolar immune microenvironment in COVID-19 patients at the single-cell resolution, and unveiled the potential immune mechanisms underlying disease progression and protection in COVID-19.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The course of clinical diagnosis and treatment of a case infected with coronavirus disease 2019</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract A pneumonia outbreak caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was first identified in Wuhan, present a major threat to public health since December 2019. There are more than 50,000 confirmed cases and 1300 dead cases worldwide for the past month or more, because of the occurrence of a highly contagious performance. Patients had clinical manifestations of fever, cough, shortness of breath, diarrhea, vomiting and so on. We herein report a case of SARS-CoV-2, describe the epidemic history, clinical diagnosis and the changes of clinical parameters during the combination therapy. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Computational and Experimental Studies of ADP-Ribosylation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The macrodomains are a multifunctional protein family that function as receptors and enzymes acting on poly(ADP-ribose), ADP-ribosylated proteins, and other metabolites of nicotinamide adenine dinucleotide (NAD(+)). Several new functions for macrodomains, such as nucleic acid binding and protein/protein interaction, have recently been identified in this family. Here, we discuss methods for the identification of new macrodomains in viruses and the prediction of their function. This is followed by the expression and purification of these proteins following overexpression in bacterial cells and confirmation of folding and function using biophysical methods.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Isolation from the First Patient with SARS-CoV-2 in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">Novel coronavirus (SARS-CoV-2) is found to cause a large outbreak started from Wuhan since December 2019 in China and SARS-CoV-2 infections have been reported with epidemiological linkage to China in 25 countries until now. We isolated SARS-CoV-2 from the oropharyngeal sample obtained from the patient with the first laboratory-confirmed SARS-CoV-2 infection in Korea. Cytopathic effects of SARS-CoV-2 in the Vero cell cultures were confluent 3 days after the first blind passage of the sample. Coronavirus was confirmed with spherical particle having a fringe reminiscent of crown on transmission electron microscopy. Phylogenetic analyses of whole genome sequences showed that it clustered with other SARS-CoV-2 reported from Wuhan.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /></body>