
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: tools and studies to monitor phenotypic change and potential adaptation of the virus</h2><br /></div><br /><br /><br /><div class="card"><h3>Kingdom of Saudi Arabia e ; Saudi Aramco Medical Services Organization, Saudi Aramco, Dhahran, Kingdom of Saudi Arabia f ; Prince Sultan Military Medical City, Riyadh, Kingdom of Saudi Arabia g</h3>
<p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first documented in the Kingdom of Saudi Arabia (KSA) in 2012 and, to date, has been identified in 180 cases with 43% mortality. In this study, we have determined the MERS-CoV evolutionary rate, documented genetic variants of the virus and their distribution throughout the Arabian peninsula, and identified the genome positions under positive selection, important features for monitoring adaptation of MERS-CoV to human transmission and for identifying the source of infections. Respiratory samples from confirmed KSA MERS cases from May to September 2013 were subjected to whole-genome deep sequencing, and 32 complete or partial sequences (20 were >99% complete, 7 were 50 to 94% complete, and 5 were 27 to 50% complete) were obtained, bringing the total available MERS-CoV genomic sequences to 65. An evolutionary rate of 1.12 ؋ 10 ؊3 substitutions per site per year (95% credible interval [95% CI], 8.76 ؋ 10 ؊4 ; 1.37 ؋ 10 ؊3 ) was estimated, bringing the time to most recent common ancestor to March 2012 (95% CI, December 2011; June 2012). Only one MERS-CoV codon, spike 1020, located in a domain required for cell entry, is under strong positive selection. Four KSA MERS-CoV phylogenetic clades were found, with 3 clades apparently no longer contributing to current cases. The size of the population infected with MERS-CoV showed a gradual increase to June 2013, followed by a decline, possibly due to increased surveillance and infection control measures combined with a basic reproduction number (R 0 ) for the virus that is less than 1. IMPORTANCE MERS-CoV adaptation toward higher rates of sustained human-to-human transmission appears not to have occurred yet. While MERS-CoV transmission currently appears weak, careful monitoring of changes in MERS-CoV genomes and of the MERS epidemic should be maintained. The observation of phylogenetically related MERS-CoV in geographically diverse locations must be taken into account in efforts to identify the animal source and transmission of the virus.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Phenotypic and Genetic Characterization of Avian Influenza H5N2 Viruses with Intra-and Inter-Duck Variations in Taiwan</h3>
<p style="text-align: justify; text-justify: inter-word;">These results demonstrate that intra-host phenotypic variation occurs even within an individual duck. In view of recent human infections by low pathogenic AIVs, this study suggests possible determinants involved in the stepwise selection of virus variants from the duck influenza virus population which may facilitate inter-species transmission.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Use of Highly Pathogenic Avian Influenza A(H5N1) Gain-Of-Function Studies for Molecular-Based Surveillance and Pandemic Preparedness</h3>
<p style="text-align: justify; text-justify: inter-word;">. Use of highly pathogenic avian influenza A(H5N1) gain-of-function studies for molecular-based surveillance and pandemic preparedness. mBio 5(6):e02431-14.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory Investigation and Phylogenetic Analysis of an Imported Middle East Respiratory Syndrome Coronavirus Case in Greece</h3>
<p style="text-align: justify; text-justify: inter-word;">Rapid and reliable laboratory diagnosis of persons suspected of Middle East respiratory syndrome coronavirus (MERS-CoV) infection is important for timely implementation of infection control practices and disease management. In addition, monitoring molecular changes in the virus can help elucidate chains of transmission and identify mutations that might influence virus transmission efficiency. This was illustrated by a recent laboratory investigation we conducted on an imported MERS-CoV case in Greece. Two oropharyngeal swab specimens were collected on the 1 st and 2 nd day of patient hospitalization and tested using two real-time RT-PCR (rRT-PCR) assays targeting the UpE and Orf-1a regions of the MERS-CoV genome and RT-PCR and partial sequencing of RNA-dependent RNA polymerase and nucleocapsid genes. Serum specimens were also collected and serological test were performed. Results from the first swab sample were inconclusive while the second swab was strongly positive for MERS-CoV RNA by rRT-PCR and confirmed positive by RT-PCR and partial gene sequencing. Positive serologic test results further confirmed MERS-CoV infection. Full-length nucleocapsid and spike gene coding sequences were later obtained from the positive swab sample. Phylogenetic analysis revealed that the virus was closely related to recent human-derived MERS-CoV strains obtained in Jeddah and Makkah, Saudi Arabia, in April 2014 and dromedary camels in Saudi Arabia and Qatar. These findings were consistent with the patient's history. We also identified a unique amino acid substitution in the spike receptor binding domain that may have implications for receptor binding efficiency. Our initial inconclusive rRT-PCR results highlight the importance of collecting multiple specimens from suspect MERS-CoV cases and particularly specimens from the lower respiratory tract.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Infectious Diseases Specific detection of H5N1 avian influenza A virus in field specimens by a one-step RT-PCR assay</h3>
<p style="text-align: justify; text-justify: inter-word;">Validation on 145 field specimens from Vietnam and Malaysia showed that the assay was specific without cross reactivity to a number of other infuenza strains as well as human respiratory related pathogens. Detection was 100% from allantoic fluid in H5N1 positive samples, suggesting it to be a reliable sampling source for accurate detection.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic and biological characterisation of Zika virus isolates from different Brazilian regions</h3>
<p style="text-align: justify; text-justify: inter-word;">MAIN CONCLUSIONS Genomic analysis of three recent ZIKV isolates revealed some nonsynonymous substitutions, which could have an impact on the viral fitness in mammalian and insect cells.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Monitoring Survivability and Infectivity of Porcine Epidemic Diarrhea Virus (PEDv) in the Infected On-Farm Earthen Manure Storages (EMS)</h3>
<p style="text-align: justify; text-justify: inter-word;">In recent years, porcine epidemic diarrhea virus (PEDv) has caused major epidemics, which has been a burden to North America's swine industry. Low infectious dose and high viability in the environment are major challenges in eradication of this virus. To further understand the viability of PEDv in the infected manure, we longitudinally monitored survivability and infectivity of PEDv in two open earthen manure storages (EMS; previously referred to as lagoon) from two different infected swine farms identified in the province of Manitoba, Canada. Our study revealed that PEDv could survive up to 9 months in the infected EMS after the initial outbreak in the farm. The viral load varied among different layers of the EMS with an average of 1.1 × 10 5 copies/ml of EMS, independent of EMS temperature and pH. In both studied EMS, the evidence of viral replication was observed through increased viral load in the later weeks of the samplings while there was no new influx of infected manure into the EMS, which was suggestive of presence of potential alternative hosts for PEDv within the EMS. Decreasing infectivity of virus over time irrespective of increased viral load suggested the possibility of PEDv evolution within the EMS and perhaps in the new host that negatively impacted virus infectivity. Viral load in the top layer of the EMS was low and mostly non-infective suggesting that environmental factors, such as UV and sunlight, could diminish the replicability and infectivity of the virus. Thus, frequent agitation of the EMS that could expose virus to UV and sunlight might be a potential strategy for reduction of PEDv load and infectivity in the infected EMS.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation</h3>
<p style="text-align: justify; text-justify: inter-word;">The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">The ongoing COVID-19 epidemic continues to spread within and outside of China, despite several social distancing measures implemented by the Chinese government. Limited epidemiological data are available, and recent changes in case definition and reporting further complicate our understanding of the impact of the epidemic, particularly in the epidemic's epicenter.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>An Opportunistic Pathogen Afforded Ample Opportunities: Middle East Respiratory Syndrome Coronavirus</h3>
<p style="text-align: justify; text-justify: inter-word;">The human coronaviruses (CoV) include HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1, some of which have been known for decades. The severe acute respiratory syndrome (SARS) CoV briefly emerged into the human population but was controlled. In 2012, another novel severely human pathogenic CoV-the Middle East Respiratory Syndrome (MERS)-CoV-was identified in the Kingdom of Saudi Arabia; 80% of over 2000 human cases have been recorded over five years. Targeted research remains key to developing control strategies for MERS-CoV, a cause of mild illness in its camel reservoir. A new therapeutic toolbox being developed in response to MERS is also teaching us more about how CoVs cause disease. Travel-related cases continue to challenge the world's surveillance and response capabilities, and more data are needed to understand unexplained primary transmission. Signs of genetic change have been recorded, but it remains unclear whether there is any impact on clinical disease. How camels came to carry the virus remains academic to the control of MERS. To date, human-to-human transmission has been inefficient, but virus surveillance, characterisation, and reporting are key to responding to any future change. MERS-CoV is not currently a pandemic threat; it is spread mainly with the aid of human habit and error.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Changes in codon-pair bias of human immunodeficiency virus type 1 have profound effects on virus replication in cell culture</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusion: These data demonstrate that SAVE is a useful strategy to phenotypically affect the replicative properties of HIV-1.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>The global spread of Middle East respiratory syndrome: an analysis fusing traditional epidemiological tracing and molecular phylodynamics</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusion: Together, these analyses can help us identify factors associated with viral evolution and spread as well as establish efficacy of infection control measures. The results are especially pertinent to countries without current MERS-CoV endemic, since their unfamiliarity makes them particularly susceptible to uncontrollable spread of a virus that may be imported by travelers.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>The effect of human mobility and control measures on the COVID-19 epidemic in China</h3>
<p style="text-align: justify; text-justify: inter-word;">Questions remain over how these interventions affected the spread of SARS-CoV-2 to locations outside of Wuhan. We here use real-time mobility data, crowdsourced line-list data of cases with reported travel . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics</h3>
<p style="text-align: justify; text-justify: inter-word;">Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Spread of Mutant Middle East Respiratory Syndrome Coronavirus with Reduced Affinity to Human CD26 during the South Korean Outbreak</h3>
<p style="text-align: justify; text-justify: inter-word;">Here, we show the rapid emergence and spread of a mutant MERS-CoV with reduced affinity to the human CD26 receptor during the South Korean outbreak. We isolated 13 new viral genomes from 14 infected patients treated at a hospital and found that 12 of these genomes possess a point mutation in the receptor-binding domain (RBD) of viral spike (S) protein. Specifically, 11 of these genomes have an I529T mutation in RBD, and 1 has a D510G mutation. Strikingly, both mutations result in reduced affinity of RBD to human CD26 compared to wild-type RBD, as measured by surface plasmon resonance analysis and cellular binding assay. Additionally, pseudotyped virus bearing an I529T mutation in S protein showed reduced entry into host cells compared to virus with wild-type S protein. These unexpected findings suggest that MERS-CoV adaptation during human-to-human spread may be driven by host immunological pressure such as neutralizing antibodies, resulting in reduced affinity to host receptor, and thereby impairs viral fitness and virulence, rather than positive selection for a better affinity to CD26.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic and biological characterisation of Zika virus isolates from different Brazilian regions</h3>
<p style="text-align: justify; text-justify: inter-word;">BACKGROUND Zika virus (ZIKV) infections reported in recent epidemics have been linked to clinical complications that had never been associated with ZIKV before. Adaptive mutations could have contributed to the successful emergence of ZIKV as a global health threat to a nonimmune population. However, the causal relationships between the ZIKV genetic determinants, the pathogenesis and the rapid spread in Latin America and in the Caribbean remain widely unknown.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Concerns about Misinterpretation of Recent Scientific Data Implicating Dromedary Camels in Epidemiology of Middle East Respiratory Syndrome (MERS)</h3>
<p style="text-align: justify; text-justify: inter-word;">T his letter addresses some concerns about two recent articles published by the same authors in mBio (1, 2), specifically many uncertainties regarding the potential applicability of their epidemiological data, which were obtained from dromedary camels (DCs) infected with Middle East respiratory syndrome coronavirus (MERS-CoV), to human public health.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical review: Update of avian influenza A infections in humans</h3>
<p style="text-align: justify; text-justify: inter-word;">Influenza A viruses have a wide host range for infection, from wild waterfowl to poultry to humans. Recently, the cross-species transmission of avian influenza A, particularly subtype H5N1, has highlighted the importance of the non-human subtypes and their incidence in the human population has increased over the past decade. During cross-species transmission, human disease can range from the asymptomatic to mild conjunctivitis to fulminant pneumonia and death. With these cases, however, the risk for genetic change and development of a novel virus increases, heightening the need for public health and hospital measures. This review discusses the epidemiology, host range, human disease, outcome, treatment, and prevention of cross-transmission of avian influenza A into humans. ARDS = acute respiratory distress syndrome; CDC = Centers for Disease Control and Prevention; HA = hemagglutinin; HPAI = highly pathogenic avian influenza; NA = neuroaminidase; WHO = World Health Organization.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Recombination and the Evolution of Science and Public Health in China</h3>
<p style="text-align: justify; text-justify: inter-word;">Citation Lipkin WI. 2015. Middle East respiratory syndrome coronavirus recombination and the evolution of science and public health in China. mBio 6(5):e01381-15.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /></body>