
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: did farmers play a role in the origin of the epidemic</h2><br /></div><br /><br /><br /><div class="card"><h3>Domestic Poultry and SARS Coronavirus, Southern China</h3>
<p style="text-align: justify; text-justify: inter-word;">SARS coronavirus injected intratracheally into chickens, turkeys, geese, ducks, and quail, or into the allantoic sac of their embryonating eggs, failed to cause disease or replicate. This finding suggests that domestic poultry were unlikely to have been the reservoir, or associated with dissemination, of SARS coronavirus in the animal markets of southern China.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic Clues to SARS Origin in China SARS Origin in China</h3>
<p style="text-align: justify; text-justify: inter-word;">An epidemic of severe acute respiratory syndrome (SARS) began in Foshan municipality, Guangdong Province, China, in November 2002. We studied SARS case reports through April 30, 2003, including data from case investigations and a case series analysis of index cases. A total of 1,454 clinically confirmed cases (and 55 deaths) occurred; the epidemic peak was in the first week of February 2003. Healthcare workers accounted for 24% of cases. Clinical signs and symptoms differed between children (<18 years) and older persons (>65 years). Several observations support the hypothesis of a wild animal origin for SARS. Cases apparently occurred independently in at least five different municipalities; early case-patients were more likely than later patients to report living near a produce market (odds ratio undefined; lower 95% confidence interval 2.39) but not near a farm; and 9 (39%) of 23 early patients, including 6 who lived or worked in Foshan, were food handlers with probable animal contact.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Agricultural and geographic factors shaped the North American 2015 highly pathogenic avian influenza H5N2 outbreak</h3>
<p style="text-align: justify; text-justify: inter-word;">Agricultural and geographic factors of the 2015 HPAI US outbreak PLOS Pathogens | https://doi.org/10.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: Study data suggest that animal workers may have been infected with the H10N8 virus before the first recognized H10N8 human infection cases. It seems prudent to continue surveillance for H10N8 viruses among animal workers.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Associations between attributes of live poultry trade and HPAI H5N1 outbreaks: a descriptive and network analysis study in northern Vietnam</h3>
<p style="text-align: justify; text-justify: inter-word;">Our results indicate that LPT's trading for less than a year and operating at retail markets are more likely to source poultry from flocks located in communes with a past history of HPAI H5N1 outbreaks during 2003 to 2006 than LPT's trading longer than a year and operating at wholesale markets. The results of the network analysis indicate that LPT's tend to link communes of similar infection status. Conclusions: Our study provides evidence which can be used for informing policies aimed at encouraging more biosecure practices of LPT's operating at authorised LBM's. The results suggest that LPT's play a role in HPAI H5N1 transmission and may contribute to perpetuating HPAI H5N1 virus circulation amongst certain groups of communes. The impact of current disease prevention and control interventions could be enhanced by disseminating information about outbreak risk and the implementation of a formal data recording scheme at LBM's for all incoming and outgoing LPT's.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Acta Veterinaria Scandinavica Experimental infection in calves with a specific subtype of verocytotoxin-producing Escherichia coli O157:H7 of bovine origin</h3>
<p style="text-align: justify; text-justify: inter-word;">In Sweden, a particular subtype of verocytotoxin-producing Escherichia coli (VTEC) O157:H7, originally defined as being of phage type 4, and carrying two vtx 2 genes, has been found to cause the majority of reported human infections during the past 15 years, including both sporadic cases and outbreaks. One plausible explanation for this could be that this particular subtype is better adapted to colonise cattle, and thereby may be excreted in greater concentrations and for longer periods than other VTEC O157:H7 subtypes.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>First human case of avian influenza A (H5N6) in Yunnan province, China 596484S Case Report</h3>
<p style="text-align: justify; text-justify: inter-word;">The findings suggest that studies on the source of the virus, transmission models, serologic investigations, vaccines, and enhancing surveillance in both humans and birds are necessary.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Travel, migration and emerging infectious diseases</h3>
<p style="text-align: justify; text-justify: inter-word;">Emerging infectious diseases (EID) threaten public health and are sustained by increasing global commerce, travel and disruption of ecological systems. Travelers could play a role in importing EIDs and could be a sentinel of major epidemics. In connection with the extension of poverty, urbanization, extensive livestock rearing and globalization, we could be exposed to a third epidemiological transition characterized by zoonotic diseases and infections with multidrugresistant bacteria. The risk appears low for emerging infectious diseases, or very low for high-risk emerging infectious diseases, but higher for multidrug-resistant enterobacteriaceae carriage with possibly limited consequences. The role played by migrants is weaker than imagined. Immigrants don't play the role of sentinel epidemic so far. They could play a role in importing multidrug-resistant enterobacteriaceae, but it is poorly evaluated.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Using heterogeneity in the population structure of U.S. swine farms to compare transmission models for porcine epidemic diarrhoea</h3>
<p style="text-align: justify; text-justify: inter-word;">In 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Predictive Power of Air Travel and Socio-Economic Data for Early Pandemic Spread</h3>
<p style="text-align: justify; text-justify: inter-word;">Methodology/Principal Findings: Here, we examine the role of travel, trade, and national healthcare resources in predicting the emergence and initial spread of 2009 A/H1N1 influenza. We find that incorporating national healthcare resource data into our analyses allowed a much greater capacity to predict the international spread of this virus. In countries with lower healthcare resources, the reporting of 2009 A/H1N1 cases was significantly delayed, likely reflecting a lower capacity for testing and reporting, as well as other socio-political issues. We also report substantial international trade in live swine and poultry in the decade preceding the pandemic which may have contributed to the emergence and mixed genotype of this pandemic strain. However, the lack of knowledge of recent evolution of each H1N1 viral gene segment precludes the use of this approach to determine viral origins.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Spillover of Swine Coronaviruses, United States</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine epidemic diarrhea virus, a pathogen first detected in US domestic swine in 2013, has rapidly spilled over into feral swine populations. A better understanding of the factors associated with pathogen emergence is needed to better manage, and ultimately prevent, future spillover events from domestic to nondomestic animals.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Medicine Editorial Initial Cluster of Novel Coronavirus (2019-nCoV) Infections in Wuhan, China Is Consistent with Substantial Human-to-Human Transmission</h3>
<p style="text-align: justify; text-justify: inter-word;">Reanalysis of the epidemic curve from the initial cluster of cases with novel coronavirus (2019-nCoV) in December 2019 indicates substantial human-to-human transmission. It is possible that the common exposure history at a seafood market in Wuhan originated from the human-to-human transmission events within the market, and the early, strong emphasis that market exposure indicated animal-to-human transmission was potentially the result of observer bias. To support the hypothesis of zoonotic origin of 2019-nCoV stemming from the Huanan seafood market, the index case should have had exposure history related to the market and the virus should have been identified from animals sold at the market. As these requirements remain unmet, zoonotic spillover at the market must not be overemphasized.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Origin, Evolution, and Genotyping of Emergent Porcine Epidemic Diarrhea Virus Strains in the United States</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronaviruses are known to infect humans and other animals and cause respiratory and gastrointestinal diseases. Here we report the emergence of porcine epidemic diarrhea virus (PEDV) in the United States and determination of its origin, evolution, and genotypes based on temporal and geographical evidence. Histological lesions in small intestine sections of affected pigs and the complete genomic sequences of three emergent strains of PEDV isolated from outbreaks in Minnesota and Iowa were characterized. Genetic and phylogenetic analyses of the three U.S. strains revealed a close relationship with Chinese PEDV strains and their likely Chinese origin. The U.S. PEDV strains underwent evolutionary divergence, which can be classified into two sublineages. The three emergent U.S. strains are most closely related to a strain isolated in 2012 from Anhui Province in China, which might be the result of multiple recombination events between different genetic lineages or sublineages of PEDV. Molecular clock analysis of the divergent time based on the complete genomic sequences is consistent with the actual time difference, approximately 2 to 3 years, of the PED outbreaks between China (December 2010) and the United States (May 2013). The finding that the emergent U.S. PEDV strains share unique genetic features at the 5=-untranslated region with a bat coronavirus provided further support of the evolutionary origin of PEDV from bats and potential cross-species transmission. The data from this study have important implications for understanding the ongoing PEDV outbreaks in the United States and will guide future efforts to develop effective preventive and control measures against PEDV. IMPORTANCE The sudden emergence of porcine epidemic diarrhea virus (PEDV), a coronavirus, for the first time in the United States causes significant economic and public health concerns. Since its recognition in May 2013, PEDV has rapidly spread across the United States, resulting in high mortality in piglets in more than 17 States now. The ongoing outbreaks of Middle East respiratory syndrome coronavirus in humans from countries in or near the Arabian Peninsula and the historical deadly nature of the 2002 outbreaks of severe acute respiratory syndrome coronavirus create further anxiety over the emergence of PEDV in the United States due to the lack of scientific information about the origin and evolution of this emerging coronavirus. Here we report the detailed genetic characterization, origin, and evolution of emergent PEDV strains in the United States. The results provide much needed information to devise effective preventive and control strategies against PEDV in the United States. Citation Huang YW, Dickerman AW, Piñeyro P, Li L, Fang L, Kiehne R, Opriessnig T, Meng XJ. 2013. Origin, evolution, and genotyping of emergent porcine epidemic diarrhea virus strains in the United States. mBio 4(5):00737-13.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Results of an online questionnaire to survey calf management practices on dairy cattle breeding farms in Austria and to estimate differences in disease incidences depending on farm structure and management practices Open Access</h3>
<p style="text-align: justify; text-justify: inter-word;">Additional file 1. The original online questionnaire in German, distributed to the farmers.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Ducks: The ''Trojan Horses'' of H5N1 influenza</h3>
<p style="text-align: justify; text-justify: inter-word;">Wild ducks are the main reservoir of influenza A viruses that can be transmitted to domestic poultry and mammals, including humans. Of the 16 hemagglutinin (HA) subtypes of influenza A viruses, only the H5 and H7 subtypes cause highly pathogenic (HP) influenza in the natural hosts. Several duck species are naturally resistant to HP Asian H5N1 influenza viruses. These duck species can shed and spread virus from both the respiratory and intestinal tracts while showing few or no disease signs. While the HP Asian H5N1 viruses are 100% lethal for chickens and other gallinaceous poultry, the absence of disease signs in some duck species has led to the concept that ducks are the ''Trojan horses'' of H5N1 in their surreptitious spread of virus. An important unresolved issue is whether the HP H5N1 viruses are maintained in the wild duck population of the world. Here, we review the ecology and pathobiology of ducks infected with influenza A viruses and ducks' role in the maintenance and spread of HP H5N1 viruses. We also identify the key questions about the role of ducks that must be resolved in order to understand the emergence and control of pandemic influenza. It is generally accepted that wild duck species can spread HP H5N1 viruses, but there is insufficient evidence to show that ducks maintain these viruses and transfer them from one generation to the next.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Phylogeographic investigation of 2014 porcine epidemic diarrhea virus (PEDV) transmission in Taiwan</h3>
<p style="text-align: justify; text-justify: inter-word;">The porcine epidemic diarrhea virus (PEDV) that emerged and spread throughout Taiwan in 2014 triggered significant concern in the country's swine industry. Acknowledging the absence of a thorough investigation at the geographic level, we used 2014 outbreak sequence information from the Taiwan government's open access databases plus GenBank records to analyze PEDV dissemination among Taiwanese pig farms. Genetic sequences, locations, and dates of identified PEDV-positive cases were used to assess spatial, temporal, clustering, GIS, and phylogeographic factors affecting PEDV dissemination. Our conclusion is that S gene sequences from 2014 PEDV-positive clinical samples collected in Taiwan were part of the same Genogroup 2 identified in the US in 2013. According to phylogenetic and phylogeographic data, viral strains collected in different areas were generally independent of each other, with certain clusters identified across different communities. Data from GIS and multiple potential infection factors were used to pinpoint cluster dissemination in areas with large numbers of swine farms in southern Taiwan. The data indicate that the 2014 Taiwan PEDV epidemic resulted from the spread of multiple strains, with strong correlations identified with pig farm numbers and sizes (measured as animal concentrations), feed mill numbers, and the number of slaughterhouses in a specifically defined geographic area.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Pathways of Foot-and-Mouth Disease Virus in the United Kingdom</h3>
<p style="text-align: justify; text-justify: inter-word;">Foot-and-mouth disease (FMD) virus causes an acute vesicular disease of domesticated and wild ruminants and pigs. Identifying sources of FMD outbreaks is often confounded by incomplete epidemiological evidence and the numerous routes by which virus can spread (movements of infected animals or their products, contaminated persons, objects, and aerosols). Here, we show that the outbreaks of FMD in the United Kingdom in August 2007 were caused by a derivative of FMDV O 1 BFS 1860, a virus strain handled at two FMD laboratories located on a single site at Pirbright in Surrey. Genetic analysis of complete viral genomes generated in real-time reveals a probable chain of transmission events, predicting undisclosed infected premises, and connecting the second cluster of outbreaks in September to those in August. Complete genome sequence analysis of FMD viruses conducted in real-time have identified the initial and intermediate sources of these outbreaks and demonstrate the value of such techniques in providing information useful to contemporary disease control programmes.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>An evaluation of contaminated complete feed as a vehicle for porcine epidemic diarrhea virus infection of naïve pigs following consumption via natural feeding behavior: proof of concept</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Since its initial detection in May 2013, porcine epidemic diarrhea virus (PEDV) has spread rapidly throughout the US swine industry. Initially, contaminated feed was proposed as a risk factor for PEDV; however, data were not available to support this theory. Here we provide proof of concept of this risk by describing a novel means for recovering PEDV-contaminated complete feed material from commercial swine sites and conducting an in vivo experiment to prove its infectivity.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">finding of the clinical and the environmental isolate (1,2). The existence of MATa in nature is also supported by recent studies designed to established the origin of the serotype AD strains (4, 5) . These studies demonstrated that AD strains were diploid or aneuploid hybrids derived from a fusion of serotype A and D parents and that several of them were harboring a serotype A MATa locus. These hybrid strains have been found fairly often in Europe (9,10).</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Parameters of the 2001 Foot and Mouth Epidemic in Great Britain</h3>
<p style="text-align: justify; text-justify: inter-word;">Despite intensive ongoing research, key aspects of the spatial-temporal evolution of the 2001 foot and mouth disease (FMD) epidemic in Great Britain (GB) remain unexplained. Here we develop a Markov Chain Monte Carlo (MCMC) method for estimating epidemiological parameters of the 2001 outbreak for a range of simple transmission models. We make the simplifying assumption that infectious farms were completely observed in 2001, equivalent to assuming that farms that were proactively culled but not diagnosed with FMD were not infectious, even if some were infected. We estimate how transmission parameters varied through time, highlighting the impact of the control measures on the progression of the epidemic. We demonstrate statistically significant evidence for assortative contact patterns between animals of the same species. Predictive risk maps of the transmission potential in different geographic areas of GB are presented for the fitted models.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /></body>