
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: calling for all potential testers using PCR in a defined area to start testing and report to a specific entity</h2><br /></div><br /><br /><br /><div class="card"><h3>A field-deployable insulated isothermal RT-PCR assay for identification of influenza A (H7N9) shows good performance in the laboratory</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Avian influenza A (H7N9) remains circulating in China. For countries at risk of introduction of H7N9, such as Vietnam, early detection of H7N9 virus is essential for the early containment of the virus. Insulated isothermal reverse transcriptase PCR (iiRT-PCR) is a portable PCR system that can be deployed under field conditions to identify pathogens at the sampling site. Applying PCR at the sampling site will greatly reduce the time to obtain a diagnostic result which allows the veterinary authority to take immediate action to contain disease spreading.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulate_PCR for amplicon prediction and annotation from multiplex, degenerate primers and probes</h3>
<p style="text-align: justify; text-justify: inter-word;">Simulate_PCR is the only freely available software that can be run from the command line for high throughput applications which can calculate all products from large lists of primers and probes compared to a large sequence database such as nt. It requires no prior knowledge of how primers should be paired. Degenerate bases are allowed and entire amplicon sequences are extracted and annotated with gene information. Examples are provided for sets of TaqMan®/Luminex® PCR signatures predicted to amplify all HIV-1 genomes, all Coronaviridae genomes, and a group of antibiotic resistance genes. The software is a command line perl script freely available as open source.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>we enrolled children with pneumonia who were admitted into wards of the De- partments of Respiratory Medicine, Infectious Diseases</h3>
<p style="text-align: justify; text-justify: inter-word;">we enrolled 4,225 hospitalized children with pneumonia in a study to determine the seasonality of respiratory syncytial virus (RSV) infection in Beijing, China. We defined season as the period during which >10% of total PCRs performed each week were RSV positive. We identified 8 distinctive RSV seasons. On average, the season onset occurred at week 41 (mid-October) and lasted 33 weeks, through week 20 of the next year (mid-May); 97% of all RSV-positive cases occurred during the season. RSV seasons occurred 3-5 weeks earlier and lasted ≈6 weeks longer in RSV subgroup A-dominant years than in RSV subgroup B-dominant years. Our analysis indicates that monitoring such RSV subgroup shifts might provide better estimates for the onset of RSV transmission. PCR-based tests could be a flexible or complementary way of determining RSV seasonality in locations where RSV surveillance is less well-established, such as local hospitals throughout China.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Single Fluorescence Channel- based Multiplex Detection of Avian Influenza Virus by Quantitative PCR with Intercalating Dye OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Since its invention in 1985 the polymerase chain reaction (PCR) has become a well-established method for amplification and detection of segments of double-stranded DNA. Incorporation of fluorogenic probe or DNA intercalating dyes (such as SYBR Green) into the PCR mixture allowed real-time reaction monitoring and extraction of quantitative information (qPCR). Probes with different excitation spectra enable multiplex qPCR of several DNA segments using multi-channel optical detection systems. Here we show multiplex qPCR using an economical EvaGreen-based system with single optical channel detection. Previously reported non quantitative multiplex realtime PCR techniques based on intercalating dyes were conducted once the PCR is completed by performing melting curve analysis (MCA). The technique presented in this paper is both qualitative and quantitative as it provides information about the presence of multiple DNA strands as well as the number of starting copies in the tested sample. Besides important internal control, multiplex qPCR also allows detecting concentrations of more than one DNA strand within the same sample. Detection of the avian influenza virus H7N9 by PCR is a well established method. Multiplex qPCR greatly enhances its specificity as it is capable of distinguishing both haemagglutinin (HA) and neuraminidase (NA) genes as well as their ratio.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Tien Giang Provincial Hospital, My Tho, Tien Giang Province, Vietnam, 6 Dong Nai Children's Hospital</h3>
<p style="text-align: justify; text-justify: inter-word;">We demonstrate that the early diagnosis of dengue can be enhanced beyond the current standard of care using a simple evidence-based algorithm. The results should support patient management and clinical trials of specific therapies.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Annals of Clinical Microbiology and Antimicrobials Predicting the sensitivity and specificity of published real-time PCR assays</h3>
<p style="text-align: justify; text-justify: inter-word;">We assessed the quality of a signature by predicting the number of true positive, false positive and false negative hits against all available public sequence data. We found real-time PCR signatures described in recent literature and used a BLAST search based approach to collect all hits to the primer-probe combinations that should be amplified by real-time PCR chemistry. We then compared our hits with the sequences in the NCBI taxonomy tree that the signature was designed to detect.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Individual sequences in large sets of gene sequences may be distinguished efficiently by combinations of shared sub-sequences</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Most current DNA diagnostic tests for identifying organisms use specific oligonucleotide probes that are complementary in sequence to, and hence only hybridise with the DNA of one target species. By contrast, in traditional taxonomy, specimens are usually identified by 'dichotomous keys' that use combinations of characters shared by different members of the target set. Using one specific character for each target is the least efficient strategy for identification. Using combinations of shared bisectionally-distributed characters is much more efficient, and this strategy is most efficient when they separate the targets in a progressively binary way.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Usefulness of multiplex PCR methods and respiratory viruses' distribution in children below 15 years old according to age, seasons and clinical units in France: A 3 years retrospective study</h3>
<p style="text-align: justify; text-justify: inter-word;">Of the 381 mPCR tests (344 children) performed, 51.4% were positive. Positivity and viral co-infection rates were higher in the 6-36 months old strata (81% and 25%, p<0.0001 and p = 0.04, respectively). Viral distribution showed strong variations across ages. During specific influenza epidemic periods, only 1/39 (2.5%) mPCR tests were positive for influenza and 19/39 (48.7%) for other viruses. During specific RSV epidemic periods, only 8/46 (17.4%) mPCR tests were positive for RSV and 14/46 (30.4%) for other viruses. 477/1529 (31.2%) of RSV immunochromatography-tests were positive. Among the negatives PLOS ONE |</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Deep Sequencing and Phylogenetic Analysis of Novel Human Betacoronavirus</h3>
<p style="text-align: justify; text-justify: inter-word;">Fifteen betaCoV full genomes with the closest homology to EMC/2012 (GenBank accession no. JX869059) were analyzed to identify all possible primer-like sequences (melting temperature 58°C-60°C, guanine plus cytosine content 35%-60%) yielding 357,198 potential primers. The 30-kb JX869059 CoV genome was divided into fifteen 2.5-kb overlapping amplicons, and the 2 highest frequency sequences mapping in the 5′ and 3′ 250 bp of each amplicon were selected. Reverse complements of the 3′ mapping sequences were prepared, resulting in a set of 60 primers for the 15 amplicons (4 primers/amplicon). Three additional primers were added for the extreme termini of the genome. The algorithm also prepared a virtual PCR map of the predicted binding and amplicon size for the primer set on CoV genomes. A map of the primer mapping positions</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics</h3>
<p style="text-align: justify; text-justify: inter-word;">Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<p style="text-align: justify; text-justify: inter-word;">is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101//doi.org/10. /2020 1 4 Discussion SARS-CoV-2, a novel betacoronavirus are a major cause of symptomatic respiratory tract infection in all age groups worldwide 11,16,25 . Timely and accurate diagnosis of the virus enables appropriate treatment of infections. RT-PCR is widely deployed in diagnostic virology. In the case of a public health emergency, proficient diagnostic laboratories can rely on this robust technology to establish new diagnostic tests within their routine services before pre-formulated assays become available.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Single Fluorescence Channel- based Multiplex Detection of Avian Influenza Virus by Quantitative PCR with Intercalating Dye OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">The influenza A virus is composed of eight RNA segments of negative-sense single-stranded RNA where segment 4 encodes the haemagglutinin (HA) gene, and segment 6 the neuraminidase (NA) gene. In order to increase specificity of H7N9 detection, besides the detection of the virus itself, one should also detect both HA and NA individually as well as their ratio. PCR is the method of choice to detect the virus as it can deliver results in tens of minutes compared to traditional methods such as ELISA where the testing takes a few days.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Geospatial Science and Point-of-Care Testing: Creating Solutions for Population Access, Emergencies, Outbreaks, and Disasters</h3>
<p style="text-align: justify; text-justify: inter-word;">Impact: Geospatial solutions generate inherently optimal and logical placement of POCT conceptually, physically, and temporally as a means to improve crisis response and spatial resilience. If public health professionals, geospatial scientists, and POCT specialists join forces, new collaborative teamwork can create faster response and higher impact during disasters, complex crises, outbreaks, and epidemics, as well as more efficient primary, urgent, and emergency community care.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Single Fluorescence Channel- based Multiplex Detection of Avian Influenza Virus by Quantitative PCR with Intercalating Dye OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">The polymerase chain reaction (PCR) was invented in 1985 1,2 to amplify double stranded DNA segments. Adding fluorogenic probe or DNA intercalating dyes (such as SYBR Green) allowed real-time 3 reaction progress monitoring and extracting of quantitative information (qPCR) 4,5 . A typical PCR process consists of three steps conducted at different temperatures: denaturation at 95 °C, annealing at 50-60 °C and extension at 72 °C. Using the Taqman probe-based PCR system the annealing and extension steps are combined into one and performed at a temperature of about 60 °C 5 . The presence of end-point PCR products can be confirmed by agarose gel electrophoresis or by capillary electrophoresis (CE) 6 . However, an end-point measurement does not necessarily correlate with the original number of copies of the amplified DNA sequence. Real-time PCR allows precise quantitative information to be extracted from the exponential phase of the reaction 4,7 . The Taqman probe-based assay format, for example the FAM TM (fluorophore) and TAMRA TM (quencher) labeled probe, is specific to its target gene, whereas the 1 Kist-Europe,</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Polymerase cross-linking spiral reaction (PCLSR) for detection of African swine fever virus (ASFV) in pigs and wild boars</h3>
<p style="text-align: justify; text-justify: inter-word;">The study reports the development of a polymerase cross-linking spiral reaction (PCLSR) for the detection of African swine fever virus (ASFV) DNA in blood collected from infected pigs and wild boars. The method uses 3 specifically designed primers. Two outer-spiral primers comprising of 3′ sequences complementary to ASFV p72 gene sequence and 5′end sequences complementary to exogenous gene of black widow alpha-latrotoxin as well as additional ASFV specific cross-linking primer. The method is specific exclusively to ASFV DNA without cross-reactions with cDNA of classical swine fever virus (CSFV), porcine reproductive respiratory syndrome (PRRSV) or porcine epidemic diarrhea virus (PEDV). The sensitivity of this technique reached 7.2 × 10 2 copies per μl −1 of plasmid containing p72 gene. The PCLSR was conducted at 65 °C creating cross-linked complex structures. The results of PCLSR were visualized using SYBR Green I dye, gel electrophoresis while the reaction progress was traced using real-time PCR system that resulted in registration of fluorescent curves and melting peaks at 85.3 °C. The developed PCLSR was examined using blood or tissue samples collected from selected 17 ASF cases from infected wild boars and 3 outbreaks in pigs. Further tests have been also conducted using 55 tissue samples from 23 outbreaks and 22 cases. These results showed that PCLSR might be further used for preliminary and cost-effective detection and surveillance of ASFV.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Make Data Sharing Routine to Prepare for Public Health Emergencies</h3>
<p style="text-align: justify; text-justify: inter-word;">• The recent outbreaks caused by Ebola and Zika viruses highlighted the importance of medical and public health research in accelerating outbreak control and prompted calls for researchers to share data rapidly and widely during public health emergencies.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Twelve years' detection of respiratory viruses by immunofluorescence in hospitalised children: impact of the introduction of a new respiratory picornavirus assay</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Direct immunofluorescence assays (DFA) are a rapid and inexpensive method for the detection of respiratory viruses and may therefore be used for surveillance. Few epidemiological studies have been published based solely on DFA and none included respiratory picornaviruses and human metapneumovirus (hMPV). We wished to evaluate the use of DFA for epidemiological studies with a long-term observation of respiratory viruses that includes both respiratory picornaviruses and hMPV. Methods: Since 1998 all children hospitalized with respiratory illness at the University Hospital Bern have been screened with DFA for common respiratory viruses including adenovirus, respiratory syncytial virus (RSV), influenza A and B, and parainfluenza virus 1-3. In 2006 assays for respiratory picornaviruses and hMPV were added. Here we describe the epidemiological pattern for these respiratory viruses detected by DFA in 10'629 nasopharyngeal aspirates collected from 8'285 patients during a 12-year period (1998)(1999)(2000)(2001)(2002)(2003)(2004)(2005)(2006)(2007)(2008)(2009)(2010). Results: Addition of assays for respiratory picornaviruses and hMPV raised the proportion of positive DFA results from 35% to 58% (p < 0.0001). Respiratory picornaviruses were the most common viruses detected among patients ≥1 year old. The seasonal patterns and age distribution for the studied viruses agreed well with those reported in the literature. In 2010, an hMPV epidemic of unexpected size was observed.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Forensic genetics and genomics: Much more than just a human affair</h3>
<p style="text-align: justify; text-justify: inter-word;">check the quality of profiles or DNA sequences before analysing the results. The genetic information obtained is then compared with other genetic information (i.e., derived from reliable databases [e.g., 10] or reference sample[s]) considering statistical analyses. The experimental workflow ends with a report describing the technical procedures applied and the answers to the question(s) of the request.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Influenza surveillance in Europe: establishing epidemic thresholds by the Moving Epidemic Method</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions The method produced a robust and specific signal to detect influenza epidemics. The good balance between the sensitivity and specificity of the epidemic threshold to detect seasonal epidemics and avoid false alerts has advantages for public health purposes. This method may serve as standard to define the start of the annual influenza epidemic in countries in Europe.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Individual sequences in large sets of gene sequences may be distinguished efficiently by combinations of shared sub-sequences</h3>
<p style="text-align: justify; text-justify: inter-word;">We have developed a practical method for finding minimal sets of sub-sequences that identify individual sequences, and could be targeted by combinations of probes, so that the efficient strategy of traditional taxonomic identification could be used in DNA diagnosis. The sizes of minimal sub-sequence sets depended mostly on sequence diversity and sub-sequence length and interactions between these parameters. We found that 201 distinct cytochrome oxidase subunit-1 (CO1) genes from moths (Lepidoptera) were distinguished using only 15 sub-sequences 20 nucleotides long, whereas only 8-10 sub-sequences 6-10 nucleotides long were required to distinguish the CO1 genes of 92 species from the 9 largest orders of insects.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /></body>