
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: tools and studies to monitor phenotypic change and potential adaptation of coronavirus virus</h2><br /></div><br /><br /><br /><div class="card"><h3>Spread, Circulation, and Evolution of the Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first documented in the Kingdom of Saudi Arabia (KSA) in 2012 and, to date, has been identified in 180 cases with 43% mortality. In this study, we have determined the MERS-CoV evolutionary rate, documented genetic variants of the virus and their distribution throughout the Arabian peninsula, and identified the genome positions under positive selection, important features for monitoring adaptation of MERS-CoV to human transmission and for identifying the source of infections. Respiratory samples from confirmed KSA MERS cases from May to September 2013 were subjected to whole-genome deep sequencing, and 32 complete or partial sequences (20 were ≥99% complete, 7 were 50 to 94% complete, and 5 were 27 to 50% complete) were obtained, bringing the total available MERS-CoV genomic sequences to 65. An evolutionary rate of 1.12 × 10(−3) substitutions per site per year (95% credible interval [95% CI], 8.76 × 10(−4); 1.37 × 10(−3)) was estimated, bringing the time to most recent common ancestor to March 2012 (95% CI, December 2011; June 2012). Only one MERS-CoV codon, spike 1020, located in a domain required for cell entry, is under strong positive selection. Four KSA MERS-CoV phylogenetic clades were found, with 3 clades apparently no longer contributing to current cases. The size of the population infected with MERS-CoV showed a gradual increase to June 2013, followed by a decline, possibly due to increased surveillance and infection control measures combined with a basic reproduction number (R(0)) for the virus that is less than 1.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>We shouldn't worry when a virus mutates during disease outbreaks</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Mutation. The word naturally conjures fears of unexpected and freakish changes. Ill-informed discussions of mutations thrive during virus outbreaks, including the ongoing spread of SARS-CoV-2. In reality, mutations are a natural part of the virus life cycle and rarely impact outbreaks dramatically.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Microevolution of Outbreak-Associated Middle East Respiratory Syndrome Coronavirus, South Korea, 2015</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">During the 2015 Middle East respiratory syndrome coronavirus outbreak in South Korea, we sequenced full viral genomes of strains isolated from 4 patients early and late during infection. Patients represented at least 4 generations of transmission. We found no evidence of changes in the evolutionary rate and no reason to suspect adaptive changes in viral proteins.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Microevolution of Outbreak-Associated Middle East Respiratory Syndrome Coronavirus, South Korea, 2015</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">During the 2015 Middle East respiratory syndrome coronavirus outbreak in South Korea, we sequenced full viral genomes of strains isolated from 4 patients early and late during infection. Patients represented at least 4 generations of transmission. We found no evidence of changes in the evolutionary rate and no reason to suspect adaptive changes in viral proteins.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Extensive Positive Selection Drives the Evolution of Nonstructural Proteins in Lineage C Betacoronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome-related coronavirus (MERS-CoV) spreads to humans via zoonotic transmission from camels. MERS-CoV belongs to lineage C of betacoronaviruses (betaCoVs), which also includes viruses isolated from bats and hedgehogs. A large portion of the betaCoV genome consists of two open reading frames (ORF1a and ORF1b) that are translated into polyproteins. These are cleaved by viral proteases to generate 16 nonstructural proteins (nsp1 to nsp16) which compose the viral replication-transcription complex. We investigated the evolution of ORF1a and ORF1b in lineage C betaCoVs. Results indicated widespread positive selection, acting mostly on ORF1a. The proportion of positively selected sites in ORF1a was much higher than that previously reported for the surface-exposed spike protein. Selected sites were unevenly distributed, with nsp3 representing the preferential target. Several pairs of coevolving sites were also detected, possibly indicating epistatic interactions; most of these were located in nsp3. Adaptive evolution at nsp3 is ongoing in MERS-CoV strains, and two selected sites (G720 and R911) were detected in the protease domain. While position 720 is variable in camel-derived viruses, suggesting that the selective event does not represent a specific adaptation to humans, the R911C substitution was observed only in human-derived MERS-CoV isolates, including the viral strain responsible for the recent South Korean outbreak. It will be extremely important to assess whether these changes affect host range or other viral phenotypes. More generally, data herein indicate that CoV nsp3 represents a major selection target and that nsp3 sequencing should be envisaged in monitoring programs and field surveys. IMPORTANCE Both severe acute respiratory syndrome coronavirus (SARS-CoV) and MERS-CoV originated in bats and spread to humans via an intermediate host. This clearly highlights the potential for coronavirus host shifting and the relevance of understanding the molecular events underlying the adaptation to new host species. We investigated the evolution of ORF1a and ORF1b in lineage C betaCoVs and in 87 sequenced MERS-CoV isolates. Results indicated widespread positive selection, stronger in ORF1a than in ORF1b. Several selected sites were found to be located in functionally relevant protein regions, and some of them corresponded to functional mutations in other coronaviruses. The proportion of selected sites we identified in ORF1a is much higher than that for the surface-exposed spike protein. This observation suggests that adaptive evolution in ORF1a might contribute to host shifts or immune evasion. Data herein also indicate that genetic diversity at nonstructural proteins should be taken into account when antiviral compounds are developed.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>How Coronaviruses Cause Infection—from Colds to Deadly Pneumonia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus outbreak raises questions about how such pathogens evolve and what makes infections mild or severe</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-like WIV1-CoV poised for human emergence</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Outbreaks from zoonotic sources represent a threat to both human disease as well as the global economy. Despite a wealth of metagenomics studies, methods to leverage these datasets to identify future threats are underdeveloped. In this study, we describe an approach that combines existing metagenomics data with reverse genetics to engineer reagents to evaluate emergence and pathogenic potential of circulating zoonotic viruses. Focusing on the severe acute respiratory syndrome (SARS)-like viruses, the results indicate that the WIV1-coronavirus (CoV) cluster has the ability to directly infect and may undergo limited transmission in human populations. However, in vivo attenuation suggests additional adaptation is required for epidemic disease. Importantly, available SARS monoclonal antibodies offered success in limiting viral infection absent from available vaccine approaches. Together, the data highlight the utility of a platform to identify and prioritize prepandemic strains harbored in animal reservoirs and document the threat posed by WIV1-CoV for emergence in human populations.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Management strategies of neonatal jaundice during the coronavirus disease 2019 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of coronavirus disease 2019 (COVID-19; formally known as 2019-nCoV) has become a most challenging health emergency. Owing to rigorous quarantine and control measures taken in China, routine neonatal health surveillance and follow-up have become challenging. Without follow-up surveillance, some rapid and progressive newborn diseases, such as bilirubin encephalopathy, may be ignored. The characteristics of onset age of kernicterus suggest that monitoring of bilirubin level at home provides a useful way to alert hospital visits and to prevent the development of extremely hyperbilirubinemia. Therefore, we developed an online follow-up program for convenient monitoring of bilirubin level of newborns that is based on our practical experiences. The aim is to make our management strategies of neonatal jaundice tailored to the infection prevention and control during the COVID-19 epidemic.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 11</span></i><p style="text-align: justify; text-justify: inter-word;">Live-attenuated RNA virus vaccines are efficacious but subject to reversion to virulence. Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy; increased fidelity is attenuating for some viruses. Coronavirus replication fidelity is approximately 20-fold greater than that of other RNA viruses and is mediated by a 3′-5′ exonuclease activity (ExoN) that likely functions in RNA proofreading. In this study, we demonstrate that engineered inactivation of SARS-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity in vivo and attenuation of pathogenesis in young, aged, and immunocompromised mouse models of human SARS. The ExoN inactivation genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo. Our approach represents a strategy with potential for broad applications for the stable attenuation of coronaviruses and possibly other RNA viruses.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 11</span></i><p style="text-align: justify; text-justify: inter-word;">Live-attenuated RNA virus vaccines are efficacious but subject to reversion to virulence. Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy; increased fidelity is attenuating for some viruses. Coronavirus replication fidelity is approximately 20-fold greater than that of other RNA viruses and is mediated by a 3′-5′ exonuclease activity (ExoN) that likely functions in RNA proofreading. In this study, we demonstrate that engineered inactivation of SARS-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity in vivo and attenuation of pathogenesis in young, aged, and immunocompromised mouse models of human SARS. The ExoN inactivation genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo. Our approach represents a strategy with potential for broad applications for the stable attenuation of coronaviruses and possibly other RNA viruses.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The kinetics-based enzyme-linked immunosorbent assay for coronavirus antibodies in cats: calibration to the indirect immunofluorescence assay and computerized standardization of results through normalization to control values.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The computer-assisted, kinetics-based enzyme-linked immunosorbent assay for coronavirus antibodies in cats was calibrated to the conventional indirect immunofluorescence assay by linear regression analysis and computerized interpolation (generation of "immunofluorescence assay-equivalent" titers). Procedures were developed for normalization and standardization of kinetics-based enzyme-linked immunosorbent assay results through incorporation of five different control sera of predetermined ("expected") titer in daily runs. When used with such sera and with computer assistance, the kinetics-based enzyme-linked immunosorbent assay minimized both within-run and between-run variability while allowing also for efficient data reduction and statistical analysis and reporting of results.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Murine Coronavirus Evolution In Vivo: Functional Compensation of a Detrimental Amino Acid Substitution in the Receptor Binding Domain of the Spike Glycoprotein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Murine coronavirus A59 strain causes mild to moderate hepatitis in mice. We have previously shown that mutants of A59, unable to induce hepatitis, may be selected by persistent infection of primary glial cells in vitro. These in vitro isolated mutants encoded two amino acids substitutions in the spike (S) gene: Q159L lies in the putative receptor binding domain of S, and H716D, within the cleavage signal of S. Here, we show that hepatotropic revertant variants may be selected from these in vitro isolated mutants (Q159L-H716D) by multiple passages in the mouse liver. One of these mutants, hr2, was chosen for more in-depth study based on a more hepatovirulent phenotype. The S gene of hr2 (Q159L-R654H-H716D-E1035D) differed from the in vitro isolates (Q159L-H716D) in only 2 amino acids (R654H and E1035D). Using targeted RNA recombination, we have constructed isogenic recombinant MHV-A59 viruses differing only in these specific amino acids in S (Q159L-R654H-H716D-E1035D). We demonstrate that specific amino acid substitutions within the spike gene of the hr2 isolate determine the ability of the virus to cause lethal hepatitis and replicate to significantly higher titers in the liver compared to wild-type A59. Our results provide compelling evidence of the ability of coronaviruses to rapidly evolve in vivo to highly virulent phenotypes by functional compensation of a detrimental amino acid substitution in the receptor binding domain of the spike glycoprotein.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Large-scale evolutionary surveillance of the 2009 H1N1 influenza A virus using resequencing arrays</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 25</span></i><p style="text-align: justify; text-justify: inter-word;">In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Infidelity of SARS-CoV Nsp14-Exonuclease Mutant Virus Replication Is Revealed by Complete Genome Sequencing</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Most RNA viruses lack the mechanisms to recognize and correct mutations that arise during genome replication, resulting in quasispecies diversity that is required for pathogenesis and adaptation. However, it is not known how viruses encoding large viral RNA genomes such as the Coronaviridae (26 to 32 kb) balance the requirements for genome stability and quasispecies diversity. Further, the limits of replication infidelity during replication of large RNA genomes and how decreased fidelity impacts virus fitness over time are not known. Our previous work demonstrated that genetic inactivation of the coronavirus exoribonuclease (ExoN) in nonstructural protein 14 (nsp14) of murine hepatitis virus results in a 15-fold decrease in replication fidelity. However, it is not known whether nsp14-ExoN is required for replication fidelity of all coronaviruses, nor the impact of decreased fidelity on genome diversity and fitness during replication and passage. We report here the engineering and recovery of nsp14-ExoN mutant viruses of severe acute respiratory syndrome coronavirus (SARS-CoV) that have stable growth defects and demonstrate a 21-fold increase in mutation frequency during replication in culture. Analysis of complete genome sequences from SARS-ExoN mutant viral clones revealed unique mutation sets in every genome examined from the same round of replication and a total of 100 unique mutations across the genome. Using novel bioinformatic tools and deep sequencing across the full-length genome following 10 population passages in vitro, we demonstrate retention of ExoN mutations and continued increased diversity and mutational load compared to wild-type SARS-CoV. The results define a novel genetic and bioinformatics model for introduction and identification of multi-allelic mutations in replication competent viruses that will be powerful tools for testing the effects of decreased fidelity and increased quasispecies diversity on viral replication, pathogenesis, and evolution.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Ebola virus glycoprotein with increased infectivity dominated the 2013–2016 epidemic</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The magnitude of the 2013–2016 Ebola virus disease (EVD) epidemic enabled an unprecedented number of viral mutations to occur over successive human-to-human transmission events, increasing the probability that adaptation to the human host occurred during the outbreak. We investigated one nonsynonymous mutation, Ebola virus (EBOV) glycoprotein (GP) mutant A82V, for its effect on viral infectivity. This mutation, located at the NPC1-binding site on EBOV GP, occurred early in the 2013–2016 outbreak and rose to high frequency. We found that GP-A82V had heightened ability to infect primate cells, including human dendritic cells. The increased infectivity was restricted to cells that have primate-specific NPC1 sequences at the EBOV interface, suggesting that this mutation was indeed an adaptation to the human host. GP-A82V was associated with increased mortality, consistent with the hypothesis that the heightened intrinsic infectivity of GP-A82V contributed to disease severity during the EVD epidemic.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations</h3>
<i class="tooltip">2017<span class="tooltiptext">2017-08-11</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3'[-&gt;]5'exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.\n\nIMPORTANCEUnique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Fitness barriers limit reversion of a proofreading-deficient coronavirus</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-04-26</span></i><p style="text-align: justify; text-justify: inter-word;">The 3'-to-5' exoribonuclease in coronavirus (CoV) nonstructural protein 14 (nsp14-ExoN) mediates RNA proofreading during genome replication. ExoN catalytic residues are arranged in three motifs: I (DE), II (E), III (D). Alanine substitution of the motif I residues (AA-E-D, four nucleotide substitutions) in murine hepatitis virus (MHV) and SARS-CoV yields viable mutants with impaired replication and fitness, increased mutation rates, and attenuated virulence in vivo. Despite these impairments, MHV- and SARS-CoV ExoN motif I AA mutants (ExoN-AA) have not reverted at motif I in diverse in vitro and in vivo environments, suggesting that profound fitness barriers prevent motif I reversion. To test this hypothesis, we engineered MHV-ExoN-AA with 1, 2 or 3 nucleotide mutations along genetic pathways to AA-to-DE reversion. We show that engineered intermediate revertants were viable but had no increased replication or competitive fitness compared to MHV-ExoN-AA. In contrast, a low passage (P10) MHV-ExoN-AA showed increased replication and competitive fitness without reversion of ExoN-AA. Finally, engineered reversion of ExoN-AA to ExoN-DE in the presence of ExoN-AA passage-adaptive mutations resulted in significant fitness loss. These results demonstrate that while reversion is possible, at least one alternative adaptive pathway is more rapidly advantageous than intermediate revertants and may alter the genetic background to render reversion detrimental to fitness. Our results provide an evolutionary rationale for lack of ExoN-AA reversion, illuminate potential multi-protein replicase interactions and coevolution, and support future studies aimed at stabilizing attenuated CoV ExoN-AA mutants.\n\nIMPORTANCECoronaviruses encode an exoribonuclease (ExoN) that is important for viral replication, fitness, and virulence, yet coronaviruses with a defective ExoN (ExoN-AA) have not reverted under diverse experimental conditions. In this study, we identify multiple impediments to MHV-ExoN-AA reversion. We show that ExoN-AA reversion is possible but evolutionarily unfavorable. Instead, compensatory mutations outside of ExoN-AA motif I are more accessible and beneficial than partial reversion. We also show that coevolution between replicase proteins over long-term passage partially compensates for ExoN-AA motif I but renders the virus inhospitable to a reverted ExoN. Our results reveal the evolutionary basis for the genetic stability of ExoN-inactivating mutations, illuminate complex functional and evolutionary relationships between coronavirus replicase proteins, and identify potential mechanisms for stabilization of ExoN-AA coronavirus mutants.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS coronavirus replicase proteins in pathogenesis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Much progress has been made in understanding the role of structural and accessory proteins in the pathogenesis of severe acute respiratory syndrome coronavirus (SARS-CoV) infections. The SARS epidemic also brought new attention to the proteins translated from ORF1a and ORF1b of the input genome RNA, also known as the replicase/transcriptase gene. Evidence for change within the ORF1ab coding sequence during the SARS epidemic, as well as evidence from studies with other coronaviruses, indicates that it is likely that the ORF1ab proteins play roles in virus pathogenesis distinct from or in addition to functions directly involved in viral replication. Recent reverse genetic studies have confirmed that proteins of ORF1ab may be involved in cellular signaling and modification of cellular gene expression, as well as virulence by mechanisms yet to be determined. Thus, the evolution of the ORF1ab proteins may be determined as much by issues of host range and virulence as they are by specific requirements for intracellular replication.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of ViroCyt(®) Virus Counter for Rapid Filovirus Quantitation</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">Development and evaluation of medical countermeasures for diagnostics, vaccines, and therapeutics requires production of standardized, reproducible, and well characterized virus preparations. For filoviruses this includes plaque assay for quantitation of infectious virus, transmission electron microscopy (TEM) for morphology and quantitation of virus particles, and real-time reverse transcription PCR for quantitation of viral RNA (qRT-PCR). The ViroCyt(®) Virus Counter (VC) 2100 (ViroCyt, Boulder, CO, USA) is a flow-based instrument capable of quantifying virus particles in solution. Using a proprietary combination of fluorescent dyes that stain both nucleic acid and protein in a single 30 min step, rapid, reproducible, and cost-effective quantification of filovirus particles was demonstrated. Using a seed stock of Ebola virus variant Kikwit, the linear range of the instrument was determined to be 2.8E+06 to 1.0E+09 virus particles per mL with coefficient of variation ranging from 9.4% to 31.5% for samples tested in triplicate. VC particle counts for various filovirus stocks were within one log of TEM particle counts. A linear relationship was established between the plaque assay, qRT-PCR, and the VC. VC results significantly correlated with both plaque assay and qRT-PCR. These results demonstrated that the VC is an easy, fast, and consistent method to quantify filoviruses in stock preparations.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can utilize GRP78 for attachment onto host cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus tropism is predominantly determined by the interaction between coronavirus spikes and the host receptors. In this regard, coronaviruses have evolved a complicated receptor-recognition system through their spike proteins. Spikes from highly related coronaviruses can recognize distinct receptors, whereas spikes of distant coronaviruses can employ the same cell-surface molecule for entry. Moreover, coronavirus spikes can recognize a broad range of cell-surface molecules in addition to the receptors and thereby can augment coronavirus attachment or entry. The receptor of Middle East respiratory syndrome coronavirus (MERS-CoV) is dipeptidyl peptidase 4 (DPP4). In this study, we identified membrane-associated 78-kDa glucose-regulated protein (GRP78) as an additional binding target of the MERS-CoV spike. Further analyses indicated that GRP78 could not independently render nonpermissive cells susceptible to MERS-CoV infection but could facilitate MERS-CoV entry into permissive cells by augmenting virus attachment. More importantly, by exploring potential interactions between GRP78 and spikes of other coronaviruses, we discovered that the highly conserved human GRP78 could interact with the spike protein of bat coronavirus HKU9 (bCoV-HKU9) and facilitate its attachment to the host cell surface. Taken together, our study has identified GRP78 as a host factor that can interact with the spike proteins of two Betacoronaviruses, the lineage C MERS-CoV and the lineage D bCoV-HKU9. The capacity of GRP78 to facilitate surface attachment of both a human coronavirus and a phylogenetically related bat coronavirus exemplifies the need for continuous surveillance of the evolution of animal coronaviruses to monitor their potential for human adaptations.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Adaptive Evolution Influences the Infectious Dose of MERS-CoV Necessary to Achieve Severe Respiratory Disease</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We recently established a mouse model (288–330(+/+)) that developed acute respiratory disease resembling human pathology following infection with a high dose (5×10(6) PFU) of mouse-adapted MERS-CoV (icMERSma1). Although this high dose conferred fatal respiratory disease in mice, achieving similar pathology at lower viral doses may more closely reflect naturally acquired infections. Through continued adaptive evolution of icMERSma1 we generated a novel mouse-adapted MERS-CoV (maM35c4) capable of achieving severe respiratory disease at doses between 10(3)–10(5) PFU. Novel mutations were identified in the maM35c4 genome that may be responsible for eliciting etiologies of acute respiratory distress syndrome at 10–1000 fold lower viral doses. Importantly, comparative genetics of the two mouse-adapted MERS strains allowed us to identify specific mutations that remained fixed through an additional 20 cycles of adaptive evolution. Our data indicate that the extent of MERS-CoV adaptation determines the minimal infectious dose required to achieve severe respiratory disease.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>An Opportunistic Pathogen Afforded Ample Opportunities: Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 2</span></i><p style="text-align: justify; text-justify: inter-word;">The human coronaviruses (CoV) include HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1, some of which have been known for decades. The severe acute respiratory syndrome (SARS) CoV briefly emerged into the human population but was controlled. In 2012, another novel severely human pathogenic CoV—the Middle East Respiratory Syndrome (MERS)-CoV—was identified in the Kingdom of Saudi Arabia; 80% of over 2000 human cases have been recorded over five years. Targeted research remains key to developing control strategies for MERS-CoV, a cause of mild illness in its camel reservoir. A new therapeutic toolbox being developed in response to MERS is also teaching us more about how CoVs cause disease. Travel-related cases continue to challenge the world’s surveillance and response capabilities, and more data are needed to understand unexplained primary transmission. Signs of genetic change have been recorded, but it remains unclear whether there is any impact on clinical disease. How camels came to carry the virus remains academic to the control of MERS. To date, human-to-human transmission has been inefficient, but virus surveillance, characterisation, and reporting are key to responding to any future change. MERS-CoV is not currently a pandemic threat; it is spread mainly with the aid of human habit and error.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus puts drug repurposing on the fast track</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Existing antivirals and knowledge gained from the SARS and MERS outbreaks gain traction as the fastest route to fight the current coronavirus epidemic. Existing antivirals and knowledge gained from the SARS and MERS outbreaks gain traction as the fastest route to fight the current coronavirus epidemic.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory Investigation and Phylogenetic Analysis of an Imported Middle East Respiratory Syndrome Coronavirus Case in Greece</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Apr 28</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid and reliable laboratory diagnosis of persons suspected of Middle East respiratory syndrome coronavirus (MERS-CoV) infection is important for timely implementation of infection control practices and disease management. In addition, monitoring molecular changes in the virus can help elucidate chains of transmission and identify mutations that might influence virus transmission efficiency. This was illustrated by a recent laboratory investigation we conducted on an imported MERS-CoV case in Greece. Two oropharyngeal swab specimens were collected on the 1(st) and 2(nd) day of patient hospitalization and tested using two real-time RT-PCR (rRT-PCR) assays targeting the UpE and Orf-1a regions of the MERS-CoV genome and RT-PCR and partial sequencing of RNA-dependent RNA polymerase and nucleocapsid genes. Serum specimens were also collected and serological test were performed. Results from the first swab sample were inconclusive while the second swab was strongly positive for MERS-CoV RNA by rRT-PCR and confirmed positive by RT-PCR and partial gene sequencing. Positive serologic test results further confirmed MERS-CoV infection. Full-length nucleocapsid and spike gene coding sequences were later obtained from the positive swab sample. Phylogenetic analysis revealed that the virus was closely related to recent human-derived MERS-CoV strains obtained in Jeddah and Makkah, Saudi Arabia, in April 2014 and dromedary camels in Saudi Arabia and Qatar. These findings were consistent with the patient’s history. We also identified a unique amino acid substitution in the spike receptor binding domain that may have implications for receptor binding efficiency. Our initial inconclusive rRT-PCR results highlight the importance of collecting multiple specimens from suspect MERS-CoV cases and particularly specimens from the lower respiratory tract.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Attenuation of replication by a 29 nucleotide deletion in SARS-coronavirus acquired during the early stages of human-to-human transmission</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 11</span></i><p style="text-align: justify; text-justify: inter-word;">A 29 nucleotide deletion in open reading frame 8 (ORF8) is the most obvious genetic change in severe acute respiratory syndrome coronavirus (SARS-CoV) during its emergence in humans. In spite of intense study, it remains unclear whether the deletion actually reflects adaptation to humans. Here we engineered full, partially deleted (−29 nt), and fully deleted ORF8 into a SARS-CoV infectious cDNA clone, strain Frankfurt-1. Replication of the resulting viruses was compared in primate cell cultures as well as Rhinolophus bat cells made permissive for SARS-CoV replication by lentiviral transduction of the human angiotensin-converting enzyme 2 receptor. Cells from cotton rat, goat, and sheep provided control scenarios that represent host systems in which SARS-CoV is neither endemic nor epidemic. Independent of the cell system, the truncation of ORF8 (29 nt deletion) decreased replication up to 23-fold. The effect was independent of the type I interferon response. The 29 nt deletion in SARS-CoV is a deleterious mutation acquired along the initial human-to-human transmission chain. The resulting loss of fitness may be due to a founder effect, which has rarely been documented in processes of viral emergence. These results have important implications for the retrospective assessment of the threat posed by SARS.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Attenuation of replication by a 29 nucleotide deletion in SARS-coronavirus acquired during the early stages of human-to-human transmission</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 11</span></i><p style="text-align: justify; text-justify: inter-word;">A 29 nucleotide deletion in open reading frame 8 (ORF8) is the most obvious genetic change in severe acute respiratory syndrome coronavirus (SARS-CoV) during its emergence in humans. In spite of intense study, it remains unclear whether the deletion actually reflects adaptation to humans. Here we engineered full, partially deleted (−29 nt), and fully deleted ORF8 into a SARS-CoV infectious cDNA clone, strain Frankfurt-1. Replication of the resulting viruses was compared in primate cell cultures as well as Rhinolophus bat cells made permissive for SARS-CoV replication by lentiviral transduction of the human angiotensin-converting enzyme 2 receptor. Cells from cotton rat, goat, and sheep provided control scenarios that represent host systems in which SARS-CoV is neither endemic nor epidemic. Independent of the cell system, the truncation of ORF8 (29 nt deletion) decreased replication up to 23-fold. The effect was independent of the type I interferon response. The 29 nt deletion in SARS-CoV is a deleterious mutation acquired along the initial human-to-human transmission chain. The resulting loss of fitness may be due to a founder effect, which has rarely been documented in processes of viral emergence. These results have important implications for the retrospective assessment of the threat posed by SARS.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Tracking the Evolution of the SARS Coronavirus Using High-Throughput, High-Density Resequencing Arrays</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Mutations in the SARS-Coronavirus (SARS-CoV) can alter its clinical presentation, and the study of its mutation patterns in human populations can facilitate contact tracing. Here, we describe the development and validation of an oligonucleotide resequencing array for interrogating the entire 30-kb SARS-CoV genome in a rapid, cost-effective fashion. Using this platform, we sequenced SARS-CoV genomes from Vero cell culture isolates of 12 patients and directly from four patient tissues. The sequence obtained from the array is highly reproducible, accurate (>99.99% accuracy) and capable of identifying known and novel variants of SARS-CoV. Notably, we applied this technology to a field specimen of probable SARS and rapidly deduced its infectious source. We demonstrate that array-based resequencing-by-hybridization is a fast, reliable, and economical alternative to capillary sequencing for obtaining SARS-CoV genomic sequence on a population scale, making this an ideal platform for the global monitoring of SARS-CoV and other small-genome pathogens.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>2019 Novel Coronavirus can be detected in urine, blood, anal swabs and oropharyngeal swabs samples</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We tested samples collected from nine patients diagnosed with coronavirus disease 2019 (COVID-19). The virus was found in urine, blood, anal swabs and oropharyngeal swabs. It is the first time for SARS-CoV-2 found in urine, though no urinary irritation was found.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Role of genomic and proteomic tools in the study of host–virus interactions and virus evolution</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viruses have short replication cycles and produce genomic variants within a host, a process that seems to adapt to their specific host and also enable them to infect new hosts. The recent emergence of viral genomic variants from the circulating pool within the host population and re-emergence of the old ones are posing serious threat to agriculture, animal husbandry and humanity as a whole. This review assesses the potential role of genomic and proteomic tools that can monitor not only the course of infection and pathogenesis, but also predict the pandemic or zoonotic epidemic potential of a virus in a previously exposed or immunologically naive biological population.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a recombination-resistant coronavirus as a broadly applicable, rapidly implementable vaccine platform</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging zoonotic viral diseases are major threats to global health, economic stability, and national security. Vaccines are key for reducing coronaviral disease burden; however, the utility of live-attenuated vaccines is limited by risks of reversion or repair. Because of their history of emergence events due to their prevalence in zoonotic pools, designing live-attenuated coronavirus vaccines that can be rapidly and broadly implemented is essential for outbreak preparedness. Here, we show that coronaviruses with completely rewired transcription regulatory networks (TRNs) are effective vaccines against SARS-CoV. The TRN-rewired viruses are attenuated and protect against lethal SARS-CoV challenge. While a 3-nt rewired TRN reverts via second-site mutation upon serial passage, a 7-nt rewired TRN is more stable, suggesting that a more extensively rewired TRN might be essential for avoiding growth selection. In summary, rewiring the TRN is a feasible strategy for limiting reversion in an effective live-attenuated coronavirus vaccine candidate that is potentially portable across the Nidovirales order.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a recombination-resistant coronavirus as a broadly applicable, rapidly implementable vaccine platform</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging zoonotic viral diseases are major threats to global health, economic stability, and national security. Vaccines are key for reducing coronaviral disease burden; however, the utility of live-attenuated vaccines is limited by risks of reversion or repair. Because of their history of emergence events due to their prevalence in zoonotic pools, designing live-attenuated coronavirus vaccines that can be rapidly and broadly implemented is essential for outbreak preparedness. Here, we show that coronaviruses with completely rewired transcription regulatory networks (TRNs) are effective vaccines against SARS-CoV. The TRN-rewired viruses are attenuated and protect against lethal SARS-CoV challenge. While a 3-nt rewired TRN reverts via second-site mutation upon serial passage, a 7-nt rewired TRN is more stable, suggesting that a more extensively rewired TRN might be essential for avoiding growth selection. In summary, rewiring the TRN is a feasible strategy for limiting reversion in an effective live-attenuated coronavirus vaccine candidate that is potentially portable across the Nidovirales order.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombination and convergent evolution led to the emergence of 2019 Wuhan coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;">The recent outbreak of a new coronavirus (2019-nCoV) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS and MERS, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the 2019-nCoV involving the RBD. Compared with bat isolates, the recent ancestors of 2019-nCoV accumulated a high number of amino acid substitutions in the RBD and likewise in a region of polyprotein Orf1a that is critical for viral replication and transcription. Among these recent mutations, we identify amino acid substitutions common to the SARS 2003 outbreak isolates in positions 427N and 436Y, indicating potential adaptive convergent evolution. Both 427N and 436Y belong to a helix that appears to interact with the human ACE2 receptor. In sum, we propose a two-hit scenario in the emergence of the 2019-nCoV virus whereby the 2019-nCoV ancestors in bats first acquired genetic characteristics of SARS by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, those recombinants underwent convergent evolution.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Amino Acid Substitutions in the S2 Region Enhance Severe Acute Respiratory Syndrome Coronavirus Infectivity in Rat Angiotensin-Converting Enzyme 2-Expressing Cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">To clarify the molecular basis of severe acute respiratory syndrome coronavirus (SARS-CoV) adaptation to different host species, we serially passaged SARS-CoV in rat angiotensin-converting enzyme 2 (ACE2)-expressing cells. After 15 passages, the virus (Rat-P15) came to replicate effectively in rat ACE2-expressing cells. Two amino acid substitutions in the S2 region were found on the Rat-P15 S gene. Analyses of the infectivity of the pseudotype-bearing S protein indicated that the two substitutions in the S2 region, especially the S950F substitution, were responsible for efficient infection. Therefore, virus adaptation to different host species can be induced by amino acid substitutions in the S2 region.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Cleavage of a Neuroinvasive Human Respiratory Virus Spike Glycoprotein by Proprotein Convertases Modulates Neurovirulence and Virus Spread within the Central Nervous System</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Nov 6</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses (HCoV) are respiratory pathogens that may be associated with the development of neurological diseases, in view of their neuroinvasive and neurotropic properties. The viral spike (S) glycoprotein is a major virulence factor for several coronavirus species, including the OC43 strain of HCoV (HCoV-OC43). In an attempt to study the role of this protein in virus spread within the central nervous system (CNS) and neurovirulence, as well as to identify amino acid residues important for such functions, we compared the sequence of the S gene found in the laboratory reference strain HCoV-OC43 ATCC VR-759 to S sequences of viruses detected in clinical isolates from the human respiratory tract. We identified one predominant mutation at amino acid 758 (from RRSR↓ G (758) to RRSR↓R (758)), which introduces a putative furin-like cleavage (↓) site. Using a molecular cDNA infectious clone to generate a corresponding recombinant virus, we show for the first time that such point mutation in the HCoV-OC43 S glycoprotein creates a functional cleavage site between the S1 and S2 portions of the S protein. While the corresponding recombinant virus retained its neuroinvasive properties, this mutation led to decreased neurovirulence while potentially modifying the mode of virus spread, likely leading to a limited dissemination within the CNS. Taken together, these results are consistent with the adaptation of HCoV-OC43 to the CNS environment, resulting from the selection of quasi-species harboring mutations that lead to amino acid changes in viral genes, like the S gene in HCoV-OC43, which may contribute to a more efficient establishment of a less pathogenic but persistent CNS infection. This adaptative mechanism could potentially be associated with human encephalitis or other neurological degenerative pathologies.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Mechanisms of Zoonotic Severe Acute Respiratory Syndrome Coronavirus Host Range Expansion in Human Airway Epithelium</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In 2003, severe acute respiratory syndrome coronavirus (SARS-CoV) emerged and caused over 8,000 human cases of infection and more than 700 deaths worldwide. Zoonotic SARS-CoV likely evolved to infect humans by a series of transmission events between humans and animals for sale in China. Using synthetic biology, we engineered the spike protein (S) from a civet strain, SZ16, into our epidemic strain infectious clone, creating the chimeric virus icSZ16-S, which was infectious but yielded progeny viruses incapable of propagating in vitro. After introducing a K479N mutation within the S receptor binding domain (RBD) of SZ16, the recombinant virus (icSZ16-S K479N) replicated in Vero cells but was severely debilitated in growth. The in vitro evolution of icSZ16-S K479N on human airway epithelial (HAE) cells produced two viruses (icSZ16-S K479N D8 and D22) with enhanced growth on HAE cells and on delayed brain tumor cells expressing the SARS-CoV receptor, human angiotensin I converting enzyme 2 (hACE2). The icSZ16-S K479N D8 and D22 virus RBDs contained mutations in ACE2 contact residues, Y442F and L472F, that remodeled S interactions with hACE2. Further, these viruses were neutralized by a human monoclonal antibody (MAb), S230.15, but the parent icSZ16-S K479N strain was eight times more resistant than the mutants. These data suggest that the human adaptation of zoonotic SARS-CoV strains may select for some variants that are highly susceptible to select MAbs that bind to RBDs. The epidemic, icSZ16-S K479N, and icSZ16-S K479N D22 viruses replicate similarly in the BALB/c mouse lung, highlighting the potential use of these zoonotic spike SARS-CoVs to assess vaccine or serotherapy efficacy in vivo.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>A new transmission route for the propagation of the SARS-CoV-2 coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.</jats:p></p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-04</span></i><p style="text-align: justify; text-justify: inter-word;">Novel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid generation of a mouse model for Middle East respiratory syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In this era of continued emergence of zoonotic virus infections, the rapid development of rodent models represents a critical barrier to public health preparedness, including the testing of antivirus therapy and vaccines. The Middle East respiratory syndrome coronavirus (MERS-CoV) was recently identified as the causative agent of a severe pneumonia. Given the ability of coronavirus to rapidly adapt to new hosts, a major public health concern is that MERS-CoV will further adapt to replication in humans, triggering a pandemic. No small-animal model for this infection is currently available, but studies suggest that virus entry factors can confer virus susceptibility. Here, we show that mice were sensitized to MERS-CoV infection by prior transduction with adenoviral vectors expressing the human host-cell receptor dipeptidyl peptidase 4. Mice developed a pneumonia characterized by extensive inflammatory-cell infiltration with virus clearance occurring 6–8 d after infection. Clinical disease and histopathological changes were more severe in the absence of type-I IFN signaling whereas the T-cell response was required for virus clearance. Using these mice, we demonstrated the efficacy of a therapeutic intervention (poly I:C) and a potential vaccine [Venezuelan equine encephalitis replicon particles expressing MERS-CoV spike protein]. We also found little protective cross-reactivity between MERS-CoV and the severe acute respiratory syndrome-CoV. Our results demonstrate that this system will be useful for MERS-CoV studies and for the rapid development of relevant animal models for emerging respiratory viral infections.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Viral Population Diversity in Adaptation of Bovine Coronavirus to New Host Environments</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 7</span></i><p style="text-align: justify; text-justify: inter-word;">The high mutation rate of RNA viruses enables a diverse genetic population of viral genotypes to exist within a single infected host. In-host genetic diversity could better position the virus population to respond and adapt to a diverse array of selective pressures such as host-switching events. Multiple new coronaviruses, including SARS, have been identified in human samples just within the last ten years, demonstrating the potential of coronaviruses as emergent human pathogens. Deep sequencing was used to characterize genomic changes in coronavirus quasispecies during simulated host-switching. Three bovine nasal samples infected with bovine coronavirus were used to infect human and bovine macrophage and lung cell lines. The virus reproduced relatively well in macrophages, but the lung cell lines were not infected efficiently enough to allow passage of non lab-adapted samples. Approximately 12 kb of the genome was amplified before and after passage and sequenced at average coverages of nearly 950×(454 sequencing) and 38,000×(Illumina). The consensus sequence of many of the passaged samples had a 12 nucleotide insert in the consensus sequence of the spike gene, and multiple point mutations were associated with the presence of the insert. Deep sequencing revealed that the insert was present but very rare in the unpassaged samples and could quickly shift to dominate the population when placed in a different environment. The insert coded for three arginine residues, occurred in a region associated with fusion entry into host cells, and may allow infection of new cell types via heparin sulfate binding. Analysis of the deep sequencing data indicated that two distinct genotypes circulated at different frequency levels in each sample, and support the hypothesis that the mutations present in passaged strains were “selected” from a pre-existing pool rather than through de novo mutation and subsequent population fixation.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>First two months of the 2019 Coronavirus Disease (COVID-19) epidemic in China: real-time surveillance and evaluation with a second derivative model</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Participation of both Host and Virus Factors in Induction of Severe Acute Respiratory Syndrome (SARS) in F344 Rats Infected with SARS Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">To understand the pathogenesis and develop an animal model of severe acute respiratory syndrome (SARS)-associated coronavirus (SARS-CoV), the Frankfurt 1 SARS-CoV isolate was passaged serially in young F344 rats. Young rats were susceptible to SARS-CoV but cleared the virus rapidly within 3 to 5 days of intranasal inoculation. After 10 serial passages, replication and virulence of SARS-CoV were increased in the respiratory tract of young rats without clinical signs. By contrast, adult rats infected with the passaged virus showed respiratory symptoms and severe pathological lesions in the lung. Levels of inflammatory cytokines in sera and lung tissues were significantly higher in adult F344 rats than in young rats. During in vivo passage of SARS-CoV, a single amino acid substitution was introduced within the binding domain of the viral spike protein recognizing angiotensin-converting enzyme 2 (ACE2), which is known as a SARS-CoV receptor. The rat-passaged virus more efficiently infected CHO cells expressing rat ACE2 than did the original isolate. These results strongly indicate that host and virus factors such as advanced age and virus adaptation are critical for the development of SARS in rats.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Inhibition, Escape, and Attenuated Growth of Severe Acute Respiratory Syndrome Coronavirus Treated with Antisense Morpholino Oligomers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recently emerged severe acute respiratory syndrome coronavirus (SARS-CoV) is a potent pathogen of humans and is capable of rapid global spread. Peptide-conjugated antisense morpholino oligomers (P-PMO) were designed to bind by base pairing to specific sequences in the SARS-CoV (Tor2 strain) genome. The P-PMO were tested for their capacity to inhibit production of infectious virus as well as to probe the function of conserved viral RNA motifs and secondary structures. Several virus-targeted P-PMO and a random-sequence control P-PMO showed low inhibitory activity against SARS coronavirus. Certain other virus-targeted P-PMO reduced virus-induced cytopathology and cell-to-cell spread as a consequence of decreasing viral amplification. Active P-PMO were effective when administered at any time prior to peak viral synthesis and exerted sustained antiviral effects while present in culture medium. P-PMO showed low nonspecific inhibitory activity against translation of nontargeted RNA or growth of the arenavirus lymphocytic choriomeningitis virus. Two P-PMO targeting the viral transcription-regulatory sequence (TRS) region in the 5′ untranslated region were the most effective inhibitors tested. After several viral passages in the presence of a TRS-targeted P-PMO, partially drug-resistant SARS-CoV mutants arose which contained three contiguous base point mutations at the binding site of a TRS-targeted P-PMO. Those partially resistant viruses grew more slowly and formed smaller plaques than wild-type SARS-CoV. These results suggest PMO compounds have powerful therapeutic and investigative potential toward coronavirus infection.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination Attenuation Offers Strategy for Live Attenuated Coronavirus Vaccines</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2′O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the severe acute respiratory syndrome-CoV mouse model, an NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2′O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2′O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live attenuated coronavirus vaccines. IMPORTANCE Emergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations such as the aged and immunocompromised. This study illustrates that despite success in young models, the 2′O methyltransferase mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the 2′O methyltransferase mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for 2′O methyltransferase attenuation as a major component of future live attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple and rapid approach for screening of SARS-coronavirus genotypes: an evaluation study</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Oct 18</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The Severe Acute Respiratory Syndrome (SARS) was a newly emerged infectious disease which caused a global epidemic in 2002–2003. Sequence analysis of SARS-coronavirus isolates revealed that specific genotypes predominated at different periods of the epidemic. This information can be used as a footprint for tracing the epidemiology of infections and monitor viral evolution. However, direct sequencing analysis of a large number of clinical samples is cumbersome and time consuming. We present here a simple and rapid assay for the screening of SARS-coronavirus genotypes based on the use of fluorogenic oligonucleotide probes for allelic discrimination. METHODS: Thirty SARS patients were recruited. Allelic discrimination assays were developed based on the use of fluorogenic oligonucleotide probes (TaqMan). Genotyping of the SARS-coronavirus isolates obtained from these patients were carried out by the allelic discrimination assays and confirmed by direct sequencing. RESULTS: Genotyping based on the allelic discrimination assays were fully concordant with direct sequencing. All of the 30 SARS-coronavirus genotypes studied were characteristic of genotypes previously documented to be associated with the latter part of the epidemic. Seven of the isolates contained a previously reported major deletion but in patients not epidemiologically related to the previously studied cohort. CONCLUSION: We have developed a simple and accurate method for the characterization and screening of SARS-coronavirus genotypes. It is a promising tool for the study of epidemiological relationships between documented cases during an outbreak.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Synthetic Reconstruction of Zoonotic and Early Human Severe Acute Respiratory Syndrome Coronavirus Isolates That Produce Fatal Disease in Aged Mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome (SARS) epidemic was characterized by high mortality rates in the elderly. The molecular mechanisms that govern enhanced susceptibility of elderly populations are not known, and robust animal models are needed that recapitulate the increased pathogenic phenotype noted with increasing age. Using synthetic biology and reverse genetics, we describe the construction of a panel of isogenic SARS coronavirus (SARS-CoV) strains bearing variant spike glycoproteins that are representative of zoonotic strains found in palm civets and raccoon dogs, as well as isolates spanning the early, middle, and late phases of the SARS-CoV epidemic. The recombinant viruses replicated efficiently in cell culture and demonstrated variable sensitivities to neutralization with antibodies. The human but not the zoonotic variants replicated efficiently in human airway epithelial cultures, supporting earlier hypotheses that zoonotic isolates are less pathogenic in humans but can evolve into highly pathogenic strains. All viruses replicated efficiently, but none produced clinical disease or death in young animals. In contrast, severe clinical disease, diffuse alveolar damage, hyaline membrane formation, alveolitis, and death were noted in 12-month-old mice inoculated with the palm civet HC/SZ/61/03 strain or early-human-phase GZ02 variants but not with related middle- and late-phase epidemic or raccoon dog strains. This panel of SARS-CoV recombinants bearing zoonotic and human epidemic spike glycoproteins will provide heterologous challenge models for testing vaccine efficacy against zoonotic reintroductions as well as provide the appropriate model system for elucidating the complex virus-host interactions that contribute to more-severe and fatal SARS-CoV disease and acute respiratory distress in the elderly.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 Vaccine Pipeline: an Overview</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19).</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Four ways researchers are responding to the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts. How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Caution: The clinical characteristics of COVID-19 patients at admission are changing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background  With the emergence of 4rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 (COVID-19) has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Methods Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. Results  Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24).  Conclusions The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.</jats:p></p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Broadening of Neutralization Activity to Directly Block a Dominant Antibody-Driven SARS-Coronavirus Evolution Pathway</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Nov 7</span></i><p style="text-align: justify; text-justify: inter-word;">Phylogenetic analyses have provided strong evidence that amino acid changes in spike (S) protein of animal and human SARS coronaviruses (SARS-CoVs) during and between two zoonotic transfers (2002/03 and 2003/04) are the result of positive selection. While several studies support that some amino acid changes between animal and human viruses are the result of inter-species adaptation, the role of neutralizing antibodies (nAbs) in driving SARS-CoV evolution, particularly during intra-species transmission, is unknown. A detailed examination of SARS-CoV infected animal and human convalescent sera could provide evidence of nAb pressure which, if found, may lead to strategies to effectively block virus evolution pathways by broadening the activity of nAbs. Here we show, by focusing on a dominant neutralization epitope, that contemporaneous- and cross-strain nAb responses against SARS-CoV spike protein exist during natural infection. In vitro immune pressure on this epitope using 2002/03 strain-specific nAb 80R recapitulated a dominant escape mutation that was present in all 2003/04 animal and human viruses. Strategies to block this nAb escape/naturally occurring evolution pathway by generating broad nAbs (BnAbs) with activity against 80R escape mutants and both 2002/03 and 2003/04 strains were explored. Structure-based amino acid changes in an activation-induced cytidine deaminase (AID) “hot spot” in a light chain CDR (complementarity determining region) alone, introduced through shuffling of naturally occurring non-immune human VL chain repertoire or by targeted mutagenesis, were successful in generating these BnAbs. These results demonstrate that nAb-mediated immune pressure is likely a driving force for positive selection during intra-species transmission of SARS-CoV. Somatic hypermutation (SHM) of a single VL CDR can markedly broaden the activity of a strain-specific nAb. The strategies investigated in this study, in particular the use of structural information in combination of chain-shuffling as well as hot-spot CDR mutagenesis, can be exploited to broaden neutralization activity, to improve anti-viral nAb therapies, and directly manipulate virus evolution.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Proofreading-Deficient Coronaviruses Adapt for Increased Fitness over Long-Term Passage without Reversion of Exoribonuclease-Inactivating Mutations</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 7</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus (CoV) RNA genome is the largest among the single-stranded positive-sense RNA viruses. CoVs encode a proofreading 3′-to-5′ exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in severe acute respiratory syndrome-associated coronavirus (SARS-CoV) and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold-decreased fidelity, and increased susceptibility to nucleoside analogues. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with wild-type MHV (WT-MHV). Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication and increased competitive fitness without reversion at the ExoN(-) active site. Furthermore, MHV-ExoN(-) P250 was less susceptible than MHV-ExoN(-) P3 to multiple nucleoside analogues, suggesting that MHV-ExoN(-) was under selection for increased replication fidelity. We subsequently identified novel amino acid changes within the RNA-dependent RNA polymerase and nsp14 of MHV-ExoN(-) P250 that partially accounted for the reduced susceptibility to nucleoside analogues. Our results suggest that increased replication fidelity is selected in ExoN(-) CoVs and that there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and indicate that multiple replicase proteins could compensate for ExoN functions during replication.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Proofreading-Deficient Coronaviruses Adapt for Increased Fitness over Long-Term Passage without Reversion of Exoribonuclease-Inactivating Mutations</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 7</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus (CoV) RNA genome is the largest among the single-stranded positive-sense RNA viruses. CoVs encode a proofreading 3′-to-5′ exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in severe acute respiratory syndrome-associated coronavirus (SARS-CoV) and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold-decreased fidelity, and increased susceptibility to nucleoside analogues. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with wild-type MHV (WT-MHV). Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication and increased competitive fitness without reversion at the ExoN(-) active site. Furthermore, MHV-ExoN(-) P250 was less susceptible than MHV-ExoN(-) P3 to multiple nucleoside analogues, suggesting that MHV-ExoN(-) was under selection for increased replication fidelity. We subsequently identified novel amino acid changes within the RNA-dependent RNA polymerase and nsp14 of MHV-ExoN(-) P250 that partially accounted for the reduced susceptibility to nucleoside analogues. Our results suggest that increased replication fidelity is selected in ExoN(-) CoVs and that there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and indicate that multiple replicase proteins could compensate for ExoN functions during replication.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Proofreading-Deficient Coronaviruses Adapt for Increased Fitness over Long-Term Passage without Reversion of Exoribonuclease-Inactivating Mutations</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 7</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus (CoV) RNA genome is the largest among the single-stranded positive-sense RNA viruses. CoVs encode a proofreading 3′-to-5′ exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in severe acute respiratory syndrome-associated coronavirus (SARS-CoV) and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold-decreased fidelity, and increased susceptibility to nucleoside analogues. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with wild-type MHV (WT-MHV). Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication and increased competitive fitness without reversion at the ExoN(-) active site. Furthermore, MHV-ExoN(-) P250 was less susceptible than MHV-ExoN(-) P3 to multiple nucleoside analogues, suggesting that MHV-ExoN(-) was under selection for increased replication fidelity. We subsequently identified novel amino acid changes within the RNA-dependent RNA polymerase and nsp14 of MHV-ExoN(-) P250 that partially accounted for the reduced susceptibility to nucleoside analogues. Our results suggest that increased replication fidelity is selected in ExoN(-) CoVs and that there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and indicate that multiple replicase proteins could compensate for ExoN functions during replication.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-22</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral metagenomics, protein structure, and reverse genetics: key strategies for investigating coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viral metagenomics, modeling of protein structure, and manipulation of viral genetics are key approaches that have laid the foundations of our understanding of coronavirus biology. In this review, we discuss the major advances each method has provided and discuss how future studies should leverage these strategies synergistically to answer novel questions.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Quarantined at home now, U.S. scientist describes his visit to China’s hot zone</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 13 February, Clifford Lane went to a Washington, D.C.–area airport to catch a flight to Japan, where he would help launch a study of an experimental drug, remdesivir, against coronavirus disease 2019 (COVID-19). Lane is a deputy director at the U.S. National Institute of Allergy and Infectious Diseases and a right-hand man to Anthony Fauci, head of NIAID and the top research scientist in the country advising the White House on the outbreak of the virus. As Lane waited to board his plane, he was told that his final destination had changed. “I get an email, ‘You need to go to China.’ It’s like, are you kidding?” Lane had been selected as one of two U.S. scientists to join a World Health Organization team of 13 international researchers who would tour five different cities with 12 Chinese colleagues to get a firsthand look at the coronavirus epidemic there. The joint mission, which ran from 16–23 February led to a report that offered more details about the clinical course of COVID-19 and the epidemiology in China than had appeared anywhere before.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Human Monoclonal Antibody 80R for Immunoprophylaxis of Severe Acute Respiratory Syndrome by an Animal Study, Epitope Mapping, and Analysis of Spike Variants</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In this report, the antiviral activity of 80R immunoglobulin G1 (IgG1), a human monoclonal antibody against severe acute respiratory syndrome coronavirus (SARS-CoV) spike (S) protein that acts as a viral entry inhibitor in vitro, was investigated in vivo in a mouse model. When 80R IgG1 was given prophylactically to mice at doses therapeutically achievable in humans, viral replication was reduced by more than 4 orders of magnitude to below assay limits. The essential core region of S protein required for 80R binding was identified as a conformationally sensitive fragment (residues 324 to 503) that overlaps the receptor ACE2-binding domain. Amino acids critical for 80R binding were identified. In addition, the effects of various 80R-binding domain amino acid substitutions which occur in SARS-like-CoV from civet cats, and which evolved during the 2002/2003 outbreak and in a 2003/2004 Guangdong index patient, were analyzed. The results demonstrated that the vast majority of SARS-CoVs are sensitive to 80R. We propose that by establishing the susceptibility and resistance profiles of newly emerging SARS-CoVs through early S1 genotyping of the core 180-amino-acid neutralizing epitope of 80R, an effective immunoprophylaxis strategy with 80R should be possible in an outbreak setting. Our study also cautions that for any prophylaxis strategy based on neutralizing antibody responses, whether by passive or active immunization, a genotyping monitor will be necessary for effective use.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Of chloroquine and COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Recent publications have brought attention to the possible benefit of chloroquine, a broadly used antimalarial drug, in the treatment of patients infected by the novel emerged coronavirus (SARS-CoV-2). The scientific community should consider this information in light of previous experiments with chloroquine in the field of antiviral research.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS to novel coronavirus – old lessons and new lessons</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The response to the novel coronavirus outbreak in China suggests that many of the lessons from the 2003 SARS epidemic have been implemented and the response improved as a consequence. Nevertheless some questions remain and not all lessons have been successful. The national and international response demonstrates the complex link between public health, science and politics when an outbreak threatens to impact on global economies and reputations. The unprecedented measures implemented in China are a bold attempt to control the outbreak – we need to understand their effectiveness to balance costs and benefits for similar events in the future.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic diversity of SARS-CoV-2 in Coronavirus Disease 2019 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) has infected more than 75,000 individuals and spread to over 20 countries. It is still unclear how fast the virus evolved and how the virus interacts with other microorganisms in the lung.We have conducted metatranscriptome sequencing for the bronchoalveolar lavage fluid of eight SARS-CoV-2 patients, 25 community-acquired pneumonia (CAP) patients, and 20 healthy controls.The median number of intra-host variants was 1-4 in SARS-CoV-2 infected patients, which ranged between 0 and 51 in different samples. The distribution of variants on genes was similar to those observed in the population data (110 sequences). However, very few intra-host variants were observed in the population as polymorphism, implying either a bottleneck or purifying selection involved in the transmission of the virus, or a consequence of the limited diversity represented in the current polymorphism data. Although current evidence did not support the transmission of intra-host variants in a person-to-person spread, the risk should not be overlooked. The microbiota in SARS-CoV-2 infected patients was similar to those in CAP, either dominated by the pathogens or with elevated levels of oral and upper respiratory commensal bacteria.SARS-CoV-2 evolves in vivo after infection, which may affect its virulence, infectivity, and transmissibility. Although how the intra-host variant spreads in the population is still elusive, it is necessary to strengthen the surveillance of the viral evolution in the population and associated clinical changes.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence</h3>
<i class="tooltip">2015<span class="tooltiptext">2015</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome (MERS)-CoV underscores the threat of cross-species transmission events leading to outbreaks in humans. Here we examine the disease potential of a SARS-like virus, SHC014-CoV, which is currently circulating in Chinese horseshoe bat populations. Using the SARS-CoV reverse genetics system, we generated and characterized a chimeric virus expressing the spike of bat coronavirus SHC014 in a mouse-adapted SARS-CoV backbone. The results indicate that group 2b viruses encoding the SHC014 spike in a wild-type backbone can efficiently use multiple orthologs of the SARS receptor human angiotensin converting enzyme II (ACE2), replicate efficiently in primary human airway cells and achieve in vitro titers equivalent to epidemic strains of SARS-CoV. Additionally, in vivo experiments demonstrate replication of the chimeric virus in mouse lung with notable pathogenesis. Evaluation of available SARS-based immune-therapeutic and prophylactic modalities revealed poor efficacy; both monoclonal antibody and vaccine approaches failed to neutralize and protect from infection with CoVs using the novel spike protein. On the basis of these findings, we synthetically re-derived an infectious full-length SHC014 recombinant virus and demonstrate robust viral replication both in vitro and in vivo. Our work suggests a potential risk of SARS-CoV re-emergence from viruses currently circulating in bat populations.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Naturally Occurring Anti-Escherichia coli Protein Antibodies in the Sera of Healthy Humans Cause Analytical Interference in a Recombinant Nucleocapsid Protein-Based Enzyme-Linked Immunosorbent Assay for Serodiagnosis of Severe Acute Respiratory Syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We reported the analytical interference of anti-Escherichia coli protein (EP) antibodies in human sera and residual EP in a recombinant nucleocapsid protein-based enzyme-linked immunosorbent assay as a possible source of false positives in severe acute respiratory syndrome serodiagnosis. The rate of false positives was significantly reduced by adding mouse anti-EP antiserum in the blocking step.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution, Safety, and Highly Pathogenic Influenza Viruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Experience with influenza has shown that predictions of virus phenotype or fitness from nucleotide sequence are imperfect, and that predicting the timing and course of evolution is extremely difficult. Such uncertainty means that the risk of experiments with mammalian-transmissible, possibly highly virulent influenza viruses remains high even if some aspects of their laboratory biology are reassuring; it also implies limitations on the ability of laboratory observations to guide interpretation of surveillance of strains in the field. Given these considerations, we propose that future experiments with virulent pathogens whose accidental or deliberate release could lead to extensive spread in human populations should be limited by explicit risk-benefit considerations, in the United States and worldwide.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: what is next for public health?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The WHO Scientific and Technical Advisory Group for Infectious Hazards (STAG-IH), working with the WHO secretariat, reviewed available information about the outbreaks of 2019 novel coronavirus disease (COVID-19) on Feb 7, 2020, in Geneva, Switzerland, and concluded that the continuing strategy of containment for elimination should continue, and that the coming 2–3 weeks through to the end of February, 2020, will be crucial to monitor the situation of community transmission to update WHO public health recommendations if required.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding loops in alphacoronavirus adaptation and evolution</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 23</span></i><p style="text-align: justify; text-justify: inter-word;">RNA viruses are characterized by a high mutation rate, a buffer against environmental change. Nevertheless, the means by which random mutation improves viral fitness is not well characterized. Here we report the X-ray crystal structure of the receptor-binding domain (RBD) of the human coronavirus, HCoV-229E, in complex with the ectodomain of its receptor, aminopeptidase N (APN). Three extended loops are solely responsible for receptor binding and the evolution of HCoV-229E and its close relatives is accompanied by changing loop–receptor interactions. Phylogenetic analysis shows that the natural HCoV-229E receptor-binding loop variation observed defines six RBD classes whose viruses have successively replaced each other in the human population over the past 50 years. These RBD classes differ in their affinity for APN and their ability to bind an HCoV-229E neutralizing antibody. Together, our results provide a model for alphacoronavirus adaptation and evolution based on the use of extended loops for receptor binding.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding loops in alphacoronavirus adaptation and evolution</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 23</span></i><p style="text-align: justify; text-justify: inter-word;">RNA viruses are characterized by a high mutation rate, a buffer against environmental change. Nevertheless, the means by which random mutation improves viral fitness is not well characterized. Here we report the X-ray crystal structure of the receptor-binding domain (RBD) of the human coronavirus, HCoV-229E, in complex with the ectodomain of its receptor, aminopeptidase N (APN). Three extended loops are solely responsible for receptor binding and the evolution of HCoV-229E and its close relatives is accompanied by changing loop–receptor interactions. Phylogenetic analysis shows that the natural HCoV-229E receptor-binding loop variation observed defines six RBD classes whose viruses have successively replaced each other in the human population over the past 50 years. These RBD classes differ in their affinity for APN and their ability to bind an HCoV-229E neutralizing antibody. Together, our results provide a model for alphacoronavirus adaptation and evolution based on the use of extended loops for receptor binding.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding loops in alphacoronavirus adaptation and evolution</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 23</span></i><p style="text-align: justify; text-justify: inter-word;">RNA viruses are characterized by a high mutation rate, a buffer against environmental change. Nevertheless, the means by which random mutation improves viral fitness is not well characterized. Here we report the X-ray crystal structure of the receptor-binding domain (RBD) of the human coronavirus, HCoV-229E, in complex with the ectodomain of its receptor, aminopeptidase N (APN). Three extended loops are solely responsible for receptor binding and the evolution of HCoV-229E and its close relatives is accompanied by changing loop–receptor interactions. Phylogenetic analysis shows that the natural HCoV-229E receptor-binding loop variation observed defines six RBD classes whose viruses have successively replaced each other in the human population over the past 50 years. These RBD classes differ in their affinity for APN and their ability to bind an HCoV-229E neutralizing antibody. Together, our results provide a model for alphacoronavirus adaptation and evolution based on the use of extended loops for receptor binding.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Amending Koch’s postulates for viral disease: when “growth in pure culture” leads to a loss of virulence</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">It is a common laboratory practice to propagate viruses in cell culture. While convenient, these methodologies often result in unintentional genetic alterations, which have lead to adaptation and even attenuation in animal models of disease. An example is the attenuation of hantaviruses (family: Bunyaviridae, genus: Hantavirus) when cultured in vitro. In this case, viruses propagated in the natural reservoir species cause disease in nonhuman primates that closely mimics the human disease, but passaging in cell culture attenuates these viruses to the extent that do not cause any measurable disease in nonhuman primates. As efforts to develop animal models progress, it will be important to take into account the influences that culture in vitro may have on the virulence of viruses. In this review we discuss this phenomenon in the context of past and recent examples in the published literature.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>A Small-Molecule Oxocarbazate Inhibitor of Human Cathepsin L Blocks Severe Acute Respiratory Syndrome and Ebola Pseudotype Virus Infection into Human Embryonic Kidney 293T cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A tetrahydroquinoline oxocarbazate (PubChem CID 23631927) was tested as an inhibitor of human cathepsin L (EC 3.4.22.15) and as an entry blocker of severe acute respiratory syndrome (SARS) coronavirus and Ebola pseudotype virus. In the cathepsin L inhibition assay, the oxocarbazate caused a time-dependent 17-fold drop in IC(50) from 6.9 nM (no preincubation) to 0.4 nM (4-h preincubation). Slowly reversible inhibition was demonstrated in a dilution assay. A transient kinetic analysis using a single-step competitive inhibition model provided rate constants of k(on) = 153,000 M(−1)s(−1) and k(off) = 4.40 × 10(−5) s(−1) (K(i) = 0.29 nM). The compound also displayed cathepsin L/B selectivity of >700-fold and was nontoxic to human aortic endothelial cells at 100 μM. The oxocarbazate and a related thiocarbazate (PubChem CID 16725315) were tested in a SARS coronavirus (CoV) and Ebola virus-pseudotype infection assay with the oxocarbazate but not the thiocarbazate, demonstrating activity in blocking both SARS-CoV (IC(50) = 273 ± 49 nM) and Ebola virus (IC(50) = 193 ± 39 nM) entry into human embryonic kidney 293T cells. To trace the intracellular action of the inhibitors with intracellular cathepsin L, the activity-based probe biotin-Lys-C5 alkyl linker-Tyr-Leu-epoxide (DCG-04) was used to label the active site of cysteine proteases in 293T lysates. The reduction in active cathepsin L in inhibitor-treated cells correlated well with the observed potency of inhibitors observed in the virus pseudotype infection assay. Overall, the oxocarbazate CID 23631927 was a subnanomolar, slow-binding, reversible inhibitor of human cathepsin L that blocked SARS-CoV and Ebola pseudotype virus entry in human cells.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombinant Protein-Based Assays for Detection of Antibodies to Severe Acute Respiratory Syndrome Coronavirus Spike and Nucleocapsid Proteins</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Recombinant severe acute respiratory syndrome (SARS) nucleocapsid and spike protein-based immunoglobulin G immunoassays were developed and evaluated. Our assays demonstrated high sensitivity and specificity to the SARS coronavirus in sera collected from patients as late as 2 years postonset of symptoms. These assays will be useful not only for routine SARS coronavirus diagnostics but also for epidemiological and antibody kinetic studies.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Sero-epidemiology of MERS coronavirus in Saudi Arabia (1993) and Australia (2014) and characterization of assay specificity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Pseudoparticle virus neutralization (ppNT) and a conventional microneutralization (MN) assays are specific for detecting antibodies to MERS coronavirus (MERS-CoV) when used in sero-epidemiological studies in animals. Genetically diverse MERS-CoV appear antigenically similar in MN tests. We confirm that MERS-CoV has been circulating in dromedaries in Saudi Arabia in 1993. Preliminary data suggests that feral Australian dromedaries may be free of MERS-CoV but larger confirmatory studies are needed.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-04-28</span></i><p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live-attenuated coronavirus vaccines.\n\nSignificanceEmergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations like the aged and immune-compromised. This study illustrates that despite success in young models, the NSP16 mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the NSP16 mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for NSP16 attenuation as a major component of future live-attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus, Saudi Arabia, 2017–2018</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">We characterized exposures and demographics of Middle East respiratory syndrome coronavirus cases reported to the Saudi Arabia Ministry of Health during July 1–October 31, 2017, and June 1–September 16, 2018. Molecular characterization of available specimens showed that circulating viruses during these periods continued to cluster within lineage 5.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Angiotensin receptor blockers as tentative SARS-CoV-2 therapeutics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">At the time of writing this commentary (February 2020), the coronavirus COVID-19 epidemic has already resulted in more fatalities compared with the SARS and MERS coronavirus epidemics combined. Therapeutics that may assist to contain its rapid spread and reduce its high mortality rates are urgently needed. Developing vaccines against the SARS-CoV-2 virus may take many months. Moreover, vaccines based on viral-encoded peptides may not be effective against future coronavirus epidemics, as virus mutations could make them futile. Indeed, new Influenza virus strains emerge every year, requiring new immunizations. A tentative suggestion based on existing therapeutics, which would likely be resistant to new coronavirus mutations, is to use available angiotensin receptor 1 (AT1R) blockers, such as losartan, as therapeutics for reducing the aggressiveness and mortality from SARS-CoV-2 virus infections. This idea is based on observations that the angiotensin-converting enzyme 2 (ACE2) very likely serves as the binding site for SARS-CoV-2, the strain implicated in the current COVID-19 epidemic, similarly to strain SARS-CoV implicated in the 2002-2003 SARS epidemic. This commentary elaborates on the idea of considering AT1R blockers as tentative treatment for SARS-CoV-2 infections, and proposes a research direction based on datamining of clinical patient records for assessing its feasibility.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Spike Protein Fusion Peptide and Feline Coronavirus Virulence</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are well known for their potential to change their host or tissue tropism, resulting in unpredictable new diseases and changes in pathogenicity; severe acute respiratory syndrome and feline coronaviruses, respectively, are the most recognized examples. Feline coronaviruses occur as 2 pathotypes: nonvirulent feline enteric coronaviruses (FECVs), which replicate in intestinal epithelium cells, and lethal feline infectious peritonitis viruses (FIPVs), which replicate in macrophages. Evidence indicates that FIPV originates from FECV by mutation, but consistent distinguishing differences have not been established. We sequenced the full genome of 11 viruses of each pathotype and then focused on the single most distinctive site by additionally sequencing hundreds of viruses in that region. As a result, we identified 2 alternative amino acid differences in the putative fusion peptide of the spike protein that together distinguish FIPV from FECV in >95% of cases. By these and perhaps other mutations, the virus apparently acquires its macrophage tropism and spreads systemically.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Coronavirus disease-2019 (COVID-19) is a rapidly escalating epidemic caused by SARS-CoV-2. Identification of a simple and effective indicator to assess disease severity and outcome is urgently needed.  Methods: This study retrospectively analyzed dynamic changes of lymphocyte percentage (LYM%) in 15 death cases, 15 severe cases as well as 40 moderate cases of COVID-19 patients. Next, prognostic role of lymphopenia in COVID-19 were verified in 92 hospitalized cases. Results: Our results from death and severe cases showed that LYM% in blood test were inversely associated with the progression and severity of COVID-19. LYM% in patients with moderate COVID-19 remained higher than 20% 10-12 days after symptom onset. In contrast, LYM% was lower than 20% in severe cases. However, LYM% in severe cases was higher than 5% 17-19 days after the onset of the disease, while it fell below 5% in death cases. Therefore, we established a reliable Time from symptom onset-LYM% model (TLM), which could be used to evaluate disease severity and predict the outcomes of hospitalized patients with COVID-19. Conclusion: Lymphopenia can be used to indicate clinical course, treatment effect and outcomes of COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Macaque Model for Severe Acute Respiratory Syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Rhesus and cynomolgus macaques were challenged with 10(7) PFU of a clinical isolate of the severe acute respiratory syndrome (SARS) coronavirus. Some of the animals developed a mild self-limited respiratory infection very different from that observed in humans with SARS. The macaque model as it currently exists will have limited utility in the study of SARS and the evaluation of therapies.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS Molecular Detection External Quality Assurance</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">Inactivated severe acute respiratory syndrome–associated coronavirus samples were used for an external quality assurance study within the World Health Organization SARS Reference and Verification Network and other reference institutions. Of 58 participants, 51 correctly detected virus in all samples >9,400 RNA copies per milliliter and none in negative samples. Commercial test kits significantly improved the outcome.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Negative Nasopharyngeal and Oropharyngeal Swab Does Not Rule Out COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus Disease 19 (COVID-19), has become the Public Health Emergency of International Concern.....</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Murine Coronavirus with an Extended Host Range Uses Heparan Sulfate as an Entry Receptor</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Only a relatively few mutations in its spike protein allow the murine coronavirus to switch from a murine-restricted tropism to an extended host range by being passaged in vitro. One such virus that we studied had acquired two putative heparan sulfate-binding sites while preserving another site in the furin-cleavage motif. The adaptation of the virus through the use of heparan sulfate as an attachment/entry receptor was demonstrated by increased heparin binding as well as by inhibition of infection through treatment of cells and the virus with heparinase and heparin, respectively.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Broad Spectrum Antiviral Agent Niclosamide and Its Therapeutic Potential</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The recent outbreak of coronavirus disease 2019 (COVID-19) highlights an urgent need for therapeutics. Through a series of drug repurposing screening campaigns, niclosamide, an FDA-approved anthelminthic drug, was found to be effective against various viral infections with nanomolar to micromolar potency such as SARS-CoV, MERS-CoV, ZIKV, HCV and human adenovirus, indicating its potential as an antiviral agent. In this brief article, we summarize the broad antiviral activities of niclosamide and highlight its potential clinical use for treatment of COVID-19.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS Coronavirus Detection Methods</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">Using clinical samples from patients with severe acute respiratory syndrome, we showed that the sensitivities of a quantitative reverse transcription–polymerase chain reaction (80% for fecal samples and 25% for urine samples) were higher than those of the polyclonal (50% and 5%) and monoclonal (35% and 8%) antibody-based nucleocapsid antigen capture enzyme-linked immunosorbent assays.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a quantitative RT-PCR assay for the detection of the emerging coronavirus SARS-CoV-2 using a high throughput system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ability to quickly confirm or clear suspected cases is crucial during global outbreak scenarios, especially when clinical manifestations are difficult to distinguish from other respiratory infections such as influenza, molecular diagnostics is key for detection of the emerging virus. A variety of suitable assays were made available early on during the course of the outbreak, notably by Corman et al. and others [4,5]. However, their implementation in the diagnostics laboratory usually relies on manual PCR setups requiring a high degree of human interaction for execution and interpretation, thus limiting their capacity to be scaled up for handling large numbers of samples. In this study we report the analytical evaluation of a laboratory-developed test for the detection of SARS-CoV-2 using the open channel (utility channel) of the cobas 6800 system.%U https://eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.9.2000152</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /></body>