
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: developing a point-of-care test</h2><br /></div><br /><br /><br /><div class="card"><h3>Rapid Detection of Mycobacterium tuberculosis by Recombinase Polymerase Amplification</h3>
<p style="text-align: justify; text-justify: inter-word;">Improved access to effective tests for diagnosing tuberculosis (TB) has been designated a public health priority by the World Health Organisation. In high burden TB countries nucleic acid based TB tests have been restricted to centralised laboratories and specialised research settings. Requirements such as a constant electrical supply, air conditioning and skilled, computer literate operators prevent implementation of such tests in many settings. Isothermal DNA amplification technologies permit the use of simpler, less energy intensive detection platforms more suited to low resource settings that allow the accurate diagnosis of a disease within a short timeframe. Recombinase Polymerase Amplification (RPA) is a rapid, low temperature isothermal DNA amplification reaction. We report here RPA-based detection of Mycobacterium tuberculosis complex (MTC) DNA in ,20 minutes at 39uC. Assays for two MTC specific targets were investigated, IS6110 and IS1081. When testing purified MTC genomic DNA, limits of detection of 6.25 fg (IS6110) and 20 fg (IS1081)were consistently achieved. When testing a convenience sample of pulmonary specimens from suspected TB patients, RPA demonstrated superior accuracy to indirect fluorescence microscopy. Compared to culture, sensitivities for the IS1081 RPA and microscopy were 91.4% (95%CI: 85, 97.9) and 86.1% (95%CI: 78.1, 94.1) respectively (n = 71). Specificities were 100% and 88.6% (95% CI: 80.8, 96.1) respectively. For the IS6110 RPA and microscopy sensitivities of 87.5% (95%CI: 81.7, 93.2) and 70.8% (95%CI: 62.9, 78.7) were obtained (n = 90). Specificities were 95.4 (95% CI: 92.3,98.1) and 88% (95% CI: 83.6, 92.4) respectively. The superior specificity of RPA for detecting tuberculosis was due to the reduced ability of fluorescence microscopy to distinguish Mtb complex from other acid fast bacteria. The rapid nature of the RPA assay and its low energy requirement compared to other amplification technologies suggest RPA-based TB assays could be of use for integration into a point-of-care test for use in resource constrained settings.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Visual detection of porcine epidemic diarrhea virus using a novel reverse transcription polymerase spiral reaction method</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Porcine epidemic diarrhea virus (PEDV) is a major etiological agent of porcine epidemic diarrhea around the world. Point-of-care testing in the field is lacking owing to the requirement for a simple, robust field applicable test that does not require professional laboratory equipment. The aim of this study was to establish a novel reverse transcription polymerase spiral reaction (RT-PSR) assay for the rapid detection of porcine epidemic diarrhea virus (PEDV). For the assay, a specific RT-PSR primer pair was designed against a conserved region in PEDV ORF3.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Discriminating Active from Latent Tuberculosis in Patients Presenting to Community Clinics</h3>
<p style="text-align: justify; text-justify: inter-word;">We show that active TB can be distinguished accurately from LTBI in symptomatic clinic attenders using a plasma proteomic fingerprint. Translation of biomarkers derived from this study into a robust and affordable point-of-care format will have significant implications for recognition and control of active TB in high prevalence settings.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>A Field-Deployable Reverse Transcription Recombinase Polymerase Amplification Assay for Rapid Detection of the Chikungunya Virus</h3>
<p style="text-align: justify; text-justify: inter-word;">Chikungunya virus (CHIKV) is a mosquito-borne virus currently transmitted in about 60 countries. CHIKV causes acute flu-like symptoms and in many cases prolonged musculoskeletal and joint pain. Detection of the infection is mostly done using RT-RCR or ELISA, which are not suitable for point-of-care diagnosis.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Special Report Challenges in identifying antibiotic resistance targets for point-of-care diagnostics in general practice</h3>
<p style="text-align: justify; text-justify: inter-word;">General practitioners stand at the front line of healthcare provision and have a pivotal role in the fight against increasing antibiotic resistance. In this respect, targeted antibiotic prescribing by general practitioners would help reduce the unnecessary use of antibiotics, leading to reduced treatment failures, fewer side-effects for patients and a reduction in the (global) spread of antibiotic resistances. Current 'gold standard' antibiotic resistance detection strategies tend to be slow, taking up to 48 h to obtain a result, although the implementation of point-of-care testing by general practitioners could help achieve the goal of targeted antibiotic prescribing practices. However, deciding on which antibiotic resistances to include in a point-of-care diagnostic is not a trivial task, as outlined in this publication.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Discriminating Active from Latent Tuberculosis in Patients Presenting to Community Clinics</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Because of the high global prevalence of latent TB infection (LTBI), a key challenge in endemic settings is distinguishing patients with active TB from patients with overlapping clinical symptoms without active TB but with coexisting LTBI. Current methods are insufficiently accurate. Plasma proteomic fingerprinting can resolve this difficulty by providing a molecular snapshot defining disease state that can be used to develop point-of-care diagnostics.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Performance of the Alere i RSV assay for point-of-care detection of respiratory syncytial virus in children</h3>
<p style="text-align: justify; text-justify: inter-word;">The Alere i RSV assay has the potential to facilitate the detection of RSV in pediatric point-of-care settings.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of VHH Antibodies against Dengue Virus Type 2 NS1 and Comparison with Monoclonal Antibodies for Use in Immunological Diagnosis</h3>
<p style="text-align: justify; text-justify: inter-word;">The possibility of using variable domain heavy-chain antibodies (VHH antibodies) as diagnostic tools for dengue virus (DENV) type 2 NS1 protein was investigated and compared with the use of conventional monoclonal antibodies. After successful expression of DENV type 2 NS1 protein, the genes of VHH antibodies against NS1 protein were biopanned from a non-immune llama library by phage display. VHH antibodies were then expressed and purified from Escherichia coli. Simultaneously, monoclonal antibodies were obtained by the conventional route. Sequence analysis of the VHH antibodies revealed novel and long complementarity determining regions 3 (CDR3). Epitope mapping was performed via a phage display peptide library using purified VHH and monoclonal antibodies as targets. Interestingly, the same region of NS1, which comprises amino acids 224 HWPKPHTLW 232 , was conserved for both kinds of antibodies displaying the consensus motif histidine-tryptophan-tryptophan or tryptophan-proline-tryptophan. The two types of antibodies were used to prepare rapid diagnostic kits based on immunochromatographic assay. The VHH antibody immobilized rapid diagnostic kit showed better sensitivity and specificity than the monoclonal antibody immobilized rapid diagnostic kit, which might be due to the long CDR3 regions of the VHH antibodies and their ability to bind to the pocket and cleft of the targeted antigen. This demonstrates that VHH antibodies are likely to be an option for developing point-of-care tests against DENV infection.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiplex logic processing isothermal diagnostic assays for an evolving virus</h3>
<p style="text-align: justify; text-justify: inter-word;">While point-of-care (POC) diagnostic assays can be performed at or near the site of sample acquisition, they have for the most part been considered to be relatively simplistic tests that provide relatively little information to a clinician or public health worker. Familiar examples include electrochemical sensors for glucose, 1 or rapid immunoassays for metabolites such as human chorionic gonadotropin (the canonical pregnancy test), 2 and pathogens, either directly (influenza viruses) or via immune responses (antibodies against HIV-1/2). 3 The range of conditions and pathogens that could be tested for could likely be greatly expanded by developing POC diagnostics for nucleic acids 3 that would have greater sensitivity and accuracy. However, the current gold standard for molecular diagnostics, the quantitative polymerase chain reaction (qPCR), requires significant technical expertise and expensive and cumbersome equipment. Even portable instruments, such the Cepheid GeneXpert Omni, cost several thousand dollars and rely on expensive qPCR cartridge consumables for individual tests.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Alere i RSV for Rapid Detection of Respiratory Syncytial Virus in Children Hospitalized with Acute Respiratory Tract Infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Alere i RSV is a novel rapid test which applies a nicking enzyme amplification reaction to detect respiratory syncytial virus in point-of-care settings. In this study, we evaluated the Alere i RSV assay by using frozen nasopharyngeal swab samples that were collected in viral transport medium from children hospitalized with acute respiratory tract infection during the 2015-2016 winter season. Alere i RSV assay results were compared to those for Altona RealStar RSV real-time reverse transcription-PCR (RT-PCR). We found that the overall sensitivity and specificity of the Alere i RSV test was 100% (95% confidence intervals [CI], 93% to 100%) and 97% (95% CI, 89% to 100%), respectively. Positive samples were identified within 5 to 7 min from sample collection. Overall, the Alere i RSV test performed well compared to the RT-PCR assay and has the potential to facilitate the detection of RSV in pointof-care settings.</p>
<p align="right"><i>score: 102</i></p>

</div><br /><br /><br /><div class="card"><h3>A Field-Deployable Reverse Transcription Recombinase Polymerase Amplification Assay for Rapid Detection of the Chikungunya Virus</h3>
<p style="text-align: justify; text-justify: inter-word;">The developed RT-RPA assay represents a promising method for the molecular detection of CHIKV at point of need.</p>
<p align="right"><i>score: 102</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a rapid recombinase polymerase amplification assay for the detection of Streptococcus pneumoniae in whole blood</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Streptococcus pneumoniae is an important cause of microbial disease in humans. The introduction of multivalent vaccines has coincided with a dramatic decrease in the number of pneumococcal-related deaths. In spite of this, at a global level, pneumococcal infection remains an important cause of death among children under 5 years of age and in adults 65 years of age or older. In order to properly manage patients and control the spread of infection, a rapid and highly sensitive diagnostic method is needed for routine implementation, especially in resource-limited regions where pneumococcal disease is most prevalent. Methods: Using the gene encoding leader peptidase A as a molecular diagnostics target, a real-time RPA assay was designed and optimised for the detection of S. pneumoniae in whole blood. The performance of the assay was compared to real-time PCR in terms of its analytical limit of detection and specificity. The inhibitory effect of human genomic DNA on amplification was investigated. The potential clinical utility of the assay was investigated using a small number of clinical samples. Results: The RPA assay has a limit of detection equivalent to PCR (4.0 and 5.1 genome equivalents per reaction, respectively) and was capable of detecting the equivalent of <1 colony forming unit of S. pneumoniae when spiked into human whole blood. The RPA assay was 100 % inclusive (38/38 laboratory reference strains and 19/19 invasive clinical isolates) and 100 % exclusive; differentiating strains of S. pneumoniae species from other viridans group streptococci, including S. pseudopneumoniae. When applied to the analysis of a small number (n = 11) of clinical samples (blood culture positive for S. pneumoniae), the RPA assay was demonstrated to be both rapid and sensitive. Conclusions: The RPA assay developed in this work is shown to be as sensitive and as specific as PCR. In terms of reaction kinetics, the RPA assay is shown to exceed those of the PCR, with the RPA running to completion in 20 minutes and capable generating a positive signal in as little as 6 minutes. This work represents a potentially suitable assay for application in point-of-care settings.</p>
<p align="right"><i>score: 102</i></p>

</div><br /><br /><br /><div class="card"><h3>Selection and Characterization of Monoclonal Antibodies Targeting Middle East Respiratory Syndrome Coronavirus through a Human Synthetic Fab Phage Display Library Panning</h3>
<p style="text-align: justify; text-justify: inter-word;">In this study, we panned a synthetic human Fab phage display library against MERS-S2P and obtained monoclonal antibodies (mAbs) that specifically detected the viral S2 subunit. We also described the use of these mAbs for the development of a novel ELISA format called ACCEL ELISA™, which we anticipate can be used as a rapid diagnostic test with a high sensitivity for point-of-care testing (POCT) of MERS-CoV infection.</p>
<p align="right"><i>score: 102</i></p>

</div><br /><br /><br /><div class="card"><h3>Loop-Mediated Isothermal Amplification (LAMP) for the Diagnosis of Zika Virus: A Review</h3>
<p style="text-align: justify; text-justify: inter-word;">Given the lack of approved vaccines and antivirals against ZIKV, a rapid and reliable point-of-care (POC) diagnostic test for detection of ZIKV is urgently required for control and prevention measures and to increase the diagnostic capacity of ZIKV-affected, mainly in low-resource areas [27, 28] . ZIKV infection is diagnosed in the laboratory by nucleic acid amplification tests or serological methods, including enzyme-linked immunosorbent assays (ELISA), plaque reduction neutralization tests (PRNT), and lateral flow assays (rapid tests) [6, 7, [29] [30] [31] [32] . Currently, RT-qPCR is considered the gold standard method to detect ZIKV from patient and mosquito samples [6, 7, 29] . Although RT-qPCR provides high-quality results, the test requires extensive sample preparation, RNA extraction, expensive equipment, and technical expertise to run and interpret the amplification of the viral RNA. Moreover, available serological methods are prone to produce false-positive results due to cross-reaction with other flaviviruses in circulation, such as DENV, and are therefore of limited value [6, 7, 33] .</p>
<p align="right"><i>score: 102</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Acute HIV-1 Infection by RT- LAMP</h3>
<p style="text-align: justify; text-justify: inter-word;">A rapid, cost-effective diagnostic test for the detection of acute HIV-1 infection is highly desired. Isothermal amplification techniques, such as reverse-transcription loop-mediated isothermal amplification (RT-LAMP), exhibit characteristics that are ideal for the development of a rapid nucleic acid amplification test (NAAT) because they are quick, easy to perform and do not require complex, dedicated equipment and laboratory space. In this study, we assessed the ability of the HIV-1 RT-LAMP assay to detect acute HIV infection as compared to a representative rapid antibody test and several FDA-approved laboratory-based assays. The HIV-1 RT-LAMP assay detected seroconverting individuals one to three weeks earlier than a rapid HIV antibody test and up to two weeks earlier than a lab-based antigen/antibody (Ag/Ab) combo enzyme immunoassay (EIA). RT-LAMP was not as sensitive as a labbased qualitative RNA assay, which could be attributed to the significantly smaller nucleic acid input volume. To our knowledge, this is the first demonstration of detecting acute HIV infection using the RT-LAMP assay. The availability of a rapid NAAT, such as the HIV-1 RT-LAMP assay, at the point of care (POC) or in laboratories that do not have access to large platform NAAT could increase the percentage of individuals who receive an acute HIV infection status or confirmation of their HIV status, while immediately linking them to counseling and medical care. In addition, early knowledge of HIV status could lead to reduced high-risk behavior at a time when individuals are at a higher risk for transmitting the virus.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Applying a Linear Amplification Strategy to Recombinase Polymerase Amplification for Uniform DNA Library Amplification</h3>
<p style="text-align: justify; text-justify: inter-word;">Recombinase polymerase amplification (RPA) is an isothermal DNA amplification method with broad applications as a point-of-care test and in molecular biology techniques. Currently, most of the applications are focused on target-specific amplification. Because RPA has the advantage of amplifying DNA under isothermal conditions, we utilized RPA as a DNA library amplification tool. In this study, we used a sheared genomic DNA library and an oligonucleotide (oligo) library for the comparison of polymerase chain reaction and RPA. For the sheared DNA library, we observed biased amplification after RPA was conducted. Thus, to amplify the sizevariable DNA library uniformly, we introduced a linear amplification strategy with RPA and successfully improved the uniformity. On the other hand, using the same-sized oligo library, we confirmed that RPA amplified this library uniformly without modification of the protocol. These results demonstrate that RPA can be applied not only to amplify a specific target as previously demonstrated but also to amplify a complex DNA library composed of a large number of different DNA molecules.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid communication Validation of an Aptima-format Finnish new variant of Chlamydia trachomatis (FI-nvCT) surveillance assay, 2019</h3>
<p style="text-align: justify; text-justify: inter-word;">To understand the geographic distribution, prevalence and transmission dynamics of the FI-nvCT strain nationally and internationally, we have developed an Aptima-format NAAT that sensitively and specifically detects FI-nvCT (23S rRNA gene C1515T mutation) but not wild-type (C1515) C. trachomatis strains. This research-use only test uses target capture and transcription-mediated amplification chemistries on the automated Panther instrument (Hologic Inc.) to isolate and amplify C. trachomatis 23S rRNA, and an acridinium ester probe to selectively detect the C→T mutation at position 1515.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid and sensitive detection of canine distemper virus by one-tube reverse transcription-insulated isothermal polymerase chain reaction</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Canine distemper virus (CDV) has been associated with outbreaks of canine infectious respiratory disease in shelters and boarding kennel environments. POCKIT TM Nucleic Acid Analyzer is a field-deployable device capable of generating automatically interpreted insulated isothermal polymerase chain reaction (iiPCR) results from extracted nucleic acid within one hour. In this study, reverse transcription iiPCR (RT-iiPCR) was developed to facilitate point-of-need diagnosis of CDV infection. Results: Analytical sensitivity (limit of detection 95%) of the established CDV RT-iiPCR was about 11 copies of in vitro transcribed RNA per reaction. CDV RT-iiPCR generated positive signals from CDV, but not Bordetella bronchiseptica, canine parvovirus, canine herpesvirus, canine adenovirus 2, canine influenza virus (subtype H3N8), canine parainfluenza virus, and canine respiratory coronavirus. To evaluate accuracy of the established reaction in canine distemper clinical diagnosis, 110 specimens from dogs, raccoons, and foxes suspected with CDV infection were tested simultaneously by CDV RT-iiPCR and real-time RT-PCR. CDV RT-iiPCR demonstrated excellent sensitivity (100%) and specificity (100%), compared to real-time RT-PCR.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Sequence specific visual detection of LAMP reactions by addition of cationic polymers</h3>
<p style="text-align: justify; text-justify: inter-word;">The presence or absence of minute amount of nucleic acid templates could be detected in a simple manner through visual assessment for the color of the LAMP amplicon-PEI complex precipitate. We conclude that this detection method may facilitate development of small and simple g-POCT device.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Isothermal Amplification Using a Chemical Heating Device for Point-of-Care Detection of HIV-1</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: To date, the use of traditional nucleic acid amplification tests (NAAT) for detection of HIV-1 DNA or RNA has been restricted to laboratory settings due to time, equipment, and technical expertise requirements. The availability of a rapid NAAT with applicability for resource-limited or point-of-care (POC) settings would fill a great need in HIV diagnostics, allowing for timely diagnosis or confirmation of infection status, as well as facilitating the diagnosis of acute infection, screening and evaluation of infants born to HIV-infected mothers. Isothermal amplification methods, such as reversetranscription, loop-mediated isothermal amplification (RT-LAMP), exhibit characteristics that are ideal for POC settings, since they are typically quicker, easier to perform, and allow for integration into low-tech, portable heating devices.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /></body>