
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: method evaluating potential complication of Antibody-Dependent Enhancement in SARS-CoV-2 vaccine recipients</h2><br /></div><br /><br /><br /><div class="card"><h3>Diagnosis of Acute Respiratory Syndrome Coronavirus 2 Infection by Detection of Nucleocapsid Protein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND Nucleic acid test and antibody assay have been employed in the diagnosis for SARS-CoV-2 infection, but the use of viral antigen for diagnosis has not been successfully developed.  Theoretically, viral antigen is the specific marker of the virus and precedes antibody appearance within the infected population. There is a clear need of detection of viral antigen for rapid and early diagnosis. METHODS We included a cohort of 239 participants with suspected SARS-CoV-2 infection from 7 centers for the study. We measured nucleocapsid protein in nasopharyngeal swab samples in parallel with the nucleic acid test. Nucleic acid test was taken as the reference standard, and statistical evaluation was taken in blind. We detected nucleocapsid protein in 20 urine samples in another center, employing nasopharyngeal swab nucleic acid test as reference standard. RESULTS We developed a fluorescence immunochromatographic assay for detecting nucleocapsid protein of SARS-CoV-2 in nasopharyngeal swab sample and urine within 10 minutes.  100% of nucleocapsid protein positive and negative participants accord with nucleic acid test for same samples. Further, earliest participant after 3 days of fever can be identified by the method.  In an additional preliminary study, we detected nucleocapsid protein in urine in 73.6% of diagnosed COVID-19 patients.  CONCLUSIONS Those findings indicate that nucleocapsid protein assay is an accurate, rapid, early and simple method for diagnosis of COVID-19. Appearance of nucleocapsid protein in urine coincides our finding of the SARS-CoV-2 invading kidney and might be of diagnostic value.</jats:p></p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Is COVID-19 Receiving ADE From Other Coronaviruses?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">One of the most perplexing questions regarding the current COVID-19 coronavirus epidemic is the discrepancy between the severity of cases observed in the Hubei province of China and those occurring elsewhere in the world. One possible answer is antibody dependent enhancement (ADE) of SARS-CoV-2 due to prior exposure to other coronaviruses. ADE modulates the immune response and can elicit sustained inflammation, lymphopenia, and/or cytokine storm, one or all of which have been documented in severe cases and deaths. ADE also requires prior exposure to similar antigenic epitopes, presumably circulating in local viruses, making it a possible explanation for the observed geographic limitation of severe cases and deaths.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Anti-Severe Acute Respiratory Syndrome Coronavirus Spike Antibodies Trigger Infection of Human Immune Cells via a pH- and Cysteine Protease-Independent FcγR Pathway</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Public health measures successfully contained outbreaks of the severe acute respiratory syndrome coronavirus (SARS-CoV) infection. However, the precursor of the SARS-CoV remains in its natural bat reservoir, and reemergence of a human-adapted SARS-like coronavirus remains a plausible public health concern. Vaccination is a major strategy for containing resurgence of SARS in humans, and a number of vaccine candidates have been tested in experimental animal models. We previously reported that antibody elicited by a SARS-CoV vaccine candidate based on recombinant full-length Spike-protein trimers potentiated infection of human B cell lines despite eliciting in vivo a neutralizing and protective immune response in rodents. These observations prompted us to investigate the mechanisms underlying antibody-dependent enhancement (ADE) of SARS-CoV infection in vitro. We demonstrate here that anti-Spike immune serum, while inhibiting viral entry in a permissive cell line, potentiated infection of immune cells by SARS-CoV Spike-pseudotyped lentiviral particles, as well as replication-competent SARS coronavirus. Antibody-mediated infection was dependent on Fcγ receptor II but did not use the endosomal/lysosomal pathway utilized by angiotensin I converting enzyme 2 (ACE2), the accepted receptor for SARS-CoV. This suggests that ADE of SARS-CoV utilizes a novel cell entry mechanism into immune cells. Different SARS vaccine candidates elicit sera that differ in their capacity to induce ADE in immune cells despite their comparable potency to neutralize infection in ACE2-bearing cells. Our results suggest a novel mechanism by which SARS-CoV can enter target cells and illustrate the potential pitfalls associated with immunization against it. These findings should prompt further investigations into SARS pathogenesis.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Lack of Interference with Immunogenicity of a Chimeric Alphavirus Replicon Particle-Based Influenza Vaccine by Preexisting Antivector Immunity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Antivector immunity has been recognized as a potential caveat of using virus-based vaccines. In the present study, an alphavirus-based replicon particle vaccine platform, which has demonstrated robust immunogenicity in animal models, was tested for effects of antivector immunity on immunogenicity against hemagglutinin of influenza virus as a target antigen and efficacy for protection against lethal challenge with the virus. Chimeric alphavirus-based replicon particles, comprising Venezuelan equine encephalitis virus nonstructural and Sindbis virus structural components, induced efficient protective antibody responses, which were not adversely influenced after multiple immunizations with the same vector expressing various antigens.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Mechanism for Antibody-Dependent Enhancement of Coronavirus Entry</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Antibody-dependent enhancement (ADE) of viral entry has been a major concern for epidemiology, vaccine development, and antibody-based drug therapy. However, the molecular mechanism behind ADE is still elusive. Coronavirus spike protein mediates viral entry into cells by first binding to a receptor on the host cell surface and then fusing viral and host membranes. In this study, we investigated how a neutralizing monoclonal antibody (MAb), which targets the receptor-binding domain (RBD) of Middle East respiratory syndrome (MERS) coronavirus spike, mediates viral entry using pseudovirus entry and biochemical assays. Our results showed that MAb binds to the virus surface spike, allowing it to undergo conformational changes and become prone to proteolytic activation. Meanwhile, MAb binds to cell surface IgG Fc receptor, guiding viral entry through canonical viral-receptor-dependent pathways. Our data suggest that the antibody/Fc-receptor complex functionally mimics viral receptor in mediating viral entry. Moreover, we characterized MAb dosages in viral-receptor-dependent, Fc-receptor-dependent, and both-receptors-dependent viral entry pathways, delineating guidelines on MAb usages in treating viral infections. Our study reveals a novel molecular mechanism for antibody-enhanced viral entry and can guide future vaccination and antiviral strategies.IMPORTANCE Antibody-dependent enhancement (ADE) of viral entry has been observed for many viruses. It was shown that antibodies target one serotype of viruses but only subneutralize another, leading to ADE of the latter viruses. Here we identify a novel mechanism for ADE: a neutralizing antibody binds to the surface spike protein of coronaviruses like a viral receptor, triggers a conformational change of the spike, and mediates viral entry into IgG Fc receptor-expressing cells through canonical viral-receptor-dependent pathways. We further evaluated how antibody dosages impacted viral entry into cells expressing viral receptor, Fc receptor, or both receptors. This study reveals complex roles of antibodies in viral entry and can guide future vaccine design and antibody-based drug therapy.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the scale of COVID-19 Epidemic in the United States: Simulations Based on Air Traffic directly from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Introduction: Coronavirus Disease 2019 (COVID-19) infection has been characterized by rapid spread and unusually large case clusters. It is important to have an estimate of the current state of COVID-19 epidemic in the U.S. to develop informed public health strategies.  Methods:  We estimated the potential scale of the COVID-19 epidemic (as of 03/01/2020) in the U.S. from cases imported directly from Wuhan area. We used simulations based on transmission dynamics parameters estimated from previous studies and air traffic data from Wuhan to the U.S and deliberately built our model based on conservative assumptions. Detection and quarantine of individual COVID-19 cases in the U.S before 03/01/2020 were also taken into account. We. A SEIR model was used to simulate the growth of the number of infected individuals in Wuhan area and in the U.S.  Results: With the most likely model, we estimated that there would be 9,484 infected cases (90%CI 2,054-24,241) as of 03/01/2020 if no intervention procedure had been taken to reduce the transmissibility in unidentified cases. Assuming current preventive procedures have reduced 25% of the transmissibility in unidentified cases, the number of infected cases would be 1,043 (90%CI 107-2,474). Conclusion: Our research indicates that, as of 03/01/2020., it is likely that there are already thousands of individuals in the US infected with SARS-CoV-2. Our model is dynamic and is available to the research community to further evaluate as the  situation becomes clearer.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a quantitative RT-PCR assay for the detection of the emerging coronavirus SARS-CoV-2 using a high throughput system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ability to quickly confirm or clear suspected cases is crucial during global outbreak scenarios, especially when clinical manifestations are difficult to distinguish from other respiratory infections such as influenza, molecular diagnostics is key for detection of the emerging virus. A variety of suitable assays were made available early on during the course of the outbreak, notably by Corman et al. and others [4,5]. However, their implementation in the diagnostics laboratory usually relies on manual PCR setups requiring a high degree of human interaction for execution and interpretation, thus limiting their capacity to be scaled up for handling large numbers of samples. In this study we report the analytical evaluation of a laboratory-developed test for the detection of SARS-CoV-2 using the open channel (utility channel) of the cobas 6800 system.%U https://eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.9.2000152</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Summary  Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients.  Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR.  Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF.  Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>We shouldn't worry when a virus mutates during disease outbreaks</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Mutation. The word naturally conjures fears of unexpected and freakish changes. Ill-informed discussions of mutations thrive during virus outbreaks, including the ongoing spread of SARS-CoV-2. In reality, mutations are a natural part of the virus life cycle and rarely impact outbreaks dramatically.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 Vaccine Pipeline: an Overview</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19).</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral-Induced Enhanced Disease Illness</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 5</span></i><p style="text-align: justify; text-justify: inter-word;">Understanding immune responses to viral infections is crucial to progress in the quest for effective infection prevention and control. The host immunity involves various mechanisms to combat viral infections. Under certain circumstances, a viral infection or vaccination may result in a subverted immune system, which may lead to an exacerbated illness. Clinical evidence of enhanced illness by preexisting antibodies from vaccination, infection or maternal passive immunity is available for several viruses and is presumptively proposed for other viruses. Multiple mechanisms have been proposed to explain this phenomenon. It has been confirmed that certain infection- and/or vaccine-induced immunity could exacerbate viral infectivity in Fc receptor- or complement bearing cells- mediated mechanisms. Considering that antibody dependent enhancement (ADE) is a major obstacle in vaccine development, there are continues efforts to understand the underlying mechanisms through identification of the epitopes and antibodies responsible for disease enhancement or protection. This review discusses the recent findings on virally induced ADE, and highlights the potential mechanisms leading to this condition.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>A new transmission route for the propagation of the SARS-CoV-2 coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.</jats:p></p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Has China faced only a herald wave of SARS-CoV-2?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The attack rate of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) calculated by mathematical models, from estimates of the basic reproduction number, R0, of 2–3, suggests that 50–60% of the population should eventually be infected because the population seems to be entirely naive to the new virus.1 The observed attack rate on board the Diamond Princess cruise ship remained slightly below 20% (705 of 3711 passengers and crew members became infected).1 It is of upmost importance to know whether the SARS-CoV-2 outbreak in China is subsiding, as local authorities and the entire international community might wish. With 80 026 COVID-19 cases officially reported from China as of March 2, 2020,2 the proportion of the population affected remains far from 50%, or even 20%, of China's 1·4 billion people. Has China just experienced a herald wave, to use terminology borrowed from those who study tsunamis, and is the big wave still to come? Serosurveys can help answer these questions precisely.3 To serosurvey the outbreak would involve testing sera of blood samples from the most representative sample of the population at the epicentre of the epidemic, Wuhan. Serology analysis with neutralising antibodies from the 1000 people could allow for the rate of SARS-CoV-2 infections to be estimated with good accuracy. This rate could be extrapolated to the city's entire population and thus inform more precisely whether the provisional attack rate during this period was a few cases per thousand or perhaps affected 1–2% of the population, 20%, or more. Serosurveys should be seen as polls before elections; they can be repeated several times,3 week after week, to monitor the epidemic precisely. There is no reason to wait for the end of the epidemic before doing serosurveys. The results would be tremendously informative to China, first and foremost, and to the entire international community, on the risk of big secondary epidemic waves</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus disease (COVID-19) and neonate: What neonatologist need to know</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About 3 neonates and more than 230 children cases are reported. The disease condition of mainly children was mild. There is currently no evidence that SARS-CoV-2can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID-19) is based on adult experience. Thus far, no deaths have been reported in the paediatric age group. This review describes the current understanding of COVID-19 infection in newborns and children. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Journey of a Thai Taxi Driver and Novel Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On January 20, 2020, a 51-year-old male taxi driver had fever, cough, and myalgia and went to a local pharmacy to get unspecified over-the-counter medications. At the time, he was not aware of the emergence of SARS-CoV-2 or the illness it causes (Covid-19). As the symptoms persisted, he decided to visit a private primary care clinic in Bangkok on January 23. The body temperature was 36.8°C (98°F). The clinic physician ordered a throat swab for influenza A and B; the swab was negative for both strains. Additional medications were prescribed for treatment of the patient’s symptoms.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using 'Feidian', Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple laboratory parameter facilitates early identification of COVID-19 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Machine learning-based CT radiomics model for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection: A multicenter study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Objectives To develop and test machine learning-based CT radiomics models for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection. Design Cross-sectional Setting Multicenter Participants A total of 52 patients with laboratory-confirmed SARS-CoV-2 infection and their initial CT images were enrolled from 5 designated hospitals in Ankang, Lishui, Zhenjiang, Lanzhou, and Linxia between January 23, 2020 and February 8, 2020. As of February 20, patients remained in hospital or with non-findings in CT were excluded. Therefore, 31 patients with 72 lesion segments were included in the final analysis.  Intervention CT radiomics models based on logistic regression (LR) and random forest (RF) were developed on features extracted from pneumonia lesions in training and inter-validation datasets. The predictive performance was further evaluated in test dataset on lung lobe- and patients-level. Main outcomes Short-term hospital stay (≤10 days) and long-term hospital stay (&gt;10 days). Results The CT radiomics models based on 6 second-order features were effective in discriminating short- and long-term hospital stay in patients with pneumonia associated with SARS-CoV-2 infection, with areas under the curves of 0.97 (95%CI 0.83-1.0) and 0.92 (95%CI 0.67-1.0) by LR and RF, respectively, in the test dataset. The LR model showed a sensitivity and specificity of 1.0 and 0.89, and the RF model showed similar performance with sensitivity and specificity of 0.75 and 1.0 in test dataset.  Conclusions  The machine learning-based CT radiomics models showed feasibility and accuracy for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Guidelines for pregnant women with suspected SARS-CoV-2 infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses responsible for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) can cause severe adverse pregnancy outcomes, such as miscarriage, premature delivery, intrauterine growth restriction, and maternal death.1 , 2 Vertical transmission of the virus responsible for 2019 novel coronavirus disease (COVID-19), severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has not yet been detected, whereas perinatal transmission has been suspected in one case.3 Consequences of infection with SARS-CoV-2 for pregnancies are uncertain, with no evidence so far of severe outcomes for mothers and infants; however, the possibility should be considered.4 The recent experience with Zika virus suggests that when a new pathogen emerges, the health-care community should be prepared for the worst-case scenario.5 Therefore, recommendations for management of pregnant women at risk of SARS-CoV-2 infection are urgently needed. To this end, we propose a detailed management algorithm for health-care providers (appendix).</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Vaccine Candidates: Cautious Optimism</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 17</span></i><p style="text-align: justify; text-justify: inter-word;">Efforts towards developing a vaccine for Middle East respiratory syndrome coronavirus (MERS-CoV) have yielded promising results. Utilizing a variety of platforms, several vaccine approaches have shown efficacy in animal models and begun to enter clinical trials. In this review, we summarize the current progress towards a MERS-CoV vaccine and highlight potential roadblocks identified from previous attempts to generate coronavirus vaccines.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Feasibility of controlling COVID-19 outbreaks by isolation of cases and contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Background Isolation of cases and contact tracing is used to control outbreaks of infectious diseases, and has been used for coronavirus disease 2019 (COVID-19). Whether this strategy will achieve control depends on characteristics of both the pathogen and the response. Here we use a mathematical model to assess if isolation and contact tracing are able to control onwards transmission from imported cases of COVID-19. Methods We developed a stochastic transmission model, parameterised to the COVID-19 outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-like pathogen. We considered scenarios that varied in the number of initial cases, the basic reproduction number (R0), the delay from symptom onset to isolation, the probability that contacts were traced, the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreaks were deemed controlled if transmission ended within 12 weeks or before 5000 cases in total. We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort. Findings Simulated outbreaks starting with five initial cases, an R0 of 1·5, and 0% transmission before symptom onset could be controlled even with low contact tracing probability; however, the probability of controlling an outbreak decreased with the number of initial cases, when R0 was 2·5 or 3·5 and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1·5 were controllable with less than 50% of contacts successfully traced. To control the majority of outbreaks, for R0 of 2·5 more than 70% of contacts had to be traced, and for an R0 of 3·5 more than 90% of contacts had to be traced. The delay between symptom onset and isolation had the largest role in determining whether an outbreak was controllable when R0 was 1·5. For R0 values of 2·5 or 3·5, if there were 40 initial cases, contact tracing and isolation were only potentially feasible when less than 1% of transmission occurred before symptom onset. Interpretation In most scenarios, highly effective contact tracing and case isolation is enough to control a new outbreak of COVID-19 within 3 months. The probability of control decreases with long delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts. Funding Wellcome Trust, Global Challenges Research Fund, and Health Data Research UK.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>A proposal of an alternative primer for the ARTIC Network's multiplex PCR to improve coverage of SARS-CoV-2 genome sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole genome analysis of the novel corona virus, SARS-CoV-2, soon after the epidemics of this pathogen was revealed. The primer set seems to have been adapted already by many researchers worldwide and contributed for the high-quality and prompt genome epidemiology of this potential pandemic virus. We have also seen the great performance of their primer set and protocol; the primer set was able to amplify all desired PCR products with fairy small amplification bias from clinical samples with relatively high viral load. However, we observed acute drop of reads derived from two particular PCR products, 18 and 76, out of the 98 designated products as sample's viral load decreases. We suspected the reason for this low coverage issue was due to dimer formation between primers used to amplify those two PCR products. Here, we propose replacing just one of those primers, nCoV-2019_76_RIGHT(-), to a newly designed primer. The result of the replacement of primer showed improvement in coverage at both regions targeted by the products, 18 and 76. We expect this simple modification will extend the limit for whole SARS-CoV-2 genome analysis to samples with lower viral load and enhance genomic epidemiology of this pathogen.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Neutralizing Antibody Response and SARS Severity</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Using the Taiwan nationwide laboratory-confirmed severe acute respiratory syndrome (SARS) database, we analyzed neutralizing antibody in relation to clinical outcomes. With a linear mixed model, neutralizing antibody titer was shown to peak between week 5 and week 8 after onset and to decline thereafter, with a half-life of 6.4 weeks. Patients with a longer illness showed a lower neutralizing antibody response than patients with a shorter illness duration (p = 0.008). When early responders were compared with most patients, who seroconverted on and after week 3 of illness, the small proportion (17.4%) of early responders (antibody detectable within 2 weeks) had a higher death rate (29.6% vs. 7.8%) (Fisher exact test, p = 0.004), had a shorter survival time of <2 weeks (Fisher exact test, p = 0.013), and were more likely to be > 60 years of age (Fisher exact test, p = 0.01). Our findings have implications for understanding the pathogenesis of SARS and for SARS vaccine research and development.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Liver injury in COVID-19: management and challenges</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 shares 82% genome sequence similarity to SARS-CoV and 50% genome sequence homology to Middle East respiratory syndrome coronavirus (MERS-CoV)—all three coronaviruses are known to cause severe respiratory symptoms. Liver impairment has been reported in up to 60% of patients with SARS3 and has also been reported in patients infected with MERS-CoV.4 At least seven relatively large-scale case studies have reported the clinical features of patients with COVID-19.1 , 5 , 6 , 7 , 8 , 9 , 10 In this Comment, we assess how the liver is affected using the available case studies and data from The Fifth Medical Center of PLS General Hospital, Beijing, China. These data indicate that 2–11% of patients with COVID-19 had liver comorbidities and 14–53% cases reported abnormal levels of alanine aminotransferase and aspartate aminotransferase (AST) during disease progression (table). Patients with severe COVID-19 seem to have higher rates of liver dysfunction. In a study in The Lancet by Huang and colleagues,5 elevation of AST was observed in eight (62%) of 13 patients in the intensive care unit (ICU) compared with seven (25%) of 28 patients who did not require care in the ICU. Moreover, in a large cohort including 1099 patients from 552 hospitals in 31 provinces or provincial municipalities, more severe patients with disease had abnormal liver aminotransferase levels than did non-severe patients with disease.1 Furthermore, in another study,8 patients who had a diagnosis of COVID-19 confirmed by CT scan while in the subclinical phase (ie, before symptom onset) had significantly lower incidence of AST abnormality than did patients diagnosed after the onset of symptoms. Therefore, liver injury is more prevalent in severe cases than in mild cases of COVID-19.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Managing neonates with respiratory failure due to SARS-CoV-2</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In their Comment in The Lancet Child & Adolescent Health, Jianhui Wang and colleagues1 suggested a plan to handle neonates with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections and outbreaks in neonatal intensive care units (NICUs). This is a timely reflection, given the public health problem represented by this infection and the need to anticipate any critical care issue, irrespective of patients' ages. However, the plan is incomplete or unsuitable in many points. We do not know anything about neonatal SARS-CoV-2 infections, and we must reasonably follow data from adult critical care. First, testing all NICU-admitted neonates for SARS-CoV-2 represents a wrongful use of resources. Neonatal respiratory failure can result from a wide range of causes, and testing everybody when other causes are reasonably suspected will divert laboratory resources from adult critical care. Tests should be done for infants from families infected by SARS-CoV-2 or exposed to other infected people, irrespective of their symptoms.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Monoclonal Antibodies Potently Neutralize Zika Virus and Select for Escape Mutations on the Lateral Ridge of the Envelope Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The mosquito-borne Zika virus (ZIKV) has been causing epidemic outbreaks on a global scale. Virus infection can result in severe disease in humans, including microcephaly in newborns and Guillain-Barré syndrome in adults. Here, we characterized monoclonal antibodies isolated from a patient with an active Zika virus infection that potently neutralized virus infection in Vero cells at the nanogram-per-milliliter range. In addition, these antibodies enhanced internalization of virions into human leukemia K562 cells in vitro, indicating their possible ability to cause antibody-dependent enhancement of disease. Escape variants of the ZIKV MR766 strain to a potently neutralizing antibody, AC10, exhibited an amino acid substitution at residue S368 in the lateral ridge region of the envelope protein. Analysis of publicly availably ZIKV sequences revealed the S368 site to be conserved among the vast majority (97.6%) of circulating strains. We validated the importance of this residue by engineering a recombinant virus with an S368R point mutation that was unable to be fully neutralized by AC10. Four out of the 12 monoclonal antibodies tested were also unable to neutralize the virus with the S368R mutation, suggesting this region to be an important immunogenic epitope during human infection. Last, a time-of-addition infection assay further validated the escape variant and showed that all monoclonal antibodies inhibited virus binding to the cell surface. Thus, the present study demonstrates that the lateral ridge region of the envelope protein is likely an immunodominant, neutralizing epitope. IMPORTANCE Zika virus (ZIKV) is a global health threat causing severe disease in humans, including microcephaly in newborns and Guillain-Barré syndrome in adults. Here, we analyzed the human monoclonal antibody response to acute ZIKV infection and found that neutralizing antibodies could not elicit Fc-mediated immune effector functions but could potentiate antibody-dependent enhancement of disease. We further identified critical epitopes involved with neutralization by generating and characterizing escape variants by whole-genome sequencing. We demonstrate that the lateral ridge region, particularly the S368 amino acid site, is critical for neutralization by domain III-specific antibodies.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.</jats:p></p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding SARS-CoV-2-Mediated Inflammatory Responses: From Mechanisms to Potential Therapeutic Tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Currently there is no effective antiviral therapy for SARS-CoV-2 infection, which frequently leads to fatal inflammatory responses and acute lung injury. Here, we discuss the various mechanisms of SARS-CoV-mediated inflammation. We also assume that SARS-CoV-2 likely shares similar inflammatory responses. Potential therapeutic tools to reduce SARS-CoV-2-induced inflammatory responses include various methods to block FcR activation. In the absence of a proven clinical FcR blocker, the use of intravenous immunoglobulin to block FcR activation may be a viable option for the urgent treatment of pulmonary inflammation to prevent severe lung injury. Such treatment may also be combined with systemic anti-inflammatory drugs or corticosteroids. However, these strategies, as proposed here, remain to be clinically tested for effectiveness.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Disease 2019: Coronaviruses and Blood Safety</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">With the outbreak of unknown pneumonia in Wuhan, China in December 2019, a new coronavirus, Severe Acute Respiratory Syndrome Corona Virus-2 (SARS-CoV-2) aroused the attention of the entire world. The current outbreak of infections with SARS-CoV-2 is termed Corona Virus Disease- 2019 (COVID-19). The World Health Organization (WHO) declared COVID-19 in China as a Public Health Emergency of International Concern (PHEIC). Two other corona virus infections-- SARS in 2002–2003 and Middle East Respiratory Syndrome (MERS) in 2012-- both caused severe respiratory syndrome in humans. All three of these emerging infectious diseases leading to a global spread are caused by beta-coronaviruses. Although coronaviruses usually infect the upper or lower respiratory tract, viral shedding in plasma or serum is common. Therefore, there is still a theoretical risk of transmission of coronaviruses through the transfusion of labile blood products. Because more and more asymptomatic infections are being found among COVID-19 cases, considerations of blood safety and coronaviruses have arisen especially in endemic areas. In this review, we detail current evidence and understanding of the transmission of SARS-CoV, MERS-CoV and SARS-CoV-2 through blood products as of February 10, 2020 and also discuss pathogen inactivation methods on coronaviruses.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>A Locally Transmitted Case of SARS-CoV-2 Infection in Taiwan</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On January 25, 2020, a 52-year-old woman with a history of type 2 diabetes presented with fever to an emergency department in central Taiwan. She was admitted to the hospital because of suspicion of pneumonia associated with SARS-CoV-2 infection. She had lived in Wuhan from October 21, 2019, to January 20, 2020. She returned to Taiwan from Wuhan on January 20 on an airplane. On the same day, a throat swab was obtained from another passenger on that flight; that passenger was confirmed to have the first known imported case of SARS-CoV-2 infection in Taiwan when the swab was found to be positive for the virus on January 21.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Study on SARS-COV-2 transmission and the effects of control measures in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and corona virus disease-2019 (COVID-19): the epidemic and the challenges</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">ABSTRACT Emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019, has caused a large global outbreak and a major public health issue. As of February 11, 2020, data from the WHO has shown that more than 43,000 confirmed cases have been identified in 28 countries/regions, with more than 99% of the cases being detected in China. On January 30, 2020, WHO has declared COVID-19 as the sixth public health emergency of international concern. The SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24-3.58. Among the patients with pneumonia caused by the SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground glass opacity was the most common finding from computerized tomography images of the chest. Although the one case of SARS-CoV-2 pneumonia in the United States responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of the SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Review of the Clinical Characteristics of Coronavirus Disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In late December 2019, a cluster of cases with 2019 Novel Coronavirus pneumonia (SARS-CoV-2) in Wuhan, China, aroused worldwide concern. Previous studies have reported epidemiological and clinical characteristics of coronavirus disease 2019 (COVID-19). The purpose of this brief review is to summarize those published studies as of late February 2020 on the clinical features, symptoms, complications, and treatments of COVID-19 and help provide guidance for frontline medical staff in the clinical management of this outbreak.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Hypokalemia and Clinical Implications in Patients with Coronavirus Disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND: SARS-CoV-2 has caused a series of COVID-19 globally. SARS-CoV-2 binds angiotensin I converting enzyme 2 (ACE2) of renin-angiotensin system (RAS) and causes prevalent hypokalemia METHODS: The patients with COVID-19 were classified into severe hypokalemia, hypokalemia, and normokalemia group. The study aimed to determine the relationship between hypokalemia and clinical features, the underlying causes and clinical implications of hypokalemia.  RESULTS: By Feb 15, 2020, 175 patients with COVID-19 (92 women and 83 men; median age, 46 [IQR, 34-54] years) were admitted to hospital in Wenzhou, China, consisting 39 severe hypokalemia-, 69 hypokalemia-, and 67 normokalemia patients.  Gastrointestinal symptoms were not associated with hypokalemia among 108 hypokalemia patients (P&gt;0.05).  Body temperature, CK, CK-MB, LDH, and CRP were significantly associated with the severity of hypokalemia (P&lt;0.01). 93% of severe and critically ill patients had hypokalemia which was most common among elevated CK, CK-MB, LDH, and CRP. Urine K+ loss was the primary cause of hypokalemia.  severe hypokalemia patients was given 3 g/day, adding up to an average of 34 (SD=4) g potassium during hospital stay. The exciting finding was that patients responded well to K+ supplements when they were inclined to recovery.  CONCLUSIONS: Hypokalemia is prevailing in patients with COVID-19. The correction of hypokalemia is challenging because of continuous renal K+ loss resulting from the degradation of ACE2. The end of urine K+ loss indicates a good prognosis and may be a reliable, in-time, and sensitive biomarker directly reflecting the end of adverse effect on RAS system.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country.  Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival &amp; traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% &amp; 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">hina continues to fight the outbreak of a novel coronavirus, called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which has infected tens of thousands and killed more than 2,100 people since the end of 2019. The outbreak’s hot spot is in the central province of Hubei, in particular, its capital city Wuhan, where there have been more than 62,000 infections and 2,029 deaths as of last week, according to the World Health Organization (WHO). There, doctors and scientists are desperately struggling to find reliable ways to diagnose infected patients to help treat them and control the spread of the virus. The struggle has involved two diagnostic options—computed tomography (CT) scans of patients’ lungs and a nucleic acid lab test—each with advantages and disadvantages. With “a suddenly rampant new virus, it is normal to have multiple test approaches,” says a virologist at the Chinese Center for Disease Control and Prevention</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 receptor ACE2 expression of maternal-fetal interface and fetalorgans by single cell transcriptome study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">Recent studies have demonstrated that SARS-CoV-2 cell entry depends on both ACE2 and TMPRSS2 genes (DOI:10.1016/j.cell.2020.02.052), but our current work only focus on ACE2, which is insufficient to support the conclusion of this paper. So the authors have withdrawn their manuscript whilst they perform additional experiments and analysis to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical characteristics of 140 patients infected by SARS-CoV-2 in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronavirus Disease 2019 (COVID-19) caused by Severe Acute Respiratory Syndrome Coronavirus -2 (SARS-CoV-2) infection has been widely spread. We aim to investigate the clinical characteristic and allergy status of patients infected by SARS-CoV-2. METHODS: Electronical medical records including demographics, clinical manifestation, comorbidities, laboratory data and radiological materials of 140 hospitalized COVID-19 patients, with confirmed result of SARS-CoV-2 viral infection were extracted and analysed. RESULTS: An approximately 1:1 ratio of male (50.7%) and female COVID-19 patients was found, with an overall median age of 57.0 years. All patients were community acquired cases. Fever (91.7%), cough (75.0%), fatigue (75.0%) and gastrointestinal symptoms (39.6%) were the most common clinical manifestations, whereas hypertension (30.0%) and diabetes mellitus (12.1%) were the most common comorbidities. Drug hypersensitivity (11.4%) and urticaria (1.4%) were self-reported by several patients. Asthma or other allergic diseases was not reported by any of the patients. Chronic obstructive pulmonary disease (COPD, 1.4%) and current smokers (1.4%) were rare. Bilateral ground glass or patchy opacity (89.6%) were the most common signs of radiological finding. Lymphopenia (75.4%) and eosinopenia (52.9%) were observed in most patients. Blood eosinophil counts correlate positively with lymphocyte counts in severe (r=0.486, p<0.001) and non-severe (r=0.469, p<0.001) patients after hospital admission. Significantly higher levels of D-dimer, C-reactive protein and procalcitonin were associated with severe patients compared to non-severe patients (all p<0.001). CONCLUSION: Detailed clinical investigation of 140 hospitalized COVID-19 cases suggest eosinopenia together with lymphopenia may be a potential indicator for diagnosis. Allergic diseases, asthma and COPD are not risk factors for SARS-CoV-2 infection. Elder age, high number of comorbidities and more prominent laboratory abnormalities were associated with severe patients.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Animal Models and Vaccines for SARS-CoV Infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We summarize findings of SARS-CoV infections in several animal models each of which support viral replication in lungs accompanied by histopathological changes and/or clinical signs of illness to varying degrees. New findings are reported on SARS-CoV replication and associated pathology in two additional strains (C57BL/6 and 129S6) of aged mice. We also provide new comparative data on viral replication and associated pathology following infection of golden Syrian hamsters with various SARS-CoV strains and report the levels of neutralizing antibody titers following these infections and the cross-protective efficacy of infection with these strains in protecting against heterologous challenge. Finally, we summarize findings of a variety of vaccine approaches and discuss the available in vitro and in vivo data addressing the potential for disease enhancement following re-infection in animals previously vaccinated against or infected with SARS-CoV.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and progression of acute respiratory distress syndrome in coronavirus disease 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) results in a cluster of coronavirus disease 2019 (COVID-19). We reported the clinical characteristics of COVID-19 patients with acute respiratory distress syndrome (ARDS), and further investigated the treatment and progression of ARDS in COVID-19. Methods: This study enrolled 109 patients with COVID-19 admitted to the Central Hospital of Wuhan, a designated hospital in Wuhan, China, from January 2 to February 1, 2020. Patients were followed up to February 12, 2020. The clinical data were collected from the electronic medical records. The differences in the treatment and progression with the time and the severity of ARDS were determined. Findings: Among 109 patients, mean age was 55 years, and 59 patients were male. With a median 15 days (range, 4 to 30 days) follow-up period, 31 patients (28.4%) died, while 78 (71.6%) survived and discharged. Of all patients, 53 (48.6%) developed ARDS. Compared to non-ARDS patients, ARDS patients were elder (mean age, 61 years vs. 49 years), and more likely to have the coexistent conditions, including diabetes (20.8% vs. 1.8%), cerebrovascular disease (11.3% vs. 0%), and chronic kidney disease (15.1% vs. 3.6%). Compared to mild ARDS patients, those with moderate and severe ARDS had higher mortality rates. No significant effect of antivirus, glucocorticoid, or immunoglobulin treatment on survival was observed in patients with ARDS. Interpretation: The mortality rate increased with the severity of ARDS in COVID-19, and the effects of current therapies on the survival for these patients were not satisfactory, which needs more attention from clinicians.  Funding: Health and Family Planning Commission of Wuhan Municipality.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP).  We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS.  We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene.  Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Trials and Novel Pathogens: Lessons Learned from SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">During the recent global outbreak of severe acute respiratory syndrome (SARS), thousands of patients received treatments of uncertain efficacy and known toxicity such as ribavirin and corticosteroids. Despite this, no controlled clinical trials assessing the efficacy of these agents were conducted. If a second global SARS outbreak occurred, clinicians would not have controlled data on which to base therapeutic decisions. We discuss the unique methodologic and logistical challenges faced by researchers who attempt to conduct controlled trials of therapeutic agents during an outbreak of a novel or unknown infectious pathogen. We draw upon our own experience in attempting to conduct a randomized controlled trial (trial) of ribavirin therapy for SARS and discuss the lessons learned. Strategies to facilitate future clinical trials during outbreaks of unknown or novel pathogens are also presented.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>2019 Novel Coronavirus can be detected in urine, blood, anal swabs and oropharyngeal swabs samples</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We tested samples collected from nine patients diagnosed with coronavirus disease 2019 (COVID-19). The virus was found in urine, blood, anal swabs and oropharyngeal swabs. It is the first time for SARS-CoV-2 found in urine, though no urinary irritation was found.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2.  Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Unique Safety Issues Associated with Virus Vectored Vaccines: Potential for and Theoretical Consequences of Recombination with Wild Type Virus Strains</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In 2003 and 2013, the World Health Organization convened informal consultations on characterization and quality aspects of vaccines based on live virus vectors. In the resulting reports, one of several issues raised for future study was the potential for recombination of virus-vectored vaccines with wild type pathogenic virus strains. This paper presents an assessment of this issue formulated by the Brighton Collaboration. To provide an appropriate context for understanding the potential for recombination of virus-vectored vaccines, we review briefly the current status of virus vectored vaccines, mechanisms of recombination between viruses, experience with recombination involving live attenuated vaccines in the field, and concerns raised previously in the literature regarding recombination of virus-vectored vaccines with wild type virus strains. We then present a discussion of the major variables that could influence recombination between a virus-vectored vaccine and circulating wild type virus and the consequences of such recombination, including intrinsic recombination properties of the parent virus used as a vector; sequence relatedness of vector and wild virus; virus host range, pathogenesis and transmission; replication competency of vector in target host; mechanism of vector attenuation; additional factors potentially affecting virulence; and circulation of multiple recombinant vectors in the same target population. Finally, we present some guiding principles for vector design and testing intended to anticipate and mitigate the potential for and consequences of recombination of virus-vectored vaccines with wild type pathogenic virus strains.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 is an appropriate name for the new coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have read with great interest the Correspondence by Shibo Jiang and colleagues,1 in which they propose a name change for the newly emerged coronavirus,2 which was recently designated severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the Coronavirus Study Group of the International Committee on Taxonomy of Viruses.3 The authors argued that the use of SARS in the virus name could confuse the public about the disease that it causes; in addition, they noted that the name SARS-CoV-2 is not consistent with the disease name chosen by WHO, coronavirus disease 2019. The authors also indicated that scientifically, SARS-CoV-2 is naturally occurring and different from other SARS-like or SARS-related coronaviruses that are mainly characterised by their genome sequences. Furthermore, given the probability of future attenuation of this virus to a low-pathogenic form, the authors predict that the use of the name SARS-CoV-2 might have adverse effects, both socially and economically. On these grounds, the authors suggest that the name of the new virus is changed to human coronavirus 2019 (HCoV-19). Although these concerns and suggestions are appreciated, we feel that the adoption of SARS-CoV-2 by the Coronavirus Study Group was appropriate. To facilitate good practice and scientific exchange, the International Committee on Taxonomy of Viruses has established standardised formats for classifying viruses. Under these rules, a newly emerged virus is normally assigned to a species based on phylogeny and taxonomy.4 Through DivErsity pArtitioning by hieRarchical Clustering-based analyses,5 the newly emerged coronavirus was deemed not sufficiently novel but is a sister virus to SARS-CoV, the primary viral isolate defining the species. The SARS-CoV species includes viruses such as SARS-CoV, SARS-CoV_PC4-227, and SARSr-CoV-btKY72. SARS-CoV-2 is the newest member of this viral species. The use of SARS in naming SARS-CoV-2 does not derive from the name of the SARS disease but is a natural extension of the taxonomic practice for viruses in the SARS species. The use of SARS for viruses in this species mainly refers to their taxonomic relationship to the founding virus of this species, SARS-CoV. In other words, viruses in this species can be named SARS regardless of whether or not they cause SARS-like diseases.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical characteristics of 82 death cases with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center.  Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms.  Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of &gt;5 (94.5%), high systemic immune-inflammation index of &gt;500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (&gt;10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed.  Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenges and countermeasures for organ donation during the SARS-CoV-2 epidemic: the experience of Sichuan Provincial People’s Hospital</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Up to 09:40 February 17, 2020, there have been 70,640 confirmed SARS-CoV-2 (aka 2019-nCoV) cases in China. Sichuan Provincial People’s Hospital acts as an organ transplant center in west of China, approximately 200 solid organ transplants performed each year and can perform heart, lung, liver, kidney, small bowel, stem cell transplantation, so it is necessary to establish a hospital-specific protocol to deal with the SARS-CoV-2 infection for the donor and recipient.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>A descriptive study of the impact of diseases control and prevention on the epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai, lessons learned for metropolis epidemics prevention</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To describe and evaluate the impact of diseases control and prevention on epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai.  Design: A retrospective descriptive study  Setting: China  Participants: Epidemiology information was collected from publicly accessible database. 265 patients admitted to Shanghai Public Health Center with confirmed COVID-19 were enrolled for clinical features analysis.  Main outcome measure: Prevention and control measures taken by Shanghai government,  epidemiological, demographic, clinical, laboratory and radiology data were collected. Weibull distribution, Chi-square test, Fisher's exact test, t test or Mann-Whitney U test were used in statistical analysis.   Results: COVID-19 transmission rate within Shanghai had reduced over 99% than previous speculated, and the exponential growth has been stopped so far. Epidemic was characterized by the first stage mainly composed of imported cases and the second stage where &gt;50% of cases were local. The incubation period was 6.4 (95% CI 5.3 to 7.6) days and the mean onset-admission interval was 5.5 days (95% CI, 5.1 to 5.9). Median time for COVID-19 progressed to severe diseases were 8.5 days (IQR: 4.8-11.0 days). By February 11th, proportion of patients being mild, moderate, severe and critically ill were 1.9%(5/265), 89.8%(238/265), 3.8%(10/265), 4.5%(12/265), respectively; 47 people in our cohort were discharged, and 1 patient died.  Conclusion: Strict controlling of the transmission rate at the early stage of an epidemic in metropolis can quickly prohibit the spread of the diseases. Controlling local clusters is the key to prevent outbreaks from imported cases. Most COVID-19 severe cases progressed within 14 days of disease onset. Multiple systemic laboratory abnormalities had been observed before significant respiratory dysfunction.   Keyword: COVID-19, SARS-CoV-2, epidemics dynamics, diseases control, clinical features</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential impact of seasonal forcing on a SARS-CoV-2 pandemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>[Potential false-positive rate among the 'asymptomatic infected individuals' in close contacts of COVID-19 patients]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: As the prevention and control of COVID-19continues to advance, the active nucleic acid test screening in the close contacts of the patients has been carrying out in many parts of China. However, the false-positive rate of positive results in the screening has not been reported up to now. But to clearify the false-positive rate during screening is important in COVID-19 control and prevention. Methods: Point values and reasonable ranges of the indicators which impact the false-positive rate of positive results were estimated based on the information available to us at present. The false-positive rate of positive results in the active screening was deduced, and univariate and multivariate-probabilistic sensitivity analyses were performed to understand the robustness of the findings. Results: When the infection rate of the close contacts and the sensitivity and specificity of reported results were taken as the point estimates, the positive predictive value of the active screening was only 19.67%, in contrast, the false-positive rate of positive results was 80.33%. The multivariate-probabilistic sensitivity analysis results supported the base-case findings, with a 75% probability for the false-positive rate of positive results over 47%. Conclusions: In the close contacts of COVID-19 patients, nearly half or even more of the 'asymptomatic infected individuals' reported in the active nucleic acid test screening might be false positives.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Clinical Application of A Rapid IgM-IgG Combined Antibody Test for SARS-CoV-2 Infection Diagnosis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of the novel coronavirus disease (COVID-19) quickly spread all over China and to more than 20 other countries. Although the virus (SARS-Cov-2) nucleic acid RT-PCR test has become the standard method for diagnosis of SARS-CoV-2 infection, these real-time PCR test kits have many limitations. In addition, high false negative rates were reported. There is an urgent need for an accurate and rapid test method to quickly identify large number of infected patients and asymptomatic carriers to prevent virus transmission and assure timely treatment of patients. We have developed a rapid and simple point-of-care lateral flow immunoassay which can detect IgM and IgG antibodies simultaneously against SARS-CoV-2 virus in human blood within 15 minutes which can detect patients at different infection stages. With this test kit, we carried out clinical studies to validate its clinical efficacy uses. The clinical detection sensitivity and specificity of this test were measured using blood samples collected from 397 PCR confirmed COVID-19 patients and 128 negative patients at 8 different clinical sites. The overall testing sensitivity was 88.66% and specificity was 90.63%. In addition, we evaluated clinical diagnosis results obtained from different types of venous and fingerstick blood samples. The results indicated great detection consistency among samples from fingerstick blood, serum and plasma of venous blood. The IgM-IgG combined assay has better utility and sensitivity compared with a single IgM or IgG test. It can be used for the rapid screening of SARS-CoV-2 carriers, symptomatic or asymptomatic, in hospitals, clinics, and test laboratories. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunization with SARS Coronavirus Vaccines Leads to Pulmonary Immunopathology on Challenge with the SARS Virus</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Apr 20</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Severe acute respiratory syndrome (SARS) emerged in China in 2002 and spread to other countries before brought under control. Because of a concern for reemergence or a deliberate release of the SARS coronavirus, vaccine development was initiated. Evaluations of an inactivated whole virus vaccine in ferrets and nonhuman primates and a virus-like-particle vaccine in mice induced protection against infection but challenged animals exhibited an immunopathologic-type lung disease. DESIGN: Four candidate vaccines for humans with or without alum adjuvant were evaluated in a mouse model of SARS, a VLP vaccine, the vaccine given to ferrets and NHP, another whole virus vaccine and an rDNA-produced S protein. Balb/c or C57BL/6 mice were vaccinated IM on day 0 and 28 and sacrificed for serum antibody measurements or challenged with live virus on day 56. On day 58, challenged mice were sacrificed and lungs obtained for virus and histopathology. RESULTS: All vaccines induced serum neutralizing antibody with increasing dosages and/or alum significantly increasing responses. Significant reductions of SARS-CoV two days after challenge was seen for all vaccines and prior live SARS-CoV. All mice exhibited histopathologic changes in lungs two days after challenge including all animals vaccinated (Balb/C and C57BL/6) or given live virus, influenza vaccine, or PBS suggesting infection occurred in all. Histopathology seen in animals given one of the SARS-CoV vaccines was uniformly a Th2-type immunopathology with prominent eosinophil infiltration, confirmed with special eosinophil stains. The pathologic changes seen in all control groups lacked the eosinophil prominence. CONCLUSIONS: These SARS-CoV vaccines all induced antibody and protection against infection with SARS-CoV. However, challenge of mice given any of the vaccines led to occurrence of Th2-type immunopathology suggesting hypersensitivity to SARS-CoV components was induced. Caution in proceeding to application of a SARS-CoV vaccine in humans is indicated.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 and the cardiovascular system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects host cells through ACE2 receptors, leading to coronavirus disease (COVID-19)-related pneumonia, while also causing acute myocardial injury and chronic damage to the cardiovascular system. Therefore, particular attention should be given to cardiovascular protection during treatment for COVID-19.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Psychological responses, behavioral changes and public perceptions during the early phase of the COVID-19 outbreak in China: a population based cross-sectional survey</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To investigate psychological and behavioral responses to the threat of SARS-CoV-2 infections and their associations with public perceptions in China  Design: Cross sectional population-based telephone survey via random digital dialing between 1 and 10 February, 2020 Setting: Wuhan (the epicentre and quarantined city), and Shanghai (a typical major city with close transportation link with Wuhan) Participants: Random sample of 510 residents in Wuhan and 501 residents in Shanghai aged above 18 Main outcome measures: Anxiety (measured by the 7-item generalized anxiety disorder [GAD-7] scale), recommended and avoidance behaviors (engaged in all six behaviors such as increasing surface cleaning and reducing going out). Results: The prevalence rates of moderate or severe anxiety (score ≥10 on GAD-7) were 32.7% (n=167) among Wuhan participants and 20.4% (n=102) among Shanghai participants. 78.6% (n=401) of Wuhan participants and 63.9% (n=320) of Shanghai participants had carried out all six precautionary behaviors. For both measures, Wuhan participants were more responsive to the outbreak (p&lt;0.001). Controlling for personal characteristics, logistic regression results suggested that risks of moderate or severe anxiety were positively associated with perceived susceptibility (odds ratio 1.6, 95% confidence interval 1.3-1.8) and severity of the disease (1.6, 1.4-1.9) and confusion about information reliability (1.6, 1.5-1.9). Having confidence in taking measures to protect oneself against the disease was associated with a lower risk (0.6, 0.5-0.7). The strongest predictor of behavioral change was perceived severity (1.2, 1.1-1.4), followed by confusion about information reliability (1.1, 1.0-1.3).   Conclusions: Psychological and behavioral responses to COVID-19 have been dramatic during the rising phase of the outbreak. Our results support efforts for timely dissemination of accurate and reliable information to address the high anxiety level.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The Time Required to Estimate the Case Fatality Ratio of Influenza Using Only the Tip of an Iceberg: Joint Estimation of the Virulence and the Transmission Potential</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 May 10</span></i><p style="text-align: justify; text-justify: inter-word;">Estimating the case fatality ratio (CFR) of a novel strain of influenza virus during the early stage of the pandemic is one of key epidemiological tasks to be conducted as rapid research response. Past experience during the epidemics of severe acute respiratory syndrome (SARS) and influenza A (H1N1-2009) posed several technical challenges in estimating the CFR in real time. The present study aimed to develop a simple method to estimate the CFR based on readily available datasets, that is, confirmed cases and deaths, while addressing some of the known technical issues. To assess the reliability and validity of the proposed method, we examined the minimum length of time required for the assigned CFR to be included within the 95% confidence intervals and for the estimated CFR to be below a prespecified cut-off value by means of Monte Carlo simulations. Overall, the smaller the transmission potential was, the longer it took to compare the estimated CFR against the cut-off value. If policymaking and public health response have to be made based on the CFR estimate derived from the proposed method and readily available data, it should be noted that the successful estimation may take longer than a few months.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Severe Acute Respiratory Syndrome Coronavirus in Blood of Infected Patients</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) has caused major outbreaks worldwide, resulting in an urgent need to obtain sensitive and accurate diagnosis of this disease. PCR-based detection methods were developed for use on a variety of samples, including blood. Eighteen subjects were investigated, and results indicated that blood samples contain sufficient virus for detection by using quantitative real-time PCR.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The outbreak of SARS-CoV-2 pneumonia calls for viral vaccines</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of 2019-novel coronavirus disease (COVID-19) that is caused by SARS-CoV-2 has spread rapidly in China, and has developed to be a Public Health Emergency of International Concern. However, no specific antiviral treatments or vaccines are available yet. This work aims to share strategies and candidate antigens to develop safe and effective vaccines against SARS-CoV-2. An outbreak of 2019-novel coronavirus (SARS-CoV-2) that causes atypical pneumonia (COVID-19) has raged in China since mid-December 2019 and has spread to 26 countries (February 20, 2020). The epidemic was identified by the first four cases confirmed on December 29, 2019 and was traced to the Huanan Seafood Wholesale Market, Wuhan city, Hubei Province, China1. A total of 75,465 cases with SARS-CoV-2 infections have been confirmed up to date (February 20, 2020), and 2,236 people have died in China2. COVID-19 spreads rapidly by human-to-human transmission with a median incubation period of 3.0 days (range, 0 to 24.0), and the time from symptom onset to developing pneumonia is 4.0 days (range, 2.0 to 7.0)3. Respiratory droplets and direct contact are conventional transmission routes for SARS-CoV-2, and fecal-to-oral transmission might also have a role3. Fever, dry cough, and fatigue are common symptoms at onset of COVID-194. Most patients have lymphopenia and bilateral ground-glass opacity changes on chest CT scans4,5. No specific antiviral treatments or vaccines are available because it is a new emerging viral disease. Development of SARS-CoV-2-based vaccines is urgently required. The entire virus particle-based preparation of vaccines, including inactivated and attenuated virus vaccines is advisable, because it is based on previous studies about the prevention and control of seasonal influenza vaccines6. The first SARS-CoV-2 (Wuhan-Hu-1) was successfully sequenced and its genomic sequence submitted to GenBank on January 5, 2020 (Accession no. MN908947.3)7. Subsequently large-scale culture of SARS-CoV-2 was quickly performed, and an inactivated virus vaccine could be prepared through the employment of established physical and chemical methods such as UV light, formaldehyde, and β-propiolactone8. The development of attenuated-virus vaccines is also possible by carefully screening the serially propagated SARS-CoV-2 with reduced pathogenesis such as induced minimal lung injury, diminished limited neutrophil influx, and increased anti-inflammatory cytokine expressions compared with the wild-type virus9. Both inactivated and attenuated virus vaccines have their own disadvantages and side effects (Table 1). Alternatively, new vaccine designs based on the putative protective antigen/peptides derived from SARS-CoV-2 should be considered</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the infection and case fatality ratio for COVID-19 using age-adjusted data from the outbreak on the Diamond Princess cruise ship</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Adjusting for delay from confirmation-to-death, we estimated case and infection fatality ratios (CFR, IFR) for COVID-19 on the Diamond Princess ship as 2.3% (0.75%-5.3%) and 1.2% (0.38-2.7%). Comparing deaths onboard with expected deaths based on naive CFR estimates using China data, we estimate IFR and CFR in China to be 0.5% (95% CI: 0.2-1.2%) and 1.1% (95% CI: 0.3-2.4%) respectively.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>The within-host viral kinetics of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>The infection evidence of SARS-COV-2 in ocular surface： a single-center cross-sectional study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Purpose: The aim of this study was to identify whether SARS-COV-2 infected in ocular surface.  Methods: Cross-sectional study of patients presenting for who received a COVID-19 diagnosis, from December 30, 2019 to February 7, 2020, at Tongji hospital, Tongji medical college, Huazhong University of Science and Technology. Demographics, temperature was recorded, blood routine test (Rt), chest Computed Tomography (CT) were took intermittently, and SARS-COV-2 real-time reverse-transcriptase-polymerase-chain-reaction (RT-PCR) assay were arranged for the nasopharyngeal and conjunctival swab samples.  Results: A total of 102 patients (48 Male [50%] and 54 Female [50%]) with clinical symptoms, Rt, and chest Computed Tomography (CT) abnormalities were identified with a clinical diagnosis of COVID-19. Patients had a mean [SD] gestational age of 57.63 [14.90] years. Of a total of 102 patients identified, 72 patients (36 men [50%] and 36 women [50%]; mean [SD] age, 58.68 [14.81] years) confirmed by laboratory diagnosis with SARS-COV-2 RT-PCR assay. Only two patients (2.78%) with conjunctivitis was identified from 72 patients with a laboratory confirmed COVID-19. However, SARS-COV-2 RNA fragments was found in ocular discharges by SARS-COV-2 RT-PCR only in one patient with conjunctivitis. Conclusions: Although we suspect the incidence of SARS-COV-2 infection through the ocular surface is extremely low, the nosocomial infection of SARS-CoV-2 through the eyes after occupational exposure is a potential route. The inefficient diagnostic method and the sampling time lag may contribute to the lower positive rate of conjunctival swab samples of SARS-COV-2. Therefore, to lower the SARS-COV-2 nosocomial infection, the protective goggles should be wore in all the health care workers.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background and Objective: To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. Design and Methods: We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. Results: The peak time of the curve for the infected newly diagnosed as COVID-19 should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei. It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese COVID-19 epidemic can be completely controlled in May,2020. Conclusions: COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global SARS-CoV-2 cases rise with the great efforts by Chinese government and its people.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Association of Cardiovascular Manifestations with In-hospital Outcomes in Patients with COVID-19: A Hospital Staff Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The outbreaks of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) remain a huge threat to the public health worldwide. Clinical data is limited up to now regarding the risk factors in favor of severe conversion of non-severe case with COVID-19. Aims: This study analyzed a hospital staff data to figure out general clinical features of COVID-19 in terms of the association of cardiovascular manifestations (CVMs) with in-hospital outcomes of COVID-19 cases.  Methods: Retrospective, single-center case series of 41 consecutive hospitalized health staff with confirmed COVID-19 were collected at the Central Hospital of Wuhan in Wuhan, China, from January 15 to January 24, 2020. Epidemiological, demographic, clinical, laboratory, radiological, treatment data and in-hospital adverse events were collected and analyzed. Final date of follow-up was March 3, 2020. A comparative study was applied between cases with CVMs and those without CVMs. Results: Of all, clinicians and clinical nurses accounted for 80.5%, while 87.8% of all had history of patient contact. The population was presented with a mean age of 39.1 +- 9.2 and less comorbidities than community population. The three most frequent symptoms of COVID-19 cases analyzed were fever (82.9%), myalgia or fatigue (80.5%) and cough (63.4%). While, the three most frequent initial symptoms were myalgia or fatigue (80.5%), fever (73.2%) and cough (41.5%). There were 95.1% cases featured as non-severe course of disease according to the official standard in China. Patients with CVMs and those without CVMs accounted for 58.5% and 41.5%, respectively. Compared with cases without CVMs, patients with CVMs were presented with lower baseline lymphocyte count (0.99 +- 0.43 and 1.55 +- 0.61, P&lt;0.001), more who had at least once positive nucleic acid detection of throat swab during admission (50.0% and 11.8%, P=0.011), and more received oxygen support (79.2% and 23.5%, P&lt;0.001). The rate of in-hospital adverse events was significantly higher in patients with CVMs group (75.0% and 23.5%, P=0.001). Multivariable logistic regression model indicated that, coexisting with CVMs in COVID-19 patients was not independently associated with in-hospital adverse events. Conclusions: Most of hospital staff with COVID-19 had history of patient contact, featured non-severe course of disease. Cases with CVMs suffered from more in-hospital adverse events than those without CVMs. But concomitant CVMs were not independently associated with in-hospital adverse events in COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus  broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>A systematic review and evaluation of Zika virus forecasting and prediction research during a public health emergency of international concern</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-05-10</span></i><p style="text-align: justify; text-justify: inter-word;">INTRODUCTIONEpidemic forecasting and prediction tools have the potential to provide actionable information in the midst of emerging epidemics. While numerous predictive studies were published during the 2016-2017 Zika Virus (ZIKV) pandemic, it remains unknown how timely, reproducible and actionable the information produced by these studies was.\n\nMETHODSTo improve the functional use of mathematical modeling in support of future infectious disease outbreaks, we conducted a systematic review of all ZIKV prediction studies published during the recent ZIKV pandemic using the PRISMA guidelines. Using MEDLINE, EMBASE and grey literature review, we identified studies that forecasted, predicted or simulated ecological or epidemiological phenomenon related to the Zika pandemic that were published as of March 01, 2017. Eligible studies underwent evaluation of objectives, data sources, methods, timeliness, reproducibility, accessibility and clarity by independent reviewers.\n\nRESULTS2034 studies were identified, of which n = 73 met eligibility criteria. Spatial spread, R0 (basic reproductive number) and epidemic dynamics were most commonly predicted, with few studies predicting Guillain-Barre Syndrome burden (4%), sexual transmission risk (4%) and intervention impact (4%). Most studies specifically examined populations in the Americas (52%), with few African-specific studies (4%). Case count (67%), vector (41%) and demographic data (37%) were the most common data sources. Real-time internet data and pathogen genomic information were used in 7% and 0% of studies, respectively, and social science and behavioral data were typically absent in modeling efforts. Deterministic models were favored over stochastic approaches. Forty percent of studies made model data entirely available, 29% provided all relevant model code, 43% presented uncertainty in all predictions and 54% provided sufficient methodological detail allowing complete reproducibility. Fifty-one percent of predictions were published after the epidemic peak in the Americas. While the use of preprints improved the accessibility of ZIKV predictions by a median 119 days sooner than journal publication dates, they were used in only 30% of studies.\n\nCONCLUSIONSMany ZIKV predictions were published during the 2016-2017 pandemic. The accessibility, reproducibility, timeliness, and incorporation of uncertainty in these published predictions varied and indicates that there is substantial room for improvement. To enhance the utility of analytical tools for outbreak response, it is essential to improve the sharing of model data, code, and preprints for future outbreaks, epidemics and pandemics.\n\nAuthor summaryResearchers published many studies which sought to predict and forecast important features of Zika virus (ZIKV) infections and their spread during the 2016-2017 ZIKV pandemic. We conducted a comprehensive review of such ZIKV prediction studies and evaluated their aims, the data sources they used, which methods were used, how timely they were published, and whether they provided sufficient information to be used or reproduced by others. Of the 73 studies evaluated, we found that the accessibility, reproducibility, timeliness, and incorporation of uncertainty in these published predictions varied and indicates that there is substantial room for improvement. We identified that the release of study findings before formal journal publication ( pre-prints) increased the timeliness of Zika prediction studies, but note they were infrequently used during this public health emergency. Addressing these areas can improve our understanding of Zika and other outbreaks and ensure that forecasts can inform preparedness and response to future outbreaks, epidemics and pandemics.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Coronavirus disease-2019 (COVID-19) is a rapidly escalating epidemic caused by SARS-CoV-2. Identification of a simple and effective indicator to assess disease severity and outcome is urgently needed.  Methods: This study retrospectively analyzed dynamic changes of lymphocyte percentage (LYM%) in 15 death cases, 15 severe cases as well as 40 moderate cases of COVID-19 patients. Next, prognostic role of lymphopenia in COVID-19 were verified in 92 hospitalized cases. Results: Our results from death and severe cases showed that LYM% in blood test were inversely associated with the progression and severity of COVID-19. LYM% in patients with moderate COVID-19 remained higher than 20% 10-12 days after symptom onset. In contrast, LYM% was lower than 20% in severe cases. However, LYM% in severe cases was higher than 5% 17-19 days after the onset of the disease, while it fell below 5% in death cases. Therefore, we established a reliable Time from symptom onset-LYM% model (TLM), which could be used to evaluate disease severity and predict the outcomes of hospitalized patients with COVID-19. Conclusion: Lymphopenia can be used to indicate clinical course, treatment effect and outcomes of COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control.  Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk.   Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 32 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days.   Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Assay Challenges for Emerging Infectious Diseases: The Zika Experience</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 2</span></i><p style="text-align: justify; text-justify: inter-word;">From the perspective of vaccine development, it is imperative to accurately diagnose target infections in order to exclude subjects with prior exposure from evaluations of vaccine effectiveness, to track incident infection during the course of a clinical trial and to differentiate immune reactions due to natural infections from responses that are vaccine related. When vaccine development is accelerated to a rapid pace in response to emerging infectious disease threats, the challenges to develop such diagnostic tools is even greater. This was observed through the recent expansion of Zika virus infections into the Western Hemisphere in 2014–2017. When initial Zika vaccine clinical trials were being designed and launched in response to the outbreak, there were no standardized sets of viral and immunological assays, and no approved diagnostic tests for Zika virus infection. The diagnosis of Zika virus infection is still an area of active research and development on many fronts. Here we review emerging infectious disease vaccine clinical assay development and trial execution with a special focus on the state of Zika virus clinical assays and diagnostics.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Neurological Manifestations of Hospitalized Patients with COVID-19 in Wuhan, China: a retrospective case series study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>OBJECTIVE: To study the neurological manifestations of patients with coronavirus disease 2019 (COVID-19).  DESIGN: Retrospective case series SETTING: Three designated COVID-19 care hospitals of the Union Hospital of Huazhong University of Science and Technology in Wuhan, China. PARTICIPANTS: Two hundred fourteen hospitalized patients with laboratory confirmed diagnosis of severe acute respiratory syndrome from coronavirus 2 (SARS-CoV-2) infection. Data were collected from 16 January 2020 to 19 February 2020. MAIN OUTCOME MEASURES: Clinical data were extracted from electronic medical records and reviewed by a trained team of physicians. Neurological symptoms fall into three categories: central nervous system (CNS) symptoms or diseases (headache, dizziness, impaired consciousness, ataxia, acute cerebrovascular disease, and epilepsy), peripheral nervous system (PNS) symptoms (hypogeusia, hyposmia, hypopsia, and neuralgia), and skeletal muscular symptoms. Data of all neurological symptoms were checked by two trained neurologists. RESULTS: Of 214 patients studied, 88 (41.1%) were severe and 126 (58.9%) were non-severe patients. Compared with non-severe patients, severe patients were older (58.7 ± 15.0 years vs 48.9 ± 14.7 years), had more underlying disorders (42 [47.7%] vs 41 [32.5%]), especially hypertension (32 [36.4%] vs 19 [15.1%]), and showed less typical symptoms such as fever (40 [45.5%] vs 92 [73%]) and cough (30 [34.1%] vs 77 [61.1%]). Seventy-eight (36.4%) patients had neurologic manifestations. More severe patients were likely to have neurologic symptoms (40 [45.5%] vs 38 [30.2%]), such as acute cerebrovascular diseases (5 [5.7%] vs 1 [0.8%]), impaired consciousness (13 [14.8%] vs 3 [2.4%]) and skeletal muscle injury (17 [19.3%] vs 6 [4.8%]). CONCLUSION: Compared with non-severe patients with COVID-19, severe patients commonly had neurologic symptoms manifested as acute cerebrovascular diseases, consciousness impairment and skeletal muscle symptoms.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 sensitive to type I interferon pretreatment.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B coronaviruses share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel coronavirus is much more sensitive to type I interferon pretreatment. We subsequently examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN response between SARS-CoV and SARS-CoV-2 that could help inform disease progression, treatment options, and animal model development.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 31st December 2019, the World Health Organization was informed of a cluster of cases of pneumonia of unknown etiology in Wuhan, China. Subsequent investigations identified a novel coronavirus, now named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), from the affected patients. Highly sensitive and specific laboratory diagnostics are important for controlling the rapidly evolving SARS-CoV-2-associated Coronavirus Disease 2019 (COVID-19) epidemic. In this study, we developed and compared the performance of three novel real-time RT-PCR assays targeting the RNA-dependent RNA polymerase (RdRp)/helicase (Hel), spike (S), and nucleocapsid (N) genes of SARS-CoV-2 with that of the reported RdRp-P2 assay which is used in >30 European laboratories. Among the three novel assays, the COVID-19-RdRp/Hel assay had the lowest limit of detection in vitro (1.8 TCID50/ml with genomic RNA and 11.2 RNA copies/reaction with in vitro RNA transcripts). Among 273 specimens from 15 patients with laboratory-confirmed COVID-19 in Hong Kong, 77 (28.2%) were positive by both the COVID-19-RdRp/Hel and RdRp-P2 assays. The COVID-19-RdRp/Hel assay was positive for an additional 42 RdRd-P2-negative specimens [119/273 (43.6%) vs 77/273 (28.2%), P<0.001], including 29/120 (24.2%) respiratory tract specimens and 13/153 (8.5%) non-respiratory tract specimens. The mean viral load of these specimens was 3.21*104 RNA copies/ml (range, 2.21*102 to 4.71*105 RNA copies/ml). The COVID-19-RdRp/Hel assay did not cross-react with other human-pathogenic coronaviruses and respiratory pathogens in cell culture and clinical specimens, whereas the RdRp-P2 assay cross-reacted with SARS-CoV in cell culture. The highly sensitive and specific COVID-19-RdRp/Hel assay may help to improve the laboratory diagnosis of COVID-19.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral vector-based influenza vaccines</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jul 25</span></i><p style="text-align: justify; text-justify: inter-word;">Antigenic drift of seasonal influenza viruses and the occasional introduction of influenza viruses of novel subtypes into the human population complicate the timely production of effective vaccines that antigenically match the virus strains that cause epidemic or pandemic outbreaks. The development of game-changing vaccines that induce broadly protective immunity against a wide variety of influenza viruses is an unmet need, in which recombinant viral vectors may provide. Use of viral vectors allows the delivery of any influenza virus antigen, or derivative thereof, to the immune system, resulting in the optimal induction of virus-specific B- and T-cell responses against this antigen of choice. This systematic review discusses results obtained with vectored influenza virus vaccines and advantages and disadvantages of the currently available viral vectors.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Precautions are Needed for COVID-19 Patients with Coinfection of Common Respiratory Pathogens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: With the ongoing outbreak of Coronavirus Disease 2019 (COVID-19), infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan. Methods: We conducted a double-centre study recruiting 68 patients with severe acute respiratory coronavirus 2 (SARS-CoV-2) infection confirmed by nucleic acid testing in Qingdao and Wuhan from January 17 to February 16, 2020. Indirect immunofluorescence was performed to detect the specific IgM antibody against common respiratory pathogens in collected acute phase serum.  Results: Of the 68 patients with SARS-CoV-2 infection, 30 (44.12%) were from Qingdao. The median age of Qingdao and Wuhan patients were 50 (IQR: 37-59) and 31 (IQR: 28-38) years, respectively, and the majority of patients were female in Qingdao (60.00%) and Wuhan (55.26%). Among COVID-19 patients in Qingdao, 24 (80.00%) of them had IgM antibodies against at least one respiratory pathogen, whereas only one (2.63%) of the patients in Wuhan had positive results for serum IgM antibody detection (P&lt;0.0001). The most common respiratory pathogens detected in Qingdao COVID-19 patients were influenza virus A (60.00%) and influenza virus B (53.33%), followed by mycoplasma pneumoniae (23.33%) and legionella pneumophila (20.00%). While the pattern for coinfection in patients with community-acquired pneumonia in Qingdao was quite different, with a positive rate of only 20.90%.  Interpretation: We reported a large proportion of COVID-19 patients with coinfection of seasonal respiratory pathogens in Qingdao, northeast China, which differed greatly from the patients in Wuhan, central China. Precautions are needed when dealing with COVID-19 patients beyond the epidemic centre who have coinfection with other respiratory pathogens. We highly recommend adding SARS-CoV-2 to routine diagnostic testing in capable hospitals to prevent misdetection of the virus.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Case fatality rate of novel coronavirus disease 2019 in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Background: A pandemic of coronavirus disease 2019 (COVID-19) which have caused more than 80 thousand persons infected globally is still ongoing. This study aims to calculate its case fatality rate (CFR). Methods: The method, termed as converged CFR calculation, was based on the formula of dividing the number of known deaths by the number of confirmed cases T days before, where T was an average time period from case confirmation to death. It was found that supposing a T, if it was smaller (bigger) than the true T, calculated CFRs would gradually increase (decrease) to infinitely near the true T with time went on. According to the law, the true T value could be determined by trends of daily CFRs calculated with different assumed T values (left of true T is decreasing, right is increasing). Then the CFR could be calculated. Results: CFR of COVID-19 in China except Hubei Province was 0.8% to 0.9%. So far, the CFR had accurately predicted the death numbers more than 3 weeks. CFR in Hubei of China was 5.4% by which the calculated death number corresponded with the reported number for 2 weeks. Conclusion: The method could be used for CFR calculating while pandemics are still ongoing. Dynamic monitoring of the daily CFRs trends could help outbreak-controller to have a clear vision in the timeliness of the case confirmation.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>The efficacy of convalescent plasma for the treatment of severe influenza</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background. Administration of convalescent plasma may be of clinical benefit for treatment of severe acute viral respiratory infections. However, no clear evidence exists to support or oppose convalescent plasma use in clinical practice. We conducted a systematic review and meta-analysis to assess the evidence of randomized controlled trials (RCTs) in the convalescent plasma for the treatment of severe influenza. Methods. Healthcare databases were searched in February 2020. All records were screened against the eligibility criteria. Data extraction and risk of bias assessments were undertaken. The primary outcome was case fatality rates by influenza. Results. We identified 5 RCTs of severe influenza. The pooled analyses showed no evidence for a reduction in mortality (Odds Ratio (OR) 1.06; p = 0.87; I2 = 35%). We also found non significant reductions in days in ICU and hospital, and days on mechanical ventilation. There seemed to have a biological benefit of increasing HAI titer levels and decreasing influenza B virus loads and cytokines after convalescent plasma treatment. No serious adverse events was reported between two groups. Studies were commonly of low risk of bias with high quality.  Conclusions. Convalescent plasma appears safe but may not reduce mortality in severe influenza. This therapy should be studied within the context of a well designed clinical trial for treatment of SARS Cov 2 infection.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Four ways researchers are responding to the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts. How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Real-time dissemination of scientific information to fight a public health emergency of international concern</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rapidly sharing scientific information is an effective way to reduce public panic about COVID-19, and doing so is the key to providing real-time guidance to epidemiologists working to contain the outbreak, clinicians managing patients, and modelers helping to understand future developments and the possible effectiveness of various interventions. This issue has rapidly reviewed and published articles describing COVID-19, including the drug treatment options for SARS-CoV-2, its clinical characteristics, and therapies involving a combination of Chinese and Western medicine, the efficacy of chloroquine phosphate in the treatment of COVID-19 associated pneumonia according to clinical studies, and reflections on the system of reserve medical supplies for public health emergencies. As an academic journal, we will continue to quickly and transparently share data with frontline healthcare workers who need to know the epidemiological and clinical features of COVID-19.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Healthcare Worker Seroconversion in SARS Outbreak</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Serum samples were obtained from healthcare workers 5 weeks after exposure to an outbreak of severe acute respiratory syndrome (SARS). A sensitive dot blot enzyme-linked immunosorbent assay, complemented by a specific neutralization test, shows that only persons in whom probable SARS was diagnosed had specific antibodies and suggests that subclinical SARS is not an important feature of the disease.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China.  Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity.  Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country.  Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 transmission in cancer patients of a tertiary hospital in Wuhan</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>In December 2019, an outbreak of atypical pneumonia known as 2019 novel coronavirus disease (COVID-19) occurred in Wuhan, China. This new type of pneumonia is characterized by rapid human-to-human transmission. Among the different disease types, cancer patients are often recalled to the hospital for treatment and disease surveillance, and the majority of cancer treatments such as chemotherapy and radiotherapy are immunosuppressive.  This prompts us to consider if cancer patients were at an elevated risk of SARS-CoV-2 infection.  A total of 1,524 cancer patients who were managed at our tertiary cancer institution-Zhongnan hospital of Wuhan University were reviewed during the period of Dec 30, 2019 to Feb 17, 2020. It was found that cancer patients had an estimated 2-fold increased risk of COVID-19 than the general population. We identified twelve patients who were infected with SARS-CoV-2, with two recorded deaths (16.7%), albeit one patient passed away from a COVID-19 unrelated cause. Interestingly, only five of these patients were ongoing treatment at the time of contracting the virus, suggesting that hospital visitation was the likely factor contributing to the elevated incidence in cancer patients. Moreover, we also observed that the incidence of severe COVID-19 was not higher than in the general population. Consequently, for cancer patients who require treatment, proper isolation protocols must be in place to mitigate the risk of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Does SARS-CoV-2 has a longer incubation period than SARS and MERS?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of a novel coronavirus (SARS-CoV-2) since Dec 2019 in Wuhan, the major transportation hub in central China, became an emergency of major international concern. While several etiological studies have begun to reveal the specific biological features of this virus, the epidemic characteristics need to be elucidated. Notably, a long incubation time was reported to be associated with SARS-CoV-2 infection, leading to adjustments in screening and control policies. To avoid the risk of virus spread, all potentially exposed subjects are required to be isolated for 14 days, which is the longest predicted incubation time. However, based on our analysis of a larger dataset available so far, we find there is no observable difference between the incubation time for SARS-CoV-2, SARS and MERS, highlighting the need for larger and well annotated datasets. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovering drugs to treat coronavirus disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 virus emerged in December 2019 and then spread rapidly worldwide, particularly to China, Japan, and South Korea. Scientists are endeavoring to find antivirals specific to the virus. Several drugs such as chloroquine, arbidol, remdesivir, and favipiravir are currently undergoing clinical studies to test their efficacy and safety in the treatment of coronavirus disease 2019 (COVID-19) in China; some promising results have been achieved thus far. This article summarizes agents with potential efficacy against SARS-CoV-2.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Critical care crisis and some recommendations during the COVID-19 epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Lack of critical care resource in face of COVID-19 epidemics Based on data reported by the National Health Commission of China, there have been about 2000 new confirmed cases and?>?4000 suspected cases daily over the past week in Wuhan [3]. About 15% of the patients have developed severe pneumonia, and about 6% need noninvasive or invasive ventilatory support. Currently, there are about 1000 patients who need ventilatory support and another 120 new patients daily who require noninvasive or invasive ventilation support in Wuhan city; however, there are only about 600 ICU beds [4]. To address this shortfall, 70 ICU beds were created from general beds and the government quickly transformed three general hospitals to critical care hospitals with a total of about 2500 beds that specialize in patients with severe SARS-CoV-2 pneumonia (equipped with monitors and high-flow nasal cannula, noninvasive ventilator or invasive ventilators). An equally great (or potentially greater) problem is the shortage of trained personnel to treat these critically ill patients. Until the crisis, there were about 300 ICU physicians and 1000 ICU nurses in Wuhan city. By the end of January, more than 600 additional ICU doctors and 1500 ICU nurses were transferred to Wuhan from the rest of China. As well, an additional 3000 staff including infectious disease, respiratory, internal medicine physicians and nurses were transferred to Wuhan by the government. There are logistical issues which make care of the patients difficult. These include donning of personal protective equipment (e.g., gloves, gowns, respiratory and eye protection), lack of instruments and disposables, and shortages of supplemental oxygen. Many severe hypoxemic patients only receive high-flow nasal oxygen (HFNO) or noninvasive mechanical ventilation rather than invasive mechanical ventilation because of intubation delay or lack of mechanical ventilators (especially at early phase). Our preliminary data show that only about 25% of patients who died were intubated and received mechanical ventilation.JO - Intensive Care Medicine</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical manifestations and outcome of SARS-CoV-2 infection during pregnancy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Tang and colleagues, in this Journal, drew readers attention to emerging COVID19.1 We focused on the pregnant COVID19 patients. Given the maternal physiologic and immune function changes in pregnancy,2 pregnant individuals might face greater risk of getting infected by SARS-CoV-2 and might have more complicated clinical events. We described epidemiological, clinical characteristics, pregnancy and perinatal outcomes of all hospitalized pregnant patients diagnosed with COVID-19 in China. We identified all hospitalized pregnant patients with laboratory-confirmed SARS-CoV-2 infection between December 8, 2019, and February 25, 2020 officially reported by the central government, in areas outside Wuhan, China. Information including age, geographic location, epidemiological history, prenatal course, maternal and newborn hospital course, discharge data and outcome were obtained by Centers for Disease Control and Prevention and Local Health Commission. When necessary, we attempted to contact local hospital or patients by telephone to supply missing information. This investigation was part of an emergency public health outbreak investigation and therefore not subject to institutional review board. There were a total of 13 Chinese patients with SARS-CoV-2 admitted to hospitals outside of Wuhan (Table 1). There were 3 patients from Zhejiang, 3 from other cities of Hubei and 1 each from Fujian, Shanxi, Beijing, Guangdong, Jiangxi, Heilongjiang and Anhui. The maternal age ranged between 22 to 36 years. Two women were less than 28 weeks of gestation and the other 11 patients were in their third trimesters at presentation. None of the patients had underlying medical disease.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamic effects of antibody-dependent enhancement on the fitness of viruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Antibody-dependent enhancement (ADE), a phenomenon in which viral replication is increased rather than decreased by immune sera, has been observed in vitro for a large number of viruses of public health importance, including flaviviruses, coronaviruses, and retroviruses. The most striking in vivo example of ADE in humans is dengue hemorrhagic fever, a disease in which ADE is thought to increase the severity of clinical manifestations of dengue virus infection by increasing virus replication. We examine the epidemiological impact of ADE on the prevalence and persistence of viral serotypes. Using a dynamical system model of n cocirculating dengue serotypes, we find that ADE may provide a competitive advantage to those serotypes that undergo enhancement compared with those that do not, and that this advantage increases with increasing numbers of cocirculating serotypes. Paradoxically, there are limits to the selective advantage provided by increasing levels of ADE, because greater levels of enhancement induce large amplitude oscillations in incidence of all dengue virus infections, threatening the persistence of both the enhanced and nonenhanced serotypes. Although the models presented here are specifically designed for dengue, our results are applicable to any epidemiological system in which partial immunity increases pathogen replication rates. Our results suggest that enhancement is most advantageous in settings where multiple serotypes circulate and where a large host population is available to support pathogen persistence during the deep troughs of ADE-induced large amplitude oscillations of virus replication.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /></body>