
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: use of best animal models and their predictive value for a human SARS-CoV-2 vaccine</h2><br /></div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 Vaccine Pipeline: an Overview</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19).</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>The outbreak of SARS-CoV-2 pneumonia calls for viral vaccines</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of 2019-novel coronavirus disease (COVID-19) that is caused by SARS-CoV-2 has spread rapidly in China, and has developed to be a Public Health Emergency of International Concern. However, no specific antiviral treatments or vaccines are available yet. This work aims to share strategies and candidate antigens to develop safe and effective vaccines against SARS-CoV-2. An outbreak of 2019-novel coronavirus (SARS-CoV-2) that causes atypical pneumonia (COVID-19) has raged in China since mid-December 2019 and has spread to 26 countries (February 20, 2020). The epidemic was identified by the first four cases confirmed on December 29, 2019 and was traced to the Huanan Seafood Wholesale Market, Wuhan city, Hubei Province, China1. A total of 75,465 cases with SARS-CoV-2 infections have been confirmed up to date (February 20, 2020), and 2,236 people have died in China2. COVID-19 spreads rapidly by human-to-human transmission with a median incubation period of 3.0 days (range, 0 to 24.0), and the time from symptom onset to developing pneumonia is 4.0 days (range, 2.0 to 7.0)3. Respiratory droplets and direct contact are conventional transmission routes for SARS-CoV-2, and fecal-to-oral transmission might also have a role3. Fever, dry cough, and fatigue are common symptoms at onset of COVID-194. Most patients have lymphopenia and bilateral ground-glass opacity changes on chest CT scans4,5. No specific antiviral treatments or vaccines are available because it is a new emerging viral disease. Development of SARS-CoV-2-based vaccines is urgently required. The entire virus particle-based preparation of vaccines, including inactivated and attenuated virus vaccines is advisable, because it is based on previous studies about the prevention and control of seasonal influenza vaccines6. The first SARS-CoV-2 (Wuhan-Hu-1) was successfully sequenced and its genomic sequence submitted to GenBank on January 5, 2020 (Accession no. MN908947.3)7. Subsequently large-scale culture of SARS-CoV-2 was quickly performed, and an inactivated virus vaccine could be prepared through the employment of established physical and chemical methods such as UV light, formaldehyde, and β-propiolactone8. The development of attenuated-virus vaccines is also possible by carefully screening the serially propagated SARS-CoV-2 with reduced pathogenesis such as induced minimal lung injury, diminished limited neutrophil influx, and increased anti-inflammatory cytokine expressions compared with the wild-type virus9. Both inactivated and attenuated virus vaccines have their own disadvantages and side effects (Table 1). Alternatively, new vaccine designs based on the putative protective antigen/peptides derived from SARS-CoV-2 should be considered</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;">Wuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available for the cure of COVID-19. Therefore, an effective vaccine against SARS-COV-2 is needed to be design. This study was conducted to design an effective multi-epitope vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as a target and epitopes (B cell, IFN-{gamma} and T cell) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected T cell epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the worlds population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and twelve MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the vaccine. The allergenicity, physiochemical properties, antigenicity and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic and antigenic. Molecular docking demonstrated a stable and strong binding affinity of MEV with TLR3 and TLR8. Codon optimization and in silico cloning ensured increased expression in the Escherichia coli K-12 system. However, to ensure its safety and immunogenic profile, the proposed vaccine needs to be experimentally validated.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to COVID-19: A previously unknown SARS-CoV-2 virus of pandemic potential infecting humans – Call for a One Health approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses continue to pose a threat to human health. The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in December 2019 which causes coronavirus disease-2019 (COVID-19), an acute respiratory disease marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. This recent ongoing emergence of a previously unknown coronavirus in China leads to huge impacts on humans globally. Here, we discuss the outbreak in a one health context, highlighting the need for the implementation of one health measures and practices to improve human health and reduce the emergence of pandemic viruses.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>The deadly coronaviruses: The 2003 SARS pandemic and the 2020 novel coronavirus epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019-nCoV is officially called SARS-CoV-2 and is the cause of the disease named COVID-19. This viral epidemic in China has led to the deaths of over 1800 people, mostly elderly or those with an underlying chronic disease or immunosuppressed state. This is the third serious Coronavirus outbreak in less than 20 years, following SARS in 2002–2003 and MERS in 2012. While human strains of Coronavirus are associated with about 15% of cases of the common cold, the SARS-CoV-2 may present with varying degrees of severity, from flu-like symptoms to death. It is currently believed that this deadly Coronavirus strain originated from wild animals at the Huanan market in Wuhan, a city in Hubei province. Bats, snakes and pangolins have been cited as potential carriers based on the sequence homology of CoV isolated from these animals and the viral nucleic acids of the virus isolated from SARS-CoV-2 infected patients. Extreme quarantine measures, including sealing off large cities, closing borders and confining people to their homes, were instituted in January 2020 to prevent spread of the virus, but by that time much of the damage had been done, as human-human transmission became evident. While these quarantine measures are necessary and have prevented a historical disaster along the lines of the Spanish flu, earlier recognition and earlier implementation of quarantine measures may have been even more effective. Lessons learned from SARS resulted in faster determination of the nucleic acid sequence and a more robust quarantine strategy. However, it is clear that finding an effective antiviral and developing a vaccine are still significant challenges. The costs of the epidemic are not limited to medical aspects, as the virus has led to significant sociological, psychological and economic effects globally. Unfortunately, emergence of SARS-CoV-2 has led to numerous reports of Asians being subjected to racist behavior and hate crimes across the world.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>A combined nucleocapsid vaccine induces vigorous SARS-CD8+ T-cell immune responses</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Aug 22</span></i><p style="text-align: justify; text-justify: inter-word;">Several studies have shown that cell-mediated immune responses play a crucial role in controlling viral replication. As such, a candidate SARS vaccine should elicit broad CD8+ T-cell immune responses. Several groups of mice were immunized alone or in combination with SARS-nucleocapsid immunogen. A high level of specific SARS-CD8+ T-cell response was demonstrated in mice that received DNA encoding the SARS-nucleocapsid, protein and XIAP as an adjuvant. We also observed that co-administration of a plasmid expressing nucleocapsid, recombinant protein and montanide/CpG induces high antibody titers in immunized mice. Moreover, this vaccine approach merits further investigation as a potential candidate vaccine against SARS.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Has China faced only a herald wave of SARS-CoV-2?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The attack rate of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) calculated by mathematical models, from estimates of the basic reproduction number, R0, of 2–3, suggests that 50–60% of the population should eventually be infected because the population seems to be entirely naive to the new virus.1 The observed attack rate on board the Diamond Princess cruise ship remained slightly below 20% (705 of 3711 passengers and crew members became infected).1 It is of upmost importance to know whether the SARS-CoV-2 outbreak in China is subsiding, as local authorities and the entire international community might wish. With 80 026 COVID-19 cases officially reported from China as of March 2, 2020,2 the proportion of the population affected remains far from 50%, or even 20%, of China's 1·4 billion people. Has China just experienced a herald wave, to use terminology borrowed from those who study tsunamis, and is the big wave still to come? Serosurveys can help answer these questions precisely.3 To serosurvey the outbreak would involve testing sera of blood samples from the most representative sample of the population at the epicentre of the epidemic, Wuhan. Serology analysis with neutralising antibodies from the 1000 people could allow for the rate of SARS-CoV-2 infections to be estimated with good accuracy. This rate could be extrapolated to the city's entire population and thus inform more precisely whether the provisional attack rate during this period was a few cases per thousand or perhaps affected 1–2% of the population, 20%, or more. Serosurveys should be seen as polls before elections; they can be repeated several times,3 week after week, to monitor the epidemic precisely. There is no reason to wait for the end of the epidemic before doing serosurveys. The results would be tremendously informative to China, first and foremost, and to the entire international community, on the risk of big secondary epidemic waves</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Current advancements and potential strategies in the development of MERS-CoV vaccines</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) is a newly emerging infectious disease caused by a novel coronavirus, MERS-coronavirus (MERS-CoV), a new member in the lineage C of β-coronavirus (β-CoV). The increased human cases and high mortality rate of MERS-CoV infection make it essential to develop safe and effective vaccines. In this review, the current advancements and potential strategies in the development of MERS vaccines, particularly subunit vaccines based on MERS-CoV spike (S) protein and its receptor-binding domain (RBD), are discussed. How to improve the efficacy of subunit vaccines through novel adjuvant formulations and routes of administration as well as currently available animal models for evaluating the in vivo efficacy of MERS-CoV vaccines are also addressed. Overall, these strategies may have important implications for the development of effective and safe vaccines for MERS-CoV in the future.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of a novel coronavirus associated with an ongoing outbreak of pneumonia (Covid-2019) resulted in infections of more than 72,000 people and claimed over 1,800 lives. Coronavirus spike (S) glycoprotein trimers promote entry into cells and are the main target of the humoral immune response. We show here that SARS-CoV-2 S mediates entry in VeroE6 cells and in BHK cells transiently transfected with human ACE2, establishing ACE2 as a functional receptor for this novel coronavirus. We further demonstrate that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, which correlates with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S1/S2 subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and other SARS-related CoVs. We determined a cryo-electron microscopy structure of the SARS-CoV-2 S ectodomain trimer, demonstrating spontaneous opening of the receptor-binding domain, and providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal sera potently inhibited SARS-CoV-2 S-mediated entry into target cells, thereby indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Engineering a stable CHO cell line for the expression of a MERS-coronavirus vaccine antigen</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) has infected at least 2,040 patients and caused 712 deaths since its first appearance in 2012, yet neither pathogen-specific therapeutics nor approved vaccines are available. To address this need, we are developing a subunit recombinant protein vaccine comprising residues 377 to 588 of the MERS-CoV spike protein receptor-binding domain (RBD), which, when formulated with the AddaVax adjuvant, it induces a significant neutralizing antibody response and protection against MERS-CoV challenge in vaccinated animals. To prepare for the manufacture and first-in-human testing of the vaccine, we have developed a process to stably produce the recombinant MERS S377–588 protein in Chinese hamster ovary (CHO) cells. To accomplish this, we transfected an adherent dihydrofolate reductase-deficient CHO cell line (adCHO) with a plasmid encoding S377–588 fused with the human IgG Fc fragment (S377-588-Fc). We then demonstrated the interleukin-2 signal peptide-directed secretion of the recombinant protein into extracellular milieu. Using a gradually increasing methotrexate (MTX) concentration to 5 μM, we increased protein yield by a factor of 40. The adCHO-expressed S377-588-Fc recombinant protein demonstrated functionality and binding specificity identical to those of the protein from transiently transfected HEK293T cells. In addition, hCD26/dipeptidyl peptidase-4 (DPP4) transgenic mice vaccinated with AddaVax-adjuvanted S377-588-Fc could produce neutralizing antibodies against MERS-CoV and survived for at least 21 days after challenge with live MERS-CoV with no evidence of immunological toxicity or eosinophilic immune enhancement. To prepare for large scale-manufacture of the vaccine antigen, we have further developed a high-yield monoclonal suspension CHO cell line.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>The within-host viral kinetics of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>A new transmission route for the propagation of the SARS-CoV-2 coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>A randomized placebo-controlled phase 1 safety and tolerability study of a novel human anti-MERS coronavirus polyclonal intravenous immunoglobulin produced from transchromosomic cattle</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Passive immunotherapy approaches are being developed to prevent and treat a variety of human medical conditions. Here we report the first use of a fully-human polyclonal IgG immunoglobulin (SAB-301) produced from hyperimmune plasma of transchromosomic (Tc) cattle immunized with a Middle East Respiratory Syndrome (MERS) coronavirus vaccine. METHODS: We conducted a double-blind, placebo-controlled, single dose escalation study in six cohorts of 3–10 participants who were randomly assigned by a computer-generated table to receive 1, 2.5, 5, 10, 20, or 50 mg/kg of SAB-301 or placebo on Day 0, and were followed by clinical, laboratory, and pharmacokinetic assessments on days 1, 3, 7, 21, 42 and 90. The primary outcome was safety whereas pharmacokinetic profile, MERS virus neutralization assay over time (pharmacodynamics), and immunogenicity were secondary outcomes. The analysis was performed on the intention to treat population. ClinicalTrials.gov Identifier: NCT02788188. FINDINGS: We randomized 38 participants (28 to SAB-301 and 10 to placebo). Ninety-seven adverse event (AEs) were reported: 64 AEs occurred in 23 of 28 participants (82%) receiving SAB-301 (mean 2.3 AEs per participant), and 33 AEs occurred in 10 of 10 participants (100%) receiving placebo (mean 3.3 AEs per participant). The most common AEs were headache, albuminuria, elevated creatine kinase, and common cold, and occurred in similar proportions as placebo. Single dose pharmacokinetics (PK) demonstrated relatively linear and dose-proportional increases in maximal concentration and area-under-the-concentration-time curve (AUC(0–24)), and the PK strongly correlated with the microneutralization assay (R(2)=0.845, p<0.001). The AUC in the 50 mg/kg dose (27498.18 μg*days/mL) is comparable to the AUC that was associated with protection in a preclinical model. The average terminal elimination half-life (t(1/2)) was ~ 28 days. INTERPRETATION: Single infusions of SAB-301 up to 50 mg/kg appear to be safe and well-tolerated in healthy participants. Tc cattle derived human immunoglobulin offers a new platform technology to produce fully-human polyclonal IgG immunoglobulin for other medical conditions. FUNDING: National Institute of Allergy and Infectious Diseases, National Institutes of Health, United States.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Mucin 4 Protects Female Mice from Coronavirus Pathogenesis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">Using incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4-/- mice and found that female, but not male, Muc4-/- mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4-/- mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4s impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis.  ImportanceGiven the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the world is witnessing the epidemic of COVID-19, a disease caused by a novel coronavirus, SARS-CoV-2, emerging genetics and clinical evidences suggest a similar path to those of SARS and MERS. The rapid genomic sequencing and open access data, together with advanced vaccine technology, are expected to give us more knowledge on the pathogen itself, including the host immune response as well as the plan for therapeutic vaccines in the near future. This review aims to provide a comparative view among SARS-CoV, MERS-CoV and the newly epidemic SARS-CoV-2, in the hope to gain a better understanding of the host-pathogen interaction, host immune responses, and the pathogen immune evasion strategies. This predictive view may help in designing an immune intervention or preventive vaccine for COVID-19 in the near future.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: combining antiviral and anti-inflammatory treatments</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Both coronavirus disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS) are characterised by an overexuberant inflammatory response and, for SARS, viral load is not correlated with the worsening of symptoms. In our previous Correspondence to The Lancet, we described how BenevolentAI's proprietary artificial intelligence (AI)-derived knowledge graph, queried by a suite of algorithms, enabled identification of a target and a potential therapeutic against SARS coronavirus 2 (SARS-CoV-2; the causative organism in COVID-19). We identified a group of approved drugs that could inhibit clathrin-mediated endocytosis and thereby inhibit viral infection of cells (appendix). The drug targets are members of the numb-associated kinase (NAK) family—including AAK1 and GAK—the inhibition of which has been shown to reduce viral infection in vitro. Baricitinib was identified as a NAK inhibitor, with a particularly high affinity for AAK1, a pivotal regulator of clathrin-mediated endocytosis. We suggested that this drug could be of use in countering SARS-CoV-2 infections, subject to appropriate clinical testing.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;">A new coronavirus SARS-CoV-2, recently discovered in Wuhan, China, has caused over 74000 infection cases and 2000 deaths. Due to the rapidly growing cases and the unavailability of specific therapy, there is a desperate need for vaccines to combat the epidemic of SARS-CoV-2. In the present study, we performed an in silico approach based on the available virus genome to identify the antigenic B-cell epitopes and human-leukocyte-antigen (HLA) restricted T-cell epitopes. A total of 61 B-cell epitopes were initially identified, 19 of which with higher potential immunogenicity were used for vaccine design. 499 T-cell epitopes were predicted that showed affinity with the 34 most popular HLA alleles in Chinese population. Based on these epitopes, 30 vaccine candidates were designed and inspected against safety risks, including potential toxicity, human homologous, pharmaceutical peptides and bioactive peptides. Majority of vaccine peptides contained both B-cell and T-cell epitopes, which may interact with the most prevalent HLA alleles accounting for ~99% of Chinese population. Docking analysis showed stable hydrogen bonds of epitopes with their corresponding HLA alleles. In conclusion, these putative antigenic peptides may elicit the resistance response to the viral infection. In vitro and in vivo experiments are required to validate the effectiveness of these peptide vaccine.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Middle East Respiratory Syndrome Coronavirus vaccines – advances and challenges</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an emerging pathogen with the potential to pose a threat to global public health. Sporadic cases and outbreaks continue to be reported in the Middle East, and case fatality rates remain high at approximately 36% globally. No specific preventive or therapeutic countermeasures currently exist. A safe and effective vaccine could play an important role in protecting against the threat from MERS-CoV. This review discusses human vaccine candidates currently under development, and explores viral characteristics, molecular epidemiology and immunology relevant to MERS-CoV vaccine development. At present, a DNA vaccine candidate has begun a human clinical trial, while two vector-based candidates will very soon begin human trials. Protein-based vaccines are still at pre-clinical stage. Challenges to successful development include incomplete understanding of viral transmission, pathogenesis and immune response (in particular at the mucosal level), no optimal animal challenge models, lack of standardized immunological assays, and insufficient sustainable funding.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of an ideal adjuvant for receptor-binding domain-based subunit vaccines against Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS), an emerging infectious disease caused by MERS coronavirus (MERS-CoV), has garnered worldwide attention as a consequence of its continuous spread and pandemic potential, making the development of effective vaccines a high priority. We previously demonstrated that residues 377–588 of MERS-CoV spike (S) protein receptor-binding domain (RBD) is a very promising MERS subunit vaccine candidate, capable of inducing potent neutralization antibody responses. In this study, we sought to identify an adjuvant that optimally enhanced the immunogenicity of S377–588 protein fused with Fc of human IgG (S377–588-Fc). Specifically, we compared several commercially available adjuvants, including Freund's adjuvant, aluminum, Monophosphoryl lipid A, Montanide ISA51 and MF59 with regard to their capacity to enhance the immunogenicity of this subunit vaccine. In the absence of adjuvant, S377–588-Fc alone induced readily detectable neutralizing antibody and T-cell responses in immunized mice. However, incorporating an adjuvant improved its immunogenicity. Particularly, among the aforementioned adjuvants evaluated, MF59 is the most potent as judged by its superior ability to induce the highest titers of IgG, IgG1 and IgG2a subtypes, and neutralizing antibodies. The addition of MF59 significantly augmented the immunogenicity of S377–588-Fc to induce strong IgG and neutralizing antibody responses as well as protection against MERS-CoV infection in mice, suggesting that MF59 is an optimal adjuvant for MERS-CoV RBD-based subunit vaccines.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding domain-based subunit vaccines against MERS-CoV</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Development of effective vaccines, in particular, subunit-based vaccines, against emerging Middle East respiratory syndrome (MERS) caused by the MERS coronavirus (MERS-CoV) will provide the safest means of preventing the continuous spread of MERS in humans and camels. This review briefly describes the structure of the MERS-CoV spike (S) protein and its receptor-binding domain (RBD), discusses the current status of MERS vaccine development, and illustrates the strategies used to develop RBD-based subunit vaccines against MERS. It also summarizes currently available animal models for MERS-CoV and proposes a future direction for MERS vaccines. Taken together, this review will assist researchers working to develop effective and safe subunit vaccines against MERS-CoV and any other emerging coronaviruses that might cause future pandemics.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We report the kinetics of the immune response in relation to clinical and virological features of a patient with mild-to-moderate coronavirus disease-19 (COVID-19) requiring hospitalisation. Increased antibody-secreting cells, follicular T-helper cells, activated CD4+ and CD8+ T-cells and IgM/IgG SARS-CoV-2-binding antibodies were detected in blood, prior to symptomatic recovery. These immunological changes persisted for at least 7 days following full resolution of symptoms, indicating substantial anti-viral immunity in this non-severe COVID-19.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of candidate vaccine approaches for MERS-CoV</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) as a cause of severe respiratory disease highlights the need for effective approaches to CoV vaccine development. Efforts focused solely on the receptor-binding domain (RBD) of the viral Spike (S) glycoprotein may not optimize neutralizing antibody (NAb) responses. Here we show that immunogens based on full-length S DNA and S1 subunit protein elicit robust serum-neutralizing activity against several MERS-CoV strains in mice and non-human primates. Serological analysis and isolation of murine monoclonal antibodies revealed that immunization elicits NAbs to RBD and, non-RBD portions of S1 and S2 subunit. Multiple neutralization mechanisms were demonstrated by solving the atomic structure of a NAb-RBD complex, through sequencing of neutralization escape viruses and by constructing MERS-CoV S variants for serological assays. Immunization of rhesus macaques confers protection against MERS-CoV-induced radiographic pneumonia, as assessed using computerized tomography, supporting this strategy as a promising approach for MERS-CoV vaccine development.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of candidate vaccine approaches for MERS-CoV</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) as a cause of severe respiratory disease highlights the need for effective approaches to CoV vaccine development. Efforts focused solely on the receptor-binding domain (RBD) of the viral Spike (S) glycoprotein may not optimize neutralizing antibody (NAb) responses. Here we show that immunogens based on full-length S DNA and S1 subunit protein elicit robust serum-neutralizing activity against several MERS-CoV strains in mice and non-human primates. Serological analysis and isolation of murine monoclonal antibodies revealed that immunization elicits NAbs to RBD and, non-RBD portions of S1 and S2 subunit. Multiple neutralization mechanisms were demonstrated by solving the atomic structure of a NAb-RBD complex, through sequencing of neutralization escape viruses and by constructing MERS-CoV S variants for serological assays. Immunization of rhesus macaques confers protection against MERS-CoV-induced radiographic pneumonia, as assessed using computerized tomography, supporting this strategy as a promising approach for MERS-CoV vaccine development.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>First respiratory transmitted food borne outbreak?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The world is faced with a remarkable coronavirus outbreak with epicentre in Wuhan, China. Altogether 40554 cases have been confirmed globally with novel coronavirus (SARS-CoV-2) until February 10, 2020. Rigorous surveillance in other countries is required to prevent further global expansion of the outbreak, but resolving the exact mechanism of the initial transmission events is crucial. Most initial cases had visited Huanan South Seafood Market in Wuhan selling also various exotic live animals. Based on the limited initial human-to-human transmission and timely clustering of cases in Huanan market among elderly men, coupled with knowledge that coronaviruses are derived from animals and relationship of SARS-CoV-2 to bat coronavirus, zoonotic transmission in the first instance is probable. To target the actions, similar epidemiological actions to human cases are needed with animal or food exposures. According to current information, an exceptionally wide contamination of seafood market might explain the initiation of the SARS-CoV-2 outbreak. Seafood tanks, air contamination by live animals or rodents are possibilities, but sold animals normally come from various sources. The mode of transmission may become clearer in future: usually in outbreak investigations, hindsight is easy, but for now information about the initial source of this outbreak is limited.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential impact of seasonal forcing on a SARS-CoV-2 pandemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Fast vaccine design and development based on correlates of protection (COPs): Influenza as a trendsetter</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">New and reemerging infectious diseases call for innovative and efficient control strategies of which fast vaccine design and development represent an important element. In emergency situations, when time is limited, identification and use of correlates of protection (COPs) may play a key role as a strategic tool for accelerated vaccine design, testing, and licensure. We propose that general rules for COP-based vaccine design can be extracted from the existing knowledge of protective immune responses against a large spectrum of relevant viral and bacterial pathogens. Herein, we focus on the applicability of this approach by reviewing the established and up-coming COPs for influenza in the context of traditional and a wide array of new vaccine concepts. The lessons learnt from this field may be applied more generally to COP-based accelerated vaccine design for emerging infections.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus and the race to distribute reliable diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">International teams worked at speed to make tests for the virus available in record time. The medical community is rallying to develop a set of rapid and reliable molecular diagnostic tests for the new human coronavirus that appeared in China — now dubbed sudden acute respiratory syndrome coronavirus-2 (SARS-CoV-2).</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the {beta}-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Vaccine Candidates: Cautious Optimism</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 17</span></i><p style="text-align: justify; text-justify: inter-word;">Efforts towards developing a vaccine for Middle East respiratory syndrome coronavirus (MERS-CoV) have yielded promising results. Utilizing a variety of platforms, several vaccine approaches have shown efficacy in animal models and begun to enter clinical trials. In this review, we summarize the current progress towards a MERS-CoV vaccine and highlight potential roadblocks identified from previous attempts to generate coronavirus vaccines.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;">The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the scale of COVID-19 Epidemic in the United States: Simulations Based on Air Traffic directly from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Introduction: Coronavirus Disease 2019 (COVID-19) infection has been characterized by rapid spread and unusually large case clusters. It is important to have an estimate of the current state of COVID-19 epidemic in the U.S. to develop informed public health strategies.  Methods:  We estimated the potential scale of the COVID-19 epidemic (as of 03/01/2020) in the U.S. from cases imported directly from Wuhan area. We used simulations based on transmission dynamics parameters estimated from previous studies and air traffic data from Wuhan to the U.S and deliberately built our model based on conservative assumptions. Detection and quarantine of individual COVID-19 cases in the U.S before 03/01/2020 were also taken into account. We. A SEIR model was used to simulate the growth of the number of infected individuals in Wuhan area and in the U.S.  Results: With the most likely model, we estimated that there would be 9,484 infected cases (90%CI 2,054-24,241) as of 03/01/2020 if no intervention procedure had been taken to reduce the transmissibility in unidentified cases. Assuming current preventive procedures have reduced 25% of the transmissibility in unidentified cases, the number of infected cases would be 1,043 (90%CI 107-2,474). Conclusion: Our research indicates that, as of 03/01/2020., it is likely that there are already thousands of individuals in the US infected with SARS-CoV-2. Our model is dynamic and is available to the research community to further evaluate as the  situation becomes clearer.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number Ro -- the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of Ro vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of Ro across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate $r$, the mean generation interval $\bar G$, and the generation-interval dispersion $\kappa$. We then apply our framework to early estimates of Ro for the SARS-CoV-2 outbreak. We show that many early Ro estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of Ro, including the shape of the generation-interval distribution, in efforts to estimate Ro at the outset of an epidemic.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome: obstacles and prospects for vaccine development</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of Middle East respiratory syndrome (MERS) highlights the need to engineer new methods for expediting vaccine development against emerging diseases. However, several obstacles prevent pursuit of a licensable MERS vaccine. First, the lack of a suitable animal model for MERS complicates in vivo testing of candidate vaccines. Second, due to the low number of MERS cases, pharmaceutical companies have little incentive to pursue MERS vaccine production as the costs of clinical trials are high. In addition, the timeline from bench research to approved vaccine use is 10 years or longer. Using novel methods and cost-saving strategies, genetically engineered vaccines can be produced quickly and cost-effectively. Along with progress in MERS animal model development, these obstacles can be circumvented or at least mitigated.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Protective efficacy of a novel simian adenovirus vaccine against lethal MERS-CoV challenge in a transgenic human DPP4 mouse model</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Oct 16</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a novel zoonotic virus that causes severe respiratory disease in humans with a case fatality rate close to 40%, but for which no vaccines are available. Here, we evaluated the utility of ChAdOx1, a promising replication-deficient simian adenovirus vaccine vector platform with an established safety profile in humans and dromedary camels, for MERS-CoV vaccine development. Using a transgenic lethal BALB/c MERS-CoV mouse model we showed that single dose intranasal or intramuscular immunisation with ChAdOx1 MERS, encoding full-length MERS-CoV Spike glycoprotein, is highly immunogenic and confers protection against lethal viral challenge. Immunogenicity and efficacy were comparable between immunisation routes. Together these data provide support for further evaluation of ChAdOx1 MERS vaccine in humans and dromedary camels, the animal reservoir of infection.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-28</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the Corona Virus Disease 2019 (COVID-19) cases in China has become a public health emergency of international concern (PHEIC). Based on angiotensin converting enzyme 2 (ACE2) as cell entry receptor of SARS-CoV, we used the hACE2 transgenic mice infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant lymphocytes and monocytes in alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, alveolar macrophages and alveolar epithelia. The phenomenon was not found in wild type mice with SARS-CoV-2 infection. The pathogenicity of SARS-CoV-2 in hACE2 mice was clarified and the Kochs postulates were fulfilled as well, and the mouse model may facilitate the development of therapeutics and vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Toward Developing a Preventive MERS-CoV Vaccine—Report from a Workshop Organized by the Saudi Arabia Ministry of Health and the International Vaccine Institute, Riyadh, Saudi Arabia, November 14–15, 2015</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) remains a serious international public health threat. With the goal of accelerating the development of countermeasures against MERS coronavirus (MERS-CoV), funding agencies, nongovernmental organizations, and researchers across the world assembled in Riyadh, Saudi Arabia, on November 14–15, 2015, to discuss vaccine development challenges. The meeting was spearheaded by the Saudi Ministry of Health and co-organized by the International Vaccine Institute, South Korea. Accelerating the development of a preventive vaccine requires a better understanding of MERS epidemiology, transmission, and pathogenesis in humans and animals. A combination of rodent and nonhuman primate models should be considered in evaluating and developing preventive and therapeutic vaccine candidates. Dromedary camels should be considered for the development of veterinary vaccines. Several vaccine technology platforms targeting the MERS-CoV spike protein were discussed. Mechanisms to maximize investment, provide robust data, and affect public health are urgently needed.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.</jats:p></p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>A Live Attenuated Severe Acute Respiratory Syndrome Coronavirus Is Immunogenic and Efficacious in Golden Syrian Hamsters</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The immunogenicity and protective efficacy of a live attenuated vaccine consisting of a recombinant severe acute respiratory syndrome (SARS) coronavirus lacking the E gene (rSARS-CoV-ΔE) were studied using hamsters. Hamsters immunized with rSARS-CoV-ΔE developed high serum-neutralizing antibody titers and were protected from replication of homologous (SARS-CoV Urbani) and heterologous (GD03) SARS-CoV in the upper and lower respiratory tract. rSARS-CoV-ΔE-immunized hamsters remained active following wild-type virus challenge, while mock-immunized hamsters displayed decreased activity. Despite being attenuated in replication in the respiratory tract, rSARS-CoV-ΔE is an immunogenic and efficacious vaccine in hamsters.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Recent Advances in the Vaccine Development Against Middle East Respiratory Syndrome-Coronavirus</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 2</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) is a deadly viral respiratory disease caused by MERS-coronavirus (MERS-CoV) infection. To date, there is no specific treatment proven effective against this viral disease. In addition, no vaccine has been licensed to prevent MERS-CoV infection thus far. Therefore, our current review focuses on the most recent studies in search of an effective MERS vaccine. Overall, vaccine candidates against MERS-CoV are mainly based upon the viral spike (S) protein, due to its vital role in the viral infectivity, although several studies focused on other viral proteins such as the nucleocapsid (N) protein, envelope (E) protein, and non-structural protein 16 (NSP16) have also been reported. In general, the potential vaccine candidates can be classified into six types: viral vector-based vaccine, DNA vaccine, subunit vaccine, nanoparticle-based vaccine, inactivated-whole virus vaccine and live-attenuated vaccine, which are discussed in detail. Besides, the immune responses and potential antibody dependent enhancement of MERS-CoV infection are extensively reviewed. In addition, animal models used to study MERS-CoV and evaluate the vaccine candidates are discussed intensively.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Antigenicity and immunogenicity of SARS-CoV S protein receptor-binding domain stably expressed in CHO cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The receptor-binding domain (RBD) of SARS coronavirus (SARS-CoV) spike (S) protein contains multiple conformation-dependent epitopes that induce neutralizing antibody responses. Here we used CHO-K1 cells to establish a cell line for stable expression of a 193-mer (residues 318–510) RBD (RBD193-CHO) and determined its antigenicity and immunogenicity. We found that RBD193-CHO reacted strongly with a panel of six monoclonal antibodies recognizing various conformational and linear epitopes in RBD, suggesting that this recombinant protein maintains intact conformation and good antigenicity. Immunization of mice with RBD193-CHO resulted in induction of high titers of RBD-specific neutralizing antibodies and potent IL-4-expressing T cell responses. RBD193-CHO induced immunity that protected a majority of the vaccinated mice from SARS-CoV challenge. These results suggest that the recombinant RBD produced in an established stable cell line maintains strong immunogenicity with high potential for use as an effective and economic subunit SARS vaccine.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">hina continues to fight the outbreak of a novel coronavirus, called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which has infected tens of thousands and killed more than 2,100 people since the end of 2019. The outbreak’s hot spot is in the central province of Hubei, in particular, its capital city Wuhan, where there have been more than 62,000 infections and 2,029 deaths as of last week, according to the World Health Organization (WHO). There, doctors and scientists are desperately struggling to find reliable ways to diagnose infected patients to help treat them and control the spread of the virus. The struggle has involved two diagnostic options—computed tomography (CT) scans of patients’ lungs and a nucleic acid lab test—each with advantages and disadvantages. With “a suddenly rampant new virus, it is normal to have multiple test approaches,” says a virologist at the Chinese Center for Disease Control and Prevention</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country.  Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival &amp; traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% &amp; 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.</jats:p></p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background and Objective: To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. Design and Methods: We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. Results: The peak time of the curve for the infected newly diagnosed as COVID-19 should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei. It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese COVID-19 epidemic can be completely controlled in May,2020. Conclusions: COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global SARS-CoV-2 cases rise with the great efforts by Chinese government and its people.</jats:p></p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Sanofi, BARDA team for coronavirus vaccine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Sanofi Pasteur is working with the US Biomedical Advanced Research and Development Authority (BARDA) to develop a vaccine against the novel coronavirus, known as severe acute respiratory syndrome-CoV-2, which as of Feb. 20 had infected more than 75,000 people and killed more than 2,100. Sanofi will build on work on a vaccine against SARS, another coronavirus that circulated in the early 2000s. In December 2019, BARDA awarded the firm a $226 million contract to expand manufacturing capacity for use in a potential flu pandemic. Sanofi says the new vaccine will use the same technology.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Generation of antibodies against COVID-19 virus for development of diagnostic tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Targeted Strategies for Henipavirus Therapeutics</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Sep 28</span></i><p style="text-align: justify; text-justify: inter-word;">Hendra and Nipah viruses are related emergent paramyxoviruses that infect and cause disease in animals and humans. Disease manifests as a generalized vasculitis affecting multiple organs, but is the most severe in the respiratory and central nervous systems. The high case fatality and person-to-person transmission associated with the most recent NiV outbreaks, and the recent re-emergence of HeV, emphasize the importance and necessity of effective therapeutics for these novel agents. In recent years henipavirus research has revealed a more complete understanding of pathogenesis and, as a consequence, viable approaches towards vaccines and therapeutics have emerged. All strategies target early steps in viral replication including receptor binding and membrane fusion. Animal models have been developed, some of which may prove more valuable than others for evaluating the efficacy of therapeutic agents and regimes. Assessments of protective host immunity and drug pharmacokinetics will be crucial to the further advancement of therapeutic compounds.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">What is COVID-19, and what do we know so far about its clinical presentation? The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1]. The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host [2]. Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Kallikrein 13: a new player in coronaviral infections.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus HKU1 (HCoV-HKU1) is associated with respiratory disease and is prevalent worldwide, but in vitro model for virus replication is lacking. Interaction between the coronaviral spike (S) protein and its receptor is the major determinant of virus tissue and host specificity, but virus entry is a complex process requiring a concerted action of multiple cellular elements. Here, we show that KLK13 is required for the infection of the human respiratory epithelium and is sufficient to mediate the entry of HCoV-HKU1 to non-permissive RD cells. We also demonstrated HCoV-HKU1 S protein cleavage by KLK13 in the S1/S2 region, proving that KLK13 is the priming enzyme for this virus. Summarizing, we show for the first time that protease distribution and specificity predetermines the tissue and cell specificity of the virus and may also regulate interspecies transmission. It is also of importance that presented data may be relevant for the emerging coronaviruses, including SARS-CoV-2 and may help to understand the differences in their zoonotic potential.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding domain of MERS-CoV with optimal immunogen dosage and immunization interval protects human transgenic mice from MERS-CoV infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) continues to raise worldwide concerns due to its pandemic potential. Increased MERS cases and no licensed MERS vaccines highlight the need to develop safe and effective vaccines against MERS. We have previously demonstrated that a receptor-binding domain (RBD) fragment containing residues 377–588 of MERS-coronavirus (MERS-CoV) spike protein is a critical neutralizing domain and an important vaccine target. Nevertheless, its optimal immunogen dosage and immunization interval, key factors for human-used vaccines that induce protective immunity, have never been investigated. In this study, we optimized these criteria using a recombinant MERS-CoV RBD protein fused with Fc (S377–588-Fc) and utilized the optimal immunization schedule to evaluate the protective efficacy of RBD against MERS-CoV infection in human dipeptidyl peptidase 4 transgenic (hDPP4-Tg) mice. Compared with one dose and 2 doses at 1-, 2-, and 3-week intervals, a regimen of 2 doses of this protein separated by an interval of 4 weeks induced the strongest antibody response and neutralizing antibodies against MERS-CoV infection, and maintained at a high level during the detection period. Notably, RBD protein at the optimal dosage and interval protected hDPP4-Tg mice against lethal MERS-CoV challenge, and the protection was positively correlated with serum neutralizing antibodies. Taken together, the optimal immunogen dosage and immunization interval identified in this study will provide useful guidelines for further development of MERS-CoV RBD-based vaccines for human use.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;">Severe Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 - a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Tailoring Subunit Vaccine Immunity with Adjuvant Combinations and Delivery Routes Using the Middle East Respiratory Coronavirus (MERS-CoV) Receptor-Binding Domain as an Antigen</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 18</span></i><p style="text-align: justify; text-justify: inter-word;">The development of an effective vaccine is critical for prevention of a Middle East respiratory syndrome coronavirus (MERS-CoV) pandemic. Some studies have indicated the receptor-binding domain (RBD) protein of MERS-CoV spike (S) is a good candidate antigen for a MERS-CoV subunit vaccine. However, highly purified proteins are typically not inherently immunogenic. We hypothesised that humoral and cell-mediated immunity would be improved with a modification of the vaccination regimen. Therefore, the immunogenicity of a novel MERS-CoV RBD-based subunit vaccine was tested in mice using different adjuvant formulations and delivery routes. Different vaccination regimens were compared in BALB/c mice immunized 3 times intramuscularly (i.m.) with a vaccine containing 10 µg of recombinant MERS-CoV RBD in combination with either aluminium hydroxide (alum) alone, alum and polyriboinosinic acid (poly I:C) or alum and cysteine-phosphate-guanine (CpG) oligodeoxynucleotides (ODN). The immune responses of mice vaccinated with RBD, incomplete Freund’s adjuvant (IFA) and CpG ODN by a subcutaneous (s.c.) route were also investigated. We evaluated the induction of RBD-specific humoral immunity (total IgG and neutralizing antibodies) and cellular immunity (ELISpot assay for IFN-γ spot-forming cells and splenocyte cytokine production). Our findings indicated that the combination of alum and CpG ODN optimized the development of RBD-specific humoral and cellular immunity following subunit vaccination. Interestingly, robust RBD-specific antibody and T-cell responses were induced in mice immunized with the rRBD protein in combination with IFA and CpG ODN, but low level of neutralizing antibodies were elicited. Our data suggest that murine immunity following subunit vaccination can be tailored using adjuvant combinations and delivery routes. The vaccination regimen used in this study is promising and could improve the protection offered by the MERS-CoV subunit vaccine by eliciting effective humoral and cellular immune responses.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative Pathogenesis of Three Human and Zoonotic SARS-CoV Strains in Cynomolgus Macaques</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 20</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome (SARS) epidemic was characterized by increased pathogenicity in the elderly due to an early exacerbated innate host response. SARS-CoV is a zoonotic pathogen that entered the human population through an intermediate host like the palm civet. To prevent future introductions of zoonotic SARS-CoV strains and subsequent transmission into the human population, heterologous disease models are needed to test the efficacy of vaccines and therapeutics against both late human and zoonotic isolates. Here we show that both human and zoonotic SARS-CoV strains can infect cynomolgus macaques and resulted in radiological as well as histopathological changes similar to those seen in mild human cases. Viral replication was higher in animals infected with a late human phase isolate compared to a zoonotic isolate. While there were significant differences in the number of host genes differentially regulated during the host responses between the three SARS-CoV strains, the top pathways and functions were similar and only apparent early during infection with the majority of genes associated with interferon signaling pathways. This study characterizes critical disease models in the evaluation and licensure of therapeutic strategies against SARS-CoV for human use.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Aerosol and surface stability of HCoV-19 (SARS-CoV-2) compared to SARS-CoV-1</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel human coronavirus, now named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, referred to as HCoV-19 here) that emerged in Wuhan, China in late 2019 is now causing a pandemic.  Here, we analyze the aerosol and surface stability of HCoV-19 and compare it with SARS-CoV-1, the most closely related human coronavirus.2 We evaluated the stability of HCoV-19 and SARS-CoV-1 in aerosols and on different surfaces and estimated their decay rates using a Bayesian regression model</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Angiotensin receptor blockers as tentative SARS-CoV-2 therapeutics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">At the time of writing this commentary (February 2020), the coronavirus COVID-19 epidemic has already resulted in more fatalities compared with the SARS and MERS coronavirus epidemics combined. Therapeutics that may assist to contain its rapid spread and reduce its high mortality rates are urgently needed. Developing vaccines against the SARS-CoV-2 virus may take many months. Moreover, vaccines based on viral-encoded peptides may not be effective against future coronavirus epidemics, as virus mutations could make them futile. Indeed, new Influenza virus strains emerge every year, requiring new immunizations. A tentative suggestion based on existing therapeutics, which would likely be resistant to new coronavirus mutations, is to use available angiotensin receptor 1 (AT1R) blockers, such as losartan, as therapeutics for reducing the aggressiveness and mortality from SARS-CoV-2 virus infections. This idea is based on observations that the angiotensin-converting enzyme 2 (ACE2) very likely serves as the binding site for SARS-CoV-2, the strain implicated in the current COVID-19 epidemic, similarly to strain SARS-CoV implicated in the 2002-2003 SARS epidemic. This commentary elaborates on the idea of considering AT1R blockers as tentative treatment for SARS-CoV-2 infections, and proposes a research direction based on datamining of clinical patient records for assessing its feasibility.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2.  Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative Pathogenesis and Systems Biology for  Biodefense Virus Vaccine Development</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Jun 6</span></i><p style="text-align: justify; text-justify: inter-word;">Developing vaccines to biothreat agents presents a number of challenges for discovery, preclinical development, and licensure. The need for high containment to work with live agents limits the amount and types of research that can be done using complete pathogens, and small markets reduce potential returns for industry. However, a number of tools, from comparative pathogenesis of viral strains at the molecular level to novel computational approaches, are being used to understand the basis of viral attenuation and characterize protective immune responses. As the amount of basic molecular knowledge grows, we will be able to take advantage of these tools not only to rationally attenuate virus strains for candidate vaccines, but also to assess immunogenicity and safety in silico. This review discusses how a basic understanding of pathogenesis, allied with systems biology and machine learning methods, can impact biodefense vaccinology.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Four ways researchers are responding to the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts. How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>A 219-mer CHO-Expressing Receptor-Binding Domain of SARS-CoV S Protein Induces Potent Immune Responses and Protective Immunity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Development of vaccines is essential for the prevention of future recurrences of severe acute respiratory syndrome (SARS), caused by the SARS coronavirus (SARS-CoV). The spike (S) protein, especially receptor-binding domain (RBD) of SARS-CoV, plays important roles in the prevention of SARS infection, and is thus an important component in SARS vaccine development. In this study, we expressed a 219-mer (residues 318–536) RBD protein in Chinese hamster ovary (CHO)-K1 cells (RBD219-CHO), and tested its immune responses and protective immunity in a mouse model. The results showed that this recombinant protein was correctly folded, being able to maintain intact conformation and authentic antigenicity. It could induce strong humoral and cellular immune responses and high titers of neutralizing antibodies in the vaccinated mice. RBD219-CHO protein elicited potent protective immunity that protected all vaccinated mice from SARS-CoV challenge. These results suggest that the recombinant RBD219-CHO protein has great potential for the development of an effective and safe SARS subunit vaccine.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 receptor ACE2 expression of maternal-fetal interface and fetalorgans by single cell transcriptome study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">Recent studies have demonstrated that SARS-CoV-2 cell entry depends on both ACE2 and TMPRSS2 genes (DOI:10.1016/j.cell.2020.02.052), but our current work only focus on ACE2, which is insufficient to support the conclusion of this paper. So the authors have withdrawn their manuscript whilst they perform additional experiments and analysis to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19 A mathematical model for simulating the phase-based transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125?nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is >?96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1].ID - Fisher2020 As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunization with SARS Coronavirus Vaccines Leads to Pulmonary Immunopathology on Challenge with the SARS Virus</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Apr 20</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Severe acute respiratory syndrome (SARS) emerged in China in 2002 and spread to other countries before brought under control. Because of a concern for reemergence or a deliberate release of the SARS coronavirus, vaccine development was initiated. Evaluations of an inactivated whole virus vaccine in ferrets and nonhuman primates and a virus-like-particle vaccine in mice induced protection against infection but challenged animals exhibited an immunopathologic-type lung disease. DESIGN: Four candidate vaccines for humans with or without alum adjuvant were evaluated in a mouse model of SARS, a VLP vaccine, the vaccine given to ferrets and NHP, another whole virus vaccine and an rDNA-produced S protein. Balb/c or C57BL/6 mice were vaccinated IM on day 0 and 28 and sacrificed for serum antibody measurements or challenged with live virus on day 56. On day 58, challenged mice were sacrificed and lungs obtained for virus and histopathology. RESULTS: All vaccines induced serum neutralizing antibody with increasing dosages and/or alum significantly increasing responses. Significant reductions of SARS-CoV two days after challenge was seen for all vaccines and prior live SARS-CoV. All mice exhibited histopathologic changes in lungs two days after challenge including all animals vaccinated (Balb/C and C57BL/6) or given live virus, influenza vaccine, or PBS suggesting infection occurred in all. Histopathology seen in animals given one of the SARS-CoV vaccines was uniformly a Th2-type immunopathology with prominent eosinophil infiltration, confirmed with special eosinophil stains. The pathologic changes seen in all control groups lacked the eosinophil prominence. CONCLUSIONS: These SARS-CoV vaccines all induced antibody and protection against infection with SARS-CoV. However, challenge of mice given any of the vaccines led to occurrence of Th2-type immunopathology suggesting hypersensitivity to SARS-CoV components was induced. Caution in proceeding to application of a SARS-CoV vaccine in humans is indicated.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 sensitive to type I interferon pretreatment.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B coronaviruses share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel coronavirus is much more sensitive to type I interferon pretreatment. We subsequently examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN response between SARS-CoV and SARS-CoV-2 that could help inform disease progression, treatment options, and animal model development.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>A highly conserved WDYPKCDRA epitope in the RNA directed RNA polymerase of human coronaviruses can be used as epitope-based universal vaccine design</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 May 29</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronaviruses are the diverse group of RNA virus. From 1960, six strains of human coronaviruses have emerged that includes SARS-CoV and the recent infection by deadly MERS-CoV which is now going to cause another outbreak. Prevention of these viruses is urgent and a universal vaccine for all strain could be a promising solution in this circumstance. In this study we aimed to design an epitope based vaccine against all strain of human coronavirus. RESULTS: Multiple sequence alignment (MSA) approach was employed among spike (S), membrane (M), enveloped (E) and nucleocapsid (N) protein and replicase polyprotein 1ab to identify which one is highly conserve in all coronaviruses strains. Next, we use various in silico tools to predict consensus immunogenic and conserved peptide. We found that conserved region is present only in the RNA directed RNA polymerase protein. In this protein we identified one epitope WDYPKCDRA is highly immunogenic and 100% conserved among all available human coronavirus strains. CONCLUSIONS: Here we suggest in vivo study of our identified novel peptide antigen in RNA directed RNA polymerase protein for universal vaccine – which may be the way to prevent all human coronavirus disease.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Optimization of antigen dose for a receptor-binding domain-based subunit vaccine against MERS coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) is an emerging infectious disease caused by MERS coronavirus (MERS-CoV). The continuous increase of MERS cases has posed a serious threat to public health worldwide, calling for development of safe and effective MERS vaccines. We have previously shown that a recombinant protein containing residues 377–588 of MERS-CoV receptor-binding domain (RBD) fused with human Fc (S377-588-Fc) induced highly potent anti-MERS-CoV neutralizing antibodies in the presence of MF59 adjuvant. Here we optimized the doses of S377-588-Fc using MF59 as an adjuvant in order to elicit strong immune responses with minimal amount of antigen. Our results showed that S377-588-Fc at 1 μg was able to induce in the immunized mice potent humoral and cellular immune responses. Particularly, S377-588-Fc at 1 μg elicited strong neutralizing antibody responses against both pseudotyped and live MERS-CoV similar to those induced at 5 and 20 μg, respectively. These results suggest that this RBD-based subunit MERS vaccine candidate at the dose as low as one μg is sufficiently potent to induce strong humoral and cellular immune responses, including neutralizing antibodies, against MERS-CoV infection, thus providing guidance for determining the optimal dosage of RBD-based MERS vaccines in the future clinical trials and for applying the dose-sparing strategy in other subunit vaccine trials.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-01</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited.  METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG).  CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be greatly relieved.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the  2019-nCoV_N2, N3 of USA and the  ORF1ab of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Chikungunya as a paradigm for emerging viral diseases: Evaluating disease impact and hurdles to vaccine development</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 17</span></i><p style="text-align: justify; text-justify: inter-word;">Chikungunya fever (CHIKF) is an emerging infectious disease caused by an alphavirus transmitted by Aedes spp. mosquitoes. Because mosquito control programs are not highly efficient for outbreak containment, vaccines are essential to reduce the burden of disease. Although no licensed vaccine against CHIKF is yet available, many highly promising candidates are undergoing preclinical studies, and a few of them have been tested in human trials of phase 1 or 2. Here, we review recent findings regarding the need for a CHIKF vaccine and provide an update on vaccines nearing or having entered clinical trials. We also address needs to tackle bottlenecks to vaccine development—including scientific and financial barriers—and to accelerate the development of vaccines; several actions should be taken: (i) design efficacy trials to be conducted during the course of outbreaks; (ii) evaluate the opportunity for adopting the “animal rule”for demonstration of efficacy for regulatory purposes; (iii) strengthen the collective commitment of nations, international organizations, potential donors and industry; (iv) stimulate public and/or private partnerships to invest in vaccine development and licensure; and (v) identify potential markets for an effective and safe CHIKF vaccine.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Antiviral drugs for managing infections with human coronaviruses are not yet approved, posing a serious challenge to current global efforts aimed at containing the outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Remdesivir (RDV) is an investigational compound with a broad spectrum of antiviral activities against RNA viruses, including SARS-CoV and Middle East respiratory syndrome (MERS-CoV). RDV is a nucleotide analog inhibitor of RNA-dependent RNA polymerases (RdRps). Here, we co-expressed the MERS-CoV nonstructural proteins nsp5, nsp7, nsp8, and nsp12 (RdRp) in insect cells as a part a polyprotein to study the mechanism of inhibition of MERS-CoV RdRp by RDV. We initially demonstrated that nsp8 and nsp12 form an active complex. The triphosphate form of the inhibitor (RDV-TP) competes with its natural counterpart ATP. Of note, the selectivity value for RDV-TP obtained here with a steady-state approach suggests that it is more efficiently incorporated than ATP and two other nucleotide analogues. Once incorporated at position i, the inhibitor caused RNA synthesis arrest at position i+3. Hence, the likely mechanism of action is delayed RNA chain termination. The additional three nucleotides may protect the inhibitor from excision by the viral 3'-5' exonuclease activity. Together, these results help to explain the high potency of RDV against RNA viruses in cell-based assays.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the cross-species evolution of 2019 novel coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Recent study reported in this journal that the threats of continuous evolution and dissemination of 2019 human coronaviruses.1 Since its emergence in December 2019, a “seventh” member of the family of 2019 human coronavirus named “SARS-CoV-2” was responsible for an outbreak of coronavirus disease (COVID-19) in Wuhan, China.2 As of February 23, 2020, China had reported more than 77,042 confirmed cases of SARS-CoV-2, with 2,445 fatalities and counting (http://www.nhc.gov.cn). Strikingly, SARS-CoV-2 had been transmitted rapidly in more than 29 countries to date (https://www.who.int), including Asia, Europe, North America, Africa, and Oceania, posing serious concerns about its pandemic potential. Despite of droplet and contact transmissions of SARS-CoV-2, recent studies demonstrated that SARS-CoV-2 might be transmitted via aerosol and fecal–oral routes 3 (Figure 1), which needs to be paid attention in particular.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid establishment of laboratory diagnostics for the novel coronavirus SARS-CoV-2 in Bavaria, Germany, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In connection with the ongoing outbreak of a novel coronavirus in the province Hubei and surrounding areas in China, it was expected that Europe would also be confronted with the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), as infections in travellers in several Asian countries outside of China were confirmed shortly after the announcement of the outbreak in Wuhan [1-4]. Therefore, it was necessary to rapidly implement adequately quick and sensitive diagnostic assays for outbreak management of SARS-CoV-2 in public health laboratories. As soon as the World Health Organization (WHO) published the first protocols for real-time RT-PCR assays, the Bavarian Food and Health Authority started to implement them. We ordered control material and oligonucleotides (see details below) in week 4 and ran our first SARS-CoV-2 assays on 27 January (week 5). On the same day, the first German case of coronavirus disease 2019 (COVID-19) was diagnosed in Bavaria [1]. In the following days, health authorities implemented comprehensive measures to prevent further transmission of SARS-CoV-2, including testing of contact persons. We initially used the protocol based on the E gene and RdRp gene developed by the German Consiliary Laboratory for Coronaviruses hosted at the Charité in Berlin [5]. Real-time RT-PCR was initially performed with the QuantiTect Virus +Rox Vial kit (QIAGEN, Hilden, Germany) on the Bio-Rad CFX96 Touch Real-Time PCR Detection System (Bio-Rad, Feldkirchen, Germany). The kit was chosen for its frequent and successful use in our laboratory with other assays and its immediate availability. Primers and probes were used as described [5] and provided by TIB Molbiol (Berlin, Germany). Control material was ordered from the European Virus Archive (EVAg) and consisted of synthetic Wuhan coronavirus 2019 E gene control (reference number 026N-03866) and SARS-CoV Frankfurt 1 RNA (reference number 004N-02005) [6]. In addition, the control of LightMix Modular Wuhan CoV RdRP-gene (TibMolbiol, Berlin, Germany) was used for the SARS-CoV-2 specific assay. Respiratory samples (nasopharyngeal swabs or sputum) were obtained from patients and contact persons. Sputum samples were diluted in 2 mL phosphate buffered saline (PBS). RNA was extracted using the QIAamp Bio Robot kit (QIAGEN) on a Hamilton Microlab Star (Hamilton, Bonaduz, Switzerland). As at 1 March, 87,024 cases and 2,979 associated deaths have been reported worldwide [1]. The vast majority of the deaths (96%) have been reported in China [1]. Despite the high number of cases reported globally, estimates of the severity pyramid of disease and case fatality rate remain very uncertain; one large study conducted in China estimated that the majority (81%) of the cases were mild (i.e. non-pneumonia or mild pneumonia), 14% were severe (e.g. with dyspnoea) and 5% were in a critical condition (i.e. respiratory failure, septic shock and/or multiple organ dysfunction/failure) [2]. The case fatality ratio was 2.3% [2]. Despite extraordinary containment measures implemented in China, including the enforced lockdown of several cities and closures of schools, the virus has spread throughout the country and internationally [2]. It is too early to predict with any certainty the epidemiological developments over the coming weeks, but the possibility of widespread community transmission becoming established throughout the EU/EEA is becoming increasingly likely.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical manifestations and outcome of SARS-CoV-2 infection during pregnancy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Tang and colleagues, in this Journal, drew readers attention to emerging COVID19.1 We focused on the pregnant COVID19 patients. Given the maternal physiologic and immune function changes in pregnancy,2 pregnant individuals might face greater risk of getting infected by SARS-CoV-2 and might have more complicated clinical events. We described epidemiological, clinical characteristics, pregnancy and perinatal outcomes of all hospitalized pregnant patients diagnosed with COVID-19 in China. We identified all hospitalized pregnant patients with laboratory-confirmed SARS-CoV-2 infection between December 8, 2019, and February 25, 2020 officially reported by the central government, in areas outside Wuhan, China. Information including age, geographic location, epidemiological history, prenatal course, maternal and newborn hospital course, discharge data and outcome were obtained by Centers for Disease Control and Prevention and Local Health Commission. When necessary, we attempted to contact local hospital or patients by telephone to supply missing information. This investigation was part of an emergency public health outbreak investigation and therefore not subject to institutional review board. There were a total of 13 Chinese patients with SARS-CoV-2 admitted to hospitals outside of Wuhan (Table 1). There were 3 patients from Zhejiang, 3 from other cities of Hubei and 1 each from Fujian, Shanxi, Beijing, Guangdong, Jiangxi, Heilongjiang and Anhui. The maternal age ranged between 22 to 36 years. Two women were less than 28 weeks of gestation and the other 11 patients were in their third trimesters at presentation. None of the patients had underlying medical disease.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 has caused tens of thousands of infections and more than one thousand deaths. There are currently no registered therapies for treating coronavirus infections. Because of time consuming process of new drug development, drug repositioning may be the only solution to the epidemic of sudden infectious diseases. We systematically analyzed all the proteins encoded by SARS-CoV-2 genes, compared them with proteins from other coronaviruses, predicted their structures, and built 19 structures that could be done by homology modeling. By performing target-based virtual ligand screening, a total of 21 targets (including two human targets) were screened against compound libraries including ZINC drug database and our own database of natural products. Structure and screening results of important targets such as 3-chymotrypsin-like protease (3CLpro), Spike, RNA-dependent RNA polymerase (RdRp), and papain like protease (PLpro) were discussed in detail. In addition, a database of 78 commonly used anti-viral drugs including those currently on the market and undergoing clinical trials for SARS-CoV-2 was constructed. Possible targets of these compounds and potential drugs acting on a certain target were predicted. This study will provide new lead compounds and targets for further in vitro and in vivo studies of SARS-CoV-2, new insights for those drugs currently ongoing clinical studies, and also possible new strategies for drug repositioning to treat SARS-CoV-2 infections.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>The Adjuvanticity of an O. volvulus-Derived rOv-ASP-1 Protein in Mice Using Sequential Vaccinations and in Non-Human Primates</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 May 17</span></i><p style="text-align: justify; text-justify: inter-word;">Adjuvants potentiate antigen-specific protective immune responses and can be key elements promoting vaccine effectiveness. We previously reported that the Onchocerca volvulus recombinant protein rOv-ASP-1 can induce activation and maturation of naïve human DCs and therefore could be used as an innate adjuvant to promote balanced Th1 and Th2 responses to bystander vaccine antigens in mice. With a few vaccine antigens, it also promoted a Th1-biased response based on pronounced induction of Th1-associated IgG2a and IgG2b antibody responses and the upregulated production of Th1 cytokines, including IL-2, IFN-γ, TNF-α and IL-6. However, because it is a protein, the rOv-ASP-1 adjuvant may also induce anti-self-antibodies. Therefore, it was important to verify that the host responses to self will not affect the adjuvanticity of rOv-ASP-1 when it is used in subsequent vaccinations with the same or different vaccine antigens. In this study, we have established rOv-ASP-1's adjuvanticity in mice during the course of two sequential vaccinations using two vaccine model systems: the receptor-binding domain (RBD) of SARS-CoV spike protein and a commercial influenza virus hemagglutinin (HA) vaccine comprised of three virus strains. Moreover, the adjuvanticity of rOv-ASP-1 was retained with an efficacy similar to that obtained when it was used for a first vaccination, even though a high level of anti-rOv-ASP-1 antibodies was present in the sera of mice before the administration of the second vaccine. To further demonstrate its utility as an adjuvant for human use, we also immunized non-human primates (NHPs) with RBD plus rOv-ASP-1 and showed that rOv-ASP-1 could induce high titres of functional and protective anti-RBD antibody responses in NHPs. Notably, the rOv-ASP-1 adjuvant did not induce high titer antibodies against self in NHPs. Thus, the present study provided a sound scientific foundation for future strategies in the development of this novel protein adjuvant.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in Chinese Han population is limited. Our objective is to describe the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in the Chinese Han population. In this case series of 752 patients, the full spectrum of cases is described. Fever was present in 86-90% of the patients. The second most common symptom was cough (49.1-51.0%), fatigue (25.2-27.1%), sputum (20.0-23.1%), and headache (9.8-11.1%). the mortality rate is 4.6% in Wuhan, 1.9% in Beijing, and 0.9% in Shanghai. Our findings showed that the levels of lymphocytes were 0.8（IQR, 0.6-1.1）109/L in Wuhan, 1.0（IQR, 0.7-1.4）109/L in Beijing, and 1.1 (IQR, 0.8-1.5) 109/L in Shanghai before admission to hospitals, respectively, indicating that cellular immune function might relate to the mortality. Based on the reference ranges of normal Chinese Han population and the data of the critically ill patients we have observed, it is recommended that reference ranges of people at high risk of COVID-19 infection are CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500 cells/mm3, and CD8+ lymphocytes below 300 cells/mm3.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Duration of quarantine in hospitalized patients with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection: a question needing an answer</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019 a new form of pneumonia was observed in the Chinese province of Hubei.[1] The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was subsequently identified as responsible of this condition, defined coronavirus disease (COVID-19).[2] The virus has now spread outside Chinese borders with 82,297 cases and 2,804 deaths worldwide at the 26th of February.[3] After infection, symptoms appear after an incubation time of 3-5 days, with 80% of those infected developing a mild disease, 15% a severe disease and 5% will require support in intensive care unit (ICU).[4] Overall, the estimated case-fatality rate is comprised between 0.4% and 2.9% and the basic reproduction number is approximately 3.28.[4,5] SARS-CoV-2 is a new pathogen for humankind and any type of immune protection exist, thus everybody can be potentially infected. Moreover, no primary prophylaxis measures (vaccination) nor effective treatments are available. If the above represented percentages are applied to the worldwide populations, it appears clear why any measure should be considered to avoid a further diffusion of the virus and prevent the saturation and collapse of health systems and the most catastrophic pandemic since 1919 Spanish flu. Isolation of those affected and the use of personal protective equipment (PPE) are the mainstay to block transmission of this pathogen, which is presumed through respiratory droplets. A 14 days quarantine is applied to subjects coming from endemic areas or who had contact with confirmed cases. It is assumed that, if in this period the subject does not develop any sign or symptoms compatible with COVID-19, he is not infected and thus the quarantine can be removed, and the subject returned to the community. Domiciliary quarantine of 14 days since a positive test is applied also for patients with a diagnosed mild disease who did not need medical support. These rules are effective in controlling infections in the community, but several doubts arise when it is necessary to transpose them in the hospital setting. Hospitals are indeed a delicate place in epidemics: they collect fragile persons who can be exposed to the virus and are subsequently readmitted to the community thus spreading the infection. Indeed, the ongoing outbreak in Northern Italy has been linked to a single infected patient who accessed to a community hospital where he transmitted the virus to several other patients and health-care operators.[6] Moreover, the isolation of patients in the hospital setting impose a significant burden in terms of PPE used by the health-care operators, space dedicated and time employed in their management. Even more complex is the situation of patients in ICU, where viral spreading is facilitated by endotracheal tubes and manoeuvres performed on the respiratory tract. Therefore, a clear definition of the infectiousness timing and intensity of viral spreading is mandatory to alleviate the burden on the health-care system. Unfortunately, the data available on the topic are scarce and composed only of measurements of viral shedding, without an assessment of the infectivity. Kim et al.[7] assessed the viral load kinetics of SARS-CoV-2 in upper and lower respiratory tract materials in the first two confirmed patients in Korea. They employed real-time reverse transcriptase polymerase chain reaction (rRT-PCR) to detect SARS-CoV-2 and converted cycle threshold (CT) values of rRT-PCR into RNA copy number. The detection limit of rRT-PCR was 2,690 copies/mL. Overall, viral load above detection limit was detected until 14 and 25 days after symptoms onset and for 13 and 11 days after the first detection, respectively.[7] Of note, both patients received treatment with lopinavir/ritonavir. Instead, Zou and colleagues analysed viral load in repeated nasal and throat swabs obtained from the 17 symptomatic patients.[8] They also employed rRT-PCR and considered a CT of 40 as detection limit. Higher viral loads were observed in nasal </p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Potent and persistent antibody responses against the receptor-binding domain of SARS-CoV spike protein in recovered patients</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Nov 4</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The spike (S) protein of SARS-CoV not only mediates receptor-binding but also induces neutralizing antibodies. We previously identified the receptor-binding domain (RBD) of S protein as a major target of neutralizing antibodies in animal models and thus proposed a RBD-based vaccine. However, the antigenicity and immunogenicity of RBD in humans need to be characterized. RESULTS: Two panels of serum samples from recovered SARS patients were included and the antibody responses against the RBD were measured by ELISA and micro-neutralization assays. We found that the RBD of S protein induced potent antibody responses in the recovered SARS patients and RBD-specific antibodies could persist at high titers over three year follow-up. Furthermore, affinity purified anti-RBD antibodies possessed robust neutralizing activity. CONCLUSION: The RBD of SARS-CoV is highly immunogenic in humans and mediates protective responses and RBD-based vaccines and diagnostic approaches can be further developed.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors' CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">The 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within COVID-19 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Systems Vaccinology Identifies an Early Innate Immune Signature as a Correlate of Antibody Responses to the Ebola Vaccine rVSV-ZEBOV</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 29</span></i><p style="text-align: justify; text-justify: inter-word;">Predicting vaccine efficacy remains a challenge. We used a systems vaccinology approach to identify early innate immune correlates of antibody induction in humans receiving the Ebola vaccine rVSV-ZEBOV. Blood samples from days 0, 1, 3, 7, and 14 were analyzed for changes in cytokine levels, innate immune cell subsets, and gene expression. Integrative statistical analyses with cross-validation identified a signature of 5 early innate markers correlating with antibody titers on day 28 and beyond. Among those, IP-10 on day 3 and MFI of CXCR6 on NK cells on day 1 were independent correlates. Consistently, we found an early gene expression signature linked to IP-10. This comprehensive characterization of early innate immune responses to the rVSV-ZEBOV vaccine in humans revealed immune signatures linked to IP-10. These results suggest correlates of vaccine-induced antibody induction and provide a rationale to explore strategies for augmenting the effectiveness of vaccines through manipulation of IP-10.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Systems Vaccinology Identifies an Early Innate Immune Signature as a Correlate of Antibody Responses to the Ebola Vaccine rVSV-ZEBOV</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 29</span></i><p style="text-align: justify; text-justify: inter-word;">Predicting vaccine efficacy remains a challenge. We used a systems vaccinology approach to identify early innate immune correlates of antibody induction in humans receiving the Ebola vaccine rVSV-ZEBOV. Blood samples from days 0, 1, 3, 7, and 14 were analyzed for changes in cytokine levels, innate immune cell subsets, and gene expression. Integrative statistical analyses with cross-validation identified a signature of 5 early innate markers correlating with antibody titers on day 28 and beyond. Among those, IP-10 on day 3 and MFI of CXCR6 on NK cells on day 1 were independent correlates. Consistently, we found an early gene expression signature linked to IP-10. This comprehensive characterization of early innate immune responses to the rVSV-ZEBOV vaccine in humans revealed immune signatures linked to IP-10. These results suggest correlates of vaccine-induced antibody induction and provide a rationale to explore strategies for augmenting the effectiveness of vaccines through manipulation of IP-10.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Systems Vaccinology Identifies an Early Innate Immune Signature as a Correlate of Antibody Responses to the Ebola Vaccine rVSV-ZEBOV</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 29</span></i><p style="text-align: justify; text-justify: inter-word;">Predicting vaccine efficacy remains a challenge. We used a systems vaccinology approach to identify early innate immune correlates of antibody induction in humans receiving the Ebola vaccine rVSV-ZEBOV. Blood samples from days 0, 1, 3, 7, and 14 were analyzed for changes in cytokine levels, innate immune cell subsets, and gene expression. Integrative statistical analyses with cross-validation identified a signature of 5 early innate markers correlating with antibody titers on day 28 and beyond. Among those, IP-10 on day 3 and MFI of CXCR6 on NK cells on day 1 were independent correlates. Consistently, we found an early gene expression signature linked to IP-10. This comprehensive characterization of early innate immune responses to the rVSV-ZEBOV vaccine in humans revealed immune signatures linked to IP-10. These results suggest correlates of vaccine-induced antibody induction and provide a rationale to explore strategies for augmenting the effectiveness of vaccines through manipulation of IP-10.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus Outbreak: What We Know and What We Don&#x2019;t</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Phylogenetic analyses reveal that SARS-CoV-2 is closely related to a group of SARS-like coronaviruses. However, it remains unclear where the virus comes from and how it was transmitted to humans in the first place. Unlike with other zoonotic agents such as hantavirus and arenavirus, thus far we haven’t found a SARS virus in animals that is the same as that in humans. Fortunately, SARS virus has not appeared in humans since 2004. In contrast, this new virus seems to have stronger transmission capabilities among people. Compared to the primary virus in humans, we still know less about whether, what, and how the virus has changed and the effect of the changes for their epidemics in humans. Control and prevention of the disease is especially difficult in China and elsewhere if there are infected individuals with no clinical signs.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The outbreak of Coronavirus Disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, December 2019, and continuously poses a serious threat to public health. Our previous study has shown that cytokine storm occurred during SARS-CoV-2 infection, while the detailed role of cytokines in the disease severity and progression remained unclear due to the limited case number. In this study, we examined 48 cytokines in the plasma samples from 53 COVID-19 cases, among whom 34 were severe cases, and the others moderate. Results showed that 14 cytokines were significantly elevated upon admission in COVID-19 cases. Moreover, IP-10, MCP-3, and IL-1ra were significantly higher in severe cases, and highly associated with the PaO2/FaO2 and Murray score. Furthermore, the three cytokines were independent predictors for the progression of COVID-19, and the combination of IP-10, MCP-3 and IL-1ra showed the biggest area under the curve (AUC) of the receiver-operating characteristics (ROC) calculations. Serial detection of IP-10, MCP-3 and IL-1ra in 14 severe cases showed that the continuous high levels of these cytokines were associated with disease deterioration and fatal outcome. In conclusion, we report three cytokines that closely associated with disease severity and outcome of COVID-19. These findings add to our understanding of the immunopathologic mechanisms of SARS-CoV-2 infection, which suggested novel therapeutic targets and strategy.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>ChAdOx1 and MVA based vaccine candidates against MERS-CoV elicit neutralising antibodies and cellular immune responses in mice</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 27</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) has infected more than 1900 humans, since 2012. The syndrome ranges from asymptomatic and mild cases to severe pneumonia and death. The virus is believed to be circulating in dromedary camels without notable symptoms since the 1980s. Therefore, dromedary camels are considered the only animal source of infection. Neither antiviral drugs nor vaccines are approved for veterinary or medical use despite active research on this area. Here, we developed four vaccine candidates against MERS-CoV based on ChAdOx1 and MVA viral vectors, two candidates per vector. All vaccines contained the full-length spike gene of MERS-CoV; ChAdOx1 MERS vaccines were produced with or without the leader sequence of the human tissue plasminogen activator gene (tPA) where MVA MERS vaccines were produced with tPA, but either the mH5 or F11 promoter driving expression of the spike gene. All vaccine candidates were evaluated in a mouse model in prime only or prime-boost regimens. ChAdOx1 MERS with tPA induced higher neutralising antibodies than ChAdOx1 MERS without tPA. A single dose of ChAdOx1 MERS with tPA elicited cellular immune responses as well as neutralising antibodies that were boosted to a significantly higher level by MVA MERS. The humoral immunogenicity of a single dose of ChAdOx1 MERS with tPA was equivalent to two doses of MVA MERS (also with tPA). MVA MERS with mH5 or F11 promoter induced similar antibody levels; however, F11 promoter enhanced the cellular immunogenicity of MVA MERS to significantly higher magnitudes. In conclusion, our study showed that MERS-CoV vaccine candidates could be optimized by utilising different viral vectors, various genetic designs of the vectors, or different regimens to increase immunogenicity. ChAdOx1 and MVA vectored vaccines have been safely evaluated in camels and humans and these MERS vaccine candidates should now be tested in camels and in clinical trials.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>The spatiotemporal estimation of the dynamic risk and the international transmission of 2019 Novel Coronavirus (COVID-19) outbreak: A global perspective</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An ongoing novel coronavirus SARS-CoV-2 pneumonia infection outbreak called COVID-19 started in Wuhan, Hubei Province, China, in December 2019. It both spread rapidly to all provinces in China and started spreading around the world quickly through international human movement from January 2020. Currently, the spatiotemporal epidemic transmission patterns, prediction models, and possible risk analysis for the future are insufficient for COVID-19 but we urgently need relevant information, particularly from the global perspective. We have developed a novel two-stage simulation model to simulate the spatiotemporal changes in the number of COVID-19 cases and estimate the future worldwide risk. Based on the connectivity of countries to China and the country's medical and epidemic prevention capabilities, different scenarios are generated to analyze the possible transmission throughout the world and use this information to evaluate each country's vulnerability to and the dynamic risk of COVID-19. Countries' vulnerability to the COVID-19 outbreak from China is calculated for 63 countries around the world. Taiwan, South Korea, Hong Kong, and Japan are the most vulnerable areas. The relationship between each country's vulnerability and days before the first imported case occurred shows a very high exponential decrease. The cumulative number of cases in each country also has a linear relationship with vulnerability, which can compare and quantify the initial epidemic prevention capabilities to various countries' management strategies. In total, 1,000 simulation results of future cases around the world are generated for the spatiotemporal risk assessment. According to the simulation results of this study, if there is no specific medicine for it, it will likely form a global pandemic. This method can be used as a preliminary risk assessment of the spatiotemporal spread for a new global epidemic. * Note: This study was completed on February 15, 2020.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficacy of an Adjuvanted Middle East Respiratory Syndrome Coronavirus Spike Protein Vaccine in Dromedary Camels and Alpacas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV is present in dromedary camels throughout the Middle East and Africa. Dromedary camels are the primary zoonotic reservoir for human infections. Interruption of the zoonotic transmission chain from camels to humans, therefore, may be an effective strategy to control the ongoing MERS-CoV outbreak. Here we show that vaccination with an adjuvanted MERS-CoV Spike protein subunit vaccine confers complete protection from MERS-CoV disease in alpaca and results in reduced and delayed viral shedding in the upper airways of dromedary camels. Protection in alpaca correlates with high serum neutralizing antibody titers. Lower titers of serum neutralizing antibodies correlate with delayed and significantly reduced shedding in the nasal turbinates of dromedary camels. Together, these data indicate that induction of robust neutralizing humoral immune responses by vaccination of naïve animals reduces shedding that potentially could diminish the risk of zoonotic transmission.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficacy of an Adjuvanted Middle East Respiratory Syndrome Coronavirus Spike Protein Vaccine in Dromedary Camels and Alpacas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV is present in dromedary camels throughout the Middle East and Africa. Dromedary camels are the primary zoonotic reservoir for human infections. Interruption of the zoonotic transmission chain from camels to humans, therefore, may be an effective strategy to control the ongoing MERS-CoV outbreak. Here we show that vaccination with an adjuvanted MERS-CoV Spike protein subunit vaccine confers complete protection from MERS-CoV disease in alpaca and results in reduced and delayed viral shedding in the upper airways of dromedary camels. Protection in alpaca correlates with high serum neutralizing antibody titers. Lower titers of serum neutralizing antibodies correlate with delayed and significantly reduced shedding in the nasal turbinates of dromedary camels. Together, these data indicate that induction of robust neutralizing humoral immune responses by vaccination of naïve animals reduces shedding that potentially could diminish the risk of zoonotic transmission.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND Corona Virus Disease 2019 (COVID-19) is spreading worldwide. Effective screening for patients is important to limit the epidemic. However, some defects make the currently applied diagnosis methods are still not very ideal for early warning of patients. We aimed to develop a diagnostic model that allows for the quick screening of highly suspected patients using easy-to-get variables. METHODS A total of 1,311 patients receiving severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) nucleicacid detection were included, whom with a positive result were classified into COVID-19 group. Multivariate logistic regression analyses were performed to construct the diagnostic model. Receiver operating characteristic (ROC) curve analysis were used for model validation. RESULTS After analysis, signs of pneumonia on CT, history of close contact, fever, neutrophil-to-lymphocyte ratio (NLR), Tmax and sex were included in the diagnostic model. Age and meaningful respiratory symptoms were enrolled into COVID-19 early warning score (COVID-19 EWS). The areas under the ROC curve (AUROC) indicated that both of the diagnostic model (training dataset 0.956 [95%CI 0.935-0.977, P &lt; 0.001]; validation dataset 0.960 [95%CI 0.919-1.0, P &lt; 0.001] ) and COVID-19 EWS (training dataset 0.956 [95%CI 0.934-0.978, P &lt; 0.001] ; validate dataset 0.966 [95%CI 0.929-1, P &lt; 0.001]) had good discrimination capacity. In addition, we also obtained the cut-off values of disease severity predictors, such as CT score, CD8+ T cell count, CD4+ T cell count, and so on. CONCLUSIONS The new developed COVID-19 EWS was a considerable tool for early and relatively accurately warning of SARS-CoV-2 infected patients.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control.  Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk.   Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 32 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days.   Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>A mathematical model for estimating the age-specific transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: A novel coronavirus named as "SARS-CoV-2" has spread widely in many countries since December 2019, especially in China. This study aimed to quantify the age-specific transmissibility by using a mathematical model. Methods: An age-specific susceptible - exposed - symptomatic - asymptomatic - recovered - seafood market (SEIARW) model was developed based on two suspected transmission routes (from market to person and person to person). The susceptible people from Wuhan City were divided into different age groups. We used the subscript i and j to represent age group 1 to 4 (1: &lt;= 14 years; 2: 15-44 years; 3: 45-64 years; 4: &gt;= 65 years) and 1 to 5 (1: &lt;= 5 years; 2: 6-14 years; 3: 15-24 years; 4: 25-59 years; 4: &gt;= 60 years), respectively. Data of reported COVID-19 cases were collected from one published literature from 26 November to 22 December, 2019 in Wuhan City, China. The age-specific transmissibility of the virus was estimated accordingly secondary attack rate (SAR).  Results: The age-specific SEIARW model fitted with the reported data well by dividing the population into four age groups (χ2 = 4.99 × 10-6, P &gt; 0.999), and five age groups (χ2 = 4.85 × 10-6, P &gt; 0.999). Based on the four-age-group SEIARW model, the highest transmissibility occurred from age group 2 to 3 (SAR23 = 17.56 per 10 million persons), followed by from age group 3 to 2 (SAR32 = 10.17 per 10 million persons). The lowest transmissibility occurred from age group 1 to 2 (SAR12 = 0.002 per 10 million persons). Based on the five-age-group SEIARW model, the highest transmissibility occurred from age group 4 to 5 (SAR45 = 12.40 per 10 million persons), followed by from age group 5 to 4 (SAR54 = 6.61 per 10 million persons). The lowest transmissibility occurred from age group 3 to 4 (SAR34 = 0.0002 per 10 million persons). Conclusions: SARS-CoV-2 has high transmissibility among adults and elder people but low transmissibility among children and young people.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /></body>