
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: universal coronavirus vaccine</h2><br /></div><br /><br /><br /><div class="card"><h3>A highly conserved WDYPKCDRA epitope in the RNA directed RNA polymerase of human coronaviruses can be used as epitope-based universal vaccine design</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 May 29</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronaviruses are the diverse group of RNA virus. From 1960, six strains of human coronaviruses have emerged that includes SARS-CoV and the recent infection by deadly MERS-CoV which is now going to cause another outbreak. Prevention of these viruses is urgent and a universal vaccine for all strain could be a promising solution in this circumstance. In this study we aimed to design an epitope based vaccine against all strain of human coronavirus. RESULTS: Multiple sequence alignment (MSA) approach was employed among spike (S), membrane (M), enveloped (E) and nucleocapsid (N) protein and replicase polyprotein 1ab to identify which one is highly conserve in all coronaviruses strains. Next, we use various in silico tools to predict consensus immunogenic and conserved peptide. We found that conserved region is present only in the RNA directed RNA polymerase protein. In this protein we identified one epitope WDYPKCDRA is highly immunogenic and 100% conserved among all available human coronavirus strains. CONCLUSIONS: Here we suggest in vivo study of our identified novel peptide antigen in RNA directed RNA polymerase protein for universal vaccine – which may be the way to prevent all human coronavirus disease.</p>
<p align="right"><i>score: 281</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a recombination-resistant coronavirus as a broadly applicable, rapidly implementable vaccine platform</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging zoonotic viral diseases are major threats to global health, economic stability, and national security. Vaccines are key for reducing coronaviral disease burden; however, the utility of live-attenuated vaccines is limited by risks of reversion or repair. Because of their history of emergence events due to their prevalence in zoonotic pools, designing live-attenuated coronavirus vaccines that can be rapidly and broadly implemented is essential for outbreak preparedness. Here, we show that coronaviruses with completely rewired transcription regulatory networks (TRNs) are effective vaccines against SARS-CoV. The TRN-rewired viruses are attenuated and protect against lethal SARS-CoV challenge. While a 3-nt rewired TRN reverts via second-site mutation upon serial passage, a 7-nt rewired TRN is more stable, suggesting that a more extensively rewired TRN might be essential for avoiding growth selection. In summary, rewiring the TRN is a feasible strategy for limiting reversion in an effective live-attenuated coronavirus vaccine candidate that is potentially portable across the Nidovirales order.</p>
<p align="right"><i>score: 229</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a recombination-resistant coronavirus as a broadly applicable, rapidly implementable vaccine platform</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging zoonotic viral diseases are major threats to global health, economic stability, and national security. Vaccines are key for reducing coronaviral disease burden; however, the utility of live-attenuated vaccines is limited by risks of reversion or repair. Because of their history of emergence events due to their prevalence in zoonotic pools, designing live-attenuated coronavirus vaccines that can be rapidly and broadly implemented is essential for outbreak preparedness. Here, we show that coronaviruses with completely rewired transcription regulatory networks (TRNs) are effective vaccines against SARS-CoV. The TRN-rewired viruses are attenuated and protect against lethal SARS-CoV challenge. While a 3-nt rewired TRN reverts via second-site mutation upon serial passage, a 7-nt rewired TRN is more stable, suggesting that a more extensively rewired TRN might be essential for avoiding growth selection. In summary, rewiring the TRN is a feasible strategy for limiting reversion in an effective live-attenuated coronavirus vaccine candidate that is potentially portable across the Nidovirales order.</p>
<p align="right"><i>score: 229</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Vaccine Candidates: Cautious Optimism</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 17</span></i><p style="text-align: justify; text-justify: inter-word;">Efforts towards developing a vaccine for Middle East respiratory syndrome coronavirus (MERS-CoV) have yielded promising results. Utilizing a variety of platforms, several vaccine approaches have shown efficacy in animal models and begun to enter clinical trials. In this review, we summarize the current progress towards a MERS-CoV vaccine and highlight potential roadblocks identified from previous attempts to generate coronavirus vaccines.</p>
<p align="right"><i>score: 218</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-04</span></i><p style="text-align: justify; text-justify: inter-word;">Novel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.</p>
<p align="right"><i>score: 214</i></p>

</div><br /><br /><br /><div class="card"><h3>A decade after SARS: Strategies to control emerging coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two novel coronaviruses have emerged in humans in the 21(st) century, Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome human coronavirus (MERS-CoV), both of which cause acute respiratory distress syndrome (ARDS) and have high mortality rates. There are no clinically approved vaccines or antiviral drugs available for either of these infections; thus, a priority in the field is the development of effective therapeutic and preventive strategies that can be readily applied to new emergent strains. This review will: describe the emergence and identification of novel human coronaviruses over the last 10 years; review their key biological features, including tropism and receptor use; and summarize approaches to develop broadly effective vaccines.</p>
<p align="right"><i>score: 212</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-04-28</span></i><p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live-attenuated coronavirus vaccines.\n\nSignificanceEmergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations like the aged and immune-compromised. This study illustrates that despite success in young models, the NSP16 mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the NSP16 mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for NSP16 attenuation as a major component of future live-attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 208</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination Attenuation Offers Strategy for Live Attenuated Coronavirus Vaccines</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved coronavirus proteins as a strategy to generate live attenuated vaccine strains against current and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a conserved 2′O methyltransferase (MTase), could provide a broad attenuation platform against future emergent strains. Using the severe acute respiratory syndrome-CoV mouse model, an NSP16 mutant vaccine was evaluated for protection from heterologous challenge, efficacy in the aging host, and potential for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and potential for reversion makes targeting NSP16 alone an untenable approach. However, combining a 2′O MTase mutation with a previously described CoV fidelity mutant produced a vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, and no evidence for reversion. Together, the results indicate that targeting the CoV 2′O MTase in parallel with other conserved attenuating mutations may provide a platform strategy for rapidly generating live attenuated coronavirus vaccines. IMPORTANCE Emergent coronaviruses remain a significant threat to global public health and rapid response vaccine platforms are needed to stem future outbreaks. However, failure of many previous CoV vaccine formulations has clearly highlighted the need to test efficacy under different conditions and especially in vulnerable populations such as the aged and immunocompromised. This study illustrates that despite success in young models, the 2′O methyltransferase mutant carries too much risk for pathogenesis and reversion in vulnerable models to be used as a stand-alone vaccine strategy. Importantly, the 2′O methyltransferase mutation can be paired with other attenuating approaches to provide robust protection from heterologous challenge and in vulnerable populations. Coupled with increased safety and reduced pathogenesis, the study highlights the potential for 2′O methyltransferase attenuation as a major component of future live attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 206</i></p>

</div><br /><br /><br /><div class="card"><h3>Airway Memory CD4(+) T Cells Mediate Protective Immunity against Emerging Respiratory Coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two zoonotic coronaviruses (CoV), SARS-CoV and MERS-CoV have crossed species to cause severe human respiratory disease. Here, we showed that induction of airway memory CD4(+) T cells specific for a conserved epitope shared by SARS-CoV and MERS-CoV is a potential strategy for developing pan-coronavirus vaccines. Airway memory CD4(+) T cells differed phenotypically and functionally from lung-derived cells and were crucial for protection against both CoVs in mice. Protection was interferon-γ-dependent and required early induction of robust innate and virus-specific CD8(+) T cell responses. The conserved epitope was also recognized in SARS-CoV and MERS-CoV-infected human leukocyte antigen DR2 and DR3 transgenic mice, indicating potential relevance in human populations. Additionally, this epitope was cross-protective between human and bat CoVs, the progenitors for many human CoVs. Vaccine strategies that induce airway memory CD4(+) T cells targeting conserved epitopes may have broad applicability in the context of new CoV and other respiratory virus outbreaks.</p>
<p align="right"><i>score: 206</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV NSP16 necessary for IFN resistance and viral pathogenesis</h3>
<i class="tooltip">2017<span class="tooltiptext">2017-08-08</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses encode a mix of highly conserved and novel genes as well as genetic elements necessary for infection and pathogenesis, raising the possibility for common targets for attenuation and therapeutic design. In this study, we focus on the highly conserved nonstructural protein (NSP) 16, a viral 2O methyl-transferase (MTase) that encodes critical functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif in the conserved KDKE motif of MERS NSP16 (D130A) and evaluated the effect on viral infection and pathogenesis. While the absence of 2O MTase activity had only marginal impact on propagation and replication in Vero cells, the MERS dNSP16 mutant demonstrated significant attenuation relative to control both in primary human airway cultures and in vivo. Further examination indicated the MERS dNSP16 mutant had a type I IFN based attenuation and was partially restored in the absence of IFIT molecules. Importantly, the robust attenuation permitted use of MERS dNSP16 as a live attenuated vaccine platform protecting from challenge with a mouse adapted MERS-CoV strain. These studies demonstrate the importance of the conserved 2O MTase activity for CoV pathogenesis and highlight NSP16 as a conserved universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting.\n\nSignificanceCoronavirus emergence in both human and livestock represents a significant threat to global public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and swine delta coronavirus in the 21st century. These studies describe an approach that effectively targets the highly conserved 2O methyl-transferase activity of coronaviruses for attenuation. With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable target for a live attenuated vaccine platform as well as therapeutic development for both current and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad applicability of targeting viral 2O MTase function across coronaviruses.</p>
<p align="right"><i>score: 205</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Non-Structural Protein 1 Is a Major Pathogenicity Factor: Implications for the Rational Design of Coronavirus Vaccines</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Aug 10</span></i><p style="text-align: justify; text-justify: inter-word;">Attenuated viral vaccines can be generated by targeting essential pathogenicity factors. We report here the rational design of an attenuated recombinant coronavirus vaccine based on a deletion in the coding sequence of the non-structural protein 1 (nsp1). In cell culture, nsp1 of mouse hepatitis virus (MHV), like its SARS-coronavirus homolog, strongly reduced cellular gene expression. The effect of nsp1 on MHV replication in vitro and in vivo was analyzed using a recombinant MHV encoding a deletion in the nsp1-coding sequence. The recombinant MHV nsp1 mutant grew normally in tissue culture, but was severely attenuated in vivo. Replication and spread of the nsp1 mutant virus was restored almost to wild-type levels in type I interferon (IFN) receptor-deficient mice, indicating that nsp1 interferes efficiently with the type I IFN system. Importantly, replication of nsp1 mutant virus in professional antigen-presenting cells such as conventional dendritic cells and macrophages, and induction of type I IFN in plasmacytoid dendritic cells, was not impaired. Furthermore, even low doses of nsp1 mutant MHV elicited potent cytotoxic T cell responses and protected mice against homologous and heterologous virus challenge. Taken together, the presented attenuation strategy provides a paradigm for the development of highly efficient coronavirus vaccines.</p>
<p align="right"><i>score: 205</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a lineage C betacoronavirus that since its emergence in 2012 has caused outbreaks in human populations with case-fatality rates of ∼36%. As in other coronaviruses, the spike (S) glycoprotein of MERS-CoV mediates receptor recognition and membrane fusion and is the primary target of the humoral immune response during infection. Here we use structure-based design to develop a generalizable strategy for retaining coronavirus S proteins in the antigenically optimal prefusion conformation and demonstrate that our engineered immunogen is able to elicit high neutralizing antibody titers against MERS-CoV. We also determined high-resolution structures of the trimeric MERS-CoV S ectodomain in complex with G4, a stem-directed neutralizing antibody. The structures reveal that G4 recognizes a glycosylated loop that is variable among coronaviruses and they define four conformational states of the trimer wherein each receptor-binding domain is either tightly packed at the membrane-distal apex or rotated into a receptor-accessible conformation. Our studies suggest a potential mechanism for fusion initiation through sequential receptor-binding events and provide a foundation for the structure-based design of coronavirus vaccines.</p>
<p align="right"><i>score: 203</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Protection against Severe Acute Respiratory Syndrome Coronavirus-Mediated Lethal Respiratory Disease in Aged Mice by Immunization with a Mouse-Adapted Virus Lacking E Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Zoonotic coronaviruses, including the one that caused severe acute respiratory syndrome (SARS), cause significant morbidity and mortality in humans. No specific therapy for any human coronavirus is available, making vaccine development critical for protection against these viruses. We previously showed that recombinant SARS coronavirus (SARS-CoV) (Urbani strain based) lacking envelope (E) protein expression (rU-ΔE) provided good but not perfect protection in young mice against challenge with virulent mouse-adapted SARS-CoV (MA15). To improve vaccine efficacy, we developed a second set of E-deleted vaccine candidates on an MA15 background (rMA15-ΔE). rMA15-ΔE is safe, causing no disease in 6-week-, 12-month-, or 18-month-old BALB/c mice. Immunization with this virus completely protected mice of three ages from lethal disease and effected more-rapid virus clearance. Compared to rU-ΔE, rMA15-ΔE immunization resulted in significantly greater neutralizing antibody and SARS-CoV-specific CD4 and CD8 T cell responses. After challenge, inflammatory cell infiltration, edema, and lung destruction were decreased in the lungs of rMA15-ΔE-immunized mice compared to those in rU-ΔE-immunized 12-month-old mice. Collectively, these results show that immunization with a species-adapted attenuated coronavirus lacking E protein expression is safe and provides optimal immunogenicity and long-term protection against challenge with lethal virus. This approach will be generally useful for development of vaccines protective against human coronaviruses as well as against coronaviruses that cause disease in domestic and companion animals.</p>
<p align="right"><i>score: 201</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus infections in the laboratory rat: degree of cross protection following immunization with a heterologous strain.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">One hundred and twenty-one specific pathogen-free male Wistar rats eight to ten weeks of age were used to evaluate the efficacy of Parker's rat coronavirus (PRC) in affording cross protection on subsequent challenge with virulent sialodacryoadenitis (SDA) virus. Sixty-two animals were inoculated intranasally on day 0 and 21 days later with approximately 10(2) median tissue culture infective doses (TCID50) of the tenth passage of PRC replicated in L-2 cells. Animals were selected at random postvaccination to evaluate the safety and efficacy of PRC by histopathology, immunohistochemistry and serology. At three and six months postvaccination (PV), vaccinated and seronegative control rats were inoculated intranasally with approximately 10(3) TCID50 doses of virulent SDA virus. Challenged rats were then killed at 6, 10 and 14 days postchallenge and necropsied. Evaluations were based on lesion indices in lacrimal and salivary glands and respiratory tract, the presence of viral antigen by immunohistochemistry, and antibody response. Lesions were observed in rats killed PV, but in general, they were significantly reduced compared with those present in seronegative animals post-exposure to virulent SDA virus (p < or = 0.05). However, they were still considered to be an unacceptable level for a routine vaccination procedure. Potvaccination antibody titers to rat coronavirus were evident in all animals tested at three or six months prior to challenge with SDA virus.(ABSTRACT TRUNCATED AT 250 WORDS)</p>
<p align="right"><i>score: 198</i></p>

</div><br /><br /><br /><div class="card"><h3>Harnessing the Power of T Cells: The Promising Hope for a Universal Influenza Vaccine</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Mar 26</span></i><p style="text-align: justify; text-justify: inter-word;">Next-generation vaccines that utilize T cells could potentially overcome the limitations of current influenza vaccines that rely on antibodies to provide narrow subtype-specific protection and are prone to antigenic mismatch with circulating strains. Evidence from animal models shows that T cells can provide heterosubtypic protection and are crucial for immune control of influenza virus infections. This has provided hope for the design of a universal vaccine able to prime against diverse influenza virus strains and subtypes. However, multiple hurdles exist for the realisation of a universal T cell vaccine. Overall primary concerns are: extrapolating human clinical studies, seeding durable effective T cell resident memory (Trm), population human leucocyte antigen (HLA) coverage, and the potential for T cell-mediated immune escape. Further comprehensive human clinical data is needed during natural infection to validate the protective role T cells play during infection in the absence of antibodies. Furthermore, fundamental questions still exist regarding the site, longevity and duration, quantity, and phenotype of T cells needed for optimal protection. Standardised experimental methods, and eventually simplified commercial assays, to assess peripheral influenza-specific T cell responses are needed for larger-scale clinical studies of T cells as a correlate of protection against influenza infection. The design and implementation of a T cell-inducing vaccine will require a consensus on the level of protection acceptable in the community, which may not provide sterilizing immunity but could protect the individual from severe disease, reduce the length of infection, and potentially reduce transmission in the community. Therefore, increasing the standard of care potentially offered by T cell vaccines should be considered in the context of pandemic preparedness and zoonotic infections, and in combination with improved antibody vaccine targeting methods. Current pandemic vaccine preparedness measures and ongoing clinical trials under-utilise T cell-inducing vaccines, reflecting the myriad questions that remain about how, when, where, and which T cells are needed to fight influenza virus infection. This review aims to bring together basic fundamentals of T cell biology with human clinical data, which need to be considered for the implementation of a universal vaccine against influenza that harnesses the power of T cells.</p>
<p align="right"><i>score: 197</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding domain as a target for developing SARS vaccines</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A decade ago, severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) caused a global pandemic with a mortality rate of 10%. Reports of recent outbreaks of a SARS-like disease caused by Middle East respiratory syndrome coronavirus (MERS-CoV) have raised serious concerns of a possible reemergence of SARS-CoV, either by laboratory escape or the presence of a natural reservoir. Therefore, the development of effective and safe SARS vaccines is still needed. Based on our previous studies, we believe that the receptor-binding domain (RBD) in the S1 subunit of the SARS-CoV spike (S) protein is the most important target for developing a SARS vaccine. In particular, RBD of S protein contains the critical neutralizing domain (CND), which is able to induce highly potent neutralizing antibody response and cross-protection against divergent SARS-CoV strains. Furthermore, a RBD-based subunit vaccine is expected to be safer than other vaccines that may induce Th2-type immunopathology. This review will discuss key advances in the development of RBD-based SARS vaccines and the possibility of using a similar strategy to develop vaccines against MERS-CoV.</p>
<p align="right"><i>score: 197</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 Vaccine Pipeline: an Overview</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19).</p>
<p align="right"><i>score: 191</i></p>

</div><br /><br /><br /><div class="card"><h3>Developing Universal Influenza Vaccines: Hitting the Nail, Not Just on the Head</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Mar 26</span></i><p style="text-align: justify; text-justify: inter-word;">Influenza viruses have a huge impact on public health. Current influenza vaccines need to be updated annually and protect poorly against antigenic drift variants or novel emerging subtypes. Vaccination against influenza can be improved in two important ways, either by inducing more broadly protective immune responses or by decreasing the time of vaccine production, which is relevant especially during a pandemic outbreak. In this review, we outline the current efforts to develop so-called “universal influenza vaccines”, describing antigens that may induce broadly protective immunity and novel vaccine production platforms that facilitate timely availability of vaccines.</p>
<p align="right"><i>score: 189</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the last decades by two global outbreaks of deadly pneumonia. Entry of coronaviruses into cells is mediated by the transmembrane spike glycoprotein S, which forms a trimer carrying receptor-binding and membrane fusion functions(1). S also contains the principal antigenic determinants and is the target of neutralizing antibodies. Here we present the structure of a murine coronavirus S trimer ectodomain determined at 4.0 Å resolution by single particle cryo-electron microscopy. It reveals the metastable pre-fusion architecture of S and highlights key interactions stabilizing it. The structure shares a common core with paramyxovirus F proteins(2,3), implicating mechanistic similarities and an evolutionary connection between these viral fusion proteins. The accessibility of the highly conserved fusion peptide at the periphery of the trimer indicates potential vaccinology strategies to elicit broadly neutralizing antibodies against coronaviruses. Finally, comparison with crystal structures of human coronavirus S domains allows rationalization of the molecular basis for species specificity based on the use of spatially contiguous but distinct domains.</p>
<p align="right"><i>score: 188</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the last decades by two global outbreaks of deadly pneumonia. Entry of coronaviruses into cells is mediated by the transmembrane spike glycoprotein S, which forms a trimer carrying receptor-binding and membrane fusion functions(1). S also contains the principal antigenic determinants and is the target of neutralizing antibodies. Here we present the structure of a murine coronavirus S trimer ectodomain determined at 4.0 Å resolution by single particle cryo-electron microscopy. It reveals the metastable pre-fusion architecture of S and highlights key interactions stabilizing it. The structure shares a common core with paramyxovirus F proteins(2,3), implicating mechanistic similarities and an evolutionary connection between these viral fusion proteins. The accessibility of the highly conserved fusion peptide at the periphery of the trimer indicates potential vaccinology strategies to elicit broadly neutralizing antibodies against coronaviruses. Finally, comparison with crystal structures of human coronavirus S domains allows rationalization of the molecular basis for species specificity based on the use of spatially contiguous but distinct domains.</p>
<p align="right"><i>score: 188</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the last decades by two global outbreaks of deadly pneumonia. Entry of coronaviruses into cells is mediated by the transmembrane spike glycoprotein S, which forms a trimer carrying receptor-binding and membrane fusion functions(1). S also contains the principal antigenic determinants and is the target of neutralizing antibodies. Here we present the structure of a murine coronavirus S trimer ectodomain determined at 4.0 Å resolution by single particle cryo-electron microscopy. It reveals the metastable pre-fusion architecture of S and highlights key interactions stabilizing it. The structure shares a common core with paramyxovirus F proteins(2,3), implicating mechanistic similarities and an evolutionary connection between these viral fusion proteins. The accessibility of the highly conserved fusion peptide at the periphery of the trimer indicates potential vaccinology strategies to elicit broadly neutralizing antibodies against coronaviruses. Finally, comparison with crystal structures of human coronavirus S domains allows rationalization of the molecular basis for species specificity based on the use of spatially contiguous but distinct domains.</p>
<p align="right"><i>score: 188</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the last decades by two global outbreaks of deadly pneumonia. Entry of coronaviruses into cells is mediated by the transmembrane spike glycoprotein S, which forms a trimer carrying receptor-binding and membrane fusion functions(1). S also contains the principal antigenic determinants and is the target of neutralizing antibodies. Here we present the structure of a murine coronavirus S trimer ectodomain determined at 4.0 Å resolution by single particle cryo-electron microscopy. It reveals the metastable pre-fusion architecture of S and highlights key interactions stabilizing it. The structure shares a common core with paramyxovirus F proteins(2,3), implicating mechanistic similarities and an evolutionary connection between these viral fusion proteins. The accessibility of the highly conserved fusion peptide at the periphery of the trimer indicates potential vaccinology strategies to elicit broadly neutralizing antibodies against coronaviruses. Finally, comparison with crystal structures of human coronavirus S domains allows rationalization of the molecular basis for species specificity based on the use of spatially contiguous but distinct domains.</p>
<p align="right"><i>score: 188</i></p>

</div><br /><br /><br /><div class="card"><h3>Polyanhydride Nanovaccine Induces Robust Pulmonary B and T Cell Immunity and Confers Protection Against Homologous and Heterologous Influenza A Virus Infections</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Aug 28</span></i><p style="text-align: justify; text-justify: inter-word;">Influenza A virus (IAV) is a major cause of respiratory illness. Given the disease severity, associated economic costs, and recent appearance of novel IAV strains, there is a renewed interest in developing novel and efficacious “universal” IAV vaccination strategies. Recent studies have highlighted that immunizations capable of generating local (i.e., nasal mucosa and lung) tissue-resident memory T and B cells in addition to systemic immunity offer the greatest protection against future IAV encounters. Current IAV vaccines are designed to largely stimulate IAV-specific antibodies, but do not generate the lung-resident memory T and B cells induced during IAV infections. Herein, we report on an intranasally administered biocompatible polyanhydride nanoparticle-based IAV vaccine (IAV-nanovax) capable of providing protection against subsequent homologous and heterologous IAV infections in both inbred and outbred populations. Our findings also demonstrate that vaccination with IAV-nanovax promotes the induction of germinal center B cells within the lungs, both systemic and lung local IAV-specific antibodies, and IAV-specific lung-resident memory CD4 and CD8 T cells. Altogether our findings show that an intranasally administered nanovaccine can induce immunity within the lungs, similar to what occurs during IAV infections, and thus could prove useful as a strategy for providing “universal” protection against IAV.</p>
<p align="right"><i>score: 186</i></p>

</div><br /><br /><br /><div class="card"><h3>Dendritic Cell-Specific Antigen Delivery by Coronavirus Vaccine Vectors Induces Long-Lasting Protective Antiviral and Antitumor Immunity</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Sep 14</span></i><p style="text-align: justify; text-justify: inter-word;">Efficient vaccination against infectious agents and tumors depends on specific antigen targeting to dendritic cells (DCs). We report here that biosafe coronavirus-based vaccine vectors facilitate delivery of multiple antigens and immunostimulatory cytokines to professional antigen-presenting cells in vitro and in vivo. Vaccine vectors based on heavily attenuated murine coronavirus genomes were generated to express epitopes from the lymphocytic choriomeningitis virus glycoprotein, or human Melan-A, in combination with the immunostimulatory cytokine granulocyte-macrophage colony-stimulating factor (GM-CSF). These vectors selectively targeted DCs in vitro and in vivo resulting in vector-mediated antigen expression and efficient maturation of DCs. Single application of only low vector doses elicited strong and long-lasting cytotoxic T-cell responses, providing protective antiviral and antitumor immunity. Furthermore, human DCs transduced with Melan-A-recombinant human coronavirus 229E efficiently activated tumor-specific CD8(+) T cells. Taken together, this novel vaccine platform is well suited to deliver antigens and immunostimulatory cytokines to DCs and to initiate and maintain protective immunity.</p>
<p align="right"><i>score: 184</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe Acute Respiratory Syndrome-Associated Coronavirus Vaccines Formulated with Delta Inulin Adjuvants Provide Enhanced Protection while Ameliorating Lung Eosinophilic Immunopathology</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although the severe acute respiratory syndrome-associated coronavirus (SARS-CoV) epidemic was controlled by nonvaccine measures, coronaviruses remain a major threat to human health. The design of optimal coronavirus vaccines therefore remains a priority. Such vaccines present major challenges: coronavirus immunity often wanes rapidly, individuals needing to be protected include the elderly, and vaccines may exacerbate rather than prevent coronavirus lung immunopathology. To address these issues, we compared in a murine model a range of recombinant spike protein or inactivated whole-virus vaccine candidates alone or adjuvanted with either alum, CpG, or Advax, a new delta inulin-based polysaccharide adjuvant. While all vaccines protected against lethal infection, addition of adjuvant significantly increased serum neutralizing-antibody titers and reduced lung virus titers on day 3 postchallenge. Whereas unadjuvanted or alum-formulated vaccines were associated with significantly increased lung eosinophilic immunopathology on day 6 postchallenge, this was not seen in mice immunized with vaccines formulated with delta inulin adjuvant. Protection against eosinophilic immunopathology by vaccines containing delta inulin adjuvants correlated better with enhanced T-cell gamma interferon (IFN-γ) recall responses rather than reduced interleukin-4 (IL-4) responses, suggesting that immunopathology predominantly reflects an inadequate vaccine-induced Th1 response. This study highlights the critical importance for development of effective and safe coronavirus vaccines of selection of adjuvants based on the ability to induce durable IFN-γ responses. IMPORTANCE Coronaviruses such as SARS-CoV and Middle East respiratory syndrome-associated coronavirus (MERS-CoV) cause high case fatality rates and remain major human public health threats, creating a need for effective vaccines. While coronavirus antigens that induce protective neutralizing antibodies have been identified, coronavirus vaccines present a unique problem in that immunized individuals when infected by virus can develop lung eosinophilic pathology, a problem that is further exacerbated by the formulation of SARS-CoV vaccines with alum adjuvants. This study shows that formulation of SARS-CoV spike protein or inactivated whole-virus vaccines with novel delta inulin-based polysaccharide adjuvants enhances neutralizing-antibody titers and protection against clinical disease but at the same time also protects against development of lung eosinophilic immunopathology. It also shows that immunity achieved with delta inulin adjuvants is long-lived, thereby overcoming the natural tendency for rapidly waning coronavirus immunity. Thus, delta inulin adjuvants may offer a unique ability to develop safer and more effective coronavirus vaccines.</p>
<p align="right"><i>score: 184</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibody-Dependent Cell-Mediated Cytotoxicity Epitopes on the Hemagglutinin Head Region of Pandemic H1N1 Influenza Virus Play Detrimental Roles in H1N1-Infected Mice</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 21</span></i><p style="text-align: justify; text-justify: inter-word;">Engaging the antibody-dependent cell-mediated cytotoxicity (ADCC) for killing of virus-infected cells and secretion of antiviral cytokines and chemokines was incorporated as one of the important features in the design of universal influenza vaccines. However, investigation of the ADCC epitopes on the highly immunogenic influenza hemagglutinin (HA) head region has been rarely reported. In this study, we determined the ADCC and antiviral activities of two putative ADCC epitopes, designated E1 and E2, on the HA head of a pandemic H1N1 influenza virus in vitro and in a lethal mouse model. Our data demonstrated that sera from the E1-vaccinated mice could induce high ADCC activities. Importantly, the induction of ADCC response modestly decreased viral load in the lungs of H1N1-infected mice. However, the elevated ADCC significantly increased mouse alveolar damage and mortality than that of the PBS-vaccinated group (P < 0.0001). The phenotype was potentially due to an exaggerated inflammatory cell infiltration triggered by ADCC, as an upregulated release of cytotoxic granules (perforin) was observed in the lung tissue of E1-vaccinated mice after H1N1 influenza virus challenge. Overall, our data suggested that ADCC elicited by certain domains of HA head region might have a detrimental rather than protective effect during influenza virus infection. Thus, future design of universal influenza vaccine shall strike a balance between the induction of protective immunity and potential side effects of ADCC.</p>
<p align="right"><i>score: 181</i></p>

</div><br /><br /><br /><div class="card"><h3>Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the past few decades by two global outbreaks of deadly pneumonia. The coronavirus spike (S) glycoprotein initiates infection by promoting fusion of the viral and cellular membranes through conformational changes that remain largely uncharacterized. Here we report the cryoEM structure of a coronavirus S glycoprotein in the postfusion state, showing large-scale secondary, tertiary, and quaternary rearrangements compared with the prefusion trimer and rationalizing the free-energy landscape of this conformational machine. We also biochemically characterized the molecular events associated with refolding of the metastable prefusion S glycoprotein to the postfusion conformation using limited proteolysis, mass spectrometry, and single-particle EM. The observed similarity between postfusion coronavirus S and paramyxovirus F structures demonstrates that a conserved refolding trajectory mediates entry of these viruses and supports the evolutionary relatedness of their fusion subunits. Finally, our data provide a structural framework for understanding the mode of neutralization of antibodies targeting the fusion machinery and for engineering next-generation subunit vaccines or inhibitors against this medically important virus family.</p>
<p align="right"><i>score: 181</i></p>

</div><br /><br /><br /><div class="card"><h3>An H5N1-based matrix protein 2 ectodomain tetrameric peptide vaccine provides cross-protection against lethal infection with H7N9 influenza virus</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Apr 8</span></i><p style="text-align: justify; text-justify: inter-word;">In March 2013, a patient infected with a novel avian influenza A H7N9 virus was reported in China. Since then, there have been 458 confirmed infection cases and 177 deaths. The virus contains several human-adapted markers, indicating that H7N9 has pandemic potential. The outbreak of this new influenza virus highlighted the need for the development of universal influenza vaccines. Previously, we demonstrated that a tetrameric peptide vaccine based on the matrix protein 2 ectodomain (M2e) of the H5N1 virus (H5N1-M2e) could protect mice from lethal infection with different clades of H5N1 and 2009 pandemic H1N1 influenza viruses. In this study, we investigated the cross-protection of H5N1-M2e against lethal infection with the new H7N9 virus. Although five amino acid differences existed at positions 13, 14, 18, 20, and 21 between M2e of H5N1 and H7N9, H5N1-M2e vaccination with either Freund's adjuvant or the Sigma adjuvant system (SAS) induced a high level of anti-M2e antibody, which cross-reacted with H7N9-M2e peptide. A mouse-adapted H7N9 strain, A/Anhui/01/2013m, was used for lethal challenge in animal experiments. H5N1-M2e vaccination provided potent cross-protection against lethal challenge of the H7N9 virus. Reduced viral replication and histopathological damage of mouse lungs were also observed in the vaccinated mice. Our results suggest that the tetrameric H5N1-M2e peptide vaccine could protect against different subtypes of influenza virus infections. Therefore, this vaccine may be an ideal candidate for developing a universal vaccine to prevent the reemergence of avian influenza A H7N9 virus and the emergence of potential novel reassortants of influenza virus.</p>
<p align="right"><i>score: 180</i></p>

</div><br /><br /><br /><div class="card"><h3>Purified coronavirus Spike protein nanoparticles induce coronavirus neutralizing antibodies in mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Development of vaccination strategies for emerging pathogens are particularly challenging because of the sudden nature of the emergence of these viruses and the long process needed for traditional vaccine development. Therefore, there is a need for development of a rapid method of vaccine development that can respond to emerging pathogens in a short time frame. The emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) in 2003 and Middle East respiratory syndrome (MERS-CoV) in late 2012 demonstrate the importance of coronaviruses as emerging pathogens. The spike glycoproteins of coronaviruses reside on the surface of the virion and are responsible for virus entry. The spike glycoprotein is the major immunodominant antigen of coronaviruses and has proven to be an excellent target for vaccine designs that seek to block coronavirus entry and promote antibody targeting of infected cells. Vaccination strategies for coronaviruses have involved live attenuated virus, recombinant viruses, non-replicative virus-like particles expressing coronavirus proteins or DNA plasmids expressing coronavirus genes. None of these strategies has progressed to an approved human coronavirus vaccine in the ten years since SARS-CoV emerged. Here we describe a novel method for generating MERS-CoV and SARS-CoV full-length spike nanoparticles, which in combination with adjuvants are able to produce high titer antibodies in mice.</p>
<p align="right"><i>score: 180</i></p>

</div><br /><br /><br /><div class="card"><h3>Protective Efficacy of the Conserved NP, PB1, and M1 Proteins as Immunogens in DNA- and Vaccinia Virus-Based Universal Influenza A Virus Vaccines in Mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The conventional hemagglutinin (HA)- and neuraminidase (NA)-based influenza vaccines need to be updated most years and are ineffective if the glycoprotein HA of the vaccine strains is a mismatch with that of the epidemic strain. Universal vaccines targeting conserved viral components might provide cross-protection and thus complement and improve conventional vaccines. In this study, we generated DNA plasmids and recombinant vaccinia viruses expressing the conserved proteins nucleoprotein (NP), polymerase basic 1 (PB1), and matrix 1 (M1) from influenza virus strain A/Beijing/30/95 (H3N2). BALB/c mice were immunized intramuscularly with a single vaccine based on NP, PB1, or M1 alone or a combination vaccine based on all three antigens and were then challenged with lethal doses of the heterologous influenza virus strain A/PR/8/34 (H1N1). Vaccines based on NP, PB1, and M1 provided complete or partial protection against challenge with 1.7 50% lethal dose (LD(50)) of PR8 in mice. Of the three antigens, NP-based vaccines induced protection against 5 LD(50) and 10 LD(50) and thus exhibited the greatest protective effect. Universal influenza vaccines based on the combination of NP, PB1, and M1 induced a strong immune response and thus might be an alternative approach to addressing future influenza virus pandemics.</p>
<p align="right"><i>score: 177</i></p>

</div><br /><br /><br /><div class="card"><h3>Mucosal Vaccination with Recombinant Adenovirus Encoding Nucleoprotein Provides Potent Protection against Influenza Virus Infection</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Sep 25</span></i><p style="text-align: justify; text-justify: inter-word;">Influenza vaccines that target the highly variable surface glycoproteins hemagglutinin and neuraminidase cause inconvenience of having vaccination every year. For this reason, development of universal vaccines targeting conserved viral components is needed. In this study, we generated recombinant adenovirus (rAd) vaccine encoding nucleoprotein (NP) of A/PR/8/34 influenza virus, designated rAd/NP. BALB/c mice were immunized intranasally or sublingually with rAd/NP vaccine and subsequently challenged with lethal doses of heterologous as well as homologous influenza viruses. We found that intranasal immunization of rAd/NP elicited strong mucosal IgA responses as well as stronger CD8 T-cell responses toward immunodominant K(d)-restricted NP(147-155) epitope than sublingual immunization. Importantly, only single intranasal but not sublingual immunization of rAd/NP provides potent protection against both homologous and heterologous influenza virus challenges. These results suggest that recombinant rAd/NP could be a universal vaccine candidate for mucosal administration against influenza virus.</p>
<p align="right"><i>score: 177</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccination to Conserved Influenza Antigens in Mice Using a Novel Simian Adenovirus Vector, PanAd3, Derived from the Bonobo Pan paniscus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Mar 11</span></i><p style="text-align: justify; text-justify: inter-word;">Among approximately 1000 adenoviruses from chimpanzees and bonobos studied recently, the Pan Adenovirus type 3 (PanAd3, isolated from a bonobo, Pan paniscus) has one of the best profiles for a vaccine vector, combining potent transgene immunogenicity with minimal pre-existing immunity in the human population. In this study, we inserted into a replication defective PanAd3 a transgene expressing a fusion protein of conserved influenza antigens nucleoprotein (NP) and matrix 1 (M1). We then studied antibody and T cell responses as well as protection from challenge infection in a mouse model. A single intranasal administration of PanAd3-NPM1 vaccine induced strong antibody and T cell responses, and protected against high dose lethal influenza virus challenge. Thus PanAd3 is a promising candidate vector for vaccines, including universal influenza vaccines.</p>
<p align="right"><i>score: 177</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-binding domain-based subunit vaccines against MERS-CoV</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Development of effective vaccines, in particular, subunit-based vaccines, against emerging Middle East respiratory syndrome (MERS) caused by the MERS coronavirus (MERS-CoV) will provide the safest means of preventing the continuous spread of MERS in humans and camels. This review briefly describes the structure of the MERS-CoV spike (S) protein and its receptor-binding domain (RBD), discusses the current status of MERS vaccine development, and illustrates the strategies used to develop RBD-based subunit vaccines against MERS. It also summarizes currently available animal models for MERS-CoV and proposes a future direction for MERS vaccines. Taken together, this review will assist researchers working to develop effective and safe subunit vaccines against MERS-CoV and any other emerging coronaviruses that might cause future pandemics.</p>
<p align="right"><i>score: 175</i></p>

</div><br /><br /><br /><div class="card"><h3>Computational Approaches and Challenges to Developing Universal Influenza Vaccines</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 28</span></i><p style="text-align: justify; text-justify: inter-word;">The traditional design of effective vaccines for rapidly-evolving pathogens, such as influenza A virus, has failed to provide broad spectrum and long-lasting protection. With low cost whole genome sequencing technology and powerful computing capabilities, novel computational approaches have demonstrated the potential to facilitate the design of a universal influenza vaccine. However, few studies have integrated computational optimization in the design and discovery of new vaccines. Understanding the potential of computational vaccine design is necessary before these approaches can be implemented on a broad scale. This review summarizes some promising computational approaches under current development, including computationally optimized broadly reactive antigens with consensus sequences, phylogenetic model-based ancestral sequence reconstruction, and immunomics to compute conserved cross-reactive T-cell epitopes. Interactions between virus-host-environment determine the evolvability of the influenza population. We propose that with the development of novel technologies that allow the integration of data sources such as protein structural modeling, host antibody repertoire analysis and advanced phylodynamic modeling, computational approaches will be crucial for the development of a long-lasting universal influenza vaccine. Taken together, computational approaches are powerful and promising tools for the development of a universal influenza vaccine with durable and broad protection.</p>
<p align="right"><i>score: 175</i></p>

</div><br /><br /><br /><div class="card"><h3>Modified Vaccinia Virus Ankara (MVA) as Production Platform for Vaccines against Influenza and Other Viral Respiratory Diseases</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jul 17</span></i><p style="text-align: justify; text-justify: inter-word;">Respiratory viruses infections caused by influenza viruses, human parainfluenza virus (hPIV), respiratory syncytial virus (RSV) and coronaviruses are an eminent threat for public health. Currently, there are no licensed vaccines available for hPIV, RSV and coronaviruses, and the available seasonal influenza vaccines have considerable limitations. With regard to pandemic preparedness, it is important that procedures are in place to respond rapidly and produce tailor made vaccines against these respiratory viruses on short notice. Moreover, especially for influenza there is great need for the development of a universal vaccine that induces broad protective immunity against influenza viruses of various subtypes. Modified Vaccinia Virus Ankara (MVA) is a replication-deficient viral vector that holds great promise as a vaccine platform. MVA can encode one or more foreign antigens and thus functions as a multivalent vaccine. The vector can be used at biosafety level 1, has intrinsic adjuvant capacities and induces humoral and cellular immune responses. However, there are some practical and regulatory issues that need to be addressed in order to develop MVA-based vaccines on short notice at the verge of a pandemic. In this review, we discuss promising novel influenza virus vaccine targets and the use of MVA for vaccine development against various respiratory viruses.</p>
<p align="right"><i>score: 173</i></p>

</div><br /><br /><br /><div class="card"><h3>Humoral and Cellular Immune Responses Induced by 3a DNA Vaccines against Severe Acute Respiratory Syndrome (SARS) or SARS-Like Coronavirus in Mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Vaccine development for severe acute respiratory syndrome coronavirus (SARS-CoV) has mainly focused on the spike (S) protein. However, the variation of the S gene between viruses may affect the efficacy of a vaccine, particularly for cross-protection against SARS-like CoV (SL-CoV). Recently, a more conserved group-specific open reading frame (ORF), the 3a gene, was found in both SARS-CoV and SL-CoV. Here, we studied the immunogenicity of human SARS-CoV 3a and bat SL-CoV 3a DNA vaccines in mice through electroporation immunization followed by enzyme-linked immunosorbent, enzyme-linked immunospot, and flow cytometry assays. Our results showed that high levels of specific humoral responses were induced by SARS-CoV 3a and SL-CoV 3a DNA vaccines. Furthermore, a strong Th1-based cellular immune response was stimulated by both DNA vaccines. The vaccines stimulated gamma interferon production mainly by CD8(+) T cells and interleukin-2 (IL-2) mainly by CD4(+) T cells. Of interest, the frequency of IL-2-positive cells elicited by the SARS-CoV 3a DNA vaccine was significantly higher than that elicited by the SL-CoV 3a DNA vaccine. In summary, our study provides a reference for designing cross-protective DNA vaccines based on the group-specific ORFs of CoVs.</p>
<p align="right"><i>score: 170</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 169</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 169</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of candidate vaccine approaches for MERS-CoV</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) as a cause of severe respiratory disease highlights the need for effective approaches to CoV vaccine development. Efforts focused solely on the receptor-binding domain (RBD) of the viral Spike (S) glycoprotein may not optimize neutralizing antibody (NAb) responses. Here we show that immunogens based on full-length S DNA and S1 subunit protein elicit robust serum-neutralizing activity against several MERS-CoV strains in mice and non-human primates. Serological analysis and isolation of murine monoclonal antibodies revealed that immunization elicits NAbs to RBD and, non-RBD portions of S1 and S2 subunit. Multiple neutralization mechanisms were demonstrated by solving the atomic structure of a NAb-RBD complex, through sequencing of neutralization escape viruses and by constructing MERS-CoV S variants for serological assays. Immunization of rhesus macaques confers protection against MERS-CoV-induced radiographic pneumonia, as assessed using computerized tomography, supporting this strategy as a promising approach for MERS-CoV vaccine development.</p>
<p align="right"><i>score: 168</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of candidate vaccine approaches for MERS-CoV</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) as a cause of severe respiratory disease highlights the need for effective approaches to CoV vaccine development. Efforts focused solely on the receptor-binding domain (RBD) of the viral Spike (S) glycoprotein may not optimize neutralizing antibody (NAb) responses. Here we show that immunogens based on full-length S DNA and S1 subunit protein elicit robust serum-neutralizing activity against several MERS-CoV strains in mice and non-human primates. Serological analysis and isolation of murine monoclonal antibodies revealed that immunization elicits NAbs to RBD and, non-RBD portions of S1 and S2 subunit. Multiple neutralization mechanisms were demonstrated by solving the atomic structure of a NAb-RBD complex, through sequencing of neutralization escape viruses and by constructing MERS-CoV S variants for serological assays. Immunization of rhesus macaques confers protection against MERS-CoV-induced radiographic pneumonia, as assessed using computerized tomography, supporting this strategy as a promising approach for MERS-CoV vaccine development.</p>
<p align="right"><i>score: 168</i></p>

</div><br /><br /><br /><div class="card"><h3>Towards a Coronavirus-Based HIV Multigene Vaccine</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Jun-Dec</span></i><p style="text-align: justify; text-justify: inter-word;">Human immunodeficiency virus (HIV) infection represents one of the major health threats in the developing world. The costly treatment of infected individuals with multiple highly efficient anti-HIV drugs is only affordable in industrialized countries. Thus, an efficient vaccination strategy is required to prevent the further spread of the infection. The molecular biology of coronaviruses and particular features of the human coronavirus 229E (HCoV 229E) indicate that HCoV 229E-based vaccine vectors can become a new class of highly efficient vaccines. First, the receptor of HCoV 229E, human aminopeptidase N (hAPN or CD13) is expressed mainly on human dendritic cells (DCs) and macrophages indicating that targeting of HCoV 229E-based vectors to professional antigen presenting cells can be achieved by receptor-mediated transduction. Second, HCoV 229E structural genes can be replaced by multiple transcriptional units encoding various antigens. These virus-like particles (VLPs) containing HCoV 229E-based vector RNA have the ability to transduce human DCs and to mediate heterologous gene expression in these cells. Finally, coronavirus infections are associated with mainly respiratory and enteric diseases, and natural transmission of coronaviruses occurs via mucosal surfaces. In humans, HCoV 229E causes common cold by infecting the upper respiratory tract. HCoV 229E infections are mainly encountered in children and re-infection occurs frequently in adults. It is thus most likely that pre-existing immunity against HCoV 229E will not significantly impact on the vaccination efficiency if HCoV 229E-based vectors are used in humans.</p>
<p align="right"><i>score: 168</i></p>

</div><br /><br /><br /><div class="card"><h3>MIDDLE EAST RESPIRATORY SYNDROME VACCINES</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East Respiratory Syndrome-coronavirus (MERS-CoV) has infected over 1600 individuals with nearly 600 deaths since it was first identified in human populations in 2012. No anti-viral therapies or vaccines for treatment and prophylaxis are available. Here, we discuss approaches to developing MERS vaccines, including a summary of previous efforts to develop vaccines useful against human and non-human coronaviruses. A striking feature of MERS is the important role that camels have in transmission. Camel vaccination may be a novel approach to preventing the human infection.</p>
<p align="right"><i>score: 166</i></p>

</div><br /><br /><br /><div class="card"><h3>Live, Attenuated Coronavirus Vaccines through the Directed Deletion of Group-Specific Genes Provide Protection against Feline Infectious Peritonitis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Feline infectious peritonitis (FIP) is a fatal immunity-mediated disease caused by mutants of a ubiquitous coronavirus. Since previous attempts to protect cats under laboratory and field conditions have been largely unsuccessful, we used our recently developed system of reverse genetics (B. J. Haijema, H. Volders, and P. J. M. Rottier, J. Virol. 77:4528-4538, 2003) for the development of a modified live FIP vaccine. With this objective, we deleted the group-specific gene cluster open reading frame 3abc or 7ab and obtained deletion mutant viruses that not only multiplied well in cell culture but also showed an attenuated phenotype in the cat. At doses at which the wild-type virus would be fatal, the mutants with gene deletions did not cause any clinical symptoms. They still induced an immune response, however, as judged from the high levels of virus-neutralizing antibodies. The FIP virus (FIPV) mutant lacking the 3abc cluster and, to a lesser extent, the mutant missing the 7ab cluster, protected cats against a lethal homologous challenge; no protection was obtained with the mutant devoid of both gene clusters. Our studies show that the deletion of group-specific genes from the coronavirus genome results in live attenuated candidate vaccines against FIPV. More generally, our approach may allow the development of vaccines against infections with other pathogenic coronaviruses, including that causing severe acute respiratory syndrome in humans.</p>
<p align="right"><i>score: 165</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral metagenomics, protein structure, and reverse genetics: key strategies for investigating coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viral metagenomics, modeling of protein structure, and manipulation of viral genetics are key approaches that have laid the foundations of our understanding of coronavirus biology. In this review, we discuss the major advances each method has provided and discuss how future studies should leverage these strategies synergistically to answer novel questions.</p>
<p align="right"><i>score: 165</i></p>

</div><br /><br /><br /><div class="card"><h3>Protection against homo and hetero-subtypic inﬂuenza A virus by optimized M2e DNA vaccine</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 16</span></i><p style="text-align: justify; text-justify: inter-word;">Current influenza vaccines provide hemagglutinin strain-specific protection, but rarely provide cross-protection against divergent strains. It is, therefore, particularly important to develop a universal vaccine against conserved proteins or conserved regions of the virus. In this study, we used N-terminal extracellular region of the influenza virus M2 protein (M2e) as the target antigen and constructed two optimized M2e DNA vaccines (p-tPA-p3M2e and p-p3M2e) with increased antigenic epitope density and enhanced antigen secretion. Both vaccines induced high M2e-specific humoral and cellular immune responses in the vaccinated mice. These two vaccines also conferred protection against a lethal infection of homo-subtypic H1N1 virus, with p-tPA-p3M2e being the most effective. In addition, p-tPA-p3M2e also showed cross-protection against different subtypes of the influenza virus (H9N2, H6N6, and H10N8) at varying rates (80%, 40%, and 20%, respectively). After passive immunization, M2e DNA vaccine-induced antibodies in the sera provided complete protection against homologous virus challenge. An analysis of the mechanism underlying this immunization-mediated protection indicates that M2e-specific IgG and T-cell immune responses may play critical roles in the prevention of infection and viral clearance. Taken together, our results indicate that this optimized M2e DNA vaccine is a promising candidate for the development of a universal, broad-spectrum influenza virus vaccine.</p>
<p align="right"><i>score: 165</i></p>

</div><br /><br /><br /><div class="card"><h3>Chimeric coronavirus-like particles carrying severe acute respiratory syndrome coronavirus (SCoV) S protein protects mice against challenge with SCoV</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We tested the efficacy of coronavirus-like particles (VLPs) for protecting mice against severe acute respiratory syndrome coronavirus (SCoV) infection. Coexpression of SCoV S protein and E, M and N proteins of mouse hepatitis virus in 293T or CHO cells resulted in the efficient production of chimeric VLPs carrying SCoV S protein. Balb/c mice inoculated with a mixture of chimeric VLPs and alum twice at an interval of four weeks were protected from SCoV challenge, as indicated by the absence of infectious virus in the lungs. The same groups of mice had high levels of SCoV-specific neutralizing antibodies, while mice in the negative control groups, which were not immunized with chimeric VLPs, failed to manifest neutralizing antibodies, suggesting that SCoV-specific neutralizing antibodies are important for the suppression of viral replication within the lungs. Despite some differences in the cellular composition of inflammatory infiltrates, we did not observe any overt lung pathology in the chimeric-VLP-treated mice, when compared to the negative control mice. Our results show that chimeric VLP can be an effective vaccine strategy against SCoV infection.</p>
<p align="right"><i>score: 164</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunization with an Attenuated Severe Acute Respiratory Syndrome Coronavirus Deleted in E Protein Protects Against Lethal Respiratory Disease</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) caused substantial morbidity and mortality in 2002-2003. Deletion of the envelope (E) protein modestly diminished virus growth in tissue culture but abrogated virulence in animals. Here, we show that immunization with rSARS-CoV-ΔE or SARS-CoV-Δ[E,6-9b] (deleted in accessory proteins (6,7a,7b,8a,8b,9b) in addition to E) nearly completely protected BALB/c mice from fatal respiratory disease caused by mouse-adapted SARS-CoV and partly protected hACE2 Tg mice from lethal disease. hACE2 Tg mice, which express the human SARS-CoV receptor, are extremely susceptible to infection. We also show that rSARS-CoV-ΔE and rSARS-CoV-Δ[E,6-9b] induced anti-virus T cell and antibody responses. Further, the E-deleted viruses were stable after 16 blind passages through tissue culture cells, with only a single mutation in the surface glycoprotein detected. The passaged virus remained avirulent in mice. These results suggest that rSARS-CoV-ΔE is an efficacious vaccine candidate that might be useful if SARS recurred.</p>
<p align="right"><i>score: 163</i></p>

</div><br /><br /><br /><div class="card"><h3>Self-Amplifying mRNA Vaccines Expressing Multiple Conserved Influenza Antigens Confer Protection against Homologous and Heterosubtypic Viral Challenge</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug 15</span></i><p style="text-align: justify; text-justify: inter-word;">Current hemagglutinin (HA)-based seasonal influenza vaccines induce vaccine strain-specific neutralizing antibodies that usually fail to provide protection against mismatched circulating viruses. Inclusion in the vaccine of highly conserved internal proteins such as the nucleoprotein (NP) and the matrix protein 1 (M1) was shown previously to increase vaccine efficacy by eliciting cross-reactive T-cells. However, appropriate delivery systems are required for efficient priming of T-cell responses. In this study, we demonstrated that administration of novel self-amplifying mRNA (SAM(®)) vectors expressing influenza NP (SAM(NP)), M1 (SAM(M1)), and NP and M1 (SAM(M1-NP)) delivered with lipid nanoparticles (LNP) induced robust polyfunctional CD4 T helper 1 cells, while NP-containing SAM also induced cytotoxic CD8 T cells. Robust expansions of central memory (T(CM)) and effector memory (T(EM)) CD4 and CD8 T cells were also measured. An enhanced recruitment of NP-specific cytotoxic CD8 T cells was observed in the lungs of SAM(NP)-immunized mice after influenza infection that paralleled with reduced lung viral titers and pathology, and increased survival after homologous and heterosubtypic influenza challenge. Finally, we demonstrated for the first time that the co-administration of RNA (SAM(M1-NP)) and protein (monovalent inactivated influenza vaccine (MIIV)) was feasible, induced simultaneously NP-, M1- and HA-specific T cells and HA-specific neutralizing antibodies, and enhanced MIIV efficacy against a heterologous challenge. In conclusion, systemic administration of SAM vectors expressing conserved internal influenza antigens induced protective immune responses in mice, supporting the SAM(®) platform as another promising strategy for the development of broad-spectrum universal influenza vaccines.</p>
<p align="right"><i>score: 162</i></p>

</div><br /><br /><br /><div class="card"><h3>Pseudotype-Based Neutralization Assays for Influenza: A Systematic Analysis</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">The use of vaccination against the influenza virus remains the most effective method of mitigating the significant morbidity and mortality caused by this virus. Antibodies elicited by currently licensed influenza vaccines are predominantly hemagglutination-inhibition (HI)-competent antibodies that target the globular head of hemagglutinin (HA) thus inhibiting influenza virus entry into target cells. These antibodies predominantly confer homosubtypic/strain specific protection and only rarely confer heterosubtypic protection. However, recent academia or pharma-led R&D toward the production of a “universal vaccine” has centered on the elicitation of antibodies directed against the stalk of the influenza HA that has been shown to confer broad protection across a range of different subtypes (H1–H16). The accurate and sensitive measurement of antibody responses elicited by these “next-generation” influenza vaccines is, however, hampered by the lack of sensitivity of the traditional influenza serological assays HI, single radial hemolysis, and microneutralization. Assays utilizing pseudotypes, chimeric viruses bearing influenza glycoproteins, have been shown to be highly efficient for the measurement of homosubtypic and heterosubtypic broadly neutralizing antibodies, making them ideal serological tools for the study of cross-protective responses against multiple influenza subtypes with pandemic potential. In this review, we will analyze and compare literature involving the production of influenza pseudotypes with particular emphasis on their use in serum antibody neutralization assays. This will enable us to establish the parameters required for optimization and propose a consensus protocol to be employed for the further deployment of these assays in influenza vaccine immunogenicity studies.</p>
<p align="right"><i>score: 162</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus nonstructural protein 15 mediates evasion of dsRNA sensors and limits apoptosis in macrophages</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-sense RNA viruses that generate double-stranded RNA (dsRNA) intermediates during replication, yet evade detection by host innate immune sensors. Here we report that coronavirus nonstructural protein 15 (nsp15), an endoribonuclease, is required for evasion of dsRNA sensors. We evaluated two independent nsp15 mutant mouse coronaviruses, designated N15m1 and N15m3, and found that these viruses replicated poorly and induced rapid cell death in mouse bone marrow-derived macrophages. Infection of macrophages with N15m1, which expresses an unstable nsp15, or N15m3, which expresses a catalysis-deficient nsp15, activated MDA5, PKR, and the OAS/RNase L system, resulting in an early, robust induction of type I IFN, PKR-mediated apoptosis, and RNA degradation. Immunofluorescence imaging of nsp15 mutant virus-infected macrophages revealed significant dispersal of dsRNA early during infection, whereas in WT virus-infected cells, the majority of the dsRNA was associated with replication complexes. The loss of nsp15 activity also resulted in greatly attenuated disease in mice and stimulated a protective immune response. Taken together, our findings demonstrate that coronavirus nsp15 is critical for evasion of host dsRNA sensors in macrophages and reveal that modulating nsp15 stability and activity is a strategy for generating live-attenuated vaccines.</p>
<p align="right"><i>score: 162</i></p>

</div><br /><br /><br /><div class="card"><h3>Natural Mutations in the Receptor Binding Domain of Spike Glycoprotein Determine the Reactivity of Cross-Neutralization between Palm Civet Coronavirus and Severe Acute Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome (SARS) outbreak of 2002 and 2003 occurred as a result of zoonotic transmission. Coronavirus (CoV) found in naturally infected palm civet (civet-CoV) represents the closest genetic relative to SARS-CoV, but the degree and the determinants of cross-neutralization among these viruses remain to be investigated. Studies indicate that the receptor binding domain (RBD) of the SARS-CoV spike (S) glycoprotein contains major determinants for viral entry and neutralization. We aim to characterize the impact of natural mutations within the RBDs of civet-CoVs on viral entry and cross-neutralization. In this study, the S glycoprotein genes were recovered from naturally infected civets in central China (Hubei province), extending the geographic distribution of civet-CoV beyond the southeastern province of Guangdong. Moreover, pseudoviruses generated in our laboratory with four civet S genes, each with a distinct RBD, infected cells expressing human receptor angiotensin-converting enzyme 2, but with 90 to 95% less efficiency compared to that of SARS-CoV. These four civet S genes were also constructed as DNA vaccines to immunize mice. Immunized sera elicited against most civet S glycoproteins displayed potent neutralizing activities against autologous viruses but were much less efficient (50% inhibitory concentration, 20- to 40-fold) at neutralizing SARS-CoV and vice versa. Convalescence-phase sera from humans were similarly ineffective against the dominant civet pseudovirus. Our findings suggest that the design of SARS vaccine should consider not only preventing the reemergence of SARS-CoV but also providing cross-protection, thus interrupting zoonotic transmission of a group of genetically divergent civet CoVs of broad geographic origin.</p>
<p align="right"><i>score: 161</i></p>

</div><br /><br /><br /><div class="card"><h3>Vulnerabilities in coronavirus glycan shields despite extensive glycosylation</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-21</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses (CoVs) are zoonotic pathogens with high fatality rates and pandemic potential. Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.</p>
<p align="right"><i>score: 161</i></p>

</div><br /><br /><br /><div class="card"><h3>Subunit Vaccines Against Emerging Pathogenic Human Coronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Seven coronaviruses (CoVs) have been isolated from humans so far. Among them, three emerging pathogenic CoVs, including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and a newly identified CoV (2019-nCoV), once caused or continue to cause severe infections in humans, posing significant threats to global public health. SARS-CoV infection in humans (with about 10% case fatality rate) was first reported from China in 2002, while MERS-CoV infection in humans (with about 34.4% case fatality rate) was first reported from Saudi Arabia in June 2012. 2019-nCoV was first reported from China in December 2019, and is currently infecting more than 70000 people (with about 2.7% case fatality rate). Both SARS-CoV and MERS-CoV are zoonotic viruses, using bats as their natural reservoirs, and then transmitting through intermediate hosts, leading to human infections. Nevertheless, the intermediate host for 2019-nCoV is still under investigation and the vaccines against this new CoV have not been available. Although a variety of vaccines have been developed against infections of SARS-CoV and MERS-CoV, none of them has been approved for use in humans. In this review, we have described the structure and function of key proteins of emerging human CoVs, overviewed the current vaccine types to be developed against SARS-CoV and MERS-CoV, and summarized recent advances in subunit vaccines against these two pathogenic human CoVs. These subunit vaccines are introduced on the basis of full-length spike (S) protein, receptor-binding domain (RBD), non-RBD S protein fragments, and non-S structural proteins, and the potential factors affecting these subunit vaccines are also illustrated. Overall, this review will be helpful for rapid design and development of vaccines against the new 2019-nCoV and any future CoVs with pandemic potential. This review was written for the topic of Antivirals for Emerging Viruses: Vaccines and Therapeutics in the Virology section of Frontiers in Microbiology.</p>
<p align="right"><i>score: 159</i></p>

</div><br /><br /><br /><div class="card"><h3>Passive immunization with influenza haemagglutinin specific monoclonal antibodies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The isolation of broadly neutralising antibodies against the influenza haemagglutinin has spurred investigation into their clinical potential, and has led to advances in influenza virus biology and universal influenza vaccine development. Studies in animal models have been invaluable for demonstrating the prophylactic and therapeutic efficacy of broadly neutralising antibodies, for comparisons with antiviral drugs used as the standard of care, and for defining their mechanism of action and potential role in providing protection from airborne infection.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 11</span></i><p style="text-align: justify; text-justify: inter-word;">Live-attenuated RNA virus vaccines are efficacious but subject to reversion to virulence. Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy; increased fidelity is attenuating for some viruses. Coronavirus replication fidelity is approximately 20-fold greater than that of other RNA viruses and is mediated by a 3′-5′ exonuclease activity (ExoN) that likely functions in RNA proofreading. In this study, we demonstrate that engineered inactivation of SARS-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity in vivo and attenuation of pathogenesis in young, aged, and immunocompromised mouse models of human SARS. The ExoN inactivation genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo. Our approach represents a strategy with potential for broad applications for the stable attenuation of coronaviruses and possibly other RNA viruses.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 11</span></i><p style="text-align: justify; text-justify: inter-word;">Live-attenuated RNA virus vaccines are efficacious but subject to reversion to virulence. Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy; increased fidelity is attenuating for some viruses. Coronavirus replication fidelity is approximately 20-fold greater than that of other RNA viruses and is mediated by a 3′-5′ exonuclease activity (ExoN) that likely functions in RNA proofreading. In this study, we demonstrate that engineered inactivation of SARS-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity in vivo and attenuation of pathogenesis in young, aged, and immunocompromised mouse models of human SARS. The ExoN inactivation genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo. Our approach represents a strategy with potential for broad applications for the stable attenuation of coronaviruses and possibly other RNA viruses.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of a novel coronavirus associated with an ongoing outbreak of pneumonia (Covid-2019) resulted in infections of more than 72,000 people and claimed over 1,800 lives. Coronavirus spike (S) glycoprotein trimers promote entry into cells and are the main target of the humoral immune response. We show here that SARS-CoV-2 S mediates entry in VeroE6 cells and in BHK cells transiently transfected with human ACE2, establishing ACE2 as a functional receptor for this novel coronavirus. We further demonstrate that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, which correlates with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S1/S2 subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and other SARS-related CoVs. We determined a cryo-electron microscopy structure of the SARS-CoV-2 S ectodomain trimer, demonstrating spontaneous opening of the receptor-binding domain, and providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal sera potently inhibited SARS-CoV-2 S-mediated entry into target cells, thereby indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.</p>
<p align="right"><i>score: 157</i></p>

</div><br /><br /><br /><div class="card"><h3>A Human Monoclonal Antibody with Neutralizing Activity against Highly Divergent Influenza Subtypes</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Dec 5</span></i><p style="text-align: justify; text-justify: inter-word;">The interest in broad-range anti-influenza A monoclonal antibodies (mAbs) has recently been strengthened by the identification of anti-hemagglutinin (HA) mAbs endowed with heterosubtypic neutralizing activity to be used in the design of “universal” prophylactic or therapeutic tools. However, the majority of the single mAbs described to date do not bind and neutralize viral isolates belonging to highly divergent subtypes clustering into the two different HA-based influenza phylogenetic groups: the group 1 including, among others, subtypes H1, H2, H5 and H9 and the group 2 including, among others, H3 subtype. Here, we describe a human mAb, named PN-SIA28, capable of binding and neutralizing all tested isolates belonging to phylogenetic group 1, including H1N1, H2N2, H5N1 and H9N2 subtypes and several isolates belonging to group 2, including H3N2 isolates from the first period of the 1968 pandemic. Therefore, PN-SIA28 is capable of neutralizing isolates belonging to subtypes responsible of all the reported pandemics, as well as other subtypes with pandemic potential. The region recognized by PN-SIA28 has been identified on the stem region of HA and includes residues highly conserved among the different influenza subtypes. A deep characterization of PN-SIA28 features may represent a useful help in the improvement of available anti-influenza therapeutic strategies and can provide new tools for the development of universal vaccinal strategies.</p>
<p align="right"><i>score: 157</i></p>

</div><br /><br /><br /><div class="card"><h3>Prefusion structure of a human coronavirus spike protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">HKU1 is a human betacoronavirus that causes mild yet prevalent respiratory disease(1) and is related to the zoonotic SARS(2) and MERS(3) betacoronaviruses that have high fatality rates and pandemic potential. Cell tropism and host range is determined in part by the coronavirus spike (S) protein(4), which binds cellular receptors and mediates membrane fusion. As the largest known class I fusion protein, its size and extensive glycosylation have hindered structural studies of the full ectodomain, thus preventing a molecular understanding of its function and limiting development of effective interventions. Here we present the 4.0 Å resolution structure of the trimeric HKU1 S protein determined using single-particle cryo-electron microscopy. In the prefusion conformation, the receptor-binding subunits, S1, rest atop the fusion-mediating subunits, S2, preventing their conformational rearrangement. Surprisingly, the S1 C-terminal domains are interdigitated and form extensive quaternary interactions that occlude surfaces known to bind protein receptors in other coronaviruses. These features, along with the location of the two protease sites known to be important for coronavirus entry, provide a structural basis to support a model of membrane fusion mediated by progressive S protein destabilization through receptor binding and proteolytic cleavage. Additionally, these studies should serve as a foundation for the structure-based design of betacoronavirus vaccine immunogens.</p>
<p align="right"><i>score: 157</i></p>

</div><br /><br /><br /><div class="card"><h3>A Recombinant Vaccine of H5N1 HA1 Fused with Foldon and Human IgG Fc Induced Complete Cross-Clade Protection against Divergent H5N1 Viruses</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Jan 27</span></i><p style="text-align: justify; text-justify: inter-word;">Development of effective vaccines to prevent influenza, particularly highly pathogenic avian influenza (HPAI) caused by influenza A virus (IAV) subtype H5N1, is a challenging goal. In this study, we designed and constructed two recombinant influenza vaccine candidates by fusing hemagglutinin 1 (HA1) fragment of A/Anhui/1/2005(H5N1) to either Fc of human IgG (HA1-Fc) or foldon plus Fc (HA1-Fdc), and evaluated their immune responses and cross-protection against divergent strains of H5N1 virus. Results showed that these two recombinant vaccines induced strong immune responses in the vaccinated mice, which specifically reacted with HA1 proteins and an inactivated heterologous H5N1 virus. Both proteins were able to cross-neutralize infections by one homologous strain (clade 2.3) and four heterologous strains belonging to clades 0, 1, and 2.2 of H5N1 pseudoviruses as well as three heterologous strains (clades 0, 1, and 2.3.4) of H5N1 live virus. Importantly, immunization with these two vaccine candidates, especially HA1-Fdc, provided complete cross-clade protection against high-dose lethal challenge of different strains of H5N1 virus covering clade 0, 1, and 2.3.4 in the tested mouse model. This study suggests that the recombinant fusion proteins, particularly HA1-Fdc, could be developed into an efficacious universal H5N1 influenza vaccine, providing cross-protection against infections by divergent strains of highly pathogenic H5N1 virus.</p>
<p align="right"><i>score: 155</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Serologic and Antigenic Relationships Between Middle Eastern Respiratory Syndrome Coronavirus and Other Coronaviruses to Develop Vaccine Platforms for the Rapid Response to Emerging Coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Background. Middle East respiratory syndrome coronavirus (MERS-CoV) emerged in 2012, causing severe acute respiratory disease and pneumonia, with 44% mortality among 136 cases to date. Design of vaccines to limit the virus spread or diagnostic tests to track newly emerging strains requires knowledge of antigenic and serologic relationships between MERS-CoV and other CoVs. Methods. Using synthetic genomics and Venezuelan equine encephalitis virus replicons (VRPs) expressing spike and nucleocapsid proteins from MERS-CoV and other human and bat CoVs, we characterize the antigenic responses (using Western blot and enzyme-linked immunosorbent assay) and serologic responses (using neutralization assays) against 2 MERS-CoV isolates in comparison with those of other human and bat CoVs. Results. Serologic and neutralization responses against the spike glycoprotein were primarily strain specific, with a very low level of cross-reactivity within or across subgroups. CoV N proteins within but not across subgroups share cross-reactive epitopes with MERS-CoV isolates. Our findings were validated using a convalescent-phase serum specimen from a patient infected with MERS-CoV (NA 01) and human antiserum against SARS-CoV, human CoV NL63, and human CoV OC43. Conclusions. Vaccine design for emerging CoVs should involve chimeric spike protein containing neutralizing epitopes from multiple virus strains across subgroups to reduce immune pathology, and a diagnostic platform should include a panel of nucleocapsid and spike proteins from phylogenetically distinct CoVs.</p>
<p align="right"><i>score: 155</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological Screening for Coronavirus Infections in Cats</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 13</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are widespread among mammals and birds and known for their potential for cross-species transmission. In cats, infections with feline coronaviruses (FCoVs) are common. Several non-feline coronaviruses have been reported to infect feline cells as well as cats after experimental infection, supported by their ability to engage the feline receptor ortholog for cell entry. However, whether cats might become naturally infected with CoVs of other species is unknown. We analyzed coronavirus infections in cats by serological monitoring. In total 137 cat serum samples and 25 FCoV type 1 or type 2-specific antisera were screened for the presence of antibodies against the S1 receptor binding subunit of the CoV spike protein, which is immunogenic and possesses low amino acid sequence identity among coronavirus species. Seventy-eight sera were positive for antibodies that recognized one or more coronavirus S1s whereas 1 serum exclusively reacted with human coronavirus 229E (HCoV-229E) and two sera exclusively reacted with porcine delta coronavirus (PDCoV). We observed antigenic cross-reactivity between S1s of type 1 and type 2 FCoVs, and between FCoV type 1 and porcine epidemic diarrhea virus (PEDV). Domain mapping of antibody epitopes indicated the presence of conserved epitope(s) particularly in the CD domains of S1. The cross-reactivity of FCoV type 1 and PEDV was also observed at the level of virus neutralization. To conclude, we provide the first evidence of antigenic cross-reactivity among S1 proteins of coronaviruses, which should be considered in the development of serological diagnoses. In addition, the potential role of cats in cross-species transmission of coronaviruses cannot be excluded.</p>
<p align="right"><i>score: 155</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological Screening for Coronavirus Infections in Cats</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 13</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are widespread among mammals and birds and known for their potential for cross-species transmission. In cats, infections with feline coronaviruses (FCoVs) are common. Several non-feline coronaviruses have been reported to infect feline cells as well as cats after experimental infection, supported by their ability to engage the feline receptor ortholog for cell entry. However, whether cats might become naturally infected with CoVs of other species is unknown. We analyzed coronavirus infections in cats by serological monitoring. In total 137 cat serum samples and 25 FCoV type 1 or type 2-specific antisera were screened for the presence of antibodies against the S1 receptor binding subunit of the CoV spike protein, which is immunogenic and possesses low amino acid sequence identity among coronavirus species. Seventy-eight sera were positive for antibodies that recognized one or more coronavirus S1s whereas 1 serum exclusively reacted with human coronavirus 229E (HCoV-229E) and two sera exclusively reacted with porcine delta coronavirus (PDCoV). We observed antigenic cross-reactivity between S1s of type 1 and type 2 FCoVs, and between FCoV type 1 and porcine epidemic diarrhea virus (PEDV). Domain mapping of antibody epitopes indicated the presence of conserved epitope(s) particularly in the CD domains of S1. The cross-reactivity of FCoV type 1 and PEDV was also observed at the level of virus neutralization. To conclude, we provide the first evidence of antigenic cross-reactivity among S1 proteins of coronaviruses, which should be considered in the development of serological diagnoses. In addition, the potential role of cats in cross-species transmission of coronaviruses cannot be excluded.</p>
<p align="right"><i>score: 155</i></p>

</div><br /><br /><br /><div class="card"><h3>T Cell Responses to Whole SARS Coronavirus in Humans</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Effective vaccines should confer long-term protection against future outbreaks of severe acute respiratory syndrome (SARS) caused by a novel zoonotic coronavirus (SARS-CoV) with unknown animal reservoirs. We conducted a cohort study examining multiple parameters of immune responses to SARS-CoV infection, aiming to identify the immune correlates of protection. We used a matrix of overlapping peptides spanning whole SARS-CoV proteome to determine T cell responses from 128 SARS convalescent samples by ex vivo IFN-γ ELISPOT assays. Approximately 50% of convalescent SARS patients were positive for T cell responses, and 90% possessed strongly neutralizing Abs. Fifty-five novel T cell epitopes were identified, with spike protein dominating total T cell responses. CD8(+) T cell responses were more frequent and of a greater magnitude than CD4(+) T cell responses (p < 0.001). Polychromatic cytometry analysis indicated that the virus-specific T cells from the severe group tended to be a central memory phenotype (CD27(+)/CD45RO(+)) with a significantly higher frequency of polyfunctional CD4(+) T cells producing IFN-γ, TNF-α, and IL-2, and CD8(+) T cells producing IFN-γ, TNF-α, and CD107a (degranulation), as compared with the mild-moderate group. Strong T cell responses correlated significantly (p < 0.05) with higher neutralizing Ab. The serum cytokine profile during acute infection indicated a significant elevation of innate immune responses. Increased Th2 cytokines were observed in patients with fatal infection. Our study provides a roadmap for the immunogenicity of SARS-CoV and types of immune responses that may be responsible for the virus clearance, and should serve as a benchmark for SARS-CoV vaccine design and evaluation.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>Roadmap to developing a recombinant coronavirus S protein receptor-binding domain vaccine for severe acute respiratory syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A subunit vaccine, RBD-S, is under development to prevent severe acute respiratory syndrome (SARS) caused by SARS coronavirus (SARS-CoV), which is classified by the US NIH as a category C pathogen. This vaccine is comprised of a recombinant receptor-binding domain (RBD) of the SARS-CoV spike (S) protein and formulated on alum, together with a synthetic glucopyranosyl lipid A. The vaccine would induce neutralizing antibodies without causing Th2-type immunopathology. Vaccine development is being led by the nonprofit product development partnership; Sabin Vaccine Institute and Texas Children’s Hospital Center for Vaccine Development in collaboration with two academic partners (the New York Blood Center and University of Texas Medical Branch); an industrial partner (Immune Design Corporation); and Walter Reed Army Institute of Research. A roadmap for the product development of the RBD-S SARS vaccine is outlined with a goal to manufacture the vaccine for clinical testing within the next 5 years.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of serologic and antigenic relationships between middle eastern respiratory syndrome coronavirus and other coronaviruses to develop vaccine platforms for the rapid response to emerging coronaviruses</h3>
<i class="tooltip">2014<span class="tooltiptext">2014</span></i><p style="text-align: justify; text-justify: inter-word;">Background: Middle East respiratory syndrome coronavirus (MERS-CoV) emerged in 2012, causing severe acute respiratory disease and pneumonia, with 44% mortality among 136 cases to date. Design of vaccines to limit the virus spread or diagnostic tests to track newly emerging strains requires knowledge of antigenic and serologic relationships between MERS-CoV and other CoVs.Methods:  Using synthetic genomics and Venezuelan equine encephalitis virus replicons (VRPs) expressing spike and nucleocapsid proteins from MERS-CoV and other human and bat CoVs, we characterize the antigenic responses (using Western blot and enzyme-linked immunosorbent assay) and serologic responses (using neutralization assays) against 2 MERS-CoV isolates in comparison with those of other human and bat CoVs.Results:  Serologic and neutralization responses against the spike glycoprotein were primarily strain specific, with a very low level of cross-reactivity within or across subgroups. CoV N proteins within but not across subgroups share cross-reactive epitopes with MERS-CoV isolates. Our findings were validated using a convalescent-phase serum specimen from a patient infected with MERS-CoV (NA 01) and human antiserum against SARS-CoV, human CoV NL63, and human CoV OC43.Conclusions:  Vaccine design for emerging CoVs should involve chimeric spike protein containing neutralizing epitopes from multiple virus strains across subgroups to reduce immune pathology, and a diagnostic platform should include a panel of nucleocapsid and spike proteins from phylogenetically distinct CoVs.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses: An Overview of Their Replication and Pathogenesis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs), enveloped positive-sense RNA viruses, are characterized by club-like spikes that project from their surface, an unusually large RNA genome, and a unique replication strategy. Coronaviruses cause a variety of diseases in mammals and birds ranging from enteritis in cows and pigs and upper respiratory disease chickens to potentially lethal human respiratory infections. Here we provide a brief introduction to coronaviruses discussing their replication and pathogenicity, and current prevention and treatment strategies. We will also discuss the outbreaks of the highly pathogenic Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the recently identified Middle Eastern Respiratory Syndrome Coronavirus (MERS-CoV).</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Jumping species – A Mechanism for Coronavirus Persistence and Survival</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Zoonotic transmission of novel viruses represents a significant threat to global public health and is fueled by globalization, the loss of natural habitats, and exposure to new hosts. For coronaviruses (CoVs), broad diversity exists within bat populations and uniquely positions them to seed future emergence events. In this review, we explore the host and viral dynamics that shape these CoV populations for survival, amplification, and possible emergence in novel hosts.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential Interventions for Novel Coronavirus in China: A Systemic Review</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of a novel coronavirus (COVID-19 or 2019-CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA virus infection and we have found that general treatments, coronavirus-specific treatments, and antiviral treatments should be useful in fighting COVID-19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID-19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID-19 if the infection is uncontrollable. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential T-cell and B-cell Epitopes of 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-21</span></i><p style="text-align: justify; text-justify: inter-word;">As of Feb 16th 2020, 2019-nCoV has infected more than 51,857 people across 26 countries and claimed 1666 lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for 2019-nCoV or any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. We validated our computational pipeline with SARS-CoV experimental data.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombinant Modified Vaccinia Virus Ankara Expressing the Spike Glycoprotein of Severe Acute Respiratory Syndrome Coronavirus Induces Protective Neutralizing Antibodies Primarily Targeting the Receptor Binding Region</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Immunization with a killed or inactivated viral vaccine provides significant protection in animals against challenge with certain corresponding pathogenic coronaviruses (CoVs). However, the promise of this approach in humans is hampered by serious concerns over the risk of leaking live severe acute respiratory syndrome (SARS) viruses. In this study, we generated a SARS vaccine candidate by using the live-attenuated modified vaccinia virus Ankara (MVA) as a vector. The full-length SARS-CoV envelope Spike (S) glycoprotein gene was introduced into the deletion III region of the MVA genome. The newly generated recombinant MVA, ADS-MVA, is replication incompetent in mammalian cells and highly immunogenic in terms of inducing potent neutralizing antibodies in mice, rabbits, and monkeys. After two intramuscular vaccinations with ADS-MVA alone, the 50% inhibitory concentration in serum was achieved with reciprocal sera dilutions of more than 1,000- to 10,000-fold in these animals. Using fragmented S genes as immunogens, we also mapped a neutralizing epitope in the region of N-terminal 400 to 600 amino acids of the S glycoprotein (S400-600), which overlaps with the angiotensin-converting enzyme 2 (ACE2) receptor-binding region (RBR; S318-510). Moreover, using a recombinant soluble RBR-Fc protein, we were able to absorb and remove the majority of the neutralizing antibodies despite observing that the full S protein tends to induce a broader spectrum of neutralizing activities in comparison with fragmented S proteins. Our data suggest that a major mechanism for neutralizing SARS-CoV likely occurs through blocking the interaction between virus and the cellular receptor ACE2. In addition, ADS-MVA induced potent immune responses which very likely protected Chinese rhesus monkeys from pathogenic SARS-CoV challenge.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccines to prevent severe acute respiratory syndrome coronavirus-induced disease</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">An important effort has been performed after the emergence of severe acute respiratory syndrome (SARS) epidemic in 2003 to diagnose and prevent virus spreading. Several types of vaccines have been developed including inactivated viruses, subunit vaccines, virus-like particles (VLPs), DNA vaccines, heterologous expression systems, and vaccines derived from SARS-CoV genome by reverse genetics. This review describes several aspects essential to develop SARS-CoV vaccines, such as the correlates of protection, virus serotypes, vaccination side effects, and bio-safeguards that can be engineered into recombinant vaccine approaches based on the SARS-CoV genome. The production of effective and safe vaccines to prevent SARS has led to the development of promising vaccine candidates, in contrast to the design of vaccines for other coronaviruses, that in general has been less successful. After preclinical trials in animal models, efficacy and safety evaluation of the most promising vaccine candidates described has to be performed in humans.</p>
<p align="right"><i>score: 152</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic Assembly of a Full-Length Infectious Clone of Human Coronavirus NL63</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Historically, coronaviruses were predominantly associated with mild upper respiratory disease in humans. More recently, three novel coronaviruses associated with severe human respiratory disease were found, including (i) the severe acute respiratory syndrome coronavirus, associated with a significant atypical pneumonia and 10% mortality; (ii) HKU-1, associated with chronic pulmonary disease; and (iii) NL63, associated with both upper and lower respiratory tract disease in children and adults worldwide. These discoveries establish coronaviruses as important human pathogens and underscore the need for continued research toward the development of platforms that will enable genetic manipulation of the viral genome, allowing for rapid and rational development and testing of candidate vaccines, vaccine vectors, and therapeutics. In this report, we describe a reverse genetics system for NL63, whereby five contiguous cDNAs that span the entire genome were used to generate a full-length cDNA. Recombinant NL63 viruses which contained the expected marker mutations replicated as efficiently as the wild-type NL63 virus. In addition, we engineered the heterologous green fluorescent protein gene in place of open reading frame 3 (ORF3) of the NL63 clone, simultaneously creating a unique marker for NL63 infection and demonstrating that the ORF3 protein product is nonessential for the replication of NL63 in cell culture. The availability of the NL63 and NL63gfp clones and recombinant viruses provides powerful tools that will help advance our understanding of this important human pathogen.</p>
<p align="right"><i>score: 152</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental inoculation of cats with human coronavirus 229E and subsequent challenge with feline infectious peritonitis virus.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Minimal-disease cats exposed to live human coronavirus 229E developed homologous antibody responses that suggested little or no replication of the virus in inoculated animals. Oronasal and subcutaneous inoculation of coronavirus 229E did not elicit an antibody response by heterologous (transmissible gastroenteritis virus, canine coronavirus) neutralization or by heterologous (transmissible gastroenteritis virus) kinetics-based enzyme-linked immunosorbent assay. No clinical signs attributable to coronavirus 229E were seen in inoculated cats. Although the number of animals in each of the five experimental groups was small (n = 2), antibodies produced in response to the virus did not appear to sensitize cats to subsequent feline infectious peritonitis virus challenge, but neither did they cross-protect cats against the challenge dose.</p>
<p align="right"><i>score: 151</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of an immunodominant linear neutralization domain on the S2 portion of the murine coronavirus spike glycoprotein and evidence that it forms part of complex tridimensional structure.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Numerous studies have demonstrated that the spike glycoprotein of coronaviruses bears major determinants of pathogenesis. To elucidate the antigenic structure of the protein, a panel of monoclonal antibodies was studied by competitive ELISA, and their reactivities were assayed against fragments of the murine coronavirus murine hepatitis virus strain A59 S gene expressed in prokaryotic vectors. An immunodominant linear domain was localized within the predicted stalk, S2, of the peplomer. It is recognized by several neutralizing antibodies. Other domains were also identified near the proteolytic cleavage site, in the predicted globular head, S1, and in another part of the stalk. Furthermore, competition results suggest that the immunodominant functional domain forms part of a complex three-dimensional structure. Surprisingly, some antibodies which have no antiviral biological activities were shown to bind the immunodominant neutralization domain.</p>
<p align="right"><i>score: 151</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibody response to equine coronavirus in horses inoculated with a bovine coronavirus vaccine</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 6</span></i><p style="text-align: justify; text-justify: inter-word;">A vaccine for equine coronavirus (ECoV) is so far unavailable. Bovine coronavirus (BCoV) is antigenically related to ECoV; it is therefore possible that BCoV vaccine will induce antibodies against ECoV in horses. This study investigated antibody response to ECoV in horses inoculated with BCoV vaccine. Virus neutralization tests showed that antibody titers against ECoV increased in all six horses tested at 14 days post inoculation, although the antibody titers were lower against ECoV than against BCoV. This study showed that BCoV vaccine provides horses with antibodies against ECoV to some extent. It is unclear whether antibodies provided by BCoV vaccine are effective against ECoV, and therefore ECoV challenge studies are needed to evaluate efficacy of the vaccine in the future.</p>
<p align="right"><i>score: 151</i></p>

</div><br /><br /><br /><div class="card"><h3>Emerging Respiratory Viruses: Challenges and Vaccine Strategies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The current threat of avian influenza to the human population, the potential for the reemergence of severe acute respiratory syndrome (SARS)-associated coronavirus, and the identification of multiple novel respiratory viruses underline the necessity for the development of therapeutic and preventive strategies to combat viral infection. Vaccine development is a key component in the prevention of widespread viral infection and in the reduction of morbidity and mortality associated with many viral infections. In this review we describe the different approaches currently being evaluated in the development of vaccines against SARS-associated coronavirus and avian influenza viruses and also highlight the many obstacles encountered in the development of these vaccines. Lessons learned from current vaccine studies, coupled with our increasing knowledge of the host and viral factors involved in viral pathogenesis, will help to increase the speed with which efficacious vaccines targeting newly emerging viral pathogens can be developed.</p>
<p align="right"><i>score: 150</i></p>

</div><br /><br /><br /><div class="card"><h3>Priming with rAAV encoding RBD of SARS-CoV S protein and boosting with RBD-specific peptides for T cell epitopes increase T cell responses and provide protection against SRAS-CoV infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Development of vaccines against severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) is crucial in the prevention of SARS reemergence. The receptor-binding domain (RBD) of SARS-CoV spike (S) protein is an important target in developing safe and effective SARS vaccines. Our previous study has demonstrated that vaccination with adeno-associated virus encoding RBD (RBD-rAAV) induces high titer of neutralizing antibodies. In this study, we further assessed the immune responses and protective effect of the immunization with RBD-rAAV prime/RBD-specific T cell peptide boost, as compared to those of vaccinations with RBD-rAAV and RBD-peptides alone. Our results indicated that RBD-rAAV prime/RBD-peptide boost induced similar Th1 and neutralizing antibody responses, but stronger Th2 and CTL responses than RBD-rAAV prime/boost. The resulting immune responses protected the vaccinated mice from subsequent SARS-CoV challenge, which was evidenced by lower level of viral replication in mouse lung tissues. However, no significant immune responses and protective effect were detected in mice vaccinated with RBD-peptides or blank AAV alone. Since T cell epitopes are highly conserved and boosting with peptides may induce the production of effector memory T cells, our results suggest that the vaccination protocol used may be ideal for providing effective, universal and long-term protection against SARS-CoV infection.</p>
<p align="right"><i>score: 148</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Viral Transmission Behavior via Protein Intrinsic Disorder Prediction: Coronaviruses</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 14</span></i><p style="text-align: justify; text-justify: inter-word;">Besides being a common threat to farm animals and poultry, coronavirus (CoV) was responsible for the human severe acute respiratory syndrome (SARS) epidemic in 2002–4. However, many aspects of CoV behavior, including modes of its transmission, are yet to be fully understood. We show that the amount and the peculiarities of distribution of the protein intrinsic disorder in the viral shell can be used for the efficient analysis of the behavior and transmission modes of CoV. The proposed model allows categorization of the various CoVs by the peculiarities of disorder distribution in their membrane (M) and nucleocapsid (N). This categorization enables quick identification of viruses with similar behaviors in transmission, regardless of genetic proximity. Based on this analysis, an empirical model for predicting the viral transmission behavior is developed. This model is able to explain some behavioral aspects of important coronaviruses that previously were not fully understood. The new predictor can be a useful tool for better epidemiological, clinical, and structural understanding of behavior of both newly emerging viruses and viruses that have been known for a long time. A potentially new vaccine strategy could involve searches for viral strains that are characterized by the evolutionary misfit between the peculiarities of the disorder distribution in their shells and their behavior.</p>
<p align="right"><i>score: 148</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a heat-stable and orally delivered recombinant M2e-expressing B. subtilis spore-based influenza vaccine</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Highly conserved ectodomain of influenza virus M2 protein (M2e) is an important target for the development of universal influenza vaccines. Today, the use of chemical or genetic fusion constructs have been undertaken to overcome the low immunogenicity of M2e in vaccine formulation. However, current M2e vaccines are neither orally delivered nor heat-stable. In this study, we evaluated the immune efficacy of an orally delivered recombinant M2e vaccine containing 3 molcules of M2e consensus sequence of influenza A viruses, termed RSM2e3. To accomplish this, CotB, a spore coat of Bacillus subtilis (B. subtilis), was used as a fusion partner, and heat-stable nonpathogenic B. subtilis spores were used as the carrier. Our results showed that CotB-M2e3 fusion had no effect on spore structure or function in the resultant recombinant RSM2e3 strain and that heterologous influenza virus M2e protein was successfully displayed on the surface of the recombinant RSM2e3 spore. Importantly, recombinant RSM2e3 spores elicited strong and long-term M2e-specific systemic and mucosal immune responses, completely protecting immunized mice from lethal challenge of A/PR/8/34(H1N1) influenza virus. Taken together, our study forms a solid basis for the development of a novel orally delivered and heat-stable influenza vaccine based on B. subtilis spore surface display.</p>
<p align="right"><i>score: 147</i></p>

</div><br /><br /><br /><div class="card"><h3>Engineering the largest RNA virus genome as an infectious bacterial artificial chromosome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The construction of cDNA clones encoding large-size RNA molecules of biological interest, like coronavirus genomes, which are among the largest mature RNA molecules known to biology, has been hampered by the instability of those cDNAs in bacteria. Herein, we show that the application of two strategies, cloning of the cDNAs into a bacterial artificial chromosome and nuclear expression of RNAs that are typically produced within the cytoplasm, is useful for the engineering of large RNA molecules. A cDNA encoding an infectious coronavirus RNA genome has been cloned as a bacterial artificial chromosome. The rescued coronavirus conserved all of the genetic markers introduced throughout the sequence and showed a standard mRNA pattern and the antigenic characteristics expected for the synthetic virus. The cDNA was transcribed within the nucleus, and the RNA translocated to the cytoplasm. Interestingly, the recovered virus had essentially the same sequence as the original one, and no splicing was observed. The cDNA was derived from an attenuated isolate that replicates exclusively in the respiratory tract of swine. During the engineering of the infectious cDNA, the spike gene of the virus was replaced by the spike gene of an enteric isolate. The synthetic virus replicated abundantly in the enteric tract and was fully virulent, demonstrating that the tropism and virulence of the recovered coronavirus can be modified. This demonstration opens up the possibility of employing this infectious cDNA as a vector for vaccine development in human, porcine, canine, and feline species susceptible to group 1 coronaviruses.</p>
<p align="right"><i>score: 147</i></p>

</div><br /><br /><br /><div class="card"><h3>A Double-Inactivated Severe Acute Respiratory Syndrome Coronavirus Vaccine Provides Incomplete Protection in Mice and Induces Increased Eosinophilic Proinflammatory Pulmonary Response upon Challenge</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus (SARS-CoV) is an important emerging virus that is highly pathogenic in aged populations and is maintained with great diversity in zoonotic reservoirs. While a variety of vaccine platforms have shown efficacy in young-animal models and against homologous viral strains, vaccine efficacy has not been thoroughly evaluated using highly pathogenic variants that replicate the acute end stage lung disease phenotypes seen during the human epidemic. Using an adjuvanted and an unadjuvanted double-inactivated SARS-CoV (DIV) vaccine, we demonstrate an eosinophilic immunopathology in aged mice comparable to that seen in mice immunized with the SARS nucleocapsid protein, and poor protection against a nonlethal heterologous challenge. In young and 1-year-old animals, we demonstrate that adjuvanted DIV vaccine provides protection against lethal disease in young animals following homologous and heterologous challenge, although enhanced immune pathology and eosinophilia are evident following heterologous challenge. In the absence of alum, DIV vaccine performed poorly in young animals challenged with lethal homologous or heterologous strains. In contrast, DIV vaccines (both adjuvanted and unadjuvanted) performed poorly in aged-animal models. Importantly, aged animals displayed increased eosinophilic immune pathology in the lungs and were not protected against significant virus replication. These data raise significant concerns regarding DIV vaccine safety and highlight the need for additional studies of the molecular mechanisms governing DIV-induced eosinophilia and vaccine failure, especially in the more vulnerable aged-animal models of human disease.</p>
<p align="right"><i>score: 146</i></p>

</div><br /><br /><br /><div class="card"><h3>Peptide Nanoparticles as Novel Immunogens: Design and Analysis of a Prototypic Severe Acute Respiratory Syndrome Vaccine</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) is an infectious disease caused by a novel coronavirus that cost nearly 800 lives. While there have been no recent outbreaks of the disease, the threat remains as SARS coronavirus (SARS-CoV) like strains still exist in animal reservoirs. Therefore, the development of a vaccine against SARS is in grave need. Here, we have designed and produced a prototypic SARS vaccine: a self-assembling polypeptide nanoparticle that repetitively displays a SARS B-cell epitope from the C-terminal heptad repeat of the virus’ spike protein. Biophysical analyses with circular dichroism, transmission electron microscopy and dynamic light scattering confirmed the computational design showing α-helcial nanoparticles with sizes of about 25 nm. Immunization experiments with no adjuvants were performed with BALB/c mice. An investigation of the binding properties of the elicited antibodies showed that they were highly conformation specific for the coiled-coil epitope because they specifically recognized the native trimeric conformation of C-terminal heptad repeat region. Consequently, the antisera exhibited neutralization activity in an in vitro infection inhibition assay. We conclude that these peptide nanoparticles represent a promising platform for vaccine design, in particular for diseases that are characterized by neutralizing epitopes with coiled-coil conformation such as SARS-CoV or other enveloped viruses.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>A synthetic consensus anti–spike protein DNA vaccine induces protective immunity against Middle East respiratory syndrome coronavirus in nonhuman primates</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">First identified in 2012, Middle East respiratory syndrome (MERS) is caused by an emerging human coronavirus, which is distinct from the severe acute respiratory syndrome coronavirus (SARS-CoV), and represents a novel member of the lineage C betacoronoviruses. Since its identification, MERS coronavirus (MERS-CoV) has been linked to more than 1372 infections manifesting with severe morbidity and, often, mortality (about 495 deaths) in the Arabian Peninsula, Europe, and, most recently, the United States. Human-to-human transmission has been documented, with nosocomial transmission appearing to be an important route of infection. The recent increase in cases of MERS in the Middle East coupled with the lack of approved antiviral therapies or vaccines to treat or prevent this infection are causes for concern. We report on the development of a synthetic DNA vaccine against MERS-CoV. An optimized DNA vaccine encoding the MERS spike protein induced potent cellular immunity and antigen-specific neutralizing antibodies in mice, macaques, and camels. Vaccinated rhesus macaques seroconverted rapidly and exhibited high levels of virus-neutralizing activity. Upon MERS viral challenge, all of the monkeys in the control-vaccinated group developed characteristic disease, including pneumonia. Vaccinated macaques were protected and failed to demonstrate any clinical or radiographic signs of pneumonia. These studies demonstrate that a consensus MERS spike protein synthetic DNA vaccine can induce protective responses against viral challenge, indicating that this strategy may have value as a possible vaccine modality against this emerging pathogen.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficacy of an Adjuvanted Middle East Respiratory Syndrome Coronavirus Spike Protein Vaccine in Dromedary Camels and Alpacas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV is present in dromedary camels throughout the Middle East and Africa. Dromedary camels are the primary zoonotic reservoir for human infections. Interruption of the zoonotic transmission chain from camels to humans, therefore, may be an effective strategy to control the ongoing MERS-CoV outbreak. Here we show that vaccination with an adjuvanted MERS-CoV Spike protein subunit vaccine confers complete protection from MERS-CoV disease in alpaca and results in reduced and delayed viral shedding in the upper airways of dromedary camels. Protection in alpaca correlates with high serum neutralizing antibody titers. Lower titers of serum neutralizing antibodies correlate with delayed and significantly reduced shedding in the nasal turbinates of dromedary camels. Together, these data indicate that induction of robust neutralizing humoral immune responses by vaccination of naïve animals reduces shedding that potentially could diminish the risk of zoonotic transmission.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficacy of an Adjuvanted Middle East Respiratory Syndrome Coronavirus Spike Protein Vaccine in Dromedary Camels and Alpacas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV is present in dromedary camels throughout the Middle East and Africa. Dromedary camels are the primary zoonotic reservoir for human infections. Interruption of the zoonotic transmission chain from camels to humans, therefore, may be an effective strategy to control the ongoing MERS-CoV outbreak. Here we show that vaccination with an adjuvanted MERS-CoV Spike protein subunit vaccine confers complete protection from MERS-CoV disease in alpaca and results in reduced and delayed viral shedding in the upper airways of dromedary camels. Protection in alpaca correlates with high serum neutralizing antibody titers. Lower titers of serum neutralizing antibodies correlate with delayed and significantly reduced shedding in the nasal turbinates of dromedary camels. Together, these data indicate that induction of robust neutralizing humoral immune responses by vaccination of naïve animals reduces shedding that potentially could diminish the risk of zoonotic transmission.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of the Mechanisms Causing Reversion to Virulence in an Attenuated SARS-CoV for the Design of a Genetically Stable Vaccine</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">A SARS-CoV lacking the full-length E gene (SARS-CoV-∆E) was attenuated and an effective vaccine. Here, we show that this mutant virus regained fitness after serial passages in cell culture or in vivo, resulting in the partial duplication of the membrane gene or in the insertion of a new sequence in gene 8a, respectively. The chimeric proteins generated in cell culture increased virus fitness in vitro but remained attenuated in mice. In contrast, during SARS-CoV-∆E passage in mice, the virus incorporated a mutated variant of 8a protein, resulting in reversion to a virulent phenotype. When the full-length E protein was deleted or its PDZ-binding motif (PBM) was mutated, the revertant viruses either incorporated a novel chimeric protein with a PBM or restored the sequence of the PBM on the E protein, respectively. Similarly, after passage in mice, SARS-CoV-∆E protein 8a mutated, to now encode a PBM, and also regained virulence. These data indicated that the virus requires a PBM on a transmembrane protein to compensate for removal of this motif from the E protein. To increase the genetic stability of the vaccine candidate, we introduced small attenuating deletions in E gene that did not affect the endogenous PBM, preventing the incorporation of novel chimeric proteins in the virus genome. In addition, to increase vaccine biosafety, we introduced additional attenuating mutations into the nsp1 protein. Deletions in the carboxy-terminal region of nsp1 protein led to higher host interferon responses and virus attenuation. Recombinant viruses including attenuating mutations in E and nsp1 genes maintained their attenuation after passage in vitro and in vivo. Further, these viruses fully protected mice against challenge with the lethal parental virus, and are therefore safe and stable vaccine candidates for protection against SARS-CoV.</p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombinant Mouse Hepatitis Virus Strain A59 from Cloned, Full-Length cDNA Replicates to High Titers In Vitro and Is Fully Pathogenic In Vivo</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Mouse hepatitis virus (MHV) is the prototype of group II coronaviruses and one of the most extensively studied coronaviruses. Here, we describe a reverse genetic system for MHV (strain A59) based upon the cloning of a full-length genomic cDNA in vaccinia virus. We show that the recombinant virus generated from cloned cDNA replicates to the same titers as the parental virus in cell culture (∼10(9) PFU/ml), has the same plaque morphology, and produces the same amounts and proportions of genomic and subgenomic mRNAs in virus-infected cells. In a mouse model of neurological infection, the recombinant and parental viruses are equally virulent, they replicate to the same titers in brain and liver, and they induce similar patterns of acute hepatitis, acute meningoencephalitis, and chronic demyelination. We also describe improvements in the use of the coronavirus reverse genetic system based on vaccinia virus cloning vectors. These modifications facilitate (i) the mutagenesis of cloned cDNA by using vaccinia virus-mediated homologous recombination and (ii) the rescue of recombinant coronaviruses by using a stable nucleocapsid protein-expressing cell line for the electroporation of infectious full-length genomes. Thus, our system represents a versatile and universal tool to study all aspects of MHV molecular biology and pathogenesis. We expect this system to provide valuable insights into the replication of group II coronaviruses that may lead to the development of novel strategies against coronavirus infections, including the related severe acute respiratory syndrome coronavirus.</p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>Early death after feline infectious peritonitis virus challenge due to recombinant vaccinia virus immunization.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The gene encoding the fusogenic spike protein of the coronavirus causing feline infectious peritonitis was recombined into the genome of vaccinia virus. The recombinant induced spike-protein-specific, in vitro neutralizing antibodies in mice. When kittens were immunized with the recombinant, low titers of neutralizing antibodies were obtained. After challenge with feline infectious peritonitis virus, these animals succumbed earlier than did the control group immunized with wild-type vaccinia virus (early death syndrome).</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus-Vectored Influenza Virus Vaccines</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 7</span></i><p style="text-align: justify; text-justify: inter-word;">Despite the availability of an inactivated vaccine that has been licensed for >50 years, the influenza virus continues to cause morbidity and mortality worldwide. Constant evolution of circulating influenza virus strains and the emergence of new strains diminishes the effectiveness of annual vaccines that rely on a match with circulating influenza strains. Thus, there is a continued need for new, efficacious vaccines conferring cross-clade protection to avoid the need for biannual reformulation of seasonal influenza vaccines. Recombinant virus-vectored vaccines are an appealing alternative to classical inactivated vaccines because virus vectors enable native expression of influenza antigens, even from virulent influenza viruses, while expressed in the context of the vector that can improve immunogenicity. In addition, a vectored vaccine often enables delivery of the vaccine to sites of inductive immunity such as the respiratory tract enabling protection from influenza virus infection. Moreover, the ability to readily manipulate virus vectors to produce novel influenza vaccines may provide the quickest path toward a universal vaccine protecting against all influenza viruses. This review will discuss experimental virus-vectored vaccines for use in humans, comparing them to licensed vaccines and the hurdles faced for licensure of these next-generation influenza virus vaccines.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Protein Engineering Strategies for the Development of Viral Vaccines and Immunotherapeutics</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Vaccines that elicit a protective broadly neutralizing antibody (bNAb) response and monoclonal antibody therapies are critical for the treatment and prevention of viral infections. However, isolation of protective neutralizing antibodies has been challenging for some viruses, notably those with high antigenic diversity or those that do not elicit a bNAb response in the course of natural infection. Here, we discuss recent work that employs protein engineering strategies to design immunogens that elicit bNAbs or engineer novel bNAbs. We highlight the use of rational, computational, and combinatorial strategies and assess the potential of these approaches for the development of new vaccines and immunotherapeutics.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural Basis for Potent Cross-Neutralizing Human Monoclonal Antibody Protection against Lethal Human and Zoonotic Severe Acute Respiratory Syndrome Coronavirus Challenge</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002, and detailed phylogenetic and epidemiological analyses have suggested that it originated from animals. The spike (S) glycoprotein has been identified as a major component of protective immunity, and 23 different amino acid changes were noted during the expanding epidemic. Using a panel of SARS-CoV recombinants bearing the S glycoproteins from isolates representing the zoonotic and human early, middle, and late phases of the epidemic, we identified 23 monoclonal antibodies (MAbs) with neutralizing activity against one or multiple SARS-CoV spike variants and determined the presence of at least six distinct neutralizing profiles in the SARS-CoV S glycoprotein. Four of these MAbs showed cross-neutralizing activity against all human and zoonotic S variants in vitro, and at least three of these were mapped in distinct epitopes using escape mutants, structure analyses, and competition assays. These three MAbs (S109.8, S227.14, and S230.15) were tested for use in passive vaccination studies using lethal SARS-CoV challenge models for young and senescent mice with four different homologous and heterologous SARS-CoV S variants. Both S227.14 and S230.15 completely protected young and old mice from weight loss and virus replication in the lungs for all viruses tested, while S109.8 completely protected mice from weight loss and clinical signs in the presence of viral titers. We conclude that a single human MAb can confer broad protection against lethal challenge with multiple zoonotic and human SARS-CoV isolates, and we identify a robust cocktail formulation that targets distinct epitopes and minimizes the likely generation of escape mutants.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Developing a vaccine for human rhinoviruses</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Oct 1</span></i><p style="text-align: justify; text-justify: inter-word;">Rhinoviruses (RV’s) are common human pathogens of the respiratory tract being the most frequent cause of mild diseases of the upper respiratory tract (common cold) but more importantly they are a major initiator of acute exacerbations of chronic airway diseases. Infections can be life threatening in the latter context however RV -induced common colds have an associated economic cost from loss of productivity due to absence from work or school. There are no appropriate antiviral therapies available and vaccine strategies have failed because of the large number of viral serotypes and the lack of cross-serotype protection generated. Here, approaches past and present for development of a vaccine to these widespread human pathogens are highlighted.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Spike Protein Delivered by Modified Vaccinia Virus Ankara Efficiently Induces Virus-Neutralizing Antibodies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) has recently emerged as a causative agent of severe respiratory disease in humans. Here, we constructed recombinant modified vaccinia virus Ankara (MVA) expressing full-length MERS-CoV spike (S) protein (MVA-MERS-S). The genetic stability and growth characteristics of MVA-MERS-S make it a suitable candidate vaccine for clinical testing. Vaccinated mice produced high levels of serum antibodies neutralizing MERS-CoV. Thus, MVA-MERS-S may serve for further development of an emergency vaccine against MERS-CoV.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>A Synthetic Porcine Reproductive and Respiratory Syndrome Virus Strain Confers Unprecedented Levels of Heterologous Protection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Current vaccines do not provide sufficient levels of protection against divergent porcine reproductive and respiratory syndrome virus (PRRSV) strains circulating in the field, mainly due to the substantial variation of the viral genome. We describe here a novel approach to generate a PRRSV vaccine candidate that could confer unprecedented levels of heterologous protection against divergent PRRSV isolates. By using a set of 59 nonredundant, full-genome sequences of type 2 PRRSVs, a consensus genome (designated PRRSV-CON) was generated by aligning these 59 PRRSV full-genome sequences, followed by selecting the most common nucleotide found at each position of the alignment. Next, the synthetic PRRSV-CON strain was generated through the use of reverse genetics. PRRSV-CON replicates as efficiently as our prototype PRRSV strain FL12, both in vitro and in vivo. Importantly, when inoculated into pigs, PRRSV-CON confers significantly broader levels of heterologous protection than does wild-type PRRSV. Collectively, our data demonstrate that PRRSV-CON can serve as an excellent candidate for the development of a broadly protective PRRSV vaccine. IMPORTANCE The extraordinary genetic variation of RNA viruses poses a monumental challenge for the development of broadly protective vaccines against these viruses. To minimize the genetic dissimilarity between vaccine immunogens and contemporary circulating viruses, computational strategies have been developed for the generation of artificial immunogen sequences (so-called “centralized” sequences) that have equal genetic distances to the circulating viruses. Thus far, the generation of centralized vaccine immunogens has been carried out at the level of individual viral proteins. We expand this concept to PRRSV, a highly variable RNA virus, by creating a synthetic PRRSV strain based on a centralized PRRSV genome sequence. This study provides the first example of centralizing the whole genome of an RNA virus to improve vaccine coverage. This concept may be significant for the development of vaccines against genetically variable viruses that require active viral replication in order to achieve complete immune protection.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Protective efficacy of a broadly cross-reactive swine influenza DNA vaccine encoding M2e, cytotoxic T lymphocyte epitope and consensus H3 hemagglutinin</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Jun 27</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Pigs have been implicated as mixing reservoir for the generation of new pandemic influenza strains, control of swine influenza has both veterinary and public health significance. Unlike human influenza vaccines, strains used for commercially available swine influenza vaccines are not regularly replaced, making the vaccines provide limited protection against antigenically diverse viruses. It is therefore necessary to develop broadly protective swine influenza vaccines that are efficacious to both homologous and heterologous virus infections. In this study, two forms of DNA vaccines were constructed, one was made by fusing M2e to consensus H3HA (MHa), which represents the majority of the HA sequences of H3N2 swine influenza viruses. Another was made by fusing M2e and a conserved CTL epitope (NP147-155) to consensus H3HA (MNHa). Their protective efficacies against homologous and heterologous challenges were tested. RESULTS: BALB/c mice were immunized twice by particle-mediated epidermal delivery (gene gun) with the two DNA vaccines. It was shown that the two vaccines elicited substantial antibody responses, and MNHa induced more significant T cell-mediated immune response than MHa did. Then two H3N2 strains representative of different evolutional and antigenic clusters were used to challenge the vaccine-immunized mice (homosubtypic challenge). Results indicated that both of the DNA vaccines prevented homosubtypic virus infections completely. The vaccines’ heterologous protective efficacies were further tested by challenging with a H1N1 swine influenza virus and a reassortant 2009 pandemic strain. It was found that MNHa reduced the lung viral titers significantly in both challenge groups, histopathological observation showed obvious reduction of lung pathogenesis as compared to MHa and control groups. CONCLUSIONS: The combined utility of the consensus HA and the conserved M2e and CTL epitope can confer complete and partial protection against homologous and heterologous challenges, respectively, in mouse model. This may provide a basis for the development of universal swine influenza vaccines.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Current advancements and potential strategies in the development of MERS-CoV vaccines</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) is a newly emerging infectious disease caused by a novel coronavirus, MERS-coronavirus (MERS-CoV), a new member in the lineage C of β-coronavirus (β-CoV). The increased human cases and high mortality rate of MERS-CoV infection make it essential to develop safe and effective vaccines. In this review, the current advancements and potential strategies in the development of MERS vaccines, particularly subunit vaccines based on MERS-CoV spike (S) protein and its receptor-binding domain (RBD), are discussed. How to improve the efficacy of subunit vaccines through novel adjuvant formulations and routes of administration as well as currently available animal models for evaluating the in vivo efficacy of MERS-CoV vaccines are also addressed. Overall, these strategies may have important implications for the development of effective and safe vaccines for MERS-CoV in the future.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Modulation of the Immune Response to the Severe Acute Respiratory Syndrome Spike Glycoprotein by Gene-Based and Inactivated Virus Immunization</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although the initial isolates of the severe acute respiratory syndrome (SARS) coronavirus (CoV) are sensitive to neutralization by antibodies through their spike (S) glycoprotein, variants of S have since been identified that are resistant to such inhibition. Optimal vaccine strategies would therefore make use of additional determinants of immune recognition, either through cellular or expanded, cross-reactive humoral immunity. Here, the cellular and humoral immune responses elicited by different combinations of gene-based and inactivated viral particles with various adjuvants have been assessed. The T-cell response was altered by different prime-boost immunizations, with the optimal CD8 immunity induced by DNA priming and replication-defective adenoviral vector boosting. The humoral immune response was enhanced most effectively through the use of inactivated virus with adjuvants, either MF59 or alum, and was associated with stimulation of the CD4 but not the CD8 response. The use of inactivated SARS virus with MF59 enhanced the CD4 and antibody response even after gene-based vaccination. Because both cellular and humoral immune responses are generated by gene-based vaccination and inactivated viral boosting, this strategy may prove useful in the generation of SARS-CoV vaccines.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>PEDV and PDCoV Pathogenesis: The Interplay Between Host Innate Immune Responses and Porcine Enteric Coronaviruses</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 22</span></i><p style="text-align: justify; text-justify: inter-word;">Enteropathogenic porcine epidemic diarrhea virus (PEDV) and porcine deltacoronavirus (PDCoV), members of the coronavirus family, account for the majority of lethal watery diarrhea in neonatal pigs in the past decade. These two viruses pose significant economic and public health burdens, even as both continue to emerge and reemerge worldwide. The ability to evade, circumvent or subvert the host’s first line of defense, namely the innate immune system, is the key determinant for pathogen virulence, survival, and the establishment of successful infection. Unfortunately, we have only started to unravel the underlying viral mechanisms used to manipulate host innate immune responses. In this review, we gather current knowledge concerning the interplay between these viruses and components of host innate immunity, focusing on type I interferon induction and signaling in particular, and the mechanisms by which virus-encoded gene products antagonize and subvert host innate immune responses. Finally, we provide some perspectives on the advantages gained from a better understanding of host-pathogen interactions. This includes their implications for the future development of PEDV and PDCoV vaccines and how we can further our knowledge of the molecular mechanisms underlying virus pathogenesis, virulence, and host coevolution.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Nonstructural Protein 16 Is Necessary for Interferon Resistance and Viral Pathogenesis</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 15</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) encode a mixture of highly conserved and novel genes, as well as genetic elements necessary for infection and pathogenesis, raising the possibility of common targets for attenuation and therapeutic design. In this study, we focused on highly conserved nonstructural protein 16 (NSP16), a viral 2′O-methyltransferase (2′O-MTase) that encodes critical functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif in the conserved KDKE motif of Middle East respiratory syndrome CoV (MERS-CoV) NSP16 (D130A) and evaluated the effect on viral infection and pathogenesis. While the absence of 2′O-MTase activity had only a marginal impact on propagation and replication in Vero cells, dNSP16 mutant MERS-CoV demonstrated significant attenuation relative to the control both in primary human airway cell cultures and in vivo. Further examination indicated that dNSP16 mutant MERS-CoV had a type I interferon (IFN)-based attenuation and was partially restored in the absence of molecules of IFN-induced proteins with tetratricopeptide repeats. Importantly, the robust attenuation permitted the use of dNSP16 mutant MERS-CoV as a live attenuated vaccine platform protecting from a challenge with a mouse-adapted MERS-CoV strain. These studies demonstrate the importance of the conserved 2′O-MTase activity for CoV pathogenesis and highlight NSP16 as a conserved universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting. IMPORTANCE Coronavirus (CoV) emergence in both humans and livestock represents a significant threat to global public health, as evidenced by the sudden emergence of severe acute respiratory syndrome CoV (SARS-CoV), MERS-CoV, porcine epidemic diarrhea virus, and swine delta CoV in the 21st century. These studies describe an approach that effectively targets the highly conserved 2′O-MTase activity of CoVs for attenuation. With clear understanding of the IFN/IFIT (IFN-induced proteins with tetratricopeptide repeats)-based mechanism, NSP16 mutants provide a suitable target for a live attenuated vaccine platform, as well as therapeutic development for both current and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-CoV functions have not yet proven effective against MERS-CoV, illustrating the broad applicability of targeting viral 2′O-MTase function across CoVs.</p>
<p align="right"><i>score: 141</i></p>

</div><br /><br /><br /></body>