
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: standardization of challenge studies for vaccine development</h2><br /></div><br /><br /><br /><div class="card"><h3>Emerging infectious disease laboratory and diagnostic preparedness to accelerate vaccine development</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 16</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid vaccine development in response to an outbreak of a new emerging infectious disease (EID) is a goal targeted by public health agencies worldwide. This goal becomes more complicated when there are no standardized sets of viral and immunological assays, no accepted and well-characterized samples, standards or reagents, and no approved diagnostic tests for the EID pathogen. The diagnosis of infections is of critical importance to public health, but also in vaccine development in order to track incident infections during clinical trials, to differentiate natural infection responses from those that are vaccine-related and, if called for by study design, to exclude subjects with prior exposure from vaccine efficacy trials. Here we review emerging infectious disease biological standards development, vaccine clinical assay development and trial execution with the recent experiences of MERS-CoV and Zika virus as examples. There is great need to establish, in advance, the standardized reagents, sample panels, controls, and assays to support the rapid advancement of vaccine development efforts in response to EID outbreaks.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Standardized Viral Hemagglutination and Hemagglutination-Inhibition Tests. II. Description and Statistical Evaluation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Standardized hemagglutination and hemagglutination-inhibition procedures are described and statistically evaluated for all animal viruses where applicable, except for rubella and the arbovirus group. The standardized tests employ a constant phosphate-buffered saline diluent and constant volumes of serum, antigen, and standardized erythrocyte suspension. The standardized hemagglutination test has a reproducibility of 84 to 96% with adenoviruses, rubeola, and the myxoviruses, and 78 to 93% with reoviruses; the standardized hemagglutination-inhibition test has a reproducibility of 95 to 100% with all viruses tested.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>The human viral challenge model: accelerating the evaluation of respiratory antivirals, vaccines and novel diagnostics</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 22</span></i><p style="text-align: justify; text-justify: inter-word;">The Human Viral Challenge (HVC) model has, for many decades, helped in the understanding of respiratory viruses and their role in disease pathogenesis. In a controlled setting using small numbers of volunteers removed from community exposure to other infections, this experimental model enables proof of concept work to be undertaken on novel therapeutics, including vaccines, immunomodulators and antivirals, as well as new diagnostics. Crucially, unlike conventional phase 1 studies, challenge studies include evaluable efficacy endpoints that then guide decisions on how to optimise subsequent field studies, as recommended by the FDA and thus licensing studies that follow. Such a strategy optimises the benefit of the studies and identifies possible threats early on, minimising the risk to subsequent volunteers but also maximising the benefit of scarce resources available to the research group investing in the research. Inspired by the principles of the 3Rs (Replacement, Reduction and Refinement) now commonly applied in the preclinical phase, HVC studies allow refinement and reduction of the subsequent development phase, accelerating progress towards further statistically powered phase 2b studies. The breadth of data generated from challenge studies allows for exploration of a wide range of variables and endpoints that can then be taken through to pivotal phase 3 studies. We describe the disease burden for acute respiratory viral infections for which current conventional development strategies have failed to produce therapeutics that meet clinical need. The Authors describe the HVC model’s utility in increasing scientific understanding and in progressing promising therapeutics through development. The contribution of the model to the elucidation of the virus-host interaction, both regarding viral pathogenicity and the body’s immunological response is discussed, along with its utility to assist in the development of novel diagnostics. Future applications of the model are also explored. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1186/s12931-018-0784-1) contains supplementary material, which is available to authorized users.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>The human viral challenge model: accelerating the evaluation of respiratory antivirals, vaccines and novel diagnostics</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 22</span></i><p style="text-align: justify; text-justify: inter-word;">The Human Viral Challenge (HVC) model has, for many decades, helped in the understanding of respiratory viruses and their role in disease pathogenesis. In a controlled setting using small numbers of volunteers removed from community exposure to other infections, this experimental model enables proof of concept work to be undertaken on novel therapeutics, including vaccines, immunomodulators and antivirals, as well as new diagnostics. Crucially, unlike conventional phase 1 studies, challenge studies include evaluable efficacy endpoints that then guide decisions on how to optimise subsequent field studies, as recommended by the FDA and thus licensing studies that follow. Such a strategy optimises the benefit of the studies and identifies possible threats early on, minimising the risk to subsequent volunteers but also maximising the benefit of scarce resources available to the research group investing in the research. Inspired by the principles of the 3Rs (Replacement, Reduction and Refinement) now commonly applied in the preclinical phase, HVC studies allow refinement and reduction of the subsequent development phase, accelerating progress towards further statistically powered phase 2b studies. The breadth of data generated from challenge studies allows for exploration of a wide range of variables and endpoints that can then be taken through to pivotal phase 3 studies. We describe the disease burden for acute respiratory viral infections for which current conventional development strategies have failed to produce therapeutics that meet clinical need. The Authors describe the HVC model’s utility in increasing scientific understanding and in progressing promising therapeutics through development. The contribution of the model to the elucidation of the virus-host interaction, both regarding viral pathogenicity and the body’s immunological response is discussed, along with its utility to assist in the development of novel diagnostics. Future applications of the model are also explored. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1186/s12931-018-0784-1) contains supplementary material, which is available to authorized users.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>Determination of the infectious titer and virulence of an original US porcine epidemic diarrhea virus PC22A strain</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Sep 25</span></i><p style="text-align: justify; text-justify: inter-word;">The infectious dose of a virus pool of original US PEDV strain PC22A was determined in 4-day-old, cesarean-derived, colostrum-deprived (CDCD) piglets. The median pig diarrhea dose (PDD(50)) of the virus pool was determined as 7.35 log(10) PDD(50)/mL, similar to the cell culture infectious titer, 7.75 log(10) plaque-forming units (PFU)/mL. 100 PDD(50) caused watery diarrhea in all conventional suckling piglets (n = 12) derived from a PEDV-naive sow, whereas 1000 and 10 000 PDD(50) did not cause diarrhea in piglets derived from two PEDV-field exposed-recovered sows. This information is important for future PEDV challenge studies and validation of PEDV vaccines.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Reliable and Standardized Animal Models to Study the Pathogenesis of Bluetongue and Schmallenberg Viruses in Ruminant Natural Host Species with Special Emphasis on Placental Crossing</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 15</span></i><p style="text-align: justify; text-justify: inter-word;">Starting in 2006, bluetongue virus serotype 8 (BTV8) was responsible for a major epizootic in Western and Northern Europe. The magnitude and spread of the disease were surprisingly high and the control of BTV improved significantly with the marketing of BTV8 inactivated vaccines in 2008. During late summer of 2011, a first cluster of reduced milk yield, fever, and diarrhoea was reported in the Netherlands. Congenital malformations appeared in March 2012 and Schmallenberg virus (SBV) was identified, becoming one of the very few orthobunyaviruses distributed in Europe. At the start of both epizootics, little was known about the pathogenesis and epidemiology of these viruses in the European context and most assumptions were extrapolated based on other related viruses and/or other regions of the World. Standardized and repeatable models potentially mimicking clinical signs observed in the field are required to study the pathogenesis of these infections, and to clarify their ability to cross the placental barrier. This review presents some of the latest experimental designs for infectious disease challenges with BTV or SBV. Infectious doses, routes of infection, inoculum preparation, and origin are discussed. Particular emphasis is given to the placental crossing associated with these two viruses.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Assay Challenges for Emerging Infectious Diseases: The Zika Experience</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 2</span></i><p style="text-align: justify; text-justify: inter-word;">From the perspective of vaccine development, it is imperative to accurately diagnose target infections in order to exclude subjects with prior exposure from evaluations of vaccine effectiveness, to track incident infection during the course of a clinical trial and to differentiate immune reactions due to natural infections from responses that are vaccine related. When vaccine development is accelerated to a rapid pace in response to emerging infectious disease threats, the challenges to develop such diagnostic tools is even greater. This was observed through the recent expansion of Zika virus infections into the Western Hemisphere in 2014–2017. When initial Zika vaccine clinical trials were being designed and launched in response to the outbreak, there were no standardized sets of viral and immunological assays, and no approved diagnostic tests for Zika virus infection. The diagnosis of Zika virus infection is still an area of active research and development on many fronts. Here we review emerging infectious disease vaccine clinical assay development and trial execution with a special focus on the state of Zika virus clinical assays and diagnostics.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Developing vaccines against epidemic-prone emerging infectious diseases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Nov 27</span></i><p style="text-align: justify; text-justify: inter-word;">Today’s world is characterized by increasing population density, human mobility, urbanization, and climate and ecological change. This global dynamic has various effects, including the increased appearance of emerging infectious diseases (EIDs), which pose a growing threat to global health security. Outbreaks of EIDs, like the 2013–2016 Ebola outbreak in West Africa or the current Ebola outbreak in Democratic Republic of the Congo (DRC), have not only put populations in low- and middle-income countries (LMIC) at risk in terms of morbidity and mortality, but they also have had a significant impact on economic growth in affected regions and beyond. The Coalition for Epidemic Preparedness Innovation (CEPI) is an innovative global partnership between public, private, philanthropic, and civil society organizations that was launched as the result of a consensus that a coordinated, international, and intergovernmental plan was needed to develop and deploy new vaccines to prevent future epidemics. CEPI is focusing on supporting candidate vaccines against the World Health Organization (WHO) Blueprint priority pathogens MERS-CoV, Nipah virus, Lassa fever virus, and Rift Valley fever virus, as well as Chikungunya virus, which is on the WHO watch list. The current vaccine portfolio contains a wide variety of technologies, ranging across recombinant viral vectors, nucleic acids, and recombinant proteins. To support and accelerate vaccine development, CEPI will also support science projects related to the development of biological standards and assays, animal models, epidemiological studies, and diagnostics, as well as build capacities for future clinical trials in risk-prone contexts.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Sampling port for real time analysis of bioaerosol in whole body exposure system for animal aerosol model development</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">INTRODUCTION: Multiple factors influence the viability of aerosolized bacteria. The delivery of aerosols is affected by chamber conditions (humidity, temperature, and pressure) and bioaerosol characteristics (particle number, particle size distribution, and viable aerosol concentration). Measurement of viable aerosol concentration and particle size is essential to optimize viability and lung delivery. The Madison chamber is widely used to expose small animals to infectious aerosols. METHODS: A multiplex sampling port was added to the Madison chamber to measure the chamber conditions and bioaerosol characteristics. Aerosols of three pathogens (Bacillus anthracis, Yersinia pestis, and Mycobacterium tuberculosis) were generated under constant conditions and their bioaerosol characteristics were analyzed. Airborne microbes were captured using an impinger or BioSampler. The particle size distribution of airborne microbes was determined using an aerodynamic particle sizer (APS). Viable aerosol concentration, spray factor (viable aerosol concentration/inoculum concentration), and dose presented to the mouse were calculated. Dose retention efficiency and viable aerosol retention rate were calculated from the sampler titers to determine the efficiency of microbe retention in lungs of mice. RESULTS: B. anthracis, Y. pestis, and M. tuberculosis aerosols were sampled through the port. The count mean aerodynamic sizes were 0.98, 0.77, and 0.78 μm with geometric standard deviations of 1.60, 1.90, and 2.37, and viable aerosol concentrations in the chamber were 211, 57, and 1 colony-forming unit (CFU)/mL, respectively. Based on the aerosol concentrations, the doses presented to mice for the three pathogens were 2.5e5, 2.2e4 and 464 CFU. DISCUSSION: Using the multiplex sampling port we determined whether the animals were challenged with an optimum bioaerosol based on dose presented and respirable particle size.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Effect of specific humoral immunity and some non-specific factors on resistance of volunteers to respiratory coronavirus infection.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Thirty-three volunteers were inoculated intranasally with coronavirus 229 E, and their responses monitored by antibody rises, symptomatology and virus excretion. These were related to their pre-trial immune status as indicated by concentrations of specific antibodies and non-specific proteins in serum and nasal washings. Both circulating and local specific antibodies were associated with protection from infection and disease, but only specific IgA antibodies of either type appeared to shorten the period of virus shedding. Although total secretory IgA was significantly associated only with reduction of symptoms, total protein in nasal washings appeared to protect against infection also, indicating that other locally produced proteins, not identified, may be associated with resistance. Two of the many factors which may affect the concentration of circulating and local protective proteins and thus influence the outcome of virus inoculation, namely, sex of the volunteer and the interval since the previous cold, were examined. Male volunteers or volunteers who had had evidence of a recent respiratory infection were less likely to be infected, but if they were infected, they had lower clinical scores and stopped shedding virus earlier than the rest. These groups possessed higher concentrations of specific antibodies and non-specific proteins in their pre-challenge sera and/or nasal washings. The significance of these findings is discussed.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of a microneutralization test in cell culture and virus neutralization test in embryonated eggs for determining infectious bronchitis virus antibodies.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A microneutralization test (MNT) system utilizing cytopathic effect end points was effective in determing neutralization indexes for infectious bronchitis virus antibodies. The system is reproducible within 1 index unit at the 95% level of probability. Comparison of the MNT to tests in eggs resulted in a positive correlation (B =0.81), which was significant (P greater than 0.01). The quantitative dose-response relationship of the MNT is linear (P greater than 0.005), with the 95% prediction limits fitting between one 10-fold dilution.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>KILchip v1.0: A Novel Plasmodium falciparum Merozoite Protein Microarray to Facilitate Malaria Vaccine Candidate Prioritization</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 11</span></i><p style="text-align: justify; text-justify: inter-word;">Passive transfer studies in humans clearly demonstrated the protective role of IgG antibodies against malaria. Identifying the precise parasite antigens that mediate immunity is essential for vaccine design, but has proved difficult. Completion of the Plasmodium falciparum genome revealed thousands of potential vaccine candidates, but a significant bottleneck remains in their validation and prioritization for further evaluation in clinical trials. Focusing initially on the Plasmodium falciparum merozoite proteome, we used peer-reviewed publications, multiple proteomic and bioinformatic approaches, to select and prioritize potential immune targets. We expressed 109 P. falciparum recombinant proteins, the majority of which were obtained using a mammalian expression system that has been shown to produce biologically functional extracellular proteins, and used them to create KILchip v1.0: a novel protein microarray to facilitate high-throughput multiplexed antibody detection from individual samples. The microarray assay was highly specific; antibodies against P. falciparum proteins were detected exclusively in sera from malaria-exposed but not malaria-naïve individuals. The intensity of antibody reactivity varied as expected from strong to weak across well-studied antigens such as AMA1 and RH5 (Kruskal–Wallis H test for trend: p < 0.0001). The inter-assay and intra-assay variability was minimal, with reproducible results obtained in re-assays using the same chip over a duration of 3 months. Antibodies quantified using the multiplexed format in KILchip v1.0 were highly correlated with those measured in the gold-standard monoplex ELISA [median (range) Spearman's R of 0.84 (0.65–0.95)]. KILchip v1.0 is a robust, scalable and adaptable protein microarray that has broad applicability to studies of naturally acquired immunity against malaria by providing a standardized tool for the detection of antibody correlates of protection. It will facilitate rapid high-throughput validation and prioritization of potential Plasmodium falciparum merozoite-stage antigens paving the way for urgently needed clinical trials for the next generation of malaria vaccines.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>PROPHYLACTIC ANTIBODY TREATMENT AND INTRAMUSCULAR IMMUNIZATION REDUCE INFECTIOUS HUMAN RHINOVIRUS 16 LOAD IN THE LOWER RESPIRATORY TRACT OF CHALLENGED COTTON RATS</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human rhinoviruses (HRV) represent the single most important etiological agents of the common cold and are the most frequent cause of acute respiratory infections in humans. Currently the performance of available animal models for immunization studies using HRV challenge is very limited. The cotton rat (Sigmodon hispidus) is a well-recognized model for the study of human respiratory viral infections. In this work we show that, without requiring any genetic modification of either the host or the virus, intranasal infection of cotton rats with HRV16 resulted in measurable lower respiratory tract pathology, mucus production, and expression of interferon-activated genes. Intramuscular immunization with live HRV16 generated robust protective immunity that correlated with high serum levels of neutralizing antibodies. In addition, cotton rats treated prophylactically with hyperimmune anti-HRV16 serum were protected against HRV16 intranasal challenge. Finally, protection by immunization was efficiently transferred from mothers to newborn animals resulting in a substantial reduction of infectious virus loads in the lung following intranasal challenge. Overall, our results demonstrate that the cotton rat provides valuable additional model development options for testing vaccines and prophylactic therapies against rhinovirus infection.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of ViroCyt(®) Virus Counter for Rapid Filovirus Quantitation</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">Development and evaluation of medical countermeasures for diagnostics, vaccines, and therapeutics requires production of standardized, reproducible, and well characterized virus preparations. For filoviruses this includes plaque assay for quantitation of infectious virus, transmission electron microscopy (TEM) for morphology and quantitation of virus particles, and real-time reverse transcription PCR for quantitation of viral RNA (qRT-PCR). The ViroCyt(®) Virus Counter (VC) 2100 (ViroCyt, Boulder, CO, USA) is a flow-based instrument capable of quantifying virus particles in solution. Using a proprietary combination of fluorescent dyes that stain both nucleic acid and protein in a single 30 min step, rapid, reproducible, and cost-effective quantification of filovirus particles was demonstrated. Using a seed stock of Ebola virus variant Kikwit, the linear range of the instrument was determined to be 2.8E+06 to 1.0E+09 virus particles per mL with coefficient of variation ranging from 9.4% to 31.5% for samples tested in triplicate. VC particle counts for various filovirus stocks were within one log of TEM particle counts. A linear relationship was established between the plaque assay, qRT-PCR, and the VC. VC results significantly correlated with both plaque assay and qRT-PCR. These results demonstrated that the VC is an easy, fast, and consistent method to quantify filoviruses in stock preparations.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Relative Immunogenicity and Protection Potential of Candidate Yersinia Pestis Antigens against Lethal Mucosal Plague Challenge in Balb/C Mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Yersinia Pestis Outer proteins, plasminogen activator protease and Yop secretion protein F are necessary for the full virulence of Yesinia pestis and have been proposed as potential protective antigens for vaccines against plague. In the current study, we used DNA immunization as a tool to study the relative protective immunity of these proteins with a standardized intranasal challenge system in mice. While the natural full length gene sequences for most of these Y. pestis proteins did not display a good level of protein expression in vitro when delivered by a DNA vaccine vector, the overall immunogenicity of these wild type gene DNA vaccines was low in eliciting antigen-specific antibody responses and gene sequence modifications improved both of these parameters. However, even modified YopD, YopO and YscF antigens were only able to partially protect immunized mice at various levels against lethal challenge with Y. pestis KIM 1001 strain while no protection was observed with either the YopB or Pla antigens. These results demonstrate that DNA immunization is effective in screening, optimizing and comparing optimal antigen designs and immunogenicity of candidate antigens for the development of a subunit-based plague vaccine.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Microneutralization test for respiratory syncytial virus based on an enzyme immunoassay.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Virus infectivity and antibody neutralization titers for respiratory syncytial virus were determined in cell cultures in microtiter plates. After an appropriate incubation period, the cells were fixed, and an enzyme-linked immunosorbent assay was performed directly in the microtiter plates for detection of virus. Results could be read and recorded automatically, which is especially helpful when running large numbers of tests.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Pathogenicity for humans of human rhinovirus type 2 mutants resistant to or dependent on chalcone Ro 09-0410.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Mutants of human rhinovirus type 2 (HRV-2) resistant to and dependent on the antirhinoviral compound chalcone Ro 09-0410 were selected in cell culture under clean laboratory conditions. A total of 42 volunteers were challenged with either the drug-resistant mutant [SR2-410(r)] (15 volunteers), the drug-dependent mutant [SR2-410(d)] (15 volunteers), or a wild-type HRV-2 which had a similar passage level in vitro as the mutants but without the drug (12 volunteers). Of volunteers challenged with the wild-type HRV-2, 33, 67, and 82% developed cold symptoms, shed virus, and showed serological evidence of infection, respectively. In contrast, only 13, 27, and 23% of volunteers challenged with the drug-resistant mutant developed colds, shed virus, and showed serological evidence of infection, respectively. None of the volunteers challenged with the drug-dependent virus became infected or had symptoms of colds. These results demonstrate that a drug-resistant rhinovirus was capable of infecting humans and producing disease, although its infectivity was reduced when compared with that of the wild type. In contrast, a drug-dependent virus had lost its ability to infect humans.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>International Biological Reference Preparations for Epidemic Infectious Diseases</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Recent years have seen unprecedented investment in research and development for countermeasures for high-threat pathogens, including specific and ambitious objectives for development of diagnostics, therapeutics, and vaccines. The inadequate availability of biological reference materials for these pathogens poses a genuine obstacle in pursuit of these objectives, and the lack of a comprehensive and equitable framework for developing reference materials is a weakness. We outline the need for internationally standardized biological materials for high-threat pathogens as a core element of global health security. We also outline the key components of a framework for addressing this deficiency.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Cellular and Humoral Immunity following Snow Mountain Virus Challenge</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Little is known about the immune response to noroviruses. To elucidate the immunobiology of norovirus infection in humans, 15 volunteers were challenged with Snow Mountain virus (SMV), a genogroup 2 norovirus. We assessed the cellular and humoral immune response and infection by analyzing stool, serum, saliva, and peripheral blood mononuclear cell (PBMC) responses pre- and postchallenge. In contrast to Norwalk virus (NV), SMV infection was not dependent upon blood group secretor status. Nine of 15 volunteers were infected and showed a ≥4-fold increase over the prechallenge anti-SMV serum immunoglobulin G (IgG) titer, mostly subclass IgG1. Although serum IgG elicited by SMV infection was cross-reactive with Hawaii virus (HV), another genogroup 2 norovirus, salivary IgA was less cross-reactive. Neither SMV-elicited serum IgG nor salivary IgA cross-reacted with NV, a genogroup 1 norovirus. Significant increases in serum gamma interferon (IFN-γ) and IL-2, but not IL-6 or IL-10, were noted on day 2 postchallenge. For the majority of volunteers, both infected and uninfected, PBMCs stimulated with norovirus virus-like particles secreted IFN-γ and other Th1 cytokines, suggesting previous norovirus exposure in most volunteers. Like the IgG antibodies, the SMV-activated T cells were cross-reactive with HV but not NV. IFN-γ production was dependent upon CD4(+) cells, consistent with a predominant, but not exclusive, Th1 response. To our knowledge, this is the first report characterizing T-cell and cytokine responses following live norovirus challenge.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Considerations for the design of vaccine efficacy trials during public health emergencies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Public Health Emergencies (PHEs) provide a complex and challenging environment for vaccine evaluation. Under the R&D Blueprint Plan of Action, the World Health Organization (WHO) has convened a group of experts to agree on standard procedures to rapidly evaluate experimental vaccines during PHEs while maintaining the highest scientific and ethical standards. The Blueprint priority diseases, selected for their likelihood to cause PHEs and the lack of adequate medical countermeasures, were used to frame our methodological discussions. Here, we outline major vaccine study designs to be used in PHEs and summarize high-level recommendations for their use in this setting. We recognize that the epidemiology and transmission dynamics of the Blueprint priority diseases may be highly uncertain and that the unique characteristics of the vaccines and outbreak settings may affect our study design. To address these challenges, our group underscores the need for novel, flexible, and responsive trial designs. We conclude that assignment to study groups using randomization is a key principle underlying rigorous study design and should be utilized except in exceptional circumstances. Advance planning for vaccine trial designs is critical for rapid and effective response to a PHE and to advance knowledge to address and mitigate future PHEs.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>The use of equine influenza pseudotypes for serological screening</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 31</span></i><p style="text-align: justify; text-justify: inter-word;">Standard assays used for influenza serology present certain practical issues, such as inter-laboratory variability, complex protocols and the necessity for handling certain virus strains in high biological containment facilities. In an attempt to address this, avian and human influenza HA pseudotyped retroviruses have been successfully employed in antibody neutralization assays. In this study we generated an equine influenza pseudotyped lentivirus for serological screening. This was achieved by co-transfection of HEK293T cells with plasmids expressing the haemagglutinin (HA) protein of an H3N8 subtype equine influenza virus strain, HIV gag-pol and firefly luciferase reporter genes and harvesting virus from supernatant. In order to produce infective pseudotype particles it was necessary to additionally co-transfect a plasmid encoding the TMPRSS2 endoprotease to cleave the HA. High titre pseudotype virus (PV) was then used in PV antibody neutralization assays (PVNAs) to successfully distinguish between vaccinated and non-vaccinated equines. The sera were also screened by single radial haemolysis (SRH) assay. There was a 65% correlation between the results of the two assays, with the PVNA assay appearing slightly more sensitive. Future work will extend the testing of the PVNA with a larger number of serum samples to assess sensitivity/specificity, inter/intra-laboratory variability and to define a protective titre.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrated, multi-cohort analysis identifies conserved transcriptional signatures across multiple respiratory viruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Respiratory viral infections are a significant burden to healthcare worldwide. Many whole genome expression profiles have identified different respiratory viral infection signatures, but these have not translated to clinical practice. Here, we performed two integrated, multi-cohort analyses of publicly available transcriptional data of viral infections. First, we identified a common host signature across different respiratory viral infections that could distinguish (a) individuals with viral infections from healthy controls and from those with bacterial infections, and (b) symptomatic from asymptomatic subjects prior to symptom onset in challenge studies. Second, we identified an influenza-specific host response signature that (a) could distinguish influenza-infected samples from those with bacterial and other respiratory viral infections, (b) was a diagnostic and prognostic marker in influenza-pneumonia patients and influenza challenge studies, and (c) was predictive of response to influenza vaccine. Our results have applications in the diagnosis, prognosis, and identification of drug targets in viral infections.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral shedding and clinical illness in naturally acquired influenza virus infections</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Volunteer challenge studies have provided detailed data on viral shedding from the respiratory tract before and through the course of experimental influenza virus infection. There are no comparable quantitative data on naturally-acquired infections. METHODS: In a community-based study in Hong Kong in 2008, we followed up initially well individuals to quantify trends in viral shedding based on culture and reverse transcription polymerase chain reaction (RT-PCR) through the course of illness associated with seasonal influenza A and B virus infection. RESULTS: Trends in symptom scores more closely matched changes in molecular viral loads measured by RT-PCR for influenza A than influenza B. For influenza A virus infections, replicating viral loads measured by culture declined to undetectable levels earlier after illness onset than molecular viral loads. Most viral shedding occurred during the first 2–3 days after illness onset and we estimated that 1–8% of infectiousness occurs prior to illness onset. Only 14% of infections with detectable shedding by RT-PCR were asymptomatic, and viral shedding was low in these cases. CONCLUSIONS: Our results suggest that ‘silent spreaders’ (i.e. individuals who are infectious while asymptomatic or pre-symptomatic) may be less important in the spread of influenza epidemics than previously thought.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Infection and Response to Rechallenge of Alpacas with Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We conducted a challenge/rechallenge trial in which 3 alpacas were infected with Middle East respiratory syndrome coronavirus. The alpacas shed virus at challenge but were refractory to further shedding at rechallenge on day 21. The trial indicates that alpacas may be suitable models for infection and shedding dynamics of this virus.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Infection and Response to Rechallenge of Alpacas with Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We conducted a challenge/rechallenge trial in which 3 alpacas were infected with Middle East respiratory syndrome coronavirus. The alpacas shed virus at challenge but were refractory to further shedding at rechallenge on day 21. The trial indicates that alpacas may be suitable models for infection and shedding dynamics of this virus.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>The effect of chemical clarification of oral fluids on porcine epidemic diarrhea virus antibody responses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Routine testing of breeding herd oral fluid (OF) samples for porcine epidemic diarrhea virus (PEDV) IgG and/or IgA is used to track levels of PEDV immunity over time. However, OFs contain particles of feed, feces, and inorganic material that detract from the quality of the sample. We clarified swine OF samples using lyophilized chitosan-based formulas (A–C) tested by PEDV IgG and IgA ELISAs. To evaluate both the immediate and residual effects of treatment on antibody detection, samples were tested immediately post-treatment, then stored at 4°C and retested at 2, 4, and 6 days post-treatment (DPT). Formulations were shown to effectively clarify samples. Statistical analysis comparing treated to untreated OF samples at 0 DPT found that neither chitosan nor Tween 20 affected the OF ELISA IgA and IgG sample-to-positive (S/P) ratio results (p > 0.05). Furthermore, pairwise comparisons of 0 DPT to 2, 4, and 6 DPT results detected no significant differences (p > 0.05) in IgA and IgG S/P ratios (i.e., treated OF samples were stable over time). Therefore, chitosan efficiently clarified OF specimens without affecting the results of the PEDV IgG and IgA antibody ELISAs.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Medical Countermeasures to Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">Preclinical development of and research on potential Middle East respiratory syndrome coronavirus (MERS-CoV) medical countermeasures remain preliminary; advancements are needed before most countermeasures are ready to be tested in human clinical trials. Research priorities include standardization of animal models and virus stocks for studying disease pathogenesis and efficacy of medical countermeasures; development of MERS-CoV diagnostics; improved access to nonhuman primates to support preclinical research; studies to better understand and control MERS-CoV disease, including vaccination studies in camels; and development of a standardized clinical trial protocol. Partnering with clinical trial networks in affected countries to evaluate safety and efficacy of investigational therapeutics will strengthen efforts to identify successful medical countermeasures.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>A Systems Framework for Vaccine Design</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Numerous challenges have been identified in vaccine development, including variable efficacy as a function of population demographics and a lack of characterization and mechanistic understanding of immune correlates of protection able to guide delivery and dosing. There is tremendous opportunity in recent technological and computational advances to elucidate systems level understanding of pathogen-host interactions and correlates of immunity. A systems biology approach to vaccinology provides a new paradigm for rational vaccine design in a “precision medicine” context.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>The personal touch: strategies toward personalized vaccines and predicting immune responses to them</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The impact of vaccines on public health and well-being has been profound. Smallpox has been eradicated, polio is nearing eradication, and multiple diseases have been eliminated from certain areas of the world. Unfortunately, we now face diseases such as: hepatitis C, malaria, or tuberculosis, as well as new and re-emerging pathogens for which lack effective vaccines. Empirical approaches to vaccine development have been successful in the past, but may not be up to the current infectious disease challenges facing us. New, directed approaches to vaccine design, development, and testing need to be developed. Ideally these approaches will capitalize on cutting-edge technologies, advanced analytical and modeling strategies, and up-to-date knowledge of both pathogen and host. These approaches will pay particular attention to the causes of inter-individual variation in vaccine response in order to develop new vaccines tailored to the unique needs of individuals and communities within the population.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of an Aerosol Model of Cryptococcus Reveals Humidity as an Important Factor Affecting the Viability of Cryptococcus during Aerosolization</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 23</span></i><p style="text-align: justify; text-justify: inter-word;">Cryptococcus is an emerging global health threat that is annually responsible for over 1,000,000 infections and one third of all AIDS patient deaths. There is an ongoing outbreak of cryptococcosis in the western United States and Canada. Cryptococcosis is a disease resulting from the inhalation of the infectious propagules from the environment. The current and most frequently used animal infection models initiate infection via liquid suspension through intranasal instillation or intravenous injection. These models do not replicate the typically dry nature of aerosol exposure and may hinder our ability to decipher the initial events that lead to clearance or the establishment of infection. We have established a standardized aerosol model of murine infection for the human fungal pathogen Cryptococcus. Aerosolized cells were generated utilizing a Collison nebulizer in a whole-body Madison Chamber at different humidity conditions. The aerosols inside the chamber were sampled using a BioSampler to determine viable aerosol concentration and spray factor (ratio of viable aerosol concentration to total inoculum concentration). We have effectively delivered yeast and yeast-spore mixtures to the lungs of mice and observed the establishment of disease. We observed that growth conditions prior to exposure and humidity within the Madison Chamber during exposure can alter Cryptococcus survival and dose retained in mice.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of immunoglobulin (Ig) A antibodies against porcine epidemic diarrhea virus (PEDV) in fecal and serum samples</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Oct 13</span></i><p style="text-align: justify; text-justify: inter-word;">Many assays for detection of antibodies against porcine epidemic diarrhea virus (PEDV) are based on detection of neutralizing antibodies or immunoglobulin (Ig) G in serum samples. However, due to the particular features of the mucosal immune system, presence of serum antibodies against enteric pathogens, such as PEDV, not always correlates with protection. In contrast, anti-PEDV IgA antibodies correlate with protection against subsequent challenges. An indirect PEDV IgA ELISA was previously developed to monitor IgA levels in colostrum and milk samples. In the present paper we describe an adaptation of the protocol for detection of IgA antibodies in serum and fecal samples. • The adapted protocol will aid in future assessment of protective levels of humoral response against PEDV infection by measuring IgA levels in serum and fecal samples. • Fecal samples are non-invasive and easy to collect at any time by animal caretakers and therefore offering advantages over the serum sample collection procedure. • A strong positive correlation between the anti-PEDV levels in fecal and serum samples was identified; however, detection of IgA antibodies was often more successful in serum than in paired fecal samples due to overall lower sample-to-positive (S/P) ratios for the latter sample type.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Accelerating vaccine development and deployment: report of a Royal Society satellite meeting</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">The Royal Society convened a meeting on the 17th and 18th November 2010 to review the current ways in which vaccines are developed and deployed, and to make recommendations as to how each of these processes might be accelerated. The meeting brought together academics, industry representatives, research sponsors, regulators, government advisors and representatives of international public health agencies from a broad geographical background. Discussions were held under Chatham House rules. High-throughput screening of new vaccine antigens and candidates was seen as a driving force for vaccine discovery. Multi-stakeholder, small-scale manufacturing facilities capable of rapid production of clinical grade vaccines are currently too few and need to be expanded. In both the human and veterinary areas, there is a need for tiered regulatory standards, differentially tailored for experimental and commercial vaccines, to allow accelerated vaccine efficacy testing. Improved cross-fertilization of knowledge between industry and academia, and between human and veterinary vaccine developers, could lead to more rapid application of promising approaches and technologies to new product development. Identification of best-practices and development of checklists for product development plans and implementation programmes were seen as low-cost opportunities to shorten the timeline for vaccine progression from the laboratory bench to the people who need it.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Conventional and Specific-Pathogen Free Rats Respond Differently to Anesthesia and Surgical Trauma</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">Specific-pathogen free (SPF) animals were introduced in the 1960s to minimize disease and infection as variables in biomedical research. Our aim was to examine differences in physiological response in rat colonies bred and housed in a conventional versus SPF facility, and implications for research. Sprague-Dawley rats were anesthetized and catheterized for blood and pressure monitoring, and electrocardiogram (ECG) leads implanted. Hematology was assessed, and coagulation profile using rotational thromboelastometry. Health screening was outsourced to Cerberus Sciences. SPF rats had significantly lower pulse pressure (38% decrease), arrhythmias and prolonged QTc (27% increase) compared to conventional rats. No arrhythmias were found in conventional rats. SPF rats had significantly higher white cell, monocyte, neutrophil and lymphocyte counts, and were hyperfibrinolytic, indicated by EXTEM maximum lysis >15%. Independent assessment revealed similar pathogen exclusion between colonies, with the exception of Proteus in SPF animals. Returning to a conventional facility restored normal host physiology. We conclude that SPF animals displayed an abnormal hemodynamic, hematological and hemostatic phenotype in response to anesthesia and surgery, and provide a number of recommendations to help standardize research outcomes and translation.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Impact of confinement housing on study end-points in the calf model of cryptosporidiosis</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 25</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Diarrhea is the second leading cause of death in children < 5 years globally and the parasite genus Cryptosporidium is a leading cause of that diarrhea. The global disease burden attributable to cryptosporidiosis is substantial and the only approved chemotherapeutic, nitazoxanide, has poor efficacy in HIV positive children. Chemotherapeutic development is dependent on the calf model of cryptosporidiosis, which is the best approximation of human disease. However, the model is not consistently applied across research studies. Data collection commonly occurs using two different methods: Complete Fecal Collection (CFC), which requires use of confinement housing, and Interval Collection (IC), which permits use of box stalls. CFC mimics human challenge model methodology but it is unknown if confinement housing impacts study end-points and if data gathered via this method is suitable for generalization to human populations. METHODS: Using a modified crossover study design we compared CFC and IC and evaluated the impact of housing on study end-points. At birth, calves were randomly assigned to confinement (n = 14) or box stall housing (n = 9), or were challenged with 5 x 10(7) C. parvum oocysts, and followed for 10 days. Study end-points included fecal oocyst shedding, severity of diarrhea, degree of dehydration, and plasma cortisol. FINDINGS: Calves in confinement had no significant differences in mean log oocysts enumerated per gram of fecal dry matter between CFC and IC samples (P = 0.6), nor were there diurnal variations in oocyst shedding (P = 0.1). Confinement housed calves shed significantly more oocysts (P = 0.05), had higher plasma cortisol (P = 0.001), and required more supportive care (P = 0.0009) than calves in box stalls. CONCLUSION: Housing method confounds study end-points in the calf model of cryptosporidiosis. Due to increased stress data collected from calves in confinement housing may not accurately estimate the efficacy of chemotherapeutics targeting C. parvum.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccines for Pandemic Influenza</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Recent outbreaks of highly pathogenic avian influenza in Asia and associated human infections have led to a heightened level of awareness and preparation for a possible influenza pandemic. Vaccination is the best option by which spread of a pandemic virus could be prevented and severity of disease reduced. Production of live attenuated and inactivated vaccine seed viruses against avian influenza viruses, which have the potential to cause pandemics, and their testing in preclinical studies and clinical trials will establish the principles and ensure manufacturing experience that will be critical in the event of the emergence of such a virus into the human population. Studies of such vaccines will also add to our understanding of the biology of avian influenza viruses and their behavior in mammalian hosts.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Studies of experimental rhinovirus type 2 infections in polar isolation and in England.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">After five months of total isolation a wintering party of seventeen British Antarctic Survey (BAS) personnel was inoculated under double blind concitions with placebo, or rhinovirus type 2 which had been propagated in tissue culture. The clinical and virological responses of these subjects were compared with those of volunteers in England who received a similar dose of the same strain. The virus used was apparently partly attenuated for man; at the dosage used its effects in England were similar to a smaller dose of an unattenuated strain, but in the Antarctic it caused relatively severe infections. Both the symptoms and the laboratory evidence of virus infection appeared to be more pronounced in the BAS subjects than in the volunteers in England who received the same challenge. In the former group the infection readily spread to those who were originally given placebo. In the BAS subjects serum antibody titres were well maintained during the isolation period but a significant fall in nasal immunoglobulin concentration was recorded during the 5 months of isolation after the virus challenge. Possible mechanisms for the increased sensitivity to rhinovirus of subjects who have been totally isolated in a small closed community are discussed.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative Pathogenesis and Systems Biology for  Biodefense Virus Vaccine Development</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Jun 6</span></i><p style="text-align: justify; text-justify: inter-word;">Developing vaccines to biothreat agents presents a number of challenges for discovery, preclinical development, and licensure. The need for high containment to work with live agents limits the amount and types of research that can be done using complete pathogens, and small markets reduce potential returns for industry. However, a number of tools, from comparative pathogenesis of viral strains at the molecular level to novel computational approaches, are being used to understand the basis of viral attenuation and characterize protective immune responses. As the amount of basic molecular knowledge grows, we will be able to take advantage of these tools not only to rationally attenuate virus strains for candidate vaccines, but also to assess immunogenicity and safety in silico. This review discusses how a basic understanding of pathogenesis, allied with systems biology and machine learning methods, can impact biodefense vaccinology.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunization with Live Human Rhinovirus (HRV) 16 Induces Protection in Cotton Rats against HRV14 Infection</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 31</span></i><p style="text-align: justify; text-justify: inter-word;">Human rhinoviruses (HRVs) are the main cause of cold-like illnesses, and currently no vaccine or antiviral therapies against HRVs are available to prevent or mitigate HRV infection. There are more than 150 antigenically heterogeneous HRV serotypes, with ∼90 HRVs belonging to major group species A and B. Development of small animal models that are susceptible to infection with major group HRVs would be beneficial for vaccine research. Previously, we showed that the cotton rat (Sigmodon hispidus) is semi-permissive to HRV16 (major group, species HRV-A virus) infection, replicating in the upper and lower respiratory tracts with measurable pathology, mucus production, and expression of inflammatory mediators. Herein, we report that intranasal infection of cotton rats with HRV14 (major group, species HRV-B virus) results in isolation of infectious virus from the nose and lung. Similar to HRV16, intramuscular immunization with live HRV14 induces homologous protection that correlated with high levels of serum neutralizing antibodies. Vaccination and challenge experiments with HRV14 and HRV16 to evaluate the development of cross-protective immunity demonstrate that intramuscular immunization with live HRV16 significantly protects animals against HRV14 challenge. Determination of the immunological mechanisms involved in heterologous protection and further characterization of infection with other major HRV serotypes in the cotton rat could enhance the robustness of the model to define heterotypic relationships between this diverse group of viruses and thereby increase its potential for development of a multi-serotype HRV vaccine.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Antigenic relationships among some bovine rotaviruses: serum neutralization and cross-protection in gnotobiotic calves.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A method was further developed to screen non-tissue-culture-adapted bovine rotaviruses for serotype, using a neutralization test with infectious fecal rotavirus. One of those rotaviruses (B223) which was not blocked by antiserum to the neonatal calf diarrhea virus (NCDV) serotype was then adapted to cell culture in the presence of the antiserum for two or more passages and hyperimmune antiserum to this isolate had a 60-fold-higher homologous neutralization titer than with the NCDV serotype rotavirus. Seventy-three isolates were serotyped and eight (11%) were not of the NCDV serotype (bovine rotavirus serotype I). Of these eight, five belonged to the new bovine rotavirus serotype II and three were not typed, indicating the existence of one or more further serotypes. Cross-protection studies in gnotobiotic calves showed that cross-protection only occurred between rotaviruses of the same serotype, and even a minor serotype difference was sufficient for the calves to show a lack of cross-protection. The serotypes (I and II and the three untyped isolates) also showed differences in the rate of migration in polyacrylamide gel electrophoresis of some of their RNA segments (no. 4, 6, 7, 8, 9, 10), indicating that they were of different electropherotypes.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus infections in the laboratory rat: degree of cross protection following immunization with a heterologous strain.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">One hundred and twenty-one specific pathogen-free male Wistar rats eight to ten weeks of age were used to evaluate the efficacy of Parker's rat coronavirus (PRC) in affording cross protection on subsequent challenge with virulent sialodacryoadenitis (SDA) virus. Sixty-two animals were inoculated intranasally on day 0 and 21 days later with approximately 10(2) median tissue culture infective doses (TCID50) of the tenth passage of PRC replicated in L-2 cells. Animals were selected at random postvaccination to evaluate the safety and efficacy of PRC by histopathology, immunohistochemistry and serology. At three and six months postvaccination (PV), vaccinated and seronegative control rats were inoculated intranasally with approximately 10(3) TCID50 doses of virulent SDA virus. Challenged rats were then killed at 6, 10 and 14 days postchallenge and necropsied. Evaluations were based on lesion indices in lacrimal and salivary glands and respiratory tract, the presence of viral antigen by immunohistochemistry, and antibody response. Lesions were observed in rats killed PV, but in general, they were significantly reduced compared with those present in seronegative animals post-exposure to virulent SDA virus (p < or = 0.05). However, they were still considered to be an unacceptable level for a routine vaccination procedure. Potvaccination antibody titers to rat coronavirus were evident in all animals tested at three or six months prior to challenge with SDA virus.(ABSTRACT TRUNCATED AT 250 WORDS)</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiplex Evaluation of Influenza Neutralizing Antibodies with Potential Applicability to In-Field Serological Studies</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jul 3</span></i><p style="text-align: justify; text-justify: inter-word;">The increased number of outbreaks of H5 and H7 LPAI and HPAI viruses in poultry has major public and animal health implications. The continuous rapid evolution of these subtypes and the emergence of new variants influence the ability to undertake effective surveillance. Retroviral pseudotypes bearing influenza haemagglutinin (HA) and neuraminidase (NA) envelope glycoproteins represent a flexible platform for sensitive, readily standardized influenza serological assays. We describe a multiplex assay for the study of neutralizing antibodies that are directed against both influenza H5 and H7 HA. This assay permits the measurement of neutralizing antibody responses against two antigenically distinct HAs in the same serum/plasma sample thus increasing the amount and quality of serological data that can be acquired from valuable sera. Sera obtained from chickens vaccinated with a monovalent H5N2 vaccine, chickens vaccinated with a bivalent H7N1/H5N9 vaccine, or turkeys naturally infected with an H7N3 virus were evaluated in this assay and the results correlated strongly with data obtained by HI assay. We show that pseudotypes are highly stable under basic cold-chain storage conditions and following multiple rounds of freeze-thaw. We propose that this robust assay may have practical utility for in-field serosurveillance and vaccine studies in resource-limited regions worldwide.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>A review of Phase I trials of Ebola virus vaccines: what can we learn from the race to develop novel vaccines?</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 May 26</span></i><p style="text-align: justify; text-justify: inter-word;">Sporadic outbreaks of Ebola virus infection have been documented since the mid-Seventies and viral exposure can lead to lethal haemorrhagic fever with case fatalities as high as 90%. There is now a comprehensive body of data from both ongoing and completed clinical trials assessing various vaccine strategies, which were rapidly advanced through clinical trials in response to the 2013–2016 Ebola virus disease (EVD) public health emergency. Careful consideration of immunogenicity post vaccination is essential but has been somewhat stifled because of the wide array of immunological assays and outputs that have been used in the numerous clinical trials. We discuss here the different aspects of the immune assays currently used in the Phase I clinical trials for Ebola virus vaccines, and draw comparisons across the immune outputs where possible; various trials have examined both cellular and humoral immunity in European and African cohorts. Assessment of the safety data, the immunological outputs and the ease of field deployment for the various vaccine modalities will help both the scientific community and policy-makers prioritize and potentially license vaccine candidates. If this can be achieved, the next outbreak of Ebola virus, or other emerging pathogen, can be more readily contained and will not have such widespread and devastating consequences. This article is part of the themed issue ‘The 2013–2016 West African Ebola epidemic: data, decision-making and disease control’.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>The time course of the immune response to experimental coronavirus infection of man.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">After preliminary trials, the detailed changes in the concentration of specific circulating and local antibodies were followed in 15 volunteers inoculated with coronavirus 229E. Ten of them, who had significantly lower concentrations of pre-existing antibody than the rest, became infected and eight of these developed colds. A limited investigation of circulating lymphocyte populations showed some lymphocytopenia in infected volunteers. In this group, antibody concentrations started to increase 1 week after inoculation and reached a maximum about 1 week later. Thereafter antibody titres slowly declined. Although concentrations were still slightly raised 1 year later, this did not always prevent reinfection when volunteers were then challenged with the homologous virus. However, the period of virus shedding was shorter than before and none developed a cold. All of the uninfected group were infected on re-challenge although they also appeared to show some resistance to disease and in the extent of infection. These results are discussed with reference to natural infections with coronavirus and with other infections, such as rhinovirus infections.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Amending Koch’s postulates for viral disease: when “growth in pure culture” leads to a loss of virulence</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">It is a common laboratory practice to propagate viruses in cell culture. While convenient, these methodologies often result in unintentional genetic alterations, which have lead to adaptation and even attenuation in animal models of disease. An example is the attenuation of hantaviruses (family: Bunyaviridae, genus: Hantavirus) when cultured in vitro. In this case, viruses propagated in the natural reservoir species cause disease in nonhuman primates that closely mimics the human disease, but passaging in cell culture attenuates these viruses to the extent that do not cause any measurable disease in nonhuman primates. As efforts to develop animal models progress, it will be important to take into account the influences that culture in vitro may have on the virulence of viruses. In this review we discuss this phenomenon in the context of past and recent examples in the published literature.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Macaque Model for Severe Acute Respiratory Syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Rhesus and cynomolgus macaques were challenged with 10(7) PFU of a clinical isolate of the severe acute respiratory syndrome (SARS) coronavirus. Some of the animals developed a mild self-limited respiratory infection very different from that observed in humans with SARS. The macaque model as it currently exists will have limited utility in the study of SARS and the evaluation of therapies.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Measurement of CD8 and CD4 T Cell Responses in Mouse Lungs</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Study of the adaptive immune response to a viral challenge in an animal model often includes analysis of the T cell response. Here we discuss in detail the methods that are used to characterize the CD8 and CD4 T cell response following viral challenge in the lung.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Standards for Sequencing Viral Genomes in the Era of High-Throughput Sequencing</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jun 17</span></i><p style="text-align: justify; text-justify: inter-word;">Thanks to high-throughput sequencing technologies, genome sequencing has become a common component in nearly all aspects of viral research; thus, we are experiencing an explosion in both the number of available genome sequences and the number of institutions producing such data. However, there are currently no common standards used to convey the quality, and therefore utility, of these various genome sequences. Here, we propose five “standard” categories that encompass all stages of viral genome finishing, and we define them using simple criteria that are agnostic to the technology used for sequencing. We also provide genome finishing recommendations for various downstream applications, keeping in mind the cost-benefit trade-offs associated with different levels of finishing. Our goal is to define a common vocabulary that will allow comparison of genome quality across different research groups, sequencing platforms, and assembly techniques.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Funding vaccines for emerging infectious diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Immunization has played a large role in substantially reducing the infected and death tolls from infectious diseases. In the case of emerging diseases, the identity of the pathogen responsible, as well as the time and location for the next outbreak, cannot be accurately predicted using current means. Coupled with disjointed efforts towards the development of vaccines and a lack of funds and desire to advance promising products against known emerging pathogens to clinical trials, there has been a shortage of approved products ready for emergency use. Recent outbreaks have exposed these weaknesses, and the Coalition for Epidemic Preparedness Innovations (CEPI) was created in 2016 to address these issues. In this commentary, we discuss the establishment of such a global vaccine fund, and provide some additional points to consider for stimulating further discussion on this comprehensive, ambitious initiative.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Cough aerosol in healthy participants: fundamental knowledge to optimize droplet-spread infectious respiratory disease management</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Mar 21</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The Influenza A H1N1 virus can be transmitted via direct, indirect, and airborne route to non-infected subjects when an infected patient coughs, which expels a number of different sized droplets to the surrounding environment as an aerosol. The objective of the current study was to characterize the human cough aerosol pattern with the aim of developing a standard human cough bioaerosol model for Influenza Pandemic control. METHOD: 45 healthy non-smokers participated in the open bench study by giving their best effort cough. A laser diffraction system was used to obtain accurate, time-dependent, quantitative measurements of the size and number of droplets expelled by the cough aerosol. RESULTS: Voluntary coughs generated droplets ranging from 0.1 - 900 microns in size. Droplets of less than one-micron size represent 97% of the total number of measured droplets contained in the cough aerosol. Age, sex, weight, height and corporal mass have no statistically significant effect on the aerosol composition in terms of size and number of droplets. CONCLUSIONS: We have developed a standard human cough aerosol model. We have quantitatively characterized the pattern, size, and number of droplets present in the most important mode of person-to-person transmission of IRD: the cough bioaerosol. Small size droplets (< 1 μm) predominated the total number of droplets expelled when coughing. The cough aerosol is the single source of direct, indirect and/or airborne transmission of respiratory infections like the Influenza A H1N1 virus. STUDY DESIGN: Open bench, Observational, Cough, Aerosol study</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Interaction between the nasal microbiota and S. pneumoniae in the context of live-attenuated influenza vaccine</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 5</span></i><p style="text-align: justify; text-justify: inter-word;">Streptococcus pneumoniae is the main bacterial pathogen involved in pneumonia. Pneumococcal acquisition and colonization density is probably affected by viral co-infections, the local microbiome composition and mucosal immunity. Here, we report the interactions between live-attenuated influenza vaccine (LAIV), successive pneumococcal challenge, and the healthy adult nasal microbiota and mucosal immunity using an experimental human challenge model. Nasal microbiota profiles at baseline are associated with consecutive pneumococcal carriage outcome (non-carrier, low-dense and high-dense pneumococcal carriage), independent of LAIV co-administration. Corynebacterium/Dolosigranulum-dominated profiles are associated with low-density colonization. Lowest rates of natural viral co-infection at baseline and post-LAIV influenza replication are detected in the low-density carriers. Also, we detected the fewest microbiota perturbations and mucosal cytokine responses in the low-density carriers compared to non-carriers or high-density carriers. These results indicate that the complete respiratory ecosystem affects pneumococcal behaviour following challenge, with low-density carriage representing the most stable ecological state.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Interaction between the nasal microbiota and S. pneumoniae in the context of live-attenuated influenza vaccine</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 5</span></i><p style="text-align: justify; text-justify: inter-word;">Streptococcus pneumoniae is the main bacterial pathogen involved in pneumonia. Pneumococcal acquisition and colonization density is probably affected by viral co-infections, the local microbiome composition and mucosal immunity. Here, we report the interactions between live-attenuated influenza vaccine (LAIV), successive pneumococcal challenge, and the healthy adult nasal microbiota and mucosal immunity using an experimental human challenge model. Nasal microbiota profiles at baseline are associated with consecutive pneumococcal carriage outcome (non-carrier, low-dense and high-dense pneumococcal carriage), independent of LAIV co-administration. Corynebacterium/Dolosigranulum-dominated profiles are associated with low-density colonization. Lowest rates of natural viral co-infection at baseline and post-LAIV influenza replication are detected in the low-density carriers. Also, we detected the fewest microbiota perturbations and mucosal cytokine responses in the low-density carriers compared to non-carriers or high-density carriers. These results indicate that the complete respiratory ecosystem affects pneumococcal behaviour following challenge, with low-density carriage representing the most stable ecological state.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Interaction between the nasal microbiota and S. pneumoniae in the context of live-attenuated influenza vaccine</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 5</span></i><p style="text-align: justify; text-justify: inter-word;">Streptococcus pneumoniae is the main bacterial pathogen involved in pneumonia. Pneumococcal acquisition and colonization density is probably affected by viral co-infections, the local microbiome composition and mucosal immunity. Here, we report the interactions between live-attenuated influenza vaccine (LAIV), successive pneumococcal challenge, and the healthy adult nasal microbiota and mucosal immunity using an experimental human challenge model. Nasal microbiota profiles at baseline are associated with consecutive pneumococcal carriage outcome (non-carrier, low-dense and high-dense pneumococcal carriage), independent of LAIV co-administration. Corynebacterium/Dolosigranulum-dominated profiles are associated with low-density colonization. Lowest rates of natural viral co-infection at baseline and post-LAIV influenza replication are detected in the low-density carriers. Also, we detected the fewest microbiota perturbations and mucosal cytokine responses in the low-density carriers compared to non-carriers or high-density carriers. These results indicate that the complete respiratory ecosystem affects pneumococcal behaviour following challenge, with low-density carriage representing the most stable ecological state.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Personalized Vaccinology: A Review</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">At the current time, the field of vaccinology remains empirical in many respects. Vaccine development, vaccine immunogenicity, and vaccine efficacy have, for the most part, historically been driven by an empiric “isolate-inactivate-inject” paradigm. In turn, a population-level public health paradigm of “the same dose for everyone for every disease” model has been the normative thinking in regard to prevention of vaccine-preventable infectious diseases. In addition, up until recently, no vaccines specifically had been designed to overcome the immunosenescence of aging, consistent with a post-WWII mentality of developing vaccines and vaccine programs for children. It is now recognized that the current lack of knowledge concerning how immune responses to vaccines are generated is a critical barrier to understanding poor vaccine responses in the elderly and in immunoimmaturity, discovery of new correlates of vaccine immunogenicity (vaccine response biomarkers), and a directed approach to new vaccine development. The new fields of vaccinomics and adversomics provide models that permit global profiling of the innate, humoral, and cellular immune responses integrated at a systems biology level. This has advanced the science beyond that of reductionist scientific approaches by revealing novel interactions between and within the immune system and other biological systems (beyond transcriptional level), which are critical to developing “downstream” adaptive humoral and cellular responses to infectious pathogens and vaccines. Others have applied systems level approaches to the study of antibody responses (a.k.a. “systems serology”), [1] high dimensional cell subset immunophenotyping through CyTOF, [2, 3] and vaccine induced metabolic changes [4]. In turn, this knowledge is being utilized to better understand the following: identifying who is at risk for which infections; the level of risk that exists regarding poor immunogenicity and/or serious adverse events; and the type or dose of vaccine needed to fully protect an individual. In toto, such approaches allow for a personalized approach to the practice of vaccinology, analogous to the substantial inroads that individualized medicine is playing in other fields of human health and medicine. Herein we briefly review the field of vaccinomics, adversomics, and personalized vaccinology.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Global Causes of Diarrheal Disease Mortality in Children <5 Years of Age: A Systematic Review</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Sep 4</span></i><p style="text-align: justify; text-justify: inter-word;">Estimation of pathogen-specific causes of child diarrhea deaths is needed to guide vaccine development and other prevention strategies. We did a systematic review of articles published between 1990 and 2011 reporting at least one of 13 pathogens in children <5 years of age hospitalized with diarrhea. We included 2011 rotavirus data from the Rotavirus Surveillance Network coordinated by WHO. We excluded studies conducted during diarrhea outbreaks that did not discriminate between inpatient and outpatient cases, reporting nosocomial infections, those conducted in special populations, not done with adequate methods, and rotavirus studies in countries where the rotavirus vaccine was used. Age-adjusted median proportions for each pathogen were calculated and applied to 712 000 deaths due to diarrhea in children under 5 years for 2011, assuming that those observed among children hospitalized for diarrhea represent those causing child diarrhea deaths. 163 articles and WHO studies done in 31 countries were selected representing 286 inpatient studies. Studies seeking only one pathogen found higher proportions for some pathogens than studies seeking multiple pathogens (e.g. 39% rotavirus in 180 single-pathogen studies vs. 20% in 24 studies with 5–13 pathogens, p<0·0001). The percentage of episodes for which no pathogen could be identified was estimated to be 34%; the total of all age-adjusted percentages for pathogens and no-pathogen cases was 138%. Adjusting all proportions, including unknowns, to add to 100%, we estimated that rotavirus caused 197 000 [Uncertainty range (UR) 110 000–295 000], enteropathogenic E. coli 79 000 (UR 31 000–146 000), calicivirus 71 000 (UR 39 000–113 000), and enterotoxigenic E. coli 42 000 (UR 20 000–76 000) deaths. Rotavirus, calicivirus, enteropathogenic and enterotoxigenic E. coli cause more than half of all diarrheal deaths in children <5 years in the world.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>The time course of the humoral immune response to rhinovirus infection.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The specific humoral immune response of 17 volunteers to infection with human rhinovirus type 2 (HRV-2) has been measured both by neutralization and by ELISA. Six volunteers who had HRV-2-specific antibodies in either serum or nasal secretions before HRV-2 inoculation were resistant to infection and illness. Of the remaining 11 volunteers who had little pre-existing HRV-2-specific antibody, one was immune but 10 became infected and displayed increases in HRV-2-specific antibodies. These antibodies first increased 1-2 weeks after infection and reached a maximum at 5 weeks. All six resistant volunteers who had high pre-existing antibody and eight of the volunteers who became infected maintained their HRV-2-specific antibody for at least 1 year. At this time they were protected against reinfection. Two volunteers showed decreases in HRV-2-specific antibodies from either serum or nasal secretions. They became infected but not ill after HRV-2 inoculation 1 year later.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid development of vaccines against emerging pathogens: The replication-deficient simian adenovirus platform technology</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 16</span></i><p style="text-align: justify; text-justify: inter-word;">Despite the fact that there had been multiple small outbreaks of Ebola Virus Disease, when a large outbreak occurred in 2014 there were no vaccines or drugs available for use. Clinical development of multiple candidate vaccines was then initiated in parallel with attempts to contain the outbreak but only one vaccine was eventually tested in a phase III trial. In order to be better prepared for future outbreaks of known human pathogens, platform technologies to accelerate vaccine development should be employed, allowing vaccine developers to take advantage of detailed knowledge of the vaccine platform and facilitating rapid progress to clinical trials and eventually to vaccine stockpiles. This review gives an example of one such vaccine platform, replication-deficient simian adenoviruses, and describes progress in human and livestock vaccine development for three outbreak pathogens, Ebola virus, Rift Valley Fever Virus and Middle East Respiratory Syndrome Coronavirus.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus antibody titres in sera of healthy adults and experimentally infected volunteers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Six coronaviruses isolated in the U.S.A. have been inoculated into volunteers and all produced colds. Between 10 and 20% of infected volunteers developed heterologous antibody responses after these and other experimental infections with coronaviruses. The haemagglutination-inhibition test with the OC43 virus strain was found to detect antibody rises after infection with a variety of strains. Studies on normal adult sera taken between 1965 and 1970 revealed a high frequency of neutralizing antibody to one strain (229 E) and a frequency of HI antibody to strain OC43 which fluctuated from year to year. Complement-fixing antibodies to these two viruses were also found, revealing an apparent increase in the activity of coronaviruses in the general population of the U.K., during the winter of 1968-9.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Isotype-specific antibody responses to foot-and-mouth disease virus in sera and secretions of "carrier' and "non-carrier' cattle.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Isotype-specific antibody responses to foot-and-mouth disease virus (FMDV) were measured in the sera and upper respiratory tract secretions of vaccinated and susceptible cattle challenged with FMDV by direct contact or by intranasal inoculation. A comparison was made between cattle that eliminated FMDV and those that developed and maintained a persistent infection. Serological and mucosal antibody responses were detected in all animals after challenge. IgA and IgM were detected before the development of IgG1 and IgG2 responses. IgM was not detected in vaccinated cattle. Challenge with FMDV elicited a prolonged biphasic secretory antibody response in FMDV "carrier' animals only. The response was detected as FMDV-specific IgA in both mucosal secretions and serum samples, which gained statistical significance (P < 0.05) by 5 weeks after challenge. This observation could represent the basis of a test to differentiate vaccinated and/or recovered convalescent cattle from FMDV "carriers'.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>The Convergence of High-Consequence Livestock and Human Pathogen Research and Development: A Paradox of Zoonotic Disease</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 May 30</span></i><p style="text-align: justify; text-justify: inter-word;">The World Health Organization (WHO) estimates that zoonotic diseases transmitted from animals to humans account for 75 percent of new and emerging infectious diseases. Globally, high-consequence pathogens that impact livestock and have the potential for human transmission create research paradoxes and operational challenges for the high-containment laboratories that conduct work with them. These specialized facilities are required for conducting all phases of research on high-consequence pathogens (basic, applied, and translational) with an emphasis on both the generation of fundamental knowledge and product development. To achieve this research mission, a highly-trained workforce is required and flexible operational methods are needed. In addition, working with certain pathogens requires compliance with regulations such as the Centers for Disease Control (CDC) and the U.S. Department of Agriculture (USDA) Select Agent regulations, which adds to the operational burden. The vast experience from the existing studies at Plum Island Animal Disease Center, other U.S. laboratories, and those in Europe and Australia with biosafety level 4 (BSL-4) facilities designed for large animals, clearly demonstrates the valuable contribution this capability brings to the efforts to detect, prepare, prevent and respond to livestock and potential zoonotic threats. To raise awareness of these challenges, which include biosafety and biosecurity issues, we held a workshop at the 2018 American Society for Microbiology (ASM) Biothreats conference to further discuss the topic with invited experts and audience participants. The workshop covered the subjects of research funding and metrics, economic sustainment of drug and vaccine development pipelines, workforce turnover, and the challenges of maintaining operational readiness of high containment laboratories.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibody Responses of Cattle with Respiratory Coronavirus Infections during Pathogenesis of Shipping Fever Pneumonia Are Lower with Antigens of Enteric Strains than with Those of a Respiratory Strain</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The serum antibody responses of cattle with respiratory coronavirus infections during the pathogenesis of shipping fever pneumonia were analyzed with different bovine coronavirus antigens, including those from a wild-type respiratory bovine coronavirus (RBCV) strain (97TXSF-Lu 15-2) directly isolated from lung tissue from a fatally infected bovine, a wild-type enteropathogenic bovine coronavirus (EBCV) strain (Ly 138-3), and the highly cell culture-adapted, enteric prototype strain (EBCV L9-81). Infectivity-neutralizing (IN) and hemagglutinin-inhibiting (HAI) activities were tested. Sequential serum samples, collected during the onset of the respiratory coronavirus infection and at weekly intervals for 5 weeks thereafter, had significantly higher IN and HAI titers for antigens of RBCV strain 97TXSF-Lu15-2 than for the wild-type and the highly cell culture-adapted EBCV strains, with P values ranging from <0.0001 to 0.0483. The IN and HAI antibody responses against the two EBCV strains did not differ significantly, but the lowest titers were detected with EBCV strain L9-81.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of an indirect ELISA, blocking ELISA, fluorescent microsphere immunoassay and fluorescent focus neutralization assay for serologic evaluation of exposure to North American strains of Porcine Epidemic Diarrhea Virus</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Recent, severe outbreaks of porcine epidemic diarrhea virus (PEDV) in Asia and North America highlight the need for well-validated diagnostic tests for the identification of PEDV infected animals and evaluation of their immune status to this virus. PEDV was first detected in the U.S. in May 2013 and spread rapidly across the country. Some serological assays for PEDV have been previously described, but few were readily available in the U.S. Several U.S. laboratories quickly developed indirect fluorescent antibody (IFA) assays for the detection of antibodies to PEDV in swine serum, indicating prior exposure. However, the IFA has several disadvantages, including low throughput and relatively subjective interpretation. Different serologic test formats have advantages and disadvantages, depending on the questions being asked, so a full repertoire of tests is useful. Therefore, the objective of this study was to develop and validate multiple improved serological assays for PEDV, including an indirect ELISA (iELISA); a highly specific monoclonal antibody-based blocking ELISA (bELISA); fluorescent microsphere immunoassays (FMIA) that can be multiplexed to monitor exposure to multiple antigens and pathogens simultaneously; and a fluorescent focus neutralization assay (FFN) to measure functional virus neutralizing antibodies. RESULTS: A recombinant North American nucleoprotein (NP) based iELISA was developed and validated along with a bELISA using newly developed PEDV-NP specific biotinylated monoclonal antibodies (mAbs) and an FMIA using magnetic beads coupled with expressed NA PEDV-NP. Receiver operating characteristic (ROC) analysis was performed using swine serum samples (iELISA n = 1486, bELISA n = 1186, FMIA n = 1420). The ROC analysis for the FMIA showed estimated sensitivity and specificity of 98.2 and 99.2 %, respectively. The iELISA and bELISA showed a sensitivity and specificity of 97.9 and 97.6 %; and 98.2 and 98.9 %, respectively. Inter-rater (kappa) agreement was calculated to be 0.941 between iELISA and IFA, 0.945 between bELISA and IFA and 0.932 between FMIA and IFA. Similar comparative kappa values were observed between the iELISA, bELISA and FMIA, which demonstrated a significant level of testing agreement among the three assays. No cross-reactivity with the closely related coronaviruses, transmissible gastroenteritis virus (TGEV) or porcine respiratory coronavirus (PRCV) was noted with these assays. All three assays detected seroconversion of naïve animals within 6–9 days post exposure. The FFN assay allows relative quantitation of functional neutralizing antibodies in serum, milk or colostrum samples. CONCLUSION: Well-validated iELISA, bELISA and FMIA assays for the detection of PEDV antibodies were developed and showed good correlation with IFA and each other. Each assay format has advantages that dictate how they will be used in the field. Newly developed mAbs to the PEDV-NP were used in the bELISA and for expediting FFN testing in the detection and quantitation of neutralizing antibodies. In addition, these PEDV mAbs are useful for immunohistochemistry, fluorescent antibody staining and other antigen-based tests. Measurement of neutralizing antibody responses using the FFN assay may provide a valuable tool for assessment of vaccine candidates or protective immunity.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>The promise and challenge of high-throughput sequencing of the antibody repertoire</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Efforts to determine the antibody repertoire encoded by B cells in the blood or lymphoid organs using high-throughput DNA sequencing technologies have been advancing at an extremely rapid pace and are transforming our understanding of humoral immune responses. Information gained from high-throughput DNA sequencing of immunoglobulin genes (Ig-seq) can be applied to detect B-cell malignancies with high sensitivity, to discover antibodies specific for antigens of interest, to guide vaccine development and to understand autoimmunity. Rapid progress in the development of experimental protocols and informatics analysis tools is helping to reduce sequencing artifacts, to achieve more precise quantification of clonal diversity and to extract the most pertinent biological information. That said, broader application of Ig-seq, especially in clinical settings, will require the development of a standardized experimental design framework that will enable the sharing and meta-analysis of sequencing data generated by different laboratories.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Complement-Mediated Neutralization of Canine Distemper Virus In Vitro: Cross-Reaction between Vaccine Onderstepoort and Field KDK-1 Strains with Different Hemagglutinin Gene Characteristics</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The properties of neutralization of antigens of canine distemper virus Onderstepoort and a recent field isolate, KDK-1, were investigated with strain-specific dog sera. A conventional neutralization assay indicated antigenic dissimilarity between the strains; however, when guinea pig complement was included in the reaction mixture, the strains were neutralized with not only the homologous but also the heterologous antibodies.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>A sensitive retroviral pseudotype assay for influenza H5N1‐neutralizing antibodies</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 May 26</span></i><p style="text-align: justify; text-justify: inter-word;">Background  The World Health Organisation (WHO) recommended the development of simple, safe, sensitive and specific neutralization assays for avian influenza antibodies. We have used retroviral pseudotypes bearing influenza H5 hemagglutinin (HA) as safe, surrogate viruses for influenza neutralization assays which can be carried out at Biosafety Level 2. Results  Using our assay, sera from patients who had recovered from infection with influenza H5N1, and sera from animals experimentally immunized or infected with H5 tested positive for the presence of neutralizing antibodies to H5N1. Pseudotype neutralizing antibody titers were compared with titers obtained by hemagglutinin inhibition (HI) assays and microneutralization (MN) assays using live virus, and showed a high degree of correlation, sensitivity and specificity. Conclusions  The pseudotype neutralization assay is as sensitive as horse erythrocyte HI and MN for the detection of antibodies to H5N1. It is safer, and can be applied in a high‐throughput format for human and animal surveillance and for the evaluation of vaccines.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Serotyping of cell culture-adapted subgroup 2 human rotavirus strains by neutralization.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Nine human rotavirus strains from stools of infants with gastroenteritis were serially propagated in MA-104 cell cultures. All strains were identified as subgroup 2 rotaviruses by RNA gel electrophoresis, complement fixation, and enzyme-linked immunosorbent assay. The human rotavirus strains were propagated for 15 to 20 passages and then used for immunization of guinea pigs and rabbits. Animal antisera were also raised against a subgroup 1 human strain purified from stools and against the cell culture-adapted Wa strain, a reference subgroup 2 rotavirus of human origin. Cross-neutralization studies revealed the existence of two distinct serotypes within the cell culture-adapted subgroup 2 human rotaviruses: strains related and unrelated to strain Wa were classified as serotypes 1 and 3, respectively. Results with convalescent-phase sera from infants with primary rotavirus infections confirmed the existence of two serotypes within subgroup 2, and the serotypes responsible for primary subgroup 2 infections could be determined on the basis of the neutralizing reactivity of convalescent sera.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of multiple vaccine vectors in a single heterologous prime boost trial</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The prevention of infectious disease via prophylactic immunization is a mainstay of global public health efforts. Vaccine design would be facilitated by a better understanding of the type and durability of immune responses generated by different vaccine vectors. We report here the results of a comparative immunogenicity trial of six different vaccine vectors expressing the same insert antigen, cowpox virus B5 (CPXV-B5). Of those vectors tested, recombinant adenovirus (rAd5) was the most immunogenic, inducing the highest titer anti-B5 antibodies and conferring protection from sublethal vaccinia virus challenge in mice after a single immunization. We tested select heterologous prime-boost combinations and identified recombinant vesicular stomatitis virus (rVSV) and recombinant Venezuelan equine encephalitis virus replicons (VRP) as the most synergistic regimen. Comparative data such as those presented here are critical to efforts to generate protective vaccines for emerging infectious diseases as well as for biothreat agents.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Proposal for detection of 2019-nCoV nucleic acid in clinical laboratories</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December, the outbreak of a novel coronavirus (2019-nCoV) in Wuhan, China, has attracted extensive global attention. On January 20, 2020,the Chinese health authorities upgraded the coronavirus to a Class B infectious disease in the Law of the People's Republic of China on the Prevention and Treatment of Infectious Diseases , and considered it as Class A infectious diseases in disease control and prevention. On January 22, 2020, the 2019-nCoV nucleic acid detection test was listed as the diagnostic criteria in the 'guidelines for diagnosis and treatment of pneumonia due to 2019-nCoV (Trial Version 2)' . Therefore, standardizing the operation process of the 2019-nCoV nucleic acid detection in clinical laboratories has become a top priority. It is of paramount importance to establish standard protocols for detection of the 2019-nCoV nucleic acids in clinical laboratories to improve the reliability of the results and ensure the biosafety of laboratory personnel.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Improving vaccine trials in infectious disease emergencies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Unprecedented global effort is underway to facilitate testing of countermeasures in infectious disease emergencies. Better understanding of the various options for trial design, as well as preliminary global agreement on the most suitable designs for the various scenarios, are needed now—in advance of outbreaks. What would enhance, then, the speed, validity, and ethics of clinical studies of such countermeasures? Focusing on studies of vaccine efficacy and effectiveness in emergencies, we highlight three needs: for formal randomized trials—even in most emergencies; for individually-randomized trials—even in many emergencies; and for six areas of innovation in trial methodology. These needs should inform current updates of protocols and roadmaps.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of porcine epidemic diarrhea virus transmission and the immune response in growing pigs</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 May 6</span></i><p style="text-align: justify; text-justify: inter-word;">Clinical disease associated with porcine epidemic diarrhea virus (PEDV) infection in naïve pigs is well chronicled; however, information on endemic PEDV infection is limited. To characterize chronic PEDV infection, the duration of infectious virus shedding and development of protective immunity was determined. On Day 0 (D0), a growing pig was challenged with PEDV and 13 contacts were commingled. On D7, 9 contact pigs (principal virus group (PG)), were selected, moved to a separate room and commingled with one sentinel pig (S1). This process was repeated weekly with S2, S3 and S4. The PG was PEDV-positive by PCR from D3-11, with some pigs intermittently positive to D42. Pigs S1 and S2 were PEDV-positive within 24 hours of commingling. Antibodies were detected in all PG by D21 and by 7 days post-contact in S1 and S2. Pigs S3 and S4 were PCR and antibody negative following commingling. To evaluate protective immunity, 5 naïve pigs (N) and the PG were challenged (N/C, PG/C) with homologous virus on D49. All N/C pigs were PEDV PCR-positive by D52 with detection out to D62 in 3/5 N/C pigs. All PG/C pigs were PEDV PCR-negative post-challenge. By D63, all N/C seroconverted. Although PEDV RNA was demonstrated in pigs after primary infection until D42, infectious PEDV capable of horizontal transmission to naïve pigs was only shed 14–16 days after infection to age-matched pigs. Homologous re-challenge 49 days post initial PEDV exposure did not result in re-infection of the pigs. This demonstrates potential for an effective PEDV vaccine.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Dissimilar Properties of Two Recombinant Lactobacillus acidophilus Strains Displaying Salmonella FliC with Different Anchoring Motifs</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Display of heterologous antigens on the cell surface is considered a useful technique for vaccine delivery by recombinant lactobacilli. In this study, two recombinant Lactobacillus acidophilus derivatives displaying Salmonella flagellin (FliC) were constructed using different anchor motifs. In one instance, the FliC protein was fused to the C-terminal region of a cell envelope proteinase (PrtP) and was bound to the cell wall by electrostatic bonds. In the other case, the same antigen was conjugated to the anchor region of mucus binding protein (Mub) and was covalently associated with the cell wall by an LPXTG motif. These two recombinant L. acidophilus cell surface displays resulted in dissimilar maturation and cytokine production by human myeloid dendritic cells. The surface-associated antigen was highly sensitive to simulated gastric and small intestinal juices. By supplementation with bicarbonate buffer and soybean trypsin inhibitor, the cell surface antigen was protected from proteolytic enzymes during gastric challenge in vitro. The protective reagents also increased the viability of the L. acidophilus cells upon challenge with simulated digestive juices. These results demonstrate the importance of protecting cells and their surface-associated antigens during oral immunization.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Virulence, Transmission, and Heterologous Protection of Four Isolates of Haemophilus parasuis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Haemophilus parasuis causes Glässer's disease, a syndrome of polyserositis, meningitis, and arthritis in swine. Previous studies with H. parasuis have revealed virulence disparity among isolates and inconsistent heterologous protection. In this study, virulence, direct transmission, and heterologous protection of 4 isolates of H. parasuis (SW114, 12939, MN-H, and 29755) were evaluated using a highly susceptible pig model. In an initial experiment, isolates 12939, MN-H, and 29755 caused Glässer's disease, while strain SW114 failed to cause any clinical signs of disease. One pig from each group challenged with MN-H or 29755 failed to develop clinical disease but was able to transmit H. parasuis to noninfected pigs, which subsequently developed Glässer's disease. Pigs colonized with SW114, 29755, or MN-H that were free of clinical disease were protected from a subsequent challenge with isolate 12939. In a following experiment, pigs vaccinated with strain SW114 given as either a bacterin intramuscularly or a live intranasal vaccine were protected from subsequent challenge with isolate 12939; however, some pigs given live SW114 developed arthritis. Overall these studies demonstrated that pigs infected with virulent isolates of H. parasuis can remain healthy and serve as reservoirs for transmission to naive pigs and that heterologous protection among H. parasuis isolates is possible. In addition, further attenuation of strain SW114 is necessary if it is to be used as a live vaccine.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Infectious Middle East Respiratory Syndrome Coronavirus Excretion and Serotype Variability Based on Live Virus Isolates from Patients in Saudi Arabia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The newly emerged Middle East respiratory syndrome coronavirus (MERS-CoV) has infected at least 1,082 people, including 439 fatalities. So far, no empirical virus isolation study has been done to elucidate infectious virus secretion or serotype variability. Here, we used 51 respiratory samples from 32 patients with confirmed MERS-CoV infection for virus isolation in Vero B4 and Caco-2 cells. We found Caco-2 cells to significantly enhance isolation success over routinely used Vero cells. Isolation success correlated with viral RNA concentration and time after diagnosis as well as with the amount of IgA antibodies secreted in respiratory samples used for isolation. Results from plaque reduction neutralization assays using a representative range of serum samples and virus isolates suggested that all circulating human MERS-CoV strains represent one single serotype. The choice of prototype strain is not likely to influence the success of candidate MERS-CoV vaccines. However, vaccine formulations should be evaluated for their potential to induce IgA.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Injection Route and TLR9 Agonist Addition Significantly Impact Heroin Vaccine Efficacy</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">[Image: see text] Active immunization is an effective means of blocking the pharmacodynamic effects of drugs and holds promise as a treatment for heroin addiction. Previously, we demonstrated the efficacy of our first-generation vaccine in blocking heroin self-administration in rats, however, many vaccine components can be modified to further improve performance. Herein we examine the effects of varying heroin vaccine injection route and adjuvant formulation. Mice immunized via subcutaneous (sc) injection exhibited inferior anti-heroin titers compared to intraperitoneal (ip) and sc/ip coadministration injection routes. Addition of TLR9 agonist cytosine-guanine oligodeoxynucleotide 1826 (CpG ODN 1826) to the original alum adjuvant elicited superior antibody titers and opioid affinities compared to alum alone. To thoroughly assess vaccine efficacy, full dose–response curves were generated for heroin-induced analgesia in both hot plate and tail immersion tests. Mice treated with CpG ODN 1826 exhibited greatly shifted dose–response curves (10–13-fold vs unvaccinated controls) while non-CpG ODN vaccine groups did not exhibit the same robust effect (2–7-fold shift for ip and combo, 2–3-fold shift for sc). Our results suggest that CpG ODN 1826 is a highly potent adjuvant, and injection routes should be considered for development of small molecule–protein conjugate vaccines. Lastly, this study has established a new standard for assessing drugs of abuse vaccines, wherein a full dose–response curve should be performed in an appropriate behavioral task.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Injection Route and TLR9 Agonist Addition Significantly Impact Heroin Vaccine Efficacy</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">[Image: see text] Active immunization is an effective means of blocking the pharmacodynamic effects of drugs and holds promise as a treatment for heroin addiction. Previously, we demonstrated the efficacy of our first-generation vaccine in blocking heroin self-administration in rats, however, many vaccine components can be modified to further improve performance. Herein we examine the effects of varying heroin vaccine injection route and adjuvant formulation. Mice immunized via subcutaneous (sc) injection exhibited inferior anti-heroin titers compared to intraperitoneal (ip) and sc/ip coadministration injection routes. Addition of TLR9 agonist cytosine-guanine oligodeoxynucleotide 1826 (CpG ODN 1826) to the original alum adjuvant elicited superior antibody titers and opioid affinities compared to alum alone. To thoroughly assess vaccine efficacy, full dose–response curves were generated for heroin-induced analgesia in both hot plate and tail immersion tests. Mice treated with CpG ODN 1826 exhibited greatly shifted dose–response curves (10–13-fold vs unvaccinated controls) while non-CpG ODN vaccine groups did not exhibit the same robust effect (2–7-fold shift for ip and combo, 2–3-fold shift for sc). Our results suggest that CpG ODN 1826 is a highly potent adjuvant, and injection routes should be considered for development of small molecule–protein conjugate vaccines. Lastly, this study has established a new standard for assessing drugs of abuse vaccines, wherein a full dose–response curve should be performed in an appropriate behavioral task.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of 3 vaccination strategies against porcine reproductive and respiratory syndrome virus, Mycoplasma hyopneumoniae, and porcine circovirus type 2 on a 3 pathogen challenge model</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The objective of this study was to compare clinical, microbiologic, immunologic, and pathologic parameters in pigs each concurrently administered porcine reproductive and respiratory syndrome virus (PRRSV), Mycoplasma hyopneumoniae, and porcine circovirus type 2 (PCV2) vaccine from 1 of 2 commercial sources at 21 days of age and challenged with field strains of each of the 3 pathogens. Pigs were challenged with PRRSV and M. hyopneumoniae at 42 days of age (−14 days post-challenge, dpc) followed by a challenge with PCV2 at 56 days of age (0 dpc). Significant differences were observed between vaccinated challenged and unvaccinated challenged groups in clinical (average daily gain and clinical signs), microbiologic (viremia and nasal shedding), immunologic (antibodies and interferon-γ secreting cells), and pathologic (lesions) outcomes. Significant differences were observed among the 3 vaccinated challenged groups in microbiologic (nasal shedding of M. hyopneumoniae and viremia of PCV2) and immunologic (M. hyopneumoniae- and PCV2-specific interferon-γ secreting cells) outcomes. The vaccination regimen for PRRSV vaccine, M. hyopneumoniae vaccine, and PCV2 vaccine is efficacious for controlling triple challenge with PRRSV, M. hyopneumoniae, and PCV2 from weaning to finishing period.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccines for emerging infectious diseases: Lessons from MERS coronavirus and Zika virus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 28</span></i><p style="text-align: justify; text-justify: inter-word;">The past decade and a half has been characterized by numerous emerging infectious diseases. With each new threat, there has been a call for rapid vaccine development. Pathogens such as the Middle East Respiratory Syndrome coronavirus (MERS-CoV) and the Zika virus represent either new viral entities or viruses emergent in new geographic locales and characterized by novel complications. Both serve as paradigms for the global spread that can accompany new pathogens. In this paper, we review the epidemiology and pathogenesis of MERS-CoV and Zika virus with respect to vaccine development. The challenges in vaccine development and the approach to clinical trial design to test vaccine candidates for disease entities with a changing epidemiology are discussed.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid identification of novel antigens of Salmonella Enteritidis by microarray-based immunoscreening</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb 14</span></i><p style="text-align: justify; text-justify: inter-word;">We report on an approach to rapidly screen thousands of Salmonella Enteritidis proteins with the goal of identifying novel immunodominant proteins. We used a microarray-based system that warrants high throughput and easy handling. Seven immunogenic candidates were selected after screening. Comparative analyses by ELISA and microarrays manifested their immunodominant character. The large repetitive protein (SEN4030) that plays a role as a putative adhesin in initial cell surface interaction and is highly specific to Salmonella is considered to be the most suitable protein for a diagnostic approach. The results further demonstrate that the strategy applied herein is convenient for specifically identifying immunogenic proteins of pathogenic microorganisms. Consequently, it enables a sound assessment of promising candidates for diagnostic applications and vaccine development. Moreover, the elucidation of immunogenic proteins may assist in unveiling unknown virulence-associated factors, thus furthering the understanding of the underlying pathogenicity of Salmonella in general, and of S. Enteritidis, one of the most frequently detected serovars of this pathogen, in particular. [Figure: see text]</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Middle East Respiratory Syndrome Coronavirus vaccines – advances and challenges</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an emerging pathogen with the potential to pose a threat to global public health. Sporadic cases and outbreaks continue to be reported in the Middle East, and case fatality rates remain high at approximately 36% globally. No specific preventive or therapeutic countermeasures currently exist. A safe and effective vaccine could play an important role in protecting against the threat from MERS-CoV. This review discusses human vaccine candidates currently under development, and explores viral characteristics, molecular epidemiology and immunology relevant to MERS-CoV vaccine development. At present, a DNA vaccine candidate has begun a human clinical trial, while two vector-based candidates will very soon begin human trials. Protein-based vaccines are still at pre-clinical stage. Challenges to successful development include incomplete understanding of viral transmission, pathogenesis and immune response (in particular at the mucosal level), no optimal animal challenge models, lack of standardized immunological assays, and insufficient sustainable funding.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental inoculation of cats with human coronavirus 229E and subsequent challenge with feline infectious peritonitis virus.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Minimal-disease cats exposed to live human coronavirus 229E developed homologous antibody responses that suggested little or no replication of the virus in inoculated animals. Oronasal and subcutaneous inoculation of coronavirus 229E did not elicit an antibody response by heterologous (transmissible gastroenteritis virus, canine coronavirus) neutralization or by heterologous (transmissible gastroenteritis virus) kinetics-based enzyme-linked immunosorbent assay. No clinical signs attributable to coronavirus 229E were seen in inoculated cats. Although the number of animals in each of the five experimental groups was small (n = 2), antibodies produced in response to the virus did not appear to sensitize cats to subsequent feline infectious peritonitis virus challenge, but neither did they cross-protect cats against the challenge dose.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Successful experimental challenge of dogs with canine parvovirus-2.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Withholding food from dogs for 24 hours prior to, and for 48 hours following oral challenge with a gut mucosal homogenate of canine parvovirus-2, was a successful means of reproducing gastroenteric signs of canine parvovirus-2 infection. Twenty-one of 24 dogs, which had previously received various vaccine preparations of mink enteritis virus or were unvaccinated, and which were starved at challenge, developed soft or liquid feces with large or without large clots of mucus. Altered feces were most frequent on postexposure day 11. Seven dogs passed frank blood in their stools on one or more occasions and seven dogs vomited sporadically. Pyrexia was noted in 71.6% of the dogs on postexposure day 6 and lymphopenia was detected on postexposure day 5 or 6 in 50% of the dogs monitored. In contrast, four dogs not starved at the time of challenge remained free of gastrointestinal signs apart from one dog which passed a soft stool with scant mucus on one day, postexposure day 6. Also four dogs vaccinated with a killed canine parvovirus-2 vaccine preparation and subsequently starved at the time of challenge, remained clinically healthy. Apart from these last mentioned four dogs, all others shed canine parvovirus-2 in their feces following challenge.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Conventional respiratory support therapy for Severe Acute Respiratory Infections (SARI): Clinical indications and nosocomial infection prevention and control</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory infection (SARI) diseases (such as SARS, MERS, pH1N1) can rapidly progress to acute respiratory failure with high lethality. The outbreak of a novel coronavirus infection can lead to 15% ~ 30% patients developing into acute respiratory distress syndrome (ARDS). Respiratory support is the most important therapy for SARI patients with respiratory failure. However, respiratory support is a high skilled technology, which means inappropriate application may bring related complications and cross infection of SARI pathogens among medical staff and non-medical personnel in hospital. Therefore, it is meaningful to established a standardized indication of respiratory support and to prevent related nosocomial transmission in SARI patients.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>A Longitudinal Cohort Study in Calves Evaluated for Rotavirus Infections from One to 12 Months of Age by Sequential Serological Assays</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Using an immunocytochemical staining assay involving six different recombinant baculoviruses with each expressing one of the major bovine rotavirus VP7 (G6, G8 and G10) and VP4 (P6[1], P7[5] and P8[11]) serotypes, we analyzed IgG antibody responses to individual proteins in archival serum samples collected from 31 calves monthly from 1 month to 12 months of age during 1974–1975 in Higley, Arizona. Seroresponses to VP7 and VP4, as determined by a 4-fold or greater antibody response, were not always elicited concurrently following infection: in some calves, (i) seroresponses to VP7 were detected earlier than to VP4 or vice versa; and (ii) a subsequent 2(nd) seroresponse was detected for VP7 or VP4 only. In addition, a second infection was more likely to be caused by different G and/or P types. Analyses of serum samples showed that the most frequent G-P combination was G8P6[1] followed by G8P7[5], G8P8[11] and G6P6[1].</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostics for Lassa fever virus: a genetically diverse pathogen found in low-resource settings</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 7</span></i><p style="text-align: justify; text-justify: inter-word;">Lassa fever virus (LASV) causes acute viral haemorrhagic fever with symptoms similar to those seen with Ebola virus infections. LASV is endemic to West Africa and is transmitted through contact with excretions of infected Mastomys natalensis rodents and other rodent species. Due to a high fatality rate, lack of treatment options and difficulties with prevention and control, LASV is one of the high-priority pathogens included in the WHO R&D Blueprint. The WHO LASV vaccine strategy relies on availability of effective diagnostic tests. Current diagnostics for LASV include in-house and commercial (primarily research-only) laboratory-based serological and nucleic acid amplification tests. There are two commercially available (for research use only) rapid diagnostic tests (RDTs), and a number of multiplex panels for differential detection of LASV infection from other endemic diseases with similar symptoms have been evaluated. However, a number of diagnostic gaps remain. Lineage detection is a challenge due to the genomic diversity of LASV, as pan-lineage sensitivity for both molecular and immunological detection is necessary for surveillance and outbreak response. While pan-lineage ELISA and RDTs are commercially available (for research use only), validation and external quality assessment (EQA) is needed to confirm detection sensitivity for all known or relevant strains. Variable sensitivity of LASV PCR tests also highlights the need for improved validation and EQA. Given that LASV outbreaks typically occur in low-resource settings, more options for point-of-care testing would be valuable. These requirements should be taken into account in target product profiles for improved LASV diagnostics.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostics for Lassa fever virus: a genetically diverse pathogen found in low-resource settings</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 7</span></i><p style="text-align: justify; text-justify: inter-word;">Lassa fever virus (LASV) causes acute viral haemorrhagic fever with symptoms similar to those seen with Ebola virus infections. LASV is endemic to West Africa and is transmitted through contact with excretions of infected Mastomys natalensis rodents and other rodent species. Due to a high fatality rate, lack of treatment options and difficulties with prevention and control, LASV is one of the high-priority pathogens included in the WHO R&D Blueprint. The WHO LASV vaccine strategy relies on availability of effective diagnostic tests. Current diagnostics for LASV include in-house and commercial (primarily research-only) laboratory-based serological and nucleic acid amplification tests. There are two commercially available (for research use only) rapid diagnostic tests (RDTs), and a number of multiplex panels for differential detection of LASV infection from other endemic diseases with similar symptoms have been evaluated. However, a number of diagnostic gaps remain. Lineage detection is a challenge due to the genomic diversity of LASV, as pan-lineage sensitivity for both molecular and immunological detection is necessary for surveillance and outbreak response. While pan-lineage ELISA and RDTs are commercially available (for research use only), validation and external quality assessment (EQA) is needed to confirm detection sensitivity for all known or relevant strains. Variable sensitivity of LASV PCR tests also highlights the need for improved validation and EQA. Given that LASV outbreaks typically occur in low-resource settings, more options for point-of-care testing would be valuable. These requirements should be taken into account in target product profiles for improved LASV diagnostics.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostics for Lassa fever virus: a genetically diverse pathogen found in low-resource settings</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 7</span></i><p style="text-align: justify; text-justify: inter-word;">Lassa fever virus (LASV) causes acute viral haemorrhagic fever with symptoms similar to those seen with Ebola virus infections. LASV is endemic to West Africa and is transmitted through contact with excretions of infected Mastomys natalensis rodents and other rodent species. Due to a high fatality rate, lack of treatment options and difficulties with prevention and control, LASV is one of the high-priority pathogens included in the WHO R&D Blueprint. The WHO LASV vaccine strategy relies on availability of effective diagnostic tests. Current diagnostics for LASV include in-house and commercial (primarily research-only) laboratory-based serological and nucleic acid amplification tests. There are two commercially available (for research use only) rapid diagnostic tests (RDTs), and a number of multiplex panels for differential detection of LASV infection from other endemic diseases with similar symptoms have been evaluated. However, a number of diagnostic gaps remain. Lineage detection is a challenge due to the genomic diversity of LASV, as pan-lineage sensitivity for both molecular and immunological detection is necessary for surveillance and outbreak response. While pan-lineage ELISA and RDTs are commercially available (for research use only), validation and external quality assessment (EQA) is needed to confirm detection sensitivity for all known or relevant strains. Variable sensitivity of LASV PCR tests also highlights the need for improved validation and EQA. Given that LASV outbreaks typically occur in low-resource settings, more options for point-of-care testing would be valuable. These requirements should be taken into account in target product profiles for improved LASV diagnostics.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostics for Lassa fever virus: a genetically diverse pathogen found in low-resource settings</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 7</span></i><p style="text-align: justify; text-justify: inter-word;">Lassa fever virus (LASV) causes acute viral haemorrhagic fever with symptoms similar to those seen with Ebola virus infections. LASV is endemic to West Africa and is transmitted through contact with excretions of infected Mastomys natalensis rodents and other rodent species. Due to a high fatality rate, lack of treatment options and difficulties with prevention and control, LASV is one of the high-priority pathogens included in the WHO R&D Blueprint. The WHO LASV vaccine strategy relies on availability of effective diagnostic tests. Current diagnostics for LASV include in-house and commercial (primarily research-only) laboratory-based serological and nucleic acid amplification tests. There are two commercially available (for research use only) rapid diagnostic tests (RDTs), and a number of multiplex panels for differential detection of LASV infection from other endemic diseases with similar symptoms have been evaluated. However, a number of diagnostic gaps remain. Lineage detection is a challenge due to the genomic diversity of LASV, as pan-lineage sensitivity for both molecular and immunological detection is necessary for surveillance and outbreak response. While pan-lineage ELISA and RDTs are commercially available (for research use only), validation and external quality assessment (EQA) is needed to confirm detection sensitivity for all known or relevant strains. Variable sensitivity of LASV PCR tests also highlights the need for improved validation and EQA. Given that LASV outbreaks typically occur in low-resource settings, more options for point-of-care testing would be valuable. These requirements should be taken into account in target product profiles for improved LASV diagnostics.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Diagnostic Performance of Three Indirect Enzyme-Linked Immunosorbent Assays for the Detection of IgG Antibodies to Ebola Virus in Human Sera</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 24</span></i><p style="text-align: justify; text-justify: inter-word;">Filovirus serological diagnosis and epidemiological investigations are hampered due to the unavailability of validated immunoassays. Diagnostic performance of three indirect enzyme-linked immunosorbent assays (I-ELISA) was evaluated for the detection of IgG antibody to Ebola virus (EBOV) in human sera. One I-ELISA was based on a whole EBOV antigen (WAg) and two utilized recombinant nucleocapsid (NP) and glycoproteins (GP), respectively. Validation data sets derived from individual sera collected in South Africa (SA), representing an EBOV non-endemic country, and from sera collected during an Ebola disease (EBOD) outbreak in Sierra Leone (SL), were categorized according to the compounded results of the three I-ELISAs and real time reverse-transcription polymerase chain reaction (RT-PCR). At the cut-off values selected at 95% accuracy level by the two-graph receiver operating characteristic analysis, specificity in the SA EBOV negative serum panel (n = 273) ranged from 98.17% (GP ELISA) to 99.27% (WAg ELISA). Diagnostic specificity in the SL EBOV negative panel (n = 676) was 100% by the three ELISAs. The diagnostic sensitivity in 423 RT-PCR confirmed EBOD patients was dependent on the time when the serum was collected after onset of disease. It significantly increased 2 weeks post-onset, reaching 100% sensitivity by WAg and NP and 98.1% by GP I-ELISA.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>“Running the Gauntlet”: Formidable challenges in advancing neglected tropical diseases vaccines from development through licensure, and a “Call to Action”</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 10</span></i><p style="text-align: justify; text-justify: inter-word;">Translational science for new biotechnologies (e.g. drugs, vaccines, devices, or diagnostics) depend on the development of a robust ‘business case’. This is driven by complex scientific, technical, logistical, financial and operational elements to determine the feasibility and probability of traversing the “valleys of death” leading to licensure. The potential results in terms of profitability and financial realization, called ‘product value proposition’ play a crucial role in establishing incentives for investment during and after development. With this review, our goal is to summarize the challenges in taking vaccines against neglected tropical diseases (NTDs) from development through licensure and provide a perspective that these vaccines can have measurable public health and economic profitability and market success. Understanding these processes and its challenges would open the opportunity to accelerate and advance these essential NTD vaccines through the last mile towards licensure and for the delivery to afflicted populations in low- and middle-income countries.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Plaque assay and improved yield of human coronaviruses in a human rhabdomyosarcoma cell line.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Propagation and plaque assay of human coronavirus prototypes were studied in two human cell lines: a diploid fetal tonsil (FT) and a heteroploid rhabdomyosarcoma (RD) cell lines. Plaques, observed within 2 to 3 days on FT cell monolayers with both 229E and OC43 viruses, appeared as colorless areas after staining with neutral red or crystal violet, whereas neutral red staining was required for visualization of plaques on RD cells. The plating efficiencies were approximately equal between the two cell lines, but virus assay by plaque formation was 15- to 30-fold more efficient than tube dilution assay with 50% endpoints. The discrepancy between 50% endpoint and plaque-forming unit values was striking and appeared to result from the fact that killing of cells (particularly RD cells) by coronaviruses was not accompanied by visible changes in the cells but killing was detected by the failure of infected cells to stain with a vital dye. The latent phase in one-step growth curves was 5 to 6 h for both viruses in either cell line, but the maximum yield of intracellular virus was reached in 18 to 20 h for FT cells and 24 to 28 h for RD cells. Virus release also differed between the two cell lines: in FT cells, the maximum yield of extracellular virus was reached 2 to 3 h later than that of intracellular virus, whereas in RD cells, the difference was 5 h for 229E virus and 10 h for OC43 virus. Although both cell lines appear equally useful for plaque assay, RD cells would be preferred for mass virus propagation because yields (5 X 10(8) plaque-forming units per ml) were 10-fold higher than in FT cells, a finding true for both virus prototypes.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a symptom score for clinical studies to identify children with a documented viral upper respiratory tract infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The objective of this study was to develop a symptom scoring system for use in clinical studies that differentiates children with cold symptoms who have an identifiable viral etiology for their upper respiratory tract infection (URI) from those in whom no virus is detected. Nasal swabs for PCR testing for identification of respiratory viruses were obtained on children 2–11 years old at baseline and when parents thought their child was developing a cold. Parental-recorded severity of specific symptoms in children with and without a documented viral URI were compared. Nasal swabs were obtained on 108 children whose parents reported their child was developing a cold. A viral etiology was identified in 62/108 (57.4%) samples. Symptom measures that best differentiated children with a viral etiology from those without were significant runny nose and significant cough on day 1 through day 4 of the illness. A URI symptom score was developed based on these symptoms, with a sensitivity of 81.4%, specificity of 61.9% and accuracy of 73.3%. Parental impression is only a moderately accurate predictor of viral URI in children. Our URI symptom score provided a more accurate method for identifying children with viral URIs for clinical studies.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of serologic and antigenic relationships between middle eastern respiratory syndrome coronavirus and other coronaviruses to develop vaccine platforms for the rapid response to emerging coronaviruses</h3>
<i class="tooltip">2014<span class="tooltiptext">2014</span></i><p style="text-align: justify; text-justify: inter-word;">Background: Middle East respiratory syndrome coronavirus (MERS-CoV) emerged in 2012, causing severe acute respiratory disease and pneumonia, with 44% mortality among 136 cases to date. Design of vaccines to limit the virus spread or diagnostic tests to track newly emerging strains requires knowledge of antigenic and serologic relationships between MERS-CoV and other CoVs.Methods:  Using synthetic genomics and Venezuelan equine encephalitis virus replicons (VRPs) expressing spike and nucleocapsid proteins from MERS-CoV and other human and bat CoVs, we characterize the antigenic responses (using Western blot and enzyme-linked immunosorbent assay) and serologic responses (using neutralization assays) against 2 MERS-CoV isolates in comparison with those of other human and bat CoVs.Results:  Serologic and neutralization responses against the spike glycoprotein were primarily strain specific, with a very low level of cross-reactivity within or across subgroups. CoV N proteins within but not across subgroups share cross-reactive epitopes with MERS-CoV isolates. Our findings were validated using a convalescent-phase serum specimen from a patient infected with MERS-CoV (NA 01) and human antiserum against SARS-CoV, human CoV NL63, and human CoV OC43.Conclusions:  Vaccine design for emerging CoVs should involve chimeric spike protein containing neutralizing epitopes from multiple virus strains across subgroups to reduce immune pathology, and a diagnostic platform should include a panel of nucleocapsid and spike proteins from phylogenetically distinct CoVs.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccination with attenuated Salmonella enterica Dublin expressing E coli O157:H7 outer membrane protein Intimin induces transient reduction of fecal shedding of E coli O157:H7 in cattle</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Jul 7</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Escherichia coli serogroup O157:H7 has emerged as an important zoonotic bacterial pathogen, causing a range of symptoms from self-limiting bloody diarrhea to severe hemorrhagic colitis and hemolytic-uremic syndrome in humans. Beef and dairy cattle are considered the most important animal reservoirs for this pathogen. One of the important virulence characteristics of E. coli O157:H7 is the eaeA gene encoding the 97 kDa surface protein intimin. Intimin is required for attachment and effacement during the interaction of enterohemorrhagic E. coli with human and bovine neonatal enterocytes. The present study was undertaken to test the hypothesis that an adaptive mucosal immune response directed against intimin will reduce or prevent enteric colonization and fecal shedding of E. coli O157:H7 in cattle. RESULTS: Cattle were orally inoculated with either milk (control), milk with live attenuated Salmonella enterica serovar Dublin (vector), or milk with live attenuated recombinant S. Dublin expressing intimin (vaccinated) on days 0, 14 and 28. On day 98, all calves were challenged orally with E. coli O157:H7 to evaluate whether vaccination with the recombinant S. Dublin expressing intimin would reduce the level of E. coli O157:H7 fecal shedding. During the first 28 days, vaccinated calves shed both the vector strain and the intimin-expressing S. Dublin strain at a similar level. The vector strain was shed for a significantly longer period as compared to the level of recombinant vaccine strain. Calves that received the intimin-expressed vaccine ceased shedding S. Dublin from day 28 to day 63. All calves were challenged with E. coli O157:H7 on day 98 to determine the effect on fecal shedding of E. coli O157:H7. The amount of E. coli O157:H7 in feces was measured for 30 days post-challenge. We observed a transient clearance of E. coli O157:H7 from the feces in the vaccinated calves. The magnitude of fecal E. coli O157:H7 shedding did not correlate with the presence of intimin-specific fecal IgA. CONCLUSION: Oral vaccination with live attenuated recombinant S. Dublin expressing intimin reduced enteric colonization and fecal shedding of E. coli O157:H7. However, the transient clearance of E. coli O157:H7 was not associated with an enhanced IgA-mediated mucosal immune response.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>A Monoclonal Antibody-Based Copro-ELISA Kit for Canine Echinococcosis to Support the PAHO Effort for Hydatid Disease Control in South America</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 10</span></i><p style="text-align: justify; text-justify: inter-word;">Cystic echinococcosis is still a major concern in South America. While some regions show advances in the control of the disease, others have among the highest incidence in the world. To reverse this situation the Pan American Health Organization (PAHO) has launched a regional project on cystic echinococcosis control and surveillance. An early concern of the program was the lack of a standardized diagnostic tool to monitor infection in dogs, a key target of control programs. Under this premise, we have developed a new copro-ELISA test after extensive screening of a large panel of monoclonal antibodies (MAbs) and polyclonal sera, which performs with high standards of sensitivity (92.6%) and specificity (86.4%) as established by necropsy diagnosis of dogs. The key component of the test, MAbEg9 has a convenient IgG isotype and reacts with a periodate-resistant epitope found in high molecular weight components of the worm. Time-course analysis of experimentally infected dogs showed that even animals with a very low number of parasites could be detected as early as day 20 post infection. The test was formulated in a ready-to-use kit format with proven stability of each component for a minimum of 3 months at room temperature. This characteristic facilitates its standardized use and shipping to other laboratories, which was demonstrated by the identical results obtained by two different laboratories in Peru and our own laboratory when a large number of field samples were analyzed independently in a blind fashion.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunogenicity of mumps virus vaccine candidates matching circulating genotypes in the United States and China</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Mumps virus (MuV) causes acute infection in humans with characteristic swelling of the parotid gland. While vaccination has greatly reduced the incidence of MuV infection, there have been multiple large outbreaks of mumps virus (MuV) in highly vaccinated populations. The most common vaccine strain, Jeryl Lynn, belongs to genotype A, which is no longer a circulating genotype. We have developed two vaccine candidates that match the circulating genotypes in the United States (genotype G) and China (genotype F). We found that there was a significant decrease in the ability of the Jeryl Lynn vaccine to produce neutralizing antibody responses to non-matched viruses, when compared to either of our vaccine candidates. Our data suggests that an updated vaccine may allow for better immunity against the circulating MuV genotypes G and F.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Neutralization Assay for Nipah Virus Using Pseudotype Particles</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Nipah virus (NiV) and Hendra virus (HeV) are zoonotic paramyxoviruses capable of causing severe disease in humans and animals. These viruses require biosafety level 4 (BSL-4) containment. Like other paramyxoviruses, the plaque reduction neutralization test (PRNT) can be used to detect antibodies to the surface glycoproteins, fusion (F) and attachment (G), and PRNT titers give an indication of protective immunity. Unfortunately, for NiV and HeV, the PRNT must be performed in BSL-4 containment and takes 5–7 days to complete. Thus, we have developed a neutralization assay using VSV pseudotype particles expressing the F and G proteins of NiV (pVSV-NiV-F/G) as target antigens. This rapid assay, which can be performed at BSL-2, was evaluated using serum samples from outbreak investigations and more than 300 serum samples from an experimental NiV vaccination study in swine. The results of the neutralization assays with pVSV-NiV-F/G as antigen showed a good correlation with those of standard PRNT. Therefore, this new method has the potential to be a rapid and cost-effective diagnostic method, especially in locations that lack high containment facilities, and will provide a valuable tool for basic research and vaccine development.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Time for T? Immunoinformatics addresses vaccine design for neglected tropical and emerging infectious diseases</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jan 2</span></i><p style="text-align: justify; text-justify: inter-word;">Vaccines have been invaluable for global health, saving lives and reducing healthcare costs, while also raising the quality of human life. However, newly emerging infectious diseases (EID) and more well-established tropical disease pathogens present complex challenges to vaccine developers; in particular, neglected tropical diseases, which are most prevalent among the world’s poorest, include many pathogens with large sizes, multistage life cycles and a variety of nonhuman vectors. EID such as MERS-CoV and H7N9 are highly pathogenic for humans. For many of these pathogens, while their genomes are available, immune correlates of protection are currently unknown. These complexities make developing vaccines for EID and neglected tropical diseases all the more difficult. In this review, we describe the implementation of an immunoinformatics-driven approach to systematically search for key determinants of immunity in newly available genome sequence data and design vaccines. This approach holds promise for the development of 21st century vaccines, improving human health everywhere.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenge models to assess new therapies in chronic obstructive pulmonary disease</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Sep 13</span></i><p style="text-align: justify; text-justify: inter-word;">Chronic obstructive pulmonary disease (COPD) is a major cause of morbidity and mortality. Current therapies confer partial benefits either by incompletely improving airflow limitation or by reducing acute exacerbations, hence new therapies are desirable. In the absence of robust early predictors of clinical efficacy, the potential success of novel therapeutic agents in COPD will not entirely be known until the drugs enter relatively large and costly clinical trials. New predictive models in humans, and new study designs are being sought to allow for confirmation of pharmacodynamic and potentially clinically meaningful effects in early development. This review focuses on human challenge models with lipopolysaccharide endotoxin, ozone, and rhinovirus, in the early clinical development phases of novel therapeutic agents for the treatment and reduction of exacerbations in COPD.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /></body>