
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: potential coronavirus risk factors</h2><br /></div><br /><br /><br /><div class="card"><h3>Coronavirus Immunoreactivity in Individuals With a Recent Onset of Psychotic Symptoms</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Prenatal influenza exposure increases the risk for schizophrenia and brings to question how other respiratory viruses may contribute to neuropsychiatric disease etiopathology. Human coronaviruses cause respiratory infections that range in seriousness from common colds to severe acute respiratory syndrome. Like influenza, coronaviruses can be neurotropic. To test for associations between coronaviruses and serious mental disorders, we utilized a recently developed assay and measured immunoglobulin G (IgG) response against 4 human coronavirus strains (229E, HKU1, NL63, and OC43) in 106 patients with a recent onset of psychotic symptoms and 196 nonpsychiatric controls. We expressed results quantitatively as antibody levels and qualitatively as seroprevalence relative to a defined seropositivity cutoff value. Patient IgG levels were higher than controls for HKU1, NL63, and OC43, with HKU1 and NL63 both showing highly significant patient-to-control differences (HKU1, P ≤ .002; NL63, P ≤ .00001). All 4 coronaviruses were more seroprevalent in patients vs controls, with greatest intergroup differences observed for HKU1 (93% vs 77%, P ≤ .0001). HKU1 and NL63 associations with the patient group were further supported by multivariate analyses that controlled for age, gender, race, socioeconomic status, and smoking status (HKU1, odds ratio [OR] = 1.32, 95% confidence interval [CI] = 1.03–1.67, P ≤ .027; NL63, OR = 2.42, 95% CI = 1.25–4.66, P ≤ .008). Among patients, NL63 was associated with schizophrenia-spectrum (OR = 3.10, 95% CI = 1.27–7.58, P ≤ .013) but not mood disorders. HKU1 and NL63 coronavirus exposures may represent comorbid risk factors in neuropsychiatric disease. Future studies should explore links between the timing of coronavirus infections and subsequent development of schizophrenia and other disorders with psychotic symptoms.</p>
<p align="right"><i>score: 175</i></p>

</div><br /><br /><br /><div class="card"><h3>Animal models for SARS and MERS coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome coronavirus (MERS-CoV), two strains of animal coronaviruses that crossed the species barrier to infect and cause severe respiratory infections in humans within the last 12 years, have taught us that coronaviruses represent a global threat that does not recognize international borders. We can expect to see other novel coronaviruses emerge in the future. An ideal animal model should reflect the clinical signs, viral replication and pathology seen in humans. In this review, we present factors to consider in establishing an animal model for the study of novel coronaviruses and compare the different animal models that have been employed to study SARS-CoV and MERS-CoV.</p>
<p align="right"><i>score: 159</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS, MERS and novel coronavirus (COVID-19) epidemics, the newest and biggest global health threats: what lessons have we learned?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China.Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made.Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally.We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>Jumping species – A Mechanism for Coronavirus Persistence and Survival</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Zoonotic transmission of novel viruses represents a significant threat to global public health and is fueled by globalization, the loss of natural habitats, and exposure to new hosts. For coronaviruses (CoVs), broad diversity exists within bat populations and uniquely positions them to seed future emergence events. In this review, we explore the host and viral dynamics that shape these CoV populations for survival, amplification, and possible emergence in novel hosts.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk Factors for Primary Middle East Respiratory Syndrome Coronavirus Illness in Humans, Saudi Arabia, 2014</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Risk factors for primary Middle East respiratory syndrome coronavirus (MERS-CoV) illness in humans are incompletely understood. We identified all primary MERS-CoV cases reported in Saudi Arabia during March–November 2014 by excluding those with history of exposure to other cases of MERS-CoV or acute respiratory illness of unknown cause or exposure to healthcare settings within 14 days before illness onset. Using a case–control design, we assessed differences in underlying medical conditions and environmental exposures among primary case-patients and 2–4 controls matched by age, sex, and neighborhood. Using multivariable analysis, we found that direct exposure to dromedary camels during the 2 weeks before illness onset, as well as diabetes mellitus, heart disease, and smoking, were each independently associated with MERS-CoV illness. Further investigation is needed to better understand animal-to-human transmission of MERS-CoV.</p>
<p align="right"><i>score: 152</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk Factors for Primary Middle East Respiratory Syndrome Coronavirus Illness in Humans, Saudi Arabia, 2014</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Risk factors for primary Middle East respiratory syndrome coronavirus (MERS-CoV) illness in humans are incompletely understood. We identified all primary MERS-CoV cases reported in Saudi Arabia during March–November 2014 by excluding those with history of exposure to other cases of MERS-CoV or acute respiratory illness of unknown cause or exposure to healthcare settings within 14 days before illness onset. Using a case–control design, we assessed differences in underlying medical conditions and environmental exposures among primary case-patients and 2–4 controls matched by age, sex, and neighborhood. Using multivariable analysis, we found that direct exposure to dromedary camels during the 2 weeks before illness onset, as well as diabetes mellitus, heart disease, and smoking, were each independently associated with MERS-CoV illness. Further investigation is needed to better understand animal-to-human transmission of MERS-CoV.</p>
<p align="right"><i>score: 152</i></p>

</div><br /><br /><br /><div class="card"><h3>Trypsin treatment unlocks barrier for zoonotic coronaviruses infection</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-09-16</span></i><p style="text-align: justify; text-justify: inter-word;">Traditionally, the emergence of coronaviruses (CoVs) has been attributed to a gain in receptor binding in a new host. Our previous work with SARS-like viruses argued that bats already harbor CoVs with the ability to infect humans without adaptation. These results suggested that additional barriers limit the emergence of zoonotic CoV. In this work, we describe overcoming host restriction of two MERS-like bat CoVs using exogenous protease treatment. We found that the spike protein of PDF2180-CoV, a MERS-like virus found in a Ugandan bat, could mediate infection of Vero and human cells in the presence of exogenous trypsin. We subsequently show that the bat virus spike can mediate infection of human gut cells, but is unable to infect human lung cells. Using receptor-blocking antibodies, we show that infection with the PDF2180 spike does not require MERS-CoV receptor DPP4 and antibodies developed against the MERS spike receptor-binding domain and S2 portion are ineffective in neutralizing the PDF2180 chimera. Finally, we found that addition of exogenous trypsin also rescues replication of HKU5-CoV, a second MERS-like group 2c CoV. Together, these results indicate that proteolytic cleavage of the spike, not receptor binding, is the primary infection barrier for these two group 2c CoVs. Coupled with receptor binding, proteolytic activation offers a new parameter to evaluate emergence potential of CoVs and offer a means to recover previously unrecoverable zoonotic CoV strains.\n\nImportanceOverall, our studies demonstrate that proteolytic cleavage is the primary barrier to infection for a subset of zoonotic coronaviruses. Moving forward, the results argue that both receptor binding and proteolytic cleavage of the spike are critical factors that must be considered for evaluating the emergence potential and risk posed by zoonotic coronaviruses. In addition, the findings also offer a novel means to recover previously uncultivable zoonotic coronavirus strains and argue that other tissues, including the digestive tract, could be a site for future coronavirus emergence events in humans.</p>
<p align="right"><i>score: 149</i></p>

</div><br /><br /><br /><div class="card"><h3>Distribution and Evolutionary History of the Mobile Genetic Element s2m in Coronaviruses</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">The mobile genetic element s2m has been described in several families of single-stranded RNA viruses. The function remains elusive, but an increasing number of s2m-containing sequences are being deposited in publicly available databases. Currently, more than 700 coronavirus sequences containing s2m can be found in GenBank, including the severe acute respiratory syndrome (SARS) coronavirus genome. This is an updated review of the pattern of s2m in coronaviruses, the possible functional implications and the evolutionary history.</p>
<p align="right"><i>score: 148</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to COVID-19: A previously unknown SARS-CoV-2 virus of pandemic potential infecting humans – Call for a One Health approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses continue to pose a threat to human health. The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in December 2019 which causes coronavirus disease-2019 (COVID-19), an acute respiratory disease marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. This recent ongoing emergence of a previously unknown coronavirus in China leads to huge impacts on humans globally. Here, we discuss the outbreak in a one health context, highlighting the need for the implementation of one health measures and practices to improve human health and reduce the emergence of pandemic viruses.</p>
<p align="right"><i>score: 148</i></p>

</div><br /><br /><br /><div class="card"><h3>How Coronaviruses Cause Infection—from Colds to Deadly Pneumonia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus outbreak raises questions about how such pathogens evolve and what makes infections mild or severe</p>
<p align="right"><i>score: 146</i></p>

</div><br /><br /><br /><div class="card"><h3>Risks of Death and Severe Disease in Patients With Middle East Respiratory Syndrome Coronavirus, 2012–2015</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging pathogen, first recognized in 2012, with a high case fatality risk, no vaccine, and no treatment beyond supportive care. We estimated the relative risks of death and severe disease among MERS-CoV patients in the Middle East between 2012 and 2015 for several risk factors, using Poisson regression with robust variance and a bootstrap-based expectation maximization algorithm to handle extensive missing data. Increased age and underlying comorbidity were risk factors for both death and severe disease, while cases arising in Saudi Arabia were more likely to be severe. Cases occurring later in the emergence of MERS-CoV and among health-care workers were less serious. This study represents an attempt to estimate risk factors for an emerging infectious disease using open data and to address some of the uncertainty surrounding MERS-CoV epidemiology.</p>
<p align="right"><i>score: 146</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel coronavirus takes flight from bats?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Two recent studies provide initial insights into a novel coronavirus that is associated with an outbreak of human respiratory disease.</p>
<p align="right"><i>score: 146</i></p>

</div><br /><br /><br /><div class="card"><h3>Unveiling the Origin and Transmission of 2019-nCoV</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus has caused thousands of human infections in China since December 2019, raising a global public health concern. Recent studies (Huang et al., Chan et al., and Zhou et al.) have provided timely insights into its origin and ability to spread among humans, informing infection prevention and control practices.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Diabetes Mellitus, Hypertension, and Death among 32 Patients with MERS-CoV Infection, Saudi Arabia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Diabetes mellitus and hypertension are recognized risk factors for severe clinical outcomes, including death, associated with Middle East respiratory syndrome coronavirus infection. Among 32 virus-infected patients in Saudi Arabia, severity of illness and frequency of death corresponded closely with presence of multiple and more severe underlying conditions.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Diabetes Mellitus, Hypertension, and Death among 32 Patients with MERS-CoV Infection, Saudi Arabia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Diabetes mellitus and hypertension are recognized risk factors for severe clinical outcomes, including death, associated with Middle East respiratory syndrome coronavirus infection. Among 32 virus-infected patients in Saudi Arabia, severity of illness and frequency of death corresponded closely with presence of multiple and more severe underlying conditions.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Burden and Risk Factors for Coronavirus Infections in Infants in Rural Nepal</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Nov 15</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Knowledge of risk factors for symptomatic human coronavirus (HCoV) infections in children in community settings is limited. We estimated the disease burden and impact of birth-related, maternal, household, and seasonal factors on HCoV infections among children from birth to 6 months old in rural Nepal. METHODS: Prospective, active, weekly surveillance for acute respiratory infections (ARIs) was conducted in infants over a period of 3 years during 2 consecutive, population-based randomized trials of maternal influenza immunization. Midnasal swabs were collected for acute respiratory symptoms and tested for HCoV and other viruses by reverse-transcription polymerase chain reaction. Association between HCoV incidence and potential risk factors was modeled using Poisson regression. RESULTS: Overall, 282 of 3505 (8%) infants experienced an HCoV ARI within the first 6 months of life. HCoV incidence overall was 255.6 (95% confidence interval [CI], 227.3–286.5) per 1000 person-years, and was more than twice as high among nonneonates than among neonates (incidence rate ratio [IRR], 2.53; 95% CI, 1.52–4.21). HCoV ARI incidence was also positively associated with the number of children <5 years of age per room in a household (IRR, 1.13; 95% CI, 1.01–1.28). Of the 296 HCoV infections detected, 46% were coinfections with other respiratory viruses. While HCoVs were detected throughout the study period, seasonal variation was also observed, with incidence peaking in 2 winters (December–February) and 1 autumn (September–November). CONCLUSIONS: HCoV is associated with a substantial proportion of illnesses among young infants in rural Nepal. There is an increased risk of HCoV infection beyond the first month of life.</p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>What is the evidence that bovine coronavirus is a biologically significant respiratory pathogen in cattle?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses, including bovine coronavirus (BCoV), are etiologically associated with enteric and respiratory disease across a wide range of mammalian and avian species. The role of BCoV in calfhood diarrhea is well-established, but its role in the bovine respiratory disease complex (BRDC) has been controversial. This review re-examines the evidence that BCoV is a significant pathogen in the BRDC.</p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Since mid-December 2019, a cluster of pneumonia-like diseases caused by a novel coronavirus, now designated COVID-19 by the WHO, emerged in Wuhan city and rapidly spread throughout China. Here we identify the clinical characteristics of COVID-19 in a cohort of patients in Shanghai.  Methods: Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, laboratory and radiological features.  Results: Of 198 patients, the median duration from disease onset to hospital admission was 4 days. The mean age of the patients was 50.1 years, and 51.0% patients were male. The most common symptom was fever. Less than half of the patients presented with respiratory systems including cough, sputum production, itchy or sore throat, shortness of breath, and chest congestion. 5.6% patients had diarrhoea. On admission, T lymphocytes were decreased in 45.8% patients. Ground glass opacity was the most common radiological finding on chest computed tomography. 9.6% were admitted to the ICU because of the development of organ dysfunction. Compared with patients not treated in ICU, patients treated in the ICU were older, had longer waiting time to admission, fever over 38.5o C, dyspnoea, reduced T lymphocytes, elevated neutrophils and organ failure. Conclusions: In this single centre cohort of COVID-19 patients, the most common symptom was fever, and the most common laboratory abnormality was decreased blood T cell counts. Older age, male, fever over 38.5oC, symptoms of dyspnoea, and underlying comorbidity, were the risk factors most associated with severity of disease. Key words: 2019 novel coronavirus; acute respiratory infection; risk factors for disease severity</jats:p></p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Nucleocapsid-Based Human Coronavirus Immunoassay and Estimates of Individuals Exposed to Coronavirus in a U.S. Metropolitan Population</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses cause respiratory infections ranging from common colds to severe acute respiratory syndrome (SARS) in humans. Estimates for exposure to non-SARS coronaviruses are high, particularly for 229E and OC43; however, less information regarding seroprevalence is available for HKU1 and NL63. To measure exposure rates to these four coronavirus strains (229E, HKU1, NL63, and OC43), we devised an immunoassay based on amino- and carboxy-terminally tagged recombinant coronavirus nucleocapsid antigens. Four human and one feline coronavirus antigen were cloned into baculoviruses expressed in insect cells and recovered proteins bound in the solid phase of an enzyme-linked immunosorbent assay-based system. We screened sera from 10 children and 196 adults and established primary cutoff points based on immunoglobulin G (IgG) antibody levels of the predominantly seronegative children. The proportion of seropositive adults for each coronavirus was as follows: 229E, 91.3%; HKU1, 59.2%; NL63, 91.8%; and OC43, 90.8%. No evidence of a significant serological response to the feline coronavirus was observed. Significant associations of coronavirus seropositivity and antibody levels with age, gender, race, socioeconomic status, smoking status, and season of the blood draw were tested with chi-square and regression analyses. The group II coronaviruses (OC43 and HKU1) were significantly associated with race (P ≤ 0.009 and P ≤ 0.03, respectively). Elevated OC43 IgG levels were further significantly associated with smoking status (P ≤ 0.03), as were high NL63 titers with socioeconomic status (P ≤ 0.04). The high-level immunoreactivity of each coronavirus was significantly associated with the summer season (P ≤ 0.01 to 0.0001). In summary, high rates of exposure to 229E, NL63, and OC43 and a moderate rate of exposure to HKU1 characterized the seroprevalence among individuals in this population. Demographic factors, such as race, smoking status, and socioeconomic status, may confer an increased risk of susceptibility to these viruses.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Human coronaviruses are uncommon in patients with gastrointestinal illness</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronaviruses infect numerous animal species causing a variety of illnesses including respiratory, neurologic and enteric disease. Human coronaviruses (HCoV) are mainly associated with respiratory tract disease but have been implicated in enteric disease. OBJECTIVES: To investigate the frequency of coronaviruses in stool samples from children and adults with gastrointestinal illness by RT-PCR. STUDY DESIGN: Clinical samples submitted for infectious diarrhea testing were collected from December 2007 through March 2008. RNA extraction and RT-PCR was performed for stools negative for Clostridium difficile using primer sets against HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1. Clinical data from samples positive for coronaviruses were reviewed and recorded. RESULTS: Samples from 479 patients were collected including 151 pediatric (<= 18yrs), and 328 adults (> 18yrs). Of these samples, 4 patients (1.3%, 2 adult; 2 pediatric) screened positive for the presence of a coronavirus. All detected coronaviruses were identified as HCoV-HKU1. No stools screened positive for either HCoV-229E, HCoV-NL63 or HCoV-OC43. All HCoV-HKU1 positive samples occurred between mid January to mid February. Clinical manifestations from HCoV-HKU1 positive patients included diarrhea, emesis and respiratory complaints. Three (75%) patients were admitted to the hospital with a median length of stay of 6 days. CONCLUSIONS: Coronaviruses as a group are not commonly identified in stool samples of patients presenting with gastrointestinal illness. HCoV-HKU1 can be identified in stool samples from children and adults with gastrointestinal disease, with most individuals having respiratory findings as well. No stool samples screened positive for HCoV-NL63, HCoV-229E, or HCoV-OC43.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The 2019-nCoV is reported to share the same entry (ACE2) as SARS-CoV according to the updated findings. Analyzing the distribution and expression level of the route of coronavirus may help reveal underlying mechanisms of viral susceptibility and post-infection modulation. In this study, we found that the expression of ACE2 in healthy populations and patients with underlying diseases was not significantly different, suggesting relatively similar susceptibility, which was consistent with current clinical observations. Moreover, based on the expression of ACE2 in smoking individuals, we inferred that long-term smoking might be a risk factor for 2019-nCoV. Analyzing the ACE2 in SARS-CoV infected cells suggested that ACE2 was more than just a receptor but also participated in post-infection regulation, including immune response, cytokine secretion, and viral genome replication. We also constructed Protein-protein interaction (PPI) networks and identified hub genes in viral activity and cytokine secretion. Our findings could explain the clinical symptoms so far and help clinicians and researchers understand the pathogenesis and design therapeutic strategies for 2019-nCoV.</jats:p></p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Animal Models Against Emerging Coronaviruses: From SARS to MERS coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two novel coronaviruses have emerged to cause severe disease in humans. While bats may be the primary reservoir for both viruses, SARS coronavirus (SARS-CoV) likely crossed into humans from civets in China, and MERS coronavirus (MERS-CoV) has been transmitted from camels in the Middle East. Unlike SARS-CoV that resolved within a year, continued introductions of MERS-CoV present an on-going public health threat. Animal models are needed to evaluate countermeasures against emerging viruses. With SARS-CoV, several animal species were permissive to infection. In contrast, most laboratory animals are refractory or only semi-permissive to infection with MERS-CoV. This host-range restriction is largely determined by sequence heterogeneity in the MERS-CoV receptor. We describe animal models developed to study coronaviruses, with a focus on host-range restriction at the level of the viral receptor and discuss approaches to consider in developing a model to evaluate countermeasures against MERS-CoV.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and Characterization of a Novel Bat Coronavirus Closely Related to the Direct Progenitor of Severe Acute Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We report the isolation and characterization of a novel bat coronavirus which is much closer to the severe acute respiratory syndrome coronavirus (SARS-CoV) in genomic sequence than others previously reported, particularly in its S gene. Cell entry and susceptibility studies indicated that this virus can use ACE2 as a receptor and infect animal and human cell lines. Our results provide further evidence of the bat origin of the SARS-CoV and highlight the likelihood of future bat coronavirus emergence in humans.</p>
<p align="right"><i>score: 141</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology of a Novel Recombinant Middle East Respiratory Syndrome Coronavirus in Humans in Saudi Arabia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe respiratory illness in humans. Fundamental questions about circulating viruses and transmission routes remain. METHODS: We assessed routinely collected epidemiologic data for MERS-CoV cases reported in Saudi Arabia during 1 January– 30 June 2015 and conducted a more detailed investigation of cases reported during February 2015. Available respiratory specimens were obtained for sequencing. RESULTS: During the study period, 216 MERS-CoV cases were reported. Full genome (n = 17) or spike gene sequences (n = 82) were obtained from 99 individuals. Most sequences (72 of 99 [73%]) formed a discrete, novel recombinant subclade (NRC-2015), which was detected in 6 regions and became predominant by June 2015. No clinical differences were noted between clades. Among 87 cases reported during February 2015, 13 had no recognized risks for secondary acquisition; 12 of these 13 also denied camel contact. Most viruses (8 of 9) from these 13 individuals belonged to NRC-2015. DISCUSSIONS: Our findings document the spread and eventual predominance of NRC-2015 in humans in Saudi Arabia during the first half of 2015. Our identification of cases without recognized risk factors but with similar virus sequences indicates the need for better understanding of risk factors for MERS-CoV transmission.</p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Trypsin Treatment Unlocks Barrier for Zoonotic Bat Coronavirus Infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Traditionally, the emergence of coronaviruses (CoVs) has been attributed to a gain in receptor binding in a new host. Our previous work with severe acute respiratory syndrome (SARS)-like viruses argued that bats already harbor CoVs with the ability to infect humans without adaptation. These results suggested that additional barriers limit the emergence of zoonotic CoV. In this work, we describe overcoming host restriction of two Middle East respiratory syndrome (MERS)-like bat CoVs using exogenous protease treatment. We found that the spike protein of PDF2180-CoV, a MERS-like virus found in a Ugandan bat, could mediate infection of Vero and human cells in the presence of exogenous trypsin. We subsequently show that the bat virus spike can mediate the infection of human gut cells but is unable to infect human lung cells. Using receptor-blocking antibodies, we show that infection with the PDF2180 spike does not require MERS-CoV receptor DPP4 and antibodies developed against the MERS spike receptor-binding domain and S2 portion are ineffective in neutralizing the PDF2180 chimera. Finally, we found that the addition of exogenous trypsin also rescues HKU5-CoV, a second bat group 2c CoV. Together, these results indicate that proteolytic cleavage of the spike, not receptor binding, is the primary infection barrier for these two group 2c CoVs. Coupled with receptor binding, proteolytic activation offers a new parameter to evaluate the emergence potential of bat CoVs and offers a means to recover previously unrecoverable zoonotic CoV strains.IMPORTANCE Overall, our studies demonstrate that proteolytic cleavage is the primary barrier to infection for a subset of zoonotic coronaviruses. Moving forward, the results argue that both receptor binding and proteolytic cleavage of the spike are critical factors that must be considered for evaluating the emergence potential and risk posed by zoonotic coronaviruses. In addition, the findings also offer a novel means to recover previously uncultivable zoonotic coronavirus strains and argue that other tissues, including the digestive tract, could be a site for future coronavirus emergence events in humans.</p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Association of Cardiovascular Manifestations with In-hospital Outcomes in Patients with COVID-19: A Hospital Staff Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The outbreaks of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) remain a huge threat to the public health worldwide. Clinical data is limited up to now regarding the risk factors in favor of severe conversion of non-severe case with COVID-19. Aims: This study analyzed a hospital staff data to figure out general clinical features of COVID-19 in terms of the association of cardiovascular manifestations (CVMs) with in-hospital outcomes of COVID-19 cases.  Methods: Retrospective, single-center case series of 41 consecutive hospitalized health staff with confirmed COVID-19 were collected at the Central Hospital of Wuhan in Wuhan, China, from January 15 to January 24, 2020. Epidemiological, demographic, clinical, laboratory, radiological, treatment data and in-hospital adverse events were collected and analyzed. Final date of follow-up was March 3, 2020. A comparative study was applied between cases with CVMs and those without CVMs. Results: Of all, clinicians and clinical nurses accounted for 80.5%, while 87.8% of all had history of patient contact. The population was presented with a mean age of 39.1 +- 9.2 and less comorbidities than community population. The three most frequent symptoms of COVID-19 cases analyzed were fever (82.9%), myalgia or fatigue (80.5%) and cough (63.4%). While, the three most frequent initial symptoms were myalgia or fatigue (80.5%), fever (73.2%) and cough (41.5%). There were 95.1% cases featured as non-severe course of disease according to the official standard in China. Patients with CVMs and those without CVMs accounted for 58.5% and 41.5%, respectively. Compared with cases without CVMs, patients with CVMs were presented with lower baseline lymphocyte count (0.99 +- 0.43 and 1.55 +- 0.61, P&lt;0.001), more who had at least once positive nucleic acid detection of throat swab during admission (50.0% and 11.8%, P=0.011), and more received oxygen support (79.2% and 23.5%, P&lt;0.001). The rate of in-hospital adverse events was significantly higher in patients with CVMs group (75.0% and 23.5%, P=0.001). Multivariable logistic regression model indicated that, coexisting with CVMs in COVID-19 patients was not independently associated with in-hospital adverse events. Conclusions: Most of hospital staff with COVID-19 had history of patient contact, featured non-severe course of disease. Cases with CVMs suffered from more in-hospital adverse events than those without CVMs. But concomitant CVMs were not independently associated with in-hospital adverse events in COVID-19 patients.</jats:p></p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Significance of Human Coronavirus in Bronchoalveolar Lavage Samples From Hematopoietic Cell Transplant Recipients and Patients With Hematologic Malignancies</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND. The possible role of human coronavirus (HCoV) in lower respiratory tract disease (LRTD) in hematopoietic cell transplant (HCT) recipients and patients with hematologic malignancies (HM) has not been well studied. METHODS. We conducted a retrospective review of HCT/HM patients with HCoV detected in bronchoalveolar lavage (BAL). HCoV strains were identified in BAL samples using strain-specific polymerase chain reaction. Mortality rates were compared among HCT recipients with LRTD caused by HCoV, respiratory syncytial virus (RSV), influenza virus, or parainfluenza virus (PIV) by multivariable Cox regression analysis. RESULTS. We identified 35 patients (37 episodes) with HCoV LRTD. Among 23 available BAL samples, 48% were strain OC43, 22% were NL63, 17% were 229E, and 13% were HKU1. Overall, 21 patients (60%) required oxygen therapy at diagnosis and 19 (54%) died within 90 days of diagnosis. Respiratory copathogens were detected in 21 episodes (57%), including viruses (n = 12), fungi (n = 10), and bacteria (n = 8). Mortality rates were not different between patients with and without copathogens (P = .65). In multivariable models, mortality associated with HCoV LRTD was similar to that seen with RSV, influenza, and PIV LRTD in HCT recipients (adjusted hazard ratio, 1.34 [95% confidence interval, .66–2.71], P = .41 vs RSV, adjusted for cell source, cytopenia, copathogens, oxygen use, and steroid use). CONCLUSIONS. HCoV LRTD in patients with HCT or HM is associated with high rates of oxygen use and mortality. Mortality associated with HCoV LRTD in HCT recipients appears to be similar to that seen with RSV, influenza virus, and PIV.</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Transmission in Extended Family, Saudi Arabia, 2014</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">Risk factors for human-to-human transmission of Middle East respiratory syndrome coronavirus (MERS-CoV) are largely unknown. After MERS-CoV infections occurred in an extended family in Saudi Arabia in 2014, relatives were tested by using real-time reverse transcription PCR (rRT-PCR) and serologic methods. Among 79 relatives, 19 (24%) were MERS-CoV positive; 11 were hospitalized, and 2 died. Eleven (58%) tested positive by rRT-PCR; 8 (42%) tested negative by rRT-PCR but positive by serology. Compared with MERS-CoV–negative adult relatives, MERS-CoV–positive adult relatives were older and more likely to be male and to have chronic medical conditions. Risk factors for household transmission included sleeping in an index patient’s room and touching respiratory secretions from an index patient. Casual contact and simple proximity were not associated with transmission. Serology was more sensitive than standard rRT-PCR for identifying infected relatives, highlighting the value of including serology in future investigations.</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology of coronavirus respiratory infections.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses were found by enzyme linked immunosorbent assay in upper respiratory tract secretions taken during 30% of 108 acute respiratory infections experienced by 30 children under age 6 years with recurrent respiratory infections (index group), and during 29% of 51 acute infections experienced by their siblings. Lower respiratory tract infection--predominantly wheezy bronchitis--occurred in 30% of the index children's coronavirus positive infections but in none of their siblings' infections. Reinfections were common. Two peaks of infection were seen each year in the late autumn/early winter and in the early summer.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 sensitive to type I interferon pretreatment.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B coronaviruses share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel coronavirus is much more sensitive to type I interferon pretreatment. We subsequently examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN response between SARS-CoV and SARS-CoV-2 that could help inform disease progression, treatment options, and animal model development.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>Occupational Exposure to Dromedaries and Risk for MERS-CoV Infection, Qatar, 2013–2014</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">We determined the presence of neutralizing antibodies to Middle East respiratory syndrome coronavirus in persons in Qatar with and without dromedary contact. Antibodies were only detected in those with contact, suggesting dromedary exposure as a risk factor for infection. Findings also showed evidence for substantial underestimation of the infection in populations at risk in Qatar.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>Occupational Exposure to Dromedaries and Risk for MERS-CoV Infection, Qatar, 2013–2014</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">We determined the presence of neutralizing antibodies to Middle East respiratory syndrome coronavirus in persons in Qatar with and without dromedary contact. Antibodies were only detected in those with contact, suggesting dromedary exposure as a risk factor for infection. Findings also showed evidence for substantial underestimation of the infection in populations at risk in Qatar.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS CORONAVIRUS AND INNATE IMMUNITY</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of the highly pathogenic SARS coronavirus (SARS-CoV) has reignited interest in coronavirus biology and pathogenesis. An emerging theme in coronavirus pathogenesis is that the interaction between specific viral genes and the host immune system, specifically the innate immune system, functions as a key determinant in regulating virulence and disease outcomes. Using SARS-CoV as a model, we will review the current knowledge of the interplay between coronavirus infection and the host innate immune system in vivo, and then discuss the mechanisms by which specific gene products antagonize the host innate immune response in cell culture models. Our data suggests that the SARS-CoV uses specific strategies to evade and antagonize the sensing and signaling arms of the interferon pathway. We summarize by identifying future points of consideration that will contribute greatly to our understanding of the molecular mechanisms governing coronavirus pathogenesis and virulence, and the development of severe disease in humans and animals.</p>
<p align="right"><i>score: 137</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An in-depth annotation of the newly discovered coronavirus (2019-nCoV) genome has revealed differences between 2019-nCoV and severe acute respiratory syndrome (SARS) or SARS-like coronaviruses. A systematic comparison identified 380 amino acid substitutions between these coronaviruses, which may have caused functional and pathogenic divergence of 2019-nCoV.</p>
<p align="right"><i>score: 137</i></p>

</div><br /><br /><br /><div class="card"><h3>Travel-related MERS-CoV cases: an assessment of exposures and risk factors in a group of Dutch travellers returning from the Kingdom of Saudi Arabia, May 2014</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Oct 17</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In May 2014, Middle East respiratory syndrome coronavirus (MERS-CoV) infection, with closely related viral genomes, was diagnosed in two Dutch residents, returning from a pilgrimage to Medina and Mecca, Kingdom of Saudi Arabia (KSA). These patients travelled with a group of 29 other Dutch travellers. We conducted an epidemiological assessment of the travel group to identify likely source(s) of infection and presence of potential risk factors. METHODS: All travellers, including the two cases, completed a questionnaire focussing on potential human, animal and food exposures to MERS-CoV. The questionnaire was modified from the WHO MERS-CoV questionnaire, taking into account the specific route and activities of the travel group. RESULTS: Twelve non-cases drank unpasteurized camel milk and had contact with camels. Most travellers, including one of the two patients (Case 1), visited local markets, where six of them consumed fruits. Two travellers, including Case 1, were exposed to coughing patients when visiting a hospital in Medina. Four travellers, including Case 1, visited two hospitals in Mecca. All travellers had been in contact with Case 1 while he was sick, with initially non-respiratory complaints. The cases were found to be older than the other travellers and both had co-morbidities. CONCLUSIONS: This epidemiological study revealed the complexity of MERS-CoV outbreak investigations with multiple potential exposures to MERS-CoV reported such as healthcare visits, camel exposure, and exposure to untreated food products. Exposure to MERS-CoV during a hospital visit is considered a likely source of infection for Case 1 but not for Case 2. For Case 2, the most likely source could not be determined. Exposure to MERS-CoV via direct contact with animals or dairy products seems unlikely for the two Dutch cases. Furthermore, exposure to a common but still unidentified source cannot be ruled out. More comprehensive research into sources of infection in the Arabian Peninsula is needed to strengthen and specify the prevention of MERS-CoV infections.</p>
<p align="right"><i>score: 137</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses: Important Emerging Human Pathogens</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The identification of Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012 reaffirmed the importance of understanding how coronaviruses emerge, infect, and cause disease. By comparing what is known about severe acute respiratory syndrome coronavirus (SARS-CoV) to what has recently been found for MERS-CoV, researchers are discovering similarities and differences that may be important for pathogenesis. Here we discuss what is known about each virus and what gaps remain in our understanding, especially concerning MERS-CoV.</p>
<p align="right"><i>score: 137</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk factors for MERS coronavirus infection in dromedary camels in Burkina Faso, Ethiopia, and Morocco, 2015</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 30</span></i><p style="text-align: justify; text-justify: inter-word;">Understanding Middle East respiratory syndrome coronavirus (MERS-CoV) transmission in dromedary camels is important, as they consitute a source of zoonotic infection to humans. To identify risk factors for MERS-CoV infection in camels bred in diverse conditions in Burkina Faso, Ethiopia and Morocco, blood samples and nasal swabs were sampled in February–March 2015. A relatively high MERS-CoV RNA rate was detected in Ethiopia (up to 15.7%; 95% confidence interval (CI): 8.2–28.0), followed by Burkina Faso (up to 12.2%; 95% CI: 7–20.4) and Morocco (up to 7.6%; 95% CI: 1.9–26.1). The RNA detection rate was higher in camels bred for milk or meat than in camels for transport (p = 0.01) as well as in younger camels (p = 0.06). High seropositivity rates (up to 100%; 95% CI: 100–100 and 99.4%; 95% CI: 95.4–99.9) were found in Morocco and Ethiopia, followed by Burkina Faso (up to 84.6%; 95% CI: 77.2–89.9). Seropositivity rates were higher in large/medium herds (≥51 camels) than small herds (p = 0.061), in camels raised for meat or milk than for transport (p = 0.01), and in nomadic or sedentary herds than in herds with a mix of these lifestyles (p < 0.005).</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus-Associated Influenza-Like Illness in the Community Setting in Peru</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We present findings describing the epidemiology of non-severe acute respiratory syndrome human coronavirus-associated influenza-like illness from a population-based active follow-up study in four different regions of Peru. In 2010, the prevalence of infections by human coronaviruses 229E, OC43, NL63, or HKU1 was 6.4% in participants with influenza-like illness who tested negative for influenza viruses. Ten of 11 human coronavirus infections were identified in the fall–winter season. Human coronaviruses are present in different regions of Peru and are relatively frequently associated with influenza-like illness in Peru.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Is COVID-19 Receiving ADE From Other Coronaviruses?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">One of the most perplexing questions regarding the current COVID-19 coronavirus epidemic is the discrepancy between the severity of cases observed in the Hubei province of China and those occurring elsewhere in the world. One possible answer is antibody dependent enhancement (ADE) of SARS-CoV-2 due to prior exposure to other coronaviruses. ADE modulates the immune response and can elicit sustained inflammation, lymphopenia, and/or cytokine storm, one or all of which have been documented in severe cases and deaths. ADE also requires prior exposure to similar antigenic epitopes, presumably circulating in local viruses, making it a possible explanation for the observed geographic limitation of severe cases and deaths.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses post-SARS: Update on replication and pathogenesis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">While coronaviruses were first identified nearly 60 years ago, they received notoriety in 2003 when one of their members was identified as the etiological agent of the Severe Acute Respiratory Syndrome (SARS-CoV) 1. Previously these viruses were known to be important agents of respiratory and enteric infections of domestic and companion animals and to cause approximately 15% of all cases of the common cold. This Review focuses on recent advances in our understanding of the mechanisms of coronavirus replication, interactions with the host immune response and disease pathogenesis and highlights the recent identification of numerous novel coronaviruses and the propensity of this virus family to cross species barriers.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>First respiratory transmitted food borne outbreak?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The world is faced with a remarkable coronavirus outbreak with epicentre in Wuhan, China. Altogether 40554 cases have been confirmed globally with novel coronavirus (SARS-CoV-2) until February 10, 2020. Rigorous surveillance in other countries is required to prevent further global expansion of the outbreak, but resolving the exact mechanism of the initial transmission events is crucial. Most initial cases had visited Huanan South Seafood Market in Wuhan selling also various exotic live animals. Based on the limited initial human-to-human transmission and timely clustering of cases in Huanan market among elderly men, coupled with knowledge that coronaviruses are derived from animals and relationship of SARS-CoV-2 to bat coronavirus, zoonotic transmission in the first instance is probable. To target the actions, similar epidemiological actions to human cases are needed with animal or food exposures. According to current information, an exceptionally wide contamination of seafood market might explain the initiation of the SARS-CoV-2 outbreak. Seafood tanks, air contamination by live animals or rodents are possibilities, but sold animals normally come from various sources. The mode of transmission may become clearer in future: usually in outbreak investigations, hindsight is easy, but for now information about the initial source of this outbreak is limited.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>The human enteric coronaviruses.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A coronarirus was seen in the faeces from 15 (4.2%) of 355 adults with diarrhoea and from 5 (5.2%) of 96 adults without diarrhoea. Similar particles were seen in the faeces from 5 (2.2%) of 227 children aged 1--14 years with gastroenteritis, but in none of those from 230 infants under one year of age with gastroenteritis. There was no evidence that the coronavirus was responsible for any of 34 outbreaks of gastroenteritis, although it possibly caused diarrhoea in patients admitted to a psycho-geriatric unit. Excretion of the virus often continued for many months. One strain was propagated in human embryo kidney monolayers and human embryo intestinal organ cultures, although serial passage could not be accomplished.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>The New Coronavirus, the Current King of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are respiratory viruses whose appearance on electron microscopy looks like tiny crowns (Figure 1). The first coronaviruses discovered were the cause of pediatric and adult respiratory infections that were not particularly dangerous. One study from Vanderbilt found coronavirus in about 5% of samples from upper respiratory illness and 8% from lower respiratory illness [1]. Most clinically significant coronavirus infections have been found in children &lt; 2 years of age, as reviewed in this journal in 2009 [2], although adults also might suffer severe infections [3, 4]. This was the picture as late as 2018: a group of respiratory viruses that could cause serious illness in the very young, but with negligible mortality [5, 6].</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Combining Clinical and Epidemiologic Features for Early Recognition of SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Early recognition and rapid initiation of infection control precautions are currently the most important strategies for controlling severe acute respiratory syndrome (SARS). No rapid diagnostic tests currently exist that can rule out SARS among patients with febrile respiratory illnesses. Clinical features alone cannot with certainty distinguish SARS from other respiratory illnesses rapidly enough to inform early management decisions. A balanced approach to screening that allows early recognition of SARS without unnecessary isolation of patients with other respiratory illnesses will require clinicians not only to look for suggestive clinical features but also to routinely seek epidemiologic clues suggestive of SARS coronavirus exposure. Key epidemiologic risk factors include 1) exposure to settings where SARS activity is suspected or documented, or 2) in the absence of such exposure, epidemiologic linkage to other persons with pneumonia (i.e., pneumonia clusters), or 3) exposure to healthcare settings. When combined with clinical findings, these epidemiologic features provide a possible strategic framework for early recognition of SARS.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>In this issue of Occupational Medicine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The recent COVID-19 outbreak in Wuhan city (Hubei Province of central China), presents significant public health challenges not only in china but across all affected countries worldwide. Koh [1] succinctly describes what coronaviruses are, and outlines six known species associated with human illnesses. The article also describes countries of origin of previous coronavirus disease outbreaks and reports number of known fatalities with associated case-fatality rates. Occupations implicated in the current COVID-19 outbreak are detailed and groups at high risk of contracting the infection e.g. healthcare workers highlighted. Social stigmatisation associated with COVID-19 is explored and suggested measures to contain the infection discussed.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Persistence of coronaviruses on inanimate surfaces and their inactivation with biocidal agents</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Currently, the emergence of a novel human coronavirus, SARS-CoV-2, has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described with incubation times between 2-10 days, facilitating its spread via droplets, contaminated hands or surfaces. We therefore reviewed the literature on all available information about the persistence of human and veterinary coronaviruses on inanimate surfaces as well as inactivation strategies with biocidal agents used for chemical disinfection, e.g. in healthcare facilities. The analysis of 22 studies reveals that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days, but can be efficiently inactivated by surface disinfection procedures with 62–71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite within 1 minute. Other biocidal agents such as 0.05–0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. As no specific therapies are available for SARS-CoV-2, early containment and prevention of further spread will be crucial to stop the ongoing outbreak and to control this novel infectious thread.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the Recent 2019 Novel Coronavirus (SARS-CoV-2) in Light of Past Human Coronavirus Outbreaks</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are RNA viruses that have become a major public health concern since the Severe Acute Respiratory Syndrome-CoV (SARS-CoV) outbreak in 2002. The continuous evolution of coronaviruses was further highlighted with the emergence of the Middle East Respiratory Syndrome-CoV (MERS-CoV) outbreak in 2012. Currently, the world is concerned about the 2019 novel CoV (SARS-CoV-2) that was initially identified in the city of Wuhan, China in December 2019. Patients presented with severe viral pneumonia and respiratory illness. The number of cases has been mounting since then. As of late February 2020, tens of thousands of cases and several thousand deaths have been reported in China alone, in addition to thousands of cases in other countries. Although the fatality rate of SARS-CoV-2 is currently lower than SARS-CoV, the virus seems to be highly contagious based on the number of infected cases to date. In this review, we discuss structure, genome organization, entry of CoVs into target cells, and provide insights into past and present outbreaks. The future of human CoV outbreaks will not only depend on how the viruses will evolve, but will also depend on how we develop efficient prevention and treatment strategies to deal with this continuous threat.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>Gain, Preservation, and Loss of a Group 1a Coronavirus Accessory Glycoprotein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-strand RNA viruses of extraordinary genetic complexity and diversity. In addition to a common set of genes for replicase and structural proteins, each coronavirus may carry multiple group-specific genes apparently acquired through relatively recent heterologous recombination events. Here we describe an accessory gene, ORF3, unique to canine coronavirus type I (CCoV-I) and characterize its product, glycoprotein gp3. Whereas ORF3 is conserved in CCoV-I, only remnants remain in CCoV-II and CCoV-II-derived porcine and feline coronaviruses. Our findings provide insight into the evolutionary history of coronavirus group 1a and into the dynamics of gain and loss of accessory genes.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Prediction of Intrinsic Disorder in MERS-CoV/HCoV-EMC Supports a High Oral-Fecal Transmission</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Nov 13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus, MERS-CoV (NCoV, HCoV-EMC/2012), originating from the Middle-East, has been discovered. Incoming data reveal that the virus is highly virulent to humans. A model that categorizes coronaviuses according to the hardness of their shells was developed before the discovery of MERS-CoV. Using protein intrinsic disorder prediction, coronaviruses were categorized into three groups that can be linked to the levels of oral-fecal and respiratory transmission regardless of genetic proximity. Using this model, MERS-CoV is placed into disorder group C, which consists of coronaviruses that have relatively hard inner and outer shells. The members of this group are likely to persist in the environment for a longer period of time and possess the highest oral-fecal components but relatively low respiratory transmission components. Oral-urine and saliva transmission are also highly possible since both require harder protective shells. Results show that disorder prediction can be used as a tool that suggests clues to look for in further epidemiological investigations.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral metagenomics, protein structure, and reverse genetics: key strategies for investigating coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viral metagenomics, modeling of protein structure, and manipulation of viral genetics are key approaches that have laid the foundations of our understanding of coronavirus biology. In this review, we discuss the major advances each method has provided and discuss how future studies should leverage these strategies synergistically to answer novel questions.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</h3>
<i class="tooltip">2012<span class="tooltiptext">2012</span></i><p style="text-align: justify; text-justify: inter-word;">A previously unknown coronavirus was isolated from the sputum of a 60-year-old man who presented with acute pneumonia and subsequent renal failure with a fatal outcome in Saudi Arabia. The virus (called HCoV-EMC) replicated readily in cell culture, producing cytopathic effects of rounding, detachment, and syncytium formation. The virus represents a novel betacoronavirus species. The closest known relatives are bat coronaviruses HKU4 and HKU5. Here, the clinical data, virus isolation, and molecular identification are presented. The clinical picture was remarkably similar to that of the severe acute respiratory syndrome (SARS) outbreak in 2003 and reminds us that animal coronaviruses can cause severe disease in humans.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus – A Snapshot of Current Knowledge</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Another animal to human transmission of a coronavirus occurred in December 2019 on a live animal market in the Chinese city of Wuhan causing an epidemic in China, reaching now different continents. This minireview summarizes the research literature on the virological, clinical and epidemiological aspects of this epidemic published until end of February 2020.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.</jats:p></p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 30 years, several cross-species transmission events, as well as changes in virus tropism, have mediated significant animal and human diseases. Most notable is severe acute respiratory syndrome (SARS), a lower respiratory tract disease of humans that was first reported in late 2002 in Guangdong Province, China. The disease, which quickly spread worldwide over a period of 4 months spanning late 2002 and early 2003, infected over 8,000 individuals and killed nearly 800 before it was successfully contained by aggressive public health intervention strategies. A coronavirus (SARS-CoV) was identified as the etiological agent of SARS, and initial assessments determined that the virus crossed to human hosts from zoonotic reservoirs, including bats, Himalayan palm civets (Paguma larvata), and raccoon dogs (Nyctereutes procyonoides), sold in exotic animal markets in Guangdong Province. In this review, we discuss the molecular mechanisms that govern coronavirus cross-species transmission both in vitro and in vivo, using the emergence of SARS-CoV as a model. We pay particular attention to how changes in the Spike attachment protein, both within and outside of the receptor binding domain, mediate the emergence of coronaviruses in new host populations.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus (MERS-CoV): A review</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As a novel coronavirus first reported by Saudi Arabia in 2012, the Middle East respiratory syndrome coronavirus (MERS-CoV) is responsible for an acute human respiratory syndrome. The virus, of 2C beta-CoV lineage, expresses the dipeptidyl peptidase 4 (DPP4) receptor and is densely endemic in dromedary camels of East Africa and the Arabian Peninsula. MERS-CoV is zoonotic but human-to-human transmission is also possible. Surveillance and phylogenetic researches indicate MERS-CoV to be closely associated with bats’ coronaviruses, suggesting bats as reservoirs, although unconfirmed. With no vaccine currently available for MERS-CoV nor approved prophylactics, its global spread to over 25 countries with high fatalities highlights its role as ongoing public health threat. An articulated action plan ought to be taken, preferably from a One Health perspective, for appropriately advanced countermeasures against MERS-CoV.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence of comorbidities in cases of Middle East respiratory syndrome coronavirus: a retrospective study</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Nov 5</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) is a life-threatening respiratory disease with a high case fatality rate; however, its risk factors remain unclear. We aimed to explore the influence of demographic factors, clinical manifestations and underlying comorbidities on mortality in MERS-CoV patients. Retrospective chart reviews were performed to identify all laboratory-confirmed cases of MERS-COV infection in Saudi Arabia that were reported to the Ministry of Health of Saudi Arabia between 23 April 2014 and 7 June 2016. Statistical analyses were conducted to assess the effect of sex, age, clinical presentation and comorbidities on mortality from MERS-CoV. A total of 281 confirmed MERS-CoV cases were identified: 167 (59.4%) patients were male and 55 (20%) died. Mortality predominantly occurred among Saudi nationals and older patients and was significantly associated with respiratory failure and shortness of breath. Of the 281 confirmed cases, 160 (56.9%) involved comorbidities, wherein diabetes mellitus, hypertension, ischemic heart disease, congestive heart failure, end-stage renal disease and chronic kidney disease were significantly associated with mortality from MERS-CoV and two or three comorbidities significantly affected the fatality rates from MERS-CoV. The findings of this study show that old age and the existence of underlying comorbidities significantly increase mortality from MERS-CoV.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Spatial modelling of contribution of individual level risk factors for mortality from Middle East respiratory syndrome coronavirus in the Arabian Peninsula</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jul 31</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Middle East respiratory syndrome coronavirus is a contagious respiratory pathogen that is contracted via close contact with an infected subject. Transmission of the pathogen has occurred through animal-to-human contact at first followed by human-to-human contact within families and health care facilities. DATA AND METHODS: This study is based on a retrospective analysis of the Middle East respiratory syndrome coronavirus outbreak in the Kingdom of Saudi Arabia between June 2012 and July 2015. A Geoadditive variable model for binary outcomes was applied to account for both individual level risk factors as well spatial variation via a fully Bayesian approach. RESULTS: Out of 959 confirmed cases, 642 (67%) were males and 317 (33%) had died. Three hundred and sixty four (38%) cases occurred in Ar Riyad province, while 325 (34%) cases occurred in Makkah. Individuals with some comorbidity had a significantly higher likelihood of dying from MERS-CoV compared with those who did not suffer comorbidity [Odds ratio (OR) = 2.071; 95% confidence interval (CI): 1.307, 3.263]. Health-care workers were significantly less likely to die from the disease compared with non-health workers [OR = 0.372, 95% CI: 0.151, 0.827]. Patients who had fatal clinical experience and those with clinical and subclinical experiences were equally less likely to die from the disease compared with patients who did not have fatal clinical experience and those without clinical and subclinical experiences respectively. The odds of dying from the disease was found to increase as age increased beyond 25 years and was much higher for individuals with any underlying comorbidities. CONCLUSION: Interventions to minimize mortality from the Middle East respiratory syndrome coronavirus should particularly focus individuals with comorbidity, non-health-care workers, patients with no clinical fatal experience, and patients without any clinical and subclinical experiences.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and laboratory inspection of novel coronavirus pneumonia (COVID-19) in Xiangyang, Hubei</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Since December 2019, a novel coronavirus pneumonia (COVID-19) rapidly spread in China, reached multiple continents currently.We aimed to reveal the infectious characteristics of COVID-19 that provide more information for the research of novel coronavirus. Methods: We performed a retrospective study on the clinical characteristics of 128 COVID-19 cases with laboratory-confirmed from Xiangyang No 1 Hospitalad during January 2020 to 16 February 2020. Results: Female patients account for 53.1%. The aged below 20 years that accounts for 1.6% of overall patients. The aged in 21~50, 51~65, over 66 years were accounts for 44.5%, 35.1%,18.8%, respectively. In the difference age spectrum, all severe groups compared with non-severe groups were difference significantly ( P &lt; 0.01 ). Fever ( 89.8% ) and Cough ( 67.2% ) were common clinical symptoms. The rate of patients with sore throats (14.1%) was rare. The rate of chest computed tomography scan showing ground glass opacity in overall, non-severe, severe groups were 63.3%, 60.7%, 76.2%, respectively. White blood cell counts in the normal range of overall patients, but severe group patients were increased significantly ( P &lt; 0.01). Lymphocytes of overall patients were decreased. Alanine transaminase (ALT) and aspartate transaminase (AST) in the normal range of overall patients, but its were elevated in the severe group. Creatinine (CR) and blood urea nitrogen (BUN) of overall patients in the normal range. C-reactive protein (CRP) level of all patients were increased markedly, but it in the severe group was significantly higher than that in the non-severe group ( P &lt; 0.01 ). Conclusions: Our data provide more information that advanced age, lower lymphocytes levels at the diagnosed COVID-19 patients may be a risk factor for unfavourable prognosis. The white blood cells and C-reactive protein level elevated in severe COVID-19 patients may be accompanying bacterial infection. 2019-nCov may be carries a risk factor of impaired liver and kidney function.</jats:p></p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Single-Amino-Acid Substitutions in Open Reading Frame (ORF) 1b-nsp14 and ORF 2a Proteins of the Coronavirus Mouse Hepatitis Virus Are Attenuating in Mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A reverse genetic system was recently established for the coronavirus mouse hepatitis virus strain A59 (MHV-A59), in which cDNA fragments of the RNA genome are assembled in vitro into a full-length genome cDNA, followed by electroporation of in vitro-transcribed genome RNA into cells with recovery of viable virus. The “in vitro-assembled” wild-type MHV-A59 virus (icMHV-A59) demonstrated replication identical to laboratory strains of MHV-A59 in tissue culture; however, icMHV-A59 was avirulent following intracranial inoculation of C57BL/6 mice. Sequencing of the cloned genome cDNA fragments identified two single-nucleotide mutations in cloned genome fragment F, encoding a Tyr6398His substitution in open reading frame (ORF) 1b p59-nsp14 and a Leu94Pro substitution in the ORF 2a 30-kDa protein. The mutations were repaired individually and together in recombinant viruses, all of which demonstrated wild-type replication in tissue culture. Following intracranial inoculation of mice, the viruses encoding Tyr6398His/Leu94Pro substitutions and the Tyr6398His substitution alone demonstrated log(10) 50% lethal dose (LD(50)) values too great to be measured. The Leu94Pro mutant virus had reduced but measurable log(10) LD(50), and the “corrected” Tyr6398/Leu94 virus had a log(10) LD(50) identical to wild-type MHV-A59. The experiments have defined residues in ORF 1b and ORF 2a that attenuate virus replication and virulence in mice but do not affect in vitro replication. The results suggest that these proteins serve roles in pathogenesis or virus survival in vivo distinct from functions in virus replication. The study also demonstrates the usefulness of the reverse genetic system to confirm the role of residues or proteins in coronavirus replication and pathogenesis.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Community Transmission of Severe Acute Respiratory Syndrome Coronavirus 2, Shenzhen, China, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since early January 2020, after the outbreak of 2019 novel coronavirus infection in Wuhan, China, ≈365 confirmed cases have been reported in Shenzhen, China. The mode of community and intrafamily transmission is threatening residents in Shenzhen. Strategies to strengthen prevention and interruption of these transmissions should be urgently addressed.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Prolonged Shedding of Human Coronavirus in Hematopoietic Cell Transplant Recipients: Risk Factors and Viral Genome Evolution</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Recent data suggest that human coronavirus (HCoV) pneumonia is associated with significant mortality in hematopoietic cell transplant (HCT) recipients. Investigation of risk factors for prolonged shedding and intrahost genome evolution may provide critical information for development of novel therapeutics. METHODS: We retrospectively reviewed HCT recipients with HCoV detected in nasal samples by polymerase chain reaction (PCR). HCoV strains were identified using strain-specific PCR. Shedding duration was defined as time between first positive and first negative sample. Logistic regression analyses were performed to evaluate factors for prolonged shedding (≥21 days). Metagenomic next-generation sequencing (mNGS) was conducted when ≥4 samples with cycle threshold values of <28 were available. RESULTS: Seventeen of 44 patients had prolonged shedding. Among 31 available samples, 35% were OC43, 32% were NL63, 19% were HKU1, and 13% were 229E; median shedding duration was similar between strains (P = .79). Bivariable logistic regression analyses suggested that high viral load, receipt of high-dose steroids, and myeloablative conditioning were associated with prolonged shedding. mNGS among 5 subjects showed single-nucleotide polymorphisms from OC43 and NL63 starting 1 month following onset of shedding. CONCLUSIONS: High viral load, high-dose steroids, and myeloablative conditioning were associated with prolonged shedding of HCoV in HCT recipients. Genome changes were consistent with the expected molecular clock of HCoV.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a pangolin niche for a 2019-nCoV-like coronavirus through an extensive meta-metagenomic search</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-14</span></i><p style="text-align: justify; text-justify: inter-word;">In numerous instances, tracking the biological significance of a nucleic acid sequence can be augmented through the identification of environmental niches in which the sequence of interest is present. Many metagenomic datasets are now available, with deep sequencing of samples from diverse biological niches. While any individual metagenomic dataset can be readily queried using web-based tools, meta-searches through all such datasets are less accessible. In this brief communication, we demonstrate such a meta-meta-genomic approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the keyword "virome". In addition to the homology to bat coronaviruses observed in descriptions of the 2019-nCoV sequence (F. Wu et al. 2020, Nature, doi.org/10.1038/s41586-020-2008-3; P. Zhou et al. 2020, Nature, doi.org/10.1038/s41586-020-2012-7), we note a strong homology to numerous sequence reads in a metavirome dataset generated from the lungs of deceased Pangolins reported by Liu et al. (Viruses 11:11, 2019, http://doi.org/10.3390/v11110979). Our observations are relevant to discussions of the derivation of 2019-nCoV and illustrate the utility and limitations of meta-metagenomic search tools in effective and rapid characterization of potentially significant nucleic acid sequences.  ImportanceMeta-metagenomic searches allow for high-speed, low-cost identification of potentially significant biological niches for sequences of interest.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</jats:p></p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Advance in human coronaviruses research of host interactions</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">2019-novel coronavirus (2019-nCoV) is a highly pathogenic human CoV that first emerged in Wuhan in 2019.2019-nCoV has a zoonotic origin and poses a major threat to public health.However, little is known about the viral factors contributing to the high virulence of 2019-nCoV.Many animal viruses, including CoVs, encode proteins that interfere with host gene expression, including those involved in antiviral immune responses, and these viral proteins are often major virulence factors.Human coronaviruses (HCoVs) are known respiratory pathogens associated with a range of respiratory infection.In the past 17 years, the onset of 2019-nCoV, severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) have thrust HCoVs into spotlight of the research community due to their high pathogenicity in humans.The recent study of HCoVs-host interactions has contributed extensively to our understanding of infection pathogenesis of 2019-nCoV.This review discuss various host physiopathologic mechanism, such as apoptosis, innate immunity, endoplasmic reticulum (ER) stress response, mitogen-activated protein kinase (MAPK) pathway and nuclear factor kappa B (NF-&kappa;B) pathway that may be modulated by HCoVs and provides evidence for the intensive investigate of 2019-nCoV infection.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Facing the COVID-19 outbreak: What should we know and what could we do?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-26</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (2019-nCov) was identified in Wuhan, Hubei Province, China in December of 2019. This new coronavirus has resulted in thousands of cases of lethal disease in China, with additional patients being identified in a rapidly growing number internationally. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public database and the state-of-the-art single-cell RNA-Seq technique, we analyzed the ACE2 RNA expression profile in the normal human lungs. The result indicates that the ACE2 virus receptor expression is concentrated in a small population of type II alveolar cells (AT2). Surprisingly, we found that this population of ACE2-expressing AT2 also highly expressed many other genes that positively regulating viral reproduction and transmission. A comparison between eight individual samples demonstrated that the Asian male one has an extremely large number of ACE2-expressing cells in the lung. This study provides a biological background for the epidemic investigation of the 2019-nCov infection disease, and could be informative for future anti-ACE2 therapeutic strategy development.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Mechanisms of Coronavirus Cell Entry Mediated by the Viral Spike Protein</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Jun 20</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are enveloped positive-stranded RNA viruses that replicate in the cytoplasm. To deliver their nucleocapsid into the host cell, they rely on the fusion of their envelope with the host cell membrane. The spike glycoprotein (S) mediates virus entry and is a primary determinant of cell tropism and pathogenesis. It is classified as a class I fusion protein, and is responsible for binding to the receptor on the host cell as well as mediating the fusion of host and viral membranes—A process driven by major conformational changes of the S protein. This review discusses coronavirus entry mechanisms focusing on the different triggers used by coronaviruses to initiate the conformational change of the S protein: receptor binding, low pH exposure and proteolytic activation. We also highlight commonalities between coronavirus S proteins and other class I viral fusion proteins, as well as distinctive features that confer distinct tropism, pathogenicity and host interspecies transmission characteristics to coronaviruses.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Relation Between Chest CT Findings and Clinical Conditions of Coronavirus Disease (COVID-19) Pneumonia: A Multicenter Study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE. The increasing number of cases of confirmed coronavirus disease (COVID-19) in China is striking. The purpose of this study was to investigate the relation between chest CT findings and the clinical conditions of COVID-19 pneumonia. MATERIALS AND METHODS. Data on 101 cases of COVID-19 pneumonia were retrospectively collected from four institutions in Hunan, China. Basic clinical characteristics and detailed imaging features were evaluated and compared between two groups on the basis of clinical status: nonemergency (mild or common disease) and emergency (severe or fatal disease). RESULTS. Patients 21-50 years old accounted for most (70.2%) of the cohort, and five (5.0%) patients had disease associated with a family outbreak. Most patients (78.2%) had fever as the onset symptom. Most patients with COVID-19 pneumonia had typical imaging features, such as ground-glass opacities (GGO) (87 [86.1%]) or mixed GGO and consolidation (65 [64.4%]), vascular enlargement in the lesion (72 [71.3%]), and traction bronchiectasis (53 [52.5%]). Lesions present on CT images were more likely to have a peripheral distribution (88 [87.1%]) and bilateral involvement (83 [82.2%]) and be lower lung predominant (55 [54.5%]) and multifocal (55 [54.5%]). Patients in the emergency group were older than those in the non-emergency group. Architectural distortion, traction bronchiectasis, and CT involvement score aided in evaluation of the severity and extent of the disease. CONCLUSION. Patients with confirmed COVID-19 pneumonia have typical imaging features that can be helpful in early screening of highly suspected cases and in evaluation of the severity and extent of disease. Most patients with COVID-19 pneumonia have GGO or mixed GGO and consolidation and vascular enlargement in the lesion. Lesions are more likely to have peripheral distribution and bilateral involvement and be lower lung predominant and multifocal. CT involvement score can help in evaluation of the severity and extent of the disease.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>In Vivo and In Vitro Models of Demyelinating Disease: Endogenous Factors Influencing Demyelinating Disease Caused by Mouse Hepatitis Virus in Rats and Mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Intracerebral inoculation of JHM virus (JHMV), the neuropathic strain of mouse hepatitis virus, into Wistar Furth, Wistar Lewis, and Fischer 344 rats at various ages indicated that Wistar Furth rats are more susceptible to the virus than are the other strains. Fischer 344 and Wistar Lewis rats were more resistant to inoculation at 2 and 5 days of age and completely resistant by 10 days of age. In contrast, Wistar Furth rats which were very susceptible at both 2 and 5 days of age remained susceptible until 21 days of age. Intracerebral challenge of an F1 cross between Wistar Furth and Wistar Lewis rats at 10 days of age indicated that resistance to JHMV infection is dominant. Cyclophosphamide treatment 28 days after intracerebral inoculation exacerbated an inapparent infection, leading to paralysis in eight of nine and death in six of nine Wistar Furth test rats. In such immunosuppressed animals, grey- and white-matter lesions were noted throughout the central nervous system, in contrast to the purely demyelinating lesions noted previously. Since rats, unlike mice, were not susceptible to disease after intracerebral injection with the serorelated viscerotropic strain MHV-3, we wished to extend our understanding of the neurological disease process elicited by the two viruses in rodents. For this reason, various mouse strains, including some with recognized immunodeficiencies, were challenged by different routes of inoculation. Intraperitoneal infection of nude and beige mice with JHMV indicated that lack of natural killer cell functions does not markedly enhance the susceptibility to virus, whereas T-cell activity appears to be essential for resisting infection. JHMV and MHV-3 replication in peritoneal macrophages from highly resistant A/J mice was reduced in comparison with that noted in macrophages from susceptible C57BL6/J mice. An initial intraperitoneal inoculation of JHMV was able to protect C57BL6/J mice against fatal intracerebral challenge within 3 days, whereas A/J mice remained susceptible beyond day 3. The protective effect did not appear to result from increased levels of circulating interferon, preceded elevation in serum JHMV-neutralizing antibody titers, and persisted for at least several weeks after intraperitoneal inoculation. Based on the combined studies described here and on previous work by us and others, it appears that the factors influencing the outcome of coronavirus disease in rodents are age at inoculation, route of challenge, genetic constitution of the virus and host, and competence of the immune system, particularly cellular immunity involving T-cells.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Caution: The clinical characteristics of COVID-19 patients at admission are changing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background  With the emergence of 4rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 (COVID-19) has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Methods Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. Results  Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24).  Conclusions The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.</jats:p></p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Human Coronavirus HCoV-NL63</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 25</span></i><p style="text-align: justify; text-justify: inter-word;">Even though coronavirus infection of humans is not normally associated with severe diseases, the identification of the coronavirus responsible for the outbreak of severe acute respiratory syndrome showed that highly pathogenic coronaviruses can enter the human population. Shortly thereafter, in Holland in 2004, another novel human coronavirus (HCoV-NL63) was isolated from a seven-month old infant suffering from respiratory symptoms. This virus has subsequently been identified in various countries, indicating a worldwide distribution. HCoV-NL63 has been shown to infect mainly children and the immunocommpromised, who presented with either mild upper respiratory symptoms (cough, fever and rhinorrhoea) or more serious lower respiratory tract involvement such as bronchiolitis and croup, which was observed mainly in younger children. In fact, HCoV-NL63 is the aetiological agent for up to 10% of all respiratory diseases. This review summarizes recent findings of human coronavirus HCoV-NL63 infections, including isolation and identification, phylogeny and taxonomy, genome structure and transcriptional regulation, transmission and pathogenesis, and detection and diagnosis.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Characteristics of Patients with Severe Pneumonia Caused by the 2019 Novel Coronavirus in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Background: A new virus broke out in Wuhan, Hubei, China, and was later named 2019 novel coronavirus (2019-nCoV). The clinical characteristics of severe pneumonia caused by 2019-nCoV are still not clear. Objectives: The aim of this study was to explore the clinical characteristics and risk factors of the severe pneumonia caused by the 2019-nCoV in Wuhan, China. Method: The study included patients hospitalized at the central hospital of Wuhan who had been diagnosed with a pneumonia caused by the novel coronavirus. Clinical features, chronic co-morbidities, demographic data, laboratory examinations, and chest computed tomography (CT) scans were reviewed through electronic medical records. SPSS was used for data analysis to explore the clinical characteristics and risk factors of the patients with the severe pneumonia. Results: A total of 110 patients diagnosed with 2019 novel coronavirus pneumonia were included in the study, including 38 with severe pneumonia and 72 with non-severe pneumonia. Statistical analysis showed that advanced age, an increase of D-dimer, and a decrease of lymphocytes were characteristics of the patients with severe pneumonia. Moreover, in the early stage of the disease, chest CT scans of patients with the severe pneumonia showed the illness can progress rapidly. Conclusions: Advanced age, lymphocyte decline, and D-dimer elevation are important characteristics of patients with severe pneumonia. Clinicians should focus on these characteristics to identify high-risk patients at an early stage.</jats:p></p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding the Influence Factors in Viral Nucleic Acid Test of 2019 novel Coronavirus (2019-nCoV)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">At present, the prevention and control of new coronavirus has entered a critical period. However, the use of quantitative real-time PCR (qRT-PCR) assays for the detection of viral nucleic acid, as a crucial diagnostic approach, has been doubted in clinical practice. Herein, we have reviewed the current status of epidemic prevention and control, latest development of detection technologies, disease characteristics, clinical sampling and transport. We have also discussed the factors that may affect the performance of viral nucleic acid detection, and suggested some effective methods to improve the detection performance of the assays.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 Update – March 2020 | Global Health | JN Learning | JAMA Ed Hub [Video recording]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus testing, mortality, vaccine development, containment vs mitigation, and more. Anthony S. Fauci, MD discusses the latest developments in the global spread of COVID-19 and the SARS-CoV-2 virus with JAMA Editor Howard Bauchner, MD. - What's the difference between COVID-19 and SARS-CoV-2? (01:15) - What's the status and accuracy of diagnostic testing in the US? (01:58) - What's the case-fatality rate for the virus? (05:31) - Scientific advances and vaccine development (25:06) - Are current clinical trials providing a picture of treatments? (13:41) - Risk communication: how do we present information so there's faith that it's accurate? (15:24) - Risk groups (children, the elderly, pregnant women) (16:26) - Containment vs mitigation vs quarantine vs isolation (19:10) - Protecting the elderly and nursing home resident (23:52) - Public health prospects in Latin America, Africa (26:35) - Will coronavirus wane in warmer months like influenza? (27:52) - Why is anxiety so high about this disease?- Does the US have capacity to care for COVID19 infection? (31:03) - What is your daily schedule like? (32:23)</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation period as part of the case definition of severe respiratory illness caused by a novel coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Due to non-specific symptoms following acute respiratory viral infections, it is difficult for many countries without on-going transmission of a novel coronavirus to rule out other possibilities including influenza in advance of isolating imported febrile individuals with a possible exposure history. The incubation period helps differential diagnosis and up to two days is suggestive of influenza. It is worth accounting for the incubation period as part of the case definition of novel coronavirus infection.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential role of inanimate surfaces for the spread of coronaviruses and their inactivation with disinfectant agents</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary The novel human coronavirus 2019-nCoV has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described, probably via droplets but possibly also via contaminated hands or surfaces. In a recent review on the persistence of human and veterinary coronaviruses on inanimate surfaces it was shown that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days. Some disinfectant agents effectively reduce coronavirus infectivity within 1 minute such 62% - 71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite. Other compounds such as 0.05% - 0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. An effective surface disinfection may help to ensure an early containment and prevention of further viral spread.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Air, Surface Environmental, and Personal Protective Equipment Contamination by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) From a Symptomatic Patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses have been implicated in nosocomial outbreaks with environmental contamination as a route of transmission. Similarly, nosocomial transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been reported. However, the mode of transmission and extent of environmental contamination are unknown.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Active Monitoring of Persons Exposed to Patients with Confirmed COVID-19 — United States, January–February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, an outbreak of coronavirus disease 2019 (COVID-19), caused by the virus SARS-CoV-2, began in Wuhan, China (1). The disease spread widely in China, and, as of February 26, 2020, COVID-19 cases had been identified in 36 other countries and territories, including the United States. Person-to-person transmission has been widely documented, and a limited number of countries have reported sustained person-to-person spread.* On January 20, state and local health departments in the United States, in collaboration with teams deployed from CDC, began identifying and monitoring all persons considered to have had close contact† with patients with confirmed COVID-19 (2). The aims of these efforts were to ensure rapid evaluation and care of patients, limit further transmission, and better understand risk factors for transmission.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-22</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of rotavirus and coronavirus shedding in two beef cow herds in Idaho</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Fecal samples were taken at the time of pregnancy examinations and at parturition from two beef herds. They were also taken from sick calves at the onset of disease, and from 25% of the healthy calves at 15 days of age. All fecal samples were examined by electron microscopy for viruses. Four cows in herd A were detected excreting coronavirus, one at the time of the pregnancy examinations and three at parturition. The first cow was removed from the herd and the others calved at the end of the season. There were no sick calves. No cows in herd B were detected excreting virus at the time of pregnancy checks, but fourteen coronavirus and two rotavirus carrier cows were found at parturition. All but two calves sampled had large numbers of virus particles in their feces. Clinical illness was associated with dams shedding virus and with nightly low temperatures.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological and clinical features of human coronavirus infections among different subsets of patients</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Nov 5</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Epidemiological and clinical data of human coronaviruses (HCoVs) infections are restricted to span 1–3 years at most. We conducted a comprehensive 9‐year study on HCoVs by analyzing 1137 respiratory samples from four subsets of patients (asymptomatic, general community, with comorbidities, and hospitalized) in São Paulo, Brazil. METHODS: A pan‐coronavirus RT‐PCR screening assay was performed, followed by species‐specific real‐time RT‐PCR monoplex assays. RESULTS: Human coronaviruses were detected in 88 of 1137 (7.7%) of the samples. The most frequently detected HCoV species were NL63 (50.0%) and OC43 (27.3%). Patients with comorbidities presented the highest risk of acquiring coronavirus infection (odds ratio = 4.17; 95% confidence interval = 1.9–9.3), and children with heart diseases revealed a significant HCoV infection presence. Dyspnea was more associated with HCoV‐229E infections (66.6%), and cyanosis was reported only in HCoV‐OC43 infections. There were interseasonal differences in the detection frequencies, with HCoV‐229E being predominant in the year 2004 (61.5%) and HCoV‐NL63 (70.8%) in 2008. CONCLUSIONS: Our data provide a novel insight into the epidemiology and clinical knowledge of HCoVs among different subsets of patients, revealing that these viruses may cause more than mild respiratory tract disease.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>A Rare Case of Human Coronavirus 229E Associated with Acute Respiratory Distress Syndrome in a Healthy Adult</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 15</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus 229E (HCoV-229E) is one of the first coronavirus strains being described. It is linked to common cold symptoms in healthy adults. Younger children and the elderly are considered vulnerable to developing lower respiratory tract infections (LRTIs). In particular, immunocompromised patients have been reported with severe and life-threatening LRTIs attributed to HCoV-229E. We report for the first time a case of LRTI and acute respiratory distress syndrome developed in a healthy adult with no comorbidities and HCoV-229E strain identified as the only causative agent. A 45-year-old female with a clear medical history presented with fever, cough, and headache. Respiratory tract infection was diagnosed, and empirical antibiotics were started. Within two days, she developed bilateral pleural effusions, diffuse consolidations, and ground glass opacities involving all lung fields. She needed immediate oxygen supply, while ABGs deteriorated and chest imaging and PaO(2)/FiO(2) indicated ARDS. Early administration of systemic corticosteroids led to gradual clinical improvement. Multiplex PCR from nasal secretions was positive only for HCoV-229E and negative for multiple other pathogens. It remains to be elucidated how an immunocompetent adult developed a life-threatening LRTI caused by a “benign considered” coronavirus strain, the HCoV-229E.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Profile of Antibodies to the Nucleocapsid Protein of the Severe Acute Respiratory Syndrome (SARS)-Associated Coronavirus in Probable SARS Patients</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Profiles of antibodies to the nucleocapsid protein of the severe acute respiratory syndrome (SARS)-associated coronavirus in 445 probable SARS patients and 3,749 healthy people or non-SARS patients were analyzed by antigen-capturing enzyme-linked immunosorbent assay. Antinucleocapsid antibodies were elucidated in 17.5% of the probable SARS patients 1 to 7 days after the onset of symptoms and in 80% of the patients 8 to 14 days after the onset. About 90% of the probable SARS patients were positive 15 or more days after illness. Antibody titers increased up to 70 days, and high antibody titers were maintained at least for another 3 months. Of the healthy people and non-SARS patients, only seven (0.187%) were weakly positive.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional Cell Receptors for Human Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Viruses infect host cells by binding to receptors on thesurface of cells. Receptor is an important factor affecting host range and interspecific transmission. In December 2019, an outbreak of unexplained pneumonia occurred in Wuhan, Hubei province. The pathogen was a new coronavirus, named 2019 NovelCoronavirus (2019-nCoV) by WHO. Angiotensin-converting enzyme 2 (ACE2) was found to be the receptor of 2019-nCoV.This review provides a brief overview of human coronavirus receptors and their applications, with a view to providing references for the tracing, cross-species transmission, epidemiological analysis and antiviral and vaccine studies of 2019-nCoV.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Histopathologic Evaluation and Scoring of Viral Lung Infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Emergent coronaviruses such as MERS-CoV and SARS-CoV can cause significant morbidity and mortality in infected individuals. Lung infection is a common clinical feature and contributes to disease severity as well as viral transmission. Animal models are often required to study viral infections and therapies, especially during an initial outbreak. Histopathology studies allow for identification of lesions and affected cell types to better understand viral pathogenesis and clarify effective therapies. Use of immunostaining allows detection of presumed viral receptors and viral tropism for cells can be evaluated to correlate with lesions. In the lung, lesions and immunostaining can be qualitatively described to define the cell types, microanatomic location, and type of changes seen. These features are important and necessary, but this approach can have limitations when comparing treatment groups. Semiquantitative and quantitative tissue scores are more rigorous as these provide the ability to statistically compare groups and increase the reproducibility and rigor of the study. This review describes principles, approaches, and resources that can be useful to evaluate coronavirus lung infection, focusing on MER-CoV infection as the principal example.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection and Prevalence Patterns of Group I Coronaviruses in Bats, Northern Germany</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Apr</span></i><p style="text-align: justify; text-justify: inter-word;">We tested 315 bats from 7 different bat species in northern Germany for coronaviruses by reverse transcription–PCR. The overall prevalence was 9.8%. There were 4 lineages of group I coronaviruses in association with 4 different species of verspertilionid bats (Myotis dasycneme, M. daubentonii, Pipistrellus nathusii, P. pygmaeus). The lineages formed a monophyletic clade of bat coronaviruses found in northern Germany. The clade of bat coronaviruses have a sister relationship with a clade of Chinese type I coronaviruses that were also associated with the Myotis genus (M. ricketti). Young age and ongoing lactation, but not sex or existing gravidity, correlated significantly with coronavirus detection. The virus is probably maintained on the population level by amplification and transmission in maternity colonies, rather than being maintained in individual bats.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-Like Coronavirus WIV1-CoV Does Not Replicate in Egyptian Fruit Bats (Rousettus aegyptiacus)</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 19</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS)-like WIV1-coronavirus (CoV) was first isolated from Rhinolophus sinicus bats and can use the human angiotensin converting enzyme 2 (ACE2) receptor. In the current study, we investigate the ability of WIV1-CoV to infect Rousettus aegyptiacus bats. No clinical signs were observed throughout the experiment. Furthermore, only four oropharyngeal swabs and two respiratory tissues, isolated on day 3 post inoculation, were found positive for viral RNA. Two out of twelve bats showed a modest increase in coronavirus specific antibodies post challenge. In conclusion, WIV1-CoV was unable to cause a robust infection in Rousettus aegyptiacus bats.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Metapneumovirus-associated Atypical Pneumonia and SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">Acute pneumonia developed in a previously healthy man during the outbreak of severe acute respiratory syndrome (SARS) in southern China in March 2003. Antibiotic treatment was ineffective, and he died 8 days after illness onset. Human metapneumovirus was isolated from lung tissue. No other pathogen was found. Other etiologic agents should thus be sought in apparent SARS cases when coronavirus infection cannot be confirmed.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Mouse Hepatitis Coronavirus A59 Nucleocapsid Protein Is a Type I Interferon Antagonist</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of several new coronaviruses, including the etiological cause of severe acute respiratory syndrome, has significantly increased the importance of understanding virus-host cell interactions of this virus family. We used mouse hepatitis virus (MHV) A59 as a model to gain insight into how coronaviruses affect the type I alpha/beta interferon (IFN) system. We demonstrate that MHV is resistant to type I IFN. Protein kinase R (PKR) and the alpha subunit of eukaryotic translation initiation factor are not phosphorylated in infected cells. The RNase L activity associated with 2′,5′-oligoadenylate synthetase is not activated or is blocked, since cellular RNA is not degraded. These results are consistent with lack of protein translation shutoff early following infection. We used a well-established recombinant vaccinia virus (VV)-based expression system that lacks the viral IFN antagonist E3L to screen viral genes for their ability to rescue the IFN sensitivity of the mutant. The nucleocapsid (N) gene rescued VVΔE3L from IFN sensitivity. N gene expression prevents cellular RNA degradation and partially rescues the dramatic translation shutoff characteristic of the VVΔE3L virus. However, it does not prevent PKR phosphorylation. The results indicate that the MHV N protein is a type I IFN antagonist that likely plays a role in circumventing the innate immune response.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and Genetic Diversity of Coronaviruses in Bats from China</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses can infect a variety of animals including poultry, livestock, and humans and are currently classified into three groups. The interspecies transmissions of coronaviruses between different hosts form a complex ecosystem of which little is known. The outbreak of severe acute respiratory syndrome (SARS) and the recent identification of new coronaviruses have highlighted the necessity for further investigation of coronavirus ecology, in particular the role of bats and other wild animals. In this study, we sampled bat populations in 15 provinces of China and reveal that approximately 6.5% of the bats, from diverse species distributed throughout the region, harbor coronaviruses. Full genomes of four coronavirues from bats were sequenced and analyzed. Phylogenetic analyses of the spike, envelope, membrane, and nucleoprotein structural proteins and the two conserved replicase domains, putative RNA-dependent RNA polymerase and RNA helicase, revealed that bat coronaviruses cluster in three different groups: group 1, another group that includes all SARS and SARS-like coronaviruses (putative group 4), and an independent bat coronavirus group (putative group 5). Further genetic analyses showed that different species of bats maintain coronaviruses from different groups and that a single bat species from different geographic locations supports similar coronaviruses. Thus, the findings of this study suggest that bats may play an integral role in the ecology and evolution of coronaviruses.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>The role of programmed-1 ribosomal frameshifting in coronavirus propagation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses have the potential to cause significant economic, agricultural and health problems. The severe acute respiratory syndrome (SARS) associated coronavirus outbreak in late 2002, early 2003 called attention to the potential damage that coronaviruses could cause in the human population. The ensuing research has enlightened many to the molecular biology of coronaviruses. A programmed -1 ribosomal frameshift is required by coronaviruses for the production of the RNA dependent RNA polymerase which in turn is essential for viral replication. The frameshifting signal encoded in the viral genome has additional features that are not essential for frameshifting. Elucidation of the differences between coronavirus frameshift signals and signals from other viruses may help our understanding of these features. Here we summarize current knowledge and add additional insight regarding the function of the programmed -1 ribosomal frameshift signal in the coronavirus lifecycle.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus (HCoV) Infection Among Adults in Cleveland, Ohio: An Increasingly Recognized Respiratory Pathogen</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Oct 4</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human Coronaviruses (CoV) have been long recognized as a common cause of respiratory tract disease including severe respiratory tract illness, yet there are few recent studies characterizing disease among adults in the United States. Here, we describe CoV infections and clinical characteristics among adults (>18 years) presenting with respiratory illness in Cleveland, Ohio. METHODS: Between February 1, 2016 and April 30, 2017, 2949 nasopharyngeal swab specimens were analyzed by NxTAG Respiratory Pathogen Panel in adults presenting with respiratory illness at MetroHealth Medical Center. Clinical data were collected on adults whose samples screened positive for CoV-HKU1, CoV-OC43, CoV-229E or CoV-NL63. RESULTS: Coronaviruses were detected in 192 (6.5%) adults including 105 (3.5%) OC43, 67 (2.3%) 229E, 13 (0.4%) HKU1 and 7 (0.2%) NL63. The majority of adults with coronavirus infection were females (66.2%) with a median age of 53 years. Common comorbidities included smoking (40.0%), asthma (38.0%), COPD (35.4%), and inhaled corticosteroid use (28.6%). Eighty-five (46.4%) required admission to the hospital. Common presenting symptoms included shortness of breath (42.7%) and cough (31.0%) whereas fever was uncommon (12.5%). Gastrointestinal symptoms were more common in HKU1 and NL63 infected adults. Seventy-three percent of coronavirus disease occurred between the months of January and March. Despite the recognition of coronavirus infection, 70 (36.5%) received antibiotics for their disease. CONCLUSION: This study provides needed insight into clinical characteristics and severity associated with coronavirus infection in adults. Coronavirus infection should be considered in differential diagnosis of respiratory tract illness in adults including those that require hospitalization, have a history of smoking and have pulmonary comorbidities. DISCLOSURES: All authors: No reported disclosures.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Systems approaches to coronavirus pathogenesis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses comprise a large group of emergent human and animal pathogens, including the highly pathogenic SARS-CoV and MERS-CoV strains that cause significant morbidity and mortality in infected individuals, especially the elderly. As emergent viruses may cause episodic outbreaks of disease over time, human samples are limited. Systems biology and genetic technologies maximize opportunities for identifying critical host and viral genetic factors that regulate susceptibility and virus-induced disease severity. These approaches provide discovery platforms that highlight and allow targeted confirmation of critical targets for prophylactics and therapeutics, especially critical in an outbreak setting. Although poorly understood, it has long been recognized that host regulation of virus-associated disease severity is multigenic. The advent of systems genetic and biology resources provide new opportunities for deconvoluting the complex genetic interactions and expression networks that regulate pathogenic or protective host response patterns following virus infection. Using SARS-CoV as a model, dynamic transcriptional network changes and disease-associated phenotypes have been identified in different genetic backgrounds, leading to the promise of population-wide discovery of the underpinnings of Coronavirus pathogenesis.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological and Genetic Correlates of Severe Acute Respiratory Syndrome Coronavirus Infection in the Hospital with the Highest Nosocomial Infection Rate in Taiwan in 2003</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Taiwan experienced a series of outbreaks of nosocomial severe acute respiratory syndrome (SARS) infections in 2003. Two months after the final outbreak, we recruited 658 employees from the hospital that suffered the first and most severe SARS infections to help us investigate epidemiological and genetic factors associated with the SARS coronavirus (SARS-CoV). SARS-CoV infections were detected by using enzyme immunoassays and confirmed by a combination of Western blot assays, neutralizing antibody tests, and commercial SARS tests. Risk factors were analyzed via questionnaire responses and sequence-specific oligonucleotide probes of human leukocyte antigen (HLA) alleles. Our results indicate that 3% (20/658) of the study participants were seropositive, with one female nurse identified as a subclinical case. Identified SARS-CoV infection risk factors include working in the same building as the hospital's emergency room and infection ward, providing direct care to SARS patients, and carrying a Cw*0801 HLA allele. The odds ratio for contracting a SARS-CoV infection among persons with either a homozygous or a heterozygous Cw*0801 genotype was 4.4 (95% confidence interval, 1.5 to 12.9; P = 0.007).</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Feline aminopeptidase N is not a functional receptor for avian infectious bronchitis virus</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Feb 26</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronaviruses are an important cause of infectious diseases in humans, including severe acute respiratory syndrome (SARS), and have the continued potential for emergence from animal species. A major factor in the host range of a coronavirus is its receptor utilization on host cells. In many cases, coronavirus-receptor interactions are well understood. However, a notable exception is the receptor utilization by group 3 coronaviruses, including avian infectious bronchitis virus (IBV). Feline aminopeptidase N (fAPN) serves as a functional receptor for most group 1 coronaviruses including feline infectious peritonitis virus (FIPV), canine coronavirus, transmissible gastroenteritis virus (TGEV), and human coronavirus 229E (HCoV-229E). A recent report has also suggested a role for fAPN during IBV entry (Miguel B, Pharr GT, Wang C: The role of feline aminopeptidase N as a receptor for infectious bronchitis virus. Brief review. Arch Virol 2002, 147:2047–2056. RESULTS: Here we show that, whereas both transient transfection and constitutive expression of fAPN on BHK-21 cells can rescue FIPV and TGEV infection in non-permissive BHK cells, fAPN expression does not rescue infection by the prototype IBV strain Mass41. To account for the previous suggestion that fAPN could serve as an IBV receptor, we show that feline cells can be infected with the prototype strain of IBV (Mass 41), but with low susceptibility compared to primary chick kidney cells. We also show that BHK-21 cells are slightly susceptible to certain IBV strains, including Ark99, Ark_DPI, CA99, and Iowa97 (<0.01% efficiency), but this level of infection is not increased by fAPN expression. CONCLUSION: We conclude that fAPN is not a functional receptor for IBV, the identity of which is currently under investigation.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 17</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world’s ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 17</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world’s ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /></body>