
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: immune response and immunity</h2><br /></div><br /><br /><br /><div class="card"><h3>Immune pathways and defence mechanisms in honey bees Apis mellifera</h3>
<p style="text-align: justify; text-justify: inter-word;">Here we focus on individual defences, by presenting a genome-wide analysis of immunity in a social insect, the honey bee Apis mellifera . We present honey bee models for each of four signalling pathways associated with immunity, identifying plausible orthologues for nearly all predicted pathway members. When compared to the sequenced Drosophila and Anopheles genomes, honey bees possess roughly one-third as many genes in 17 gene families implicated in insect immunity. We suggest that an implied reduction in immune flexibility in bees reflects either the strength of social barriers to disease, or a tendency for bees to be attacked by a limited set of highly coevolved pathogens.</p>
<p align="right"><i>score: 165</i></p>

</div><br /><br /><br /><div class="card"><h3>Brief review of the chicken Major Histocompatibility Complex: the genes, their distribution on chromosome 16, and their contributions to disease resistance</h3>
<p style="text-align: justify; text-justify: inter-word;">Nearly all genes presently mapped to chicken chromosome 16 (GGA 16) have either a demonstrated role in immune responses or are considered to serve in immunity by reason of sequence homology with immune system genes defined in other species. The genes are best described in regional units. Among these, the best known is the polymorphic major histocompatibility complex-B (MHC-B) region containing genes for classical peptide antigen presentation. Nearby MHC-B is a small region containing two CD1 genes, which encode molecules known to bind lipid antigens and which will likely be found in chickens to present lipids to specialized T cells, as occurs with CD1 molecules in other species. Another region is the MHC-Y region, separated from MHC-B by an intervening region of tandem repeats. Like MHC-B, MHC-Y is polymorphic. It contains specialized class I and class II genes and c-type lectin-like genes. Yet another region, separated from MHC-Y by the single nucleolar organizing region (NOR) in the chicken genome, contains olfactory receptor genes and scavenger receptor genes, which are also thought to contribute to immunity. The structure, distribution, linkages and patterns of polymorphism in these regions, suggest GGA 16 evolves as a microchromosome devoted to immune defense. Many GGA 16 genes are polymorphic and polygenic. At the moment most disease associations are at the haplotype level. Roles of individual MHC genes in disease resistance are documented in only a very few instances. Provided suitable experimental stocks persist, the availability of increasingly detailed maps of GGA 16 genes combined with new means for detecting genetic variability will lead to investigations defining the contributions of individual loci and more applications for immunogenetics in breeding healthy poultry.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>non-coding RNA Long Non-Coding RNAs Regulating Immunity in Insects</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent advances in modern technology have led to the understanding that not all genetic information is coded into protein and that the genomes of each and every organism including insects produce non-coding RNAs that can control different biological processes. Among RNAs identified in the last decade, long non-coding RNAs (lncRNAs) represent a repertoire of a hidden layer of internal signals that can regulate gene expression in physiological, pathological, and immunological processes. Evidence shows the importance of lncRNAs in the regulation of host-pathogen interactions. In this review, an attempt has been made to view the role of lncRNAs regulating immune responses in insects.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Roles of aminoacyl-tRNA synthetases in immune regulation and immune diseases</h3>
<p style="text-align: justify; text-justify: inter-word;">Aminoacyl-tRNA synthetases (ARSs) play a vital role in protein synthesis by linking amino acids to their cognate transfer RNAs (tRNAs). This typical function has been well recognized over the past few decades. However, accumulating evidence reveals that ARSs are involved in a wide range of physiological and pathological processes apart from translation. Strikingly, certain ARSs are closely related to different types of immune responses. In this review, we address the infection and immune responses induced by pathogen ARSs, as well as the potential anti-infective compounds that target pathogen ARSs. Meanwhile, we describe the functional mechanisms of ARSs in the development of immune cells. In addition, we focus on the roles of ARSs in certain immune diseases, such as autoimmune diseases, infectious diseases, and tumor immunity. Although our knowledge of ARSs in the immunological context is still in its infancy, research in this field may provide new ideas for the treatment of immunerelated diseases.</p>
<p align="right"><i>score: 147</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification and characterisation of the CD40-ligand of Sigmodon hispidus</h3>
<p style="text-align: justify; text-justify: inter-word;">Cotton rats are an important animal model to study infectious diseases. They have demonstrated higher susceptibility to a wider variety of human pathogens than other rodents and are also the animal model of choice for pre-clinical evaluations of some vaccine candidates. However, the genome of cotton rats remains to be fully sequenced, with much fewer genes cloned and characterised compared to other rodent species. Here we report the cloning and characterization of CD40 ligand, whose human and murine counterparts are known to be expressed on a range of cell types including activated T cells and B cells, dendritic cells, granulocytes, macrophages and platelets and exerts a broad array of immune responses. The cDNA for cotton rat CD40L we isolated is comprised of 1104 nucleotides with an open reading frame (ORF) of 783bp coding for a 260 amino acid protein. The recombinant cotton rat CD40L protein was recognized by an antibody against mouse CD40L. Moreover, it demonstrated functional activities on immature bone marrow dendritic cells by upregulating surface maturation markers (CD40, CD54, CD80, and CD86), and increasing IL-6 gene and protein expression. The availability of CD40L gene identity could greatly facilitate mechanistic research on pathogen-induced-immunopathogenesis and vaccine-elicited immune responses.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Natural immune response to Plasmodium vivax alpha-helical coiled coil protein motifs and its association with the risk of P. vivax malaria</h3>
<p style="text-align: justify; text-justify: inter-word;">Protein α-helical coiled coil structures are known to induce antibodies able to block critical functions in different pathogens. In a previous study, a total of 50 proteins of Plasmodium vivax erythrocytic asexual stages containing α-helical coiled coil structural motifs were identified in silico, and the corresponding peptides were chemically synthesized. A total of 43 peptides were recognized by naturally acquired antibodies in plasma samples from both Papua New Guinea (PNG) and Colombian adult donors. In this study, the association between IgG antibodies to these peptides and clinical immunity was further explored by measuring total IgG antibody levels to 24 peptides in baseline samples from a longitudinal study of children aged 1-3 years (n = 164) followed for 16 months. Samples were reactive to all peptides tested. Eight peptides were recognized by >50% of individuals, whereas only one peptide had < 20% reactivity. Children infected at baseline were seropositive to 23/24 peptides. No significant association was observed between antibody titers and age or molecular force of infection, suggesting that antibody levels had already reached an equilibrium. There was a strong association between antibody levels to all peptides and protection against P. vivax clinical episodes during the 16 months follow-up. These results suggest that the selected coiled coil antigens might be good markers of both exposure and acquired immunity to P. vivax malaria, and further preclinical investigation should be performed to determine their potential as P. vivax vaccine antigens.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Gene Regulatory Network Inference of Immunoresponsive Gene 1 (IRG1) Identifies Interferon Regulatory Factor 1 (IRF1) as Its Transcriptional Regulator in Mammalian Macrophages</h3>
<p style="text-align: justify; text-justify: inter-word;">Immunoresponsive gene 1 (IRG1) is one of the highest induced genes in macrophages under pro-inflammatory conditions. Its function has been recently described: it codes for immune-responsive gene 1 protein/cis-aconitic acid decarboxylase (IRG1/CAD), an enzyme catalysing the production of itaconic acid from cis-aconitic acid, a tricarboxylic acid (TCA) cycle intermediate. Itaconic acid possesses specific antimicrobial properties inhibiting isocitrate lyase, the first enzyme of the glyoxylate shunt, an anaplerotic pathway that bypasses the TCA cycle and enables bacteria to survive on limited carbon conditions. To elucidate the mechanisms underlying itaconic acid production through IRG1 induction in macrophages, we examined the transcriptional regulation of IRG1. To this end, we studied IRG1 expression in human immune cells under different inflammatory stimuli, such as TNFα and IFNγ, in addition to lipopolysaccharides. Under these conditions, as previously shown in mouse macrophages, IRG1/CAD accumulates in mitochondria. Furthermore, using literature information and transcription factor prediction models, we re-constructed raw gene regulatory networks (GRNs) for IRG1 in mouse and human macrophages. We further implemented a contextualization algorithm that relies on genome-wide gene expression data to infer putative cell type-specific gene regulatory interactions in mouse and human macrophages, which allowed us to predict potential transcriptional regulators of IRG1. Among the computationally identified regulators, siRNA-mediated gene silencing of interferon regulatory factor 1 (IRF1) in macrophages significantly decreased the expression of IRG1/CAD at the gene and protein level, which correlated with a reduced production of itaconic acid. Using a synergistic approach of both computational and experimental methods, PLOS ONE | we here shed more light on the transcriptional machinery of IRG1 expression and could pave the way to therapeutic approaches targeting itaconic acid levels.</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk factors associated with exposure to bovine respiratory disease pathogens during the peri-weaning period in dairy bull calves</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Bovine respiratory disease (BRD) remains among the leading causes of death of cattle internationally. The objective of this study was to identify risk factors associated with exposure to BRD pathogens during the peri-weaning period (day (d)-14 to d 14 relative to weaning at 0) in dairy bull calves using serological responses to these pathogens as surrogate markers of exposure. Clinically normal Holstein-Friesian and Jersey breed bull calves (n = 72) were group housed in 4 pens using a factorial design with calves of different breeds and planes of nutrition in each pen. Intrinsic, management and clinical data were collected during the pre-weaning (d − 56 to d − 14) period. Calves were gradually weaned over 14 days (d − 14 to d 0). Serological analysis for antibodies against key BRD pathogens (BRSV, BPI3V, BHV-1, BHV-4, BCoV, BVDV and H. somni) was undertaken at d − 14 and d 14. Linear regression models (for BVDV, BPI3V, BHV-1, BHV-4, BCoV and H. somni) and a single mixed effect random variable model (for BRSV) were used to identify risk factors for changes in antibody levels to these pathogens. Results: BRSV was the only pathogen which demonstrated clustering by pen. Jersey calves experienced significantly lower changes in BVDV S/P than Holstein-Friesian calves. Animals with a high maximum respiratory score (≥8) recorded significant increases in H. somni S/P during the peri-weaning period when compared to those with respiratory scores of ≤3. Haptoglobin levels of between 1.32 and 1.60 mg/ml at d − 14 were significantly associated with decreases in BHV-1 S/N during the peri-weaning period. Higher BVDV S/P ratios at d − 14 were significantly correlated with increased changes in serological responses to BHV-4 over the peri-weaning period. Conclusions: Haptoglobin may have potential as a predictor of exposure to BHV-1. BRSV would appear to play a more significant role at the 'group' rather than 'individual animal' level. The significant associations between the pre-weaning levels of antibodies to certain BRD pathogens and changes in the levels of antibodies to the various pathogens during the peri-weaning period may reflect a cohort of possibly genetically linked 'better responders' among the study population.</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunity Traits in Pigs: Substantial Genetic Variation and Limited Covariation</h3>
<p style="text-align: justify; text-justify: inter-word;">Methodology/Principal Findings: Fifty four ITs were studied on 443 Large White pigs vaccinated against Mycoplasma hyopneumoniae and analyzed by combining a principal component analysis (PCA) and genetic parameter estimation. ITs include specific and non specific antibodies, seric inflammatory proteins, cell subsets by hemogram and flow cytometry, ex vivo production of cytokines (IFNa, TNFa, IL6, IL8, IL12, IFNc, IL2, IL4, IL10), phagocytosis and lymphocyte proliferation. While six ITs had heritabilities that were weak or not significantly different from zero, 18 and 30 ITs had moderate (0.1,h2#0.4) or high (h2.0.4) heritability values, respectively. Phenotypic and genetic correlations between ITs were weak except for a few traits that mostly include cell subsets. PCA revealed no cluster of innate or adaptive ITs.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>Composition and Function of Haemolymphatic Tissues in the European Common Shrew</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Studies of wild animals responding to their native parasites are essential if we are to understand how the immune system functions in the natural environment. While immune defence may bring increased survival, this may come at a resource cost to other physiological traits, including reproduction. Here, we tested the hypothesis that wild common shrews (Sorex araneus), which produce large numbers of offspring during the one breeding season of their short life span, forgo investment in immunity and immune system maintenance, as increased longevity is unlikely to bring further opportunities for mating. In particular, we predicted that adult shrews, with shorter expected lifespans, would not respond as effectively as young animals to infection.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>Deciphering desirable immune responses from disease models with resistant and susceptible chickens</h3>
<p style="text-align: justify; text-justify: inter-word;">Coccidiosis and necrotic enteritis (NE) are among the most significant diseases affecting the poultry industry. These diseases have become more prominent in the wake of policies to reduce the use of antibiotics in animal production. This has led to more research focused on better understanding the immune system and its responses to pathogen challenge, and thus developing informed strategies to exploit immune responses that can support enhanced disease resistance and growth performance. Some chicken breeds and lines show greater resistance or susceptibility to various diseases, and thus these birds maybe able to shed light on immune processes or pathways that contribute to the more resistant/susceptible state. This review attempts to identify potentially important genes that show some consistency in (relative) up or downregulation in key tissues between the resistant and susceptible chickens. For coccidiosis and NE, relative downregulation of IL-10 and (slightly less consistently) upregulation of IFNγ appear to be features of more resistant birds. Data for IFN-α, IL-12, and IL-17D are currently less consistent. Gene expression data from NE studies have identified some potentially interesting, perhaps less well understood, immune-related genes (e.g., TCF12, BCL2, IRF2, TRAF3, TAB3, etc.,) that maybe associated with the resistant and/or susceptible phenotype. Salmonella and Campylobacter are important foodborne pathogens harbored by the chicken intestinal tract, while infectious bursal disease and infectious bronchitis are also important viral diseases of poultry. We, therefore, consider whether there are consistent features from resistant/susceptible disease models with these pathogens that relate to findings from the coccidiosis and NE studies. It is not anticipated that ideal immune responses to these pathogens will be identical but rather that consistent elements maybe identified that could help inform breeding or alternative strategies to support general disease resistance and enhanced (and efficient) flock productivity.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Toll-Like Receptor Responses to Peste des petits ruminants Virus in Goats and Water Buffalo</h3>
<p style="text-align: justify; text-justify: inter-word;">Ovine rinderpest or goat plague is an economically important and contagious viral disease of sheep and goats, caused by the Peste des petits ruminants virus (PPRV). Differences in susceptibility to goat plague among different breeds and water buffalo exist. The host innate immune system discriminates between pathogen associated molecular patterns and self antigens through surveillance receptors known as Toll like receptors (TLR). We investigated the role of TLR and cytokines in differential susceptibility of goat breeds and water buffalo to PPRV. We examined the replication of PPRV in peripheral blood mononuclear cells (PBMC) of Indian domestic goats and water buffalo and demonstrated that the levels of TLR3 and TLR7 and downstream signalling molecules correlation with susceptibility vs resistance. Naturally susceptible goat breeds, Barbari and Tellichery, had dampened innate immune responses to PPRV and increased viral loads with lower basal expression levels of TLR 3/7. Upon stimulation of PBMC with synthetic TLR3 and TLR7 agonists or PPRV, the levels of proinflammatory cytokines were found to be significantly higher while immunosuppressive interleukin (IL) 10 levels were lower in PPRV resistant Kanni and Salem Black breeds and water buffalo at transcriptional level, correlating with reduced viralloads in infected PBMC. Water buffalo produced higher levels of interferon (IFN) a in comparison with goats at transcriptional and translational levels. Pre-treatment of Vero cells with human IFNa resulted in reduction of PPRV replication, confirming the role of IFNa in limiting PPRV replication. Treatment with IRS66, a TLR7 antagonist, resulted in the reduction of IFNa levels, with increased PPRV replication confirming the role of TLR7. Single nucleotide polymorphism analysis of TLR7 of these goat breeds did not show any marked nucleotide differences that might account for susceptibility vs resistance to PPRV. Analyzing other host genetic factors might provide further insights on susceptibility to PPRV and genetic polymorphisms in the host.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h3>
<p style="text-align: justify; text-justify: inter-word;">The outbreak of the 2019 Novel Coronavirus (2019-nCoV) has rapidly spread from Wuhan, China to multiple countries, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of 2019-nCoV is lacking, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development. In this study, we predicted the potential of all the 2019-nCoV viral proteins to induce class I and II MHC presentation and form linear antibody epitopes. We showed that the enrichment for T cell and B cell epitopes is not uniform on the viral genome, with several focused regions that generate abundant epitopes and may be more targetable. We showed that genetic variations in 2019-nCoV, though fewer for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus, which should be considered in the development of therapies. We create an online database to broadly share our research outcome. Overall, we present an immunological resource for 2019-nCoV that could significantly promote both therapeutic development and mechanistic research.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>High-Throughput Sequencing-Based immune Repertoire Study during infectious Disease</h3>
<p style="text-align: justify; text-justify: inter-word;">The selectivity of the adaptive immune response is based on the enormous diversity of T and B cell antigen-specific receptors. The immune repertoire, the collection of T and B cells with functional diversity in the circulatory system at any given time, is dynamic and reflects the essence of immune selectivity. In this article, we review the recent advances in immune repertoire study of infectious diseases, which were achieved by traditional techniques and high-throughput sequencing (HTS) techniques. HTS techniques enable the determination of complementary regions of lymphocyte receptors with unprecedented efficiency and scale. This progress in methodology enhances the understanding of immunologic changes during pathogen challenge and also provides a basis for further development of novel diagnostic markers, immunotherapies, and vaccines.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Transcriptomic analysis of immune response to bacterial lipopolysaccharide in zebra finch (Taeniopygia guttata)</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Despite the convergence of rapid technological advances in genomics and the maturing field of ecoimmunology, our understanding of the genes that regulate immunity in wild populations is still nascent. Previous work to assess immune function has relied upon relatively crude measures of immunocompetence. However, with next-generation RNA-sequencing, it is now possible to create a profile of gene expression in response to an immune challenge. In this study, captive zebra finch (Taeniopygia guttata; adult males) were challenged with bacterial lipopolysaccharide (LPS) or vehicle to stimulate the innate immune system. 2 hours after injection, birds were euthanized and hypothalami, spleen, and red blood cells (RBCs) were collected. Taking advantage of the fully sequenced genome of zebra finch, total RNA was isolated, sequenced, and partially annotated in these tissue/cells. Results: In hypothalamus, there were 707 significantly upregulated transcripts, as well as 564 and 144 in the spleen and RBCs, respectively, relative to controls. Also, 155 transcripts in the hypothalamus, 606 in the spleen, and 61 in the RBCs were significantly downregulated. More specifically, a number of immunity-related transcripts (e.g., IL-1β, RSAD2, SOCS3) were upregulated among tissues/cells. Additionally, transcripts involved in metabolic processes (APOD, LRAT, RBP4) were downregulated. Conclusions: These results suggest a potential trade-off in expression of genes that regulate immunity and metabolism in birds challenged with LPS. This finding is consistent with a hypothermic response to LPS treatment in small birds. Unlike mammals, birds have nucleated RBCs, and these results support a novel transcriptomic response of avian RBCs to immune challenge.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Mechanisms of protective immune responses induced by the Plasmodium falciparum circumsporozoite protein-based, self-assembling protein nanoparticle vaccine Mechanisms of protective immune responses induced by the Plasmodium falciparum circumsporozoite protein-based, self-assembling protein nanoparticle vaccine</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: A lack of defined correlates of immunity for malaria, combined with the inability to induce long-lived sterile immune responses in a human host, demonstrate a need for improved understanding of potentially protective immune mechanisms for enhanced vaccine efficacy. Protective sterile immunity (>90%) against the Plasmodium falciparum circumsporozoite protein (CSP) has been achieved using a transgenically modified Plasmodium berghei sporozoite (Tg-Pb/PfCSP) and a self-assembling protein nanoparticle (SAPN) vaccine presenting CSP epitopes (PfCSP-SAPN). Here, several possible mechanisms involved in the independently protective humoral and cellular responses induced following SAPN immunization are described. Methods: Inbred mice were vaccinated with PfCSP-SAPN in PBS. Serum antibodies were harvested and effects on P. falciparum sporozoites mobility and integrity were examined using phase contrast microscopy. The functionality of SAPN-induced antibodies on inhibition of sporozoite invasion and growth within primary human hepatocytes was also examined. The internal processing of SAPN by bone marrow-derived dendritic cells (BMDDC), using organelle-specific, fluorescent-tagged antibody or gold-encapsulated SAPN, was observed using confocal or electron microscopy, respectively. Results: The results of this work demonstrate that PfCSP-SAPN induces epitope-specific antibody titers, predominantly of the Th2 isotype IgG1, and that serum antibodies from PfCSP-SAPN-immunized mice appear to target P. falciparum sporozoites via the classical pathway of complement. This results in sporozoite death as indicated by cessation of motility and the circumsporozoite precipitation reaction. Moreover, PfCSP-SAPN-induced antibodies are able to inhibit wild-type P. falciparum sporozoite invasion and growth within cultured primary human hepatocytes. In addition, the observation that PfCSP-SAPN are processed (and presented) to the immune system by dendritic cells in a slow and continuous fashion via transporter associated with antigen processing (TAP) recruitment to the early endosome (EE), and have partially delayed processing through the endoplasmic reticulum, has the potential to induce the long-lived, effector memory CD8 + T-cells as described previously.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Genomics Understanding Haemophilus parasuis infection in porcine spleen through a transcriptomics approach</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: A total of 931 differentially expressed (DE) transcripts were identified in the porcine spleen 7 days after HPS infection; of these, 92 unique genes showed differential expression patterns based on analysis using BLASTX and Gene Ontology. The DE genes involved in the immune response included genes for inflammasomes (RETN, S100A8, S100A9, S100A12), adhesion molecules (CLDN3, CSPG2, CD44, LGALS8), transcription factors (ZBTB16, SLC39A14, CEBPD, CEBPB), acute-phase proteins and complement (SAA1, LTF, HP, C3), differentiation genes for epithelial cells and keratinocytes (TGM1, MS4A8B, CSTA), and genes related to antigen processing and presentation (HLA-B, HLA-DRB1). Further immunostimulation analyses indicated that mRNA levels of S100A8, S100A9, and S100A12 in porcine PK-15 cells increased within 48 h and were sustained after administration of lipopolysaccharide (LPS) and Poly(I:C) respectively. In addition, mapping of DE genes to porcine health traits QTL regions showed that 70 genes were distributed in 7 different known porcine QTL regions. Finally, 10 DE genes were validated by quantitative PCR.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of the immune response to vaccination with pigeon circovirus recombinant capsid protein (PiCV rCP) in pigeons uninfected and subclinically infected with PiCV</h3>
<p style="text-align: justify; text-justify: inter-word;">Infections with immunosuppressive pigeon circovirus (PiCV) pose the most severe health problem to the global pigeon breeding. The vaccination with immunogenic PiCV recombinant capsid protein (PiCV rCP) is a potential tool for disease control. Because of the high prevalence of PiCV asymptomatic infections, the subclinically infected pigeons will be vaccinated in practice. The aim of this study was to answer a question if vaccination of asymptomatic, infected with PiCV pigeons induces a similar immune response to PiCV rCP as in uninfected birds. One hundred and twenty 6-week-old carrier pigeons were divided into 4 groups (2 groups of naturally infected and uninfected with PiCV individuals). Birds from groups V and V1 were vaccinated twice with PiCV rCP mixed with an adjuvant, whereas pigeons from groups C and C1 were immunized with an adjuvant only. The expression of genes encoding IFN-γ, CD4, and CD8 T lymphocyte receptors; the number of anti-PiCV rCP IgY-secreting B cells (SBC) and anti-PiCV rCP IgY were evaluated 2, 21, 39 and 46 days post vaccination (dpv). Study results showed that the expression of CD8 and IFN-γ genes was higher in both groups of infected pigeons than in the uninfected birds, irrespective of vaccination. In the uninfected birds, the expression of these genes was insignificantly higher in the vaccinated pigeons. The anti-PiCV rCP IgY-SBC were detected on 2 and 23 dpv and seroconversion was noted on 23 and 39 dpv in V and V1 groups, respectively. In the light of the results obtained, it could be concluded that pigeon circovirus recombinant capsid protein elicits the immune response in both naturally infected and uninfected pigeons, but its rate varies depending on PiCV infectious status. The infection with PiCV masks the potential cellular immune response to the vaccination with PiCV rCP and leads to the suppression of humoral immunity.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Associations between Single Nucleotide Polymorphisms in Cellular Viral Receptors and Attachment Factor- Related Genes and Humoral Immunity to Rubella Vaccination</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: Our analysis revealed evidence for multiple associations between genetic variants in the PVR, PVRL2, CD209/DC-SIGN, RARB, MOG, IL6 and other immune function-related genes and rubella-specific neutralizing antibodies after vaccination (meta p-value ,0.05).</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Transcriptomics of In Vitro Immune-Stimulated Hemocytes from the Manila Clam Ruditapes philippinarum Using High-Throughput Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: This study represents the first transcriptome analysis using 454-pyrosequencing conducted on R. philippinarum focused on its immune system. Our results will provide a rich source of data to discover and identify new genes, which will serve as a basis for microarray construction and the study of gene expression as well as for the identification of genetic markers. The discovery of new immune sequences was very productive and resulted in a large variety of contigs that may play a role in the defense mechanisms of Ruditapes philippinarum.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /></body>