
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: incubation period of the coronavirus</h2><br /></div><br /><br /><br /><div class="card"><h3>Estimate the incubation period of coronavirus 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Motivation: Wuhan pneumonia is an acute infectious disease caused by the 2019 novel coronavirus (COVID-19). It is being treated as a Class A infectious disease though it was classified as Class B according to the Infectious Disease Prevention Act of China. Accurate estimation of the incubation period of the coronavirus is essential to the prevention and control. However, it remains unclear about its exact incubation period though it is believed that symptoms of COVID-19 can appear in as few as 2 days or as long as 14 or even more after exposure. The accurate incubation period calculation requires original chain-of-infection data that may not be fully available in the Wuhan regions. In this study, we aim to accurately calculate the incubation period of COVID-19 by taking advantage of the chain-of-infection data, which is well-documented and epidemiologically informative, outside the Wuhan regions. Methods: We acquired and collected officially reported COVID-19 data from 10 regions in China except for Hubei province. To achieve the accurate calculation of the incubation period, we only involved the officially confirmed cases with a clear history of exposure and time of onset. We excluded those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time. We proposed a Monte Caro simulation approach to estimate the incubation of COVID-19 as well as employed nonparametric ways. We also employed manifold learning and related statistical analysis to decipher the incubation relationships between different age/gender groups. Result: The incubation period of COVID-19 did not follow general incubation distributions such as lognormal, Weibull, and Gamma distributions. We estimated that the mean and median of its incubation were 5.84 and 5.0 days via bootstrap and proposed Monte Carlo simulations.  We found that the incubation periods of the groups with age&gt;=40 years and age&lt;40 years demonstrated a statistically significant difference. The former group had a longer incubation period and a larger variance than the latter. It further suggested that different quarantine time should be applied to the groups for their different incubation periods. Our machine learning analysis also showed that the two groups were linearly separable. incubation of COVID-19 along with previous statistical analysis. Our results further indicated that the incubation difference between males and females did not demonstrate a statistical significance.</jats:p></p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of incubation period distribution of human infections with MERS-CoV in South Korea and Saudi Arabia</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Oct 24</span></i><p style="text-align: justify; text-justify: inter-word;">The incubation period is an important epidemiologic distribution, it is often incorporated in case definitions, used to determine appropriate quarantine periods, and is an input to mathematical modeling studies. Middle East Respiratory Syndrome coronavirus (MERS) is an emerging infectious disease in the Arabian Peninsula. There was a large outbreak of MERS in South Korea in 2015. We examined the incubation period distribution of MERS coronavirus infection for cases in South Korea and in Saudi Arabia. Using parametric and nonparametric methods, we estimated a mean incubation period of 6.9 days (95% credibility interval: 6.3–7.5) for cases in South Korea and 5.0 days (95% credibility interval: 4.0–6.6) among cases in Saudi Arabia. In a log-linear regression model, the mean incubation period was 1.42 times longer (95% credibility interval: 1.18–1.71) among cases in South Korea compared to Saudi Arabia. The variation that we identified in the incubation period distribution between locations could be associated with differences in ascertainment or reporting of exposure dates and illness onset dates, differences in the source or mode of infection, or environmental differences.</p>
<p align="right"><i>score: 118</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background:  The current outbreak of coronavirus disease 2019 (COVID-19) has quickly spread across countries and become a global crisis. However, one of the most important clinical characteristics in epidemiology, the distribution of the incubation period, remains unclear. Different estimates of the incubation period of COVID-19 were reported in recent published studies, but all have their own limitations. In this study, we propose a novel low-cost and accurate method to estimate the incubation distribution.   Methods: We have conducted a cross-sectional and forward follow-up study by identifying those asymptomatic individuals at their time of departure from Wuhan and then following them until their symptoms developed. The renewal process is hence adopted by considering the incubation period as a renewal and the duration between departure and symptom onset as a forward recurrence time. Under mild assumptions, the observations of selected forward times can be used to consistently estimate the parameters in the distribution of the incubation period. Such a method enhances the accuracy of estimation by reducing recall bias and utilizing the abundant and readily available forward time data.   Findings:  The estimated distribution of forward time fits the observations in the collected data well. The estimated median of incubation period is 8.13 days (95% confidence interval [CI]: 7.37-8.91), the mean is 8.62 days (95% CI: 8.02-9.28), the 90th percentile is 14.65 days (95% CI: 14.00-15.26), and the 99th percentile is 20.59 days (95% CI: 19.47, 21.62). Compared with results in other studies, the incubation period estimated in this study is longer.  Interpretation: Based on the estimated incubation distribution in this study, about 10% of patients with COVID-19 would not develop symptoms until 14 days after infection. Further study of the incubation distribution is warranted to directly estimate the proportion with long incubation periods.</jats:p></p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the distribution of the incubation period of 2019 novel coronavirus (COVID-19) infection between travelers to Hubei, China and non-travelers</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objectives: Amid the continuing spread of the novel coronavirus (COVID-19), the incubation period of COVID-19 should be regularly re-assessed as more information is available upon the increase in reported cases. The present work estimated the distribution of incubation periods of patients infected in and outside Hubei province of China.  Methods: Clinical data were collected from the individual cases reported by the media as they were not fully available on the official pages of the Chinese health authorities. MLE was used to estimate the distributions of the incubation period. Results: It was found that the incubation period of patients with no travel history to Hubei was longer and more volatile. Conclusion: It is recommended that the duration of quarantine should be extended to at least 3 weeks.</jats:p></p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-28</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</jats:p></p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2&ndash;14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3&ndash;4 days without truncation and at 5&ndash;9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period Duration and Severity of Clinical Disease Following Severe Acute Respiratory Syndrome Coronavirus Infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Few previous studies have investigated the association between the severity of an infectious disease and the length of incubation period. METHODS: We estimated the association between the length of the incubation period and the severity of infection with the severe acute respiratory syndrome (SARS) coronavirus, using data from the epidemic in 2003 in Hong Kong. RESULTS: We estimated the incubation period of SARS based on a subset of patients with available data on exposure periods and a separate subset of patients in a putative common source outbreak, and we found significant associations between shorter incubation period and greater severity in both groups after adjusting for potential confounders. CONCLUSIONS: Our findings suggest that patients with a shorter incubation period proceeded to have more severe disease. Further studies are needed to investigate potential biological mechanisms for this association.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>The incubation period of 2019-nCoV infections among travellers from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-28</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Currently, a novel coronavirus 2019-nCoV causes an outbreak of viral pneumonia in Wuhan, China. Little is known about its epidemiological characteristics. Using the travel history and symptom onset of 88 confirmed cases that were detected outside Wuhan, we estimate the mean incubation period to be 6.4 (5.6 - 7.7, 95% CI) days, ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile). These values help to inform case definitions for 2019-nCoV and appropriate durations for quarantine.</jats:p></p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Does SARS-CoV-2 has a longer incubation period than SARS and MERS?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of a novel coronavirus (SARS-CoV-2) since Dec 2019 in Wuhan, the major transportation hub in central China, became an emergency of major international concern. While several etiological studies have begun to reveal the specific biological features of this virus, the epidemic characteristics need to be elucidated. Notably, a long incubation time was reported to be associated with SARS-CoV-2 infection, leading to adjustments in screening and control policies. To avoid the risk of virus spread, all potentially exposed subjects are required to be isolated for 14 days, which is the longest predicted incubation time. However, based on our analysis of a larger dataset available so far, we find there is no observable difference between the incubation time for SARS-CoV-2, SARS and MERS, highlighting the need for larger and well annotated datasets. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiple Contact Dates and SARS Incubation Periods</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Many severe acute respiratory syndrome (SARS) patients have multiple possible incubation periods due to multiple contact dates. Multiple contact dates cannot be used in standard statistical analytic techniques, however. I present a simple spreadsheet-based method that uses multiple contact dates to calculate the possible incubation periods of SARS.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation periods of acute respiratory viral infections: a systematic review</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Knowledge of the incubation period is essential in the investigation and control of infectious disease, but statements of incubation period are often poorly referenced, inconsistent, or based on limited data. In a systematic review of the literature on nine respiratory viral infections of public-health importance, we identified 436 articles with statements of incubation period and 38 with data for pooled analysis. We fitted a log-normal distribution to pooled data and found the median incubation period to be 5·6 days (95% CI 4·8–6·3) for adenovirus, 3·2 days (95% CI 2·8–3·7) for human coronavirus, 4·0 days (95% CI 3·6–4·4) for severe acute respiratory syndrome coronavirus, 1·4 days (95% CI 1·3–1·5) for influenza A, 0·6 days (95% CI 0·5–0·6) for influenza B, 12·5 days (95% CI 11·8–13·3) for measles, 2·6 days (95% CI 2·1–3·1) for parainfluenza, 4·4 days (95% CI 3·9–4·9) for respiratory syncytial virus, and 1·9 days (95% CI 1·4–2·4) for rhinovirus. When using the incubation period, it is important to consider its full distribution: the right tail for quarantine policy, the central regions for likely times and sources of infection, and the full distribution for models used in pandemic planning. Our estimates combine published data to give the detail necessary for these and other applications.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic Parameters of the Middle East Respiratory Syndrome Outbreak in Korea, 2015</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun 30</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Epidemiologic parameters are important in planning infection control policies during the outbreak of emerging infections. Korea experienced an outbreak of Middle East Respiratory Syndrome coronavirus (MERS-CoV) infection in 2015, which was characterized by superspreading events in healthcare settings. We aimed to estimate the epidemiologic parameters over time during the outbreak to assess the effectiveness of countermeasures. MATERIALS AND METHODS: Publicly available data pertaining to the MERS outbreak in Korea were collected. We estimated the incubation periods of 162 cases whose sources of exposure were identified and the temporal trend was evaluated. Factors influencing incubation duration were analyzed. The generational reproduction number (R(g)) and case reproduction number (R(c)) were estimated over time. RESULTS: The estimated median incubation period was 7.4 days (95% CI, 6.9-8.0). Median incubation periods tended to be longer over time as the disease generation progressed: 6.16 days (95% CI, 5.38-6.97), 7.68 days (95% CI, 7.04-8.44), and 7.95 days (95% CI, 6.25-9.88) in the first, second, and third generations, respectively. The number of days of illness in the source cases at the time of exposure inversely correlated with the incubation periods in the receiving cases (HR 0.91 [95% CI, 0.84-0.99] per one illness day increase; P=0.026). This relationship was consistent (HR 0.83 [95% CI, 0.74-0.93] per one illness day increase) in the multivariable analysis incorporating clinical characteristics, the order of generation, and a link to superspreaders. Because the third generation cases were exposed to their source cases in the early stage (median one day) compared to the second generation cases (median 6 days), the temporal trend of incubation periods appears to be influenced by early isolation of symptomatic cases and reduction of potential exposure to source cases in the later stage. R(g) declined rapidly from 28 to 0.23 in two generations. R(c) dropped below the epidemic threshold at one on May 31, 2015, which approximately coincided with the initiation of the stringent countermeasures. CONCLUSIONS: Despite the initial delay, the stringent countermeasures targeted towards second generation cases appeared to effectively contain the MERS outbreak in Korea as suggested by the decline of R(c) shortly after implementation. Except for superspreading events, the transmission potential for MERS-CoV seems to be low. Further research should be focused on characterizing superspreaders in comparison to non-transmitting cases with regard to environmental, behavioral, and virologic and host genetic factors in order to better prepare for future outbreaks of MERS-CoV.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic Parameters of the Middle East Respiratory Syndrome Outbreak in Korea, 2015</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun 30</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Epidemiologic parameters are important in planning infection control policies during the outbreak of emerging infections. Korea experienced an outbreak of Middle East Respiratory Syndrome coronavirus (MERS-CoV) infection in 2015, which was characterized by superspreading events in healthcare settings. We aimed to estimate the epidemiologic parameters over time during the outbreak to assess the effectiveness of countermeasures. MATERIALS AND METHODS: Publicly available data pertaining to the MERS outbreak in Korea were collected. We estimated the incubation periods of 162 cases whose sources of exposure were identified and the temporal trend was evaluated. Factors influencing incubation duration were analyzed. The generational reproduction number (R(g)) and case reproduction number (R(c)) were estimated over time. RESULTS: The estimated median incubation period was 7.4 days (95% CI, 6.9-8.0). Median incubation periods tended to be longer over time as the disease generation progressed: 6.16 days (95% CI, 5.38-6.97), 7.68 days (95% CI, 7.04-8.44), and 7.95 days (95% CI, 6.25-9.88) in the first, second, and third generations, respectively. The number of days of illness in the source cases at the time of exposure inversely correlated with the incubation periods in the receiving cases (HR 0.91 [95% CI, 0.84-0.99] per one illness day increase; P=0.026). This relationship was consistent (HR 0.83 [95% CI, 0.74-0.93] per one illness day increase) in the multivariable analysis incorporating clinical characteristics, the order of generation, and a link to superspreaders. Because the third generation cases were exposed to their source cases in the early stage (median one day) compared to the second generation cases (median 6 days), the temporal trend of incubation periods appears to be influenced by early isolation of symptomatic cases and reduction of potential exposure to source cases in the later stage. R(g) declined rapidly from 28 to 0.23 in two generations. R(c) dropped below the epidemic threshold at one on May 31, 2015, which approximately coincided with the initiation of the stringent countermeasures. CONCLUSIONS: Despite the initial delay, the stringent countermeasures targeted towards second generation cases appeared to effectively contain the MERS outbreak in Korea as suggested by the decline of R(c) shortly after implementation. Except for superspreading events, the transmission potential for MERS-CoV seems to be low. Further research should be focused on characterizing superspreaders in comparison to non-transmitting cases with regard to environmental, behavioral, and virologic and host genetic factors in order to better prepare for future outbreaks of MERS-CoV.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic Characteristics of COVID-19 in Guizhou, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>At the end of 2019, a coronavirus disease 2019 (COVID-19) outbroke in Wuhan, China, and spread to Guizhou province on January of 2020. To acquire the epidemiologic characteristics of COVID-19 in Guizhou, China, we collected data on 162 laboratory-confirmed cases related to COVID-19. We described the demographic characteristics of the cases and estimated the incubation period, serial interval and basic reproduction number. We also presented two representative case studies in Guizhou province -- Case Study 1 was an example of asymptomatic carrier; and Case Study 2 was an example of a large and complex infection chain that involved four different districts spanning three provinces and eight families. With an estimation of 8 days incubation period and 6 days serial interval, our results indicate that there may exist infectiousness during the incubation period for 2019-nCoV. This increases the difficulty of screening or identifying cases related to COVID-19.</jats:p></p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Serial interval of novel coronavirus (COVID-19) infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n = 28) and a subset of pairs with highest certainty in reporting (n = 18). In addition, we adjust for right truncation of the data as the epidemic is still in its growth phase. Results Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions The serial interval of COVID-19 is close to or shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation period as part of the case definition of severe respiratory illness caused by a novel coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Due to non-specific symptoms following acute respiratory viral infections, it is difficult for many countries without on-going transmission of a novel coronavirus to rule out other possibilities including influenza in advance of isolating imported febrile individuals with a possible exposure history. The incubation period helps differential diagnosis and up to two days is suggestive of influenza. It is worth accounting for the incubation period as part of the case definition of novel coronavirus infection.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Study on assessing early epidemiological parameters of coronavirus disease epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: To study the early dynamics of the epidemic of coronavirus disease (COVID-19) in China from 15 to 31 January, 2020, and estimate the corresponding epidemiological parameters (incubation period, generation interval and basic reproduction number) of the epidemic. Methods: By means of Weibull, Gamma and Lognormal distributions methods, we estimated the probability distribution of the incubation period and generation interval data obtained from the reported COVID-19 cases. Moreover, the AIC criterion was used to determine the optimal distribution. Considering the epidemic is ongoing, the exponential growth model was used to fit the incidence data of COVID-19 from 10 to 31 January, 2020, and exponential growth method, maximum likelihood method and SEIR model were used to estimate the basic reproduction number. Results: Early COVID-19 cases kept an increase in exponential growth manner before 26 January, 2020, then the increase trend became slower. The average incubation period was 5.01 (95%CI: 4.31-5.69) days; the average generation interval was 6.03 (95%CI: 5.20-6.91) days. The basic reproduction number was estimated to be 3.74 (95%CI: 3.63-3.87), 3.16 (95%CI: 2.90-3.43), and 3.91 (95%CI: 3.71-4.11) by three methods, respectively. Conclusions: The Gamma distribution fits both the generation interval and incubation period best, and the mean value of generation interval is 1.02 day longer than that of incubation period. The relatively high basic reproduction number indicates that the epidemic is still serious; Based on our analysis, the turning point of the epidemic would be seen on 26 January, the growth rate would be lower afterwards.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The ongoing outbreak of novel corona virus disease 2019 (COVID-19) in Wuhan, China, is arousing international concern. This study evaluated whether and when the infected but asymptomatic cases during the incubation period could infect others.  Methods: We collected data on demographic characteristics, exposure history, and symptom onset day of the confirmed cases, which had been announced by the Chinese local authorities. We evaluated the potential of transmission during the incubation period in 50 infection clusters, including 124 cases. All the secondary cases had a history of contact with their first-generation cases prior to symptom onset.  Results: The estimated mean incubation period for COVID-19 was 4.9 days (95% confidence interval [CI], 4.4 to 5.4) days, ranging from 0.8 to 11.1 days (2.5th to 97.5th percentile). The observed mean and standard deviation (SD) of serial interval was 4.1±3.3 days, with the 2.5th and 97.5th percentiles at -1 and 13 days. The infectious curve showed that in 73.0% of the secondary cases, their date of getting infected was before symptom onset of the first-generation cases, particularly in the last three days of the incubation period.  Conclusions: The results indicated the transmission of COVID-9 occurs among close contacts during the incubation period, which may lead to a quarantine loophole. Strong and effective countermeasures should be implemented to prevent or mitigate asymptomatic transmission during the incubation period in populations at high risk.</jats:p></p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Serial interval of novel coronavirus (2019-nCoV) infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.</jats:p></p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Association between Severity of MERS-CoV Infection and Incubation Period</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">We analyzed data for 170 patients in South Korea who had laboratory-confirmed infection with Middle East respiratory syndrome coronavirus. A longer incubation period was associated with a reduction in the risk for death (adjusted odds ratio/1-day increase in incubation period 0.83, 95% credibility interval 0.68–1.03).</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Association between Severity of MERS-CoV Infection and Incubation Period</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">We analyzed data for 170 patients in South Korea who had laboratory-confirmed infection with Middle East respiratory syndrome coronavirus. A longer incubation period was associated with a reduction in the risk for death (adjusted odds ratio/1-day increase in incubation period 0.83, 95% credibility interval 0.68–1.03).</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission and clinical characteristics of coronavirus disease 2019 in 104 outside-Wuhan patients, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Cases with coronavirus disease 2019 (COVID-19) emigrated from Wuhan escalated the risk of spreading in other cities. This report focused on the outside-Wuhan patients to assess the transmission and clinical characteristics of this illness. Methods: Contact investigation was conducted on each patient who admitted to the assigned hospitals in Hunan Province (geographically adjacent to Wuhan) from Jan 22, 2020 to Feb 12, 2020. Demographic, clinical, laboratory and radiological characteristics, medication therapy and outcomes were collected and analyzed. Patients were confirmed by PCR test. Results: Of the 104 patients, 48 (46.15%) were imported cases and 56 (53.85%) were indigenous cases; 93 (89.42%) had a definite contact history with infections. Family clusters were the major body of patients. Transmission along the chain of 3 &amp;ldquo:generations” was observed. Mean age was 43 (rang, 8-84) years (including 3 children) and 49 (47.12%) were male. Most patients had typical symptoms, 5 asymptomatic infections were found and 2 of them infected their relatives. The median incubation period was 6 (rang, 1-32) days, of 8 patients ranged from 18 to 32 days. Just 9 of 16 severe patients required ICU care. Until Feb 12, 2020, 40 (38.46%) discharged and 1 (0.96%) died. For the antiviral treatment, 80 (76.92%) patients received traditional Chinese medicine therapy. Conclusions: Family but not community transmission occupied the main body of infections in the two centers. Asymptomatic transmission demonstrated here warned us that it may bring more risk to the spread of COVID-19. The incubation period of 8 patients exceeded 14 days.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the serial interval of the novel coronavirus disease (COVID-19): A statistical analysis using the public data in Hong Kong from January 16 to February 15, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Backgrounds: The emerging virus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has caused a large outbreak of novel coronavirus disease (COVID-19) in Wuhan, China since December 2019. Based on the publicly available surveillance data, we identified 21 transmission chains in Hong Kong and estimated the serial interval (SI) of COVID-19. Methods: Index cases were identified and reported after symptoms onset, and contact tracing was conducted to collect the data of the associated secondary cases. An interval censored likelihood framework is adopted to fit a Gamma distribution function to govern the SI of COVID-19.  Findings: Assuming a Gamma distributed model, we estimated the mean of SI at 4.4 days (95%CI: 2.9−6.7) and SD of SI at 3.0 days (95%CI: 1.8−5.8) by using the information of all 21 transmission chains in Hong Kong.  Conclusion: The SI of COVID-19 may be shorter than the preliminary estimates in previous works. Given the likelihood that SI could be shorter than the incubation period, pre-symptomatic transmission may occur, and extra efforts on timely contact tracing and quarantine are recommended in combating the COVID-19 outbreak.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>A Chinese Case of COVID-19 Did Not Show Infectivity During the Incubation Period: Based on an Epidemiological Survey</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Controversy remains over whether the novel coronavirus 2019 (COVID-19) virus may have infectivity during the incubation period before the onset of symptoms. The author had the opportunity to examine the infectivity of COVID-19 during the incubation period by conducting an epidemiological survey on a confirmed patient who had visited Jeju Island during the incubation period. The epidemiological findings support the claim that the COVID-19 virus does not have infectivity during the incubation period.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary epidemiologic assessment of MERS-CoV outbreak in South Korea, May–June 2015</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">South Korea is experiencing the largest outbreak of Middle East respiratory syndrome coronavirus infections outside the Arabian Peninsula, with 166 laboratory-confirmed cases, including 24 deaths as of 19 June 2015. We estimated that the mean incubation period was 6.7 days and the mean serial interval 12.6 days. We found it unlikely that infectiousness precedes symptom onset. Based on currently available data, we predict an overall case fatality risk of 21.3% (95% credible interval: 14%–31%).</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Age-dependent risks of Incidence and Mortality of COVID-19 in Hubei Province and Other Parts of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>New coronavirus SARS-CoV-2 poses a big challenge for global public health in early 2020. Coronavirus Disease 2019 (COVID-19) caused by the virus rapidly spreads all over the world and takes thousands of lives in just two months. It is critical to refine the incidence and mortality risks of COVID-19 for the effective management of the general public and patients in the outbreak. In this report, we investigate the incidence and mortality risks of the infection by analyzing the age composition of 5319 infected patients, 76 fatal cases, and 1,144,648 individuals of the general public in China. Our result shows a relatively low incidence risk for young people but a very high mortality risk for seniors. Notably, mortality risk could be as high as 0.48 for people older than 80 years. Furthermore, our study suggests that a good medical service can effectively reduce the mortality rate of the viral infection to 1% or less.</jats:p></p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission interval estimates suggest pre-symptomatic spread of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods:  We collected outbreak information from Singapore and Tianjin,  China,  reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring.  Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.</jats:p></p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Surveillance operation for the 141st confirmed case of Middle East Respiratory Syndrome coronavirus in response to the patient’s prior travel to Jeju Island</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug 7</span></i><p style="text-align: justify; text-justify: inter-word;">The provincial government of Jeju, South Korea, was notified that a 42-year-old man infected with the Middle East Respiratory Syndrome (MERS) coronavirus had gone sightseeing in Jeju Island. Although the visiting period might be interpreted as the incubation period of MERS, the province decided to conduct active surveillance to prevent a worst-case scenario. Based on the channel of movement of the patient, healthy isolation and active monitoring were conducted for persons who came in contact with the patient. During the active surveillance, none of the 56 persons in self-isolation and 123 persons under active monitoring became infected. This fact supports that MERS is not contagious during the incubation period.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Surveillance operation for the 141st confirmed case of Middle East Respiratory Syndrome coronavirus in response to the patient’s prior travel to Jeju Island</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug 7</span></i><p style="text-align: justify; text-justify: inter-word;">The provincial government of Jeju, South Korea, was notified that a 42-year-old man infected with the Middle East Respiratory Syndrome (MERS) coronavirus had gone sightseeing in Jeju Island. Although the visiting period might be interpreted as the incubation period of MERS, the province decided to conduct active surveillance to prevent a worst-case scenario. Based on the channel of movement of the patient, healthy isolation and active monitoring were conducted for persons who came in contact with the patient. During the active surveillance, none of the 56 persons in self-isolation and 123 persons under active monitoring became infected. This fact supports that MERS is not contagious during the incubation period.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of the epidemic growth of the early 2019-nCoV outbreak using internationally confirmed cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On January 23, 2020, a quarantine was imposed on travel in and out of Wuhan, where the 2019 novel coronavirus (2019-nCoV) outbreak originated from. Previous analyses estimated the basic epidemiological parameters using symptom onset dates of the confirmed cases in Wuhan and outside China.  Methods: We obtained information on the 46 coronavirus cases who traveled from Wuhan before January 23 and have been subsequently confirmed in Hong Kong, Japan, Korea, Macau, Singapore, and Taiwan as of February 5, 2020. Most cases have detailed travel history and disease progress. Compared to previous analyses, an important distinction is that we used this data to informatively simulate the infection time of each case using the symptom onset time, previously reported incubation interval, and travel history. We then fitted a simple exponential growth model with adjustment for the January 23 travel ban to the distribution of the simulated infection time. We used a Bayesian analysis with diffuse priors to quantify the uncertainty of the estimated epidemiological parameters. We performed sensitivity analysis to different choices of incubation interval and the hyperparameters in the prior specification.  Results: We found that our model provides good fit to the distribution of the infection time. Assuming the travel rate to the selected countries and regions is constant over the study period, we found that the epidemic was doubling in size every 2.9 days (95% credible interval [CrI], 2 days--4.1 days). Using previously reported serial interval for 2019-nCoV, the estimated basic reproduction number is 5.7 (95% CrI, 3.4--9.2). The estimates did not change substantially if we assumed the travel rate doubled in the last 3 days before January 23, when we used previously reported incubation interval for severe acute respiratory syndrome (SARS), or when we changed the hyperparameters in our prior specification.  Conclusions: Our estimated epidemiological parameters are higher than an earlier report using confirmed cases in Wuhan. This indicates the 2019-nCoV could have been spreading faster than previous estimates.</jats:p></p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Buying Time—The Immune System Determinants of the Incubation Period to Respiratory Viruses</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Nov 18</span></i><p style="text-align: justify; text-justify: inter-word;">Respiratory viruses cause disease in humans characterized by an abrupt onset of symptoms. Studies in humans and animal models have shown that symptoms are not immediate and appear days or even weeks after infection. Since the initial symptoms are a manifestation of virus recognition by elements of the innate immune response, early virus replication must go largely undetected. The interval between infection and the emergence of symptoms is called the incubation period and is widely used as a clinical score. While incubation periods have been described for many virus infections the underlying mechanism for this asymptomatic phase has not been comprehensively documented. Here we review studies of the interaction between human pathogenic respiratory RNA viruses and the host with a particular emphasis on the mechanisms used by viruses to inhibit immunity. We discuss the concept of the “stealth phase”, defined as the time between infection and the earliest detectable inflammatory response. We propose that the “stealth phase” phenomenon is primarily responsible for the suppression of symptoms during the incubation period and results from viral antagonism that inhibits major pathways of the innate immune system allowing an extended time of unhindered virus replication.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation period of 2019 novel coronavirus (2019-nCoV) infections among travellers from Wuhan, China, 20–28 January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In January 2020, an increasing number of cases confirmed to be infected with 2019-nCoV were detected outside Wuhan. For 88 cases detected between 20 and 28 January, the travel history (to and) from Wuhan is known, as well as their symptom onset date. Their ages range from 2 to 72 years of age (information missing for four cases); 31 were female and 57 were male. During this initial stage of the epidemic, it is most likely that these travellers were infected in Wuhan. Consequently, their time spent in Wuhan can be taken as the duration of exposure to infection. Of these 88 cases with known travel history, 63 were Wuhan residents who travelled elsewhere and 25 were visitors who stayed in Wuhan for a limited time. By taking the date of symptom onset and travel history together, we inferred the possible incubation period for each of these cases. The data used for this analysis has been translated from Chinese sources such as provincial centres of disease control, and made publicly available [8]. We took the data as available on 29 January 2020 (Supplementary Material S1).</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Singapore's experience of SARS</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus that causes severe acute respiratory syndrome (SARS) is transmitted mainly via respiratory droplets. Typical presenting symptoms are akin to those of ordinary pneumonia. Young patients start with fever, chills, malaise, headache, or myalgia; cough and dyspnoea follow. Older persons and those taking corticosteroids may have neither fever nor respiratory symptoms. Exceptional suspicion is needed to identify SARS early in the illness. During an outbreak, even patients with low suspicion of SARS should be promptly isolated, and all contacts quarantined. Health workers need training in the use of appropriate barriers against droplets and other body fluids. Any fever cluster in patients or carers requires immediate action: discharges, visits, and transfers between wards and hospitals should be stopped. Halting hospital admissions and ten-day quarantine of suspected cases create wide buffer zones. To counter a possible resurgence of SARS, a system of prepared isolation and quarantine facilities is important.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>The serial interval of COVID-19 from publicly reported confirmed cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.1% of reports indicating pre-symptomatic transmission.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of meteorological conditions and prediction of epidemic trend of 2019-nCoV infection in 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To investigate the meteorological condition for incidence and spread of 2019-nCoV infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease.  Methods: The meteorological factors during the outbreak period of the novel coronavirus pneumonia in Wuhan in 2019 were collected and analyzed, and were confirmed with those of Severe Acute Respiratory Syndrome (SARS) in China in 2003. Data of patients infected with 2019-nCoV and SARS coronavirus were collected from WHO website and other public sources. Results: This study found that the suitable temperature range for 2019-nCoV coronavirus survival is (13-24 degree Celsius), among which 19 degree Celsius lasting about 60 days is conducive to the spread between the vector and humans; the humidity range is 50%-80%, of which about 75% humidity is conducive to the survival of the coronavirus; the suitable precipitation range is below 30 mm/ month. Cold air and continuous low temperature over one week are helpful for the elimination of the virus. The prediction results show that with the approach of spring, the temperature in north China gradually rises, and the coronavirus spreads to middle and high latitudes along the temperature line of 13-18 degree Celsius. The population of new coronavirus infections is concentrated in Beijing, Tianjin, Hebei, Jiangsu, Zhejiang, Shanghai and other urban agglomerations. Starting from May 2020, the Beijing-Tianjin-Hebei urban agglomeration, the Central China Zhengzhou-Wuhan urban agglomeration, the eastern Jiangsu-Zhejiang-Shanghai urban agglomeration, and the southern Pearl River Delta urban agglomeration are all under a high temperature above 24 degree Celsius, which is not conducive to the survival and reproduction of coronaviruses, so the epidemic is expected to end. Conclusions: A wide range of continuous warm and dry weather is conducive to the survival of 2019-nCoV. The coming of spring, in addition to the original Wuhan-Zhengzhou urban agglomeration in central China, means that the prevention and control measures in big cities located in mid-latitude should be strengthened, especially the monitoring of transportation hubs. The Pearl River Delta urban agglomeration is a concentrated area of population in south China, with a faster temperature rise than those in mid-high latitudes, and thus the prevention in this area should be prioritized. From a global perspective, cities with a mean temperature below 24 degree Celsius are all high-risk cities for 2019-nCoV transmission before June.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemic Situation of Novel Coronavirus Pneumonia in China mainland</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>[Objective] Analyze the occurrence of novel coronavirus pneumonia(NCP) in China mainland, explore the epidemiological rules, and evaluate the effect of prevention and control. [Methods] From December 1, 2019 to February 14, 2020, Analysis of 66,492 confirmed cases of NCP in China mainland. [Results] From December 1, 2019 to February 14, 2020, a total of 66,492 cases of NCP were confirmed in China mainland, a total of 54,406 cases were confirmed in Hubei Province, a total of 37,914 cases were confirmed in Wuhan city. On February 5, 2020, the number of suspected cases of NCP in China mainland reached a maximum of 5,328. Since then, the suspected cases have shown a significant downward trend. On February 4, 2020, the number of confirmed cases has gradually decreased in China mainland since then. On February 12, 2020, the number of confirmed cases of NCP in China mainland increased explosively to 15,152, and then began to decline. From February 3, 2020, except for Hubei Province, In China mainland, the number of newly confirmed cases of NCP has continued to decline; From December 1, 2019 to February 14, 2020, a total of 1,523 cases of NCP deaths in China mainland, a cumulative cure of 8096 cases, and  mortality and cure rate were 2.29% (1523/66492) and 12.18% (8096/66492) respectively; Starting from January 27, 2020, The spread index of NCP gradually declined, and the extinction index of NCP rose  little by little from January 29, 2020. [Conclusion] Starting from February 5, 2020, the number of suspected cases of the NCP is gradually decrease in China mainland. From February 3, 2020, the number of newly confirmed cases of NCP is continuous decline in China mainland except Hubei Province, This shows that the control of the transmission of NCP has achieved. Judged from the number of confirmed cases of NCP, controlling the outbreak in China mainland is to control the epidemic in Hubei province, and the key to controlling the outbreak in Hubei province is to control the epidemic in Wuhan city; Judging from the gradual increase in the number and cure rate of the NCP, and the gradual decline in the number of deaths and mortality, the trend of outbreak control and treatment is getting better; A decrease of the NCP spread index indicates a slowdown in the spread of the virus, but the cumulative confirmed cases is still increase, so the epidemic will continue for some time. However, the turning point of the epidemic in mainland China has not yet occurred. On February 12, 2020, the turning point of the epidemic in China mainland except Hubei province, has turned up.</jats:p></p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the Distribution of the Incubation Periods of Human Avian Influenza A(H7N9) Virus Infections</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A novel avian influenza virus, influenza A(H7N9), emerged in China in early 2013 and caused severe disease in humans, with infections occurring most frequently after recent exposure to live poultry. The distribution of A(H7N9) incubation periods is of interest to epidemiologists and public health officials, but estimation of the distribution is complicated by interval censoring of exposures. Imputation of the midpoint of intervals was used in some early studies, resulting in estimated mean incubation times of approximately 5 days. In this study, we estimated the incubation period distribution of human influenza A(H7N9) infections using exposure data available for 229 patients with laboratory-confirmed A(H7N9) infection from mainland China. A nonparametric model (Turnbull) and several parametric models accounting for the interval censoring in some exposures were fitted to the data. For the best-fitting parametric model (Weibull), the mean incubation period was 3.4 days (95% confidence interval: 3.0, 3.7) and the variance was 2.9 days; results were very similar for the nonparametric Turnbull estimate. Under the Weibull model, the 95th percentile of the incubation period distribution was 6.5 days (95% confidence interval: 5.9, 7.1). The midpoint approximation for interval-censored exposures led to overestimation of the mean incubation period. Public health observation of potentially exposed persons for 7 days after exposure would be appropriate.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation periods of viral gastroenteritis: a systematic review</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Sep 25</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Accurate knowledge of incubation period is important to investigate and to control infectious diseases and their transmission, however statements of incubation period in the literature are often uncited, inconsistent, and/or not evidence based. METHODS: In a systematic review of the literature on five enteric viruses of public health importance, we found 256 articles with incubation period estimates, including 33 with data for pooled analysis. RESULTS: We fit a log-normal distribution to pooled data and found the median incubation period to be 4.5 days (95% CI 3.9-5.2 days) for astrovirus, 1.2 days (95% CI 1.1-1.2 days) for norovirus genogroups I and II, 1.7 days (95% CI 1.5-1.8 days) for sapovirus, and 2.0 days (95% CI 1.4-2.4 days) for rotavirus. CONCLUSIONS: Our estimates combine published data and provide sufficient quantitative detail to allow for these estimates to be used in a wide range of clinical and modeling applications. This can translate into improved prevention and control efforts in settings with transmission or the risk of transmission.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the generation interval for COVID-19 based on symptom onset data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies. Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided. Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers. Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32-67%) for Singapore and 62% (95%CI 50-76%) for Tianjin, China. Estimates of the reproduction number based on the generation interval distribution were slightly higher than those based on the serial interval distribution. Conclusions: Estimating generation and serial interval distributions from outbreak data requires careful investigation of the underlying transmission network. Detailed contact tracing information is essential for correctly estimating these quantities.</jats:p></p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>The human enteric coronaviruses.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A coronarirus was seen in the faeces from 15 (4.2%) of 355 adults with diarrhoea and from 5 (5.2%) of 96 adults without diarrhoea. Similar particles were seen in the faeces from 5 (2.2%) of 227 children aged 1--14 years with gastroenteritis, but in none of those from 230 infants under one year of age with gastroenteritis. There was no evidence that the coronavirus was responsible for any of 34 outbreaks of gastroenteritis, although it possibly caused diarrhoea in patients admitted to a psycho-geriatric unit. Excretion of the virus often continued for many months. One strain was propagated in human embryo kidney monolayers and human embryo intestinal organ cultures, although serial passage could not be accomplished.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the Asymptomatic Proportion of 2019 Novel Coronavirus onboard the Princess Cruises Ship, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The potential infectiousness of asymptomatic COVID-19 cases together with a substantial fraction of asymptomatic infections among all infections, have been highlighted in clinical studies. We conducted statistical modeling analysis to derive the delay-adjusted asymptomatic proportion of the positive COVID-19 infections onboard the Princess Cruises ship along with the timeline of infections. We estimated the asymptomatic proportion at 17.9% (95% CrI: 15.5%-20.2%), with most of the infections occurring before the start of the 2-week quarantine.</jats:p></p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Coronavirus Disease 2019 (COVID-19): An Emerging Infectious Disease in the 21st Century</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background: At the beginning of the New Year 2020, China alerted the world health organization (WHO) to a cluster of unusual pneumonia cases in Wuhan. After extensive speculation, eventually a new species of coronavirus introduced as the causative pathogen of the disease. Coronavirus disease 2019 (COVID-19) is a name for the disease, and the virus that causes it is known SARS-CoV-2. The very rapid spread of the COVID-19 in China and in many other countries has caused fear among people across the world. The novel coronavirus outbreak declared a Public Health Emergency of International Concern on 30 January 2020. Materials and Methods: Several databases such as PubMed, Scopus, Google scholar, and BioRxiv were searched for publications reporting on the novel coronavirus up to 29 February 2020. Literature searches were performed using keywords including “Coronavirus 2019”, “2019-nCoV”, “COVID-19”, and “SARS-CoV-2”. Moreover, websites such as the World Health Organization (WHO) and the Centers for Disease Control and Prevention (CDC) were searched to retrieve updated data and statistics regarding the novel coronavirus. We extracted data on the epidemiology, pathogenesis, virology, clinical manifestations, transmission routes, diagnosis, treatment, and prevention measures. Results: From the 1416 articles identified in the initial search, 53 were remained after title and abstract screening. After full-text review, 37 articles were eligible to include in our study. Incubation period for COVID-19 is between 2-10 days, according to the World Health Organization (WHO). The case fatality rate in patients infected with SARC-CoV-2 is 4.3%, and the results indicate that the mortality is higher in elderly individuals and patients with chronic conditions including patients with coronary artery disease, diabetes, chronic pulmonary disease, and hypertension. The mortality rate in healthy subjects is less than 1%. Conclusion: The outbreak caused by the novel coronavirus is larger than the previous human coronaviruses, showing that the SARS-CoV-2 is an extremely contagious virus. However, the mortality rate of COVID-19 is lower than that of other coronaviruses diseases such as SARS or MERS and other viruses like HIV and Ebola. Currently, due to the lack of an effective treatment and vaccine, the best way to deal with the COVID-19 disease is to prevent transmission and spread of the virus and to execute personal protective measures.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>[Epidemiological analysis on a family cluster of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: To understand the possible transmission route of a family cluster of COVID-19 in Zhengzhou and the potential infectivity of COVID-19 in incubation period, and provide scientific evidence for the timely control of infectious source and curb the spread of the epidemic. Methods: Epidemiological investigation was conducted for a family cluster of COVID-19 (8 cases) with descriptive epidemiological method, and respiratory tract samples of the cases were collected for the nucleic acid detection of 2019-nCoV by RT-PCR. Results: Two primary cases, which occurred on 31 January and 1 February, 2020, respectively, had a common exposure history in Wuhan. The other six family members had onsets on 30 January, 31 January, 1 February (three cases) and 3 February, 2020. Conclusions: In this family cluster of COVID-19, six family members were infected through common family exposure to the 2 primary cases. Five secondary cases had onsets earlier than or on the same day as the primary cases, indicating that COVID-19 is contagious in incubation period, and the home isolation in the early phase of the epidemic might lead to the risk of family cluster of COVID-19.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>News</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Chinese scientists released genetic information on the coronavirus causing an outbreak of SARS-like illness in Wuhan, China. The Vaccine Research Institute at the US National Institutes of Health (NIH) immediately began development of a new vaccine. After the 2003 SARS outbreak, it took 20 months from the release of the viral genome to get a vaccine ready for human trials. The zika virus took six months. Now, the NIH is pushing to have a vaccine ready for testing in three to six months. The race is on. The NIH team took the template for a SARS vaccine and swapped enough code from the new virus to start a new vaccine. Researchers, having experience with SARS, already knew which part of the virus to choose for the vaccine’s antigen, a sort of red flag for the immune system. This synthetic antigen will prod the body to make antibodies.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Effectiveness of airport screening at detecting travellers infected with 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.</jats:p></p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>[Advances on presymptomatic or asymptomatic carrier transmission of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is rapidly spreading. Patients in incubation period and healthy carriers are possible sources for transmission. However, such sources of infection cannot be effectively identified due to the symptoms absent. The research evidence is very lacking so far, although there are a few studies suggesting that presymptomatic or asymptomatic carrier may cause COVID-19 transmission. Nearly half of the literature is in the state of preprint without peer review. The question of "the degree to which presymptomatic or asymptomatic infections can transmit" is not fully understood. There is an urgent need to screen infected carriers in larger close contacts or in the general population, and assess their risk for transmission.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus.</jats:p></p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolving Epidemiology and Impact of Non-pharmaceutical Interventions on the Outbreak of Coronavirus Disease 2019 in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND We described the epidemiological features of the coronavirus disease 2019 (Covid-19) outbreak, and evaluated the impact of non-pharmaceutical interventions on the epidemic in Wuhan, China.  METHODS Individual-level data on 25,961 laboratory-confirmed Covid-19 cases reported through February 18, 2020 were extracted from the municipal Notifiable Disease Report System. Based on key events and interventions, we divided the epidemic into four periods: before January 11, January 11-22, January 23 - February 1, and February 2-18. We compared epidemiological characteristics across periods and different demographic groups. We developed a susceptible-exposed-infectious-recovered model to study the epidemic and evaluate the impact of interventions.   RESULTS The median age of the cases was 57 years and 50.3% were women. The attack rate peaked in the third period and substantially declined afterwards across geographic regions, sex and age groups, except for children (age &lt;20) whose attack rate continued to increase. Healthcare workers and elderly people had higher attack rates and severity risk increased with age. The effective reproductive number dropped from 3.86 (95% credible interval 3.74 to 3.97) before interventions to 0.32 (0.28 to 0.37) post interventions. The interventions were estimated to prevent 94.5% (93.7 to 95.2%) infections till February 18. We found that at least 59% of infected cases were unascertained in Wuhan, potentially including asymptomatic and mild-symptomatic cases.  CONCLUSIONS	 Considerable countermeasures have effectively controlled the Covid-19 outbreak in Wuhan. Special efforts are needed to protect vulnerable populations, including healthcare workers, elderly and children. Estimation of unascertained cases has important implications on continuing surveillance and interventions.</jats:p></p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Health protection guideline of passenger transport stations and transportation facilities during novel coronavirus pneumonia (NCP) outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">During the coronavirus pneumonia (NCP) outbreak, the transportation industries are faced with the more burdensome tasks of outbreak prevention and control as well as ensuring smooth transportation. It is important to organize transportation in order to restore the order of production and life, ensure the normal economic and social operation, and control the outbreak in the whole society. From the perspective of health, this guideline puts forward technical requirements on the operation management, personnel requirements and health protection of passenger transportation places such as aviation, railway, subway, bus, taxi, ship, etc., which reduces the impact of the NCP outbreak on the transportation industry and personal health risks.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemic trend of corona virus disease 2019 (COVID-19) in mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: In order to master the epidemic trend of corona virus disease 2019 (COVID-19) and evaluate the effect of prevention and control, we evaluate the epidemic dynamics of COVID-19 in mainland China, Hubei province, Wuhan city and other provinces outside Hubei from January 16 to February 14, 2020. Methods: We collected the daily number of new confirmed COVID-19 cases by nucleic acid detection reported by the National Health Commission from January 16, 2020 to February 14, 2020. The analysis includes the epidemic curve of the new confirmed cases, multiple of the new confirmed cases for period-over-period, multiple of the new confirmed cases for fixed-base, and the period-over-period growth rate of the new confirmed cases. Results: From January 16 to February 14, 2020, the cumulative number of new confirmed cases of COVID-19 in mainland China was 50 031, including 37 930 in Hubei province, 22 883 in Wuhan city and 12 101 in other provinces outside Hubei. The peak of the number of new confirmed cases in other provinces outside Hubei was from January 31 to February 4, 2020, and the peak of new confirmed cases in Wuhan city and Hubei province was from February 5 to February 9, 2020. The number of new confirmed cases in other provinces outside Hubei showed a significant decline (23% compared with the peak) from February 5 to February 9, 2020, while the number of new confirmed cases in Wuhan city (30% compared with the peak) and Hubei Province (37% compared with the peak) decreased significantly from February 10 to February 14, 2020. Conclusion: The epidemic prevention and control measures taken by the state and governments at all levels have shown very significant effects, effectively curbing the spread of the COVID-19 epidemic in China.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Early efforts in modeling the incubation period of infectious diseases with an acute course of illness</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 May 11</span></i><p style="text-align: justify; text-justify: inter-word;">The incubation period of infectious diseases, the time from infection with a microorganism to onset of disease, is directly relevant to prevention and control. Since explicit models of the incubation period enhance our understanding of the spread of disease, previous classic studies were revisited, focusing on the modeling methods employed and paying particular attention to relatively unknown historical efforts. The earliest study on the incubation period of pandemic influenza was published in 1919, providing estimates of the incubation period of Spanish flu using the daily incidence on ships departing from several ports in Australia. Although the study explicitly dealt with an unknown time of exposure, the assumed periods of exposure, which had an equal probability of infection, were too long, and thus, likely resulted in slight underestimates of the incubation period. After the suggestion that the incubation period follows lognormal distribution, Japanese epidemiologists extended this assumption to estimates of the time of exposure during a point source outbreak. Although the reason why the incubation period of acute infectious diseases tends to reveal a right-skewed distribution has been explored several times, the validity of the lognormal assumption is yet to be fully clarified. At present, various different distributions are assumed, and the lack of validity in assuming lognormal distribution is particularly apparent in the case of slowly progressing diseases. The present paper indicates that (1) analysis using well-defined short periods of exposure with appropriate statistical methods is critical when the exact time of exposure is unknown, and (2) when assuming a specific distribution for the incubation period, comparisons using different distributions are needed in addition to estimations using different datasets, analyses of the determinants of incubation period, and an understanding of the underlying disease mechanisms.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Early efforts in modeling the incubation period of infectious diseases with an acute course of illness</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 May 11</span></i><p style="text-align: justify; text-justify: inter-word;">The incubation period of infectious diseases, the time from infection with a microorganism to onset of disease, is directly relevant to prevention and control. Since explicit models of the incubation period enhance our understanding of the spread of disease, previous classic studies were revisited, focusing on the modeling methods employed and paying particular attention to relatively unknown historical efforts. The earliest study on the incubation period of pandemic influenza was published in 1919, providing estimates of the incubation period of Spanish flu using the daily incidence on ships departing from several ports in Australia. Although the study explicitly dealt with an unknown time of exposure, the assumed periods of exposure, which had an equal probability of infection, were too long, and thus, likely resulted in slight underestimates of the incubation period. After the suggestion that the incubation period follows lognormal distribution, Japanese epidemiologists extended this assumption to estimates of the time of exposure during a point source outbreak. Although the reason why the incubation period of acute infectious diseases tends to reveal a right-skewed distribution has been explored several times, the validity of the lognormal assumption is yet to be fully clarified. At present, various different distributions are assumed, and the lack of validity in assuming lognormal distribution is particularly apparent in the case of slowly progressing diseases. The present paper indicates that (1) analysis using well-defined short periods of exposure with appropriate statistical methods is critical when the exact time of exposure is unknown, and (2) when assuming a specific distribution for the incubation period, comparisons using different distributions are needed in addition to estimations using different datasets, analyses of the determinants of incubation period, and an understanding of the underlying disease mechanisms.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Early efforts in modeling the incubation period of infectious diseases with an acute course of illness</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 May 11</span></i><p style="text-align: justify; text-justify: inter-word;">The incubation period of infectious diseases, the time from infection with a microorganism to onset of disease, is directly relevant to prevention and control. Since explicit models of the incubation period enhance our understanding of the spread of disease, previous classic studies were revisited, focusing on the modeling methods employed and paying particular attention to relatively unknown historical efforts. The earliest study on the incubation period of pandemic influenza was published in 1919, providing estimates of the incubation period of Spanish flu using the daily incidence on ships departing from several ports in Australia. Although the study explicitly dealt with an unknown time of exposure, the assumed periods of exposure, which had an equal probability of infection, were too long, and thus, likely resulted in slight underestimates of the incubation period. After the suggestion that the incubation period follows lognormal distribution, Japanese epidemiologists extended this assumption to estimates of the time of exposure during a point source outbreak. Although the reason why the incubation period of acute infectious diseases tends to reveal a right-skewed distribution has been explored several times, the validity of the lognormal assumption is yet to be fully clarified. At present, various different distributions are assumed, and the lack of validity in assuming lognormal distribution is particularly apparent in the case of slowly progressing diseases. The present paper indicates that (1) analysis using well-defined short periods of exposure with appropriate statistical methods is critical when the exact time of exposure is unknown, and (2) when assuming a specific distribution for the incubation period, comparisons using different distributions are needed in addition to estimations using different datasets, analyses of the determinants of incubation period, and an understanding of the underlying disease mechanisms.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Individual quarantine and active monitoring of contacts are core disease control strategies, particularly for emerging infectious diseases such as Coronavirus Disease 2019 (COVID-19). To estimate the comparative efficacy of these interventions to control COVID-19, we fit a stochastic branching model, comparing two sets of reported parameters for the dynamics of the disease. Our results suggest that individual quarantine may contain an outbreak of COVID-19 with a short serial interval (4.8 days) only in settings with high intervention performance where at least three-quarters of infected contacts are individually quarantined. However, in settings where this performance is unrealistically high and the outbreak of COVID-19 continues to grow, so too will the burden of the number of contacts traced for active monitoring or quarantine. In such circumstances where resources are prioritized for scalable interventions such as social distancing, we show active monitoring or individual quarantine of high-risk contacts can contribute synergistically to social distancing. To the extent that interventions based on contact tracing can be implemented, therefore, they can help mitigate the spread of COVID-19. Our model highlights the urgent need for more data on the serial interval and the extent of presymptomatic transmission in order to make data-driven policy decisions regarding the cost-benefit comparisons of individual quarantine vs. active monitoring of contacts.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>The impact of social distancing and epicenter lockdown on the COVID-19 epidemic in mainland China: A data-driven SEIQR model study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The outbreak of coronavirus disease 2019 (COVID-19) which originated in Wuhan, China, constitutes a public health emergency of international concern with a very high risk of spread and impact at the global level. We developed data-driven susceptible-exposed-infectious-quarantine-recovered (SEIQR) models to simulate the epidemic with the interventions of social distancing and epicenter lockdown. Population migration data combined with officially reported data were used to estimate model parameters, and then calculated the daily exported infected individuals by estimating the daily infected ratio and daily susceptible population size. As of Jan 01, 2020, the estimated initial number of latently infected individuals was 380.1 (95%-CI: 379.8~381.0). With 30 days of substantial social distancing, the reproductive number in Wuhan and Hubei was reduced from 2.2 (95%-CI: 1.4~3.9) to 1.58 (95%-CI: 1.34~2.07), and in other provinces from 2.56 (95%-CI: 2.43~2.63) to 1.65 (95%-CI: 1.56~1.76). We found that earlier intervention of social distancing could significantly limit the epidemic in mainland China. The number of infections could be reduced up to 98.9%, and the number of deaths could be reduced by up to 99.3% as of Feb 23, 2020. However, earlier epicenter lockdown would partially neutralize this favorable effect. Because it would cause in situ deteriorating, which overwhelms the improvement out of the epicenter. To minimize the epidemic size and death, stepwise implementation of social distancing in the epicenter city first, then in the province, and later the whole nation without the epicenter lockdown would be practical and cost-effective.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>A familial cluster of infection associated with the 2019 novel coronavirus indicating potential person-to-person transmission during the incubation period</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An ongoing outbreak of pneumonia associated with 2019 novel coronavirus (2019-nCoV) was reported in China. It is unclear if the infectivity exists during the incubation period, although a person-to-person transmission has been reported in previous studies. We report the epidemiological features of a familial cluster of four patients in Shanghai, of which one was 88 years old man with moving difficulties and was only exposed to his asymptomatic family members who developed symptoms later. The epidemiological evidence has shown a potential transmission of the 2019-nCoV during the incubation period.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>First respiratory transmitted food borne outbreak?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The world is faced with a remarkable coronavirus outbreak with epicentre in Wuhan, China. Altogether 40554 cases have been confirmed globally with novel coronavirus (SARS-CoV-2) until February 10, 2020. Rigorous surveillance in other countries is required to prevent further global expansion of the outbreak, but resolving the exact mechanism of the initial transmission events is crucial. Most initial cases had visited Huanan South Seafood Market in Wuhan selling also various exotic live animals. Based on the limited initial human-to-human transmission and timely clustering of cases in Huanan market among elderly men, coupled with knowledge that coronaviruses are derived from animals and relationship of SARS-CoV-2 to bat coronavirus, zoonotic transmission in the first instance is probable. To target the actions, similar epidemiological actions to human cases are needed with animal or food exposures. According to current information, an exceptionally wide contamination of seafood market might explain the initiation of the SARS-CoV-2 outbreak. Seafood tanks, air contamination by live animals or rodents are possibilities, but sold animals normally come from various sources. The mode of transmission may become clearer in future: usually in outbreak investigations, hindsight is easy, but for now information about the initial source of this outbreak is limited.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>A spatial model of CoVID-19 transmission in England and Wales: early spread and peak timing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-14</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: An outbreak of a novel coronavirus, named CoVID-19, was first reported in China on 31 December 2019. As of 9 February 2020, cases have been reported in 25 countries, including probable cases of human-to-human transmission in England.    Methods: We adapted an existing national-scale metapopulation model to capture the spread of CoVID-19 in England and Wales. We used 2011 census data to capture population sizes and population movement, together with parameter estimates from the current outbreak in China.   Results:  We predict that a CoVID-19 outbreak will peak 126 to 147 days (~4 months) after the start of person-to-person transmission in England and Wales in the absence of controls, assuming biological parameters remain unchanged. Therefore, if person-to-person transmission persists from February, we predict the epidemic peak would occur in June. The starting location has minimal impact on peak timing, and model stochasticity varies peak timing by 10 days. Incorporating realistic parameter uncertainty leads to estimates of peak time ranging from 78 days to 241 days after person-to-person transmission has been established.  Seasonal changes in transmission rate substantially impact the timing and size of the epidemic peak, as well as the total attack rate.   Discussion: We provide initial estimates of the potential course of CoVID-19 in England and Wales in the absence of control measures. These results can be refined with improved estimates of epidemiological parameters, and permit investigation of control measures and cost effectiveness analyses. Seasonal changes in transmission rate could shift the timing of the peak into winter months, which will have important implications for healthcare capacity planning.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Severe Acute Respiratory Syndrome Coronavirus in Stool Specimens by Commercially Available Real-Time Reverse Transcriptase PCR Assays</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Three commercially available real-time reverse transcriptase PCR assays (the Artus RealArt HPA coronavirus LightCycler, the Artus RealArt HPA coronavirus Rotor-Gene, and the EraGen severe acute respiratory syndrome coronavirus POL assay) and three RNA extraction methodologies were evaluated for the detection of severe acute respiratory syndrome coronavirus RNA from 91 stool specimens. The assays' sensitivities were highest (58% to 75%) for specimens obtained 8 to 21 days after symptom onset. The assays were less sensitive when specimens were obtained less than 8 days or more than 21 days after the onset of symptoms. All assays were 100% specific.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Characteristics of Coronavirus Disease 2019 in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND Since December 2019, when coronavirus disease 2019 (Covid-19) emerged in Wuhan city and rapidly spread throughout China, data have been needed on the clinical characteristics of the affected patients. METHODS We extracted data regarding 1099 patients with laboratory-confirmed Covid-19 from 552 hospitals in 30 provinces, autonomous regions, and municipalities in China through January 29, 2020. The primary composite end point was admission to an intensive care unit (ICU), the use of mechanical ventilation, or death. RESULTS The median age of the patients was 47 years; 41.9% of the patients were female. The primary composite end point occurred in 67 patients (6.1%), including 5.0% who were admitted to the ICU, 2.3% who underwent invasive mechanical ventilation, and 1.4% who died. Only 1.9% of the patients had a history of direct contact with wildlife. Among nonresidents of Wuhan, 72.3% had contact with residents of Wuhan, including 31.3% who had visited the city. The most common symptoms were fever (43.8% on admission and 88.7% during hospitalization) and cough (67.8%). Diarrhea was uncommon (3.8%). The median incubation period was 4 days (interquartile range, 2 to 7). On admission, ground-glass opacity was the most common radiologic finding on chest computed tomography (CT) (56.4%). No radiographic or CT abnormality was found in 157 of 877 patients (17.9%) with nonsevere disease and in 5 of 173 patients (2.9%) with severe disease. Lymphocytopenia was present in 83.2% of the patients on admission. CONCLUSIONS During the first 2 months of the current outbreak, Covid-19 spread rapidly throughout China and caused varying degrees of illness. Patients often presented without fever, and many did not have abnormal radiologic findings. (Funded by the National Health Commission of China and others.)</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Early dynamics of transmission and control of COVID-19: a mathematical modelling study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: An outbreak of the novel coronavirus SARS-CoV-2 has led to 46,997 confirmed cases as of 13th February 2020. Understanding the early transmission dynamics of the infection and evaluating the effectiveness of control measures is crucial for assessing the potential for sustained transmission to occur in new areas.   Methods: We combined a stochastic transmission model with data on cases of novel coronavirus disease (COVID-19) in Wuhan and international cases that originated in Wuhan to estimate how transmission had varied over time during January and February 2020. Based on these estimates, we then calculated the probability that newly introduced cases might generate outbreaks in other areas.   Findings: We estimated that the median daily reproduction number, Rt , declined from 2.35 (95% CI: 1.15-4.77) one week before travel restrictions were introduced on 23rd January to 1.05 (95% CI: 0.413-2.39) one week after. Based on our estimates of Rt,we calculated that in locations with similar transmission potential as Wuhan in early January, once there are at least four independently introduced cases, there is a more than 50% chance the infection will establish within that population.   Interpretation: Our results show that COVID-19 transmission likely declined in Wuhan during late January 2020, coinciding with the introduction of control measures. As more cases arrive in international locations with similar transmission potential to Wuhan pre-control, it is likely many chains of transmission will fail to establish initially, but may still cause new outbreaks eventually.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Phase adjusted estimation of the number of 2019 novel coronavirus cases in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV / SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of Rt. In the first assumption, Rt was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with Rt = 1.9, 2.6 or 3.1. The number of infections would reach 11,044, 70,258 and 227,989, respectively, by 29 February 2020. In the second assumption, Rt was assumed to gradually decrease at different phases from high level of transmission (Rt = 3.1, 2.6 and 1.9) to below 1 (Rt = 0.9 or 0.5) owing to increasingly implemented public heath intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce Rt to an ideal level and control the infection.</jats:p></p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>How Ophthalmologists Should Understand and Respond to the Current Epidemic of Novel Coronavirus Pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The new coronavirus pneumonia that first appeared in Wuhan, China, in December 2019 has attracted great attention from both the Chinese government and the international community. The International Committee on Viral Classification named the virus 'Severe Acute Respiratory Syndrome Coronavirus 2' (SARS-CoV-2), and the WHO named the pneumonia it causes 'Coronavirus Disease 2019' (COVID-19). At present, the disease is centered in Wuhan City and is spreading rapidly to all parts of China, as well as twenty other countries. About 20% of the people infected during the SARS epidemic in 2003 were employees in hospital environments. COVID-19 has infected an even greater number of heath care workers. Therefore, ophthalmologists need to understand the disease and recognize the importance of taking preventive measures. Although ophthalmologists do not work on the front lines of the outbreak, due to their area of expertise, a variety of situations, such as infection consultations or ophthalmic emergency treatments, can lead to the exposure of ophthalmologists to high-risk environments. This risk will only increase as the number of infected patients continues to increase. When dealing with seemingly normal ophthalmic patients, the vigilance of ophthalmologists and associated staff tends to be significantly reduced. To better protect patients, families, and health care workers, it is strongly recommended that in addition to the standard precautions for the care of all patients, strict contact precautions and droplet precautions need to be taken by ophthalmologists. These measures include 1) wearing an efficient mask (an N95 mask); 2) always performing hand hygiene before and after examining a patient; (3) wearing sterile gloves when entering a patient&rsquo;s room and touching a patient; (4) wearing a gown when contact is expected with items and environmental surfaces surrounding a patient or when the patient is incontinent or has diarrhea or a surgical or other invasive wound with oozing fluid; 5) cleaning and disinfecting ophthalmic equipment and correctly handling medical waste after examination to prevent transmission to patients who are subsequently examined; 6) wearing goggles and a disposable mask to cover the front and sides of the face before touching a patient, as the virus could spread through the ocular surface; 7) performing the relevant screening for novel coronavirus pneumonia for regular patients who have conjunctivitis and respiratory symptoms at the same time; 8) prohibiting the use of infected patients as potential donors for corneal transplants and temporarily adding donor SARS-CoV-2 screening to the medical standard of the eye bank during the outbreak; and 9) for the purposes of scientific research, diagnosis, and other special needs, packing, shipping, and transporting collected specimens according to the relevant dangerous biological goods regulations.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Association between the Severity of Influenza A(H7N9) Virus Infections and Length of the Incubation Period</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb 17</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In early 2013, a novel avian-origin influenza A(H7N9) virus emerged in China, and has caused sporadic human infections. The incubation period is the delay from infection until onset of symptoms, and varies from person to person. Few previous studies have examined whether the duration of the incubation period correlates with subsequent disease severity. METHODS AND FINDINGS: We analyzed data of period of exposure on 395 human cases of laboratory-confirmed influenza A(H7N9) virus infection in China in a Bayesian framework using a Weibull distribution. We found a longer incubation period for the 173 fatal cases with a mean of 3.7 days (95% credibility interval, CrI: 3.4–4.1), compared to a mean of 3.3 days (95% CrI: 2.9–3.6) for the 222 non-fatal cases, and the difference in means was marginally significant at 0.47 days (95% CrI: -0.04, 0.99). There was a statistically significant correlation between a longer incubation period and an increased risk of death after adjustment for age, sex, geographical location and underlying medical conditions (adjusted odds ratio 1.70 per day increase in incubation period; 95% credibility interval 1.47–1.97). CONCLUSIONS: We found a significant association between a longer incubation period and a greater risk of death among human H7N9 cases. The underlying biological mechanisms leading to this association deserve further exploration.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Distribution of the COVID-19 epidemic and correlation with population emigration from wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The ongoing new coronavirus pneumonia (Corona Virus Disease 2019, COVID-19) outbreak is spreading in China, but it has not yet reached its peak. Five million people emigrated from Wuhan before lockdown, potentially representing a source of virus infection. Determining case distribution and its correlation with population emigration from Wuhan in the early stage of the epidemic is of great importance for early warning and for the prevention of future outbreaks. METHODS: The official case report on the COVID-19 epidemic was collected as of January 30, 2020. Time and location information on COVID-19 cases was extracted and analyzed using ArcGIS and WinBUGS software. Data on population migration from Wuhan city and Hubei province were extracted from Baidu Qianxi, and their correlation with the number of cases was analyzed. RESULTS: The COVID-19 confirmed and death cases in Hubei province accounted for 59.91% (5806/9692) and 95.77% (204/213) of the total cases in China respectively. Hot spot provinces included Sichuan and Yunnan, which are adjacent to Hubei. The time risk of Hubei province on the following day was 1.960 times that on the previous day. The number of cases in some cities was relatively low, but the time risk appeared to be continuously rising. The correlation coefficient between the provincial number of cases and emigration from Wuhan was up to 0.943. The lockdown of 17 cities in Hubei province and the implementation of nationwide control measures efficiently prevented an exponential growth in the number of cases. CONCLUSIONS: The population that emigrated from Wuhan was the main infection source in other cities and provinces. Some cities with a low number of cases showed a rapid increase in case load. Owing to the upcoming Spring Festival return wave, understanding the risk trends in different regions is crucial to ensure preparedness at both the individual and organization levels and to prevent new outbreaks.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Modeling the Epidemic Dynamics and Control of COVID-19 Outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The coronavirus disease 2019 (COVID-19) is rapidly spreading over China and more than 30 countries in last two months. COVID-19 has multiple characteristics distinct from other infectious diseases, including a high infectivity during incubation, time delay between real dynamics and daily observed case numbers, and the effects from multiple quarantine and control measures. We develop a model SUQC to adequately characterizes the dynamics of COVID-19 and explicitly model the control by artificial measures, which is more suitable for analysis than other existing epidemic models.  The SUQC model is applied to the daily released data of the confirmed infected to analyze the outbreak of COVID-19 in Wuhan, Hubei (excluding Wuhan), China (excluding Hubei) and four first-tier cities of China. We find that, before January 30, 2020, all these regions except Beijing have a reproductive number R&gt;1, and after January 30, all regions have a reproductive number R&lt;1, indicating the effectiveness of the quarantine and control measures in inhibiting COVID-19. The confirmation rate of Wuhan is 0.0643, significantly lower than 0.1914 of Hubei (excluding Wuhan) and 0.2189 of China (excluding Hubei), but increases to 0.3229 after Feb 12th when clinical diagnosis was adopted. The un-quarantined infected individuals in Wuhan on February 12, 2020 is as high as 3,509 and decreases to 334 on February 21th, 2020.   After fitting the model with recent data, we predict that the end times of COVID-19 of Wuhan and Hubei are around late-March, of China (excluding Hubei) around mid-March, and of the four tier-one cities before March 2020. A total of 80,511 individuals of the whole country are infected, among which 49,510 are from Wuhan, 17,679 from Hubei(excluding Wuhan), and the rest 13,322 from other regions of China (excluding Hubei). We suggest the rigorous quarantine and control measures should be kept before March in Beijing, Shanghai, Guangzhou and Shenzhen, and before late-March in Hubei. The model can also be useful to predict the trend of epidemic and provide quantitative guide for other counties in a high risk of outbreak, such as South Korea, Japan and Iran.</jats:p></p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk Management of COVID-19 by Universities in China Short-term Forecasts of the COVID-19 Epidemic in Guangdong and Zhejiang, China: February 13–23, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid spread of new coronaviruses throughout China and the world in 2019&ndash;2020 has had a great impact on China&rsquo;s economic and social development. As the backbone of Chinese society, Chinese universities have made significant contributions to emergency risk management. Such contributions have been made primarily in the following areas: alumni resource collection, medical rescue and emergency management, mental health maintenance, control of staff mobility, and innovation in online education models. Through the support of these methods, Chinese universities have played a positive role in the prevention and control of the epidemic situation. However, they also face the problems of alumni&rsquo;s economic development difficulties, the risk of deadly infection to medical rescue teams and health workers, infection of teachers and students, and the unsatisfactory application of information technology in resolving the crisis. In response to these risks and emergency problems, we propose some corresponding solutions for public dissemination, including issues related to medical security, emergency research, professional assistance, positive communication, and hierarchical information-based teaching. The ongoing COVID-19 epidemic continues to spread within and outside of China, despite several social distancing measures implemented by the Chinese government. Limited epidemiological data are available, and recent changes in case definition and reporting further complicate our understanding of the impact of the epidemic, particularly in the epidemic&rsquo;s epicenter. Here we use previously validated phenomenological models to generate short-term forecasts of cumulative reported cases in Guangdong and Zhejiang, China. Using daily reported cumulative case data up until 13 February 2020 from the National Health Commission of China, we report 5- and 10-day ahead forecasts of cumulative case reports. Specifically, we generate forecasts using a generalized logistic growth model, the Richards growth model, and a sub-epidemic wave model, which have each been previously used to forecast outbreaks due to different infectious diseases. Forecasts from each of the models suggest the outbreaks may be nearing extinction in both Guangdong and Zhejiang; however, the sub-epidemic model predictions also include the potential for further sustained transmission, particularly in Zhejiang. Our 10-day forecasts across the three models predict an additional 65&ndash;81 cases (upper bounds: 169&ndash;507) in Guangdong and an additional 44&ndash;354 (upper bounds: 141&ndash;875) cases in Zhejiang by February 23, 2020. In the best-case scenario, current data suggest that transmission in both provinces is slowing down.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background Estimation of the fraction and contagiousness of undocumented novel coronavirus (COVID-19) infections is critical for understanding the overall prevalence and pandemic potential of this disease. Many mild infections are typically not reported and, depending on their contagiousness, may support stealth transmission and the spread of documented infection.   Methods Here we use observations of reported infection and spread within China in conjunction with mobility data, a networked dynamic metapopulation model and Bayesian inference, to infer critical epidemiological characteristics associated with the emerging coronavirus, including the fraction of undocumented infections and their contagiousness.   Results We estimate 86% of all infections were undocumented (95% CI: [82%-90%]) prior to the Wuhan travel shutdown (January 23, 2020). Per person, these undocumented infections were 52% as contagious as documented infections ([44%-69%]) and were the source of infection for two-thirds of documented cases. Our estimate of the reproductive number (2.23; [1.77-3.00]) aligns with earlier findings; however, after travel restrictions and control measures were imposed this number falls considerably. Conclusions A majority of COVID-19 infections were undocumented prior to implementation of control measures on January 23, and these undocumented infections substantially contributed to virus transmission. These findings explain the rapid geographic spread of COVID-19 and indicate containment of this virus will be particularly challenging. Our findings also indicate that heightened awareness of the outbreak, increased use of personal protective measures, and travel restriction have been associated with reductions of the overall force of infection; however, it is unclear whether this reduction will be sufficient to stem the virus spread.</jats:p></p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating the infected population and spread trend of 2019-nCov under different policy by EIR model</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported official data, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared.</jats:p></p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Active Monitoring of Persons Exposed to Patients with Confirmed COVID-19 — United States, January–February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, an outbreak of coronavirus disease 2019 (COVID-19), caused by the virus SARS-CoV-2, began in Wuhan, China (1). The disease spread widely in China, and, as of February 26, 2020, COVID-19 cases had been identified in 36 other countries and territories, including the United States. Person-to-person transmission has been widely documented, and a limited number of countries have reported sustained person-to-person spread.* On January 20, state and local health departments in the United States, in collaboration with teams deployed from CDC, began identifying and monitoring all persons considered to have had close contact† with patients with confirmed COVID-19 (2). The aims of these efforts were to ensure rapid evaluation and care of patients, limit further transmission, and better understand risk factors for transmission.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Backgrounds An ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number,R0, of 2019-nCoV in the early phase of the outbreak. Methods Accounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (γ), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI. Findings The early outbreak data largely follows the exponential growth. We estimated that the meanR0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0. Conclusion The mean estimate ofR0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>A Well Infant with Coronavirus Disease 2019 (COVID-19) with High Viral Load</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A well 6-month-old infant with coronavirus disease 2019 (COVID-19) had persistently positive nasopharyngeal swabs to day 16 of admission. This case highlights the difficulties in establishing the true incidence of COVID-19 as asymptomatic individuals can excrete the virus. These patients may play important roles in human-to-human transmission in the community.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical findings in a group of patients infected with the 2019 novel coronavirus (SARS-Cov-2) outside of Wuhan, China: retrospective case series</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective To study the clinical characteristics of patients in Zhejiang province, China, infected with the 2019 severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2) responsible for coronavirus disease 2019 (covid-2019).Design Retrospective case series.Setting Seven hospitals in Zhejiang province, China.Participants 62 patients admitted to hospital with laboratory confirmed SARS-Cov-2 infection. Data were collected from 10 January 2020 to 26 January 2020.Main outcome measures Clinical data, collected using a standardised case report form, such as temperature, history of exposure, incubation period. If information was not clear, the working group in Hangzhou contacted the doctor responsible for treating the patient for clarification.Results Of the 62 patients studied (median age 41 years), only one was admitted to an intensive care unit, and no patients died during the study. According to research, none of the infected patients in Zhejiang province were ever exposed to the Huanan seafood market, the original source of the virus; all studied cases were infected by human to human transmission. The most common symptoms at onset of illness were fever in 48 (77%) patients, cough in 50 (81%), expectoration in 35 (56%), headache in 21 (34%), myalgia or fatigue in 32 (52%), diarrhoea in 3 (8%), and haemoptysis in 2 (3%). Only two patients (3%) developed shortness of breath on admission. The median time from exposure to onset of illness was 4 days (interquartile range 3-5 days), and from onset of symptoms to first hospital admission was 2 (1-4) days.Conclusion As of early February 2020, compared with patients initially infected with SARS-Cov-2 in Wuhan, the symptoms of patients in Zhejiang province are relatively mild.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-29</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundsAn ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.  MethodsAccounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate ({gamma}), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI.  FindingsThe early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.  ConclusionThe mean estimate of R0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>2019-nCoV: new challenges from coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of pneumonia caused by the novel coronavirus 2019-nCoV in Wuhan, Hubei province of China, at the end of 2019 shaped tremendous challenges to China's public health and clinical treatment. The virus belongs to the β genus Coronavirus in the family Corornaviridae, and is closely related to SARS-CoV and MERS-CoV, causing severe symptoms of pneumonia. The virus is transmitted through droplets, close contact, and other means, and patients in the incubation period could potentially transmit the virus to other persons. According to current observations, 2019-nCoV is weaker than SARS in pathogenesis, but has stronger transmission competence; it's mechanism of cross-species spread might be related with angiotensin-converting enzyme Ⅱ (ACE2), which is consistent with the receptor SARS-CoV. After the outbreak of this disease, Chinese scientists invested a lot of energy to carry out research by developing rapid diagnostic reagents, identifying the characters of the pathogen, screening out clinical drugs that may inhibit the virus, and are rapidly developing vaccines. The emergence of 2019-nCoV reminds us once again of the importance of establishing a systematic coronavirus surveillance network. It also poses new challenges to prevention and control of the emerging epidemic and rapidly responses on scientific research.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Proposal for detection of 2019-nCoV nucleic acid in clinical laboratories</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December, the outbreak of a novel coronavirus (2019-nCoV) in Wuhan, China, has attracted extensive global attention. On January 20, 2020,the Chinese health authorities upgraded the coronavirus to a Class B infectious disease in the Law of the People's Republic of China on the Prevention and Treatment of Infectious Diseases , and considered it as Class A infectious diseases in disease control and prevention. On January 22, 2020, the 2019-nCoV nucleic acid detection test was listed as the diagnostic criteria in the 'guidelines for diagnosis and treatment of pneumonia due to 2019-nCoV (Trial Version 2)' . Therefore, standardizing the operation process of the 2019-nCoV nucleic acid detection in clinical laboratories has become a top priority. It is of paramount importance to establish standard protocols for detection of the 2019-nCoV nucleic acids in clinical laboratories to improve the reliability of the results and ensure the biosafety of laboratory personnel.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission characteristics of the COVID-19 outbreak in China: a study driven by data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-01</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days.  It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.</jats:p></p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Hospital Outbreak of Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In September 2012, the World Health Organization reported the first cases of pneumonia caused by the novel Middle East respiratory syndrome coronavirus (MERS-CoV). We describe a cluster of health care–acquired MERS-CoV infections. METHODS: Medical records were reviewed for clinical and demographic information and determination of potential contacts and exposures. Case patients and contacts were interviewed. The incubation period and serial interval (the time between the successive onset of symptoms in a chain of transmission) were estimated. Viral RNA was sequenced. RESULTS: Between April 1 and May 23, 2013, a total of 23 cases of MERS-CoV infection were reported in the eastern province of Saudi Arabia. Symptoms included fever in 20 patients (87%), cough in 20 (87%), shortness of breath in 11 (48%), and gastrointestinal symptoms in 8 (35%); 20 patients (87%) presented with abnormal chest radiographs. As of June 12, a total of 15 patients (65%) had died, 6 (26%) had recovered, and 2 (9%) remained hospitalized. The median incubation period was 5.2 days (95% confidence interval [CI], 1.9 to 14.7), and the serial interval was 7.6 days (95% CI, 2.5 to 23.1). A total of 21 of the 23 cases were acquired by person-to-person transmission in hemodialysis units, intensive care units, or in-patient units in three different health care facilities. Sequencing data from four isolates revealed a single monophyletic clade. Among 217 household contacts and more than 200 health care worker contacts whom we identified, MERS-CoV infection developed in 5 family members (3 with laboratory-confirmed cases) and in 2 health care workers (both with laboratory-confirmed cases). CONCLUSIONS: Person-to-person transmission of MERS-CoV can occur in health care settings and may be associated with considerable morbidity. Surveillance and infection-control measures are critical to a global public health response.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>2019-nCoV: new challenges from coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of pneumonia caused by the novel coronavirus 2019-nCoV in Wuhan, Hubei province of China, at the end of 2019 shaped tremendous challenges to China's public health and clinical treatment. The virus belongs to the &beta; genus Coronavirus in the family Corornaviridae, and is closely related to SARS-CoV and MERS-CoV, causing severe symptoms of pneumonia. The virus is transmitted through droplets, close contact, and other means, and patients in the incubation period could potentially transmit the virus to other persons. According to current observations, 2019-nCoV is weaker than SARS in pathogenesis, but has stronger transmission competence; it's mechanism of cross-species spread might be related with angiotensin-converting enzyme &#8545; (ACE2), which is consistent with the receptor SARS-CoV. After the outbreak of this disease, Chinese scientists invested a lot of energy to carry out research by developing rapid diagnostic reagents, identifying the characters of the pathogen, screening out clinical drugs that may inhibit the virus, and are rapidly developing vaccines. The emergence of 2019-nCoV reminds us once again of the importance of establishing a systematic coronavirus surveillance network. It also poses new challenges to prevention and control of the emerging epidemic and rapidly responses on scientific research.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Containing 2019-nCoV (Wuhan) coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus 2019-nCoV first appeared in December 2019 in Wuhan, China. While most of the initial cases were linked to the Huanan Seafood Wholesale Market, person-to-person transmission has been verified. Given that a vaccine cannot be developed and deployed for at least a year, preventing further transmission relies upon standard principles of containment, two of which are the isolation of known cases and the quarantine of persons believed at high risk of exposure. This note presents probability models for assessing the effectiveness of case isolation and quarantine within a community during the initial phase of an outbreak with illustrations based on early observations from Wuhan.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Investigation of a nosocomial outbreak of severe acute respiratory syndrome (SARS) in Toronto, Canada</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Severe acute respiratory syndrome (SARS) was introduced into Canada by a visitor to Hong Kong who returned to Toronto on Feb. 23, 2003. Transmission to a family member who was later admitted to a community hospital in Toronto led to a large nosocomial outbreak. In this report we summarize the preliminary results of the epidemiological investigation into the transmission of SARS between 128 cases associated with this hospital outbreak. METHODS: We collected epidemiologic data on 128 probable and suspect cases of SARS associated with the hospital outbreak, including those who became infected in hospital and the next generation of illness arising among their contacts. Incubation periods were calculated based on cases with a single known exposure. Transmission chains from the index family to hospital contacts and within the hospital were mapped. Attack rates were calculated for nurses in 3 hospital wards where transmission occurred. RESULTS: The cases ranged in age from 21 months to 86 years; 60.2% were female. Seventeen deaths were reported (case-fatality rate 13.3%). Of the identified cases, 36.7% were hospital staff. Other cases were household or social contacts of SARS cases (29.6%), hospital patients (14.1%), visitors (14.1%) or other health care workers (5.5%). Of the 128 cases, 120 (93.8%) had documented contact with a SARS case or with a ward where there was a known SARS case. The remaining 8 cases without documented exposure are believed to have had exposure to an unidentified case and remain under investigation. The attack rates among nurses who worked in the emergency department, intensive care unit and coronary care unit ranged from 10.3% to 60.0%. Based on 42 of the 128 cases with a single known contact with a SARS case, the mean incubation period was 5 days (range 2 to 10 days). INTERPRETATION: Evidence to date suggests that SARS is a severe respiratory illness spread mainly by respiratory droplets. There has been no evidence of further transmission within the hospital after the elapse of 2 full incubation periods (20 days).</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China.  Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity.  Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country.  Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.</jats:p></p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Domestic Poultry and SARS Coronavirus, Southern China</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">SARS coronavirus injected intratracheally into chickens, turkeys, geese, ducks, and quail, or into the allantoic sac of their embryonating eggs, failed to cause disease or replicate. This finding suggests that domestic poultry were unlikely to have been the reservoir, or associated with dissemination, of SARS coronavirus in the animal markets of southern China.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">1The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Scientists question China’s decision not to report symptom-free coronavirus cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Researchers say that excluding these people could conceal the epidemic’s true extent, but others say the practice makes sense. Researchers are concerned that China’s official reports on the number of coronavirus infections have not been including people who have tested positive for the virus but who have no symptoms. They fear the practice is masking the epidemic’s true scale. But public health experts say China is right to prioritize tracking sick patients who are spreading the disease.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Intrinsic growth rules of patients infected, dead and cured with 2019 novel coronavirus in mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background An outbreak of a novel coronavirus (SARS-CoV-2)-infected pneumonia (COVID-19) was first occurred in Wuhan, China, in December 2019 and then spread rapidly to other regions. It has been declared that at least one confirmed case infected by SARS-CoV-2 was found in each province of China by late January 2020. Methods We collected the time series data of the cumulative number of confirmed infected, dead, and cured cases from the health commissions in 31 provinces in mainland China. A simple descriptive model in a logistic form was formulated to infer the intrinsic epidemic rules of COVID-19. Furthermore, we compared the intrinsic epidemic rules of COVID-19 in Hubei with that of the severe acute respiratory syndrome (SARS) in Beijing, which was obtained from the Ministry of Public Health of China in 2003.  Results The feature of data of three population groups in all the provinces could be well captured (R-square&gt;0.95) by the proposed descriptive model with slight variation among different provinces. By comparing fitted parameters, we found that the fitted curve of the infected case is the earliest to be saturated and has the lowest semi-saturation period in both COVID-19 and SARS. While all three population groups of SARS are later to saturate and have a longer semi-saturation period than that of COVID-19. Conclusions Our model is robust and stable to depict the intrinsic growth rule for the cumulative number of confirmed infected, dead, and cured cases in 30 provinces, which could depict the epidemic situation in mainland China very well. Despite the virus caused SARS (SARS-CoV) and the virus caused COVID-19 (SARS-CoV-2) are homologous, their epidemiologic characteristics are quite different. The duration of the outbreak would be shorter for COVID-19.</jats:p></p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: global infections pass 90,000</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Updates on the respiratory illness that has infected tens of thousands of people. The number of people worldwide who have been infected with the coronavirus has passed 90,000. More than 3,000 people have died since the outbreak began in December. The vast majority of cases — more than 80,000 — have occurred in China, but around 60 other countries are now also dealing with outbreaks. Many nations are preparing for a global pandemic, as reports of cases caused by community spread — rather than importation from China — rise. South Korea, Italy and Iran are fighting the largest outbreaks outside of China.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>The Effects of "Fangcang, Huoshenshan, and Leishenshan" Makeshift Hospitals and Temperature on the Mortality of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background  In December 2019, a novel coronavirus disease (COVID-19) broke out in Wuhan, China, however, the factors affecting the mortality remain unclear. Methods  Thirty-two days of data that were shared by China National Health Commission and China Weather Net were collected using standard forms. The difference in the mortality of confirmed and severe cases before and after the use of Fangcang, Huoshenshan, and Leishenshan makeshift hospitals (MSHs) was tested using Mann-Whitney U test. We also studied whether air temperature (AT) could affect the above outcomes of COVID-19 cases by performing Spearman analysis. Results The mortality of confirmed cases was significantly decreased both in Wuhan (U = 1, P &lt; 0.001) and Hubei (U = 0, P &lt; 0.001), while in non-Hubei regions, as a contrast, the mortality of confirmed cases remained unchanged (U = 40, P = 0.139). However, another eight days later, changes in the mortality in non-Hubei regions also became significant (U = 73, P = 0.039). Mortality of confirmed cases was found to be significantly correlated with temperature both in Wuhan (r = -0.441, P = 0.012) and Hubei (r = -0.440, P = 0.012). Conclusions Our findings indicated that both the use of MSHs and the rise of AT were beneficial to the survival of COVID-19 cases.</jats:p></p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-associated Coronavirus Transmission, United States</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">To better assess the risk for transmission of the severe acute respiratory syndrome–associated coronavirus (SARS-CoV), we obtained serial specimens and clinical and exposure data from seven confirmed U.S. SARS patients and their 10 household contacts. SARS-CoV was detected in a day-14 sputum specimen from one case-patient and in five stool specimens from two case-patients. In one case-patient, SARS-CoV persisted in stool for at least 26 days after symptom onset. The highest amounts of virus were in the day-14 sputum sample and a day-14 stool sample. Residual respiratory symptoms were still present in recovered SARS case-patients 2 months after illness onset. Possible transmission of SARS-CoV occurred in one household contact, but this person had also traveled to a SARS-affected area. The data suggest that SARS-CoV is not always transmitted efficiently. Laboratory diagnosis of SARS-CoV infection is difficult; thus, sputum and stool specimens should be included in the diagnostic work-up for SARS-CoV infection.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis on the epidemic factors for the Corona Virus Disease</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, corona virus disease 2019 (COVID-19) , an emerging infection disease occurred in Wuhan, has spread in the mainland China. The epidemic factors on the basis of knowledge of SARS-CoV-2 were discussed in this paper. This puts a lot of pressure on clinical resources and care. SARS-CoV-2 is a novel corona virus, the onset of COVID-19 is slow, and the pathogenesis of SARS-CoV-2 remains unclear and may lead to multiple organ damage. These put a lot of pressure on clinical resources and care. Source of infection including the patients, asymptomatic carrier and patients in the incubation period are contagious. It is difficult to control source of infection. Routes of SARS-CoV-2 transmission are diversified and the main routes of transmission for COVID-19 are droplet transmission and close contact transmission. All population have susceptibility to SARS-CoV-2. Social factors such population movements and aggregation accelerated the spread of SARS-CoV-2. The Chinese government's adopted measures are positive and effective, and are accepted by the expert group from the World Health Organization. However, it will be a long-term hard work in the future to seriously summarize and think deeply to achieve public health security in China.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>The Effects of Temperature and Relative Humidity on the Viability of the SARS Coronavirus</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Oct 1</span></i><p style="text-align: justify; text-justify: inter-word;">The main route of transmission of SARS CoV infection is presumed to be respiratory droplets. However the virus is also detectable in other body fluids and excreta. The stability of the virus at different temperatures and relative humidity on smooth surfaces were studied. The dried virus on smooth surfaces retained its viability for over 5 days at temperatures of 22–25°C and relative humidity of 40–50%, that is, typical air-conditioned environments. However, virus viability was rapidly lost (>3 log(10)) at higher temperatures and higher relative humidity (e.g., 38°C, and relative humidity of >95%). The better stability of SARS coronavirus at low temperature and low humidity environment may facilitate its transmission in community in subtropical area (such as Hong Kong) during the spring and in air-conditioned environments. It may also explain why some Asian countries in tropical area (such as Malaysia, Indonesia or Thailand) with high temperature and high relative humidity environment did not have major community outbreaks of SARS.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>The effect of travel restrictions on the spread of the 2019 novel coronavirus (2019-nCoV) outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Motivated by the rapid spread of a novel coronavirus (2019-nCoV) in Mainland China, we use a global metapopulation disease transmission model to project the impact of both domestic and international travel limitations on the national and international spread of the epidemic. The model is calibrated on the evidence of internationally imported cases before the implementation of the travel quarantine of Wuhan. By assuming a generation time of 7.5 days, the reproduction number is estimated to be 2.4 [90% CI 2.2-2.6]. The median estimate for number of cases before the travel ban implementation on January 23, 2020 is 58,956 [90% CI 40,759 - 87,471] in Wuhan and 3,491 [90% CI 1,924 - 7,360] in other locations in Mainland China.  The model shows that as of January 23, most Chinese cities had already received a considerable number of infected cases, and the travel quarantine delays the overall epidemic progression by only 3 to 5 days. The travel quarantine has a more marked effect at the international scale, where we estimate the number of case importations to be reduced by 80% until the end of February. Modeling results also indicate that sustained 90% travel restrictions to and from Mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher reduction of transmission in the community.</jats:p></p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the daily trend in the size of the COVID-19 infected population in Wuhan</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>There has been an outbreak of coronavirus disease (COVID-19) in Wuhan city, Hubei province, China since December 2019. Cases have been exported to other parts of China and more than 20 countries. We provide estimates of the daily trend in the size of the epidemic in Wuhan based on detailed information of 10,940 confirmed cases outside Hubei province.</jats:p></p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of COVID-2019 burden and potential for international dissemination of infection from Iran</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The Coronavirus Disease 2019 (COVID-19) epidemic began in Wuhan, China in late 2019 and continues to spread globally, with exported cases confirmed in 28 countries at the time of writing. During the interval between February 19 and 23, 2020, Iran reported its first 43 cases with eight deaths.  Three exported cases originating in Iran were identified, suggesting a underlying burden of disease in that country than is indicated by reported cases.  A large epidemic in Iran could further fuel global dissemination of COVID-19.  We sought to estimate COVID-19 outbreak size in Iran based on known exported case counts and air travel links between Iran and other countries, and to anticipate where infections originating in Iran may spread to next.  We assessed interconnectivity between Iran and other countries using using International Air Transport Association (IATA) data.  We used the methods of Fraser et al. to estimate the size of the underlying epidemic that would result in cases being observed  in the United Arab Emirates (UAE), Lebanon, and Canada.  Time at risk estimates were based on a presumed 6 week epidemic age, and length of stay data for visitors to Iran derived from the United Nations World Tourism Organization (UNWTO).  We evaluated the relationship between the strength of travel links with Iran, and destination country rankings on the Infectious Disease Vulnerability Index (IDVI), a validated metric that estimates the capacity of a country to respond to an infectious disease outbreak . Scores range between 0-1, with higher scores reflecting greater capacity to manage infectious outbreaks. UAE, Lebanon, and Canada ranked 3rd, 21st, and 31st, respectively, for outbound air travel volume from Iran in February 2019. We estimated that 18,300 (95% confidence interval: 3770 to 53,470) COVID-19 cases would have had to occur in Iran, assuming an outbreak duration of 1.5 months in the country, in order to observe these three internationally exported cases reported at the time of writing.  Results were robust under varying assumptions about undiagnosed case numbers in Syria, Azerbaijan and Iraq.  Even if it were assumed that all cases were identified in all countries with certainty, the "best case" outbreak size was substantial (1820, 95% CI: 380-5320 cases), and far higher than reported case counts.  Given the low volumes of air travel to countries with identified cases of COVID-19 with origin in Iran (such as Canada), it is likely that Iran is currently experiencing a COVID-19 epidemic of significant size for such exportations to be occurring. This is concerning, both for public health in Iran itself, and because of the high likelihood for outward dissemination of the epidemic to neighbouring countries with lower capacity to respond to infectious diseases epidemics.</jats:p></p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /></body>