
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: basic reproductive number of SARS-CoV-2</h2><br /></div><br /><br /><br /><div class="card"><h3>The within-host viral kinetics of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</p>
<p align="right"><i>score: 159</i></p>

</div><br /><br /><br /><div class="card"><h3>Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number Ro -- the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of Ro vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of Ro across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate $r$, the mean generation interval $\bar G$, and the generation-interval dispersion $\kappa$. We then apply our framework to early estimates of Ro for the SARS-CoV-2 outbreak. We show that many early Ro estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of Ro, including the shape of the generation-interval distribution, in efforts to estimate Ro at the outset of an epidemic.</jats:p></p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>A mathematical model for simulating the phase-based transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus. METHODS: In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from Huanan Seafood Wholesale Market (reservoir) to people, we simplified the model as Reservoir-People (RP) transmission network model. The next generation matrix approach was adopted to calculate the basic reproduction number (R(0)) from the RP model to assess the transmissibility of the SARS-CoV-2. RESULTS: The value of R(0) was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.58. CONCLUSIONS: Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>A mathematical model for simulating the phase-based transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus. Methods In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from Huanan Seafood Wholesale Market (reservoir) to people, we simplified the model as Reservoir-People (RP) transmission network model. The next generation matrix approach was adopted to calculate the basic reproduction number (R 0) from the RP model to assess the transmissibility of the SARS-CoV-2. Results The value of R 0 was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.58. Conclusions Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating the secondary transmission pattern and epidemic prediction of the COVID-19 in metropolitan areas of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Understanding the transmission dynamics of COVID-19 is crucial for evaluating the spread pattern of it, especially in metropolitan areas of China which may cause secondary outbreaks outside Wuhan, the center of the new coronavirus disease outbreak. We used reported data from Jan 24, 2020, to Feb 23, 2020, fitted the model of infection, and on the number of cases reported to estimate likely number of infections in four high risk metropolitan areas, as well as facilitate understanding the COVID-19's spread pattern. A group of SERI model statistical parameters were estimated using Markov Chain Monte Carlo (MCMC) methods, and our modeling integrated the effect of the official quarantine regulation and travel restriction of China. As a result, we estimated that the basic reproductive number R0 ​is 3.11 in Beijing, 2.78 in Shanghai, 2.02 in Guangzhou, and 1.75 in Shenzhen. In addition, we inferred the prediction results and compared the results of different level of parameters, For example, In Beijing, the predicated peak number of cases is around 466 at the peak time Feb 29, 2020; however, when the city conducts different levels (strict, mild, or weak) of travel restrictions or regulation measures, the estimation results show that transmission dynamics will change and the peek number of cases shows the changing proportion is between 56%~159%. We concluded that public health interventions would reduce the risks of COVID-19 spreading and more rigorous control and prevention measures will effectively contain its further spread, but risk increases when businesses and social activities returning back before the ending date. Besides, the experiences gained and lessons learned from China are potential to provide evidences supporting for other metropolitan areas and big cities with emerging cases outside China.</jats:p></p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and corona virus disease-2019 (COVID-19): the epidemic and the challenges</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">ABSTRACT Emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019, has caused a large global outbreak and a major public health issue. As of February 11, 2020, data from the WHO has shown that more than 43,000 confirmed cases have been identified in 28 countries/regions, with more than 99% of the cases being detected in China. On January 30, 2020, WHO has declared COVID-19 as the sixth public health emergency of international concern. The SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24-3.58. Among the patients with pneumonia caused by the SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground glass opacity was the most common finding from computerized tomography images of the chest. Although the one case of SARS-CoV-2 pneumonia in the United States responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of the SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>A new transmission route for the propagation of the SARS-CoV-2 coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.</jats:p></p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>The reproductive number of COVID-19 is higher compared to SARS coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Teaser: Our review found the average R0 for 2019-nCoV to be 3.28, which exceeds WHO estimates of 1.4 to 2.5.Here we review the basic reproduction number (R0) of the 2019-nCoV virus. R0 is an indication of the transmissibility of a virus, representing the average number of new infections generated by an infectious person in a totally naïve population. For R0 greater than one the number infected is likely to increase, and for R0 less than one transmission is likely to die out. The basic reproduction number is a central concept in infectious disease epidemiology, indicating the risk of an infectious agent with respect to epidemic spread.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19 A mathematical model for simulating the phase-based transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125?nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is >?96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1].ID - Fisher2020 As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential impact of seasonal forcing on a SARS-CoV-2 pandemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.</jats:p></p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Early dynamics of transmission and control of COVID-19: a mathematical modelling study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: An outbreak of the novel coronavirus SARS-CoV-2 has led to 46,997 confirmed cases as of 13th February 2020. Understanding the early transmission dynamics of the infection and evaluating the effectiveness of control measures is crucial for assessing the potential for sustained transmission to occur in new areas.   Methods: We combined a stochastic transmission model with data on cases of novel coronavirus disease (COVID-19) in Wuhan and international cases that originated in Wuhan to estimate how transmission had varied over time during January and February 2020. Based on these estimates, we then calculated the probability that newly introduced cases might generate outbreaks in other areas.   Findings: We estimated that the median daily reproduction number, Rt , declined from 2.35 (95% CI: 1.15-4.77) one week before travel restrictions were introduced on 23rd January to 1.05 (95% CI: 0.413-2.39) one week after. Based on our estimates of Rt,we calculated that in locations with similar transmission potential as Wuhan in early January, once there are at least four independently introduced cases, there is a more than 50% chance the infection will establish within that population.   Interpretation: Our results show that COVID-19 transmission likely declined in Wuhan during late January 2020, coinciding with the introduction of control measures. As more cases arrive in international locations with similar transmission potential to Wuhan pre-control, it is likely many chains of transmission will fail to establish initially, but may still cause new outbreaks eventually.</jats:p></p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological research priorities for public health control of the ongoing global novel coronavirus (2019-nCoV) outbreak</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Infections with 2019-nCoV can spread from person to person, and in the earliest phase of the outbreak the basic reproductive number was estimated to be around 2.2, assuming a mean serial interval of 7.5 days [2]. The serial interval was not precisely estimated, and a potentially shorter mean serial interval would have corresponded to a slightly lower basic reproductive number. Control measures and changes in population behaviour later in January should have reduced the effective reproductive number. However, it is too early to estimate whether the effective reproductive number has been reduced to below the critical threshold of 1 because cases currently being detected and reported would have mostly been infected in mid- to late-January. Average delays between infection and illness onset have been estimated at around 5–6 days, with an upper limit of around 11-14 days [2,5], and delays from illness onset to laboratory confirmation added a further 10 days on average [2].</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Study on SARS-COV-2 transmission and the effects of control measures in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.</jats:p></p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control.  Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk.   Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 32 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days.   Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.</jats:p></p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the basic reproduction number of COVID-19 in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: The number of confirmed and suspected cases of the COVID-19 in Hubei province is still increasing. However, the estimations of the basic reproduction number of COVID-19 varied greatly across studies. The objectives of this study are 1) to estimate the basic reproduction number (R(0)) of COVID-19 reflecting the infectiousness of the virus and 2) to assess the effectiveness of a range of controlling intervention. Method: The reported number of daily confirmed cases from January 17 to February 8, 2020 in Hubei province were collected and used for model fit. Four methods, the exponential growth (EG), maximum likelihood estimation (ML), sequential Bayesian method (SB) and time dependent reproduction numbers (TD), were applied to estimate the R(0). Result: Among the four methods, the EG method fitted the data best. The estimated R(0) was 3.49 (95% CI: 3.42-3.58) by using EG method. The R(0) was estimated to be 2.95 (95%CI: 2.86-3.03) after taking control measures. Conclusion: In the early stage of the epidemic, it is appropriate to estimate R(0) using the EG method. Meanwhile, timely and effective control measures were warranted to further reduce the spread of COVID-19.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.</jats:p></p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the scale of COVID-19 Epidemic in the United States: Simulations Based on Air Traffic directly from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Introduction: Coronavirus Disease 2019 (COVID-19) infection has been characterized by rapid spread and unusually large case clusters. It is important to have an estimate of the current state of COVID-19 epidemic in the U.S. to develop informed public health strategies.  Methods:  We estimated the potential scale of the COVID-19 epidemic (as of 03/01/2020) in the U.S. from cases imported directly from Wuhan area. We used simulations based on transmission dynamics parameters estimated from previous studies and air traffic data from Wuhan to the U.S and deliberately built our model based on conservative assumptions. Detection and quarantine of individual COVID-19 cases in the U.S before 03/01/2020 were also taken into account. We. A SEIR model was used to simulate the growth of the number of infected individuals in Wuhan area and in the U.S.  Results: With the most likely model, we estimated that there would be 9,484 infected cases (90%CI 2,054-24,241) as of 03/01/2020 if no intervention procedure had been taken to reduce the transmissibility in unidentified cases. Assuming current preventive procedures have reduced 25% of the transmissibility in unidentified cases, the number of infected cases would be 1,043 (90%CI 107-2,474). Conclusion: Our research indicates that, as of 03/01/2020., it is likely that there are already thousands of individuals in the US infected with SARS-CoV-2. Our model is dynamic and is available to the research community to further evaluate as the  situation becomes clearer.</jats:p></p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the serial interval of the novel coronavirus disease (COVID-19): A statistical analysis using the public data in Hong Kong from January 16 to February 15, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Backgrounds: The emerging virus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has caused a large outbreak of novel coronavirus disease (COVID-19) in Wuhan, China since December 2019. Based on the publicly available surveillance data, we identified 21 transmission chains in Hong Kong and estimated the serial interval (SI) of COVID-19. Methods: Index cases were identified and reported after symptoms onset, and contact tracing was conducted to collect the data of the associated secondary cases. An interval censored likelihood framework is adopted to fit a Gamma distribution function to govern the SI of COVID-19.  Findings: Assuming a Gamma distributed model, we estimated the mean of SI at 4.4 days (95%CI: 2.9−6.7) and SD of SI at 3.0 days (95%CI: 1.8−5.8) by using the information of all 21 transmission chains in Hong Kong.  Conclusion: The SI of COVID-19 may be shorter than the preliminary estimates in previous works. Given the likelihood that SI could be shorter than the incubation period, pre-symptomatic transmission may occur, and extra efforts on timely contact tracing and quarantine are recommended in combating the COVID-19 outbreak.</jats:p></p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Duration of quarantine in hospitalized patients with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection: a question needing an answer</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019 a new form of pneumonia was observed in the Chinese province of Hubei.[1] The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was subsequently identified as responsible of this condition, defined coronavirus disease (COVID-19).[2] The virus has now spread outside Chinese borders with 82,297 cases and 2,804 deaths worldwide at the 26th of February.[3] After infection, symptoms appear after an incubation time of 3-5 days, with 80% of those infected developing a mild disease, 15% a severe disease and 5% will require support in intensive care unit (ICU).[4] Overall, the estimated case-fatality rate is comprised between 0.4% and 2.9% and the basic reproduction number is approximately 3.28.[4,5] SARS-CoV-2 is a new pathogen for humankind and any type of immune protection exist, thus everybody can be potentially infected. Moreover, no primary prophylaxis measures (vaccination) nor effective treatments are available. If the above represented percentages are applied to the worldwide populations, it appears clear why any measure should be considered to avoid a further diffusion of the virus and prevent the saturation and collapse of health systems and the most catastrophic pandemic since 1919 Spanish flu. Isolation of those affected and the use of personal protective equipment (PPE) are the mainstay to block transmission of this pathogen, which is presumed through respiratory droplets. A 14 days quarantine is applied to subjects coming from endemic areas or who had contact with confirmed cases. It is assumed that, if in this period the subject does not develop any sign or symptoms compatible with COVID-19, he is not infected and thus the quarantine can be removed, and the subject returned to the community. Domiciliary quarantine of 14 days since a positive test is applied also for patients with a diagnosed mild disease who did not need medical support. These rules are effective in controlling infections in the community, but several doubts arise when it is necessary to transpose them in the hospital setting. Hospitals are indeed a delicate place in epidemics: they collect fragile persons who can be exposed to the virus and are subsequently readmitted to the community thus spreading the infection. Indeed, the ongoing outbreak in Northern Italy has been linked to a single infected patient who accessed to a community hospital where he transmitted the virus to several other patients and health-care operators.[6] Moreover, the isolation of patients in the hospital setting impose a significant burden in terms of PPE used by the health-care operators, space dedicated and time employed in their management. Even more complex is the situation of patients in ICU, where viral spreading is facilitated by endotracheal tubes and manoeuvres performed on the respiratory tract. Therefore, a clear definition of the infectiousness timing and intensity of viral spreading is mandatory to alleviate the burden on the health-care system. Unfortunately, the data available on the topic are scarce and composed only of measurements of viral shedding, without an assessment of the infectivity. Kim et al.[7] assessed the viral load kinetics of SARS-CoV-2 in upper and lower respiratory tract materials in the first two confirmed patients in Korea. They employed real-time reverse transcriptase polymerase chain reaction (rRT-PCR) to detect SARS-CoV-2 and converted cycle threshold (CT) values of rRT-PCR into RNA copy number. The detection limit of rRT-PCR was 2,690 copies/mL. Overall, viral load above detection limit was detected until 14 and 25 days after symptoms onset and for 13 and 11 days after the first detection, respectively.[7] Of note, both patients received treatment with lopinavir/ritonavir. Instead, Zou and colleagues analysed viral load in repeated nasal and throat swabs obtained from the 17 symptomatic patients.[8] They also employed rRT-PCR and considered a CT of 40 as detection limit. Higher viral loads were observed in nasal </p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Since first identified, the epidemic scale of the recently emerged novel coronavirus (2019-nCoV) in Wuhan, China, has increased rapidly, with cases arising across China and other countries and regions. using a transmission model, we estimate a basic reproductive number of 3.11 (95%CI, 2.39-4.13); 58-76% of transmissions must be prevented to stop increasing; Wuhan case ascertainment of 5.0% (3.6-7.4); 21022 (11090-33490) total infections in Wuhan 1 to 22 January.</jats:p></p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterizing the transmission and identifying the control strategy for COVID-19 through epidemiological modeling</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended.</jats:p></p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Beyond R0: the importance of contact tracing when predicting epidemics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The basic reproductive number --- R<jats:sub>0</jats:sub> --- is one of the most common and most commonly misapplied numbers in public health.  Nevertheless, estimating R<jats:sub>0</jats:sub> for every transmissible pathogen, emerging or endemic, remains a priority for epidemiologists the world over.  Although often used to compare outbreaks and forecast pandemic risk, this single number belies the complexity that two different pathogens can exhibit, even when they have the same R<jats:sub>0</jats:sub>. Here, we show how predicting outbreak size requires both an estimate of R<jats:sub>0</jats:sub> and an estimate of the heterogeneity in the number of secondary infections.  To facilitate rapid determination of outbreak risk, we propose a reformulation of a classic result from random network theory that relies on contact tracing data to simultaneously determine the first moment (R<jats:sub>0</jats:sub>) and the higher moments (representing the heterogeneity) in the distribution of secondary infections.  Further, we show how this framework is robust in the face of the typically limited amount of data for emerging pathogens. Lastly, we demonstrate that without data on the heterogeneity in secondary infections for emerging pathogens like 2019-nCoV, the uncertainty in outbreak size ranges dramatically, in the case of 2019-nCoV from 5-40% of susceptible individuals. Taken together, our work highlights the critical need for contact tracing during emerging infectious disease outbreaks and the need to look beyond R<jats:sub>0</jats:sub> when predicting epidemic size.</jats:p></p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of the epidemic properties of the 2019 novel coronavirus: A mathematical modeling study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The 2019 novel Coronavirus (COVID-19) emerged in Wuhan, China in December 2019 and has been spreading rapidly in China. Decisions about its pandemic threat and the appropriate level of public health response depend heavily on estimates of its basic reproduction number and assessments of interventions conducted in the early stages of the epidemic.  Methods We conducted a mathematical modeling study using five independent methods to assess the basic reproduction number (R0) of COVID-19, using data on confirmed cases obtained from the China National Health Commission for the period 10th January to 8th February. We analyzed the data for the period before the closure of Wuhan city (10th January to 23rd January) and the post-closure period (23rd January to 8th February) and for the whole period, to assess both the epidemic risk of the virus and the effectiveness of the closure of Wuhan city on spread of COVID-19.  Findings Before the closure of Wuhan city the basic reproduction number of COVID-19 was 4.38 (95% CI: 3.63-5.13), dropping to 3.41 (95% CI: 3.16-3.65) after the closure of Wuhan city. Over the entire epidemic period COVID-19 had a basic reproduction number of 3.39 (95% CI: 3.09-3.70), indicating it has a very high transmissibility.  Interpretation COVID-19 is a highly transmissible virus with a very high risk of epidemic outbreak once it emerges in metropolitan areas. The closure of Wuhan city was effective in reducing the severity of the epidemic, but even after closure of the city and the subsequent expansion of that closure to other parts of Hubei the virus remained extremely infectious. Emergency planners in other cities should consider this high infectiousness when considering responses to this virus.</jats:p></p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Intrinsic growth rules of patients infected, dead and cured with 2019 novel coronavirus in mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background An outbreak of a novel coronavirus (SARS-CoV-2)-infected pneumonia (COVID-19) was first occurred in Wuhan, China, in December 2019 and then spread rapidly to other regions. It has been declared that at least one confirmed case infected by SARS-CoV-2 was found in each province of China by late January 2020. Methods We collected the time series data of the cumulative number of confirmed infected, dead, and cured cases from the health commissions in 31 provinces in mainland China. A simple descriptive model in a logistic form was formulated to infer the intrinsic epidemic rules of COVID-19. Furthermore, we compared the intrinsic epidemic rules of COVID-19 in Hubei with that of the severe acute respiratory syndrome (SARS) in Beijing, which was obtained from the Ministry of Public Health of China in 2003.  Results The feature of data of three population groups in all the provinces could be well captured (R-square&gt;0.95) by the proposed descriptive model with slight variation among different provinces. By comparing fitted parameters, we found that the fitted curve of the infected case is the earliest to be saturated and has the lowest semi-saturation period in both COVID-19 and SARS. While all three population groups of SARS are later to saturate and have a longer semi-saturation period than that of COVID-19. Conclusions Our model is robust and stable to depict the intrinsic growth rule for the cumulative number of confirmed infected, dead, and cured cases in 30 provinces, which could depict the epidemic situation in mainland China very well. Despite the virus caused SARS (SARS-CoV) and the virus caused COVID-19 (SARS-CoV-2) are homologous, their epidemiologic characteristics are quite different. The duration of the outbreak would be shorter for COVID-19.</jats:p></p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</jats:p></p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>A mathematical model for estimating the age-specific transmissibility of a novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: A novel coronavirus named as "SARS-CoV-2" has spread widely in many countries since December 2019, especially in China. This study aimed to quantify the age-specific transmissibility by using a mathematical model. Methods: An age-specific susceptible - exposed - symptomatic - asymptomatic - recovered - seafood market (SEIARW) model was developed based on two suspected transmission routes (from market to person and person to person). The susceptible people from Wuhan City were divided into different age groups. We used the subscript i and j to represent age group 1 to 4 (1: &lt;= 14 years; 2: 15-44 years; 3: 45-64 years; 4: &gt;= 65 years) and 1 to 5 (1: &lt;= 5 years; 2: 6-14 years; 3: 15-24 years; 4: 25-59 years; 4: &gt;= 60 years), respectively. Data of reported COVID-19 cases were collected from one published literature from 26 November to 22 December, 2019 in Wuhan City, China. The age-specific transmissibility of the virus was estimated accordingly secondary attack rate (SAR).  Results: The age-specific SEIARW model fitted with the reported data well by dividing the population into four age groups (χ2 = 4.99 × 10-6, P &gt; 0.999), and five age groups (χ2 = 4.85 × 10-6, P &gt; 0.999). Based on the four-age-group SEIARW model, the highest transmissibility occurred from age group 2 to 3 (SAR23 = 17.56 per 10 million persons), followed by from age group 3 to 2 (SAR32 = 10.17 per 10 million persons). The lowest transmissibility occurred from age group 1 to 2 (SAR12 = 0.002 per 10 million persons). Based on the five-age-group SEIARW model, the highest transmissibility occurred from age group 4 to 5 (SAR45 = 12.40 per 10 million persons), followed by from age group 5 to 4 (SAR54 = 6.61 per 10 million persons). The lowest transmissibility occurred from age group 3 to 4 (SAR34 = 0.0002 per 10 million persons). Conclusions: SARS-CoV-2 has high transmissibility among adults and elder people but low transmissibility among children and young people.</jats:p></p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country.  Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival &amp; traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% &amp; 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.</jats:p></p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of country-level basic reproductive ratios for novel Coronavirus (COVID-19) using synthetic contact matrices</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The outbreak of novel coronavirus (COVID-19) has the potential for global spread, infecting large numbers in all countries. In this case, estimating the country-specific basic reproductive ratio is a vital first step in public-health planning. The basic reproductive ratio (<jats:italic>R</jats:italic><jats:sub>0</jats:sub>) is determined by both the nature of pathogen and the network of contacts through which the disease can spread - with this network determined by socio-demographics including age-structure and household composition. Here we focus on the age-structured transmission within the population, using data from China to inform age-dependent susceptibility and synthetic age-mixing matrices to inform the contact network. This allows us to determine the country-specific basic reproductive ratio as a multiplicative scaling of the value from China. We predict that <jats:italic>R</jats:italic><jats:sub>0</jats:sub> will be highest across Eastern Europe and Japan, and lowest across Africa, Central America and South-Western Asia. This pattern is largely driven by the ratio of children to older adults in each country and the observed propensity of clinical cases in the elderly.</jats:p></p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Has China faced only a herald wave of SARS-CoV-2?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The attack rate of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) calculated by mathematical models, from estimates of the basic reproduction number, R0, of 2–3, suggests that 50–60% of the population should eventually be infected because the population seems to be entirely naive to the new virus.1 The observed attack rate on board the Diamond Princess cruise ship remained slightly below 20% (705 of 3711 passengers and crew members became infected).1 It is of upmost importance to know whether the SARS-CoV-2 outbreak in China is subsiding, as local authorities and the entire international community might wish. With 80 026 COVID-19 cases officially reported from China as of March 2, 2020,2 the proportion of the population affected remains far from 50%, or even 20%, of China's 1·4 billion people. Has China just experienced a herald wave, to use terminology borrowed from those who study tsunamis, and is the big wave still to come? Serosurveys can help answer these questions precisely.3 To serosurvey the outbreak would involve testing sera of blood samples from the most representative sample of the population at the epicentre of the epidemic, Wuhan. Serology analysis with neutralising antibodies from the 1000 people could allow for the rate of SARS-CoV-2 infections to be estimated with good accuracy. This rate could be extrapolated to the city's entire population and thus inform more precisely whether the provisional attack rate during this period was a few cases per thousand or perhaps affected 1–2% of the population, 20%, or more. Serosurveys should be seen as polls before elections; they can be repeated several times,3 week after week, to monitor the epidemic precisely. There is no reason to wait for the end of the epidemic before doing serosurveys. The results would be tremendously informative to China, first and foremost, and to the entire international community, on the risk of big secondary epidemic waves</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Phase adjusted estimation of the number of 2019 novel coronavirus cases in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV / SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of Rt. In the first assumption, Rt was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with Rt = 1.9, 2.6 or 3.1. The number of infections would reach 11,044, 70,258 and 227,989, respectively, by 29 February 2020. In the second assumption, Rt was assumed to gradually decrease at different phases from high level of transmission (Rt = 3.1, 2.6 and 1.9) to below 1 (Rt = 0.9 or 0.5) owing to increasingly implemented public heath intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce Rt to an ideal level and control the infection.</jats:p></p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>The role of absolute humidity on transmission rates of the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (COVID-19) was identified in Wuhan, Hubei Province, China, in December 2019 and has caused over 40,000 cases worldwide to date. Previous studies have supported an epidemiological hypothesis that cold and dry (low absolute humidity) environments facilitate the survival and spread of droplet-mediated viral diseases, and warm and humid (high absolute humidity) environments see attenuated viral transmission (i.e., influenza). However, the role of absolute humidity in transmission of COVID-19 has not yet been established. Here, we examine province-level variability of the basic reproductive numbers of COVID-19 across China and find that changes in weather alone (i.e., increase of temperature and humidity as spring and summer months arrive in the North Hemisphere) will not necessarily lead to declines in COVID-19 case counts without the implementation of extensive public health interventions.</jats:p></p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents.</jats:p></p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China.  Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity.  Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country.  Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.</jats:p></p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimating the reproduction number of the coronavirus disease (COVID-19) outbreak in Republic of Korea and Italy by 5 March 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus disease 2019 (COVID-19) outbreak and Italy has caused 6088 cases and 41 deaths in Republic of Korea and 3144 cases and 107 death in Italy by 5 March 2020. We modeled the transmission process in Republic of Korea and Italy with a stochastic model and estimated the basic reproduction number R0 as 2.6 (95% CI: 2.3-2.9) or 3.2 (95% CI: 2.9-3.5) in Republic of Korea, under the assumption that the exponential growth starting on 31 January or 5 February 2020, and 2.6 (95% CI: 2.3-2.9) or 3.3 (95% CI: 3.0-3.6) in Italy, under the assumption that the exponential growth starting on 5 February or 10 February 2020. Estimates of dispersion term (k) were 10 (95% CI: 5-56) or 22 (95% CI: 8-61) in Republic of Korea, and 13 (95% CI: 5-61) or 37 (95% CI: 13-61) in Italy, and all of which imply few super-spreading events.</jats:p></p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Phase-adjusted estimation of the number of Coronavirus Disease 2019 cases in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV/SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious, and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of Rt. In the first assumption, Rt was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with Rt = 1.9, 2.6, or 3.1. The number of infections would reach 11,044, 70,258, and 227,989, respectively, by 29 February 2020. In the second assumption, Rt was assumed to gradually decrease at different phases from high level of transmission (Rt = 3.1, 2.6, and 1.9) to below 1 (Rt = 0.9 or 0.5) owing to increasingly implemented public health intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077–84,520 or 55,869–81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce Rt to an ideal level and control the infection.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Early Phylogenetic Estimate of the Effective Reproduction Number Of Sars-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">To reconstruct the evolutionary dynamics of the 2019 novel coronavirus recently causing an outbreak in Wuhan, China, 52 SARS-CoV-2 genomes available on 04 February 2020 at GISAID were analysed. The two models used to estimate the reproduction number (coalescent-based exponential growth and a birth-death skyline method) indicated an estimated mean evolutionary rate of 7.8 x 10(-4) subs/site/year (range 1.1x10(-4) -15x10(-4) ) and a mean tMRCA of the tree root of 73 days. The estimated R value was 2.6 (range 2.1-5.1), and increased from 0.8 to 2.4 in December 2019. The estimated mean doubling time of the epidemic was between 3.6 and 4.1 days. This study proves the usefulness of phylogeny in supporting the surveillance of emerging new infections even as the epidemic is growing. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Model Parameters and Outbreak Control for SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">Control of the 2002–2003 severe acute respiratory syndrome (SARS) outbreak was based on rapid diagnosis coupled with effective patient isolation. We used uncertainty and sensitivity analysis of the basic reproductive number R(0) to assess the role that model parameters play in outbreak control. The transmission rate and isolation effectiveness have the largest fractional effect on R(0). We estimated the distribution of the reproductive number R(0) under perfect isolation conditions. The distribution lies in the interquartile range 0.19–1.08, with a median of 0.49. Even though the median of R(0) is <1, we found that 25% of our R(0) distribution lies at R(0) > 1, even with perfect isolation. This implies the need to simultaneously apply more than one method of control.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the generation interval for COVID-19 based on symptom onset data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies. Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided. Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers. Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32-67%) for Singapore and 62% (95%CI 50-76%) for Tianjin, China. Estimates of the reproduction number based on the generation interval distribution were slightly higher than those based on the serial interval distribution. Conclusions: Estimating generation and serial interval distributions from outbreak data requires careful investigation of the underlying transmission network. Detailed contact tracing information is essential for correctly estimating these quantities.</jats:p></p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>We shouldn't worry when a virus mutates during disease outbreaks</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Mutation. The word naturally conjures fears of unexpected and freakish changes. Ill-informed discussions of mutations thrive during virus outbreaks, including the ongoing spread of SARS-CoV-2. In reality, mutations are a natural part of the virus life cycle and rarely impact outbreaks dramatically.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background and Objective: To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. Design and Methods: We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. Results: The peak time of the curve for the infected newly diagnosed as COVID-19 should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei. It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese COVID-19 epidemic can be completely controlled in May,2020. Conclusions: COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global SARS-CoV-2 cases rise with the great efforts by Chinese government and its people.</jats:p></p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world.With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Age-dependent risks of Incidence and Mortality of COVID-19 in Hubei Province and Other Parts of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>New coronavirus SARS-CoV-2 poses a big challenge for global public health in early 2020. Coronavirus Disease 2019 (COVID-19) caused by the virus rapidly spreads all over the world and takes thousands of lives in just two months. It is critical to refine the incidence and mortality risks of COVID-19 for the effective management of the general public and patients in the outbreak. In this report, we investigate the incidence and mortality risks of the infection by analyzing the age composition of 5319 infected patients, 76 fatal cases, and 1,144,648 individuals of the general public in China. Our result shows a relatively low incidence risk for young people but a very high mortality risk for seniors. Notably, mortality risk could be as high as 0.48 for people older than 80 years. Furthermore, our study suggests that a good medical service can effectively reduce the mortality rate of the viral infection to 1% or less.</jats:p></p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-29</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundsAn ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.  MethodsAccounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate ({gamma}), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI.  FindingsThe early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.  ConclusionThe mean estimate of R0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Backgrounds An ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number,R0, of 2019-nCoV in the early phase of the outbreak. Methods Accounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (γ), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI. Findings The early outbreak data largely follows the exponential growth. We estimated that the meanR0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0. Conclusion The mean estimate ofR0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Study on assessing early epidemiological parameters of coronavirus disease epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: To study the early dynamics of the epidemic of coronavirus disease (COVID-19) in China from 15 to 31 January, 2020, and estimate the corresponding epidemiological parameters (incubation period, generation interval and basic reproduction number) of the epidemic. Methods: By means of Weibull, Gamma and Lognormal distributions methods, we estimated the probability distribution of the incubation period and generation interval data obtained from the reported COVID-19 cases. Moreover, the AIC criterion was used to determine the optimal distribution. Considering the epidemic is ongoing, the exponential growth model was used to fit the incidence data of COVID-19 from 10 to 31 January, 2020, and exponential growth method, maximum likelihood method and SEIR model were used to estimate the basic reproduction number. Results: Early COVID-19 cases kept an increase in exponential growth manner before 26 January, 2020, then the increase trend became slower. The average incubation period was 5.01 (95%CI: 4.31-5.69) days; the average generation interval was 6.03 (95%CI: 5.20-6.91) days. The basic reproduction number was estimated to be 3.74 (95%CI: 3.63-3.87), 3.16 (95%CI: 2.90-3.43), and 3.91 (95%CI: 3.71-4.11) by three methods, respectively. Conclusions: The Gamma distribution fits both the generation interval and incubation period best, and the mean value of generation interval is 1.02 day longer than that of incubation period. The relatively high basic reproduction number indicates that the epidemic is still serious; Based on our analysis, the turning point of the epidemic would be seen on 26 January, the growth rate would be lower afterwards.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Feasibility of controlling COVID-19 outbreaks by isolation of cases and contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Background Isolation of cases and contact tracing is used to control outbreaks of infectious diseases, and has been used for coronavirus disease 2019 (COVID-19). Whether this strategy will achieve control depends on characteristics of both the pathogen and the response. Here we use a mathematical model to assess if isolation and contact tracing are able to control onwards transmission from imported cases of COVID-19. Methods We developed a stochastic transmission model, parameterised to the COVID-19 outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-like pathogen. We considered scenarios that varied in the number of initial cases, the basic reproduction number (R0), the delay from symptom onset to isolation, the probability that contacts were traced, the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreaks were deemed controlled if transmission ended within 12 weeks or before 5000 cases in total. We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort. Findings Simulated outbreaks starting with five initial cases, an R0 of 1·5, and 0% transmission before symptom onset could be controlled even with low contact tracing probability; however, the probability of controlling an outbreak decreased with the number of initial cases, when R0 was 2·5 or 3·5 and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1·5 were controllable with less than 50% of contacts successfully traced. To control the majority of outbreaks, for R0 of 2·5 more than 70% of contacts had to be traced, and for an R0 of 3·5 more than 90% of contacts had to be traced. The delay between symptom onset and isolation had the largest role in determining whether an outbreak was controllable when R0 was 1·5. For R0 values of 2·5 or 3·5, if there were 40 initial cases, contact tracing and isolation were only potentially feasible when less than 1% of transmission occurred before symptom onset. Interpretation In most scenarios, highly effective contact tracing and case isolation is enough to control a new outbreak of COVID-19 within 3 months. The probability of control decreases with long delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts. Funding Wellcome Trust, Global Challenges Research Fund, and Health Data Research UK.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world. With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Outbreak investigation for COVID-19 in northern Vietnam</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two Vietnamese adults returned to their home province of Vinh Phuc in northern Vietnam on Jan 17, 2020, from Wuhan, China, where they had been living since Nov 15, 2019, for a business trip. They presented with mild respiratory symptoms to their local health facilities at 4 days and 8 days, respectively, after arrival in Vinh Phuc. Both individuals were initially placed into respiratory isolation in hospital. Case 1 tested positive for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative organism of coronavirus disease 2019 (COVID-19), on Jan 30, 2020, and remained in isolation until recovery. Case 2 was discharged from isolation in hospital after having one negative test result on Jan 28 (11 days after returning from Wuhan). Following discharge, the patient attended a family social function. 2 days later, she was readmitted after a second nasal swab for SARS-CoV-2 taken during her time in hospital was reported as positive. Screening of 79 individuals who had been in contact with these two patients (namely, family members in the same household and anyone who had been within 2 m of them) was initiated on Jan 31. Six individuals from the same work team, who had also travelled from Wuhan on Jan 17, were isolated, and four of them tested positive for SARS-CoV-2 (cases 3, 4, and 8 in Vinh Phuc, and one case from another province). Five secondary cases were diagnosed within the social network of case 2. These included three household members (cases 6, 7, and 11) and two people who had attended the social function (cases 5 and 9; appendix p 1). Four of these individuals reported mild respiratory symptoms; the remaining patient was asymptomatic (case 7) at the time of diagnosis.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission characteristics of the COVID-19 outbreak in China: a study driven by data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-01</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days.  It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.</jats:p></p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic variations of SARS-CoV-2 suggest multiple outbreak sources of transmission</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We examined 169 genomes of SARS-CoV-2 and found that they can be classified into two major genotypes, Type I and Type II. Type I can be further divided into Type IA and IB. Our phylogenetic analysis showed that the Type IA resembles the ancestral SARS-CoV-2 most. Type II was likely evolved from Type I and predominant in the infections. Our results suggest that Type II SARS-CoV-2 was the source of the outbreak in the Wuhan Huanan market and it was likely originated from a super-spreader. The outbreak caused by the Type I virus should have occurred somewhere else, because the patients had no direct link to the market. Furthermore, by analyzing three genomic sites that distinguish Type I and Type II strains, we found that synonymous changes at two of the three sites confer higher protein translational efficiencies in Type II strains than in Type I strains, which might explain why Type II strains are predominant, implying that Type II is more contagious (transmissible) than Type I. These findings could be valuable for the current epidemic prevention and control.</jats:p></p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Phase-adjusted estimation of the number of Coronavirus Disease 2019 cases in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 24</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV/SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R(0) of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious, and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of R(t). In the first assumption, R(t) was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with R(t) = 1.9, 2.6, or 3.1. The number of infections would reach 11,044, 70,258, and 227,989, respectively, by 29 February 2020. In the second assumption, R(t) was assumed to gradually decrease at different phases from high level of transmission (R(t) = 3.1, 2.6, and 1.9) to below 1 (R(t) = 0.9 or 0.5) owing to increasingly implemented public health intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077–84,520 or 55,869–81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce R(t) to an ideal level and control the infection.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus Outbreak: What We Know and What We Don&#x2019;t</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Phylogenetic analyses reveal that SARS-CoV-2 is closely related to a group of SARS-like coronaviruses. However, it remains unclear where the virus comes from and how it was transmitted to humans in the first place. Unlike with other zoonotic agents such as hantavirus and arenavirus, thus far we haven’t found a SARS virus in animals that is the same as that in humans. Fortunately, SARS virus has not appeared in humans since 2004. In contrast, this new virus seems to have stronger transmission capabilities among people. Compared to the primary virus in humans, we still know less about whether, what, and how the virus has changed and the effect of the changes for their epidemics in humans. Control and prevention of the disease is especially difficult in China and elsewhere if there are infected individuals with no clinical signs.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of the Time-Varying Reproduction Number of COVID-19 Outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The 2019-nCoV outbreak in Wuhan, China has attracted world-wide attention. As of February 11, 2020, a total of 44730 cases of novel coronavirus-infected pneumonia associated with COVID-19 were confirmed by the National Health Commission of China. Methods: Three approaches, namely Poisson likelihood-based method (ML), exponential growth rate-based method (EGR) and stochastic Susceptible-Infected-Removed dynamic model-based method (SIR), were implemented to estimate the basic and controlled reproduction numbers. Results: A total of 71 chains of transmission together with dates of symptoms onset and 67 dates of infections were identified among 5405 confirmed cases outside Hubei as reported by February 2, 2020. Based on this information, we find the serial interval having an average of 4.41 days with a standard deviation of 3.17 days and the infectious period having an average of 10.91 days with a standard deviation of 3.95 days. Conclusions: The controlled reproduction number is declining. It is lower than one in most regions of China, but is still larger than one in Hubei Province. Sustained efforts are needed to further reduce the Rc to below one in order to end the current epidemic.</jats:p></p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the cross-species evolution of 2019 novel coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Recent study reported in this journal that the threats of continuous evolution and dissemination of 2019 human coronaviruses.1 Since its emergence in December 2019, a “seventh” member of the family of 2019 human coronavirus named “SARS-CoV-2” was responsible for an outbreak of coronavirus disease (COVID-19) in Wuhan, China.2 As of February 23, 2020, China had reported more than 77,042 confirmed cases of SARS-CoV-2, with 2,445 fatalities and counting (http://www.nhc.gov.cn). Strikingly, SARS-CoV-2 had been transmitted rapidly in more than 29 countries to date (https://www.who.int), including Asia, Europe, North America, Africa, and Oceania, posing serious concerns about its pandemic potential. Despite of droplet and contact transmissions of SARS-CoV-2, recent studies demonstrated that SARS-CoV-2 might be transmitted via aerosol and fecal–oral routes 3 (Figure 1), which needs to be paid attention in particular.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Assessing the Tendency of 2019-nCoV (COVID-19) Outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Since December 8, 2019, the spread of COVID-19 is increasing every day. It is particularly important to predict the trend of the epidemic for the timely adjustment of the economy and industries. We proposed a Flow-SEHIR model in this paper, based on which we further analyzed the trends of 2019-nCoV (COVID-19) in China.    The results show that the basic reproductive numbers R0 of COVID-19 is 3.56 (95% CI: 2.31-4.81). The number of daily confirmed new cases reaches the inflection point on Feb. 6-10 outside Hubei. For the maximum of infected cases number, the predicted peak value in China except Hubei was estimated to be 13806 (95% CI: 11926-15845). The peak arrival time is on March 3-9. The temporal number of patients in most areas of China outside Hubei will peak from March 12 to March 15. The peak values of more than 73.5% provinces or regions in China will be controlled within 1000. According to Flow-SEHIR model and estimations from the data of evacuation of nationals from Wuhan, the peak cumulative number of patients in Hubei was estimated to be 403481 (95% CI: 143284-1166936).</jats:p></p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>A deterministic epidemic model for the emergence of COVID-19 in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Coronavirus disease (COVID-19) broke out in Wuhan, Hubei province,China, in December 2019 and soon after Chinese health authorities tookunprecedented prevention and control measures to curb the spreading ofthe novel coronavirus-related pneumonia.  We develop a mathematicalmodel based on daily updates of reported cases to study the evolutionof the epidemic.  With the model, on 95% confidence level, we estimatethe basic reproduction number, R0 = 2.82 ± 0.11, time between March19 and March 21 when the effective reproduction number becoming lessthan one, the epidemic ending after April 2 and the total number ofconfirmed cases approaching 14408 ± 429 on the Chinese mainlandexcluding Hubei province.</jats:p></p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>The reproductive number R0 of COVID-19 based on estimate of a statistical time delay dynamical system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>In this paper, we estimate the reproductive number <jats:italic>R</jats:italic><jats:sub>0</jats:sub> of COVID-19 based on Wallinga and Lipsitch framework {11} and a novel statistical time delay dynamic system. We use the observed data reported in CCDC's paper to estimate distribution of the generation interval of the infection and apply  the simulation results from the time delay dynamic system as well as released data from CCDC  to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate <jats:italic>r</jats:italic> of COVID-19 is almost in [0.30, 0.32] which is  larger than  the growth rate 0.1  estimated by CCDC {9},   and the  reproductive number <jats:italic>R</jats:italic><jats:sub>0</jats:sub> of COVID-19 is estimated by 3.25≤ <jats:italic>R</jats:italic><jats:sub>0</jats:sub> ≤3.4 if we simply use <jats:italic>R</jats:italic>=1+<jats:italic>r</jats:italic>*<jats:italic>T</jats:italic><jats:sub>c</jats:sub> with <jats:italic>T</jats:italic><jats:sub>c</jats:sub>=7.5, which is bigger than that of SARS. Some evolutions and predictions are listed.</jats:p></p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome-wide data inferring the evolution and population demography of the novel pneumonia coronavirus (SARS-CoV-2)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, coronavirus disease 2019 (COVID-19) emerged in Wuhan, Central China and rapidly spread throughout China. Up to March 3, 2020, SARS-CoV-2 has infected more than 89,000 people in China and other 66 countries across six continents. In this study, we used 10 new sequenced genomes of SARS-CoV-2 and combined 136 genomes from GISAID database to investigate the genetic variation and population demography through different analysis approaches (e.g. Network, EBSP, Mismatch, and neutrality tests) in the previous three months. The results showed that eighty haplotypes had 183 substitution sites, including 27 parsimony-informative and 156 singletons. Sliding window analyses of genetic diversity suggested a certain range in mutations and variation in genomic abundance of SARS-CoV-2, which could be explaining the existing widespread and high adaptation of the deadly virus. Phylogenetic analysis showed that pangolin may not be an intermediate host. The network indicated that, in the original haplotype (H14), one patient sample lived near the Huanan seafood market (approximately 2 km), which indicating high possibility of the patient having a history of unconscious contact with this market. However, based on this clue, we cannot accurately conclude that whether this market was the origin center of SARS-CoV-2. Additionally, 16 genomes, collected from this market, assigned to 10 haplotypes, indicated a circulating infection within the market in a short term, which would have led to the outbreak of SARS-CoV-2 in Wuhan and other areas. The EBSP results showed that the first estimated expansion date began from 7 December 2019, which indicated that the transmission of SARS-CoV-2 could have begun from person to person in mid to late November.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus.</jats:p></p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 sensitive to type I interferon pretreatment.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B coronaviruses share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel coronavirus is much more sensitive to type I interferon pretreatment. We subsequently examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN response between SARS-CoV and SARS-CoV-2 that could help inform disease progression, treatment options, and animal model development.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.</jats:p></p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Uncertainties about the transmission routes of 2019 novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019 novel coronavirus (now named as SARS‐CoV‐2) caused an outbreak of SARS‐like illness in the late of December 2019. At present, the origin, susceptible population, and infection sources already have been clear.1, 2 However, the transmission routes, a key step to the epidemic control, have not yet been fully ascertained. Here, we focus on the potential transmission routes that have been investigated in the SARS‐CoV‐2 epidemic recently. SARS‐CoV‐2, similar to SARS and MERS, is predominantly spread via respiratory tract with high infectivity. It is commonly recognized that droplet transmission is the main route. Spread by aerosol is suspected to be another important route of transmission but unestablished now. Epidemiological experts, as well as the WHO, consider more evidence is needed to confirm.3 Besides, there are other routes except respiratory transmission. The previous study indicated that different human coronaviruses, such as SARS‐CoV and MERS‐CoV, can maintain infectious for a different time on inanimate surfaces.4 Meanwhile, it was reported that SARS‐CoV‐2 was also founded on the surface of the door handles, cell phones, and other items in the residential sites of confirmed cases.5 Therefore, individuals will be probably infected if they touch the nose, mouth, or eyes after contacting the contaminated items.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple ecological model captures the transmission pattern of the coronavirus COVID-19 outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The rapid spread of the 2019 novel coronavirus (COVID-19), initially reported in the city of Wuhan in China, and quickly transmitted to the entire nation and beyond, has become an international public health emergency. Estimating the final number of infection cases and the turning point (time with the fastest spreading rate) is crucial to assessing and improving the national and international control measures currently being applied. In this paper we develop a simple model based on infectious growth with a time-varying infection rate, and estimate the final number of infections and the turning point using data updated daily from 3 February 2020, when China escalated its initial public health measures, to 10 February. Our model provides an extremely good fit to the existing data and therefore a reasonable estimate of the time-varying infection rate that has largely captured the transmission pattern of this epidemic outbreak. Our estimation suggests that (i) the final number of infections in China could reach 78,000 with an upper 95% confidence limit of 88,880; (ii) the turning point of the fastest spread was on the 4th or the 5th of February; and (iii) the projected period for the end of the outbreak (i.e., when 95% of the final predicted number of infection is reached) will be the 24th of February, with an upper 95% confidence limit on the 19th of March. This suggests that the current control measures in China are excellent, and more than sufficient to contain the spread of this highly infectious novel coronavirus, and that the application of such measures could be considered internationally for the global control of this outbreak.</jats:p></p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Facing the COVID-19 outbreak: What should we know and what could we do?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Early Transmissibility Assessment of a Novel Coronavirus in Wuhan, China</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Between December 1, 2019 and January 26, 2020, nearly 3000 cases of respiratory illness caused by a novel coronavirus originating in Wuhan, China have been reported. In this short analysis, we combine publicly available cumulative case data from the ongoing outbreak with phenomenological modeling methods to conduct an early transmissibility assessment. Our model suggests that the basic reproduction number associated with the outbreak (at time of writing) may range from 2.0 to 3.1. Though these estimates are preliminary and subject to change, they are consistent with previous findings regarding the transmissibility of the related SARS-Coronavirus and indicate the possibility of epidemic potential.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological Development of Novel Coronavirus Pneumonia in China and Its Forecast</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>ABSTRACT  BACKGROUND AND OBJECTIVE The novel coronavirus (SARS-Cov-2) infected coronavirus disease 2019 (COVID-19) was broken out in Wuhan and Hubei province for more than a month. It severely threats people's health of thousands in Chin and even other countries. In order to prevent its wide spread, it is necessary to understand the development of the epidemic with precise mathematical language.  METHODS    The various data of novel coronavirus pneumonia were collected from the official websites of the National Health Committee of the People's Republic of China. According to epidemic and administrative division, three groups were divided to analyze the data, Hubei Province (including Wuhan), nationwide without Hubei and Henan Province. With classic SIR models, the fitting epidemiological curves of incidence have made, and basic reproduction number (R0) was also calculated as well. Therefore the disease's infection intensity, peak time and the epidemiological end time can be deduced.</jats:p></p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission interval estimates suggest pre-symptomatic spread of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods:  We collected outbreak information from Singapore and Tianjin,  China,  reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring.  Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.</jats:p></p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of the reproductive number of Novel Coronavirus (COVID-19) and the probable outbreak size on the Diamond Princess cruise ship: A data-driven analysis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Backgrounds Up to February 16, 2020, 355 cases have been confirmed as having COVID-19 infection on the Diamond Princess cruise ship. It is of crucial importance to estimate the reproductive number (R0) of the novel virus in the early stage of outbreak and make a prediction of daily new cases on the ship. Method We fitted the reported serial interval (Mean and standard deviation) with a gamma distribution and applied “earlyR” package in R to estimate the R0 in the early stage of COVID-19 outbreak. We applied “projections” package in R to simulate the plausible cumulative epidemic trajectories and future daily incidence by fitting the data of existing daily incidence, a serial interval distribution, and the estimated R0 into a model based on the assumption that daily incidence obeys approximately Poisson distribution determined by daily infectiousness. Results The Maximum-Likelihood (ML) value of R0 was 2.28 for COVID-19 outbreak at early stage on the ship. The median with 95% confidence interval (CI) of R0 values was 2.28 (2.06-2.52) estimated by the bootstrap resampling method. The probable number of new cases for the next ten days would gradually increase, and the estimated cumulative cases would reach 1514 (1384-1656) at the tenth day in the future. However, if R0 value was reduced by 25% and 50%, the estimated total number of cumulative cases would be reduced to 1081 (981-1177) and 758 (697-817), respectively. Conclusion The median with 95% CI of R0 of COVID-19 was about 2.28 (2.06-2.52) during the early stage experienced on the Diamond Princess cruise ship. The future daily incidence and probable outbreak size is largely dependent on the change of R0. Unless strict infection management and control are taken, our findings indicate the potential of COVID-19 to cause greater outbreak on the ship.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2.  Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</jats:p></p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis on the epidemic factors for the Corona Virus Disease</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, corona virus disease 2019 (COVID-19) , an emerging infection disease occurred in Wuhan, has spread in the mainland China. The epidemic factors on the basis of knowledge of SARS-CoV-2 were discussed in this paper. This puts a lot of pressure on clinical resources and care. SARS-CoV-2 is a novel corona virus, the onset of COVID-19 is slow, and the pathogenesis of SARS-CoV-2 remains unclear and may lead to multiple organ damage. These put a lot of pressure on clinical resources and care. Source of infection including the patients, asymptomatic carrier and patients in the incubation period are contagious. It is difficult to control source of infection. Routes of SARS-CoV-2 transmission are diversified and the main routes of transmission for COVID-19 are droplet transmission and close contact transmission. All population have susceptibility to SARS-CoV-2. Social factors such population movements and aggregation accelerated the spread of SARS-CoV-2. The Chinese government's adopted measures are positive and effective, and are accepted by the expert group from the World Health Organization. However, it will be a long-term hard work in the future to seriously summarize and think deeply to achieve public health security in China.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>First respiratory transmitted food borne outbreak?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The world is faced with a remarkable coronavirus outbreak with epicentre in Wuhan, China. Altogether 40554 cases have been confirmed globally with novel coronavirus (SARS-CoV-2) until February 10, 2020. Rigorous surveillance in other countries is required to prevent further global expansion of the outbreak, but resolving the exact mechanism of the initial transmission events is crucial. Most initial cases had visited Huanan South Seafood Market in Wuhan selling also various exotic live animals. Based on the limited initial human-to-human transmission and timely clustering of cases in Huanan market among elderly men, coupled with knowledge that coronaviruses are derived from animals and relationship of SARS-CoV-2 to bat coronavirus, zoonotic transmission in the first instance is probable. To target the actions, similar epidemiological actions to human cases are needed with animal or food exposures. According to current information, an exceptionally wide contamination of seafood market might explain the initiation of the SARS-CoV-2 outbreak. Seafood tanks, air contamination by live animals or rodents are possibilities, but sold animals normally come from various sources. The mode of transmission may become clearer in future: usually in outbreak investigations, hindsight is easy, but for now information about the initial source of this outbreak is limited.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Assessing the plausibility of subcritical transmission of 2019-nCoV in the United States</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract: The 2019-nCoV outbreak has raised concern of global spread.  While person-to-person transmission within the Wuhan district has led to a large outbreak, the transmission potential outside of the region remains unclear.  Here we present a simple approach for determining whether the upper limit of the confidence interval for the reproduction number exceeds one for transmission in the United States, which would allow endemic transmission.  As of February 7, 2020, the number of cases in the United states support subcritical transmission, rather than ongoing transmission.  However, this conclusion can change if pre-symptomatic cases resulting from human-to-human transmission have not yet been identified.</jats:p></p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Modelling the coronavirus disease (COVID-19) outbreak on the Diamond Princess ship using the public surveillance data from January 20 to February 20, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus disease 2019 (COVID-19) outbreak on the Diamond Princess ship has caused over 634 cases as of February 20, 2020. We model the transmission process on the ship with a stochastic model and estimate the basic reproduction number at 2.2 (95%CI: 2.1−2.4). We estimate a large dispersion parameter than other coronaviruses, which implies that the virus is difficult to go extinction. The epidemic doubling time is at 4.6 days (95%CI: 3.0−9.3), and thus timely actions were crucial. The lesson learnt on the ship is generally applicable in other settings.</jats:p></p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating the infected population and spread trend of 2019-nCov under different policy by EIR model</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported official data, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared.</jats:p></p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Escalating infection control response to the rapidly evolving epidemiology of the Coronavirus disease 2019 (COVID-19) due to SARS-CoV-2 in Hong Kong</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Background:To describe the infection control preparedness for Coronavirus Disease (COVID-19) due to SARS-CoV-2 [previously known as 2019-novel coronavirus] in the first 42 days after announcement of a cluster of pneumonia in China, on 31 December 2019 (day 1) in Hong Kong.Methods:A bundle approach of active and enhanced laboratory surveillance, early airborne infection isolation, rapid molecular diagnostic testing, and contact tracing for healthcare workers (HCWs) with unprotected exposure in the hospitals was implemented. Epidemiological characteristics of confirmed cases, environmental and air samples were collected and analyzed.Results:From day 1 to day 42, forty-two (3.3%) of 1275 patients fulfilling active (n=29) and enhanced laboratory surveillance (n=13) confirmed to have SARS-CoV-2 infection. The number of locally acquired case significantly increased from 1 (7.7%) of 13 [day 22 to day 32] to 27 (93.1%) of 29 confirmed case [day 33 to day 42] (p&lt;0.001). Twenty-eight patients (66.6%) came from 8 family clusters. Eleven (2.7%) of 413 HCWs caring these confirmed cases were found to have unprotected exposure requiring quarantine for 14 days. None of them was infected and nosocomial transmission of SARS-CoV-2 was not observed. Environmental surveillance performed in a patient with viral load of 3.3x106 copies/ml (pooled nasopharyngeal/ throat swab) and 5.9x106 copies/ml (saliva) respectively. SARS-CoV-2 revealed in 1 (7.7%) of 13 environmental samples, but not in 8 air samples collected at a distance of 10 cm from patient's chin with or without wearing a surgical mask.Conclusion:Appropriate hospital infection control measures could prevent nosocomial transmission of SARS-CoV-2.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the Recent 2019 Novel Coronavirus (SARS-CoV-2) in Light of Past Human Coronavirus Outbreaks</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are RNA viruses that have become a major public health concern since the Severe Acute Respiratory Syndrome-CoV (SARS-CoV) outbreak in 2002. The continuous evolution of coronaviruses was further highlighted with the emergence of the Middle East Respiratory Syndrome-CoV (MERS-CoV) outbreak in 2012. Currently, the world is concerned about the 2019 novel CoV (SARS-CoV-2) that was initially identified in the city of Wuhan, China in December 2019. Patients presented with severe viral pneumonia and respiratory illness. The number of cases has been mounting since then. As of late February 2020, tens of thousands of cases and several thousand deaths have been reported in China alone, in addition to thousands of cases in other countries. Although the fatality rate of SARS-CoV-2 is currently lower than SARS-CoV, the virus seems to be highly contagious based on the number of infected cases to date. In this review, we discuss structure, genome organization, entry of CoVs into target cells, and provide insights into past and present outbreaks. The future of human CoV outbreaks will not only depend on how the viruses will evolve, but will also depend on how we develop efficient prevention and treatment strategies to deal with this continuous threat.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>The spatiotemporal estimation of the dynamic risk and the international transmission of 2019 Novel Coronavirus (COVID-19) outbreak: A global perspective</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An ongoing novel coronavirus SARS-CoV-2 pneumonia infection outbreak called COVID-19 started in Wuhan, Hubei Province, China, in December 2019. It both spread rapidly to all provinces in China and started spreading around the world quickly through international human movement from January 2020. Currently, the spatiotemporal epidemic transmission patterns, prediction models, and possible risk analysis for the future are insufficient for COVID-19 but we urgently need relevant information, particularly from the global perspective. We have developed a novel two-stage simulation model to simulate the spatiotemporal changes in the number of COVID-19 cases and estimate the future worldwide risk. Based on the connectivity of countries to China and the country's medical and epidemic prevention capabilities, different scenarios are generated to analyze the possible transmission throughout the world and use this information to evaluate each country's vulnerability to and the dynamic risk of COVID-19. Countries' vulnerability to the COVID-19 outbreak from China is calculated for 63 countries around the world. Taiwan, South Korea, Hong Kong, and Japan are the most vulnerable areas. The relationship between each country's vulnerability and days before the first imported case occurred shows a very high exponential decrease. The cumulative number of cases in each country also has a linear relationship with vulnerability, which can compare and quantify the initial epidemic prevention capabilities to various countries' management strategies. In total, 1,000 simulation results of future cases around the world are generated for the spatiotemporal risk assessment. According to the simulation results of this study, if there is no specific medicine for it, it will likely form a global pandemic. This method can be used as a preliminary risk assessment of the spatiotemporal spread for a new global epidemic. * Note: This study was completed on February 15, 2020.</jats:p></p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-Time Estimation of the Risk of Death from Novel Coronavirus (COVID-19) Infection: Inference Using Exported Cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The exported cases of 2019 novel coronavirus (COVID-19) infection that were confirmed outside China provide an opportunity to estimate the cumulative incidence and confirmed case fatality risk (cCFR) in mainland China. Knowledge of the cCFR is critical to characterize the severity and understand the pandemic potential of COVID-19 in the early stage of the epidemic. Using the exponential growth rate of the incidence, the present study statistically estimated the cCFR and the basic reproduction number&mdash;the average number of secondary cases generated by a single primary case in a na&iuml;ve population. We modeled epidemic growth either from a single index case with illness onset on 8 December, 2019 (Scenario 1), or using the growth rate fitted along with the other parameters (Scenario 2) based on data from 20 exported cases reported by 24 January 2020. The cumulative incidence in China by 24 January was estimated at 6924 cases (95% confidence interval [CI]: 4885, 9211) and 19,289 cases (95% CI: 10,901, 30,158), respectively. The latest estimated values of the cCFR were 5.3% (95% CI: 3.5%, 7.5%) for Scenario 1 and 8.4% (95% CI: 5.3%, 12.3%) for Scenario 2. The basic reproduction number was estimated to be 2.1 (95% CI: 2.0, 2.2) and 3.2 (95% CI: 2.7, 3.7) for Scenarios 1 and 2, respectively. Based on these results, we argued that the current COVID-19 epidemic has a substantial potential for causing a pandemic. The proposed approach provides insights in early risk assessment using publicly available data.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Composition and divergence of coronavirus spike proteins and host ACE2 receptors predict potential intermediate hosts of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) crossed the species barriers to infect humans caused thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS-CoV-2) causes the outbreaks of Coronavirus Disease 2019 (COVID-19) was discovered. Until February 18, 2020, there are 72533 confirmed COVID-19 cases (including 10644 severe cases) and 1872 deaths in China. SARS-CoV-2 is surging in the public and caused substantial burdens due to its human-to-human transmission. However, the intermediate host of SARS-CoV-2 is still unclear. Finding the possible intermediate host of SARS-CoV-2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor binding domain (RBD) of coronavirus spike protein and the host receptor, Angiotensin Converting Enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that like previous suggested pangolins and snacks, the turtles (C. picta bellii, C. mydas, and P. sinensis) may act as the potential intermediate hosts transmitting SARS-CoV-2 to human. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of the epidemic growth of the early 2019-nCoV outbreak using internationally confirmed cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On January 23, 2020, a quarantine was imposed on travel in and out of Wuhan, where the 2019 novel coronavirus (2019-nCoV) outbreak originated from. Previous analyses estimated the basic epidemiological parameters using symptom onset dates of the confirmed cases in Wuhan and outside China.  Methods: We obtained information on the 46 coronavirus cases who traveled from Wuhan before January 23 and have been subsequently confirmed in Hong Kong, Japan, Korea, Macau, Singapore, and Taiwan as of February 5, 2020. Most cases have detailed travel history and disease progress. Compared to previous analyses, an important distinction is that we used this data to informatively simulate the infection time of each case using the symptom onset time, previously reported incubation interval, and travel history. We then fitted a simple exponential growth model with adjustment for the January 23 travel ban to the distribution of the simulated infection time. We used a Bayesian analysis with diffuse priors to quantify the uncertainty of the estimated epidemiological parameters. We performed sensitivity analysis to different choices of incubation interval and the hyperparameters in the prior specification.  Results: We found that our model provides good fit to the distribution of the infection time. Assuming the travel rate to the selected countries and regions is constant over the study period, we found that the epidemic was doubling in size every 2.9 days (95% credible interval [CrI], 2 days--4.1 days). Using previously reported serial interval for 2019-nCoV, the estimated basic reproduction number is 5.7 (95% CrI, 3.4--9.2). The estimates did not change substantially if we assumed the travel rate doubled in the last 3 days before January 23, when we used previously reported incubation interval for severe acute respiratory syndrome (SARS), or when we changed the hyperparameters in our prior specification.  Conclusions: Our estimated epidemiological parameters are higher than an earlier report using confirmed cases in Wuhan. This indicates the 2019-nCoV could have been spreading faster than previous estimates.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission potential of COVID-19 in Iran</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We computed reproduction number of COVID-19 epidemic in Iran using two different methods. We estimated R0 at 3.6 (95% CI, 3.2, 4.2) (generalized growth model) and at 3.58 (95% CI, 1.29, 8.46) (estimated epidemic doubling time of 1.20 (95% CI, 1.05, 1.44) days) respectively. Immediate social distancing measures are recommended.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Aerodynamic Characteristics and RNA Concentration of SARS-CoV-2 Aerosol in Wuhan Hospitals during COVID-19 Outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">Background: The ongoing outbreak of COVID-19 has spread rapidly and sparked global concern. While the transmission of SARS-CoV-2 through human respiratory droplets and contact with infected persons is clear, the aerosol transmission of SARS-CoV-2 has been little studied. Methods: Thirty-five aerosol samples of three different types (total suspended particle, size segregated and deposition aerosol) were collected in Patient Areas (PAA) and Medical Staff Areas (MSA) of Renmin Hospital of Wuhan University (Renmin) and Wuchang Fangcang Field Hospital (Fangcang), and Public Areas (PUA) in Wuhan, China during COVID-19 outbreak. A robust droplet digital polymerase chain reaction (ddPCR) method was employed to quantitate the viral SARS-CoV-2 RNA genome and determine aerosol RNA concentration. Results: The ICU, CCU and general patient rooms inside Renmin, patient hall inside Fangcang had undetectable or low airborne SARS-CoV-2 concentration but deposition samples inside ICU and air sample in Fangcang patient toilet tested positive. The airborne SARS-CoV-2 in Fangcang MSA had bimodal distribution with higher concentration than those in Renmin during the outbreak but turned negative after patients number reduced and rigorous sanitization implemented. PUA had undetectable airborne SARS-CoV-2 concentration but obviously increased with accumulating crowd flow. Conclusions: Room ventilation, open space, proper use and disinfection of toilet can effectively limit aerosol transmission of SARS-CoV-2. Gathering of crowds with asymptomatic carriers is a potential source of airborne SARS-CoV-2. The virus aerosol deposition on protective apparel or floor surface and their subsequent resuspension is a potential transmission pathway and effective sanitization is critical in minimizing aerosol transmission of SARS-CoV-2.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>The serial interval of COVID-19 from publicly reported confirmed cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.1% of reports indicating pre-symptomatic transmission.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Individual quarantine and active monitoring of contacts are core disease control strategies, particularly for emerging infectious diseases such as Coronavirus Disease 2019 (COVID-19). To estimate the comparative efficacy of these interventions to control COVID-19, we fit a stochastic branching model, comparing two sets of reported parameters for the dynamics of the disease. Our results suggest that individual quarantine may contain an outbreak of COVID-19 with a short serial interval (4.8 days) only in settings with high intervention performance where at least three-quarters of infected contacts are individually quarantined. However, in settings where this performance is unrealistically high and the outbreak of COVID-19 continues to grow, so too will the burden of the number of contacts traced for active monitoring or quarantine. In such circumstances where resources are prioritized for scalable interventions such as social distancing, we show active monitoring or individual quarantine of high-risk contacts can contribute synergistically to social distancing. To the extent that interventions based on contact tracing can be implemented, therefore, they can help mitigate the spread of COVID-19. Our model highlights the urgent need for more data on the serial interval and the extent of presymptomatic transmission in order to make data-driven policy decisions regarding the cost-benefit comparisons of individual quarantine vs. active monitoring of contacts.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>An updated estimation of the risk of transmission of the novel coronavirus (2019-nCov)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 11</span></i><p style="text-align: justify; text-justify: inter-word;">The basic reproduction number of an infectious agent is the average number of infections one case can generate over the course of the infectious period, in a naïve, uninfected population. It is well-known that the estimation of this number may vary due to several methodological issues, including different assumptions and choice of parameters, utilized models, used datasets and estimation period. With the spreading of the novel coronavirus (2019-nCoV) infection, the reproduction number has been found to vary, reflecting the dynamics of transmission of the coronavirus outbreak as well as the case reporting rate. Due to significant variations in the control strategies, which have been changing over time, and thanks to the introduction of detection technologies that have been rapidly improved, enabling to shorten the time from infection/symptoms onset to diagnosis, leading to faster confirmation of the new coronavirus cases, our previous estimations on the transmission risk of the 2019-nCoV need to be revised. By using time-dependent contact and diagnose rates, we refit our previously proposed dynamics transmission model to the data available until January 29th(,) 2020 and re-estimated the effective daily reproduction ratio that better quantifies the evolution of the interventions. We estimated when the effective daily reproduction ratio has fallen below 1 and when the epidemics will peak. Our updated findings suggest that the best measure is persistent and strict self-isolation. The epidemics will continue to grow, and can peak soon with the peak time depending highly on the public health interventions practically implemented.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Incorporating Human Movement Data to Improve Epidemiological Estimates for 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Estimating the key epidemiological features of the novel coronavirus (2019-nCoV) epidemic proves to be challenging, given incompleteness and delays in early data reporting, in particular, the severe under-reporting bias in the epicenter, Wuhan, Hubei Province, China. As a result, the current literature reports widely varying estimates. We developed an alternative geo-stratified debiasing estimation framework by incorporating human mobility with case reporting data in three stratified zones, i.e., Wuhan, Hubei Province excluding Wuhan, and mainland China excluding Hubei. We estimated the latent infection ratio to be around 0.12% (18,556 people) and the basic reproduction number to be 3.24 in Wuhan before the city's lockdown on January 23, 2020. The findings based on this debiasing framework have important implications to prioritization of control and prevention efforts.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Does SARS-CoV-2 has a longer incubation period than SARS and MERS?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of a novel coronavirus (SARS-CoV-2) since Dec 2019 in Wuhan, the major transportation hub in central China, became an emergency of major international concern. While several etiological studies have begun to reveal the specific biological features of this virus, the epidemic characteristics need to be elucidated. Notably, a long incubation time was reported to be associated with SARS-CoV-2 infection, leading to adjustments in screening and control policies. To avoid the risk of virus spread, all potentially exposed subjects are required to be isolated for 14 days, which is the longest predicted incubation time. However, based on our analysis of a larger dataset available so far, we find there is no observable difference between the incubation time for SARS-CoV-2, SARS and MERS, highlighting the need for larger and well annotated datasets. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3-to-5 exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the distribution of the incubation period of 2019 novel coronavirus (COVID-19) infection between travelers to Hubei, China and non-travelers</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objectives: Amid the continuing spread of the novel coronavirus (COVID-19), the incubation period of COVID-19 should be regularly re-assessed as more information is available upon the increase in reported cases. The present work estimated the distribution of incubation periods of patients infected in and outside Hubei province of China.  Methods: Clinical data were collected from the individual cases reported by the media as they were not fully available on the official pages of the Chinese health authorities. MLE was used to estimate the distributions of the incubation period. Results: It was found that the incubation period of patients with no travel history to Hubei was longer and more volatile. Conclusion: It is recommended that the duration of quarantine should be extended to at least 3 weeks.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-28</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamic basic reproduction number based evaluation for current prevention and control of COVID-19 outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: To evaluate the current status of the prevention and control of coronavirus disease (COVID-19) outbreak in China, establish a predictive model to evaluate the effects of the current prevention and control strategies, and provide scientific information for decision- making departments. Methods: Based on the epidemic data of COVID-19 openly accessed from national health authorities, we estimated the dynamic basic reproduction number R(0)(t) to evaluate the effects of the current COVID-19 prevention and control strategies in all the provinces (municipalities and autonomous regions) as well as in Wuhan and the changes in infectivity of COVID-19 over time. Results: For the stability of the results, 24 provinces (municipality) with more than 100 confirmed COVID-19 cases were included in the analysis. At the beginning of the outbreak, the R(0)(t) showed unstable trend with big variances. As the strengthening of the prevention and control strategies, R(0)(t) began to show a downward trend in late January, and became stable in February. By the time of data analysis, 18 provinces (municipality) (75%) had the R(0)(t)s less than 1. The results could be used for the decision making to free population floating conditionally. Conclusions: Dynamic R(0)(t) is useful in the evaluation of the change in infectivity of COVID-19, the prevention and control strategies for the COVID-19 outbreak have shown preliminary effects, if continues, it is expected to control the COVID-19 outbreak in China in near future.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Active Monitoring of Persons Exposed to Patients with Confirmed COVID-19 — United States, January–February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, an outbreak of coronavirus disease 2019 (COVID-19), caused by the virus SARS-CoV-2, began in Wuhan, China (1). The disease spread widely in China, and, as of February 26, 2020, COVID-19 cases had been identified in 36 other countries and territories, including the United States. Person-to-person transmission has been widely documented, and a limited number of countries have reported sustained person-to-person spread.* On January 20, state and local health departments in the United States, in collaboration with teams deployed from CDC, began identifying and monitoring all persons considered to have had close contact† with patients with confirmed COVID-19 (2). The aims of these efforts were to ensure rapid evaluation and care of patients, limit further transmission, and better understand risk factors for transmission.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Distinguishing Between Reservoir Exposure and Human-to-Human Transmission for Emerging Pathogens Using Case Onset Data</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 7</span></i><p style="text-align: justify; text-justify: inter-word;">Pathogens such as MERS-CoV, influenza A/H5N1 and influenza A/H7N9 are currently generating sporadic clusters of spillover human cases from animal reservoirs. The lack of a clear human epidemic suggests that the basic reproductive number R0 is below or very close to one for all three infections. However, robust cluster-based estimates for low R0 values are still desirable so as to help prioritise scarce resources between different emerging infections and to detect significant changes between clusters and over time. We developed an inferential transmission model capable of distinguishing the signal of human-to-human transmission from the background noise of direct spillover transmission (e.g. from markets or farms). By simulation, we showed that our approach could obtain unbiased estimates of R0, even when the temporal trend in spillover exposure was not fully known, so long as the serial interval of the infection and the timing of a sudden drop in spillover exposure were known (e.g. day of market closure). Applying our method to data from the three largest outbreaks of influenza A/H7N9 outbreak in China in 2013, we found evidence that human-to-human transmission accounted for 13% (95% credible interval 1%–32%) of cases overall. We estimated R0 for the three clusters to be: 0.19 in Shanghai (0.01-0.49), 0.29 in Jiangsu (0.03-0.73); and 0.03 in Zhejiang (0.00-0.22). If a reliable temporal trend for the spillover hazard could be estimated, for example by implementing widespread routine sampling in sentinel markets, it should be possible to estimate sub-critical values of R0 even more accurately. Should a similar strain emerge with R0>1, these methods could give a real-time indication that sustained transmission is occurring with well-characterised uncertainty.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical outcomes of 402 patients with COVID-2019 from a single center in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The SARS-CoV-2 outbreak is causing widespread infections and significant mortality. Previous studies describing clinical characteristics of the disease contained small cohorts from individual centers or larger series consisting of mixed cases from different hospitals. We report analyses of mortality and disease severity among 402 patients from a single hospital. The cohort included 297 patients with confirmed and 105 with suspected diagnosis. The latter group met the criteria for clinical diagnosis but nucleic acid tests results were initially interpreted as suspicious. Data were compared between genders and among different age groups. The overall case fatality is 5.2%. However, patients 70 years of age or older suffered a significantly higher mortality (17.8%), associated with more patients having severe or critical illness (57.5%). Patients 50 years of age or older had a mortality of 8.0%, and those younger than 50 years, 1.2%. Male patients had a mortality of 7.6% versus 2.9% in females.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background:  The current outbreak of coronavirus disease 2019 (COVID-19) has quickly spread across countries and become a global crisis. However, one of the most important clinical characteristics in epidemiology, the distribution of the incubation period, remains unclear. Different estimates of the incubation period of COVID-19 were reported in recent published studies, but all have their own limitations. In this study, we propose a novel low-cost and accurate method to estimate the incubation distribution.   Methods: We have conducted a cross-sectional and forward follow-up study by identifying those asymptomatic individuals at their time of departure from Wuhan and then following them until their symptoms developed. The renewal process is hence adopted by considering the incubation period as a renewal and the duration between departure and symptom onset as a forward recurrence time. Under mild assumptions, the observations of selected forward times can be used to consistently estimate the parameters in the distribution of the incubation period. Such a method enhances the accuracy of estimation by reducing recall bias and utilizing the abundant and readily available forward time data.   Findings:  The estimated distribution of forward time fits the observations in the collected data well. The estimated median of incubation period is 8.13 days (95% confidence interval [CI]: 7.37-8.91), the mean is 8.62 days (95% CI: 8.02-9.28), the 90th percentile is 14.65 days (95% CI: 14.00-15.26), and the 99th percentile is 20.59 days (95% CI: 19.47, 21.62). Compared with results in other studies, the incubation period estimated in this study is longer.  Interpretation: Based on the estimated incubation distribution in this study, about 10% of patients with COVID-19 would not develop symptoms until 14 days after infection. Further study of the incubation distribution is warranted to directly estimate the proportion with long incubation periods.</jats:p></p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to COVID-19: A previously unknown SARS-CoV-2 virus of pandemic potential infecting humans – Call for a One Health approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses continue to pose a threat to human health. The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in December 2019 which causes coronavirus disease-2019 (COVID-19), an acute respiratory disease marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. This recent ongoing emergence of a previously unknown coronavirus in China leads to huge impacts on humans globally. Here, we discuss the outbreak in a one health context, highlighting the need for the implementation of one health measures and practices to improve human health and reduce the emergence of pandemic viruses.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /></body>