
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: access to geographic and temporal diverse sample sets to understand geographic distribution and genomic differences</h2><br /></div><br /><br /><br /><div class="card"><h3>Regional genetic diversity for NNV grouper viruses across the Indo- Asian region -implications for selecting virus resistance in farmed groupers OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">This study uses statistical approaches and assessment of "characteristic attributes" (i.e. nucleotide positions that discriminate among strains) to assess whether published and new NNV RNA2 cds sequences show genetic differentiation over geography, host species and years. Rather clear evidence was found for regional strains of NNV. Interestingly, most of the geographic defining "characteristic attributes" were in codon position three, and not translated into differences for the protein capsid (i.e. they were synonymous variations), suggesting that while NNV strains were geographically isolated and had diverged in different regions for RNA sequences, selection had largely conserved the protein sequences among regions. The apparent selection constraint on the capsid protein may mitigate the risk that despite geographic subdivision, NNV strain variability will confound genetic selection for host resistance. The existence of regional Asian NNV strains may suggest that hatcheries are at risk from NNV not only from imported material but also from endemic reservoirs.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Etiology of fever in Ugandan children: identification of microbial pathogens using 1 metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic 2 analysis 3 4</h3>
<p style="text-align: justify; text-justify: inter-word;">Uganda. 50 51 Results: The most common pathogens identified were Plasmodium falciparum (51.1% of 52 samples) and parvovirus B19 (4.4%) from blood; human rhinoviruses A and C (40%), 53 respiratory syncytial virus (10%), and human herpesvirus 5 (10%) from nasopharyngeal swabs; 54 and rotavirus A (50% of those with diarrhea) from stool. Among other potential pathogens, we 55 identified one novel orthobunyavirus, tentatively named Nyangole virus, from the blood of a 56 child diagnosed with malaria and pneumonia, and Bwamba orthobunyavirus in the nasopharynx 57 of a child with rash and sepsis. We also identified two novel human rhinovirus C species. 58 59 Conclusions: This exploratory pilot study demonstrates the utility of mNGS and the IDseq 60 platform for defining the molecular landscape of febrile infectious diseases in resource limited 61 areas. These methods, supported by a robust data analysis and sharing platform, offer a new tool 62 for the surveillance, diagnosis, and ultimately treatment and prevention of infectious diseases. 63</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Meta-genomic analysis of toilet waste from long distance flights; a step towards global surveillance of infectious diseases and antimicrobial resistance OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Human populations worldwide are increasingly confronted with infectious diseases and antimicrobial resistance spreading faster and appearing more frequently. Knowledge regarding their occurrence and worldwide transmission is important to control outbreaks and prevent epidemics. Here, we performed shotgun sequencing of toilet waste from 18 international airplanes arriving in Copenhagen, Denmark, from nine cities in three world regions. An average of 18.6 Gb (14.8 to 25.7 Gb) of raw Illumina paired end sequence data was generated, cleaned, trimmed and mapped against reference sequence databases for bacteria and antimicrobial resistance genes. An average of 106,839 (0.06%) reads were assigned to resistance genes with genes encoding resistance to tetracycline, macrolide and beta-lactam resistance genes as the most abundant in all samples. We found significantly higher abundance and diversity of genes encoding antimicrobial resistance, including critical important resistance (e.g. bla CTX-M ) carried on airplanes from South Asia compared to North America. Presence of Salmonella enterica and norovirus were also detected in higher amounts from South Asia, whereas Clostridium difficile was most abundant in samples from North America. Our study provides a first step towards a potential novel strategy for global surveillance enabling simultaneous detection of multiple human health threatening genetic elements, infectious agents and resistance genes.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Enterovirus serotypes in patients with central nervous system and respiratory infections in Viet</h3>
<p style="text-align: justify; text-justify: inter-word;">We describe the diverse spectrum of enteroviruses from patients with CNS and respiratory infections in Viet Nam between 1997 and 2010. These data confirm the global circulation of Enterovirus genera and their associations and are important for clinical diagnostics, patient management, and outbreak response.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Therapeutic Recommendations in the Context of a Global Public Health Crisis using Translational Bioinformatics Approaches: A proof-of-concept study using Nipah Virus Infection</h3>
<p style="text-align: justify; text-justify: inter-word;">We live in a world of emerging new diseases and old diseases resurging in more aggressive forms. Drug development by pharmaceutical companies is a market-driven and costly endeavor, and thus it is often a challenge when drugs are needed for diseases endemic only to certain regions or which affect only a few patients. However, biomedical open data is accessible and reusable for reanalysis and generation of a new hypotheses and discovery.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrated sequence and immunology filovirus database at Los Alamos</h3>
<p style="text-align: justify; text-justify: inter-word;">The Ebola outbreak of 2013-15 infected more than 28 000 people and claimed more lives than all previous filovirus outbreaks combined. Governmental agencies, clinical teams, and the world scientific community pulled together in a multifaceted response ranging from prevention and disease control, to evaluating vaccines and therapeutics in human trials. As this epidemic is finally coming to a close, refocusing on long-term prevention strategies becomes paramount. Given the very real threat of future filovirus outbreaks, and the inherent uncertainty of the next outbreak virus and geographic location, it is prudent to consider the extent and implications of known natural diversity in advancing vaccines and therapeutic approaches. To facilitate such consideration, we have updated and enhanced the content of the filovirus portion of Los Alamos Hemorrhagic Fever Viruses Database. We have integrated and performed baseline analysis of all family Filoviridae sequences deposited into GenBank, with associated immune response data, and metadata, and we have added new computational tools with web-interfaces to assist users with analysis. Here, we (i) describe the main features of updated database, (ii) provide integrated views and some basic analyses summarizing evolutionary patterns as they relate to geo-temporal data captured in the database and (iii) highlight the most conserved regions in the proteome that may be useful for a T cell vaccine strategy.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Title page Observational Research in Childhood Infectious Diseases (ORChID): a dynamic birth cohort study Author affiliations</h3>
<p style="text-align: justify; text-justify: inter-word;">The large biobank of specimens to be collated will act as a rich source of material to answer targeted research questions, including the role of virus acquisition and shedding on clinical illness and the discovery of new infectious agents.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics</h3>
<p style="text-align: justify; text-justify: inter-word;">Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>A Reverse Genetics Platform That Spans the Zika Virus Family Tree</h3>
<p style="text-align: justify; text-justify: inter-word;">Zika virus (ZIKV), a mosquito-borne flavivirus discovered in 1947, has only recently caused large outbreaks and emerged as a significant human pathogen. In 2015, ZIKV was detected in Brazil, and the resulting epidemic has spread throughout the Western Hemisphere. Severe complications from ZIKV infection include neurological disorders such as Guillain-Barré syndrome in adults and a variety of fetal abnormalities, including microcephaly, blindness, placental insufficiency, and fetal demise. There is an urgent need for tools and reagents to study the pathogenesis of epidemic ZIKV and for testing vaccines and antivirals. Using a reverse genetics platform, we generated six ZIKV infectious clones and derivative viruses representing diverse temporal and geographic origins. These include three versions of MR766, the prototype 1947 strain (with and without a glycosylation site in the envelope protein), and H/PF/2013, a 2013 human isolate from French Polynesia representative of the virus introduced to Brazil. In the course of synthesizing a clone of a circulating Brazilian strain, phylogenetic studies identified two distinct ZIKV clades in Brazil. We reconstructed viable clones of strains SPH2015 and BeH819015, representing ancestral members of each clade. We assessed recombinant virus replication, binding to monoclonal antibodies, and virulence in mice. This panel of molecular clones and recombinant virus isolates will enable targeted studies of viral determinants of pathogenesis, adaptation, and evolution, as well as the rational attenuation of contemporary outbreak strains to facilitate the design of vaccines and therapeutics.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Virology Journal Comparative analysis of full genomic sequences among different genotypes of dengue virus type 3</h3>
<p style="text-align: justify; text-justify: inter-word;">Although the previous study demonstrated the envelope protein of dengue viruses is under purifying selection pressure, little is known about the genetic differences of full-length viral genomes of DENV-3. In our study, complete genomic sequencing of DENV-3 strains collected from different geographical locations and isolation years were determined and the sequence diversity as well as selection pressure sites in the DENV genome other than within the E gene were also analyzed.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Respiratory Syncytial Virus whole- genome sequencing identifies convergent evolution of sequence duplication in the C-terminus of the G gene OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">To better understand RSV evolutionary dynamics, we sequenced RSV whole genomes from acutely infected infants from middle Tennessee who were enrolled as part of the Infant Susceptibility to Pulmonary Infections and Asthma Following RSV Exposure (INSPIRE) longitudinal observational birth cohort 25 . The objective of our sequencing efforts was to assess the epidemiological and evolutionary dynamics of RSV from Tennessee within a global context, which is important for identifying RSV strains that feed into the global circulation.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Whole genome sequencing and phylogenetic analysis of human metapneumovirus strains from Kenya and Zambia</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusion: This is the first genomic characterization of HMPV genomes from African patients.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Quality and quantity of dromedary camel DNA sampled from whole-blood, saliva, and tail- hair</h3>
<p style="text-align: justify; text-justify: inter-word;">So far, there have been no thorough investigation of the molecular basis of camels' traits, including desert adaptions, behavioral attributes, aesthetic traits, athletic traits, and traits of economic importance [16] . The dromedary camel genome has been sequenced recently [17, 18] , which opens the door to study the camel genomically. The reference camel genome enabled the identification of genes associated with white-spotting coat color [19] , solid coat colors [20] , and "tameness" [21] . The aforementioned studies employed an approach that consists of sequencing candidate genes [22]-this approach is limited in utility and does not allow for exploring the wide-range of camel phenotypes. The candidate gene approach can be applied only when the biology of the phenotype is fully understood and a prior knowledge of candidate genes and their functions is available [22] . A biobank of camel DNA samples, with associated phenotypes, is needed to utilize genome sequencing and SNP genotyping technologies in linkage analyses [23] or genome-wide association studies [24] . Moreover, validating the association and possibly the causation, of the genetic variants of a given phenotype requires genotyping a large number of individuals (positive and negative for the phenotype) and examining the agreement between their genotype and phenotype [25] . This validation method will benefit greatly from a camel DNA biobank.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Predominant Bacteria Detected from the Middle Ear Fluid of Children Experiencing Otitis Media: A Systematic Review</h3>
<p style="text-align: justify; text-justify: inter-word;">Otitis media (OM) is amongst the most common childhood diseases and is associated with multiple microbial pathogens within the middle ear. Global and temporal monitoring of predominant bacterial pathogens is important to inform new treatment strategies, vaccine development and to monitor the impact of vaccine implementation to improve progress toward global OM prevention.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Phylogenetic nomenclature and evolution of mannose-binding lectin (MBL2) haplotypes</h3>
<p style="text-align: justify; text-justify: inter-word;">In this work, we applied the same method in 288 and 470 chromosomes from Gabonese and European adults, respectively, and found three new haplotypes in the last group. We propose a phylogenetic nomenclature to standardize MBL2 studies and found two major phylogenetic branches due to six strongly linked polymorphisms associated with high MBL production. They presented high Fst values and were imbedded in regions with high nucleotide diversity and significant Tajima's D values. Compared to others using small sample sizes and unphased genotypic data, we found differences in haplotyping, frequency estimation, Fu and Li's D* and Fst results.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Specificity and Strain-Typing Capabilities of Nanorod Array-Surface Enhanced Raman Spectroscopy for Mycoplasma pneumoniae Detection</h3>
<p style="text-align: justify; text-justify: inter-word;">Mycoplasma pneumoniae is a cell wall-less bacterial pathogen of the human respiratory tract that accounts for > 20% of all community-acquired pneumonia (CAP). At present the most effective means for detection and strain-typing is quantitative polymerase chain reaction (qPCR), which can exhibit excellent sensitivity and specificity but requires separate tests for detection and genotyping, lacks standardization between available tests and between labs, and has limited practicality for widespread, point-of-care use. We have developed and previously described a silver nanorod array-surface enhanced Raman Spectroscopy (NA-SERS) biosensing platform capable of detecting M. pneumoniae with statistically significant specificity and sensitivity in simulated and true clinical throat swab samples, and the ability to distinguish between reference strains of the two main genotypes of M. pneumoniae. Furthermore, we have established a qualitative lower endpoint of detection for NA-SERS of < 1 genome equivalent (cell/μl) and a quantitative multivariate detection limit of 5.3 ± 1 cells/μl. Here we demonstrate using partial least squares-discriminatory analysis (PLS-DA) of sample spectra that NA-SERS correctly identified M. pneumoniae clinical isolates from globally diverse origins and distinguished these from a panel of 12 other human commensal and pathogenic mycoplasma species with 100% cross-validated statistical accuracy. Furthermore, PLS-DA correctly classified by strain type all 30 clinical isolates with 96% cross-validated accuracy for type 1 strains, 98% cross-validated accuracy for type 2 strains, and 90% cross-validated accuracy for type 2V strains.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Public health preparedness requires effective surveillance of and rapid response to infectious disease outbreaks. Inclusion of research activities within the outbreak setting provides important opportunities to maximize limited resources, to enhance gains in scientific knowledge, and ultimately to increase levels of preparedness. With rapid advances in laboratory technologies, banking and analysis of human genomic specimens can be conducted as part of public health investigations, enabling valuable research well into the future.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Quality and quantity of dromedary camel DNA sampled from whole-blood, saliva, and tail- hair</h3>
<p style="text-align: justify; text-justify: inter-word;">DNA sequencing for molecular studies were successful in using each of camel blood [17] [18] [19] [20] , tissue [29] , saliva [30] , and hair [19, 21] . However, the question of which DNA source is most useful for camel sample biobanking still remains. We think that for the DNA source to be optimal in camel biobanking, it should have the following attributes: (1) it should be easy and non-invasive to collect, (2) the collection should require no special training, (3) it should neither be pathogenic nor harmful, (4) it should provide adequate DNA quantity and quality, (5) it should be amenable to long-term storage, and (6) it should be easy to transport to the lab and distribute to interested researchers.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent years have seen unprecedented investment in research and development for countermeasures for high-threat pathogens, including specific and ambitious objectives for development of diagnostics, therapeutics, and vaccines. The inadequate availability of biological reference materials for these pathogens poses a genuine obstacle in pursuit of these objectives, and the lack of a comprehensive and equitable framework for developing reference materials is a weakness. We outline the need for internationally standardized biological materials for high-threat pathogens as a core element of global health security. We also outline the key components of a framework for addressing this deficiency.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences, and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and identify touching points and temporal pathways that are candidates for deeper investigation by geneticists and epidemiologists.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /></body>