
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: tracking variations of coronavirus over time</h2><br /></div><br /><br /><br /><div class="card"><h3>Genetic Variation of SARS Coronavirus in Beijing Hospital</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">To characterize genetic variation of severe acute respiratory syndrome–associated coronavirus (SARS-CoV) transmitted in the Beijing area during the epidemic outbreak of 2003, we sequenced 29 full-length S genes of SARS-CoV from 20 hospitalized SARS patients on our unit, the Beijing 302 Hospital. Viral RNA templates for the S-gene amplification were directly extracted from raw clinical samples, including plasma, throat swab, sputum, and stool, during the course of the epidemic in the Beijing area. We used a TA-cloning assay with direct analysis of nested reverse transcription–polymerase chain reaction products in sequence. One hundred thirteen sequence variations with nine recurrent variant sites were identified in analyzed S-gene sequences compared with the BJ01 strain of SARS-CoV. Among them, eight variant sites were, we think, the first documented. Our findings demonstrate the coexistence of S-gene sequences with and without substitutions (referred to BJ01) in samples analyzed from some patients.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</jats:p></p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic diversity of SARS-CoV-2 in Coronavirus Disease 2019 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) has infected more than 75,000 individuals and spread to over 20 countries. It is still unclear how fast the virus evolved and how the virus interacts with other microorganisms in the lung.We have conducted metatranscriptome sequencing for the bronchoalveolar lavage fluid of eight SARS-CoV-2 patients, 25 community-acquired pneumonia (CAP) patients, and 20 healthy controls.The median number of intra-host variants was 1-4 in SARS-CoV-2 infected patients, which ranged between 0 and 51 in different samples. The distribution of variants on genes was similar to those observed in the population data (110 sequences). However, very few intra-host variants were observed in the population as polymorphism, implying either a bottleneck or purifying selection involved in the transmission of the virus, or a consequence of the limited diversity represented in the current polymorphism data. Although current evidence did not support the transmission of intra-host variants in a person-to-person spread, the risk should not be overlooked. The microbiota in SARS-CoV-2 infected patients was similar to those in CAP, either dominated by the pathogens or with elevated levels of oral and upper respiratory commensal bacteria.SARS-CoV-2 evolves in vivo after infection, which may affect its virulence, infectivity, and transmissibility. Although how the intra-host variant spreads in the population is still elusive, it is necessary to strengthen the surveillance of the viral evolution in the population and associated clinical changes.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>The 2019 novel coronavirus resource</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An ongoing outbreak of a novel coronavirus infection in Wuhan, China since December 2019 has led to 31,516 infected persons and 638 deaths across 25 countries (till 16:00 on February 7, 2020). The virus causing this pneumonia was then named as the 2019 novel coronavirus (2019-nCoV) by the World Health Organization. To promote the data sharing and make all relevant information of 2019-nCoV publicly available, we construct the 2019 Novel Coronavirus Resource (2019nCoVR, https://bigd.big.ac.cn/ncov). 2019nCoVR features comprehensive integration of genomic and proteomic sequences as well as their metadata information from the Global Initiative on Sharing All Influenza Data, National Center for Biotechnology Information, China National GeneBank, National Microbiology Data Center and China National Center for Bioinformation (CNCB)/National Genomics Data Center (NGDC). It also incorporates a wide range of relevant information including scientific literatures, news, and popular articles for science dissemination, and provides visualization functionalities for genome variation analysis results based on all collected 2019-nCoV strains. Moreover, by linking seamlessly with related databases in CNCB/NGDC, 2019nCoVR offers virus data submission and sharing services for raw sequence reads and assembled sequences. In this report, we provide comprehensive descriptions on data deposition, management, release and utility in 2019nCoVR, laying important foundations in aid of studies on virus classification and origin, genome variation and evolution, fast detection, drug development and pneumonia precision prevention and therapy.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>A database resource for Genome-wide dynamics analysis of Coronaviruses on a historical and global scale</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-07</span></i><p style="text-align: justify; text-justify: inter-word;">The recent outbreak of a new zoonotic origin Coronavirus has ring the bell for the potential spread of epidemic Coronavirus crossing the species. With the urgent needs to assist the control of the Coronavirus spread and to provide valuable scientific information, we developed a coronavirus database (CoVdb), an online genomics and proteomics analysis platform. Based on public available coronavirus genomic information, the database annotates the genome of every strain and identifies 780 possible ORFs of all strains available in Genebank. In addition, the comprehensive evaluation of all the published genomes of Coronavirus strains, including population genetics analysis, functional analysis and structural analysis on a historical and global scale were presented in the CoVdb. In the database, the researcher can easily obtain the basic information of a Coronavirus gene with the distribution of the gene among strains, conserved or high mutation regions, possible subcellular location and topology of the gene. Moreover, sliding windows for population genetics analysis results is provided, thereby facilitating genetics and evolutional analysis at the genomic level. CoVdb can be accessed freely at http://covdb.popgenetics.net.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>The New Coronavirus, the Current King of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are respiratory viruses whose appearance on electron microscopy looks like tiny crowns (Figure 1). The first coronaviruses discovered were the cause of pediatric and adult respiratory infections that were not particularly dangerous. One study from Vanderbilt found coronavirus in about 5% of samples from upper respiratory illness and 8% from lower respiratory illness [1]. Most clinically significant coronavirus infections have been found in children &lt; 2 years of age, as reviewed in this journal in 2009 [2], although adults also might suffer severe infections [3, 4]. This was the picture as late as 2018: a group of respiratory viruses that could cause serious illness in the very young, but with negligible mortality [5, 6].</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">hina continues to fight the outbreak of a novel coronavirus, called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which has infected tens of thousands and killed more than 2,100 people since the end of 2019. The outbreak’s hot spot is in the central province of Hubei, in particular, its capital city Wuhan, where there have been more than 62,000 infections and 2,029 deaths as of last week, according to the World Health Organization (WHO). There, doctors and scientists are desperately struggling to find reliable ways to diagnose infected patients to help treat them and control the spread of the virus. The struggle has involved two diagnostic options—computed tomography (CT) scans of patients’ lungs and a nucleic acid lab test—each with advantages and disadvantages. With “a suddenly rampant new virus, it is normal to have multiple test approaches,” says a virologist at the Chinese Center for Disease Control and Prevention</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>Variations in Spike Glycoprotein Gene of MERS-CoV, South Korea, 2015</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of nosocomial infections with Middle East respiratory syndrome coronavirus occurred in South Korea in May 2015. Spike glycoprotein genes of virus strains from South Korea were closely related to those of strains from Riyadh, Saudi Arabia. However, virus strains from South Korea showed strain-specific variations.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Scientists question China’s decision not to report symptom-free coronavirus cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Researchers say that excluding these people could conceal the epidemic’s true extent, but others say the practice makes sense. Researchers are concerned that China’s official reports on the number of coronavirus infections have not been including people who have tested positive for the virus but who have no symptoms. They fear the practice is masking the epidemic’s true scale. But public health experts say China is right to prioritize tracking sick patients who are spreading the disease.</p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Background: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Diagnostic test based on real-time reverse transcription polymerase chain reaction assay (qRT-PCR) was the main means of confirmation, and sample collection was mostly throat swabs, which was easy to miss the diagnosis. It is necessary to seek specimen types with higher detection efficiency and accuracy. Methods: Paired specimens of throat swabs and sputum were obtained from 54 cases, and RNA was extracted and tested for 2019-nCoV (equated with SARS-CoV-2) by qRT-PCR assay. Results: The positive rates of 2019-nCoV from sputum specimens and throat swabs were 76.9% and 44.2%, respectively. Sputum specimens showed a significantly higher positive rate than throat swabs in detecting viral nucleic acid using qRT-PCR assay (P=0.001). Conclusions: The detection rates of 2019-nCoV from sputum specimens are significantly higher than throat swabs. We suggest that sputum would benefit for the detection of 2019-nCoV in patients who produce sputum. The results can facilitate the selection of specimens and increase the accuracy of diagnosis.</jats:p></p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /><div class="card"><h3>An eight-year epidemiologic study based on baculovirus-expressed type-specific spike proteins for the differentiation of type I and II feline coronavirus infections</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 15</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Feline infectious peritonitis (FIP) is a fatal disease caused by feline coronavirus (FCoV). FCoVs are divided into two serotypes with markedly different infection rates among cat populations around the world. A baculovirus-expressed type-specific domain of the spike proteins of FCoV was used to survey the infection of the two viruses over the past eight years in Taiwan. RESULTS: An immunofluorescence assay based on cells infected with the recombinant viruses that was capable of distinguishing between the two types of viral infection was established. A total of 833 cases from a teaching hospital was surveyed for prevalence of different FCoV infections. Infection of the type I FCoV was dominant, with a seropositive rate of 70.4%, whereas 3.5% of cats were infected with the type II FCoV. In most cases, results derived from serotyping and genotyping were highly agreeable. However, 16.7% (4/24) FIP cats and 9.8% (6/61) clinically healthy cats were found to possess antibodies against both viruses. Moreover, most of the cats (84.6%, 22/26) infected with a genotypic untypable virus bearing a type I FCoV antibody. CONCLUSION: A relatively simple serotyping method to distinguish between two types of FCoV infection was developed. Based on this method, two types of FCoV infection in Taiwan was first carried out. Type I FCoV was found to be predominant compared with type II virus. Results derived from serotyping and genotyping support our current understanding of evolution of disease-related FCoV and transmission of FIP.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic drift of human coronavirus OC43 spike gene during adaptive evolution</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jun 22</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) continuously threaten human health. However, to date, the evolutionary mechanisms that govern CoV strain persistence in human populations have not been fully understood. In this study, we characterized the evolution of the major antigen-spike (S) gene in the most prevalent human coronavirus (HCoV) OC43 using phylogenetic and phylodynamic analysis. Among the five known HCoV-OC43 genotypes (A to E), higher substitution rates and dN/dS values as well as more positive selection sites were detected in the S gene of genotype D, corresponding to the most dominant HCoV epidemic in recent years. Further analysis showed that the majority of substitutions were located in the S1 subunit. Among them, seven positive selection sites were chronologically traced in the temporal evolution routes of genotype D, and six were located around the critical sugar binding region in the N-terminal domain (NTD) of S protein, an important sugar binding domain of CoV. These findings suggest that the genetic drift of the S gene may play an important role in genotype persistence in human populations, providing insights into the mechanisms of HCoV-OC43 adaptive evolution.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic drift of human coronavirus OC43 spike gene during adaptive evolution</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jun 22</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) continuously threaten human health. However, to date, the evolutionary mechanisms that govern CoV strain persistence in human populations have not been fully understood. In this study, we characterized the evolution of the major antigen-spike (S) gene in the most prevalent human coronavirus (HCoV) OC43 using phylogenetic and phylodynamic analysis. Among the five known HCoV-OC43 genotypes (A to E), higher substitution rates and dN/dS values as well as more positive selection sites were detected in the S gene of genotype D, corresponding to the most dominant HCoV epidemic in recent years. Further analysis showed that the majority of substitutions were located in the S1 subunit. Among them, seven positive selection sites were chronologically traced in the temporal evolution routes of genotype D, and six were located around the critical sugar binding region in the N-terminal domain (NTD) of S protein, an important sugar binding domain of CoV. These findings suggest that the genetic drift of the S gene may play an important role in genotype persistence in human populations, providing insights into the mechanisms of HCoV-OC43 adaptive evolution.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: global infections pass 90,000</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Updates on the respiratory illness that has infected tens of thousands of people. The number of people worldwide who have been infected with the coronavirus has passed 90,000. More than 3,000 people have died since the outbreak began in December. The vast majority of cases — more than 80,000 — have occurred in China, but around 60 other countries are now also dealing with outbreaks. Many nations are preparing for a global pandemic, as reports of cases caused by community spread — rather than importation from China — rise. South Korea, Italy and Iran are fighting the largest outbreaks outside of China.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Co-Circulation of Canine Coronavirus I and IIa/b with High Prevalence and Genetic Diversity in Heilongjiang Province, Northeast China</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 15</span></i><p style="text-align: justify; text-justify: inter-word;">To trace the evolution of canine coronavirus (CCoV), 201 stool samples from diarrheic dogs in northeast China were subjected to reverse transcription-polymerase chain reactions (RT-PCRs) targeting the partial M and S genes of CCoV, followed by an epidemiological analysis. M gene RT-PCRs showed that 28.36% (57/201) of the samples were positive for CCoV; of the 57 positive samples, CCoV-I and CCoV-II accounted for 15.79% (9/57) and 84.21% (48/57), respectively. A sequence comparison of the partial M gene revealed nucleotide homologies of 88.4%–100% among the 57 CCoV strains, and 88.7%–96.2% identity between the 57 CCoV strains and the Chinese reference strain HF3. The CCoV-I and CCoV-II strains exhibited genetic diversity when compared with reference strains from China and other countries. The 57 CCoV strains exhibited high co-infection rates with canine kobuvirus (CaKV) (33.33%) and canine parvovirus-2 (CPV-2) (31.58%). The CCoV prevalence in diarrheic dogs differed significantly with immunization status, regions, seasons, and ages. Moreover, 28 S genes were amplified from the 57 CCoV-positive samples, including 26 CCoV-IIa strains, one CCoV-IIb strain, and one CCoV-I strain. A sequence comparison of the partial S gene revealed 86.3%–100% nucleotide identity among the 26 CCoV-IIa strains, and 89.6%–92.2% identity between the 26 CCoV-IIa strains and the Chinese reference strain V1. The 26 CCoV-IIa strains showed genetic diversity when compared with reference strains from China and other countries. Our data provide evidence that CCoV-I, CCoV-IIa, and CCoV-IIb strains co-circulate in the diarrhoetic dogs in northeast China, high co-infection rates with CaKV and CPV-2 were observed, and the CCoV-II strains exhibited high prevalence and genetic diversity.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple and rapid approach for screening of SARS-coronavirus genotypes: an evaluation study</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Oct 18</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The Severe Acute Respiratory Syndrome (SARS) was a newly emerged infectious disease which caused a global epidemic in 2002–2003. Sequence analysis of SARS-coronavirus isolates revealed that specific genotypes predominated at different periods of the epidemic. This information can be used as a footprint for tracing the epidemiology of infections and monitor viral evolution. However, direct sequencing analysis of a large number of clinical samples is cumbersome and time consuming. We present here a simple and rapid assay for the screening of SARS-coronavirus genotypes based on the use of fluorogenic oligonucleotide probes for allelic discrimination. METHODS: Thirty SARS patients were recruited. Allelic discrimination assays were developed based on the use of fluorogenic oligonucleotide probes (TaqMan). Genotyping of the SARS-coronavirus isolates obtained from these patients were carried out by the allelic discrimination assays and confirmed by direct sequencing. RESULTS: Genotyping based on the allelic discrimination assays were fully concordant with direct sequencing. All of the 30 SARS-coronavirus genotypes studied were characteristic of genotypes previously documented to be associated with the latter part of the epidemic. Seven of the isolates contained a previously reported major deletion but in patients not epidemiologically related to the previously studied cohort. CONCLUSION: We have developed a simple and accurate method for the characterization and screening of SARS-coronavirus genotypes. It is a promising tool for the study of epidemiological relationships between documented cases during an outbreak.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and genetic diversity analysis of human coronaviruses among cross-border children</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: More than a decade after the outbreak of human coronaviruses (HCoVs) SARS in Guangdong province and Hong Kong SAR of China in 2002, there is still no reoccurrence, but the evolution and recombination of the coronaviruses in this region are still unknown. Therefore, surveillance on the prevalence and the virus variation of HCoVs circulation in this region is conducted. METHODS: A total of 3298 nasopharyngeal swabs samples were collected from cross-border children (<6 years, crossing border between Southern China and Hong Kong SAR) showing symptoms of respiratory tract infection, such as fever (body temperature > 37.5 °C), from 2014 May to 2015 Dec. Viral nucleic acids were analyzed and sequenced to study the prevalence and genetic diversity of the four human coronaviruses. The statistical significance of the data was evaluated with Fisher chi-square test. RESULTS: 78 (2.37%; 95%CI 1.8-2.8%) out of 3298 nasopharyngeal swabs specimens were found to be positive for OC43 (36;1.09%), HKU1 (34; 1.03%), NL63 (6; 0.18%) and 229E (2;0.01%). None of SARS or MERS was detected. The HCoVs predominant circulating season was in transition of winter to spring, especially January and February and NL63 detected only in summer and fall. Complex population with an abundant genetic diversity of coronaviruses was circulating and they shared homology with the published strains (99-100%). Besides, phylogenetic evolutionary analysis indicated that OC43 coronaviruses were clustered into three clades (B,D,E), HKU1 clustered into two clades(A,B) and NL63 clustered into two clades(A,B). Moreover, several novel mutations including nucleotides substitution and the insertion of spike of the glycoprotein on the viral surface were discovered. CONCLUSIONS: The detection rate and epidemic trend of coronaviruses were stable and no obvious fluctuations were found. The detected coronaviruses shared a conserved gene sequences in S and RdRp. However, mutants of the epidemic strains were detected, suggesting continuous monitoring of the human coronaviruses is in need among cross-border children, who are more likely to get infected and transmit the viruses across the border easily, in addition to the general public.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>The role of absolute humidity on transmission rates of the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (COVID-19) was identified in Wuhan, Hubei Province, China, in December 2019 and has caused over 40,000 cases worldwide to date. Previous studies have supported an epidemiological hypothesis that cold and dry (low absolute humidity) environments facilitate the survival and spread of droplet-mediated viral diseases, and warm and humid (high absolute humidity) environments see attenuated viral transmission (i.e., influenza). However, the role of absolute humidity in transmission of COVID-19 has not yet been established. Here, we examine province-level variability of the basic reproductive numbers of COVID-19 across China and find that changes in weather alone (i.e., increase of temperature and humidity as spring and summer months arrive in the North Hemisphere) will not necessarily lead to declines in COVID-19 case counts without the implementation of extensive public health interventions.</jats:p></p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of SARS-associated Coronavirus, Beijing</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Single nucleotide variations (SNVs) at 5 loci (17564, 21721, 22222, 23823, and 27827) were used to define the molecular epidemiologic characteristics of severe acute respiratory syndrome–associated coronavirus (SARS-CoV) from Beijing patients. Five fragments targeted at the SNV loci were amplified directly from clinical samples by using reverse transcription–polymerase chain reaction (RT-PCR), before sequencing the amplified products. Analyses of 45 sequences obtained from 29 patients showed that the GGCTC motif dominated among samples collected from March to early April 2003; the TGTTT motif predominanted afterwards. The switch from GGCTC to TGTTT was observed among patients belonging to the same cluster, which ruled out the possibility of the coincidental superposition of 2 epidemics running in parallel in Beijing. The Beijing isolates underwent the same change pattern reported from Guangdong Province. The same series of mutations occurring in separate geographic locations and at different times suggests a dominant process of viral adaptation to the host.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>First two months of the 2019 Coronavirus Disease (COVID-19) epidemic in China: real-time surveillance and evaluation with a second derivative model</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: Scientists clash over virus name</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Some researchers in China are unhappy with the designated name for the new coronavirus, SARS-CoV-2. They worry that the use of ‘SARS-CoV’ will confuse the public and impede efforts to control the pathogen’s spread.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Characteristics of Coronavirus Pneumonia 2019 (COVID-19): An Updated Systematic Review</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>OBJECTIVE Clinical characteristics of novel coronavirus pneumonia (COVID-19) have been described in numerous studies but yielded varying results. We aimed to conduct a systematic review on scientific literatures and to synthesize critical data on clinical traits of COVID-19 from its initial outbreak to pandemic. METHODS Systematic searches were conducted to identify retrospective observational study that contained clinical characteristics on COVID-19 through multiple databases. Two reviewers independently evaluated eligible publications. Data on clinical characteristics of COVID-19 were extracted and analyzed. RESULTS Seventy-two retrospective studies demonstrating the clinical characteristics of COVID-19 were included. A total of 3470 COVID-19 patients were synthesized to the final analysis in an unbiased manner. The most common symptom was fever (2878 [83.0%]), and 63.4% of the patients presented fever as onset symptom. There were 2528 [88.2%] of 2866 cases had abnormal lung findings on chest CT scan. Laboratory findings showed that 1498 [62.8%] of 2387 cases had lymphopenia, and 1354 [64.8%] of 2091 cases had an increased level of C-reactive protein (CRP). A total of 185 [11.5%] patients were admitted to intensive care unit (ICU) while the overall case fatality rate (CFR) was 3.7%. Compared to patients admitted outside of Hubei, China, those from Hubei had a significant higher ICU admission rate (21.9% vs. 2.5%, p&lt;0.001). Also, CFR attributed to COVID-19 was significantly higher in Hubei than that of non-Hubei admissions (10.4% vs. 0.6%, p&lt;0.001).  INTERPRETATION This large patient-based systematic review presents a more precise profiling of the COVID-19 from its outbreak to current pandemic. Dynamic evolvements of COVID-19 are needed to be characterized in future studies.</jats:p></p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Trends in Transmissibility of 2019 Novel Coronavirus-infected Pneumonia in Wuhan and 29 Provinces in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The 2019 novel coronavirus infected pneumonia (COVID- 19) represents a significant public health threat. The COVID-19 emerged in December 2019 in Wuhan, China and rapidly spread to other regions and countries. The variation in transmission patterns and disease spread in regard to time or among different locations, partially reflecting the public health in- tervention effects, remains to be quantified. As most transmissibility-related epidemic parameters are unknown, we sought, with minimal assumptions, to estimate real-time transmissibility and forecast new cases using dynamic modelling. Method: Using the cases reported from the National Health Commission of China and transportation data, including the total number of travelling hours through railway, airplane, and car outbound from Wuhan, we have built a time-series model to estimate real-time underlying transmission rates of newly generated cases sequentially from January 20, 2020 to Feb 13, 2020 in Wuhan, Hubei province and other 28 provinces in China. We quantified the instantaneous transmission rate and relative reproduction number (R_t) of COVID-19, and evaluated whether public health intervention affected the case transmissibility in each province. Based on the current estimates, we have predicted the trend of disease spread with a high level of certainty. Findings: We estimated that R_t declined from the range of 4 to 5 towards 1 and remained below unity, while there was an initial growth followed by a decline in a shorter period in Hubei and other provinces. The ratio of transmission rates decreased dramatically from January 23 to 27 likely due to the rigorous public health intervention implemented by the government beginning on January 23, 2020. The mean duration of the infectious period was 6 to 9 days. We have predicted the trend of infection sizes which be- came stable in provinces around February 19 to 24, 2020, and the date of containment would be one-week later in Wuhan. Interpretation: Public health interventions implemented at both the so- cial and personal levels are effective in preventing outbreaks of COVID-19 in Wuhan and other provinces. Model prediction results suggested that COVID- 19 will be contained around the end of February 2020 in China. Keywords: Coronavirus, COVID-19, transmissibility, dynamic reproduction number R, statistical modelling, pneumonia outbreak</jats:p></p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution and variation of 2019-novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-30</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.  MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.  ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1{middle dot}05 x 10-2 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 x 10-4 to 2.72 x 10-2) for N, 5.34 x 10-3 (5.10 x 10-4, 1.28 x 10-2) for S, 1.69 x 10-3 (3.94 x 10-4, 3.60 x 10-3) for P, 1.65 x 10-3 (4.47 x 10-4, 3.24 x 10-3) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.  ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>M gene analysis of canine coronavirus strains detected in Korea</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Dec 15</span></i><p style="text-align: justify; text-justify: inter-word;">The purpose of this study was to investigate the genetic features of canine coronavirus (CCV) strains detected in Korea. M gene sequences obtained for isolates from 22 dogs with enteritis over a 5-year period were evaluated. Sequence comparison revealed that the 22 Korean CCV strains had an 87.2 to 100% nucleotide homology. Comparing to the typical reference CCV strains (type II), the nucleotide sequence of Korean strains had homology ranged from 86.3% to 98.3% (89.1% to 99.2% for the amino acid sequence) and 87.7% to 97.8% (92.4% to 100% for the amino acid sequence) when compared to FCoV-like CCV strains (type I). Three amino acid variations in the M gene were characteristic for the Korean CCV strains. Phylogenetic analysis demonstrated that the 22 Korean CCV strains belonged to four typical CCV clusters (i.e., a unique Korean CCV cluster, a type II and transmissible gastroenteritis virus cluster, an intermediate cluster between type I and II, and a type I cluster). This study was the first to identify genetic differences of the M gene from Korean CCV strains and provided a platform for molecular identification of different Korean CCV strains.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>How Coronaviruses Cause Infection—from Colds to Deadly Pneumonia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus outbreak raises questions about how such pathogens evolve and what makes infections mild or severe</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>The novel coronavirus outbreak: what can be learned from China in public reporting?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The new coronavirus outbreak gets everyone’s attention. China’s national actions against the outbreak have contributed great contributions to the world. China has been learning from practice for better reporting and is fast to adapt itself. In this article we discuss China’s practice in public reporting and its implications to global health. Confirmed cases, dynamic suspected cases, recovered cases, and deaths have been reported both in accumulative numbers and their daily updates. Some ratio indictors reporting (fatality rate, recovery rate, etc.), trend reporting, and global surveillance have been applied as well. Some improvements can still be made. It is necessary to further explore the influential factors behind the indicators for interventions. Recommendations are made to the World Health Organization and other countries for better public reporting and surveillance.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of intrapatient heterogeneity uncovers the microevolution of Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Genome sequence analysis of Middle East respiratory syndrome coronavirus (MERS-CoV) variants from patient specimens has revealed the evolutionary dynamics and mechanisms of pathogenesis of the virus. However, most studies have analyzed the consensus sequences of MERS-CoVs, precluding an investigation of intrapatient heterogeneity. Here, we analyzed non–consensus sequences to characterize intrapatient heterogeneity in cases associated with the 2015 outbreak of MERS in South Korea. Deep-sequencing analysis of MERS-CoV genomes performed on specimens from eight patients revealed significant intrapatient variation; therefore, sequence heterogeneity was further analyzed using targeted deep sequencing. A total of 35 specimens from 24 patients (including a super-spreader) were sequenced to detect and analyze variants displaying intrapatient heterogeneity. Based on the analysis of non–consensus sequences, we demonstrated the intrapatient heterogeneity of MERS-CoVs, with the highest level in the super-spreader specimen. The heterogeneity could be transmitted in a close association with variation in the consensus sequences, suggesting the occurrence of multiple MERS-CoV infections. Analysis of intrapatient heterogeneity revealed a relationship between D510G and I529T mutations in the receptor-binding domain (RBD) of the viral spike glycoprotein. These two mutations have been reported to reduce the affinity of the RBD for human CD26. Notably, although the frequency of both D510G and I529T varied greatly among specimens, the combined frequency of the single mutants was consistently high (87.7% ± 1.9% on average). Concurrently, the frequency of occurrence of the wild type at the two positions was only 6.5% ± 1.7% on average, supporting the hypothesis that selection pressure exerted by the host immune response played a critical role in shaping genetic variants and their interaction in human MERS-CoVs during the outbreak.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory Diagnosis of SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">The virologic test results of 415 patients with severe acute respiratory syndrome (SARS) were examined. The peak detection rate for SARS-associated coronavirus occurred at week 2 after illness onset for respiratory specimens, at weeks 2 to 3 for stool or rectal swab specimens, and at week 4 for urine specimens. The latest stool sample that was positive by reverse transcription–polymerase chain reaction (RT-PCR) was collected on day 75 while the patient was receiving intensive care. Tracheal aspirate and stool samples had a higher diagnostic yield (RT-PCR average positive rate for first 2 weeks: 66.7% and 56.5%, respectively). Pooled throat and nasal swabs, rectal swab, nasal swab, throat swab, and nasopharyngeal aspirate specimens provided a moderate yield (29.7%–40.0%), whereas throat washing and urine specimens showed a lower yield (17.3% and 4.5%). The collection procedures for stool and pooled nasal and throat swab specimens were the least likely to transmit infection, and the combination gave the highest yield for coronavirus detection by RT-PCR. Positive virologic test results in patient groups were associated with mechanical ventilation or death (p < 0.001), suggesting a correlation between viral load and disease severity.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic Analysis of 15 Human Coronaviruses OC43 (HCoV-OC43s) Circulating in France from 2001 to 2013 Reveals a High Intra-Specific Diversity with New Recombinant Genotypes</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 May 7</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus OC43 (HCoV-OC43) is one of five currently circulating human coronaviruses responsible for respiratory infections. Like all coronaviruses, it is characterized by its genome’s high plasticity. The objectives of the current study were to detect genetically distinct genotypes and eventually recombinant genotypes in samples collected in Lower Normandy between 2001 and 2013. To this end, we sequenced complete nsp12, S, and N genes of 15 molecular isolates of HCoV-OC43 from clinical samples and compared them to available data from the USA, Belgium, and Hong-Kong. A new cluster E was invariably detected from nsp12, S, and N data while the analysis of nsp12 and N genes revealed the existence of new F and G clusters respectively. The association of these different clusters of genes in our specimens led to the description of thirteen genetically distinct genotypes, among which eight recombinant viruses were discovered. Identification of these recombinant viruses, together with temporal analysis and tMRCA estimation, provides important information for understanding the dynamics of the evolution of these epidemic coronaviruses.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Open-source analytics tools for studying the COVID-19 coronavirus outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>To provide convenient access to epidemiological data on the coronavirus outbreak, we developed an R package, nCov2019 (https://github.com/GuangchuangYu/nCov2019). Besides detailed real-time statistics, it offers access to three data sources with detailed daily statistics from December 1, 2019, for 43 countries and more than 500 Chinese cities. We also developed a web app (http://www.bcloud.org/e/) with interactive plots and simple time-series forecasts. These analytics tools could be useful in informing the public and studying how this and similar viruses spread in populous countries.</jats:p></p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology characteristics of human coronaviruses in patients with respiratory infection symptoms and phylogenetic analysis of HCoV-OC43 during 2010-2015 in Guangzhou</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus (HCoV) is one of the most common causes of respiratory tract infection throughout the world. To investigate the epidemiological and genetic variation of HCoV in Guangzhou, south China, we collected totally 13048 throat and nasal swab specimens from adults and children with fever and acute upper respiratory infection symptoms in Gunazhou, south China between July 2010 and June 2015, and the epidemiological features of HCoV and its species were studied. Specimens were screened for HCoV by real-time RT-PCR, and 7 other common respiratory viruses were tested simultaneously by PCR or real-time PCR. HCoV was detected in 294 cases (2.25%) of the 13048 samples, with most of them inpatients (251 cases, 85.4% of HCoV positive cases) and young children not in nursery (53.06%, 156 out of 294 HCoV positive cases). Four HCoVs, as OC43, 229E, NL63 and HKU1 were detected prevalent during 2010–2015 in Guangzhou, and among the HCoV positive cases, 60.20% were OC43, 16.67% were 229E, 14.97% were NL63 and 7.82% were HKU1. The month distribution showed that totally HCoV was prevalent in winter, but differences existed in different species. The 5 year distribution of HCoV showed a peak-valley distribution trend, with the detection rate higher in 2011 and 2013 whereas lower in 2010, 2012 and 2014. The age distribution revealed that children (especially those <3 years old) and old people (>50 years) were both high risk groups to be infected by HCoV. Of the 294 HCoV positive patients, 34.69% (101 cases) were co-infected by other common respiratory viruses, and influenza virus was the most common co-infecting virus (30/101, 29.70%). Fifteen HCoV-OC43 positive samples of 2013–2014 were selected for S gene sequencing and phylogenetic analysis, and the results showed that the 15 strains could be divided into 2 clusters in the phylogenetic tree, 12 strains of which formed a separate cluster that was closer to genotype G found in Malaysia. It was revealed for the first time that genotype B and genotype G of HCoV-OC43 co-circulated and the newly defined genotype G was epidemic as a dominant genotype during 2013–2014 in Guanzhou, south China.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential T-cell and B-cell Epitopes of 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-21</span></i><p style="text-align: justify; text-justify: inter-word;">As of Feb 16th 2020, 2019-nCoV has infected more than 51,857 people across 26 countries and claimed 1666 lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for 2019-nCoV or any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. We validated our computational pipeline with SARS-CoV experimental data.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequences of Two Chinese Virulent Avian Coronavirus Infectious Bronchitis Virus Variants</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Avian coronavirus infectious bronchitis virus (IBV) is variable, which causes many serotypes. Here we reported the complete genome sequences of two virulent IBV variants from China, GX-YL5 and GX-YL9, belonging to different serotypes. Differences between GX-YL5 and GX-YL9 were found mainly in stem-loop structure I in the predicted RNA secondary structure of open reading frame (ORF) 1b and the S protein gene fusion region, which will help us understand the molecular evolutionary mechanism of IBV and the disconcordance between the genotypes and serotypes of coronavirus.</p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Characterization of Major Structural Protein Genes of Avian Coronavirus Infectious Bronchitis Virus Isolates in Southern China</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Dec 4</span></i><p style="text-align: justify; text-justify: inter-word;">To gain comprehensive genetic information of circulating avian coronavirus infectious bronchitis virus (IBV) isolates in China, analysis of the phylogenetic tree, entropy of the amino acid sequences, and the positive selection as well as computational recombinations of S1, M and N genes of 23 IBV isolates was conducted in the present study. The phylogenetic trees based on the S1, M and N genes exhibited considerably different topology and the CK/CH/LSC/99I-type isolates were the predominant IBVs based on the phylogenetic analysis of S1 gene. Results of entropy of amino acid sequences revealed that the S1 gene had the largest variation; the M gene had less variation than the N gene. Positive selections were detected in not only S1 but also M and N gene proteins. In addition, five S1 gene recombinants between vaccine strain 4/91 and CK/CH/LSC/99I-type field isolate were confirmed. In conclusion, multiple IBV genotypes co-circulated; genetic diversity and positive selections existed in S1, M and N genes; 4/91 vaccine recombinants emerged in China. Our results show that field IBVs in China are continuing to evolve and vaccine strains may have an important role in the appearance of new IBV strains via recombination. In addition, the present study indicates that IBV evolution is driven by both generations of genetic diversity and selection.</p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus nixes conference, twilight zone beckons and a faded star brightens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus enters dangerous new phase The new coronavirus has spread to more than 70 nations and the total number of infections worldwide had passed 90,000 as Nature went to press (see ‘Rapid spread’). Researchers have warned that the surge in outbreaks outside China, where the virus emerged and most cases have occurred, means that the coronavirus is becoming unstoppable. The World Health Organization has resisted describing the situation as a pandemic. Director-general Tedros Adhanom Ghebreyesus said on 2 March that there was still a chance of containing the virus. Mike Ryan, director of the WHO’s emergencies programme, said that using the word pandemic would mean that efforts to contain and slow the spread of the virus have failed, which has proved untrue in China, Singapore and other regions. But other scientists say the surge in international cases marks a tipping point. “I think the epidemiological conditions for a pandemic are met,” says Marc Lipsitch, an infectious-disease epidemiologist at the Harvard T.H. Chan School of Public Health in Boston, Massachusetts. He and others say that although containment measures seem to have kept outbreaks from escalating outside China for more than a month, such procedures might soon become unfeasible on a broader scale. Those efforts have involved quickly identifying infected people and their close contacts, and isolating them to prevent further transmission. “We’ve got to think more carefully about what measures might be sustainable in terms of reducing transmission without shutting down cities completely and stopping people from moving,” says Ben Cowling, an infectious-disease epidemiologist at the University of Hong Kong. The efforts include ‘social distancing’, which reduces the average chances that uninfected people will encounter an infected person. But some epidemiologists say too little is known about the outbreak to deploy this effectively.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Novo Coronavirus (nCoV)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Os Coronavírus (CoV) compõem uma grande família de vírus, conhecidos desde meados da década de 1960. Podem causar desde um resfriado comum até síndromes respiratórias graves, como a síndrome respiratória aguda grave (SARS - Severe Acute Respiratory Syndrome) e a síndrome respiratória do Oriente Médio (MERS - Middle East Respiratory Syndrome). Os casos agora identificados estão relacionados a uma nova variante do Coronavírus, denominada 2019-nCoV, até então não identificada em humanos.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Gene Variations in Cis-Acting Elements between the Taiwan and Prototype Strains of Porcine Epidemic Diarrhea Virus Alter Viral Gene Expression</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Nov 29</span></i><p style="text-align: justify; text-justify: inter-word;">In 2013, the outbreak of porcine epidemic diarrhea (PED) in Taiwan caused serious economic losses. In this study, we examined whether the variations of the cis-acting elements between the porcine epidemic diarrhea virus (PEDV) Taiwan (TW) strain and the prototype strain CV777 alter gene expression. For this aim, we analyzed the variations of the cis-acting elements in the 5′ and 3′ untranslated regions (UTRs) between the PEDV TW, CV777, and other reference strains. We also determined the previously unidentified transcription regulatory sequence (TRS), a sequence motif required for coronavirus transcription, and found that a nucleotide deletion in the TW strain, in comparison with CV777 strain, immediately downstream of the leader core sequence alters the identity between the leader TRS and the body TRS. Functional analyses using coronavirus defective interfering (DI) RNA revealed that such variations in cis-acting elements for the TW strain compared with the CV777 strain have an influence on the efficiency of gene expression. The current data show for the first time the evolution of PEDV in terms of cis-acting elements and their effects on gene expression, and thus may contribute to our understanding of recent PED outbreaks worldwide.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Gene Variations in Cis-Acting Elements between the Taiwan and Prototype Strains of Porcine Epidemic Diarrhea Virus Alter Viral Gene Expression</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Nov 29</span></i><p style="text-align: justify; text-justify: inter-word;">In 2013, the outbreak of porcine epidemic diarrhea (PED) in Taiwan caused serious economic losses. In this study, we examined whether the variations of the cis-acting elements between the porcine epidemic diarrhea virus (PEDV) Taiwan (TW) strain and the prototype strain CV777 alter gene expression. For this aim, we analyzed the variations of the cis-acting elements in the 5′ and 3′ untranslated regions (UTRs) between the PEDV TW, CV777, and other reference strains. We also determined the previously unidentified transcription regulatory sequence (TRS), a sequence motif required for coronavirus transcription, and found that a nucleotide deletion in the TW strain, in comparison with CV777 strain, immediately downstream of the leader core sequence alters the identity between the leader TRS and the body TRS. Functional analyses using coronavirus defective interfering (DI) RNA revealed that such variations in cis-acting elements for the TW strain compared with the CV777 strain have an influence on the efficiency of gene expression. The current data show for the first time the evolution of PEDV in terms of cis-acting elements and their effects on gene expression, and thus may contribute to our understanding of recent PED outbreaks worldwide.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>The epidemiology and pathogenesis of coronavirus disease (COVID-19) outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease (COVID-19) is caused by SARS-COV2 and represents the causative agent of a potentially fatal disease that is of great global public health concern. Based on the large number of infected people that were exposed to the wet animal market in Wuhan City, China, it is suggested that this is likely the zoonotic origin of COVID-19. Person-to-person transmission of COVID-19 infection led to the isolation of patients that were subsequently administered a variety of treatments. Extensive measures to reduce person-to-person transmission of COVID-19 have been implemented to control the current outbreak. Special attention and efforts to protect or reduce transmission should be applied in susceptible populations including children, health care providers, and elderly people. In this review, we highlights the symptoms, epidemiology, transmission, pathogenesis, phylogenetic analysis and future directions to control the spread of this fatal disease.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus Infections in Israel: Epidemiology, Clinical Symptoms and Summer Seasonality of HCoV-HKU1</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Sep 21</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses (HCoVs) cause mild to severe respiratory diseases. Six types of HCoVs have been discovered, the most recent one termed the Middle East respiratory syndrome coronavirus (MERS-CoV). The aim of this study is to monitor the circulation of HCoV types in the population during 2015–2016 in Israel. HCoVs were detected by real-time PCR analysis in 1910 respiratory samples, collected from influenza-like illness (ILI) patients during the winter sentinel influenza survey across Israel. Moreover, 195 HCoV-positive samples from hospitalized patients were detected during one year at Soroka University Medical Center. While no MERS-CoV infections were detected, 10.36% of patients in the survey were infected with HCoV-OC43 (43.43%), HCoV-NL63 (44.95%), and HCoV-229E (11.62%) viruses. The HCoVs were shown to co-circulate with respiratory syncytial virus (RSV) and to appear prior to influenza virus infections. HCoV clinical symptoms were more severe than those of RSV infections but milder than influenza symptoms. Hospitalized patients had similar HCoV types percentages. However, while it was absent from the public winter survey, 22.6% of the patients were HCoV-HKU1 positives, mainly during the spring-summer period.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses in animals and humans</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Controlling outbreaks will require detailed knowledge of their biology and behaviourCoronaviruses have been around for many years and were first discovered in the 1960s. They include viruses contributing to the common cold (HCoV-229E) and a variety of animal and avian coronaviruses, such as infectious bronchitis virus (IBV), which infects poultry. Coronaviruses typically cause respiratory or gastrointestinal illness, but strains of IBV have been shown to target the oviduct in chickens, and others can cause severe kidney disease.Animal and avian coronaviruses can have high mortality rates among infected animals and illustrate the difficulties in developing vaccines. Similar to influenza viruses, despite many decades of research there is no vaccine that protects against all strains of IBV coronavirus. This is due in part to the continuously shifting diversity in the virus spike glycoprotein, a major immunogenic target and hence a good vaccine candidate for animal and human infections.During the mid-1990s these viruses were described as the backwater of virology, since none caused serious disease in humans. However, this changed in 2002-03 with …</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of the 2019 Novel Coronavirus (2019-nCoV) has rapidly spread from Wuhan, China to multiple countries, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of 2019-nCoV is lacking, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development. In this study, we predicted the potential of all the 2019-nCoV viral proteins to induce class I and II MHC presentation and form linear antibody epitopes. We showed that the enrichment for T cell and B cell epitopes is not uniform on the viral genome, with several focused regions that generate abundant epitopes and may be more targetable. We showed that genetic variations in 2019-nCoV, though fewer for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus, which should be considered in the development of therapies. We create an online database to broadly share our research outcome. Overall, we present an immunological resource for 2019-nCoV that could significantly promote both therapeutic development and mechanistic research.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Application of the ARIMA model on the COVID-2019 epidemic dataset</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-2019) has been recognized as a global threat, and several studies are being conducted using various mathematical models to predict the probable evolution of this epidemic. These mathematical models based on various factors and analyses are subject to potential bias. Here, we propose a simple econometric model that could be useful to predict the spread of COVID-2019. We performed Auto Regressive Integrated Moving Average (ARIMA) model prediction on the Johns Hopkins epidemiological data to predict the epidemiological trend of the prevalence and incidence of COVID-2019. For further comparison or for future perspective, case definition and data collection have to be maintained in real time.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Statistics-Based Predictions of Coronavirus Epidemic Spreading in Mainland China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Information about the open-access article 'Statistics-Based Predictions of Coronavirus Epidemic Spreading in Mainland China' in DOAJ. DOAJ is an online directory that indexes and provides access to quality open access, peer-reviewed journals.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although human coronavirus OC43-OC43 (HCoV-OC43) is the coronavirus most commonly associated with human infections, little is known about its molecular epidemiology and evolution. We conducted a molecular epidemiology study to investigate different genotypes and potential recombination in HCoV-OC43. Twenty-nine HCoV-OC43 strains from nasopharyngeal aspirates, collected from 2004 to 2011, were subjected to RNA-dependent RNA polymerase (RdRp), spike, and nucleocapsid gene analysis. Phylogenetic analysis showed at least three distinct clusters of HCoV-OC43, although 10 unusual strains displayed incongruent phylogenetic positions between RdRp and spike genes. This suggested the presence of four HCoV-OC43 genotypes (A to D), with genotype D most likely arising from recombination. The complete genome sequencing of two genotype C and D strains and bootscan analysis showed recombination events between genotypes B and C in the generation of genotype D. Of the 29 strains, none belonged to the more ancient genotype A, 5 from 2004 belonged to genotype B, 15 from 2004 to 2006 belonged to genotype C, and 1 from 2004 and all 8 from 2008 to 2011 belonged to the recombinant genotype D. Molecular clock analysis using spike and nucleocapsid genes dated the most recent common ancestor of all genotypes to the 1950s, genotype B and C to the 1980s, genotype B to the 1990s, and genotype C to the late 1990s to early 2000s, while the recombinant genotype D strains were detected as early as 2004. This represents the first study to describe natural recombination in HCoV-OC43 and the evolution of different genotypes over time, leading to the emergence of novel genotype D, which is associated with pneumonia in our elderly population.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular characterization of human coronaviruses and their circulation dynamics in Kenya, 2009–2012</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human Coronaviruses (HCoV) are a common cause of respiratory illnesses and are responsible for considerable morbidity and hospitalization across all age groups especially in individuals with compromised immunity. There are six known species of HCoV: HCoV-229E, HCoV-NL63, HCoV-HKU1, HCoV-OC43, MERS-CoV and SARS-HCoV. Although studies have shown evidence of global distribution of HCoVs, there is limited information on their presence and distribution in Kenya. METHODS: HCoV strains that circulated in Kenya were retrospectively diagnosed and molecularly characterized. A total of 417 nasopharyngeal specimens obtained between January 2009 and December 2012 from around Kenya were analyzed by a real time RT-PCR using HCoV-specific primers. HCoV-positive specimens were subsequently inoculated onto monolayers of LL-CMK2 cells. The isolated viruses were characterized by RT-PCR amplification and sequencing of the partial polymerase (pol) gene. RESULTS: The prevalence of HCoV infection was as follows: out of the 417 specimens, 35 (8.4 %) were positive for HCoV, comprising 10 (2.4 %) HCoV-NL63, 12 (2.9 %) HCoV-OC43, 9 (2.1 %) HCoV-HKU1, and 4 (1 %) HCoV-229E. The Kenyan HCoV strains displayed high sequence homology to the prototypes and contemporaneous strains. Evolution analysis showed that the Kenyan HCoV-OC43 and HCoV-NL63 isolates were under purifying selection. Phylogenetic evolutionary analyses confirmed the identities of three HCoV-HKU1, five HCoV-NL63, eight HCoV-OC43 and three HCoV-229E. CONCLUSIONS: There were yearly variations in the prevalence and circulation patterns of individual HCoVs in Kenya. This paper reports on the first molecular characterization of human Coronaviruses in Kenya, which play an important role in causing acute respiratory infections among children.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: death toll passes 2,000</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Updates on the respiratory illness that has infected tens of thousands of people. Scientists are concerned about a new virus that has infected tens of thousands of people and killed more than 2,000. The virus, which emerged in the Chinese city of Wuhan in December, is a coronavirus and belongs to the same family as the pathogen that causes severe acute respiratory syndrome, or SARS. It causes a respiratory illness called COVID-19, which can spread from person to person.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Prolonged Shedding of Human Coronavirus in Hematopoietic Cell Transplant Recipients: Risk Factors and Viral Genome Evolution</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Recent data suggest that human coronavirus (HCoV) pneumonia is associated with significant mortality in hematopoietic cell transplant (HCT) recipients. Investigation of risk factors for prolonged shedding and intrahost genome evolution may provide critical information for development of novel therapeutics. METHODS: We retrospectively reviewed HCT recipients with HCoV detected in nasal samples by polymerase chain reaction (PCR). HCoV strains were identified using strain-specific PCR. Shedding duration was defined as time between first positive and first negative sample. Logistic regression analyses were performed to evaluate factors for prolonged shedding (≥21 days). Metagenomic next-generation sequencing (mNGS) was conducted when ≥4 samples with cycle threshold values of <28 were available. RESULTS: Seventeen of 44 patients had prolonged shedding. Among 31 available samples, 35% were OC43, 32% were NL63, 19% were HKU1, and 13% were 229E; median shedding duration was similar between strains (P = .79). Bivariable logistic regression analyses suggested that high viral load, receipt of high-dose steroids, and myeloablative conditioning were associated with prolonged shedding. mNGS among 5 subjects showed single-nucleotide polymorphisms from OC43 and NL63 starting 1 month following onset of shedding. CONCLUSIONS: High viral load, high-dose steroids, and myeloablative conditioning were associated with prolonged shedding of HCoV in HCT recipients. Genome changes were consistent with the expected molecular clock of HCoV.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Unveiling the Origin and Transmission of 2019-nCoV</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus has caused thousands of human infections in China since December 2019, raising a global public health concern. Recent studies (Huang et al., Chan et al., and Zhou et al.) have provided timely insights into its origin and ability to spread among humans, informing infection prevention and control practices.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk estimation and prediction by modeling the transmission of the novel coronavirus (COVID-19) in mainland China excluding Hubei province</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In December 2019, an outbreak of novel coronavirus disease (COVID-19) emerged in Wuhan, China and has swiftly spread to other parts of China and a number of oversea countries. Our aim is to evaluate the effectiveness of the evolution of interventions and self-protection measures, estimate the risk of partial lifting control measures and predict the epidemic trend of the virus in mainland China excluding Hubei province based on the published data and a novel mathematical model. Methods: A novel COVID-19 transmission dynamic model incorporating the intervention measures implemented in China is proposed. We parameterize the model by using the Markov Chain Monte Carlo (MCMC) method and estimate the control reproduction number Rc, as well as the effective daily reproduction ratio Re(t), of the disease transmission in mainland China excluding Hubei province. Results: The estimation outcomes indicate that the control reproduction number is 3.36 (95% CI 3.20-3.64) and Re(t) has dropped below 1 since January 31st, 2020, which implies that the containment strategies implemented by the Chinese government in mainland China excluding Hubei province are indeed effective and magnificently suppressed COVID-19 transmission. Moreover, our results show that relieving personal protection too early may lead to the spread of disease for a longer time and more people would be infected, and may even cause epidemic or outbreak again. By calculating the effective reproduction ratio, we proved that the contact rate should be kept at least less than 30% of the normal level by April, 2020. Conclusions: To ensure the epidemic ending rapidly, it is necessary to maintain the current integrated restrict interventions and self-protection measures, including travel restriction, quarantine of entry, contact tracing followed by quarantine and isolation and reduction of contact, like wearing masks, etc. People should be fully aware of the real-time epidemic situation and keep sufficient personal protection until April. If all the above conditions are met, the outbreak is expected to be ended by April in mainland China apart from Hubei province.</jats:p></p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and Genetic Diversity Analysis of Human Coronavirus OC43 among Adult Patients with Acute Respiratory Infections in Beijing, 2012</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jul 2</span></i><p style="text-align: justify; text-justify: inter-word;">To determine the prevalence, epidemiology and genetic diversity of human coronavirus OC43 (HCoV-OC43) among adult patients with acute respiratory infections (ARI) in Beijing,five hundred and fifty-nine nasopharyngeal swab samples were collected from adult patients with ARI in Beijing. The prevalence of HCoV-OC43 infection among these patients was assessed using two different OneStep reverse transcription polymerase chain reaction (RT-PCR) assays. The epidemiological profiles of the patients with HCoV-OC43 infection were described. Partial S and N genes of HCoV-OC43 circulating strains were sequenced followed by phylogenetic analysis and amino acid alignment. Our results showed that the prevalence of HCoV-OC43 infection was 12.52% (95% CI: 9.78–15.26%), and the epidemic peak occurred in autumn. Fifty partial S and 40 partial N fragments were obtained from these patients. Phylogenetic analysis based on neighbour-joining method showed that at least three distinct clusters (A, B, C/D) of HCoV-OC43 strains were circulating among adult patients with ARI in Beijing. In addition, some novel unique clusters (UNT) of HCoV-OC43 were found in the S- and N-based phylogenetic trees. Furthermore, consensus amino acids substitutes for each cluster were also found after alignment of partial S or N sequence coding region in this study. In conclusion, we herein describe the prevalence of HCoV-OC43 among adult patients and provide substantial evidence for the genetic diversity of HCoV-OC43 circulating in Beijing.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: WHO officially names disease COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The World Health Organization has officially named the disease caused by the coronavirus COVID-19. This will replace various monikers and hashtags given to the emerging illness over the past few weeks. Most recently, on 8 February, China’s National Health Commission decided to temporarily call the disease novel coronavirus pneumonia, or NCP. But because viruses continue to spread from animals to people, this coronavirus won’t be novel for long.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>2019 novel coronavirus is undergoing active recombination</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019 novel coronavirus (2019-nCoV) outbreak in Wuhan since December 2019 [1] has quickly spread to twenty-five countries [2], caused more than 44,000 cases and 1,000 deaths so far [3]. The fast sharing of 2019-nCoV genomes in GISAID (https://www.gisaid.org) provides a valuable dataset for 2019-nCoV haplotype network analysis, which is crucial for transmission and evolutionary track surveillance and secondary outbreak prevention. We called single-nucleotide variations (SNVs) for all the 84 2019-nCov genomes in GISAID using MN908947 as the reference. Genomes with same SNVs are grouped into a haplotype (shown as a pie chart node in Fig. 1, see Tab. S1 for the accessions), then the network is constructed using the median join [4] method in popART [5]. We found the 2019-nCoV haplotype network has obvious characteristics of single origin from haplotype hap_011: first, the network is star-like, centralized on the haplotype hap_011; second, hap_011 has the largest sample size and majority of the samples are from Hubei province—where outbreak originated (Tab. S1); third, most of satellite haplotypes are also from Hubei (Fig. 1); fourth, the average collection dates of hap_011 (has 0 mutation relative to MN908947) is earlier than all other mutation groups (Fig. S1). The single origin of 2019-nCov indicates a persistent animal to human transmission is unlikely, otherwise, multiple nodes with above characteristics should be observed.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Why does the coronavirus spread so easily between people?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the number of coronavirus infections approaches 100,000 people worldwide, researchers are racing to understand what makes it spread so easily. A handful of genetic and structural analyses have identified a key feature of the virus — a protein on its surface — that might explain why it infects human cells so readily. Other groups are investigating the doorway through which the new coronavirus enters human tissues — a receptor on cell membranes. Both the cell receptor and the virus protein offer potential targets for drugs to block the pathogen, but researchers say it is too early to be sure. “Understanding transmission of the virus is key to its containment and future prevention,” says David Veesler, a structural virologist at the University of Washington in Seattle, who posted his team’s findings about the virus protein on the biomedical preprint server bioRxiv on 20 February1. The new virus spreads much more readily than the one that caused severe acute respiratory syndrome, or SARS (also a coronavirus), and has infected more than ten times the number of people who contracted SARS.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-length genome sequences of porcine epidemic diarrhoea virus strain CV777; Use of NGS to analyse genomic and sub-genomic RNAs</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Mar 1</span></i><p style="text-align: justify; text-justify: inter-word;">Porcine epidemic diarrhoea virus, strain CV777, was initially characterized in 1978 as the causative agent of a disease first identified in the UK in 1971. This coronavirus has been widely distributed among laboratories and has been passaged both within pigs and in cell culture. To determine the variability between different stocks of the PEDV strain CV777, sequencing of the full-length genome (ca. 28kb) has been performed in 6 different laboratories, using different protocols. Not surprisingly, each of the different full genome sequences were distinct from each other and from the reference sequence (Accession number AF353511) but they are >99% identical. Unique and shared differences between sequences were identified. The coding region for the surface-exposed spike protein showed the highest proportion of variability including both point mutations and small deletions. The predicted expression of the ORF3 gene product was more dramatically affected in three different variants of this virus through either loss of the initiation codon or gain of a premature termination codon. The genome of one isolate had a substantially rearranged 5´-terminal sequence. This rearrangement was validated through the analysis of sub-genomic mRNAs from infected cells. It is clearly important to know the features of the specific sample of CV777 being used for experimental studies.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe acute respiratory syndrome diagnostics using a coronavirus protein microarray</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">To monitor severe acute respiratory syndrome (SARS) infection, a coronavirus protein microarray that harbors proteins from SARS coronavirus (SARS-CoV) and five additional coronaviruses was constructed. These microarrays were used to screen ≈400 Canadian sera from the SARS outbreak, including samples from confirmed SARS-CoV cases, respiratory illness patients, and healthcare professionals. A computer algorithm that uses multiple classifiers to predict samples from SARS patients was developed and used to predict 206 sera from Chinese fever patients. The test assigned patients into two distinct groups: those with antibodies to SARS-CoV and those without. The microarray also identified patients with sera reactive against other coronavirus proteins. Our results correlated well with an indirect immunofluorescence test and demonstrated that viral infection can be monitored for many months after infection. We show that protein microarrays can serve as a rapid, sensitive, and simple tool for large-scale identification of viral-specific antibodies in sera.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological factors and worldwide pattern of Middle East respiratory syndrome coronavirus from 2013 to 2016</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 6</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging threat to global health security with high intensity and lethality. This study was conducted to investigate epidemiological factors and patterns related to this disease. METHODS: Full details of MERS-CoV cases available on the disease outbreak news section of the World Health Organization official website from January 2013 to November 2016 were retrieved; demographic and clinical information, global distribution status, potential contacts, and probable risk factors for the mortality of laboratory-confirmed MERS-CoV cases were extracted and analyzed by following standard statistical methods. RESULTS: Details of 1,094 laboratory-confirmed cases were recorded, including 421 related deaths. Significant differences were observed in the presentation of the disease from year to year, and all studied parameters differed during the years under study (all P-values <0.05). Evaluation of the effects of various potential risk factors of the final outcome (dead/survived) revealed that two factors, namely, the morbid case being native and travel history, are significant based on a unifactorial analysis (P <0.05). From 2013 to 2016, these factors remained important. However, factors that were significant in predicting mortality varied in different years. CONCLUSION: These findings point to interesting potential dimensions in the dynamic of this disease. Furthermore, effective national and international preparedness plans and actions are essential to prevent, control, and predict such viral outbreaks; improve patient management; and ensure global health security.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Caution: The clinical characteristics of COVID-19 patients at admission are changing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background  With the emergence of 4rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 (COVID-19) has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Methods Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. Results  Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24).  Conclusions The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.</jats:p></p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An in-depth annotation of the newly discovered coronavirus (2019-nCoV) genome has revealed differences between 2019-nCoV and severe acute respiratory syndrome (SARS) or SARS-like coronaviruses. A systematic comparison identified 380 amino acid substitutions between these coronaviruses, which may have caused functional and pathogenic divergence of 2019-nCoV.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution of mouse hepatitis virus: detection and characterization of spike deletion variants during persistent infection.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">High-frequency RNA recombination has been proposed as an important mechanism for generating viral deletion variants of murine coronavirus. Indeed, a number of variants with deletions in the spike glycoprotein have been isolated from persistently infected animals. However, the significance of generating and potentially accumulating deletion variants in the persisting viral RNA population is unclear. To study this issue, we evaluated the evolution of spike variants by examining the population of spike RNA sequences detected in the brains and spinal cords of mice inoculated with coronavirus and sacrificed at 4, 42, or 100 days postinoculation. We focused on the S1 hypervariable region since previous investigators had shown that this region is subject to recombination and deletion. RNA isolated from the brains or spinal cords of infected mice was rescued by reverse transcription-PCR, and the amplified products were cloned and used in differential colony hybridizations to identify individual isolates with deletions. We found that 11 of 20 persistently infected mice harbored spike deletion variants (SDVs), indicating that deletions are common but not required for persistent infection. To determine if a specific type of SDV accumulated during persistence, we sequenced 106 of the deletion isolates. We identified 23 distinct patterns of SDVs, including 5 double-deletion variants. Furthermore, we found that each mouse harbored distinct variants in its central nervous system (CNS), suggesting that SDVs are generated during viral replication in the CNS. Interestingly, mice with the most severe and persisting neurological disease harbored the most prevalent and diverse quasispecies of SDVs. Overall, these findings illustrate the complexity of the population of persisting viral RNAs which may contribute to chronic disease.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Two deletion variants of Middle East respiratory syndrome coronavirus found in a patient with characteristic symptoms</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Apr 18</span></i><p style="text-align: justify; text-justify: inter-word;">Significant sequence variation of Middle East respiratory syndrome coronavirus (MERS CoV) has never been detected since it was first reported in 2012. A MERS patient came from Korea to China in late May 2015. The patient was 44 years old and had symptoms including high fever, dry cough with a little phlegm, and shortness of breath, which are roughly consistent with those associated with MERS, and had had close contact with individuals with confirmed cases of MERS.After one month of therapy with antiviral, anti-infection, and immune-enhancing agents, the patient recovered in the hospital and was discharged. A nasopharyngeal swab sample was collected for direct sequencing, which revealed two deletion variants of MERS CoV. Deletions of 414 and 419 nt occurred between ORF5 and the E protein, resulting in a partial protein fusion or truncation of ORF5 and the E protein. Functional analysis by bioinformatics and comparison to previous studies implied that the two variants might be defective in their ability to package MERS CoV. However, the mechanism of how these deletions occurred and what effects they have need to be further investigated. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1007/s00705-017-3361-x) contains supplementary material, which is available to authorized users.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolutionary Dynamics of MERS-CoV: Potential Recombination, Positive Selection and Transmission</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 May 4</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) belongs to beta group of coronavirus and was first discovered in 2012. MERS-CoV can infect multiple host species and cause severe diseases in human. We conducted a series of phylogenetic and bioinformatic analyses to study the evolution dynamics of MERS-CoV among different host species with genomic data. Our analyses show: 1) 28 potential recombinant sequences were detected and they can be classified into seven potential recombinant types; 2) The spike (S) protein of MERS-CoV was under strong positive selection when MERS-CoV transmitted from their natural host to human; 3) Six out of nine positive selection sites detected in spike (S) protein are located in its receptor-binding domain which is in direct contact with host cells; 4) MERS-CoV frequently transmitted back and forth between human and camel after it had acquired the human-camel infection capability. Together, these results suggest that potential recombination events might have happened frequently during MERS-CoV’s evolutionary history and the positive selection sites in MERS-CoV’s S protein might enable it to infect human.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-04</span></i><p style="text-align: justify; text-justify: inter-word;">Novel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome-wide data inferring the evolution and population demography of the novel pneumonia coronavirus (SARS-CoV-2)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, coronavirus disease 2019 (COVID-19) emerged in Wuhan, Central China and rapidly spread throughout China. Up to March 3, 2020, SARS-CoV-2 has infected more than 89,000 people in China and other 66 countries across six continents. In this study, we used 10 new sequenced genomes of SARS-CoV-2 and combined 136 genomes from GISAID database to investigate the genetic variation and population demography through different analysis approaches (e.g. Network, EBSP, Mismatch, and neutrality tests) in the previous three months. The results showed that eighty haplotypes had 183 substitution sites, including 27 parsimony-informative and 156 singletons. Sliding window analyses of genetic diversity suggested a certain range in mutations and variation in genomic abundance of SARS-CoV-2, which could be explaining the existing widespread and high adaptation of the deadly virus. Phylogenetic analysis showed that pangolin may not be an intermediate host. The network indicated that, in the original haplotype (H14), one patient sample lived near the Huanan seafood market (approximately 2 km), which indicating high possibility of the patient having a history of unconscious contact with this market. However, based on this clue, we cannot accurately conclude that whether this market was the origin center of SARS-CoV-2. Additionally, 16 genomes, collected from this market, assigned to 10 haplotypes, indicated a circulating infection within the market in a short term, which would have led to the outbreak of SARS-CoV-2 in Wuhan and other areas. The EBSP results showed that the first estimated expansion date began from 7 December 2019, which indicated that the transmission of SARS-CoV-2 could have begun from person to person in mid to late November.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamic basic reproduction number based evaluation for current prevention and control of COVID-19 outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: To evaluate the current status of the prevention and control of coronavirus disease (COVID-19) outbreak in China, establish a predictive model to evaluate the effects of the current prevention and control strategies, and provide scientific information for decision- making departments. Methods: Based on the epidemic data of COVID-19 openly accessed from national health authorities, we estimated the dynamic basic reproduction number R(0)(t) to evaluate the effects of the current COVID-19 prevention and control strategies in all the provinces (municipalities and autonomous regions) as well as in Wuhan and the changes in infectivity of COVID-19 over time. Results: For the stability of the results, 24 provinces (municipality) with more than 100 confirmed COVID-19 cases were included in the analysis. At the beginning of the outbreak, the R(0)(t) showed unstable trend with big variances. As the strengthening of the prevention and control strategies, R(0)(t) began to show a downward trend in late January, and became stable in February. By the time of data analysis, 18 provinces (municipality) (75%) had the R(0)(t)s less than 1. The results could be used for the decision making to free population floating conditionally. Conclusions: Dynamic R(0)(t) is useful in the evaluation of the change in infectivity of COVID-19, the prevention and control strategies for the COVID-19 outbreak have shown preliminary effects, if continues, it is expected to control the COVID-19 outbreak in China in near future.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Measures for diagnosing and treating infections by a novel coronavirus responsible for a pneumonia outbreak originating in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 10 January 2020, a new coronavirus causing a pneumonia outbreak in Wuhan City in central China was denoted as 2019-nCoV by the World Health Organization (WHO). As of 24 January 2020, there were 887 confirmed cases of 2019-nCoV infection, including 26 deaths, reported in China and other countries. Therefore, combating this new virus and stopping the epidemic is a matter of urgency. Here, we focus on advances in research and development of fast diagnosis methods, as well as potential prophylactics and therapeutics to prevent or treat 2019-nCoV infection.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19.  Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Disease 2019 (COVID-19): A Perspective from China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, an outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection occurred in Wuhan, Hubei Province, China and spread across China and beyond. On February 12, 2020, WHO officially named the disease caused by the novel coronavirus as Coronavirus Disease 2019 (COVID-19). Since most COVID-19 infected patients were diagnosed with pneumonia and characteristic CT imaging patterns, radiological examinations have become vital in early diagnosis and assessment of disease course. To date, CT findings have been recommended as major evidence for clinical diagnosis of COVID-19 in Hubei, China. This review focuses on the etiology, epidemiology, and clinical symptoms of COVID-19, while highlighting the role of chest CT in prevention and disease control. A full translation of this article in Chinese is available.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular epidemiology and characterization of human coronavirus in Thailand, 2012–2013</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug 26</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronavirus causes respiratory infections in humans. To determine the prevalence of human coronavirus (HCoV) infection among patients with influenza-like illness, 5833 clinical samples from nasopharyngeal swabs and aspirates collected between January 2012 and December 2013 were examined. RESULTS: HCoV was found in 46 (0.79 %) samples. There were 19 (0.32 %) HCoV-HKU1, 19 (0.32 %) HCoV-NL63, 5 (0.09 %) HCoV-229E, and 3 (0.05 %) HCoV-OC43. None of the sample tested positive for MERS-CoV. The majority (54 %) of the HCoV-positive patients were between the ages of 0 and 5 years. HCoV was detected throughout the 2-year period and generally peaked from May to October, which coincided with the rainy season. Phylogenetic trees based on the alignment of the spike (S) gene sequences suggest an emergence of a new clade for HCoV-229E. CONCLUSIONS: The data in this study provide an insight into the prevalence of the recent circulating HCoVs in the region.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>2019-nCoV: new challenges from coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of pneumonia caused by the novel coronavirus 2019-nCoV in Wuhan, Hubei province of China, at the end of 2019 shaped tremendous challenges to China's public health and clinical treatment. The virus belongs to the &beta; genus Coronavirus in the family Corornaviridae, and is closely related to SARS-CoV and MERS-CoV, causing severe symptoms of pneumonia. The virus is transmitted through droplets, close contact, and other means, and patients in the incubation period could potentially transmit the virus to other persons. According to current observations, 2019-nCoV is weaker than SARS in pathogenesis, but has stronger transmission competence; it's mechanism of cross-species spread might be related with angiotensin-converting enzyme &#8545; (ACE2), which is consistent with the receptor SARS-CoV. After the outbreak of this disease, Chinese scientists invested a lot of energy to carry out research by developing rapid diagnostic reagents, identifying the characters of the pathogen, screening out clinical drugs that may inhibit the virus, and are rapidly developing vaccines. The emergence of 2019-nCoV reminds us once again of the importance of establishing a systematic coronavirus surveillance network. It also poses new challenges to prevention and control of the emerging epidemic and rapidly responses on scientific research.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Age-dependent risks of Incidence and Mortality of COVID-19 in Hubei Province and Other Parts of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>New coronavirus SARS-CoV-2 poses a big challenge for global public health in early 2020. Coronavirus Disease 2019 (COVID-19) caused by the virus rapidly spreads all over the world and takes thousands of lives in just two months. It is critical to refine the incidence and mortality risks of COVID-19 for the effective management of the general public and patients in the outbreak. In this report, we investigate the incidence and mortality risks of the infection by analyzing the age composition of 5319 infected patients, 76 fatal cases, and 1,144,648 individuals of the general public in China. Our result shows a relatively low incidence risk for young people but a very high mortality risk for seniors. Notably, mortality risk could be as high as 0.48 for people older than 80 years. Furthermore, our study suggests that a good medical service can effectively reduce the mortality rate of the viral infection to 1% or less.</jats:p></p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of a subgenotype of human coronavirus NL63 associated with severe lower respiratory tract infection in China, 2018</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus NL63 (HCoV-NL63) is primarily associated with common cold in children, elderly and immunocompromised individuals. Outbreaks caused by HCoV-NL63 are rare. Here we report a cluster of HCoV-NL63 cases with severe lower respiratory tract infection that arose in Guangzhou, China, in 2018. Twenty-three hospitalized children were confirmed to be HCoV-NL63 positive, and most of whom were hospitalized with severe pneumonia or acute bronchitis. Whole genomes of HCoV-NL63 were obtained using next-generation sequencing. Phylogenetic and single amino acid polymorphism analyses showed that this outbreak was associated with two subgenotypes (C3 and B) of HCoV-NL63. Half of patients were identified to be related to a new subgenotype C3. One unique amino acid mutation at I507 L in spike protein receptor binding domain (RBD) was detected, which segregated this subgenotype C3 from other known subgenotypes. Pseudotyped virus bearing the I507 L mutation in RBD showed enhanced entry into host cells as compared to the prototype virus. This study proved that HCoV-NL63 was undergoing continuous mutation and has the potential to cause severe lower respiratory disease in humans.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Mutations, Recombination and Insertion in the Evolution of 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.  MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.  ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.  ConclusionsOur studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.  One Sentence SummaryNovel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-01</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited.  METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG).  CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be greatly relieved.</jats:p></p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: children are as susceptible as adults, study suggests</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Updates on the respiratory illness that has infected tens of thousands of people and killed thousands. Children are just as likely to get infected with the new coronavirus as adults, finds one of the most detailed studies yet published on the spread of the virus, known as SARS-CoV-2. The analysis — based on data from Shenzhen, China — provides a partial answer to one of the most pressing questions surrounding the outbreak: the role of children.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</jats:p></p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>The Evolutionary Processes of Canine Coronaviruses</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Jul 7</span></i><p style="text-align: justify; text-justify: inter-word;">Since the first identification of the virus in 1971, the disease caused by canine coronavirus (CCoV) has not been adequately investigated, and the role that the virus plays in canine enteric illness has not been well established. Only after the emergence in 2002 of SARS in human has new attention been focused on coronaviruses. As a consequence of the relatively high mutation frequency of RNA-positive stranded viruses, CCoV has evolved and, with the biomolecular techniques developed over the last two decades, new virus strains, serotypes, and subtypes have been identified in infected dogs. Considering the widespread nature of CCoV infections among dog populations, several studies have been carried out, focusing upon the epidemiological relevance of these viruses and underlining the need for further investigation into the biology of CCoVs and into the pathogenetic role of the infections. This paper reports the evolutionary processes of CCoVs with a note onto recent diagnostic methods.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS to novel coronavirus – old lessons and new lessons</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The response to the novel coronavirus outbreak in China suggests that many of the lessons from the 2003 SARS epidemic have been implemented and the response improved as a consequence. Nevertheless some questions remain and not all lessons have been successful. The national and international response demonstrates the complex link between public health, science and politics when an outbreak threatens to impact on global economies and reputations. The unprecedented measures implemented in China are a bold attempt to control the outbreak – we need to understand their effectiveness to balance costs and benefits for similar events in the future.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>An epidemiological forecast model and software assessing interventions on COVID-19 epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We develop a health informatics toolbox that enables public health workers to timely analyze and evaluate the time-course dynamics of the novel coronavirus (COVID-19) infection using the public available data from the China CDC. This toolbox is built upon a hierarchical epidemiological model in which two observed time series of daily proportions of infected and removed cases are emitted from the underlying infection dynamics governed by a Markov SIR infectious disease process. We extend the SIR model to incorporate various types of time-varying quarantine protocols, including government-level macro isolation policies and community-level micro inspection measures. We develop a calibration procedure for under-reported infected cases. This toolbox provides forecast, in both online and offline forms, of turning points of interest, including the time when daily infected proportion becomes smaller than the previous ones and the time when daily infected proportions becomes smaller than that of daily removed proportion, as well as the ending time of the epidemic. An R software is made available for the public, and examples on the use of this software are illustrated. Some possible extensions of our novel epidemiological models are discussed.</jats:p></p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic analysis of 16 Colorado human NL63 coronaviruses identifies a new genotype, high sequence diversity in the N-terminal domain of the spike gene and evidence of recombination</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">This study compared the complete genome sequences of 16 NL63 strain human coronaviruses (hCoVs) from respiratory specimens of paediatric patients with respiratory disease in Colorado, USA, and characterized the epidemiology and clinical characteristics associated with circulating NL63 viruses over a 3-year period. From 1 January 2009 to 31 December 2011, 92 of 9380 respiratory specimens were found to be positive for NL63 RNA by PCR, an overall prevalence of 1 %. NL63 viruses were circulating during all 3 years, but there was considerable yearly variation in prevalence and the month of peak incidence. Phylogenetic analysis comparing the genome sequences of the 16 Colorado NL63 viruses with those of the prototypical hCoV-NL63 and three other NL63 viruses from the Netherlands demonstrated that there were three genotypes (A, B and C) circulating in Colorado from 2005 to 2010, and evidence of recombination between virus strains was found. Genotypes B and C co-circulated in Colorado in 2005, 2009 and 2010, but genotype A circulated only in 2005 when it was the predominant NL63 strain. Genotype C represents a new lineage that has not been described previously. The greatest variability in the NL63 virus genomes was found in the N-terminal domain (NTD) of the spike gene (nt 1–600, aa 1–200). Ten different amino acid sequences were found in the NTD of the spike protein among these NL63 strains and the 75 partial published sequences of NTDs from strains found at different times throughout the world.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Cross-host evolution of severe acute respiratory syndrome coronavirus in palm civet and human</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The genomic sequences of severe acute respiratory syndrome coronaviruses from human and palm civet of the 2003/2004 outbreak in the city of Guangzhou, China, were nearly identical. Phylogenetic analysis suggested an independent viral invasion from animal to human in this new episode. Combining all existing data but excluding singletons, we identified 202 single-nucleotide variations. Among them, 17 are polymorphic in palm civets only. The ratio of nonsynonymous/synonymous nucleotide substitution in palm civets collected 1 yr apart from different geographic locations is very high, suggesting a rapid evolving process of viral proteins in civet as well, much like their adaptation in the human host in the early 2002–2003 epidemic. Major genetic variations in some critical genes, particularly the Spike gene, seemed essential for the transition from animal-to-human transmission to human-to-human transmission, which eventually caused the first severe acute respiratory syndrome outbreak of 2002/2003.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>The clinical and virological features of the first imported case causing MERS-CoV outbreak in South Korea, 2015</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jul 14</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In 2015, the largest outbreak of Middle East respiratory syndrome coronavirus (MERS-CoV) infection outside the Middle East occurred in South Korea. We summarized the epidemiological, clinical, and laboratory findings of the first Korean case of MERS-CoV and analyzed whole-genome sequences of MERS-CoV derived from the patient. CASE PRESENTATION: A 68-year-old man developed fever and myalgia 7 days after returning to Korea, following a 10-day trip to the Middle East. Before diagnosis, he visited 4 hospitals, potentially resulting in secondary transmission to 28 patients. On admission to the National Medical Center (day 9, post-onset of clinical illness), he presented with drowsiness, hypoxia, and multiple patchy infiltrations on the chest radiograph. He was intubated (day 12) because of progressive acute respiratory distress syndrome (ARDS) and INF-α2a and ribavirin treatment was commenced. The treatment course was prolonged by superimposed ventilator associated pneumonia. MERS-CoV PCR results converted to negative from day 47 and the patient was discharged (day 137), following rehabilitation therapy. The complete genome sequence obtained from a sputum sample (taken on day 11) showed the highest sequence similarity (99.59%) with the virus from an outbreak in Riyadh, Saudi Arabia, in February 2015. CONCLUSIONS: The first case of MERS-CoV infection had high transmissibility and was associated with a severe clinical course. The patient made a successful recovery after early treatment with antiviral agents and adequate supportive care. This first case in South Korea became a super-spreader because of improper infection control measures, rather than variations of the virus.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating the accuracy of different respiratory specimens in the laboratory diagnosis and monitoring the viral shedding of 2019-nCoV infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background:  The outbreak of novel coronavirus pneumonia (NCP) caused by 2019-nCoV spread rapidly, and elucidation the diagnostic accuracy of different respiratory specimens is crucial for the control and treatment of this diseases.   Methods:  Respiratory samples including nasal swabs, throat swabs, sputum and bronchoalveolar lavage fluid (BALF) were collected from Guangdong CDC confirmed NCP patients, and viral RNAs were detected using a CFDA approved detection kit. Results were analyzed in combination with sample collection date and clinical information.  Finding:  Except for BALF, the sputum possessed the highest positive rate (74.4%~88.9%), followed by nasal swabs (53.6%~73.3%) for both severe and mild cases during the first 14 days after illness onset (d.a.o). For samples collected ≥ 15 d.a.o, sputum and nasal swabs still possessed a high positive rate ranging from 42.9%~61.1%. The positive rate of throat swabs collected ≥ 8 d.a.o was low, especially in samples from mild cases. Viral RNAs could be detected in all the lower respiratory tract of severe cases, but not the mild cases. CT scan of cases 02, 07 and 13 showed typical viral pneumonia with ground glass opacity, while no viral RNAs were detected in first three or all the upper respiratory samples.     Interpretation:  Sputum is most accurate for laboratory diagnosis of NCP, followed by nasal swabs. Detection of viral RNAs in BLAF is necessary for diagnosis and monitoring of viruses in severe cases. CT scan could serve as an important make up for the diagnosis of NCP.  Funding  National Science and Technology Major Project, Sanming Project of Medicine and China Postdoctoral Science Foundation.</jats:p></p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating the secondary transmission pattern and epidemic prediction of the COVID-19 in metropolitan areas of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Understanding the transmission dynamics of COVID-19 is crucial for evaluating the spread pattern of it, especially in metropolitan areas of China which may cause secondary outbreaks outside Wuhan, the center of the new coronavirus disease outbreak. We used reported data from Jan 24, 2020, to Feb 23, 2020, fitted the model of infection, and on the number of cases reported to estimate likely number of infections in four high risk metropolitan areas, as well as facilitate understanding the COVID-19's spread pattern. A group of SERI model statistical parameters were estimated using Markov Chain Monte Carlo (MCMC) methods, and our modeling integrated the effect of the official quarantine regulation and travel restriction of China. As a result, we estimated that the basic reproductive number R0 ​is 3.11 in Beijing, 2.78 in Shanghai, 2.02 in Guangzhou, and 1.75 in Shenzhen. In addition, we inferred the prediction results and compared the results of different level of parameters, For example, In Beijing, the predicated peak number of cases is around 466 at the peak time Feb 29, 2020; however, when the city conducts different levels (strict, mild, or weak) of travel restrictions or regulation measures, the estimation results show that transmission dynamics will change and the peek number of cases shows the changing proportion is between 56%~159%. We concluded that public health interventions would reduce the risks of COVID-19 spreading and more rigorous control and prevention measures will effectively contain its further spread, but risk increases when businesses and social activities returning back before the ending date. Besides, the experiences gained and lessons learned from China are potential to provide evidences supporting for other metropolitan areas and big cities with emerging cases outside China.</jats:p></p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Characteristics of Imported Cases of COVID-19 in Jiangsu Province: A Multicenter Descriptive Study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: We aimed to report the clinical characteristics of imported coronavirus disease-19 (COVID-19) in Jiangsu Province. METHODS: We retrospectively investigated the clinical, imaging, and laboratory characteristics of confirmed cases of COVID-19 with WHO interim guidance in three Grade A hospitals of Jiangsu from Jan 22 to Feb 14, 2020. Real time RT-PCR was used to detect the new coronavirus in respiratory samples. RESULTS: Of the 80 patients infected with COVID-19, 41 patients were female, with a median age of 46.1 years. Except for 3 severe patients, the rest of the 77 patients exhibited mild or moderate symptoms. 9 patients were unconfirmed until a third-time nucleic acid test. 38 cases had a history of chronic diseases. The main clinical manifestations of the patients were fever and cough, which accounted for 63 cases (78.75%) and 51 cases (-63.75%) respectively. Only 3 patients (3.75%) showed liver dysfunction. Imaging examination showed that 55 patients (-68.75%) showed abnormal, 25 cases (31.25%) had no abnormal density shadow in the parenchyma of both lungs. Up to now, 21 cases were discharged from the hospital, and no patient died. The average length of stay for discharged patients was 8 days. CONCLUSIONS: Compared with the cases in Wuhan, the cases in Jiangsu exhibited mild or moderate symptoms and no obvious gender susceptivity. The proportion of patients having liver dysfunction and abnormal CT imaging was relatively lower than that of Wuhan. Notably, infected patients may be falsely excluded based on two consecutively negative respiratory pathogenic nucleic acid test results.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>The 2019 Novel Coronavirus Outbreak - A Global Threat</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019 Novel Corona virus infection (COVID 19) is an ongoing public health emergency of international significance. There are significant knowledge gaps in the epidemiology, transmission dynamics, investigation tools and management. In this article, we review the available evidence about this disease. Every decade has witnessed the evolution of a new coronavirus epidemic since the last three decades. The varying transmission patterns, namely, nosocomial transmission and spread through mildly symptomatic cases is an area of concern. There is a spectrum of clinical features from mild to severe life threatening disease with major complications like severe pneumonia, ARDS, acute cardiac injury and septic shock. Presence of bilateral ground glass opacity and consolidation on imaging in appropriate clinical background should raise a suspicion about COVID 19. Poor prognostic factors include Multilobular infiltration on chest imaging, Lymphopenia, Bacterial co-infection, Smoking history, Chronic medical conditions like Hypertension and age >60 years (MuLBSTA score). Diagnosis is confirmed with PCR based testing of appropriate respiratory samples. Management is primarily supportive, with newer antivirals (lopinavir ritonavir and Remdesivir) under investigation. Role of steroids is still inconclusive. Standard infection control and prevention techniques should be followed. Vigilant screening of suspected cases and their contacts is important. Isolation of symptomatic cases and home quarantine of asymptomatic contacts is recommended. To conclude, controlling this highly transmissible disease requires international co-ordination.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus covid-19 has killed more people than SARS and MERS combined, despite lower case fatality rate</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus that has so far spread from China to 26 countries around the world does not seem to be as “deadly as other coronaviruses including SARS and MERS,” the World Health Organization has said.At a briefing on 17 February WHO’s director general, Tedros Adhanom Ghebreyesus, said that more than 80% of patients with covid-19 have a “mild disease and will recover” and that it is fatal in 2% of reported cases. In comparison, the 2003 outbreak of severe acute respiratory syndrome (SARS) had a case fatality rate of around 10% (8098 cases and 774 deaths), while Middle East respiratory syndrome (MERS) killed 34% of people with the illness …</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS Transmission Pattern in Singapore Reassessed by Viral Sequence Variation Analysis</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Feb 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Epidemiological investigations of infectious disease are mainly dependent on indirect contact information and only occasionally assisted by characterization of pathogen sequence variation from clinical isolates. Direct sequence analysis of the pathogen, particularly at a population level, is generally thought to be too cumbersome, technically difficult, and expensive. We present here a novel application of mass spectrometry (MS)–based technology in characterizing viral sequence variations that overcomes these problems, and we apply it retrospectively to the severe acute respiratory syndrome (SARS) outbreak in Singapore. METHODS AND FINDINGS: The success rate of the MS-based analysis for detecting SARS coronavirus (SARS-CoV) sequence variations was determined to be 95% with 75 copies of viral RNA per reaction, which is sufficient to directly analyze both clinical and cultured samples. Analysis of 13 SARS-CoV isolates from the different stages of the Singapore outbreak identified nine sequence variations that could define the molecular relationship between them and pointed to a new, previously unidentified, primary route of introduction of SARS-CoV into the Singapore population. Our direct determination of viral sequence variation from a clinical sample also clarified an unresolved epidemiological link regarding the acquisition of SARS in a German patient. We were also able to detect heterogeneous viral sequences in primary lung tissues, suggesting a possible coevolution of quasispecies of virus within a single host. CONCLUSION: This study has further demonstrated the importance of improving clinical and epidemiological studies of pathogen transmission through the use of genetic analysis and has revealed the MS-based analysis to be a sensitive and accurate method for characterizing SARS-CoV genetic variations in clinical samples. We suggest that this approach should be used routinely during outbreaks of a wide variety of agents, in order to allow the most effective control.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Networks of information token recurrences derived from genomic sequences may reveal hidden patterns in epidemic outbreaks: A case study of the 2019-nCoV coronavirus.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and suggest touching points and temporal pathways that are of interest for deeper investigation by geneticists and epidemiologists.</jats:p></p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /></body>