
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: a mechanism for coordinating the rapid dissemination of COVID-19 genome tracking information</h2><br /></div><br /><br /><br /><div class="card"><h3>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Preprints bring ‘firehose’ of outbreak data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn't even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>With $115 million, more than 80 Boston researchers will collaborate to tackle COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A $115 million collaboration to tackle the rapidly spreading viral disease COVID-19, led by heavy hitters of Boston science and funded by a Chinese property development company, kicked off today as the group’s leaders pledged to take on the virus on many fronts. The project brings together researchers at many of the city’s top academic institutions, along with local biotechnology companies such as Moderna. Those leading it hope they can quickly funnel money into studies that will build off a new repository of samples from infected people and community surveillance, materials that can be rapidly shared among scientists. The project, they anticipate, should answer critical questions about how COVID-19 is spreading and how best to prevent and treat infections.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>‘A completely new culture of doing research.’ Coronavirus outbreak changes how scientists communicate | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 22 January, Dave O’Connor and Tom Friedrich invited several dozen colleagues around the United States to join a new workspace on the instant messaging platform Slack. The scientists, both at the Wisconsin National Primate Research Center, had seen news about a new disease emerging in China and realized researchers would need a primate model if they were going to answer some important questions about its biology. “We put out a call to a bunch of investigators and basically said: ‘Hey, let’s talk,’” O’Connor says. The idea is to coordinate research and make sure results are comparable, Friedrich adds. (They named the Slack workspace the Wu-han Clan, a play on the hip-hop group Wu-Tang Clan.) The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn’t even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodias first case of COVID-2019.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearmans rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>The United States badly bungled coronavirus testing—but things may soon improve | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? The World Health Organization (WHO) has shipped testing kits to 57 countries. China had five commercial tests on the market 1 month ago and can now do up to 1.6 million tests a week; South Korea has tested 65,000 people so far. The U. S. Centers for Disease Control and Prevention (CDC), in contrast, has done only 459 tests since the epidemic began. The rollout of a CDC-designed test kit to state and local labs has become a fiasco because it contained a faulty reagent. Labs around the country eager to test more suspected cases—and test them faster—have been unable to do so. No commercial or state labs have the approval to use their own tests. In what is already an infamous snafu, CDC initially refused a request to test a patient in Northern California who turned out to be the first probable COVID19 case without known links to an infected person.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Can an Infectious Disease Genomics Project Predict and Prevent the Next Pandemic?</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Oct 26</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases need a globally coordinated genomic-based movement linking sequencing efforts to development of response tools to mitigate the impact of existing and emerging threats.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in ‘lockdown’ directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation’s economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and advances how smart city networks should work towards enhancing standardization protocols for increased data sharing in the event of outbreaks or disasters, leading to better global understanding and management of the same.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Real-time dissemination of scientific information to fight a public health emergency of international concern</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rapidly sharing scientific information is an effective way to reduce public panic about COVID-19, and doing so is the key to providing real-time guidance to epidemiologists working to contain the outbreak, clinicians managing patients, and modelers helping to understand future developments and the possible effectiveness of various interventions. This issue has rapidly reviewed and published articles describing COVID-19, including the drug treatment options for SARS-CoV-2, its clinical characteristics, and therapies involving a combination of Chinese and Western medicine, the efficacy of chloroquine phosphate in the treatment of COVID-19 associated pneumonia according to clinical studies, and reflections on the system of reserve medical supplies for public health emergencies. As an academic journal, we will continue to quickly and transparently share data with frontline healthcare workers who need to know the epidemiological and clinical features of COVID-19.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease. AVAILABILITY: https://www.genomedetective.com/app/typingtool/cov. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Are high-performing health systems resilient against the COVID-19 epidemic?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As of March 5, 2020, there has been sustained local transmission of coronavirus disease 2019 (COVID-19) in Hong Kong, Singapore, and Japan.1 Containment strategies seem to have prevented smaller transmission chains from amplifying into widespread community transmission. The health systems in these locations have generally been able to adapt,2 , 3 but their resilience could be affected if the COVID-19 epidemic continues for many more months and increasing numbers of people require services. We outline some of the core dimensions of these resilient health systems4 and their responses to the COVID-19 epidemic. First, after variable periods of adaptation, the three locations took actions to manage the outbreak of a new pathogen. Surveillance systems were readjusted to identify potential cases while public health staff identified their contacts. National laboratory networks developed diagnostic tests once the COVID-19 genetic sequences were published5 and laboratory testing capacity was increased in all three locations, although expansion of the diagnostic capacity to university and large private laboratories in Japan is still ongoing. In Hong Kong, initially, only pneumonia patients without a microbiological diagnosis were tested, but surveillance has been broadened to include all inpatients with pneumonia and a purposively sampled proportion of outpatients and emergency attendees totalling about 1500 per day (Leung GM, unpublished). Japan's testing strategy has also evolved with diagnostic tests now offered to all suspected cases irrespective of their travel history; however, there are reports of cases that should have been tested but were not.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>The novel coronavirus (SARS-CoV-2) infections in China: prevention, control and challenges</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, an outbreak of novel coronavirus (SARS-CoV-2) that began from Wuhan, Hubei Province, has rapidly spread to 34 provincial-level divisions of China [1] and 25 countries around the world [2]. Up to February 15, 2020, 68,586 cases in China and 526 cases in other countries have been identified as having COVID-19 (eFigure 1). The estimated mortality rate is 3.1% in Wuhan, 2.8% in Hubei Province, 0.6% in other provinces of China, 2.4% in China and 0.4% in other countries, respectively. Faced with such a grim situation, the Chinese government has taken a series of unprecedented rigorous measures. First, all the 31 provincial-level divisions in China mainland have launched the highest level of responding mechanism for major public health emergency. Second, Wuhan and other 12 cities in Hubei Province have consecutively shut down all outbound transportation channels and suspended public transportation. Third, the State Council announced that both the Spring Festival holiday and winter vacation will be extended. Fourth, two emergency makeshift hospitals (Huoshenshan and Leishenshan hospitals) with a total capacity of 2600 beds were built and more than ten cabin hospitals were renovated with more than 10,000 beds in Wuhan. Fifth, more than 30,000 members from military and public hospitals have successively headed out to Wuhan and other cities in Hubei Province.ER -</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus and the race to distribute reliable diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">International teams worked at speed to make tests for the virus available in record time. The medical community is rallying to develop a set of rapid and reliable molecular diagnostic tests for the new human coronavirus that appeared in China — now dubbed sudden acute respiratory syndrome coronavirus-2 (SARS-CoV-2).</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Top stories: Bungled coronavirus testing, disarming ‘atomic bomb’ cells, and jet stream blocking</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? One answer is that a test kit designed by the U.S. Centers for Disease Control and Prevention (CDC) and rolled out to state and local labs late last month contained a faulty reagent. The problems have led many to doubt the accuracy of confirmed tallies of the disease. But the situation may soon improve, as the Food and Drug Administration comes up with a workaround for the faulty CDC kit.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>From Isolation to Coordination: How Can Telemedicine Help Combat the COVID-19 Outbreak?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The rapid spread of Coronavirus disease 2019 (COVID-19) presents China with a critical challenge. As normal capacity of the Chinese hospitals is exceeded, healthcare professionals struggling to manage this unprecedented crisis face the difficult question of how best to coordinate the medical resources used in highly separated locations. Responding rapidly to this crisis, the National Telemedicine Center of China (NTCC), located in Zhengzhou, Henan Province, has established the Emergency Telemedicine Consultation System (ETCS), a telemedicine-enabled outbreak alert and response network. ETCS is built upon a doctor-to-doctor (D2D) approach, in which health services can be accessed remotely through terminals across hospitals. The system architecture of ETCS comprises three major architectural layers: (1) telemedicine service platform layer, (2) telemedicine cloud layer, and (3) telemedicine service application layer. Our ETCS has demonstrated substantial benefits in terms of the effectiveness of consultations and remote patient monitoring, multidisciplinary care, and prevention education and training.</jats:p></p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Response to COVID-19 in Taiwan: Big Data Analytics, New Technology, and Proactive Testing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Taiwan is 81 miles off the coast of mainland China and was expected to have the second highest number of cases of coronavirus disease 2019 (COVID-19) due to its proximity to and number of flights between China.1 The country has 23 million citizens of which 850 000 reside in and 404 000 work in China.2,3 In 2019, 2.71 million visitors from the mainland traveled to Taiwan.4 As such, Taiwan has been on constant alert and ready to act on epidemics arising from China ever since the severe acute respiratory syndrome (SARS) epidemic in 2003. Given the continual spread of COVID-19 around the world, understanding the action items that were implemented quickly in Taiwan and assessing the effectiveness of these actions in preventing a large-scale epidemic may be instructive for other countries. COVID-19 occurred just before the Lunar New Year during which time millions of Chinese and Taiwanese were expected to travel for the holidays. Taiwan quickly mobilized and instituted specific approaches for case identification, containment, and resource allocation to protect the public health. Taiwan leveraged its national health insurance database and integrated it with its immigration and customs database to begin the creation of big data for analytics; it generated real-time alerts during a clinical visit based on travel history and clinical symptoms to aid case identification. It also used new technology, including QR code scanning and online reporting of travel history and health symptoms to classify travelers’ infectious risks based on flight origin and travel history in the past 14 days. Persons with low risk (no travel to level 3 alert areas) were sent a health declaration border pass via SMS (short message service) messaging to their phones for faster immigration clearance; those with higher risk (recent travel to level 3 alert areas) were quarantined at home and tracked through their mobile phone to ensure that they remained at home during the incubation period. Moreover, Taiwan enhanced COVID-19 case finding by proactively seeking out patients with severe respiratory symptoms (based on information from the National Health Insurance [NHI] database) who had tested negative for influenza and retested them for COVID-19; 1 was found of 113 cases. The toll-free number 1922 served as a hotline for citizens to report suspicious symptoms or cases in themselves or others; as the disease progressed, this hotline has reached full capacity, so each major city was asked to create its own hotline as an alternative. It is not known how often this hotline has been used. The government addressed the issue of disease stigma and compassion for those affected by providing food, frequent health checks, and encouragement for those under quarantine. This rapid response included hundreds of action items (eTable in the Supplement).</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>The ongoing crises in China illustrate that the assessment of epidemics in isolation is no longer sufficient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary The interplay of simultaneous COVID-19, African swine fever, and avian influenza emergencies on global health and industries is constantly evolving and difficult to predict, and therefore warrants further scrutiny. The need for a health network of global scope for the rapid and open exchange of information needs to be strengthened in order to address ongoing and future epidemics under competing resources.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>The race to unravel the United States’ biggest coronavirus outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rohit Shankar left the virology laboratory at 2 a.m. on Wednesday, and was back at the lab bench by 7 a.m. the same day. “It’s okay,” he says, “I had a doughnut and a coffee.” Shankar, a medical scientist, and his colleagues at the University of Washington in Seattle are poised to exponentially drive up the number of confirmed cases of the coronavirus disease COVID-19 around the city, in western Washington state. That’s because this week, they began analysing a mountain of nose and throat swabs collected from hospitals in the region. Already, the researchers are seeing clear signs that the virus has infected vastly more people than have been formally detected. Scientists fear coronavirus spread in countries least able to contain it Washington state has become the United States’ ground zero for COVID-19, which has now spread to more than 90 countries worldwide in what seems to be a new and dangerous phase of the outbreak. Washington has declared a state emergency, and ten people there have died from the disease. But the number of confirmed cases in Washington — 70 — is still an underestimate resulting from a lack of testing, researchers agree. A genomic analysis posted online on 29 February suggested that hundreds of people in western Washington might already be infected. Academic scientists have mostly been prevented from measuring the extent of the US outbreak because of federal rules restricting the number of labs qualified to run diagnostic tests. But that is changing now, and helps account for why the state's caseload jumped from 10 to 70 this week. Dozens of virologists and genomicists have now kicked into high gear in Seattle, dropping or adapting projects to devote resources to the outbreak. Researchers are working around the clock to find out how many people have the disease in the area. Others are analysing genomes to reveal how the virus is transmitted or developing new therapies. The scientists are racing to help Washington avoid the fate of Hubei province in China, where more than 2,900 people have died of COVID-19 so far. The coronavirus emerged in the province’s city Wuhan in December, and the initial response from officials was slow. “We are past the point of containment,” says Helen Chu, an infectious-disease specialist at the University of Washington School of Medicine (UW Medicine) in Seattle. “So now we need to keep the people who are vulnerable from getting sick.”</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Integrating Genome-based Informatics to Modernize Global Disease Monitoring, Information Sharing, and Response</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid advancement of genome technologies holds great promise for improving the quality and speed of clinical and public health laboratory investigations and for decreasing their cost. The latest generation of genome DNA sequencers can provide highly detailed and robust information on disease-causing microbes, and in the near future these technologies will be suitable for routine use in national, regional, and global public health laboratories. With additional improvements in instrumentation, these next- or third-generation sequencers are likely to replace conventional culture-based and molecular typing methods to provide point-of-care clinical diagnosis and other essential information for quicker and better treatment of patients. Provided there is free-sharing of information by all clinical and public health laboratories, these genomic tools could spawn a global system of linked databases of pathogen genomes that would ensure more efficient detection, prevention, and control of endemic, emerging, and other infectious disease outbreaks worldwide.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Computers and viral diseases. Preliminary bioinformatics studies on the design of a synthetic vaccine and a preventative peptidomimetic antagonist against the SARS-CoV-2 (2019-nCoV, COVID-19) coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of novel coronavirus’s spike protein solved</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">n a piece of research accomplished with astonishing speed, scientists from the University of Texas at Austin and the National Institutes of Health have released a cryo-electron microscopy (cryo-EM) structure of part of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the novel coronavirus that has infected tens of thousands of people and killed more than 2,000 since the end of December (Science 2020, DOI: 10.1126/science.abb2507). The part of the virus imaged, called the spike protein, helps the virus attach to and infect human cells, and the determination of its structure comes just weeks after the virus’s genome sequence was published. The breakthrough is a huge step toward developing a vaccine against the virus as well as treatments for COVID-19, the disease that it causes, the researchers say. UT Austin’s Jason McLellan and his colleagues, who have spent many years studying other coronaviruses, had already figured out how to use select</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome sequences published outside of Standards in Genomic Sciences, October - November 2012</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 12</span></i><p style="text-align: justify; text-justify: inter-word;">The purpose of this table is to provide the community with a citable record of publications of ongoing genome sequencing projects that have led to a publication in the scientific literature. While our goal is to make the list complete, there is no guarantee that we may have omitted one or more publications appearing in this time frame. Readers and authors who wish to have publications added to subsequent versions of this list are invited to provide the bibliographic data for such references to the SIGS editorial office.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome sequences published outside of Standards in Genomic Sciences, July - October 2012</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 10</span></i><p style="text-align: justify; text-justify: inter-word;">The purpose of this table is to provide the community with a citable record of publications of ongoing genome sequencing projects that have led to a publication in the scientific literature. While our goal is to make the list complete, there is no guarantee that we may have omitted one or more publications appearing in this time frame. Readers and authors who wish to have publications added to subsequent versions of this list are invited to provide the bibliographic data for such references to the SIGS editorial office.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>The management of biosafety risk in clinical laboratory of hospital during the outbreak of 2019 Novel Coronavirus disease</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">During the outbreak of coronavirus disease-19 (COVID-19), the clinical laboratories of hospitals designated for the disease treatment is undertaking a lot of clinical testing work of infectious specimens. How to manage the biosafety risk is a major problem that the clinical laboratory and the nosocomial infection control department are facing. This article introduces the hierarchical prevention and control biosafety measures in the clinical laboratory from the perspective of the laboratory, with a view to provide reasonable and feasible methods for the clinical laboratories of hospitals at various levels during the outbreak.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Quarantined at home now, U.S. scientist describes his visit to China’s hot zone</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 13 February, Clifford Lane went to a Washington, D.C.–area airport to catch a flight to Japan, where he would help launch a study of an experimental drug, remdesivir, against coronavirus disease 2019 (COVID-19). Lane is a deputy director at the U.S. National Institute of Allergy and Infectious Diseases and a right-hand man to Anthony Fauci, head of NIAID and the top research scientist in the country advising the White House on the outbreak of the virus. As Lane waited to board his plane, he was told that his final destination had changed. “I get an email, ‘You need to go to China.’ It’s like, are you kidding?” Lane had been selected as one of two U.S. scientists to join a World Health Organization team of 13 international researchers who would tour five different cities with 12 Chinese colleagues to get a firsthand look at the coronavirus epidemic there. The joint mission, which ran from 16–23 February led to a report that offered more details about the clinical course of COVID-19 and the epidemiology in China than had appeared anywhere before.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 30</span></i><p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Standards for Sequencing Viral Genomes in the Era of High-Throughput Sequencing</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jun 17</span></i><p style="text-align: justify; text-justify: inter-word;">Thanks to high-throughput sequencing technologies, genome sequencing has become a common component in nearly all aspects of viral research; thus, we are experiencing an explosion in both the number of available genome sequences and the number of institutions producing such data. However, there are currently no common standards used to convey the quality, and therefore utility, of these various genome sequences. Here, we propose five “standard” categories that encompass all stages of viral genome finishing, and we define them using simple criteria that are agnostic to the technology used for sequencing. We also provide genome finishing recommendations for various downstream applications, keeping in mind the cost-benefit trade-offs associated with different levels of finishing. Our goal is to define a common vocabulary that will allow comparison of genome quality across different research groups, sequencing platforms, and assembly techniques.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Labs scramble to spot hidden coronavirus infections | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The seeming precision of the global tallies of cases and deaths caused by the novel coronavirus now spreading from Wuhan, China belies an alarming fact. The world is in the dark about the epidemic’s real scale and speed, because existing tests have limited powers—and testing is far too spotty. “We are underestimating how common this infection is,” cautions Jeremy Farrar, head of the Wellcome Trust. Within days of Chinese researchers releasing the sequence of the virus on 11 January, scientists developed tests capable of detecting genetic sequences that distinguish the new agent from other coronaviruses circulating in humans. By 28 January, China’s National Medical Products Administration had approved diagnostic test kits from five companies. It was an astonishing pace for the response to a pathogen never seen before—and yet it was only a beginning.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: Establishing a social contact data sharing initiative and an online tool to assess mitigation strategies for COVID-19. Results: Using our online tool and the available social contact data, we illustrate that social distancing could have a considerable impact on reducing transmission for COVID-19. The effect itself depends on assumptions made about disease-specific characteristics and the choice of intervention(s).  Keywords: Social contact data, Data sharing initiative, User interface.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>[The importance of strengthening the ability of fundamental disease prevention and control system from the perspective of the epidemic situation of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 has been in epidemic for nearly two months. The prevention and control measures have achieved remarkable results. From the response and disposal process of this epidemic, it is exposed that fundamental disease prevention and control system are insufficient in human resources and ability of laboratory testing. It is suggested that the disease control institutions should strengthen the construction of these aspects in future.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS, MERS and novel coronavirus (COVID-19) epidemics, the newest and biggest global health threats: what lessons have we learned?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China.Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made.Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally.We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Labs scramble to produce new coronavirus diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The seeming precision of the global tallies of cases and deaths caused by the novel coronavirus now spreading from Wuhan, China, belies an alarming fact. The world is in the dark about the epidemic's real scale and speed, because existing tests have limited powers—and testing is far too spotty. “We are underestimating how common this infection is,” cautions Jeremy Farrar, head of the Wellcome Trust. Within days of Chinese researchers releasing the sequence of the virus on 11 January, scientists developed tests capable of detecting genetic sequences that distinguish the new agent from other coronaviruses circulating in humans. By 28 January, China's National Medical Products Administration had approved diagnostic test kits from five companies. It was an astonishing pace for the response to a pathogen never seen before—and yet it was only a beginning</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Networks of information token recurrences derived from genomic sequences may reveal hidden patterns in epidemic outbreaks: A case study of the 2019-nCoV coronavirus.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and suggest touching points and temporal pathways that are of interest for deeper investigation by geneticists and epidemiologists.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>CoVDB: a comprehensive database for comparative analysis of coronavirus genes and genomes</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Jan 2</span></i><p style="text-align: justify; text-justify: inter-word;">The recent SARS epidemic has boosted interest in the discovery of novel human and animal coronaviruses. By July 2007, more than 3000 coronavirus sequence records, including 264 complete genomes, are available in GenBank. The number of coronavirus species with complete genomes available has increased from 9 in 2003 to 25 in 2007, of which six, including coronavirus HKU1, bat SARS coronavirus, group 1 bat coronavirus HKU2, groups 2c and 2d coronaviruses, were sequenced by our laboratory. To overcome the problems we encountered in the existing databases during comparative sequence analysis, we built a comprehensive database, CoVDB (http://covdb.microbiology.hku.hk), of annotated coronavirus genes and genomes. CoVDB provides a convenient platform for rapid and accurate batch sequence retrieval, the cornerstone and bottleneck for comparative gene or genome analysis. Sequences can be directly downloaded from the website in FASTA format. CoVDB also provides detailed annotation of all coronavirus sequences using a standardized nomenclature system, and overcomes the problems of duplicated and identical sequences in other databases. For complete genomes, a single representative sequence for each species is available for comparative analysis such as phylogenetic studies. With the annotated sequences in CoVDB, more specific blast search results can be generated for efficient downstream analysis.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Database and Online Inquiry System Based on Natural Vectors</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 17</span></i><p style="text-align: justify; text-justify: inter-word;">We construct a virus database called VirusDB (http://yaulab.math.tsinghua.edu.cn/VirusDB/) and an online inquiry system to serve people who are interested in viral classification and prediction. The database stores all viral genomes, their corresponding natural vectors, and the classification information of the single/multiple-segmented viral reference sequences downloaded from National Center for Biotechnology Information. The online inquiry system serves the purpose of computing natural vectors and their distances based on submitted genomes, providing an online interface for accessing and using the database for viral classification and prediction, and back-end processes for automatic and manual updating of database content to synchronize with GenBank. Submitted genomes data in FASTA format will be carried out and the prediction results with 5 closest neighbors and their classifications will be returned by email. Considering the one-to-one correspondence between sequence and natural vector, time efficiency, and high accuracy, natural vector is a significant advance compared with alignment methods, which makes VirusDB a useful database in further research.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Reconsideration on the multiple value of Behavior determining Health: in the perspective of the situation of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Why has the concept of behavior determining health created more and more extensive and far-reaching influence ever since it was put forward? The reason lies in its multiple values. It is of great practical significance and has important implications for long-term health care to explore and analyze in the perspective of the situation of COVID-19 its philosophical values, cultural values, methodological values, social values and the national strategic value of 'healthy China'.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Singapore claims first use of antibody test to track coronavirus infections | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In what appears to be a first, disease trackers in Singapore have used an experimental antibody test for COVID-19 to confirm that a suspected patient was infected with the coronavirus. The patient was one of two people who together formed a missing link between two clusters of cases that each occurred in a Singaporean church. Researchers around the world are racing to develop antibody tests, also called serological tests, that can confirm whether someone was infected even after their immune system has cleared the virus that causes COVID-19. The group that developed the test, at Duke-NUS Medical School in Singapore, is among the front-runners, although its assay has to be validated before it is taken into production and deployed widely.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>What we know so far: COVID-19 current clinical knowledge and research</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, health authorities in Wuhan, China, identified a cluster of pneumonia cases of unknown aetiology linked to the city's South China Seafood Market. Subsequent investigations revealed a novel coronavirus, SARS-CoV-2, as the causative agent now at the heart of a major outbreak. The rising case numbers have been accompanied by unprecedented public health action, including the wholesale isolation of Wuhan. Alongside this has been a robust scientific response, including early publication of the pathogen genome, and rapid development of highly specific diagnostics. This article will review the new knowledge of SARS-CoV-2 COVID-19 acute respiratory disease, and summarise its clinical features.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Recommended psychological crisis intervention response to the 2019 novel coronavirus pneumonia outbreak in China: a model of West China Hospital</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus pneumonia (COVID-19)epidemic has brought serious social psychological impact to the Chinese people, especially those quarantined and thus with limited access to face-to-face communication and traditional social psychological interventions. To better deal with the urgent psychological problems of people involved in the COVID-19 epidemic, we developed a new psychological crisis intervention model by utilizing internet technology. This new model, one of West China Hospital, integrates physicians, psychiatrists, psychologists and social workers into Internet platforms to carry out psychological intervention to patients, their families and medical staff. We hope this model will make a sound basis for developing a more comprehensive psychological crisis intervention response system that is applicable for urgent social and psychological problems.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>How to fight an infodemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">WHO's newly launched platform aims to combat misinformation around COVID-19. John Zarocostas reports from WHO is leading the effort to slow the spread of the 2019 coronavirus disease (COVID-19) outbreak. But a global epidemic of misinformation—spreading rapidly through social media platforms and other outlets—poses a serious problem for public health. “We’re not just fighting an epidemic; we’re fighting an infodemic”, said WHO Director-General Tedros Adhanom Ghebreyesus at the Munich Security Conference on Feb 15. Immediately after COVID-19 was declared a Public Health Emergency of International Concern, WHO's risk communication team launched a new information platform called WHO Information Network for Epidemics (EPI-WIN), with the aim of using a series of amplifiers to share tailored information with specific target groups. Sylvie Briand, director of Infectious Hazards Management at WHO's Health Emergencies Programme and architect of WHO's strategy to counter the infodemic risk, told The Lancet, “We know that every outbreak will be accompanied by a kind of tsunami of information, but also within this information you always have misinformation, rumours, etc. We know that even in the Middle Ages there was this phenomenon”. “But the difference now with social media is that this phenomenon is amplified, it goes faster and further, like the viruses that travel with people and go faster and further. So it is a new challenge, and the challenge is the [timing] because you need to be faster if you want to fill the void…What is at stake during an outbreak is making sure people will do the right thing to control the disease or to mitigate its impact. So it is not only information to make sure people are informed; it is also making sure people are informed to act appropriately.” About 20 staff and some consultants are involved in WHO's communications teams globally, at any given time. This includes social media personnel at each of WHO's six regional offices, risk communications consultants, and WHO communications officers.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>NCBI Viral Genomes Resource</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jan 28</span></i><p style="text-align: justify; text-justify: inter-word;">Recent technological innovations have ignited an explosion in virus genome sequencing that promises to fundamentally alter our understanding of viral biology and profoundly impact public health policy. Yet, any potential benefits from the billowing cloud of next generation sequence data hinge upon well implemented reference resources that facilitate the identification of sequences, aid in the assembly of sequence reads and provide reference annotation sources. The NCBI Viral Genomes Resource is a reference resource designed to bring order to this sequence shockwave and improve usability of viral sequence data. The resource can be accessed at http://www.ncbi.nlm.nih.gov/genome/viruses/ and catalogs all publicly available virus genome sequences and curates reference genome sequences. As the number of genome sequences has grown, so too have the difficulties in annotating and maintaining reference sequences. The rapid expansion of the viral sequence universe has forced a recalibration of the data model to better provide extant sequence representation and enhanced reference sequence products to serve the needs of the various viral communities. This, in turn, has placed increased emphasis on leveraging the knowledge of individual scientific communities to identify important viral sequences and develop well annotated reference virus genome sets.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Hospital Emergency Management Plan During the COVID-19 Epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The confirmed and suspected cases of the 2019 novel coronavirus disease (COVID-19) have increased not only in Wuhan, Hubei Province but also China and the world. Enormous demand for handling the COVID-19 outbreak challenged both the healthcare personnel and medical supply system. In West China Hospital, Emergency Department (ED) undertook the mission of clinical reception, primary diagnosis, and interim treatment for the suspected cases of COVID-19.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Visualising the expansion and spread of coronavirus disease 2019 by cartograms</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) has emerged as a growing focus of global attention and a critical factor in public-health decision making. Towards fighting the COVID-19 outbreak, countries worldwide and international organisations have taken various actions, including promoting the transparency of and public access to disease data. In such public communications, maps have played an important role in that a map is worth a thousand words. Most of these have taken the form of a choropleth map. Here, we propose employing cartograms to visualise both the expansion and spread of COVID-19. We designed a combination of six circular cartograms containing the data of confirmed cases every 48 hours from 24 January to 3 February 2020. Such a design conveys both spatial and temporal information more intuitively and efficiently, so it can be expected to facilitate better public participation in the fight against COVID-19.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Effectiveness for the Response to COVID-19: The MERS Outbreak Containment Procedures</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In the current issue of Osong Public Health and Research Perspectives, 3 studies are presented dealing with COVID-19. A study by Kim et al, analyzed the genetic information of the COVID-19 virus isolated from a patient in South Korea. The virus was identified by real-time reverse transcriptase (RT) PCR followed by Next Generation Sequencing of the full-length of the genome [5]. In an interim report by the COVID-19 National Emergency Response Center of the KCDC, epidemiological and clinical characteristics of COVID-19 in 28 cases in South Korea were reported. There were 53.9% of cases who were males, with 16 cases in patients who had traveled abroad, of which 11 cases (39.3%) were from Wuhan, China, and 12 of the remaining cases were believed to have been infected in South Korea. The incubation period was 4.8 days. Most secondary infected cases were from family members outside the family home (66.7%) and within the family home (75%) [6].</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Early epidemiological analysis of the coronavirus disease 2019 outbreak based on crowdsourced data: a population-level observational study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Background As the outbreak of coronavirus disease 2019 (COVID-19) progresses, epidemiological data are needed to guide situational awareness and intervention strategies. Here we describe efforts to compile and disseminate epidemiological information on COVID-19 from news media and social networks. Methods In this population-level observational study, we searched DXY.cn, a health-care-oriented social network that is currently streaming news reports on COVID-19 from local and national Chinese health agencies. We compiled a list of individual patients with COVID-19 and daily province-level case counts between Jan 13 and Jan 31, 2020, in China. We also compiled a list of internationally exported cases of COVID-19 from global news media sources (Kyodo News, The Straits Times, and CNN), national governments, and health authorities. We assessed trends in the epidemiology of COVID-19 and studied the outbreak progression across China, assessing delays between symptom onset, seeking care at a hospital or clinic, and reporting, before and after Jan 18, 2020, as awareness of the outbreak increased. All data were made publicly available in real time. Findings We collected data for 507 patients with COVID-19 reported between Jan 13 and Jan 31, 2020, including 364 from mainland China and 143 from outside of China. 281 (55%) patients were male and the median age was 46 years (IQR 35–60). Few patients (13 [3%]) were younger than 15 years and the age profile of Chinese patients adjusted for baseline demographics confirmed a deficit of infections among children. Across the analysed period, delays between symptom onset and seeking care at a hospital or clinic were longer in Hubei province than in other provinces in mainland China and internationally. In mainland China, these delays decreased from 5 days before Jan 18, 2020, to 2 days thereafter until Jan 31, 2020 (p=0·0009). Although our sample captures only 507 (5·2%) of 9826 patients with COVID-19 reported by official sources during the analysed period, our data align with an official report published by Chinese authorities on Jan 28, 2020. Interpretation News reports and social media can help reconstruct the progression of an outbreak and provide detailed patient-level data in the context of a health emergency. The availability of a central physician-oriented social network facilitated the compilation of publicly available COVID-19 data in China. As the outbreak progresses, social media and news reports will probably capture a diminishing fraction of COVID-19 cases globally due to reporting fatigue and overwhelmed health-care systems. In the early stages of an outbreak, availability of public datasets is important to encourage analytical efforts by independent teams and provide robust evidence to guide interventions. Funding Fogarty International Center, US National Institutes of Health.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Indonesia finally reports two coronavirus cases. Scientists worry it has many more</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Indonesia finally reported its first two COVID-19 cases on Monday. At a press conference in Jakarta, President Joko Widodo announced that two women aged 31 and 64 from Depok had contracted the virus. But some scientists believe the country, which has close ties to China, almost certainly has a “silent epidemic” within its borders and should urgently boost its surveillance efforts. Scientists at the Eijkman Institute for Molecular Biology tell Science they have offered to the Indonesian Ministry of Health to help test more people, but have been snubbed so far. The detection of the first two cases, along with measures to prevent the spread of the disease, demonstrate Indonesia’s seriousness and capability to deal with the COVID-19 epidemic, Widodo said yesterday. “Since the beginning, we have been serious in strictly following WHO [World Health Organization] guidelines regarding the coronavirus issue,” he said.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Chiropteran types I and II interferon genes inferred from genome sequencing traces by a statistical gene-family assembler</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Jul 21</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The rate of emergence of human pathogens is steadily increasing; most of these novel agents originate in wildlife. Bats, remarkably, are the natural reservoirs of many of the most pathogenic viruses in humans. There are two bat genome projects currently underway, a circumstance that promises to speed the discovery host factors important in the coevolution of bats with their viruses. These genomes, however, are not yet assembled and one of them will provide only low coverage, making the inference of most genes of immunological interest error-prone. Many more wildlife genome projects are underway and intend to provide only shallow coverage. RESULTS: We have developed a statistical method for the assembly of gene families from partial genomes. The method takes full advantage of the quality scores generated by base-calling software, incorporating them into a complete probabilistic error model, to overcome the limitation inherent in the inference of gene family members from partial sequence information. We validated the method by inferring the human IFNA genes from the genome trace archives, and used it to infer 61 type-I interferon genes, and single type-II interferon genes in the bats Pteropus vampyrus and Myotis lucifugus. We confirmed our inferences by direct cloning and sequencing of IFNA, IFNB, IFND, and IFNK in P. vampyrus, and by demonstrating transcription of some of the inferred genes by known interferon-inducing stimuli. CONCLUSION: The statistical trace assembler described here provides a reliable method for extracting information from the many available and forthcoming partial or shallow genome sequencing projects, thereby facilitating the study of a wider variety of organisms with ecological and biomedical significance to humans than would otherwise be possible.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Straining the System: Novel Coronavirus (COVID-19) and Preparedness for Concomitant Disasters</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Just a few weeks before the first confirmed case of novel coronavirus (COVID-19) was reported in the United States, the US Centers for Disease Control and Prevention (CDC) issued a bold promise to the nation: the agency will use its scientific expertise to bring a new level of preparedness in the United States and global health security against current and growing threats, finally eliminate certain diseases, and bring an end to the devastation of epidemics.(1) The current outbreak of COVID-19 reminds us how urgent this promise is and just how critical it is to continue to sustain and strengthen our nation's public health infrastructure. The unprecedented pace of the public health response to COVID-19 has only been possible because of prior investments in public health preparedness. To accelerate our pace and meet the challenges of current and future health threats, we must advance our world-class data and analytics capabilities; maintain and expand our state-of-the-art public health laboratory capacity; continue building a workforce of trusted, expert, public health professionals; sustain our capacity to rapidly respond to outbreaks at their source; and assure a strong global and domestic preparedness capacity. (Am J Public Health. Published online ahead of print February 13, 2020: e1-e2. doi:10.2105/AJPH.2020.305618).</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic Review of the Registered Clinical Trials of Coronavirus Diseases 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Since the outbreak of coronavirus diseases 2019 (COVID-19), many researchers in China have immediately carried out clinical research scheme of the COVID-19. But, there is still a lack of systematic review of registered clinical trials. Therefore, we made the first systematic review of the clinical trials of COVID-19 in order to provide evidence for the control of the COVID-19. Methods: The database from the Chinese Clinical Registration Center and the ClinicalTrials.gov were searched to collect the registered clinical trials of COVID-19. The retrieval inception date is February 9, 2020. Two evaluators independently selected literature, extracted data and evaluated the risk of bias. This study is based on the recommendations of PRISMA in Cochrane handbook. Results: A total of 75 COVID-19 registered clinical trials (63 interventional studies and 12 observational studies) were obtained. 97.3% of clinical trials were initiated by Chinese organizations. Only 11 trials have begun to recruit patients, and all registered clinical trials have not been completed. Most of the trials are early clinical exploratory trials or in pre-experiment stage (only two trials of Remdesivir in Ⅲ stage), and the sample size of subjects recruited is small. The main intervention methods include traditional Chinese medicine treatment, western medicine treatment and integrated Chinese and Western medicine treatment. The subjects were mainly non severe adult patients (≥ 18 years old). The main outcomes were clinical observation and examination. The duration of most trials was more than 5 months, and the median of the intervention study was 180 d (95% CI: 146.3 - 328.9 d); the median of the observation period was 334 d (95% CI: 166.6 - 363.4 d). Overall, both the methodology quality of intervention register trials and observational trials are low. Conclusions: Disorderly and intensive clinical trials of COVID-19 using traditional Chinese medicine and Western medicine are ongoing or will be carried out in China. However, based on the poor quality and small sample size and long completion period, we will not be able to obtain reliable, high-quality clinical evidence about COVID-19 treatment for quite a long time in the future. Improving the quality of study design, prioritizing promising drugs, and using different designs and statistical methods are worth advocating and recommending for the clinical trials of COVID-19 in China.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>“The disruption is enormous.” Coronavirus epidemic snarls science worldwide | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Normal daily life has come to a virtual standstill in large parts of China as a result of the epidemic of COVID-19—and so has science. Universities across the country remain closed; access to labs is restricted, projects have been mothballed, field work interrupted, and travel severely curtailed. But scientists elsewhere in the world are noticing an impact as well, as collaborations with China are on pause and scientific meetings for the next five months have been canceled or postponed. The damage to science pales compared to the human suffering; the total number of cases has risen to 71,429, the World Health Organization (WHO) reported today, almost 99% of them in China, and there have been 1775 deaths. Still, for individual researchers the losses can be serious—and stressful. “Basically, everything has completely stopped,” says John Speakman, who runs an animal behavior lab at the Chinese Academy of Sciences (CAS) in Beijing that has effectively been shut since the Lunar New Year on 25 January. “The disruption is enormous. The stress on the staff is really high.” But Speakman says he understands why the Chinese government took the measures. “It’s annoying, but I completely support what they have done,” he says.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>The 2019 novel coronavirus resource</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An ongoing outbreak of a novel coronavirus infection in Wuhan, China since December 2019 has led to 31,516 infected persons and 638 deaths across 25 countries (till 16:00 on February 7, 2020). The virus causing this pneumonia was then named as the 2019 novel coronavirus (2019-nCoV) by the World Health Organization. To promote the data sharing and make all relevant information of 2019-nCoV publicly available, we construct the 2019 Novel Coronavirus Resource (2019nCoVR, https://bigd.big.ac.cn/ncov). 2019nCoVR features comprehensive integration of genomic and proteomic sequences as well as their metadata information from the Global Initiative on Sharing All Influenza Data, National Center for Biotechnology Information, China National GeneBank, National Microbiology Data Center and China National Center for Bioinformation (CNCB)/National Genomics Data Center (NGDC). It also incorporates a wide range of relevant information including scientific literatures, news, and popular articles for science dissemination, and provides visualization functionalities for genome variation analysis results based on all collected 2019-nCoV strains. Moreover, by linking seamlessly with related databases in CNCB/NGDC, 2019nCoVR offers virus data submission and sharing services for raw sequence reads and assembled sequences. In this report, we provide comprehensive descriptions on data deposition, management, release and utility in 2019nCoVR, laying important foundations in aid of studies on virus classification and origin, genome variation and evolution, fast detection, drug development and pneumonia precision prevention and therapy.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevent and predict</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the COVID-19 outbreak continues, the next pandemic could be prevented by ending the wildlife trade and reinvesting in the monitoring of potential zoonoses.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>The 2019-nCoV epidemic control strategies and future challenges of building healthy smart cities</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">reported and confirmed on 31 December 2019, in Wuhan, Hubei Province, China, which is one of China’s largest cities and a major domestic transport hub (located in the central part of China, as shown in Figure 1).1 The epidemic is attracting worldwide concern due to its rapid spread and transmission rate between humans. On 30 January 2020, the International Health Regulations, Emergency Committee of the World Health Organization (WHO) declared the outbreak – a ‘public health emergency of international concern’.2 On 8 February 2020 (24:00 GMT?+?8), there were 37,198 confirmed infections in China (including 811 deaths with a death ratio of 2.1%; and 6188 patients were confirmed in serious condition and 28,942 suspected cases).3 The COVID 19 infections were also reported in 26 other countries on 7 February 2020, including Canada, the USA, Australia, India, Sri-Lanka, Cambodia, Thailand, Vietnam, Malaysia, Singapore, Taiwan, the Republic of Korea, Sri Lanka, Japan, Philippines, Nepal, United Arab Emirates, Russia, Italy, Germany, Sweden, Finland, Belgium, Spain, France and the UK.4</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory readiness and response for novel coronavirus (2019-nCoV) in expert laboratories in 30 EU/EEA countries, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Timely detection of novel coronavirus (2019-nCoV) infection cases is crucial to interrupt the spread of this virus. We assessed the required expertise and capacity for molecular detection of 2019-nCoV in specialised laboratories in 30 European Union/European Economic Area (EU/EEA) countries. Thirty-eight laboratories in 24 EU/EEA countries had diagnostic tests available by 29 January 2020. A coverage of all EU/EEA countries was expected by mid-February. Availability of primers/probes, positive controls and personnel were main implementation barriers.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory readiness and response for novel coronavirus (2019-nCoV) in expert laboratories in 30 EU/EEA countries, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">Timely detection of novel coronavirus (2019-nCoV) infection cases is crucial to interrupt the spread of this virus. We assessed the required expertise and capacity for molecular detection of 2019-nCoV in specialised laboratories in 30 European Union/European Economic Area (EU/EEA) countries. Thirty-eight laboratories in 24 EU/EEA countries had diagnostic tests available by 29 January 2020. A coverage of all EU/EEA countries was expected by mid-February. Availability of primers/probes, positive controls and personnel were main implementation barriers.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Strengthening laboratory capacity for detection of respiratory viral pathogens through the Global Health Security Agenda (GHSA) framework</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Endemic and emerging respiratory viruses are a threat to public health, and a robust public health laboratory system is essential to ensure global health security. OBJECTIVE: This program sought to expand molecular laboratory testing capacity to detect a broad range of respiratory pathogens in clinical respiratory specimens collected during disease surveillance and outbreak investigations. METHODS: As a part of the Global Health Security Agenda (GHSA), the United States Centers for Disease Control and Prevention utilised the equipment and training infrastructure already in place at the World Health Organization National Influenza Centers to expand testing capacity for respiratory viruses in laboratories in GHSA partner countries. This was done through the provision of quality assured reagents, including multiplex platforms and technical guidance for laboratory staff, as well as the assessment of laboratory testing accuracy. CONCLUSION: Early findings illustrated that GHSA laboratories have been able to expand testing capacity using specimens from routine surveillance, as well as from outbreak situations.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects and challenges of Open Science - Academic movements related to the Novel coronavirus (2019-nCoV) and COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">オープンサイエンスの効果はさまざまに論じられてき た。たとえば経済協力開発機構（OECD）の報告書 1) では， 科学研究の効率化，研究の透明性や質の向上，技術革新の 加速，経済への波及効果，地球規模の課題への取り組み， 共同研究の推進などが挙げられている。 折しも 2019 年 12 月以降，新型コロナウイルス（2019- nCoV）の感染が中国の武漢市から世界中に拡大して「地 球規模の課題」となっている。2020 年 1 月 30 日には世 界保健機関（WHO）が国際的な緊急事態であると宣言し た 2)。これに対して各国・地域の研究者，助成機関，学術 出版社といったステークホルダーが最新の研究成果の公開 と再利用を進めて対策にあたるという，まさに「オープン サイエンス」というべき動きが起きている。そこで本稿は 新型コロナウイルスに関する学術界の動向を概観した上 で，今回の事例におけるオープンサイエンスの効果と課題 について考察したい。なお，本稿の記述は 2020 年 2 月 6 日時点の情報に基づいている（情報が古い箇所もあろうか と思いますが，ご容赦を）。</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19's graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control.  Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation.  Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus  broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>A database resource for Genome-wide dynamics analysis of Coronaviruses on a historical and global scale</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-07</span></i><p style="text-align: justify; text-justify: inter-word;">The recent outbreak of a new zoonotic origin Coronavirus has ring the bell for the potential spread of epidemic Coronavirus crossing the species. With the urgent needs to assist the control of the Coronavirus spread and to provide valuable scientific information, we developed a coronavirus database (CoVdb), an online genomics and proteomics analysis platform. Based on public available coronavirus genomic information, the database annotates the genome of every strain and identifies 780 possible ORFs of all strains available in Genebank. In addition, the comprehensive evaluation of all the published genomes of Coronavirus strains, including population genetics analysis, functional analysis and structural analysis on a historical and global scale were presented in the CoVdb. In the database, the researcher can easily obtain the basic information of a Coronavirus gene with the distribution of the gene among strains, conserved or high mutation regions, possible subcellular location and topology of the gene. Moreover, sliding windows for population genetics analysis results is provided, thereby facilitating genetics and evolutional analysis at the genomic level. CoVdb can be accessed freely at http://covdb.popgenetics.net.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur.AimWe aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available.MethodsHere we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology.ResultsThe workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive - Global (EVAg), a European Union infrastructure project.ConclusionThe present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Representing virus-host interactions and other multi-organism processes in the Gene Ontology</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The Gene Ontology project is a collaborative effort to provide descriptions of gene products in a consistent and computable language, and in a species-independent manner. The Gene Ontology is designed to be applicable to all organisms but up to now has been largely under-utilized for prokaryotes and viruses, in part because of a lack of appropriate ontology terms. METHODS: To address this issue, we have developed a set of Gene Ontology classes that are applicable to microbes and their hosts, improving both coverage and quality in this area of the Gene Ontology. Describing microbial and viral gene products brings with it the additional challenge of capturing both the host and the microbe. Recognising this, we have worked closely with annotation groups to test and optimize the GO classes, and we describe here a set of annotation guidelines that allow the controlled description of two interacting organisms. CONCLUSIONS: Building on the microbial resources already in existence such as ViralZone, UniProtKB keywords and MeGO, this project provides an integrated ontology to describe interactions between microbial species and their hosts, with mappings to the external resources above. Housing this information within the freely-accessible Gene Ontology project allows the classes and annotation structure to be utilized by a large community of biologists and users.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Maybe not an overreaction</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A mathematical simulation of coronavirus COVID-19 estimated the number of infections to be far worse than reported. Starting in December 2019, a novel coronavirus, known as COVID-19, caused growing cases of atypical pneumonia in the city of Wuhan, China. The virus spread. As of 12 February 2020, a total of 45,204 confirmed cases had been reported from China and 519 confirmed cases from 27 countries worldwide. Among the reported victims, 1116 died. When facing an epidemic as such, a dangerous mistake that countries over the world can make is to rely entirely on confirmed data but underestimate the actual size of infections. Causes of underestimation include the limitation of resources, as there can be insufficient quarantine spaces, detection reagents, and medical personnel to identify infections in real time. Additionally, the lack of symptoms from the virus in its dormant state can delay confirmation. There is also a fear of data being manipulated or downplayed by officials due to economic and political concerns.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>VAPiD: a lightweight cross-platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 23</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: With sequencing technologies becoming cheaper and easier to use, more groups are able to obtain whole genome sequences of viruses of public health and scientific importance. Submission of genomic data to NCBI GenBank is a requirement prior to publication and plays a critical role in making scientific data publicly available. GenBank currently has automatic prokaryotic and eukaryotic genome annotation pipelines but has no viral annotation pipeline beyond influenza virus. Annotation and submission of viral genome sequence is a non-trivial task, especially for groups that do not routinely interact with GenBank for data submissions. RESULTS: We present Viral Annotation Pipeline and iDentification (VAPiD), a portable and lightweight command-line tool for annotation and GenBank deposition of viral genomes. VAPiD supports annotation of nearly all unsegmented viral genomes. The pipeline has been validated on human immunodeficiency virus, human parainfluenza virus 1–4, human metapneumovirus, human coronaviruses (229E/OC43/NL63/HKU1/SARS/MERS), human enteroviruses/rhinoviruses, measles virus, mumps virus, Hepatitis A-E Virus, Chikungunya virus, dengue virus, and West Nile virus, as well the human polyomaviruses BK/JC/MCV, human adenoviruses, and human papillomaviruses. The program can handle individual or batch submissions of different viruses to GenBank and correctly annotates multiple viruses, including those that contain ribosomal slippage or RNA editing without prior knowledge of the virus to be annotated. VAPiD is programmed in Python and is compatible with Windows, Linux, and Mac OS systems. CONCLUSIONS: We have created a portable, lightweight, user-friendly, internet-enabled, open-source, command-line genome annotation and submission package to facilitate virus genome submissions to NCBI GenBank. Instructions for downloading and installing VAPiD can be found at https://github.com/rcs333/VAPiD.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;">SummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.  AvailabilityAvailable online: https://www.genomedetective.com/app/typingtool/cov  * Contactkoen@emweb.be and deoliveira@ukzn.ac.za  Supplementary informationSupplementary data is available online.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Annotation Transfer Utility (GATU): rapid annotation of viral genomes using a closely related reference genome</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Jun 13</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Since DNA sequencing has become easier and cheaper, an increasing number of closely related viral genomes have been sequenced. However, many of these have been deposited in GenBank without annotations, severely limiting their value to researchers. While maintaining comprehensive genomic databases for a set of virus families at the Viral Bioinformatics Resource Center and Viral Bioinformatics – Canada , we found that researchers were unnecessarily spending time annotating viral genomes that were close relatives of already annotated viruses. We have therefore designed and implemented a novel tool, Genome Annotation Transfer Utility (GATU), to transfer annotations from a previously annotated reference genome to a new target genome, thereby greatly reducing this laborious task. RESULTS: GATU transfers annotations from a reference genome to a closely related target genome, while still giving the user final control over which annotations should be included. GATU also detects open reading frames present in the target but not the reference genome and provides the user with a variety of bioinformatics tools to quickly determine if these ORFs should also be included in the annotation. After this process is complete, GATU saves the newly annotated genome as a GenBank, EMBL or XML-format file. The software is coded in Java and runs on a variety of computer platforms. Its user-friendly Graphical User Interface is specifically designed for users trained in the biological sciences. CONCLUSION: GATU greatly simplifies the initial stages of genome annotation by using a closely related genome as a reference. It is not intended to be a gene prediction tool or a "complete" annotation system, but we have found that it significantly reduces the time required for annotation of genes and mature peptides as well as helping to standardize gene names between related organisms by transferring reference genome annotations to the target genome. The program is freely available under the General Public License and can be accessed along with documentation and tutorial from .</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Looming threat of COVID-19 infection in Africa: act collectively, and fast</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Models that enable the continent to better allocate scarce resources to better prepare and respond to the COVID-19 epidemic are crucial. The modelling study by Marius Gilbert and colleagues in The Lancet identifies each African country's risk of importation of COVID-19 from China, using data on the volume of air travel from three airports in provinces in China to African countries. Gilbert and colleagues use two indicators to determine the capacity of countries to detect and respond to cases: preparedness, using the WHO International Health Regulations Monitoring and Evaluation Framework; and vulnerability, using the Infectious Disease Vulnerability Index. Based on their analysis, Egypt, Algeria, and South Africa had the highest importation risk, and a moderate to high capacity to respond to outbreaks. Nigeria, Ethiopia, Sudan, Angola, Tanzania, Ghana, and Kenya had moderate risk with variable capacity and high vulnerability. In the model, the risk mainly originates from Guangdong, Fujian, and Beijing. The study provides a valuable tool that can help countries in Africa prioritise and allocate resources as they prepare to respond to the potential introduction and spread of COVID-19.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>US Emergency Legal Responses to Novel Coronavirus: Balancing Public Health and Civil Liberties</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">With increasing numbers of cases of coronavirus disease 2019 (COVID-19) globally and in the United States, Health and Human Services (HHS) Secretary Alex Azar declared a national public health emergency on January 31. The emergency declaration of the HHS authorizes additional resources, enhanced federal powers, interjurisdictional coordination, and waivers of specific regulations. State and local public health emergency declarations are also likely. During crises, government has a special responsibility to thoughtfully balance public health protections and civil liberties.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Lessons from SARS and MERS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Within weeks of the first cases, a series of papers were released detailing the epidemiology of the disease (now termed COVID‐19) [1-3]. By early January 2020 the virus was identified and the sequence determined. The virus (termed SARS‐CoV‐2) shares 88% sequence identity to two coronaviruses found in bats, bat‐SLCoVZC45 and bat‐SL‐CoVZXC21, 79% identity with the Severe Acute Respiratory Syndrome (SARS) coronavirus and 50% identity with Middle Eastern Respiratory Syndrome (MERS) coronavirus [4]. From the first cohort of patients, 8 complete genomes were 99.9% identical in sequence. Given that the typical RNA coronavirus evolves at a rate of 104 nucleotide substitutions per year, this suggests a recent single source emergence in early December or late November 2019 [4]. SARS‐CoV‐2 is thought to be transmitted via contaminated hands, surfaces and aerosolised droplets; extensive human‐to‐human transmission is evident, with clusters of infected families and medical staff [5]. The number of confirmed cases has increased rapidly, at a rate that far outstripped the rate of rise of cases of SARS in 2002/3, raising serious global health concerns. By the 21st January, COVID‐19 cases were widespread across mainland China, soon spreading beyond the Chinese borders.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Mitigate the effects of home confinement on children during the COVID-19 outbreak</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In response to the coronavirus disease 2019 (COVID-19) outbreak, the Chinese Government has ordered a nationwide school closure as an emergency measure to prevent spreading of the infection. Public activities are discouraged. The Ministry of Education estimates that more than 220 million children and adolescents are confined to their homes; this includes 180 million primary and secondary students and 47 million preschool children).1 Thanks to the strong administrative system in China, the emergency home schooling plan has been rigorously implemented.2 Massive efforts are being made by schools and teachers at all levels to create online courses and deliver them through TV broadcasts and the internet in record time. The new virtual semester has just started in many parts of the country, and various courses are offered online in a well organised manner. These actions are helping to alleviate many parents' concerns about their children's educational attainment by ensuring that school learning is largely undisrupted. Although these measures and efforts are highly commendable and necessary, there are reasons to be concerned because prolonged school closure and home confinement during a disease outbreak might have negative effects on children's physical and mental health.3 , 4 Evidence suggests that when children are out of school (eg, weekends and summer holidays), they are physically less active, have much longer screen time, irregular sleep patterns, and less favourable diets, resulting in weight gain and a loss of cardiorespiratory fitness.3 , 5 Such negative effects on health are likely to be much worse when children are confined to their homes without outdoor activities and interaction with same aged friends during the outbreak.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Why airport screening won’t stop the spread of coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">If you have traveled internationally the past 2 months, you may have encountered them: health officers briefly pointing a thermometer gun at your forehead or watching as you go by to check for signs of a cough or difficulty breathing. Many countries are now watching arriving and departing air passengers who might suffer from the viral disease COVID-19; some require passengers to fill out health declarations. (Some also simply ban or quarantine those who have recently been in outbreak hot spots.) Exit and entry screening may look reassuring, but experience with other diseases shows it’s exceedingly rare for screeners to detect infected passengers. Just last week, eight passengers who later tested positive for COVID-19 arrived in Shanghai from Italy and passed the airport screeners unnoticed, for example. And even if screeners do find the occasional case, it has almost no impact on the course of an outbreak.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>[Potential false-positive rate among the 'asymptomatic infected individuals' in close contacts of COVID-19 patients]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: As the prevention and control of COVID-19continues to advance, the active nucleic acid test screening in the close contacts of the patients has been carrying out in many parts of China. However, the false-positive rate of positive results in the screening has not been reported up to now. But to clearify the false-positive rate during screening is important in COVID-19 control and prevention. Methods: Point values and reasonable ranges of the indicators which impact the false-positive rate of positive results were estimated based on the information available to us at present. The false-positive rate of positive results in the active screening was deduced, and univariate and multivariate-probabilistic sensitivity analyses were performed to understand the robustness of the findings. Results: When the infection rate of the close contacts and the sensitivity and specificity of reported results were taken as the point estimates, the positive predictive value of the active screening was only 19.67%, in contrast, the false-positive rate of positive results was 80.33%. The multivariate-probabilistic sensitivity analysis results supported the base-case findings, with a 75% probability for the false-positive rate of positive results over 47%. Conclusions: In the close contacts of COVID-19 patients, nearly half or even more of the 'asymptomatic infected individuals' reported in the active nucleic acid test screening might be false positives.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential association between COVID-19 mortality and health-care resource availability</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As recorded by the Chinese Center for Disease Control and Prevention (China CDC), by Feb 16, 2020, there had been 70 641 confirmed cases and 1772 deaths due to COVID-19, with an average mortality of about 2·5%. However, in-depth analysis of these data show clear disparities in mortality rates between Wuhan (>3%), different regions of Hubei (about 2·9% on average), and across the other provinces of China (about 0·7% on average). We postulate that this is likely to be related to the rapid escalation in the number of infections around the epicentre of the outbreak, which has resulted in an insufficiency of health-care resources, thereby negatively affecting patient outcomes in Hubei, while this has not yet been the situation for the other parts of China (figure A, B). If we assume that average levels of health care are similar throughout China, higher numbers of infections in a given population can be considered an indirect indicator of a heavier health-care burden. Plotting mortality against the incidence of COVID-19 (cumulative number of confirmed cases since the start of the outbreak, per 10 000 population) showed a significant positive correlation (figure C), suggesting that mortality is correlated with health-care burden.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Prediction of COVID-19 Spreading Profiles in South Korea, Italy and Iran by Data-Driven Coding</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>This work applies a data-driven coding method for prediction of the COVID-19 spreading profile in any given population that shows an initial phase of epidemic progression. Based on the historical data collected for COVID-19 spreading in 367 cities in China and the set of parameters of the augmented Susceptible-Exposed-Infected-Removed (SEIR) model obtained for each city, a set of profile codes representing a variety of transmission mechanisms and contact topologies is formed. By comparing the data of an early outbreak of a given population with the complete set of historical profiles, the best fit profiles are selected and the corresponding sets of profile codes are used for prediction of the future progression of the epidemic in that population. Application of the method to the data collected for South Korea, Italy and Iran shows that peaks of infection cases are expected to occur before the end of March 2020, and that the percentage of population infected in each city will be less than 0.01%, 0.05% and 0.02%, for South Korea, Italy and Iran, respectively.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Scientific Collections in Scientific Preparedness</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">Building on the findings and recommendations of the Interagency Working Group on Scientific Collections, Scientific Collections International (SciColl) aims to improve the rapid access to science collections across disciplines within the federal government and globally, between government agencies and private research institutions. SciColl offered a novel opportunity for the US Department of Health and Human Services, Office of the Assistant Secretary for Preparedness and Response, to explore the value of scientific research collections under the science preparedness initiative and integrate it as a research resource at each stage in the emergence of the infectious diseases cycle. Under the leadership of SciColl’s executive secretariat at the Smithsonian Institution, and with multiple federal and international partners, a workshop during October 2014 fully explored the intersections of the infectious disease cycle and the role scientific collections could play as an evidentiary scientific resource to mitigate risks associated with emerging infectious diseases.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Developing international open science collaborations: Funder reflections on the Open Science Prize</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 1</span></i><p style="text-align: justify; text-justify: inter-word;">The Open Science Prize was established with the following objectives: first, to encourage the crowdsourcing of open data to make breakthroughs that are of biomedical significance; second, to illustrate that funders can indeed work together when scientific interests are aligned; and finally, to encourage international collaboration between investigators with the intent of achieving important innovations that would not be possible otherwise. The process for running the competition and the successes and challenges that arose are presented.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>SEED Servers: High-Performance Access to the SEED Genomes, Annotations, and Metabolic Models</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 24</span></i><p style="text-align: justify; text-justify: inter-word;">The remarkable advance in sequencing technology and the rising interest in medical and environmental microbiology, biotechnology, and synthetic biology resulted in a deluge of published microbial genomes. Yet, genome annotation, comparison, and modeling remain a major bottleneck to the translation of sequence information into biological knowledge, hence computational analysis tools are continuously being developed for rapid genome annotation and interpretation. Among the earliest, most comprehensive resources for prokaryotic genome analysis, the SEED project, initiated in 2003 as an integration of genomic data and analysis tools, now contains >5,000 complete genomes, a constantly updated set of curated annotations embodied in a large and growing collection of encoded subsystems, a derived set of protein families, and hundreds of genome-scale metabolic models. Until recently, however, maintaining current copies of the SEED code and data at remote locations has been a pressing issue. To allow high-performance remote access to the SEED database, we developed the SEED Servers (http://www.theseed.org/servers): four network-based servers intended to expose the data in the underlying relational database, support basic annotation services, offer programmatic access to the capabilities of the RAST annotation server, and provide access to a growing collection of metabolic models that support flux balance analysis. The SEED servers offer open access to regularly updated data, the ability to annotate prokaryotic genomes, the ability to create metabolic reconstructions and detailed models of metabolism, and access to hundreds of existing metabolic models. This work offers and supports a framework upon which other groups can build independent research efforts. Large integrations of genomic data represent one of the major intellectual resources driving research in biology, and programmatic access to the SEED data will provide significant utility to a broad collection of potential users.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>SEED Servers: High-Performance Access to the SEED Genomes, Annotations, and Metabolic Models</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 24</span></i><p style="text-align: justify; text-justify: inter-word;">The remarkable advance in sequencing technology and the rising interest in medical and environmental microbiology, biotechnology, and synthetic biology resulted in a deluge of published microbial genomes. Yet, genome annotation, comparison, and modeling remain a major bottleneck to the translation of sequence information into biological knowledge, hence computational analysis tools are continuously being developed for rapid genome annotation and interpretation. Among the earliest, most comprehensive resources for prokaryotic genome analysis, the SEED project, initiated in 2003 as an integration of genomic data and analysis tools, now contains >5,000 complete genomes, a constantly updated set of curated annotations embodied in a large and growing collection of encoded subsystems, a derived set of protein families, and hundreds of genome-scale metabolic models. Until recently, however, maintaining current copies of the SEED code and data at remote locations has been a pressing issue. To allow high-performance remote access to the SEED database, we developed the SEED Servers (http://www.theseed.org/servers): four network-based servers intended to expose the data in the underlying relational database, support basic annotation services, offer programmatic access to the capabilities of the RAST annotation server, and provide access to a growing collection of metabolic models that support flux balance analysis. The SEED servers offer open access to regularly updated data, the ability to annotate prokaryotic genomes, the ability to create metabolic reconstructions and detailed models of metabolism, and access to hundreds of existing metabolic models. This work offers and supports a framework upon which other groups can build independent research efforts. Large integrations of genomic data represent one of the major intellectual resources driving research in biology, and programmatic access to the SEED data will provide significant utility to a broad collection of potential users.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Analyzing the Epidemiological Outbreak of COVID-19: A Visual Exploratory Data Analysis (EDA) Approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">There is an obvious concern globally regarding the fact about the emerging coronavirus 2019-nCoV as a worldwide public health threat. As the outbreak of COVID-19 causes by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) progresses within China and beyond, rapidly available epidemiological data are needed to guide strategies for situational awareness and intervention. The recent outbreak of pneumonia in Wuhan, China caused by the SARS-CoV-2 emphasizes the importance of analyzing the epidemiological data of this novel virus and predicting their risks of infecting people all around the globe. In this article, we present an effort to compile and analyze epidemiological outbreak information on COVID-19 based on the several open datasets on Novel Corona Virus 2019 provided by the Johns Hopkins University, World Health Organization (WHO), Chinese Center for Disease Control and Prevention (CDC), National Health Commission (NHC), and DXY. An Exploratory Data Analysis (EDA) with visualizations has been made in order to understand the number of different cases reported (confirmed, death, recovered) in different provinces of China and outside of China. Overall, at the outset of an outbreak like this, it is highly important to readily provide information in order to begin the evaluation necessary to understand the risks and begin containment activities. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Deep learning Enables Accurate Diagnosis of Novel Coronavirus (COVID-19) with CT images</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background A novel coronavirus (COVID-19) has emerged recently as an acute respiratory syndrome. The outbreak was originally reported in Wuhan, China, but has subsequently been spread world-widely. As the COVID-19 continues to spread rapidly across the world, computed tomography (CT) has become essentially important for fast diagnoses. Thus, it is urgent to develop an accurate computer-aided method to assist clinicians to identify COVID-19-infected patients by CT images.     Materials and Methods We collected chest CT scans of 88 patients diagnosed with the COVID-19 from hospitals of two provinces in China, 101 patients infected with bacteria pneumonia, and 86 healthy persons for comparison and modeling. Based on the collected dataset, a deep learning-based CT diagnosis system (DeepPneumonia) was developed to identify patients with COVID-19.  Results The experimental results showed that our model can accurately identify the COVID-19 patients from others with an excellent AUC of 0.99 and recall (sensitivity) of 0.93. In addition, our model was capable of discriminating the COVID-19 infected patients and bacteria pneumonia-infected patients with an AUC of 0.95, recall (sensitivity) of 0.96. Moreover, our model could localize the main lesion features, especially the ground-glass opacity (GGO) that is of great help to assist doctors in diagnosis. The diagnosis for a patient could be finished in 30 seconds, and the implementation on Tianhe-2 supercompueter enables a parallel executions of thousands of tasks simultaneously. An online server is available for online diagnoses with CT images by http://biomed.nscc-gz.cn/server/Ncov2019.  Conclusions The established models can achieve a rapid and accurate identification of COVID-19 in human samples, thereby allowing identification of patients.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>World Health Organization declares Global Emergency: A review of the 2019 Novel Coronavirus (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An unprecedented outbreak of pneumonia of unknown aetiology in Wuhan City, Hubei province in China emerged in December of 2019. A novel coronavirus was identified as the causative agent and was subsequently termed COVID-19 by the World Health Organization (WHO). Considered a relative of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), COVID-19 is a betacoronavirus that affects the lower respiratory tract and manifests as pneumonia in humans. Despite rigorous global containment and quarantine efforts, the incidence of COVID-19 continues to rise, with 50,580 laboratory-confirmed cases and 1,526 deaths worldwide. In response to this global outbreak, we summarise the current state of knowledge surrounding COVID-19.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>GenomeFingerprinter: The Genome Fingerprint and the Universal Genome Fingerprint Analysis for Systematic Comparative Genomics</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: No attention has been paid on comparing a set of genome sequences crossing genetic components and biological categories with far divergence over large size range. We define it as the systematic comparative genomics and aim to develop the methodology. RESULTS: First, we create a method, GenomeFingerprinter, to unambiguously produce a set of three-dimensional coordinates from a sequence, followed by one three-dimensional plot and six two-dimensional trajectory projections, to illustrate the genome fingerprint of a given genome sequence. Second, we develop a set of concepts and tools, and thereby establish a method called the universal genome fingerprint analysis (UGFA). Particularly, we define the total genetic component configuration (TGCC) (including chromosome, plasmid, and phage) for describing a strain as a systematic unit, the universal genome fingerprint map (UGFM) of TGCC for differentiating strains as a universal system, and the systematic comparative genomics (SCG) for comparing a set of genomes crossing genetic components and biological categories. Third, we construct a method of quantitative analysis to compare two genomes by using the outcome dataset of genome fingerprint analysis. Specifically, we define the geometric center and its geometric mean for a given genome fingerprint map, followed by the Euclidean distance, the differentiate rate, and the weighted differentiate rate to quantitatively describe the difference between two genomes of comparison. Moreover, we demonstrate the applications through case studies on various genome sequences, giving tremendous insights into the critical issues in microbial genomics and taxonomy. CONCLUSIONS: We have created a method, GenomeFingerprinter, for rapidly computing, geometrically visualizing, intuitively comparing a set of genomes at genome fingerprint level, and hence established a method called the universal genome fingerprint analysis, as well as developed a method of quantitative analysis of the outcome dataset. These have set up the methodology of systematic comparative genomics based on the genome fingerprint analysis.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>The novel coronavirus (COVID-19) infection in Hangzhou: An experience to share</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Current situation in Hangzhou Hangzhou, the capital of Zhejiang province in China, was confronted with the pandemic of a novel coronavirus (COVID-19) that originated in Wuhan, Hubei province1 . According to the Health Commission of Zhejiang Province2 , six cases was first reported on January 19, 2020, and the cumulative cases reached 169 as of February 20, 2020. The situation in Hangzhou was once rather severe as it was once the top ranking city with respect to number of confirmed cases in Zhejiang province at the beginning of the epidemic. Since Hangzhou government took rigorous measures to contain the epidemic, positive trends have been seen. The daily number of newly confirmed cases has sharply decreased within the last week and there was only one confirmed case from February 17 to 20. Similarly, another point to be emphasized was that Hangzhou reported no deaths in its administrative region. We used a regression of log-incidence over time model3 ,which could provide a fitted trajectory for the actual daily incidence to verify the control effect. As show in Fig.1, the optimal splitting point, which was defined as the peak in number of daily new cases simulated by the model, occurred on January 25. That date was just about a week after launching the highest level of emergency public health alert and response in Hangzhou, which indicates that the prevention and control measures may be effective</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Base-By-Base: Single nucleotide-level analysis of whole viral genome alignments</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jul 14</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: With ever increasing numbers of closely related virus genomes being sequenced, it has become desirable to be able to compare two genomes at a level more detailed than gene content because two strains of an organism may share the same set of predicted genes but still differ in their pathogenicity profiles. For example, detailed comparison of multiple isolates of the smallpox virus genome (each approximately 200 kb, with 200 genes) is not feasible without new bioinformatics tools. RESULTS: A software package, Base-By-Base, has been developed that provides visualization tools to enable researchers to 1) rapidly identify and correct alignment errors in large, multiple genome alignments; and 2) generate tabular and graphical output of differences between the genomes at the nucleotide level. Base-By-Base uses detailed annotation information about the aligned genomes and can list each predicted gene with nucleotide differences, display whether variations occur within promoter regions or coding regions and whether these changes result in amino acid substitutions. Base-By-Base can connect to our mySQL database (Virus Orthologous Clusters; VOCs) to retrieve detailed annotation information about the aligned genomes or use information from text files. CONCLUSION: Base-By-Base enables users to quickly and easily compare large viral genomes; it highlights small differences that may be responsible for important phenotypic differences such as virulence. It is available via the Internet using Java Web Start and runs on Macintosh, PC and Linux operating systems with the Java 1.4 virtual machine.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Data on Hospital Environmental Hygiene Monitoring and Medical Staff Protection during the Coronavirus Disease 2019 Outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The outbreak of coronavirus disease 2019 (COVID-19) has placed unprecedented challenges on hospital environmental hygiene and medical staff protection. It is crucial to assess hospital environmental hygiene to understand the most important environmental issues for controlling the spread of COVID-19 in hospitals.  Objective: To detect the presence of COVID-19 in the samples from the area at risk of contamination in the First Hospital of Jilin University. Methods: Viruses in the air were collected by natural sedimentation and air particle sampler methods. Predetermined environmental surfaces were sampled using swabs at seven o'clock in the morning before disinfection. The real-time reverse-transcription PCR method was used to detect the existence of COVID-19 pathogens. Results: Viruses could be detected on the surfaces of the nurse station in the isolation area with suspected patients and in the air of the isolation ward with an intensive care patient. Conclusion: Comprehensive monitoring of hospital environmental hygiene during pandemic outbreaks is conducive to the refinement of hospital infection control. It is of great significance to ensure the safety of medical treatment and the quality of hospital infection control through the monitoring of environmental hygiene.</jats:p></p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundNew endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a "public-health emergency of international concern" due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.  MethodSeveral techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.  ResultsExtensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.  ConclusionT cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>News at a glance</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Just weeks after a novel coronavirus emerged in Wuhan, China, the outbreak continued to expand in gravity and scope. This week, the cumulative death toll surpassed 1000, compared with the more than 800 killed by another coronavirus in the 2002–03 outbreaks of severe acute respiratory syndrome (SARS). Authorities assigned official names to the virus, SARS-CoV-2, and the resulting disease, COVID-19. Li Wenliang, a 34-year-old doctor who sounded an early alarm in Wuhan about the disease only to be summoned by local police for doing so, succumbed to the virus last week.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Exploring the coronavirus epidemic using the new WashU Virus Genome Browser</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;">Since its debut in mid-December, 2019, the novel coronavirus (2019-nCoV) has rapidly spread from its origin in Wuhan, China, to several countries across the globe, leading to a global health crisis. As of February 7, 2020, 44 strains of the virus have been sequenced and uploaded to NCBIs GenBank [1], providing insight into the viruss evolutionary history and pathogenesis. Here, we present the WashU Virus Genome Browser, a web-based portal for viewing virus genomic data. The browser is home to 16 complete 2019-nCoV genome sequences, together with hundreds of related viral sequences including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and Ebola virus. In addition, the browser features unique customizability, supporting user-provided upload of novel viral sequences in various formats. Sequences can be viewed in both a track-based representation as well as a phylogenetic tree-based view, allowing the user to easily compare sequence features across multiple strains. The WashU Virus Genome Browser inherited many features and track types from the WashU Epigenome Browser, and additionally incorporated a new type of SNV track to address the specific needs of viral research. Our Virus Browser portal can be accessed at https://virusgateway.wustl.edu, and documentation is available at https://virusgateway.readthedocs.io/.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /></body>