
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: real-time tracking of whole COVID-19 genomes</h2><br /></div><br /><br /><br /><div class="card"><h3>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearmans rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease. AVAILABILITY: https://www.genomedetective.com/app/typingtool/cov. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Singapore claims first use of antibody test to track coronavirus infections | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In what appears to be a first, disease trackers in Singapore have used an experimental antibody test for COVID-19 to confirm that a suspected patient was infected with the coronavirus. The patient was one of two people who together formed a missing link between two clusters of cases that each occurred in a Singaporean church. Researchers around the world are racing to develop antibody tests, also called serological tests, that can confirm whether someone was infected even after their immune system has cleared the virus that causes COVID-19. The group that developed the test, at Duke-NUS Medical School in Singapore, is among the front-runners, although its assay has to be validated before it is taken into production and deployed widely.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Preprints bring ‘firehose’ of outbreak data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn't even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Cyberbiosecurity Challenges of Pathogen Genome Databases</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 15</span></i><p style="text-align: justify; text-justify: inter-word;">Pathogen detection, identification, and tracking is shifting from non-molecular methods, DNA fingerprinting methods, and single gene methods to methods relying on whole genomes. Viral Ebola and influenza genome data are being used for real-time tracking, while food-borne bacterial pathogen outbreaks and hospital outbreaks are investigated using whole genomes in the UK, Canada, the USA and the other countries. Also, plant pathogen genomes are starting to be used to investigate plant disease epidemics such as the wheat blast outbreak in Bangladesh. While these genome-based approaches provide never-seen advantages over all previous approaches with regard to public health and biosecurity, they also come with new vulnerabilities and risks with regard to cybersecurity. The more we rely on genome databases, the more likely these databases will become targets for cyber-attacks to interfere with public health and biosecurity systems by compromising their integrity, taking them hostage, or manipulating the data they contain. Also, while there is the potential to collect pathogen genomic data from infected individuals or agricultural and food products during disease outbreaks to improve disease modeling and forecast, how to protect the privacy of individuals, growers, and retailers is another major cyberbiosecurity challenge. As data become linkable to other data sources, individuals and groups become identifiable and potential malicious activities targeting those identified become feasible. Here, we define a number of potential cybersecurity weaknesses in today's pathogen genome databases to raise awareness, and we provide potential solutions to strengthen cyberbiosecurity during the development of the next generation of pathogen genome databases.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodias first case of COVID-2019.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Private collection: high correlation of sample collection and patient admission date in clinical microbiological testing complicates sharing of phylodynamic metadata</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 27</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious pathogens are known for their rapid evolutionary rates with new mutations arising over days to weeks. The ability to rapidly recover whole genome sequences and analyze the spread and evolution of pathogens using genetic information and pathogen collection dates has lead to interest in real-time tracking of infectious transmission and outbreaks. However, the level of temporal resolution afforded by these analyses may conflict with definitions of what constitutes protected health information (PHI) and privacy requirements for de-identification for publication and public sharing of research data and metadata. In the United States, dates and locations associated with patient care that provide greater resolution than year or the first three digits of the zip code are generally considered patient identifiers. Admission and discharge dates are specifically named as identifiers in Department of Health and Human Services guidance. To understand the degree to which one can impute admission dates from specimen collection dates, we examined sample collection dates and patient admission dates associated with more than 270,000 unique microbiological results from the University of Washington Laboratory Medicine Department between 2010 and 2017. Across all positive microbiological tests, the sample collection date exactly matched the patient admission date in 68.8% of tests. Collection dates and admission dates were identical from emergency department and outpatient testing 86.7% and 96.5% of the time, respectively, with >99% of tests collected within 1 day from the patient admission date. Samples from female patients were significantly more likely to be collected closer to admission date that those from male patients. We show that PHI-associated dates such as admission date can confidently be imputed from deposited collection date. We suggest that publicly depositing microbiological collection dates at greater resolution than the year may not meet routine Safe Harbor-based requirements for patient de-identification. We recommend the use of Expert Determination to determine PHI for a given study and/or direct patient consent if clinical laboratories or phylodynamic practitioners desire to make these data available.</p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;">SummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.  AvailabilityAvailable online: https://www.genomedetective.com/app/typingtool/cov  * Contactkoen@emweb.be and deoliveira@ukzn.ac.za  Supplementary informationSupplementary data is available online.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>‘A completely new culture of doing research.’ Coronavirus outbreak changes how scientists communicate | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 22 January, Dave O’Connor and Tom Friedrich invited several dozen colleagues around the United States to join a new workspace on the instant messaging platform Slack. The scientists, both at the Wisconsin National Primate Research Center, had seen news about a new disease emerging in China and realized researchers would need a primate model if they were going to answer some important questions about its biology. “We put out a call to a bunch of investigators and basically said: ‘Hey, let’s talk,’” O’Connor says. The idea is to coordinate research and make sure results are comparable, Friedrich adds. (They named the Slack workspace the Wu-han Clan, a play on the hip-hop group Wu-Tang Clan.) The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn’t even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>A proposal of an alternative primer for the ARTIC Network's multiplex PCR to improve coverage of SARS-CoV-2 genome sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole genome analysis of the novel corona virus, SARS-CoV-2, soon after the epidemics of this pathogen was revealed. The primer set seems to have been adapted already by many researchers worldwide and contributed for the high-quality and prompt genome epidemiology of this potential pandemic virus. We have also seen the great performance of their primer set and protocol; the primer set was able to amplify all desired PCR products with fairy small amplification bias from clinical samples with relatively high viral load. However, we observed acute drop of reads derived from two particular PCR products, 18 and 76, out of the 98 designated products as sample's viral load decreases. We suspected the reason for this low coverage issue was due to dimer formation between primers used to amplify those two PCR products. Here, we propose replacing just one of those primers, nCoV-2019_76_RIGHT(-), to a newly designed primer. The result of the replacement of primer showed improvement in coverage at both regions targeted by the products, 18 and 76. We expect this simple modification will extend the limit for whole SARS-CoV-2 genome analysis to samples with lower viral load and enhance genomic epidemiology of this pathogen.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of novel coronavirus’s spike protein solved</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">n a piece of research accomplished with astonishing speed, scientists from the University of Texas at Austin and the National Institutes of Health have released a cryo-electron microscopy (cryo-EM) structure of part of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the novel coronavirus that has infected tens of thousands of people and killed more than 2,000 since the end of December (Science 2020, DOI: 10.1126/science.abb2507). The part of the virus imaged, called the spike protein, helps the virus attach to and infect human cells, and the determination of its structure comes just weeks after the virus’s genome sequence was published. The breakthrough is a huge step toward developing a vaccine against the virus as well as treatments for COVID-19, the disease that it causes, the researchers say. UT Austin’s Jason McLellan and his colleagues, who have spent many years studying other coronaviruses, had already figured out how to use select</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Straining the System: Novel Coronavirus (COVID-19) and Preparedness for Concomitant Disasters</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Just a few weeks before the first confirmed case of novel coronavirus (COVID-19) was reported in the United States, the US Centers for Disease Control and Prevention (CDC) issued a bold promise to the nation: the agency will use its scientific expertise to bring a new level of preparedness in the United States and global health security against current and growing threats, finally eliminate certain diseases, and bring an end to the devastation of epidemics.(1) The current outbreak of COVID-19 reminds us how urgent this promise is and just how critical it is to continue to sustain and strengthen our nation's public health infrastructure. The unprecedented pace of the public health response to COVID-19 has only been possible because of prior investments in public health preparedness. To accelerate our pace and meet the challenges of current and future health threats, we must advance our world-class data and analytics capabilities; maintain and expand our state-of-the-art public health laboratory capacity; continue building a workforce of trusted, expert, public health professionals; sustain our capacity to rapidly respond to outbreaks at their source; and assure a strong global and domestic preparedness capacity. (Am J Public Health. Published online ahead of print February 13, 2020: e1-e2. doi:10.2105/AJPH.2020.305618).</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>With $115 million, more than 80 Boston researchers will collaborate to tackle COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A $115 million collaboration to tackle the rapidly spreading viral disease COVID-19, led by heavy hitters of Boston science and funded by a Chinese property development company, kicked off today as the group’s leaders pledged to take on the virus on many fronts. The project brings together researchers at many of the city’s top academic institutions, along with local biotechnology companies such as Moderna. Those leading it hope they can quickly funnel money into studies that will build off a new repository of samples from infected people and community surveillance, materials that can be rapidly shared among scientists. The project, they anticipate, should answer critical questions about how COVID-19 is spreading and how best to prevent and treat infections.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Unique epidemiological and clinical features of the emerging 2019 novel coronavirus pneumonia (COVID-19) implicate special control measures</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract By Feb 27th, 2020, the outbreak of COVID-19 caused 82623 confirmed cases and 2858 deaths globally, more than Severe Acute Respiratory Syndrome (SARS) (8273 cases, 775 deaths) and Middle East Respiratory Syndrome (MERS) (1139 cases, 431 deaths) caused in 2003 and 2013 respectively. COVID-19 has spread to 46 countries internationally. Total fatality rate of COVID-19 is estimated at 3.46% by far based on published data from Chinese Center for Disease Control and Prevention (China CDC). Average incubation period of COVID-19 is around 6.4 days, ranges from 0-24 days. The basic reproductive number (R0) of COVID-19 ranges from 2-3.5 at the early phase regardless of different prediction models, which is higher than SARS and MERS. A study from China CDC showed majority of patients (80.9%) were considered asymptomatic or mild pneumonia but released large amounts of viruses at the early phase of infection, which posed enormous challenges for containing the spread of COVID-19. Nosocomial transmission was another severe problem. 3019 health workers were infected by Feb 12, 2020, which accounted for 3.83% of total number of infections, and extremely burdened the health system, especially in Wuhan. Limited epidemiological and clinical data suggest that the disease spectrum of COVID-19 may differ from SARS or MERS. We summarize latest literatures on genetic, epidemiological, and clinical features of COVID-19 in comparison to SARS and MERS and emphasize special measures on diagnosis and potential interventions. This review will improve our understanding of the unique features of COVID-19 and enhance our control measures in the future. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Mystery deepens over animal source of coronavirus Podcast Extra: ‘There is lots of anxiety’: a scientist’s view from South Korea Coronavirus latest: Brazil reports first case in South America</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Pangolins are a prime suspect, but a slew of genetic analyses has yet to find conclusive proof. Pangolins are a prime suspect, but a slew of genetic analyses has yet to find conclusive proof. Nick Howe speaks to chemist Bartosz Grzybowski about his experience with the coronavirus outbreak in South Korea. Nick Howe speaks to chemist Bartosz Grzybowski about his experience with the coronavirus outbreak in South Korea. A case of COVID-19 has been confirmed in Brazil — the first in South America. On 26 February, Brazil’s minister of health, Luiz Henrique Mandetta, confirmed that a man who traveled to northern Italy between 9 and 21 February has the disease. Italy’s outbreak has escalated to 324 cases and 12 deaths, according to a virus tracker maintained by Johns Hopkins University in Baltimore, Maryland.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>The effect of human mobility and control measures on the COVID-19 epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing COVID-19 outbreak has expanded rapidly throughout China. Major behavioral, clinical, and state interventions are underway currently to mitigate the epidemic and prevent the persistence of the virus in human populations in China and worldwide. It remains unclear how these unprecedented interventions, including travel restrictions, have affected COVID-19 spread in China. We use real-time mobility data from Wuhan and detailed case data including travel history to elucidate the role of case importation on transmission in cities across China and ascertain the impact of control measures. Early on, the spatial distribution of COVID-19 cases in China was well explained by human mobility data. Following the implementation of control measures, this correlation dropped and growth rates became negative in most locations, although shifts in the demographics of reported cases are still indicative of local chains of transmission outside Wuhan. This study shows that the drastic control measures implemented in China have substantially mitigated the spread of COVID-19.</jats:p></p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>(18)F-FDG PET/CT findings of COVID-19: a series of four highly suspected cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">PURPOSE: The aim of this case series is to illustrate the (18)F-FDG PET/CT findings of patients with acute respiratory disease caused by COVID-19 in Wuhan, Hubei province of China. METHODS: We describe the (18)F-FDG PET/CT results from four patients who were admitted to the hospital with respiratory symptoms and fever between January 13 and January 20, 2020, when the COVID-19 outbreak was still unrecognized and the virus infectivity was unknown. A retrospective review of the patients' medical history, clinical and laboratory data, as well as imaging findings strongly suggested a diagnosis of COVID-19. RESULTS: All patients had peripheral ground-glass opacities and/or lung consolidations in more than two pulmonary lobes. Lung lesions were characterized by a high (18)F-FDG uptake and there was evidence of lymph node involvement. Conversely, disseminated disease was absent, a finding suggesting that COVID-19 has pulmonary tropism. CONCLUSIONS: Although (18)F-FDG PET/CT cannot be routinely used in an emergency setting and is generally not recommended for infectious diseases, our pilot data shed light on the potential clinical utility of this imaging technique in the differential diagnosis of complex cases.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>2019 novel coronavirus is undergoing active recombination</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019 novel coronavirus (2019-nCoV) outbreak in Wuhan since December 2019 [1] has quickly spread to twenty-five countries [2], caused more than 44,000 cases and 1,000 deaths so far [3]. The fast sharing of 2019-nCoV genomes in GISAID (https://www.gisaid.org) provides a valuable dataset for 2019-nCoV haplotype network analysis, which is crucial for transmission and evolutionary track surveillance and secondary outbreak prevention. We called single-nucleotide variations (SNVs) for all the 84 2019-nCov genomes in GISAID using MN908947 as the reference. Genomes with same SNVs are grouped into a haplotype (shown as a pie chart node in Fig. 1, see Tab. S1 for the accessions), then the network is constructed using the median join [4] method in popART [5]. We found the 2019-nCoV haplotype network has obvious characteristics of single origin from haplotype hap_011: first, the network is star-like, centralized on the haplotype hap_011; second, hap_011 has the largest sample size and majority of the samples are from Hubei province—where outbreak originated (Tab. S1); third, most of satellite haplotypes are also from Hubei (Fig. 1); fourth, the average collection dates of hap_011 (has 0 mutation relative to MN908947) is earlier than all other mutation groups (Fig. S1). The single origin of 2019-nCov indicates a persistent animal to human transmission is unlikely, otherwise, multiple nodes with above characteristics should be observed.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Sequence analyses of RNA virus genomes remain challenging owing to the exceptional genetic plasticity of these viruses. Because of high mutation and recombination rates, genome replication by viral RNA-dependent RNA polymerases leads to populations of closely related viruses, so-called “quasispecies.” Standard (short-read) sequencing technologies are ill-suited to reconstruct large numbers of full-length haplotypes of (1) RNA virus genomes and (2) subgenome-length (sg) RNAs composed of noncontiguous genome regions. Here, we used a full-length, direct RNA sequencing (DRS) approach based on nanopores to characterize viral RNAs produced in cells infected with a human coronavirus. By using DRS, we were able to map the longest (∼26-kb) contiguous read to the viral reference genome. By combining Illumina and Oxford Nanopore sequencing, we reconstructed a highly accurate consensus sequence of the human coronavirus (HCoV)-229E genome (27.3 kb). Furthermore, by using long reads that did not require an assembly step, we were able to identify, in infected cells, diverse and novel HCoV-229E sg RNAs that remain to be characterized. Also, the DRS approach, which circumvents reverse transcription and amplification of RNA, allowed us to detect methylation sites in viral RNAs. Our work paves the way for haplotype-based analyses of viral quasispecies by showing the feasibility of intra-sample haplotype separation. Even though several technical challenges remain to be addressed to exploit the potential of the nanopore technology fully, our work illustrates that DRS may significantly advance genomic studies of complex virus populations, including predictions on long-range interactions in individual full-length viral RNA haplotypes.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Dynamics Simulations Indicate the COVID-19 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed molecular dynamics simulations of the main protease (Mpro). We compared and contrasted the Mpro for COVID-19 with a highly similar SARS protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the pockets time-dependence indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus mutability will pose a further challenge to the rational design of small-molecule inhibitors.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>The United States badly bungled coronavirus testing—but things may soon improve | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? The World Health Organization (WHO) has shipped testing kits to 57 countries. China had five commercial tests on the market 1 month ago and can now do up to 1.6 million tests a week; South Korea has tested 65,000 people so far. The U. S. Centers for Disease Control and Prevention (CDC), in contrast, has done only 459 tests since the epidemic began. The rollout of a CDC-designed test kit to state and local labs has become a fiasco because it contained a faulty reagent. Labs around the country eager to test more suspected cases—and test them faster—have been unable to do so. No commercial or state labs have the approval to use their own tests. In what is already an infamous snafu, CDC initially refused a request to test a patient in Northern California who turned out to be the first probable COVID19 case without known links to an infected person.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">1The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Generation of antibodies against COVID-19 virus for development of diagnostic tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Exploring the coronavirus epidemic using the new WashU Virus Genome Browser</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-11</span></i><p style="text-align: justify; text-justify: inter-word;">Since its debut in mid-December, 2019, the novel coronavirus (2019-nCoV) has rapidly spread from its origin in Wuhan, China, to several countries across the globe, leading to a global health crisis. As of February 7, 2020, 44 strains of the virus have been sequenced and uploaded to NCBIs GenBank [1], providing insight into the viruss evolutionary history and pathogenesis. Here, we present the WashU Virus Genome Browser, a web-based portal for viewing virus genomic data. The browser is home to 16 complete 2019-nCoV genome sequences, together with hundreds of related viral sequences including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and Ebola virus. In addition, the browser features unique customizability, supporting user-provided upload of novel viral sequences in various formats. Sequences can be viewed in both a track-based representation as well as a phylogenetic tree-based view, allowing the user to easily compare sequence features across multiple strains. The WashU Virus Genome Browser inherited many features and track types from the WashU Epigenome Browser, and additionally incorporated a new type of SNV track to address the specific needs of viral research. Our Virus Browser portal can be accessed at https://virusgateway.wustl.edu, and documentation is available at https://virusgateway.readthedocs.io/.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Computers and viral diseases. Preliminary bioinformatics studies on the design of a synthetic vaccine and a preventative peptidomimetic antagonist against the SARS-CoV-2 (2019-nCoV, COVID-19) coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>The epidemiology and pathogenesis of coronavirus disease (COVID-19) outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease (COVID-19) is caused by SARS-COV2 and represents the causative agent of a potentially fatal disease that is of great global public health concern. Based on the large number of infected people that were exposed to the wet animal market in Wuhan City, China, it is suggested that this is likely the zoonotic origin of COVID-19. Person-to-person transmission of COVID-19 infection led to the isolation of patients that were subsequently administered a variety of treatments. Extensive measures to reduce person-to-person transmission of COVID-19 have been implemented to control the current outbreak. Special attention and efforts to protect or reduce transmission should be applied in susceptible populations including children, health care providers, and elderly people. In this review, we highlights the symptoms, epidemiology, transmission, pathogenesis, phylogenetic analysis and future directions to control the spread of this fatal disease.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Closed environments facilitate secondary transmission of coronavirus disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Commissioned by the Minister of the Ministry of Health, Labour, and Welfare of Japan, we collected secondary transmission data with the aim of identifying high risk transmission settings. We show that closed environments contribute to secondary transmission of COVID-19 and promote superspreading events. Closed environments are consistent with large-scale COVID-19 transmission events such as that of the ski chalet-associated cluster in France and the church- and hospital-associated clusters in South Korea. Our findings are also consistent with the declining incidence of COVID-19 cases in China, as gathering in closed environments was prohibited in the wake of the rapid spread of the disease. Reduction of unnecessary close contact in closed environments may help prevent large case clusters and superspreading events.</jats:p></p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-08-14</span></i><p style="text-align: justify; text-justify: inter-word;">Sequence analyses of RNA virus genomes remain challenging due to the exceptional genetic plasticity of these viruses. Because of high mutation and recombination rates, genome replication by viral RNA-dependent RNA polymerases leads to populations of closely related viruses, so-called  quasispecies. Standard (short-read) sequencing technologies are ill-suited to reconstruct large numbers of full-length haplotypes of (i) RNA virus genomes and (ii) subgenome-length (sg) RNAs comprised of noncontiguous genome regions. Here, we used a full-length, direct RNA sequencing (DRS) approach based on nanopores to characterize viral RNAs produced in cells infected with a human coronavirus.\n\nUsing DRS, we were able to map the longest ([~]26 kb) contiguous read to the viral reference genome. By combining Illumina and nanopore sequencing, we reconstructed a highly accurate consensus sequence of the human coronavirus (HCoV) 229E genome (27.3 kb). Furthermore, using long reads that did not require an assembly step, we were able to identify, in infected cells, diverse and novel HCoV-229E sg RNAs that remain to be characterized. Also, the DRS approach, which circumvents reverse transcription and amplification of RNA, allowed us to detect methylation sites in viral RNAs. Our work paves the way for haplotype-based analyses of viral quasispecies by demonstrating the feasibility of intra-sample haplotype separation.\n\nEven though several technical challenges remain to be addressed to exploit the potential of the nanopore technology fully, our work illustrates that direct RNA sequencing may significantly advance genomic studies of complex virus populations, including predictions on long-range interactions in individual full-length viral RNA haplotypes.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>What we know so far: COVID-19 current clinical knowledge and research</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, health authorities in Wuhan, China, identified a cluster of pneumonia cases of unknown aetiology linked to the city's South China Seafood Market. Subsequent investigations revealed a novel coronavirus, SARS-CoV-2, as the causative agent now at the heart of a major outbreak. The rising case numbers have been accompanied by unprecedented public health action, including the wholesale isolation of Wuhan. Alongside this has been a robust scientific response, including early publication of the pathogen genome, and rapid development of highly specific diagnostics. This article will review the new knowledge of SARS-CoV-2 COVID-19 acute respiratory disease, and summarise its clinical features.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Angiotensin-converting enzyme 2 (ACE2) is the cellular receptor for SARS coronavirus (SARS-CoV) and the new coronavirus (SARS-CoV-2) that is causing the serious epidemic COVID-19. Here we present cryo-EM structures of full-length human ACE2, in the presence of a neutral amino acid transporter B(0)AT1, with or without the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) of SARS-CoV-2, both at an overall resolution of 2.9 A, with a local resolution of 3.5 A at the ACE2-RBD interface. The ACE2-B(0)AT1 complex is assembled as a dimer of heterodimers, with the Collectrin-like domain (CLD) of ACE2 mediating homo-dimerization. The RBD is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. These findings provide important insights to the molecular basis for coronavirus recognition and infection.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the world is witnessing the epidemic of COVID-19, a disease caused by a novel coronavirus, SARS-CoV-2, emerging genetics and clinical evidences suggest a similar path to those of SARS and MERS. The rapid genomic sequencing and open access data, together with advanced vaccine technology, are expected to give us more knowledge on the pathogen itself, including the host immune response as well as the plan for therapeutic vaccines in the near future. This review aims to provide a comparative view among SARS-CoV, MERS-CoV and the newly epidemic SARS-CoV-2, in the hope to gain a better understanding of the host-pathogen interaction, host immune responses, and the pathogen immune evasion strategies. This predictive view may help in designing an immune intervention or preventive vaccine for COVID-19 in the near future.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural basis for the recognition of the 2019-nCoV by human ACE2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">Angiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 (COVID-19). Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B0AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 [A] in the presence of B0AT1. The local resolution at the ACE2-RBD interface is 3.5 [A], allowing analysis of the detailed interactions between the RBD and the receptor. Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>[Advances on presymptomatic or asymptomatic carrier transmission of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is rapidly spreading. Patients in incubation period and healthy carriers are possible sources for transmission. However, such sources of infection cannot be effectively identified due to the symptoms absent. The research evidence is very lacking so far, although there are a few studies suggesting that presymptomatic or asymptomatic carrier may cause COVID-19 transmission. Nearly half of the literature is in the state of preprint without peer review. The question of "the degree to which presymptomatic or asymptomatic infections can transmit" is not fully understood. There is an urgent need to screen infected carriers in larger close contacts or in the general population, and assess their risk for transmission.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>The role of absolute humidity on transmission rates of the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel coronavirus (COVID-19) was identified in Wuhan, Hubei Province, China, in December 2019 and has caused over 40,000 cases worldwide to date. Previous studies have supported an epidemiological hypothesis that cold and dry (low absolute humidity) environments facilitate the survival and spread of droplet-mediated viral diseases, and warm and humid (high absolute humidity) environments see attenuated viral transmission (i.e., influenza). However, the role of absolute humidity in transmission of COVID-19 has not yet been established. Here, we examine province-level variability of the basic reproductive numbers of COVID-19 across China and find that changes in weather alone (i.e., increase of temperature and humidity as spring and summer months arrive in the North Hemisphere) will not necessarily lead to declines in COVID-19 case counts without the implementation of extensive public health interventions.</jats:p></p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus (COVID-19) Outbreak: What the Department of Radiology Should Know</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, a novel coronavirus (COVID-19) pneumonia emerged in Wuhan, China. Since then, this highly contagious COVID-19 has been spreading worldwide, with a rapid rise in the number of deaths. Novel COVID-19-infected pneumonia (NCIP) is characterized by fever, fatigue, dry cough, and dyspnea. A variety of chest imaging features have been reported, similar to those found in other types of COVID-19 syndromes. The purpose of the present review is to briefly discuss the known epidemiology and the imaging findings of COVID-19 syndromes, with a focus on the reported imaging findings of NCIP. Moreover, the authors review precautions and safety measures for radiology department personnel to manage patients with known or suspected NCIP. Implementation of a robust plan in the radiology department is required to prevent further transmission of the virus to patients and department staff members.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission characteristics of the COVID-19 outbreak in China: a study driven by data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-01</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days.  It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.</jats:p></p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Fecal specimen diagnosis 2019 Novel Coronavirus-Infected Pneumonia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and spread of 2019 Novel Coronavirus-Infected Pneumonia (COVID-19) from Wuhan, China, it has spread globally. We extracted the data on 14 patients with laboratory-confirmed COVID-19 from Jinhua Municipal Central hospital through January 27th, 2020. We found that compared to pharyngeal swab specimens, nucleic acid detection of COVID-19 in fecal specimens was equally accurate. And we found that patients with a positive stool test did not experience gastrointestinal symptoms and had nothing to do with the severity of the lung infection. These results may help to understand the clinical diagnosis and the changes of clinical parameters of COVID-19. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.</jats:p></p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Deep learning-based model for detecting 2019 novel coronavirus pneumonia on high-resolution computed tomography: a prospective study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Computed tomography (CT) is the preferred imaging method for diagnosing 2019 novel coronavirus (COVID19) pneumonia. Our research aimed to construct a system based on deep learning for detecting COVID-19 pneumonia on high resolution CT, relieve working pressure of radiologists and contribute to the control of the epidemic. Methods: For model development and validation, 46,096 anonymous images from 106 admitted patients, including 51 patients of laboratory confirmed COVID-19 pneumonia and 55 control patients of other diseases in Renmin Hospital of Wuhan University (Wuhan, Hubei province, China) were retrospectively collected and processed. Twenty-seven consecutive patients undergoing CT scans in Feb, 5, 2020 in Renmin Hospital of Wuhan University were prospectively collected to evaluate and compare the efficiency of radiologists against 2019-CoV pneumonia with that of the model. Findings: The model achieved a per-patient sensitivity of 100%, specificity of 93.55%, accuracy of 95.24%, PPV of 84.62%, and NPV of 100%; a per-image sensitivity of 94.34%, specificity of 99.16%, accuracy of 98.85%, PPV of 88.37%, and NPV of 99.61% in retrospective dataset. For 27 prospective patients, the model achieved a comparable performance to that of expert radiologist. With the assistance of the model, the reading time of radiologists was greatly decreased by 65%. Conclusion: The deep learning model showed a comparable performance with expert radiologist, and greatly improve the efficiency of radiologists in clinical practice. It holds great potential to relieve the pressure of frontline radiologists, improve early diagnosis, isolation and treatment, and thus contribute to the control of the epidemic.</jats:p></p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>The race to unravel the United States’ biggest coronavirus outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rohit Shankar left the virology laboratory at 2 a.m. on Wednesday, and was back at the lab bench by 7 a.m. the same day. “It’s okay,” he says, “I had a doughnut and a coffee.” Shankar, a medical scientist, and his colleagues at the University of Washington in Seattle are poised to exponentially drive up the number of confirmed cases of the coronavirus disease COVID-19 around the city, in western Washington state. That’s because this week, they began analysing a mountain of nose and throat swabs collected from hospitals in the region. Already, the researchers are seeing clear signs that the virus has infected vastly more people than have been formally detected. Scientists fear coronavirus spread in countries least able to contain it Washington state has become the United States’ ground zero for COVID-19, which has now spread to more than 90 countries worldwide in what seems to be a new and dangerous phase of the outbreak. Washington has declared a state emergency, and ten people there have died from the disease. But the number of confirmed cases in Washington — 70 — is still an underestimate resulting from a lack of testing, researchers agree. A genomic analysis posted online on 29 February suggested that hundreds of people in western Washington might already be infected. Academic scientists have mostly been prevented from measuring the extent of the US outbreak because of federal rules restricting the number of labs qualified to run diagnostic tests. But that is changing now, and helps account for why the state's caseload jumped from 10 to 70 this week. Dozens of virologists and genomicists have now kicked into high gear in Seattle, dropping or adapting projects to devote resources to the outbreak. Researchers are working around the clock to find out how many people have the disease in the area. Others are analysing genomes to reveal how the virus is transmitted or developing new therapies. The scientists are racing to help Washington avoid the fate of Hubei province in China, where more than 2,900 people have died of COVID-19 so far. The coronavirus emerged in the province’s city Wuhan in December, and the initial response from officials was slow. “We are past the point of containment,” says Helen Chu, an infectious-disease specialist at the University of Washington School of Medicine (UW Medicine) in Seattle. “So now we need to keep the people who are vulnerable from getting sick.”</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-2019: the role of the nsp2 and nsp3 in its pathogenesis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Last December 2019, a new virus, named COVID-2019 causing many cases of severe pneumonia was reported in Wuhan, China. The virus knowledge is limited and especially about COVID-2019 pathogenesis. The Open Reading Frame 1ab (ORF1ab) of COVID-2019 has been analyzed to evidence the presence of mutation caused by selective pressure on the virus. For selective pressure analysis fast-unconstrained Bayesian approximation (FUBAR) was used. Homology modelling has been performed by SwissModel and HHPred servers. The presence of transmembrane helical segments in Coronavirus ORF1ab nsp2 and nsp3, was tested by TMHMM, MEMSAT and MEMPACK tools. Three-dimensional structures have been analyzed and displayed using PyMOL. FUBAR analysis revealed the presence of potential sites under positive selective pressure (p-value < 0.05). Position 723 in the COVID-2019 has a serine instead a glycine residue, while at aminoacidic position 1010 a proline instead an isoleucine. Significant (p < 0.05) pervasive negative selection in 2416 sites (55%) was found. The positive selective pressure could account for some clinical features of this virus compared to SARS and Bat SARS-like CoV. The stabilizing mutation falling in the endosome-associated-protein-like domain of the nsp2 protein could account for COVID-2019 high ability of contagious, while the destabilizing mutation in nsp3 proteins could suggest a potential mechanism differentiating COVID-2019 from SARS. These data could be helpful for further investigation aimed to identify potential therapeutic targets or vaccine strategy, especially in the actual moment when the epidemic is ongoing and the scientific community is trying to enrich knowledge about this new viral pathogen. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Capturing sequence diversity in metagenomes with comprehensive and scalable probe design</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 4</span></i><p style="text-align: justify; text-justify: inter-word;">Metagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. Here we present CATCH, a computational method to enhance nucleic-acid capture for enrichment of diverse microbial taxa. CATCH designs optimal probe sets, with a specified number of oligonucleotides, that achieve full coverage of and scale well with known sequence diversity. We focus on applying CATCH to capture viral genomes in complex metagenomic samples. We design, synthesize, and validate multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriches unique viral content on average 18-fold, allowing us to assemble genomes that could not be recovered without enrichment, and accurately preserves within-sample diversity. We also use these probe sets to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of uncharacterized viral infections in human and mosquito samples. The results demonstrate that CATCH enables more sensitive and cost-effective metagenomic sequencing.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Capturing sequence diversity in metagenomes with comprehensive and scalable probe design</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 4</span></i><p style="text-align: justify; text-justify: inter-word;">Metagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. Here we present CATCH, a computational method to enhance nucleic-acid capture for enrichment of diverse microbial taxa. CATCH designs optimal probe sets, with a specified number of oligonucleotides, that achieve full coverage of and scale well with known sequence diversity. We focus on applying CATCH to capture viral genomes in complex metagenomic samples. We design, synthesize, and validate multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriches unique viral content on average 18-fold, allowing us to assemble genomes that could not be recovered without enrichment, and accurately preserves within-sample diversity. We also use these probe sets to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of uncharacterized viral infections in human and mosquito samples. The results demonstrate that CATCH enables more sensitive and cost-effective metagenomic sequencing.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Capturing sequence diversity in metagenomes with comprehensive and scalable probe design</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 4</span></i><p style="text-align: justify; text-justify: inter-word;">Metagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. Here we present CATCH, a computational method to enhance nucleic-acid capture for enrichment of diverse microbial taxa. CATCH designs optimal probe sets, with a specified number of oligonucleotides, that achieve full coverage of and scale well with known sequence diversity. We focus on applying CATCH to capture viral genomes in complex metagenomic samples. We design, synthesize, and validate multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriches unique viral content on average 18-fold, allowing us to assemble genomes that could not be recovered without enrichment, and accurately preserves within-sample diversity. We also use these probe sets to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of uncharacterized viral infections in human and mosquito samples. The results demonstrate that CATCH enables more sensitive and cost-effective metagenomic sequencing.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Capturing sequence diversity in metagenomes with comprehensive and scalable probe design</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 4</span></i><p style="text-align: justify; text-justify: inter-word;">Metagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. Here we present CATCH, a computational method to enhance nucleic-acid capture for enrichment of diverse microbial taxa. CATCH designs optimal probe sets, with a specified number of oligonucleotides, that achieve full coverage of and scale well with known sequence diversity. We focus on applying CATCH to capture viral genomes in complex metagenomic samples. We design, synthesize, and validate multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriches unique viral content on average 18-fold, allowing us to assemble genomes that could not be recovered without enrichment, and accurately preserves within-sample diversity. We also use these probe sets to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of uncharacterized viral infections in human and mosquito samples. The results demonstrate that CATCH enables more sensitive and cost-effective metagenomic sequencing.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential T-cell and B-cell Epitopes of 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-21</span></i><p style="text-align: justify; text-justify: inter-word;">As of Feb 16th 2020, 2019-nCoV has infected more than 51,857 people across 26 countries and claimed 1666 lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for 2019-nCoV or any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. We validated our computational pipeline with SARS-CoV experimental data.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Reconsideration on the multiple value of Behavior determining Health: in the perspective of the situation of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Why has the concept of behavior determining health created more and more extensive and far-reaching influence ever since it was put forward? The reason lies in its multiple values. It is of great practical significance and has important implications for long-term health care to explore and analyze in the perspective of the situation of COVID-19 its philosophical values, cultural values, methodological values, social values and the national strategic value of 'healthy China'.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Why airport screening won’t stop the spread of coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">If you have traveled internationally the past 2 months, you may have encountered them: health officers briefly pointing a thermometer gun at your forehead or watching as you go by to check for signs of a cough or difficulty breathing. Many countries are now watching arriving and departing air passengers who might suffer from the viral disease COVID-19; some require passengers to fill out health declarations. (Some also simply ban or quarantine those who have recently been in outbreak hot spots.) Exit and entry screening may look reassuring, but experience with other diseases shows it’s exceedingly rare for screeners to detect infected passengers. Just last week, eight passengers who later tested positive for COVID-19 arrived in Shanghai from Italy and passed the airport screeners unnoticed, for example. And even if screeners do find the occasional case, it has almost no impact on the course of an outbreak.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterizing the transmission and identifying the control strategy for COVID-19 through epidemiological modeling</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended.</jats:p></p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.</jats:p></p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Maybe not an overreaction</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A mathematical simulation of coronavirus COVID-19 estimated the number of infections to be far worse than reported. Starting in December 2019, a novel coronavirus, known as COVID-19, caused growing cases of atypical pneumonia in the city of Wuhan, China. The virus spread. As of 12 February 2020, a total of 45,204 confirmed cases had been reported from China and 519 confirmed cases from 27 countries worldwide. Among the reported victims, 1116 died. When facing an epidemic as such, a dangerous mistake that countries over the world can make is to rely entirely on confirmed data but underestimate the actual size of infections. Causes of underestimation include the limitation of resources, as there can be insufficient quarantine spaces, detection reagents, and medical personnel to identify infections in real time. Additionally, the lack of symptoms from the virus in its dormant state can delay confirmation. There is also a fear of data being manipulated or downplayed by officials due to economic and political concerns.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Asymptomatic novel coronavirus pneumonia patient outside WuHan: The value of CT images in the course of the disease</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The purpose of this case report is to describe the imaging and associated clinical features of an asymptomatic novel coronavirus pneumonia (COVID-19) patient outside WuHan, China. The principle findings are that in this patient with laboratory-confirmed COVID-19, CT findings preceded symptoms and included bilateral pleural effusions, previously not reported in association with COVID-19. The role of this case report is promotion of potential recognition amongst radiologists of this new disease, which has been declared a global health emergency by the World Health Organization (WHO).</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Time to use the p-word? Coronavirus enters dangerous new phase</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As outbreaks surge worldwide, scientists fear that COVID-19 might soon become pandemic. As outbreaks surge worldwide, scientists fear that COVID-19 might soon become pandemic.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Pathways of Foot-and-Mouth Disease Virus in the United Kingdom in 2007</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Apr 18</span></i><p style="text-align: justify; text-justify: inter-word;">Foot-and-mouth disease (FMD) virus causes an acute vesicular disease of domesticated and wild ruminants and pigs. Identifying sources of FMD outbreaks is often confounded by incomplete epidemiological evidence and the numerous routes by which virus can spread (movements of infected animals or their products, contaminated persons, objects, and aerosols). Here, we show that the outbreaks of FMD in the United Kingdom in August 2007 were caused by a derivative of FMDV O(1) BFS 1860, a virus strain handled at two FMD laboratories located on a single site at Pirbright in Surrey. Genetic analysis of complete viral genomes generated in real-time reveals a probable chain of transmission events, predicting undisclosed infected premises, and connecting the second cluster of outbreaks in September to those in August. Complete genome sequence analysis of FMD viruses conducted in real-time have identified the initial and intermediate sources of these outbreaks and demonstrate the value of such techniques in providing information useful to contemporary disease control programmes.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Pathways of Foot-and-Mouth Disease Virus in the United Kingdom in 2007</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Apr 18</span></i><p style="text-align: justify; text-justify: inter-word;">Foot-and-mouth disease (FMD) virus causes an acute vesicular disease of domesticated and wild ruminants and pigs. Identifying sources of FMD outbreaks is often confounded by incomplete epidemiological evidence and the numerous routes by which virus can spread (movements of infected animals or their products, contaminated persons, objects, and aerosols). Here, we show that the outbreaks of FMD in the United Kingdom in August 2007 were caused by a derivative of FMDV O(1) BFS 1860, a virus strain handled at two FMD laboratories located on a single site at Pirbright in Surrey. Genetic analysis of complete viral genomes generated in real-time reveals a probable chain of transmission events, predicting undisclosed infected premises, and connecting the second cluster of outbreaks in September to those in August. Complete genome sequence analysis of FMD viruses conducted in real-time have identified the initial and intermediate sources of these outbreaks and demonstrate the value of such techniques in providing information useful to contemporary disease control programmes.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Pathways of Foot-and-Mouth Disease Virus in the United Kingdom in 2007</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Apr 18</span></i><p style="text-align: justify; text-justify: inter-word;">Foot-and-mouth disease (FMD) virus causes an acute vesicular disease of domesticated and wild ruminants and pigs. Identifying sources of FMD outbreaks is often confounded by incomplete epidemiological evidence and the numerous routes by which virus can spread (movements of infected animals or their products, contaminated persons, objects, and aerosols). Here, we show that the outbreaks of FMD in the United Kingdom in August 2007 were caused by a derivative of FMDV O(1) BFS 1860, a virus strain handled at two FMD laboratories located on a single site at Pirbright in Surrey. Genetic analysis of complete viral genomes generated in real-time reveals a probable chain of transmission events, predicting undisclosed infected premises, and connecting the second cluster of outbreaks in September to those in August. Complete genome sequence analysis of FMD viruses conducted in real-time have identified the initial and intermediate sources of these outbreaks and demonstrate the value of such techniques in providing information useful to contemporary disease control programmes.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in ‘lockdown’ directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation’s economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and advances how smart city networks should work towards enhancing standardization protocols for increased data sharing in the event of outbreaks or disasters, leading to better global understanding and management of the same.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Pathogen surveillance in the informal settlement, Kibera, Kenya, using a metagenomics approach</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Oct 10</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Worldwide, the number of emerging and re-emerging infectious diseases is increasing, highlighting the importance of global disease pathogen surveillance. Traditional population-based methods may fail to capture important events, particularly in settings with limited access to health care, such as urban informal settlements. In such environments, a mixture of surface water runoff and human feces containing pathogenic microorganisms could be used as a surveillance surrogate. METHOD: We conducted a temporal metagenomic analysis of urban sewage from Kibera, an urban informal settlement in Nairobi, Kenya, to detect and quantify bacterial and associated antimicrobial resistance (AMR) determinants, viral and parasitic pathogens. Data were examined in conjunction with data from ongoing clinical infectious disease surveillance. RESULTS: A large variation of read abundances related to bacteria, viruses, and parasites of medical importance, as well as bacterial associated antimicrobial resistance genes over time were detected. Significant increased abundances were observed for a number of bacterial pathogens coinciding with higher abundances of AMR genes. Vibrio cholerae as well as rotavirus A, among other virus peaked in several weeks during the study period whereas Cryptosporidium spp. and Giardia spp, varied more over time. CONCLUSION: The metagenomic surveillance approach for monitoring circulating pathogens in sewage was able to detect putative pathogen and resistance loads in an urban informal settlement. Thus, valuable if generated in real time to serve as a comprehensive infectious disease agent surveillance system with the potential to guide disease prevention and treatment. The approach may lead to a paradigm shift in conducting real-time global genomics-based surveillance in settings with limited access to health care.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>[Potential false-positive rate among the 'asymptomatic infected individuals' in close contacts of COVID-19 patients]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objective: As the prevention and control of COVID-19continues to advance, the active nucleic acid test screening in the close contacts of the patients has been carrying out in many parts of China. However, the false-positive rate of positive results in the screening has not been reported up to now. But to clearify the false-positive rate during screening is important in COVID-19 control and prevention. Methods: Point values and reasonable ranges of the indicators which impact the false-positive rate of positive results were estimated based on the information available to us at present. The false-positive rate of positive results in the active screening was deduced, and univariate and multivariate-probabilistic sensitivity analyses were performed to understand the robustness of the findings. Results: When the infection rate of the close contacts and the sensitivity and specificity of reported results were taken as the point estimates, the positive predictive value of the active screening was only 19.67%, in contrast, the false-positive rate of positive results was 80.33%. The multivariate-probabilistic sensitivity analysis results supported the base-case findings, with a 75% probability for the false-positive rate of positive results over 47%. Conclusions: In the close contacts of COVID-19 patients, nearly half or even more of the 'asymptomatic infected individuals' reported in the active nucleic acid test screening might be false positives.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 Update From China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">By mid-February 2020 there were 60,000 confirmed cases of COVID-19, the vast majority diagnosed in Hubei Province (including Wuhan city) in mainland China. China CDC Chief Epidemiologist Zunyou Wu, MD, PhD discusses the latest COVID-19 developments in the country with JAMA Editor in Chief Howard Bauchner, MD.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Zoonotic origin and transmission of Middle East respiratory syndrome coronavirus in the UAE</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 May 13</span></i><p style="text-align: justify; text-justify: inter-word;">Since the emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012, there have been a number of clusters of human-to-human transmission. These cases of human-to-human transmission involve close contact and have occurred primarily in healthcare settings, and they are suspected to result from repeated zoonotic introductions. In this study, we sequenced whole MERS-CoV genomes directly from respiratory samples collected from 23 confirmed MERS cases in the United Arab Emirates (UAE). These samples included cases from three nosocomial and three household clusters. The sequences were analysed for changes and relatedness with regard to the collected epidemiological data and other available MERS-CoV genomic data. Sequence analysis supports the epidemiological data within the clusters, and further, suggests that these clusters emerged independently. To understand how and when these clusters emerged, respiratory samples were taken from dromedary camels, a known host of MERS-CoV, in the same geographic regions as the human clusters. Middle East respiratory syndrome coronavirus genomes from six virus-positive animals were sequenced, and these genomes were nearly identical to those found in human patients from corresponding regions. These data demonstrate a genetic link for each of these clusters to a camel and support the hypothesis that human MERS-CoV diversity results from multiple zoonotic introductions.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Rigidity of the Outer Shell Predicted by a Protein Intrinsic Disorder Model Sheds Light on the COVID-19 (Wuhan-2019-nCoV) Infectivity</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The world is currently witnessing an outbreak of a new coronavirus spreading quickly across China and affecting at least 24 other countries. With almost 65,000 infected, a worldwide death toll of at least 1370 (as of 14 February 2020), and with the potential to affect up to two-thirds of the world population, COVID-19 is considered by the World Health Organization (WHO) to be a global health emergency. The speed of spread and infectivity of COVID-19 (also known as Wuhan-2019-nCoV) are dramatically exceeding those of the Middle East respiratory syndrome coronavirus (MERS-CoV) and severe acute respiratory syndrome coronavirus (SARS-CoV). In fact, since September 2012, the WHO has been notified of 2494 laboratory-confirmed cases of infection with MERS-CoV, whereas the 2002&ndash;2003 epidemic of SARS affected 26 countries and resulted in more than 8000 cases. Therefore, although SARS, MERS, and COVID-19 are all the result of coronaviral infections, the causes of the coronaviruses differ dramatically in their transmissibility. It is likely that these differences in infectivity of coronaviruses can be attributed to the differences in the rigidity of their shells which can be evaluated using computational tools for predicting intrinsic disorder predisposition of the corresponding viral proteins.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevent and predict</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the COVID-19 outbreak continues, the next pandemic could be prevented by ending the wildlife trade and reinvesting in the monitoring of potential zoonoses.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Looming threat of COVID-19 infection in Africa: act collectively, and fast</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Models that enable the continent to better allocate scarce resources to better prepare and respond to the COVID-19 epidemic are crucial. The modelling study by Marius Gilbert and colleagues in The Lancet identifies each African country's risk of importation of COVID-19 from China, using data on the volume of air travel from three airports in provinces in China to African countries. Gilbert and colleagues use two indicators to determine the capacity of countries to detect and respond to cases: preparedness, using the WHO International Health Regulations Monitoring and Evaluation Framework; and vulnerability, using the Infectious Disease Vulnerability Index. Based on their analysis, Egypt, Algeria, and South Africa had the highest importation risk, and a moderate to high capacity to respond to outbreaks. Nigeria, Ethiopia, Sudan, Angola, Tanzania, Ghana, and Kenya had moderate risk with variable capacity and high vulnerability. In the model, the risk mainly originates from Guangdong, Fujian, and Beijing. The study provides a valuable tool that can help countries in Africa prioritise and allocate resources as they prepare to respond to the potential introduction and spread of COVID-19.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Role of temperature and humidity in the modulation of the doubling time of COVID-19 cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is having a great impact on public health, mortality and economy worldwide, in spite of the efforts to prevent its epidemy. The SARS-CoV-2 genome is different from that of MERS-CoV and SARS-CoV, although also expected to spread differently according to meteorological conditions. Our main goal is to investigate the role of some meteorological variables on the expansion of this outbreak. In this study, an exponential model relating the number of accumulated confirmed cases and time was considered. The rate of COVID-19 spread, using as criterion the doubling time of the number of confirmed cases, was used as dependent variable in a linear model that took four independent meteorological variables: temperature, humidity, precipitation and wind speed. Only China cases were considered, to control both cultural aspects and containment policies. Confirmed cases and the 4 meteorological variables were gathered between January 23 and March 1 (39 days) for the 31 provinces of Mainland China. Several periods of time were sampled for each province, obtaining more than one value for the rate of disease progression. Two different periods of time were tested, of 12 and 15 days, along with 3 and 5 different starting points in time, randomly chosen. The median value for each meteorological variable was computed, using the same time period; models with adjusted R square above 0.75 were selected. The rate of progression and doubling time were computed and used to fit a linear regression model. Models were evaluated using alpha=0.05. Results indicate that the doubling time correlates positively with temperature and inversely with humidity, suggesting that a decrease in the rate of progression of COVID-19 with the arrival of spring and summer in the north hemisphere. A 20oC increase is expected to delay the doubling time in 1.8 days. Those variables explain 18% of the variation in disease doubling time; the remaining 82% may be related to containment measures, general health policies, population density, transportation or cultural aspects.</jats:p></p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>World Health Organization declares Global Emergency: A review of the 2019 Novel Coronavirus (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An unprecedented outbreak of pneumonia of unknown aetiology in Wuhan City, Hubei province in China emerged in December of 2019. A novel coronavirus was identified as the causative agent and was subsequently termed COVID-19 by the World Health Organization (WHO). Considered a relative of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), COVID-19 is a betacoronavirus that affects the lower respiratory tract and manifests as pneumonia in humans. Despite rigorous global containment and quarantine efforts, the incidence of COVID-19 continues to rise, with 50,580 laboratory-confirmed cases and 1,526 deaths worldwide. In response to this global outbreak, we summarise the current state of knowledge surrounding COVID-19.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>A bioinformatic filter for improved base-call accuracy and polymorphism detection using the Affymetrix GeneChip® whole-genome resequencing platform</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Dec 15</span></i><p style="text-align: justify; text-justify: inter-word;">DNA resequencing arrays enable rapid acquisition of high-quality sequence data. This technology represents a promising platform for rapid high-resolution genotyping of microorganisms. Traditional array-based resequencing methods have relied on the use of specific PCR-amplified fragments from the query samples as hybridization targets. While this specificity in the target DNA population reduces the potential for artifacts caused by cross-hybridization, the subsampling of the query genome limits the sequence coverage that can be obtained and therefore reduces the technique's resolution as a genotyping method. We have developed and validated an Affymetrix Inc. GeneChip® array-based, whole-genome resequencing platform for Francisella tularensis, the causative agent of tularemia. A set of bioinformatic filters that targeted systematic base-calling errors caused by cross-hybridization between the whole-genome sample and the array probes and by deletions in the sample DNA relative to the chip reference sequence were developed. Our approach eliminated 91% of the false-positive single-nucleotide polymorphism calls identified in the SCHU S4 query sample, at the cost of 10.7% of the true positives, yielding a total base-calling accuracy of 99.992%.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-10-05</span></i><p style="text-align: justify; text-justify: inter-word;">The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Deep learning Enables Accurate Diagnosis of Novel Coronavirus (COVID-19) with CT images</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background A novel coronavirus (COVID-19) has emerged recently as an acute respiratory syndrome. The outbreak was originally reported in Wuhan, China, but has subsequently been spread world-widely. As the COVID-19 continues to spread rapidly across the world, computed tomography (CT) has become essentially important for fast diagnoses. Thus, it is urgent to develop an accurate computer-aided method to assist clinicians to identify COVID-19-infected patients by CT images.     Materials and Methods We collected chest CT scans of 88 patients diagnosed with the COVID-19 from hospitals of two provinces in China, 101 patients infected with bacteria pneumonia, and 86 healthy persons for comparison and modeling. Based on the collected dataset, a deep learning-based CT diagnosis system (DeepPneumonia) was developed to identify patients with COVID-19.  Results The experimental results showed that our model can accurately identify the COVID-19 patients from others with an excellent AUC of 0.99 and recall (sensitivity) of 0.93. In addition, our model was capable of discriminating the COVID-19 infected patients and bacteria pneumonia-infected patients with an AUC of 0.95, recall (sensitivity) of 0.96. Moreover, our model could localize the main lesion features, especially the ground-glass opacity (GGO) that is of great help to assist doctors in diagnosis. The diagnosis for a patient could be finished in 30 seconds, and the implementation on Tianhe-2 supercompueter enables a parallel executions of thousands of tasks simultaneously. An online server is available for online diagnoses with CT images by http://biomed.nscc-gz.cn/server/Ncov2019.  Conclusions The established models can achieve a rapid and accurate identification of COVID-19 in human samples, thereby allowing identification of patients.</jats:p></p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents.</jats:p></p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-14</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore. We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% - 64%) of Singapore′s capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 - 4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% -  67%), among intermediate surveillance locations it is 37% (95% HPDI 18% - 68%), and among low surveillance locations it is 11% (95% HPDI 0% - 42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers may be underestimated by several fold, and severity correspondingly overestimated by several fold. Undetected cases are likely in countries around the world, with greater risk in countries of low detection capacity and high connectivity to the epicenter of the outbreak.</jats:p></p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Quarantined at home now, U.S. scientist describes his visit to China’s hot zone</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 13 February, Clifford Lane went to a Washington, D.C.–area airport to catch a flight to Japan, where he would help launch a study of an experimental drug, remdesivir, against coronavirus disease 2019 (COVID-19). Lane is a deputy director at the U.S. National Institute of Allergy and Infectious Diseases and a right-hand man to Anthony Fauci, head of NIAID and the top research scientist in the country advising the White House on the outbreak of the virus. As Lane waited to board his plane, he was told that his final destination had changed. “I get an email, ‘You need to go to China.’ It’s like, are you kidding?” Lane had been selected as one of two U.S. scientists to join a World Health Organization team of 13 international researchers who would tour five different cities with 12 Chinese colleagues to get a firsthand look at the coronavirus epidemic there. The joint mission, which ran from 16–23 February led to a report that offered more details about the clinical course of COVID-19 and the epidemiology in China than had appeared anywhere before.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Pulmonary pathology of early phase 2019 novel coronavirus (COVID-19) pneumonia in two patients with lung cancer</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">There is currently a lack of pathologic data on the novel coronavirus (SARS-CoV-2) pneumonia, or COVID-19, from autopsy or biopsy. Two patients who recently underwent lung lobectomies for adenocarcinoma were retrospectively found to have had COVID-19 at the time of surgery. These two cases thus provide important first opportunities to study the pathology of COVID-19. Pathologic examinations revealed that, apart from the tumors, the lungs of both patients exhibited edema, proteinaceous exudate, focal reactive hyperplasia of pneumocytes with patchy inflammatory cellular infiltration, and multinucleated giant cells. Hyaline membranes were not prominent. Since both patients did not exhibit symptoms of pneumonia at the time of surgery, these changes likely represent an early phase of the lung pathology of COVID-19 pneumonia.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors' CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">The 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within COVID-19 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical characteristics of 101 non-surviving hospitalized patients with COVID-19: A single center, retrospective study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Rationale: COVID-19 have deprived many lives in Wuhan, China and caused global concerns. Few studies focused on clinical characteristics non-survivors hospitalized with COVID-19.   Objectives: We aimed to clarify the clinical characteristics of died patients with COVID-19 and tracked the causes of the rapid disease progression and death.  Methods: 101 non-surviving patients with confirmed COVID-19 were included in this retrospective study. Clinical data were collected and compared between non-survivors who died within 3 days and after 3 days of admission. Multivariable regression analysis was used to analyze risk factors associated with rapid disease progression and death. Measurements and Main Results: Among included patients, median age was 71 years (IQR, 59-80), 60 (59.41%) were men. 82 (81.19%) had one or more comorbidities including hypertension (59 [58.42%]), diabetes (22 [21.78%]) etc. 100 (99.01%) suffered respiratory failure, 53 (52.48%) developed acute cardiac injury, acute kidney and liver injury occurred in 23 (22.77%) and 18 (17.82%) patients, respectively. Multivariable regression analysis showed that elevated high sensitivity troponin I (OR 2.68, 95%CI 1.31-5.49, P=0.007), neutrophils (OR 1.14, 95%CI 1.01-1.28, P=0.033) and depressed oxygen saturation (OR 0.94, 95%CI 0.89-0.99, P=0.027) on admission were associated with rapid death of patients with COVID-19. Conclusions: Older patients (&gt;70 years) with comorbidities had a steeply increased risk of death with COVID-19. Respiratory failure, acute cardiac and kidney injury played a crucial role in the death of patients. Elevated high sensitivity troponin, neutrophils and depressed oxygen saturation predicted the rapid death of patients.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Top stories: Bungled coronavirus testing, disarming ‘atomic bomb’ cells, and jet stream blocking</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? One answer is that a test kit designed by the U.S. Centers for Disease Control and Prevention (CDC) and rolled out to state and local labs late last month contained a faulty reagent. The problems have led many to doubt the accuracy of confirmed tallies of the disease. But the situation may soon improve, as the Food and Drug Administration comes up with a workaround for the faulty CDC kit.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. METHODS: Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. RESULTS: The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. CONCLUSION: SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of the Severe Acute Respiratory Syndrome Coronavirus by Simultaneous Multigene DNA Sequencing</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recent severe acute respiratory syndrome (SARS) outbreak resulted in calls for an accurate diagnostic test that can be used not only for routine testing but also for generating nucleotide sequences to monitor the epidemic. Although the identity of the SARS coronavirus (SARS-CoV) genome was confirmed by DNA sequencing, it is impractical to sequence the entire 29-kb SARS-CoV genome on a routine basis. Therefore, alternative assay methods such as the enzyme-linked immunosorbent assay and PCR have been pursued for routine testing, primarily to resolve probable cases. We report here a modification of standard DNA sequencing technology for accurate identification of SARS-CoV in routine testing. Instead of requiring the sequencing of the whole SARS-CoV genome, our modification enables the simultaneous sequencing of three regions of the SARS-CoV genome, the spike protein-encoding gene (35 nucleotides), gene M (43 nucleotides), and gene N (45 nucleotides), in a single electropherogram. Comparing these nucleotide sequences to DNA databank entries (National Institutes of Health) conclusively identified them as SARS-CoV sequences.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The novel coronavirus SARS-CoV-2, etiological agent of recently named Coronavirus infected disease (COVID-19) by WHO, has caused more than 2, 000 deaths worldwide since its emergency in Wuhan City, Hubei province, China, in December, 2019. The symptoms of COVID-19 varied from modest, mild to acute respiratory distress syndrome (ARDS), and the latter of which is generally associated with deregulated immune cytokine production; however, we currently know little as to the interplay between the extent of clinical symptoms and the compositions of lung immune microenvironment. Here, we comprehensively characterized the lung immune microenvironment with the bronchoalveolar lavage fluid (BALF) from 3 severe and 3 mild COVID-19 patients and 8 previously reported healthy lung controls through single-cell RNA sequence (scRNA-seq) combined with TCR-seq. Our data shows that monocyte-derived FCN1+ macrophages, whereas notFABP4+ alveolar macrophages that represent a predominant macrophage subset in BALF from patients with mild diseases, overwhelm in the severely damaged lungs from patients with ARDS. These cells are highly inflammatory and enormous chemokine producers implicated in cytokine storm. Furthermore, the formation of tissue resident, highly expanded clonal CD8+ T cells in the lung microenvironment of mild symptom patients suggests a robust adaptive immune response connected to a better control of COVID-19. This study first reported the cellular atlas of lung bronchoalveolar immune microenvironment in COVID-19 patients at the single-cell resolution, and unveiled the potential immune mechanisms underlying disease progression and protection in COVID-19.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Respiratory support for patients with COVID-19 infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As of Feb 27, 2020, coronavirus disease 2019 (COVID-19) has affected 47 countries and territories around the world.1 Xiaobo Yang and colleagues2 described 52 of 710 patients with confirmed COVID-19 admitted to an intensive care unit (ICU) in Wuhan, China. 29 (56%) of 52 patients were given non-invasive ventilation at ICU admission, of whom 22 (76%) required further orotracheal intubation and invasive mechanical ventilation. The ICU mortality rate among those who required non-invasive ventilation was 23 (79%) of 29 and among those who required invasive mechanical ventilation was 19 (86%) of 22.2 Jonathan Chun-Hei Cheung and colleagues3 do not recommend use of a high-flow nasal cannula or non-invasive ventilation until the patient has viral clearance. Supporting the recommendation of the authors, I would like to add some points in relation to the use of high-flow nasal oxygen therapy and non-invasive ventilation in patients with COVID-19 infection: First, although exhaled air dispersion during high-flow nasal oxygen therapy and non-invasive ventilation via different interfaces is restricted, provided that there is a good mask interface fitting,4 not all hospitals around the world have access to such interfaces or enough personal-protective equipment of sufficiently high quality (ie, considered fit-tested particulate respirators, N95 or equivalent, or higher level of protection) for aerosol-generating procedures, and several hospitals do not have a negative pressure isolation room. Of 1688 health-care workers who have become infected with COVID-19, five (0·3%) have died;5 a sign of the vastly difficult working conditions for health-care workers.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 Update – March 2020 | Global Health | JN Learning | JAMA Ed Hub [Video recording]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus testing, mortality, vaccine development, containment vs mitigation, and more. Anthony S. Fauci, MD discusses the latest developments in the global spread of COVID-19 and the SARS-CoV-2 virus with JAMA Editor Howard Bauchner, MD. - What's the difference between COVID-19 and SARS-CoV-2? (01:15) - What's the status and accuracy of diagnostic testing in the US? (01:58) - What's the case-fatality rate for the virus? (05:31) - Scientific advances and vaccine development (25:06) - Are current clinical trials providing a picture of treatments? (13:41) - Risk communication: how do we present information so there's faith that it's accurate? (15:24) - Risk groups (children, the elderly, pregnant women) (16:26) - Containment vs mitigation vs quarantine vs isolation (19:10) - Protecting the elderly and nursing home resident (23:52) - Public health prospects in Latin America, Africa (26:35) - Will coronavirus wane in warmer months like influenza? (27:52) - Why is anxiety so high about this disease?- Does the US have capacity to care for COVID19 infection? (31:03) - What is your daily schedule like? (32:23)</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS, MERS and novel coronavirus (COVID-19) epidemics, the newest and biggest global health threats: what lessons have we learned?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China.Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made.Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally.We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /></body>