
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: understanding whether there is more than one strain of coronavirus in circulation</h2><br /></div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63, France</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Aug</span></i><p style="text-align: justify; text-justify: inter-word;">The human coronavirus NL63 (HCoV-NL63) was first identified in the Netherlands, and its circulation in France has not been investigated. We studied HCoV-NL63 infection in hospitalized children diagnosed with respiratory tract infections. From November 2002 to April 2003, we evaluated 300 respiratory specimens for HCoV-NL63. Of the 300 samples, 28 (9.3%) were positive for HCoV-NL63. The highest prevalence was found in February (18%). The main symptoms were fever (61%), rhinitis (39%), bronchiolitis (39%), digestive problems (33%), otitis (28%), pharyngitis (22%), and conjunctivitis (17%). A fragment of the spike protein gene was sequenced to determine the variety of circulating HCoV-NL63. Phylogenetic analysis indicated that strains with different genetic markers cocirculate in France.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Etiology of respiratory tract infections in the community and clinic in Ilorin, Nigeria</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 7</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE: Recognizing increasing interest in community disease surveillance globally, the goal of this study was to investigate whether respiratory viruses circulating in the community may be represented through clinical (hospital) surveillance in Nigeria. RESULTS: Children were selected via convenience sampling from communities and a tertiary care center (n = 91) during spring 2017 in Ilorin, Nigeria. Nasal swabs were collected and tested using polymerase chain reaction. The majority (79.1%) of subjects were under 6 years old, of whom 46 were infected (63.9%). A total of 33 of the 91 subjects had one or more respiratory tract virus; there were 10 cases of triple infection and 5 of quadruple. Parainfluenza virus 4, respiratory syncytial virus B and enterovirus were the most common viruses in the clinical sample; present in 93.8% (15/16) of clinical subjects, and 6.7% (5/75) of community subjects (significant difference, p < 0.001). Coronavirus OC43 was the most common virus detected in community members (13.3%, 10/75). A different strain, Coronavirus OC 229 E/NL63 was detected among subjects from the clinic (2/16) and not detected in the community. This pilot study provides evidence that data from the community can potentially represent different information than that sourced clinically, suggesting the need for community surveillance to enhance public health efforts and scientific understanding of respiratory infections.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Get Real</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since this is going to be a post about the coronavirus, let’s start off with this PSA: wash your hands. ... OK, either tomorrow or Friday I hope to do a post on all the things that are going on in the biopharma industry for a possible coronavirus treatment. ... It was clearly related to the virus from the first case (reported on January 19 in the same county in Washington state), descended from it in a way that makes it almost certain that the coronavirus has been spreading undetected among that population for weeks.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Respiratory Coronaviruses Detected In Patients with Influenza-Like Illness in Arkansas, USA</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Dec 26</span></i><p style="text-align: justify; text-justify: inter-word;">Acute respiratory viruses often result in significant morbidity and mortality. The potential impact of human respiratory coronavirus (CoV) infections was underestimated until the severe acute respiratory syndrome (SARS-CoV) outbreak in 2003, which showed that new, highly pathogenic coronaviruses could be introduced to humans, highlighting the importance of monitoring the circulating coronaviruses. The use of sensitive molecular methods has contributed to the differential diagnosis of viruses circulating in humans. Our study aim was to investigate the molecular epidemiology of human CoV strains circulating in Arkansas, their genetic variability and their association with reported influenza-like symptoms. We analyzed 200 nasal swab samples, collected by the Arkansas Department of Health in 2010, for influenza diagnosis. All samples were from patients showing acute respiratory symptoms while testing negative for influenza. Samples were pre-screened, using a quantitative reverse transcriptase polymerase chain reaction (qRT-PCR) multiprobe for coronavirus, and subjected to confirmatory pancoronavirus and/or strain-specific reverse transcriptase (RT)-PCR followed by sequence analysis. Seventy-nine samples (39.5%) were positive by qRT-PCR and 35 samples (17.5%) were confirmed by conventional RT-PCR. Twenty-three of the confirmed samples (59%) were sequenced. The most frequent strain detected was HCoV-OC43-like followed by NL63-like; only one sample was positive for HCoV-229E and one for HCoV-HKU1. Feline-like CoV strains were detected in three samples, representing possible evidence of interspecies transmission or a new human strain. Seventeen percent of the coronavirus positive samples were also positive for other respiratory viruses, such as Respiratory Syncytial Virus (RSV), Parainfluenza 2 and 3, and Rhinovirus. Thus, HCoV-OC43, NL63, HKU1 and new feline-like strains were circulating in Arkansas in 2010. HCoV was the sole respiratory virus detected in 16% of the patients who showed acute respiratory symptoms with negative diagnoses for influenza virus.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic analysis of 16 Colorado human NL63 coronaviruses identifies a new genotype, high sequence diversity in the N-terminal domain of the spike gene and evidence of recombination</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">This study compared the complete genome sequences of 16 NL63 strain human coronaviruses (hCoVs) from respiratory specimens of paediatric patients with respiratory disease in Colorado, USA, and characterized the epidemiology and clinical characteristics associated with circulating NL63 viruses over a 3-year period. From 1 January 2009 to 31 December 2011, 92 of 9380 respiratory specimens were found to be positive for NL63 RNA by PCR, an overall prevalence of 1 %. NL63 viruses were circulating during all 3 years, but there was considerable yearly variation in prevalence and the month of peak incidence. Phylogenetic analysis comparing the genome sequences of the 16 Colorado NL63 viruses with those of the prototypical hCoV-NL63 and three other NL63 viruses from the Netherlands demonstrated that there were three genotypes (A, B and C) circulating in Colorado from 2005 to 2010, and evidence of recombination between virus strains was found. Genotypes B and C co-circulated in Colorado in 2005, 2009 and 2010, but genotype A circulated only in 2005 when it was the predominant NL63 strain. Genotype C represents a new lineage that has not been described previously. The greatest variability in the NL63 virus genomes was found in the N-terminal domain (NTD) of the spike gene (nt 1–600, aa 1–200). Ten different amino acid sequences were found in the NTD of the spike protein among these NL63 strains and the 75 partial published sequences of NTDs from strains found at different times throughout the world.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Two antigenic groups of human coronaviruses detected by using enzyme-linked immunosorbent assay.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Paired sera from volunteers inoculated with one of the five recently isolated strains of human coronavirus (HCV), AD, GI, HO, PA, and RO, none of which has been grown in tissue culture, or with strain OC38 were tested against coronavirus antigens by enzyme-linked immunosorbent assay. When HCV strains OC43, 229E, or the 229E-related tissue culture-adapted strains PR and TO were used as antigens, it was shown that all strains fell into one of two antigenic groups. The HCV OC43 group was comprised of strains OC43, GI, HO, and RO, and the HCV 229E group contained strains AD and PA as well as the tissue culture-adapted strains PR, TO, and KI. Enzyme-linked immunosorbent assay of the paired sera with the coronavirus mouse hepatitis virus strain 3 as antigen confirmed the relationship of this virus to the HCV OC43 group but not to the HCV 229E group.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>Infectious Middle East Respiratory Syndrome Coronavirus Excretion and Serotype Variability Based on Live Virus Isolates from Patients in Saudi Arabia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The newly emerged Middle East respiratory syndrome coronavirus (MERS-CoV) has infected at least 1,082 people, including 439 fatalities. So far, no empirical virus isolation study has been done to elucidate infectious virus secretion or serotype variability. Here, we used 51 respiratory samples from 32 patients with confirmed MERS-CoV infection for virus isolation in Vero B4 and Caco-2 cells. We found Caco-2 cells to significantly enhance isolation success over routinely used Vero cells. Isolation success correlated with viral RNA concentration and time after diagnosis as well as with the amount of IgA antibodies secreted in respiratory samples used for isolation. Results from plaque reduction neutralization assays using a representative range of serum samples and virus isolates suggested that all circulating human MERS-CoV strains represent one single serotype. The choice of prototype strain is not likely to influence the success of candidate MERS-CoV vaccines. However, vaccine formulations should be evaluated for their potential to induce IgA.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution and variation of 2019-novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-30</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.  MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.  ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1{middle dot}05 x 10-2 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 x 10-4 to 2.72 x 10-2) for N, 5.34 x 10-3 (5.10 x 10-4, 1.28 x 10-2) for S, 1.69 x 10-3 (3.94 x 10-4, 3.60 x 10-3) for P, 1.65 x 10-3 (4.47 x 10-4, 3.24 x 10-3) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.  ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Variation of SARS Coronavirus in Beijing Hospital</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">To characterize genetic variation of severe acute respiratory syndrome–associated coronavirus (SARS-CoV) transmitted in the Beijing area during the epidemic outbreak of 2003, we sequenced 29 full-length S genes of SARS-CoV from 20 hospitalized SARS patients on our unit, the Beijing 302 Hospital. Viral RNA templates for the S-gene amplification were directly extracted from raw clinical samples, including plasma, throat swab, sputum, and stool, during the course of the epidemic in the Beijing area. We used a TA-cloning assay with direct analysis of nested reverse transcription–polymerase chain reaction products in sequence. One hundred thirteen sequence variations with nine recurrent variant sites were identified in analyzed S-gene sequences compared with the BJ01 strain of SARS-CoV. Among them, eight variant sites were, we think, the first documented. Our findings demonstrate the coexistence of S-gene sequences with and without substitutions (referred to BJ01) in samples analyzed from some patients.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Calculating virus spread</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Getting a full picture of the coronavirus outbreak is extremely difficult. Maths can help plug some of the gaps, says Adam Kucharski</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus antibody titres in sera of healthy adults and experimentally infected volunteers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Six coronaviruses isolated in the U.S.A. have been inoculated into volunteers and all produced colds. Between 10 and 20% of infected volunteers developed heterologous antibody responses after these and other experimental infections with coronaviruses. The haemagglutination-inhibition test with the OC43 virus strain was found to detect antibody rises after infection with a variety of strains. Studies on normal adult sera taken between 1965 and 1970 revealed a high frequency of neutralizing antibody to one strain (229 E) and a frequency of HI antibody to strain OC43 which fluctuated from year to year. Complement-fixing antibodies to these two viruses were also found, revealing an apparent increase in the activity of coronaviruses in the general population of the U.K., during the winter of 1968-9.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 14</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes severe acute respiratory illness with a case fatality rate (CFR) of 35,5%. The highest number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this disease. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its epidemiological cycle, prevention and containment measures can be derived from mathematical models of disease epidemiology. We constructed 2-strain models to predict past outbreaks in the interval 2012–2016 and derive key epidemiological information for Macca, Madina and Riyadh. We approached variability in infection through three different disease incidence functions capturing social behavior in response to an epidemic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal incidence and the overall shape of the epidemic cycle. The evolution in the basic reproduction number (R(0)) warns that MERS-CoV may easily take an epidemic form. The best model correctly captures this feature, indicating a high epidemic risk (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be aware that measures for strict containment are urgently needed before new epidemics take off in the region.</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Phylogenetic investigation of enteric bovine coronavirus in Ireland reveals partitioning between European and global strains</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 30</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Bovine coronavirus is a primary cause of neonatal calf diarrhea worldwide, and is also associated with acute diarrhea in adult cattle during the winter season. There are no reports on molecular characterization of bovine coronavirus in Ireland, and little data exists apart from serological studies. FINDINGS: In this study, 11 neonatal (mean age 9 days) calf BCoV strains from the south of Ireland were collected over a one year period and characterized using molecular methods. The spike gene which encodes a protein involved in viral entry, infectivity and immune response shows the most variability amongst the isolates and was subsequently selected for in depth analysis. Phylogenetic analysis of the spike gene revealed that the Irish strains clustered with novel BCoV strains from Europe in a unique clade, possibly indicating lineage partitioning. Direct analysis of alignments identified amino acid changes in the spike protein unique to the Irish clade. CONCLUSION: Thus, monitoring of bovine coronavirus in Ireland is important as the current isolates in circulation in the south of Ireland may be diverging from the available vaccine strain, which may have implications regarding future BCoV vaccine efficacy.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU1 Infection in the United States</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 May</span></i><p style="text-align: justify; text-justify: inter-word;">In 2005, a new human coronavirus, HCoV-HKU1, was identified in Hong Kong. We screened respiratory specimens collected from December 16, 2001, to December 15, 2002, from children <5 years of age who tested negative for respiratory syncytial virus, parainfluenza viruses, influenza virus, and adenovirus for HCoV-HKU1 by reverse transcription–polymerase chain reaction. Overall, 1,048 respiratory specimens from 851 children were tested, and 9 HCoV-HKU1–positive children (1%) were identified, 2 of whom had 2 positive specimens. Children who had HCoV-HKU1 infection had evidence of either upper or lower respiratory tract infection or both. Two patients had disease beyond the respiratory tract. HCoV-HKU1 was identified from December 2001 to February 2002. Sequence analyses suggest that a single strain was circulating. HCoV-HKU1 is therefore likely circulating in the United States and is associated with upper and lower respiratory tract disease.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and genetic diversity analysis of human coronaviruses among cross-border children</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: More than a decade after the outbreak of human coronaviruses (HCoVs) SARS in Guangdong province and Hong Kong SAR of China in 2002, there is still no reoccurrence, but the evolution and recombination of the coronaviruses in this region are still unknown. Therefore, surveillance on the prevalence and the virus variation of HCoVs circulation in this region is conducted. METHODS: A total of 3298 nasopharyngeal swabs samples were collected from cross-border children (<6 years, crossing border between Southern China and Hong Kong SAR) showing symptoms of respiratory tract infection, such as fever (body temperature > 37.5 °C), from 2014 May to 2015 Dec. Viral nucleic acids were analyzed and sequenced to study the prevalence and genetic diversity of the four human coronaviruses. The statistical significance of the data was evaluated with Fisher chi-square test. RESULTS: 78 (2.37%; 95%CI 1.8-2.8%) out of 3298 nasopharyngeal swabs specimens were found to be positive for OC43 (36;1.09%), HKU1 (34; 1.03%), NL63 (6; 0.18%) and 229E (2;0.01%). None of SARS or MERS was detected. The HCoVs predominant circulating season was in transition of winter to spring, especially January and February and NL63 detected only in summer and fall. Complex population with an abundant genetic diversity of coronaviruses was circulating and they shared homology with the published strains (99-100%). Besides, phylogenetic evolutionary analysis indicated that OC43 coronaviruses were clustered into three clades (B,D,E), HKU1 clustered into two clades(A,B) and NL63 clustered into two clades(A,B). Moreover, several novel mutations including nucleotides substitution and the insertion of spike of the glycoprotein on the viral surface were discovered. CONCLUSIONS: The detection rate and epidemic trend of coronaviruses were stable and no obvious fluctuations were found. The detected coronaviruses shared a conserved gene sequences in S and RdRp. However, mutants of the epidemic strains were detected, suggesting continuous monitoring of the human coronaviruses is in need among cross-border children, who are more likely to get infected and transmit the viruses across the border easily, in addition to the general public.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU1 and Other Coronavirus Infections in Hong Kong</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have recently described the discovery of a novel coronavirus, coronavirus HKU1 (CoV-HKU1), associated with community-acquired pneumonia. However, the clinical spectrum of disease and the epidemiology of CoV-HKU1 infections in relation to infections with other respiratory viruses are unknown. In this 12-month prospective study, 4,181 nasopharyngeal aspirates from patients with acute respiratory tract infections were subjected to reverse transcription-PCRs specific for CoV-HKU1 and human coronaviruses NL63 (HCoV-NL63), OC43 (HCoV-OC43), and 229E (HCoV-229E). Coronaviruses were detected in 87 (2.1%) patients, with 13 (0.3%) positive for CoV-HKU1, 17 (0.4%) positive for HCoV-NL63, 53 (1.3%) positive for HCoV-OC43, and 4 (0.1%) positive for HCoV-229E. Of the 13 patients with CoV-HKU1 infections, 11 were children and 8 had underlying diseases. Similar to the case for other coronaviruses, upper respiratory infection was the most common presentation of CoV-HKU1 infections, although pneumonia, acute bronchiolitis, and asthmatic exacerbation also occurred. Despite a shorter duration of fever (mean, 1.7 days) and no difference in maximum temperature in children with CoV-HKU1 infections compared to patients with most other respiratory virus infections, a high incidence of febrile seizures (50%) was noted, which was significantly higher than those for HCoV-OC43 (14%), adenovirus (9%), human parainfluenza virus 1 (0%), and respiratory syncytial virus (8%) infections. CoV-HKU1 and HCoV-OC43 infections peaked in winter, although cases of the former also occurred in spring to early summer. This is in contrast to HCoV-NL63 infections, which mainly occurred in early summer and autumn but were absent in winter. Two genotypes of CoV-HKU1 cocirculated during the study period. Continuous studies over a longer period are warranted to ascertain the seasonal variation and relative importance of the different coronaviruses. Similar studies in other countries are required to better determine the epidemiology and genetic diversity of CoV-HKU1.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetics and Pathogenesis of Feline Infectious Peritonitis Virus</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is endemic in feral cat populations and cat colonies, frequently preceding outbreaks of fatal feline infectious peritonitis (FIP). FCoV exhibits 2 biotypes: the pathogenic disease and a benign infection with feline enteric coronavirus (FECV). Uncertainty remains regarding whether genetically distinctive avirulent and virulent forms coexist or whether an avirulent form mutates in vivo, causing FIP. To resolve these alternative hypotheses, we isolated viral sequences from FCoV-infected clinically healthy and sick cats (8 FIP cases and 48 FECV-asymptomatic animals); 735 sequences from 4 gene segments were generated and subjected to phylogenetic analyses. Viral sequences from healthy cats were distinct from sick cats on the basis of genetic distances observed in the membrane and nonstructural protein 7b genes. These data demonstrate distinctive circulating virulent and avirulent strains in natural populations. In addition, 5 membrane protein amino acid residues with functional potential differentiated healthy cats from cats with FIP. These findings may have potential as diagnostic markers for virulent FIP-associated FCoV.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetics and Pathogenesis of Feline Infectious Peritonitis Virus</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is endemic in feral cat populations and cat colonies, frequently preceding outbreaks of fatal feline infectious peritonitis (FIP). FCoV exhibits 2 biotypes: the pathogenic disease and a benign infection with feline enteric coronavirus (FECV). Uncertainty remains regarding whether genetically distinctive avirulent and virulent forms coexist or whether an avirulent form mutates in vivo, causing FIP. To resolve these alternative hypotheses, we isolated viral sequences from FCoV-infected clinically healthy and sick cats (8 FIP cases and 48 FECV-asymptomatic animals); 735 sequences from 4 gene segments were generated and subjected to phylogenetic analyses. Viral sequences from healthy cats were distinct from sick cats on the basis of genetic distances observed in the membrane and nonstructural protein 7b genes. These data demonstrate distinctive circulating virulent and avirulent strains in natural populations. In addition, 5 membrane protein amino acid residues with functional potential differentiated healthy cats from cats with FIP. These findings may have potential as diagnostic markers for virulent FIP-associated FCoV.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetics and Pathogenesis of Feline Infectious Peritonitis Virus</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is endemic in feral cat populations and cat colonies, frequently preceding outbreaks of fatal feline infectious peritonitis (FIP). FCoV exhibits 2 biotypes: the pathogenic disease and a benign infection with feline enteric coronavirus (FECV). Uncertainty remains regarding whether genetically distinctive avirulent and virulent forms coexist or whether an avirulent form mutates in vivo, causing FIP. To resolve these alternative hypotheses, we isolated viral sequences from FCoV-infected clinically healthy and sick cats (8 FIP cases and 48 FECV-asymptomatic animals); 735 sequences from 4 gene segments were generated and subjected to phylogenetic analyses. Viral sequences from healthy cats were distinct from sick cats on the basis of genetic distances observed in the membrane and nonstructural protein 7b genes. These data demonstrate distinctive circulating virulent and avirulent strains in natural populations. In addition, 5 membrane protein amino acid residues with functional potential differentiated healthy cats from cats with FIP. These findings may have potential as diagnostic markers for virulent FIP-associated FCoV.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetics and Pathogenesis of Feline Infectious Peritonitis Virus</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is endemic in feral cat populations and cat colonies, frequently preceding outbreaks of fatal feline infectious peritonitis (FIP). FCoV exhibits 2 biotypes: the pathogenic disease and a benign infection with feline enteric coronavirus (FECV). Uncertainty remains regarding whether genetically distinctive avirulent and virulent forms coexist or whether an avirulent form mutates in vivo, causing FIP. To resolve these alternative hypotheses, we isolated viral sequences from FCoV-infected clinically healthy and sick cats (8 FIP cases and 48 FECV-asymptomatic animals); 735 sequences from 4 gene segments were generated and subjected to phylogenetic analyses. Viral sequences from healthy cats were distinct from sick cats on the basis of genetic distances observed in the membrane and nonstructural protein 7b genes. These data demonstrate distinctive circulating virulent and avirulent strains in natural populations. In addition, 5 membrane protein amino acid residues with functional potential differentiated healthy cats from cats with FIP. These findings may have potential as diagnostic markers for virulent FIP-associated FCoV.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetics and Pathogenesis of Feline Infectious Peritonitis Virus</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is endemic in feral cat populations and cat colonies, frequently preceding outbreaks of fatal feline infectious peritonitis (FIP). FCoV exhibits 2 biotypes: the pathogenic disease and a benign infection with feline enteric coronavirus (FECV). Uncertainty remains regarding whether genetically distinctive avirulent and virulent forms coexist or whether an avirulent form mutates in vivo, causing FIP. To resolve these alternative hypotheses, we isolated viral sequences from FCoV-infected clinically healthy and sick cats (8 FIP cases and 48 FECV-asymptomatic animals); 735 sequences from 4 gene segments were generated and subjected to phylogenetic analyses. Viral sequences from healthy cats were distinct from sick cats on the basis of genetic distances observed in the membrane and nonstructural protein 7b genes. These data demonstrate distinctive circulating virulent and avirulent strains in natural populations. In addition, 5 membrane protein amino acid residues with functional potential differentiated healthy cats from cats with FIP. These findings may have potential as diagnostic markers for virulent FIP-associated FCoV.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation of Coronavirus NL63 from Blood from Children in Rural Haiti: Phylogenetic Similarities with Recent Isolates from Malaysia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus (HCoV) NL63 is recognized as a common cause of upper respiratory infections and influenza-like illness. In screening children with acute undifferentiated febrile illness in a school cohort in rural Haiti, we identified HCoV-NL63 in blood samples from four children. Cases clustered over an 11-day period; children did not have respiratory symptoms, but two had gastrointestinal complaints. On phylogenetic analysis, the Haitian HCoV-NL63 strains cluster together in a highly supported monophyletic clade linked most closely with recently reported strains from Malaysia; two respiratory HCoV-NL63 strains identified in north Florida in the same general period form a separate clade, albeit again with close linkages with the Malaysian strains. Our data highlight the variety of presentations that may be seen with HCoV-NL63, and underscore the apparent ease with which CoV strains move among countries, with our data consistent with recurrent introduction of strains into the Caribbean (Haiti and Florida) from Asia.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>A novel pancoronavirus RT-PCR assay: frequent detection of human coronavirus NL63 in children hospitalized with respiratory tract infections in Belgium</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Feb 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Four human coronaviruses are currently known to infect the respiratory tract: human coronaviruses OC43 (HCoV-OC43) and 229E (HCoV-229E), SARS associated coronavirus (SARS-CoV) and the recently identified human coronavirus NL63 (HCoV-NL63). In this study we explored the incidence of HCoV-NL63 infection in children diagnosed with respiratory tract infections in Belgium. METHODS: Samples from children hospitalized with respiratory diseases during the winter seasons of 2003 and 2004 were evaluated for the presence of HCoV-NL63 using a optimized pancoronavirus RT-PCR assay. RESULTS: Seven HCoV-NL63 positive samples were identified, six were collected during January/February 2003 and one at the end of February 2004. CONCLUSIONS: Our results support the notation that HCoV-NL63 can cause serious respiratory symptoms in children. Sequence analysis of the S gene showed that our isolates could be classified into two subtypes corresponding to the two prototype HCoV-NL63 sequences isolated in The Netherlands in 1988 and 2003, indicating that these two subtypes may currently be cocirculating.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Intrahost Diversity of Feline Coronavirus: A Consensus between the Circulating Virulent/Avirulent Strains and the Internal Mutation Hypotheses?</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Mar 27</span></i><p style="text-align: justify; text-justify: inter-word;">To evaluate the most controversial issue concerning current feline coronavirus (FCoV) virology, the coexisting hypotheses of the intrahost and interhost origins of feline infectious peritonitis virus (FIPV) in regard to the pathogenesis of feline infectious peritonitis (FIP), this study aimed to assess the molecular diversity of the membrane gene FCoVs in 190 samples from 10 cats with signs of FIP and in 5 faecal samples from cats without signs of FIP. All samples from the non-FIP cats and 25.26% of the samples from the FIP cats were positive for the FCoV membrane (M) gene. Mutations in this gene consisted of SNP changes randomly scattered among the sequences; few mutations resulted in amino acid changes. No geographic pattern was observed. Of the cats without FIP that harboured FECoV, the amino acid sequence identities for the M gene were 100% among cats (Cats 1–3) from the same cattery, and the overall sequence identity for the M gene was ≥91%. In one cat, two different lineages of FCoV, one enteric and one systemic, were found that segregated apart in the M gene tree. In conclusion, the in vivo mutation transition hypothesis and the circulating high virulent-low virulent FCoV hypothesis have been found to be plausible according to the results obtained from sequencing the M gene.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Sequences of Human Coronavirus OC43 and NL63, Associated with Respiratory Infections in Kilifi, Kenya</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 14</span></i><p style="text-align: justify; text-justify: inter-word;">Coding-complete genomes of two human coronavirus OC43 strains and one NL63 strain were obtained by metagenomic sequencing of clinical samples collected in 2017 and 2018 in Kilifi, Kenya. Maximum likelihood phylogenies showed that the OC43 strains were genetically dissimilar and that the NL63 strain was closely related to NL63 genotype B viruses.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and Genetic Diversity Analysis of Human Coronavirus OC43 among Adult Patients with Acute Respiratory Infections in Beijing, 2012</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jul 2</span></i><p style="text-align: justify; text-justify: inter-word;">To determine the prevalence, epidemiology and genetic diversity of human coronavirus OC43 (HCoV-OC43) among adult patients with acute respiratory infections (ARI) in Beijing,five hundred and fifty-nine nasopharyngeal swab samples were collected from adult patients with ARI in Beijing. The prevalence of HCoV-OC43 infection among these patients was assessed using two different OneStep reverse transcription polymerase chain reaction (RT-PCR) assays. The epidemiological profiles of the patients with HCoV-OC43 infection were described. Partial S and N genes of HCoV-OC43 circulating strains were sequenced followed by phylogenetic analysis and amino acid alignment. Our results showed that the prevalence of HCoV-OC43 infection was 12.52% (95% CI: 9.78–15.26%), and the epidemic peak occurred in autumn. Fifty partial S and 40 partial N fragments were obtained from these patients. Phylogenetic analysis based on neighbour-joining method showed that at least three distinct clusters (A, B, C/D) of HCoV-OC43 strains were circulating among adult patients with ARI in Beijing. In addition, some novel unique clusters (UNT) of HCoV-OC43 were found in the S- and N-based phylogenetic trees. Furthermore, consensus amino acids substitutes for each cluster were also found after alignment of partial S or N sequence coding region in this study. In conclusion, we herein describe the prevalence of HCoV-OC43 among adult patients and provide substantial evidence for the genetic diversity of HCoV-OC43 circulating in Beijing.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic diversity of SARS-CoV-2 in Coronavirus Disease 2019 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) has infected more than 75,000 individuals and spread to over 20 countries. It is still unclear how fast the virus evolved and how the virus interacts with other microorganisms in the lung.We have conducted metatranscriptome sequencing for the bronchoalveolar lavage fluid of eight SARS-CoV-2 patients, 25 community-acquired pneumonia (CAP) patients, and 20 healthy controls.The median number of intra-host variants was 1-4 in SARS-CoV-2 infected patients, which ranged between 0 and 51 in different samples. The distribution of variants on genes was similar to those observed in the population data (110 sequences). However, very few intra-host variants were observed in the population as polymorphism, implying either a bottleneck or purifying selection involved in the transmission of the virus, or a consequence of the limited diversity represented in the current polymorphism data. Although current evidence did not support the transmission of intra-host variants in a person-to-person spread, the risk should not be overlooked. The microbiota in SARS-CoV-2 infected patients was similar to those in CAP, either dominated by the pathogens or with elevated levels of oral and upper respiratory commensal bacteria.SARS-CoV-2 evolves in vivo after infection, which may affect its virulence, infectivity, and transmissibility. Although how the intra-host variant spreads in the population is still elusive, it is necessary to strengthen the surveillance of the viral evolution in the population and associated clinical changes.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Backcalculating the Incidence of Infection with COVID-19 on the Diamond Princess Comparative Seasonal Respiratory Virus Epidemic Timing in Utah</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">To understand the time-dependent risk of infection on a cruise ship, the Diamond Princess, I estimated the incidence of infection with novel coronavirus (COVID-19). The epidemic curve of a total of 199 confirmed cases was drawn, classifying individuals into passengers with and without close contact and crew members. A backcalculation method was employed to estimate the incidence of infection. The peak time of infection was seen for the time period from 2 to 4 February 2020, and the incidence has abruptly declined afterwards. The estimated number of new infections among passengers without close contact was very small from 5 February on which a movement restriction policy was imposed. Without the intervention from 5 February, it was predicted that the cumulative incidence with and without close contact would have been as large as 1373 (95% CI: 570, 2176) and 766 (95% CI: 587, 946) cases, respectively, while these were kept to be 102 and 47 cases, respectively. Based on an analysis of illness onset data on board, the risk of infection among passengers without close contact was considered to be very limited. Movement restriction greatly reduced the number of infections from 5 February onwards. Previous studies have found evidence of viral interference between seasonal respiratory viruses. Using laboratory-confirmed data from a Utah-based healthcare provider, Intermountain Health Care, we analyzed the time-specific patterns of respiratory syncytial virus (RSV), influenza A, influenza B, human metapneumovirus, rhinovirus, and enterovirus circulation from 2004 to 2018, using descriptive methods and wavelet analysis (n = 89,462) on a local level. The results showed that RSV virus dynamics in Utah were the most consistent of any of the viruses studied, and that the other seasonal viruses were generally in synchrony with RSV, except for enterovirus (which mostly occurs late summer to early fall) and influenza A and B during pandemic years.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human coronavirus NL63 (HCoV-NL63) is a globally endemic pathogen causing mild and severe respiratory tract infections with reinfections occurring repeatedly throughout a lifetime. METHODS: Nasal samples were collected in coastal Kenya through community-based and hospital-based surveillance. HCoV-NL63 was detected with multiplex real-time reverse transcription PCR, and positive samples were targeted for nucleotide sequencing of the spike (S) protein. Additionally, paired samples from 25 individuals with evidence of repeat HCoV-NL63 infection were selected for whole-genome virus sequencing. RESULTS: HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching. CONCLUSIONS: In this coastal Kenya setting, HCoV-NL63 exhibited low prevalence in hospital pediatric pneumonia admissions. Clade persistence with low genetic diversity suggest limited immune selection, and absence of detectable clade switching in reinfections indicates initial exposure was insufficient to elicit a protective immune response.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human coronavirus NL63 (HCoV-NL63) is a globally endemic pathogen causing mild and severe respiratory tract infections with reinfections occurring repeatedly throughout a lifetime. METHODS: Nasal samples were collected in coastal Kenya through community-based and hospital-based surveillance. HCoV-NL63 was detected with multiplex real-time reverse transcription PCR, and positive samples were targeted for nucleotide sequencing of the spike (S) protein. Additionally, paired samples from 25 individuals with evidence of repeat HCoV-NL63 infection were selected for whole-genome virus sequencing. RESULTS: HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching. CONCLUSIONS: In this coastal Kenya setting, HCoV-NL63 exhibited low prevalence in hospital pediatric pneumonia admissions. Clade persistence with low genetic diversity suggest limited immune selection, and absence of detectable clade switching in reinfections indicates initial exposure was insufficient to elicit a protective immune response.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jun 1</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Human coronavirus NL63 (HCoV-NL63) is a globally endemic pathogen causing mild and severe respiratory tract infections with reinfections occurring repeatedly throughout a lifetime. METHODS: Nasal samples were collected in coastal Kenya through community-based and hospital-based surveillance. HCoV-NL63 was detected with multiplex real-time reverse transcription PCR, and positive samples were targeted for nucleotide sequencing of the spike (S) protein. Additionally, paired samples from 25 individuals with evidence of repeat HCoV-NL63 infection were selected for whole-genome virus sequencing. RESULTS: HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching. CONCLUSIONS: In this coastal Kenya setting, HCoV-NL63 exhibited low prevalence in hospital pediatric pneumonia admissions. Clade persistence with low genetic diversity suggest limited immune selection, and absence of detectable clade switching in reinfections indicates initial exposure was insufficient to elicit a protective immune response.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of SARS-associated Coronavirus, Beijing</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">Single nucleotide variations (SNVs) at 5 loci (17564, 21721, 22222, 23823, and 27827) were used to define the molecular epidemiologic characteristics of severe acute respiratory syndrome–associated coronavirus (SARS-CoV) from Beijing patients. Five fragments targeted at the SNV loci were amplified directly from clinical samples by using reverse transcription–polymerase chain reaction (RT-PCR), before sequencing the amplified products. Analyses of 45 sequences obtained from 29 patients showed that the GGCTC motif dominated among samples collected from March to early April 2003; the TGTTT motif predominanted afterwards. The switch from GGCTC to TGTTT was observed among patients belonging to the same cluster, which ruled out the possibility of the coincidental superposition of 2 epidemics running in parallel in Beijing. The Beijing isolates underwent the same change pattern reported from Guangdong Province. The same series of mutations occurring in separate geographic locations and at different times suggests a dominant process of viral adaptation to the host.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>A double epidemic model for the SARS propagation</h3>
<i class="tooltip">2003<span class="tooltiptext">2003 Sep 10</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior. It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior. RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread. We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region. We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses. This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS. CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS. Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected. We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients. Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic. The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus HKU15 in respiratory tract of pigs and first discovery of coronavirus quasispecies in 5′-untranslated region</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus HKU15 is a deltacoronavirus that was discovered in fecal samples of pigs in Hong Kong in 2012. Over the past three years, Coronavirus HKU15 has been widely detected in pigs in East/Southeast Asia and North America and has been associated with fatal outbreaks. In all such epidemiological studies, the virus was generally only detected in fecal/intestinal samples. In this molecular epidemiology study, we detected Coronavirus HKU15 in 9.6% of the nasopharyngeal samples obtained from 249 pigs in Hong Kong. Samples that tested positive were mostly collected during winter. Complete genome sequencing of the Coronavirus HKU15 in two nasopharyngeal samples revealed quasispecies in one of the samples. Two of the polymorphic sites involved indels, but the other two involved transition substitutions. Phylogenetic analysis showed that the two nasopharyngeal strains in the present study were most closely related to the strains PDCoV/CHJXNI2/2015 from Jiangxi, China, and CH/Sichuan/S27/2012 from Sichuan, China. The outbreak strains in the United States possessed highly similar genome sequences and were clustered monophyletically, whereas the Asian strains were more diverse and paraphyletic. The detection of Coronavirus HKU15 in respiratory tracts of pigs implies that in addition to enteric infections, Coronavirus HKU15 may be able to cause respiratory infections in pigs and that in addition to fecal-oral transmission, the virus could possibly spread through the respiratory route. The presence of the virus in respiratory samples provides an alternative clinical sample to confirm the diagnosis of Coronavirus HKU15 infection. Quasispecies were unprecedentedly observed in the 5′-untranslated region of coronavirus genomes.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus Infections in Israel: Epidemiology, Clinical Symptoms and Summer Seasonality of HCoV-HKU1</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Sep 21</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses (HCoVs) cause mild to severe respiratory diseases. Six types of HCoVs have been discovered, the most recent one termed the Middle East respiratory syndrome coronavirus (MERS-CoV). The aim of this study is to monitor the circulation of HCoV types in the population during 2015–2016 in Israel. HCoVs were detected by real-time PCR analysis in 1910 respiratory samples, collected from influenza-like illness (ILI) patients during the winter sentinel influenza survey across Israel. Moreover, 195 HCoV-positive samples from hospitalized patients were detected during one year at Soroka University Medical Center. While no MERS-CoV infections were detected, 10.36% of patients in the survey were infected with HCoV-OC43 (43.43%), HCoV-NL63 (44.95%), and HCoV-229E (11.62%) viruses. The HCoVs were shown to co-circulate with respiratory syncytial virus (RSV) and to appear prior to influenza virus infections. HCoV clinical symptoms were more severe than those of RSV infections but milder than influenza symptoms. Hospitalized patients had similar HCoV types percentages. However, while it was absent from the public winter survey, 22.6% of the patients were HCoV-HKU1 positives, mainly during the spring-summer period.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of intrapatient heterogeneity uncovers the microevolution of Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Genome sequence analysis of Middle East respiratory syndrome coronavirus (MERS-CoV) variants from patient specimens has revealed the evolutionary dynamics and mechanisms of pathogenesis of the virus. However, most studies have analyzed the consensus sequences of MERS-CoVs, precluding an investigation of intrapatient heterogeneity. Here, we analyzed non–consensus sequences to characterize intrapatient heterogeneity in cases associated with the 2015 outbreak of MERS in South Korea. Deep-sequencing analysis of MERS-CoV genomes performed on specimens from eight patients revealed significant intrapatient variation; therefore, sequence heterogeneity was further analyzed using targeted deep sequencing. A total of 35 specimens from 24 patients (including a super-spreader) were sequenced to detect and analyze variants displaying intrapatient heterogeneity. Based on the analysis of non–consensus sequences, we demonstrated the intrapatient heterogeneity of MERS-CoVs, with the highest level in the super-spreader specimen. The heterogeneity could be transmitted in a close association with variation in the consensus sequences, suggesting the occurrence of multiple MERS-CoV infections. Analysis of intrapatient heterogeneity revealed a relationship between D510G and I529T mutations in the receptor-binding domain (RBD) of the viral spike glycoprotein. These two mutations have been reported to reduce the affinity of the RBD for human CD26. Notably, although the frequency of both D510G and I529T varied greatly among specimens, the combined frequency of the single mutants was consistently high (87.7% ± 1.9% on average). Concurrently, the frequency of occurrence of the wild type at the two positions was only 6.5% ± 1.7% on average, supporting the hypothesis that selection pressure exerted by the host immune response played a critical role in shaping genetic variants and their interaction in human MERS-CoVs during the outbreak.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>How Coronaviruses Cause Infection—from Colds to Deadly Pneumonia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus outbreak raises questions about how such pathogens evolve and what makes infections mild or severe</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of Bovine Coronavirus on the Basis of Comparative Analyses of the S Gene</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Bovine coronavirus (BCoV), a group 2 member of the genus Coronavirus in the family Coronaviridae, is an important pathogen in cattle worldwide. It causes diarrhea in adult animals (winter dysentery), as well as enteric and respiratory diseases in calves. The annual occurrence of BCoV epidemics in Sweden and Denmark led to this investigation, with the aim to deepen the knowledge of BCoV epidemiology at the molecular level. A total of 43 samples from outbreaks in both countries were used for PCR amplification and direct sequencing of a 624-nucleotide fragment of the BCoV S gene. Sequence comparison and phylogenetic studies were performed. The results showed (i) identical sequences from different animals in the same herds and from paired nasal and fecal samples, suggesting a dominant virus circulating in each herd at a given time; (ii) sequence differences among four outbreaks in different years in the same herd, indicating new introduction of virus; (iii) identical sequences in four different Danish herds in samples obtained within 2 months, implying virus transmission between herds; and (iv) that at least two different virus strains were involved in the outbreaks of BCoV in Denmark during the spring of 2003. This study presents molecular data of BCoV infections that will contribute to an increased understanding of BCoV epidemiology in cattle populations.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>An outbreak of feline infectious peritonitis in a Taiwanese shelter: epidemiologic and molecular evidence for horizontal transmission of a novel type II feline coronavirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 17</span></i><p style="text-align: justify; text-justify: inter-word;">Feline infectious peritonitis (FIP) is a fatal disease caused by feline coronavirus (FCoV) infection. FCoV can be divided into serotypes I and II. The virus that causes FIP (FIPV) is believed to occur sporadically and spread infrequently from cat to cat. Recently, an FIP outbreak from an animal shelter was confirmed in Taiwan. FCoV from all the cats in this shelter were analyzed to determine the epidemiology of this outbreak. Thirteen of 46 (28.2%) cats with typical signs of FIP were identified. Among them, seven cats were confirmed by necropsy and/or histopathological examinations. Despite the fact that more than one FCoV was identified in this multi-cat environment, the eight FIP cats were invariably found to be infected with a type II FCoV. Sequence analysis revealed that the type II FIPV detected from fecal samples, body effusions and granulomatous tissue homogenates from the cats that succumbed to FIP all harbored an identical recombination site in their S gene. Two of the cats that succumbed to FIP were found to harbor an identical nonsense mutation in the 3c gene. Fecal shedding of this type II virus in the effusive form of FIP can be detected up to six days before death. Taken together, our data demonstrate that horizontal transmission of FIPV is possible and that FIP cats can pose a potential risk to other cats living in the same environment.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: global infections pass 90,000</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Updates on the respiratory illness that has infected tens of thousands of people. The number of people worldwide who have been infected with the coronavirus has passed 90,000. More than 3,000 people have died since the outbreak began in December. The vast majority of cases — more than 80,000 — have occurred in China, but around 60 other countries are now also dealing with outbreaks. Many nations are preparing for a global pandemic, as reports of cases caused by community spread — rather than importation from China — rise. South Korea, Italy and Iran are fighting the largest outbreaks outside of China.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although human coronavirus OC43-OC43 (HCoV-OC43) is the coronavirus most commonly associated with human infections, little is known about its molecular epidemiology and evolution. We conducted a molecular epidemiology study to investigate different genotypes and potential recombination in HCoV-OC43. Twenty-nine HCoV-OC43 strains from nasopharyngeal aspirates, collected from 2004 to 2011, were subjected to RNA-dependent RNA polymerase (RdRp), spike, and nucleocapsid gene analysis. Phylogenetic analysis showed at least three distinct clusters of HCoV-OC43, although 10 unusual strains displayed incongruent phylogenetic positions between RdRp and spike genes. This suggested the presence of four HCoV-OC43 genotypes (A to D), with genotype D most likely arising from recombination. The complete genome sequencing of two genotype C and D strains and bootscan analysis showed recombination events between genotypes B and C in the generation of genotype D. Of the 29 strains, none belonged to the more ancient genotype A, 5 from 2004 belonged to genotype B, 15 from 2004 to 2006 belonged to genotype C, and 1 from 2004 and all 8 from 2008 to 2011 belonged to the recombinant genotype D. Molecular clock analysis using spike and nucleocapsid genes dated the most recent common ancestor of all genotypes to the 1950s, genotype B and C to the 1980s, genotype B to the 1990s, and genotype C to the late 1990s to early 2000s, while the recombinant genotype D strains were detected as early as 2004. This represents the first study to describe natural recombination in HCoV-OC43 and the evolution of different genotypes over time, leading to the emergence of novel genotype D, which is associated with pneumonia in our elderly population.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>An outbreak of <scp>COVID</scp> ‐19 caused by a new coronavirus: what we know so far</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of a novel coronavirus, now formally named severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) and causing coronavirus disease 2019 (COVID‐19), emerged in the city of Wuhan in Hubei province in central China in December 2019. The first cases were noted as a cluster of patients with pneumonia who were all linked to a live animal market, and testing found the presence of a previously unknown coronavirus. Coronaviruses are a group of viruses that affect both animals and humans, and several (OC43, 229E, HKU1 and NL63) are a cause of the common cold.1, 2 However, two coronaviruses have previously caused significant outbreaks associated with more severe disease: the SARS coronavirus in 2002–2003 and the Middle East respiratory syndrome coronavirus that emerged in 2012.1, 2 Chinese authorities and researchers should be commended for their rapid sharing of viral sequences which enabled laboratories worldwide to develop diagnostic tests within weeks of discovery of the pathogen.3 An Australian laboratory subsequently isolated the virus from a clinical sample (the first to do so outside of China), and rapidly shared this virus with relevant global agencies, further aiding diagnostic, therapeutic and vaccine development efforts. Information on the new virus and its impact is being updated constantly. We know that SARS‐CoV‐2 can cause severe disease, although active surveillance of contacts is required to define the milder end of the disease spectrum and to estimate the true hospitalisation and case fatality ratio. The cases reported to date suggest that most are older adults; it is currently unclear whether comorbidities reflect the age group affected or whether they are risk factors for severe disease.4, 5 Early studies using data before the institution of public health interventions in China suggest that SARS‐CoV‐2 is as transmissible as SARS coronavirus and probably more transmissible than influenza viruses.6, 7 The timing of infectiousness relative to symptom onset is a particularly important parameter with implications for public health control. While reports suggest that asymptomatic infection and transmission may result from minimally symptomatic cases, the contribution of this to transmission is not yet known.8 Careful analysis of early data suggests that the mean incubation period is 6 days, with a range of up to 14 days.9 Reports of large outbreaks, particularly associated with hospitals and closed communities, raise the possibility of “superspreading” events, a feature of previous coronavirus outbreaks.10 The importance of infection control is also reinforced by a report that 41% of cases in Wuhan were acquired nosocomially (including 40 health care workers and 17 patients).4 As at 26 February 2020, there were 23 confirmed cases of COVID‐19 in Australia, across five jurisdictions. Although reported case numbers in China are slowing, large outbreaks on a cruise ship in Japan, and in South Korea, Iran and Italy are concerning and highlight our interconnected world. However, it is likely that this will change in the days or weeks ahead. Although absolute case numbers are small in Australia, the public health, political and societal ramifications have already been considerable, ranging from travel restrictions on non‐Australians coming from China, the use of offshore and remote quarantine facilities, and disturbing reports of racism against members of our Asian community (https://insig​htplus.mja.com.au/2020/5/coron​avirus-no-place-for-racism-xenop​hobia/).</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Descriptive distribution and phylogenetic analysis of feline infectious peritonitis virus isolates of Malaysia</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Jan 6</span></i><p style="text-align: justify; text-justify: inter-word;">The descriptive distribution and phylogeny of feline coronaviruses (FCoVs) were studied in cats suspected of having feline infectious peritonitis (FIP) in Malaysia. Ascitic fluids and/or biopsy samples were subjected to a reverse transcription polymerase chain reaction (RT-PCR) targeted for a conserved region of 3'untranslated region (3'UTR) of the FCoV genome. Eighty nine percent of the sampled animals were positive for the presence of FCoV. Among the FCoV positive cats, 80% of cats were males and 64% were below 2 years of age. The FCoV positive cases included 56% domestic short hair (DSH), 40% Persian, and 4% Siamese cats. The nucleotide sequences of 10 selected amplified products from FIP cases were determined. The sequence comparison revealed that the field isolates had 96% homology with a few point mutations. The extent of homology decreased to 93% when compared with reference strains. The overall branching pattern of phylogenetic tree showed two distinct clusters, where all Malaysian isolates fall into one main genetic cluster. These findings provided the first genetic information of FCoV in Malaysia.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Sensitivity and specificity of a real-time reverse transcriptase polymerase chain reaction detecting feline coronavirus mutations in effusion and serum/plasma of cats to diagnose feline infectious peritonitis</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Aug 2</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Feline coronavirus (FCoV) exists as two pathotypes, and FCoV spike gene mutations are considered responsible for the pathotypic switch in feline infectious peritonitis (FIP) pathogenesis. The aim of this study was to evaluate sensitivity and specificity of a real-time reverse transcriptase polymerase chain reaction (RT-PCR) specifically designed to detect FCoV spike gene mutations at two nucleotide positions. It was hypothesized that this test would correctly discriminate feline infectious peritonitis virus (FIPV) and feline enteric coronavirus (FECV). METHODS: The study included 63 cats with signs consistent with FIP. FIP was confirmed in 38 cats. Twenty-five control cats were definitively diagnosed with a disease other than FIP. Effusion and/or serum/plasma samples were examined by real-time RT-PCR targeting the two FCoV spike gene fusion peptide mutations M1058 L and S1060A using an allelic discrimination approach. Sensitivity, specificity, negative and positive predictive values including 95% confidence intervals (95% CI) were calculated. RESULTS: FIPV was detected in the effusion of 25/59 cats, one of them being a control cat with chronic kidney disease. A mixed population of FIPV/FECV was detected in the effusion of 2/59 cats; all of them had FIP. RT-PCR was negative or the pathotype could not be determined in 34/59 effusion samples. In effusion, sensitivity was 68.6% (95% CI 50.7–83.2), specificity was 95.8% (95% CI 78.9–99.9). No serum/plasma samples were positive for FIPV. CONCLUSIONS: Although specificity of the test in effusions was high, one false positive result occurred. The use of serum/plasma cannot be recommended due to a low viral load in blood.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Co-Circulation of Canine Coronavirus I and IIa/b with High Prevalence and Genetic Diversity in Heilongjiang Province, Northeast China</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 15</span></i><p style="text-align: justify; text-justify: inter-word;">To trace the evolution of canine coronavirus (CCoV), 201 stool samples from diarrheic dogs in northeast China were subjected to reverse transcription-polymerase chain reactions (RT-PCRs) targeting the partial M and S genes of CCoV, followed by an epidemiological analysis. M gene RT-PCRs showed that 28.36% (57/201) of the samples were positive for CCoV; of the 57 positive samples, CCoV-I and CCoV-II accounted for 15.79% (9/57) and 84.21% (48/57), respectively. A sequence comparison of the partial M gene revealed nucleotide homologies of 88.4%–100% among the 57 CCoV strains, and 88.7%–96.2% identity between the 57 CCoV strains and the Chinese reference strain HF3. The CCoV-I and CCoV-II strains exhibited genetic diversity when compared with reference strains from China and other countries. The 57 CCoV strains exhibited high co-infection rates with canine kobuvirus (CaKV) (33.33%) and canine parvovirus-2 (CPV-2) (31.58%). The CCoV prevalence in diarrheic dogs differed significantly with immunization status, regions, seasons, and ages. Moreover, 28 S genes were amplified from the 57 CCoV-positive samples, including 26 CCoV-IIa strains, one CCoV-IIb strain, and one CCoV-I strain. A sequence comparison of the partial S gene revealed 86.3%–100% nucleotide identity among the 26 CCoV-IIa strains, and 89.6%–92.2% identity between the 26 CCoV-IIa strains and the Chinese reference strain V1. The 26 CCoV-IIa strains showed genetic diversity when compared with reference strains from China and other countries. Our data provide evidence that CCoV-I, CCoV-IIa, and CCoV-IIb strains co-circulate in the diarrhoetic dogs in northeast China, high co-infection rates with CaKV and CPV-2 were observed, and the CCoV-II strains exhibited high prevalence and genetic diversity.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Extremely Low Exposure of a Community to Severe Acute Respiratory Syndrome Coronavirus: False Seropositivity due to Use of Bacterially Derived Antigens</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Estimates of seropositivity to a new infectious agent in a community are useful to public health. For severe acute respiratory syndrome (SARS), the figures are conflicting. Herein, we screened 12,000 people in a community stricken by SARS 10 months previously and found 53 individuals (0.44%) who had immunoglobulin G antibodies to the SARS coronavirus (SARS-CoV) nucleocapsid (N) produced in bacteria. However, only seven of these (group 1) had sera which also reacted with the native N antigen expressed in SARS-CoV-infected Vero cells, N-transfected 293T cells, and tissues of infected SARS patients. Of these, six individuals had had SARS previously. The remaining person, as well as the 46 other individuals (group 2), were healthy and had no history of SARS. Group 1 antibodies recognized epitopes located slightly differently in N from those of group 2 antibodies, and a mouse hybridoma antibody resembling the former type was generated. Unusually, group 2 antibodies appeared to recognize cross-reactive bacterial epitopes that presumably were posttranslationally modified in eukaryotes and hence were probably not induced by SARS-CoV or related coronaviruses but rather by bacteria. The N antigen is thus highly unique. The extremely low rate (0.008%) of asymptomatic SARS infection found attests to the high virulence of the SARS-CoV virus.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">What is COVID-19, and what do we know so far about its clinical presentation? The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1]. The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host [2]. Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological evidence of MERS-CoV and HKU8-related CoV co-infection in Kenyan camels</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Oct 24</span></i><p style="text-align: justify; text-justify: inter-word;">Dromedary camels are important reservoir hosts of various coronaviruses, including Middle East respiratory syndrome coronavirus (MERS-CoV) that cause human infections. CoV genomes regularly undergo recombination during infection as observed in bat SARS-related CoVs. Here we report for the first time that only a small proportion of MERS-CoV receptor-binding domain positive (RBD) of spike protein positive camel sera in Kenya were also seropositive to MERS-CoV nucleocapsid (NP). In contrast, many of them contain antibodies against bat HKU8-related (HKU8r)-CoVs. Among 584 camel samples that were positive against MERS-CoV RBD, we found only 0.48 (8.22%) samples were also positive for NP. Furthermore, we found bat HKU8r-CoV NP antibody in 73 (12.5%) of the MERS-CoV RBD positive and NP negative samples, yet found only 3 (0.43%) of the HKU8r-CoV S1 antibody in the same samples. These findings may indicate co-infection with MERS-CoV and a HKU8r-CoV in camels. It may also raise the possibility of the circulation of a recombinant coronavirus virus with the spike of MERS-CoV and the NP of a HKU8r-CoV in Kenya. We failed to find molecular evidence of an HKU8r-CoV or a putative recombinant virus. Our findings should alert other investigators to look for molecular evidence of HKU8r-CoV or recombinants.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Polyphyletic origin of MERS coronaviruses and isolation of a novel clade A strain from dromedary camels in the United Arab Emirates</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Dec 21</span></i><p style="text-align: justify; text-justify: inter-word;">Little is known regarding the molecular epidemiology of Middle East respiratory syndrome coronavirus (MERS-CoV) circulating in dromedaries outside Saudi Arabia. To address this knowledge gap, we sequenced 10 complete genomes of MERS-CoVs isolated from 2 live and 8 dead dromedaries from different regions in the United Arab Emirates (UAE). Phylogenetic analysis revealed one novel clade A strain, the first detected in the UAE, and nine clade B strains. Strain D998/15 had a distinct phylogenetic position within clade A, being more closely related to the dromedary isolate NRCE-HKU205 from Egypt than to the human isolates EMC/2012 and Jordan-N3/2012. A comparison of predicted protein sequences also demonstrated the existence of two clade A lineages with unique amino acid substitutions, A1 (EMC/2012 and Jordan-N3/2012) and A2 (D998/15 and NRCE-HKU205), circulating in humans and camels, respectively. The nine clade B isolates belong to three distinct lineages: B1, B3 and B5. Two B3 strains, D1271/15 and D1189.1/15, showed evidence of recombination between lineages B4 and B5 in ORF1ab. Molecular clock analysis dated the time of the most recent common ancestor (tMRCA) of clade A to March 2011 and that of clade B to November 2011. Our data support a polyphyletic origin of MERS-CoV in dromedaries and the co-circulation of diverse MERS-CoVs including recombinant strains in the UAE.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Radiology Perspective of Coronavirus Disease 2019 (COVID-19): Lessons From Severe Acute Respiratory Syndrome and Middle East Respiratory Syndrome</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE. Since the outbreak of the novel coronavirus pulmonary illness coronavirus disease 2019 (COVID-19) in China, more than 79,000 people have contracted the virus worldwide. The virus is rapidly spreading with human-to-human transmission despite imposed precautions. Because similar pulmonary syndromes have been reported from other strains of the coronavirus family, our aim is to review the lessons from imaging studies obtained during severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) outbreaks. CONCLUSION. The review of experiences with the MERS and SARS outbreaks will help us better understand the role of the radiologist in combating the outbreak of COVID-19. The known imaging manifestations of the novel coronavirus and the possible unknowns will also be discussed.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequence of Human Coronavirus OC43 Isolated from Mexico</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Nov 10</span></i><p style="text-align: justify; text-justify: inter-word;">We report the complete genome sequence of the first Mexican human coronavirus (HCoV) OC43, obtained by new-generation sequencing and a metagenomic approach, isolated from a child hospitalized with pneumonia. The genome is closely related to the other OC43 genome sequences available, ranging from 99.8% to 98.2% nucleotide sequence identity.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Why does the coronavirus spread so easily between people?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the number of coronavirus infections approaches 100,000 people worldwide, researchers are racing to understand what makes it spread so easily. A handful of genetic and structural analyses have identified a key feature of the virus — a protein on its surface — that might explain why it infects human cells so readily. Other groups are investigating the doorway through which the new coronavirus enters human tissues — a receptor on cell membranes. Both the cell receptor and the virus protein offer potential targets for drugs to block the pathogen, but researchers say it is too early to be sure. “Understanding transmission of the virus is key to its containment and future prevention,” says David Veesler, a structural virologist at the University of Washington in Seattle, who posted his team’s findings about the virus protein on the biomedical preprint server bioRxiv on 20 February1. The new virus spreads much more readily than the one that caused severe acute respiratory syndrome, or SARS (also a coronavirus), and has infected more than ten times the number of people who contracted SARS.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>The recent ancestry of Middle East respiratory syndrome coronavirus in Korea has been shaped by recombination</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 6</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe cases of human respiratory disease. Since 2012, the victims have mainly come from the Middle East countries or sporadically from some other geographical regions seeded by the travelers who visited the Middle East. Such an introduction through travelling led to the emergence of a MERS-CoV outbreak in Korea in May 2015, which caused more than 140 confirmed human cases in less than a month. Using 70 complete genome sequences of MERS-CoV isolates, including the most recent sequences for the Korean and Chinese isolates, we reconstructed the phylogenetic relationships of the complete genome and the individual protein coding regions. The Korean MERS-CoV strain clustered in the previously established Hafr-Al-Batin-1_2013 clade together with two Saudi Arabian and one Chinese strain sampled in 2015. Although these four strains remained monophyletic in the entire protein-coding region, this clade showed different phylogenetic relationships across the genome, indicating a shared unique recombination pattern that is different from previously reported putative recombination strains. Our findings suggest that the recent ancestor of the Korean and its related MERS-CoV strains is characterized by unique mosaic genome pattern that is different from other putative recombinants.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>The recent ancestry of Middle East respiratory syndrome coronavirus in Korea has been shaped by recombination</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 6</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe cases of human respiratory disease. Since 2012, the victims have mainly come from the Middle East countries or sporadically from some other geographical regions seeded by the travelers who visited the Middle East. Such an introduction through travelling led to the emergence of a MERS-CoV outbreak in Korea in May 2015, which caused more than 140 confirmed human cases in less than a month. Using 70 complete genome sequences of MERS-CoV isolates, including the most recent sequences for the Korean and Chinese isolates, we reconstructed the phylogenetic relationships of the complete genome and the individual protein coding regions. The Korean MERS-CoV strain clustered in the previously established Hafr-Al-Batin-1_2013 clade together with two Saudi Arabian and one Chinese strain sampled in 2015. Although these four strains remained monophyletic in the entire protein-coding region, this clade showed different phylogenetic relationships across the genome, indicating a shared unique recombination pattern that is different from previously reported putative recombination strains. Our findings suggest that the recent ancestor of the Korean and its related MERS-CoV strains is characterized by unique mosaic genome pattern that is different from other putative recombinants.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus and the race to distribute reliable diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">International teams worked at speed to make tests for the virus available in record time. The medical community is rallying to develop a set of rapid and reliable molecular diagnostic tests for the new human coronavirus that appeared in China — now dubbed sudden acute respiratory syndrome coronavirus-2 (SARS-CoV-2).</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic lesions within the 3a gene of SARS-CoV</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Jun 20</span></i><p style="text-align: justify; text-justify: inter-word;">A series of frameshift mutations within the 3a gene has been observed in culture-derived severe acute respiratory syndrome coronavirus (SARS-CoV). We report here that viral RNA from clinical samples obtained from SARS-CoV infected patients also contains a heterogeneous population of wild-type and mutant 3a transcripts.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Mutation of the S and 3c genes in genomes of feline coronaviruses</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 17</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is classified into two biotypes based on its pathogenicity in cats: a feline enteric coronavirus of low pathogenicity and a highly virulent feline infectious peritonitis virus. It has been suspected that FCoV alters its biotype via mutations in the viral genome. The S and 3c genes of FCoV have been considered the candidates for viral pathogenicity conversion. In the present study, FCoVs were analyzed for the frequency and location of mutations in the S and 3c genes from faecal samples of cats in an animal shelter and the faeces, effusions, and tissues of cats that were referred to veterinary hospitals. Our results indicated that approximately 95% FCoVs in faeces did not carry mutations in the two genes. However, 80% FCoVs in effusion samples exhibited mutations in the S and 3c genes with remainder displaying a mutation in the S or 3c gene. It was also suggested that mutational analysis of the 3c gene could be useful for studying the horizontal transmission of FCoVs in multi-cat environments.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Mutation of the S and 3c genes in genomes of feline coronaviruses</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 17</span></i><p style="text-align: justify; text-justify: inter-word;">Feline coronavirus (FCoV) is classified into two biotypes based on its pathogenicity in cats: a feline enteric coronavirus of low pathogenicity and a highly virulent feline infectious peritonitis virus. It has been suspected that FCoV alters its biotype via mutations in the viral genome. The S and 3c genes of FCoV have been considered the candidates for viral pathogenicity conversion. In the present study, FCoVs were analyzed for the frequency and location of mutations in the S and 3c genes from faecal samples of cats in an animal shelter and the faeces, effusions, and tissues of cats that were referred to veterinary hospitals. Our results indicated that approximately 95% FCoVs in faeces did not carry mutations in the two genes. However, 80% FCoVs in effusion samples exhibited mutations in the S and 3c genes with remainder displaying a mutation in the S or 3c gene. It was also suggested that mutational analysis of the 3c gene could be useful for studying the horizontal transmission of FCoVs in multi-cat environments.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Characterization of Major Structural Protein Genes of Avian Coronavirus Infectious Bronchitis Virus Isolates in Southern China</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Dec 4</span></i><p style="text-align: justify; text-justify: inter-word;">To gain comprehensive genetic information of circulating avian coronavirus infectious bronchitis virus (IBV) isolates in China, analysis of the phylogenetic tree, entropy of the amino acid sequences, and the positive selection as well as computational recombinations of S1, M and N genes of 23 IBV isolates was conducted in the present study. The phylogenetic trees based on the S1, M and N genes exhibited considerably different topology and the CK/CH/LSC/99I-type isolates were the predominant IBVs based on the phylogenetic analysis of S1 gene. Results of entropy of amino acid sequences revealed that the S1 gene had the largest variation; the M gene had less variation than the N gene. Positive selections were detected in not only S1 but also M and N gene proteins. In addition, five S1 gene recombinants between vaccine strain 4/91 and CK/CH/LSC/99I-type field isolate were confirmed. In conclusion, multiple IBV genotypes co-circulated; genetic diversity and positive selections existed in S1, M and N genes; 4/91 vaccine recombinants emerged in China. Our results show that field IBVs in China are continuing to evolve and vaccine strains may have an important role in the appearance of new IBV strains via recombination. In addition, the present study indicates that IBV evolution is driven by both generations of genetic diversity and selection.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>De-isolating COVID-19 Suspect Cases: A Continuing Challenge</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of 15 February 2020, Singapore had screened a total of 991 suspect cases for COVID-19, of which 72 cases tested positive, 812 cases tested negative, while the remaining 107 had pending results.(1) Besides optimising sample type to increase yield, (2) the challenge in clinical management of suspect cases lies in deciding whether they may be de-isolated or if further isolation and repeat testing is required. No single indicator may be effectively used to decide on de-isolation of a suspect case. In our series of positive cases, samples from one suspect case only returned positive on the fifth repeated sample (nasopharyngeal swab), on the seventh day of clinical illness. Current evidence suggests that transmission of COVID-19 may be possible even from asymptomatic contacts,(3) and polymerase chain reaction (PCR) testing may not return positive initially. (4) Our suspect case was kept isolated because of a high index of clinical suspicion, with a clinically compatible illness and history of close contact with a laboratory-proven COVID-19 case. While multiplex respiratory virus panels, in general, may be helpful in the evaluation of other viral acute respiratory infections (ARIs), even the detection of an alternate respiratory pathogen may not definitively exclude COVID-19 infection. Dual infections can occur in 10- 20% of viral acute respiratory infections, as has been reported with SARS-CoV and MERSCoV. (5) In our case series, one patient with confirmed COVID-19 by nasopharyngeal aspirate also exhibited clinical symptoms compatible with dengue fever. This was laboratory confirmed by dengue NS1 antigen test. (PL Lim, personal communication).</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological Responses in Patients with Severe Acute Respiratory Syndrome Coronavirus Infection and Cross-Reactivity with Human Coronaviruses 229E, OC43, and NL63</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The serological response profile of severe acute respiratory syndrome (SARS) coronavirus (CoV) infection was defined by neutralization tests and subclass-specific immunofluorescent (IF) tests using serial sera from 20 patients. SARS CoV total immunoglobulin (Ig) (IgG, IgA, and IgM [IgGAM]) was the first antibody to be detectable. There was no difference in time to seroconversion between the patients who survived (n = 14) and those who died (n = 6). Although SARS CoV IgM was still detectable by IF tests with 8 of 11 patients at 7 months postinfection, the geometric mean titers dropped from 282 at 1 month postinfection to 19 at 7 months (P = 0.001). In contrast, neutralizing antibody and SARS CoV IgGAM and IgG antibody titers remained stable over this period. The SARS CoV antibody response was sometimes associated with an increase in preexisting IF IgG antibody titers for human coronaviruses OC43, 229E, and NL63. There was no change in IF IgG titer for virus capsid antigen from the herpesvirus that was used as an unrelated control, Epstein-Barr virus. In contrast, patients who had OC43 infections, and probably also 229E infections, without prior exposure to SARS CoV had increases of antibodies specific for the infecting virus but not for SARS CoV. There is a need for awareness of cross-reactive antibody responses between coronaviruses when interpreting IF serology.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects of Coronavirus Infections in Children</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">The isolation of the coronavirus (CoV) identified as the cause of severe acute respiratory syndrome and the detection of 2 new human CoVs (HCoV-NL63 and HCoV-HKU1) have led to studies of the epidemiology and clinical and socioeconomic effects of infections caused by all HCoVs, including those known since the late 1960s (HCoV-229E and HCoV-OC43). HCoV infections can be associated with respiratory and extrarespiratory manifestations, including central nervous system involvement. Furthermore, unlike other RNA viruses, HCoVs can easily mutate and recombine when different strains infect the same cells and give rise to a novel virus with unpredictable host ranges and pathogenicity. Thus, circulating HCoVs should be closely monitored to detect the spread of particularly virulent strains in the community at an early stage and to facilitate the development of adequate preventive and therapeutic measures.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects of Coronavirus Infections in Children</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">The isolation of the coronavirus (CoV) identified as the cause of severe acute respiratory syndrome and the detection of 2 new human CoVs (HCoV-NL63 and HCoV-HKU1) have led to studies of the epidemiology and clinical and socioeconomic effects of infections caused by all HCoVs, including those known since the late 1960s (HCoV-229E and HCoV-OC43). HCoV infections can be associated with respiratory and extrarespiratory manifestations, including central nervous system involvement. Furthermore, unlike other RNA viruses, HCoVs can easily mutate and recombine when different strains infect the same cells and give rise to a novel virus with unpredictable host ranges and pathogenicity. Thus, circulating HCoVs should be closely monitored to detect the spread of particularly virulent strains in the community at an early stage and to facilitate the development of adequate preventive and therapeutic measures.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic variance of the 2019-nCoV coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;">There is rising global concern for the recently emerged novel Coronavirus (2019-nCov). Full genomic sequences have been released by the worldwide scientific community in the last few weeks in order to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and SARS. We confirm high sequence similarity (>99%) between all sequenced 2019-nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019-nCoV. Despite the low heterogeneity of the 2019-nCoV genomes, we could identify at least two hyper-variable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8-encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti-coronavirus approaches.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of serologic and antigenic relationships between middle eastern respiratory syndrome coronavirus and other coronaviruses to develop vaccine platforms for the rapid response to emerging coronaviruses</h3>
<i class="tooltip">2014<span class="tooltiptext">2014</span></i><p style="text-align: justify; text-justify: inter-word;">Background: Middle East respiratory syndrome coronavirus (MERS-CoV) emerged in 2012, causing severe acute respiratory disease and pneumonia, with 44% mortality among 136 cases to date. Design of vaccines to limit the virus spread or diagnostic tests to track newly emerging strains requires knowledge of antigenic and serologic relationships between MERS-CoV and other CoVs.Methods:  Using synthetic genomics and Venezuelan equine encephalitis virus replicons (VRPs) expressing spike and nucleocapsid proteins from MERS-CoV and other human and bat CoVs, we characterize the antigenic responses (using Western blot and enzyme-linked immunosorbent assay) and serologic responses (using neutralization assays) against 2 MERS-CoV isolates in comparison with those of other human and bat CoVs.Results:  Serologic and neutralization responses against the spike glycoprotein were primarily strain specific, with a very low level of cross-reactivity within or across subgroups. CoV N proteins within but not across subgroups share cross-reactive epitopes with MERS-CoV isolates. Our findings were validated using a convalescent-phase serum specimen from a patient infected with MERS-CoV (NA 01) and human antiserum against SARS-CoV, human CoV NL63, and human CoV OC43.Conclusions:  Vaccine design for emerging CoVs should involve chimeric spike protein containing neutralizing epitopes from multiple virus strains across subgroups to reduce immune pathology, and a diagnostic platform should include a panel of nucleocapsid and spike proteins from phylogenetically distinct CoVs.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Spread, Circulation, and Evolution of the Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first documented in the Kingdom of Saudi Arabia (KSA) in 2012 and, to date, has been identified in 180 cases with 43% mortality. In this study, we have determined the MERS-CoV evolutionary rate, documented genetic variants of the virus and their distribution throughout the Arabian peninsula, and identified the genome positions under positive selection, important features for monitoring adaptation of MERS-CoV to human transmission and for identifying the source of infections. Respiratory samples from confirmed KSA MERS cases from May to September 2013 were subjected to whole-genome deep sequencing, and 32 complete or partial sequences (20 were ≥99% complete, 7 were 50 to 94% complete, and 5 were 27 to 50% complete) were obtained, bringing the total available MERS-CoV genomic sequences to 65. An evolutionary rate of 1.12 × 10(−3) substitutions per site per year (95% credible interval [95% CI], 8.76 × 10(−4); 1.37 × 10(−3)) was estimated, bringing the time to most recent common ancestor to March 2012 (95% CI, December 2011; June 2012). Only one MERS-CoV codon, spike 1020, located in a domain required for cell entry, is under strong positive selection. Four KSA MERS-CoV phylogenetic clades were found, with 3 clades apparently no longer contributing to current cases. The size of the population infected with MERS-CoV showed a gradual increase to June 2013, followed by a decline, possibly due to increased surveillance and infection control measures combined with a basic reproduction number (R(0)) for the virus that is less than 1.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>A Multigene Approach for Comparing Genealogy of Betacoronavirus from Cattle and Horses</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Nov 17</span></i><p style="text-align: justify; text-justify: inter-word;">Gastroenteritis is one of the leading causes of morbidity and mortality among young and newborn animals and is often caused by multiple intestinal infections, with rotavirus and bovine coronavirus (BCoV) being the main viral causes in cattle. Given that BCoV is better studied than equine coronaviruses and given the possibility of interspecies transmission of these viruses, this research was designed to compare the partial sequences of the spike glycoprotein (S), hemagglutinin-esterase protein (HE), and nucleoprotein (N) genes from coronaviruses from adult cattle with winter dysentery, calves with neonatal diarrhea, and horses. To achieve this, eleven fecal samples from dairy cows with winter dysentery, three from calves, and two from horses, all from Brazil, were analysed. It could be concluded that the enteric BCoV genealogy from newborn and adult cattle is directly associated with geographic distribution patterns, when S and HE genes are taken into account. A less-resolved genealogy exists for the HE and N genes in cattle, with a trend for an age-related segregation pattern. The coronavirus strains from horses revealed Betacoronavirus sequences indistinguishable from those found in cattle, a fact previously unknown.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombinant Canine Coronaviruses in Dogs, Europe</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses of potential recombinant origin with porcine transmissible gastroenteritis virus (TGEV), referred to as a new subtype (IIb) of canine coronavirus (CCoV), were recently identified in dogs in Europe. To assess the distribution of the TGEV-like CCoV subtype, during 2001–2008 we tested fecal samples from dogs with gastroenteritis. Of 1,172 samples, 493 (42.06%) were positive for CCoV. CCoV-II was found in 218 samples, and CCoV-I and CCoV-II genotypes were found in 182. Approximately 20% of the samples with CCoV-II had the TGEV-like subtype; detection rates varied according to geographic origin. The highest and lowest rates of prevalence for CCoV-II infection were found in samples from Hungary and Greece (96.87% and 3.45%, respectively). Sequence and phylogenetic analyses showed that the CCoV-IIb strains were related to prototype TGEV-like strains in the 5′ and the 3′ ends of the spike protein gene.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-27</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundA novel coronavirus (2019-nCoV) associated with human to human transmission and severe human infection has been recently reported from the city of Wuhan in China. Our objectives were to characterize the genetic relationships of the 2019-nCoV and to search for putative recombination within the subgenus of sarbecovirus.  MethodsPutative recombination was investigated by RDP4 and Simplot v3.5.1 and discordant phylogenetic clustering in individual genomic fragments was confirmed by phylogenetic analysis using maximum likelihood and Bayesian methods.  ResultsOur analysis suggests that the 2019-nCoV although closely related to BatCoV RaTG13 sequence throughout the genome (sequence similarity 96.3%), shows discordant clustering with the Bat-SARS-like coronavirus sequences. Specifically, in the 5-part spanning the first 11,498 nucleotides and the last 3-part spanning 24,341-30,696 positions, 2019-nCoV and RaTG13 formed a single cluster with Bat-SARS-like coronavirus sequences, whereas in the middle region spanning the 3-end of ORF1a, the ORF1b and almost half of the spike regions, 2019-nCoV and RaTG13 grouped in a separate distant lineage within the sarbecovirus branch.  ConclusionsThe levels of genetic similarity between the 2019-nCoV and RaTG13 suggest that the latter does not provide the exact variant that caused the outbreak in humans, but the hypothesis that 2019-nCoV has originated from bats is very likely. We show evidence that the novel coronavirus (2019-nCov) is not-mosaic consisting in almost half of its genome of a distinct lineage within the betacoronavirus. These genomic features and their potential association with virus characteristics and virulence in humans need further attention.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Mouse hepatitis virus utilizes two carcinoembryonic antigens as alternative receptors.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The cellular receptor for the murine coronavirus mouse hepatitis virus (MHV) has been identified as a member of the murine carcinoembryonic antigen (CEA) family (R. K. Williams, G. S. Jiang, and K. V. Holmes, Proc. Natl. Acad. Sci. USA 88:5533-5536, 1991). However, the receptor protein was not detected in some of the susceptible mouse tissues. We therefore examined whether other types of MHV receptor might exist. By polymerase chain reaction with the conserved sequences of murine CEA gene family members (mmCGM) as primers, we detected two CEA-encoding RNAs in the mouse liver. One of them (1.3 kb) encodes mmCGM1, which has previously been identified as the receptor for MHV, and the other one (0.8 kb) was shown to encode another member of mouse CGM, mmCGM2. The sequence analysis showed that mmCGM2 lacks 564 nucleotides in the middle of the gene compared with mmCGM1. These two CEA transcripts are probably derived from the same gene by an alternative splicing mechanism. Expression of either of these cDNA clones in COS-7 cells rendered these cells susceptible to MHV infection, suggesting that not only mmCGM1 but also mmCGM2 serves as a receptor for MHV. The mmCGM2 was the major CEA species in the mouse brain, which is a main target organ for the neurotropic strains of MHV. Very little mmCGM1 was detected in the mouse brain or in cells of the susceptible mouse astrocytoma cell line DBT. This result indicates that MHV may utilize different CEA molecules as the major receptor in the mouse brain and in the liver. This is a first identification of multiple receptors for a single virus. The presence of different receptors in different tissues may explain the target cell specificity of certain MHVs.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of canine coronavirus spread among domestic dogs in Vietnam</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Feb 14</span></i><p style="text-align: justify; text-justify: inter-word;">Canine coronavirus (CCoV) is an important pathogen that causes enteritis in dogs, but there is no information on CCoV infection in Vietnam. To examine the prevalence of CCoV infection among Vietnamese dogs, 201 serum samples were analyzed by virus-neutralization (VN) test. The results showed that antibody against CCoV-II was present in 87 dogs (43.3%). To detect genes of CCoV, fecal samples collected from 30 diarrheic and 50 healthy dogs were examinated by RT-PCR, confirming that 2 diarrheic dogs and 5 healthy dogs were positive for CCoV. Nucleotide sequences of N-terminal region of spike (S) gene indicated that CCoV strains were divided into two subgenotypes, CCoV-IIa and -IIb, respectively. Furthemore, we succeeded in isolating CCoV/dog/HCM47/2015, the isolate was plaque-purified three times, and 3’-terminal one-third of the genome was analyzed. Interestingly, the plaque-purified virus had a large deletion in ORF3abc and E genes (1,165 nt), and a short deletion in ORF7b gene (60 nt), suggesting that these regions are not necessary for in vitro replication of CCoV. Next, the antigenicity between the isolated CCoV-IIb and the other CCoV-IIa was compared by VN test, revealing that antigenicty of the isolated CCoV is equal or higher than that of the other CCoV. In summary, two subgenotypes of CCoV-II are spreading among Vietnamese dogs. The isolated virus with a large deletion after in vitro passage may be useful for the development of vaccine, owing to its antigenicity and efficient viral growth in vitro.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of canine coronavirus spread among domestic dogs in Vietnam</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Feb 14</span></i><p style="text-align: justify; text-justify: inter-word;">Canine coronavirus (CCoV) is an important pathogen that causes enteritis in dogs, but there is no information on CCoV infection in Vietnam. To examine the prevalence of CCoV infection among Vietnamese dogs, 201 serum samples were analyzed by virus-neutralization (VN) test. The results showed that antibody against CCoV-II was present in 87 dogs (43.3%). To detect genes of CCoV, fecal samples collected from 30 diarrheic and 50 healthy dogs were examinated by RT-PCR, confirming that 2 diarrheic dogs and 5 healthy dogs were positive for CCoV. Nucleotide sequences of N-terminal region of spike (S) gene indicated that CCoV strains were divided into two subgenotypes, CCoV-IIa and -IIb, respectively. Furthemore, we succeeded in isolating CCoV/dog/HCM47/2015, the isolate was plaque-purified three times, and 3’-terminal one-third of the genome was analyzed. Interestingly, the plaque-purified virus had a large deletion in ORF3abc and E genes (1,165 nt), and a short deletion in ORF7b gene (60 nt), suggesting that these regions are not necessary for in vitro replication of CCoV. Next, the antigenicity between the isolated CCoV-IIb and the other CCoV-IIa was compared by VN test, revealing that antigenicty of the isolated CCoV is equal or higher than that of the other CCoV. In summary, two subgenotypes of CCoV-II are spreading among Vietnamese dogs. The isolated virus with a large deletion after in vitro passage may be useful for the development of vaccine, owing to its antigenicity and efficient viral growth in vitro.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Co-circulation of Four Human Coronaviruses (HCoVs) in Queensland Children with Acute Respiratory Tract Illnesses in 2004</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Apr 23</span></i><p style="text-align: justify; text-justify: inter-word;">Acute respiratory illnesses (ARIs) with unconfirmed infectious aetiologies peak at different times of the year. Molecular diagnostic assays reduce the number of unconfirmed ARIs compared to serology- or culture-based techniques. Screening of 888 inpatient and outpatient respiratory specimens spanning late autumn through to early spring, 2004, identified the presence of a human coronavirus (HCoV) on 74 occasions (8.3% of all specimens and 26.3% of all respiratory virus detections). Prevalence peaked in August (late winter in the southern hemisphere) when they were detected in 21.9% of specimens tested. HCoV-HKU1 and HCoV-OC43 comprised 82.4% of all HCoVs detected. Positive specimens were used to develop novel reverse transcriptase real-time PCRs (RT-rtPCRs) for HCoV detection. An objective clinical severity score was assigned to each positive HCoV patient. Severity scores were similar to those from a random selection of young children who were positive for respiratory syncytial virus at a different time but from the same specimen population. During the cooler months of 2004, sensitive and specific RT-rtPCRs identified the concurrent circulation of all four HCoVs, a quarter of which co-occurred with another virus and most of which were from children under the age of two years.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Tracking the Evolution of the SARS Coronavirus Using High-Throughput, High-Density Resequencing Arrays</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Mutations in the SARS-Coronavirus (SARS-CoV) can alter its clinical presentation, and the study of its mutation patterns in human populations can facilitate contact tracing. Here, we describe the development and validation of an oligonucleotide resequencing array for interrogating the entire 30-kb SARS-CoV genome in a rapid, cost-effective fashion. Using this platform, we sequenced SARS-CoV genomes from Vero cell culture isolates of 12 patients and directly from four patient tissues. The sequence obtained from the array is highly reproducible, accurate (>99.99% accuracy) and capable of identifying known and novel variants of SARS-CoV. Notably, we applied this technology to a field specimen of probable SARS and rapidly deduced its infectious source. We demonstrate that array-based resequencing-by-hybridization is a fast, reliable, and economical alternative to capillary sequencing for obtaining SARS-CoV genomic sequence on a population scale, making this an ideal platform for the global monitoring of SARS-CoV and other small-genome pathogens.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Metapneumovirus Infection in Japanese Children</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human metapneumovirus (hMPV) has been recently discovered as an etiological agent of acute respiratory infections. Our purpose was to asses the virological and clinical features of children with respiratory infections caused by hMPV. We examined 658 nasopharyngeal swab samples obtained from 637 children with respiratory infections for hMPV by using reverse transcription-PCR (RT-PCR). A total of 268 samples from 637 children were inoculated onto tertiary monkey kidney cells. A total of 36 serum samples (26 in the acute phase and 10 in the convalescent phase) from the 26 hMPV-positive children were tested for immunoglobulin G (IgG) and IgM antibodies to hMPV by using an indirect immunofluorescence assay. We detected hMPV in 57 (8.9%) of the 637 samples by using RT-PCR and isolated 7 (2.6%) hMPV strains of the 268 samples in cell cultures. A total of 12 (46.2%) of 26 hMPV-positive children were suspected to have primary infection with hMPV as determined by an indirect immunofluorescence assay. The infected children were diagnosed as having wheezy bronchitis (36.8%), upper respiratory tract infection (26.3%), bronchitis (22.8%), and pneumonia (14.0%). We showed that two hMPV groups were circulating in different regions during the same period and that reinfection with hMPV frequently occurs in childhood. The RT-PCR test is the most sensitive test for detection of hMPV, and a serological test may be useful to differentiate between primary infection and reinfection with hMPV.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world.With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Canine Enteric Coronaviruses: Emerging Viral Pathogens with Distinct Recombinant Spike Proteins</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 22</span></i><p style="text-align: justify; text-justify: inter-word;">Canine enteric coronavirus (CCoV) is an alphacoronavirus infecting dogs that is closely related to enteric coronaviruses of cats and pigs. While CCoV has traditionally caused mild gastro-intestinal clinical signs, there are increasing reports of lethal CCoV infections in dogs, with evidence of both gastrointestinal and systemic viral dissemination. Consequently, CCoV is now considered to be an emerging infectious disease of dogs. In addition to the two known serotypes of CCoV, novel recombinant variants of CCoV have been found containing spike protein N-terminal domains (NTDs) that are closely related to those of feline and porcine strains. The increase in disease severity in dogs and the emergence of novel CCoVs can be attributed to the high level of recombination within the spike gene that can occur during infection by more than one CCoV type in the same host.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology characteristics of human coronaviruses in patients with respiratory infection symptoms and phylogenetic analysis of HCoV-OC43 during 2010-2015 in Guangzhou</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus (HCoV) is one of the most common causes of respiratory tract infection throughout the world. To investigate the epidemiological and genetic variation of HCoV in Guangzhou, south China, we collected totally 13048 throat and nasal swab specimens from adults and children with fever and acute upper respiratory infection symptoms in Gunazhou, south China between July 2010 and June 2015, and the epidemiological features of HCoV and its species were studied. Specimens were screened for HCoV by real-time RT-PCR, and 7 other common respiratory viruses were tested simultaneously by PCR or real-time PCR. HCoV was detected in 294 cases (2.25%) of the 13048 samples, with most of them inpatients (251 cases, 85.4% of HCoV positive cases) and young children not in nursery (53.06%, 156 out of 294 HCoV positive cases). Four HCoVs, as OC43, 229E, NL63 and HKU1 were detected prevalent during 2010–2015 in Guangzhou, and among the HCoV positive cases, 60.20% were OC43, 16.67% were 229E, 14.97% were NL63 and 7.82% were HKU1. The month distribution showed that totally HCoV was prevalent in winter, but differences existed in different species. The 5 year distribution of HCoV showed a peak-valley distribution trend, with the detection rate higher in 2011 and 2013 whereas lower in 2010, 2012 and 2014. The age distribution revealed that children (especially those <3 years old) and old people (>50 years) were both high risk groups to be infected by HCoV. Of the 294 HCoV positive patients, 34.69% (101 cases) were co-infected by other common respiratory viruses, and influenza virus was the most common co-infecting virus (30/101, 29.70%). Fifteen HCoV-OC43 positive samples of 2013–2014 were selected for S gene sequencing and phylogenetic analysis, and the results showed that the 15 strains could be divided into 2 clusters in the phylogenetic tree, 12 strains of which formed a separate cluster that was closer to genotype G found in Malaysia. It was revealed for the first time that genotype B and genotype G of HCoV-OC43 co-circulated and the newly defined genotype G was epidemic as a dominant genotype during 2013–2014 in Guanzhou, south China.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world. With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS Transmission Pattern in Singapore Reassessed by Viral Sequence Variation Analysis</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Feb 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Epidemiological investigations of infectious disease are mainly dependent on indirect contact information and only occasionally assisted by characterization of pathogen sequence variation from clinical isolates. Direct sequence analysis of the pathogen, particularly at a population level, is generally thought to be too cumbersome, technically difficult, and expensive. We present here a novel application of mass spectrometry (MS)–based technology in characterizing viral sequence variations that overcomes these problems, and we apply it retrospectively to the severe acute respiratory syndrome (SARS) outbreak in Singapore. METHODS AND FINDINGS: The success rate of the MS-based analysis for detecting SARS coronavirus (SARS-CoV) sequence variations was determined to be 95% with 75 copies of viral RNA per reaction, which is sufficient to directly analyze both clinical and cultured samples. Analysis of 13 SARS-CoV isolates from the different stages of the Singapore outbreak identified nine sequence variations that could define the molecular relationship between them and pointed to a new, previously unidentified, primary route of introduction of SARS-CoV into the Singapore population. Our direct determination of viral sequence variation from a clinical sample also clarified an unresolved epidemiological link regarding the acquisition of SARS in a German patient. We were also able to detect heterogeneous viral sequences in primary lung tissues, suggesting a possible coevolution of quasispecies of virus within a single host. CONCLUSION: This study has further demonstrated the importance of improving clinical and epidemiological studies of pathogen transmission through the use of genetic analysis and has revealed the MS-based analysis to be a sensitive and accurate method for characterizing SARS-CoV genetic variations in clinical samples. We suggest that this approach should be used routinely during outbreaks of a wide variety of agents, in order to allow the most effective control.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of severe acute respiratory syndrome coronavirus genomes in Taiwan: Molecular epidemiology and genome evolution</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Since early March 2003, the severe acute respiratory syndrome (SARS) coronavirus (CoV) infection has claimed 346 cases and 37 deaths in Taiwan. The epidemic occurred in two stages. The first stage caused limited familial or hospital infections and lasted from early March to mid-April. All cases had clear contact histories, primarily from Guangdong or Hong Kong. The second stage resulted in a large outbreak in a municipal hospital, and quickly spread to northern and southern Taiwan from late April to mid-June. During this stage, there were some sporadic cases with untraceable contact histories. To investigate the origin and transmission route of SARS-CoV in Taiwan's epidemic, we conducted a systematic viral lineage study by sequencing the entire viral genome from ten SARS patients. SARS-CoV viruses isolated from Taiwan were found closely related to those from Guangdong and Hong Kong. In addition, all cases from the second stage belonged to the same lineage after the municipal hospital outbreak, including the patients without an apparent contact history. Analyses of these full-length sequences showed a positive selection occurring during SARS-CoV virus evolution. The mismatch distribution indicated that SARS viral genomes did not reach equilibrium and suggested a recent introduction of the viruses into human populations. The estimated genome mutation rate was ≈0.1 per genome, demonstrating possibly one of the lowest rates among known RNA viruses.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Seroepidemiological Study on SARS-CoV IgG Antibodies of Different Populations from Several Areas</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE: In order to investigate the clinical and epidemiological rules of severe acute respiratory syndrome (SARS), rates and levels SARS coronavirus (SARS-CoV) IgG antibodies of the patients and community populations from several areas were detected. METHODS: Indirect immunofluorescent assay (IFA) and double-antigen sandwich enzyme-linked immunosorbent assay (ELISA) were used to detect the SARS coronavirus-specific IgG antibodies in sera of 1700, including 1453 general populations from Hongkong, Marco, Guangzhou and Peking and 257 SARS patients from Guangzhou and Peking. The dynamics of the serum antibodies of SARS patients were observed from 3 to 360 days after onset of symptoms. RESULTS: 90% of 257 patient serum specimens after 20 days of disease onset showed positive SARS-CoV IgG either using ELISA or IFA. 257 SARS patients, antibodies titers increased steadily in early 4 to 6 months after onset of SARS. The titers of most cases came to the peak in the 6th month. then antibodies titers declined rapidly in some cases. However, all specimens still were positive for SARS-CoV IgG in the 48th month. CONCLUSIONS: This study suggest that few inapparent infectious patients exist during SARS epidemic. Serum IgG antibodies has diagnostic value for SARS in the late course of disease and the antibodies present more than 48 months.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Mutations, Recombination and Insertion in the Evolution of 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.  MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.  ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.  ConclusionsOur studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.  One Sentence SummaryNovel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequences of Four Novel Human Coronavirus OC43 Isolates Associated with Severe Acute Respiratory Infection</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 May 24</span></i><p style="text-align: justify; text-justify: inter-word;">We report here the complete genome sequences of four human coronavirus (HCoV) OC43 isolates generated using targeted viral nucleic acid capture and next-generation sequencing; the isolates were collected in New Mexico and Arkansas, USA, in February (HCoV-OC43/USA/TCNP_0070/2016) and March (HCoV-OC43/USA/ACRI_0052/2016) 2016 and January 2017 (HCoV-OC43/USA/TCNP_00204/2017 and HCoV-OC43/USA/TCNP_00212/2017).</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory Investigation and Phylogenetic Analysis of an Imported Middle East Respiratory Syndrome Coronavirus Case in Greece</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Apr 28</span></i><p style="text-align: justify; text-justify: inter-word;">Rapid and reliable laboratory diagnosis of persons suspected of Middle East respiratory syndrome coronavirus (MERS-CoV) infection is important for timely implementation of infection control practices and disease management. In addition, monitoring molecular changes in the virus can help elucidate chains of transmission and identify mutations that might influence virus transmission efficiency. This was illustrated by a recent laboratory investigation we conducted on an imported MERS-CoV case in Greece. Two oropharyngeal swab specimens were collected on the 1(st) and 2(nd) day of patient hospitalization and tested using two real-time RT-PCR (rRT-PCR) assays targeting the UpE and Orf-1a regions of the MERS-CoV genome and RT-PCR and partial sequencing of RNA-dependent RNA polymerase and nucleocapsid genes. Serum specimens were also collected and serological test were performed. Results from the first swab sample were inconclusive while the second swab was strongly positive for MERS-CoV RNA by rRT-PCR and confirmed positive by RT-PCR and partial gene sequencing. Positive serologic test results further confirmed MERS-CoV infection. Full-length nucleocapsid and spike gene coding sequences were later obtained from the positive swab sample. Phylogenetic analysis revealed that the virus was closely related to recent human-derived MERS-CoV strains obtained in Jeddah and Makkah, Saudi Arabia, in April 2014 and dromedary camels in Saudi Arabia and Qatar. These findings were consistent with the patient’s history. We also identified a unique amino acid substitution in the spike receptor binding domain that may have implications for receptor binding efficiency. Our initial inconclusive rRT-PCR results highlight the importance of collecting multiple specimens from suspect MERS-CoV cases and particularly specimens from the lower respiratory tract.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Moderate mutation rate in the SARS coronavirus genome and its implications</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Moderate mutation rate in the SARS coronavirus genome and its implications</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Moderate mutation rate in the SARS coronavirus genome and its implications</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jun 28</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the  2019-nCoV_N2, N3 of USA and the  ORF1ab of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background A novel coronavirus (2019-nCoV) associated with human to human transmission and severe human infection has been recently reported from the city of Wuhan in China. Our objectives were to characterize the genetic relationships of the 2019-nCoV and to search for putative recombination within the subgenus of sarbecovirus. Methods Putative recombination was investigated by RDP4 and Simplot v3.5.1 and discordant phylogenetic clustering in individual genomic fragments was confirmed by phylogenetic analysis using maximum likelihood and Bayesian methods. Results Our analysis suggests that the 2019-nCoV although closely related to BatCoV RaTG13 sequence throughout the genome (sequence similarity 96.3%), shows discordant clustering with the Bat_SARS-like coronavirus sequences. Specifically, in the 5′-part spanning the first 11,498 nucleotides and the last 3′-part spanning 24,341–30,696 positions, 2019-nCoV and RaTG13 formed a single cluster with Bat_SARS-like coronavirus sequences, whereas in the middle region spanning the 3′-end of ORF1a, the ORF1b and almost half of the spike regions, 2019-nCoV and RaTG13 grouped in a separate distant lineage within the sarbecovirus branch. Conclusions The levels of genetic similarity between the 2019-nCoV and RaTG13 suggest that the latter does not provide the exact variant that caused the outbreak in humans, but the hypothesis that 2019-nCoV has originated from bats is very likely. We show evidence that the novel coronavirus (2019-nCov) is not-mosaic consisting in almost half of its genome of a distinct lineage within the betacoronavirus. These genomic features and their potential association with virus characteristics and virulence in humans need further attention.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Singapore claims first use of antibody test to track coronavirus infections | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In what appears to be a first, disease trackers in Singapore have used an experimental antibody test for COVID-19 to confirm that a suspected patient was infected with the coronavirus. The patient was one of two people who together formed a missing link between two clusters of cases that each occurred in a Singaporean church. Researchers around the world are racing to develop antibody tests, also called serological tests, that can confirm whether someone was infected even after their immune system has cleared the virus that causes COVID-19. The group that developed the test, at Duke-NUS Medical School in Singapore, is among the front-runners, although its assay has to be validated before it is taken into production and deployed widely.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic RNA sequence of feline coronavirus strain FCoV C1Je</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">This paper reports the first genomic RNA sequence of a field strain feline coronavirus (FCoV). Viral RNA was isolated at post mortem from the jejunum and liver of a cat with feline infectious peritonitis (FIP). A consensus sequence of the jejunum-derived genomic RNA (FCoV C1Je) was determined from overlapping cDNA fragments produced by reverse transcriptase polymerase chain reaction (RT-PCR) amplification. RT-PCR products were sequenced by a reiterative sequencing strategy and the genomic RNA termini were determined using a rapid amplification of cDNA ends PCR strategy. The FCoV C1Je genome was found to be 29,255 nucleotides in length, excluding the poly(A) tail. Comparison of the FCoV C1Je genomic RNA sequence with that of the laboratory strain FCoV FIP virus (FIPV) 79-1146 showed that both viruses have a similar genome organisation and predictions made for the open reading frames and cis-acting elements of the FIPV 79-1146 genome hold true for FCoV C1Je. In addition, the sequence of the 3′-proximal third of the liver derived genomic RNA (FCoV C1Li), which encompasses the structural and accessory protein genes of the virus, was also determined. Comparisons of the enteric (jejunum) and non-enteric (liver) derived viral RNA sequences revealed 100% nucleotide identity, a finding that questions the well accepted ‘internal mutation theory’ of FIPV pathogenicity.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19, Australia: Epidemiology Report 2 (Reporting week ending 19:00 AEDT 8 February 2020)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">This is the second epidemiological report for coronavirus disease (COVID-19), previously known as novel coronavirus (2019-nCoV), reported in Australia as at 19:00 Australian Eastern Daylight Time [AEDT] 8 February 2020. It includes data on Australian cases notified during the week ending 19:00 AEDT 8 February 2020, the international situation and current information on the severity, transmission and spread of the COVID-19 infection.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /></body>