
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: bioinformatics for a target regions of genome</h2><br /></div><br /><br /><br /><div class="card"><h3>Citation: Targeted NGS Platforms for Genetic Screening and Gene Discovery in Primary Immunodeficiencies</h3>
<p style="text-align: justify; text-justify: inter-word;">Objective: To evaluate the strength and weakness of targeted NGS sequencing using custom-made Ion Torrent and Haloplex (Agilent) panels for diagnostics and research purposes.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Mathematical Analysis of Copy Number Variation in a DNA Sample Using Digital PCR on a Nanofluidic Device</h3>
<p style="text-align: justify; text-justify: inter-word;">Copy Number Variations (CNVs) of regions of the human genome have been associated with multiple diseases. We present an algorithm which is mathematically sound and computationally efficient to accurately analyze CNV in a DNA sample utilizing a nanofluidic device, known as the digital array. This numerical algorithm is utilized to compute copy number variation and the associated statistical confidence interval and is based on results from probability theory and statistics. We also provide formulas which can be used as close approximations.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparisons of substitution, insertion and deletion probes for resequencing and mutational analysis using oligonucleotide microarrays</h3>
<p style="text-align: justify; text-justify: inter-word;">Although oligonucleotide probes complementary to single nucleotide substitutions are commonly used in microarray-based screens for genetic variation, little is known about the hybridization properties of probes complementary to small insertions and deletions. It is necessary to define the hybridization properties of these latter probes in order to improve the specificity and sensitivity of oligonucleotide microarray-based mutational analysis of disease-related genes. Here, we compare and contrast the hybridization properties of oligonucleotide microarrays consisting of 25mer probes complementary to all possible single nucleotide substitutions and insertions, and one and two base deletions in the 9168 bp coding region of the ATM (ataxia telangiectasia mutated) gene. Over 68 different dye-labeled single-stranded nucleic acid targets representing all ATM coding exons were applied to these microarrays. We assess hybridization specificity by comparing the relative hybridization signals from probes perfectly matched to ATM sequences to those containing mismatches. Probes complementary to two base substitutions displayed the highest average specificity followed by those complementary to single base substitutions, single base deletions and single base insertions. In all the cases, hybridization specificity was strongly influenced by sequence context and possible intra-and intermolecular probe and/or target structure. Furthermore, single nucleotide substitution probes displayed the most consistent hybridization specificity data followed by single base deletions, two base deletions and single nucleotide insertions. Overall, these studies provide valuable empirical data that can be used to more accurately model the hybridization properties of insertion and deletion probes and improve the design and interpretation of oligonucleotide microarray-based resequencing and mutational analysis. e33 of specific sequence variants by sequencing even when dealing with slightly ambiguous hybridization data.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Genomics Microarray-based identification of antigenic variants of foot-and-mouth disease virus: a bioinformatics quality assessment</h3>
<p style="text-align: justify; text-justify: inter-word;">A FMDV microarray was designed and optimized to detect SNPs at a major antigenic site of the virus. A screening of point mutants of the genomic region encoding antigenic site A of FMDV C-S8c1 was achieved. The hybridization pattern of a mutant includes specific positive and negative signals as well as crosshybridization signals, which are of different intensity depending on the thermodynamic stability of each probe-target pair. Moreover, an array bioinformatic classification method was developed to evaluate the hybridization signals. This statistical analysis shows that the procedure allows a very accurate classification per variant genome.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated degenerate PCR primer design for high-throughput sequencing improves efficiency of viral sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: Augmenting our previously developed and published JCVI Primer Design Pipeline, we achieved similarly high sequencing success rates with only minor software modifications. The recommended methodology for the construction of the consensus sequence that encapsulates the allelic variation of the targeted population and is a key step prior to designing degenerate primers is also formally described.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Virology Journal Universal primers that amplify RNA from all three flavivirus subgroups</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: Bioinformatic analysis of 257 published full-length Flavivirus genomes revealed conserved regions not previously targeted by primers. Two degenerate primers, Flav100F and Flav200R were designed from these regions and used to generate an 800 base pair cDNA product. The region amplified encoded part of the methyltransferase and most of the RNA-dependent-RNApolymerase (NS5) coding sequence. One-step RT-PCR testing was successful using standard conditions with RNA from over 60 different flavivirus strains representing about 50 species. The cDNA from each virus isolate was sequenced then used in phylogenetic analyses and database searches to confirm the identity of the template RNA.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulate_PCR for amplicon prediction and annotation from multiplex, degenerate primers and probes</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Pairing up primers to amplify desired targets and avoid undesired cross reactions can be a combinatorial challenge. Effective prediction of specificity and inclusivity from multiplexed primers and TaqMan®/ Luminex® probes is a critical step in PCR design. Results: Code is described to identify all primer and probe combinations from a list of unpaired, unordered candidates that should produce a product. It predicts and extracts all amplicon sequences in a large sequence database from a list of primers and probes, allowing degenerate bases and user-specified levels of primer-target mismatch tolerance. Amplicons hit by TaqMan®/Luminex® probes are indicated, and products may be annotated with gene information from NCBI. Fragment length distributions are calculated to predict electrophoretic gel banding patterns.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Accurate prediction of functional, structural, and stability changes in PITX2 mutations using in silico bioinformatics algorithms</h3>
<p style="text-align: justify; text-justify: inter-word;">Mutations in PITX2 have been implicated in several genetic disorders, particularly Axenfeld-Rieger syndrome. In order to determine the most reliable bioinformatics tools to assess the likely pathogenicity of PITX2 variants, the results of bioinformatics predictions were compared to the impact of variants on PITX2 structure and function. The MutPred, Provean, and PMUT bioinformatic tools were found to have the highest performance in predicting the pathogenicity effects of all 18 characterized missense variants in PITX2, all with sensitivity and specificity >93%. Applying these three programs to assess the likely pathogenicity of 13 previously uncharacterized PITX2 missense variants predicted 12/13 variants as deleterious, except A30V which was predicted as benign variant for all programs. Molecular modeling of the PITX2 homoedomain predicts that of the 31 known PITX2 variants, L54Q, F58L, V83F, V83L, W86C, W86S, and R91P alter PITX2's structure. In contrast, the remaining 24 variants are not predicted to change PITX2's structure. The results of molecular modeling, performed on all the PITX2 missense mutations located in the homeodomain, were compared with the findings of eight protein stability programs. CUPSAT was found to be the most reliable in predicting the effect of missense mutations on PITX2 stability. Our results showed that for PITX2, and likely other members of this homeodomain transcription factor family, MutPred, Provean, PMUT, molecular modeling, and CUPSAT can reliably be used to predict PITX2 missense variants pathogenicity.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved Algorithmic Complexity for the 3SEQ Recombination Detection Algorithm</h3>
<p style="text-align: justify; text-justify: inter-word;">Identifying recombinant sequences in an era of large genomic databases is challenging as it requires an efficient algorithm to identify candidate recombinants and parents, as well as appropriate statistical methods to correct for the large number of comparisons performed. In 2007, a computation was introduced for an exact nonparametric mosaicism statistic that gave high-precision P values for putative recombinants. This exact computation meant that multiplecomparisons corrected P values also had high precision, which is crucial when performing millions or billions of tests in large databases. Here, we introduce an improvement to the algorithmic complexity of this computation from O(mn 3 ) to O(mn 2 ), where m and n are the numbers of recombination-informative sites in the candidate recombinant. This new computation allows for recombination analysis to be performed in alignments with thousands of polymorphic sites. Benchmark runs are presented on viral genome sequence alignments, new features are introduced, and applications outside recombination analysis are discussed.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>An optimized microarray platform for assaying genomic variation in Plasmodium falciparum field populations</h3>
<p style="text-align: justify; text-justify: inter-word;">We present an optimized probe design for copy number variation (CNV) and SNP genotyping in the Plasmodium falciparum genome. We demonstrate that variable length and isothermal probes are superior to static length probes. We show that sample preparation and hybridization conditions mitigate the effects of host DNA contamination in field samples. The microarray and workflow presented can be used to identify CNVs and SNPs with 95% accuracy in a single hybridization, in field samples containing up to 92% human DNA contamination.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>ANDES: Statistical tools for the ANalyses of DEep Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">As new sequencing technologies evolve, deep sequencing will become increasingly cost-efficient and the inter and intra-sample comparisons of largely homogeneous sequences will become more common. We have provided a software package and demonstrated its application on various empirically-derived datasets. Investigators may download the software from Sourceforge at https://sourceforge.net/projects/andestools.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>An analysis of the feasibility of short read sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Several methods for ultra high-throughput DNA sequencing are currently under investigation. Many of these methods yield very short blocks of sequence information (reads). Here we report on an analysis showing the level of genome sequencing possible as a function of read length. It is shown that resequencing and de novo sequencing of the majority of a bacterial genome is possible with read lengths of 20-30 nt, and that reads of 50 nt can provide reconstructed contigs (a contiguous fragment of sequence data) of 1000 nt and greater that cover 80% of human chromosome 1.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Defective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both in vitro and in vivo. However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of a subset of RSV cbDVG species. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus unable to produce a subset of cbDVGs due to mutations introduced in this sequence. The identified sequence was also found as a site for cbDVG generation during natural RSV infections, and common cbDVGs originated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome determine the location of cbDVG formation and, therefore, the generation of cbDVGs is not a stochastic process. These findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>GenomeLandscaper: Landscape analysis of genome-fingerprints maps assessing chromosome architecture OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Assessing correctness of an assembled chromosome architecture is a central challenge. We create a geometric analysis method (called GenomeLandscaper) to conduct landscape analysis of genomefingerprints maps (GFM), trace large-scale repetitive regions, and assess their impacts on the global architectures of assembled chromosomes. We develop an alignment-free method for phylogenetics analysis. The human Y chromosomes (GRCh.chrY, HuRef.chrY and YH.chrY) are analysed as a proofof-concept study. We construct a galaxy of genome-fingerprints maps (GGFM) for them, and a landscape compatibility among relatives is observed. But a long sharp straight line on the GGFM breaks such a landscape compatibility, distinguishing GRCh38p1.chrY (and throughout GRCh38p7. chrY) from GRCh37p13.chrY, HuRef.chrY and YH.chrY. We delete a 1.30-Mbp target segment to rescue the landscape compatibility, matching the antecedent GRCh37p13.chrY. We re-locate it into the modelled centromeric and pericentromeric region of GRCh38p10.chrY, matching a gap placeholder of GRCh37p13.chrY. We decompose it into sub-constituents (such as BACs, interspersed repeats, and tandem repeats) and trace their homologues by phylogenetics analysis. We elucidate that most examined tandem repeats are of reasonable quality, but the BAC-sized repeats, 173U1020C (176.46 Kbp) and 5U41068C (205.34 Kbp), are likely over-repeated. These results offer unique insights into the centromeric and pericentromeric regions of the human Y chromosomes.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>A reverse genetics system for avian coronavirus infectious bronchitis virus based on targeted RNA recombination</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: Targeted RNA recombination allowed for the modification of the 3′-end of the IBV genome, encoding all structural and accessory genes. A wild-type recombinant IBV was constructed, containing several synonymous marker mutations. The in ovo growth kinetics and in vivo characteristics of the recombinant virus were similar to those of the parental IBV strain H52. Conclusions: Targeted RNA recombination allows for the generation of recombinant IBV strains that are not able to infect and propagate in continuous cell lines. The ability to introduce specific mutations holds promise for the development of rationally designed live-attenuated IBV vaccines and for studies into the biology of IBV in general.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into N-calls of mitochondrial DNA sequencing using MitoChip v2.0</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Developments in DNA resequencing microarrays include mitochondrial DNA (mtDNA) sequencing and mutation detection. Failure by the microarray to identify a base, compared to the reference sequence, is designated an 'N-call.' This study re-examined the N-call distribution of mtDNA samples sequenced by the Affymetrix MitoChip v.2.0, based on the hypothesis that N-calls may represent insertions or deletions (indels) in mtDNA. Findings: We analysed 16 patient mtDNA samples using MitoChip. N-calls by the proprietary GSEQ software were significantly reduced when either of the freeware on-line algorithms ResqMi or sPROFILER was utilized. With sPROFILER, this decrease in N-calls had no effect on the homoplasmic or heteroplasmic mutation levels compared to GSEQ software, but ResqMi produced a significant change in mutation load, as well as producing longer N-cell stretches. For these reasons, further analysis using ResqMi was not attempted. Conventional DNA sequencing of the longer N-calls stretches from sPROFILER revealed 7 insertions and 12 point mutations. Moreover, analysis of single-base N-calls of one mtDNA sample found 3 other point mutations. Conclusions: Our study is the first to analyse N-calls produced from GSEQ software for the MitoChipv2.0. By narrowing the focus to longer stretches of N-calls revealed by sPROFILER, conventional sequencing was able to identify unique insertions and point mutations. Shorter N-calls also harboured point mutations, but the absence of deletions among N-calls suggests that probe confirmation affects binding and thus N-calling. This study supports the contention that the GSEQ is more capable of assigning bases when used in conjunction with sPROFILER.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>VIRsiRNApred: a web server for predicting inhibition efficacy of siRNAs targeting human viruses</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: During 10-fold cross validation on T 1380 using hybrid approach, we achieved a maximum Pearson Correlation Coefficient (PCC) of 0.55 between predicted and actual efficacy of viral siRNAs. On V 345 independent dataset, our best model achieved a maximum correlation of 0.50 while existing general siRNA prediction methods showed PCC from 0.05 to 0.18. However, using leave one out cross validation PCC was improved to 0.58 and 0.55 on training and validation datasets respectively. SVM performed better than other machine learning techniques used like ANN, KNN and REP Tree.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Information theory-based algorithm for in silico prediction of PCR products with whole genomic sequences as templates</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: Primer sequence is converted to a vector of the full potential hydrogen numbers (3 for G or C, 2 for A or T), while template sequence is converted to a vector of the actual hydrogen bond numbers formed after primer annealing. The former is considered as source information and the latter destination information. An information coefficient is calculated as a measure for fidelity of this information transfer process and thus a measure of similarity between primer and potential annealing site on template.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryptococcus gattii VGIIb-like Variant in White-Tailed Deer, Nova Scotia, Canada</h3>
<p style="text-align: justify; text-justify: inter-word;">In conclusion, we suggest that an accurate NGSbased prediction requires an integrative approach to all relevant information obtained by in silico analysis of a given genome locus. In particular, not only presence of CRISPR spacers but also presence and location of potentially interfering IS6110 insertion(s) should be considered for correct NGS-based assignment to internationally recognized spoligotypes.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>High-throughput detection of mutations responsible for childhood hearing loss using resequencing microarrays</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Despite current knowledge of mutations in 45 genes that can cause nonsyndromic sensorineural hearing loss (SNHL), no unified clinical test has been developed that can comprehensively detect mutations in multiple genes. We therefore designed Affymetrix resequencing microarrays capable of resequencing 13 genes mutated in SNHL (GJB2, GJB6, CDH23, KCNE1, KCNQ1, MYO7A, OTOF, PDS, MYO6, SLC26A5, TMIE, TMPRSS3, USH1C). We present results from hearing loss arrays developed in two different research facilities and highlight some of the approaches we adopted to enhance the applicability of resequencing arrays in a clinical setting. Results: We leveraged sequence and intensity pattern features responsible for diminished coverage and accuracy and developed a novel algorithm, sPROFILER, which resolved >80% of no-calls from GSEQ and allowed 99.6% (range: 99.2-99.8%) of sequence to be called, while maintaining overall accuracy at >99.8% based upon dideoxy sequencing comparison. Conclusions: Together, these findings provide insight into critical issues for disease-centered resequencing protocols suitable for clinical application and support the use of array-based resequencing technology as a valuable molecular diagnostic tool for pediatric SNHL and other genetic diseases with substantial genetic heterogeneity.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /></body>