
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: can livestock serve as a reservoir after the COVID-19 epidemic appears to be over</h2><br /></div><br /><br /><br /><div class="card"><h3>The timing of one-shot interventions for epidemic control</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The apparent early success in China's large-scale intervention to control the COVID-19 epidemic has led to interest in whether other countries can replicate it as well as concerns about a resurgence of the epidemic if or when China relaxes the interventions.  In this paper we look at the impact of a single short-term intervention on an epidemic.  We see that if an intervention cannot be sustained long-term, it has the greatest impact if it is imposed once infection levels have become large enough that there is an appreciable number of infections present.  For minimising the total number infected it should start close to the peak so that there is no rebound once the intervention is stopped, while to minimise the peak prevalence, it should start earlier, allowing two peaks of comparable size rather than one very large peak.  In populations with distinct subgroups, synchronized interventions are less effective than targeting the interventions in each sub-population separately.      We do not attempt to clearly determine what makes an intervention sustainable or not.  We believe that is a policy question.  If an intervention is sustainable, it should be kept in place.  Our intent is to offer insight into how best to time an intervention whose impact on society is too great to maintain.</jats:p></p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Inoculation of Goats, Sheep, and Horses with MERS-CoV Does Not Result in Productive Viral Shedding</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug 19</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first recognized in 2012 and can cause severe disease in infected humans. Dromedary camels are the reservoir for the virus, although, other than nasal discharge, these animals do not display any overt clinical disease. Data from in vitro experiments suggest that other livestock such as sheep, goats, and horses might also contribute to viral transmission, although field data has not identified any seropositive animals. In order to understand if these animals could be infected, we challenged young goats and horses and adult sheep with MERS-CoV by intranasal inoculation. Minimal or no virus shedding was detected in all of the animals. During the four weeks following inoculation, neutralizing antibodies were detected in the young goats, but not in sheep or horses.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Searching for animal models and potential target species for emerging pathogens: Experience gained from Middle East respiratory syndrome (MERS) coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>A Well Infant with Coronavirus Disease 2019 (COVID-19) with High Viral Load</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A well 6-month-old infant with coronavirus disease 2019 (COVID-19) had persistently positive nasopharyngeal swabs to day 16 of admission. This case highlights the difficulties in establishing the true incidence of COVID-19 as asymptomatic individuals can excrete the virus. These patients may play important roles in human-to-human transmission in the community.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Domestic Pig Unlikely Reservoir for MERS-CoV</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We tested the suitability of the domestic pig as a model for Middle East respiratory syndrome coronavirus (MERS-CoV) infection. Inoculation did not cause disease, but a low level of virus replication, shedding, and seroconversion were observed. Pigs do not recapitulate human MERS-CoV and are unlikely to constitute a reservoir in nature.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Livestock Susceptibility to Infection with Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) cases continue to be reported, predominantly in Saudi Arabia and occasionally other countries. Although dromedaries are the main reservoir, other animal species might be susceptible to MERS coronavirus (MERS-CoV) infection and potentially serve as reservoirs. To determine whether other animals are potential reservoirs, we inoculated MERS-CoV into llamas, pigs, sheep, and horses and collected nasal and rectal swab samples at various times. The presence of MERS-CoV in the nose of pigs and llamas was confirmed by PCR, titration of infectious virus, immunohistochemistry, and in situ hybridization; seroconversion was detected in animals of both species. Conversely, in sheep and horses, virus-specific antibodies did not develop and no evidence of viral replication in the upper respiratory tract was found. These results prove the susceptibility of llamas and pigs to MERS-CoV infection. Thus, the possibility of MERS-CoV circulation in animals other than dromedaries, such as llamas and pigs, is not negligible.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Livestock Susceptibility to Infection with Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) cases continue to be reported, predominantly in Saudi Arabia and occasionally other countries. Although dromedaries are the main reservoir, other animal species might be susceptible to MERS coronavirus (MERS-CoV) infection and potentially serve as reservoirs. To determine whether other animals are potential reservoirs, we inoculated MERS-CoV into llamas, pigs, sheep, and horses and collected nasal and rectal swab samples at various times. The presence of MERS-CoV in the nose of pigs and llamas was confirmed by PCR, titration of infectious virus, immunohistochemistry, and in situ hybridization; seroconversion was detected in animals of both species. Conversely, in sheep and horses, virus-specific antibodies did not develop and no evidence of viral replication in the upper respiratory tract was found. These results prove the susceptibility of llamas and pigs to MERS-CoV infection. Thus, the possibility of MERS-CoV circulation in animals other than dromedaries, such as llamas and pigs, is not negligible.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Is COVID-19 Receiving ADE From Other Coronaviruses?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">One of the most perplexing questions regarding the current COVID-19 coronavirus epidemic is the discrepancy between the severity of cases observed in the Hubei province of China and those occurring elsewhere in the world. One possible answer is antibody dependent enhancement (ADE) of SARS-CoV-2 due to prior exposure to other coronaviruses. ADE modulates the immune response and can elicit sustained inflammation, lymphopenia, and/or cytokine storm, one or all of which have been documented in severe cases and deaths. ADE also requires prior exposure to similar antigenic epitopes, presumably circulating in local viruses, making it a possible explanation for the observed geographic limitation of severe cases and deaths.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential Intermediate Hosts for Coronavirus Transmission: No Evidence of Clade 2c Coronaviruses in Domestic Livestock from Ghana</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 10</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV), nearly a decade ago with worldwide distribution, was believed to be of zoonotic origin from bats with dromedary camels as intermediate hosts. There is a likelihood of other domestic livestock serving as intermediate hosts for this virus. The presence of coronaviruses, closely related to MERS-CoV in Ghanaian bats, presented the opportunity to test the hypothesis of transmissibility of this virus through domestic livestock species. The possible interactions between livestock and bats in 31 household farms were accessed by observation and interviews with farmers. Rectal swabs and serum from cattle, sheep, goats, donkeys, and swine from commercial and household farms were tested for MERS-CoV and a Nycteris sp. bat coronavirus, previously detected in Ghana. A pan-PCR assay to detect clade 2c viruses and recombinant immunofluorescence assay to detect anti-spike IgG antibodies against the target viruses were used. Likely contact between livestock and bats was determined for 13 farms (41.9%) that reported confining their livestock and also observing bats in their homes. Livestock were left unconfined on eight farms (25.8%) that also observed bats roosting in trees close to their homes. No viral RNA or antibodies against the two coronaviruses were detected in any of the livestock species tested. Cattle, sheep, goats, donkeys, and swine are not likely hosts of clade 2c coronaviruses.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>[Advances on presymptomatic or asymptomatic carrier transmission of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is rapidly spreading. Patients in incubation period and healthy carriers are possible sources for transmission. However, such sources of infection cannot be effectively identified due to the symptoms absent. The research evidence is very lacking so far, although there are a few studies suggesting that presymptomatic or asymptomatic carrier may cause COVID-19 transmission. Nearly half of the literature is in the state of preprint without peer review. The question of "the degree to which presymptomatic or asymptomatic infections can transmit" is not fully understood. There is an urgent need to screen infected carriers in larger close contacts or in the general population, and assess their risk for transmission.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy.  Methods We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level.  Results   The median age of the cases was 44 years, with an increasing of cases in younger age groups and the elderly as the epidemic progressed. The delay from symptom onset to hospital admission decreased from 4.4 days (95%CI: 0.0-14.0) until January 27 to 2.6 days (0.0-9.0) from January 28 to February 17. The mean incubation period was estimated at 5.2 days (1.8-12.4) and the mean serial interval at 5.1 days (1.3-11.6). The epidemic dynamics in provinces outside Hubei was highly variable, but consistently included a mix of case importations and local transmission. We estimate that the epidemic was self-sustained for less than three weeks with Rt reaching peaks between 1.40 (1.04-1.85) in Shenzhen City of Guangdong Province and 2.17 (1.69-2.76) in Shandong Province. In all the analyzed locations (n=10) Rt was estimated to be below the epidemic threshold since the end of January.  Conclusion   Our findings suggest that the strict containment measures and movement restrictions in place may contribute to the interruption of local COVID-19 transmission outside Hubei Province. The shorter serial interval estimated here implies that transmissibility is not as high as initial estimates suggested.</jats:p></p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the Asymptomatic Proportion of 2019 Novel Coronavirus onboard the Princess Cruises Ship, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The potential infectiousness of asymptomatic COVID-19 cases together with a substantial fraction of asymptomatic infections among all infections, have been highlighted in clinical studies. We conducted statistical modeling analysis to derive the delay-adjusted asymptomatic proportion of the positive COVID-19 infections onboard the Princess Cruises ship along with the timeline of infections. We estimated the asymptomatic proportion at 17.9% (95% CrI: 15.5%-20.2%), with most of the infections occurring before the start of the 2-week quarantine.</jats:p></p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Susceptibility of Pigs and Chickens to SARS Coronavirus</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of severe acute respiratory syndrome (SARS) in humans, associated with a new coronavirus, was reported in Southeast Asia, Europe, and North America in early 2003. To address speculations that the virus originated in domesticated animals, or that domestic species were susceptible to the virus, we inoculated 6-week-old pigs and chickens intravenously, intranasally, ocularly, and orally with 10(6) PFU of SARS-associated coronavirus (SARS-CoV). Clinical signs did not develop in any animal, nor were gross pathologic changes evident on postmortem examinations. Attempts at virus isolation were unsuccessful; however, viral RNA was detected by reverse transcriptase-polymerase chain reaction in blood of both species during the first week after inoculation, and in chicken organs at 2 weeks after inoculation. Virus-neutralizing antibodies developed in the pigs. Our results indicate that these animals do not play a role as amplifying hosts for SARS-CoV.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Domestic Poultry and SARS Coronavirus, Southern China</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">SARS coronavirus injected intratracheally into chickens, turkeys, geese, ducks, and quail, or into the allantoic sac of their embryonating eggs, failed to cause disease or replicate. This finding suggests that domestic poultry were unlikely to have been the reservoir, or associated with dissemination, of SARS coronavirus in the animal markets of southern China.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficacy of an Adjuvanted Middle East Respiratory Syndrome Coronavirus Spike Protein Vaccine in Dromedary Camels and Alpacas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV is present in dromedary camels throughout the Middle East and Africa. Dromedary camels are the primary zoonotic reservoir for human infections. Interruption of the zoonotic transmission chain from camels to humans, therefore, may be an effective strategy to control the ongoing MERS-CoV outbreak. Here we show that vaccination with an adjuvanted MERS-CoV Spike protein subunit vaccine confers complete protection from MERS-CoV disease in alpaca and results in reduced and delayed viral shedding in the upper airways of dromedary camels. Protection in alpaca correlates with high serum neutralizing antibody titers. Lower titers of serum neutralizing antibodies correlate with delayed and significantly reduced shedding in the nasal turbinates of dromedary camels. Together, these data indicate that induction of robust neutralizing humoral immune responses by vaccination of naïve animals reduces shedding that potentially could diminish the risk of zoonotic transmission.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficacy of an Adjuvanted Middle East Respiratory Syndrome Coronavirus Spike Protein Vaccine in Dromedary Camels and Alpacas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV is present in dromedary camels throughout the Middle East and Africa. Dromedary camels are the primary zoonotic reservoir for human infections. Interruption of the zoonotic transmission chain from camels to humans, therefore, may be an effective strategy to control the ongoing MERS-CoV outbreak. Here we show that vaccination with an adjuvanted MERS-CoV Spike protein subunit vaccine confers complete protection from MERS-CoV disease in alpaca and results in reduced and delayed viral shedding in the upper airways of dromedary camels. Protection in alpaca correlates with high serum neutralizing antibody titers. Lower titers of serum neutralizing antibodies correlate with delayed and significantly reduced shedding in the nasal turbinates of dromedary camels. Together, these data indicate that induction of robust neutralizing humoral immune responses by vaccination of naïve animals reduces shedding that potentially could diminish the risk of zoonotic transmission.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">What is COVID-19, and what do we know so far about its clinical presentation? The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1]. The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host [2]. Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>The ongoing crises in China illustrate that the assessment of epidemics in isolation is no longer sufficient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary The interplay of simultaneous COVID-19, African swine fever, and avian influenza emergencies on global health and industries is constantly evolving and difficult to predict, and therefore warrants further scrutiny. The need for a health network of global scope for the rapid and open exchange of information needs to be strengthened in order to address ongoing and future epidemics under competing resources.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Co‐localization of Middle East respiratory syndrome coronavirus (MERS‐CoV) and dipeptidyl peptidase‐4 in the respiratory tract and lymphoid tissues of pigs and llamas</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 28</span></i><p style="text-align: justify; text-justify: inter-word;">This study investigated the co‐localization of the Middle East respiratory syndrome coronavirus (MERS‐CoV) and its receptor dipeptidyl peptidase‐4 (DPP4) by immunohistochemistry (IHC) across respiratory and lymphoid organs of experimentally MERS‐CoV infected pigs and llamas. Also, scanning electron microscopy was performed to assess the ciliary integrity of respiratory epithelial cells in both species. In pigs, on day 2 post‐inoculation (p.i.), DPP4‐MERS‐CoV co‐localization was detected in medial turbinate epithelium. On day 4 p.i., the virus/receptor co‐localized in frontal and medial turbinate epithelial cells in pigs, and epithelial cells distributed unevenly through the whole nasal cavity and in the cervical lymph node in llamas. MERS‐CoV viral nucleocapsid was mainly detected in upper respiratory tract sites on days 2 and 4 p.i. in pigs and day 4 p.i. in llamas. No MERS‐CoV was detected on day 24 p.i. in any tissue by IHC. While pigs showed severe ciliary loss in the nasal mucosa both on days 2 and 4 p.i. and moderate loss in the trachea on days 4 and 24 p.i., ciliation of respiratory organs in llamas was not significantly affected. Obtained data confirm the role of DPP4 for MERS‐CoV entry in respiratory epithelial cells of llamas. Notably, several nasal epithelial cells in pigs were found to express viral antigen but not DPP4, suggesting the possible existence of other molecule/s facilitating virus entry or down regulation of DPP4 upon infection.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Tropical Medicine & International Health March 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Editorial An editorial on the COVID-19 epidemic summarizes the current knowledge about the disease and the novel SARS-CoV-2 virus. In particular, characteristics of the virus and treatment options are addressed</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission interval estimates suggest pre-symptomatic spread of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods:  We collected outbreak information from Singapore and Tianjin,  China,  reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring.  Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Host Species Restriction of Middle East Respiratory Syndrome Coronavirus through Its Receptor, Dipeptidyl Peptidase 4</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) emerged in 2012. Recently, the MERS-CoV receptor dipeptidyl peptidase 4 (DPP4) was identified and the specific interaction of the receptor-binding domain (RBD) of MERS-CoV spike protein and DPP4 was determined by crystallography. Animal studies identified rhesus macaques but not hamsters, ferrets, or mice to be susceptible for MERS-CoV. Here, we investigated the role of DPP4 in this observed species tropism. Cell lines of human and nonhuman primate origin were permissive of MERS-CoV, whereas hamster, ferret, or mouse cell lines were not, despite the presence of DPP4. Expression of human DPP4 in nonsusceptible BHK and ferret cells enabled MERS-CoV replication, whereas expression of hamster or ferret DPP4 did not. Modeling the binding energies of MERS-CoV spike protein RBD to DPP4 of human (susceptible) or hamster (nonsusceptible) identified five amino acid residues involved in the DPP4-RBD interaction. Expression of hamster DPP4 containing the five human DPP4 amino acids rendered BHK cells susceptible to MERS-CoV, whereas expression of human DPP4 containing the five hamster DPP4 amino acids did not. Using the same approach, the potential of MERS-CoV to utilize the DPP4s of common Middle Eastern livestock was investigated. Modeling of the DPP4 and MERS-CoV RBD interaction predicted the ability of MERS-CoV to bind the DPP4s of camel, goat, cow, and sheep. Expression of the DPP4s of these species on BHK cells supported MERS-CoV replication. This suggests, together with the abundant DPP4 presence in the respiratory tract, that these species might be able to function as a MERS-CoV intermediate reservoir. IMPORTANCE The ongoing outbreak of Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 701 laboratory-confirmed cases to date, with 249 fatalities. Although bats and dromedary camels have been identified as potential MERS-CoV hosts, the virus has so far not been isolated from any species other than humans. The inability of MERS-CoV to infect commonly used animal models, such as hamster, mice, and ferrets, indicates the presence of a species barrier. We show that the MERS-CoV receptor DPP4 plays a pivotal role in the observed species tropism of MERS-CoV and subsequently identified the amino acids in DPP4 responsible for this restriction. Using a combined modeling and experimental approach, we predict that, based on the ability of MERS-CoV to utilize the DPP4 of common Middle East livestock species, such as camels, goats, sheep, and cows, these form a potential MERS-CoV intermediate host reservoir species.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel corona virus disease (COVID-19) in pregnancy: What clinical recommendations to follow?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Interim guidance has been issued by the World Health Organization (WHO) and Centers for Disease Control and Prevention (CDC) on managing COVID-19, which include some recommendations specific to pregnant women mostly drawn on experience from previous coronavirus outbreaks.8, 9 Chinese expert recommendations for the care of pregnant women with suspected and confirmed COVID-9 were developed and disseminated in China quite early following the outbreak in Wuhan.10 These recommendations have been dynamic, evolving as more knowledge about epidemiology, pathogenesis, disease progression and clinical course among infected pregnant patients has been gathered. Limited clinical experience in managing pregnant women with COVID-19 and their neonates has been reported from China recently based on a case series of nine pregnancies with confirmed COVID-19 treated in Zhongnan Hospital of Wuhan University and 10 neonates (nine pregnancies) delivered at five different hospitals,11, 12 although many more cases (>100) of suspected or confirmed COVID-19 have been treated and delivered in several hospitals in China according to the news releases and media reports. So far, no maternal deaths have been reported. There appears to be some risk of premature rupture of membranes, preterm delivery, fetal tachycardia and fetal distress when the infection occurs in the third trimester of pregnancy. However, there is no evidence suggesting transplacental transmission based on very limited data, as the analysis of amniotic fluid, cord blood, neonatal throat swab, and breast milk samples available from six of the nine patients were found to be negative for SARS-COV-2. Whether virus shedding occurs vaginally is also not known.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Persistence and clearance of viral RNA in 2019 novel coronavirus disease rehabilitation patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background: A patient&rsquo;s infectivity is determined by the presence of the virus in different body fluids, secretions, and excreta. The persistence and clearance of viral RNA from different specimens of patients with 2019 novel coronavirus disease (COVID-19) remain unclear. This study analyzed the clearance time and factors influencing 2019 novel coronavirus (2019-nCoV) RNA in different samples from patients with COVID-19, providing further evidence to improve the management of patients during convalescence. Methods: The clinical data and laboratory test results of convalescent patients with COVID-19 who were admitted to from January 20, 2020 to February 10, 2020 were collected retrospectively. The reverse transcription polymerase chain reaction (RT-PCR) results for patients&rsquo; oropharyngeal swab, stool, urine, and serum samples were collected and analyzed. Convalescent patients refer to recovered non-febrile patients without respiratory symptoms who had two successive (minimum 24 h sampling interval) negative RT-PCR results for viral RNA from oropharyngeal swabs. The effects of cluster of differentiation 4 (CD4)+ T lymphocytes, inflammatory indicators, and glucocorticoid treatment on viral nucleic acid clearance were analyzed. Results: In the 292 confirmed cases, 66 patients recovered after treatment and were included in our study. In total, 28 (42.4%) women and 38 men (57.6%) with a median age of 44.0 (34.0&ndash;62.0) years were analyzed. After in-hospital treatment, patients&rsquo; inflammatory indicators decreased with improved clinical condition. The median time from the onset of symptoms to first negative RT-PCR results for oropharyngeal swabs in convalescent patients was 9.5 (6.0&ndash;11.0) days. By February 10, 2020, 11 convalescent patients (16.7%) still tested positive for viral RNA from stool specimens and the other 55 patients&rsquo; stool specimens were negative for 2019-nCoV following a median duration of 11.0 (9.0&ndash;16.0) days after symptom onset. Among these 55 patients, 43 had a longer duration until stool specimens were negative for viral RNA than for throat swabs, with a median delay of 2.0 (1.0&ndash;4.0) days. Results for only four (6.9%) urine samples were positive for viral nucleic acid out of 58 cases; viral RNA was still present in three patients&rsquo; urine specimens after throat swabs were negative. Using a multiple linear regression model ( F =2.669, P =0.044, and adjusted R 2 =0.122), the analysis showed that the CD4+ T lymphocyte count may help predict the duration of viral RNA detection in patients&rsquo; stools ( t =-2.699, P =0.010). The duration of viral RNA detection from oropharyngeal swabs and fecal samples in the glucocorticoid treatment group was longer than that in the non-glucocorticoid treatment group (15 days vs 8.0 days, respectively; t =2.550, P =0.013) and the duration of viral RNA detection in fecal samples in the glucocorticoid treatment group was longer than that in the non-glucocorticoid treatment group (20 days vs 11 days, respectively; t =4.631, P &lt;0.001). There was no statistically significant difference in inflammatory indicators between patients with positive fecal viral RNA test results and those with negative results ( P &gt;0.05). Conclusions: In brief, as the clearance of viral RNA in patients&rsquo; stools was delayed compared to that in oropharyngeal swabs, it is important to identify viral RNA in feces during convalescence. Because of the delayed clearance of viral RNA in the glucocorticoid treatment group, glucocorticoids are not recommended in the treatment of COVID-19, especially for mild disease. The duration of RNA detection may relate to host cell immunity.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Cross-Protection against a Human Enteric Coronavirus and a Virulent Bovine Enteric Coronavirus in Gnotobiotic Calves</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A group 2 human coronavirus designated HECV-4408 was isolated from a child with acute diarrhea and is antigenically and genetically more closely related to bovine coronavirus (BCoV) than to human coronavirus OC43 (X. M. Zhang, W. Herbst, K. G. Kousoulas, and J. Storz, J. Med. Virol. 44:152-161, 1994). To determine whether HECV-4408 infects gnotobiotic calves and induces cross-protective immunity against the virulent enteric BCoV DB2 strain, gnotobiotic calves (n = 4) were orally inoculated with HECV-4408 and then challenged with BCoV DB2 at postinoculation day (PID) 21. All calves inoculated with HECV-4408 developed diarrhea at PID 3 to 4 lasting 5 to 9 days. Fecal and nasal virus shedding were first detected by reverse transcription-PCR at PID 3 to 4 and at PID 2 to 4, respectively. After challenge with bovine coronavirus, no diarrhea or virus shedding was detected in calves inoculated with HECV-4408, but a mock-inoculated calf developed diarrhea and fecal and nasal shedding. Fecal immunoglobulin A (IgA) and serum IgG antibodies were first detected at PID 7 and PID 14, respectively. At postchallenge day 7, serum IgG and fecal IgA antibody titers remained the same or increased only twofold compared to prechallenge titers. An additional two gnotobiotic calves were inoculated with HECV-4408 and euthanized at PID 5. Moderate villous atrophy was observed in the small intestines, and viral antigen was detected in villous enterocytes of the small and large intestines by immunohistochemistry. These results support and extend the previous report that HECV-4408 is likely a variant of bovine coronavirus. They confirm its infectivity for calves and complete cross-protection against a bovine coronavirus (DB2 strain) showing 98.2% amino acid identity to HECV-4408 in the S protein.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Asymptomatic Middle East Respiratory Syndrome Coronavirus Infection in Rabbits</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The ability of Middle East respiratory syndrome coronavirus (MERS-CoV) to infect small animal species may be restricted given the fact that mice, ferrets, and hamsters were shown to resist MERS-CoV infection. We inoculated rabbits with MERS-CoV. Although virus was detected in the lungs, neither significant histopathological changes nor clinical symptoms were observed. Infectious virus, however, was excreted from the upper respiratory tract, indicating a potential route of MERS-CoV transmission in some animal species.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The ongoing outbreak of novel corona virus disease 2019 (COVID-19) in Wuhan, China, is arousing international concern. This study evaluated whether and when the infected but asymptomatic cases during the incubation period could infect others.  Methods: We collected data on demographic characteristics, exposure history, and symptom onset day of the confirmed cases, which had been announced by the Chinese local authorities. We evaluated the potential of transmission during the incubation period in 50 infection clusters, including 124 cases. All the secondary cases had a history of contact with their first-generation cases prior to symptom onset.  Results: The estimated mean incubation period for COVID-19 was 4.9 days (95% confidence interval [CI], 4.4 to 5.4) days, ranging from 0.8 to 11.1 days (2.5th to 97.5th percentile). The observed mean and standard deviation (SD) of serial interval was 4.1±3.3 days, with the 2.5th and 97.5th percentiles at -1 and 13 days. The infectious curve showed that in 73.0% of the secondary cases, their date of getting infected was before symptom onset of the first-generation cases, particularly in the last three days of the incubation period.  Conclusions: The results indicated the transmission of COVID-9 occurs among close contacts during the incubation period, which may lead to a quarantine loophole. Strong and effective countermeasures should be implemented to prevent or mitigate asymptomatic transmission during the incubation period in populations at high risk.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk Management of COVID-19 by Universities in China Short-term Forecasts of the COVID-19 Epidemic in Guangdong and Zhejiang, China: February 13–23, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid spread of new coronaviruses throughout China and the world in 2019&ndash;2020 has had a great impact on China&rsquo;s economic and social development. As the backbone of Chinese society, Chinese universities have made significant contributions to emergency risk management. Such contributions have been made primarily in the following areas: alumni resource collection, medical rescue and emergency management, mental health maintenance, control of staff mobility, and innovation in online education models. Through the support of these methods, Chinese universities have played a positive role in the prevention and control of the epidemic situation. However, they also face the problems of alumni&rsquo;s economic development difficulties, the risk of deadly infection to medical rescue teams and health workers, infection of teachers and students, and the unsatisfactory application of information technology in resolving the crisis. In response to these risks and emergency problems, we propose some corresponding solutions for public dissemination, including issues related to medical security, emergency research, professional assistance, positive communication, and hierarchical information-based teaching. The ongoing COVID-19 epidemic continues to spread within and outside of China, despite several social distancing measures implemented by the Chinese government. Limited epidemiological data are available, and recent changes in case definition and reporting further complicate our understanding of the impact of the epidemic, particularly in the epidemic&rsquo;s epicenter. Here we use previously validated phenomenological models to generate short-term forecasts of cumulative reported cases in Guangdong and Zhejiang, China. Using daily reported cumulative case data up until 13 February 2020 from the National Health Commission of China, we report 5- and 10-day ahead forecasts of cumulative case reports. Specifically, we generate forecasts using a generalized logistic growth model, the Richards growth model, and a sub-epidemic wave model, which have each been previously used to forecast outbreaks due to different infectious diseases. Forecasts from each of the models suggest the outbreaks may be nearing extinction in both Guangdong and Zhejiang; however, the sub-epidemic model predictions also include the potential for further sustained transmission, particularly in Zhejiang. Our 10-day forecasts across the three models predict an additional 65&ndash;81 cases (upper bounds: 169&ndash;507) in Guangdong and an additional 44&ndash;354 (upper bounds: 141&ndash;875) cases in Zhejiang by February 23, 2020. In the best-case scenario, current data suggest that transmission in both provinces is slowing down.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus infection in non-camelid domestic mammals</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 16</span></i><p style="text-align: justify; text-justify: inter-word;">Dromedary camels are natural host of the Middle East respiratory syndrome coronavirus (MERS-CoV). However, there are limited studies of MERS-CoV infection of other domestic mammals exposed to infected dromedaries. We expanded our surveillance among camels in Egypt, Tunisia, and Senegal to include other domestic mammalian species in contact with infected camels. A total of 820 sera and 823 nasal swabs from cattle, sheep, goats, donkeys, buffaloes, mules, and horses were collected. Swabs were tested using RT-PCR and virus RNA-positive samples were genetically sequenced and phylogenetically analysed. Sera were screened using virus microneutralization tests and positive sera (where available) were confirmed using plaque reduction neutralization tests (PRNT). We detected 90% PRNT confirmed MERS-CoV antibody in 35 (55.6%) of 63 sera from sheep collected from Senegal, two sheep (1.8%) of 114 in Tunisia and a goat (0.9%) of 107 in Egypt, with titres ranging from 1:80 to ≥1:320. We detected MERS-CoV RNA in swabs from three sheep (1.2%) of 254 and five goats (4.1%) of 121 from Egypt and Senegal, as well as one cow (1.9%) of 53 and three donkeys (7.1%) of 42 from Egypt. Partial sequences of the RT-PCR amplicons confirmed specificity of the results. This study showed that domestic livestock in contact with MERS-CoV infected camels may be at risk of infection. We recommend expanding current MERS-CoV surveillance in animals to include other livestock in close contact with dromedary camels. The segregation of camels from other livestock in farms and live animal markets may need to be considered.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Species-Specific Colocalization of Middle East Respiratory Syndrome Coronavirus Attachment and Entry Receptors</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) uses the S1(B) domain of its spike protein to bind to dipeptidyl peptidase 4 (DPP4), its functional receptor, and its S1(A) domain to bind to sialic acids. The tissue localization of DPP4 in humans, bats, camelids, pigs, and rabbits generally correlates with MERS-CoV tropism, highlighting the role of DPP4 in virus pathogenesis and transmission. However, MERS-CoV S1(A) does not indiscriminately bind to all α2,3-sialic acids, and the species-specific binding and tissue distribution of these sialic acids in different MERS-CoV-susceptible species have not been investigated. We established a novel method to detect these sialic acids on tissue sections of various organs of different susceptible species by using nanoparticles displaying multivalent MERS-CoV S1(A). We found that the nanoparticles specifically bound to the nasal epithelial cells of dromedary camels, type II pneumocytes in human lungs, and the intestinal epithelial cells of common pipistrelle bats. Desialylation by neuraminidase abolished nanoparticle binding and significantly reduced MERS-CoV infection in primary susceptible cells. In contrast, S1(A) nanoparticles did not bind to the intestinal epithelium of serotine bats and frugivorous bat species, nor did they bind to the nasal epithelium of pigs and rabbits. Both pigs and rabbits have been shown to shed less infectious virus than dromedary camels and do not transmit the virus via either contact or airborne routes. Our results depict species-specific colocalization of MERS-CoV entry and attachment receptors, which may be relevant in the transmission and pathogenesis of MERS-CoV. IMPORTANCE MERS-CoV uses the S1(B) domain of its spike protein to attach to its host receptor, dipeptidyl peptidase 4 (DPP4). The tissue localization of DPP4 has been mapped in different susceptible species. On the other hand, the S1(A) domain, the N-terminal domain of this spike protein, preferentially binds to several glycotopes of α2,3-sialic acids, the attachment factor of MERS-CoV. Here we show, using a novel method, that the S1(A) domain specifically binds to the nasal epithelium of dromedary camels, alveolar epithelium of humans, and intestinal epithelium of common pipistrelle bats. In contrast, it does not bind to the nasal epithelium of pigs or rabbits, nor does it bind to the intestinal epithelium of serotine bats and frugivorous bat species. This finding supports the importance of the S1(A) domain in MERS-CoV infection and tropism, suggests its role in transmission, and highlights its potential use as a component of novel vaccine candidates.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevent and predict</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the COVID-19 outbreak continues, the next pandemic could be prevented by ending the wildlife trade and reinvesting in the monitoring of potential zoonoses.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Replicative Capacity of MERS Coronavirus in Livestock Cell Lines</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Replicative capacity of Middle East respiratory syndrome coronavirus (MERS-CoV) was assessed in cell lines derived from livestock and peridomestic small mammals on the Arabian Peninsula. Only cell lines originating from goats and camels showed efficient replication of MERS-CoV. These results provide direction in the search for the intermediate host of MERS-CoV.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Animal Models and Vaccines for SARS-CoV Infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We summarize findings of SARS-CoV infections in several animal models each of which support viral replication in lungs accompanied by histopathological changes and/or clinical signs of illness to varying degrees. New findings are reported on SARS-CoV replication and associated pathology in two additional strains (C57BL/6 and 129S6) of aged mice. We also provide new comparative data on viral replication and associated pathology following infection of golden Syrian hamsters with various SARS-CoV strains and report the levels of neutralizing antibody titers following these infections and the cross-protective efficacy of infection with these strains in protecting against heterologous challenge. Finally, we summarize findings of a variety of vaccine approaches and discuss the available in vitro and in vivo data addressing the potential for disease enhancement following re-infection in animals previously vaccinated against or infected with SARS-CoV.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Disease 2019 (COVID-19) and Pregnancy: What obstetricians need to know</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus Disease 2019 (COVID-19) is an emerging disease with a rapid increase in cases and deaths since its first identification in Wuhan, China, in December 2019. Limited data are available about COVID-19 during pregnancy; however, information on illnesses associated with other highly pathogenic coronaviruses (i.e., severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS)) might provide insights into COVID-19’s effects during pregnancy.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND The coronavirus disease 2019 (COVID-19) is a newly emerged infection from the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Apart from the respiratory system, it is unclear whether SARS-CoV-2 can also directly infect other tissues such as the kidney or induce acute renal failure. METHODS We conducted a retrospective analysis of estimated glomerular filtration rate (eGFR), plasma creatinine, and urea concentrations along with other clinical parameters from 85 patients with laboratory-confirmed COVID-19 admitted to a hospital in Wuhan from January 17, 2020 to March 3, 2020. Kidney tissues from six other patients with postmortem examinations were analyzed by Hematoxylin and Eosin (H&amp;E) and in situ expression of viral nucleocaspid protein (NP) antigen was detected by immunohistochemistry. RESULTS Among these 85 COVID-19 cases, 27.06% (23/85) patients exhibited acute renal failure (ARF), and elder patients (≥ 60 years old) are easier to develop ARF (65.22% vs 24.19%, p&lt; 0.001). Comorbidities of disorders like hypertension and heart failure were more common in patients who developed ARF (69.57% vs 11.29%, p&lt; 0.001). Dynamic observation of eGFR revealed that perished cases have a rapid decrease of eGFR but quickly boosting plasma creatinine and urea. On the other hand, enhancement of eGFR and persistence low level of plasma creatinine and urea was observed in recovery patients following diuretic treatment. H&amp;E staining demonstrated kidney tissues from 6 cases of postmortems have severe acute tubular necrosis but no evidence of glomerular pathology or tubulointerstitial lymphocyte infiltration. Immunohistochemistry showed that SARS-CoV-2 NP antigen was accumulated in kidney tubules.   CONCLUSIONS The development of ARF during the course of disease is an important negative prognostic indicator for survival with COVID-19. SARS-CoV-2 virus directly infects human kidney tubules by thus induces acute tubular damage, ARF and probably also lead to urine transmission.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Sero-epidemiology of MERS coronavirus in Saudi Arabia (1993) and Australia (2014) and characterization of assay specificity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Pseudoparticle virus neutralization (ppNT) and a conventional microneutralization (MN) assays are specific for detecting antibodies to MERS coronavirus (MERS-CoV) when used in sero-epidemiological studies in animals. Genetically diverse MERS-CoV appear antigenically similar in MN tests. We confirm that MERS-CoV has been circulating in dromedaries in Saudi Arabia in 1993. Preliminary data suggests that feral Australian dromedaries may be free of MERS-CoV but larger confirmatory studies are needed.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-associated Coronavirus Transmitted from Human to Pig</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">SEVERE ACUTE RESPIRATORY SYNDROME–ASSOCIATED: coronavirus (SARS-CoV) was isolated from a pig during a survey for possible routes of viral transmission after a SARS epidemic. Sequence and epidemiology analyses suggested that the pig was infected by a SARS-CoV of human origin.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although the finding of severe acute respiratory syndrome coronavirus (SARS-CoV) in caged palm civets from live animal markets in China has provided evidence for interspecies transmission in the genesis of the SARS epidemic, subsequent studies suggested that the civet may have served only as an amplification host for SARS-CoV. In a surveillance study for CoV in noncaged animals from the wild areas of the Hong Kong Special Administration Region, we identified a CoV closely related to SARS-CoV (bat-SARS-CoV) from 23 (39%) of 59 anal swabs of wild Chinese horseshoe bats (Rhinolophus sinicus) by using RT-PCR. Sequencing and analysis of three bat-SARS-CoV genomes from samples collected at different dates showed that bat-SARS-CoV is closely related to SARS-CoV from humans and civets. Phylogenetic analysis showed that bat-SARS-CoV formed a distinct cluster with SARS-CoV as group 2b CoV, distantly related to known group 2 CoV. Most differences between the bat-SARS-CoV and SARS-CoV genomes were observed in the spike genes, ORF 3 and ORF 8, which are the regions where most variations also were observed between human and civet SARS-CoV genomes. In addition, the presence of a 29-bp insertion in ORF 8 of bat-SARS-CoV genome, not in most human SARS-CoV genomes, suggests that it has a common ancestor with civet SARS-CoV. Antibody against recombinant bat-SARS-CoV nucleocapsid protein was detected in 84% of Chinese horseshoe bats by using an enzyme immunoassay. Neutralizing antibody to human SARS-CoV also was detected in bats with lower viral loads. Precautions should be exercised in the handling of these animals.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the generation interval for COVID-19 based on symptom onset data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies. Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided. Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers. Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32-67%) for Singapore and 62% (95%CI 50-76%) for Tianjin, China. Estimates of the reproduction number based on the generation interval distribution were slightly higher than those based on the serial interval distribution. Conclusions: Estimating generation and serial interval distributions from outbreak data requires careful investigation of the underlying transmission network. Detailed contact tracing information is essential for correctly estimating these quantities.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel coronavirus infection and pregnancy</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Clinical manifestation of COVID-19 infection during pregnancy Due to the physiological changes in their immune and cardiopulmonary systems, pregnant women are more likely to develop severe illness after infection with respiratory viruses. In 2009, pregnant women accounted for 1% of patients infected with influenza A subtype H1N1 virus, but they accounted for 5% of all H1N1-related deaths6. In addition, severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV), two notable strains of the coronavirus family, are both known to be responsible for severe complications during pregnancy, including the need for endotracheal intubation, admission to an intensive care unit (ICU), renal failure and death7, 8. Interestingly, the impact of COVID-19 infection on pregnant women appears to be less severe. Chen et al.9 reported the clinical characteristics of nine pregnant women with laboratory-confirmed COVID-19 in the third trimester, which comprised mainly fever and cough. Other symptoms included myalgia, malaise, sore throat, diarrhea and shortness of breath. Data from laboratory tests showed that the majority of patients had lymphopenia and increased C-reactive protein, and chest CT scans showed multiple patchy ground-glass shadows in the lungs. Pregnancy complications that appeared after the onset of COVID-19 infection included fetal distress in two of nine patients and premature rupture of the membranes in two of nine patients. None of the patients developed severe COVID-19 pneumonia or died. Another series10 of nine pregnant women with COVID-19 pneumonia presenting from mid-trimester onwards, or during the postpartum period, reported similar findings except for one woman requiring ICU care and ventilation for acute respiratory distress syndrome after the infection was diagnosed 2?days postpartum. In general, both studies reported that the clinical characteristics of the pregnant women with COVID-19 pneumonia were similar to those of non-pregnant adult patients who developed COVID-19 pneumonia2-4. These observations are also in line with what has been learned about COVID-19 pneumonia in pregnancy in several other hospitals in Wuhan, China. Pregnant healthcare professionals should follow risk-assessment and infection-control guidelines following exposure to patients with suspected or confirmed COVID-19. Adherence to recommended infection prevention and control practices is an important part of protecting all healthcare professionals in clinical settings11.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 9</span></i><p style="text-align: justify; text-justify: inter-word;">Known human coronaviruses are believed to have originated in animals and made use of intermediate hosts for transmission to humans. The intermediate hosts of most of the human coronaviruses are known, but not for HCoV-NL63. This study aims to assess the possible role of some major domestic livestock species as intermediate hosts of HCoV-NL63. We developed a testing algorithm for high throughput screening of livestock sera with ELISA and confirmation with recombinant immunofluorescence assay testing for antibodies against HCoV-NL63 in livestock. Optimization of the ELISA showed a capability of the assay to significantly distinguish HCoV-NL63 from HCoV-229E (U = 27.50, p < 0.001) and HCoV-OC43 (U = 55.50, p < 0.001) in coronavirus-characterized sera. Evaluation of the assay with collected human samples showed no significant difference in mean optical density values of immunofluorescence-classified HCoV-NL63-positive and HCoV-NL63-negative samples (F (1, 215) = 0.437, p = 0.509). All the top 5% (n = 8) most reactive human samples tested by ELISA were HCoV-NL63 positive by immunofluorescence testing. In comparison, only a proportion (84%, n = 42) of the top 25% were positive by immunofluorescence testing, indicating an increased probability of the highly ELISA reactive samples testing positive by the immunofluorescence assay. None of the top 5% most ELISA reactive livestock samples were positive for HCoV-NL63-related viruses by immunofluorescence confirmation. Ghanaian domestic livestock are not likely intermediate hosts of HCoV-NL63-related coronaviruses.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 (Novel Coronavirus 2019) - recent trends</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The World Health Organization (WHO) has issued a warning that, although the 2019 novel coronavirus (COVID-19) from Wuhan City (China), is not pandemic, it should be contained to prevent the global spread. The COVID-19 virus was known earlier as 2019-nCoV. As of 12 February 2020, WHO reported 45,171 cases and 1115 deaths related to COVID-19. COVID-19 is similar to Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) virus in its pathogenicity, clinical spectrum, and epidemiology. Comparison of the genome sequences of COVID-19, SARS-CoV, and Middle East Respiratory Syndrome coronavirus (MERS-CoV) showed that COVID-19 has a better sequence identity with SARS-CoV compared to MERS CoV. However, the amino acid sequence of COVID-19 differs from other coronaviruses specifically in the regions of 1ab polyprotein and surface glycoprotein or S-protein. Although several animals have been speculated to be a reservoir for COVID-19, no animal reservoir has been already confirmed. COVID-19 causes COVID-19 disease that has similar symptoms as SARS-CoV. Studies suggest that the human receptor for COVID-19 may be angiotensin-converting enzyme 2 (ACE2) receptor similar to that of SARS-CoV. The nucleocapsid (N) protein of COVID-19 has nearly 90% amino acid sequence identity with SARS-CoV. The N protein antibodies of SARS-CoV may cross react with COVID-19 but may not provide cross-immunity. In a similar fashion to SARS-CoV, the N protein of COVID-19 may play an important role in suppressing the RNA interference (RNAi) to overcome the host defense. This mini-review aims at investigating the most recent trend of COVID-19.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Cotton Rats as a Model for Severe Acute Respiratory Syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Experimental studies were conducted to evaluate two species of cotton rats, Sigmodon hispidus and Sigmodon fulviventer, as a model for severe acute respiratory syndrome (SARS). Blood and turbinate wash samples, and lung tissue were collected from each animal at different time points after SARS coronavirus (CoV) infection for determining the growth curve of virus, if any, by the standard infectivity assay in Vero E6 cells. In addition, sections of the lung, liver, spleen, and kidney were taken and used for histology analysis. All animals were observed daily for signs of illness, and in some experiments, animals were weighed on the day when they were sacrificed. The results indicated that the cotton rat species, S. hispidus and S. fulviventer, were not a useful model for either SARS-CoV infection or disease. This observation was supported by the absence of any signs of illness, the failure to consistently demonstrate virus in the blood and tissues, and the absent of any notable histopathology. However, infected animals were capable of producing neuralizing antibodies against SARS-CoV, suggesting the serocon-version did occur. Further studies are warranted to consider other animal species in efforts to find better animal models for the evaluation of SARS-CoV vaccines and antiviral drugs. Key Words: Cotton rats—SARS—Model animal.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Lack of Middle East Respiratory Syndrome Coronavirus Transmission in Rabbits</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) transmission from dromedaries to humans has resulted in major outbreaks in the Middle East. Although some other livestock animal species have been shown to be susceptible to MERS-CoV, it is not fully understood why the spread of the virus in these animal species has not been observed in the field. In this study, we used rabbits to further characterize the transmission potential of MERS-CoV. In line with the presence of MERS-CoV receptor in the rabbit nasal epithelium, high levels of viral RNA were shed from the nose following virus inoculation. However, unlike MERS-CoV-infected dromedaries, these rabbits did not develop clinical manifestations including nasal discharge and did shed only limited amounts of infectious virus from the nose. Consistently, no transmission by contact or airborne routes was observed in rabbits. Our data indicate that despite relatively high viral RNA levels produced, low levels of infectious virus are excreted in the upper respiratory tract of rabbits as compared to dromedary camels, thus resulting in a lack of viral transmission.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Time Course of MERS-CoV Infection and Immunity in Dromedary Camels</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">Knowledge about immunity to Middle East respiratory syndrome coronavirus (MERS-CoV) in dromedary camels is essential for infection control and vaccination. A longitudinal study of 11 dam–calf pairs showed that calves lose maternal MERS-CoV antibodies 5–6 months postparturition and are left susceptible to infection, indicating a short window of opportunity for vaccination.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe Acute Respiratory Syndrome Coronavirus Infection of Golden Syrian Hamsters</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Small animal models are needed in order to evaluate the efficacy of candidate vaccines and antivirals directed against the severe acute respiratory syndrome coronavirus (SARS CoV). We investigated the ability of SARS CoV to infect 5-week-old Golden Syrian hamsters. When administered intranasally, SARS CoV replicates to high titers in the lungs and nasal turbinates. Peak replication in the lower respiratory tract was noted on day 2 postinfection (p.i.) and was cleared by day 7 p.i. Low levels of virus were present in the nasal turbinates of a few hamsters at 14 days p.i. Viral replication in epithelial cells of the respiratory tract was accompanied by cellular necrosis early in infection, followed by an inflammatory response coincident with viral clearance, focal consolidation in pulmonary tissue, and eventual pulmonary tissue repair. Despite high levels of virus replication and associated pathology in the respiratory tract, the hamsters showed no evidence of disease. Neutralizing antibodies were detected in sera at day 7 p.i., and mean titers at day 28 p.i. exceeded 1:400. Hamsters challenged with SARS CoV at day 28 p.i. were completely protected from virus replication and accompanying pathology in the respiratory tract. Comparing these data to the mouse model, SARS CoV replicates to a higher titer and for a longer duration in the respiratory tract of hamsters and is accompanied by significant pathology that is absent in mice. Viremia and extrapulmonary spread of SARS CoV to liver and spleen, which are seen in hamsters, were not detected in mice. The hamster, therefore, is superior to the mouse as a model for the evaluation of antiviral agents and candidate vaccines against SARS CoV replication.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>A long-term animal experiment indicating persistent infection of bovine coronavirus in cattle</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">A long-term animal experiment involving inoculation with bovine coronavirus (BCoV) was conducted to verify its persistent infection in cattle. Three colostrum-deprived Holstein calves were housed separately in individual rooms of a high-containment facility and inoculated with the BCoV strain Kumamoto/1/07. Until the end of the experiment (1,085, 700 and 280 days, respectively), viral RNAs were detected sporadically by RT-PCR and nested PCR from plasma, nasal discharge, and feces. Seroconversion and titer changes were validated by hemagglutination inhibition tests and neutralization tests. Among the samples, nasal discharge showed a higher viral positivity than feces, which seemed to be associated with positive detection in the plasma. These data demonstrate the existence of persistent infection of BCoV in the respiratory tissues of cattle.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Norwalk-like calicivirus genes in farm animals.</h3>
<i class="tooltip">2000<span class="tooltiptext">2000 Jan-Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Viruses closely related to Norwalk-like viruses (NLVs) were recently found in stored stool samples from two calves (United Kingdom and Germany) and four pigs (Japan), sparking discussions about the potential for zoonotic transmission. To investigate if NLVs are commonly present in farm animals, pooled stool samples from 100 pig farms, 48 chicken farms, 43 dairy cow herds, and 75 veal calf farms from the Netherlands were assayed by reverse transcription- polymerase chain reaction amplification, using primers specific for the detection of NLVs from humans. NLV RNA was detected in 33 (44%) of the specimens from veal calf farms and two (2%) specimens from pig farms. Our data show that NLV infections until recently thought to be restricted to humans occur often in calves and sometimes in pigs. While zoonotic transmission has not been proven, these findings suggest that calves and pigs may be reservoir hosts of NLVs.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>The UCUAAAC promoter motif is not required for high-frequency leader recombination in bovine coronavirus defective interfering RNA.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The 65-nucleotide leader on the cloned bovine coronavirus defective interfering (DI) RNA, when marked by mutations, has been shown to rapidly convert to the wild-type leader of the helper virus following DI RNA transfection into helper virus-infected cells. A model of leader-primed transcription in which free leader supplied in trans by the helper virus interacts by way of its flanking 5'UCUAAAC3' sequence element with the 3'-proximal 3'AGAUUUG5' promoter on the DI RNA minus strand to prime RNA replication has been used to explain this phenomenon. To test this model, the UCUAAAC element which occurs only once in the BCV 5' untranslated region was either deleted or completely substituted in input DI RNA template, and evidence of leader conversion was sought. In both cases, leader conversion occurred rapidly, indicating that this element is not required on input RNA for the conversion event. Substitution mutations mapped the crossover region to a 24-nucleotide segment that begins within the UCUAAAC sequence and extends downstream. Although structure probing of the bovine coronavirus 5' untranslated region indicated that the UCUAAAC element is in the loop of a prominent stem and thus theoretically available for base pair-directed priming, no evidence of an unattached leader early in infection that might have served as a primer for transcription was found by RNase protection studies. These results together suggest that leader conversion on the DI RNA 5' terminus is not guided by the UCUAAAC element and might arise instead from a high-frequency, region-specific, homologous recombination event perhaps during minus-strand synthesis rather than by leader priming during plus-strand synthesis.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Time to use the p-word? Coronavirus enters dangerous new phase</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As outbreaks surge worldwide, scientists fear that COVID-19 might soon become pandemic. As outbreaks surge worldwide, scientists fear that COVID-19 might soon become pandemic.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Humoral Immunogenicity and Efficacy of a Single Dose of ChAdOx1 MERS Vaccine Candidate in Dromedary Camels</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 8</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV seronegative and seropositive camels received a single intramuscular dose of ChAdOx1 MERS, a replication-deficient adenoviral vectored vaccine expressing MERS-CoV spike protein, with further groups receiving control vaccinations. Infectious camels with active naturally acquired MERS-CoV infection, were co-housed with the vaccinated camels at a ratio of 1:2 (infected:vaccinated); nasal discharge and virus titres were monitored for 14 days. Overall, the vaccination reduced virus shedding and nasal discharge (p = 0.0059 and p = 0.0274, respectively). Antibody responses in seropositive camels were enhancedby the vaccine; these camels had a higher average age than seronegative. Older seronegative camels responded more strongly to vaccination than younger animals; and neutralising antibodies were detected in nasal swabs. Further work is required to optimise vaccine regimens for younger seronegative camels.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Humoral Immunogenicity and Efficacy of a Single Dose of ChAdOx1 MERS Vaccine Candidate in Dromedary Camels</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 8</span></i><p style="text-align: justify; text-justify: inter-word;">MERS-CoV seronegative and seropositive camels received a single intramuscular dose of ChAdOx1 MERS, a replication-deficient adenoviral vectored vaccine expressing MERS-CoV spike protein, with further groups receiving control vaccinations. Infectious camels with active naturally acquired MERS-CoV infection, were co-housed with the vaccinated camels at a ratio of 1:2 (infected:vaccinated); nasal discharge and virus titres were monitored for 14 days. Overall, the vaccination reduced virus shedding and nasal discharge (p = 0.0059 and p = 0.0274, respectively). Antibody responses in seropositive camels were enhancedby the vaccine; these camels had a higher average age than seronegative. Older seronegative camels responded more strongly to vaccination than younger animals; and neutralising antibodies were detected in nasal swabs. Further work is required to optimise vaccine regimens for younger seronegative camels.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Short-term Forecasts of the COVID-19 Epidemic in Guangdong and Zhejiang, China: February 13–23, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing COVID-19 epidemic continues to spread within and outside of China, despite several social distancing measures implemented by the Chinese government. Limited epidemiological data are available, and recent changes in case definition and reporting further complicate our understanding of the impact of the epidemic, particularly in the epidemic&rsquo;s epicenter. Here we use previously validated phenomenological models to generate short-term forecasts of cumulative reported cases in Guangdong and Zhejiang, China. Using daily reported cumulative case data up until 13 February 2020 from the National Health Commission of China, we report 5- and 10-day ahead forecasts of cumulative case reports. Specifically, we generate forecasts using a generalized logistic growth model, the Richards growth model, and a sub-epidemic wave model, which have each been previously used to forecast outbreaks due to different infectious diseases. Forecasts from each of the models suggest the outbreaks may be nearing extinction in both Guangdong and Zhejiang; however, the sub-epidemic model predictions also include the potential for further sustained transmission, particularly in Zhejiang. Our 10-day forecasts across the three models predict an additional 65&ndash;81 cases (upper bounds: 169&ndash;507) in Guangdong and an additional 44&ndash;354 (upper bounds: 141&ndash;875) cases in Zhejiang by February 23, 2020. In the best-case scenario, current data suggest that transmission in both provinces is slowing down.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>In this issue of Occupational Medicine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The recent COVID-19 outbreak in Wuhan city (Hubei Province of central China), presents significant public health challenges not only in china but across all affected countries worldwide. Koh [1] succinctly describes what coronaviruses are, and outlines six known species associated with human illnesses. The article also describes countries of origin of previous coronavirus disease outbreaks and reports number of known fatalities with associated case-fatality rates. Occupations implicated in the current COVID-19 outbreak are detailed and groups at high risk of contracting the infection e.g. healthcare workers highlighted. Social stigmatisation associated with COVID-19 is explored and suggested measures to contain the infection discussed.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological Evidence of MERS-CoV Antibodies in Dromedary Camels (Camelus dromedaries) in Laikipia County, Kenya</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Oct 16</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a recently identified virus causing severe viral respiratory illness in people. Little is known about the reservoir in the Horn of Africa. In Kenya, where no human MERS cases have been reported, our survey of 335 dromedary camels, representing nine herds in Laikipia County, showed a high seroprevalence (46.9%) to MERS-CoV antibodies. Between herd differences were present (14.3%– 82.9%), but was not related to management type or herd isolation. Further research should focus on identifying similarity between MERS-CoV viral isolates in Kenya and clinical isolates from the Middle East and elsewhere.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Experiences after Twenty Months with Pandemic Influenza A (H1N1) 2009 Infection in the Naïve Norwegian Pig Population</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Jan 3</span></i><p style="text-align: justify; text-justify: inter-word;">Pandemic (H1N1) 2009 influenza A virus was detected in Norwegian pigs in October 2009. Until then, Norway was regarded free of swine influenza. Intensified screening revealed 91 positive herds within three months. The virus was rapidly transmitted to the susceptible population, including closed breeding herds with high biosecurity. Humans were important for the introduction as well as spread of the virus to pigs. Mild or no clinical signs were observed in infected pigs. Surveillance of SIV in 2010 revealed that 41% of all the Norwegian pig herds had antibodies to pandemic (H1N1) 2009 virus. Furthermore, this surveillance indicated that pigs born in positive herds after the active phase did not seroconvert, suggesting no ongoing infection in the herds. However, results from surveillance in 2011 show a continuing spread of the infection in many herds, either caused by new introduction or by virus circulation since 2009.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Infection and Response to Rechallenge of Alpacas with Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We conducted a challenge/rechallenge trial in which 3 alpacas were infected with Middle East respiratory syndrome coronavirus. The alpacas shed virus at challenge but were refractory to further shedding at rechallenge on day 21. The trial indicates that alpacas may be suitable models for infection and shedding dynamics of this virus.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Infection and Response to Rechallenge of Alpacas with Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We conducted a challenge/rechallenge trial in which 3 alpacas were infected with Middle East respiratory syndrome coronavirus. The alpacas shed virus at challenge but were refractory to further shedding at rechallenge on day 21. The trial indicates that alpacas may be suitable models for infection and shedding dynamics of this virus.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>High Prevalence of Middle East Respiratory Coronavirus in Young Dromedary Camels in Jordan</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Prevalence of Middle East respiratory syndrome coronavirus (MERS-CoV) was determined in 45 dromedary camels from two geographically separated herds in Jordan. Virus shedding was only detected in swabs obtained from the respiratory tract and primarily observed in camels younger than 3 years. MERS-CoV seroprevalence increased with age of camels. Bovine and sheep sera were seronegative. Phylogenetic analysis of partial S2 clustered the Jordanian MERS-CoV strains with contemporary MERS-CoV strains associated with nosocomial outbreaks.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We report the kinetics of the immune response in relation to clinical and virological features of a patient with mild-to-moderate coronavirus disease-19 (COVID-19) requiring hospitalisation. Increased antibody-secreting cells, follicular T-helper cells, activated CD4+ and CD8+ T-cells and IgM/IgG SARS-CoV-2-binding antibodies were detected in blood, prior to symptomatic recovery. These immunological changes persisted for at least 7 days following full resolution of symptoms, indicating substantial anti-viral immunity in this non-severe COVID-19.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Differential Expression of the Middle East Respiratory Syndrome Coronavirus Receptor in the Upper Respiratory Tracts of Humans and Dromedary Camels</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is not efficiently transmitted between humans, but it is highly prevalent in dromedary camels. Here we report that the MERS-CoV receptor—dipeptidyl peptidase 4 (DPP4)—is expressed in the upper respiratory tract epithelium of camels but not in that of humans. Lack of DPP4 expression may be the primary cause of limited MERS-CoV replication in the human upper respiratory tract and hence restrict transmission.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Dual Infection of Gnotobiotic Calves with Bovine Strains of Group A and Porcine-Like Group C Rotaviruses Influences Pathogenesis of the Group C Rotavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">There is serological evidence that bovine group C rotaviruses exist in the United States, but there are no reports of their isolation. Ninety fecal samples from calves with diarrhea, 81 samples from adult cows with diarrhea (winter dysentery), and 20 fecal samples from healthy adult cows were tested for group C rotaviruses by polyacrylamide gel electrophoresis, immune electron microscopy, and reverse transcription-PCR (RT-PCR). Three samples from adult cow diarrhea cases were positive only by RT-PCR, and a group C rotavirus was isolated from a positive sample in monkey kidney (MA104) cells (WD534tc/C). Genetically and serologically, the WD534tc/C strain was more closely related to the Cowden porcine group C strain than to the Shintoku bovine strain. Because the original cow feces also contained a group A rotavirus (detected after passage in cell culture), we hypothesized that such dual-rotavirus infections might play a role in the pathogenesis and host adaptation of rotaviruses. Thus, we examined the pathogenesis of WD534tc/C alone or combined with virulent (IND/A) or attenuated (NCDV/A) bovine group A rotaviruses in gnotobiotic calves. WD534tc/C alone induced diarrhea without (or with limited) virus shedding in inoculated calves (n = 3). In contrast, all calves coinfected with WD534tc/C and IND/A (n = 2) developed diarrhea and shed both viruses, whereas calves coinfected with WD534tc/C and NCDV/A (n = 3) developed diarrhea but did not shed either virus. Infection with WD534tc/C or NCDV/A alone caused only mild villous atrophy (jejunum and/or ileum), whereas dual infection with both viruses induced lesions throughout the small intestine. Although IND/A alone caused villous atrophy, more-widespread small intestinal lesions occurred in calves coinfected with WD534tc/C and IND/A. In conclusion, coinfection of calves with group A rotaviruses enhanced fecal shedding of a bovine group C rotavirus and the extent of histopathological lesions in the small intestines. Thus, our findings suggest a potential novel hypothesis involving dual infections for the adaptation of heterologous rotaviruses to new host species.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Quasispecies That Include Homologues of Human Isolates Revealed through Whole-Genome Analysis and Virus Cultured from Dromedary Camels in Saudi Arabia</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Complete Middle East respiratory syndrome coronavirus (MERS-CoV) genome sequences were obtained from nasal swabs of dromedary camels sampled in the Kingdom of Saudi Arabia through direct analysis of nucleic acid extracts or following virus isolation in cell culture. Consensus dromedary MERS-CoV genome sequences were the same with either template source and identical to published human MERS-CoV sequences. However, in contrast to individual human cases, where only clonal genomic sequences are reported, detailed population analyses revealed the presence of more than one genomic variant in individual dromedaries. If humans are truly infected only with clonal virus populations, we must entertain a model for interspecies transmission of MERS-CoV wherein only specific genotypes are capable of passing bottleneck selection.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Quasispecies That Include Homologues of Human Isolates Revealed through Whole-Genome Analysis and Virus Cultured from Dromedary Camels in Saudi Arabia</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Apr 29</span></i><p style="text-align: justify; text-justify: inter-word;">Complete Middle East respiratory syndrome coronavirus (MERS-CoV) genome sequences were obtained from nasal swabs of dromedary camels sampled in the Kingdom of Saudi Arabia through direct analysis of nucleic acid extracts or following virus isolation in cell culture. Consensus dromedary MERS-CoV genome sequences were the same with either template source and identical to published human MERS-CoV sequences. However, in contrast to individual human cases, where only clonal genomic sequences are reported, detailed population analyses revealed the presence of more than one genomic variant in individual dromedaries. If humans are truly infected only with clonal virus populations, we must entertain a model for interspecies transmission of MERS-CoV wherein only specific genotypes are capable of passing bottleneck selection.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolutionary relationship analysis of Middle East respiratory syndrome coronavirus 4a and 4b protein coding sequences</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 19</span></i><p style="text-align: justify; text-justify: inter-word;">The 4a and 4b proteins of the Middle East respiratory syndrome coronavirus (MERS-CoV) have been described for their antagonism on host innate immunity. However, unlike clustering patterns of the complete gene sequences of human and camel MERS-CoVs, the 4a and 4b protein coding regions did not constitute species-specific phylogenetic groups. Moreover, given the estimated evolutionary rates of the complete, 4a, and 4b gene sequences, the 4a and 4b proteins might be less affected by species-specific innate immune pressures. These results suggest that the 4a and 4b proteins of MERS-CoV may function against host innate immunity in a manner independent of host species and/or evolutionary clustering patterns.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Serial interval of novel coronavirus (2019-nCoV) infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.</jats:p></p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jan 4</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe human infections and dromedary camels are considered an intermediary host. The dynamics of natural infection in camels are not well understood. Through systematic surveillance in Egypt, nasal, rectal, milk, urine and serum samples were collected from camels between June 2014 and February 2016. Locations included quarantines, markets, abattoirs, free-roaming herds and farmed breeding herds. The overall seroprevalence was 71% and RNA detection rate was 15%. Imported camels had higher seroprevalence (90% vs 61%) and higher RT-PCR detection rates (21% vs 12%) than locally raised camels. Juveniles had lower seroprevalence than adults (37% vs 82%) but similar RT-PCR detection rates (16% vs 15%). An outbreak in a breeding herd, showed that antibodies rapidly wane, that camels become re-infected, and that outbreaks in a herd are sustained for an extended time. Maternal antibodies titers were very low in calves regardless of the antibody titers of the mothers. Our results support the hypothesis that camels are a reservoir for MERS-CoV and that camel trade is an important route of introducing the virus into importing countries. Findings related to waning antibodies and re-infection have implications for camel vaccine development, disease management and zoonotic threat.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jan 4</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe human infections and dromedary camels are considered an intermediary host. The dynamics of natural infection in camels are not well understood. Through systematic surveillance in Egypt, nasal, rectal, milk, urine and serum samples were collected from camels between June 2014 and February 2016. Locations included quarantines, markets, abattoirs, free-roaming herds and farmed breeding herds. The overall seroprevalence was 71% and RNA detection rate was 15%. Imported camels had higher seroprevalence (90% vs 61%) and higher RT-PCR detection rates (21% vs 12%) than locally raised camels. Juveniles had lower seroprevalence than adults (37% vs 82%) but similar RT-PCR detection rates (16% vs 15%). An outbreak in a breeding herd, showed that antibodies rapidly wane, that camels become re-infected, and that outbreaks in a herd are sustained for an extended time. Maternal antibodies titers were very low in calves regardless of the antibody titers of the mothers. Our results support the hypothesis that camels are a reservoir for MERS-CoV and that camel trade is an important route of introducing the virus into importing countries. Findings related to waning antibodies and re-infection have implications for camel vaccine development, disease management and zoonotic threat.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jan 4</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe human infections and dromedary camels are considered an intermediary host. The dynamics of natural infection in camels are not well understood. Through systematic surveillance in Egypt, nasal, rectal, milk, urine and serum samples were collected from camels between June 2014 and February 2016. Locations included quarantines, markets, abattoirs, free-roaming herds and farmed breeding herds. The overall seroprevalence was 71% and RNA detection rate was 15%. Imported camels had higher seroprevalence (90% vs 61%) and higher RT-PCR detection rates (21% vs 12%) than locally raised camels. Juveniles had lower seroprevalence than adults (37% vs 82%) but similar RT-PCR detection rates (16% vs 15%). An outbreak in a breeding herd, showed that antibodies rapidly wane, that camels become re-infected, and that outbreaks in a herd are sustained for an extended time. Maternal antibodies titers were very low in calves regardless of the antibody titers of the mothers. Our results support the hypothesis that camels are a reservoir for MERS-CoV and that camel trade is an important route of introducing the virus into importing countries. Findings related to waning antibodies and re-infection have implications for camel vaccine development, disease management and zoonotic threat.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jan 4</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe human infections and dromedary camels are considered an intermediary host. The dynamics of natural infection in camels are not well understood. Through systematic surveillance in Egypt, nasal, rectal, milk, urine and serum samples were collected from camels between June 2014 and February 2016. Locations included quarantines, markets, abattoirs, free-roaming herds and farmed breeding herds. The overall seroprevalence was 71% and RNA detection rate was 15%. Imported camels had higher seroprevalence (90% vs 61%) and higher RT-PCR detection rates (21% vs 12%) than locally raised camels. Juveniles had lower seroprevalence than adults (37% vs 82%) but similar RT-PCR detection rates (16% vs 15%). An outbreak in a breeding herd, showed that antibodies rapidly wane, that camels become re-infected, and that outbreaks in a herd are sustained for an extended time. Maternal antibodies titers were very low in calves regardless of the antibody titers of the mothers. Our results support the hypothesis that camels are a reservoir for MERS-CoV and that camel trade is an important route of introducing the virus into importing countries. Findings related to waning antibodies and re-infection have implications for camel vaccine development, disease management and zoonotic threat.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Systematic, active surveillance for Middle East respiratory syndrome coronavirus in camels in Egypt</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jan 4</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes severe human infections and dromedary camels are considered an intermediary host. The dynamics of natural infection in camels are not well understood. Through systematic surveillance in Egypt, nasal, rectal, milk, urine and serum samples were collected from camels between June 2014 and February 2016. Locations included quarantines, markets, abattoirs, free-roaming herds and farmed breeding herds. The overall seroprevalence was 71% and RNA detection rate was 15%. Imported camels had higher seroprevalence (90% vs 61%) and higher RT-PCR detection rates (21% vs 12%) than locally raised camels. Juveniles had lower seroprevalence than adults (37% vs 82%) but similar RT-PCR detection rates (16% vs 15%). An outbreak in a breeding herd, showed that antibodies rapidly wane, that camels become re-infected, and that outbreaks in a herd are sustained for an extended time. Maternal antibodies titers were very low in calves regardless of the antibody titers of the mothers. Our results support the hypothesis that camels are a reservoir for MERS-CoV and that camel trade is an important route of introducing the virus into importing countries. Findings related to waning antibodies and re-infection have implications for camel vaccine development, disease management and zoonotic threat.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Is it super-spreading?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">If the covid-19 virus is transmitted largely by superspreaders, it might not go pandemic, reports Debora MacKenzie</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Preprints bring ‘firehose’ of outbreak data</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Wu-han Clan is just one example of how the COVID-19 outbreak is transforming how scientists communicate about fast-moving health crises. A torrent of data is being released daily by preprint servers that didn't even exist a decade ago, then dissected on platforms such as Slack and Twitter, and in the media, before formal peer review begins. Journal staffers are working overtime to get manuscripts reviewed, edited, and published at record speeds. The venerable New England Journal of Medicine (NEJM) posted one COVID-19 paper within 48 hours of submission. Viral genomes posted on a platform named GISAID, more than 200 so far, are analyzed instantaneously by a phalanx of evolutionary biologists who share their phylogenetic trees in preprints and on social media.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Passive Immunotherapy with Dromedary Immune Serum in an Experimental Animal Model for Middle East Respiratory Syndrome Coronavirus Infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome (MERS) is a highly lethal pulmonary infection. Serum from convalescent MERS patients may provide some benefit but is not readily available. In contrast, nearly all camels in the Middle East have been infected with MERS-CoV. Here, we show that sera obtained from MERS-immune camels augment the kinetics of MERS-CoV clearance and reduce the severity of pathological changes in infected lungs, with efficacy proportional to the titer of MERS-CoV-neutralizing serum antibody. IMPORTANCE Middle East respiratory syndrome, caused by a coronavirus, is highly lethal, with a case fatality rate of 35 to 40%. No specific therapy is available, and care is generally supportive. One promising approach is passive administration of sera from convalescent human MERS patients or other animals to exposed or infected patients. The vast majority of, if not all, camels in the Middle East have been infected with MERS-CoV, and some contain high titers of antibody to the virus. Here, we show that this antibody is protective if delivered either prophylactically or therapeutically to mice infected with MERS-CoV, indicating that this may be a useful intervention in infected patients.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk of COVID-19 for patients with cancer</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The authors concluded by use of epidemiological statistics that because the proportion of patients with cancer histories was higher in a cohort with COVID-19 than in the population in China, patients with cancer were more likely to develop COVID-19. They found 18 COVID-19 patients with cancer histories among 1590 COVID-19 patients from 575 hospitals in 31 provincial regions. Of these 16 patients (two of the 18 patients had unknown treatment status), only four had undergone surgery or chemotherapy within the previous month; 12 had recovered from initial cancer treatments (eg, surgery or chemotherapy) and had no obvious immunosuppression. We therefore do not think the COVID-19 infections in the 12 survivors of previous cancers were associated with their cancers. COVID-19 is a highly contagious infection to which everyone, to our knowledge, is susceptible; the most important morbidity factor is exposure to an infection source.2 Furthermore, although the authors indicate that patients with cancer had worse outcomes from COVID-19, they also reported the median age of these patients (63·1 years) to be significantly higher than for those without cancer (48·7 years), suggesting that older age is associated with worse COVID-19 outcomes</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Broad Spectrum Antiviral Agent Niclosamide and Its Therapeutic Potential</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The recent outbreak of coronavirus disease 2019 (COVID-19) highlights an urgent need for therapeutics. Through a series of drug repurposing screening campaigns, niclosamide, an FDA-approved anthelminthic drug, was found to be effective against various viral infections with nanomolar to micromolar potency such as SARS-CoV, MERS-CoV, ZIKV, HCV and human adenovirus, indicating its potential as an antiviral agent. In this brief article, we summarize the broad antiviral activities of niclosamide and highlight its potential clinical use for treatment of COVID-19.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. METHODS: Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. RESULTS: The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. CONCLUSION: SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>A case of 2019 Novel Coronavirus in a pregnant woman with preterm delivery</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We presented a case of a 30-week pregnant woman with COVID-19 delivering a healthy baby with no evidence of COVID-19.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Blocking transmission of Middle East respiratory syndrome coronavirus (MERS-CoV) in llamas by vaccination with a recombinant spike protein</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 12</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing Middle East respiratory syndrome coronavirus (MERS-CoV) outbreaks pose a worldwide public health threat. Blocking MERS-CoV zoonotic transmission from dromedary camels, the animal reservoir, could potentially reduce the number of primary human cases. Here we report MERS-CoV transmission from experimentally infected llamas to naïve animals. Directly inoculated llamas shed virus for at least 6 days and could infect all in-contact naïve animals 4–5 days after exposure. With the aim to block virus transmission, we examined the efficacy of a recombinant spike S1-protein vaccine. In contrast to naïve animals, in-contact vaccinated llamas did not shed infectious virus upon exposure to directly inoculated llamas, consistent with the induction of strong virus neutralizing antibody responses. Our data provide further evidence that vaccination of the reservoir host may impede MERS-CoV zoonotic transmission to humans.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Host Determinants of MERS-CoV Transmission and Pathogenesis</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 19</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic pathogen that causes respiratory infection in humans, ranging from asymptomatic to severe pneumonia. In dromedary camels, the virus only causes a mild infection but it spreads efficiently between animals. Differences in the behavior of the virus observed between individuals, as well as between humans and dromedary camels, highlight the role of host factors in MERS-CoV pathogenesis and transmission. One of these host factors, the MERS-CoV receptor dipeptidyl peptidase-4 (DPP4), may be a critical determinant because it is variably expressed in MERS-CoV-susceptible species as well as in humans. This could partially explain inter- and intraspecies differences in the tropism, pathogenesis, and transmissibility of MERS-CoV. In this review, we explore the role of DPP4 and other host factors in MERS-CoV transmission and pathogenesis—such as sialic acids, host proteases, and interferons. Further characterization of these host determinants may potentially offer novel insights to develop intervention strategies to tackle ongoing outbreaks.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Adjusted age-specific case fatality ratio during the COVID-19 epidemic in Hubei, China, January and February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The coronavirus disease 2019 (COVID-19) epidemic that originated in Wuhan, China has spread to more than 60 countries. We estimated the age-specific case fatality ratio (CFR) by fitting a transmission model to data from China, accounting for underreporting of cases and the time delay to death. Overall CFR among all infections was 1.6% (1.4-1.8%) and increased considerably for the elderly, highlighting the expected burden for populations with further expansion of the COVID-19 epidemic around the globe.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission potential of the novel coronavirus (COVID-19) onboard the Diamond Princess Cruises Ship, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of COVID-19 developed aboard the Princess Cruises Ship during January-February 2020. Using mathematical modeling and time-series incidence data describing the trajectory of the outbreak among passengers and crew members, we characterize how the transmission potential varied over the course of the outbreak. Our estimate of the mean reproduction number in the confined setting reached values as high as ∼11, which is higher than mean estimates reported from community-level transmission dynamics in China and Singapore (approximate range: 1.1-7). Our findings suggest that Rt decreased substantially compared to values during the early phase after the Japanese government implemented an enhanced quarantine control. Most recent estimates of Rt reached values largely below the epidemic threshold, indicating that a secondary outbreak of the novel coronavirus was unlikely to occur aboard the Diamond Princess Ship.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Coronavirus Infection in Hospitalized Infants Under 1 Year of Age in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Previous studies suggest that COVID-19 is more likely to infect older adult men, particularly those with chronic comorbidities.2-4 Few infections in children have been reported. We identified all infected infants in China and described demographic, epidemiologic, and clinical features.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Longitudinal study of Middle East Respiratory Syndrome coronavirus infection in dromedary camel herds in Saudi Arabia, 2014–2015</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 21</span></i><p style="text-align: justify; text-justify: inter-word;">Two herds of dromedary camels were longitudinally sampled with nasal and rectal swabs and serum, between September 2014 and May 2015, and the samples were tested for Middle East Respiratory Syndrome (MERS) coronavirus RNA and antibodies. Evidence of MERS-CoV infection was confirmed in one herd on the basis of detection of virus RNA in nasal swabs from three camels and significant increases in the antibody titers from three others. The three viruses were genetically identical, thus indicating introduction of a single virus into this herd. There was evidence of reinfection of camels that were previously seropositive, thus suggesting that prior infection does not provide complete immunity from reinfection, a finding that is relevant to camel vaccination strategies as a means to prevent zoonotic transmission.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>A new transmission route for the propagation of the SARS-CoV-2 coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of a Novel Influenza Virus in Cattle and Swine: Proposal for a New Genus in the Orthomyxoviridae Family</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 4</span></i><p style="text-align: justify; text-justify: inter-word;">We have recently reported the isolation of a novel virus, provisionally designated C/swine/Oklahoma/1334/2011 (C/OK), with 50% overall homology to human influenza C viruses (ICV), from a pig in Oklahoma. Deep RNA sequencing of C/OK virus found a matrix 1 (M1) protein expression strategy that differed from that of ICV. The novelty of C/OK virus prompted us to investigate whether C/OK virus could exist in a nonswine species. Significantly, we found that C/OK virus was widespread in U.S. bovine herds, as demonstrated by reverse transcription (RT)-PCR and serological assays. Genome sequencing of three bovine viruses isolated from two herds in different states further confirmed these findings. To determine whether swine/bovine C/OK viruses can undergo reassortment with human ICV, and to clarify the taxonomic status of C/OK, in vitro reassortment and serological typing by agar gel immunodiffusion (AGID) were conducted. In vitro reassortment using two human ICV and two swine and bovine C/OK viruses demonstrated that human ICV and C/OK viruses were unable to reassort and produce viable progeny. Antigenically, no cross-recognition of detergent split virions was observed in AGID between human and nonhuman viruses by using polyclonal antibodies that were reactive to cognate antigens. Taken together, these results demonstrate that C/OK virus is genetically and antigenically distinct from ICV. The classification of the new virus in a separate genus of the Orthomyxoviridae family is proposed. The finding of C/OK virus in swine and bovine indicates that this new virus may spread and establish infection in other mammals, including humans.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of a Novel Influenza Virus in Cattle and Swine: Proposal for a New Genus in the Orthomyxoviridae Family</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 4</span></i><p style="text-align: justify; text-justify: inter-word;">We have recently reported the isolation of a novel virus, provisionally designated C/swine/Oklahoma/1334/2011 (C/OK), with 50% overall homology to human influenza C viruses (ICV), from a pig in Oklahoma. Deep RNA sequencing of C/OK virus found a matrix 1 (M1) protein expression strategy that differed from that of ICV. The novelty of C/OK virus prompted us to investigate whether C/OK virus could exist in a nonswine species. Significantly, we found that C/OK virus was widespread in U.S. bovine herds, as demonstrated by reverse transcription (RT)-PCR and serological assays. Genome sequencing of three bovine viruses isolated from two herds in different states further confirmed these findings. To determine whether swine/bovine C/OK viruses can undergo reassortment with human ICV, and to clarify the taxonomic status of C/OK, in vitro reassortment and serological typing by agar gel immunodiffusion (AGID) were conducted. In vitro reassortment using two human ICV and two swine and bovine C/OK viruses demonstrated that human ICV and C/OK viruses were unable to reassort and produce viable progeny. Antigenically, no cross-recognition of detergent split virions was observed in AGID between human and nonhuman viruses by using polyclonal antibodies that were reactive to cognate antigens. Taken together, these results demonstrate that C/OK virus is genetically and antigenically distinct from ICV. The classification of the new virus in a separate genus of the Orthomyxoviridae family is proposed. The finding of C/OK virus in swine and bovine indicates that this new virus may spread and establish infection in other mammals, including humans.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of a Novel Influenza Virus in Cattle and Swine: Proposal for a New Genus in the Orthomyxoviridae Family</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 4</span></i><p style="text-align: justify; text-justify: inter-word;">We have recently reported the isolation of a novel virus, provisionally designated C/swine/Oklahoma/1334/2011 (C/OK), with 50% overall homology to human influenza C viruses (ICV), from a pig in Oklahoma. Deep RNA sequencing of C/OK virus found a matrix 1 (M1) protein expression strategy that differed from that of ICV. The novelty of C/OK virus prompted us to investigate whether C/OK virus could exist in a nonswine species. Significantly, we found that C/OK virus was widespread in U.S. bovine herds, as demonstrated by reverse transcription (RT)-PCR and serological assays. Genome sequencing of three bovine viruses isolated from two herds in different states further confirmed these findings. To determine whether swine/bovine C/OK viruses can undergo reassortment with human ICV, and to clarify the taxonomic status of C/OK, in vitro reassortment and serological typing by agar gel immunodiffusion (AGID) were conducted. In vitro reassortment using two human ICV and two swine and bovine C/OK viruses demonstrated that human ICV and C/OK viruses were unable to reassort and produce viable progeny. Antigenically, no cross-recognition of detergent split virions was observed in AGID between human and nonhuman viruses by using polyclonal antibodies that were reactive to cognate antigens. Taken together, these results demonstrate that C/OK virus is genetically and antigenically distinct from ICV. The classification of the new virus in a separate genus of the Orthomyxoviridae family is proposed. The finding of C/OK virus in swine and bovine indicates that this new virus may spread and establish infection in other mammals, including humans.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of a Novel Influenza Virus in Cattle and Swine: Proposal for a New Genus in the Orthomyxoviridae Family</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 4</span></i><p style="text-align: justify; text-justify: inter-word;">We have recently reported the isolation of a novel virus, provisionally designated C/swine/Oklahoma/1334/2011 (C/OK), with 50% overall homology to human influenza C viruses (ICV), from a pig in Oklahoma. Deep RNA sequencing of C/OK virus found a matrix 1 (M1) protein expression strategy that differed from that of ICV. The novelty of C/OK virus prompted us to investigate whether C/OK virus could exist in a nonswine species. Significantly, we found that C/OK virus was widespread in U.S. bovine herds, as demonstrated by reverse transcription (RT)-PCR and serological assays. Genome sequencing of three bovine viruses isolated from two herds in different states further confirmed these findings. To determine whether swine/bovine C/OK viruses can undergo reassortment with human ICV, and to clarify the taxonomic status of C/OK, in vitro reassortment and serological typing by agar gel immunodiffusion (AGID) were conducted. In vitro reassortment using two human ICV and two swine and bovine C/OK viruses demonstrated that human ICV and C/OK viruses were unable to reassort and produce viable progeny. Antigenically, no cross-recognition of detergent split virions was observed in AGID between human and nonhuman viruses by using polyclonal antibodies that were reactive to cognate antigens. Taken together, these results demonstrate that C/OK virus is genetically and antigenically distinct from ICV. The classification of the new virus in a separate genus of the Orthomyxoviridae family is proposed. The finding of C/OK virus in swine and bovine indicates that this new virus may spread and establish infection in other mammals, including humans.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Sequence of the Novel Reassortant Mammalian Orthoreovirus Strain MRV00304/13, Isolated from a Calf with Diarrhea from the United States</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 May 29</span></i><p style="text-align: justify; text-justify: inter-word;">Mammalian orthoreovirus (MRV) strain MRV00304/13 was isolated from diarrheic calves. The serotype-specific antigen σ1 was found to be 95% identical to that of bovine MRV1. All predicted viral proteins had >92% identity to those of MRV except µ2 and σ1s (80 and 72% identities, respectively), suggesting that MRV00304/13 is a novel reassortant MRV1.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>CORONAVIRUS IN PREGNANCY AND DELIVERY: RAPID REVIEW AND EXPERT CONSENSUS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND Person to person spread of COIVD-19 in the UK has now been confirmed. There are limited case series reporting the impact on women affected by coronaviruses (CoV) during pregnancy. In women affected by SARS and MERS, the case fatality rate appeared higher in women affected in pregnancy compared with non-pregnant women. We conducted a rapid, review to guide management of women affected by COVID -19 during pregnancy and developed interim practice guidance with the RCOG and RCPCH to inform maternity and neonatal service planning METHODS Searches were conducted in PubMed and MedRxiv to identify primary case reports, case series, observational studies or randomised-controlled trial describing women affected by coronavirus in pregnancy and on neonates. Data was extracted from relevant papers and the review was drafted with representatives of the RCPCH and RCOG who also provided expert consensus on areas where data were lacking  RESULTS From 9964 results on PubMed and 600 on MedRxiv, 18 relevant studies (case reports and case series) were identified. There was inconsistent reporting of maternal, perinatal and neonatal outcomes across case reports and series concerning COVID-19, SARS, MERS and other coronaviruses. From reports of 19 women to date affected by COVID-19 in pregnancy, delivering 20 babies, 3 (16%) were asymptomatic, 1 (5%) was admitted to ICU and no maternal deaths have been reported. Deliveries were 17 by caesarean section, 2 by vaginal delivery, 8 (42%) delivered pre-term. There was one neonatal death, in 15 babies who were tested there was no evidence of vertical transmission. CONCLUSIONS Morbidity and mortality from COVID-19 appears less marked than for SARS and MERS, acknowledging the limited number of cases reported to date. Pre-term delivery affected 42% of women hospitalised with COVID-19, which may put considerable pressure on neonatal services if the UK reasonable worse-case scenario of 80% of the population affected is realised. There has been no evidence of vertical transmission to date.  The RCOG and RCPCH have provided interim guidance to help maternity and neonatal services plan their response to COVID-19.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Chimeric camel/human heavy-chain antibodies protect against MERS-CoV infection</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Aug 8</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) continues to cause outbreaks in humans as a result of spillover events from dromedaries. In contrast to humans, MERS-CoV–exposed dromedaries develop only very mild infections and exceptionally potent virus-neutralizing antibody responses. These strong antibody responses may be caused by affinity maturation as a result of repeated exposure to the virus or by the fact that dromedaries—apart from conventional antibodies—have relatively unique, heavy chain–only antibodies (HCAbs). These HCAbs are devoid of light chains and have long complementarity-determining regions with unique epitope binding properties, allowing them to recognize and bind with high affinity to epitopes not recognized by conventional antibodies. Through direct cloning and expression of the variable heavy chains (VHHs) of HCAbs from the bone marrow of MERS-CoV–infected dromedaries, we identified several MERS-CoV–specific VHHs or nanobodies. In vitro, these VHHs efficiently blocked virus entry at picomolar concentrations. The selected VHHs bind with exceptionally high affinity to the receptor binding domain of the viral spike protein. Furthermore, camel/human chimeric HCAbs—composed of the camel VHH linked to a human Fc domain lacking the CH1 exon—had an extended half-life in the serum and protected mice against a lethal MERS-CoV challenge. HCAbs represent a promising alternative strategy to develop novel interventions not only for MERS-CoV but also for other emerging pathogens.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Chimeric camel/human heavy-chain antibodies protect against MERS-CoV infection</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Aug 8</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) continues to cause outbreaks in humans as a result of spillover events from dromedaries. In contrast to humans, MERS-CoV–exposed dromedaries develop only very mild infections and exceptionally potent virus-neutralizing antibody responses. These strong antibody responses may be caused by affinity maturation as a result of repeated exposure to the virus or by the fact that dromedaries—apart from conventional antibodies—have relatively unique, heavy chain–only antibodies (HCAbs). These HCAbs are devoid of light chains and have long complementarity-determining regions with unique epitope binding properties, allowing them to recognize and bind with high affinity to epitopes not recognized by conventional antibodies. Through direct cloning and expression of the variable heavy chains (VHHs) of HCAbs from the bone marrow of MERS-CoV–infected dromedaries, we identified several MERS-CoV–specific VHHs or nanobodies. In vitro, these VHHs efficiently blocked virus entry at picomolar concentrations. The selected VHHs bind with exceptionally high affinity to the receptor binding domain of the viral spike protein. Furthermore, camel/human chimeric HCAbs—composed of the camel VHH linked to a human Fc domain lacking the CH1 exon—had an extended half-life in the serum and protected mice against a lethal MERS-CoV challenge. HCAbs represent a promising alternative strategy to develop novel interventions not only for MERS-CoV but also for other emerging pathogens.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /></body>