
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: surveillance of mixed wildlife-livestock farms for coronaviruses</h2><br /></div><br /><br /><br /><div class="card"><h3>Potential Intermediate Hosts for Coronavirus Transmission: No Evidence of Clade 2c Coronaviruses in Domestic Livestock from Ghana</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 10</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV), nearly a decade ago with worldwide distribution, was believed to be of zoonotic origin from bats with dromedary camels as intermediate hosts. There is a likelihood of other domestic livestock serving as intermediate hosts for this virus. The presence of coronaviruses, closely related to MERS-CoV in Ghanaian bats, presented the opportunity to test the hypothesis of transmissibility of this virus through domestic livestock species. The possible interactions between livestock and bats in 31 household farms were accessed by observation and interviews with farmers. Rectal swabs and serum from cattle, sheep, goats, donkeys, and swine from commercial and household farms were tested for MERS-CoV and a Nycteris sp. bat coronavirus, previously detected in Ghana. A pan-PCR assay to detect clade 2c viruses and recombinant immunofluorescence assay to detect anti-spike IgG antibodies against the target viruses were used. Likely contact between livestock and bats was determined for 13 farms (41.9%) that reported confining their livestock and also observing bats in their homes. Livestock were left unconfined on eight farms (25.8%) that also observed bats roosting in trees close to their homes. No viral RNA or antibodies against the two coronaviruses were detected in any of the livestock species tested. Cattle, sheep, goats, donkeys, and swine are not likely hosts of clade 2c coronaviruses.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>A strategy to prevent future pandemics similar to the 2019-nCoV outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel bat-origin coronavirus emerged in Wuhan, China in December 2019 and continues to spread across China and the world. At the time of writing, a massive global response has been implemented to control the disease as it spreads from person to person. Yet the high-risk human-wildlife interactions and interfaces that led to the emergence of SARS-CoV and of 2019-nCoV continue to exist in emerging disease hotspots globally. To prevent the next pandemic related to these interfaces, we call for research and investment in three areas: 1) surveillance among wildlife to identify the high-risk pathogens they carry; 2) surveillance among people who have contact with wildlife to identify early spillover events; and 3) improvement of market biosecurity regarding the wildlife trade. As the emergence of a novel virus anywhere can impact the furthest reaches of our connected world, international collaboration among scientists is essential to address these risks and prevent the next pandemic.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Alpha and Beta Coronavirus in Wildlife Species in France: Bats, Rodents, Rabbits, and Hedgehogs</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 29</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are closely monitored in the context of emerging diseases and, as illustrated with Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome-coronavirus (MERS-CoV), are known to cross the species barrier and eventually to move from wildlife to humans. Knowledge of the diversity of coronaviruses in wildlife is therefore essential to better understand and prevent emergence events. This study explored the presence of coronaviruses in four wild mammal orders in France: Bats, rodents, lagomorphs, and hedgehogs. Betacoronavirus and Alphacoronavirus genera were identified. The results obtained suggest the circulation of potentially evolving virus strains, with the potential to cross the species barrier.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin</h3>
<i class="tooltip">2018<span class="tooltiptext">2018</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of viruses from wildlife animal reservoirs poses a marked threat to human and animal health (1) . Bats have been recognized as one of the most important reservoirs for emerging viruses and the transmission of a coronavirus that originated in bats to humans via intermediate hosts was responsible for the high-impact emerging zoonosis, severe acute respiratory syndrome (SARS) (2-10) . Here we provide virological, epidemiological, evolutionary and experimental evidence that a novel HKU2-related bat coronavirus, swine acute diarrhoea syndrome coronavirus (SADS-CoV), is the aetiological agent that was responsible for a large-scale outbreak of fatal disease in pigs in China that has caused the death of 24,693 piglets across four farms. Notably, the outbreak began in Guangdong province in the vicinity of the origin of the SARS pandemic. Furthermore, we identified SADS-related CoVs with 96-98% sequence identity in 9.8% (58 out of 591) of anal swabs collected from bats in Guangdong province during 2013-2016, predominantly in horseshoe bats (Rhinolophus spp.) that are known reservoirs of SARS-related CoVs. We found that there were striking similarities between the SADS and SARS outbreaks in geographical, temporal, ecological and aetiological settings. This study highlights the importance of identifying coronavirus diversity and distribution in bats to mitigate future outbreaks that could threaten livestock, public health and economic growth.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Searching for animal models and potential target species for emerging pathogens: Experience gained from Middle East respiratory syndrome (MERS) coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Anticipating the Species Jump: Surveillance for Emerging Viral Threats</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Zoonotic disease surveillance is typically triggered after animal pathogens have already infected humans. Are there ways to identify high-risk viruses before they emerge in humans? If so, then how and where can identifications be made and by what methods? These were the fundamental questions driving a workshop to examine the future of predictive surveillance for viruses that might jump from animals to infect humans. Virologists, ecologists and computational biologists from academia, federal government and non-governmental organizations discussed opportunities as well as obstacles to the prediction of species jumps using genetic and ecological data from viruses and their hosts, vectors and reservoirs. This workshop marked an important first step towards envisioning both scientific and organizational frameworks for this future capability. Canine parvoviruses as well as seasonal H3N2 and pandemic H1N1 influenza viruses are discussed as exemplars that suggest what to look for in anticipating species jumps. To answer the question of where to look, prospects for discovering emerging viruses among wildlife, bats, rodents, arthropod vectors and occupationally exposed humans are discussed. Finally, opportunities and obstacles are identified and accompanied by suggestions for how to look for species jumps. Taken together, these findings constitute the beginnings of a conceptual framework for achieving a virus surveillance capability that could predict future species jumps.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Avian Coronavirus and Fulminating Disease in Guinea Fowl, France</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">For decades, French guinea fowl have been affected by fulminating enteritis of unclear origin. By using metagenomics, we identified a novel avian gammacoronavirus associated with this disease that is distantly related to turkey coronaviruses. Fatal respiratory diseases in humans have recently been caused by coronaviruses of animal origin.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Severe Acute Respiratory Syndrome Coronavirus-Like Virus in a Leaf-Nosed Bat in Nigeria</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for emerging zoonotic viruses that can have a profound impact on human and animal health, including lyssaviruses, filoviruses, paramyxoviruses, and severe acute respiratory syndrome coronaviruses (SARS-CoVs). In the course of a project focused on pathogen discovery in contexts where human-bat contact might facilitate more efficient interspecies transmission of viruses, we surveyed gastrointestinal tissue obtained from bats collected in caves in Nigeria that are frequented by humans. Coronavirus consensus PCR and unbiased high-throughput pyrosequencing revealed the presence of coronavirus sequences related to those of SARS-CoV in a Commerson’s leaf-nosed bat (Hipposideros commersoni). Additional genomic sequencing indicated that this virus, unlike subgroup 2b CoVs, which includes SARS-CoV, is unique, comprising three overlapping open reading frames between the M and N genes and two conserved stem-loop II motifs. Phylogenetic analyses in conjunction with these features suggest that this virus represents a new subgroup within group 2 CoVs.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Alphacoronaviruses Detected in French Bats Are Phylogeographically Linked to Coronaviruses of European Bats</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 2</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are a reservoir for a diverse range of viruses, including coronaviruses (CoVs). To determine the presence of CoVs in French bats, fecal samples were collected between July and August of 2014 from four bat species in seven different locations around the city of Bourges in France. We present for the first time the presence of alpha-CoVs in French Pipistrellus pipistrellus bat species with an estimated prevalence of 4.2%. Based on the analysis of a fragment of the RNA-dependent RNA polymerase (RdRp) gene, phylogenetic analyses show that alpha-CoVs sequences detected in French bats are closely related to other European bat alpha-CoVs. Phylogeographic analyses of RdRp sequences show that several CoVs strains circulate in European bats: (i) old strains detected that have probably diverged a long time ago and are detected in different bat subspecies; (ii) strains detected in Myotis and Pipistrellus bat species that have more recently diverged. Our findings support previous observations describing the complexity of the detected CoVs in bats worldwide.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Sequence Analysis of Four Deltacoronaviruses from Birds in the Middle East Reveal Interspecies Jumping with Recombination as a Potential Mechanism for Avian-to-Avian and Avian-to-Mammalian Transmission</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome showed once again that coronaviruses (CoVs) in animals are potential source for epidemics in humans. To explore the diversity of deltacoronaviruses in animals in the Middle East, we tested fecal samples from 1,356 mammals and birds in Dubai, The United Arab Emirates. Four novel deltacoronaviruses were detected from eight birds of four species by reverse transcription-PCR (RT-PCR): FalCoV UAE-HKU27 from a falcon, HouCoV UAE-HKU28 from a houbara bustard, PiCoV UAE-HKU29 from a pigeon, and QuaCoV UAE-HKU30 from five quails. Complete genome sequencing showed that FalCoV UAE-HKU27, HouCoV UAE-HKU28, and PiCoV UAE-HKU29 belong to the same CoV species, suggesting recent interspecies transmission between falcons and their prey, houbara bustards and pigeons, possibly along the food chain. Western blotting detected specific anti-FalCoV UAE-HKU27 antibodies in 33 (75%) of 44 falcon serum samples, supporting genuine infection in falcons after virus acquisition. QuaCoV UAE-HKU30 belongs to the same CoV species as porcine coronavirus HKU15 (PorCoV HKU15) and sparrow coronavirus HKU17 (SpCoV HKU17), discovered previously from swine and tree sparrows, respectively, supporting avian-to-swine transmission. Recombination involving the spike protein is common among deltacoronaviruses, which may facilitate cross-species transmission. FalCoV UAE-HKU27, HouCoV UAE-HKU28, and PiCoV UAE-HKU29 originated from recombination between white-eye coronavirus HKU16 (WECoV HKU16) and magpie robin coronavirus HKU18 (MRCoV HKU18), QuaCoV UAE-HKU30 from recombination between PorCoV HKU15/SpCoV HKU17 and munia coronavirus HKU13 (MunCoV HKU13), and PorCoV HKU15 from recombination between SpCoV HKU17 and bulbul coronavirus HKU11 (BuCoV HKU11). Birds in the Middle East are hosts for diverse deltacoronaviruses with potential for interspecies transmission. IMPORTANCE During an attempt to explore the diversity of deltacoronaviruses among mammals and birds in Dubai, four novel deltacoronaviruses were detected in fecal samples from eight birds of four different species: FalCoV UAE-HKU27 from a falcon, HouCoV UAE-HKU28 from a houbara bustard, PiCoV UAE-HKU29 from a pigeon, and QuaCoV UAE-HKU30 from five quails. Genome analysis revealed evidence of recent interspecies transmission between falcons and their prey, houbara bustards and pigeons, possibly along the food chain, as well as avian-to-swine transmission. Recombination, which is known to occur frequently in some coronaviruses, was also common among these deltacoronaviruses and occurred predominantly at the spike region. Such recombination, involving the receptor binding protein, may contribute to the emergence of new viruses capable of infecting new hosts. Birds in the Middle East are hosts for diverse deltacoronaviruses with potential for interspecies transmission.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Alphacoronaviruses in New World Bats: Prevalence, Persistence, Phylogeny, and Potential for Interaction with Humans</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 May 12</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoirs for many different coronaviruses (CoVs) as well as many other important zoonotic viruses. We sampled feces and/or anal swabs of 1,044 insectivorous bats of 2 families and 17 species from 21 different locations within Colorado from 2007 to 2009. We detected alphacoronavirus RNA in bats of 4 species: big brown bats (Eptesicus fuscus), 10% prevalence; long-legged bats (Myotis volans), 8% prevalence; little brown bats (Myotis lucifugus), 3% prevalence; and western long-eared bats (Myotis evotis), 2% prevalence. Overall, juvenile bats were twice as likely to be positive for CoV RNA as adult bats. At two of the rural sampling sites, CoV RNAs were detected in big brown and long-legged bats during the three sequential summers of this study. CoV RNA was detected in big brown bats in all five of the urban maternity roosts sampled throughout each of the periods tested. Individually tagged big brown bats that were positive for CoV RNA and later sampled again all became CoV RNA negative. Nucleotide sequences in the RdRp gene fell into 3 main clusters, all distinct from those of Old World bats. Similar nucleotide sequences were found in amplicons from gene 1b and the spike gene in both a big-brown and a long-legged bat, indicating that a CoV may be capable of infecting bats of different genera. These data suggest that ongoing evolution of CoVs in bats creates the possibility of a continued threat for emergence into hosts of other species. Alphacoronavirus RNA was detected at a high prevalence in big brown bats in roosts in close proximity to human habitations (10%) and known to have direct contact with people (19%), suggesting that significant potential opportunities exist for cross-species transmission of these viruses. Further CoV surveillance studies in bats throughout the Americas are warranted.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Jumping species – A Mechanism for Coronavirus Persistence and Survival</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Zoonotic transmission of novel viruses represents a significant threat to global public health and is fueled by globalization, the loss of natural habitats, and exposure to new hosts. For coronaviruses (CoVs), broad diversity exists within bat populations and uniquely positions them to seed future emergence events. In this review, we explore the host and viral dynamics that shape these CoV populations for survival, amplification, and possible emergence in novel hosts.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of Novel Alphacoronaviruses in European Rodents and Shrews</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 18</span></i><p style="text-align: justify; text-justify: inter-word;">Eight hundred and thirteen European rodents and shrews encompassing seven different species were screened for alphacoronaviruses using PCR detection. Novel alphacoronaviruses were detected in the species Rattus norvegicus, Microtus agrestis, Sorex araneus and Myodes glareolus. These, together with the recently described Lucheng virus found in China, form a distinct rodent/shrew-specific clade within the coronavirus phylogeny. Across a highly conserved region of the viral polymerase gene, the new members of this clade were up to 22% dissimilar at the nucleotide level to the previously described Lucheng virus. As such they might represent distinct species of alphacoronaviruses. These data greatly extend our knowledge of wildlife reservoirs of alphacoronaviruses.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Multiple Novel Viruses, Including a Parvovirus and a Hepevirus, in Feces of Red Foxes</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Red foxes (Vulpes vulpes) are the most widespread members of the order of Carnivora. Since they often live in (peri)urban areas, they are a potential reservoir of viruses that transmit from wildlife to humans or domestic animals. Here we evaluated the fecal viral microbiome of 13 red foxes by random PCR in combination with next-generation sequencing. Various novel viruses, including a parvovirus, bocavirus, adeno-associated virus, hepevirus, astroviruses, and picobirnaviruses, were identified.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of a Highly Divergent Coronavirus in the Asian House Shrew from China Illuminates the Origin of the Alphacoronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although shrews are one of the largest groups of mammals, little is known about their role in the evolution and transmission of viral pathogens, including coronaviruses (CoVs). We captured 266 Asian house shrews (Suncus murinus) in Jiangxi and Zhejiang Provinces, China, during 2013 to 2015. CoV RNA was detected in 24 Asian house shrews, with an overall prevalence of 9.02%. Complete viral genome sequences were successfully recovered from the RNA-positive samples. The newly discovered shrew CoV fell into four lineages reflecting their geographic origins, indicative of largely allopatric evolution. Notably, these viruses were most closely related to alphacoronaviruses but sufficiently divergent that they should be considered a novel member of the genus Alphacoronavirus, which we denote Wénchéng shrew virus (WESV). Phylogenetic analysis revealed that WESV was a highly divergent member of the alphacoronaviruses and, more dramatically, that the S gene of WESV fell in a cluster that was genetically distinct from that of known coronaviruses. The divergent position of WESV suggests that coronaviruses have a long association with Asian house shrews. In addition, the genome of WESV contains a distinct NS7 gene that exhibits no sequence similarity to genes of any known viruses. Together, these data suggest that shrews are natural reservoirs for coronaviruses and may have played an important and long-term role in CoV evolution. IMPORTANCE The subfamily Coronavirinae contains several notorious human and animal pathogens, including severe acute respiratory syndrome coronavirus, Middle East respiratory syndrome coronavirus, and porcine epidemic diarrhea virus. Because of their genetic diversity and phylogenetic relationships, it has been proposed that the alphacoronaviruses likely have their ultimate ancestry in the viruses residing in bats. Here, we describe a novel alphacoronavirus (Wénchéng shrew virus [WESV]) that was sampled from Asian house shrews in China. Notably, WESV is a highly divergent member of the alphacoronaviruses and possesses an S gene that is genetically distinct from those of all known coronaviruses. In addition, the genome of WESV contains a distinct NS7 gene that exhibits no sequence similarity to those of any known viruses. Together, these data suggest that shrews are important and longstanding hosts for coronaviruses that merit additional research and surveillance.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology of Deltacoronaviruses (δ-CoV) and Gammacoronaviruses (γ-CoV) in Wild Birds in the United States</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Sep 26</span></i><p style="text-align: justify; text-justify: inter-word;">Porcine deltacoronavirus (δ-CoV) is the object of extensive research in several countries including the United States. In contrast, the epidemiology of δ-CoVs in wild birds in the US is largely unknown. Our aim was to comparatively assess the prevalence of δ- and γ-CoVs in wild migratory terrestrial and aquatic birds in Arkansas, Illinois, Indiana, Maryland, Mississippi, Missouri, Ohio, Tennessee and Wisconsin. A total of 1236 cloacal/fecal swabs collected during the period 2015–2018 were tested for γ- and δ-CoVs using genus-specific reverse transcription-PCR assays. A total of 61 (4.99%) samples were γ-CoV positive, with up to 29 positive samples per state. In contrast, only 14 samples were positive for δ-CoV (1.14%) with only 1–4 originating from the same state. Thus, unlike previous reports from Asia, γ-CoVs are more prevalent than δ-CoVs in the US, suggesting that δ-CoVs may spread in birds with lower efficiency. This may indicate δ-CoV emerging status and incomplete adaptation to new host species limiting its spread. Phylogenetic analysis of the partial N gene revealed that the newly identified δ-CoV strains were most closely related to the HKU20 (wigeon) strain. Further studies are necessary to investigate the role of aquatic bird δ-CoVs in the epidemiology of δ-CoVs in swine and terrestrial birds.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Diversity and Phylodynamics of Avian Coronaviruses in Egyptian Wild Birds</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 14</span></i><p style="text-align: justify; text-justify: inter-word;">Avian coronaviruses (ACoVs) are continuously evolving and causing serious economic consequences in the poultry industry and around the globe. Owing to their extensive genetic diversity and high mutation rates, controlling ACoVs has become a challenge. In this context, the potential contribution of wild birds in the disease dynamics, especially in domesticated birds, remains largely unknown. In the present study, five hundred fifty-seven (n = 557) cloacal/fecal swabs were collected from four different wild bird species from eight Egyptian governorates during 2016 and a total of fourteen positive isolates were used for phylodynamics and evolutionary analysis. Genetic relatedness based on spike (S1) gene demonstrated the clustering of majority of these isolates where nine isolates grouped within Egy/variant 2 (IS/885 genotype) and five isolates clustered within Egy/variant 1 (IS/1494/06 genotype). Interestingly, these isolates showed noticeable genetic diversity and were clustered distal to the previously characterized Egy/variant 1 and Egy/variant 2 in Egyptian commercial poultry. The S1 gene based comparison of nucleotide identity percentages revealed that all fourteen isolates reported in this study were genetically related to the variant GI-23 lineage with 92–100% identity. Taken together, our results demonstrate that ACoVs are circulating in Egyptian wild birds and highlight their possible contributions in the disease dynamics. The study also proposes that regular monitoring of the ACoVs in wild birds is required to effectively assess the role of wild birds in disease spread, and the emergence of ACoVs strains in the country.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Novel Coronavirus from a Beluga Whale by Using a Panviral Microarray</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of viruses such as severe acute respiratory syndrome coronavirus and Nipah virus has underscored the role of animal reservoirs in human disease and the need for reservoir surveillance. Here, we used a panviral DNA microarray to investigate the death of a captive beluga whale in an aquatic park. A highly divergent coronavirus, tentatively named coronavirus SW1, was identified in liver tissue from the deceased whale. Subsequently, the entire genome of SW1 was sequenced, yielding a genome of 31,686 nucleotides. Phylogenetic analysis revealed SW1 to be a novel virus distantly related to but most similar to group III coronaviruses.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Coronaviruses in Bats of Various Species in Italy</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Oct 31</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are natural reservoirs for many mammalian coronaviruses, which have received renewed interest after the discovery of the severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) CoV in humans. This study describes the identification and molecular characterization of alphacoronaviruses and betacoronaviruses in bats in Italy, from 2010 to 2012. Sixty-nine faecal samples and 126 carcasses were tested using pan-coronavirus RT-PCR. Coronavirus RNAs were detected in seven faecal samples and nine carcasses. A phylogenetic analysis of RNA-dependent RNA polymerase sequence fragments aided in identifying two alphacoronaviruses from Kuhl’s pipistrelle (Pipistrellus kuhlii), three clade 2b betacoronaviruses from lesser horseshoe bats (Rhinolophus hipposideros), and 10 clade 2c betacoronaviruses from Kuhl’s pipistrelle, common noctule (Nyctalus noctula), and Savi’s pipistrelle (Hypsugo savii). This study fills a substantive gap in the knowledge on bat-CoV ecology in Italy, and extends the current knowledge on clade 2c betacoronaviruses with new sequences obtained from bats that have not been previously described as hosts of these viruses.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Rotavirus strains in neglected animal species including lambs, goats and camelids</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Surveillance of rotavirus infections and circulating strains in small ruminants (i.e. lambs, goats and camelids) has been a neglected research area in the past. However, recent years that have seen an intensification of surveillance in humans and livestock animals, where vaccines to reduce disease burden caused by Rotavirus A (RVA) are available, led to the efforts to better understand the epidemiology, ecology and evolution of RVA strains in other hosts, including lambs, goats and camelids. The aim of this review is to provide an update of the epidemiology and strain diversity of RV strains in these species through searching for relevant information in public data bases.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus (MERS-CoV): A review</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As a novel coronavirus first reported by Saudi Arabia in 2012, the Middle East respiratory syndrome coronavirus (MERS-CoV) is responsible for an acute human respiratory syndrome. The virus, of 2C beta-CoV lineage, expresses the dipeptidyl peptidase 4 (DPP4) receptor and is densely endemic in dromedary camels of East Africa and the Arabian Peninsula. MERS-CoV is zoonotic but human-to-human transmission is also possible. Surveillance and phylogenetic researches indicate MERS-CoV to be closely associated with bats’ coronaviruses, suggesting bats as reservoirs, although unconfirmed. With no vaccine currently available for MERS-CoV nor approved prophylactics, its global spread to over 25 countries with high fatalities highlights its role as ongoing public health threat. An articulated action plan ought to be taken, preferably from a One Health perspective, for appropriately advanced countermeasures against MERS-CoV.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and genetic diversity of coronaviruses in wild birds, Finland</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 28</span></i><p style="text-align: justify; text-justify: inter-word;">Introduction: Migratory birds act as hosts for a number of zoonotic viruses, and have the ability to disperse these viruses to distant geographic locations. Coronaviruses (CoVs) represent a family of zoonotic viruses with wide variety of animal hosts, including birds and humans. The infections caused by coronaviruses vary from mild to severe, depending on the viral species and the host. Since the coronaviruses exhibit extraordinary large RNA genome, also the rate of homologous recombination is high, which in turn contributes to the genetic diversity and interspecies host-switches of CoVs. The emergence of novel CoVs has been rich during the last decades, and wild birds seem to serve as reservoirs for a variety of CoV strains. We examined the CoVs circulating among wild birds in Finland. Materials and methods: Samples (cloacal swab, tracheal swab, oropharyngeal swab, or tissue) representing 61 bird species were collected during 2010-2013, and examined by RT-PCR targeting the RdRp gene for the presence of CoV RNA. Results: Altogether 51/939 (5.4%) of the examined birds were found positive by RT-PCR. Diverse gamma- and deltacoronavirus sequences were detected. Discussion: Gamma- and deltacoronaviruses circulate among wild birds in Finland. The number of CoV-positive birds detected each year varies greatly.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Bat Coronaviruses in China</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 2</span></i><p style="text-align: justify; text-justify: inter-word;">During the past two decades, three zoonotic coronaviruses have been identified as the cause of large-scale disease outbreaks–Severe Acute Respiratory Syndrome (SARS), Middle East Respiratory Syndrome (MERS), and Swine Acute Diarrhea Syndrome (SADS). SARS and MERS emerged in 2003 and 2012, respectively, and caused a worldwide pandemic that claimed thousands of human lives, while SADS struck the swine industry in 2017. They have common characteristics, such as they are all highly pathogenic to humans or livestock, their agents originated from bats, and two of them originated in China. Thus, it is highly likely that future SARS- or MERS-like coronavirus outbreaks will originate from bats, and there is an increased probability that this will occur in China. Therefore, the investigation of bat coronaviruses becomes an urgent issue for the detection of early warning signs, which in turn minimizes the impact of such future outbreaks in China. The purpose of the review is to summarize the current knowledge on viral diversity, reservoir hosts, and the geographical distributions of bat coronaviruses in China, and eventually we aim to predict virus hotspots and their cross-species transmission potential.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Absence of MERS-CoV antibodies in feral camels in Australia: Implications for the pathogen's origin and spread</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Nov 2</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) infections continue to be a serious emerging disease problem internationally with well over 1000 cases and a major outbreak outside of the Middle East region. While the hypothesis that dromedary camels are the likely major source of MERS-CoV infection in humans is gaining acceptance, conjecture continues over the original natural reservoir host(s) and specifically the role of bats in the emergence of the virus. Dromedary camels were imported to Australia, principally between 1880 and 1907 and have since become a large feral population inhabiting extensive parts of the continent. Here we report that during a focussed surveillance study, no serological evidence was found for the presence of MERS-CoV in the camels in the Australian population. This finding presents various hypotheses about the timing of the emergence and spread of MERS-CoV throughout populations of camels in Africa and Asia, which can be partially resolved by testing sera from camels from the original source region, which we have inferred was mainly northwestern Pakistan. In addition, we identify bat species which overlap (or neighbour) the range of the Australian camel population with a higher likelihood of carrying CoVs of the same lineage as MERS-CoV. Both of these proposed follow-on studies are examples of “proactive surveillance”, a concept that has particular relevance to a One Health approach to emerging zoonotic diseases with a complex epidemiology and aetiology.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Absence of MERS-CoV antibodies in feral camels in Australia: Implications for the pathogen's origin and spread</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Nov 2</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) infections continue to be a serious emerging disease problem internationally with well over 1000 cases and a major outbreak outside of the Middle East region. While the hypothesis that dromedary camels are the likely major source of MERS-CoV infection in humans is gaining acceptance, conjecture continues over the original natural reservoir host(s) and specifically the role of bats in the emergence of the virus. Dromedary camels were imported to Australia, principally between 1880 and 1907 and have since become a large feral population inhabiting extensive parts of the continent. Here we report that during a focussed surveillance study, no serological evidence was found for the presence of MERS-CoV in the camels in the Australian population. This finding presents various hypotheses about the timing of the emergence and spread of MERS-CoV throughout populations of camels in Africa and Asia, which can be partially resolved by testing sera from camels from the original source region, which we have inferred was mainly northwestern Pakistan. In addition, we identify bat species which overlap (or neighbour) the range of the Australian camel population with a higher likelihood of carrying CoVs of the same lineage as MERS-CoV. Both of these proposed follow-on studies are examples of “proactive surveillance”, a concept that has particular relevance to a One Health approach to emerging zoonotic diseases with a complex epidemiology and aetiology.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification and characterization of Coronaviridae genomes from Vietnamese bats and rats based on conserved protein domains</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 15</span></i><p style="text-align: justify; text-justify: inter-word;">The Coronaviridae family of viruses encompasses a group of pathogens with a zoonotic potential as observed from previous outbreaks of the severe acute respiratory syndrome coronavirus and Middle East respiratory syndrome coronavirus. Accordingly, it seems important to identify and document the coronaviruses in animal reservoirs, many of which are uncharacterized and potentially missed by more standard diagnostic assays. A combination of sensitive deep sequencing technology and computational algorithms is essential for virus surveillance, especially for characterizing novel- or distantly related virus strains. Here, we explore the use of profile Hidden Markov Model-defined Pfam protein domains (Pfam domains) encoded by new sequences as a Coronaviridae sequence classification tool. The encoded domains are used first in a triage to identify potential Coronaviridae sequences and then processed using a Random Forest method to classify the sequences to the Coronaviridae genus level. The application of this algorithm on Coronaviridae genomes assembled from agnostic deep sequencing data from surveillance of bats and rats in Dong Thap province (Vietnam) identified thirty-four Alphacoronavirus and eleven Betacoronavirus genomes. This collection of bat and rat coronaviruses genomes provided essential information on the local diversity of coronaviruses and substantially expanded the number of coronavirus full genomes available from bat and rats and may facilitate further molecular studies on this group of viruses.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses in Avian Species – Review with Focus on Epidemiology and Diagnosis in Wild Birds</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 10</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are a large group of enveloped viruses with a single-strand RNA genome, which continuously circulate in mammals and birds and pose a threat to livestock, companion animals, and humans. CoVs harboured by avian species are classified to the genera gamma- and deltacoronaviruses. Within the gamma-CoVs the main representative is avian coronavirus, a taxonomic name which includes the highly contagious infectious bronchitis viruses (IBVs) in chickens and similar viruses infecting other domestic birds such as turkeys, guinea fowls, or quails. Additionally, IBVs have been detected in healthy wild birds, demonstrating that they may act as the vector between domestic and free-living birds. Moreover, CoVs other than IBVs, are identified in wild birds, which suggests that wild birds play a key role in the epidemiology of other gammaCoVs and deltaCoVs. Development of molecular techniques has significantly improved knowledge of the prevalence of CoVs in avian species. The methods adopted in monitoring studies of CoVs in different avian species are mainly based on detection of conservative regions within the viral replicase, nucleocapsid genes, and 3’UTR or 5’UTR. The purpose of this review is to summarise recent discoveries in the areas of epidemiology and diagnosis of CoVs in avian species and to understand the role of wild birds in the virus distribution.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ecological dynamics of emerging bat virus spillover</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viruses that originate in bats may be the most notorious emerging zoonoses that spill over from wildlife into domestic animals and humans. Understanding how these infections filter through ecological systems to cause disease in humans is of profound importance to public health. Transmission of viruses from bats to humans requires a hierarchy of enabling conditions that connect the distribution of reservoir hosts, viral infection within these hosts, and exposure and susceptibility of recipient hosts. For many emerging bat viruses, spillover also requires viral shedding from bats, and survival of the virus in the environment. Focusing on Hendra virus, but also addressing Nipah virus, Ebola virus, Marburg virus and coronaviruses, we delineate this cross-species spillover dynamic from the within-host processes that drive virus excretion to land-use changes that increase interaction among species. We describe how land-use changes may affect co-occurrence and contact between bats and recipient hosts. Two hypotheses may explain temporal and spatial pulses of virus shedding in bat populations: episodic shedding from persistently infected bats or transient epidemics that occur as virus is transmitted among bat populations. Management of livestock also may affect the probability of exposure and disease. Interventions to decrease the probability of virus spillover can be implemented at multiple levels from targeting the reservoir host to managing recipient host exposure and susceptibility.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ghosts of infections past: using archival samples to understand a century of monkeypox virus prevalence among host communities across space and time</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 31</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases that originate from multiple wildlife hosts can be complex and problematic to manage. A full understanding is further limited by large temporal and spatial gaps in sampling. However, these limitations can be overcome, in part, by using historical samples, such as those derived from museum collections. Here, we screened over 1000 museum specimens collected over the past 120 years to examine the historical distribution and prevalence of monkeypox virus (MPXV) in five species of African rope squirrel (Funisciurus sp.) collected across Central Africa. We found evidence of MPXV infections in host species as early as 1899, half a century earlier than the first recognized case of MPXV in 1958, supporting the suggestion that historic pox-like outbreaks in humans and non-human primates may have been caused by MPXV rather than smallpox as originally thought. MPX viral DNA was found in 93 of 1038 (9.0%) specimens from five Funisciurus species (F. anerythrus, F. carruthersi, F. congicus, F. lemniscatus and F. pyrropus), of which F. carruthersi and pyrropus had not previously been identified as potential MPXV hosts. We additionally documented relative prevalence rates of infection in museum specimens of Funisciurus and examined the spatial and temporal distribution of MPXV in these potential host species across nearly a hundred years (1899–1993).</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ghosts of infections past: using archival samples to understand a century of monkeypox virus prevalence among host communities across space and time</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 31</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases that originate from multiple wildlife hosts can be complex and problematic to manage. A full understanding is further limited by large temporal and spatial gaps in sampling. However, these limitations can be overcome, in part, by using historical samples, such as those derived from museum collections. Here, we screened over 1000 museum specimens collected over the past 120 years to examine the historical distribution and prevalence of monkeypox virus (MPXV) in five species of African rope squirrel (Funisciurus sp.) collected across Central Africa. We found evidence of MPXV infections in host species as early as 1899, half a century earlier than the first recognized case of MPXV in 1958, supporting the suggestion that historic pox-like outbreaks in humans and non-human primates may have been caused by MPXV rather than smallpox as originally thought. MPX viral DNA was found in 93 of 1038 (9.0%) specimens from five Funisciurus species (F. anerythrus, F. carruthersi, F. congicus, F. lemniscatus and F. pyrropus), of which F. carruthersi and pyrropus had not previously been identified as potential MPXV hosts. We additionally documented relative prevalence rates of infection in museum specimens of Funisciurus and examined the spatial and temporal distribution of MPXV in these potential host species across nearly a hundred years (1899–1993).</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ghosts of infections past: using archival samples to understand a century of monkeypox virus prevalence among host communities across space and time</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 31</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases that originate from multiple wildlife hosts can be complex and problematic to manage. A full understanding is further limited by large temporal and spatial gaps in sampling. However, these limitations can be overcome, in part, by using historical samples, such as those derived from museum collections. Here, we screened over 1000 museum specimens collected over the past 120 years to examine the historical distribution and prevalence of monkeypox virus (MPXV) in five species of African rope squirrel (Funisciurus sp.) collected across Central Africa. We found evidence of MPXV infections in host species as early as 1899, half a century earlier than the first recognized case of MPXV in 1958, supporting the suggestion that historic pox-like outbreaks in humans and non-human primates may have been caused by MPXV rather than smallpox as originally thought. MPX viral DNA was found in 93 of 1038 (9.0%) specimens from five Funisciurus species (F. anerythrus, F. carruthersi, F. congicus, F. lemniscatus and F. pyrropus), of which F. carruthersi and pyrropus had not previously been identified as potential MPXV hosts. We additionally documented relative prevalence rates of infection in museum specimens of Funisciurus and examined the spatial and temporal distribution of MPXV in these potential host species across nearly a hundred years (1899–1993).</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ghosts of infections past: using archival samples to understand a century of monkeypox virus prevalence among host communities across space and time</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 31</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases that originate from multiple wildlife hosts can be complex and problematic to manage. A full understanding is further limited by large temporal and spatial gaps in sampling. However, these limitations can be overcome, in part, by using historical samples, such as those derived from museum collections. Here, we screened over 1000 museum specimens collected over the past 120 years to examine the historical distribution and prevalence of monkeypox virus (MPXV) in five species of African rope squirrel (Funisciurus sp.) collected across Central Africa. We found evidence of MPXV infections in host species as early as 1899, half a century earlier than the first recognized case of MPXV in 1958, supporting the suggestion that historic pox-like outbreaks in humans and non-human primates may have been caused by MPXV rather than smallpox as originally thought. MPX viral DNA was found in 93 of 1038 (9.0%) specimens from five Funisciurus species (F. anerythrus, F. carruthersi, F. congicus, F. lemniscatus and F. pyrropus), of which F. carruthersi and pyrropus had not previously been identified as potential MPXV hosts. We additionally documented relative prevalence rates of infection in museum specimens of Funisciurus and examined the spatial and temporal distribution of MPXV in these potential host species across nearly a hundred years (1899–1993).</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ghosts of infections past: using archival samples to understand a century of monkeypox virus prevalence among host communities across space and time</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 31</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases that originate from multiple wildlife hosts can be complex and problematic to manage. A full understanding is further limited by large temporal and spatial gaps in sampling. However, these limitations can be overcome, in part, by using historical samples, such as those derived from museum collections. Here, we screened over 1000 museum specimens collected over the past 120 years to examine the historical distribution and prevalence of monkeypox virus (MPXV) in five species of African rope squirrel (Funisciurus sp.) collected across Central Africa. We found evidence of MPXV infections in host species as early as 1899, half a century earlier than the first recognized case of MPXV in 1958, supporting the suggestion that historic pox-like outbreaks in humans and non-human primates may have been caused by MPXV rather than smallpox as originally thought. MPX viral DNA was found in 93 of 1038 (9.0%) specimens from five Funisciurus species (F. anerythrus, F. carruthersi, F. congicus, F. lemniscatus and F. pyrropus), of which F. carruthersi and pyrropus had not previously been identified as potential MPXV hosts. We additionally documented relative prevalence rates of infection in museum specimens of Funisciurus and examined the spatial and temporal distribution of MPXV in these potential host species across nearly a hundred years (1899–1993).</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Ghosts of infections past: using archival samples to understand a century of monkeypox virus prevalence among host communities across space and time</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 31</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases that originate from multiple wildlife hosts can be complex and problematic to manage. A full understanding is further limited by large temporal and spatial gaps in sampling. However, these limitations can be overcome, in part, by using historical samples, such as those derived from museum collections. Here, we screened over 1000 museum specimens collected over the past 120 years to examine the historical distribution and prevalence of monkeypox virus (MPXV) in five species of African rope squirrel (Funisciurus sp.) collected across Central Africa. We found evidence of MPXV infections in host species as early as 1899, half a century earlier than the first recognized case of MPXV in 1958, supporting the suggestion that historic pox-like outbreaks in humans and non-human primates may have been caused by MPXV rather than smallpox as originally thought. MPX viral DNA was found in 93 of 1038 (9.0%) specimens from five Funisciurus species (F. anerythrus, F. carruthersi, F. congicus, F. lemniscatus and F. pyrropus), of which F. carruthersi and pyrropus had not previously been identified as potential MPXV hosts. We additionally documented relative prevalence rates of infection in museum specimens of Funisciurus and examined the spatial and temporal distribution of MPXV in these potential host species across nearly a hundred years (1899–1993).</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Lack of detection of Middle East respiratory syndrome coronavirus in mild and severe respiratory infections in Catalonia, northeastern Spain</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jan 28</span></i><p style="text-align: justify; text-justify: inter-word;">Surveillance of Middle East respiratory syndrome coronavirus (MERS-CoV) was conducted to explore the possible introduction and circulation of this novel virus in Catalonia, northeastern Spain. Five hundred and sixty-three samples from mild and severe respiratory infections collected between January 2012 and April 2013 were screened using real-time RT-PCR. All samples were negative, suggesting that MERS-CoV is not circulating silently in Catalonia.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of new genetic variants of Betacoronaviruses in Endemic Frugivorous Bats of Madagascar</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Mar 12</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Bats are amongst the natural reservoirs of many coronaviruses (CoVs) of which some can lead to severe infection in human. African bats are known to harbor a range of pathogens (e.g., Ebola and Marburg viruses) that can infect humans and cause disease outbreaks. A recent study in South Africa isolated a genetic variant closely related to MERS-CoV from an insectivorous bat. Though Madagascar is home to 44 bat species (41 insectivorous and 3 frugivorous) of which 34 are endemic, no data exists concerning the circulation of CoVs in the island’s chiropteran fauna. Certain Malagasy bats can be frequently found in close contact with humans and frugivorous bats feed in the same trees where people collect and consume fruits and are hunted and consumed as bush meat. The purpose of our study is to detect and identify CoVs from frugivorous bats in Madagascar to evaluate the risk of human infection from infected bats. METHODS: Frugivorous bats belonging to three species were captured in four different regions of Madagascar. We analyzed fecal and throat swabs to detect the presence of virus through amplification of the RNA-dependent RNA polymerase (RdRp) gene, which is highly conserved in all known coronaviruses. Phylogenetic analyses were performed from positive specimens. RESULTS: From 351 frugivorous bats, we detected 14 coronaviruses from two endemic bats species, of which 13 viruses were identified from Pteropus rufus and one from Eidolon dupreanum, giving an overall prevalence of 4.5%. Phylogenetic analysis revealed that the Malagasy strains belong to the genus Betacoronavirus but form three distinct clusters, which seem to represent previously undescribed genetic lineages. CONCLUSIONS: Our findings suggest that CoVs circulate in frugivorous bats of Madagascar, demonstrating the needs to evaluate spillover risk to human populations especially for individuals that hunt and consume infected bats. Possible dispersal mechanisms as to how coronaviruses arrived on Madagascar are discussed.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Alphacoronavirus vRNA in the Feces of Brazilian Free-Tailed Bats (Tadarida brasiliensis) from a Colony in Florida, USA</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Feb 27</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are natural reservoirs of coronaviruses and other viruses with zoonotic potential. Florida has indigenous non-migratory populations of Brazilian free-tailed bats (Tadarida brasiliensis) that mostly roost in colonies in artificial structures. Unlike their counterparts in Brazil and Mexico, the viruses harbored by the Florida bats have been underexplored. We report the detection of an alphacoronavirus RNA-dependent RNA polymerase (RdRp) gene sequence in the feces of two of 19 different T. brasiliensis that were capture/release bats that had been evaluated for overall health. The RdRp sequence is similar but not identical to previously detected sequences in the feces of two different species of bats (T. brasiliensis and Molossus molossus) in Brazil. In common with the experience of others doing similar work, attempts to isolate the virus in cell cultures were unsuccessful. We surmise that this and highly related alphacoronavirus are carried by Brazilian free-tailed bats living in a wide eco-spatial region. As various coronaviruses (CoVs) that affect humans emerged from bats, our study raises the question whether CoVs such as the one detected in our work are yet-to-be-detected pathogens of humans and animals other than bats.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Emerging Infectious Diseases in Free-Ranging Wildlife–Australian Zoo Based Wildlife Hospitals Contribute to National Surveillance</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 May 1</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging infectious diseases are increasingly originating from wildlife. Many of these diseases have significant impacts on human health, domestic animal health, and biodiversity. Surveillance is the key to early detection of emerging diseases. A zoo based wildlife disease surveillance program developed in Australia incorporates disease information from free-ranging wildlife into the existing national wildlife health information system. This program uses a collaborative approach and provides a strong model for a disease surveillance program for free-ranging wildlife that enhances the national capacity for early detection of emerging diseases.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection and Phylogenetic Analysis of Group 1 Coronaviruses in South American Bats</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">Bat coronaviruses (Bt-CoVs) are thought to be the precursors of severe acute respiratory syndrome coronavirus. We detected Bt-CoVs in 2 bat species from Trinidad. Phylogenetic analysis of the RNA-dependent RNA polymerase gene and helicase confirmed them as group 1 coronaviruses.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-associated Coronavirus Transmitted from Human to Pig</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">SEVERE ACUTE RESPIRATORY SYNDROME–ASSOCIATED: coronavirus (SARS-CoV) was isolated from a pig during a survey for possible routes of viral transmission after a SARS epidemic. Sequence and epidemiology analyses suggested that the pig was infected by a SARS-CoV of human origin.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Emerging coronaviruses: Genome structure, replication, and pathogenesis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of a novel coronavirus (2019-nCoV), which is causing an outbreak of unusual viral pneumonia in patients in Wuhan, a central city in China, is another warning of the risk of CoVs posed to public health. In this minireview, we provide a brief introduction of the general features of CoVs and describe diseases caused by different CoVs in humans and animals. This review will help understand the biology and potential risk of CoVs that exist in richness in wildlife such as bats.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Metagenomic Survey for Viruses in Western Arctic Caribou, Alaska, through Iterative Assembly of Taxonomic Units</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 20</span></i><p style="text-align: justify; text-justify: inter-word;">Pathogen surveillance in animals does not provide a sufficient level of vigilance because it is generally confined to surveillance of pathogens with known economic impact in domestic animals and practically nonexistent in wildlife species. As most (re-)emerging viral infections originate from animal sources, it is important to obtain insight into viral pathogens present in the wildlife reservoir from a public health perspective. When monitoring living, free-ranging wildlife for viruses, sample collection can be challenging and availability of nucleic acids isolated from samples is often limited. The development of viral metagenomics platforms allows a more comprehensive inventory of viruses present in wildlife. We report a metagenomic viral survey of the Western Arctic herd of barren ground caribou (Rangifer tarandus granti) in Alaska, USA. The presence of mammalian viruses in eye and nose swabs of 39 free-ranging caribou was investigated by random amplification combined with a metagenomic analysis approach that applied exhaustive iterative assembly of sequencing results to define taxonomic units of each metagenome. Through homology search methods we identified the presence of several mammalian viruses, including different papillomaviruses, a novel parvovirus, polyomavirus, and a virus that potentially represents a member of a novel genus in the family Coronaviridae.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and Phylogeny of Coronaviruses in Wild Birds from the Bering Strait Area (Beringia)</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Oct 29</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) can cause mild to severe disease in humans and animals, their host range and environmental spread seem to have been largely underestimated, and they are currently being investigated for their potential medical relevance. Infectious bronchitis virus (IBV) belongs to gamma-coronaviruses and causes a costly respiratory viral disease in chickens. The role of wild birds in the epidemiology of IBV is poorly understood. In the present study, we examined 1,002 cloacal and faecal samples collected from 26 wild bird species in the Beringia area for the presence of CoVs, and then we performed statistical and phylogenetic analyses. We detected diverse CoVs by RT-PCR in wild birds in the Beringia area. Sequence analysis showed that the detected viruses are gamma-coronaviruses related to IBV. These findings suggest that wild birds are able to carry gamma-coronaviruses asymptomatically. We concluded that CoVs are widespread among wild birds in Beringia, and their geographic spread and frequency is higher than previously realised. Thus, Avian CoV can be efficiently disseminated over large distances and could be a genetic reservoir for future emerging pathogenic CoVs. Considering the great animal health and economic impact of IBV as well as the recent emergence of novel coronaviruses such as SARS-coronavirus, it is important to investigate the role of wildlife reservoirs in CoV infection biology and epidemiology.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Diverse Bat Alphacoronaviruses and Betacoronaviruses in China Provides New Insights Into the Evolution and Origin of Coronavirus-Related Diseases</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 14</span></i><p style="text-align: justify; text-justify: inter-word;">Outbreaks of severe acute respiratory syndrome (SARS) in 2002, Middle East respiratory syndrome in 2012 and fatal swine acute diarrhea syndrome in 2017 caused serious infectious diseases in humans and in livestock, resulting in serious public health threats and huge economic losses. All such coronaviruses (CoVs) were confirmed to originate from bats. To continuously monitor the epidemic-related CoVs in bats, virome analysis was used to classify CoVs from 831 bats of 15 species in Yunnan, Guangxi, and Sichuan Provinces between August 2016 and May 2017. We identified 11 CoV strains from 22 individual samples of four bat species. Identification of four alpha-CoVs from Scotophilus kuhlii in Guangxi, which was closely related to a previously reported bat CoV and porcine epidemic diarrhea virus (PEDV), revealed a bat-swine lineage under the genus Alphacoronavirus. A recombinant CoV showed that the PEDV probably originated from the CoV of S. kuhlii. Another alpha-CoV, α-YN2018, from Rhinolophus affinis in Yunnan, suggested that this alpha-CoV lineage had multiple host origins, and α-YN2018 had recombined with CoVs of other bat species over time. We identified five SARS-related CoVs (SARSr-CoVs) in Rhinolophus bats from Sichuan and Yunnan and confirmed that angiotensin-converting enzyme 2 usable SARSr-CoVs were continuously circulating in Rhinolophus spp. in Yunnan. The other beta-CoV, strain β-GX2018, found in Cynopterus sphinx of Guangxi, represented an independently evolved lineage different from known CoVs of Rousettus and Eonycteris bats. The identification of diverse CoVs here provides new genetic data for understanding the distribution and source of pathogenic CoVs in China.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to COVID-19: A previously unknown SARS-CoV-2 virus of pandemic potential infecting humans – Call for a One Health approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses continue to pose a threat to human health. The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in December 2019 which causes coronavirus disease-2019 (COVID-19), an acute respiratory disease marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. This recent ongoing emergence of a previously unknown coronavirus in China leads to huge impacts on humans globally. Here, we discuss the outbreak in a one health context, highlighting the need for the implementation of one health measures and practices to improve human health and reduce the emergence of pandemic viruses.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Orthoreovirus from Mink, China, 2011</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">We identified a novel mink orthoreovirus, MRV1HB-A, which seems to be closely related to human strain MRV2tou05, which was isolated from 2 children with acute necrotizing encephalopathy in 2005. Evolution of this virus should be closely monitored so that prevention and control measures can be taken should it become more virulent.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Orthoreovirus from Mink, China, 2011</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">We identified a novel mink orthoreovirus, MRV1HB-A, which seems to be closely related to human strain MRV2tou05, which was isolated from 2 children with acute necrotizing encephalopathy in 2005. Evolution of this virus should be closely monitored so that prevention and control measures can be taken should it become more virulent.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus infection in non-camelid domestic mammals</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 16</span></i><p style="text-align: justify; text-justify: inter-word;">Dromedary camels are natural host of the Middle East respiratory syndrome coronavirus (MERS-CoV). However, there are limited studies of MERS-CoV infection of other domestic mammals exposed to infected dromedaries. We expanded our surveillance among camels in Egypt, Tunisia, and Senegal to include other domestic mammalian species in contact with infected camels. A total of 820 sera and 823 nasal swabs from cattle, sheep, goats, donkeys, buffaloes, mules, and horses were collected. Swabs were tested using RT-PCR and virus RNA-positive samples were genetically sequenced and phylogenetically analysed. Sera were screened using virus microneutralization tests and positive sera (where available) were confirmed using plaque reduction neutralization tests (PRNT). We detected 90% PRNT confirmed MERS-CoV antibody in 35 (55.6%) of 63 sera from sheep collected from Senegal, two sheep (1.8%) of 114 in Tunisia and a goat (0.9%) of 107 in Egypt, with titres ranging from 1:80 to ≥1:320. We detected MERS-CoV RNA in swabs from three sheep (1.2%) of 254 and five goats (4.1%) of 121 from Egypt and Senegal, as well as one cow (1.9%) of 53 and three donkeys (7.1%) of 42 from Egypt. Partial sequences of the RT-PCR amplicons confirmed specificity of the results. This study showed that domestic livestock in contact with MERS-CoV infected camels may be at risk of infection. We recommend expanding current MERS-CoV surveillance in animals to include other livestock in close contact with dromedary camels. The segregation of camels from other livestock in farms and live animal markets may need to be considered.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection and Genetic Characterization of Deltacoronavirus in Pigs, Ohio, USA, 2014</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">In Ohio, United States, in early 2014, a deltacoronavirus was detected in feces and intestine samples from pigs with diarrheal disease. The complete genome sequence and phylogenetic analysis of the virus confirmed that the virus is closely related to a porcine deltacoronavirus (porcine coronavirus HKU15) reported in Hong Kong in 2012.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Highly Pathogenic Swine Getah Virus in Blue Foxes, Eastern China, 2017</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We isolated Getah virus from infected foxes in Shandong Province, eastern China. We sequenced the complete Getah virus genome, and phylogenetic analysis revealed a close relationship with a highly pathogenic swine epidemic strain in China. Epidemiologic investigation showed that pigs might play a pivotal role in disease transmission to foxes.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Highly Pathogenic Swine Getah Virus in Blue Foxes, Eastern China, 2017</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We isolated Getah virus from infected foxes in Shandong Province, eastern China. We sequenced the complete Getah virus genome, and phylogenetic analysis revealed a close relationship with a highly pathogenic swine epidemic strain in China. Epidemiologic investigation showed that pigs might play a pivotal role in disease transmission to foxes.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Novel SARS-like and Other Coronaviruses in Bats from Kenya</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">Diverse coronaviruses have been identified in bats from several continents but not from Africa. We identified group 1 and 2 coronaviruses in bats in Kenya, including SARS-related coronaviruses. The sequence diversity suggests that bats are well-established reservoirs for and likely sources of coronaviruses for many species, including humans.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 17</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world’s ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 17</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world’s ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of a Novel Coronavirus in Bats</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Exotic wildlife can act as reservoirs of diseases that are endemic in the area or can be the source of new emerging diseases through interspecies transmission. The recent emergence of severe acute respiratory syndrome-associated coronavirus (SARS-CoV) highlights the importance of virus surveillance in wild animals. Here, we report the identification of a novel bat coronavirus through surveillance of coronaviruses in wildlife. Analyses of the RNA sequence from the ORF1b and S-gene regions indicated that the virus is a group 1 coronavirus. The virus was detected in fecal and respiratory samples from three bat species (Miniopterus spp.). In particular, 63% (12 of 19) of fecal samples from Miniopterus pusillus were positive for the virus. These findings suggest that this virus might be commonly circulating in M. pusillus in Hong Kong.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>High Prevalence and Putative Lineage Maintenance of Avian Coronaviruses in Scandinavian Waterfowl</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are found in a wide variety of wild and domestic animals, and constitute a risk for zoonotic and emerging infectious disease. In poultry, the genetic diversity, evolution, distribution and taxonomy of some coronaviruses have been well described, but little is known about the features of CoVs in wild birds. In this study we screened 764 samples from 22 avian species of the orders Anseriformes and Charadriiformes in Sweden collected in 2006/2007 for CoV, with an overall CoV prevalence of 18.7%, which is higher than many other wild bird surveys. The highest prevalence was found in the diving ducks—mainly Greater Scaup (Aythya marila; 51.5%)—and the dabbling duck Mallard (Anas platyrhynchos; 19.2%). Sequences from two of the Greater Scaup CoV fell into an infrequently detected lineage, shared only with a Tufted Duck (Aythya fuligula) CoV. Coronavirus sequences from Mallards in this study were highly similar to CoV sequences from the sample species and location in 2011, suggesting long-term maintenance in this population. A single Black-headed Gull represented the only positive sample from the order Charadriiformes. Globally, Anas species represent the largest fraction of avian CoV sequences, and there seems to be no host species, geographical or temporal structure. To better understand the eitiology, epidemiology and ecology of these viruses more systematic surveillance of wild birds and subsequent sequencing of detected CoV is imperative.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Extension of the known distribution of a novel clade C betacoronavirus in a wildlife host</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 3</span></i><p style="text-align: justify; text-justify: inter-word;">Disease surveillance in wildlife populations presents a logistical challenge, yet is critical in gaining a deeper understanding of the presence and impact of wildlife pathogens. Erinaceus coronavirus (EriCoV), a clade C Betacoronavirus, was first described in Western European hedgehogs (Erinaceus europaeus) in Germany. Here, our objective was to determine whether EriCoV is present, and if it is associated with disease, in Great Britain (GB). An EriCoV-specific BRYT-Green(®) real-time reverse transcription PCR assay was used to test 351 samples of faeces or distal large intestinal tract contents collected from casualty or dead hedgehogs from a wide area across GB. Viral RNA was detected in 10.8% (38) samples; however, the virus was not detected in any of the 61 samples tested from Scotland. The full genome sequence of the British EriCoV strain was determined using next generation sequencing; it shared 94% identity with a German EriCoV sequence. Multivariate statistical models using hedgehog case history data, faecal specimen descriptions and post-mortem examination findings found no significant associations indicative of disease associated with EriCoV in hedgehogs. These findings indicate that the Western European hedgehog is a reservoir host of EriCoV in the absence of apparent disease.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of potentially novel paramyxovirus and coronavirus viral RNA in bats and rats in the Mekong Delta region of southern Viet Nam</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">Bats and rodents are being increasingly recognized as reservoirs of emerging zoonotic viruses. Various studies have investigated bat viruses in tropical regions, but to date there are no data regarding viruses with zoonotic potential that circulate in bat and rat populations in Viet Nam. To address this paucity of data, we sampled three bat farms and three wet markets trading in rat meat in the Mekong Delta region of southern Viet Nam. Faecal and urine samples were screened for the presence of RNA from paramyxoviruses, coronaviruses and filoviruses. Paramyxovirus RNA was detected in 4 of 248 (1%) and 11 of 222 (4.9%) bat faecal and urine samples, respectively. Coronavirus RNA was detected in 55 of 248 (22%) of bat faecal samples; filovirus RNA was not detected in any of the bat samples. Further, coronavirus RNA was detected in 12 of 270 (4.4%) of rat faecal samples; all samples tested negative for paramyxovirus. Phylogenetic analysis revealed that the bat paramyxoviruses and bat and rat coronaviruses were related to viruses circulating in bat and rodent populations globally, but showed no cross‐species mixing of viruses between bat and rat populations within Viet Nam. Our study shows that potentially novel variants of paramyxoviruses and coronaviruses commonly circulate in bat and rat populations in Viet Nam. Further characterization of the viruses and additional human and animal surveillance is required to evaluate the likelihood of viral spillover and to assess whether these viruses pose a risk to human health.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS: emergence of a novel human coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (CoV) that causes a severe lower respiratory tract infection in humans, emerged in the Middle East region in 2012. This virus, named Middle East respiratory syndrome (MERS)-CoV, is phylogenetically related to bat CoVs, but other animal species like dromedary camels may potentially act as intermediate hosts by spreading the virus to humans. Although human to human transmission has been demonstrated, analysis of human MERS clusters indicated that chains of transmission were not self-sustaining, especially when infection control was implemented. Thus, timely identification of new MERS cases followed by their quarantine, combined with measures to limit spread of the virus from the (intermediate) host to humans, may be crucial in controlling the outbreak of this emerging CoV.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetically Diverse Coronaviruses in Wild Bird Populations of Northern England</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious bronchitis virus (IBV) causes a costly respiratory viral disease of chickens. The role of wild birds in the epidemiology of IBV is poorly understood. We detected diverse coronaviruses by PCR in wildfowl and wading birds in England. Sequence analysis showed some viruses to be related to IBV.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>The emergence of the Middle East Respiratory Syndrome coronavirus (MERS-CoV)</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">On September 20, 2012, a Saudi Arabian physician reported the isolation of a novel coronavirus from a patient with pneumonia on ProMED-mail. Within a few days the same virus was detected in a Qatari patient receiving intensive care in a London hospital, a situation reminiscent of the role air travel played in the spread of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) in 2002. SARS-CoV originated in China’s Guangdong Province and affected more than 8000 patients in 26 countries before it was contained six months later. Over a year after the emergence of this novel coronavirus—Middle East Respiratory Syndrome coronavirus (MERS-CoV)—it has caused 178 laboratory confirmed cases and 76 deaths The emergence of a second highly pathogenic coronavirus within a decade highlights the importance of a coordinated global response incorporating reservoir surveillance, high-containment capacity with fundamental and applied research programs, and dependable communication pathways to ensure outbreak containment. Here we review the current state of knowledge on the epidemiology, ecology, molecular biology, clinical features and intervention strategies of the novel coronavirus, MERS-CoV.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses in animals and humans</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Controlling outbreaks will require detailed knowledge of their biology and behaviourCoronaviruses have been around for many years and were first discovered in the 1960s. They include viruses contributing to the common cold (HCoV-229E) and a variety of animal and avian coronaviruses, such as infectious bronchitis virus (IBV), which infects poultry. Coronaviruses typically cause respiratory or gastrointestinal illness, but strains of IBV have been shown to target the oviduct in chickens, and others can cause severe kidney disease.Animal and avian coronaviruses can have high mortality rates among infected animals and illustrate the difficulties in developing vaccines. Similar to influenza viruses, despite many decades of research there is no vaccine that protects against all strains of IBV coronavirus. This is due in part to the continuously shifting diversity in the virus spike glycoprotein, a major immunogenic target and hence a good vaccine candidate for animal and human infections.During the mid-1990s these viruses were described as the backwater of virology, since none caused serious disease in humans. However, this changed in 2002-03 with …</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection and Prevalence Patterns of Group I Coronaviruses in Bats, Northern Germany</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Apr</span></i><p style="text-align: justify; text-justify: inter-word;">We tested 315 bats from 7 different bat species in northern Germany for coronaviruses by reverse transcription–PCR. The overall prevalence was 9.8%. There were 4 lineages of group I coronaviruses in association with 4 different species of verspertilionid bats (Myotis dasycneme, M. daubentonii, Pipistrellus nathusii, P. pygmaeus). The lineages formed a monophyletic clade of bat coronaviruses found in northern Germany. The clade of bat coronaviruses have a sister relationship with a clade of Chinese type I coronaviruses that were also associated with the Myotis genus (M. ricketti). Young age and ongoing lactation, but not sex or existing gravidity, correlated significantly with coronavirus detection. The virus is probably maintained on the population level by amplification and transmission in maternity colonies, rather than being maintained in individual bats.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Distinct spread of DNA and RNA viruses among mammals amid prominent role of domestic species</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-09-19</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging infectious diseases arising from pathogen spillover from mammals to humans comprise a substantial health threat. Tracing virus origin and predicting the most likely host species for future spillover events are major objectives in One Health disciplines. However, the species that share pathogens most widely with other mammals, and the role of different wildlife groups in sharing viruses with humans remain poorly identified. To address this challenge, we applied network analysis and Bayesian hierarchical models to a global database of mammal-virus associations. We show that domesticated mammals and some primates hold the most central positions in networks of known mammal-virus associations. We revealed strong evidence that DNA viruses were phylogenetically more host specific than RNA viruses, while the frequencies of sharing viruses among hosts and the proportion of zoonotic viruses in hosts were larger for RNA than DNA viruses. Among entire host-virus networks, Carnivora and Chiroptera hold central positions for mainly sharing RNA viruses with other host species, while network centrality of Primates scored relatively high for sharing DNA viruses. Ungulates hold central positions for sharing both RNA and DNA viruses. Acknowledging the role of domestic species in addition to host and virus traits in patterns of virus sharing is necessary to improve our understanding of virus spread and spillover in times of global change.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular characterization of a new species in the genus Alphacoronavirus associated with mink epizootic catarrhal gastroenteritis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A coronavirus (CoV) previously shown to be associated with catarrhal gastroenteritis in mink (Mustela vison) was identified by electron microscopy in mink faeces from two fur farms in Wisconsin and Minnesota in 1998. A pan-coronavirus and a genus-specific RT-PCR assay were used initially to demonstrate that the newly discovered mink CoVs (MCoVs) were members of the genus Alphacoronavirus. Subsequently, using a random RT-PCR approach, full-genomic sequences were generated that further confirmed that, phylogenetically, the MCoVs belonged to the genus Alphacoronavirus, with closest relatedness to the recently identified but only partially sequenced (fragments of the polymerase, and full-length spike, 3c, envelope, nucleoprotein, membrane, 3x and 7b genes) ferret enteric coronavirus (FRECV) and ferret systemic coronavirus (FRSCV). The molecular data presented in this study provide the first genetic evidence for a new coronavirus associated with epizootic catarrhal gastroenteritis outbreaks in mink and demonstrate that MCoVs possess high genomic variability and relatively low overall nucleotide sequence identities (91.7 %) between contemporary strains. Additionally, the new MCoVs appeared to be phylogenetically distant from human (229E and NL63) and other alphacoronaviruses and did not belong to the species Alphacoronavirus 1. It is proposed that, together with the partially sequenced FRECV and FRSCV, they comprise a new species within the genus Alphacoronavirus.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus and paramyxovirus in bats from Northwest Italy</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Bat-borne virus surveillance is necessary for determining inter-species transmission risks and is important due to the wide-range of bat species which may harbour potential pathogens. This study aimed to monitor coronaviruses (CoVs) and paramyxoviruses (PMVs) in bats roosting in northwest Italian regions. Our investigation was focused on CoVs and PMVs due to their proven ability to switch host and their zoonotic potential. Here we provide the phylogenetic characterization of the highly conserved polymerase gene fragments. RESULTS: Family-wide PCR screenings were used to test 302 bats belonging to 19 different bat species. Thirty-eight animals from 12 locations were confirmed as PCR positive, with an overall detection rate of 12.6% [95% CI: 9.3–16.8]. CoV RNA was found in 36 bats belonging to eight species, while PMV RNA in three Pipistrellus spp. Phylogenetic characterization have been obtained for 15 alpha- CoVs, 5 beta-CoVs and three PMVs; moreover one P. pipistrellus resulted co-infected with both CoV and PMV. A divergent alpha-CoV clade from Myotis nattereri SpA is also described. The compact cluster of beta-CoVs from R. ferrumequinum roosts expands the current viral sequence database, specifically for this species in Europe. To our knowledge this is the first report of CoVs in Plecotus auritus and M. oxygnathus, and of PMVs in P. kuhlii. CONCLUSIONS: This study identified alpha and beta-CoVs in new bat species and in previously unsurveyed Italian regions. To our knowledge this represents the first and unique report of PMVs in Italy. The 23 new bat genetic sequences presented will expand the current molecular bat-borne virus databases. Considering the amount of novel bat-borne PMVs associated with the emergence of zoonotic infections in animals and humans in the last years, the definition of viral diversity within European bat species is needed. Performing surveillance studies within a specific geographic area can provide awareness of viral burden where bats roost in close proximity to spillover hosts, and form the basis for the appropriate control measures against potential threats for public health and optimal management of bats and their habitats.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Bat Astroviruses: Towards Understanding the Transmission Dynamics of a Neglected Virus Family</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Feb 21</span></i><p style="text-align: justify; text-justify: inter-word;">Bats belong to the order Chiroptera that represents the second largest order of mammals with more than 1200 species and an almost global distribution. Environmental changes and deforestation have severely influenced many ecosystems, intensifying the contact between wildlife and humans. In recent years, bats have been found to harbor a number of different viruses with zoonotic potential, as well as a great diversity of astroviruses, for which the question of zoonotic potential remains unanswered to date. Human astroviruses have been identified as the causative agent for diarrhea in children and immunocompromised patients. For a long time, astroviruses have been considered to be strictly species-specific. However, a great genetic diversity has recently been discovered among animal and human astroviruses that might indicate the potential of these viruses to cross species barriers. Furthermore, our knowledge about the tissue tropism of astroviruses has been expanded to some neurotropic strains that have recently been shown to be responsible for encephalitis in humans and livestock. This review gives an overview on what is known about astroviruses in bats, humans and livestock, especially bovines and pigs. Future research activities are suggested to unravel astrovirus infection dynamics in bat populations to further assess the zoonotic potential of these viruses.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>First Complete Genome Sequence of a Feline Alphacoronavirus 1 Strain from Brazil</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 7</span></i><p style="text-align: justify; text-justify: inter-word;">We identified a strain of Alphacoronavirus 1, FCoV-SB22, from a pool of fecal samples from domestic cats from a rural settlement in the municipality of Santa Bárbara, Pará, Brazil. The nucleotide identity with feline coronavirus was 91.5%. The present study reports the first complete genome sequence of a feline coronavirus from Brazil.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>A48 Identification and full-genome characterization of Alpha- and Beta-Coronaviruses viruses from bats in Italy</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 22</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are the natural reservoir of Coronaviruses (CoVs). Human CoVs cause mild respiratory diseases worldwide, but, in the last decade, two Beta-CoVs [Middle East respiratory syndrome (MERS)-CoV and severe acute respiratory syndrome] caused thousands of deaths and cases worldwide. Phylogenetic analysis suggested the evolutionary origin of mammalian CoVs is derived from bats. In this study, we characterized three Alpha-CoVs and two Beta-CoVs demonstrating the circulation of bat strains in Italy. Isolates were sequenced using a next-generation sequencing approach and genomes reconstructed using the online tool Galaxy Aries. Phylogenetic analyses were conducted using MEGA7 and MrBayes. Similarity plots were generated using SSE v1.2. The structure of the receptor binding domain (RBD) in the S protein was predicted by sequence-homology method using the protein data bank. Bioinformatics analysis permitted the identification of 2 Beta-CoV complete genomes of 30 kb and three Alpha-CoV of 28 kb (named BatCoV-ITA1-5). BatCoV-ITA1 and 2 formed a monophyletic group with MERS-CoV sequences. The comparison of the concatenated domains within ORF1ab confirmed their classification into the MERS-CoV species. The 3D structure of RBD of Italian strains showed two amino acid deletions located in a region corresponding to the external subdomain of MERS-RBD. BatCoV-Ita3 and BatCoV-Ita4/5 were classified into two novel Alpha-CoV species by comparison of concatenated domains within ORF1ab. Due to the high divergence with the Alpha human spike protein strains, it was impossible to establish the protein structure and the potential affinity to human receptor. The Italian strains showed the typical organization of Alpha and Beta-CoVs. We reported two Beta-CoVs closely related to MERS-CoVs from bats belonging to common Italian species (Pipistrellus kuhlii and Hypsugo savii). The analysis of the RBD in the spike protein indicates significant differences from human RBD known to date. The three Alpha-CoV strains were classified into two novel species, confirming the high heterogeneity of CoV strains in bats. Although the studies conducted cannot confirm a risk for humans, surveillance studies are needed to investigate the genetic diversity of CoVs in bats. Because this exceeds what is known for other hosts, it is compatible with bats being the major reservoir of mammalian CoVs.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Bats have been identified as the natural reservoir of severe acute respiratory syndrome (SARS)-like and SARS coronaviruses (SLCoV and SCoV). However, previous studies suggested that none of the currently sampled bat SLCoVs is the descendant of the direct ancestor of SCoV, based on their relatively distant phylogenetic relationship. In this study, evidence of the recombinant origin of the genome of a bat SLCoV is demonstrated. We identified a potential recombination breakpoint immediately after the consensus intergenic sequence between open reading frame 1 and the S coding region, suggesting the replication intermediates may participate in the recombination event, as previously speculated for other CoVs. Phylogenetic analysis of its parental regions suggests the presence of an uncharacterized SLCoV lineage that is phylogenetically closer to SCoVs than any of the currently sampled bat SLCoVs. Using various Bayesian molecular-clock models, interspecies transfer of this SLCoV lineage from bats to the amplifying host (e.g., civets) was estimated to have happened a median of 4.08 years before the SARS outbreak. Based on this relatively short window period, we speculate that this uncharacterized SLCoV lineage may contain the direct ancestor of SCoV. This study sheds light on the possible host bat species of the direct ancestor of SCoV, providing valuable information on the scope and focus of surveillance for the origin of SCoV.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Pangolin homology associated with 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;">To explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of Bat Coronaviruses through Surveillance and Probe Capture-Based Next-Generation Sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 29</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) of bat origin have caused two pandemics in this century. Severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV both originated from bats, and it is highly likely that bat coronaviruses will cause future outbreaks. Active surveillance is both urgent and essential to predict and mitigate the emergence of these viruses in humans. Next-generation sequencing (NGS) is currently the preferred methodology for virus discovery to ensure unbiased sequencing of bat CoVs, considering their high genetic diversity. However, unbiased NGS is an expensive methodology and is prone to missing low-abundance CoV sequences due to the high background level of nonviral sequences present in surveillance field samples. Here, we employ a capture-based NGS approach using baits targeting most of the CoV species. Using this technology, we effectively reduced sequencing costs by increasing the sensitivity of detection. We discovered nine full genomes of bat CoVs in this study and revealed great genetic diversity for eight of them. IMPORTANCE Active surveillance is both urgent and essential to predict and mitigate the emergence of bat-origin CoV in humans and livestock. However, great genetic diversity increases the chance of homologous recombination among CoVs. Performing targeted PCR, a common practice for many surveillance studies, would not reflect this diversity. NGS, on the other hand, is an expensive methodology and is prone to missing low-abundance CoV sequences. Here, we employ a capture-based NGS approach using baits targeting all CoVs. Our work demonstrates that targeted, cost-effective, large-scale, genome-level surveillance of bat CoVs is now highly feasible.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation of coronaviruses antigenically indistinguishable from bovine coronavirus from wild ruminants with diarrhea.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Diarrheal feces from three sambar deer and one waterbuck in a wild animal habitat and one white-tailed deer on a wildlife farm in Ohio contained coronavirus particles which were agglutinated by antiserum to bovine coronavirus (BCV) in immune electron microscopy. Three coronavirus strains were isolated in human rectal tumor cells from the feces of the sambar and white-tailed deer and the waterbuck, respectively. Hemagglutination, receptor-destroying enzyme activity, indirect immunofluorescence, hemagglutination inhibition, virus neutralization, and Western blot (immunoblot) tests showed close biological and antigenic relationships among the isolates and with selected BCV strains. Gnotobiotic and colostrum-deprived calves inoculated with each of these isolates developed diarrhea and shed coronavirus in their feces and from their nasal passages. In a serological survey of coronavirus infections among wild deer, 8.7 and 6.6% of sera from mule deer in Wyoming and from white-tailed deer in Ohio, respectively, were seropositive against both of the isolates and selected BCV isolates by indirect immunofluorescence tests. These results confirm the existence of coronaviruses in wild ruminants and suggest that these species may harbor coronavirus strains transmissible to cattle.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Origin and evolution of pathogenic coronaviruses</h3>
<i class="tooltip">2019<span class="tooltiptext">2019</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) are two highly transmissible and pathogenic viruses that emerged in humans at the beginning of the 21st century. Both viruses likely originated in bats, and genetically diverse coronaviruses that are related to SARS-CoV and MERS-CoV were discovered in bats worldwide. In this Review, we summarize the current knowledge on the origin and evolution of these two pathogenic coronaviruses and discuss their receptor usage; we also highlight the diversity and potential of spillover of bat-borne coronaviruses, as evidenced by the recent spillover of swine acute diarrhoea syndrome coronavirus (SADS-CoV) to pigs.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of a Novel Betacoronavirus Related to Middle East Respiratory Syndrome Coronavirus in European Hedgehogs</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are known to host viruses closely related to important human coronaviruses (HCoVs), such as HCoV-229E, severe-acute respiratory syndrome coronavirus (SARS-CoV), and Middle East respiratory syndrome CoV (MERS-CoV). As RNA viruses may coevolve with their hosts, we sought to investigate the closest sister taxon to bats, the Eulipotyphla, and screened European hedgehogs (Erinaceus europaeus) from Germany for CoV by nested reverse transcriptase PCR. A novel betacoronavirus species in a phylogenetic sister relationship to MERS-CoV and clade c bat CoVs was detected and characterized on the whole-genome level. A total of 58.9% of hedgehog fecal specimens were positive for the novel CoV (EriCoV) at 7.9 log(10) mean RNA copies per ml. EriCoV RNA concentrations were higher in the intestine than in other solid organs, blood, or urine. Detailed analyses of the full hedgehog intestine showed the highest EriCoV concentrations in lower gastrointestinal tract specimens, compatible with viral replication in the lower intestine and fecal-oral transmission. Thirteen of 27 (48.2%) hedgehog sera contained non-neutralizing antibodies against MERS-CoV. The animal origins of this betacoronavirus clade that includes MERS-CoV may thus include both bat and nonbat hosts.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection and Characterization of Distinct Alphacoronaviruses in Five Different Bat Species in Denmark</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Sep 11</span></i><p style="text-align: justify; text-justify: inter-word;">Bat populations harbour a multitude of viruses; some of these are pathogenic or potentially pathogenic in other animals or humans. Therefore, it is important to monitor the populations and characterize these viruses. In this study, the presence of coronaviruses (CoVs) in different species of Danish bats was investigated using active surveillance at different geographical locations in Denmark. Faecal samples were screened for the presence of CoVs using pan-CoV real-time RT-PCR assays. The amplicons, obtained from five different species of bats, were sequenced. Phylogenetic analysis revealed a species-specific clustering with the samples from Myotis daubentonii, showing a close resemblance to coronavirus sequences obtained from the same species of bat in Germany and the United Kingdom. Our results show, for the first time, that multiple, distinct alphacoronaviruses are present in the Danish bat populations.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>A new coronavirus associated with human respiratory disease in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging infectious diseases, such as SARS and Zika, present a major threat to public health(1-3). Despite intense research efforts, how, when and where new diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city Wuhan, Hubei province, China. Up to 25th of January 2020, at least 1,975 cases have been reported since the first patient was hospitalized on the 12th of December 2019. Epidemiological investigation suggested that the outbreak was associated with a seafood market in Wuhan. We studied one patient who was a worker at the market, and who was admitted to Wuhan Central Hospital on 26th of December 2019 experiencing a severe respiratory syndrome including fever, dizziness and cough. Metagenomic RNA sequencing(4) of a bronchoalveolar lavage fluid sample identified a novel RNA virus from the family Coronaviridae, designed here as WH-Human-1 coronavirus. Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequence of a Severe Acute Respiratory Syndrome-Related Coronavirus from Kenyan Bats</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jul 11</span></i><p style="text-align: justify; text-justify: inter-word;">We identified a strain of betacoronavirus BtKY72/Rhinolophus sp./Kenya/2007 (here BtKY72) from rectal swab samples in Kenyan bats. This paper reports the complete genomic sequence of BtKY72, which is closely related to BtCoV/BM48-31/Bulgaria/2008, a severe acute respiratory syndrome (SARS)-related virus from Rhinolophus bats in Europe.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-02</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1-3. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).  Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Dromedary camels and the transmission of Middle East Respiratory Syndrome Coronavirus (MERS-CoV)</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an existential threat to global public health. The virus has been repeatedly detected in dromedary camels (Camelus dromedarius). Adult animals in many countries in the Middle East as well as in North and East Africa showed high (>90%) sero-prevalence to the virus. MERS-CoV isolated from dromedaries is genetically and phenotypically similar to viruses from humans. We summarise current understanding of the ecology of MERS-CoV in animals and transmission at the animal-human interface. We review aspects of husbandry, animal movements and trade and the use and consumption of camel dairy and meat products in the Middle East that may be relevant to the epidemiology of MERS. We also highlight the gaps in understanding the transmission of this virus in animals and from animals to humans.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Influenza as a model system for studying the cross-species transfer and evolution of the SARS coronavirus.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus (SARS-CoV) moved into humans from a reservoir species and subsequently caused an epidemic in its new host. We know little about the processes that allowed the cross-species transfer of this previously unknown virus. I discuss what we have learned about the movement of viruses into humans from studies of influenza A, both how it crossed from birds to humans and how it subsequently evolved within the human population. Starting with a brief review of severe acute respiratory syndrome to highlight the kinds of problems we face in learning about this viral disease, I then turn to influenza A, focusing on three topics. First, I present a reanalysis of data used to test the hypothesis that swine served as a "mixing vessel" or intermediate host in the transmission of avian influenza to humans during the 1918 "Spanish flu" pandemic. Second, I review studies of archived viruses from the three recent influenza pandemics. Third, I discuss current limitations in using molecular data to study the evolution of infectious disease. Although influenza A and SARS-CoV differ in many ways, our knowledge of influenza A may provide important clues about what limits or favours cross-species transfers and subsequent epidemics of newly emerging pathogens.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of Seven Novel Mammalian and Avian Coronaviruses in the Genus Deltacoronavirus Supports Bat Coronaviruses as the Gene Source of Alphacoronavirus and Betacoronavirus and Avian Coronaviruses as the Gene Source of Gammacoronavirus and Deltacoronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Recently, we reported the discovery of three novel coronaviruses, bulbul coronavirus HKU11, thrush coronavirus HKU12, and munia coronavirus HKU13, which were identified as representatives of a novel genus, Deltacoronavirus, in the subfamily Coronavirinae. In this territory-wide molecular epidemiology study involving 3,137 mammals and 3,298 birds, we discovered seven additional novel deltacoronaviruses in pigs and birds, which we named porcine coronavirus HKU15, white-eye coronavirus HKU16, sparrow coronavirus HKU17, magpie robin coronavirus HKU18, night heron coronavirus HKU19, wigeon coronavirus HKU20, and common moorhen coronavirus HKU21. Complete genome sequencing and comparative genome analysis showed that the avian and mammalian deltacoronaviruses have similar genome characteristics and structures. They all have relatively small genomes (25.421 to 26.674 kb), the smallest among all coronaviruses. They all have a single papain-like protease domain in the nsp3 gene; an accessory gene, NS6 open reading frame (ORF), located between the M and N genes; and a variable number of accessory genes (up to four) downstream of the N gene. Moreover, they all have the same putative transcription regulatory sequence of ACACCA. Molecular clock analysis showed that the most recent common ancestor of all coronaviruses was estimated at approximately 8100 BC, and those of Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and Deltacoronavirus were at approximately 2400 BC, 3300 BC, 2800 BC, and 3000 BC, respectively. From our studies, it appears that bats and birds, the warm blooded flying vertebrates, are ideal hosts for the coronavirus gene source, bats for Alphacoronavirus and Betacoronavirus and birds for Gammacoronavirus and Deltacoronavirus, to fuel coronavirus evolution and dissemination.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Longitudinal study of age-specific pattern of coronavirus infection in Lyle’s flying fox (Pteropus lylei) in Thailand</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Bats are natural reservoirs for several highly pathogenic and novel viruses including coronaviruses (CoVs) (mainly Alphacoronavirus and Betacoronavirus). Lyle’s flying fox (Pteropus lylei)‘s roosts and foraging sites are usually in the proximity to humans and animals. Knowledge about age-specific pattern of CoV infection in P. lylei, prevalence, and viral shedding at roosts and foraging sites may have an impact on infection-age-structure model to control CoV outbreak. METHODS: P. lylei bats were captured monthly during January–December 2012 for detection of CoV at three areas in Chonburi province; two human dwellings, S1 and S2, where few fruit trees were located with an open pig farm, 0.6 km and 5.5 km away from the bat roost, S3. Nested RT-PCR of RNA-dependent RNA polymerase (RdRp) gene from rectal swabs was used for CoV detection. The strain of CoV was confirmed by sequencing and phylogenetic analysis. RESULTS: CoV infection was found in both juveniles and adult bats between May and October (January, in adults only and April, in juveniles only). Of total rectal swab positives (68/367, 18.5%), ratio was higher in bats captured at S1 (11/44, 25.0%) and S2 (35/99, 35.4%) foraging sites than at roost (S3) (22/224, 9.8%). Juveniles (forearm length ≤ 136 mm) were found with more CoV infection than adults at all three sites; S1 (9/24, 37.5% vs 2/20, 10%), S2 (22/49, 44.9% vs 13/50, 26.0%), and S3 (10/30, 33.3% vs 12/194, 6.2%). The average BCI of CoV infected bats was significantly lower than uninfected bats. No gender difference related to infection was found at the sites. Phylogenetic analysis of conserved RdRp gene revealed that the detected CoVs belonged to group D betacoronavirus (n = 64) and alphacoronavirus (n = 4). CONCLUSIONS: The fact that CoV infection and shedding was found in more juvenile than adult bats may suggest transmission from mother during peripartum period. Whether viral reactivation during parturition period or stress is responsible in maintaining transmission in the bat colony needs to be explored.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>A New Bat-HKU2–like Coronavirus in Swine, China, 2017</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">We identified from suckling piglets with diarrhea in China a new bat-HKU2–like porcine coronavirus (porcine enteric alphacoronavirus). The GDS04 strain of this coronavirus shares high aa identities (>90%) with the reported bat-HKU2 strains in Coronaviridae-wide conserved domains, suggesting that the GDS04 strain belongs to the same species as HKU2.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification and Characterization of a Novel Alpaca Respiratory Coronavirus Most Closely Related to the Human Coronavirus 229E</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Dec 12</span></i><p style="text-align: justify; text-justify: inter-word;">In 2007, a novel coronavirus associated with an acute respiratory disease in alpacas (Alpaca Coronavirus, ACoV) was isolated. Full-length genomic sequencing of the ACoV demonstrated the genome to be consistent with other Alphacoronaviruses. A putative additional open-reading frame was identified between the nucleocapsid gene and 3'UTR. The ACoV was genetically most similar to the common human coronavirus (HCoV) 229E with 92.2% nucleotide identity over the entire genome. A comparison of spike gene sequences from ACoV and from HCoV-229E isolates recovered over a span of five decades showed the ACoV to be most similar to viruses isolated in the 1960’s to early 1980’s. The true origin of the ACoV is unknown, however a common ancestor between the ACoV and HCoV-229E appears to have existed prior to the 1960’s, suggesting virus transmission, either as a zoonosis or anthroponosis, has occurred between alpacas and humans.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Biologic, Antigenic, and Full-Length Genomic Characterization of a Bovine-Like Coronavirus Isolated from a Giraffe</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) possess large RNA genomes and exist as quasispecies, which increases the possibility of adaptive mutations and interspecies transmission. Recently, CoVs were recognized as important pathogens in captive wild ruminants. This is the first report of the isolation and detailed genetic, biologic, and antigenic characterization of a bovine-like CoV from a giraffe (Giraffa camelopardalis) in a wild-animal park in the United States. CoV particles were detected by immune electron microscopy in fecal samples from three giraffes with mild-to-severe diarrhea. From one of the three giraffe samples, a CoV (GiCoV-OH3) was isolated and successfully adapted to serial passage in human rectal tumor 18 cell cultures. Hemagglutination assays, receptor-destroying enzyme activity, hemagglutination inhibition, and fluorescence focus neutralization tests revealed close biological and antigenic relationships between the GiCoV-OH3 isolate and selected respiratory and enteric bovine CoV (BCoV) strains. When orally inoculated into a BCoV-seronegative gnotobiotic calf, GiCoV-OH3 caused severe diarrhea and virus shedding within 2 to 3 days. Sequence comparisons and phylogenetic analyses were performed to assess its genetic relatedness to other CoVs. Molecular characterization confirmed that the new isolate belongs to group 2a of the mammalian CoVs and revealed closer genetic relatedness between GiCoV-OH3 and the enteric BCoVs BCoV-ENT and BCoV-DB2, whereas BCoV-Mebus was more distantly related. Detailed sequence analysis of the GiCoV-OH3 spike gene demonstrated the presence of a deletion in the variable region of the S1 subunit (from amino acid 543 to amino acid 547), which is a region associated with pathogenicity and tissue tropism for other CoVs. The point mutations identified in the structural proteins (by comparing GiCoV-OH3, BCoV-ENT, BCoV-DB2, and BCoV-Mebus) were most conserved among GiCoV-OH3, BCoV-ENT, and BCoV-DB2, whereas most of the point mutations in the nonstructural proteins were unique to GiCoV-OH3. Our results confirm the existence of a bovine-like CoV transmissible to cattle from wild ruminants, namely, giraffes, but with certain genetic properties different from those of BCoVs.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of a coronavirus from elk calves with diarrhea.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">This is the first report of the isolation of a coronavirus from elk calves. Two fecal samples from elk calves with diarrhea were shown to be positive for coronavirus-like particles by electron microscopy, and the particles were propagated in the human rectal tumor-18 cell line. After 24 h, syncytia were observed, and cell culture supernatants from both samples showed hemagglutinating activity with mouse erythrocytes. Cells infected with both elk coronavirus (ECV) isolates reacted with Z3A5, a monoclonal antibody against the spike protein of bovine coronavirus (BCV), on an indirect fluorescent antibody test. The protein profiles of both ECV isolates were similar to that of BCV as determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis analysis. On Northern blot analysis, the transcriptional pattern of ECV was typical of coronaviruses, with a nested set of transcripts with common 3' end sequences. Based on a published nucleoprotein gene sequence for BCV (Mebus isolate), we arbitrarily designed two primers for amplification by PCR. After cloning, the nucleoprotein was sequenced and a high degree of homology (99%) between the nucleoprotein gene sequences of ECV and BCV was observed. Thus, ECV is closely related genetically and antigenically to BCV and will be a new member of antigenic group 2 of the mammalian coronaviruses, which possess hemagglutinin-esterase protein.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Production, Characterization, and Uses of  Monoclonal Antibodies against Recombinant Nucleoprotein of  Elk Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">This is the first report of the production of monoclonal antibodies against elk coronavirus. The nucleoprotein gene of elk coronavirus was amplified by PCR and was cloned and expressed in a prokaryotic expression vector. Recombinant nucleocapsid protein was used to immunize mice for the production of hybridomas. Twelve hybridomas that produced monoclonal antibodies against the nucleocapsid protein of elk coronavirus were selected by an indirect fluorescent-antibody test, an enzyme-linked immunosorbent assay, and a Western blot assay. Ten of the monoclonal antibodies were of the immunoglobulin G1 (IgG1) isotype, one was IgG2a, and one was IgM. All had kappa light chains. By immunohistochemistry four monoclonal antibodies detected bovine coronavirus and elk coronavirus in formalin-fixed intestinal tissues. Antinucleoprotein monoclonal antibodies were found to be better at ruminant coronavirus detection than the anti-spike protein monoclonal antibodies. Because nucleoprotein is a more abundant antigen than spike protein in infected cells, this was not an unexpected finding.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Infection in Dromedary Camels in Saudi Arabia</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb 25</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome (MERS) is proposed to be a zoonotic disease; however, the reservoir and mechanism for transmission of the causative agent, the MERS coronavirus, are unknown. Dromedary camels have been implicated through reports that some victims have been exposed to camels, camels in areas where the disease has emerged have antibodies to the virus, and viral sequences have been recovered from camels in association with outbreaks of the disease among humans. Nonetheless, whether camels mediate transmission to humans is unresolved. Here we provide evidence from a geographic and temporal survey of camels in the Kingdom of Saudi Arabia that MERS coronaviruses have been circulating in camels since at least 1992, are distributed countrywide, and can be phylogenetically classified into clades that correlate with outbreaks of the disease among humans. We found no evidence of infection in domestic sheep or domestic goats.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Global patterns in coronavirus diversity</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun 12</span></i><p style="text-align: justify; text-justify: inter-word;">Since the emergence of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrom Coronavirus (MERS-CoV) it has become increasingly clear that bats are important reservoirs of CoVs. Despite this, only 6% of all CoV sequences in GenBank are from bats. The remaining 94% largely consist of known pathogens of public health or agricultural significance, indicating that current research effort is heavily biased towards describing known diseases rather than the ‘pre-emergent’ diversity in bats. Our study addresses this critical gap, and focuses on resource poor countries where the risk of zoonotic emergence is believed to be highest. We surveyed the diversity of CoVs in multiple host taxa from twenty countries to explore the factors driving viral diversity at a global scale. We identified sequences representing 100 discrete phylogenetic clusters, ninety-one of which were found in bats, and used ecological and epidemiologic analyses to show that patterns of CoV diversity correlate with those of bat diversity. This cements bats as the major evolutionary reservoirs and ecological drivers of CoV diversity. Co-phylogenetic reconciliation analysis was also used to show that host switching has contributed to CoV evolution, and a preliminary analysis suggests that regional variation exists in the dynamics of this process. Overall our study represents a model for exploring global viral diversity and advances our fundamental understanding of CoV biodiversity and the potential risk factors associated with zoonotic emergence.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Global trends in infectious diseases at the wildlife–livestock interface</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The role and significance of wildlife–livestock interfaces in disease ecology has largely been neglected, despite recent interest in animals as origins of emerging diseases in humans. Scoping review methods were applied to objectively assess the relative interest by the scientific community in infectious diseases at interfaces between wildlife and livestock, to characterize animal species and regions involved, as well as to identify trends over time. An extensive literature search combining wildlife, livestock, disease, and geographical search terms yielded 78,861 publications, of which 15,998 were included in the analysis. Publications dated from 1912 to 2013 and showed a continuous increasing trend, including a shift from parasitic to viral diseases over time. In particular there was a significant increase in publications on the artiodactyls–cattle and bird–poultry interface after 2002 and 2003, respectively. These trends could be traced to key disease events that stimulated public interest and research funding. Among the top 10 diseases identified by this review, the majority were zoonoses. Prominent wildlife–livestock interfaces resulted largely from interaction between phylogenetically closely related and/or sympatric species. The bird–poultry interface was the most frequently cited wildlife–livestock interface worldwide with other interfaces reflecting regional circumstances. This review provides the most comprehensive overview of research on infectious diseases at the wildlife–livestock interface to date.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV in Camels but Not Camel Handlers, Sudan, 2015 and 2017</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">We tested samples collected from camels, camel workers, and other animals in Sudan and Qatar in 2015 and 2017 for evidence of Middle East respiratory syndrome coronavirus (MERS-CoV) infection. MERS-CoV antibodies were abundant in Sudan camels, but we found no evidence of MERS-CoV infection in camel workers, other livestock, or bats.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Replicative Capacity of MERS Coronavirus in Livestock Cell Lines</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Replicative capacity of Middle East respiratory syndrome coronavirus (MERS-CoV) was assessed in cell lines derived from livestock and peridomestic small mammals on the Arabian Peninsula. Only cell lines originating from goats and camels showed efficient replication of MERS-CoV. These results provide direction in the search for the intermediate host of MERS-CoV.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /></body>