
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: experimental infections to test host range for COVID-19 pathogen</h2><br /></div><br /><br /><br /><div class="card"><h3>Changes in temperature alter susceptibility to a virus following a host shift</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-06-28</span></i><p style="text-align: justify; text-justify: inter-word;">Host shifts - where a pathogen jumps between different host species - are an important source of emerging infectious disease. With ongoing climate change there is an increasing need to understand the effect changes in temperature may have on emerging infectious disease. We investigated whether species susceptibilities change with temperature and ask if susceptibility is greatest at different temperatures in different species. We infected 45 species of Drosophilidae with an RNA virus and measured how viral load changes with temperature. We found the host phylogeny explained a large proportion of the variation in viral load at each temperature, with strong phylogenetic correlations between viral loads across temperature. The variance in viral load increased with temperature, whilst the mean viral load did not, such that as temperature increased the most susceptible species become more susceptible, and the least susceptible less so. We found no significant relationship between a species susceptibility across temperatures and proxies for thermal optima; critical thermal maximum and minimum or basal metabolic rate. These results suggest that whilst the rank order of species susceptibilities can remain the same with changes in temperature, the likelihood of host shifts into a given species may increase or decrease.\n\nAuthor SummaryEmerging infectious diseases are often the result of a host shift, where a pathogen jumps from one host species into another. Understanding the factors underlying host shifts is a major goal for infectious disease researchers. This effort has been further complicated by the fact that host-parasite interactions are now taking place in a period of unprecedented global climatic warming. Here, we ask how host shifts are affected by temperature by carrying out experimental infections using an RNA virus across a wide range of related species, at three different temperatures. We find that as temperature increases the most susceptible species become more susceptible, and the least susceptible less so. This has important consequences for our understanding of host shift events in a changing climate, and suggests that temperature changes may affect the likelihood of a host shift into certain species.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Bacterial Pathogen Emergence Requires More than Direct Contact with a Novel Passerine Host</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">While direct contact may sometimes be sufficient to allow a pathogen to jump into a new host species, in other cases, fortuitously adaptive mutations that arise in the original donor host are also necessary. Viruses have been the focus of most host shift studies, so less is known about the importance of ecological versus evolutionary processes to successful bacterial host shifts. Here we tested whether direct contact with the novel host was sufficient to enable the mid-1990s jump of the bacterium Mycoplasma gallisepticum from domestic poultry to house finches (Haemorhous mexicanus). We experimentally inoculated house finches with two genetically distinct M. gallisepticum strains obtained either from poultry (Rlow) or from house finches (HF1995) during an epizootic outbreak. All 15 house finches inoculated with HF1995 became infected, whereas Rlow successfully infected 12 of 15 (80%) inoculated house finches. Comparisons among infected birds showed that, relative to HF1995, Rlow achieved substantially lower bacterial loads in the host respiratory mucosa and was cleared faster. Furthermore, Rlow-infected finches were less likely to develop clinical symptoms than HF1995-infected birds and, when they did, displayed milder conjunctivitis. The lower infection success of Rlow relative to HF1995 was not, however, due to a heightened host antibody response to Rlow. Taken together, our results indicate that contact between infected poultry and house finches was not, by itself, sufficient to explain the jump of M. gallisepticum to house finches. Instead, mutations arising in the original poultry host would have been necessary for successful pathogen emergence in the novel finch host.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Susceptibility of Pigs and Chickens to SARS Coronavirus</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of severe acute respiratory syndrome (SARS) in humans, associated with a new coronavirus, was reported in Southeast Asia, Europe, and North America in early 2003. To address speculations that the virus originated in domesticated animals, or that domestic species were susceptible to the virus, we inoculated 6-week-old pigs and chickens intravenously, intranasally, ocularly, and orally with 10(6) PFU of SARS-associated coronavirus (SARS-CoV). Clinical signs did not develop in any animal, nor were gross pathologic changes evident on postmortem examinations. Attempts at virus isolation were unsuccessful; however, viral RNA was detected by reverse transcriptase-polymerase chain reaction in blood of both species during the first week after inoculation, and in chicken organs at 2 weeks after inoculation. Virus-neutralizing antibodies developed in the pigs. Our results indicate that these animals do not play a role as amplifying hosts for SARS-CoV.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>[Advances on presymptomatic or asymptomatic carrier transmission of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is rapidly spreading. Patients in incubation period and healthy carriers are possible sources for transmission. However, such sources of infection cannot be effectively identified due to the symptoms absent. The research evidence is very lacking so far, although there are a few studies suggesting that presymptomatic or asymptomatic carrier may cause COVID-19 transmission. Nearly half of the literature is in the state of preprint without peer review. The question of "the degree to which presymptomatic or asymptomatic infections can transmit" is not fully understood. There is an urgent need to screen infected carriers in larger close contacts or in the general population, and assess their risk for transmission.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Cross-Species Virus Transmission and the Emergence of New Epidemic Diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Summary: Host range is a viral property reflecting natural hosts that are infected either as part of a principal transmission cycle or, less commonly, as “spillover” infections into alternative hosts. Rarely, viruses gain the ability to spread efficiently within a new host that was not previously exposed or susceptible. These transfers involve either increased exposure or the acquisition of variations that allow them to overcome barriers to infection of the new hosts. In these cases, devastating outbreaks can result. Steps involved in transfers of viruses to new hosts include contact between the virus and the host, infection of an initial individual leading to amplification and an outbreak, and the generation within the original or new host of viral variants that have the ability to spread efficiently between individuals in populations of the new host. Here we review what is known about host switching leading to viral emergence from known examples, considering the evolutionary mechanisms, virus-host interactions, host range barriers to infection, and processes that allow efficient host-to-host transmission in the new host population.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Tacaribe Virus Causes Fatal Infection of An Ostensible Reservoir Host, the Jamaican Fruit Bat</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Tacaribe virus (TCRV) was first isolated from 11 Artibeus species bats captured in Trinidad in the 1950s during a rabies virus surveillance program. Despite significant effort, no evidence of infection of other mammals, mostly rodents, was found, suggesting that no other vertebrates harbored TCRV. For this reason, it was hypothesized that TCRV was naturally hosted by artibeus bats. This is in stark contrast to other arenaviruses with known hosts, all of which are rodents. To examine this hypothesis, we conducted experimental infections of Jamaican fruit bats (Artibeus jamaicensis) to determine whether they could be persistently infected without substantial pathology. We subcutaneously or intranasally infected bats with TCRV strain TRVL-11573, the only remaining strain of TCRV, and found that low-dose (10(4) 50% tissue culture infective dose [TCID(50)]) inoculations resulted in asymptomatic and apathogenic infection and virus clearance, while high-dose (10(6) TCID(50)) inoculations caused substantial morbidity and mortality as early as 10 days postinfection. Uninoculated cage mates failed to seroconvert, and viral RNA was not detected in their tissues, suggesting that transmission did not occur. Together, these data suggest that A. jamaicensis bats may not be a reservoir host for TCRV.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Changes in temperature alter the potential outcomes of virus host shifts</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Oct 19</span></i><p style="text-align: justify; text-justify: inter-word;">Host shifts–where a pathogen jumps between different host species–are an important source of emerging infectious disease. With on-going climate change there is an increasing need to understand the effect changes in temperature may have on emerging infectious disease. We investigated whether species’ susceptibilities change with temperature and ask if susceptibility is greatest at different temperatures in different species. We infected 45 species of Drosophilidae with an RNA virus and measured how viral load changes with temperature. We found the host phylogeny explained a large proportion of the variation in viral load at each temperature, with strong phylogenetic correlations between viral loads across temperature. The variance in viral load increased with temperature, while the mean viral load did not. This suggests that as temperature increases the most susceptible species become more susceptible, and the least susceptible less so. We found no significant relationship between a species’ susceptibility across temperatures, and proxies for thermal optima (critical thermal maximum and minimum or basal metabolic rate). These results suggest that whilst the rank order of species susceptibilities may remain the same with changes in temperature, some species may become more susceptible to a novel pathogen, and others less so.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Searching for animal models and potential target species for emerging pathogens: Experience gained from Middle East respiratory syndrome (MERS) coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Domestic Pig Unlikely Reservoir for MERS-CoV</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">We tested the suitability of the domestic pig as a model for Middle East respiratory syndrome coronavirus (MERS-CoV) infection. Inoculation did not cause disease, but a low level of virus replication, shedding, and seroconversion were observed. Pigs do not recapitulate human MERS-CoV and are unlikely to constitute a reservoir in nature.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">What is COVID-19, and what do we know so far about its clinical presentation? The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1]. The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host [2]. Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficient Replication of Severe Acute Respiratory Syndrome Coronavirus in Mouse Cells Is Limited by Murine Angiotensin-Converting Enzyme 2</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Replication of viruses in species other than their natural hosts is frequently limited by entry and postentry barriers. The coronavirus that causes severe acute respiratory syndrome (SARS-CoV) utilizes the receptor angiotensin-converting enzyme 2 (ACE2) to infect cells. Here we compare human, mouse, and rat ACE2 molecules for their ability to serve as receptors for SARS-CoV. We found that, compared to human ACE2, murine ACE2 less efficiently bound the S1 domain of SARS-CoV and supported less-efficient S protein-mediated infection. Rat ACE2 was even less efficient, at near background levels for both activities. Murine 3T3 cells expressing human ACE2 supported SARS-CoV replication, whereas replication was less than 10% as efficient in the same cells expressing murine ACE2. These data imply that a mouse transgenically expressing human ACE2 may be a useful animal model of SARS.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of Novel Human and Animal Cells Infected by the Severe Acute Respiratory Syndrome Coronavirus by Replication-Specific Multiplex Reverse Transcription-PCR</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of the recent outbreak of severe acute respiratory syndrome. VeroE6 cells, fetal rhesus monkey kidney cells, and human peripheral blood mononuclear cells were the only cells known to be susceptible to SARS-CoV. We developed a multiplex reverse transcription-PCR assay to analyze the susceptibility of cells derived from a variety of tissues and species to SARS-CoV. Additionally, productive infection was determined by titration of cellular supernatants. Cells derived from three species of monkey were susceptible to SARS-CoV. However, the levels of SARS-CoV produced differed by 4 log(10). Mink lung epithelial cells (Mv1Lu) and R-Mix, a mixed monolayer of human lung-derived cells (A549) and mink lung-derived cells (Mv1Lu), are used by diagnostic laboratories to detect respiratory viruses (e.g., influenza virus); they were also infected with SARS-CoV, indicating that the practices of diagnostic laboratories should be examined to ensure appropriate biosafety precautions. Mv1Lu cells produce little SARS-CoV compared to that produced by VeroE6 cells, which indicates that they are a safer alternative for SARS-CoV diagnostics. Evaluation of cells permissive to other coronaviruses indicated that these cell types are not infected by SARS-CoV, providing additional evidence that SARS-CoV binds an alternative receptor. Analysis of human cells derived from lung, kidney, liver, and intestine led to the discovery that human cell lines were productively infected by SARS-CoV. This study identifies new cell lines that may be used for SARS-CoV diagnostics and/or basic research. Our data and other in vivo studies indicate that SARS-CoV has a wide host range, suggesting that the cellular receptor(s) utilized by SARS-CoV is highly conserved and is expressed by a variety of tissues.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Replication and shedding of MERS-CoV in Jamaican fruit bats (Artibeus jamaicensis)</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb 22</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) highlights the zoonotic potential of Betacoronaviruses. Investigations into the origin of MERS-CoV have focused on two potential reservoirs: bats and camels. Here, we investigated the role of bats as a potential reservoir for MERS-CoV. In vitro, the MERS-CoV spike glycoprotein interacted with Jamaican fruit bat (Artibeus jamaicensis) dipeptidyl peptidase 4 (DPP4) receptor and MERS-CoV replicated efficiently in Jamaican fruit bat cells, suggesting there is no restriction at the receptor or cellular level for MERS-CoV. To shed light on the intrinsic host-virus relationship, we inoculated 10 Jamaican fruit bats with MERS-CoV. Although all bats showed evidence of infection, none of the bats showed clinical signs of disease. Virus shedding was detected in the respiratory and intestinal tract for up to 9 days. MERS-CoV replicated transiently in the respiratory and, to a lesser extent, the intestinal tracts and internal organs; with limited histopathological changes observed only in the lungs. Analysis of the innate gene expression in the lungs showed a moderate, transient induction of expression. Our results indicate that MERS-CoV maintains the ability to replicate in bats without clinical signs of disease, supporting the general hypothesis of bats as ancestral reservoirs for MERS-CoV.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Replication and shedding of MERS-CoV in Jamaican fruit bats (Artibeus jamaicensis)</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb 22</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) highlights the zoonotic potential of Betacoronaviruses. Investigations into the origin of MERS-CoV have focused on two potential reservoirs: bats and camels. Here, we investigated the role of bats as a potential reservoir for MERS-CoV. In vitro, the MERS-CoV spike glycoprotein interacted with Jamaican fruit bat (Artibeus jamaicensis) dipeptidyl peptidase 4 (DPP4) receptor and MERS-CoV replicated efficiently in Jamaican fruit bat cells, suggesting there is no restriction at the receptor or cellular level for MERS-CoV. To shed light on the intrinsic host-virus relationship, we inoculated 10 Jamaican fruit bats with MERS-CoV. Although all bats showed evidence of infection, none of the bats showed clinical signs of disease. Virus shedding was detected in the respiratory and intestinal tract for up to 9 days. MERS-CoV replicated transiently in the respiratory and, to a lesser extent, the intestinal tracts and internal organs; with limited histopathological changes observed only in the lungs. Analysis of the innate gene expression in the lungs showed a moderate, transient induction of expression. Our results indicate that MERS-CoV maintains the ability to replicate in bats without clinical signs of disease, supporting the general hypothesis of bats as ancestral reservoirs for MERS-CoV.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-like WIV1-CoV poised for human emergence</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Outbreaks from zoonotic sources represent a threat to both human disease as well as the global economy. Despite a wealth of metagenomics studies, methods to leverage these datasets to identify future threats are underdeveloped. In this study, we describe an approach that combines existing metagenomics data with reverse genetics to engineer reagents to evaluate emergence and pathogenic potential of circulating zoonotic viruses. Focusing on the severe acute respiratory syndrome (SARS)-like viruses, the results indicate that the WIV1-coronavirus (CoV) cluster has the ability to directly infect and may undergo limited transmission in human populations. However, in vivo attenuation suggests additional adaptation is required for epidemic disease. Importantly, available SARS monoclonal antibodies offered success in limiting viral infection absent from available vaccine approaches. Together, the data highlight the utility of a platform to identify and prioritize prepandemic strains harbored in animal reservoirs and document the threat posed by WIV1-CoV for emergence in human populations.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Inoculation of Goats, Sheep, and Horses with MERS-CoV Does Not Result in Productive Viral Shedding</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug 19</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first recognized in 2012 and can cause severe disease in infected humans. Dromedary camels are the reservoir for the virus, although, other than nasal discharge, these animals do not display any overt clinical disease. Data from in vitro experiments suggest that other livestock such as sheep, goats, and horses might also contribute to viral transmission, although field data has not identified any seropositive animals. In order to understand if these animals could be infected, we challenged young goats and horses and adult sheep with MERS-CoV by intranasal inoculation. Minimal or no virus shedding was detected in all of the animals. During the four weeks following inoculation, neutralizing antibodies were detected in the young goats, but not in sheep or horses.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND The coronavirus disease 2019 (COVID-19) is a newly emerged infection from the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Apart from the respiratory system, it is unclear whether SARS-CoV-2 can also directly infect other tissues such as the kidney or induce acute renal failure. METHODS We conducted a retrospective analysis of estimated glomerular filtration rate (eGFR), plasma creatinine, and urea concentrations along with other clinical parameters from 85 patients with laboratory-confirmed COVID-19 admitted to a hospital in Wuhan from January 17, 2020 to March 3, 2020. Kidney tissues from six other patients with postmortem examinations were analyzed by Hematoxylin and Eosin (H&amp;E) and in situ expression of viral nucleocaspid protein (NP) antigen was detected by immunohistochemistry. RESULTS Among these 85 COVID-19 cases, 27.06% (23/85) patients exhibited acute renal failure (ARF), and elder patients (≥ 60 years old) are easier to develop ARF (65.22% vs 24.19%, p&lt; 0.001). Comorbidities of disorders like hypertension and heart failure were more common in patients who developed ARF (69.57% vs 11.29%, p&lt; 0.001). Dynamic observation of eGFR revealed that perished cases have a rapid decrease of eGFR but quickly boosting plasma creatinine and urea. On the other hand, enhancement of eGFR and persistence low level of plasma creatinine and urea was observed in recovery patients following diuretic treatment. H&amp;E staining demonstrated kidney tissues from 6 cases of postmortems have severe acute tubular necrosis but no evidence of glomerular pathology or tubulointerstitial lymphocyte infiltration. Immunohistochemistry showed that SARS-CoV-2 NP antigen was accumulated in kidney tubules.   CONCLUSIONS The development of ARF during the course of disease is an important negative prognostic indicator for survival with COVID-19. SARS-CoV-2 virus directly infects human kidney tubules by thus induces acute tubular damage, ARF and probably also lead to urine transmission.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>The Evolution and Genetics of Virus Host Shifts</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 6</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging viral diseases are often the product of a host shift, where a pathogen jumps from its original host into a novel species. Phylogenetic studies show that host shifts are a frequent event in the evolution of most pathogens, but why pathogens successfully jump between some host species but not others is only just becoming clear. The susceptibility of potential new hosts can vary enormously, with close relatives of the natural host typically being the most susceptible. Often, pathogens must adapt to successfully infect a novel host, for example by evolving to use different cell surface receptors, to escape the immune response, or to ensure they are transmitted by the new host. In viruses there are often limited molecular solutions to achieve this, and the same sequence changes are often seen each time a virus infects a particular host. These changes may come at a cost to other aspects of the pathogen's fitness, and this may sometimes prevent host shifts from occurring. Here we examine how these evolutionary factors affect patterns of host shifts and disease emergence.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological and clinical characteristics of SARS-CoV-2 and SARS-CoV: a system review</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background:  In this article, we summarized the early epidemiologic and clinical characteristics of SARS and COVID-19 in different countries. Aim to provide recommendations for the understanding and prevention of COVID-19.  Methods: By searching pubmed, we analyzed and compared the typical cases of SARS and COVID-19 in different countries in the early stage of the outbreak. Clinical records, laboratory results, imageological diagnosis and pathologic condition were retrospectively reviewed for these cases. This paper adopts the method of descriptive statistics and tables and graphs. Findings: Six SARS-related articles (N=337 participants) and 12 COVID-19-related articles (N=50,096 participants) were eligible for this systematic review. Fever, cough and Malaise/Fatigue were the most common symptoms in SARS and COVID-19. But in general, the clinical symptoms and signs of COVID-19 were not obvious. Compared with SARS, COVID-19 was transmitted in a more diverse way, from person to person, asymptomatic infected people and possible fecal-oral transmission, created the conditions for a large-scale spread. The mortality rates of SARS and COVID-19 were (7.7%) and (2.2%) respectively, but the overall infection rate of healthcare worker of COVID-19 (3.9%) was lower than that of SARS (40.0%). We also summarize the current reports on the pathology of COVID-19, we found that the pathological features of COVID-19 have greatly similar with SARS, which manifested as acute respiratory distress syndrome (ARDS). Interpretation: The epidemiological and clinical characteristics of SARS and COVID-19 in China are very similar, but also difference. In general, COVID-19 is transmitted in more diverse ways and is more infectious, so the early recognition of disease by healthcare worker and patient is very important. Active and effective isolation measures for suspected and close contacts are necessary.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Interferon Regulatory Factor 3-Mediated Signaling Limits Middle-East Respiratory Syndrome (MERS) Coronavirus Propagation in Cells from an Insectivorous Bat</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">Insectivorous bats are speculated to be ancestral hosts of Middle-East respiratory syndrome (MERS) coronavirus (CoV). MERS-CoV causes disease in humans with thirty-five percent fatality, and has evolved proteins that counteract human antiviral responses. Since bats experimentally infected with MERS-CoV do not develop signs of disease, we tested the hypothesis that MERS-CoV would replicate less efficiently in bat cells than in human cells because of its inability to subvert antiviral responses in bat cells. We infected human and bat (Eptesicus fuscus) cells with MERS-CoV and observed that the virus grew to higher titers in human cells. MERS-CoV also effectively suppressed the antiviral interferon beta (IFNβ) response in human cells, unlike in bat cells. To determine if IRF3, a critical mediator of the interferon response, also regulated the response in bats, we examined the response of IRF3 to poly(I:C), a synthetic analogue of viral double-stranded RNA. We observed that bat IRF3 responded to poly(I:C) by nuclear translocation and post-translational modifications, hallmarks of IRF3 activation. Suppression of IRF3 by small-interfering RNA (siRNA) demonstrated that IRF3 was critical for poly(I:C) and MERS-CoV induced induction of IFNβ in bat cells. Our study demonstrates that innate antiviral signaling in E. fuscus bat cells is resistant to MERS-CoV-mediated subversion.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Interferon Regulatory Factor 3-Mediated Signaling Limits Middle-East Respiratory Syndrome (MERS) Coronavirus Propagation in Cells from an Insectivorous Bat</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">Insectivorous bats are speculated to be ancestral hosts of Middle-East respiratory syndrome (MERS) coronavirus (CoV). MERS-CoV causes disease in humans with thirty-five percent fatality, and has evolved proteins that counteract human antiviral responses. Since bats experimentally infected with MERS-CoV do not develop signs of disease, we tested the hypothesis that MERS-CoV would replicate less efficiently in bat cells than in human cells because of its inability to subvert antiviral responses in bat cells. We infected human and bat (Eptesicus fuscus) cells with MERS-CoV and observed that the virus grew to higher titers in human cells. MERS-CoV also effectively suppressed the antiviral interferon beta (IFNβ) response in human cells, unlike in bat cells. To determine if IRF3, a critical mediator of the interferon response, also regulated the response in bats, we examined the response of IRF3 to poly(I:C), a synthetic analogue of viral double-stranded RNA. We observed that bat IRF3 responded to poly(I:C) by nuclear translocation and post-translational modifications, hallmarks of IRF3 activation. Suppression of IRF3 by small-interfering RNA (siRNA) demonstrated that IRF3 was critical for poly(I:C) and MERS-CoV induced induction of IFNβ in bat cells. Our study demonstrates that innate antiviral signaling in E. fuscus bat cells is resistant to MERS-CoV-mediated subversion.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the case fatality ratio of the COVID-19 epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Corona Virus Disease 2019 (COVID-19) due to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan city and rapidly spread throughout China since late December 2019. Crude case fatality ratio (CFR) with dividing the number of known deaths by the number of confirmed cases does not represent the true CFR and might be off by orders of magnitude. We aim to provide a precise estimate of the CFR of COVID-19 using statistical models at the early stage of the epidemic. Methods: We extracted data from the daily released epidemic report published by the National Health Commission P. R. China from 20 Jan 2020, to 1 March 2020. Competing risk model was used to obtain the cumulative hazards for death, cure, and cure-death hazard ratio. Then the CFR was estimated based on the slope of the last piece in joinpoint regression model, which reflected the most recent trend of the epidemic.  Results: As of 1 March 2020, totally 80,369 cases were diagnosed as COVID-19 in China. The CFR of COVID-19 were estimated to be 70.9% (95% CI: 66.8%-75.6%) during Jan 20-Feb 2, 20.2% (18.6%-22.1%) during Feb 3-14, 6.9% (6.4%-7.4%) during Feb 15-23, 1.5% (1.4%-1.6%) during Feb 24-March 1 in Hubei province, and 20.3% (17.0%-25.3%) during Jan 20-28, 1.9% (1.8%-2.1%) during Jan 29-Feb 12, 0.9% (0.8%-1.1%) during Feb 13-18, 0.4% (0.4%-0.5%) during Feb 19-March 1 in other areas of China, respectively.  Conclusions: Based on analyses of public data, we found that the CFR in Hubei was much higher than that of other regions in China, over 3 times in all estimation. The CFR would follow a downwards trend based on our estimation from recently released data. Nevertheless, at early stage of outbreak, CFR estimates should be viewed cautiously because of limited data source on true onset and recovery time.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Exogenous ACE2 Expression Allows Refractory Cell Lines To Support Severe Acute Respiratory Syndrome Coronavirus Replication</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Of 30 cell lines and primary cells examined, productive severe acute respiratory syndrome coronavirus (Urbani strain) (SARS-CoV) infection after low-multiplicity inoculation was detected in only six: three African green monkey kidney epithelial cell lines (Vero, Vero E6, and MA104), a human colon epithelial line (CaCo-2), a porcine kidney epithelial line [PK(15)], and mink lung epithelial cells (Mv 1 Lu). SARS-CoV produced a lytic infection in Vero, Vero E6, and MA104 cells, but there was no visible cytopathic effect in Caco-2, Mv 1 Lu, or PK(15) cells. Multistep growth kinetics were identical in Vero E6 and MA104 cells, with maximum titer reached 24 h postinoculation (hpi). Virus titer was maximal 96 hpi in CaCo-2 cells, and virus was continually produced from infected CaCo-2 cells for at least 6 weeks after infection. CaCo-2 was the only human cell type of 13 tested that supported efficient SARS-CoV replication. Expression of the SARS-CoV receptor, angiotensin-converting enzyme 2 (ACE2), resulted in SARS-CoV replication in all refractory cell lines examined. Titers achieved were variable and dependent upon the method of ACE2 expression.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Canine and feline host ranges of canine parvovirus and feline panleukopenia virus: distinct host cell tropisms of each virus in vitro and in vivo.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Canine parvovirus (CPV) emerged as an apparently new virus during the mid-1970s. The origin of CPV is unknown, but a variation from feline panleukopenia virus (FPV) or another closely related parvovirus is suspected. Here we examine the in vitro and in vivo canine and feline host ranges of CPV and FPV. Examination of three canine and six feline cell lines and mitogen-stimulated canine and feline peripheral blood lymphocytes revealed that CPV replicates in both canine and feline cells, whereas FPV replicates efficiently only in feline cells. The in vivo host ranges were unexpectedly complex and distinct from the in vitro host ranges. Inoculation of dogs with FPV revealed efficient replication in the thymus and, to some degree, in the bone marrow, as shown by virus isolation, viral DNA recovery, and Southern blotting and by strand-specific in situ hybridization. FPV replication could not be demonstrated in mesenteric lymph nodes or in the small intestine, which are important target tissues in CPV infection. Although CPV replicated well in all the feline cells tested in vitro, it did not replicate in any tissue of cats after intramuscular or intravenous inoculation. These results indicate that these viruses have complex and overlapping host ranges and that distinct tissue tropisms exist in the homologous and heterologous hosts.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Viral Population Diversity in Adaptation of Bovine Coronavirus to New Host Environments</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jan 7</span></i><p style="text-align: justify; text-justify: inter-word;">The high mutation rate of RNA viruses enables a diverse genetic population of viral genotypes to exist within a single infected host. In-host genetic diversity could better position the virus population to respond and adapt to a diverse array of selective pressures such as host-switching events. Multiple new coronaviruses, including SARS, have been identified in human samples just within the last ten years, demonstrating the potential of coronaviruses as emergent human pathogens. Deep sequencing was used to characterize genomic changes in coronavirus quasispecies during simulated host-switching. Three bovine nasal samples infected with bovine coronavirus were used to infect human and bovine macrophage and lung cell lines. The virus reproduced relatively well in macrophages, but the lung cell lines were not infected efficiently enough to allow passage of non lab-adapted samples. Approximately 12 kb of the genome was amplified before and after passage and sequenced at average coverages of nearly 950×(454 sequencing) and 38,000×(Illumina). The consensus sequence of many of the passaged samples had a 12 nucleotide insert in the consensus sequence of the spike gene, and multiple point mutations were associated with the presence of the insert. Deep sequencing revealed that the insert was present but very rare in the unpassaged samples and could quickly shift to dominate the population when placed in a different environment. The insert coded for three arginine residues, occurred in a region associated with fusion entry into host cells, and may allow infection of new cell types via heparin sulfate binding. Analysis of the deep sequencing data indicated that two distinct genotypes circulated at different frequency levels in each sample, and support the hypothesis that the mutations present in passaged strains were “selected” from a pre-existing pool rather than through de novo mutation and subsequent population fixation.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution, Safety, and Highly Pathogenic Influenza Viruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Experience with influenza has shown that predictions of virus phenotype or fitness from nucleotide sequence are imperfect, and that predicting the timing and course of evolution is extremely difficult. Such uncertainty means that the risk of experiments with mammalian-transmissible, possibly highly virulent influenza viruses remains high even if some aspects of their laboratory biology are reassuring; it also implies limitations on the ability of laboratory observations to guide interpretation of surveillance of strains in the field. Given these considerations, we propose that future experiments with virulent pathogens whose accidental or deliberate release could lead to extensive spread in human populations should be limited by explicit risk-benefit considerations, in the United States and worldwide.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Bovine-Like Coronaviruses Isolated from Four Species of Captive Wild Ruminants Are Homologous to Bovine Coronaviruses, Based on Complete Genomic Sequences</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We sequenced and analyzed the full-length genomes of four coronaviruses (CoVs), each from a distinct wild-ruminant species in Ohio: sambar deer (Cervus unicolor), a waterbuck (Kobus ellipsiprymnus), a sable antelope (Hippotragus niger), and a white-tailed deer (Odocoileus virginianus). The fecal samples from the sambar deer, the waterbuck, and the white-tailed deer were collected during winter dysentery outbreaks and sporadic diarrhea cases in 1993 and 1994 (H. Tsunemitsu, Z. R. el-Kanawati, D. R. Smith, H. H. Reed, and L. J. Saif, J. Clin. Microbiol. 33:3264-3269, 1995). A fecal sample from a sable antelope was collected in 2003 from an Ohio wild-animal habitat during the same outbreak when a bovine-like CoV from a giraffe (GiCoV) was isolated (M. Hasoksuz, K. Alekseev, A. Vlasova, X. Zhang, D. Spiro, R. Halpin, S. Wang, E. Ghedin, and L. J. Saif, J. Virol. 81:4981-4990, 2007). For two of the CoVs (sambar deer and waterbuck), complete genomes from both the cell culture-adapted and gnotobiotic-calf-passaged strains were also sequenced and analyzed. Phylogenetically, wild-ruminant CoVs belong to group 2a CoVs, with the closest relatedness to recent bovine CoV (BCoV) strains. High nucleotide identities (99.4 to 99.6%) among the wild-ruminant strains and recent BCoV strains (BCoV-LUN and BCoV-ENT, isolated in 1998) further confirm the close relatedness. Comparative genetic analysis of CoVs of captive wild ruminants with BCoV strains suggests that no specific genomic markers are present that allow discrimination between the bovine strains and bovine-like CoVs from captive wild ruminants; furthermore, no specific genetic markers were identified that defined cell cultured or calf-passaged strains or the host origin of strains. The results of this study confirm prior reports of biologic and antigenic similarities between bovine and wild-ruminant CoVs and suggest that cattle may be reservoirs for CoVs that infect captive wild ruminants or vice versa and that these CoVs may represent host range variants of an ancestral CoV.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>CD26/DPP4 Cell-Surface Expression in Bat Cells Correlates with Bat Cell Susceptibility to Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Infection and Evolution of Persistent Infection</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 19</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a recently isolated betacoronavirus identified as the etiologic agent of a frequently fatal disease in Western Asia, Middle East respiratory syndrome. Attempts to identify the natural reservoirs of MERS-CoV have focused in part on dromedaries. Bats are also suspected to be reservoirs based on frequent detection of other betacoronaviruses in these mammals. For this study, ten distinct cell lines derived from bats of divergent species were exposed to MERS-CoV. Plaque assays, immunofluorescence assays, and transmission electron microscopy confirmed that six bat cell lines can be productively infected. We found that the susceptibility or resistance of these bat cell lines directly correlates with the presence or absence of cell surface-expressed CD26/DPP4, the functional human receptor for MERS-CoV. Human anti-CD26/DPP4 antibodies inhibited infection of susceptible bat cells in a dose-dependent manner. Overexpression of human CD26/DPP4 receptor conferred MERS-CoV susceptibility to resistant bat cell lines. Finally, sequential passage of MERS-CoV in permissive bat cells established persistent infection with concomitant downregulation of CD26/DPP4 surface expression. Together, these results imply that bats indeed could be among the MERS-CoV host spectrum, and that cellular restriction of MERS-CoV is determined by CD26/DPP4 expression rather than by downstream restriction factors.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Infection and Titration of SARS-CoV in Mouse Lung</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two critical steps when investigating an animal model of a virus infection are consistently successfully infecting animals and accurately determining viral titers in tissue throughout the course of infection. Here we discuss in detail how to infect mice with SARS-CoV and then quantify the titer of virus in the lung.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Cross-Species Infection of Common Marmosets by Titi Monkey Adenovirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 24</span></i><p style="text-align: justify; text-justify: inter-word;">Adenoviruses are DNA viruses that infect a number of vertebrate hosts and are associated with both sporadic and epidemic disease in humans. We previously identified a novel adenovirus, titi monkey adenovirus (TMAdV), as the cause of a fulminant pneumonia outbreak in a colony of titi monkeys (Callicebus cupreus) at a national primate center in 2009. Serological evidence of infection by TMAdV was also found in a human researcher at the facility and household family member, raising concerns for potential cross-species transmission of the virus. Here we present experimental evidence of cross-species TMAdV infection in common marmosets (Callithrix jacchus). Nasal inoculation of a cell cultured-adapted TMAdV strain into three marmosets produced an acute, mild respiratory illness characterized by low-grade fever, reduced activity, anorexia, and sneezing. An increase in virus-specific neutralization antibody titers accompanied the development of clinical signs. Although serially collected nasal swabs were positive for TMAdV for at least 8 days, all 3 infected marmosets spontaneously recovered by day 12 post-inoculation, and persistence of the virus in tissues could not be established. Thus, the pathogenesis of experimental inoculation of TMAdV in common marmosets resembled the mild, self-limiting respiratory infection typically seen in immunocompetent human hosts rather than the rapidly progressive, fatal pneumonia observed in 19 of 23 titi monkeys during the prior 2009 outbreak. These findings further establish the potential for adenovirus cross-species transmission and provide the basis for development of a monkey model useful for assessing the zoonotic potential of adenoviruses.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Cross-Species Infection of Common Marmosets by Titi Monkey Adenovirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 24</span></i><p style="text-align: justify; text-justify: inter-word;">Adenoviruses are DNA viruses that infect a number of vertebrate hosts and are associated with both sporadic and epidemic disease in humans. We previously identified a novel adenovirus, titi monkey adenovirus (TMAdV), as the cause of a fulminant pneumonia outbreak in a colony of titi monkeys (Callicebus cupreus) at a national primate center in 2009. Serological evidence of infection by TMAdV was also found in a human researcher at the facility and household family member, raising concerns for potential cross-species transmission of the virus. Here we present experimental evidence of cross-species TMAdV infection in common marmosets (Callithrix jacchus). Nasal inoculation of a cell cultured-adapted TMAdV strain into three marmosets produced an acute, mild respiratory illness characterized by low-grade fever, reduced activity, anorexia, and sneezing. An increase in virus-specific neutralization antibody titers accompanied the development of clinical signs. Although serially collected nasal swabs were positive for TMAdV for at least 8 days, all 3 infected marmosets spontaneously recovered by day 12 post-inoculation, and persistence of the virus in tissues could not be established. Thus, the pathogenesis of experimental inoculation of TMAdV in common marmosets resembled the mild, self-limiting respiratory infection typically seen in immunocompetent human hosts rather than the rapidly progressive, fatal pneumonia observed in 19 of 23 titi monkeys during the prior 2009 outbreak. These findings further establish the potential for adenovirus cross-species transmission and provide the basis for development of a monkey model useful for assessing the zoonotic potential of adenoviruses.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Cross-Species Infection of Common Marmosets by Titi Monkey Adenovirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 24</span></i><p style="text-align: justify; text-justify: inter-word;">Adenoviruses are DNA viruses that infect a number of vertebrate hosts and are associated with both sporadic and epidemic disease in humans. We previously identified a novel adenovirus, titi monkey adenovirus (TMAdV), as the cause of a fulminant pneumonia outbreak in a colony of titi monkeys (Callicebus cupreus) at a national primate center in 2009. Serological evidence of infection by TMAdV was also found in a human researcher at the facility and household family member, raising concerns for potential cross-species transmission of the virus. Here we present experimental evidence of cross-species TMAdV infection in common marmosets (Callithrix jacchus). Nasal inoculation of a cell cultured-adapted TMAdV strain into three marmosets produced an acute, mild respiratory illness characterized by low-grade fever, reduced activity, anorexia, and sneezing. An increase in virus-specific neutralization antibody titers accompanied the development of clinical signs. Although serially collected nasal swabs were positive for TMAdV for at least 8 days, all 3 infected marmosets spontaneously recovered by day 12 post-inoculation, and persistence of the virus in tissues could not be established. Thus, the pathogenesis of experimental inoculation of TMAdV in common marmosets resembled the mild, self-limiting respiratory infection typically seen in immunocompetent human hosts rather than the rapidly progressive, fatal pneumonia observed in 19 of 23 titi monkeys during the prior 2009 outbreak. These findings further establish the potential for adenovirus cross-species transmission and provide the basis for development of a monkey model useful for assessing the zoonotic potential of adenoviruses.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Cross-Species Infection of Common Marmosets by Titi Monkey Adenovirus</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 24</span></i><p style="text-align: justify; text-justify: inter-word;">Adenoviruses are DNA viruses that infect a number of vertebrate hosts and are associated with both sporadic and epidemic disease in humans. We previously identified a novel adenovirus, titi monkey adenovirus (TMAdV), as the cause of a fulminant pneumonia outbreak in a colony of titi monkeys (Callicebus cupreus) at a national primate center in 2009. Serological evidence of infection by TMAdV was also found in a human researcher at the facility and household family member, raising concerns for potential cross-species transmission of the virus. Here we present experimental evidence of cross-species TMAdV infection in common marmosets (Callithrix jacchus). Nasal inoculation of a cell cultured-adapted TMAdV strain into three marmosets produced an acute, mild respiratory illness characterized by low-grade fever, reduced activity, anorexia, and sneezing. An increase in virus-specific neutralization antibody titers accompanied the development of clinical signs. Although serially collected nasal swabs were positive for TMAdV for at least 8 days, all 3 infected marmosets spontaneously recovered by day 12 post-inoculation, and persistence of the virus in tissues could not be established. Thus, the pathogenesis of experimental inoculation of TMAdV in common marmosets resembled the mild, self-limiting respiratory infection typically seen in immunocompetent human hosts rather than the rapidly progressive, fatal pneumonia observed in 19 of 23 titi monkeys during the prior 2009 outbreak. These findings further establish the potential for adenovirus cross-species transmission and provide the basis for development of a monkey model useful for assessing the zoonotic potential of adenoviruses.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Distribution of O-Acetylated Sialic Acids among Target Host Tissues for Influenza Virus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Sep 6</span></i><p style="text-align: justify; text-justify: inter-word;">Sialic acids (Sias) are important glycans displayed on the cells and tissues of many different animals and are frequent targets for binding and modification by pathogens, including influenza viruses. Influenza virus hemagglutinins bind Sias during the infection of their normal hosts, while the encoded neuraminidases and/or esterases remove or modify the Sia to allow virion release or to prevent rebinding. Sias naturally occur in a variety of modified forms, and modified Sias can alter influenza virus host tropisms through their altered interactions with the viral glycoproteins. However, the distribution of modified Sia forms and their effects on pathogen-host interactions are still poorly understood. Here we used probes developed from viral Sia-binding proteins to detect O-acetylated (4-O-acetyl, 9-O-acetyl, and 7,9-O-acetyl) Sias displayed on the tissues of some natural or experimental hosts for influenza viruses. These modified Sias showed highly variable displays between the hosts and tissues examined. The 9-O-acetyl (and 7,9-) modified Sia forms were found on cells and tissues of many hosts, including mice, humans, ferrets, guinea pigs, pigs, horses, dogs, as well as in those of ducks and embryonated chicken egg tissues and membranes, although in variable amounts. The 4-O-acetyl Sias were found in the respiratory tissues of fewer animals, being primarily displayed in the horse and guinea pig, but were not detected in humans or pigs. The results suggest that these Sia variants may influence virus tropisms by altering and selecting their cell interactions. IMPORTANCE Sialic acids (Sias) are key glycans that control or modulate many normal cell and tissue functions while also interacting with a variety of pathogens, including many different viruses. Sias are naturally displayed in a variety of different forms, with modifications at several positions that can alter their functional interactions with pathogens. In addition, Sias are often modified or removed by enzymes such as host or pathogen esterases or sialidases (neuraminidases), and Sia modifications can alter those enzymatic activities to impact pathogen infections. Sia chemical diversity in different hosts and tissues likely alters the pathogen-host interactions and influences the outcome of infection. Here we explored the display of 4-O-acetyl, 9-O-acetyl, and 7,9-O-acetyl modified Sia forms in some target tissues for influenza virus infection in mice, humans, birds, guinea pigs, ferrets, swine, horses, and dogs, which encompass many natural and laboratory hosts of those viruses.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Daily briefing: When a tiny lab accident almost leads to amputation</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Three weeks ago, on the basis of genetic analyses, pangolins became the prime suspect as the animal source of the coronavirus that causes COVID-19. Further analysis of those data — alongside three other genome studies of pangolin coronaviruses — shows that although the scaly anteater is still a contender, the link is far from conclusive. Whatever the source, scientists emphasize that animals should not be vilified for their role in the outbreak. “The problem is not the animals, it’s that we get in contact with them,” says animal-behaviour researcher Sara Platto. (Nature | 4 min read)</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of COVID-2019 burden and potential for international dissemination of infection from Iran</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The Coronavirus Disease 2019 (COVID-19) epidemic began in Wuhan, China in late 2019 and continues to spread globally, with exported cases confirmed in 28 countries at the time of writing. During the interval between February 19 and 23, 2020, Iran reported its first 43 cases with eight deaths.  Three exported cases originating in Iran were identified, suggesting a underlying burden of disease in that country than is indicated by reported cases.  A large epidemic in Iran could further fuel global dissemination of COVID-19.  We sought to estimate COVID-19 outbreak size in Iran based on known exported case counts and air travel links between Iran and other countries, and to anticipate where infections originating in Iran may spread to next.  We assessed interconnectivity between Iran and other countries using using International Air Transport Association (IATA) data.  We used the methods of Fraser et al. to estimate the size of the underlying epidemic that would result in cases being observed  in the United Arab Emirates (UAE), Lebanon, and Canada.  Time at risk estimates were based on a presumed 6 week epidemic age, and length of stay data for visitors to Iran derived from the United Nations World Tourism Organization (UNWTO).  We evaluated the relationship between the strength of travel links with Iran, and destination country rankings on the Infectious Disease Vulnerability Index (IDVI), a validated metric that estimates the capacity of a country to respond to an infectious disease outbreak . Scores range between 0-1, with higher scores reflecting greater capacity to manage infectious outbreaks. UAE, Lebanon, and Canada ranked 3rd, 21st, and 31st, respectively, for outbound air travel volume from Iran in February 2019. We estimated that 18,300 (95% confidence interval: 3770 to 53,470) COVID-19 cases would have had to occur in Iran, assuming an outbreak duration of 1.5 months in the country, in order to observe these three internationally exported cases reported at the time of writing.  Results were robust under varying assumptions about undiagnosed case numbers in Syria, Azerbaijan and Iraq.  Even if it were assumed that all cases were identified in all countries with certainty, the "best case" outbreak size was substantial (1820, 95% CI: 380-5320 cases), and far higher than reported case counts.  Given the low volumes of air travel to countries with identified cases of COVID-19 with origin in Iran (such as Canada), it is likely that Iran is currently experiencing a COVID-19 epidemic of significant size for such exportations to be occurring. This is concerning, both for public health in Iran itself, and because of the high likelihood for outward dissemination of the epidemic to neighbouring countries with lower capacity to respond to infectious diseases epidemics.</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Domestic Poultry and SARS Coronavirus, Southern China</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 May</span></i><p style="text-align: justify; text-justify: inter-word;">SARS coronavirus injected intratracheally into chickens, turkeys, geese, ducks, and quail, or into the allantoic sac of their embryonating eggs, failed to cause disease or replicate. This finding suggests that domestic poultry were unlikely to have been the reservoir, or associated with dissemination, of SARS coronavirus in the animal markets of southern China.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic and Clinical Characteristics of 91 Hospitalized Patients with COVID-19 in Zhejiang, China: A retrospective, multi-centre case series</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background Recent studies have focused initial clinical and Epidemiologic characteristics on the COVID-19, mainly revealing situation in Wuhan, Hubei.  Aim To reveal more data on the epidemiologic and clinical characteristics of COVID-19 patients outside of Wuhan, in Zhejiang, China.  Design  Retrospective case series.  Methods 88 cases of laboratory-confirmed and 3 cases of clinical-confirmed COVID-19 were admitted to five hospitals in Zhejiang province, China. Data were collected from 20 January 2020 to 11 February 2020.  Results  Of all 91 patients, 88 (96.70%) were laboratory-confirmed COVID-19 with throat swab samples that tested positive for SARS-Cov-2 while 3 (3.30%) were clinical-diagnosed COVID-19 cases. The median age of the patients was 50 (36.5-57) years, and female accounted for 59.34%. In this sample 40 (43.96%) patients had contracted the diseases from local cases, 31 (34.07%) patients had been to Wuhan/Hubei, 8 (8.79%) cases had contacted with people from Wuhan, 11 (12.09%) cases were confirmed aircraft transmission. In particular within the city of Ningbo, 60.52% cases can be traced back to an event held in a temple. The most common symptoms were fever (71.43%), cough (60.44%) and fatigue (43.96%). The median of incubation period was 6 (IQR, 3-8) days and the median time from first visit to a doctor to confirmed diagnosis was 1 (1-2) days. According to the Chest computed tomography scans, 67.03% cases had bilateral pneumonia.  Conclusions Social activity cluster, family cluster and travel by airplane were how COVID-19 patients get transmitted and could be rapidly diagnosed COVID-19 in Zhejiang.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Susceptibility of Aedes albopictus C6/36 cells to viral infection.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The susceptibility of the C6/36 clone of Aedes albopictus monolayer cell cultures was determined with 46 prototype viruses passed through three subcultures. Viral growth was confirmed by titration of the passage material in other susceptible host systems. Nineteen viruses demonstrated good growth in C6/36 cells: coxsackievirus group A type 10 and group B types 2, 3, 4, and 5; enterovirus 69; mumps virus; poliovirus types 1 to 3; reovirus types 1 to 3; vaccinia virus; dengue virus type 2; eastern equine encephalomyelitis virus; La Crosse virus; Rocio virus; and St. Louis encephalitis virus. Ten viruses did not adapt to growth in the C6/36 cultures. Seventeen other virus strains displayed only limited growth which was primarily restricted to the initial C6/36 passage or was detected by hemagglutinin reactions without observable cell degeneration. Of the 46 viruses, 33 (72%) were capable of initiating infection with a demonstrable cytopathic effect in the initial C6/36 passage. Hemagglutination or complement fixation titers or both were obtained with dengue virus type 2, eastern equine encephalomyelitis virus, La Crosse virus, mumps virus, reovirus types 1 to 3, and Rocio, St. Louis encephalitis, and vaccinia viruses.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Replication of MERS and SARS coronaviruses in bat cells offers insights to their ancestral origins</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-05-20</span></i><p style="text-align: justify; text-justify: inter-word;">Previous findings of Middle East Respiratory Syndrome coronavirus (MERS-CoV)-related viruses in bats, and the ability of Tylonycteris-BatCoV HKU4 spike protein to utilize MERS-CoV receptor, human dipeptidyl peptidase 4 hDPP4, suggest a bat ancestral origin of MERS-CoV. We developed 12 primary bat cell lines from seven bat species, including Tylonycteris pachypus, Pipistrellus abramus and Rhinolophus sinicus (hosts of Tylonycteris-BatCoV HKU4, Pipistrellus-BatCoV HKU5 and SARS-related-CoV respectively), and tested their susceptibilities to MERS-CoVs, SARS-CoV and human coronavirus 229E (HCoV-229E). Five cell lines, including P. abramus and R. sinicus but not T. pachypus cells, were susceptible to human MERS-CoV EMC/2012. However, three tested camel MERS-CoV strains showed different infectivities, with only two strains capable of infecting three and one cell lines respectively. SARS-CoV can only replicate in R. sinicus cells, while HCoV-229E cannot replicate in any bat cells. Bat dipeptidyl peptidase 4 (DPP4) sequences were closely related to those of human and non-human primates but distinct from dromedary DPP4 sequence. Critical residues for binding to MERS-CoV spike protein were mostly conserved in bat DPP4. DPP4 was expressed in the five bat cells susceptible to MERS-CoV, with significantly higher mRNA expression levels than those in non-susceptible cells (P=0.0174), supporting that DPP4 expression is critical for MERS-CoV infection in bats. However, overexpression of T. pachypus DPP4 failed to confer MERS-CoV susceptibility in T. pachypus cells, suggesting other cellular factors in determining viral replication. The broad cellular tropism of MERS-CoV should prompt further exploration of host diversity of related viruses to identify its ancestral origin.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Cotton Rats as a Model for Severe Acute Respiratory Syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Experimental studies were conducted to evaluate two species of cotton rats, Sigmodon hispidus and Sigmodon fulviventer, as a model for severe acute respiratory syndrome (SARS). Blood and turbinate wash samples, and lung tissue were collected from each animal at different time points after SARS coronavirus (CoV) infection for determining the growth curve of virus, if any, by the standard infectivity assay in Vero E6 cells. In addition, sections of the lung, liver, spleen, and kidney were taken and used for histology analysis. All animals were observed daily for signs of illness, and in some experiments, animals were weighed on the day when they were sacrificed. The results indicated that the cotton rat species, S. hispidus and S. fulviventer, were not a useful model for either SARS-CoV infection or disease. This observation was supported by the absence of any signs of illness, the failure to consistently demonstrate virus in the blood and tissues, and the absent of any notable histopathology. However, infected animals were capable of producing neuralizing antibodies against SARS-CoV, suggesting the serocon-version did occur. Further studies are warranted to consider other animal species in efforts to find better animal models for the evaluation of SARS-CoV vaccines and antiviral drugs. Key Words: Cotton rats—SARS—Model animal.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy.  Methods We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level.  Results   The median age of the cases was 44 years, with an increasing of cases in younger age groups and the elderly as the epidemic progressed. The delay from symptom onset to hospital admission decreased from 4.4 days (95%CI: 0.0-14.0) until January 27 to 2.6 days (0.0-9.0) from January 28 to February 17. The mean incubation period was estimated at 5.2 days (1.8-12.4) and the mean serial interval at 5.1 days (1.3-11.6). The epidemic dynamics in provinces outside Hubei was highly variable, but consistently included a mix of case importations and local transmission. We estimate that the epidemic was self-sustained for less than three weeks with Rt reaching peaks between 1.40 (1.04-1.85) in Shenzhen City of Guangdong Province and 2.17 (1.69-2.76) in Shandong Province. In all the analyzed locations (n=10) Rt was estimated to be below the epidemic threshold since the end of January.  Conclusion   Our findings suggest that the strict containment measures and movement restrictions in place may contribute to the interruption of local COVID-19 transmission outside Hubei Province. The shorter serial interval estimated here implies that transmissibility is not as high as initial estimates suggested.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>De-isolating COVID-19 Suspect Cases: A Continuing Challenge</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of 15 February 2020, Singapore had screened a total of 991 suspect cases for COVID-19, of which 72 cases tested positive, 812 cases tested negative, while the remaining 107 had pending results.(1) Besides optimising sample type to increase yield, (2) the challenge in clinical management of suspect cases lies in deciding whether they may be de-isolated or if further isolation and repeat testing is required. No single indicator may be effectively used to decide on de-isolation of a suspect case. In our series of positive cases, samples from one suspect case only returned positive on the fifth repeated sample (nasopharyngeal swab), on the seventh day of clinical illness. Current evidence suggests that transmission of COVID-19 may be possible even from asymptomatic contacts,(3) and polymerase chain reaction (PCR) testing may not return positive initially. (4) Our suspect case was kept isolated because of a high index of clinical suspicion, with a clinically compatible illness and history of close contact with a laboratory-proven COVID-19 case. While multiplex respiratory virus panels, in general, may be helpful in the evaluation of other viral acute respiratory infections (ARIs), even the detection of an alternate respiratory pathogen may not definitively exclude COVID-19 infection. Dual infections can occur in 10- 20% of viral acute respiratory infections, as has been reported with SARS-CoV and MERSCoV. (5) In our case series, one patient with confirmed COVID-19 by nasopharyngeal aspirate also exhibited clinical symptoms compatible with dengue fever. This was laboratory confirmed by dengue NS1 antigen test. (PL Lim, personal communication).</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Human, Nonhuman Primate, and Bat Cells Are Broadly Susceptible to Tibrovirus Particle Cell Entry</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 26</span></i><p style="text-align: justify; text-justify: inter-word;">In 2012, the genome of a novel rhabdovirus, Bas-Congo virus (BASV), was discovered in the acute-phase serum of a Congolese patient with presumed viral hemorrhagic fever. In the absence of a replicating virus isolate, fulfilling Koch’s postulates to determine whether BASV is indeed a human virus and/or pathogen has been impossible. However, experiments with vesiculoviral particles pseudotyped with Bas-Congo glycoprotein suggested that BASV particles can enter cells from multiple animals, including humans. In 2015, genomes of two related viruses, Ekpoma virus 1 (EKV-1) and Ekpoma virus 2 (EKV-2), were detected in human sera in Nigeria. Isolates could not be obtained. Phylogenetic analyses led to the classification of BASV, EKV-1, and EKV-2 in the same genus, Tibrovirus, together with five biting midge-borne rhabdoviruses [i.e., Beatrice Hill virus (BHV), Bivens Arm virus (BAV), Coastal Plains virus (CPV), Sweetwater Branch virus (SWBV), and Tibrogargan virus (TIBV)] not known to infect humans. Using individual recombinant vesiculoviruses expressing the glycoproteins of all eight known tibroviruses and more than 75 cell lines representing different animal species, we demonstrate that the glycoproteins of all tibroviruses can mediate vesiculovirus particle entry into human, bat, nonhuman primate, cotton rat, boa constrictor, and Asian tiger mosquito cells. Using four of five isolated authentic tibroviruses (i.e., BAV, CPV, SWBV, and TIBV), our experiments indicate that many cell types may be partially resistant to tibrovirus replication after virion cell entry. Consequently, experimental data solely obtained from experiments using tibrovirus surrogate systems (e.g., vesiculoviral pseudotypes, recombinant vesiculoviruses) cannot be used to predict whether BASV, or any other tibrovirus, infects humans.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of the first cluster of cases in a family of novel coronavirus pneumonia in Gansu Province</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The epidemiological history and clinical characteristics of 7 cases of COVID-19 and 1 case of close contact in the first family aggregation epidemic of COVID-19 in Gansu Province were analyzed. The first patient A developed on January 22, 2020, with a history of residence in Wuhan, and confirmed severe cases of NCP on January 24, 2020; patient B, on January 23, 2020, diagnosed on January 31, severe cases; patient C, asymptomatic, diagnosed on January 27; patient D, asymptomatic, diagnosed on January 27; patient E, on January 24, diagnosed on January 28; patient F, asymptomatic, diagnosed on January 31; Patient G was asymptomatic and was diagnosed on January 31. In close contact, H was asymptomatic, PCR test was negative and asymptomatic, and he was discharged early. Among the 7 patients, 1 case died of (B) aggravation, and the other patients' condition was effectively controlled after active treatment. Except for the discharged cases, 5 cases were positive for COVID-19 specific IgM antibody and 1 case was negative. In this clustering outbreak, 4 patients remained asymptomatic, but PCR and IgM antibodies were positive, indicating that asymptomatic patients may be the key point to control the epidemic. Specific IgM antibody screening for patients whose pharyngeal swab nucleic acid test is negative but with ground glass-like lung lesions is very important for early detection and early isolation.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 20</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 (Novel Coronavirus 2019) - recent trends</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The World Health Organization (WHO) has issued a warning that, although the 2019 novel coronavirus (COVID-19) from Wuhan City (China), is not pandemic, it should be contained to prevent the global spread. The COVID-19 virus was known earlier as 2019-nCoV. As of 12 February 2020, WHO reported 45,171 cases and 1115 deaths related to COVID-19. COVID-19 is similar to Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) virus in its pathogenicity, clinical spectrum, and epidemiology. Comparison of the genome sequences of COVID-19, SARS-CoV, and Middle East Respiratory Syndrome coronavirus (MERS-CoV) showed that COVID-19 has a better sequence identity with SARS-CoV compared to MERS CoV. However, the amino acid sequence of COVID-19 differs from other coronaviruses specifically in the regions of 1ab polyprotein and surface glycoprotein or S-protein. Although several animals have been speculated to be a reservoir for COVID-19, no animal reservoir has been already confirmed. COVID-19 causes COVID-19 disease that has similar symptoms as SARS-CoV. Studies suggest that the human receptor for COVID-19 may be angiotensin-converting enzyme 2 (ACE2) receptor similar to that of SARS-CoV. The nucleocapsid (N) protein of COVID-19 has nearly 90% amino acid sequence identity with SARS-CoV. The N protein antibodies of SARS-CoV may cross react with COVID-19 but may not provide cross-immunity. In a similar fashion to SARS-CoV, the N protein of COVID-19 may play an important role in suppressing the RNA interference (RNAi) to overcome the host defense. This mini-review aims at investigating the most recent trend of COVID-19.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Composition and divergence of coronavirus spike proteins and host ACE2 receptors predict potential intermediate hosts of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) crossed the species barriers to infect humans caused thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS-CoV-2) causes the outbreaks of Coronavirus Disease 2019 (COVID-19) was discovered. Until February 18, 2020, there are 72533 confirmed COVID-19 cases (including 10644 severe cases) and 1872 deaths in China. SARS-CoV-2 is surging in the public and caused substantial burdens due to its human-to-human transmission. However, the intermediate host of SARS-CoV-2 is still unclear. Finding the possible intermediate host of SARS-CoV-2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor binding domain (RBD) of coronavirus spike protein and the host receptor, Angiotensin Converting Enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that like previous suggested pangolins and snacks, the turtles (C. picta bellii, C. mydas, and P. sinensis) may act as the potential intermediate hosts transmitting SARS-CoV-2 to human. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Human, Nonhuman Primate, and Bat Cells Are Broadly Susceptible to Tibrovirus Particle Cell Entry</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-12-27</span></i><p style="text-align: justify; text-justify: inter-word;">In 2012, the genome of a novel rhabdovirus, Bas-Congo virus, was discovered in the acute-phase serum of a Congolese patient with presumed viral hemorrhagic fever. In the absence of a replicating virus isolate, fulfilling Kochs postulates to determine whether Bas-Congo virus is indeed a human virus and/or pathogen has been impossible. However, experiments with vesiculoviral particles pseudotyped with Bas-Congo glycoprotein suggested that Bas-Congo virus particles can enter cells from multiple animals, including humans. In 2015, genomes of two related viruses, Ekpoma virus 1 and Ekpoma virus 2, were detected in human sera in Nigeria. Isolates could not be obtained. Phylogenetic analyses led to the classification of Bas-Congo virus, Ekpoma virus 1, and Ekpoma virus 2 in the same genus, Tibrovirus, together with five biting midge-borne rhabdoviruses (i.e., Beatrice Hill virus, Bivens Arm virus, Coastal Plains virus, Sweetwater Branch virus, and Tibrogargan virus) not known to infect humans. Using individual recombinant vesiculoviruses expressing the glycoproteins of all eight known tibroviruses and more than 75 cell lines representing different animal species, we demonstrate that the glycoproteins of all tibroviruses can mediate vesiculovirus particle entry into human, bat, nonhuman primate, cotton rat, boa constrictor, and Asian tiger mosquito cells. Using four of five isolated authentic tibroviruses (i.e., Bivens Arm virus, Coastal Plains virus, Sweetwater Branch virus, and Tibrogargan virus), our experiments indicate that many cell types may be partially resistant to tibrovirus replication after virion cell entry. Consequently, experimental data solely obtained from experiments using tibrovirus surrogate systems (e.g., vesiculoviral pseudotypes, recombinant vesiculoviruses) cannot be used to predict whether Bas-Congo virus, or any other tibrovirus, infects humans.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Paramyxovirus Infections in Ex Vivo Lung Slice Cultures of Different Host Species</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Mar 27</span></i><p style="text-align: justify; text-justify: inter-word;">In vivo experiments in animal models of disease are of crucial importance for viral tropism and pathogenesis studies. However, these experiments must be complemented with in vitro and ex vivo experiments. Here, we describe a protocol for the preparation and ex vivo infection of lung slices from different mammalian host species with various respiratory paramyxoviruses expressing fluorescent reporter proteins, and suggest follow-up experiments including immunohistochemistry, flow cytometry and confocal microscopy.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>A Severe Acute Respiratory Syndrome-Associated Coronavirus-Specific Protein Enhances Virulence of an Attenuated Murine Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Most animal species that can be infected with the severe acute respiratory syndrome-associated coronavirus (SARS-CoV) do not reproducibly develop clinical disease, hindering studies of pathogenesis. To develop an alternative system for the study of SARS-CoV, we introduced individual SARS-CoV genes (open reading frames [ORFs]) into the genome of an attenuated murine coronavirus. One protein, the product of SARS-CoV ORF6, converted a sublethal infection to a uniformly lethal encephalitis and enhanced virus growth in tissue culture cells, indicating that SARS-CoV proteins function in the context of a heterologous coronavirus infection. Furthermore, these results suggest that the attenuated murine coronavirus lacks a virulence gene residing in SARS-CoV. Recombinant murine coronaviruses cause a reproducible and well-characterized clinical disease, offer virtually no risk to laboratory personnel, and should be useful for elucidating the role of SARS-CoV nonstructural proteins in viral replication and pathogenesis.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Viruses from animals</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Infections that cross over from other species are a deadly problem</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Replication of MERS and SARS coronaviruses in bat cells offers insights to their ancestral origins</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 10</span></i><p style="text-align: justify; text-justify: inter-word;">Previous findings of Middle East Respiratory Syndrome coronavirus (MERS-CoV)-related viruses in bats, and the ability of Tylonycteris-BatCoV HKU4 spike protein to utilize MERS-CoV receptor, human dipeptidyl peptidase 4 hDPP4, suggest a bat ancestral origin of MERS-CoV. We developed 12 primary bat cell lines from seven bat species, including Tylonycteris pachypus, Pipistrellus abramus and Rhinolophus sinicus (hosts of Tylonycteris-BatCoV HKU4, Pipistrellus-BatCoV HKU5, and SARS-related-CoV respectively), and tested their susceptibilities to MERS-CoVs, SARS-CoV, and human coronavirus 229E (HCoV-229E). Five cell lines, including P. abramus and R. sinicus but not T. pachypus cells, were susceptible to human MERS-CoV EMC/2012. However, three tested camel MERS-CoV strains showed different infectivities, with only two strains capable of infecting three and one cell lines respectively. SARS-CoV can only replicate in R. sinicus cells, while HCoV-229E cannot replicate in any bat cells. Bat dipeptidyl peptidase 4 (DPP4) sequences were closely related to those of human and non-human primates but distinct from dromedary DPP4 sequence. Critical residues for binding to MERS-CoV spike protein were mostly conserved in bat DPP4. DPP4 was expressed in the five bat cells susceptible to MERS-CoV, with significantly higher mRNA expression levels than those in non-susceptible cells (P = 0.0174), supporting that DPP4 expression is critical for MERS-CoV infection in bats. However, overexpression of T. pachypus DPP4 failed to confer MERS-CoV susceptibility in T. pachypus cells, suggesting other cellular factors in determining viral replication. The broad cellular tropism of MERS-CoV should prompt further exploration of host diversity of related viruses to identify its ancestral origin.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS–associated Coronavirus Replication in Cell Lines</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Jan</span></i><p style="text-align: justify; text-justify: inter-word;">Given the potential for laboratory-associated severe acute respiratory syndrome–associated coronavirus (SARS-CoV) infections, we must know which cell lines are susceptible to the virus. We investigated 21 cell lines routinely used for virus isolation or research. After infection with SARS-CoV, cells were observed for cytopathic effects, and quantitative real-time polymerase chain reaction was used to measure ongoing viral replication. An indirect immunofluorescence assay was also used as a confirmatory test. The study identified 10 new cell lines capable of supporting the replication of SARS-CoV and confirmed the susceptibility of 4 cell lines previously reported. This study shows that SARS-CoV can be isolated in several cell lines commonly used for diagnostic or research purposes. It also shows that SARS-CoV can achieve high titers in several cell lines, sometimes in the absence of specific cytopathic effects.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-22</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Replication and Shedding of MERS-CoV in Upper Respiratory Tract of Inoculated Dromedary Camels</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">In 2012, a novel coronavirus associated with severe respiratory disease in humans emerged in the Middle East. Epidemiologic investigations identified dromedary camels as the likely source of zoonotic transmission of Middle East respiratory syndrome coronavirus (MERS-CoV). Here we provide experimental support for camels as a reservoir for MERS-CoV. We inoculated 3 adult camels with a human isolate of MERS-CoV and a transient, primarily upper respiratory tract infection developed in each of the 3 animals. Clinical signs of the MERS-CoV infection were benign, but each of the camels shed large quantities of virus from the upper respiratory tract. We detected infectious virus in nasal secretions through 7 days postinoculation, and viral RNA up to 35 days postinoculation. The pattern of shedding and propensity for the upper respiratory tract infection in dromedary camels may help explain the lack of systemic illness among naturally infected camels and the means of efficient camel-to-camel and camel-to-human transmission.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Mouse Dipeptidyl Peptidase 4 Is Not a Functional Receptor for Middle East Respiratory Syndrome Coronavirus Infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human dipeptidyl peptidase 4 (hDPP4) was recently identified as the receptor for Middle East respiratory syndrome coronavirus (MERS-CoV) infection, suggesting that other mammalian DPP4 orthologs may also support infection. We demonstrate that mouse DPP4 cannot support MERS-CoV infection. However, employing mouse DPP4 as a scaffold, we identified two critical amino acids (A288L and T330R) that regulate species specificity in the mouse. This knowledge can support the rational design of a mouse-adapted MERS-CoV for rapid assessment of therapeutics.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Fruits of Virus Discovery: New Pathogens and New Experimental Models</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The advent of high-throughput sequencing has led to a tremendous increase in the rate of discovery of viral sequences. In some instances, novel pathogens have been identified. What has been less well appreciated is that novel virus discoveries in distinct hosts have led to the establishment of unique experimental systems to define host-virus interactions. These new systems have opened new frontiers in the study of fundamental virology and infectious disease.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The present outbreak of a coronavirus-associated acute respiratory disease called coronavirus disease 19 (COVID-19) is the third documented spillover of an animal coronavirus to humans in only two decades that has resulted in a major epidemic. The Coronaviridae Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the classification of viruses and taxon nomenclature of the family Coronaviridae, has assessed the placement of the human pathogen, tentatively named 2019-nCoV, within the Coronaviridae. Based on phylogeny, taxonomy and established practice, the CSG recognizes this virus as forming a sister clade to the prototype human and bat severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus, and designates it as SARS-CoV-2. In order to facilitate communication, the CSG proposes to use the following naming convention for individual isolates: SARS-CoV-2/host/location/isolate/date. While the full spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined, the independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying viruses at the species level to complement research focused on individual pathogenic viruses of immediate significance. This will improve our understanding of virus–host interactions in an ever-changing environment and enhance our preparedness for future outbreaks.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-14</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore. We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% - 64%) of Singapore′s capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 - 4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% -  67%), among intermediate surveillance locations it is 37% (95% HPDI 18% - 68%), and among low surveillance locations it is 11% (95% HPDI 0% - 42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers may be underestimated by several fold, and severity correspondingly overestimated by several fold. Undetected cases are likely in countries around the world, with greater risk in countries of low detection capacity and high connectivity to the epicenter of the outbreak.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>MicroRNA-based strategy to mitigate the risk of gain-of-function influenza studies</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Sep 11</span></i><p style="text-align: justify; text-justify: inter-word;">Recent gain-of-function studies in influenza A virus H5N1 strains revealed that as few as three amino-acid changes in the hemagglutinin protein confer the capacity for viral transmission between ferrets(1, 2). As transmission between ferrets is considered a surrogate indicator of transmissibility between humans, these studies raised concerns about the risks of gain-of-function influenza A virus research. Here we present an approach to strengthen the biosafety of gain-of-function influenza experiments. We exploit species-specific endogenous small RNAs to restrict influenza A virus tropism. In particular, we found that the microRNA miR-192 was expressed in primary human respiratory tract epithelial cells as well as mouse lungs but absent from the ferret respiratory tract. Incorporation of miR-192 target sites into influenza A virus did not prevent influenza replication and transmissibility in ferrets, but did attenuate influenza pathogenicity in mice. This molecular biocontainment approach should be applicable beyond influenza A virus to minimize the risk of experiments involving other pathogenic viruses.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Lessons from SARS and MERS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Within weeks of the first cases, a series of papers were released detailing the epidemiology of the disease (now termed COVID‐19) [1-3]. By early January 2020 the virus was identified and the sequence determined. The virus (termed SARS‐CoV‐2) shares 88% sequence identity to two coronaviruses found in bats, bat‐SLCoVZC45 and bat‐SL‐CoVZXC21, 79% identity with the Severe Acute Respiratory Syndrome (SARS) coronavirus and 50% identity with Middle Eastern Respiratory Syndrome (MERS) coronavirus [4]. From the first cohort of patients, 8 complete genomes were 99.9% identical in sequence. Given that the typical RNA coronavirus evolves at a rate of 104 nucleotide substitutions per year, this suggests a recent single source emergence in early December or late November 2019 [4]. SARS‐CoV‐2 is thought to be transmitted via contaminated hands, surfaces and aerosolised droplets; extensive human‐to‐human transmission is evident, with clusters of infected families and medical staff [5]. The number of confirmed cases has increased rapidly, at a rate that far outstripped the rate of rise of cases of SARS in 2002/3, raising serious global health concerns. By the 21st January, COVID‐19 cases were widespread across mainland China, soon spreading beyond the Chinese borders.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-Like Coronavirus WIV1-CoV Does Not Replicate in Egyptian Fruit Bats (Rousettus aegyptiacus)</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Dec 19</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS)-like WIV1-coronavirus (CoV) was first isolated from Rhinolophus sinicus bats and can use the human angiotensin converting enzyme 2 (ACE2) receptor. In the current study, we investigate the ability of WIV1-CoV to infect Rousettus aegyptiacus bats. No clinical signs were observed throughout the experiment. Furthermore, only four oropharyngeal swabs and two respiratory tissues, isolated on day 3 post inoculation, were found positive for viral RNA. Two out of twelve bats showed a modest increase in coronavirus specific antibodies post challenge. In conclusion, WIV1-CoV was unable to cause a robust infection in Rousettus aegyptiacus bats.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. METHODS: Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. RESULTS: The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. CONCLUSION: SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunology of Bats and Their Viruses: Challenges and Opportunities</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Dec 8</span></i><p style="text-align: justify; text-justify: inter-word;">Bats are reservoir hosts of several high-impact viruses that cause significant human diseases, including Nipah virus, Marburg virus and rabies virus. They also harbor many other viruses that are thought to have caused disease in humans after spillover into intermediate hosts, including SARS and MERS coronaviruses. As is usual with reservoir hosts, these viruses apparently cause little or no pathology in bats. Despite the importance of bats as reservoir hosts of zoonotic and potentially zoonotic agents, virtually nothing is known about the host/virus relationships; principally because few colonies of bats are available for experimental infections, a lack of reagents, methods and expertise for studying bat antiviral responses and immunology, and the difficulty of conducting meaningful field work. These challenges can be addressed, in part, with new technologies that are species-independent that can provide insight into the interactions of bats and viruses, which should clarify how the viruses persist in nature, and what risk factors might facilitate transmission to humans and livestock.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objectives Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. Methods Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. Results The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. Conclusion SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The ongoing outbreak of novel corona virus disease 2019 (COVID-19) in Wuhan, China, is arousing international concern. This study evaluated whether and when the infected but asymptomatic cases during the incubation period could infect others.  Methods: We collected data on demographic characteristics, exposure history, and symptom onset day of the confirmed cases, which had been announced by the Chinese local authorities. We evaluated the potential of transmission during the incubation period in 50 infection clusters, including 124 cases. All the secondary cases had a history of contact with their first-generation cases prior to symptom onset.  Results: The estimated mean incubation period for COVID-19 was 4.9 days (95% confidence interval [CI], 4.4 to 5.4) days, ranging from 0.8 to 11.1 days (2.5th to 97.5th percentile). The observed mean and standard deviation (SD) of serial interval was 4.1±3.3 days, with the 2.5th and 97.5th percentiles at -1 and 13 days. The infectious curve showed that in 73.0% of the secondary cases, their date of getting infected was before symptom onset of the first-generation cases, particularly in the last three days of the incubation period.  Conclusions: The results indicated the transmission of COVID-9 occurs among close contacts during the incubation period, which may lead to a quarantine loophole. Strong and effective countermeasures should be implemented to prevent or mitigate asymptomatic transmission during the incubation period in populations at high risk.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission interval estimates suggest pre-symptomatic spread of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods:  We collected outbreak information from Singapore and Tianjin,  China,  reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring.  Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Liver injury in COVID-19: management and challenges</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 shares 82% genome sequence similarity to SARS-CoV and 50% genome sequence homology to Middle East respiratory syndrome coronavirus (MERS-CoV)—all three coronaviruses are known to cause severe respiratory symptoms. Liver impairment has been reported in up to 60% of patients with SARS3 and has also been reported in patients infected with MERS-CoV.4 At least seven relatively large-scale case studies have reported the clinical features of patients with COVID-19.1 , 5 , 6 , 7 , 8 , 9 , 10 In this Comment, we assess how the liver is affected using the available case studies and data from The Fifth Medical Center of PLS General Hospital, Beijing, China. These data indicate that 2–11% of patients with COVID-19 had liver comorbidities and 14–53% cases reported abnormal levels of alanine aminotransferase and aspartate aminotransferase (AST) during disease progression (table). Patients with severe COVID-19 seem to have higher rates of liver dysfunction. In a study in The Lancet by Huang and colleagues,5 elevation of AST was observed in eight (62%) of 13 patients in the intensive care unit (ICU) compared with seven (25%) of 28 patients who did not require care in the ICU. Moreover, in a large cohort including 1099 patients from 552 hospitals in 31 provinces or provincial municipalities, more severe patients with disease had abnormal liver aminotransferase levels than did non-severe patients with disease.1 Furthermore, in another study,8 patients who had a diagnosis of COVID-19 confirmed by CT scan while in the subclinical phase (ie, before symptom onset) had significantly lower incidence of AST abnormality than did patients diagnosed after the onset of symptoms. Therefore, liver injury is more prevalent in severe cases than in mild cases of COVID-19.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Is COVID-19 Receiving ADE From Other Coronaviruses?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">One of the most perplexing questions regarding the current COVID-19 coronavirus epidemic is the discrepancy between the severity of cases observed in the Hubei province of China and those occurring elsewhere in the world. One possible answer is antibody dependent enhancement (ADE) of SARS-CoV-2 due to prior exposure to other coronaviruses. ADE modulates the immune response and can elicit sustained inflammation, lymphopenia, and/or cytokine storm, one or all of which have been documented in severe cases and deaths. ADE also requires prior exposure to similar antigenic epitopes, presumably circulating in local viruses, making it a possible explanation for the observed geographic limitation of severe cases and deaths.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical characteristics and imaging manifestations of the 2019 novel coronavirus disease (COVID-19):A multi-center study in Wenzhou city, Zhejiang, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background Little is known about COVID-19 outside Hubei. The aim of this paper was to describe the clinical characteristics and imaging manifestations of hospitalized patients with confirmed COVID-19 infection in Wenzhou, Zhejiang, China. Methods In this retrospective cohort study, 149 RT-PCR confirmed positive patients were consecutively enrolled from January 17th to February 10th, 2020 in three tertiary hospitals of Wenzhou. Outcomes were followed up until Feb 15th, 2020. Findings A total of 85 patients had Hubei travel/residence history, while another 49 had contact with people from Hubei and 15 had no traceable exposure history to Hubei. Fever, cough and expectoration were the most common symptoms, 14 patients had decreased oxygen saturation, 33 had leukopenia, 53 had lymphopenia, and 82 had elevated C reactive protein. On chest computed tomography, lung segments 6 and 10 were mostly involved. A total of 287 segments presented ground glass opacity, 637 presented mixed opacity and 170 presented consolidation. Lesions were more localized in the peripheral lung with a patchy form. No significant difference was found between patients with or without Hubei exposure history. Seventeen patients had normal CT on admission of these, 12 had negative findings even10 days later. Interpretation Most patients presented with a mild infection in our study. The imaging pattern of multifocal peripheral ground glass or mixed opacity with predominance in the lower lung is highly suspicious of COVID-19 in the first week of disease onset. Nevetheless, some patients can present with a normal chest finding despite testing positive for COVID-19. Funding: We did not receive any fundings.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Enhanced Growth of a Murine Coronavirus in Transformed Mouse Cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Plaque formation by A59 virus, a murine coronavirus, was facilitated in AL/N and Balb mouse cells transformed by polyoma virus, simian virus 40, murine sarcoma virus, or mammary tumor virus. In these virus-transformed cells, A59 virus plaques were larger, they appeared earlier, and plaquing efficiencies were higher than in normal, untransformed cells. “Spontaneously” transformed AL/N cells behaved similarly to untransformed cells, whereas “spontaneously” transformed Balb cells resembled virus-transformed cell hosts. Both untransformed and transformed AL/N and Balb cells were permissive hosts for A59 virus. However, multiplication of A59 virus was enhanced at least fivefold in the virus-transformed AL/N cell hosts. Larger differences (100-fold or greater) in A59 virus production were obtained during the first cycle of infection in Balb cells at low multiplicities and in AL/N cells after multiple cycles of virus growth. In virus-transformed and “spontaneously” transformed Balb cells, A59 virus induced extensive syncytia formation.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of the first cluster of cases in a family of novel coronavirus pneumonia in Gansu Province</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The epidemiological history and clinical characteristics of 7 cases of COVID-19 and 1 case of close contact in the first family aggregation epidemic of COVID-19 in Gansu Province were analyzed. The first patient A developed on January 22, 2020, with a history of residence in Wuhan, and confirmed severe cases of NCP on January 24, 2020; patient B, on January 23, 2020, diagnosed on January 31, severe cases; patient C, asymptomatic, diagnosed on January 27; patient D, asymptomatic, diagnosed on January 27; patient E, on January 24, diagnosed on January 28; patient F, asymptomatic, diagnosed on January 31; Patient G was asymptomatic and was diagnosed on January 31. In close contact, H was asymptomatic, PCR test was negative and asymptomatic, and he was discharged early. Among the 7 patients, 1 case died of (B) aggravation, and the other patients' condition was effectively controlled after active treatment. Except for the discharged cases, 5 cases were positive for COVID-19 specific IgM antibody and 1 case was negative. In this clustering outbreak, 4 patients remained asymptomatic, but PCR and IgM antibodies were positive, indicating that asymptomatic patients may be the key point to control the epidemic. Specific IgM antibody screening for patients whose pharyngeal swab nucleic acid test is negative but with ground glass-like lung lesions is very important for early detection and early isolation.;</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>POXVIRUS TROPISM</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Despite the success of the WHO-led smallpox eradication programme a quarter of a century ago, there remains considerable fear that variola virus, or other related pathogenic poxviruses such as monkeypox, could re-emerge and spread disease in the human population. Even today, we are still mostly ignorant about why most poxvirus infections of vertebrate hosts show strict species specificity, or how zoonotic poxvirus infections occur when poxviruses occasionally leap into novel host species. Poxvirus tropism at the cellular level seems to be regulated by intracellular events downstream of virus binding and entry, rather than at the level of specific host receptors as is the case for many other viruses. This review summarizes our current understanding of poxvirus tropism and host range, and discusses the prospects of exploiting host-restricted poxvirus vectors for vaccines, gene therapy or tissue-targeted oncolytic viral therapies for the treatment of human cancers.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Influenza as a model system for studying the cross-species transfer and evolution of the SARS coronavirus.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus (SARS-CoV) moved into humans from a reservoir species and subsequently caused an epidemic in its new host. We know little about the processes that allowed the cross-species transfer of this previously unknown virus. I discuss what we have learned about the movement of viruses into humans from studies of influenza A, both how it crossed from birds to humans and how it subsequently evolved within the human population. Starting with a brief review of severe acute respiratory syndrome to highlight the kinds of problems we face in learning about this viral disease, I then turn to influenza A, focusing on three topics. First, I present a reanalysis of data used to test the hypothesis that swine served as a "mixing vessel" or intermediate host in the transmission of avian influenza to humans during the 1918 "Spanish flu" pandemic. Second, I review studies of archived viruses from the three recent influenza pandemics. Third, I discuss current limitations in using molecular data to study the evolution of infectious disease. Although influenza A and SARS-CoV differ in many ways, our knowledge of influenza A may provide important clues about what limits or favours cross-species transfers and subsequent epidemics of newly emerging pathogens.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>The serial interval of COVID-19 from publicly reported confirmed cases</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.1% of reports indicating pre-symptomatic transmission.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Unique epidemiological and clinical features of the emerging 2019 novel coronavirus pneumonia (COVID-19) implicate special control measures</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract By Feb 27th, 2020, the outbreak of COVID-19 caused 82623 confirmed cases and 2858 deaths globally, more than Severe Acute Respiratory Syndrome (SARS) (8273 cases, 775 deaths) and Middle East Respiratory Syndrome (MERS) (1139 cases, 431 deaths) caused in 2003 and 2013 respectively. COVID-19 has spread to 46 countries internationally. Total fatality rate of COVID-19 is estimated at 3.46% by far based on published data from Chinese Center for Disease Control and Prevention (China CDC). Average incubation period of COVID-19 is around 6.4 days, ranges from 0-24 days. The basic reproductive number (R0) of COVID-19 ranges from 2-3.5 at the early phase regardless of different prediction models, which is higher than SARS and MERS. A study from China CDC showed majority of patients (80.9%) were considered asymptomatic or mild pneumonia but released large amounts of viruses at the early phase of infection, which posed enormous challenges for containing the spread of COVID-19. Nosocomial transmission was another severe problem. 3019 health workers were infected by Feb 12, 2020, which accounted for 3.83% of total number of infections, and extremely burdened the health system, especially in Wuhan. Limited epidemiological and clinical data suggest that the disease spectrum of COVID-19 may differ from SARS or MERS. We summarize latest literatures on genetic, epidemiological, and clinical features of COVID-19 in comparison to SARS and MERS and emphasize special measures on diagnosis and potential interventions. This review will improve our understanding of the unique features of COVID-19 and enhance our control measures in the future. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Nov 28</span></i><p style="text-align: justify; text-justify: inter-word;">The 2002–3 pandemic caused by severe acute respiratory syndrome coronavirus (SARS-CoV) was one of the most significant public health events in recent history(1). An ongoing outbreak of Middle East respiratory syndrome coronavirus (MERS-CoV)(2) suggests that this group of viruses remains a major threat and that their distribution is wider than previously recognized. Although bats have been suggested as the natural reservoirs of both viruses(3–5), attempts to isolate the progenitor virus of SARS-CoV from bats have been unsuccessful. Diverse SARS-like coronaviruses (SL-CoVs) have now been reported from bats in China, Europe and Africa(5–8), but none are considered a direct progenitor of SARS-CoV because of their phylogenetic disparity from this virus and the inability of their spike proteins (S) to use the SARS-CoV cellular receptor molecule, the human angiotensin converting enzyme II (ACE2)(9,10). Here, we report whole genome sequences of two novel bat CoVs from Chinese horseshoe bats (Family: Rhinolophidae) in Yunnan, China; RsSHC014 and Rs3367. These viruses are far more closely related to SARS-CoV than any previously identified bat CoVs, particularly in the receptor binding domain (RDB) of the S protein. Most importantly, we report the first recorded isolation of a live SL-CoV (bat SL-CoV-WIV1) from bat fecal samples in Vero E6 cells, which has typical coronavirus morphology, 99.9% sequence identity to Rs3367 and uses the ACE2s from human, civet and Chinese horseshoe bat for cell entry. Preliminary in vitro testing indicates that WIV1 also has a broad species tropism. Our results provide the strongest evidence to date that Chinese horseshoe bats are natural reservoirs of SARS-CoV, and that intermediate hosts may not be necessary for direct human infection by some bat SL-CoVs. They also highlight the importance of pathogen discovery programs targeting high-risk wildlife groups in emerging disease hotspots as a strategy for pandemic preparedness.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>It Takes a Community to Raise the Prevalence of a Zoonotic Pathogen</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Nov 21</span></i><p style="text-align: justify; text-justify: inter-word;">By definition, zoonotic pathogens are not strict host-species specialists in that they infect humans and at least one nonhuman reservoir species. The majority of zoonotic pathogens infect and are amplified by multiple vertebrate species in nature, each of which has a quantitatively different impact on the distribution and abundance of the pathogen and thus on disease risk. Unfortunately, when new zoonotic pathogens emerge, the dominant response by public health scientists is to search for a few, or even the single, most important reservoirs and to ignore other species that might strongly influence transmission. This focus on the single “primary” reservoir host species can delay biological understanding, and potentially public health interventions as species important in either amplifying or regulating the pathogen are overlooked. Investigating the evolutionary and ecological strategy of newly discovered or emerging pathogens within the community of potential and actual host species will be fruitful to both biological understanding and public health.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemic doubling time of the COVID-19 epidemic by Chinese province</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 epidemic doubling time by Chinese province was increasing from January 20 through February 9, 2020. Yet, the harmonic mean doubling time was relatively short, ranging from 1.4 (Hunan, 95% CI, 1.2-2.0) to 3.0 (Xinjiang, 95% CI, 2.0-4.9) days, with an estimate of 2.5 days (95% CI, 2.4-2.7) for Hubei.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential association between COVID-19 mortality and health-care resource availability</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As recorded by the Chinese Center for Disease Control and Prevention (China CDC), by Feb 16, 2020, there had been 70 641 confirmed cases and 1772 deaths due to COVID-19, with an average mortality of about 2·5%. However, in-depth analysis of these data show clear disparities in mortality rates between Wuhan (>3%), different regions of Hubei (about 2·9% on average), and across the other provinces of China (about 0·7% on average). We postulate that this is likely to be related to the rapid escalation in the number of infections around the epicentre of the outbreak, which has resulted in an insufficiency of health-care resources, thereby negatively affecting patient outcomes in Hubei, while this has not yet been the situation for the other parts of China (figure A, B). If we assume that average levels of health care are similar throughout China, higher numbers of infections in a given population can be considered an indirect indicator of a heavier health-care burden. Plotting mortality against the incidence of COVID-19 (cumulative number of confirmed cases since the start of the outbreak, per 10 000 population) showed a significant positive correlation (figure C), suggesting that mortality is correlated with health-care burden.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Spread and control of COVID-19 in China and their associations with population movement, public health emergency measures, and medical resources</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>ABSTRACT  BACKGROUND The COVID-19 epidemic, first emerged in Wuhan during December 2019, has spread globally. While the mass population movement for Chinese New Year has significantly influenced spreading the disease, little direct evidence exists about the relevance to epidemic and its control of population movement from Wuhan, local emergency response, and medical resources in China.  METHODS Spearman's correlation analysis was performed between official data of confirmed COVID-19 cases from Jan 20th to Feb 19th, 2020 and real-time travel data and health resources data.   RESULTS There were 74,675 confirmed COVID-19 cases in China by Feb 19th, 2020. The overall fatality rate was 2.84%, much higher in Hubei than in other regions (3.27% vs 0.73%). The index of population inflow from Hubei was positively correlated with total (Provincial r=0.9159, p&lt;0.001; City r=0.6311, p&lt;0.001) and primary cases (Provincial r=0.8702, p&lt;0.001; City r=0.6358, p&lt;0.001). The local health emergency measures (eg, city lockdown and traffic control) were associated with reduced infections nationwide. Moreover, the number of public health employees per capita was inversely correlated with total cases (r=-0.6295, p&lt;0.001) and infection rates (r=-0.4912, p&lt;0.01). Similarly, cities with less medical resources had higher fatality (r=-0.4791, p&lt;0.01) and lower cure rates (r=0.5286, p&lt;0.01) among the confirmed cases.  CONCLUSIONS The spread of the COVID-19 in China in its early phase was attributed primarily to population movement from Hubei, and effective governmental health emergency measures and adequate medical resources played important roles in subsequent control of epidemic and improved prognosis of affected individuals.</jats:p></p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 outbreak on the Diamond Princess cruise ship: estimating the epidemic potential and effectiveness of public health countermeasures</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Cruise ships carry a large number of people in confined spaces with relative homogeneous mixing. On 3 February, 2020, an outbreak of COVID-19 on cruise ship Diamond Princess was reported with 10 initial cases, following an index case on board around 21-25 January. By 4 February, public health measures such as removal and isolation of ill passengers and quarantine of non-ill passengers were implemented. By 20 February, 619 of 3,700 passengers and crew (17%) were tested positive.We estimated the basic reproduction number from the initial period of the outbreak using (SEIR) models. We calibrated the models with transient functions of countermeasures to incidence data. We additionally estimated a counterfactual scenario in absence of countermeasures, and established a model stratified by crew and guests to study the impact of differential contact rates among the groups. We also compared scenarios of an earlier versus later evacuation of the ship.The basic reproduction rate was initially 4 times higher on-board compared to the ${R}_0$ in the epicentre in Wuhan, but the countermeasures lowered it substantially. Based on the modeled initial ${R}_0$ of 14.8, we estimated that without any interventions within the time period of 21 January to 19 February, 2920 out of the 3700 (79%) would have been infected. Isolation and quarantine therefore prevented 2307 cases, and lowered the ${R}_0$ to 1.78. We showed that an early evacuation of all passengers on 3 February would have been associated with 76 infected persons in their incubation time.The cruise ship conditions clearly amplified an already highly transmissible disease. The public health measures prevented more than 2000 additional cases compared to no interventions. However, evacuating all passengers and crew early on in the outbreak would have prevented many more passengers and crew from infection.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Nov 30</span></i><p style="text-align: justify; text-justify: inter-word;">A large number of SARS-related coronaviruses (SARSr-CoV) have been detected in horseshoe bats since 2005 in different areas of China. However, these bat SARSr-CoVs show sequence differences from SARS coronavirus (SARS-CoV) in different genes (S, ORF8, ORF3, etc) and are considered unlikely to represent the direct progenitor of SARS-CoV. Herein, we report the findings of our 5-year surveillance of SARSr-CoVs in a cave inhabited by multiple species of horseshoe bats in Yunnan Province, China. The full-length genomes of 11 newly discovered SARSr-CoV strains, together with our previous findings, reveals that the SARSr-CoVs circulating in this single location are highly diverse in the S gene, ORF3 and ORF8. Importantly, strains with high genetic similarity to SARS-CoV in the hypervariable N-terminal domain (NTD) and receptor-binding domain (RBD) of the S1 gene, the ORF3 and ORF8 region, respectively, were all discovered in this cave. In addition, we report the first discovery of bat SARSr-CoVs highly similar to human SARS-CoV in ORF3b and in the split ORF8a and 8b. Moreover, SARSr-CoV strains from this cave were more closely related to SARS-CoV in the non-structural protein genes ORF1a and 1b compared with those detected elsewhere. Recombination analysis shows evidence of frequent recombination events within the S gene and around the ORF8 between these SARSr-CoVs. We hypothesize that the direct progenitor of SARS-CoV may have originated after sequential recombination events between the precursors of these SARSr-CoVs. Cell entry studies demonstrated that three newly identified SARSr-CoVs with different S protein sequences are all able to use human ACE2 as the receptor, further exhibiting the close relationship between strains in this cave and SARS-CoV. This work provides new insights into the origin and evolution of SARS-CoV and highlights the necessity of preparedness for future emergence of SARS-like diseases.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>The 2019-nCoV epidemic control strategies and future challenges of building healthy smart cities</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">reported and confirmed on 31 December 2019, in Wuhan, Hubei Province, China, which is one of China’s largest cities and a major domestic transport hub (located in the central part of China, as shown in Figure 1).1 The epidemic is attracting worldwide concern due to its rapid spread and transmission rate between humans. On 30 January 2020, the International Health Regulations, Emergency Committee of the World Health Organization (WHO) declared the outbreak – a ‘public health emergency of international concern’.2 On 8 February 2020 (24:00 GMT?+?8), there were 37,198 confirmed infections in China (including 811 deaths with a death ratio of 2.1%; and 6188 patients were confirmed in serious condition and 28,942 suspected cases).3 The COVID 19 infections were also reported in 26 other countries on 7 February 2020, including Canada, the USA, Australia, India, Sri-Lanka, Cambodia, Thailand, Vietnam, Malaysia, Singapore, Taiwan, the Republic of Korea, Sri Lanka, Japan, Philippines, Nepal, United Arab Emirates, Russia, Italy, Germany, Sweden, Finland, Belgium, Spain, France and the UK.4</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2.  Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Kolente virus, a rhabdovirus species isolated from ticks and bats in the Republic of Guinea</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Kolente virus (KOLEV) is a rhabdovirus originally isolated from ticks and a bat in Guinea, West Africa, in 1985. Although tests at the time of isolation suggested that KOLEV is a novel rhabdovirus, it has remained largely uncharacterized. We assembled the complete genome sequence of the prototype strain DakAr K7292, which was found to encode the five canonical rhabdovirus structural proteins (N, P, M, G and L) with alternative ORFs (>180 nt) in the P and L genes. Serologically, KOLEV exhibited a weak antigenic relationship with Barur and Fukuoka viruses in the Kern Canyon group. Phylogenetic analysis revealed that KOLEV represents a distinct and divergent lineage that shows no clear relationship to any rhabdovirus except Oita virus, although with limited phylogenetic resolution. In summary, KOLEV represents a novel species in the family Rhabdoviridae.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Zoonotic Potential of Simian Arteriviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Wild nonhuman primates are immediate sources and long-term reservoirs of human pathogens. However, ethical and technical challenges have hampered the identification of novel blood-borne pathogens in these animals. We recently examined RNA viruses in plasma from wild African monkeys and discovered several novel, highly divergent viruses belonging to the family Arteriviridae. Close relatives of these viruses, including simian hemorrhagic fever virus, have caused sporadic outbreaks of viral hemorrhagic fever in captive macaque monkeys since the 1960s. However, arterivirus infection in wild nonhuman primates had not been described prior to 2011. The arteriviruses recently identified in wild monkeys have high sequence and host species diversity, maintain high viremia, and are prevalent in affected populations. Taken together, these features suggest that the simian arteriviruses may be “preemergent” zoonotic pathogens. If not, this would imply that biological characteristics of RNA viruses thought to facilitate zoonotic transmission may not, by themselves, be sufficient for such transmission to occur.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and Selection of Duck Primary Cells as Pathogenic and Innate Immunologic Cell Models for Duck Plague Virus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 28</span></i><p style="text-align: justify; text-justify: inter-word;">Duck plague virus (DPV) is a representative pathogen transmitted among aquatic animals that causes gross lesions and immune inhibition in geese and ducks. The mechanism of organ tropism and innate immune evasion of DPV has not been completely deciphered due to a lack of cell models to study the innate immune manipulation and pathogenicity of aquatic viruses. In the present study, we isolated five types of duck primary cells [duck embryo fibroblasts (DEFs), neurons, astrocytes, peripheral blood mononuclear cells (PBMCs), and monocytes/macrophages] to identify appropriate cell models for DPV, using tropism infection and innate immunologic assays. Cells responded differently to stimulation with DNA viruses or RNA virus analogs. DPV infection exhibited broad tropism, as the recombinant virulent strain (CHv-GFP) infected DEFs, neurons, astrocytes, and monocytes/macrophages, but not the PBMCs, as the expression of EGFP was negligible. The basal levels of innate immunity molecules were highest in monocytes/macrophages and lower in DEFs and astrocytes. Conversely, the titer and genomic copy number of the attenuated virus strain was higher in DEFs and astrocytes than in neurons and monocytes/macrophages. The titer and genomic copy number of the attenuated virus strain were higher compared with the virulent strain in DEFs, neurons, and astrocytes. The innate immune response was not significantly induced by either DPV strain in DEFs, neurons, or astrocytes. The virulent strain persistently infected monocytes/macrophages, but the attenuated strain did so abortively, and this was accompanied by the phenomenon of innate immune inhibition and activation by the virulent and attenuated strains, respectively. Blockage of IFNAR signaling promoted replication of the attenuated strain. Pre-activation of IFNAR signaling inhibited infection by the virulent strain. The selection assay results indicated that induction of innate immunity plays an essential role in controlling DPV infection, and monocytes/macrophages are an important cell model for further investigations. Our study provided practical methods for isolating and culturing duck primary cells, and our results will facilitate further investigations of organ tropism, innate immune responses, latent infection, and the effectiveness of antiviral drugs for treating DPV and potentially other aerial bird pathogens.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Rethinking Biosafety in Research on Potential Pandemic Pathogens</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 9</span></i><p style="text-align: justify; text-justify: inter-word;">If accidentally released, mammalian-transmissible influenza A/H5N1 viruses could pose a greater threat to public health than possibly any other infectious agent currently under study in laboratories, because of such viruses’ likely combination of transmissibility and virulence to humans. We advocate explicit risk-benefit assessments before work on such pathogens is permitted or funded, improvement of biosafety practices and enforcement, and harmonization of criteria for permitting such experiments across government agencies, as well as internationally. Such potential pandemic pathogens, as they have been called, jeopardize not only laboratory workers and their contacts, but also the wider population, who should be involved in assessments of when such risks are acceptable in the service of scientific knowledge that may itself bear major public health benefits.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /></body>