
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: charge distribution of the coronavirus</h2><br /></div><br /><br /><br /><div class="card"><h3>Analysis of SARS-CoV E protein ion channel activity by tuning the protein and lipid charge</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A partial characterization of the ion channels formed by the SARS coronavirus (CoV) envelope (E) protein was previously reported [C. Verdiá-Báguena et al., 2012]. Here, we provide new significant insights on the involvement of lipids in the structure and function of the CoV E protein channel on the basis of three series of experiments. First, reversal potential measurements over a wide range of pH allows the dissection of the contributions to channel selectivity coming from ionizable residues of the protein transmembrane domain and also from the negatively charged groups of diphytanoyl phosphatidylserine (DPhPS) lipid. The corresponding effective pKa’s are consistent with the model pKa’s of the acidic residues candidates for titration. Second, the change of channel conductance with salt concentration reveals two distinct regimes (Donnan-controlled electrodiffusion and bulk-like electrodiffusion) fully compatible with the outcomes of selectivity experiments. Third, by measuring channel conductance in mixtures of neutral diphytanoyl phosphatidylcholine (DPhPC) lipids and negatively charged DPhPS lipids in low and high salt concentrations we conclude that the protein-lipid conformation in the channel is likely the same in charged and neutral lipids. Overall, the whole set of experiments supports the proteolipidic structure of SARS-CoV E channels and explains the large difference in channel conductance observed between neutral and charged membranes.</p>
<p align="right"><i>score: 171</i></p>

</div><br /><br /><br /><div class="card"><h3>Ribonucleocapsid Formation of Severe Acute Respiratory Syndrome Coronavirus through Molecular Action of the N-Terminal Domain of N Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Conserved among all coronaviruses are four structural proteins: the matrix (M), small envelope (E), and spike (S) proteins that are embedded in the viral membrane and the nucleocapsid phosphoprotein (N), which exists in a ribonucleoprotein complex in the lumen. The N-terminal domain of coronaviral N proteins (N-NTD) provides a scaffold for RNA binding, while the C-terminal domain (N-CTD) mainly acts as oligomerization modules during assembly. The C terminus of the N protein anchors it to the viral membrane by associating with M protein. We characterized the structures of N-NTD from severe acute respiratory syndrome coronavirus (SARS-CoV) in two crystal forms, at 1.17 Å (monoclinic) and at 1.85 Å (cubic), respectively, resolved by molecular replacement using the homologous avian infectious bronchitis virus (IBV) structure. Flexible loops in the solution structure of SARS-CoV N-NTD are now shown to be well ordered around the β-sheet core. The functionally important positively charged β-hairpin protrudes out of the core, is oriented similarly to that in the IBV N-NTD, and is involved in crystal packing in the monoclinic form. In the cubic form, the monomers form trimeric units that stack in a helical array. Comparison of crystal packing of SARS-CoV and IBV N-NTDs suggests a common mode of RNA recognition, but they probably associate differently in vivo during the formation of the ribonucleoprotein complex. Electrostatic potential distribution on the surface of homology models of related coronaviral N-NTDs suggests that they use different modes of both RNA recognition and oligomeric assembly, perhaps explaining why their nucleocapsids have different morphologies.</p>
<p align="right"><i>score: 168</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Functionally Important Negatively Charged Residues in the Carboxy End of Mouse Hepatitis Coronavirus A59 Nucleocapsid Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) protein is a multifunctional viral gene product that encapsidates the RNA genome and also plays some as yet not fully defined role in viral RNA replication and/or transcription. A number of conserved negatively charged amino acids are located within domain III in the carboxy end of all coronavirus N proteins. Previous studies suggested that the negatively charged residues are involved in virus assembly by mediating interaction between the membrane (M) protein carboxy tail and nucleocapsids. To determine the importance of these negatively charged residues, a series of alanine and other charged-residue substitutions were introduced in place of those in the N gene within a mouse hepatitis coronavirus A59 infectious clone. Aspartic acid residues 440 and 441 were identified as functionally important. Viruses could not be isolated when both residues were replaced by positively charged amino acids. When either amino acid was replaced by a positively charged residue or both were changed to alanine, viruses were recovered that contained second-site changes within N, but not in the M or envelope protein. The compensatory role of the new changes was confirmed by the construction of new viruses. A few viruses were recovered that retained the D(441)-to-arginine change and no compensatory changes. These viruses exhibited a small-plaque phenotype and produced significantly less virus. Overall, results from our analysis of a large panel of plaque-purified recovered viruses indicate that the negatively charged residues at positions 440 and 441 are key residues that appear to be involved in virus assembly.</p>
<p align="right"><i>score: 165</i></p>

</div><br /><br /><br /><div class="card"><h3>Negatively charged residues in the endodomain are critical for specific assembly of spike protein into murine coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus spike (S) protein assembles into virions via its carboxy-terminus, which is composed of a transmembrane domain and an endodomain. Here, the carboxy-terminal charge-rich motif in the endodomain was verified to be critical for the specificity of S assembly into mouse hepatitis virus (MHV). Recombinant MHVs exhibited a range of abilities to accommodate the homologous S endodomains from the betacoronaviruses bovine coronavirus and human SARS-associated coronavirus, the alphacoronavirus porcine transmissible gastroenteritis virus (TGEV), and the gammacoronavirus avian infectious bronchitis virus respectively. Interestingly, in TGEV endodomain chimeras the reverting mutations resulted in stronger S incorporation into virions, and a net gain of negatively charged residues in the charge-rich motif accounted for the improvement. Additionally, MHV S assembly could also be rescued by the acidic carboxy-terminal domain of the nucleocapsid protein. These results indicate an important role for negatively charged endodomain residues in the incorporation of MHV S protein into assembled virions.</p>
<p align="right"><i>score: 163</i></p>

</div><br /><br /><br /><div class="card"><h3>Importance of the Penultimate Positive Charge in Mouse Hepatitis Coronavirus A59 Membrane Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein carboxy tail interacts with the nucleocapsid during virus assembly. Previous studies demonstrated that the two terminal residues are important, and the charged residue (R227) in the penultimate position in the mouse hepatitis coronavirus (MHV) A59 M protein was suggested to participate in intermolecular interactions with negative charges in the nucleocapsid (N) protein. To determine the significance of the positive charge at position 227, we substituted the arginine with lysine (K), aspartic acid (D), glutamic acid (E), or alanine (A) and studied these by reverse genetics in the context of a MHV full-length infectious clone. Viruses with wild-type phenotype were readily recovered with the K or A substitutions. In contrast, negative-charge substitutions were not tolerated as well. In all recovered R227D viruses the negative charge was replaced with heterologous residues resulting from apparent template switching during negative-strand synthesis of subgenomic RNA 7. An additional second-site compensatory V202I substitution was present in some viruses. Recovered R227E viruses had second-site changes within the M protein carboxy tail that were partially compensatory. Significantly, most of the second site changes in the R227E mutant viruses were previously shown to compensate for the removal of negative charges in the N protein. Our results strongly indicate that a positive charge is not absolutely required. It is clear that other regions within the tail must also be involved in helping mediate interactions between the M protein and the nucleocapsid.</p>
<p align="right"><i>score: 159</i></p>

</div><br /><br /><br /><div class="card"><h3>Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Oct 13</span></i><p style="text-align: justify; text-justify: inter-word;">The N-terminal domain of the coronavirus nucleocapsid (N) protein adopts a fold resembling a right hand with a flexible, positively charged β-hairpin and a hydrophobic palm. This domain was shown to interact with the genomic RNA for coronavirus infectious bronchitis virus (IBV) and severe acute respiratory syndrome coronavirus (SARS-CoV). Based on its 3D structure, we used site-directed mutagenesis to identify residues essential for the RNA-binding activity of the IBV N protein and viral infectivity. Alanine substitution of either Arg-76 or Tyr-94 in the N-terminal domain of IBV N protein led to a significant decrease in its RNA-binding activity and a total loss of the infectivity of the viral RNA to Vero cells. In contrast, mutation of amino acid Gln-74 to an alanine, which does not affect the binding activity of the N-terminal domain, showed minimal, if any, detrimental effect on the infectivity of IBV. This study thus identifies residues critical for RNA binding on the nucleocapsid surface, and presents biochemical and genetic evidence that directly links the RNA binding capacity of the coronavirus N protein to the viral infectivity in cultured cells. This information would be useful in development of preventive and treatment approaches against coronavirus infection.</p>
<p align="right"><i>score: 150</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural Genomics of the Severe Acute Respiratory Syndrome Coronavirus: Nuclear Magnetic Resonance Structure of the Protein nsP7</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Here, we report the three-dimensional structure of severe acute respiratory syndrome coronavirus (SARS-CoV) nsP7, a component of the SARS-CoV replicase polyprotein. The coronavirus replicase carries out regulatory tasks involved in the maintenance, transcription, and replication of the coronavirus genome. nsP7 was found to assume a compact architecture in solution, which is comprised primarily of helical secondary structures. Three helices (α2 to α4) form a flat up-down-up antiparallel α-helix sheet. The N-terminal segment of residues 1 to 22, containing two turns of α-helix and one turn of 3(10)-helix, is packed across the surface of α2 and α3 in the helix sheet, with the α-helical region oriented at a 60° angle relative to α2 and α3. The surface charge distribution is pronouncedly asymmetrical, with the flat surface of the helical sheet showing a large negatively charged region adjacent to a large hydrophobic patch and the opposite side containing a positively charged groove that extends along the helix α1. Each of these three areas is thus implicated as a potential site for protein-protein interactions.</p>
<p align="right"><i>score: 150</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV Fusion Peptide forms an Extended Bipartite Fusion Platform that Perturbs Membrane Order in a Calcium-Dependent Manner</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are a major infectious disease threat, and include the pathogenic human pathogens of zoonotic origin: SARS-CoV and MERS-CoV. Entry of coronaviruses into host cells is mediated by the viral spike (S) protein, which is structurally categorized as a class I viral fusion protein, within the same group as influenza virus and HIV. However, S proteins have two distinct cleavage sites that can be activated by a much wider range of proteases. The exact location of the coronavirus fusion peptide (FP) has been disputed. However, most evidence suggests that the domain immediately downstream of the S2′ cleavage site is the FP (amino acids 798-818 SFIEDLLFNKVTLADAGFMKQY for SARS-CoV, FP1). In our previous ESR spectroscopic studies, the membrane ordering effect of influenza virus, HIV and Dengue virus FPs have been consistently observed. In this study, we used this effect as a criterion to identify and characterize the bona fide SARS-CoV FP. Our results indicate that both FP1 and the region immediately downstream (amino acids 816-835 KQYGECLGDINARDLICAQKF, FP2) induce significant membrane ordering. Furthermore, their effects are calcium-dependent, which is consistent with in vivo data showing that calcium is required for SARS-CoV S-mediated fusion. Isothermal titration calorimetry showed a direct interaction between calcium cations and both FPs. This Ca(2+)-dependency membrane ordering was not observed with influenza FP, indicating that the coronavirus FP exhibits a mechanistically different behavior. Membrane ordering effects are greater and penetrate deeper into membranes when FP1 and FP2 act in a concerted manner, suggesting that they form an extended fusion “platform”.</p>
<p align="right"><i>score: 149</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Analysis of Determinants for Spike Glycoprotein Assembly into Murine Coronavirus Virions: Distinct Roles for Charge-Rich and Cysteine-Rich Regions of the Endodomain</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus spike protein (S) forms the distinctive virion surface structures that are characteristic of this viral family, appearing in negatively stained electron microscopy as stems capped with spherical bulbs. These structures are essential for the initiation of infection through attachment of the virus to cellular receptors followed by fusion to host cell membranes. The S protein can also mediate the formation of syncytia in infected cells. The S protein is a type I transmembrane protein that is very large compared to other viral fusion proteins, and all except a short carboxy-terminal segment of the S molecule constitutes the ectodomain. For the prototype coronavirus mouse hepatitis virus (MHV), it has previously been established that S protein assembly into virions is specified by the carboxy-terminal segment, which comprises the transmembrane domain and the endodomain. We have genetically dissected these domains in the MHV S protein to localize the determinants of S incorporation into virions. Our results establish that assembly competence maps to the endodomain of S, which was shown to be sufficient to target a heterologous integral membrane protein for incorporation into MHV virions. In particular, mutational analysis indicated a major role for the charge-rich carboxy-terminal region of the endodomain. Additionally, we found that the adjacent cysteine-rich region of the endodomain is critical for fusion of infected cells, confirming results previously obtained with S protein expression systems.</p>
<p align="right"><i>score: 144</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of Constructed E Gene Mutants of Mouse Hepatitis Virus Confirms a Pivotal Role for E Protein in Coronavirus Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Expression studies have shown that the coronavirus small envelope protein E and the much more abundant membrane glycoprotein M are both necessary and sufficient for the assembly of virus-like particles in cells. As a step toward understanding the function of the mouse hepatitis virus (MHV) E protein, we carried out clustered charged-to-alanine mutagenesis on the E gene and incorporated the resulting mutations into the MHV genome by targeted recombination. Of the four possible clustered charged-to-alanine E gene mutants, one was apparently lethal and one had a wild-type phenotype. The two other mutants were partially temperature sensitive, forming small plaques at the nonpermissive temperature. Revertant analyses of these two mutants demonstrated that the created mutations were responsible for the temperature-sensitive phenotype of each and provided support for possible interactions among E protein monomers. Both temperature-sensitive mutants were also found to be markedly thermolabile when grown at the permissive temperature, suggesting that there was a flaw in their assembly. Most significantly, when virions of one of the mutants were examined by electron microscopy, they were found to have strikingly aberrant morphology in comparison to the wild type: most mutant virions had pinched and elongated shapes that were rarely seen among wild-type virions. These results demonstrate an important, probably essential, role for the E protein in coronavirus morphogenesis.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus E Protein: Assembly and Beyond</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Mar 8</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein that has an important role in the assembly of virions. Recent studies have indicated that the E protein has functions during infection beyond assembly, including in virus egress and in the host stress response. Additionally, the E protein has ion channel activity, interacts with host proteins, and may have multiple membrane topologies. The goal of this review is to highlight the properties and functions of the E protein, and speculate on how they may be related.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and Inhibition of the SARS Coronavirus Envelope Protein Ion Channel</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul 10</span></i><p style="text-align: justify; text-justify: inter-word;">The envelope (E) protein from coronaviruses is a small polypeptide that contains at least one α-helical transmembrane domain. Absence, or inactivation, of E protein results in attenuated viruses, due to alterations in either virion morphology or tropism. Apart from its morphogenetic properties, protein E has been reported to have membrane permeabilizing activity. Further, the drug hexamethylene amiloride (HMA), but not amiloride, inhibited in vitro ion channel activity of some synthetic coronavirus E proteins, and also viral replication. We have previously shown for the coronavirus species responsible for severe acute respiratory syndrome (SARS-CoV) that the transmembrane domain of E protein (ETM) forms pentameric α-helical bundles that are likely responsible for the observed channel activity. Herein, using solution NMR in dodecylphosphatidylcholine micelles and energy minimization, we have obtained a model of this channel which features regular α-helices that form a pentameric left-handed parallel bundle. The drug HMA was found to bind inside the lumen of the channel, at both the C-terminal and the N-terminal openings, and, in contrast to amiloride, induced additional chemical shifts in ETM. Full length SARS-CoV E displayed channel activity when transiently expressed in human embryonic kidney 293 (HEK-293) cells in a whole-cell patch clamp set-up. This activity was significantly reduced by hexamethylene amiloride (HMA), but not by amiloride. The channel structure presented herein provides a possible rationale for inhibition, and a platform for future structure-based drug design of this potential pharmacological target.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and Inhibition of the SARS Coronavirus Envelope Protein Ion Channel</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul 10</span></i><p style="text-align: justify; text-justify: inter-word;">The envelope (E) protein from coronaviruses is a small polypeptide that contains at least one α-helical transmembrane domain. Absence, or inactivation, of E protein results in attenuated viruses, due to alterations in either virion morphology or tropism. Apart from its morphogenetic properties, protein E has been reported to have membrane permeabilizing activity. Further, the drug hexamethylene amiloride (HMA), but not amiloride, inhibited in vitro ion channel activity of some synthetic coronavirus E proteins, and also viral replication. We have previously shown for the coronavirus species responsible for severe acute respiratory syndrome (SARS-CoV) that the transmembrane domain of E protein (ETM) forms pentameric α-helical bundles that are likely responsible for the observed channel activity. Herein, using solution NMR in dodecylphosphatidylcholine micelles and energy minimization, we have obtained a model of this channel which features regular α-helices that form a pentameric left-handed parallel bundle. The drug HMA was found to bind inside the lumen of the channel, at both the C-terminal and the N-terminal openings, and, in contrast to amiloride, induced additional chemical shifts in ETM. Full length SARS-CoV E displayed channel activity when transiently expressed in human embryonic kidney 293 (HEK-293) cells in a whole-cell patch clamp set-up. This activity was significantly reduced by hexamethylene amiloride (HMA), but not by amiloride. The channel structure presented herein provides a possible rationale for inhibition, and a platform for future structure-based drug design of this potential pharmacological target.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Transient Oligomerization of the SARS-CoV N Protein – Implication for Virus Ribonucleoprotein Packaging</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 May 23</span></i><p style="text-align: justify; text-justify: inter-word;">The nucleocapsid (N) phosphoprotein of the severe acute respiratory syndrome coronavirus (SARS-CoV) packages the viral genome into a helical ribonucleocapsid and plays a fundamental role during viral self-assembly. The N protein consists of two structural domains interspersed between intrinsically disordered regions and dimerizes through the C-terminal structural domain (CTD). A key activity of the protein is the ability to oligomerize during capsid formation by utilizing the dimer as a building block, but the structural and mechanistic bases of this activity are not well understood. By disulfide trapping technique we measured the amount of transient oligomers of N protein mutants with strategically located cysteine residues and showed that CTD acts as a primary transient oligomerization domain in solution. The data is consistent with the helical oligomer packing model of N protein observed in crystal. A systematic study of the oligomerization behavior revealed that altering the intermolecular electrostatic repulsion through changes in solution salt concentration or phosphorylation-mimicking mutations affects oligomerization propensity. We propose a biophysical mechanism where electrostatic repulsion acts as a switch to regulate N protein oligomerization.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Transient Oligomerization of the SARS-CoV N Protein – Implication for Virus Ribonucleoprotein Packaging</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 May 23</span></i><p style="text-align: justify; text-justify: inter-word;">The nucleocapsid (N) phosphoprotein of the severe acute respiratory syndrome coronavirus (SARS-CoV) packages the viral genome into a helical ribonucleocapsid and plays a fundamental role during viral self-assembly. The N protein consists of two structural domains interspersed between intrinsically disordered regions and dimerizes through the C-terminal structural domain (CTD). A key activity of the protein is the ability to oligomerize during capsid formation by utilizing the dimer as a building block, but the structural and mechanistic bases of this activity are not well understood. By disulfide trapping technique we measured the amount of transient oligomers of N protein mutants with strategically located cysteine residues and showed that CTD acts as a primary transient oligomerization domain in solution. The data is consistent with the helical oligomer packing model of N protein observed in crystal. A systematic study of the oligomerization behavior revealed that altering the intermolecular electrostatic repulsion through changes in solution salt concentration or phosphorylation-mimicking mutations affects oligomerization propensity. We propose a biophysical mechanism where electrostatic repulsion acts as a switch to regulate N protein oligomerization.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the past few decades by two global outbreaks of deadly pneumonia. The coronavirus spike (S) glycoprotein initiates infection by promoting fusion of the viral and cellular membranes through conformational changes that remain largely uncharacterized. Here we report the cryoEM structure of a coronavirus S glycoprotein in the postfusion state, showing large-scale secondary, tertiary, and quaternary rearrangements compared with the prefusion trimer and rationalizing the free-energy landscape of this conformational machine. We also biochemically characterized the molecular events associated with refolding of the metastable prefusion S glycoprotein to the postfusion conformation using limited proteolysis, mass spectrometry, and single-particle EM. The observed similarity between postfusion coronavirus S and paramyxovirus F structures demonstrates that a conserved refolding trajectory mediates entry of these viruses and supports the evolutionary relatedness of their fusion subunits. Finally, our data provide a structural framework for understanding the mode of neutralization of antibodies targeting the fusion machinery and for engineering next-generation subunit vaccines or inhibitors against this medically important virus family.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>A structural analysis of M protein in coronavirus assembly and morphology</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The M protein of coronavirus plays a central role in virus assembly, turning cellular membranes into workshops where virus and host factors come together to make new virus particles. We investigated how M structure and organization is related to virus shape and size using cryo-electron microscopy, tomography and statistical analysis. We present evidence that suggests M can adopt two conformations and that membrane curvature is regulated by one M conformer. Elongated M protein is associated with rigidity, clusters of spikes and a relatively narrow range of membrane curvature. In contrast, compact M protein is associated with flexibility and low spike density. Analysis of several types of virus-like particles and virions revealed that S protein, N protein and genomic RNA each help to regulate virion size and variation, presumably through interactions with M. These findings provide insight into how M protein functions to promote virus assembly.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of coronavirus disease (COVID-19) in China caused by SARS-CoV-2 virus continually lead to worldwide human infections and deaths. It is currently no specific viral protein targeted therapeutics yet. Viral nucleocapsid protein is a potential antiviral drug target, serving multiple critical functions during the viral life cycle. However, the structural information of SARS-CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the crystal structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. Although overall structure is similar with other reported coronavirus nucleocapsid protein N-terminal domain, the surface electrostatic potential characteristics between them are distinct. Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique potential RNA binding pocket alongside the beta-sheet core. Complemented by in vitro binding studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to SARS-CoV-2.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Mouse Hepatitis Coronavirus A59 Nucleocapsid Protein Phosphorylation Sites</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) is a multifunctional phosphoprotein that encapsidates the genomic RNA into a helical nucleocapsid within the mature virion. The protein also plays roles in viral RNA transcription and/or replication and possibly viral mRNA translation. Phosphorylation is one of the most common post-translation modifications that plays important regulatory roles in modulating protein functions. It has been speculated for sometime that phosphorylation could play an important role in regulation of coronavirus N protein functions. As a first step toward positioning to address this we have identified the amino acids that are phosphorylated on the mouse hepatitis coronavirus (MHV) A59 N protein. High performance liquid chromatography coupled with electrospray ionization tandem mass spectrometry (HPLC-ESI-MS/MS) was used to identify phosphorylated sites on the N protein from both infected cells and purified extracellular virions. A total of six phosphorylated sites (S162, S170, T177, S389, S424 and T428) were identified on the protein from infected cells. The same six sites were also phosphorylated on the extracellular mature virion N protein. This is the first identification of phosphorylated sites for a group II coronavirus N protein.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of mouse hepatitis coronavirus A59 nucleocapsid protein phosphorylation sites</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) is a multifunctional phosphoprotein that encapsidates the genomic RNA into a helical nucleocapsid within the mature virion. The protein also plays roles in viral RNA transcription and/or replication and possibly viral mRNA translation. Phosphorylation is one of the most common post-translation modifications that plays important regulatory roles in modulating protein functions. It has been speculated for sometime that phosphorylation could play an important role in regulation of coronavirus N protein functions. As a first step toward positioning to address this we have identified the amino acids that are phosphorylated on the mouse hepatitis coronavirus (MHV) A59 N protein. High performance liquid chromatography coupled with electrospray ionization tandem mass spectrometry (HPLC-ESI-MS/MS) was used to identify phosphorylated sites on the N protein from both infected cells and purified extracellular virions. A total of six phosphorylated sites (S162, S170, T177, S389, S424 and T428) were identified on the protein from infected cells. The same six sites were also phosphorylated on the extracellular mature virion N protein. This is the first identification of phosphorylated sites for a group II coronavirus N protein.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Conformational change of the coronavirus peplomer glycoprotein at pH 8.0 and 37 degrees C correlates with virus aggregation and virus-induced cell fusion.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have obtained biochemical and electron microscopic evidence of conformational changes at pH 8.0 and 37 degrees C in the coronavirus spike glycoprotein E2 (S). The importance of these changes is reflected in the loss of virus infectivity, the aggregation of virions, and increased virus-induced cell fusion at the same pH. Coronavirus (MHV-A59) infectivity is exquisitely sensitive to pH. The virus was quite stable at pH 6.0 and 37 degrees C (half-life, approximately 24 h) but was rapidly and irreversibly inactivated by brief treatment at pH 8.0 and 37 degrees C (half-life, approximately 30 min). Virions treated at pH 8.0 and 37 degrees C formed clumps and large aggregates. With virions treated at pH 8.0 and 37 degrees C, the amino-terminal peptide E2N (or S1) was released from virions and the remaining peptide, E2C (S2), was aggregated. Viral spikes isolated from detergent-treated virions also aggregated at pH 8.0 and 37 degrees C. Loss of virus infectivity and E2 (S) aggregation at pH 8.0 and 37 degrees C were markedly enhanced in the presence of dithiothreitol. On the basis of the effects of dithiothreitol on the reactions of the peplomer, we propose that release of E2N (S1) and aggregation of E2C (S2) may be triggered by rearrangement of intramolecular disulfide bonds. The aggregation of virions and the isolated E2 (S) glycoprotein at pH 8.0 and 37 degrees C or following treatment with guanidine and urea at pH 6.0 and 37 degrees C indicate that an irreversible conformational change has been induced in the peplomer glycoprotein by these conditions. It is interesting that coronavirus-induced cell fusion also occurred under mildly alkaline conditions and at 37 degrees C. Some enveloped viruses, including influenza viruses and alphaviruses, show conformational changes of spike glycoproteins at a low pH, which correlates with fusion and penetration of those viruses in acidified endocytic vesicles. For coronavirus MHV-A59, comparable conformational change of the spike glycoprotein E2 (S) and cell fusion occurred at a mildly alkaline condition, suggesting that coronavirus infection-penetration, like that of paramyxoviruses and lentiviruses, may occur at the plasma membrane, rather than within endocytic vesicles.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystallization and preliminary X-ray diffraction analysis of the N-terminal domain of human coronavirus OC43 nucleocapsid protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The N-terminal domain of nucleocapsid protein from human coronavirus OC43 (HCoV-OC43 N-NTD) mostly contains positively charged residues and has been identified as being responsible for RNA binding during ribonucleocapsid formation in the coronavirus. In this study, the crystallization and preliminary crystallographic analysis of HCoV-OC43 N-NTD (amino acids 58–195) with a molecular weight of 20 kDa are reported. HCoV-OC43 N-NTD was crystallized at 293 K using PEG 1500 as a precipitant and a 99.9% complete native data set was collected to 1.7 Å resolution at 100 K with an overall R (merge) of 5.0%. The crystals belonged to the hexagonal space group P6(5), with unit-cell parameters a = 81.57, c = 42.87 Å. Solvent-content calculations suggest that there is likely to be one subunit of N-NTD in the asymmetric unit.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiple Nucleic Acid Binding Sites and Intrinsic Disorder of Severe Acute Respiratory Syndrome Coronavirus Nucleocapsid Protein: Implications for Ribonucleocapsid Protein Packaging</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The nucleocapsid protein (N) of the severe acute respiratory syndrome coronavirus (SARS-CoV) packages the viral genomic RNA and is crucial for viability. However, the RNA-binding mechanism is poorly understood. We have shown previously that the N protein contains two structural domains—the N-terminal domain (NTD; residues 45 to 181) and the C-terminal dimerization domain (CTD; residues 248 to 365)—flanked by long stretches of disordered regions accounting for almost half of the entire sequence. Small-angle X-ray scattering data show that the protein is in an extended conformation and that the two structural domains of the SARS-CoV N protein are far apart. Both the NTD and the CTD have been shown to bind RNA. Here we show that all disordered regions are also capable of binding to RNA. Constructs containing multiple RNA-binding regions showed Hill coefficients greater than 1, suggesting that the N protein binds to RNA cooperatively. The effect can be explained by the “coupled-allostery” model, devised to explain the allosteric effect in a multidomain regulatory system. Although the N proteins of different coronaviruses share very low sequence homology, the physicochemical features described above may be conserved across different groups of Coronaviridae. The current results underscore the important roles of multisite nucleic acid binding and intrinsic disorder in N protein function and RNP packaging.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>A Single Polar Residue and Distinct Membrane Topologies Impact the Function of the Infectious Bronchitis Coronavirus E Protein</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 May 3</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein with a single predicted hydrophobic domain (HD), and has a poorly defined role in infection. The E protein is thought to promote virion assembly, which occurs in the Golgi region of infected cells. It has also been implicated in the release of infectious particles after budding. The E protein has ion channel activity in vitro, although a role for channel activity in infection has not been established. Furthermore, the membrane topology of the E protein is of considerable debate, and the protein may adopt more than one topology during infection. We previously showed that the HD of the infectious bronchitis virus (IBV) E protein is required for the efficient release of infectious virus, an activity that correlated with disruption of the secretory pathway. Here we report that a single residue within the hydrophobic domain, Thr16, is required for secretory pathway disruption. Substitutions of other residues for Thr16 were not tolerated. Mutations of Thr16 did not impact virus assembly as judged by virus-like particle production, suggesting that alteration of secretory pathway and assembly are independent activities. We also examined how the membrane topology of IBV E affected its function by generating mutant versions that adopted either a transmembrane or membrane hairpin topology. We found that a transmembrane topology was required for disrupting the secretory pathway, but was less efficient for virus-like particle production. The hairpin version of E was unable to disrupt the secretory pathway or produce particles. The findings reported here identify properties of the E protein that are important for its function, and provide insight into how the E protein may perform multiple roles during infection.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron tomography of mouse hepatitis virus: Insights into the structure of the coronavirion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are enveloped viruses containing the largest reported RNA genomes. As a result of their pleomorphic nature, our structural insight into the coronavirion is still rudimentary, and it is based mainly on 2D electron microscopy. Here we report the 3D virion structure of coronaviruses obtained by cryo-electron tomography. Our study focused primarily on the coronavirus prototype murine hepatitis virus (MHV). MHV particles have a distinctly spherical shape and a relatively homogenous size (≈85 nm envelope diameter). The viral envelope exhibits an unusual thickness (7.8 ± 0.7 nm), almost twice that of a typical biological membrane. Focal pairs revealed the existence of an extra internal layer, most likely formed by the C-terminal domains of the major envelope protein M. In the interior of the particles, coiled structures and tubular shapes are observed, consistent with a helical nucleocapsid model. Our reconstructions provide no evidence of a shelled core. Instead, the ribonucleoprotein seems to be extensively folded onto itself, assuming a compact structure that tends to closely follow the envelope at a distance of ≈4 nm. Focal contact points and thread-like densities connecting the envelope and the ribonucleoprotein are revealed in the tomograms. Transmissible gastroenteritis coronavirion tomograms confirm all the general features and global architecture observed for MHV. We propose a general model for the structure of the coronavirion in which our own and published observations are combined.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Nucleocapsid Is a Multifunctional Protein</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 7</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) is a structural protein that forms complexes with genomic RNA, interacts with the viral membrane protein during virion assembly and plays a critical role in enhancing the efficiency of virus transcription and assembly. Recent studies have confirmed that N is a multifunctional protein. The aim of this review is to highlight the properties and functions of the N protein, with specific reference to (i) the topology; (ii) the intracellular localization and (iii) the functions of the protein.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Viral Transmission Behavior via Protein Intrinsic Disorder Prediction: Coronaviruses</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 14</span></i><p style="text-align: justify; text-justify: inter-word;">Besides being a common threat to farm animals and poultry, coronavirus (CoV) was responsible for the human severe acute respiratory syndrome (SARS) epidemic in 2002–4. However, many aspects of CoV behavior, including modes of its transmission, are yet to be fully understood. We show that the amount and the peculiarities of distribution of the protein intrinsic disorder in the viral shell can be used for the efficient analysis of the behavior and transmission modes of CoV. The proposed model allows categorization of the various CoVs by the peculiarities of disorder distribution in their membrane (M) and nucleocapsid (N). This categorization enables quick identification of viruses with similar behaviors in transmission, regardless of genetic proximity. Based on this analysis, an empirical model for predicting the viral transmission behavior is developed. This model is able to explain some behavioral aspects of important coronaviruses that previously were not fully understood. The new predictor can be a useful tool for better epidemiological, clinical, and structural understanding of behavior of both newly emerging viruses and viruses that have been known for a long time. A potentially new vaccine strategy could involve searches for viral strains that are characterized by the evolutionary misfit between the peculiarities of the disorder distribution in their shells and their behavior.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural Flexibility of the Pentameric SARS Coronavirus Envelope Protein Ion Channel</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses contain a small envelope membrane protein with cation-selective ion channel activity mediated by its transmembrane domain (ETM). In a computational study, we proposed that ion channel activity can be explained by either of two similar ETM homopentameric transmembrane α-helical bundles, related by a ∼50° rotation of the helices. Later, we tested this prediction, using site-specific infrared dichroism of a lysine-flanked isotopically labeled ETM peptide from the virus responsible for the severe acute respiratory syndrome, SARS, reconstituted in lipid bilayers. However, the data were consistent with the presence of a kink at the center of the ETM α-helix, and it did not fit completely either computational model. Herein, we have used native ETM, without flanking lysines, and show that the helix orientation is now consistent with one of the predicted models. ETM only produced one oligomeric form, pentamers, in the lipid-mimic detergent dodecylphosphocholine and in perfluorooctanoic acid. We thus report the correct backbone model for the pentameric α-helical bundle of ETM. The disruptive effects caused by terminal lysines probably highlight the conformational flexibility required during ion channel function.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>A Major Determinant for Membrane Protein Interaction Localizes to the Carboxy-Terminal Domain of the Mouse Coronavirus Nucleocapsid Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The two major constituents of coronavirus virions are the membrane (M) and nucleocapsid (N) proteins. The M protein is anchored in the viral envelope by three transmembrane segments flanked by a short amino-terminal ectodomain and a large carboxy-terminal endodomain. The M endodomain interacts with the viral nucleocapsid, which consists of the positive-strand RNA genome helically encapsidated by N protein monomers. In previous work with the coronavirus mouse hepatitis virus (MHV), a highly defective M protein mutant, MΔ2, was constructed. This mutant contained a 2-amino-acid carboxy-terminal truncation of the M protein. Analysis of second-site revertants of MΔ2 revealed mutations in the carboxy-terminal region of the N protein that compensated for the defect in the M protein. To seek further genetic evidence corroborating this interaction, we generated a comprehensive set of clustered charged-to-alanine mutants in the carboxy-terminal domain 3 of N protein. One of these mutants, CCA4, had a highly defective phenotype similar to that of MΔ2. Transfer of the CCA4 mutation into a partially diploid MHV genome showed that CCA4 was a loss-of-function mutation rather than a dominant-negative mutation. Analysis of multiple second-site revertants of CCA4 revealed mutations in both the M protein and the N protein that could compensate for the original lesion in N. These data more precisely define the region of the N protein that interacts with the M protein. Further, we found that fusion of domain 3 of the N protein to the carboxy terminus of a heterologous protein caused it to be incorporated into MHV virions.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Crucial steps in the structure determination of a coronavirus spike glycoprotein using cryo‐electron microscopy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the last 15 years by two global outbreaks of deadly pneumonia. Entry of coronaviruses into cells is mediated by the transmembrane spike glycoprotein S, which forms a trimer carrying receptor‐binding and membrane fusion functions. Despite their biomedical importance, coronavirus S glycoproteins have proven difficult targets for structural characterization, precluding high‐resolution studies of the biologically relevant trimer. Recent technological developments in single particle cryo‐electron microscopy allowed us to determine the first structure of a coronavirus S glycoprotein trimer which provided a framework to understand the mechanisms of viral entry and suggested potential inhibition strategies for this family of viruses. Here, we describe the key factors that enabled this breakthrough.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Mesodynamics in the SARS nucleocapsid measured by NMR field cycling</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Sep 30</span></i><p style="text-align: justify; text-justify: inter-word;">Protein motions on all timescales faster than molecular tumbling are encoded in the spectral density. The dissection of complex protein dynamics is typically performed using relaxation rates determined at high and ultra-high field. Here we expand this range of the spectral density to low fields through field cycling using the nucleocapsid protein of the SARS coronavirus as a model system. The field-cycling approach enables site-specific measurements of R(1) at low fields with the sensitivity and resolution of a high-field magnet. These data, together with high-field relaxation and heteronuclear NOE, provide evidence for correlated rigid-body motions of the entire β-hairpin, and corresponding motions of adjacent loops with a time constant of 0.8 ns (mesodynamics). MD simulations substantiate these findings and provide direct verification of the time scale and collective nature of these motions.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Importance of Conserved Cysteine Residues in the Coronavirus Envelope Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus envelope (E) proteins play an important, not fully understood role(s) in the virus life cycle. All E proteins have conserved cysteine residues located on the carboxy side of the long hydrophobic domain, suggesting functional significance. In this study, we confirmed that mouse hepatitis coronavirus A59 E protein is palmitoylated. To understand the role of the conserved residues and the necessity of palmitoylation, three cysteines at positions 40, 44, and 47 were changed singly and in various combinations to alanine. Double- and triple-mutant E proteins resulted in decreased virus-like particle output when coexpressed with the membrane (M) protein. Mutant E proteins were also studied in the context of a full-length infectious clone. Single-substitution viruses exhibited growth characteristics virtually identical to those of the wild-type virus, while the double-substitution mutations gave rise to viruses with less robust growth phenotypes indicated by smaller plaques and decreased virus yields. In contrast, replacement of all three cysteines resulted in crippled virus with significantly reduced yields. Triple-mutant viruses did not exhibit impairment in entry. Mutant E proteins localized properly in infected cells. A comparison of intracellular and extracellular virus yields suggested that release is only slightly impaired. E protein lacking all three cysteines exhibited an increased rate of degradation compared to that of the wild-type protein, suggesting that palmitoylation is important for the stability of the protein. Altogether, the results indicate that the conserved cysteines and presumably palmitoylation are functionally important for virus production.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure-based exploration of the important role of Arg106 in the RNA-binding domain of human coronavirus OC43 nucleocapsid protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus OC43 (HCoV-OC43) is a causative agent of the common cold. The nucleocapsid (N) protein, which is a major structural protein of CoVs, binds to the viral RNA genome to form the virion core and results in the formation of the ribonucleoprotein (RNP) complex. We have solved the crystal structure of the N-terminal domain of HCoV-OC43 N protein (N-NTD) (residues 58 to 195) to a resolution of 2.0Å. The HCoV-OC43 N-NTD is a single domain protein composed of a five-stranded β-sheet core and a long extended loop, similar to that observed in the structures of N-NTDs from other coronaviruses. The positively charged loop of the HCoV-OC43 N-NTD contains a structurally well-conserved positively charged residue, R106. To assess the role of R106 in RNA binding, we undertook a series of site-directed mutagenesis experiments and docking simulations to characterize the interaction between R106 and RNA. The results show that R106 plays an important role in the interaction between the N protein and RNA. In addition, we showed that, in cells transfected with plasmids that encoded the mutant (R106A) N protein and infected with virus, the level of the matrix protein gene was decreased by 7-fold compared to cells that were transfected with the wild-type N protein. This finding suggests that R106, by enhancing binding of the N protein to viral RNA plays a critical role in the viral replication. The results also indicate that the strength of N protein/RNA interactions is critical for HCoV-OC43 replication.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 23</span></i><p style="text-align: justify; text-justify: inter-word;">As cell-invading molecular machinery, coronavirus spike proteins pose an evolutionary conundrum due to their high divergence. In this study, we determined the cryo-EM structure of avian infectious bronchitis coronavirus (IBV) spike protein from the γ-genus. The trimeric IBV spike ectodomain contains three receptor-binding S1 heads and a trimeric membrane-fusion S2 stalk. While IBV S2 is structurally similar to those from the other genera, IBV S1 possesses structural features that are unique to different other genera, thereby bridging these diverse spikes into an evolutionary spectrum. Specifically, among different genera, the two domains of S1, the N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), diverge from simpler tertiary structures and quaternary packing to more complex ones, leading to different functions of the spikes in receptor usage and membrane fusion. Based on the above structural and functional comparisons, we propose that the evolutionary spectrum of coronavirus spikes follows the order of α-, δ-, γ-, and β-genus. This study has provided insight into the evolutionary relationships among coronavirus spikes and deepened our understanding of their structural and functional diversity.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystallographic analysis of the N-terminal domain of Middle East respiratory syndrome coronavirus nucleocapsid protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The N-terminal domain of the nucleocapsid protein from Middle East respiratory syndrome coronavirus (MERS-CoV NP-NTD) contains many positively charged residues and has been identified to be responsible for RNA binding during ribonucleocapsid formation by the virus. In this study, the crystallization and crystallographic analysis of MERS-CoV NP-NTD (amino acids 39–165), with a molecular weight of 14.7 kDa, are reported. MERS-CoV NP-NTD was crystallized at 293 K using PEG 3350 as a precipitant and a 94.5% complete native data set was collected from a cooled crystal at 77 K to 2.63 Å resolution with an overall R (merge) of 9.6%. The crystals were monoclinic and belonged to space group P2(1), with unit-cell parameters a = 35.60, b = 109.64, c = 91.99 Å, β = 101.22°. The asymmetric unit contained four MERS-CoV NP-NTD molecules.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>A Conserved Domain in the Coronavirus Membrane Protein Tail Is Important for Virus Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus membrane (M) proteins play key roles in virus assembly, through M-M, M-spike (S), and M-nucleocapsid (N) protein interactions. The M carboxy-terminal endodomain contains a conserved domain (CD) following the third transmembrane (TM) domain. The importance of the CD (SWWSFNPETNNL) in mouse hepatitis virus was investigated with a panel of mutant proteins, using genetic analysis and transient-expression assays. A charge reversal for negatively charged E(121) was not tolerated. Lysine (K) and arginine (R) substitutions were replaced in recovered viruses by neutrally charged glutamine (Q) and leucine (L), respectively, after only one passage. E(121)Q and E(121)L M proteins were capable of forming virus-like particles (VLPs) when coexpressed with E, whereas E(121)R and E(121)K proteins were not. Alanine substitutions for the first four or the last four residues resulted in viruses with significantly crippled phenotypes and proteins that failed to assemble VLPs or to be rescued into the envelope. All recovered viruses with alanine substitutions in place of SWWS residues had second-site, partially compensating, changes in the first TM of M. Alanine substitution for proline had little impact on the virus. N protein coexpression with some M mutants increased VLP production. The results overall suggest that the CD is important for formation of the viral envelope by helping mediate fundamental M-M interactions and that the presence of the N protein may help stabilize M complexes during virus assembly.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus nucleocapsid proteins assemble constitutively in high molecular oligomers</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoV) are enveloped viruses and rely on their nucleocapsid N protein to incorporate the positive-stranded genomic RNA into the virions. CoV N proteins form oligomers but the mechanism and relevance underlying their multimerization remain to be fully understood. Using in vitro pull-down experiments and density glycerol gradients, we found that at least 3 regions distributed over its entire length mediate the self-interaction of mouse hepatitis virus (MHV) and severe acute respiratory syndrome coronavirus (SARS-CoV) N protein. The fact that these regions can bind reciprocally between themselves provides a possible molecular basis for N protein oligomerization. Interestingly, cytoplasmic N molecules of MHV-infected cells constitutively assemble into oligomers through a process that does not require binding to genomic RNA. Based on our data, we propose a model where constitutive N protein oligomerization allows the optimal loading of the genomic viral RNA into a ribonucleoprotein complex via the presentation of multiple viral RNA binding motifs.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus nucleocapsid proteins assemble constitutively in high molecular oligomers</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoV) are enveloped viruses and rely on their nucleocapsid N protein to incorporate the positive-stranded genomic RNA into the virions. CoV N proteins form oligomers but the mechanism and relevance underlying their multimerization remain to be fully understood. Using in vitro pull-down experiments and density glycerol gradients, we found that at least 3 regions distributed over its entire length mediate the self-interaction of mouse hepatitis virus (MHV) and severe acute respiratory syndrome coronavirus (SARS-CoV) N protein. The fact that these regions can bind reciprocally between themselves provides a possible molecular basis for N protein oligomerization. Interestingly, cytoplasmic N molecules of MHV-infected cells constitutively assemble into oligomers through a process that does not require binding to genomic RNA. Based on our data, we propose a model where constitutive N protein oligomerization allows the optimal loading of the genomic viral RNA into a ribonucleoprotein complex via the presentation of multiple viral RNA binding motifs.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Role of the Coronavirus E Viroporin Protein Transmembrane Domain in Virus Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus envelope (E) proteins are small (∼75- to 110-amino-acid) membrane proteins that have a short hydrophilic amino terminus, a relatively long hydrophobic membrane domain, and a long hydrophilic carboxy-terminal domain. The protein is a minor virion structural component that plays an important, not fully understood role in virus production. It was recently demonstrated that the protein forms ion channels. We investigated the importance of the hydrophobic domain of the mouse hepatitis coronavirus (MHV) A59 E protein. Alanine scanning insertion mutagenesis was used to examine the effect of disruption of the domain on virus production in the context of the virus genome by using a MHV A59 infectious clone. Mutant viruses exhibited smaller plaque phenotypes, and virus production was significantly crippled. Analysis of recovered viruses suggested that the structure of the presumed α-helical structure and positioning of polar hydrophilic residues within the predicted transmembrane domain are important for virus production. Generation of viruses with restored wild-type helical pitch resulted in increased virus production, but some exhibited decreased virus release. Viruses with the restored helical pitch were more sensitive to treatment with the ion channel inhibitor hexamethylene amiloride than were the more crippled parental viruses with the single alanine insertions, suggesting that disruption of the transmembrane domain affects the functional activity of the protein. Overall the results indicate that the transmembrane domain plays a crucial role during biogenesis of virions.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>SEVERE ACUTE RESPIRATORY SYNDROME CORONAVIRUS E PROTEIN TRANSPORTS CALCIUM IONS AND ACTIVATES THE NLRP3 INFLAMMASOME</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus (SARS-CoV) envelope (E) protein is a viroporin involved in virulence. E protein ion channel (IC) activity is specifically correlated with enhanced pulmonary damage, edema accumulation and death. IL-1β driven proinflammation is associated with those pathological signatures, however its link to IC activity remains unknown. In this report, we demonstrate that SARS-CoV E protein forms protein-lipid channels in ERGIC/Golgi membranes that are permeable to calcium ions, a highly relevant feature never reported before. Calcium ions together with pH modulated E protein pore charge and selectivity. Interestingly, E protein IC activity boosted the activation of the NLRP3 inflammasome, leading to IL-1β overproduction. Calcium transport through the E protein IC was the main trigger of this process. These findings strikingly link SARS-CoV E protein IC induced ionic disturbances at the cell level to immunopathological consequences and disease worsening in the infected organism.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Supramolecular Architecture of Severe Acute Respiratory Syndrome Coronavirus Revealed by Electron Cryomicroscopy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus particles are enveloped and pleomorphic and are thus refractory to crystallization and symmetry-assisted reconstruction. A novel methodology of single-particle image analysis was applied to selected virus features to obtain a detailed model of the oligomeric state and spatial relationships among viral structural proteins. Two-dimensional images of the S, M, and N structural proteins of severe acute respiratory syndrome coronavirus and two other coronaviruses were refined to a resolution of ∼4 nm. Proteins near the viral membrane were arranged in overlapping lattices surrounding a disordered core. Trimeric glycoprotein spikes were in register with four underlying ribonucleoprotein densities. However, the spikes were dispensable for ribonucleoprotein lattice formation. The ribonucleoprotein particles displayed coiled shapes when released from the viral membrane. Our results contribute to the understanding of the assembly pathway used by coronaviruses and other pleomorphic viruses and provide the first detailed view of coronavirus ultrastructure.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of In Vivo-Interacting Domains of the Murine Coronavirus Nucleocapsid Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid protein (N), together with the large, positive-strand RNA viral genome, forms a helically symmetric nucleocapsid. This ribonucleoprotein structure becomes packaged into virions through association with the carboxy-terminal endodomain of the membrane protein (M), which is the principal constituent of the virion envelope. Previous work with the prototype coronavirus mouse hepatitis virus (MHV) has shown that a major determinant of the N-M interaction maps to the carboxy-terminal domain 3 of the N protein. To explore other domain interactions of the MHV N protein, we expressed a series of segments of the MHV N protein as fusions with green fluorescent protein (GFP) during the course of viral infection. We found that two of these GFP-N-domain fusion proteins were selectively packaged into virions as the result of tight binding to the N protein in the viral nucleocapsid, in a manner that did not involve association with either M protein or RNA. The nature of each type of binding was further explored through genetic analysis. Our results defined two strongly interacting regions of the N protein. One is the same domain 3 that is critical for M protein recognition during assembly. The other is domain N1b, which corresponds to the N-terminal domain that has been structurally characterized in detail for two other coronaviruses, infectious bronchitis virus and the severe acute respiratory syndrome coronavirus.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Palmitoylation of SARS-CoV S protein is necessary for partitioning into detergent-resistant membranes and cell-cell fusion but not interaction with M protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are enveloped RNA viruses that generally cause mild disease in humans. However, the recently emerged coronavirus that caused severe acute respiratory syndrome (SARS-CoV) is the most pathogenic human coronavirus discovered to date. The SARS-CoV spike (S) protein mediates virus entry by binding cellular receptors and inducing fusion between the viral envelope and the host cell membrane. Coronavirus S proteins are palmitoylated, which may affect function. Here, we created a non-palmitoylated SARS-CoV S protein by mutating all nine cytoplasmic cysteine residues. Palmitoylation of SARS-CoV S was required for partitioning into detergent-resistant membranes and for cell-cell fusion. Surprisingly, however, palmitoylation of S was not required for interaction with SARS-CoV M protein. This contrasts with the requirement for palmitoylation of mouse hepatitis virus S protein for interaction with M protein, and may point to important differences in assembly and infectivity of these two coronaviruses.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>The Structure Analysis and Antigenicity Study of the N Protein of SARS-CoV</h3>
<i class="tooltip">2003<span class="tooltiptext">2003 May 28</span></i><p style="text-align: justify; text-justify: inter-word;">The Coronaviridae family is characterized by a nucleocapsid that is composed of the genome RNA molecule in combination with the nucleoprotein (N protein) within a virion. The most striking physiochemical feature of the N protein of SARS-CoV is that it is a typical basic protein with a high predicted pI and high hydrophilicity, which is consistent with its function of binding to the ribophosphate backbone of the RNA molecule. The predicted high extent of phosphorylation of the N protein on multiple candidate phosphorylation sites demonstrates that it would be related to important functions, such as RNA-binding and localization to the nucleolus of host cells. Subsequent study shows that there is an SR-rich region in the N protein and this region might be involved in the protein-protein interaction. The abundant antigenic sites predicted in the N protein, as well as experimental evidence with synthesized polypeptides, indicate that the N protein is one of the major antigens of the SARS-CoV. Compared with other viral structural proteins, the low variation rate of the N protein with regards to its size suggests its importance to the survival of the virus.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation of coronavirus envelope glycoproteins and interaction with the viral nucleocapsid.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The two envelope glycoproteins and the viral nucleocapsid of the coronavirus A59 were isolated by solubilization of the viral membrane with Nonidet P-40 at 4 degrees C followed by sucrose density gradient sedimentation. Isolated E2 consisted of rosettes of peplomers, whereas E1, the membrane glycoprotein, was irregular and amorphous. Under certain conditions significant interactions occurred between components of Nonidet P-40-disrupted virions. Incubation of the Nonidet P-40-disrupted virus at 37 degrees C resulted in formation of a complex between one of the viral glycoproteins, E1, and the viral nucleocapsid. This was caused by a temperature-dependent conformational change in E1, resulting in aggregation of E1 and interaction with the viral RNA in the nucleocapsid. E1 also bound rRNA. The E1-nucleocapsid complexes can be distinguished on sucrose and Renografin density gradients from native viral nucleocapsids. The separation of the membrane glycoprotein E1 from the peplomeric glycoprotein E2 permitted preparation of antisera against these isolated proteins. A model is proposed for the arrangement of the three major structural proteins in the coronavirus A59 virion in relation to the viral envelope and RNA.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Nuclear Magnetic Resonance Structure of the Nucleic Acid-Binding Domain of Severe Acute Respiratory Syndrome Coronavirus Nonstructural Protein 3</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The nuclear magnetic resonance (NMR) structure of a globular domain of residues 1071 to 1178 within the previously annotated nucleic acid-binding region (NAB) of severe acute respiratory syndrome coronavirus nonstructural protein 3 (nsp3) has been determined, and N- and C-terminally adjoining polypeptide segments of 37 and 25 residues, respectively, have been shown to form flexibly extended linkers to the preceding globular domain and to the following, as yet uncharacterized domain. This extension of the structural coverage of nsp3 was obtained from NMR studies with an nsp3 construct comprising residues 1066 to 1181 [nsp3(1066-1181)] and the constructs nsp3(1066-1203) and nsp3(1035-1181). A search of the protein structure database indicates that the globular domain of the NAB represents a new fold, with a parallel four-strand β-sheet holding two α-helices of three and four turns that are oriented antiparallel to the β-strands. Two antiparallel two-strand β-sheets and two 3(10)-helices are anchored against the surface of this barrel-like molecular core. Chemical shift changes upon the addition of single-stranded RNAs (ssRNAs) identified a group of residues that form a positively charged patch on the protein surface as the binding site responsible for the previously reported affinity for nucleic acids. This binding site is similar to the ssRNA-binding site of the sterile alpha motif domain of the Saccharomyces cerevisiae Vts1p protein, although the two proteins do not share a common globular fold.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Palmitoylations on Murine Coronavirus Spike Proteins Are Essential for Virion Assembly and Infectivity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus spike (S) proteins are palmitoylated at several cysteine residues clustered near their transmembrane-spanning domains. This is achieved by cellular palmitoyl acyltransferases (PATs), which can modify newly synthesized S proteins before they are assembled into virion envelopes at the intermediate compartment of the exocytic pathway. To address the importance of these fatty acylations to coronavirus infection, we exposed infected cells to 2-bromopalmitate (2-BP), a specific PAT inhibitor. 2-BP profoundly reduced the specific infectivities of murine coronaviruses at very low, nontoxic doses that were inert to alphavirus and rhabdovirus infections. 2-BP effected only two- to fivefold reductions in S palmitoylation, yet this correlated with reduced S complexing with virion membrane (M) proteins and consequent exclusion of S from virions. At defined 2-BP doses, underpalmitoylated S proteins instead trafficked to infected cell surfaces and elicited cell-cell membrane fusions, suggesting that the acyl chain adducts are more critical to virion assembly than to S-induced syncytial developments. These studies involving pharmacologic inhibition of S protein palmitoylation were complemented with molecular genetic analyses in which cysteine acylation substrates were mutated. Notably, some mutations (C1347F and C1348S) did not interfere with S incorporation into virions, indicating that only a subset of the cysteine-rich region provides the essential S-assembly functions. However, the C1347F/C1348S mutant viruses exhibited relatively low specific infectivities, similar to virions secreted from 2-BP-treated cultures. Our collective results indicate that the palmitate adducts on coronavirus S proteins are necessary in assembly and also in positioning the assembled envelope proteins for maximal infectivity.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural basis for human coronavirus attachment to sialic acid receptors</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jun 3</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses cause respiratory tract infections in humans and outbreaks of deadly pneumonia worldwide. Infections are initiated by the transmembrane spike (S) glycoprotein, which binds to host receptors and fuses the viral and cellular membranes. To understand the molecular basis of coronavirus attachment to oligosaccharide receptors, we determined cryo-EM structures of coronavirus OC43 S glycoprotein trimer in isolation and in complex with a 9-O-acetylated sialic acid. We demonstrate that the ligand binds with fast kinetics to a surface-exposed groove and interactions at the identified site are essential for S-mediated viral entry into host cells, but free monosaccharide did not trigger fusogenic conformational changes. The receptor-interacting site is conserved in all coronavirus S glycoproteins that engage 9-O-acetyl-sialogycans, with an architecture similar to the ligand-binding pockets of coronavirus hemagglutinin esterases and influenza virus C/D hemagglutinin-esterase-fusion glycoproteins. Our results demonstrate these viruses evolved similar strategies to engage sialoglycans at the surface of target cells.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of the post-fusion core of the Human coronavirus 229E spike protein at 1.86 Å resolution</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Sep 3</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus 229E (HCoV-229E) usually causes mild upper respiratory infections in heathy adults, but may lead to severe complications or mortality in individuals with weakened immune systems. Virus entry of HCoV-229E is mediated by its spike (S) protein, where the S1 domain facilitates attachment to host cells and the S2 domain is involved in subsequent fusion of the virus and host membranes. During the fusion process, two heptad repeats, HR1 and HR2, in the S2 domain assemble into a six-helix membrane-fusion structure termed the fusion core. Here, the complete fusion-core structure of HCoV-229E has been determined at 1.86 Å resolution, representing the most complete post-fusion conformation thus far among published human alphacoronavirus (α-HCoV) fusion-core structures. The overall structure of the HCoV-229E fusion core is similar to those of SARS, MERS and HCoV-NL63, but the packing of its 3HR1 core differs from those of SARS and MERS in that it contains more noncanonical ‘x’ and ‘da’ layers. Side-by-side electrostatic surface comparisons reveal that the electrostatic surface potentials are opposite in α-HCoVs and β-HCoVs at certain positions and that the HCoV-229E surface also appears to be the most hydrophobic among the various HCoVs. In addition to the highly conserved hydrophobic interactions between HR1 and HR2, some polar and electrostatic interactions are also well preserved across different HCoVs. This study adds to the structural profiling of HCoVs to aid in the structure-based design of pan-coronavirus small molecules or peptides to inhibit viral fusion.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Mass Spectroscopic Characterization of the Coronavirus Infectious Bronchitis Virus Nucleoprotein and Elucidation of the Role of Phosphorylation in RNA Binding by Using Surface Plasmon Resonance</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Phosphorylation of the coronavirus nucleoprotein (N protein) has been predicted to play a role in RNA binding. To investigate this hypothesis, we examined the kinetics of RNA binding between nonphosphorylated and phosphorylated infectious bronchitis virus N protein with nonviral and viral RNA by surface plasmon resonance (Biacore). Mass spectroscopic analysis of N protein identified phosphorylation sites that were proximal to RNA binding domains. Kinetic analysis, by surface plasmon resonance, indicated that nonphosphorylated N protein bound with the same affinity to viral RNA as phosphorylated N protein. However, phosphorylated N protein bound to viral RNA with a higher binding affinity than nonviral RNA, suggesting that phosphorylation of N protein determined the recognition of virus RNA. The data also indicated that a known N protein binding site (involved in transcriptional regulation) consisting of a conserved core sequence present near the 5′ end of the genome (in the leader sequence) functioned by promoting high association rates of N protein binding. Further analysis of the leader sequence indicated that the core element was not the only binding site for N protein and that other regions functioned to promote high-affinity binding.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Role of Spike Protein Endodomains in Regulating Coronavirus Entry</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Enveloped viruses enter cells by viral glycoprotein-mediated binding to host cells and subsequent fusion of virus and host cell membranes. For the coronaviruses, viral spike (S) proteins execute these cell entry functions. The S proteins are set apart from other viral and cellular membrane fusion proteins by their extensively palmitoylated membrane-associated tails. Palmitate adducts are generally required for protein-mediated fusions, but their precise roles in the process are unclear. To obtain additional insights into the S-mediated membrane fusion process, we focused on these acylated carboxyl-terminal intravirion tails. Substituting alanines for the cysteines that are subject to palmitoylation had effects on both S incorporation into virions and S-mediated membrane fusions. In specifically dissecting the effects of endodomain mutations on the fusion process, we used antiviral heptad repeat peptides that bind only to folding intermediates in the S-mediated fusion process and found that mutants lacking three palmitoylated cysteines remained in transitional folding states nearly 10 times longer than native S proteins. This slower refolding was also reflected in the paucity of postfusion six-helix bundle configurations among the mutant S proteins. Viruses with fewer palmitoylated S protein cysteines entered cells slowly and had reduced specific infectivities. These findings indicate that lipid adducts anchoring S proteins into virus membranes are necessary for the rapid, productive S protein refolding events that culminate in membrane fusions. These studies reveal a previously unappreciated role for covalently attached lipids on the endodomains of viral proteins eliciting membrane fusion reactions.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Infectious Bronchitis Virus Nucleoprotein Localizes to the Nucleolus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleoprotein (N) has been reported to be involved in various aspects of virus replication. We examined by confocal microscopy the subcellular localization of the avian infectious bronchitis virus N protein both in the absence and in the context of an infected cell and found that N protein localizes both to the cytoplasmic and nucleolar compartments.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe acute respiratory syndrome coronavirus (SARS-CoV) infection inhibition using spike protein heptad repeat-derived peptides</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus SARS-CoV is the primary cause of the life-threatening severe acute respiratory syndrome (SARS). With the aim of developing therapeutic agents, we have tested peptides derived from the membrane-proximal (HR2) and membrane-distal (HR1) heptad repeat region of the spike protein as inhibitors of SARS-CoV infection of Vero cells. It appeared that HR2 peptides, but not HR1 peptides, were inhibitory. Their efficacy was, however, significantly lower than that of corresponding HR2 peptides of the murine coronavirus mouse hepatitis virus (MHV) in inhibiting MHV infection. Biochemical and electron microscopical analyses showed that, when mixed, SARS-CoV HR1 and HR2 peptides assemble into a six-helix bundle consisting of HR1 as a central triple-stranded coiled coil in association with three HR2 α-helices oriented in an antiparallel manner. The stability of this complex, as measured by its resistance to heat dissociation, appeared to be much lower than that of the corresponding MHV complex, which may explain the different inhibitory potencies of the HR2 peptides. Analogous to other class I viral fusion proteins, the six-helix complex supposedly represents a postfusion conformation that is formed after insertion of the fusion peptide, proposed here for coronaviruses to be located immediately upstream of HR1, into the target membrane. The resulting close apposition of fusion peptide and spike transmembrane domain facilitates membrane fusion. The inhibitory potency of the SARS-CoV HR2-peptides provides an attractive basis for the development of a therapeutic drug for SARS.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Model of a Putative Pore: The Pentameric α-Helical Bundle of SARS Coronavirus E Protein in Lipid Bilayers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus responsible for the severe acute respiratory syndrome contains a small envelope protein, E, with putative involvement in host apoptosis and virus morphogenesis. To perform these functions, it has been suggested that protein E can form a membrane destabilizing transmembrane (TM) hairpin, or homooligomerize to form a TM pore. Indeed, in a recent study we reported that the α-helical putative transmembrane domain of E protein (ETM) forms several SDS-resistant TM interactions: a dimer, a trimer, and two pentameric forms. Further, these interactions were found to be evolutionarily conserved. Herein, we have studied multiple isotopically labeled ETM peptides reconstituted in model lipid bilayers, using the orientational parameters derived from infrared dichroic data. We show that the topology of ETM is consistent with a regular TM α-helix. Further, the orientational parameters obtained unequivocally correspond to a homopentameric model, by comparison with previous predictions. We have independently confirmed that the full polypeptide of E protein can also aggregate as pentamers after expression in Escherichia coli. This interaction must be stabilized, at least partially, at the TM domain. The model we report for this pentameric α-helical bundle may explain some of the permabilizing properties of protein E, and should be the basis of mutagenesis efforts in future functional studies.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Crowning Proteins: Modulating the Protein Surface Properties using Crown Ethers**</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 24</span></i><p style="text-align: justify; text-justify: inter-word;">Crown ethers are small, cyclic polyethers that have found wide-spread use in phase-transfer catalysis and, to a certain degree, in protein chemistry. Crown ethers readily bind metallic and organic cations, including positively charged amino acid side chains. We elucidated the crystal structures of several protein-crown ether co-crystals grown in the presence of 18-crown-6. We then employed biophysical methods and molecular dynamics simulations to compare these complexes with the corresponding apoproteins and with similar complexes with ring-shaped low-molecular-weight polyethylene glycols. Our studies show that crown ethers can modify protein surface behavior dramatically by stabilizing either intra- or intermolecular interactions. Consequently, we propose that crown ethers can be used to modulate a wide variety of protein surface behaviors, such as oligomerization, domain–domain interactions, stabilization in organic solvents, and crystallization.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Crowning Proteins: Modulating the Protein Surface Properties using Crown Ethers**</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 24</span></i><p style="text-align: justify; text-justify: inter-word;">Crown ethers are small, cyclic polyethers that have found wide-spread use in phase-transfer catalysis and, to a certain degree, in protein chemistry. Crown ethers readily bind metallic and organic cations, including positively charged amino acid side chains. We elucidated the crystal structures of several protein-crown ether co-crystals grown in the presence of 18-crown-6. We then employed biophysical methods and molecular dynamics simulations to compare these complexes with the corresponding apoproteins and with similar complexes with ring-shaped low-molecular-weight polyethylene glycols. Our studies show that crown ethers can modify protein surface behavior dramatically by stabilizing either intra- or intermolecular interactions. Consequently, we propose that crown ethers can be used to modulate a wide variety of protein surface behaviors, such as oligomerization, domain–domain interactions, stabilization in organic solvents, and crystallization.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Assembly of coronavirus spike protein into trimers and its role in epitope expression.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The folding and oligomerization of coronavirus spike protein were explored using a panel of monoclonal antibodies. Chemical cross-linking and sedimentation experiments showed that the spike of transmissible gastroenteritis virus is a homotrimer of the S membrane glycoprotein. The spike protein was synthesized as a 175,000-apparent-molecular-weight (175K) monomer subunit that is sensitive to endo-beta-N-acetylglucosaminidase H. Assembly of monomers into a trimeric structure was found to occur on a partially trimmed polypeptide and to be a rate-limiting step, since large amounts of monomers failed to trimerize 1 h after completion of synthesis. Terminal glycosylation of newly assembled trimers, resulting in the biosynthesis of three 220K oligomers, occurred with a half time of approximately 20 min. Monomeric (230K to 240K) processed forms were also observed in cells and in virions. The 175K monomeric form expressed four major antigenic sites previously localized within the amino-terminal half of the S polypeptide chain; however, two classes of trimer-restricted epitopes (borne by three 220K and/or three 175K oligomers) were identified. The S glycoprotein of coronavirus might be a valuable model system for discovering new aspects of the maturation of membrane glycoproteins.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus N protein N-terminal domain (NTD) specifically binds the transcriptional regulatory sequence (TRS) and melts TRS-cTRS RNA duplexes</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">All coronaviruses (CoVs), including the causative agent of severe acute respiratory syndrome (SARS), encode a nucleocapsid (N) protein that harbors two independent RNA binding domains of known structure, but poorly characterized RNA binding properties. We show here that the N-terminal domain (NTD) of N protein from mouse hepatitis virus (MHV), a virus most closely related to SARS-CoV, employs aromatic amino acid-nucleobase stacking interactions with a triple adenosine motif to mediate high-affinity binding to single-stranded RNAs containing the transcriptional regulatory sequence (TRS) or its complement (cTRS). Stoichiometric NTD fully unwinds a TRS-cTRS duplex that mimics a transiently formed transcription intermediate in viral subgenomic RNA synthesis. Mutation of the solvent-exposed Y127, positioned on the β-platform surface of our 1.75 Å structure, binds the TRS far less tightly and is severely crippled in its RNA unwinding activity. In contrast, the C-terminal domain (CTD) exhibits no RNA unwinding activity. Viruses harboring Y127A N mutation are strongly selected against and Y127A N does not support an accessory function in MHV replication. We propose that the helix melting activity of the coronavirus N protein NTD plays a critical accessory role in subgenomic RNA synthesis and other processes requiring RNA remodeling.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Rigidity of the Outer Shell Predicted by a Protein Intrinsic Disorder Model Sheds Light on the COVID-19 (Wuhan-2019-nCoV) Infectivity</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The world is currently witnessing an outbreak of a new coronavirus spreading quickly across China and affecting at least 24 other countries. With almost 65,000 infected, a worldwide death toll of at least 1370 (as of 14 February 2020), and with the potential to affect up to two-thirds of the world population, COVID-19 is considered by the World Health Organization (WHO) to be a global health emergency. The speed of spread and infectivity of COVID-19 (also known as Wuhan-2019-nCoV) are dramatically exceeding those of the Middle East respiratory syndrome coronavirus (MERS-CoV) and severe acute respiratory syndrome coronavirus (SARS-CoV). In fact, since September 2012, the WHO has been notified of 2494 laboratory-confirmed cases of infection with MERS-CoV, whereas the 2002&ndash;2003 epidemic of SARS affected 26 countries and resulted in more than 8000 cases. Therefore, although SARS, MERS, and COVID-19 are all the result of coronaviral infections, the causes of the coronaviruses differ dramatically in their transmissibility. It is likely that these differences in infectivity of coronaviruses can be attributed to the differences in the rigidity of their shells which can be evaluated using computational tools for predicting intrinsic disorder predisposition of the corresponding viral proteins.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Assembly in vitro of a spanning membrane protein of the endoplasmic reticulum: the E1 glycoprotein of coronavirus mouse hepatitis virus A59.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The E1 glycoprotein of coronavirus mouse hepatitis virus A59 was synthesized in vitro by translation of viral mRNA in the presence of dog pancreatic microsomes. Its disposition in the membrane was investigated by digestion with proteases and by selective NH2-terminal labeling. The protein spans the membrane, but only small portions from the NH2 and COOH terminus are exposed respectively in the lumenal and cytoplasmic domains; the bulk of the molecule is apparently buried in the membrane. The protein lacks a cleavable leader sequence and does not acquire its characteristic O-linked oligosaccharides in rough microsomes. It may enter the membrane at any stage during synthesis of the first 150 amino acid residues. These unusual features of the protein might help to explain why it is not transported to the cell surface in vivo but remains in intracellular membranes, causing the virus to bud there.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>Topographic Changes in SARS Coronavirus–infected Cells at Late Stages of Infection</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Scanning electron and atomic force microscopy was used for the first time to view the maturation of the severe acute respiratory syndrome–associated coronavirus at the cell surface. The surface form of the cells at advanced infection displayed prolific pseudopodia that, in addition to the rest of the plasma membrane, were also active sites of virus release. High magnification of the maturing virus particles showed a rosette appearance with short knoblike spikes under both the scanning electron and atomic force microscopes. The final expulsion step of the maturing virus particles seemed to result in some disruptions to the plasma membrane. The cytoskeletal network along the edge of the infected cells was enhanced and could be involved in transporting and expelling the progeny virus particles. Thickening of the actin filaments at the cell edge provided the bending force to extrude the virus particles.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and immune recognition of the porcine epidemic diarrhea virus spike protein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-19</span></i><p style="text-align: justify; text-justify: inter-word;">Porcine epidemic diarrhea virus is an alphacoronavirus responsible for significant morbidity and mortality in pigs. A key determinant of viral tropism and entry, the PEDV spike protein is a key target for the host antibody response and a good candidate for a protein-based vaccine immunogen. We used electron microscopy to evaluate the PEDV spike structure, as well as pig polyclonal antibody responses to viral infection. The structure of the PEDV spike reveals a configuration similar to that of HuCoV-NL63. Several PEDV protein-protein interfaces are mediated by non-protein components including a glycan at Asn264 and two bound palmitoleic acid molecules. The polyclonal antibody response to PEDV infection shows a dominance of epitopes in the S1 region. This structural and immune characterization provides new insights into coronavirus spike stability determinants and explores the immune landscape of viral spike proteins.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>The human coronavirus HCoV-229E S-protein structure and receptor binding</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus S-protein mediates receptor binding and fusion of the viral and host cell membranes. In HCoV-229E, its receptor binding domain (RBD) shows extensive sequence variation but how S-protein function is maintained is not understood. Reported are the X-ray crystal structures of Class III-V RBDs in complex with human aminopeptidase N (hAPN), as well as the electron cryomicroscopy structure of the 229E S-protein. The structures show that common core interactions define the specificity for hAPN and that the peripheral RBD sequence variation is accommodated by loop plasticity. The results provide insight into immune evasion and the cross-species transmission of 229E and related coronaviruses. We also find that the 229E S-protein can expose a portion of its helical core to solvent. This is undoubtedly facilitated by hydrophilic subunit interfaces that we show are conserved among coronaviruses. These interfaces likely play a role in the S-protein conformational changes associated with membrane fusion.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>NMR assignment of the nonstructural protein nsp3(1066–1181) from SARS-CoV</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Sequence-specific NMR assignments of the globular core comprising the residues 1066–1181 within the non-structural protein nsp3e from the SARS coronavirus have been obtained using triple-resonance NMR experiments with the uniformly [(13)C,(15)N]-labeled protein. The backbone and side chain assignments are nearly complete, providing the basis for the ongoing NMR structure determination. A preliminary identification of regular secondary structures has been derived from the (13)C chemical shifts.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>Receptor-Induced Conformational Changes of Murine Coronavirus Spike Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although murine coronavirus mouse hepatitis virus (MHV) enters cells by virus-cell membrane fusion triggered by its spike (S) protein, it is not well known how the S protein participates in fusion events. We reported that the soluble form of MHV receptor (soMHVR) transformed a nonfusogenic S protein into a fusogenic one (F. Taguchi and S. Matsuyama, J. Virol. 76:950-958, 2002). In the present study, we demonstrate that soMHVR induces the conformational changes of the S protein, as shown by the proteinase digestion test. A cl-2 mutant, srr7, of the MHV JHM virus (JHMV) was digested with proteinase K after treatment with soMHVR, and the resultant S protein was analyzed by Western blotting using monoclonal antibody (MAb) 10G, specific for the membrane-anchored S2 subunit. A 58-kDa fragment, encompassing the two heptad repeats in S2, was detected when srr7 was digested after soMHVR treatment, while no band was seen when the virus was untreated. The appearance of the proteinase-resistant fragment was dependent on the temperature and time of srr7 incubation with soMHVR and also on the concentration of soMHVR. Coimmunoprecipitation indicated that the direct binding of soMHVR to srr7 S protein induced these conformational changes; this was also suggested by the inhibition of the changes following pretreatment of soMHVR with anti-MHVR MAb CC1. soMHVR induced conformational changes of the S proteins of wild-type (wt) JHMV cl-2, as well as revertants from srr7, srr7A and srr7B; however, a major proportion of these S proteins were resistant to proteinase K even without soMHVR treatment. The implications of this proteinase-resistant fraction are discussed. This is the first report on receptor-induced conformational changes of the membrane-anchored fragment of the coronavirus S protein.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of an immunodominant linear neutralization domain on the S2 portion of the murine coronavirus spike glycoprotein and evidence that it forms part of complex tridimensional structure.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Numerous studies have demonstrated that the spike glycoprotein of coronaviruses bears major determinants of pathogenesis. To elucidate the antigenic structure of the protein, a panel of monoclonal antibodies was studied by competitive ELISA, and their reactivities were assayed against fragments of the murine coronavirus murine hepatitis virus strain A59 S gene expressed in prokaryotic vectors. An immunodominant linear domain was localized within the predicted stalk, S2, of the peplomer. It is recognized by several neutralizing antibodies. Other domains were also identified near the proteolytic cleavage site, in the predicted globular head, S1, and in another part of the stalk. Furthermore, competition results suggest that the immunodominant functional domain forms part of a complex three-dimensional structure. Surprisingly, some antibodies which have no antiviral biological activities were shown to bind the immunodominant neutralization domain.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Spike Protein Is a Class I Virus Fusion Protein: Structural and Functional Characterization of the Fusion Core Complex</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus entry is mediated by the viral spike (S) glycoprotein. The 180-kDa oligomeric S protein of the murine coronavirus mouse hepatitis virus strain A59 is posttranslationally cleaved into an S1 receptor binding unit and an S2 membrane fusion unit. The latter is thought to contain an internal fusion peptide and has two 4,3 hydrophobic (heptad) repeat regions designated HR1 and HR2. HR2 is located close to the membrane anchor, and HR1 is some 170 amino acids (aa) upstream of it. Heptad repeat (HR) regions are found in fusion proteins of many different viruses and form an important characteristic of class I viral fusion proteins. We investigated the role of these regions in coronavirus membrane fusion. Peptides HR1 (96 aa) and HR2 (39 aa), corresponding to the HR1 and HR2 regions, were produced in Escherichia coli. When mixed together, the two peptides were found to assemble into an extremely stable oligomeric complex. Both on their own and within the complex, the peptides were highly alpha helical. Electron microscopic analysis of the complex revealed a rod-like structure ∼14.5 nm in length. Limited proteolysis in combination with mass spectrometry indicated that HR1 and HR2 occur in the complex in an antiparallel fashion. In the native protein, such a conformation would bring the proposed fusion peptide, located in the N-terminal domain of HR1, and the transmembrane anchor into close proximity. Using biological assays, the HR2 peptide was shown to be a potent inhibitor of virus entry into the cell, as well as of cell-cell fusion. Both biochemical and functional data show that the coronavirus spike protein is a class I viral fusion protein.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Evidence for a Structural Interaction between the Carboxy Termini of the Membrane and Nucleocapsid Proteins of Mouse Hepatitis Virus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein is the most abundant virion protein and the key component in viral assembly and morphogenesis. The M protein of mouse hepatitis virus (MHV) is an integral membrane protein with a short ectodomain, three transmembrane segments, and a large carboxy-terminal endodomain facing the interior of the viral envelope. The carboxy terminus of MHV M has previously been shown to be extremely sensitive to mutation, both in a virus-like particle expression system and in the intact virion. We have constructed a mutant, MΔ2, containing a two-amino-acid truncation of the M protein that was previously thought to be lethal. This mutant was isolated by means of targeted RNA recombination with a powerful host range-based selection allowed by the interspecies chimeric virus fMHV (MHV containing the ectodomain of the feline infectious peritonitis virus S protein). Analysis of multiple second-site revertants of the MΔ2 mutant has revealed changes in regions of both the M protein and the nucleocapsid (N) protein that can compensate for the loss of the last two residues of the M protein. Our data thus provide the first genetic evidence for a structural interaction between the carboxy termini of the M and N proteins of MHV. In addition, this work demonstrates the efficacy of targeted recombination with fMHV for the systematic genetic analysis of coronavirus structural protein interactions.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of the Role of Hexamer AGUAAA and Poly(A) Tail in Coronavirus Polyadenylation</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Oct 19</span></i><p style="text-align: justify; text-justify: inter-word;">Similar to eukaryotic mRNA, the positive-strand coronavirus genome of ~30 kilobases is 5’-capped and 3’-polyadenylated. It has been demonstrated that the length of the coronaviral poly(A) tail is not static but regulated during infection; however, little is known regarding the factors involved in coronaviral polyadenylation and its regulation. Here, we show that during infection, the level of coronavirus poly(A) tail lengthening depends on the initial length upon infection and that the minimum length to initiate lengthening may lie between 5 and 9 nucleotides. By mutagenesis analysis, it was found that (i) the hexamer AGUAAA and poly(A) tail are two important elements responsible for synthesis of the coronavirus poly(A) tail and may function in concert to accomplish polyadenylation and (ii) the function of the hexamer AGUAAA in coronaviral polyadenylation is position dependent. Based on these findings, we propose a process for how the coronaviral poly(A) tail is synthesized and undergoes variation. Our results provide the first genetic evidence to gain insight into coronaviral polyadenylation.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Conductance and amantadine binding of a pore formed by a lysine-flanked transmembrane domain of SARS coronavirus envelope protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus responsible for the severe acute respiratory syndrome (SARS-CoV) contains a small envelope protein, E, with putative involvement in host cell apoptosis and virus morphogenesis. It has been suggested that E protein can form a membrane destabilizing transmembrane (TM) hairpin, or homooligomerize to form a regular TM α-helical bundle. We have shown previously that the topology of the α-helical putative TM domain of E protein (ETM), flanked by two lysine residues at C and N termini to improve solubility, is consistent with a regular TM α-helix, with orientational parameters in lipid bilayers that are consistent with a homopentameric model. Herein, we show that this peptide, reconstituted in lipid bilayers, shows sodium conductance. Channel activity is inhibited by the anti-influenza drug amantadine, which was found to bind our preparation with moderate affinity. Results obtained from single or double mutants indicate that the organization of the transmembrane pore is consistent with our previously reported pentameric α-helical bundle model.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Phosphoproteins of murine hepatitis viruses.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Four strains of the coronavirus murine hepatitis virus were examined for the presence of phosphorylated proteins. The nucleocapsid protein was determined to contain phosphate covalently linked to serine but not to threonine residues. The nucleocapsid protein was the only phosphorylated protein detected in these strains of murine hepatitis virus.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Regulation of the ER Stress Response by the Ion Channel Activity of the Infectious Bronchitis Coronavirus Envelope Protein Modulates Virion Release, Apoptosis, Viral Fitness, and Pathogenesis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 24</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus (CoV) envelope (E) protein is a small structural protein critical for virion morphogenesis and release. The recently characterized E protein ion channel activity (EIC) has also been implicated in modulating viral pathogenesis. In this study, we used infectious bronchitis coronavirus (IBV) as a model to study EIC. Two recombinant IBVs (rIBVs) harboring EIC-inactivating mutations – rT16A and rA26F – were serially passaged, and several compensatory mutations were identified in the transmembrane domain (TMD). Two rIBVs harboring these putative EIC-reverting mutations – rT16A/A26V and rA26F/F14N – were recovered. Compared with the parental rIBV-p65 control, all four EIC mutants exhibited comparable levels of intracellular RNA synthesis, structural protein production, and virion assembly. Our results showed that the IBV EIC contributed to the induction of ER stress response, as up-regulation of ER stress-related genes was markedly reduced in cells infected with the EIC-defective mutants. EIC-defective mutants also formed smaller plaques, released significantly less infectious virions into the culture supernatant, and had lower levels of viral fitness in cell culture. Significantly, all these defective phenotypes were restored in cells infected with the putative EIC revertants. EIC mutations were also implicated in regulating IBV-induced apoptosis, induction of pro-inflammatory cytokines, and viral pathogenicity in vivo. Taken together, this study highlights the importance of CoV EIC in modulating virion release and various aspects of CoV – host interaction.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Envelope Protein Palmitoylations Are Crucial for Murine Coronavirus Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus assembly process encloses a ribonucleoprotein genome into vesicles containing the lipid-embedded proteins S (spike), E (envelope), and M (membrane). This process depends on interactions with membranes that may involve palmitoylation, a common posttranslational lipidation of cysteine residues. To determine whether specific palmitoylations influence coronavirus assembly, we introduced plasmid DNAs encoding mouse hepatitis coronavirus (MHV) S, E, M, and N (nucleocapsid) into 293T cells and found that virus-like particles (VLPs) were robustly assembled and secreted into culture medium. Palmitate adducts predicted on cysteines 40, 44, and 47 of the 83-residue E protein were then evaluated by constructing mutant cDNAs with alanine or glycine codon substitutions at one or more of these positions. Triple-substituted proteins (E.Ts) lacked palmitate adducts. Both native E and E.T proteins localized at identical perinuclear locations, and both copurified with M proteins, but E.T was entirely incompetent for VLP production. In the presence of the E.T proteins, the M protein subunits accumulated into detergent-insoluble complexes that failed to secrete from cells, while native E proteins mobilized M into detergent-soluble secreted forms. Many of these observations were corroborated in the context of natural MHV infections, with native E, but not E.T, complementing debilitated recombinant MHVs lacking E. Our findings suggest that palmitoylations are essential for E to act as a vesicle morphogenetic protein and further argue that palmitoylated E proteins operate by allowing the primary coronavirus assembly subunits to assume configurations that can mobilize into secreted lipid vesicles and virions.</p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Envelope (E) Protein Remains at the Site of Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) assemble at endoplasmic reticulum Golgi intermediate compartment (ERGIC) membranes and egress from cells in cargo vesicles. Only a few molecules of the envelope (E) protein are assembled into virions. The role of E in morphogenesis is not fully understood. The cellular localization and dynamics of mouse hepatitis CoV A59 (MHV) E protein were investigated to further understanding of its role during infection. E protein localized in the ERGIC and Golgi with the amino and carboxy termini in the lumen and cytoplasm, respectively. E protein does not traffic to the cell surface. MHV was genetically engineered with a tetracysteine tag at the carboxy end of E. Fluorescence recovery after photobleaching (FRAP) showed that E is mobile in ERGIC/Golgi membranes. Correlative light electron microscopy (CLEM) confirmed the presence of E in Golgi cisternae. The results provide strong support that E proteins carry out their function(s) at the site of budding/assembly.</p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional Transcriptional Regulatory Sequence (TRS) RNA Binding and Helix Destabilizing Determinants of Murine Hepatitis Virus (MHV) Nucleocapsid (N) Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus (CoV) nucleocapsid (N) protein contains two structurally independent RNA binding domains. These are denoted N-terminal domain (NTD) and C-terminal domain and are joined by a charged linker region rich in serine and arginine residues (SR linker). In mouse hepatitis virus (MHV), the NTD binds the transcriptional regulatory sequence (TRS) RNA, a conserved hexanucleotide sequence required for subgenomic RNA synthesis. The NTD is also capable of disrupting a short RNA duplex. We show here that three residues on the β3 (Arg-125 and Tyr-127) and β5 (Tyr-190) strands play key roles in TRS RNA binding and helix destabilization with Ala substitutions of these residues lethal to the virus. NMR studies of the MHV NTD·TRS complex revealed that this region defines a major RNA binding interface in MHV with site-directed spin labeling studies consistent with a model in which the adenosine-rich 3′-region of TRS is anchored by Arg-125, Tyr-127, and Tyr-190 in a way that is critical for efficient subgenomic RNA synthesis in MHV. Characterization of CoV N NTDs from infectious bronchitis virus and from severe acute respiratory syndrome CoV revealed that, although detailed NTD-TRS determinants are distinct from those of MHV NTD, rapid helix destabilization activity of CoV N NTDs is most strongly correlated with CoV function and virus viability.</p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /><div class="card"><h3>Ca2+ ions promote fusion of Middle East Respiratory Syndrome coronavirus with host cells and increase infectivity</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-12-19</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a major emerging zoonotic infectious disease. Since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. To date, there are no efficacious drugs and vaccines against MERS-CoV, increasing its potential to cause a public health emergency. A critical step in the life cycle of MERS-CoV is the fusion with the host cell with its spike (S) protein as main determinant of viral entry. Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca2+) plays an important role for fusogenic activity via a Ca2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudo-particles (PPs) on host cells without and with Ca2+. Our results demonstrate that intracellular Ca2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca2+, as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca2+ ions interacting with the FP has implications on the fusion dynamics of the virus.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Vulnerabilities in coronavirus glycan shields despite extensive glycosylation</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-21</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses (CoVs) are zoonotic pathogens with high fatality rates and pandemic potential. Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Intracellular transport of recombinant coronavirus spike proteins: implications for virus assembly.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus spike protein genes were expressed in vitro by using the recombinant vaccinia virus expression system. Recombinant spike proteins were expressed at the cell surface and induced cell fusion in a host-cell-dependent fashion. The intracellular transport of recombinant spike proteins was studied. The half time of acquisition of resistance to endo-beta-N-acetylglucosaminidase H was approximately 3 h for the recombinant feline infectious peritonitis virus S protein. The S protein in feline infectious peritonitis virus-infected cells was found to have a half time of acquisition of resistance to endo-beta-N-acetylglucosaminidase H of approximately 1 h. This difference can be explained by the fact that coronavirus budding takes place at intracellular membranes and that the oligosaccharides of the spike protein are modified after budding. Apparently, spike protein incorporated into budded virions is transported faster through the Golgi apparatus than is spike protein alone. These findings provide new insights into the mechanism of coronavirus budding and are discussed in relation to current models of intracellular transport and sorting of proteins.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>The coronavirus nucleocapsid protein is ADP-ribosylated</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">ADP-ribosylation is a common post-translational modification, although how it modulates RNA virus infection is not well understood. While screening for ADP-ribosylated proteins during coronavirus (CoV) infection, we detected a ~55 kDa ADP-ribosylated protein in mouse hepatitis virus (MHV)-infected cells and in virions, which we identified as the viral nucleocapsid (N) protein. The N proteins of porcine epidemic diarrhea virus (PEDV), severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV were also ADP-ribosylated. ADP-ribosylation of N protein was also observed in cells exogenously expressing N protein by transduction using Venezuelan equine encephalitis virus replicon particles (VRPs). However, plasmid-derived N protein was not ADP-ribosylated following transient transfection but was ADP-ribosylated after MHV infection, indicating that this modification requires virus infection. In conclusion, we have identified a novel post-translation modification of the CoV N protein that may play a regulatory role for this important structural protein.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>The New Coronavirus, the Current King of China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are respiratory viruses whose appearance on electron microscopy looks like tiny crowns (Figure 1). The first coronaviruses discovered were the cause of pediatric and adult respiratory infections that were not particularly dangerous. One study from Vanderbilt found coronavirus in about 5% of samples from upper respiratory illness and 8% from lower respiratory illness [1]. Most clinically significant coronavirus infections have been found in children &lt; 2 years of age, as reviewed in this journal in 2009 [2], although adults also might suffer severe infections [3, 4]. This was the picture as late as 2018: a group of respiratory viruses that could cause serious illness in the very young, but with negligible mortality [5, 6].</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>The transmissible gastroenteritis coronavirus contains a spherical core shell consisting of M and N proteins.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are enveloped RNA viruses involved in a variety of pathologies that affect animals and humans. Existing structural models of these viruses propose a helical nucleocapsid under the virion envelope as the unique internal structure. In the present work, we have analyzed the structure of the transmissible gastroenteritis coronavirus. The definition of its organization supports a new structural model for coronaviruses, since a spherical, probably icosahedral, internal core has been characterized. Disruption of these cores induces the release of N-protein-containing helical nucleocapsids. Immunogold mapping and protein analysis of purified cores showed that they consist of M and N proteins, M being the main core shell component. This surprising finding, together with the fact that M protein molecules are also located in the virion envelope, indicates that a reconsideration of the assembly and maturation of coronaviruses, as well as a study of potential M-protein subclasses, is needed.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>NMR Structure of the SARS Coronavirus Nonstructural Protein 7 in Solution at pH 6.5</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The NMR structure of the severe acute respiratory syndrome coronavirus (SARS-CoV) non-structural protein 7 (nsp7) in aqueous solution at pH 6.5 was determined and compared with the results of previous structure determinations of nsp7 in solution at pH 7.5 and in crystals of a hexadecameric nsp7/nsp8 complex obtained from a solution at pH 7.5. All three structures contain four helices as the only regular secondary structures, but there are differences in the lengths and sequence locations of the four helices, as well as between the tertiary folds. The present study includes data on conformational equilibria and intramolecular rate processes in nsp7 in solution at pH 6.5, which provides further insights into the polymorphisms implicated by comparison of the presently available three nsp7 structures.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Nucleocapsid Protein Is Dynamically Associated with the Replication-Transcription Complexes</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) protein is a virion structural protein. It also functions, however, in an unknown way in viral replication and localizes to the viral replication-transcription complexes (RTCs). Here we investigated, using recombinant murine coronaviruses expressing green fluorescent protein (GFP)-tagged versions of the N protein, the dynamics of its interactions with the RTCs and the domain(s) involved. Using fluorescent recovery after photobleaching, we showed that the N protein, unlike the nonstructural protein 2, is dynamically associated with the RTCs. Recruitment of the N protein to the RTCs requires the C-terminal N2b domain, which interacts with other N proteins in an RNA-independent manner.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of the Membrane-Active Regions of the Severe Acute Respiratory Syndrome Coronavirus Spike Membrane Glycoprotein Using a 16/18-Mer Peptide Scan: Implications for the Viral Fusion Mechanism</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have identified the membrane-active regions of the severe acute respiratory syndrome coronavirus (SARS CoV) spike glycoprotein by determining the effect on model membrane integrity of a 16/18-mer SARS CoV spike glycoprotein peptide library. By monitoring the effect of this peptide library on membrane leakage in model membranes, we have identified three regions on the SARS CoV spike glycoprotein with membrane-interacting capabilities: region 1, located immediately upstream of heptad repeat 1 (HR1) and suggested to be the fusion peptide; region 2, located between HR1 and HR2, which would be analogous to the loop domain of human immunodeficiency virus type 1; and region 3, which would correspond to the pretransmembrane region. The identification of these membrane-active regions, which are capable of modifying the biophysical properties of phospholipid membranes, supports their direct role in SARS CoV-mediated membrane fusion, as well as facilitating the future development of SARS CoV entry inhibitors.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>X-Ray Structures of the N- and C-Terminal Domains of a Coronavirus Nucleocapsid Protein: Implications for Nucleocapsid Formation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses cause a variety of respiratory and enteric diseases in animals and humans including severe acute respiratory syndrome. In these enveloped viruses, the filamentous nucleocapsid is formed by the association of nucleocapsid (N) protein with single-stranded viral RNA. The N protein is a highly immunogenic phosphoprotein also implicated in viral genome replication and in modulating cell signaling pathways. We describe the structure of the two proteolytically resistant domains of the N protein from infectious bronchitis virus (IBV), a prototype coronavirus. These domains are located at its N- and C-terminal ends (NTD and CTD, respectively). The NTD of the IBV Gray strain at 1.3-Å resolution exhibits a U-shaped structure, with two arms rich in basic residues, providing a module for specific interaction with RNA. The CTD forms a tightly intertwined dimer with an intermolecular four-stranded central β-sheet platform flanked by α helices, indicating that the basic building block for coronavirus nucleocapsid formation is a dimeric assembly of N protein. The variety of quaternary arrangements of the NTD and CTD revealed by the analysis of the different crystal forms delineates possible interfaces that could be used for the formation of a flexible filamentous ribonucleocapsid. The striking similarity between the dimeric structure of CTD and the nucleocapsid-forming domain of a distantly related arterivirus indicates a conserved mechanism of nucleocapsid formation for these two viral families.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>CORONAVIRUS E PROTEIN FORMS ION CHANNELS WITH FUNCTIONALLY AND STRUCTURALLY-INVOLVED MEMBRANE LIPIDS</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus (CoV) envelope (E) protein ion channel activity was determined in channels formed in planar lipid bilayers by peptides representing either the transmembrane domain of severe acute respiratory syndrome CoV (SARS-CoV) E protein, or the full-length E protein. Both of them formed a voltage independent ion conductive pore with symmetric ion transport properties. Mutations N15A and V25F located in the transmembrane domain, prevented the ion conductivity. E protein derived channels showed no cation preference in non-charged lipid membranes, whereas they behaved as pores with mild cation selectivity in negatively-charged lipid membranes. The ion conductance was also controlled by the lipid composition of the membrane. Lipid charge also regulated the selectivity of a HCoV-229E E protein derived peptide. These results suggested that the lipids are functionally involved in E protein ion channel activity, forming a protein-lipid pore, a novel concept for CoV E protein ion channel entity.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Role of Endocytosis and Low pH in Murine Hepatitis Virus Strain A59 Cell Entry</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Infection by the coronavirus mouse hepatitis virus strain A59 (MHV-A59) requires the release of the viral genome by fusion with the respective target membrane of the host cell. Fusion is mediated by the viral S protein. Here, the entry pathway of MHV-A59 into murine fibroblast cells was studied by independent approaches. Infection of cells assessed by plaque reduction assay was strongly inhibited by lysosomotropic compounds and substances that interfere with clathrin-dependent endocytosis, suggesting that MHV-A59 is taken up via endocytosis and delivered to acidic endosomal compartments. Infection was only slightly reduced in the presence of substances inhibiting proteases of endosomal compartments, precluding that the endocytic uptake is required to activate the fusion potential of the S protein by its cleavage. Fluorescence confocal microscopy of labeled MHV-A59 confirmed that virus is taken up via endocytosis. Bright labeling of intracellular compartments suggests their fusion with the viral envelope. No fusion with the plasma membrane was observed at neutral pH conditions. However, when virus was bound to cells and the pH was lowered to 5.0, we observed a strong labeling of the plasma membrane. Electron microscopy revealed low pH triggered conformational alterations of the S ectodomain. Very likely, these alterations are irreversible because low-pH treatment of viruses in the absence of target membranes caused an irreversible loss of the fusion activity. The results imply that endocytosis plays a major role in MHV-A59 infection and the acidic pH of the endosomal compartment triggers a conformational change of the S protein mediating fusion.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>The Infectious Bronchitis Virus Coronavirus Envelope Protein Alters Golgi pH to Protect Spike Protein and Promote Release of Infectious Virus</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-10-11</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) are important human pathogens with significant zoonotic potential. Progress has been made toward identifying potential vaccine candidates for highly pathogenic human CoVs, including use of attenuated viruses that lack the CoV envelope (E) protein or express E mutants. However, no approved vaccines or anti-viral therapeutics exist. CoVs assemble by budding into the lumen of the early Golgi prior to exocytosis. The small CoV E protein plays roles in assembly, virion release, and pathogenesis. CoV E has a single hydrophobic domain (HD), is targeted to Golgi membranes, and has cation channel activity in vitro. The E protein from the avian infectious bronchitis virus (IBV) has dramatic effects on the secretory system, which requires residues in the HD. Mutation of the HD of IBV E during infection results in impaired growth kinetics, impaired release of infectious virions, accumulation of IBV S protein on the plasma membrane when compared IBV WT infected cells, and aberrant cleavage of IBV S on the surface of virions. We previously reported the formation of two distinct oligomeric pools of IBV E in transfected and infected cells. Disruption of the secretory pathway by IBV E correlates with a form that is likely monomeric, suggesting that the effects on the secretory pathway are independent of E ion channel activity. Here, we present evidence suggesting that the monomeric form of IBV E correlates with a rise in the pH of the Golgi lumen. We demonstrate that infection with IBV induces neutralization of Golgi luminal pH, promoting a model in which IBV E alters the secretory pathway through interaction with host cells factors, protecting IBV spike protein (S) from premature cleavage and leading to the efficient release of infectious virus from the cells.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV envelope protein palmitoylation or nucleocapid association is not required for promoting virus-like particle production</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Apr 27</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronavirus membrane (M) proteins are capable of interacting with nucleocapsid (N) and envelope (E) proteins. Severe acute respiratory syndrome coronavirus (SARS-CoV) M co-expression with either N or E is sufficient for producing virus-like particles (VLPs), although at a lower level compared to M, N and E co-expression. Whether E can release from cells or E/N interaction exists so as to contribute to enhanced VLP production is unknown. It also remains to be determined whether E palmitoylation or disulfide bond formation plays a role in SARS-CoV virus assembly. RESULTS: SARS-CoV N is released from cells through an association with E protein-containing vesicles. Further analysis suggests that domains involved in E/N interaction are largely located in both carboxyl-terminal regions. Changing all three E cysteine residues to alanines did not exert negative effects on E release, E association with N, or E enhancement of VLP production, suggesting that E palmitoylation modification or disulfide bond formation is not required for SARS-CoV virus assembly. We found that removal of the last E carboxyl-terminal residue markedly affected E release, N association, and VLP incorporation, but did not significantly compromise the contribution of E to efficient VLP production. CONCLUSIONS: The independence of the SARS-CoV E enhancement effect on VLP production from its viral packaging capacity suggests a distinct SARS-CoV E role in virus assembly.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>The Avian Coronavirus Infectious Bronchitis Virus Undergoes Direct Low-pH-Dependent Fusion Activation during Entry into Host Cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are the causative agents of respiratory disease in humans and animals, including severe acute respiratory syndrome. Fusion of coronaviruses is generally thought to occur at neutral pH, although there is also evidence for a role of acidic endosomes during entry of a variety of coronaviruses. Therefore, the molecular basis of coronavirus fusion during entry into host cells remains incompletely defined. Here, we examined coronavirus-cell fusion and entry employing the avian coronavirus infectious bronchitis virus (IBV). Virus entry into cells was inhibited by acidotropic bases and by other inhibitors of pH-dependent endocytosis. We carried out fluorescence-dequenching fusion assays of R18-labeled virions and show that for IBV, coronavirus-cell fusion occurs in a low-pH-dependent manner, with a half-maximal rate of fusion occurring at pH 5.5. Fusion was reduced, but still occurred, at lower temperatures (20°C). We observed no effect of inhibitors of endosomal proteases on the fusion event. These data are the first direct measure of virus-cell fusion for any coronavirus and demonstrate that the coronavirus IBV employs a direct, low-pH-dependent virus-cell fusion activation reaction. We further show that IBV was not inactivated, and fusion was unaffected, by prior exposure to pH 5.0 buffer. Virions also showed evidence of reversible conformational changes in their surface proteins, indicating that aspects of the fusion reaction may be reversible in nature.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The global outbreak of SARS in 2002-2003 was caused by the infection of a new human coronavirus SARS-CoV. The infection of SARS-CoV is mediated mainly through the viral surface glycoproteins, which consist of S1 and S2 subunits and form trimer spikes on the envelope of the virions. Here we report the ectodomain structures of the SARS-CoV surface spike trimer in different conformational states determined by single-particle cryo-electron microscopy. The conformation 1 determined at 4.3 Å resolution is three-fold symmetric and has all the three receptor-binding C-terminal domain 1 (CTD1s) of the S1 subunits in “down” positions. The binding of the “down” CTD1s to the SARS-CoV receptor ACE2 is not possible due to steric clashes, suggesting that the conformation 1 represents a receptor-binding inactive state. Conformations 2-4 determined at 7.3, 5.7 and 6.8 Å resolutions are all asymmetric, in which one RBD rotates away from the “down” position by different angles to an “up” position. The “up” CTD1 exposes the receptor-binding site for ACE2 engagement, suggesting that the conformations 2-4 represent a receptor-binding active state. This conformational change is also required for the binding of SARS-CoV neutralizing antibodies targeting the CTD1. This phenomenon could be extended to other betacoronaviruses utilizing CTD1 of the S1 subunit for receptor binding, which provides new insights into the intermediate states of coronavirus pre-fusion spike trimer during infection.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure, Function, and Evolution of Coronavirus Spike Proteins</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus spike protein is a multifunctional molecular machine that mediates coronavirus entry into host cells. It first binds to a receptor on the host cell surface through its S1 subunit and then fuses viral and host membranes through its S2 subunit. Two domains in S1 from different coronaviruses recognize a variety of host receptors, leading to viral attachment. The spike protein exists in two structurally distinct conformations, prefusion and postfusion. The transition from prefusion to postfusion conformation of the spike protein must be triggered, leading to membrane fusion. This article reviews current knowledge about the structures and functions of coronavirus spike proteins, illustrating how the two S1 domains recognize different receptors and how the spike proteins are regulated to undergo conformational transitions. I further discuss the evolution of these two critical functions of coronavirus spike proteins, receptor recognition and membrane fusion, in the context of the corresponding functions from other viruses and host cells.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structures of the X-domains of a Group-1 and a Group-3 coronavirus reveal that ADP-ribose-binding may not be a conserved property</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The polyproteins of coronaviruses are cleaved by viral proteases into at least 15 nonstructural proteins (Nsps). Consisting of five domains, Nsp3 is the largest of these (180–210 kDa). Among these domains, the so-called X-domain is believed to act as ADP-ribose-1″-phosphate phosphatase or to bind poly(ADP-ribose). However, here we show that the X-domain of Infectious Bronchitis Virus (strain Beaudette), a Group-3 coronavirus, fails to bind ADP-ribose. This is explained on the basis of the crystal structure of the protein, determined at two different pH values. For comparison, we also describe the crystal structure of the homologous X-domain from Human Coronavirus 229E, a Group-1 coronavirus, which does bind ADP-ribose.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>The Severe Acute Respiratory Syndrome Coronavirus Nucleocapsid Protein Is Phosphorylated and Localizes in the Cytoplasm by 14-3-3-Mediated Translocation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome coronavirus(SARS-CoV) nucleocapsid (N) protein is one of the four structural proteins of the virus and is predicted to be a 46-kDa phosphoprotein. Our in silico analysis predicted N to be heavily phosphorylated at multiple residues. Experimentally, we have shown in this report that the N protein of the SARS-CoV gets serine-phosphorylated by multiple kinases, in both the cytoplasm and the nucleus. The phosphoprotein is stable and localizes in the cytoplasm and coprecipitates with the membrane fraction. Also, using specific inhibitors of phosphorylation and an in vitro phosphorylation assay, we show that the nucleocapsid protein is a substrate of cyclin-dependent kinase (CDK), glycogen synthase kinase, mitogen-activated protein kinase, and casein kinase II. Further, we show that the phosphorylated protein is translocated to the cytoplasm by binding to 14-3-3 (tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein). 14-3-3 proteins are a family of highly conserved, ubiquitously expressed eukaryotic proteins that function primarily as adapters that modulate interactions between components of various cellular signaling and cell cycle regulatory pathways through phosphorylation-dependent protein-protein interactions. Coincidentally, the N protein was also found to downregulate the expression of the theta isoform of 14-3-3 (14-3-3θ), leading to the accumulation of phosphorylated N protein in the nucleus, in the absence of growth factors. Using short interfering RNA specific to 14-3-3θ we have inhibited its expression to show accumulation of phosphorylated N protein in the nucleus. Thus, the data presented here provide a possible mechanism for phosphorylation-dependent nucleocytoplasmic shuttling of the N protein. This 14-3-3-mediated transport of the phosphorylated N protein and its possible implications in interfering with the cellular machinery are discussed.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamically-Driven Inactivation of the Catalytic Machinery of the SARS 3C-Like Protease by the N214A Mutation on the Extra Domain</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Feb 24</span></i><p style="text-align: justify; text-justify: inter-word;">Despite utilizing the same chymotrypsin fold to host the catalytic machinery, coronavirus 3C-like proteases (3CLpro) noticeably differ from picornavirus 3C proteases in acquiring an extra helical domain in evolution. Previously, the extra domain was demonstrated to regulate the catalysis of the SARS-CoV 3CLpro by controlling its dimerization. Here, we studied N214A, another mutant with only a doubled dissociation constant but significantly abolished activity. Unexpectedly, N214A still adopts the dimeric structure almost identical to that of the wild-type (WT) enzyme. Thus, we conducted 30-ns molecular dynamics (MD) simulations for N214A, WT, and R298A which we previously characterized to be a monomer with the collapsed catalytic machinery. Remarkably, three proteases display distinctive dynamical behaviors. While in WT, the catalytic machinery stably retains in the activated state; in R298A it remains largely collapsed in the inactivated state, thus implying that two states are not only structurally very distinguishable but also dynamically well separated. Surprisingly, in N214A the catalytic dyad becomes dynamically unstable and many residues constituting the catalytic machinery jump to sample the conformations highly resembling those of R298A. Therefore, the N214A mutation appears to trigger the dramatic change of the enzyme dynamics in the context of the dimeric form which ultimately inactivates the catalytic machinery. The present MD simulations represent the longest reported so far for the SARS-CoV 3CLpro, unveiling that its catalysis is critically dependent on the dynamics, which can be amazingly modulated by the extra domain. Consequently, mediating the dynamics may offer a potential avenue to inhibit the SARS-CoV 3CLpro.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamically-Driven Inactivation of the Catalytic Machinery of the SARS 3C-Like Protease by the N214A Mutation on the Extra Domain</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Feb 24</span></i><p style="text-align: justify; text-justify: inter-word;">Despite utilizing the same chymotrypsin fold to host the catalytic machinery, coronavirus 3C-like proteases (3CLpro) noticeably differ from picornavirus 3C proteases in acquiring an extra helical domain in evolution. Previously, the extra domain was demonstrated to regulate the catalysis of the SARS-CoV 3CLpro by controlling its dimerization. Here, we studied N214A, another mutant with only a doubled dissociation constant but significantly abolished activity. Unexpectedly, N214A still adopts the dimeric structure almost identical to that of the wild-type (WT) enzyme. Thus, we conducted 30-ns molecular dynamics (MD) simulations for N214A, WT, and R298A which we previously characterized to be a monomer with the collapsed catalytic machinery. Remarkably, three proteases display distinctive dynamical behaviors. While in WT, the catalytic machinery stably retains in the activated state; in R298A it remains largely collapsed in the inactivated state, thus implying that two states are not only structurally very distinguishable but also dynamically well separated. Surprisingly, in N214A the catalytic dyad becomes dynamically unstable and many residues constituting the catalytic machinery jump to sample the conformations highly resembling those of R298A. Therefore, the N214A mutation appears to trigger the dramatic change of the enzyme dynamics in the context of the dimeric form which ultimately inactivates the catalytic machinery. The present MD simulations represent the longest reported so far for the SARS-CoV 3CLpro, unveiling that its catalysis is critically dependent on the dynamics, which can be amazingly modulated by the extra domain. Consequently, mediating the dynamics may offer a potential avenue to inhibit the SARS-CoV 3CLpro.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamically-Driven Inactivation of the Catalytic Machinery of the SARS 3C-Like Protease by the N214A Mutation on the Extra Domain</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Feb 24</span></i><p style="text-align: justify; text-justify: inter-word;">Despite utilizing the same chymotrypsin fold to host the catalytic machinery, coronavirus 3C-like proteases (3CLpro) noticeably differ from picornavirus 3C proteases in acquiring an extra helical domain in evolution. Previously, the extra domain was demonstrated to regulate the catalysis of the SARS-CoV 3CLpro by controlling its dimerization. Here, we studied N214A, another mutant with only a doubled dissociation constant but significantly abolished activity. Unexpectedly, N214A still adopts the dimeric structure almost identical to that of the wild-type (WT) enzyme. Thus, we conducted 30-ns molecular dynamics (MD) simulations for N214A, WT, and R298A which we previously characterized to be a monomer with the collapsed catalytic machinery. Remarkably, three proteases display distinctive dynamical behaviors. While in WT, the catalytic machinery stably retains in the activated state; in R298A it remains largely collapsed in the inactivated state, thus implying that two states are not only structurally very distinguishable but also dynamically well separated. Surprisingly, in N214A the catalytic dyad becomes dynamically unstable and many residues constituting the catalytic machinery jump to sample the conformations highly resembling those of R298A. Therefore, the N214A mutation appears to trigger the dramatic change of the enzyme dynamics in the context of the dimeric form which ultimately inactivates the catalytic machinery. The present MD simulations represent the longest reported so far for the SARS-CoV 3CLpro, unveiling that its catalysis is critically dependent on the dynamics, which can be amazingly modulated by the extra domain. Consequently, mediating the dynamics may offer a potential avenue to inhibit the SARS-CoV 3CLpro.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</jats:p></p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural Characterization of Human Coronavirus NL63 N Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are responsible for upper and lower respiratory tract infections in humans. It is estimated that 1 to 10% of the population suffers annually from cold-like symptoms related to infection with human coronavirus NL63 (HCoV-NL63), an alphacoronavirus. The nucleocapsid (N) protein, the major structural component of the capsid, facilitates RNA packing, links the capsid to the envelope, and is also involved in multiple other processes, including viral replication and evasion of the immune system. Although the role of N protein in viral replication is relatively well described, no structural data are currently available regarding the N proteins of alphacoronaviruses. Moreover, our understanding of the mechanisms of RNA binding and nucleocapsid formation remains incomplete. In this study, we solved the crystal structures of the N- and C-terminal domains (NTD, residues 10 to 140, and CTD, residues 221 to 340, respectively) of the N protein of HCoV-NL63, both at a 1.5-Å resolution. Based on our structure of NTD solved here, we proposed and experimentally evaluated a model of RNA binding. The structure of the CTD reveals the mode of N protein dimerization. Overall, this study expands our understanding of the initial steps of N protein-nucleic acid interaction and may facilitate future efforts to control the associated infections. IMPORTANCE Coronaviruses are responsible for the common cold and other respiratory tract infections in humans. According to multiple studies, 1 to 10% of the population is infected each year with HCoV-NL63. Viruses are relatively simple organisms composed of a few proteins and the nucleic acids that carry the information determining their composition. The nucleocapsid (N) protein studied in this work protects the nucleic acid from the environmental factors during virus transmission. This study investigated the structural arrangement of N protein, explaining the first steps of its interaction with nucleic acid at the initial stages of virus structure assembly. The results expand our understanding of coronavirus physiology and may facilitate future efforts to control the associated infections.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Mapping of the Coronavirus Membrane Protein Domains Involved in Interaction with the Spike Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein is the key player in virion assembly. One of its functions is to mediate the incorporation of the spikes into the viral envelope. Heterotypic interactions between M and the spike (S) protein can be demonstrated by coimmunoprecipitation and by immunofluorescence colocalization, after coexpression of their genes in eukaryotic cells. Using these assays in a mutagenetic approach, we have mapped the domains in the M protein that are involved in complex formation between M and S. It appeared that the 25-residue luminally exposed amino-terminal domain of the M protein is not important for M-S interaction. A 15-residue deletion, the insertion of a His tag, and replacement of the ectodomain by that of another coronavirus M protein did not affect the ability of the M protein to associate with the S protein. However, complex formation was sensitive to changes in the transmembrane domains of this triple-spanning protein. Deletion of either the first two or the last two transmembrane domains, known not to affect the topology of the protein, led to a considerable decrease in complex formation, but association was not completely abrogated. Various effects of changes in the part of the M protein that is located at the cytoplasmic face of the membrane were observed. Deletions of the extreme carboxy-terminal tail appeared not to interfere with M-S complex formation. However, deletions in the amphipathic domain severely affected M-S interaction. Interestingly, changes in the amino-terminal and extreme carboxy-terminal domains of M, which did not disrupt the interaction with S, are known to be fatal to the ability of the protein to engage in virus particle formation (C. A. M. de Haan, L. Kuo, P. S. Masters, H. Vennema, and P. J. M. Rottier, J. Virol. 72:6838–6850, 1998). Apparently, the structural requirements of the M protein for virus particle assembly differ from the requirements for the formation of M-S complexes.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /></body>