
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: coupling genomics and diagnostic testing on a large scale</h2><br /></div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>2 Division of Global Health Protection</h3>
<p style="text-align: justify; text-justify: inter-word;">The metagenomic surveillance approach for monitoring circulating pathogens in sewage was able to detect putative pathogen and resistance loads in an urban informal settlement. Thus, valuable if generated in real time to serve as a comprehensive infectious disease agent surveillance system with the potential to guide disease prevention and treatment. The approach may lead to a paradigm shift in conducting real-time global genomics-based surveillance in settings with limited access to health care.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: This paper presents the CEGH information system that is a Laboratory Information Management System (LIMS) based on a formal framework to support genetic testing management for Mendelian disorder studies. We have proved the feasibility and showed usability benefits of a rigorous approach that is able to specify, validate, and perform genetic testing using easy end user interfaces.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Citation: Targeted NGS Platforms for Genetic Screening and Gene Discovery in Primary Immunodeficiencies</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: NGS technology represents a powerful approach in the complex field of rare disorders but its different application should be weighted. A relatively small NGS target panel can be successfully applied for a robust diagnostic suspicion, while when the spectrum of clinical phenotypes overlaps more than one PID an in-depth NGS analysis is required, including also whole exome/genome sequencing to identify the causative gene.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Public health preparedness requires effective surveillance of and rapid response to infectious disease outbreaks. Inclusion of research activities within the outbreak setting provides important opportunities to maximize limited resources, to enhance gains in scientific knowledge, and ultimately to increase levels of preparedness. With rapid advances in laboratory technologies, banking and analysis of human genomic specimens can be conducted as part of public health investigations, enabling valuable research well into the future.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid molecular detection of macrolide resistance</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Emerging antimicrobial resistance is a significant threat to human health. However, methods for rapidly diagnosing antimicrobial resistance generally require multi-day culture-based assays. Macrolide efflux gene A, mef(A), provides resistance against erythromycin and azithromycin and is known to be laterally transferred among a wide range of bacterial species. Methods: We use Recombinase Polymerase Assay (RPA) to detect the antimicrobial resistance gene mef(A) from raw lysates without nucleic acid purification. To validate these results we performed broth dilution assays to assess antimicrobial resistance to erythromycin and ampicillin (a negative control). Results: We validate the detection of mef(A) in raw lysates of Streptococcus pyogenes, S. pneumoniae, S. salivarius, and Enterococcus faecium bacterial lysates within 7-10 min of assay time. We show that detection of mef(A) accurately predicts real antimicrobial resistance assessed by traditional culture methods, and that the assay is robust to high levels of spiked-in non-specific nucleic acid contaminant. The assay was unaffected by single-nucleotide polymorphisms within divergent mef(A) gene sequences, strengthening its utility as a robust diagnostic tool. Conclusions: This finding opens the door to implementation of rapid genomic diagnostics in a clinical setting, while providing researchers a rapid, cost-effective tool to track antibiotic resistance in both pathogens and commensal strains.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic data and pathogen genome sequences: a powerful synergy for public health</h3>
<p style="text-align: justify; text-justify: inter-word;">Epidemiologists aim to inform the design of public health interventions with evidence on the evolution, emergence and spread of infectious diseases. Sequencing of pathogen genomes, together with date, location, clinical manifestation and other relevant data about sample origins, can contribute to describing nearly every aspect of transmission dynamics, including local transmission and global spread. The analyses of these data have implications for all levels of clinical and public health practice, from institutional infection control to policies for surveillance, prevention and treatment. This review highlights the range of epidemiological questions that can be addressed from the combination of genome sequence and traditional 'line lists' (tables of epidemiological data where each line includes demographic and clinical features of infected individuals). We identify opportunities for these data to inform interventions that reduce disease incidence and prevalence. By considering current limitations of, and challenges to, interpreting these data, we aim to outline a research agenda to accelerate the genomics-driven transformation in public health microbiology.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>QTL-mapping and genomic prediction for bovine respiratory disease in U.S. Holsteins using sequence imputation and feature selection</h3>
<p style="text-align: justify; text-justify: inter-word;">We demonstrate the utility of sequence-based and biology-driven model development for genomic selection. Disease phenotypes cannot be routinely recorded in most livestock species and the observed phenotypes may vary in their genomic architecture due to variation in the pathogen composition across environments. Elucidation of trait biology and genetic architecture may guide the development of prediction models with utility across breeds and environments.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">It is time to shake up public health surveillance. New technologies for sequencing, aided by friction-free approaches to data sharing, could have an impact on public health efforts.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">A rigorous approach to facilitate and guarantee the correctness of the genetic testing management in human genome information systems Abstract Background: Recent medical and biological technology advances have stimulated the development of new testing systems that have been providing huge, varied amounts of molecular and clinical data. Growing data volumes pose significant challenges for information processing systems in research centers. Additionally, the routines of genomics laboratory are typically characterized by high parallelism in testing and constant procedure changes. Results: This paper describes a formal approach to address this challenge through the implementation of a genetic testing management system applied to human genome laboratory. We introduced the Human Genome Research Center Information System (CEGH) in Brazil, a system that is able to support constant changes in human genome testing and can provide patients updated results based on the most recent and validated genetic knowledge. Our approach uses a common repository for process planning to ensure reusability, specification, instantiation, monitoring, and execution of processes, which are defined using a relational database and rigorous control flow specifications based on process algebra (ACP). The main difference between our approach and related works is that we were able to join two important aspects: 1) process scalability achieved through relational database implementation, and 2) correctness of processes using process algebra. Furthermore, the software allows end users to define genetic testing without requiring any knowledge about business process notation or process algebra.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>SHORT REPORT Use of linked electronic health records to assess mortality and length of stay associated with pandemic influenza A(H1N1)pdm09 at a UK teaching hospital</h3>
<p style="text-align: justify; text-justify: inter-word;">Effective use of data linkage is becoming an increasingly important focus in the new healthcare system in England. We linked data from the results of a multiplex PCR assay for respiratory viruses for a population of 230 inpatients at a UK teaching hospital with their patient administrative system records in order to compare the mortality and length of stay of patients who tested positive for influenza A(H1N1)pdm09 with those positive for another influenza A virus. The results indicated a reduced risk of death among influenza A(H1N1)pdm09 patients compared to other influenza A strains, with an adjusted risk ratio of 0·25 (95% confidence interval 0·08-0·75, P = 0·01), while no significant differences were found between the lengths of stay in the hospital for these two groups. Further development of such methods to link hospital data in a routine fashion could provide a rapid means of gaining epidemiological insights into emerging infectious diseases.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Genomic Loci Important in Common Infectious Diseases: Role of High-Throughput Sequencing and Genome-Wide Association Studies</h3>
<p style="text-align: justify; text-justify: inter-word;">HIV/AIDS, tuberculosis (TB), and malaria are 3 major global public health threats that undermine development in many resource-poor settings. Recently, the notion that positive selection during epidemics or longer periods of exposure to common infectious diseases may have had a major effect in modifying the constitution of the human genome is being interrogated at a large scale in many populations around the world. is positive selection from infectious diseases increases power to detect associations in genome-wide association studies (GWASs). High-throughput sequencing (HTS) has transformed both the management of infectious diseases and continues to enable large-scale functional characterization of host resistance/susceptibility alleles and loci; a paradigm shift from single candidate gene studies. Application of genome sequencing technologies and genomics has enabled us to interrogate the host-pathogen interface for improving human health. Human populations are constantly locked in evolutionary arms races with pathogens; therefore, identification of common infectious disease-associated genomic variants/markers is important in therapeutic, vaccine development, and screening susceptible individuals in a population. is review describes a range of host-pathogen genomic loci that have been associated with disease susceptibility and resistant patterns in the era of HTS. We further highlight potential opportunities for these genetic markers.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<p style="text-align: justify; text-justify: inter-word;">Citation Chan JM, Rabadan R. 2013. Quantifying pathogen surveillance using temporal genomic data. mBio 4(1):e00524-12.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Journal of Biomedical Discovery and Collaboration Biological information specialists for biological informatics</h3>
<p style="text-align: justify; text-justify: inter-word;">Data management and integration are complicated and ongoing problems that will require commitment of resources and expertise from the various biological science communities. Primary components of successful cross-scale integration are smooth information management and migration from one context to another. We call for a broadening of the definition of bioinformatics and bioinformatics training to span biological disciplines and biological scales. Training programs are needed that educate a new kind of informatics professional, Biological Information Specialists, to work in collaboration with various discipline-specific research personnel. Biological Information Specialists are an extension of the informationist movement that began within library and information science (LIS) over 30 years ago as a professional position to fill a gap in clinical medicine. These professionals will help advance science by improving access to scientific information and by freeing scientists who are not interested in data management to concentrate on their science.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Translating Lung Microbiome Profiles into the Next- Generation Diagnostic Gold Standard for Pneumonia: a Clinical Investigator's Perspective</h3>
<p style="text-align: justify; text-justify: inter-word;">Severe bacterial pneumonia is a major global cause of morbidity and mortality, yet current diagnostic approaches rely on identification of causative pathogens by cultures, which require extended incubation periods and often fail to detect relevant pathogens. Consequently, patients are prescribed broad-spectrum antibiotics in a "one-size-fits-all" manner, which may be inappropriate for their individual needs and promote antibiotic resistance. My research focuses on leveraging next-generation sequencing of microbial DNA directly from patient samples for the development of new, culture-independent definitions of pneumonia. In this perspective article, I discuss the current state of the field and focus on the conceptual and research design challenges for clinical translation. With ongoing technological advancements and application of computational biology methods for assessing clinical validity and utility, I anticipate that sequencing-based diagnostics will soon be able to positively disrupt the way we think about, diagnose, and treat pulmonary infections.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Targeted Enrichment for Pathogen Detection and Characterization in Three Felid Species</h3>
<p style="text-align: justify; text-justify: inter-word;">Traditional diagnostic assays often lack sensitivity and can be difficult to multiplex across many pathogens. Next-generation sequencing (NGS) can overcome some of these problems but has limited application in the detection of low-copynumber pathogens in complex samples. Targeted genome capture (TGC) utilizes oligonucleotide probes to enrich specific nucleic acids in heterogeneous extracts and can therefore increase the proportion of NGS reads for low-abundance targets. While earlier studies have demonstrated the utility of this technology for detection of novel pathogens in human clinical samples, the capacity and practicality of TGC-NGS in a veterinary diagnostic setting have not yet been evaluated. Here we report the use of TGC-NGS assays for the detection and characterization of diverse feline pathogen taxa. We detected 31 pathogens comprising nine pathogen taxa in 28 felid samples analyzed. This included 20 pathogens detected via traditional PCR and 11 additional pathogens that had not been previously detected in the same samples. Most of the pathogens detected were sequenced at sufficient breadth and depth to confidently classify them at the species or subspecies level. Target nucleic acids were enriched from a low of 58-fold to 56 million-fold relative to host nucleic acids. Despite the promising performance of these assays, a number of pathogens detected by conventional PCR or serology were not isolated by TGC-NGS, suggesting that further validation is required before this technology can be used in lieu of quality-controlled standard assays. We conclude that TGC-NGS offers great potential as a broad multiplex pathogen characterization assay in veterinary diagnostic and research settings.</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Network-based analysis of comorbidities risk during an infection: SARS and HIV case studies</h3>
<p style="text-align: justify; text-justify: inter-word;">Our method presents a ripe opportunity to use data-driven approaches for advancing our current knowledge on disease mechanism and predicting disease comorbidities in a quantitative way.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Citation: Targeted NGS Platforms for Genetic Screening and Gene Discovery in Primary Immunodeficiencies</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: The complementary use of the two custom-made targeted sequencing approaches allowed the identification of causative variants in 28.6% (n = 30) of patients. Twenty-two out of 73 (34.6%) patients were diagnosed by Ion Torrent. In this group 20 were included in the SCID/CID category. Eight out of 50 (16%) patients were diagnosed by Haloplex workflow. Ion Torrent method was highly successful for those cases with welldefined phenotypes for immunological and clinical presentation. The Haloplex approach was able to diagnose 4 SCID/CID patients and 4 additional patients with complex and extended phenotypes, embracing all three PID categories in which this approach was more efficient. Both technologies showed good gene coverage.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Genomics Software for optimization of SNP and PCR-RFLP genotyping to discriminate many genomes with the fewest assays</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusion: This is the first software to optimize the selection of forensic markers to maximize information gained from the fewest assays, accepting whole or partial genome sequence data as input. As more sequence data becomes available for multiple strains and isolates of a species, automated, computational approaches such as those described here will be essential to make sense of large amounts of information, and to guide and optimize efforts in the laboratory. The software and source code for SPR Opt is publicly available and free for non-profit use at</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<p style="text-align: justify; text-justify: inter-word;">Comprehensive and uniform diagnostic test data will aid in the identification of potential outbreaks. A combination of broad respiratory pathogen testing and an internal electronic surveillance system enabled the rapid dissemination of data across the largest health care system 80 in New York, the North Shore-LIJ Health System (now Northwell Health) during the influenza A H1N1-2009 pandemic in the New York City area. Pathogen-specific molecular testing permitted rapid a) notification to state epidemiologists; b) tracking of the virus so that health care resources could be managed effectively; and c) evaluation of influenza diagnostics [27, 28] .</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /></body>