
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: physical science of the coronavirus</h2><br /></div><br /><br /><br /><div class="card"><h3>Coronavirus just caused the American Physical Society to cancel its biggest meeting of the year | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Citing the growing threat of the coronavirus, the American Physical Society (APS), the 55,000 member professional society for physicists and researchers in associated fields, cancelled its largest meeting of the year just 34 hours before it was supposed to begin. APS’s March Meeting was to be held this week at the Colorado Convention Center in Denver, and the society anticipated more than 10,000 people from all over the world would attend. However, late yesterday, APS issued a statement abruptly calling off the meeting. “The decision to cancel was based on the latest scientific data being reported, and the fact that a large number of attendees at this meeting are coming from outside the U.S.,” including countries where the virus is circulating and for which the U.S. Centers for Disease Control and Prevention have advised people to avoid non-essential travel, the APS statement says. “[T]his decision was made out of deep concern for the health and well-being of our registrants, staff, vendors, and the Denver community.”</p>
<p align="right"><i>score: 164</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron tomography of mouse hepatitis virus: Insights into the structure of the coronavirion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are enveloped viruses containing the largest reported RNA genomes. As a result of their pleomorphic nature, our structural insight into the coronavirion is still rudimentary, and it is based mainly on 2D electron microscopy. Here we report the 3D virion structure of coronaviruses obtained by cryo-electron tomography. Our study focused primarily on the coronavirus prototype murine hepatitis virus (MHV). MHV particles have a distinctly spherical shape and a relatively homogenous size (≈85 nm envelope diameter). The viral envelope exhibits an unusual thickness (7.8 ± 0.7 nm), almost twice that of a typical biological membrane. Focal pairs revealed the existence of an extra internal layer, most likely formed by the C-terminal domains of the major envelope protein M. In the interior of the particles, coiled structures and tubular shapes are observed, consistent with a helical nucleocapsid model. Our reconstructions provide no evidence of a shelled core. Instead, the ribonucleoprotein seems to be extensively folded onto itself, assuming a compact structure that tends to closely follow the envelope at a distance of ≈4 nm. Focal contact points and thread-like densities connecting the envelope and the ribonucleoprotein are revealed in the tomograms. Transmissible gastroenteritis coronavirion tomograms confirm all the general features and global architecture observed for MHV. We propose a general model for the structure of the coronavirion in which our own and published observations are combined.</p>
<p align="right"><i>score: 147</i></p>

</div><br /><br /><br /><div class="card"><h3>Palindromes in SARS and Other Coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">With the identification of a novel coronavirus associated with the severe acute respiratory syndrome (SARS), computational analysis of its RNA genome sequence is expected to give useful clues to help elucidate the origin, evolution, and pathogenicity of the virus. In this paper, we study the collective counts of palindromes in the SARS genome along with all the completely sequenced coronaviruses. Based on a Markov-chain model for the genome sequence, the mean and standard deviation for the number of palindromes at or above a given length are derived. These theoretical results are complemented by extensive simulations to provide empirical estimates. Using a z score obtained from these mathematical and empirical means and standard deviations, we have observed that palindromes of length four are significantly underrepresented in all the coronaviruses in our data set. In contrast, length-six palindromes are significantly underrepresented only in the SARS coronavirus. Two other features are unique to the SARS sequence. First, there is a length-22 palindrome TCTTTAACAAGCTTGTTAAAGA spanning positions 25962–25983. Second, there are two repeating length-12 palindromes TTATAATTATAA spanning positions 22712–22723 and 22796–22807. Some further investigations into possible biological implications of these palindrome features are proposed.</p>
<p align="right"><i>score: 146</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal Structures of Two Coronavirus ADP-Ribose-1″-Monophosphatases and Their Complexes with ADP-Ribose: a Systematic Structural Analysis of the Viral ADRP Domain</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronaviruses are a large family of plus-strand RNA viruses that cause a wide variety of diseases both in humans and in other organisms. The coronaviruses are composed of three main lineages and have a complex organization of nonstructural proteins (nsp's). In the coronavirus, nsp3 resides a domain with the macroH2A-like fold and ADP-ribose-1"-monophosphatase (ADRP) activity, which is proposed to play a regulatory role in the replication process. However, the significance of this domain for the coronaviruses is still poorly understood due to the lack of structural information from different lineages. We have determined the crystal structures of two viral ADRP domains, from the group I human coronavirus 229E and the group III avian infectious bronchitis virus, as well as their respective complexes with ADP-ribose. The structures were individually solved to elucidate the structural similarities and differences of the ADRP domains among various coronavirus species. The active-site residues responsible for mediating ADRP activity were found to be highly conserved in terms of both sequence alignment and structural superposition, whereas the substrate binding pocket exhibited variations in structure but not in sequence. Together with data from a previous analysis of the ADRP domain from the group II severe acute respiratory syndrome coronavirus and from other related functional studies of ADRP domains, a systematic structural analysis of the coronavirus ADRP domains was realized for the first time to provide a structural basis for the function of this domain in the coronavirus replication process.</p>
<p align="right"><i>score: 143</i></p>

</div><br /><br /><br /><div class="card"><h3>Assembly of the Coronavirus Envelope: Homotypic Interactions between the M Proteins</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The viral membrane proteins M and E are the minimal requirements for the budding of coronavirus particles. Since the E protein occurs in particles only in trace amounts, the lateral interactions between the M proteins apparently generate the major driving force for envelope formation. By using coimmunoprecipitation and envelope incorporation assays, we provide extensive evidence for the existence of such M-M interactions. In addition, we determined which domains of the M protein are involved in this homotypic association, using a mutagenetic approach. Mutant M proteins which were not able to assemble into viruslike particles (VLPs) by themselves (C. A. M. de Haan, L. Kuo, P. S. Masters, H. Vennema, and P. J. M. Rottier, J. Virol. 72:6838–6850, 1998) were tested for the ability to associate with other M proteins and to be rescued into VLPs formed by assembly-competent M proteins. We found that M proteins lacking parts of the transmembrane cluster, of the amphipathic domain, or of the hydrophilic carboxy-terminal tail, or M proteins that had their luminal domain replaced by heterologous ectodomains, were still able to associate with assembly-competent M proteins, resulting in their coincorporation into VLPs. Only a mutant M protein in which all three transmembrane domains had been replaced lost this ability. The results indicate that M protein molecules interact with each other through multiple contact sites, particularly at the transmembrane level. Finally, we tested the stringency with which membrane proteins are selected for incorporation into the coronavirus envelope by probing the coassembly of some foreign proteins. The observed efficient exclusion from budding of the vesicular stomatitis virus G protein and the equine arteritis virus M protein indicates that envelope assembly is indeed a highly selective sorting process. The low but detectable incorporation of CD8 molecules, however, demonstrated that this process is not perfect.</p>
<p align="right"><i>score: 142</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation of coronavirus envelope glycoproteins and interaction with the viral nucleocapsid.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The two envelope glycoproteins and the viral nucleocapsid of the coronavirus A59 were isolated by solubilization of the viral membrane with Nonidet P-40 at 4 degrees C followed by sucrose density gradient sedimentation. Isolated E2 consisted of rosettes of peplomers, whereas E1, the membrane glycoprotein, was irregular and amorphous. Under certain conditions significant interactions occurred between components of Nonidet P-40-disrupted virions. Incubation of the Nonidet P-40-disrupted virus at 37 degrees C resulted in formation of a complex between one of the viral glycoproteins, E1, and the viral nucleocapsid. This was caused by a temperature-dependent conformational change in E1, resulting in aggregation of E1 and interaction with the viral RNA in the nucleocapsid. E1 also bound rRNA. The E1-nucleocapsid complexes can be distinguished on sucrose and Renografin density gradients from native viral nucleocapsids. The separation of the membrane glycoprotein E1 from the peplomeric glycoprotein E2 permitted preparation of antisera against these isolated proteins. A model is proposed for the arrangement of the three major structural proteins in the coronavirus A59 virion in relation to the viral envelope and RNA.</p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional analysis of the stem loop S3 and S4 structures in the coronavirus 3'UTR</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We designed a series of mutations to separately destabilize two helical stems (designated S3 and S4) predicted by a covariation-based model of the coronavirus 3'UTR (Zust et al., 2008). Mouse hepatitis virus genomes containing three or four nucleotide mutations that destabilize either S3 or S4 were viable, whereas genomes carrying these mutations in both S3 and S4 were not viable. A genome carrying these mutations in S3 and S4 plus compensatory mutations restoring base-pairing yielded a virus with wild type phenotype. Larger mutations which completely disrupt S3 or S4 generated various phenotypes. Mutations opening up S3 were lethal. Disruptions of S4 generated both viable and lethal mutants. Genomes carrying the original mutations in S3 or S4 plus compensatory mutations restoring base pairing were viable and had robust growth phenotypes. These results support the Zust model for the coronavirus 3'UTR and suggest that the S3 stem is required for virus viability.</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>‘We had to act.’ How coronavirus fears forced physics society to nix giant meeting</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Harold Dorwin/Center for Astrophysics/Harvard & Smithsonian The March Meeting of the American Physical Society (APS) wasn’t the first event canceled for fear of the coronavirus, but it was nixed in dramatic fashion.</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>A structural analysis of M protein in coronavirus assembly and morphology</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The M protein of coronavirus plays a central role in virus assembly, turning cellular membranes into workshops where virus and host factors come together to make new virus particles. We investigated how M structure and organization is related to virus shape and size using cryo-electron microscopy, tomography and statistical analysis. We present evidence that suggests M can adopt two conformations and that membrane curvature is regulated by one M conformer. Elongated M protein is associated with rigidity, clusters of spikes and a relatively narrow range of membrane curvature. In contrast, compact M protein is associated with flexibility and low spike density. Analysis of several types of virus-like particles and virions revealed that S protein, N protein and genomic RNA each help to regulate virion size and variation, presumably through interactions with M. These findings provide insight into how M protein functions to promote virus assembly.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiple Nucleic Acid Binding Sites and Intrinsic Disorder of Severe Acute Respiratory Syndrome Coronavirus Nucleocapsid Protein: Implications for Ribonucleocapsid Protein Packaging</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The nucleocapsid protein (N) of the severe acute respiratory syndrome coronavirus (SARS-CoV) packages the viral genomic RNA and is crucial for viability. However, the RNA-binding mechanism is poorly understood. We have shown previously that the N protein contains two structural domains—the N-terminal domain (NTD; residues 45 to 181) and the C-terminal dimerization domain (CTD; residues 248 to 365)—flanked by long stretches of disordered regions accounting for almost half of the entire sequence. Small-angle X-ray scattering data show that the protein is in an extended conformation and that the two structural domains of the SARS-CoV N protein are far apart. Both the NTD and the CTD have been shown to bind RNA. Here we show that all disordered regions are also capable of binding to RNA. Constructs containing multiple RNA-binding regions showed Hill coefficients greater than 1, suggesting that the N protein binds to RNA cooperatively. The effect can be explained by the “coupled-allostery” model, devised to explain the allosteric effect in a multidomain regulatory system. Although the N proteins of different coronaviruses share very low sequence homology, the physicochemical features described above may be conserved across different groups of Coronaviridae. The current results underscore the important roles of multisite nucleic acid binding and intrinsic disorder in N protein function and RNP packaging.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV Fusion Peptide forms an Extended Bipartite Fusion Platform that Perturbs Membrane Order in a Calcium-Dependent Manner</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are a major infectious disease threat, and include the pathogenic human pathogens of zoonotic origin: SARS-CoV and MERS-CoV. Entry of coronaviruses into host cells is mediated by the viral spike (S) protein, which is structurally categorized as a class I viral fusion protein, within the same group as influenza virus and HIV. However, S proteins have two distinct cleavage sites that can be activated by a much wider range of proteases. The exact location of the coronavirus fusion peptide (FP) has been disputed. However, most evidence suggests that the domain immediately downstream of the S2′ cleavage site is the FP (amino acids 798-818 SFIEDLLFNKVTLADAGFMKQY for SARS-CoV, FP1). In our previous ESR spectroscopic studies, the membrane ordering effect of influenza virus, HIV and Dengue virus FPs have been consistently observed. In this study, we used this effect as a criterion to identify and characterize the bona fide SARS-CoV FP. Our results indicate that both FP1 and the region immediately downstream (amino acids 816-835 KQYGECLGDINARDLICAQKF, FP2) induce significant membrane ordering. Furthermore, their effects are calcium-dependent, which is consistent with in vivo data showing that calcium is required for SARS-CoV S-mediated fusion. Isothermal titration calorimetry showed a direct interaction between calcium cations and both FPs. This Ca(2+)-dependency membrane ordering was not observed with influenza FP, indicating that the coronavirus FP exhibits a mechanistically different behavior. Membrane ordering effects are greater and penetrate deeper into membranes when FP1 and FP2 act in a concerted manner, suggesting that they form an extended fusion “platform”.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Functionally Important Negatively Charged Residues in the Carboxy End of Mouse Hepatitis Coronavirus A59 Nucleocapsid Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) protein is a multifunctional viral gene product that encapsidates the RNA genome and also plays some as yet not fully defined role in viral RNA replication and/or transcription. A number of conserved negatively charged amino acids are located within domain III in the carboxy end of all coronavirus N proteins. Previous studies suggested that the negatively charged residues are involved in virus assembly by mediating interaction between the membrane (M) protein carboxy tail and nucleocapsids. To determine the importance of these negatively charged residues, a series of alanine and other charged-residue substitutions were introduced in place of those in the N gene within a mouse hepatitis coronavirus A59 infectious clone. Aspartic acid residues 440 and 441 were identified as functionally important. Viruses could not be isolated when both residues were replaced by positively charged amino acids. When either amino acid was replaced by a positively charged residue or both were changed to alanine, viruses were recovered that contained second-site changes within N, but not in the M or envelope protein. The compensatory role of the new changes was confirmed by the construction of new viruses. A few viruses were recovered that retained the D(441)-to-arginine change and no compensatory changes. These viruses exhibited a small-plaque phenotype and produced significantly less virus. Overall, results from our analysis of a large panel of plaque-purified recovered viruses indicate that the negatively charged residues at positions 440 and 441 are key residues that appear to be involved in virus assembly.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Ribonucleocapsid Formation of Severe Acute Respiratory Syndrome Coronavirus through Molecular Action of the N-Terminal Domain of N Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Conserved among all coronaviruses are four structural proteins: the matrix (M), small envelope (E), and spike (S) proteins that are embedded in the viral membrane and the nucleocapsid phosphoprotein (N), which exists in a ribonucleoprotein complex in the lumen. The N-terminal domain of coronaviral N proteins (N-NTD) provides a scaffold for RNA binding, while the C-terminal domain (N-CTD) mainly acts as oligomerization modules during assembly. The C terminus of the N protein anchors it to the viral membrane by associating with M protein. We characterized the structures of N-NTD from severe acute respiratory syndrome coronavirus (SARS-CoV) in two crystal forms, at 1.17 Å (monoclinic) and at 1.85 Å (cubic), respectively, resolved by molecular replacement using the homologous avian infectious bronchitis virus (IBV) structure. Flexible loops in the solution structure of SARS-CoV N-NTD are now shown to be well ordered around the β-sheet core. The functionally important positively charged β-hairpin protrudes out of the core, is oriented similarly to that in the IBV N-NTD, and is involved in crystal packing in the monoclinic form. In the cubic form, the monomers form trimeric units that stack in a helical array. Comparison of crystal packing of SARS-CoV and IBV N-NTDs suggests a common mode of RNA recognition, but they probably associate differently in vivo during the formation of the ribonucleoprotein complex. Electrostatic potential distribution on the surface of homology models of related coronaviral N-NTDs suggests that they use different modes of both RNA recognition and oligomeric assembly, perhaps explaining why their nucleocapsids have different morphologies.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Assembly of coronavirus spike protein into trimers and its role in epitope expression.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The folding and oligomerization of coronavirus spike protein were explored using a panel of monoclonal antibodies. Chemical cross-linking and sedimentation experiments showed that the spike of transmissible gastroenteritis virus is a homotrimer of the S membrane glycoprotein. The spike protein was synthesized as a 175,000-apparent-molecular-weight (175K) monomer subunit that is sensitive to endo-beta-N-acetylglucosaminidase H. Assembly of monomers into a trimeric structure was found to occur on a partially trimmed polypeptide and to be a rate-limiting step, since large amounts of monomers failed to trimerize 1 h after completion of synthesis. Terminal glycosylation of newly assembled trimers, resulting in the biosynthesis of three 220K oligomers, occurred with a half time of approximately 20 min. Monomeric (230K to 240K) processed forms were also observed in cells and in virions. The 175K monomeric form expressed four major antigenic sites previously localized within the amino-terminal half of the S polypeptide chain; however, two classes of trimer-restricted epitopes (borne by three 220K and/or three 175K oligomers) were identified. The S glycoprotein of coronavirus might be a valuable model system for discovering new aspects of the maturation of membrane glycoproteins.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure, Function, and Evolution of Coronavirus Spike Proteins</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus spike protein is a multifunctional molecular machine that mediates coronavirus entry into host cells. It first binds to a receptor on the host cell surface through its S1 subunit and then fuses viral and host membranes through its S2 subunit. Two domains in S1 from different coronaviruses recognize a variety of host receptors, leading to viral attachment. The spike protein exists in two structurally distinct conformations, prefusion and postfusion. The transition from prefusion to postfusion conformation of the spike protein must be triggered, leading to membrane fusion. This article reviews current knowledge about the structures and functions of coronavirus spike proteins, illustrating how the two S1 domains recognize different receptors and how the spike proteins are regulated to undergo conformational transitions. I further discuss the evolution of these two critical functions of coronavirus spike proteins, receptor recognition and membrane fusion, in the context of the corresponding functions from other viruses and host cells.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and Inhibition of the SARS Coronavirus Envelope Protein Ion Channel</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul 10</span></i><p style="text-align: justify; text-justify: inter-word;">The envelope (E) protein from coronaviruses is a small polypeptide that contains at least one α-helical transmembrane domain. Absence, or inactivation, of E protein results in attenuated viruses, due to alterations in either virion morphology or tropism. Apart from its morphogenetic properties, protein E has been reported to have membrane permeabilizing activity. Further, the drug hexamethylene amiloride (HMA), but not amiloride, inhibited in vitro ion channel activity of some synthetic coronavirus E proteins, and also viral replication. We have previously shown for the coronavirus species responsible for severe acute respiratory syndrome (SARS-CoV) that the transmembrane domain of E protein (ETM) forms pentameric α-helical bundles that are likely responsible for the observed channel activity. Herein, using solution NMR in dodecylphosphatidylcholine micelles and energy minimization, we have obtained a model of this channel which features regular α-helices that form a pentameric left-handed parallel bundle. The drug HMA was found to bind inside the lumen of the channel, at both the C-terminal and the N-terminal openings, and, in contrast to amiloride, induced additional chemical shifts in ETM. Full length SARS-CoV E displayed channel activity when transiently expressed in human embryonic kidney 293 (HEK-293) cells in a whole-cell patch clamp set-up. This activity was significantly reduced by hexamethylene amiloride (HMA), but not by amiloride. The channel structure presented herein provides a possible rationale for inhibition, and a platform for future structure-based drug design of this potential pharmacological target.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and Inhibition of the SARS Coronavirus Envelope Protein Ion Channel</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul 10</span></i><p style="text-align: justify; text-justify: inter-word;">The envelope (E) protein from coronaviruses is a small polypeptide that contains at least one α-helical transmembrane domain. Absence, or inactivation, of E protein results in attenuated viruses, due to alterations in either virion morphology or tropism. Apart from its morphogenetic properties, protein E has been reported to have membrane permeabilizing activity. Further, the drug hexamethylene amiloride (HMA), but not amiloride, inhibited in vitro ion channel activity of some synthetic coronavirus E proteins, and also viral replication. We have previously shown for the coronavirus species responsible for severe acute respiratory syndrome (SARS-CoV) that the transmembrane domain of E protein (ETM) forms pentameric α-helical bundles that are likely responsible for the observed channel activity. Herein, using solution NMR in dodecylphosphatidylcholine micelles and energy minimization, we have obtained a model of this channel which features regular α-helices that form a pentameric left-handed parallel bundle. The drug HMA was found to bind inside the lumen of the channel, at both the C-terminal and the N-terminal openings, and, in contrast to amiloride, induced additional chemical shifts in ETM. Full length SARS-CoV E displayed channel activity when transiently expressed in human embryonic kidney 293 (HEK-293) cells in a whole-cell patch clamp set-up. This activity was significantly reduced by hexamethylene amiloride (HMA), but not by amiloride. The channel structure presented herein provides a possible rationale for inhibition, and a platform for future structure-based drug design of this potential pharmacological target.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Infectious Bronchitis Virus E Protein Is Targeted to the Golgi Complex and Directs Release of Virus-Like Particles</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a poorly characterized small envelope protein present in low levels in virions. We are interested in the role of E in the intracellular targeting of infectious bronchitis virus (IBV) membrane proteins. We generated a cDNA clone of IBV E and antibodies to the E protein to study its cell biological properties in the absence of virus infection. We show that IBV E is an integral membrane protein when expressed in cells from cDNA. Epitope-specific antibodies revealed that the C terminus of IBV E is cytoplasmic and the N terminus is translocated. The short luminal N terminus of IBV E contains a consensus site for N-linked glycosylation, but the site is not used. When expressed using recombinant vaccinia virus, the IBV E protein is released from cells at low levels in sedimentable particles that have a density similar to that of coronavirus virions. The IBV M protein is incorporated into these particles when present. Indirect immunofluorescence microscopy showed that E is localized to the Golgi complex in cells transiently expressing IBV E. When coexpressed with IBV M, both from cDNA and in IBV infection, the two proteins are colocalized in Golgi membranes, near the coronavirus budding site. Thus, even though IBV E is present at low levels in virions, it is apparently expressed at high levels in infected cells near the site of virus assembly.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Regulation of Coronaviral Poly(A) Tail Length during Infection</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Jul 29</span></i><p style="text-align: justify; text-justify: inter-word;">The positive-strand coronavirus genome of ~30 kilobase in length and subgenomic (sg) mRNAs of shorter lengths, are 5’ and 3’-co-terminal by virtue of a common 5’-capped leader and a common 3’-polyadenylated untranslated region. Here, by ligating head-to-tail viral RNAs from bovine coronavirus-infected cells and sequencing across the ligated junctions, it was learned that at the time of peak viral RNA synthesis [6 hours postinfection (hpi)] the 3’ poly(A) tail on genomic and sgmRNAs is ~65 nucleotides (nt) in length. Surprisingly, this length was found to vary throughout infection from ~45 nt immediately after virus entry (at 0 to 4 hpi) to ~65 nt later on (at 6 h to 9 hpi) and from ~65 nt (at 6 h to 9 hpi) to ~30 nt (at 120-144 hpi). With the same method, poly(U) sequences of the same lengths were simultaneously found on the ligated viral negative-strand RNAs. Functional analyses of poly(A) tail length on specific viral RNA species, furthermore, revealed that translation, in vivo, of RNAs with the longer poly(A) tail was enhanced over those with the shorter poly(A). Although the mechanisms by which the tail lengths vary is unknown, experimental results together suggest that the length of the poly(A) and poly(U) tails is regulated. One potential function of regulated poly(A) tail length might be that for the coronavirus genome a longer poly(A) favors translation. The regulation of coronavirus translation by poly(A) tail length resembles that during embryonal development suggesting there may be mechanistic parallels.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of Constructed E Gene Mutants of Mouse Hepatitis Virus Confirms a Pivotal Role for E Protein in Coronavirus Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Expression studies have shown that the coronavirus small envelope protein E and the much more abundant membrane glycoprotein M are both necessary and sufficient for the assembly of virus-like particles in cells. As a step toward understanding the function of the mouse hepatitis virus (MHV) E protein, we carried out clustered charged-to-alanine mutagenesis on the E gene and incorporated the resulting mutations into the MHV genome by targeted recombination. Of the four possible clustered charged-to-alanine E gene mutants, one was apparently lethal and one had a wild-type phenotype. The two other mutants were partially temperature sensitive, forming small plaques at the nonpermissive temperature. Revertant analyses of these two mutants demonstrated that the created mutations were responsible for the temperature-sensitive phenotype of each and provided support for possible interactions among E protein monomers. Both temperature-sensitive mutants were also found to be markedly thermolabile when grown at the permissive temperature, suggesting that there was a flaw in their assembly. Most significantly, when virions of one of the mutants were examined by electron microscopy, they were found to have strikingly aberrant morphology in comparison to the wild type: most mutant virions had pinched and elongated shapes that were rarely seen among wild-type virions. These results demonstrate an important, probably essential, role for the E protein in coronavirus morphogenesis.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Supramolecular Architecture of Severe Acute Respiratory Syndrome Coronavirus Revealed by Electron Cryomicroscopy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus particles are enveloped and pleomorphic and are thus refractory to crystallization and symmetry-assisted reconstruction. A novel methodology of single-particle image analysis was applied to selected virus features to obtain a detailed model of the oligomeric state and spatial relationships among viral structural proteins. Two-dimensional images of the S, M, and N structural proteins of severe acute respiratory syndrome coronavirus and two other coronaviruses were refined to a resolution of ∼4 nm. Proteins near the viral membrane were arranged in overlapping lattices surrounding a disordered core. Trimeric glycoprotein spikes were in register with four underlying ribonucleoprotein densities. However, the spikes were dispensable for ribonucleoprotein lattice formation. The ribonucleoprotein particles displayed coiled shapes when released from the viral membrane. Our results contribute to the understanding of the assembly pathway used by coronaviruses and other pleomorphic viruses and provide the first detailed view of coronavirus ultrastructure.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Topographic Changes in SARS Coronavirus–infected Cells at Late Stages of Infection</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Nov</span></i><p style="text-align: justify; text-justify: inter-word;">Scanning electron and atomic force microscopy was used for the first time to view the maturation of the severe acute respiratory syndrome–associated coronavirus at the cell surface. The surface form of the cells at advanced infection displayed prolific pseudopodia that, in addition to the rest of the plasma membrane, were also active sites of virus release. High magnification of the maturing virus particles showed a rosette appearance with short knoblike spikes under both the scanning electron and atomic force microscopes. The final expulsion step of the maturing virus particles seemed to result in some disruptions to the plasma membrane. The cytoskeletal network along the edge of the infected cells was enhanced and could be involved in transporting and expelling the progeny virus particles. Thickening of the actin filaments at the cell edge provided the bending force to extrude the virus particles.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Importance of the Penultimate Positive Charge in Mouse Hepatitis Coronavirus A59 Membrane Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein carboxy tail interacts with the nucleocapsid during virus assembly. Previous studies demonstrated that the two terminal residues are important, and the charged residue (R227) in the penultimate position in the mouse hepatitis coronavirus (MHV) A59 M protein was suggested to participate in intermolecular interactions with negative charges in the nucleocapsid (N) protein. To determine the significance of the positive charge at position 227, we substituted the arginine with lysine (K), aspartic acid (D), glutamic acid (E), or alanine (A) and studied these by reverse genetics in the context of a MHV full-length infectious clone. Viruses with wild-type phenotype were readily recovered with the K or A substitutions. In contrast, negative-charge substitutions were not tolerated as well. In all recovered R227D viruses the negative charge was replaced with heterologous residues resulting from apparent template switching during negative-strand synthesis of subgenomic RNA 7. An additional second-site compensatory V202I substitution was present in some viruses. Recovered R227E viruses had second-site changes within the M protein carboxy tail that were partially compensatory. Significantly, most of the second site changes in the R227E mutant viruses were previously shown to compensate for the removal of negative charges in the N protein. Our results strongly indicate that a positive charge is not absolutely required. It is clear that other regions within the tail must also be involved in helping mediate interactions between the M protein and the nucleocapsid.</p>
<p align="right"><i>score: 134</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV NSP12 polymerase bound to NSP7 and NSP8 co-factors</h3>
<i class="tooltip">2019<span class="tooltiptext">2019-02-15</span></i><p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS- and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (NSP) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 [A] resolution structure of the SARS-CoV NSP12 polymerase bound to its essential co-factors, NSP7 and NSP8, using single particle cryo-electron microscopy. NSP12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is unexpectedly bound by two NSP8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into NSP12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultrastructural Characterization of SARS Coronavirus</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) was first described during a 2002–2003 global outbreak of severe pneumonia associated with human deaths and person-to-person disease transmission. The etiologic agent was initially identified as a coronavirus by thin-section electron microscopic examination of a virus isolate. Virions were spherical, 78 nm in mean diameter, and composed of a helical nucleocapsid within an envelope with surface projections. Herein, we show that infection with the SARS-associated coronavirus resulted in distinct ultrastructural features: double-membrane vesicles, nucleocapsid inclusions, and large granular areas of cytoplasm. These three structures and the coronavirus particles were shown to be positive for viral proteins and RNA by using ultrastructural immunogold and in situ hybridization assays. In addition, ultrastructural examination of a bronchiolar lavage specimen from a SARS patient showed numerous coronavirus-infected cells with features similar to those in infected culture cells. Electron microscopic studies were critical in identifying the etiologic agent of the SARS outbreak and in guiding subsequent laboratory and epidemiologic investigations.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Analysis of Determinants for Spike Glycoprotein Assembly into Murine Coronavirus Virions: Distinct Roles for Charge-Rich and Cysteine-Rich Regions of the Endodomain</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus spike protein (S) forms the distinctive virion surface structures that are characteristic of this viral family, appearing in negatively stained electron microscopy as stems capped with spherical bulbs. These structures are essential for the initiation of infection through attachment of the virus to cellular receptors followed by fusion to host cell membranes. The S protein can also mediate the formation of syncytia in infected cells. The S protein is a type I transmembrane protein that is very large compared to other viral fusion proteins, and all except a short carboxy-terminal segment of the S molecule constitutes the ectodomain. For the prototype coronavirus mouse hepatitis virus (MHV), it has previously been established that S protein assembly into virions is specified by the carboxy-terminal segment, which comprises the transmembrane domain and the endodomain. We have genetically dissected these domains in the MHV S protein to localize the determinants of S incorporation into virions. Our results establish that assembly competence maps to the endodomain of S, which was shown to be sufficient to target a heterologous integral membrane protein for incorporation into MHV virions. In particular, mutational analysis indicated a major role for the charge-rich carboxy-terminal region of the endodomain. Additionally, we found that the adjacent cysteine-rich region of the endodomain is critical for fusion of infected cells, confirming results previously obtained with S protein expression systems.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>A Single Polar Residue and Distinct Membrane Topologies Impact the Function of the Infectious Bronchitis Coronavirus E Protein</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 May 3</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein with a single predicted hydrophobic domain (HD), and has a poorly defined role in infection. The E protein is thought to promote virion assembly, which occurs in the Golgi region of infected cells. It has also been implicated in the release of infectious particles after budding. The E protein has ion channel activity in vitro, although a role for channel activity in infection has not been established. Furthermore, the membrane topology of the E protein is of considerable debate, and the protein may adopt more than one topology during infection. We previously showed that the HD of the infectious bronchitis virus (IBV) E protein is required for the efficient release of infectious virus, an activity that correlated with disruption of the secretory pathway. Here we report that a single residue within the hydrophobic domain, Thr16, is required for secretory pathway disruption. Substitutions of other residues for Thr16 were not tolerated. Mutations of Thr16 did not impact virus assembly as judged by virus-like particle production, suggesting that alteration of secretory pathway and assembly are independent activities. We also examined how the membrane topology of IBV E affected its function by generating mutant versions that adopted either a transmembrane or membrane hairpin topology. We found that a transmembrane topology was required for disrupting the secretory pathway, but was less efficient for virus-like particle production. The hairpin version of E was unable to disrupt the secretory pathway or produce particles. The findings reported here identify properties of the E protein that are important for its function, and provide insight into how the E protein may perform multiple roles during infection.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells</h3>
<i class="tooltip">2006<span class="tooltiptext">2006 Oct 13</span></i><p style="text-align: justify; text-justify: inter-word;">The N-terminal domain of the coronavirus nucleocapsid (N) protein adopts a fold resembling a right hand with a flexible, positively charged β-hairpin and a hydrophobic palm. This domain was shown to interact with the genomic RNA for coronavirus infectious bronchitis virus (IBV) and severe acute respiratory syndrome coronavirus (SARS-CoV). Based on its 3D structure, we used site-directed mutagenesis to identify residues essential for the RNA-binding activity of the IBV N protein and viral infectivity. Alanine substitution of either Arg-76 or Tyr-94 in the N-terminal domain of IBV N protein led to a significant decrease in its RNA-binding activity and a total loss of the infectivity of the viral RNA to Vero cells. In contrast, mutation of amino acid Gln-74 to an alanine, which does not affect the binding activity of the N-terminal domain, showed minimal, if any, detrimental effect on the infectivity of IBV. This study thus identifies residues critical for RNA binding on the nucleocapsid surface, and presents biochemical and genetic evidence that directly links the RNA binding capacity of the coronavirus N protein to the viral infectivity in cultured cells. This information would be useful in development of preventive and treatment approaches against coronavirus infection.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Biogenesis and Dynamics of the Coronavirus Replicative Structures</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Nov 21</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-strand RNA viruses that are important infectious agents of both animals and humans. A common feature among positive-strand RNA viruses is their assembly of replication-transcription complexes in association with cytoplasmic membranes. Upon infection, coronaviruses extensively rearrange cellular membranes into organelle-like replicative structures that consist of double-membrane vesicles and convoluted membranes to which the nonstructural proteins involved in RNA synthesis localize. Double-stranded RNA, presumably functioning as replicative intermediate during viral RNA synthesis, has been detected at the double-membrane vesicle interior. Recent studies have provided new insights into the assembly and functioning of the coronavirus replicative structures. This review will summarize the current knowledge on the biogenesis of the replicative structures, the membrane anchoring of the replication-transcription complexes, and the location of viral RNA synthesis, with particular focus on the dynamics of the coronavirus replicative structures and individual replication-associated proteins.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural Characterization of Human Coronavirus NL63 N Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are responsible for upper and lower respiratory tract infections in humans. It is estimated that 1 to 10% of the population suffers annually from cold-like symptoms related to infection with human coronavirus NL63 (HCoV-NL63), an alphacoronavirus. The nucleocapsid (N) protein, the major structural component of the capsid, facilitates RNA packing, links the capsid to the envelope, and is also involved in multiple other processes, including viral replication and evasion of the immune system. Although the role of N protein in viral replication is relatively well described, no structural data are currently available regarding the N proteins of alphacoronaviruses. Moreover, our understanding of the mechanisms of RNA binding and nucleocapsid formation remains incomplete. In this study, we solved the crystal structures of the N- and C-terminal domains (NTD, residues 10 to 140, and CTD, residues 221 to 340, respectively) of the N protein of HCoV-NL63, both at a 1.5-Å resolution. Based on our structure of NTD solved here, we proposed and experimentally evaluated a model of RNA binding. The structure of the CTD reveals the mode of N protein dimerization. Overall, this study expands our understanding of the initial steps of N protein-nucleic acid interaction and may facilitate future efforts to control the associated infections. IMPORTANCE Coronaviruses are responsible for the common cold and other respiratory tract infections in humans. According to multiple studies, 1 to 10% of the population is infected each year with HCoV-NL63. Viruses are relatively simple organisms composed of a few proteins and the nucleic acids that carry the information determining their composition. The nucleocapsid (N) protein studied in this work protects the nucleic acid from the environmental factors during virus transmission. This study investigated the structural arrangement of N protein, explaining the first steps of its interaction with nucleic acid at the initial stages of virus structure assembly. The results expand our understanding of coronavirus physiology and may facilitate future efforts to control the associated infections.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 28</span></i><p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS- and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (nsp) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV nsp12 polymerase bound to its essential co-factors, nsp7 and nsp8, using single particle cryo-electron microscopy. nsp12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is bound by two nsp8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into nsp12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 28</span></i><p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS- and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (nsp) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV nsp12 polymerase bound to its essential co-factors, nsp7 and nsp8, using single particle cryo-electron microscopy. nsp12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is bound by two nsp8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into nsp12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 28</span></i><p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS- and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (nsp) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV nsp12 polymerase bound to its essential co-factors, nsp7 and nsp8, using single particle cryo-electron microscopy. nsp12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is bound by two nsp8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into nsp12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 28</span></i><p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS- and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (nsp) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV nsp12 polymerase bound to its essential co-factors, nsp7 and nsp8, using single particle cryo-electron microscopy. nsp12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is bound by two nsp8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into nsp12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Envelope Protein Palmitoylations Are Crucial for Murine Coronavirus Assembly</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus assembly process encloses a ribonucleoprotein genome into vesicles containing the lipid-embedded proteins S (spike), E (envelope), and M (membrane). This process depends on interactions with membranes that may involve palmitoylation, a common posttranslational lipidation of cysteine residues. To determine whether specific palmitoylations influence coronavirus assembly, we introduced plasmid DNAs encoding mouse hepatitis coronavirus (MHV) S, E, M, and N (nucleocapsid) into 293T cells and found that virus-like particles (VLPs) were robustly assembled and secreted into culture medium. Palmitate adducts predicted on cysteines 40, 44, and 47 of the 83-residue E protein were then evaluated by constructing mutant cDNAs with alanine or glycine codon substitutions at one or more of these positions. Triple-substituted proteins (E.Ts) lacked palmitate adducts. Both native E and E.T proteins localized at identical perinuclear locations, and both copurified with M proteins, but E.T was entirely incompetent for VLP production. In the presence of the E.T proteins, the M protein subunits accumulated into detergent-insoluble complexes that failed to secrete from cells, while native E proteins mobilized M into detergent-soluble secreted forms. Many of these observations were corroborated in the context of natural MHV infections, with native E, but not E.T, complementing debilitated recombinant MHVs lacking E. Our findings suggest that palmitoylations are essential for E to act as a vesicle morphogenetic protein and further argue that palmitoylated E proteins operate by allowing the primary coronavirus assembly subunits to assume configurations that can mobilize into secreted lipid vesicles and virions.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of the Role of Hexamer AGUAAA and Poly(A) Tail in Coronavirus Polyadenylation</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Oct 19</span></i><p style="text-align: justify; text-justify: inter-word;">Similar to eukaryotic mRNA, the positive-strand coronavirus genome of ~30 kilobases is 5’-capped and 3’-polyadenylated. It has been demonstrated that the length of the coronaviral poly(A) tail is not static but regulated during infection; however, little is known regarding the factors involved in coronaviral polyadenylation and its regulation. Here, we show that during infection, the level of coronavirus poly(A) tail lengthening depends on the initial length upon infection and that the minimum length to initiate lengthening may lie between 5 and 9 nucleotides. By mutagenesis analysis, it was found that (i) the hexamer AGUAAA and poly(A) tail are two important elements responsible for synthesis of the coronavirus poly(A) tail and may function in concert to accomplish polyadenylation and (ii) the function of the hexamer AGUAAA in coronaviral polyadenylation is position dependent. Based on these findings, we propose a process for how the coronaviral poly(A) tail is synthesized and undergoes variation. Our results provide the first genetic evidence to gain insight into coronaviral polyadenylation.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</p>
<p align="right"><i>score: 130</i></p>

</div><br /><br /><br /><div class="card"><h3>The Cytoplasmic Tail of Infectious Bronchitis Virus E Protein Directs Golgi Targeting</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have previously shown that the E protein of the coronavirus infectious bronchitis virus (IBV) is localized to the Golgi complex when expressed exogenously from cDNA. Here, we report that neither the transmembrane domain nor the short lumenal domain of IBV E is required for Golgi targeting. However, an N-terminal truncation containing only the cytoplasmic domain (CTE) was efficiently localized to the Golgi complex, and this domain could retain a reporter protein in the Golgi. Thus, the cytoplasmic tail of the E protein is necessary and sufficient for Golgi targeting. The IBV E protein is palmitoylated on one or two cysteine residues adjacent to its transmembrane domain, but palmitoylation was not required for proper Golgi targeting. Using C-terminal truncations, we determined that the IBV E Golgi targeting information is present between tail amino acids 13 and 63. Upon treatment with brefeldin A, both the E and CTE proteins redistributed to punctate structures that colocalized with the Golgi matrix proteins GM130 and p115 instead of being localized to the endoplasmic reticulum like Golgi glycosylation enzymes. This suggests that IBV E is associated with the Golgi matrix through interactions of its cytoplasmic tail and may have interesting implications for coronavirus assembly in early Golgi compartments.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-coronavirus spike S2 domain flanked by cysteine residues C822 and C833 is important for activation of membrane fusion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The S2 domain of the coronavirus spike (S) protein is known to be responsible for mediating membrane fusion. In addition to a well-recognized cleavage site at the S1–S2 boundary, a second proteolytic cleavage site has been identified in the severe acute respiratory syndrome coronavirus (SARS-CoV) S2 domain (R797). C terminal to this S2 cleavage site is a conserved region flanked by cysteine residues C822 and C833. Here, we investigated the importance of this well conserved region for SARS-CoV S-mediated fusion activation. We show that the residues between C822-C833 are well conserved across all coronaviruses. Mutagenic analysis of SARS-CoV S, combined with cell–cell fusion and pseudotyped virion infectivity assays, showed a critical role for the core-conserved residues C822, D830, L831, and C833. Based on available predictive models, we propose that the conserved domain flanked by cysteines 822 and 833 forms a loop structure that interacts with components of the SARS-CoV S trimer to control the activation of membrane fusion.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Nucleocapsid Is a Multifunctional Protein</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Aug 7</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) is a structural protein that forms complexes with genomic RNA, interacts with the viral membrane protein during virion assembly and plays a critical role in enhancing the efficiency of virus transcription and assembly. Recent studies have confirmed that N is a multifunctional protein. The aim of this review is to highlight the properties and functions of the N protein, with specific reference to (i) the topology; (ii) the intracellular localization and (iii) the functions of the protein.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>The Transmembrane Oligomers of Coronavirus Protein E</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have tested the hypothesis that severe acute respiratory syndrome (SARS) coronavirus protein E (SCoVE) and its homologs in other coronaviruses associate through their putative transmembrane domain to form homooligomeric α-helical bundles in vivo. For this purpose, we have analyzed the results of molecular dynamics simulations where all possible conformational and aggregational space was systematically explored. Two main assumptions were considered; the first is that protein E contains one transmembrane α-helical domain, with its N- and C-termini located in opposite faces of the lipid bilayer. The second is that protein E forms the same type of transmembrane oligomer and with identical backbone structure in different coronaviruses. The models arising from the molecular dynamics simulations were tested for evolutionary conservation using 13 coronavirus protein E homologous sequences. It is extremely unlikely that if any of our assumptions were not correct we would find a persistent structure for all the sequences tested. We show that a low energy dimeric, trimeric and two pentameric models appear to be conserved through evolution, and are therefore likely to be present in vivo. In support of this, we have observed only dimeric, trimeric, and pentameric aggregates for the synthetic transmembrane domain of SARS protein E in SDS. The models obtained point to residues essential for protein E oligomerization in the life cycle of the SARS virus, specifically N15. In addition, these results strongly support a general model where transmembrane domains transiently adopt many aggregation states necessary for function.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus spike protein (S) plays a key role in the early steps of viral infection, with the S1 domain responsible for receptor binding and the S2 domain mediating membrane fusion. In some cases, the S protein is proteolytically cleaved at the S1–S2 boundary. In the case of the severe acute respiratory syndrome coronavirus (SARS-CoV), it has been shown that virus entry requires the endosomal protease cathepsin L; however, it was also found that infection of SARS-CoV could be strongly induced by trypsin treatment. Overall, in terms of how cleavage might activate membrane fusion, proteolytic processing of the SARS-CoV S protein remains unclear. Here, we identify a proteolytic cleavage site within the SARS-CoV S2 domain (S2′, R797). Mutation of R797 specifically inhibited trypsin-dependent fusion in both cell–cell fusion and pseudovirion entry assays. We also introduced a furin cleavage site at both the S2′ cleavage site within S2 793-KPTKR-797 (S2′), as well as at the junction of S1 and S2. Introduction of a furin cleavage site at the S2′ position allowed trypsin-independent cell–cell fusion, which was strongly increased by the presence of a second furin cleavage site at the S1–S2 position. Taken together, these data suggest a novel priming mechanism for a viral fusion protein, with a critical proteolytic cleavage event on the SARS-CoV S protein at position 797 (S2′), acting in concert with the S1–S2 cleavage site to mediate membrane fusion and virus infectivity.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>The role of programmed-1 ribosomal frameshifting in coronavirus propagation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses have the potential to cause significant economic, agricultural and health problems. The severe acute respiratory syndrome (SARS) associated coronavirus outbreak in late 2002, early 2003 called attention to the potential damage that coronaviruses could cause in the human population. The ensuing research has enlightened many to the molecular biology of coronaviruses. A programmed -1 ribosomal frameshift is required by coronaviruses for the production of the RNA dependent RNA polymerase which in turn is essential for viral replication. The frameshifting signal encoded in the viral genome has additional features that are not essential for frameshifting. Elucidation of the differences between coronavirus frameshift signals and signals from other viruses may help our understanding of these features. Here we summarize current knowledge and add additional insight regarding the function of the programmed -1 ribosomal frameshift signal in the coronavirus lifecycle.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>The 3′-Terminal 55 Nucleotides of Bovine Coronavirus Defective Interfering RNA Harbor Cis-Acting Elements Required for Both Negative- and Positive-Strand RNA Synthesis</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 May 22</span></i><p style="text-align: justify; text-justify: inter-word;">The synthesis of the negative-strand [(−)-strand] complement of the ∼30 kilobase, positive-strand [(+)-strand] coronaviral genome is a necessary early step for genome replication. The identification of cis-acting elements required for (−)-strand RNA synthesis in coronaviruses, however, has been hampered due to insufficiencies in the techniques used to detect the (−)-strand RNA species. Here, we employed a method of head-to-tail ligation and real-time quantitative reverse transcription polymerase chain reaction (qRT-PCR) to detect and quantitate the synthesis of bovine coronavirus (BCoV) defective interfering (DI) RNA (−) strands. Furthermore, using the aforementioned techniques along with Northern blot assay, we specifically defined the cis-acting RNA elements within the 3′-terminal 55 nucleotides (nts) which function in the synthesis of (−)- or (+)-strand BCoV DI RNA. The major findings are as follows: (i) nts from -5 to -39 within the 3′-terminal 55 nts are the cis-acting elements responsible for (−)-strand BCoV DI RNA synthesis, (ii) nts from −3 to −34 within the 3′-terminal 55 nts are cis-acting elements required for (+)-strand BCoV DI RNA synthesis, and (iii) the nucleotide species at the 3′-most position (−1) is important, but not critical, for both (−)- and (+)-strand BCoV DI RNA synthesis. These results demonstrate that the 3′-terminal 55 nts in BCoV DI RNA harbor cis-acting RNA elements required for both (−)- and (+)-strand DI RNA synthesis and extend our knowledge on the mechanisms of coronavirus replication. The method of head-to-tail ligation and qRT-PCR employed in the study may also be applied to identify other cis-acting elements required for (−)-strand RNA synthesis in coronaviruses.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Processing of the Human Coronavirus 229E Replicase Polyproteins by the Virus-Encoded 3C-Like Proteinase: Identification of Proteolytic Products and Cleavage Sites Common to pp1a and pp1ab</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Replicase gene expression by the human coronavirus 229E involves the synthesis of two large polyproteins, pp1a and pp1ab. Experimental evidence suggests that these precursor molecules are subject to extensive proteolytic processing. In this study, we show that a chymotrypsin-like enzyme, the virus-encoded 3C-like proteinase (3CL(pro)), cleaves within a common region of pp1a and pp1ab (amino acids 3490 to 4068) at four sites. trans-cleavage assays revealed that polypeptides of 5, 23, 12, and 16 kDa are processed from pp1a/pp1ab by proteolysis of the peptide bonds Q3546/S3547, Q3629/S3630, Q3824/N3825, and Q3933/A3934. Relative rate constants for the 3CL(pro)-mediated cleavages Q2965/A2966, Q3267/S3268, Q3824/N3825, and Q3933/A3934 were derived by competition experiments using synthetic peptides and recombinant 3CL(pro). The results indicate that coronavirus cleavage sites differ significantly with regard to their susceptibilities to proteolysis by 3CL(pro). Finally, immunoprecipitation with specific rabbit antisera was used to detect the pp1a/pp1ab processing end products in virus-infected cells, and immunofluorescence data that suggest an association of these polypeptides with intracellular membranes were obtained.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence for a Common Evolutionary Origin of Coronavirus Spike Protein Receptor-Binding Subunits</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Among different coronavirus genera, the receptor-binding S1 subunits of their spike proteins differ in primary, secondary, and tertiary structures. This study identified shared structural topologies (connectivity of secondary structural elements) in S1 domains of different coronavirus genera. The results suggest that coronavirus S1 subunits share a common evolutionary origin but have attained diverse sequences and structures following extensive divergent evolution. The results also increase understanding of the structures and functions of coronavirus S1 domains whose tertiary structures are currently unknown.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus E Protein: Assembly and Beyond</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Mar 8</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein that has an important role in the assembly of virions. Recent studies have indicated that the E protein has functions during infection beyond assembly, including in virus egress and in the host stress response. Additionally, the E protein has ion channel activity, interacts with host proteins, and may have multiple membrane topologies. The goal of this review is to highlight the properties and functions of the E protein, and speculate on how they may be related.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>The Hydrophobic Domain of Infectious Bronchitis Virus E Protein Alters the Host Secretory Pathway and Is Important for Release of Infectious Virus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus (CoV) E protein plays an important role in virus assembly. The E protein is made in excess during infection and has been shown to have ion channel activity in planar lipid bilayers. However, a role in infection for the unincorporated E or its ion channel activity has not been described. To further investigate the function of the infectious bronchitis virus (IBV) E protein, we developed a recombinant version of IBV in which the E protein was replaced by a mutant containing a heterologous hydrophobic domain. The mutant virus, IBV-EG3, was defective in release of infectious virus particles. Further characterization of IBV-EG3 revealed that damaged particles appeared to accumulate intracellularly. The phenotype of IBV-EG3 suggested that the hydrophobic domain of IBV E may be important for the forward trafficking of cargo, so we determined whether IBV E facilitated the delivery of cargo to the plasma membrane. Surprisingly, we found that IBV E, but not EG3, dramatically reduced the delivery of cargo to the plasma membrane by impeding movement through the Golgi complex. Furthermore, we observed that overexpression of IBV E, but not EG3, induced the disassembly of the Golgi complex. Finally, we determined that the delivery of IBV S to the plasma membrane was reduced in cells infected with wild-type-IBV compared to those infected with IBV-EG3. Our results indicated that the hydrophobic domain of IBV E alters the host secretory pathway to the apparent advantage of the virus.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>X-Ray Structures of the N- and C-Terminal Domains of a Coronavirus Nucleocapsid Protein: Implications for Nucleocapsid Formation</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses cause a variety of respiratory and enteric diseases in animals and humans including severe acute respiratory syndrome. In these enveloped viruses, the filamentous nucleocapsid is formed by the association of nucleocapsid (N) protein with single-stranded viral RNA. The N protein is a highly immunogenic phosphoprotein also implicated in viral genome replication and in modulating cell signaling pathways. We describe the structure of the two proteolytically resistant domains of the N protein from infectious bronchitis virus (IBV), a prototype coronavirus. These domains are located at its N- and C-terminal ends (NTD and CTD, respectively). The NTD of the IBV Gray strain at 1.3-Å resolution exhibits a U-shaped structure, with two arms rich in basic residues, providing a module for specific interaction with RNA. The CTD forms a tightly intertwined dimer with an intermolecular four-stranded central β-sheet platform flanked by α helices, indicating that the basic building block for coronavirus nucleocapsid formation is a dimeric assembly of N protein. The variety of quaternary arrangements of the NTD and CTD revealed by the analysis of the different crystal forms delineates possible interfaces that could be used for the formation of a flexible filamentous ribonucleocapsid. The striking similarity between the dimeric structure of CTD and the nucleocapsid-forming domain of a distantly related arterivirus indicates a conserved mechanism of nucleocapsid formation for these two viral families.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>The Cellular Interactome of the Coronavirus Infectious Bronchitis Virus Nucleocapsid Protein and Functional Implications for Virus Biology</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) protein plays a multifunctional role in the virus life cycle, from regulation of replication and transcription and genome packaging to modulation of host cell processes. These functions are likely to be facilitated by interactions with host cell proteins. The potential interactome of the infectious bronchitis virus (IBV) N protein was mapped using stable isotope labeling with amino acids in cell culture (SILAC) coupled to a green fluorescent protein-nanotrap pulldown methodology and liquid chromatography-tandem mass spectrometry. The addition of the SILAC label allowed discrimination of proteins that were likely to specifically bind to the N protein over background binding. Overall, 142 cellular proteins were selected as potentially binding to the N protein, many as part of larger possible complexes. These included ribosomal proteins, nucleolar proteins, translation initiation factors, helicases, and hnRNPs. The association of selected cellular proteins with IBV N protein was confirmed by immunoblotting, cosedimentation, and confocal microscopy. Further, the localization of selected proteins in IBV-infected cells as well as their activity during virus infection was assessed by small interfering RNA-mediated depletion, demonstrating the functional importance of cellular proteins in the biology of IBV. This interactome not only confirms previous observations made with other coronavirus and IBV N proteins with both overexpressed proteins and infectious virus but also provides novel data that can be exploited to understand the interaction between the virus and the host cell.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>The Porcine Deltacoronavirus Replication Organelle Comprises Double-Membrane Vesicles and Zippered Endoplasmic Reticulum with Double-Membrane Spherules</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 5</span></i><p style="text-align: justify; text-justify: inter-word;">Porcine deltacoronavirus (PDCoV) was first identified in Hong Kong in 2012 from samples taken from pigs in 2009. PDCoV was subsequently identified in the USA in 2014 in pigs with a history of severe diarrhea. The virus has now been detected in pigs in several countries around the world. Following the development of tissue culture adapted strains of PDCoV, it is now possible to address questions regarding virus–host cell interactions for this genera of coronavirus. Here, we presented a detailed study of PDCoV-induced replication organelles. All positive-strand RNA viruses induce the rearrangement of cellular membranes during virus replication to support viral RNA synthesis, forming the replication organelle. Replication organelles for the Alpha-, Beta-, and Gammacoronavirus genera have been characterized. All coronavirus genera induced the formation of double-membrane vesicles (DMVs). In addition, Alpha- and Betacoronaviruses induce the formation of convoluted membranes, while Gammacoronaviruses induce the formation of zippered endoplasmic reticulum (ER) with tethered double-membrane spherules. However, the structures induced by Deltacoronaviruses, particularly the presence of convoluted membranes or double-membrane spherules, are unknown. Initially, the dynamics of PDCoV strain OH-FD22 replication were assessed with the onset of viral RNA synthesis, protein synthesis, and progeny particle release determined. Subsequently, virus-induced membrane rearrangements were identified in infected cells by electron microscopy. As has been observed for all other coronaviruses studied to date, PDCoV replication was found to induce the formation of double-membrane vesicles. Significantly, however, PDCoV replication was also found to induce the formation of regions of zippered endoplasmic reticulum, small associated tethered vesicles, and double-membrane spherules. These structures strongly resemble the replication organelle induced by avian Gammacoronavirus infectious bronchitis virus.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Proteolytic Processing at the Amino Terminus of Human Coronavirus 229E Gene 1-Encoded Polyproteins: Identification of  a Papain-Like Proteinase and Its Substrate</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Expression of the coronavirus gene 1-encoded polyproteins, pp1a and pp1ab, is linked to a series of proteolytic events involving virus-encoded proteinases. In this study, we used transfection and immunoprecipitation assays to show that the human coronavirus 229E-encoded papain-like cysteine proteinase, PCP1, is responsible for the release of an amino-terminal protein, p9, from the gene 1-encoded polyproteins. The same protein, p9, has also been identified in virus-infected cells. Furthermore, using an in vitro trans-cleavage assay, we defined the proteolytic cleavage site at the carboxyl terminus of p9 as pp1a-pp1ab amino acids Gly-111 and Asn-112. These results and a comparative sequence analysis suggest that substrate positions P1 and P5 seem to be the major determinants of the PCP1 cleavage site and that the latter can occupy a variable position at the amino terminus of the coronavirus pp1a and pp1ab polyproteins. By combining the trans-cleavage assay with deletion mutagenesis, we were also able to locate the boundaries of the active PCP1 domain between pp1a-pp1ab amino acids Gly-861–Glu-975 and Asn-1209–Gln-1285. Finally, codon mutagenesis was used to show that Cys-1054 and His-1205 are essential for PCP1 proteolytic activity, suggesting that these amino acids most likely have a catalytic function.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Viral Transmission Behavior via Protein Intrinsic Disorder Prediction: Coronaviruses</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Oct 14</span></i><p style="text-align: justify; text-justify: inter-word;">Besides being a common threat to farm animals and poultry, coronavirus (CoV) was responsible for the human severe acute respiratory syndrome (SARS) epidemic in 2002–4. However, many aspects of CoV behavior, including modes of its transmission, are yet to be fully understood. We show that the amount and the peculiarities of distribution of the protein intrinsic disorder in the viral shell can be used for the efficient analysis of the behavior and transmission modes of CoV. The proposed model allows categorization of the various CoVs by the peculiarities of disorder distribution in their membrane (M) and nucleocapsid (N). This categorization enables quick identification of viruses with similar behaviors in transmission, regardless of genetic proximity. Based on this analysis, an empirical model for predicting the viral transmission behavior is developed. This model is able to explain some behavioral aspects of important coronaviruses that previously were not fully understood. The new predictor can be a useful tool for better epidemiological, clinical, and structural understanding of behavior of both newly emerging viruses and viruses that have been known for a long time. A potentially new vaccine strategy could involve searches for viral strains that are characterized by the evolutionary misfit between the peculiarities of the disorder distribution in their shells and their behavior.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-Coronavirus Replication Is Supported by a Reticulovesicular Network of Modified Endoplasmic Reticulum</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Sep 16</span></i><p style="text-align: justify; text-justify: inter-word;">Positive-strand RNA viruses, a large group including human pathogens such as SARS-coronavirus (SARS-CoV), replicate in the cytoplasm of infected host cells. Their replication complexes are commonly associated with modified host cell membranes. Membrane structures supporting viral RNA synthesis range from distinct spherular membrane invaginations to more elaborate webs of packed membranes and vesicles. Generally, their ultrastructure, morphogenesis, and exact role in viral replication remain to be defined. Poorly characterized double-membrane vesicles (DMVs) were previously implicated in SARS-CoV RNA synthesis. We have now applied electron tomography of cryofixed infected cells for the three-dimensional imaging of coronavirus-induced membrane alterations at high resolution. Our analysis defines a unique reticulovesicular network of modified endoplasmic reticulum that integrates convoluted membranes, numerous interconnected DMVs (diameter 200–300 nm), and “vesicle packets” apparently arising from DMV merger. The convoluted membranes were most abundantly immunolabeled for viral replicase subunits. However, double-stranded RNA, presumably revealing the site of viral RNA synthesis, mainly localized to the DMV interior. Since we could not discern a connection between DMV interior and cytosol, our analysis raises several questions about the mechanism of DMV formation and the actual site of SARS-CoV RNA synthesis. Our data document the extensive virus-induced reorganization of host cell membranes into a network that is used to organize viral replication and possibly hide replicating RNA from antiviral defense mechanisms. Together with biochemical studies of the viral enzyme complex, our ultrastructural description of this “replication network” will aid to further dissect the early stages of the coronavirus life cycle and its virus-host interactions.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Interaction of the Coronavirus Infectious Bronchitis Virus Membrane Protein with β-Actin and Its Implication in Virion Assembly and Budding</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Mar 16</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus M protein is an essential component of virion and plays pivotal roles in virion assembly, budding and maturation. The M protein is integrated into the viral envelope with three transmembrane domains flanked by a short amino-terminal ectodomain and a large carboxy-terminal endodomain. In this study, we showed co-purification of the M protein from coronavirus infectious bronchitis virus (IBV) with actin. To understand the cellular factors that may be involved in virion assembly, budding and maturation processes, IBV M was used as the bait in a yeast two-hybrid screen, resulting in the identification of β-actin as a potentially interacting partner. This interaction was subsequently confirmed by coimmunoprecipitation and immunofluorescence microscopy in mammalian cells, and mutation of amino acids A159 and K160 on the M protein abolished the interaction. Introduction of the A159-K160 mutation into an infectious IBV clone system blocks the infectivity of the clone, although viral RNA replication and subgenomic mRNA transcription were actively detected. Disruption of actin filaments with cell-permeable agent cytochalasin D at early stages of the infection cycle led to the detection of viral protein synthesis in infected cells but not release of virus particles to the cultured media. However, the same treatment at late stages of the infection cycle did not affect the release of virus particles to the media, suggesting that disruption of the actin filaments might block virion assembly and budding, but not release of the virus particles. This study reveals an essential function of actin in the replication cycle of coronavirus.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of second-site revertants of a murine coronavirus nucleocapsid protein deletion mutant and construction of nucleocapsid protein mutants by targeted RNA recombination.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Alb4 mutant of the coronavirus mouse hepatitis virus (MHV) is both temperature sensitive and thermolabile owing to a deletion in the gene encoding its nucleocapsid (N) protein. The deletion removes 29 amino acids that constitute a putative spacer region preceding the carboxyl-terminal domain of the protein. As a step toward understanding the structure and function of the MHV N protein, we isolated multiple independent revertants of Alb4 that totally or partially regained the ability to form large (wild-type-sized) plaques at the nonpermissive temperature. The N proteins of these revertant viruses concomitantly regained the ability to bind to RNA in vitro at a temperature that was restrictive for RNA binding by Alb4 N protein. Sequence analysis of the N genes of the revertants revealed that each contained a single second-site point mutation that compensated for the effects of the deletion. All reverting mutations were clustered within a stretch of 40 amino acids centered some 80 residues on the amino side of the Alb4 deletion, within a domain to which the RNA-binding activity of N had been previously mapped. By means of a targeted RNA recombination method that we have recently developed, two of the reverting mutations were introduced into a wild-type MHV genomic background. The resulting recombinants were stable and showed no gross phenotypic differences from the wild type. A detailed analysis of one, however, revealed that it was at a selective disadvantage with respect to the wild type.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Dodecamer Structure of Severe Acute Respiratory Syndrome Coronavirus Nonstructural Protein nsp10</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome coronavirus (SARS-CoV) nonstructural proteins nsp1 to nsp16 have been implicated by genetic analysis in the assembly of a functional replication/transcription complex. We report the crystal structure of nsp10 from SARS-CoV at 2.1-Å resolution. The nsp10 structure has a novel fold, and 12 identical subunits assemble to form a unique spherical dodecameric architecture. Two zinc fingers have been identified from the nsp10 monomer structure with the sequence motifs C-(X)(2)-C-(X)(5)-H-(X)(6)-C and C-(X)(2)-C-(X)(7)-C-(X)-C. The nsp10 crystal structure is the first of a new class of zinc finger protein three-dimensional structures to be revealed experimentally. The zinc finger sequence motifs are conserved among all three coronavirus antigenic groups, implicating an essential function for nsp10 in all coronaviruses. Based on the structure, we propose that nsp10 is a transcription factor for coronavirus replication/transcription.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Nucleocapsid-independent assembly of coronavirus-like particles by co-expression of viral envelope protein genes.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Budding of enveloped viruses has been shown to be driven by interactions between a nucleocapsid and a proteolipid membrane. By contrast, we here describe the assembly of viral envelopes independent of a nucleocapsid. Membrane particles containing coronaviral envelope proteins were assembled in and released from animal cells co-expressing these proteins' genes from transfected plasmids. Of the three viral membrane proteins only two were required for particle formation, the membrane glycoprotein (M) and the small envelope protein (E). The spike (S) protein was dispensable but was incorporated when present. Importantly, the nucleocapsid protein (N) was neither required not taken into the particles when present. The E protein, recently recognized to be a structural protein, was shown to be an integral membrane protein. The envelope vesicles were found by immunogold labelling and electron microscopy to form a homogeneous population of spherical particles indistinguishable from authentic coronavirions in size (approximately 100 nm in diameter) and shape. They were less dense than virions and sedimented slightly slower than virions in sucrose velocity gradients. The nucleocapsid-independent formation of apparently bona fide viral envelopes represents a novel mode of virus assembly.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 23</span></i><p style="text-align: justify; text-justify: inter-word;">As cell-invading molecular machinery, coronavirus spike proteins pose an evolutionary conundrum due to their high divergence. In this study, we determined the cryo-EM structure of avian infectious bronchitis coronavirus (IBV) spike protein from the γ-genus. The trimeric IBV spike ectodomain contains three receptor-binding S1 heads and a trimeric membrane-fusion S2 stalk. While IBV S2 is structurally similar to those from the other genera, IBV S1 possesses structural features that are unique to different other genera, thereby bridging these diverse spikes into an evolutionary spectrum. Specifically, among different genera, the two domains of S1, the N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), diverge from simpler tertiary structures and quaternary packing to more complex ones, leading to different functions of the spikes in receptor usage and membrane fusion. Based on the above structural and functional comparisons, we propose that the evolutionary spectrum of coronavirus spikes follows the order of α-, δ-, γ-, and β-genus. This study has provided insight into the evolutionary relationships among coronavirus spikes and deepened our understanding of their structural and functional diversity.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure-Function Analysis of Severe Acute Respiratory Syndrome Coronavirus RNA Cap Guanine-N7-Methyltransferase</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses possess a cap structure at the 5′ ends of viral genomic RNA and subgenomic RNAs, which is generated through consecutive methylations by virally encoded guanine-N7-methyltransferase (N7-MTase) and 2′-O-methyltransferase (2′-O-MTase). The coronaviral N7-MTase is unique for its physical linkage with an exoribonuclease (ExoN) harbored in nonstructural protein 14 (nsp14) of coronaviruses. In this study, the structure-function relationships of the N7-MTase were analyzed by deletion and site-directed mutagenesis of severe acute respiratory syndrome coronavirus (SARS-CoV) nsp14. The results showed that the ExoN domain is closely involved in the activity of the N7-MTase, suggesting that coronavirus N7-MTase is different from all other viral N7-MTases, which are separable from other structural domains located in the same polypeptide. Two of the 12 critical residues identified to be essential for the N7-MTase were located at the N terminus of the core ExoN domain, reinforcing a role of the ExoN domain in the N7-MTase activity of nsp14. The other 10 critical residues were distributed throughout the N7-MTase domain but localized mainly in the S-adenosyl-l-methionine (SAM)-binding pocket and key structural elements of the MTase fold of nsp14. The sequence motif DxGxPxA (amino acids [aa] 331 to 338) was identified as the key part of the SAM-binding site. These results provide insights into the structure and functional mechanisms of coronaviral nsp14 N7-MTase.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Glycosylation of the Severe Acute Respiratory Syndrome Coronavirus Triple-Spanning Membrane Proteins 3a and M</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome coronavirus (SARS-CoV) open reading frame 3a protein has recently been shown to be a structural protein. The protein is encoded by one of the so-called group-specific genes and has no sequence homology with any of the known structural or group-specific proteins of coronaviruses. It does, however, have several similarities to the coronavirus M proteins; (i) they are triple membrane spanning with the same topology, (ii) they have similar intracellular localizations (predominantly Golgi), (iii) both are viral structural proteins, and (iv) they appear to interact with the E and S proteins, as well as with each other. The M protein plays a crucial role in coronavirus assembly and is glycosylated in all coronaviruses, either by N-linked or by O-linked oligosaccharides. The conserved glycosylation of the coronavirus M proteins and the resemblance of the 3a protein to them led us to investigate the glycosylation of these two SARS-CoV membrane proteins. The proteins were expressed separately using the vaccinia virus T7 expression system, followed by metabolic labeling. Pulse-chase analysis showed that both proteins were modified, although in different ways. While the M protein acquired cotranslationally oligosaccharides that could be removed by PNGaseF, the 3a protein acquired its modifications posttranslationally, and they were not sensitive to the N-glycosidase enzyme. The SARS-CoV 3a protein, however, was demonstrated to contain sialic acids, indicating the presence of oligosaccharides. O-glycosylation of the 3a protein was indeed confirmed using an in situ O-glycosylation assay of endoplasmic reticulum-retained mutants. In addition, we showed that substitution of serine and threonine residues in the ectodomain of the 3a protein abolished the addition of the O-linked sugars. Thus, the SARS-CoV 3a protein is an O-glycosylated glycoprotein, like the group 2 coronavirus M proteins but unlike the SARS-CoV M protein, which is N glycosylated.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>The transmissible gastroenteritis coronavirus contains a spherical core shell consisting of M and N proteins.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are enveloped RNA viruses involved in a variety of pathologies that affect animals and humans. Existing structural models of these viruses propose a helical nucleocapsid under the virion envelope as the unique internal structure. In the present work, we have analyzed the structure of the transmissible gastroenteritis coronavirus. The definition of its organization supports a new structural model for coronaviruses, since a spherical, probably icosahedral, internal core has been characterized. Disruption of these cores induces the release of N-protein-containing helical nucleocapsids. Immunogold mapping and protein analysis of purified cores showed that they consist of M and N proteins, M being the main core shell component. This surprising finding, together with the fact that M protein molecules are also located in the virion envelope, indicates that a reconsideration of the assembly and maturation of coronaviruses, as well as a study of potential M-protein subclasses, is needed.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>The nsp2 Replicase Proteins of Murine Hepatitis Virus and Severe Acute Respiratory Syndrome Coronavirus Are Dispensable for Viral Replication</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The positive-stranded RNA genome of the coronaviruses is translated from ORF1 to yield polyproteins that are proteolytically processed into intermediate and mature nonstructural proteins (nsps). Murine hepatitis virus (MHV) and severe acute respiratory syndrome coronavirus (SARS-CoV) polyproteins incorporate 16 protein domains (nsps), with nsp1 and nsp2 being the most variable among the coronaviruses and having no experimentally confirmed or predicted functions in replication. To determine if nsp2 is essential for viral replication, MHV and SARS-CoV genome RNA was generated with deletions of the nsp2 coding sequence (MHVΔnsp2 and SARSΔnsp2, respectively). Infectious MHVΔnsp2 and SARSΔnsp2 viruses recovered from electroporated cells had 0.5 to 1 log(10) reductions in peak titers in single-cycle growth assays, as well as a reduction in viral RNA synthesis that was not specific for any positive-stranded RNA species. The Δnsp2 mutant viruses lacked expression of both nsp2 and an nsp2-nsp3 precursor, but cleaved the engineered chimeric nsp1-nsp3 cleavage site as efficiently as the native nsp1-nsp2 cleavage site. Replication complexes in MHVΔnsp2-infected cells lacked nsp2 but were morphologically indistinguishable from those of wild-type MHV by immunofluorescence. nsp2 expressed in cells by stable retroviral transduction was specifically recruited to viral replication complexes upon infection with MHVΔnsp2. These results demonstrate that while nsp2 of MHV and SARS-CoV is dispensable for viral replication in cell culture, deletion of the nsp2 coding sequence attenuates viral growth and RNA synthesis. These findings also provide a system for the study of determinants of nsp targeting and function.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Deubiquitinating and Interferon Antagonism Activities of Coronavirus Papain-Like Proteases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses encode multifunctional proteins that are critical for viral replication and for blocking the innate immune response to viral infection. One such multifunctional domain is the coronavirus papain-like protease (PLP), which processes the viral replicase polyprotein, has deubiquitinating (DUB) activity, and antagonizes the induction of type I interferon (IFN). Here we characterized the DUB and IFN antagonism activities of the PLP domains of human coronavirus NL63 and severe acute respiratory syndrome (SARS) coronavirus to determine if DUB activity mediates interferon antagonism. We found that NL63 PLP2 deconjugated ubiquitin (Ub) and the Ub-line molecule ISG15 from cellular substrates and processed both lysine-48- and lysine-63- linked polyubiquitin chains. This PLP2 DUB activity was dependent on an intact catalytic cysteine residue. We demonstrated that in contrast to PLP2 DUB activity, PLP2-mediated interferon antagonism did not require enzymatic activity. Furthermore, addition of an inhibitor that blocks coronavirus protease/DUB activity did not abrogate interferon antagonism. These results indicated that a component of coronavirus PLP-mediated interferon antagonism was independent of protease and DUB activity. Overall, these results demonstrate the multifunctional nature of the coronavirus PLP domain as a viral protease, DUB, and IFN antagonist and suggest that these independent activities may provide multiple targets for antiviral therapies.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Why does the coronavirus spread so easily between people?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the number of coronavirus infections approaches 100,000 people worldwide, researchers are racing to understand what makes it spread so easily. A handful of genetic and structural analyses have identified a key feature of the virus — a protein on its surface — that might explain why it infects human cells so readily. Other groups are investigating the doorway through which the new coronavirus enters human tissues — a receptor on cell membranes. Both the cell receptor and the virus protein offer potential targets for drugs to block the pathogen, but researchers say it is too early to be sure. “Understanding transmission of the virus is key to its containment and future prevention,” says David Veesler, a structural virologist at the University of Washington in Seattle, who posted his team’s findings about the virus protein on the biomedical preprint server bioRxiv on 20 February1. The new virus spreads much more readily than the one that caused severe acute respiratory syndrome, or SARS (also a coronavirus), and has infected more than ten times the number of people who contracted SARS.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Model of a Putative Pore: The Pentameric α-Helical Bundle of SARS Coronavirus E Protein in Lipid Bilayers</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus responsible for the severe acute respiratory syndrome contains a small envelope protein, E, with putative involvement in host apoptosis and virus morphogenesis. To perform these functions, it has been suggested that protein E can form a membrane destabilizing transmembrane (TM) hairpin, or homooligomerize to form a TM pore. Indeed, in a recent study we reported that the α-helical putative transmembrane domain of E protein (ETM) forms several SDS-resistant TM interactions: a dimer, a trimer, and two pentameric forms. Further, these interactions were found to be evolutionarily conserved. Herein, we have studied multiple isotopically labeled ETM peptides reconstituted in model lipid bilayers, using the orientational parameters derived from infrared dichroic data. We show that the topology of ETM is consistent with a regular TM α-helix. Further, the orientational parameters obtained unequivocally correspond to a homopentameric model, by comparison with previous predictions. We have independently confirmed that the full polypeptide of E protein can also aggregate as pentamers after expression in Escherichia coli. This interaction must be stabilized, at least partially, at the TM domain. The model we report for this pentameric α-helical bundle may explain some of the permabilizing properties of protein E, and should be the basis of mutagenesis efforts in future functional studies.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Severe Acute Respiratory Syndrome Coronavirus Replicase Products and Characterization of Papain-Like Protease Activity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Gene 1 of the coronavirus associated with severe acute respiratory syndrome (SARS) encodes replicase polyproteins that are predicted to be processed into 16 nonstructural proteins (nsps 1 to 16) by two viral proteases, a papain-like protease (PLpro) and a 3C-like protease (3CLpro). Here, we identify SARS coronavirus amino-terminal replicase products nsp1, nsp2, and nsp3 and describe trans-cleavage assays that characterize the protease activity required to generate these products. We generated polyclonal antisera to glutathione S-transferase-replicase fusion proteins and used the antisera to detect replicase intermediates and products in pulse-chase experiments. We found that nsp1 (p20) is rapidly processed from the replicase polyprotein. In contrast, processing at the nsp2/3 site is less efficient, since a ≈300-kDa intermediate (NSP2-3) is detected, but ultimately nsp2 (p71) and nsp3 (p213) are generated. We found that SARS coronavirus replicase products can be detected by 4 h postinfection in the cytoplasm of infected cells and that nsps 1 to 3 colocalize with newly synthesized viral RNA in punctate, perinuclear sites consistent with their predicted role in viral RNA synthesis. To determine if PLpro is responsible for processing these products, we cloned and expressed the PLpro domain and the predicted substrates and established PLpro trans-cleavage assays. We found that the PLpro domain is sufficient for processing the predicted nsp1/2 and nsp2/3 sites. Interestingly, expression of an extended region of PLpro that includes the downstream hydrophobic domain was required for processing at the predicted nsp3/4 site. We found that the hydrophobic domain is inserted into membranes and that the lumenal domain is glycosylated at asparagine residues 2249 and 2252. Thus, the hydrophobic domain may anchor the replication complex to intracellular membranes. These studies revealed that PLpro can cleave in trans at the three predicted cleavage sites and that it requires membrane association to process the nsp3/4 cleavage site.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>The Avian Coronavirus Infectious Bronchitis Virus Undergoes Direct Low-pH-Dependent Fusion Activation during Entry into Host Cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are the causative agents of respiratory disease in humans and animals, including severe acute respiratory syndrome. Fusion of coronaviruses is generally thought to occur at neutral pH, although there is also evidence for a role of acidic endosomes during entry of a variety of coronaviruses. Therefore, the molecular basis of coronavirus fusion during entry into host cells remains incompletely defined. Here, we examined coronavirus-cell fusion and entry employing the avian coronavirus infectious bronchitis virus (IBV). Virus entry into cells was inhibited by acidotropic bases and by other inhibitors of pH-dependent endocytosis. We carried out fluorescence-dequenching fusion assays of R18-labeled virions and show that for IBV, coronavirus-cell fusion occurs in a low-pH-dependent manner, with a half-maximal rate of fusion occurring at pH 5.5. Fusion was reduced, but still occurred, at lower temperatures (20°C). We observed no effect of inhibitors of endosomal proteases on the fusion event. These data are the first direct measure of virus-cell fusion for any coronavirus and demonstrate that the coronavirus IBV employs a direct, low-pH-dependent virus-cell fusion activation reaction. We further show that IBV was not inactivated, and fusion was unaffected, by prior exposure to pH 5.0 buffer. Virions also showed evidence of reversible conformational changes in their surface proteins, indicating that aspects of the fusion reaction may be reversible in nature.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Investigation of the control of coronavirus subgenomic mRNA transcription by using T7-generated negative-sense RNA transcripts.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The subgenomic mRNAs of the coronavirus transmissible gastroenteritis virus (TGEV) are not produced in equimolar amounts. We have developed a reporter gene system to investigate the control of this differential subgenomic mRNA synthesis. Transcription of mRNAs by the TGEV polymerase was obtained from negative-sense RNA templates generated in situ from DNA containing a T7 promoter. A series of gene cassettes was produced; these cassettes comprised the reporter chloramphenicol acetyltransferase (CAT) gene downstream of transcription-associated sequences (TASs) (also referred to as intergenic sequences and promoters) believed to be involved in the synthesis of TGEV subgenomic mRNAs 6 and 7. The gene cassettes were designed so that negative-sense RNA copies of the CAT gene with sequences complementary to the TGEV TASs, or modified versions, at the 3' end would be synthesized in situ by T7 RNA polymerase. Using this system, we have demonstrated that CAT was expressed from mRNAs derived from the T7-generated negative-sense RNA transcripts only in TGEV-infected cells and only from transcripts possessing a TGEV negative-sense TAS. Analysis of the CAT mRNAs showed the presence of the TGEV leader RNA sequence at the 5' end, in keeping with observations that all coronavirus mRNAs have a 5' leader sequence corresponding to the 5' end of the genomic RNA. Our results indicated that the CAT mRNAs were transcribed from the in situ-synthesized negative-sense RNA templates without the requirement of TGEV genomic 5' or 3' sequences on the T7-generated negative-sense transcripts (3'-TAS-CAT-5'). Modification of the TGEV TASs indicated (i) that the degree of potential base pairing between the 3' end of the leader RNA and the TGEV negative-sense TAS was not the sole determinant of the amount of subgenomic mRNA transcribed and (ii) that other factors, including nucleotides flanking the TAS, are involved in the regulation of transcription of TGEV subgenomic mRNAs.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Mapping of the Coronavirus Membrane Protein Domains Involved in Interaction with the Spike Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein is the key player in virion assembly. One of its functions is to mediate the incorporation of the spikes into the viral envelope. Heterotypic interactions between M and the spike (S) protein can be demonstrated by coimmunoprecipitation and by immunofluorescence colocalization, after coexpression of their genes in eukaryotic cells. Using these assays in a mutagenetic approach, we have mapped the domains in the M protein that are involved in complex formation between M and S. It appeared that the 25-residue luminally exposed amino-terminal domain of the M protein is not important for M-S interaction. A 15-residue deletion, the insertion of a His tag, and replacement of the ectodomain by that of another coronavirus M protein did not affect the ability of the M protein to associate with the S protein. However, complex formation was sensitive to changes in the transmembrane domains of this triple-spanning protein. Deletion of either the first two or the last two transmembrane domains, known not to affect the topology of the protein, led to a considerable decrease in complex formation, but association was not completely abrogated. Various effects of changes in the part of the M protein that is located at the cytoplasmic face of the membrane were observed. Deletions of the extreme carboxy-terminal tail appeared not to interfere with M-S complex formation. However, deletions in the amphipathic domain severely affected M-S interaction. Interestingly, changes in the amino-terminal and extreme carboxy-terminal domains of M, which did not disrupt the interaction with S, are known to be fatal to the ability of the protein to engage in virus particle formation (C. A. M. de Haan, L. Kuo, P. S. Masters, H. Vennema, and P. J. M. Rottier, J. Virol. 72:6838–6850, 1998). Apparently, the structural requirements of the M protein for virus particle assembly differ from the requirements for the formation of M-S complexes.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal Structure of the C-Terminal Cytoplasmic Domain of Non-Structural Protein 4 from Mouse Hepatitis Virus A59</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul 10</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The replication of coronaviruses takes place on cytoplasmic double membrane vesicles (DMVs) originating in the endoplasmic reticulum (ER). Three trans-membrane non-structural proteins, nsp3, nsp4 and nsp6, are understood to be membrane anchors of the coronavirus replication complex. Nsp4 is localized to the ER membrane when expressed alone but is recruited into the replication complex in infected cells. It is revealed to contain four trans-membrane regions and its N- and C-termini are exposed to the cytosol. METHODOLOGY/PRINCIPAL FINDINGS: We have determined the crystal structures of the C-terminal hydrophilic domain of nsp4 (nsp4C) from MHV strain A59 and a C425S site-directed mutant. The highly conserved 89 amino acid region from T408 to Q496 is shown to possess a new fold. The wild-type (WT) structure features two monomers linked by a Cys425-Cys425 disulfide bond in one asymmetric unit. The monomers are arranged with their N- and C-termini in opposite orientations to form an “open” conformation. Mutation of Cys425 to Ser did not affect the monomer structure, although the mutant dimer adopts strikingly different conformations by crystal packing, with the cross-linked C-termini and parallel N-termini of two monomers forming a “closed” conformation. The WT nsp4C exists as a dimer in solution and can dissociate easily into monomers in a reducing environment. CONCLUSIONS/SIGNIFICANCE: As nsp4C is exposed in the reducing cytosol, the monomer of nsp4C should be physiological. This structure may serve as a basis for further functional studies of nsp4.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus JHM: Cell-Free Synthesis of Structural Protein p60</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Sac(-) cells infected with murine coronavirus strain JHM shut off host cell protein synthesis and synthesized polypeptides with molecular weights of 150,000, 60,000, and 23,000. The 60,000- and 23,000-molecular-weight polypeptides comigrated with virion structural proteins p60 and p23, and the 60,000-molecular-weight protein was identified as p60 by tryptic peptide fingerprinting. Polyadenylate-containing RNA [poly(A) RNA] extracted from the cytoplasm of infected cells directed the synthesis of both 60,000- and 23,000-molecular-weight polypeptides in messenger-dependent cell-free systems derived from mouse L-cells and rabbit reticulocytes. The reticulocyte system also synthesized a 120,000-molecular-weight polypeptide that was specifically immunoprecipitated by antiserum raised against JHM virions. The identity of the 60,000- and 23,000-molecular-weight in vitro products was established by comigration with virion proteins, immunoprecipitation, and in the case of p60, tryptic peptide fingerprinting. The cytoplasmic poly(A) RNAs which encoded p60 and p23 sedimented in sucroseformamide gradients at 17S and 19S, respectively, and were clearly separable. These RNAs were among the major poly(A) RNA species synthesized in the cytoplasm of actinomycin D-treated cells late in infection, and the in vitro translation of size-fractionated RNA released from polysomes confirmed that they represent physiological mRNA's. These results suggest that the expression of the coronavirus JHM genome involves more than one subgenomic mRNA.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of In Vivo-Interacting Domains of the Murine Coronavirus Nucleocapsid Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid protein (N), together with the large, positive-strand RNA viral genome, forms a helically symmetric nucleocapsid. This ribonucleoprotein structure becomes packaged into virions through association with the carboxy-terminal endodomain of the membrane protein (M), which is the principal constituent of the virion envelope. Previous work with the prototype coronavirus mouse hepatitis virus (MHV) has shown that a major determinant of the N-M interaction maps to the carboxy-terminal domain 3 of the N protein. To explore other domain interactions of the MHV N protein, we expressed a series of segments of the MHV N protein as fusions with green fluorescent protein (GFP) during the course of viral infection. We found that two of these GFP-N-domain fusion proteins were selectively packaged into virions as the result of tight binding to the N protein in the viral nucleocapsid, in a manner that did not involve association with either M protein or RNA. The nature of each type of binding was further explored through genetic analysis. Our results defined two strongly interacting regions of the N protein. One is the same domain 3 that is critical for M protein recognition during assembly. The other is domain N1b, which corresponds to the N-terminal domain that has been structurally characterized in detail for two other coronaviruses, infectious bronchitis virus and the severe acute respiratory syndrome coronavirus.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Infectious Bronchitis Virus Nucleoprotein Localizes to the Nucleolus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleoprotein (N) has been reported to be involved in various aspects of virus replication. We examined by confocal microscopy the subcellular localization of the avian infectious bronchitis virus N protein both in the absence and in the context of an infected cell and found that N protein localizes both to the cytoplasmic and nucleolar compartments.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Analyses of Coronavirus Assembly Interactions with Interspecies Membrane and Nucleocapsid Protein Chimeras</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein is the central actor in virion morphogenesis. M organizes the components of the viral membrane, and interactions of M with itself and with the nucleocapsid (N) protein drive virus assembly and budding. In order to further define M-M and M-N interactions, we constructed mutants of the model coronavirus mouse hepatitis virus (MHV) in which all or part of the M protein was replaced by its phylogenetically divergent counterpart from severe acute respiratory syndrome coronavirus (SARS-CoV). We were able to obtain viable chimeras containing the entire SARS-CoV M protein as well as mutants with intramolecular substitutions that partitioned M protein at the boundaries between the ectodomain, transmembrane domains, or endodomain. Our results show that the carboxy-terminal domain of N protein, N3, is necessary and sufficient for interaction with M protein. However, despite some previous genetic and biochemical evidence that mapped interactions with N to the carboxy terminus of M, it was not possible to define a short linear region of M protein sufficient for assembly with N. Thus, interactions with N protein likely involve multiple linearly discontiguous regions of the M endodomain. The SARS-CoV M chimera exhibited a conditional growth defect that was partially suppressed by mutations in the envelope (E) protein. Moreover, virions of the M chimera were markedly deficient in spike (S) protein incorporation. These findings suggest that the interactions of M protein with both E and S protein are more complex than previously thought. IMPORTANCE The assembly of coronavirus virions entails concerted interactions among the viral structural proteins and the RNA genome. One strategy to study this process is through construction of interspecies chimeras that preserve or disrupt particular inter- or intramolecular associations. In this work, we replaced the membrane (M) protein of the model coronavirus mouse hepatitis virus with its counterpart from a heterologous coronavirus. The results clarify our understanding of the interaction between the coronavirus M protein and the nucleocapsid protein. At the same time, they reveal unanticipated complexities in the interactions of M with the viral spike and envelope proteins.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Mass Spectroscopic Characterization of the Coronavirus Infectious Bronchitis Virus Nucleoprotein and Elucidation of the Role of Phosphorylation in RNA Binding by Using Surface Plasmon Resonance</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Phosphorylation of the coronavirus nucleoprotein (N protein) has been predicted to play a role in RNA binding. To investigate this hypothesis, we examined the kinetics of RNA binding between nonphosphorylated and phosphorylated infectious bronchitis virus N protein with nonviral and viral RNA by surface plasmon resonance (Biacore). Mass spectroscopic analysis of N protein identified phosphorylation sites that were proximal to RNA binding domains. Kinetic analysis, by surface plasmon resonance, indicated that nonphosphorylated N protein bound with the same affinity to viral RNA as phosphorylated N protein. However, phosphorylated N protein bound to viral RNA with a higher binding affinity than nonviral RNA, suggesting that phosphorylation of N protein determined the recognition of virus RNA. The data also indicated that a known N protein binding site (involved in transcriptional regulation) consisting of a conserved core sequence present near the 5′ end of the genome (in the leader sequence) functioned by promoting high association rates of N protein binding. Further analysis of the leader sequence indicated that the core element was not the only binding site for N protein and that other regions functioned to promote high-affinity binding.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-Electron Microscopy Structure of Porcine Deltacoronavirus Spike Protein in the Prefusion State</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus spike proteins from different genera are divergent, although they all mediate coronavirus entry into cells by binding to host receptors and fusing viral and cell membranes. Here, we determined the cryo-electron microscopy structure of porcine deltacoronavirus (PdCoV) spike protein at 3.3-Å resolution. The trimeric protein contains three receptor-binding S1 subunits that tightly pack into a crown-like structure and three membrane fusion S2 subunits that form a stalk. Each S1 subunit contains two domains, an N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD). PdCoV S1-NTD has the same structural fold as alpha- and betacoronavirus S1-NTDs as well as host galectins, and it recognizes sugar as its potential receptor. PdCoV S1-CTD has the same structural fold as alphacoronavirus S1-CTDs, but its structure differs from that of betacoronavirus S1-CTDs. PdCoV S1-CTD binds to an unidentified receptor on host cell surfaces. PdCoV S2 is locked in the prefusion conformation by structural restraint of S1 from a different monomeric subunit. PdCoV spike possesses several structural features that may facilitate immune evasion by the virus, such as its compact structure, concealed receptor-binding sites, and shielded critical epitopes. Overall, this study reveals that deltacoronavirus spikes are structurally and evolutionally more closely related to alphacoronavirus spikes than to betacoronavirus spikes; it also has implications for the receptor recognition, membrane fusion, and immune evasion by deltacoronaviruses as well as coronaviruses in general. IMPORTANCE In this study, we determined the cryo-electron microscopy structure of porcine deltacoronavirus (PdCoV) spike protein at a 3.3-Å resolution. This is the first atomic structure of a spike protein from the deltacoronavirus genus, which is divergent in amino acid sequences from the well-studied alpha- and betacoronavirus spike proteins. Here, we described the overall structure of the PdCoV spike and the detailed structure of each of its structural elements. Moreover, we analyzed the functions of each of the structural elements. Based on the structures and functions of these structural elements, we discussed the evolution of PdCoV spike protein in relation to the spike proteins from other coronavirus genera. This study combines the structure, function, and evolution of PdCoV spike protein and provides many insights into its receptor recognition, membrane fusion, and immune evasion.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS‐CoV papain-like protease (PL(pro)): expression, purification, and spectroscopic/thermodynamic characterization</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Within a decade, MERS-CoV emerged with nearly four times higher case fatality rate than an earlier outbreak of SARS-CoV and spread out in 27 countries in short span of time. As an emerging virus, combating it requires an in-depth understanding of its molecular machinery. Therefore, conformational characterization studies of coronavirus proteins are necessary to advance our knowledge of the matter for the development of antiviral therapies. In this study, MERS-CoV papain-like protease (PL(pro)) was recombinantly expressed and purified. Thermal folding pathway and thermodynamic properties were characterized using dynamic multimode spectroscopy (DMS) and thermal shift assay. DMS study showed that the PL(pro) undergoes a single thermal transition and follows a pathway of two-state folding with T (m) and van’t Hoff enthalpy values of 54.4 ± 0.1 °C and 317.1 ± 3.9 kJ/mol, respectively. An orthogonal technique based on intrinsic tryptophan fluorescence also showed that MERS-CoV PL(pro) undergoes a single thermal transition and unfolds via a pathway of two-state folding with a T (m) value of 51.4 °C. Our findings provide significant understandings of the thermodynamic and structural properties of MERS-CoV PL(pro).</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Ribosomal pausing during translation of an RNA pseudoknot.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The genomic RNA of the coronavirus infectious bronchitis virus contains an efficient ribosomal frameshift signal which comprises a heptanucleotide slippery sequence followed by an RNA pseudoknot structure. The presence of the pseudoknot is essential for high-efficiency frameshifting, and it has been suggested that its function may be to slow or stall the ribosome in the vicinity of the slippery sequence. To test this possibility, we have studied translational elongation in vitro on mRNAs engineered to contain a well-defined pseudoknot-forming sequence. Insertion of the pseudoknot at a specific location within the influenza virus PB1 mRNA resulted in the production of a new translational intermediate corresponding to the size expected for ribosomal arrest at the pseudoknot. The appearance of this protein was transient, indicating that it was a true paused intermediate rather than a dead-end product, and mutational analysis confirmed that its appearance was dependent on the presence of a pseudoknot structure within the mRNA. These observations raise the possibility that a pause is required for the frameshift process. The extent of pausing at the pseudoknot was compared with that observed at a sequence designed to form a simple stem-loop structure with the same base pairs as the pseudoknot. This structure proved to be a less effective barrier to the elongating ribosome than the pseudoknot and in addition was unable to direct efficient ribosomal frameshifting, as would be expected if pausing plays an important role in frameshifting. However, the stem-loop was still able to induce significant pausing, and so this effect alone may be insufficient to account for the contribution of the pseudoknot to frameshifting.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Major genetic marker of nidoviruses encodes a replicative endoribonuclease</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are important pathogens that cause acute respiratory diseases in humans. Replication of the ≈30-kb positive-strand RNA genome of coronaviruses and discontinuous synthesis of an extensive set of subgenome-length RNAs (transcription) are mediated by the replicase-transcriptase, a barely characterized protein complex that comprises several cellular proteins and up to 16 viral subunits. The coronavirus replicase-transcriptase was recently predicted to contain RNA-processing enzymes that are extremely rare or absent in other RNA viruses. Here, we established and characterized the activity of one of these enzymes, replicative nidoviral uridylate-specific endoribonuclease (NendoU). It is considered a major genetic marker that discriminates nidoviruses (Coronaviridae, Arteriviridae, and Roniviridae) from all other RNA virus families. Bacterially expressed forms of NendoU of severe acute respiratory syndrome coronavirus and human coronavirus 229E were revealed to cleave single-stranded and double-stranded RNA in a Mn(2+)-dependent manner. Single-stranded RNA was cleaved less specifically and effectively, suggesting that double-stranded RNA is the biologically relevant NendoU substrate. Double-stranded RNA substrates were cleaved upstream and downstream of uridylates at GUU or GU sequences to produce molecules with 2′-3′ cyclic phosphate ends. 2′-O-ribose-methylated RNA substrates proved to be resistant to cleavage by NendoU, indicating a functional link with the 2′-O-ribose methyltransferase located adjacent to NendoU in the coronavirus replicative polyprotein. A mutagenesis study verified potential active-site residues and allowed us to inactivate NendoU in the full-length human coronavirus 229E clone. Substitution of D6408 by Ala was shown to abolish viral RNA synthesis, demonstrating that NendoU has critical functions in viral replication and transcription.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Assembled coronavirus from complementation of two defective interfering RNAs.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In the presence of an RNA- temperature-sensitive (ts) mutant helper virus, two coronavirus mouse hepatitis virus (MHV) defective interfering (DI) RNAs complemented each other, resulting in the assembly of MHV particles; we used this ability to complement as a means to study coronavirus assembly. One of the two DI RNAs was DIssA, a naturally occurring self-replicating DI RNA encoding N protein and the gene 1 proteins that encode RNA polymerase function; DIssA supports the replication and transcription of other non-self-replicating DI RNAs. The other DI was a genetically engineered DI RNA that encoded sM and M proteins. Coinfection of these two DIs at the nonpermissive temperature for the ts helper virus resulted in replication and transcription of both DI RNAs but not in synthesis of the helper virus RNAs. MHV particles containing DI RNAs, N protein, and M protein, all of which were exclusively derived from the two DI RNAs, were released from the coinfected cells; the amount of sM protein was below the limits of detection. Analyses of DI RNAs with mutations in the two envelope protein genes demonstrated that M and sM proteins appeared to be required for assembly and release of MHV particles that contained DI RNAs and N protein, while S protein was not required for assembly and release of MHV particles.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the past few decades by two global outbreaks of deadly pneumonia. The coronavirus spike (S) glycoprotein initiates infection by promoting fusion of the viral and cellular membranes through conformational changes that remain largely uncharacterized. Here we report the cryoEM structure of a coronavirus S glycoprotein in the postfusion state, showing large-scale secondary, tertiary, and quaternary rearrangements compared with the prefusion trimer and rationalizing the free-energy landscape of this conformational machine. We also biochemically characterized the molecular events associated with refolding of the metastable prefusion S glycoprotein to the postfusion conformation using limited proteolysis, mass spectrometry, and single-particle EM. The observed similarity between postfusion coronavirus S and paramyxovirus F structures demonstrates that a conserved refolding trajectory mediates entry of these viruses and supports the evolutionary relatedness of their fusion subunits. Finally, our data provide a structural framework for understanding the mode of neutralization of antibodies targeting the fusion machinery and for engineering next-generation subunit vaccines or inhibitors against this medically important virus family.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-11</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of coronavirus disease (COVID-19) in China caused by SARS-CoV-2 virus continually lead to worldwide human infections and deaths. It is currently no specific viral protein targeted therapeutics yet. Viral nucleocapsid protein is a potential antiviral drug target, serving multiple critical functions during the viral life cycle. However, the structural information of SARS-CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the crystal structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. Although overall structure is similar with other reported coronavirus nucleocapsid protein N-terminal domain, the surface electrostatic potential characteristics between them are distinct. Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique potential RNA binding pocket alongside the beta-sheet core. Complemented by in vitro binding studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to SARS-CoV-2.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Nonstructural Protein 16 Is a Cap-0 Binding Enzyme Possessing (Nucleoside-2′O)-Methyltransferase Activity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus family of positive-strand RNA viruses includes important pathogens of livestock, companion animals, and humans, including the severe acute respiratory syndrome coronavirus that was responsible for a worldwide outbreak in 2003. The unusually complex coronavirus replicase/transcriptase is comprised of 15 or 16 virus-specific subunits that are autoproteolytically derived from two large polyproteins. In line with bioinformatics predictions, we now show that feline coronavirus (FCoV) nonstructural protein 16 (nsp16) possesses an S-adenosyl-l-methionine (AdoMet)-dependent RNA (nucleoside-2′O)-methyltransferase (2′O-MTase) activity that is capable of cap-1 formation. Purified recombinant FCoV nsp16 selectively binds to short capped RNAs. Remarkably, an N7-methyl guanosine cap ((7Me)GpppAC(3-6)) is a prerequisite for binding. High-performance liquid chromatography analysis demonstrated that nsp16 mediates methyl transfer from AdoMet to the 2′O position of the first transcribed nucleotide, thus converting (7Me)GpppAC(3-6) into (7Me)GpppA(2′)(O)(Me)C(3-6). The characterization of 11 nsp16 mutants supported the previous identification of residues K45, D129, K169, and E202 as the putative K-D-K-E catalytic tetrad of the enzyme. Furthermore, residues Y29 and F173 of FCoV nsp16, which may be the functional counterparts of aromatic residues involved in substrate recognition by the vaccinia virus MTase VP39, were found to be essential for both substrate binding and 2′O-MTase activity. Finally, the weak inhibition profile of different AdoMet analogues indicates that nsp16 has evolved an atypical AdoMet binding site. Our results suggest that coronavirus mRNA carries a cap-1, onto which 2′O methylation follows an order of events in which 2′O-methyl transfer must be preceded by guanine N7 methylation, with the latter step being performed by a yet-unknown N7-specific MTase.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Mouse Hepatitis Coronavirus A59 Nucleocapsid Protein Phosphorylation Sites</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) is a multifunctional phosphoprotein that encapsidates the genomic RNA into a helical nucleocapsid within the mature virion. The protein also plays roles in viral RNA transcription and/or replication and possibly viral mRNA translation. Phosphorylation is one of the most common post-translation modifications that plays important regulatory roles in modulating protein functions. It has been speculated for sometime that phosphorylation could play an important role in regulation of coronavirus N protein functions. As a first step toward positioning to address this we have identified the amino acids that are phosphorylated on the mouse hepatitis coronavirus (MHV) A59 N protein. High performance liquid chromatography coupled with electrospray ionization tandem mass spectrometry (HPLC-ESI-MS/MS) was used to identify phosphorylated sites on the N protein from both infected cells and purified extracellular virions. A total of six phosphorylated sites (S162, S170, T177, S389, S424 and T428) were identified on the protein from infected cells. The same six sites were also phosphorylated on the extracellular mature virion N protein. This is the first identification of phosphorylated sites for a group II coronavirus N protein.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of mouse hepatitis coronavirus A59 nucleocapsid protein phosphorylation sites</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nucleocapsid (N) is a multifunctional phosphoprotein that encapsidates the genomic RNA into a helical nucleocapsid within the mature virion. The protein also plays roles in viral RNA transcription and/or replication and possibly viral mRNA translation. Phosphorylation is one of the most common post-translation modifications that plays important regulatory roles in modulating protein functions. It has been speculated for sometime that phosphorylation could play an important role in regulation of coronavirus N protein functions. As a first step toward positioning to address this we have identified the amino acids that are phosphorylated on the mouse hepatitis coronavirus (MHV) A59 N protein. High performance liquid chromatography coupled with electrospray ionization tandem mass spectrometry (HPLC-ESI-MS/MS) was used to identify phosphorylated sites on the N protein from both infected cells and purified extracellular virions. A total of six phosphorylated sites (S162, S170, T177, S389, S424 and T428) were identified on the protein from infected cells. The same six sites were also phosphorylated on the extracellular mature virion N protein. This is the first identification of phosphorylated sites for a group II coronavirus N protein.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>The proteome of the infectious bronchitis virus Beau-R virion</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious bronchitis is a highly contagious respiratory disease of poultry caused by the coronavirus infectious bronchitis virus (IBV). It was thought that coronavirus virions were composed of three major viral structural proteins until investigations of other coronaviruses showed that the virions also include viral non-structural and genus-specific accessory proteins as well as host-cell proteins. To study the proteome of IBV virions, virus was grown in embryonated chicken eggs, purified by sucrose-gradient ultracentrifugation and analysed by mass spectrometry. Analysis of three preparations of purified IBV yielded the three expected structural proteins plus 35 additional virion-associated host proteins. The virion-associated host proteins had a diverse range of functional attributions, being involved in cytoskeleton formation, RNA binding and protein folding pathways. Some of these proteins were unique to this study, while others were found to be orthologous to proteins identified in severe acute respiratory syndrome coronavirus virions and also virions from a number of other RNA and DNA viruses.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Acquisition of Cell–Cell Fusion Activity by Amino Acid Substitutions in Spike Protein Determines the Infectivity of a Coronavirus in Cultured Cells</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Jul 2</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus host and cell specificities are determined by specific interactions between the viral spike (S) protein and host cell receptor(s). Avian coronavirus infectious bronchitis (IBV) has been adapted to embryonated chicken eggs, primary chicken kidney (CK) cells, monkey kidney cell line Vero, and other human and animal cells. Here we report that acquisition of the cell–cell fusion activity by amino acid mutations in the S protein determines the infectivity of IBV in cultured cells. Expression of S protein derived from Vero- and CK-adapted strains showed efficient induction of membrane fusion. However, expression of S protein cloned from the third passage of IBV in chicken embryo (EP3) did not show apparent syncytia formation. By construction of chimeric S constructs and site-directed mutagenesis, a point mutation (L857-F) at amino acid position 857 in the heptad repeat 1 region of S protein was shown to be responsible for its acquisition of the cell–cell fusion activity. Furthermore, a G405-D point mutation in the S1 domain, which was acquired during further propagation of Vero-adapted IBV in Vero cells, could enhance the cell–cell fusion activity of the protein. Re-introduction of L857 back to the S gene of Vero-adapted IBV allowed recovery of variants that contain the introduced L857. However, compensatory mutations in S1 and some distant regions of S2 were required for restoration of the cell–cell fusion activity of S protein carrying L857 and for the infectivity of the recovered variants in cultured cells. This study demonstrates that acquisition of the cell–cell fusion activity in S protein determines the selection and/or adaptation of a coronavirus from chicken embryo to cultured cells of human and animal origins.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Mouse Hepatitis Virus Replicase Proteins Associate with Two Distinct Populations of Intracellular Membranes</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus replicase gene (gene 1) is translated into two co-amino-terminal polyproteins that are proteolytically processed to yield more than 15 mature proteins. Several gene 1 proteins have been shown to localize at sites of viral RNA synthesis in the infected cell cytoplasm, notably on late endosomes at early times of infection. However, both immunofluorescence and electron microscopic studies have also detected gene 1 proteins at sites distinct from the putative sites of viral RNA synthesis or virus assembly. In this study, mouse hepatitis virus (MHV)-infected cells were fractionated and analyzed to determine if gene 1 proteins segregated to more than one membrane population. Following differential centrifugation of lysates of MHV-infected DBT cells, gene 1 proteins as well as the structural N and M proteins were detected almost exclusively in a high-speed small membrane pellet. Following fractionation of the small membrane pellet on an iodixanol density gradient, the gene 1 proteins p28 and helicase cofractionated with dense membranes (1.12 to 1.13 g/ml) that also contained peak concentrations of N. In contrast, p65 and p1a-22 were detected in a distinct population of less dense membranes (1.05 to 1.09 g/ml). Viral RNA was detected in membrane fractions containing helicase, p28, and N but not in the fractions containing p65 and p1a-22. LAMP-1, a marker for late endosomes and lysosomes, was detected in both membrane populations. These results demonstrate that multiple gene 1 proteins segregate into two biochemically distinct but tightly associated membrane populations and that only one of these populations appears to be a site for viral RNA synthesis. The results further suggest that p28 is a component of the viral replication complex whereas the gene 1 proteins p1a-22 and p65 may serve roles during infection that are distinct from viral RNA transcription or replication.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The global outbreak of SARS in 2002-2003 was caused by the infection of a new human coronavirus SARS-CoV. The infection of SARS-CoV is mediated mainly through the viral surface glycoproteins, which consist of S1 and S2 subunits and form trimer spikes on the envelope of the virions. Here we report the ectodomain structures of the SARS-CoV surface spike trimer in different conformational states determined by single-particle cryo-electron microscopy. The conformation 1 determined at 4.3 Å resolution is three-fold symmetric and has all the three receptor-binding C-terminal domain 1 (CTD1s) of the S1 subunits in “down” positions. The binding of the “down” CTD1s to the SARS-CoV receptor ACE2 is not possible due to steric clashes, suggesting that the conformation 1 represents a receptor-binding inactive state. Conformations 2-4 determined at 7.3, 5.7 and 6.8 Å resolutions are all asymmetric, in which one RBD rotates away from the “down” position by different angles to an “up” position. The “up” CTD1 exposes the receptor-binding site for ACE2 engagement, suggesting that the conformations 2-4 represent a receptor-binding active state. This conformational change is also required for the binding of SARS-CoV neutralizing antibodies targeting the CTD1. This phenomenon could be extended to other betacoronaviruses utilizing CTD1 of the S1 subunit for receptor binding, which provides new insights into the intermediate states of coronavirus pre-fusion spike trimer during infection.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS‐unique fold in the Rousettus bat coronavirus HKU9</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The coronavirus nonstructural protein 3 (nsp3) is a multifunctional protein that comprises multiple structural domains. This protein assists viral polyprotein cleavage, host immune interference, and may play other roles in genome replication or transcription. Here, we report the solution NMR structure of a protein from the “SARS‐unique region” of the bat coronavirus HKU9. The protein contains a frataxin fold or double‐wing motif, which is an α + β fold that is associated with protein/protein interactions, DNA binding, and metal ion binding. High structural similarity to the human severe acute respiratory syndrome (SARS) coronavirus nsp3 is present. A possible functional site that is conserved among some betacoronaviruses has been identified using bioinformatics and biochemical analyses. This structure provides strong experimental support for the recent proposal advanced by us and others that the “SARS‐unique” region is not unique to the human SARS virus, but is conserved among several different phylogenetic groups of coronaviruses and provides essential functions.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Topology and Membrane Anchoring of the Coronavirus Replication Complex: Not All Hydrophobic Domains of nsp3 and nsp6 Are Membrane Spanning</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses express two very large replicase polyproteins, the 16 autoproteolytic cleavage products of which collectively form the membrane-anchored replication complexes. How these structures are assembled is still largely unknown, but it is likely that the membrane-spanning members of these nonstructural proteins (nsps) are responsible for the induction of the double-membrane vesicles and for anchoring the replication complexes to these membranes. For 3 of the 16 coronavirus nsps—nsp3, nsp4, and nsp6—multiple transmembrane domains are predicted. Previously we showed that, consistent with predictions, nsp4 occurs in membranes with both of its termini exposed in the cytoplasm (M. Oostra et al., J. Virol. 81:12323-12336, 2007). Strikingly, however, for both nsp3 and nsp6, predictions based on a multiple alignment of 27 coronavirus genome sequences indicate an uneven number of transmembrane domains. As a consequence, the proteinase domains present in nsp3 and nsp5 would be separated from their target sequences by the lipid bilayer. To look into this incongruity, we studied the membrane disposition of nsp3 and nsp6 of the severe acute respiratory syndrome coronavirus and murine hepatitis virus by analyzing tagged forms of the proteins expressed in cultured cells. Contrary to the predictions, in both viruses, both proteins had their amino terminus, as well as their carboxy terminus, exposed in the cytoplasm. We established that two of the three hydrophobic domains in nsp3 and six of the seven in nsp6 are membrane spanning. Subsequently, we verified that in nsp4, all four hydrophobic domains span the lipid bilayer. The occurrence of conserved non-membrane-spanning hydrophobic domains in nsp3 and nsp6 suggests an important function for these domains in coronavirus replication.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Repair and mutagenesis of the genome of a deletion mutant of the coronavirus mouse hepatitis virus by targeted RNA recombination.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The genetic characterization of a nucleocapsid (N) protein mutant of the coronavirus mouse hepatitis virus (MHV) is described. The mutant, Albany 4 (Alb4), is both temperature sensitive and thermolabile. Analysis of the progeny of a mixed infection showed that the defective Alb4 allele is recessive to wild type, and its gene product is diffusible. The N protein of Alb4 was found to be smaller than its wild-type counterpart, and sequence analysis of the Alb4 N gene revealed that it contains an internal deletion of 87 nucleotides, producing an in-frame deletion of 29 amino acids. All of these properties of Alb4 made it ideal for use as a recipient in a targeted RNA recombination experiment in which the deletion in Alb4 was repaired by recombination with synthetic RNA7, the smallest MHV subgenomic mRNA. Progeny from a cotransfection of Alb4 genomic RNA and synthetic RNA7 were selected for thermal stability. Polymerase chain reaction analysis of candidate recombinants showed that they had regained the material that is deleted in the Alb4 mutant. They also had acquired a five-nucleotide insertion in the 3' untranslated region, which had been incorporated into the synthetic RNA7 as a molecular tag. The presence of the tag was directly verified, as well, by sequencing the genomic RNA of purified recombinant viruses. This provided a clear genetic proof that the Alb4 phenotype was due to the observed deletion in the N gene. In addition, these results demonstrated that it is possible to obtain stable, independently replicating progeny from recombination between coronavirus genomic RNA and a tailored, synthetic RNA species.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>A Major Determinant for Membrane Protein Interaction Localizes to the Carboxy-Terminal Domain of the Mouse Coronavirus Nucleocapsid Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The two major constituents of coronavirus virions are the membrane (M) and nucleocapsid (N) proteins. The M protein is anchored in the viral envelope by three transmembrane segments flanked by a short amino-terminal ectodomain and a large carboxy-terminal endodomain. The M endodomain interacts with the viral nucleocapsid, which consists of the positive-strand RNA genome helically encapsidated by N protein monomers. In previous work with the coronavirus mouse hepatitis virus (MHV), a highly defective M protein mutant, MΔ2, was constructed. This mutant contained a 2-amino-acid carboxy-terminal truncation of the M protein. Analysis of second-site revertants of MΔ2 revealed mutations in the carboxy-terminal region of the N protein that compensated for the defect in the M protein. To seek further genetic evidence corroborating this interaction, we generated a comprehensive set of clustered charged-to-alanine mutants in the carboxy-terminal domain 3 of N protein. One of these mutants, CCA4, had a highly defective phenotype similar to that of MΔ2. Transfer of the CCA4 mutation into a partially diploid MHV genome showed that CCA4 was a loss-of-function mutation rather than a dominant-negative mutation. Analysis of multiple second-site revertants of CCA4 revealed mutations in both the M protein and the N protein that could compensate for the original lesion in N. These data more precisely define the region of the N protein that interacts with the M protein. Further, we found that fusion of domain 3 of the N protein to the carboxy terminus of a heterologous protein caused it to be incorporated into MHV virions.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Mutation of Asn28 disrupts the dimerization and enzymatic activity of SARS 3CL(pro)</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are responsible for a significant proportion of annual respiratory and enteric infections in humans and other mammals. The most prominent of these viruses is the Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) which causes acute respiratory and gastrointestinal infection in humans. The coronavirus main protease, 3CL(pro), is a key target for broad-spectrum antiviral development due to its critical role in viral maturation and high degree of structural conservation among coronaviruses. Dimerization is an indispensable requirement for the function of SARS 3CL(pro) and is regulated through mechanisms involving both direct and long-range interactions in the enzyme. While many of the binding interactions at the dimerization interface have been extensively studied, those that are important for long-range control are not well understood. Characterization of these dimerization mechanisms is important for the structure-based design of new treatments targeting coronavirus-based infections. Here we report that Asn28, a residue located 11 Å away from the closest residue in the opposing monomer, is essential for the enzymatic activity and dimerization of SARS 3CL(pro). Mutation of this residue to alanine almost completely inactivates the enzyme and results in a 19.2-fold decrease in the dimerization K(d). The crystallographic structure of the N28A mutant determined at a 2.35 Å resolution reveals the critical role of Asn28 in maintaining the structural integrity of the active site and in orienting key residues involved in binding at the dimer interface and substrate catalysis. These findings provide a deeper insight into complex mechanisms regulating the activity and dimerization of SARS 3CL(pro).</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Importance of Conserved Cysteine Residues in the Coronavirus Envelope Protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus envelope (E) proteins play an important, not fully understood role(s) in the virus life cycle. All E proteins have conserved cysteine residues located on the carboxy side of the long hydrophobic domain, suggesting functional significance. In this study, we confirmed that mouse hepatitis coronavirus A59 E protein is palmitoylated. To understand the role of the conserved residues and the necessity of palmitoylation, three cysteines at positions 40, 44, and 47 were changed singly and in various combinations to alanine. Double- and triple-mutant E proteins resulted in decreased virus-like particle output when coexpressed with the membrane (M) protein. Mutant E proteins were also studied in the context of a full-length infectious clone. Single-substitution viruses exhibited growth characteristics virtually identical to those of the wild-type virus, while the double-substitution mutations gave rise to viruses with less robust growth phenotypes indicated by smaller plaques and decreased virus yields. In contrast, replacement of all three cysteines resulted in crippled virus with significantly reduced yields. Triple-mutant viruses did not exhibit impairment in entry. Mutant E proteins localized properly in infected cells. A comparison of intracellular and extracellular virus yields suggested that release is only slightly impaired. E protein lacking all three cysteines exhibited an increased rate of degradation compared to that of the wild-type protein, suggesting that palmitoylation is important for the stability of the protein. Altogether, the results indicate that the conserved cysteines and presumably palmitoylation are functionally important for virus production.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>New nsp8 isoform suggests mechanism for tuning viral RNA synthesis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">During severe acute respiratory syndrome coronavirus (SARS-CoV) infection, the activity of the replication/transcription complexes (RTC) quickly peaks at 6 hours post infection (h.p.i) and then diminishes significantly in the late post-infection stages. This “down-up-down” regulation of RNA synthesis distinguishes different viral stages: primary translation, genome replication, and finally viron assembly. Regarding the nsp8 as the primase in RNA synthesis, we confirmed that the proteolysis product of the primase (nsp8) contains the globular domain (nsp8C), and indentified the resectioning site that is notably conserved in all the three groups of coronavirus. We subsequently crystallized the complex of SARS-CoV nsp8C and nsp7, and the 3-D structure of this domain revealed its capability to interfuse into the hexadecamer super-complex. This specific proteolysis may indicate one possible mechanism by which coronaviruses to switch from viral infection to genome replication and viral assembly stages.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Elucidation of the stability and functional regions of the human coronavirus OC43 nucleocapsid protein</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronavirus OC43 (HCoV-OC43) is one of the causes of the “common cold” in human during seasons of cold weather. The primary function of the HCoV-OC43 nucleocapsid protein (N protein) is to recognize viral genomic RNA, which leads to ribonucleocapsid formation. Here, we characterized the stability and identified the functional regions of the recombinant HCoV-OC43 N protein. Circular dichroism and fluorescence measurements revealed that the HCoV-OC43 N protein is more highly ordered and stabler than the SARS-CoV N protein previously studied. Surface plasmon resonance (SPR) experiments showed that the affinity of HCoV-OC43 N protein for RNA was approximately fivefold higher than that of N protein for DNA. Moreover, we found that the HCoV-OC43 N protein contains three RNA-binding regions in its N-terminal region (residues 1–173) and central-linker region (residues 174–232 and 233–300). The binding affinities of the truncated N proteins and RNA follow the order: residues 1–173–residues 233–300 > residues 174–232. SPR experiments demonstrated that the C-terminal region (residues 301–448) of HCoV-OC43 N protein lacks RNA-binding activity, while crosslinking and gel filtration analyses revealed that the C-terminal region is mainly involved in the oligomerization of the HCoV-OC43 N protein. This study may benefit the understanding of the mechanism of HCoV-OC43 nucleocapsid formation.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>Reverse Genetics System for the Avian Coronavirus Infectious Bronchitis Virus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Major advances in the study of the molecular biology of RNA viruses have resulted from the ability to generate and manipulate full-length genomic cDNAs of the viral genomes with the subsequent synthesis of infectious RNA for the generation of recombinant viruses. Coronaviruses have the largest RNA virus genomes and, together with genetic instability of some cDNA sequences in Escherichia coli, this has hampered the generation of a reverse-genetics system for this group of viruses. In this report, we describe the assembly of a full-length cDNA from the positive-sense genomic RNA of the avian coronavirus, infectious bronchitis virus (IBV), an important poultry pathogen. The IBV genomic cDNA was assembled immediately downstream of a T7 RNA polymerase promoter by in vitro ligation and cloned directly into the vaccinia virus genome. Infectious IBV RNA was generated in situ after the transfection of restricted recombinant vaccinia virus DNA into primary chick kidney cells previously infected with a recombinant fowlpox virus expressing T7 RNA polymerase. Recombinant IBV, containing two marker mutations, was recovered from the transfected cells. These results describe a reverse-genetics system for studying the molecular biology of IBV and establish a paradigm for generating genetically defined vaccines for IBV.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-Coronavirus papain-like protease: Structure, function and inhibition by designed antiviral compounds</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Over ten years have passed since the deadly human coronavirus that causes severe acute respiratory syndrome (SARS-CoV) emerged from the Guangdong Province of China. Despite the fact that the SARS-CoV pandemic infected over 8,500 individuals, claimed over 800 lives and cost billions of dollars in economic loss worldwide, there still are no clinically approved antiviral drugs, vaccines or monoclonal antibody therapies to treat SARS-CoV infections. The recent emergence of the deadly human coronavirus that causes Middle East respiratory syndrome (MERS-CoV) is a sobering reminder that new and deadly coronaviruses can emerge at any time with the potential to become pandemics. Therefore, the continued development of therapeutic and prophylactic countermeasures to potentially deadly coronaviruses is warranted. The coronaviral proteases, papain-like protease (PLpro) and 3C-like protease (3CLpro), are attractive antiviral drug targets because they are essential for coronaviral replication. Although the primary function of PLpro and 3CLpro are to process the viral polyprotein in a coordinated manner, PLpro has the additional function of stripping ubiquitin and ISG15 from host-cell proteins to aid coronaviruses in their evasion of the host innate immune responses. Therefore, targeting PLpro with antiviral drugs may have an advantage in not only inhibiting viral replication but also inhibiting the dysregulation of signaling cascades in infected cells that may lead to cell death in surrounding, uninfected cells. This review provides an up-to-date discussion on the SARS-CoV papain-like protease including a brief overview of the SARS-CoV genome and replication followed by a more in-depth discussion on the structure and catalytic mechanism of SARS-CoV PLpro, the multiple cellular functions of SARS-CoV PLpro, the inhibition of SARS-CoV PLpro by small molecule inhibitors, and the prospect of inhibiting papain-like protease from other coronaviruses. This paper forms part of a series of invited articles in Antiviral Research on “ From SARS to MERS: 10 years of research on highly pathogenic human coronaviruses.”</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /></body>