
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: using CRISPR to improve response times to future diseases</h2><br /></div><br /><br /><br /><div class="card"><h3>Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus</h3>
<p style="text-align: justify; text-justify: inter-word;">40 Methods 41 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was 42 analyzed and used for the design and development of an isothermal, CRISPR-based 43 diagnostic for the novel virus. The diagnostic performance of CRISPR-nCoV was assessed 44 and also compared across three technology platforms (mNGS, RT-PCR and CRISPR) 45 Results 46 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared 47 certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral 48 All rights reserved. No reuse allowed without permission.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/ .</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>A Toolkit for Rapid Modular Construction of Biological Circuits in Mammalian Cells</h3>
<p style="text-align: justify; text-justify: inter-word;">As a proof of concept, we built a hAAVS1 landing pad for single integration of genetic circuits, created and delivered libraries of protein variants and sgRNAs and rapidly compared multi-species Cas9-based genetic circuits. Finally, we used Ebola virus (EBOV) as a test case to address an emerging need to streamline the generation of BSL2 reagents during outbreaks of novel BSL4 viruses. Using the MTK, we rapidly built a non-infectious multicistronic ribonucleoprotein expression system for EBOV in a mammalian host. This system recapitulates the transcription and replication sub-lifecycle of the virus in a mammalian host, and can therefore be used to accelerate the study of viral biology and discovery of antiviral molecules.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Cas12a-Based On-Site and Rapid Nucleic Acid Detection of African Swine Fever</h3>
<p style="text-align: justify; text-justify: inter-word;">The mortality rate of hemorrhagic African swine fever (ASF), which targets domestic pigs and wild boars is caused by African swine fever virus (ASFV), can reach 100%. Since the first confirmed ASF outbreak in China on 3 August 2018, 156 ASF outbreaks were detected in 32 provinces. About 1,170,000 pigs were culled in order to halt further spread. There is no effective treatment or vaccine for it and the present molecular diagnosis technologies have trade-offs in sensitivity, specificity, cost and speed, and none of them cater perfectly to ASF control. Thus, a technology that overcomes the need for laboratory facilities, is relatively low cost, and rapidly and sensitively detects ASFV would be highly valuable. Here, we describe an RAA-Cas12a-based system that combines recombinase aided amplification (RAA) and CRISPR/Cas12a for ASFV detection. The fluorescence intensity readout of this system detected ASFV p72 gene levels as low as 10 aM. For on-site ASFV detection, lateral-flow strip readout was introduced for the first time in the RAA-Cas12a based system (named CORDS, Cas12abased On-site and Rapid Detection System). We used CORDS to detect target DNA highly specifically using the lateral-flow strip readout and the assay displayed no crossreactivity to other 13 swine viruses including classical swine fever (CSF). CORDS could identify the ASFV DNA target at femtomolar sensitivity in an hour at 37 • C, and only requires an incubator. For ease of use, the reagents of CORDS were lyophilized to three tubes and remained the same sensitivity when stored at 4 • C for at least 7 days. Thus, CORDS provide a rapid, sensitive and easily operable method for ASFV on-site detection. Lyophilized CORDS can withstand long-term transportation and storage, and is ready for field-based applications.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Consensus statement The species Severe acute respiratory syndrome- related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2 Coronaviridae Study Group of the International Committee on Taxonomy of Viruses*</h3>
<p style="text-align: justify; text-justify: inter-word;">significance. This will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Access to</h3>
<p style="text-align: justify; text-justify: inter-word;">Given the interconnected nature of our world today, emerging pathogens and pandemic outbreaks are an ever-growing threat to the health and economic stability of the global community. This is evident by the recent 2009 Influenza A (H1N1) pandemic, the SARS outbreak, as well as the ever-present threat of global bioterrorism. Fortunately, the biomedical community has been able to rapidly generate sequence data so these pathogens can be readily identified. To date, however, the utilization of this sequence data to rapidly produce relevant experimental results or actionable treatments is lagging in spite of obtained sequence data. Thus, a pathogenic threat that has emerged and/or developed into a pandemic can be rapidly identified; however, translating this identification into a targeted therapeutic or treatment that is rapidly available has not yet materialized. This commentary suggests that the growing technology of DNA synthesis should be fully implemented as a means to rapidly generate in vivo data and possibly actionable therapeutics soon after sequence data becomes available.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostic System for Rapid and Sensitive Differential Detection of Pathogens</h3>
<p style="text-align: justify; text-justify: inter-word;">Naturally emerging and deliberately released pathogens demand new detection strategies to allow early recognition and containment. We describe a diagnostic system for rapid, sensitive, multiplex discrimination of microbial gene sequences and report its application for detecting 22 respiratory pathogens in clinical samples.</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>The network structure and eco-evolutionary dynamics of CRISPR-induced immune diversification</h3>
<p style="text-align: justify; text-justify: inter-word;">As a heritable sequence-specific adaptive immune system, CRISPR-Cas is a powerful force shaping strain diversity in host-virus systems. While the diversity of CRISPR alleles has been explored, the associated structure and dynamics of host-virus interactions has not. We develop theory on the role of CRISPR immunity in mediating the interplay between host-virus interaction structure and eco-evolutionary dynamics in a computational model and three natural systems. We show that the structures of networks describing who infects whom and who is protected from whom are modular and weighted-nested, respectively. The dynamic interplay between these networks influences transitions between dynamical regimes of virus diversification and host control,leading to eventual virus extinction. The three empirical systems exhibit weighted nestedness, a pattern our theory shows is indicative of viral control by hosts. Previously missing from studies of microbial host-pathogen systems, the protection network plays a key role in the coevolutionary dynamics.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>CRIspR-mediated activation of endogenous BST-2/tetherin expression inhibits wild-type HIV-1 production OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">The CRISPR technology not only can knock out target genes by using the RNA-guided Cas9 nuclease but also can activate their expression when a nuclease-deficient Cas9 (dCas9) is employed. Using the latter function, we here show the effect of the CRISPR-mediated pinpoint activation of endogenous expression of BST-2 (also known as tetherin), a virus restriction factor with a broad antiviral spectrum. Single-guide RNA (sgRNA) sequences targeting the BST-2 promoter were selected by promoter assays. Potential sgRNAs and dCas9 fused to the VP64 transactivation domain, along with an accessory transcriptional activator complex, were introduced into cells by lentiviral transduction. Increased expression of BST-2 mRNA in transduced cells was confirmed by real-time RT-PCR. Cells in which BST-2 expression was highly enhanced showed the effective inhibition of HIV-1 production and replication even in the presence of the viral antagonist Vpu against BST-2. These findings confirm that the physiological stoichiometry between host restriction factors and viral antagonists may determine the outcome of the battle with viruses.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</h3>
<p style="text-align: justify; text-justify: inter-word;">We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple Platform for the Rapid Development of Antimicrobials</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent infectious outbreaks highlight the need for platform technologies that can be quickly deployed to develop therapeutics needed to contain the outbreak. We present a simple concept for rapid development of new antimicrobials. The goal was to produce in as little as one week thousands of doses of an intervention for a new pathogen. We tested the feasibility of a system based on antimicrobial synbodies. The system involves creating an array of 100 peptides that have been selected for broad capability to bind and/or kill viruses and bacteria. The peptides are pre-screened for low cell toxicity prior to large scale synthesis. Any pathogen is then assayed on the chip to find peptides that bind or kill it. Peptides are combined in pairs as synbodies and further screened for activity and toxicity. The lead synbody can be quickly produced in large scale, with completion of the entire process in one week.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficient generation of recombinant RNA viruses using targeted recombination-mediated mutagenesis of bacterial artificial chromosomes containing full-length cDNA</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: These results show that targeted recombination-mediated mutagenesis provides a powerful tool for expediting the construction of novel RNA genomes and should be applicable to the manipulation of other RNA viruses.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>The network structure and eco-evolutionary dynamics of CRISPR-induced immune diversification</h3>
<p style="text-align: justify; text-justify: inter-word;">The diversity of CRISPR alleles in natural, experimental and simulated populations has been explored [30] [31] [32] [33] [34] . It has been shown, for example, that increased strain diversity of Pseudomonas aeruginosa (with strains being clones with different spacer compositions) promotes the ability of the bacterium to control the DMS3vir virus [32] . Nevertheless, how such strain diversity evolves, the emergence of structure of host-virus interaction networks, and how they are assembled and disassembled remain unexplored. Advancing this predictive framework for CRISPR-Cas in microbial populations provides a basis for further study of the relative importance and interactions of different resistance mechanisms, and is essential for effective experimental manipulation of microbial systems, including applications to health and engineering [35, 36] . It can open new frontiers in studies of immune diversity and, more generally, contribute to tighten the connection between microbiology and evolutionary biology [37] .</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-Cas Targeting of Host Genes as an Antiviral Strategy</h3>
<p style="text-align: justify; text-justify: inter-word;">Currently, a new gene editing tool-the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) associated (Cas) system-is becoming a promising approach for genetic manipulation at the genomic level. This simple method, originating from the adaptive immune defense system in prokaryotes, has been developed and applied to antiviral research in humans. Based on the characteristics of virus-host interactions and the basic rules of nucleic acid cleavage or gene activation of the CRISPR-Cas system, it can be used to target both the virus genome and host factors to clear viral reservoirs and prohibit virus infection or replication. Here, we summarize recent progress of the CRISPR-Cas technology in editing host genes as an antiviral strategy.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<p style="text-align: justify; text-justify: inter-word;">According to the traditional Koch's postulates, virus isolation is the gold standard for virus diagnosis in the laboratory. Thus, based on SARS-CoV-2 possesses a strong capability to infect humans, CDC recommends that clinical virology laboratories should not attempt viral isolation from specimens collected from COVID-19 patients under investigation. Because SARS-CoV-2 is a newly discovered virus, the spectrum of the available diagnostic tools is tight. In the early stage, SARS-CoV-2 has been detected in human clinical specimens by next-generation sequencing, cell culture, and electron microscopy 11 . Further development of accurate and rapid methods to identify this emergency . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Make Data Sharing Routine to Prepare for Public Health Emergencies</h3>
<p style="text-align: justify; text-justify: inter-word;">• Ongoing research that does not directly relate to an emergency now may be critical for the next unpredictable outbreak.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>The N-Terminal Domain of Spike Protein Is Not the Enteric Tropism Determinant for Transmissible Gastroenteritis Virus in Piglets</h3>
<p style="text-align: justify; text-justify: inter-word;">CRISPR/Cas systems, which constitute a recent new gene editing technology developed from the RNA-mediated adaptive defense systems evolved by bacteria and archaea, have been applied to a wide variety of organisms for the in vivo editing of large genomes [39] [40] [41] . Furthermore, some studies have investigated the in vitro editing of DNA fragments or plasmids [42] [43] [44] . In general, the CRISPR/Cas system from Streptococcus pyogenes, namely, the SpyCas9 protein with a mature single-guide RNA (sgRNA), is used for in vitro DNA cleavage [42] . However, the in vitro editing of RNA virus genomes, particularly viruses with large RNA genomes, such as CoVs, using this method has not been reported.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Survey of Clinical Laboratory Practices for 2015 Middle East Respiratory Syndrome Coronavirus Outbreak in the Republic of Korea</h3>
<p style="text-align: justify; text-justify: inter-word;">Clinical laboratories were able to quickly expand their diagnostic capacity in response to the 2015 MERS-CoV outbreak. Our results show that clinical laboratories play an important role in the maintenance and enhancement of laboratory response in preparation for future emerging infections.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Point-of-View Making vaccines "on demand" A potential solution for emerging pathogens and biodefense?</h3>
<p style="text-align: justify; text-justify: inter-word;">Emergency Medical Countermeasures Enterprise (PHEMCE) has made great strides in strategic preparedness and response capabilities. There have been numerous advances in planning, biothreat countermeasure development, licensure, manufacturing, stockpiling and deployment. Increased biodefense surveillance capability has dramatically improved, while new tools and increased awareness have fostered rapid identification of new potential public health pathogens. Unfortunately, structural delays in vaccine design, development, manufacture, clinical testing and licensure processes remain significant obstacles to an effective national biodefense rapid response capability. This is particularly true for the very real threat of "novel pathogens" such as the avian-origin influenzas H7N9 and H5N1, and new coronaviruses such as hCoV-EMC. Conventional approaches to vaccine development, production, clinical testing and licensure are incompatible with the prompt deployment needed for an effective public health response. An alternative approach, proposed here, is to apply computational vaccine design tools and rapid production technologies that now make it possible to engineer vaccines for novel emerging pathogen and WMD biowarfare agent countermeasures in record time. These new tools have the potential to significantly reduce the time needed to design string-of-epitope vaccines for previously unknown</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for lineage B β- coronaviruses, including 2019-nCoV</h3>
<p style="text-align: justify; text-justify: inter-word;">Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV. author/funder. All rights reserved. No reuse allowed without permission.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /></body>