
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: coronavirus persistence and stability on different substrates and sources (e.g. nasal discharge, sputum, urine, fecal matter, blood)</h2><br /></div><br /><br /><br /><div class="card"><h3>Human Coronaviruses: Insights into Environmental Resistance and Its Influence on the Development of New Antiseptic Strategies</h3>
<i class="tooltip">2012<span class="tooltiptext">2012 Nov 12</span></i><p style="text-align: justify; text-justify: inter-word;">The Coronaviridae family, an enveloped RNA virus family, and, more particularly, human coronaviruses (HCoV), were historically known to be responsible for a large portion of common colds and other upper respiratory tract infections. HCoV are now known to be involved in more serious respiratory diseases, i.e. bronchitis, bronchiolitis or pneumonia, especially in young children and neonates, elderly people and immunosuppressed patients. They have also been involved in nosocomial viral infections. In 2002–2003, the outbreak of severe acute respiratory syndrome (SARS), due to a newly discovered coronavirus, the SARS-associated coronavirus (SARS-CoV); led to a new awareness of the medical importance of the Coronaviridae family. This pathogen, responsible for an emerging disease in humans, with high risk of fatal outcome; underline the pressing need for new approaches to the management of the infection, and primarily to its prevention. Another interesting feature of coronaviruses is their potential environmental resistance, despite the accepted fragility of enveloped viruses. Indeed, several studies have described the ability of HCoVs (i.e. HCoV 229E, HCoV OC43 (also known as betacoronavirus 1), NL63, HKU1 or SARS-CoV) to survive in different environmental conditions (e.g. temperature and humidity), on different supports found in hospital settings such as aluminum, sterile sponges or latex surgical gloves or in biological fluids. Finally, taking into account the persisting lack of specific antiviral treatments (there is, in fact, no specific treatment available to fight coronaviruses infections), the Coronaviridae specificities (i.e. pathogenicity, potential environmental resistance) make them a challenging model for the development of efficient means of prevention, as an adapted antisepsis-disinfection, to prevent the environmental spread of such infective agents. This review will summarize current knowledge on the capacity of human coronaviruses to survive in the environment and the efficacy of well-known antiseptic-disinfectants against them, with particular focus on the development of new methodologies to evaluate the activity of new antiseptic-disinfectants on viruses.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>The Effects of Temperature and Relative Humidity on the Viability of the SARS Coronavirus</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Oct 1</span></i><p style="text-align: justify; text-justify: inter-word;">The main route of transmission of SARS CoV infection is presumed to be respiratory droplets. However the virus is also detectable in other body fluids and excreta. The stability of the virus at different temperatures and relative humidity on smooth surfaces were studied. The dried virus on smooth surfaces retained its viability for over 5 days at temperatures of 22–25°C and relative humidity of 40–50%, that is, typical air-conditioned environments. However, virus viability was rapidly lost (>3 log(10)) at higher temperatures and higher relative humidity (e.g., 38°C, and relative humidity of >95%). The better stability of SARS coronavirus at low temperature and low humidity environment may facilitate its transmission in community in subtropical area (such as Hong Kong) during the spring and in air-conditioned environments. It may also explain why some Asian countries in tropical area (such as Malaysia, Indonesia or Thailand) with high temperature and high relative humidity environment did not have major community outbreaks of SARS.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Stability of bovine coronavirus on lettuce surfaces under household refrigeration conditions</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Fecal suspensions with an aerosol route of transmission were responsible for a cluster of severe acute respiratory syndrome (SARS) cases in 2003 in Hong Kong. Based on that event, the World Health Organization recommended that research be implemented to define modes of transmission of SARS coronavirus through sewage, feces, food and water. Environmental studies have shown that animal coronaviruses remain infectious in water and sewage for up to a year depending on the temperature and humidity. In this study, we examined coronavirus stability on lettuce surfaces. A cell culture adapted bovine coronavirus, diluted in growth media or in bovine fecal suspensions to simulate fecal contamination was used to spike romaine lettuce. qRT-PCR detected viral RNA copy number ranging from 6.6 × 10(4) to 1.7 × 10(6) throughout the experimental period of 30 days. Whereas infectious viruses were detected for at least 14 days, the amount of infectious virus varied, depending upon the diluent used for spiking the lettuce. UV and confocal microscopic observation indicated attachment of residual labeled virions to the lettuce surface after the elution procedure, suggesting that rates of inactivation or detection of the virus may be underestimated. Thus, it is possible that contaminated vegetables may be potential vehicles for coronavirus zoonotic transmission to humans.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>A long-term animal experiment indicating persistent infection of bovine coronavirus in cattle</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 18</span></i><p style="text-align: justify; text-justify: inter-word;">A long-term animal experiment involving inoculation with bovine coronavirus (BCoV) was conducted to verify its persistent infection in cattle. Three colostrum-deprived Holstein calves were housed separately in individual rooms of a high-containment facility and inoculated with the BCoV strain Kumamoto/1/07. Until the end of the experiment (1,085, 700 and 280 days, respectively), viral RNAs were detected sporadically by RT-PCR and nested PCR from plasma, nasal discharge, and feces. Seroconversion and titer changes were validated by hemagglutination inhibition tests and neutralization tests. Among the samples, nasal discharge showed a higher viral positivity than feces, which seemed to be associated with positive detection in the plasma. These data demonstrate the existence of persistent infection of BCoV in the respiratory tissues of cattle.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Persistence of coronaviruses on inanimate surfaces and their inactivation with biocidal agents</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Currently, the emergence of a novel human coronavirus, SARS-CoV-2, has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described with incubation times between 2-10 days, facilitating its spread via droplets, contaminated hands or surfaces. We therefore reviewed the literature on all available information about the persistence of human and veterinary coronaviruses on inanimate surfaces as well as inactivation strategies with biocidal agents used for chemical disinfection, e.g. in healthcare facilities. The analysis of 22 studies reveals that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days, but can be efficiently inactivated by surface disinfection procedures with 62–71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite within 1 minute. Other biocidal agents such as 0.05–0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. As no specific therapies are available for SARS-CoV-2, early containment and prevention of further spread will be crucial to stop the ongoing outbreak and to control this novel infectious thread.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects of Air Temperature and Relative Humidity on Coronavirus Survival on Surfaces</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Assessment of the risks posed by severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) on surfaces requires data on survival of this virus on environmental surfaces and on how survival is affected by environmental variables, such as air temperature (AT) and relative humidity (RH). The use of surrogate viruses has the potential to overcome the challenges of working with SARS-CoV and to increase the available data on coronavirus survival on surfaces. Two potential surrogates were evaluated in this study; transmissible gastroenteritis virus (TGEV) and mouse hepatitis virus (MHV) were used to determine effects of AT and RH on the survival of coronaviruses on stainless steel. At 4°C, infectious virus persisted for as long as 28 days, and the lowest level of inactivation occurred at 20% RH. Inactivation was more rapid at 20°C than at 4°C at all humidity levels; the viruses persisted for 5 to 28 days, and the slowest inactivation occurred at low RH. Both viruses were inactivated more rapidly at 40°C than at 20°C. The relationship between inactivation and RH was not monotonic, and there was greater survival or a greater protective effect at low RH (20%) and high RH (80%) than at moderate RH (50%). There was also evidence of an interaction between AT and RH. The results show that when high numbers of viruses are deposited, TGEV and MHV may survive for days on surfaces at ATs and RHs typical of indoor environments. TGEV and MHV could serve as conservative surrogates for modeling exposure, the risk of transmission, and control measures for pathogenic enveloped viruses, such as SARS-CoV and influenza virus, on health care surfaces.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Aerosol and surface stability of HCoV-19 (SARS-CoV-2) compared to SARS-CoV-1</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A novel human coronavirus, now named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, referred to as HCoV-19 here) that emerged in Wuhan, China in late 2019 is now causing a pandemic.  Here, we analyze the aerosol and surface stability of HCoV-19 and compare it with SARS-CoV-1, the most closely related human coronavirus.2 We evaluated the stability of HCoV-19 and SARS-CoV-1 in aerosols and on different surfaces and estimated their decay rates using a Bayesian regression model</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Some One Health based control strategies for the Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 21</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) presents an ideal example for developing One Health concepts. Dromedary camels are the principal reservoir for the virus. Infected camels shed the virus in body secretions, particularly nasal discharges. MERS-CoV has the potential to remain active in the environment for some time under optimum conditions of temperature and humidity. This shedding sustains the virus in endemic communities and thus contact with camels is considered a major risk factor for human infection. Reducing virus shedding from camels will have a great positive impact on reducing the human risk of infection. Our main objective is to highlight the potential aspects of reducing virus shedding from camels to the environment, thereby reducing the possibility of human infection. We will focus on the potential roles of camel markets, camel shows, importation, transportation and grazing in the amplification and shedding of the virus, providing some novel concepts for the control approaches for the MERS-CoV.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Prediction of Intrinsic Disorder in MERS-CoV/HCoV-EMC Supports a High Oral-Fecal Transmission</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Nov 13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus, MERS-CoV (NCoV, HCoV-EMC/2012), originating from the Middle-East, has been discovered. Incoming data reveal that the virus is highly virulent to humans. A model that categorizes coronaviuses according to the hardness of their shells was developed before the discovery of MERS-CoV. Using protein intrinsic disorder prediction, coronaviruses were categorized into three groups that can be linked to the levels of oral-fecal and respiratory transmission regardless of genetic proximity. Using this model, MERS-CoV is placed into disorder group C, which consists of coronaviruses that have relatively hard inner and outer shells. The members of this group are likely to persist in the environment for a longer period of time and possess the highest oral-fecal components but relatively low respiratory transmission components. Oral-urine and saliva transmission are also highly possible since both require harder protective shells. Results show that disorder prediction can be used as a tool that suggests clues to look for in further epidemiological investigations.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Temporary carriage of bovine coronavirus and bovine respiratory syncytial virus by fomites and human nasal mucosa after exposure to infected calves</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jan 22</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: In order to prevent spread of the endemic pathogens bovine coronavirus (BCoV) and bovine respiratory syncytial virus (BRSV) between herds, knowledge of indirect transmission by personnel and fomites is fundamental. The aims of the study were to determine the duration of viral RNA carriage and the infectivity of viral particles on fomites and human nasal mucosa after exposure to BCoV and BRSV. During two animal infection experiments, swabs were collected from personnel (nasal mucosa) and their clothes, boots and equipment after contact with calves shedding either virus. Viral RNA was quantified by RT-qPCR or droplet digital RT-PCR (RT-ddPCR), and selected samples with high levels of viral RNA were tested by cell culture for infectivity. RESULTS: For BCoV, 46% (n = 80) of the swabs from human nasal mucosa collected 30 min after exposure were positive by RT-qPCR. After two, four and six hours, 15%, 5% and 0% of the swabs were positive, respectively. Infective virions were not detected in mucosal swabs (n = 2). A high viral RNA load was detected on 97% (n = 44) of the fomites 24 h after exposure, and infective virions were detected in two of three swabs. For BRSV, 35% (n = 26) of the human nasal mucosa swabs collected 30 min after exposure, were positive by RT-ddPCR, but none were positive for infective virions. Of the fomites, 89% (n = 38) were positive for BRSV RNA 24 h after exposure, but all were negative for infective viruses. CONCLUSIONS: The results indicate that human nasal mucosa can carry both BCoV and BRSV RNA after exposure to virus shedding calves, but the carriage seems short-lived and the transmission potential is likely limited. High viral loads on contaminates fomites 24 h after exposure to infected animals, and detection of infective BCoV, indicate that contaminated fomites represent a significant risk for indirect transmission between herds.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Thermal stability and inactivation of hepatitis C virus grown in cell culture</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Hepatitis C virus (HCV) is a blood-borne flavivirus that infects many millions of people worldwide. Relatively little is known, however, concerning the stability of HCV and reliable procedures for inactivating this virus. METHODS: In the current study, the thermostability of cell culture-derived HCV (HCVcc, JFH-1 strain) under different environmental temperatures (37°C, room temperature, and 4°C) and the ability of heat, UVC light irradiation, and aldehyde and detergent treatments to inactivate HCVcc were evaluated. The infectious titers of treated viral samples were determined by focus-forming unit (FFU) assay using an indirect immunofluorescence assay for HCV NS3 in hepatoma Huh7-25-CD81 cells highly permissive for HCVcc infection. MTT cytotoxicity assay was performed to determine the concentrations of aldehydes or detergents at which they were no longer cytotoxic. RESULTS: HCVcc in culture medium was found to survive 37°C and room temperature (RT, 25 ± 2°C) for 2 and 16 days, respectively, while the virus was relatively stable at 4°C without drastic loss of infectivity for at least 6 weeks. HCVcc in culture medium was sensitive to heat and could be inactivated in 8 and 4 min when incubated at 60°C and 65°C, respectively. However, at 56°C, 40 min were required to eliminate HCVcc infectivity. Addition of normal human serum to HCVcc did not significantly alter viral stability at RT or its susceptibility to heat. UVC light irradiation (wavelength = 253.7 nm) with an intensity of 450 μW/cm(2 )efficiently inactivated HCVcc within 2 min. Exposures to formaldehyde, glutaraldehyde, ionic or nonionic detergents all destroyed HCVcc infectivity effectively, regardless of whether the treatments were conducted in the presence of cell culture medium or human serum. CONCLUSIONS: The results provide quantitative evidence for the potential use of a variety of approaches for inactivating HCV. The ability of HCVcc to survive ambient temperatures warrants precautions in handling and disposing of objects and materials that may have been contaminated with HCV.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Inoculation of Goats, Sheep, and Horses with MERS-CoV Does Not Result in Productive Viral Shedding</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Aug 19</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first recognized in 2012 and can cause severe disease in infected humans. Dromedary camels are the reservoir for the virus, although, other than nasal discharge, these animals do not display any overt clinical disease. Data from in vitro experiments suggest that other livestock such as sheep, goats, and horses might also contribute to viral transmission, although field data has not identified any seropositive animals. In order to understand if these animals could be infected, we challenged young goats and horses and adult sheep with MERS-CoV by intranasal inoculation. Minimal or no virus shedding was detected in all of the animals. During the four weeks following inoculation, neutralizing antibodies were detected in the young goats, but not in sheep or horses.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological Evidence of MERS-CoV Antibodies in Dromedary Camels (Camelus dromedaries) in Laikipia County, Kenya</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Oct 16</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a recently identified virus causing severe viral respiratory illness in people. Little is known about the reservoir in the Horn of Africa. In Kenya, where no human MERS cases have been reported, our survey of 335 dromedary camels, representing nine herds in Laikipia County, showed a high seroprevalence (46.9%) to MERS-CoV antibodies. Between herd differences were present (14.3%– 82.9%), but was not related to management type or herd isolation. Further research should focus on identifying similarity between MERS-CoV viral isolates in Kenya and clinical isolates from the Middle East and elsewhere.</p>
<p align="right"><i>score: 33</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Background: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Diagnostic test based on real-time reverse transcription polymerase chain reaction assay (qRT-PCR) was the main means of confirmation, and sample collection was mostly throat swabs, which was easy to miss the diagnosis. It is necessary to seek specimen types with higher detection efficiency and accuracy. Methods: Paired specimens of throat swabs and sputum were obtained from 54 cases, and RNA was extracted and tested for 2019-nCoV (equated with SARS-CoV-2) by qRT-PCR assay. Results: The positive rates of 2019-nCoV from sputum specimens and throat swabs were 76.9% and 44.2%, respectively. Sputum specimens showed a significantly higher positive rate than throat swabs in detecting viral nucleic acid using qRT-PCR assay (P=0.001). Conclusions: The detection rates of 2019-nCoV from sputum specimens are significantly higher than throat swabs. We suggest that sputum would benefit for the detection of 2019-nCoV in patients who produce sputum. The results can facilitate the selection of specimens and increase the accuracy of diagnosis.</jats:p></p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV in Upper Respiratory Tract and Lungs of Dromedary Camels, Saudi Arabia, 2013–2014</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">To assess the temporal dynamics of Middle East respiratory syndrome coronavirus (MERS-CoV) infection in dromedary camels, specimens were collected at 1–2 month intervals from 2 independent groups of animals during April 2013–May 2014 in Al-Ahsa Province, Saudi Arabia, and tested for MERS-CoV RNA by reverse transcription PCR. Of 96 live camels, 28 (29.2%) nasal swab samples were positive; of 91 camel carcasses, 56 (61.5%) lung tissue samples were positive. Positive samples were more commonly found among young animals (<4 years of age) than adults (>4 years of age). The proportions of positive samples varied by month for both groups; detection peaked during November 2013 and January 2014 and declined in March and May 2014. These findings further our understanding of MERS-CoV infection in dromedary camels and may help inform intervention strategies to reduce zoonotic infections.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus – A Snapshot of Current Knowledge</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Summary Another animal to human transmission of a coronavirus occurred in December 2019 on a live animal market in the Chinese city of Wuhan causing an epidemic in China, reaching now different continents. This minireview summarizes the research literature on the virological, clinical and epidemiological aspects of this epidemic published until end of February 2020.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus and the One Health concept</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 22</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is one of the major threats to the healthcare systems in some countries, especially in the Arabian Peninsula. MERS-CoV is considered an ideal example of the One Health concept. This is due to the animals, especially dromedary camels, play important roles in the transmission and sustainability of the virus, and the virus can be transmitted through aerosols of infected patients into the environment. However, there is some debate regarding the origin of MERS-CoV either from bats or other unknown reservoirs. The dromedary camel is the only identified animal reservoir to date. These animals play important roles in sustaining the virus in certain communities and may act as an amplifier of the virus by secreting it in their body fluids, especially in nasal and rectal discharges. MERS-CoV has been detected in the nasal and rectal secretions of infected camels, and MERS-CoV of this origin has full capacity to infect human airway epithelium in both in vitro and in vivo models. Other evidence confirms the direct transmission of MERS-CoV from camels to humans, though the role of camel meat and milk products has yet to be well studied. Human-to-human transmission is well documented through contact with an active infected patient or some silently infected persons. Furthermore, there are some significant risk factors of individuals in close contact with a positive MERS-CoV patient, including sleeping in the same patient room, removing patient waste (urine, stool, and sputum), and touching respiratory secretions from the index case. Outbreaks within family clusters have been reported, whereby some blood relative patients were infected through their wives in the same house were not infected. Some predisposing genetic factors favor MERS-CoV infection in some patients, which is worth investigating in the near future. The presence of other comorbidities may be another factor. Overall, there are many unknown/confirmed aspects of the virus/human/animal network. Here, the most recent advances in this context are discussed, and the possible reasons behind the emergence and sustainability of MERS-CoV in certain regions are presented. Identification of the exact mechanism of transmission of MERS-CoV from camels to humans and searching for new reservoir/s are of high priority. This will reduce the shedding of the virus into the environment, and thus the risk of human infection can be mitigated.</p>
<p align="right"><i>score: 32</i></p>

</div><br /><br /><br /><div class="card"><h3>Searching for animal models and potential target species for emerging pathogens: Experience gained from Middle East respiratory syndrome (MERS) coronavirus</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 3</span></i><p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus disinfection in histopathology</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019 Coronavirus epidemic, provisionally called 2019-nCoV, was first identified in Wuhan, China, in persons exposed to a seafood or wet market. There is an international push to contain the virus and prevent its spread. It is feasible that potentially infectious samples may be received in histopathology laboratories for diagnosis. This technical note presents disinfection procedures and histotechnology processes that should alleviate the risk of infection to laboratory staff. Using data obtained from similar coronaviruses, e.g. severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), experts are confident that 70% ethanol and 0.1% sodium hypochlorite should inactivate the virus. Formalin fixation and heating samples to 56oC, as used in routine tissue processing, were found to inactivate several coronaviruses and it is believed that 2019-nCoV would be similarly affected.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Replication and Shedding of MERS-CoV in Upper Respiratory Tract of Inoculated Dromedary Camels</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Dec</span></i><p style="text-align: justify; text-justify: inter-word;">In 2012, a novel coronavirus associated with severe respiratory disease in humans emerged in the Middle East. Epidemiologic investigations identified dromedary camels as the likely source of zoonotic transmission of Middle East respiratory syndrome coronavirus (MERS-CoV). Here we provide experimental support for camels as a reservoir for MERS-CoV. We inoculated 3 adult camels with a human isolate of MERS-CoV and a transient, primarily upper respiratory tract infection developed in each of the 3 animals. Clinical signs of the MERS-CoV infection were benign, but each of the camels shed large quantities of virus from the upper respiratory tract. We detected infectious virus in nasal secretions through 7 days postinoculation, and viral RNA up to 35 days postinoculation. The pattern of shedding and propensity for the upper respiratory tract infection in dromedary camels may help explain the lack of systemic illness among naturally infected camels and the means of efficient camel-to-camel and camel-to-human transmission.</p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 31</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Sequence of Human Betacoronavirus 2c Jordan-N3/2012 after Serial Passage in Mammalian Cells</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 May 29</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is the etiologic agent of a highly lethal pneumonia. Here, we report the full-genome sequence of the Jordan-N3/2012 strain after serial passage in two distinct mammalian cell lines. The genome exhibits noteworthy stability, which may inform the development of vaccines and therapeutics used to treat infection with this virus.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparison of different antigen preparations as substrates for use in passive hemagglutination and enzyme-linked immunosorbent assays for detection of antibody against bovine enteric coronavirus.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Purified coronavirus, detergent extracts of purified coronavirus, and virus-infected Madin-Darby bovine kidney cells were evaluated as antigen substrates in enzyme-linked immunosorbent assay (ELISA) and passive hemagglutination systems. Only detergent-extracted and -unextracted, purified viruses were reactive as antigen substrates in ELISA, whereas all three antigen preparations could be used for sensitization of erythrocytes in the passive hemagglutination assay. The passive hemagglutination system with infected cell extracts exhibited a similar level of sensitivity and specificity to the ELISA system employing purified coronavirus but enabled 300 times more tests to be performed per volume of virus-infected cell culture.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Infectious Middle East Respiratory Syndrome Coronavirus Excretion and Serotype Variability Based on Live Virus Isolates from Patients in Saudi Arabia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The newly emerged Middle East respiratory syndrome coronavirus (MERS-CoV) has infected at least 1,082 people, including 439 fatalities. So far, no empirical virus isolation study has been done to elucidate infectious virus secretion or serotype variability. Here, we used 51 respiratory samples from 32 patients with confirmed MERS-CoV infection for virus isolation in Vero B4 and Caco-2 cells. We found Caco-2 cells to significantly enhance isolation success over routinely used Vero cells. Isolation success correlated with viral RNA concentration and time after diagnosis as well as with the amount of IgA antibodies secreted in respiratory samples used for isolation. Results from plaque reduction neutralization assays using a representative range of serum samples and virus isolates suggested that all circulating human MERS-CoV strains represent one single serotype. The choice of prototype strain is not likely to influence the success of candidate MERS-CoV vaccines. However, vaccine formulations should be evaluated for their potential to induce IgA.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Severe Acute Respiratory Syndrome (SARS) Coronavirus Nucleocapsid Protein in SARS Patients by Enzyme-Linked Immunosorbent Assay</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We report the development of an enzyme-linked immunosorbent assay (ELISA) for the detection of severe acute respiratory syndrome (SARS) coronavirus (CoV) nucleocapsid protein. The assay was carried out with hyperimmune polyclonal nucleocapsid-specific antibodies from guinea pigs and rabbits immunized with recombinant His(6)-tagged SARS CoV nucleocapsid protein. The assay was used for the detection of SARS CoV nucleocapsid protein in nasopharyngeal aspirate, urine, and fecal samples collected from patients with confirmed SARS between days 2 and 33 after the onset of illness. The ELISA was capable of detecting this protein in SARS CoV cell culture lysates at 15 50% tissue culture infective doses/ml but did not produce positive signals when tested with cell culture lysates of human coronaviruses OC43 and 229E. When tested with 120 nasopharyngeal aspirate, 100 urine, and 100 fecal specimens from hospitalized patients without SARS, the assay was shown to have high specificities—96.7, 99, and 96%, respectively. In an evaluation of clinical specimens from SARS patients, 34 (52%) of 66 nasopharyngeal aspirate samples from 50 patients, 5 (5%) of 94 urine samples from 94 patients, and 36 (55%) of 65 fecal samples from 65 patients tested positive for SARS CoV nucleocapsid protein. Nucleocapsid protein could be detected from days 6 to 24 in nasopharyngeal aspirate specimens, from days 11 to 31 in urine specimens, and from days 8 to 32 in fecal specimens after the onset of illness. Moreover, the protein could be detected in 25 (83%) of 30 nasopharyngeal aspirate specimens obtained from days 11 to 15 and in all 7 fecal specimens obtained from days 21 to 32. Since the present ELISA is more convenient and economical than reverse transcription-PCR, it may serve as an alternative tool for the early diagnosis of SARS CoV infection in laboratories with limited resources and expertise and for mass screening for the reservoir of SARS CoV. Further studies on serial clinical specimens should reveal the duration of nucleocapsid protein shedding and may reveal a higher detection rate in SARS patients.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Nucleocapsid-Based Human Coronavirus Immunoassay and Estimates of Individuals Exposed to Coronavirus in a U.S. Metropolitan Population</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses cause respiratory infections ranging from common colds to severe acute respiratory syndrome (SARS) in humans. Estimates for exposure to non-SARS coronaviruses are high, particularly for 229E and OC43; however, less information regarding seroprevalence is available for HKU1 and NL63. To measure exposure rates to these four coronavirus strains (229E, HKU1, NL63, and OC43), we devised an immunoassay based on amino- and carboxy-terminally tagged recombinant coronavirus nucleocapsid antigens. Four human and one feline coronavirus antigen were cloned into baculoviruses expressed in insect cells and recovered proteins bound in the solid phase of an enzyme-linked immunosorbent assay-based system. We screened sera from 10 children and 196 adults and established primary cutoff points based on immunoglobulin G (IgG) antibody levels of the predominantly seronegative children. The proportion of seropositive adults for each coronavirus was as follows: 229E, 91.3%; HKU1, 59.2%; NL63, 91.8%; and OC43, 90.8%. No evidence of a significant serological response to the feline coronavirus was observed. Significant associations of coronavirus seropositivity and antibody levels with age, gender, race, socioeconomic status, smoking status, and season of the blood draw were tested with chi-square and regression analyses. The group II coronaviruses (OC43 and HKU1) were significantly associated with race (P ≤ 0.009 and P ≤ 0.03, respectively). Elevated OC43 IgG levels were further significantly associated with smoking status (P ≤ 0.03), as were high NL63 titers with socioeconomic status (P ≤ 0.04). The high-level immunoreactivity of each coronavirus was significantly associated with the summer season (P ≤ 0.01 to 0.0001). In summary, high rates of exposure to 229E, NL63, and OC43 and a moderate rate of exposure to HKU1 characterized the seroprevalence among individuals in this population. Demographic factors, such as race, smoking status, and socioeconomic status, may confer an increased risk of susceptibility to these viruses.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Transmission</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) infection causes a spectrum of respiratory illness, from asymptomatic to mild to fatal. MERS-CoV is transmitted sporadically from dromedary camels to humans and occasionally through human-to-human contact. Current epidemiologic evidence supports a major role in transmission for direct contact with live camels or humans with symptomatic MERS, but little evidence suggests the possibility of transmission from camel products or asymptomatic MERS cases. Because a proportion of case-patients do not report direct contact with camels or with persons who have symptomatic MERS, further research is needed to conclusively determine additional mechanisms of transmission, to inform public health practice, and to refine current precautionary recommendations.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Transmission</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) infection causes a spectrum of respiratory illness, from asymptomatic to mild to fatal. MERS-CoV is transmitted sporadically from dromedary camels to humans and occasionally through human-to-human contact. Current epidemiologic evidence supports a major role in transmission for direct contact with live camels or humans with symptomatic MERS, but little evidence suggests the possibility of transmission from camel products or asymptomatic MERS cases. Because a proportion of case-patients do not report direct contact with camels or with persons who have symptomatic MERS, further research is needed to conclusively determine additional mechanisms of transmission, to inform public health practice, and to refine current precautionary recommendations.</p>
<p align="right"><i>score: 30</i></p>

</div><br /><br /><br /><div class="card"><h3>Heat inactivation of the Middle East respiratory syndrome coronavirus</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Sep 24</span></i><p style="text-align: justify; text-justify: inter-word;">The culture supernatants of the emerging Middle East respiratory syndrome coronavirus (MERS-CoV) were submitted to three temperatures over time and tested for infectivity by TCID(50) method on Vero E6 cells. At 56°C, almost 25 minutes were necessary to reduce the initial titre by 4 log(10). Increasing temperature to 65°C had a strong negative effect on viral infectivity as virucidy dropped significantly to 1 minute. On the contrary, no significant decrease in titre was observed after 2 hours at 25°C. These data might be useful in establishing biosafety measures in laboratories against MERS-CoV.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS: emergence of a novel human coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (CoV) that causes a severe lower respiratory tract infection in humans, emerged in the Middle East region in 2012. This virus, named Middle East respiratory syndrome (MERS)-CoV, is phylogenetically related to bat CoVs, but other animal species like dromedary camels may potentially act as intermediate hosts by spreading the virus to humans. Although human to human transmission has been demonstrated, analysis of human MERS clusters indicated that chains of transmission were not self-sustaining, especially when infection control was implemented. Thus, timely identification of new MERS cases followed by their quarantine, combined with measures to limit spread of the virus from the (intermediate) host to humans, may be crucial in controlling the outbreak of this emerging CoV.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Inactivation and safety testing of Middle East Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is a recently emerged virus that has caused a number of human infections and deaths, primarily in the Middle East. The transmission of MERS-CoV to humans has been proposed to be as a result of contact with camels, but evidence of human-to-human transmission also exists. In order to work with MERS-CoV in a laboratory setting, the US Centers for Disease Control and Prevention (CDC) has determined that MERS-CoV should be handled at a biosafety level (BSL) 3 (BSL-3) biocontainment level. Many processes and procedures used to characterize MERS-CoV and to evaluate samples from MERS-CoV infected animals are more easily and efficiently completed at BSL-2 or lower containment. In order to complete experimental work at BSL-2, demonstration or proof of inactivation is required before removal of specimens from biocontainment laboratories. In the studies presented here, we evaluated typical means of inactivating viruses prior to handling specimens at a lower biocontainment level. We found that Trizol, AVL buffer and gamma irradiation were effective at inactivating MERS-CoV, that formaldehyde-based solutions required at least 30 minutes of contact time in a cell culture system while a mixture of methanol and acetone required 60 minutes to inactivate MERS-CoV. Together, these data provide a foundation for safely inactivating MERS-CoV, and potentially other coronaviruses, prior to removal from biocontainment facilities.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to MERS, Thrusting Coronaviruses into the Spotlight</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Jan 14</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) have formerly been regarded as relatively harmless respiratory pathogens to humans. However, two outbreaks of severe respiratory tract infection, caused by the severe acute respiratory syndrome coronavirus (SARS-CoV) and the Middle East respiratory syndrome coronavirus (MERS-CoV), as a result of zoonotic CoVs crossing the species barrier, caused high pathogenicity and mortality rates in human populations. This brought CoVs global attention and highlighted the importance of controlling infectious pathogens at international borders. In this review, we focus on our current understanding of the epidemiology, pathogenesis, prevention, and treatment of SARS-CoV and MERS-CoV, as well as provides details on the pivotal structure and function of the spike proteins (S proteins) on the surface of each of these viruses. For building up more suitable animal models, we compare the current animal models recapitulating pathogenesis and summarize the potential role of host receptors contributing to diverse host affinity in various species. We outline the research still needed to fully elucidate the pathogenic mechanism of these viruses, to construct reproducible animal models, and ultimately develop countermeasures to conquer not only SARS-CoV and MERS-CoV, but also these emerging coronaviral diseases.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive result of Sars-Cov-2 in sputum from a cured patient with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">ince December 2019, an outbreak of the novel coronavirus (SARS-CoV-2) infection has spread rapidly in Wuhan, China [1]1. Over a month since the outbreak, more than 13,000 patients with COVID-19 have be cured and discharged from hospital until now. For the clinical cure criteria in China, twice successive negative results of Sars-Cov-2 nucleic acid detection are the important index, in addition to normal body temperature for 3 days as well as obvious improvement in respiratory symptoms and CT scan [2]. In the present work, we reported that Sars-Cov-2 nucleic acid was still detectable in sputum obtained by nebulization from a cured patient. On January 22, a 49-year-old man presented himself with fever for 4 days to a clinic. Throat swab detection was positive for SARS-CoV-2 nucleic acid by real-time RT-PCR. Subsequently, the patient was diagnosed with COVID-19 according to the diagnostic criteria [2] as follows (1): the positive result of SARS-CoV-2 nucleic acid detection (2); a history of short stay in Wuhan within 14 days; and (3) symptoms of fever, and multiple patchy areas of ground-glass opacity on CT scan. After the active treatment, the patient recovered from fever and other respiratory symptoms on February 4. On February 9 and February 10, the SARS-CoV-2 nucleic acid detection was successively negative in his throat swab samples. The CT scan result showed that the inflammation was significantly decreased in both lungs. Both the results of SARS-CoV-2 nucleic acid detection and CT scans indicated a recovery trend, and the patient was ready for discharge. On February 13, the throat swab and sputum by nebulization were collected before the patient was discharged. Notably, SARS-CoV-2 nucleic acid was still detected in sputum from the patient although negative result of throat swab detection.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>The battle against SARS and MERS coronaviruses: Reservoirs and Animal Models</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Jul 28</span></i><p style="text-align: justify; text-justify: inter-word;">In humans, infection with the coronavirus, especially the severe acute respiratory syndrome coronavirus (SARS‐CoV) and the emerging Middle East respiratory syndrome coronavirus (MERS‐CoV), induces acute respiratory failure, resulting in high mortality. Irregular coronavirus related epidemics indicate that the evolutionary origins of these two pathogens need to be identified urgently and there are still questions related to suitable laboratory animal models. Thus, in this review we aim to highlight key discoveries concerning the animal origin of the virus and summarize and compare current animal models.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Community Case Clusters of Middle East Respiratory Syndrome Coronavirus in Hafr Al-Batin, Kingdom of Saudi Arabia: A Descriptive Genomic study</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first described in September 2012 and to date 86 deaths from a total of 206 cases of MERS-CoV infection have been reported to the WHO. Camels have been implicated as the reservoir of MERS-CoV, but the exact source and mode of transmission for most patients remain unknown. During a 3 month period, June to August 2013, there were 12 positive MERS-CoV cases reported from the Hafr Al-Batin region district in the north east region of the Kingdom of Saudi Arabia. In addition to the different regional camel festivals in neighboring countries, Hafr Al-Batin has the biggest camel market in the entire Kingdom and hosts an annual camel festival. Thus, we conducted a detailed epidemiological, clinical and genomic study to ascertain common exposure and transmission patterns of all cases of MERS-CoV reported from Hafr Al-Batin. Analysis of previously reported genetic data indicated that at least two of the infected contacts could not have been directly infected from the index patient and alternate source should be considered. While camels appear as the likely source, other sources have not been ruled out. More detailed case control studies with detailed case histories, epidemiological information and genomic analysis are being conducted to delineate the missing pieces in the transmission dynamics of MERS-CoV outbreak.</p>
<p align="right"><i>score: 29</i></p>

</div><br /><br /><br /><div class="card"><h3>Feline and Canine Coronaviruses: Common Genetic and Pathobiological Features</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Jul 31</span></i><p style="text-align: justify; text-justify: inter-word;">A new human coronavirus responsible for severe acute respiratory syndrome (SARS) was identified in 2003, which raised concern about coronaviruses as agents of serious infectious disease. Nevertheless, coronaviruses have been known for about 50 years to be major agents of respiratory, enteric, or systemic infections of domestic and companion animals. Feline and canine coronaviruses are widespread among dog and cat populations, sometimes leading to the fatal diseases known as feline infectious peritonitis (FIP) and pantropic canine coronavirus infection in cats and dogs, respectively. In this paper, different aspects of the genetics, host cell tropism, and pathogenesis of the feline and canine coronaviruses (FCoV and CCoV) will be discussed, with a view to illustrating how study of FCoVs and CCoVs can improve our general understanding of the pathobiology of coronaviruses.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus 229E Remains Infectious on Common Touch Surface Materials</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Nov 10</span></i><p style="text-align: justify; text-justify: inter-word;">The evolution of new and reemerging historic virulent strains of respiratory viruses from animal reservoirs is a significant threat to human health. Inefficient human-to-human transmission of zoonotic strains may initially limit the spread of transmission, but an infection may be contracted by touching contaminated surfaces. Enveloped viruses are often susceptible to environmental stresses, but the human coronaviruses responsible for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) have recently caused increasing concern of contact transmission during outbreaks. We report here that pathogenic human coronavirus 229E remained infectious in a human lung cell culture model following at least 5 days of persistence on a range of common nonbiocidal surface materials, including polytetrafluoroethylene (Teflon; PTFE), polyvinyl chloride (PVC), ceramic tiles, glass, silicone rubber, and stainless steel. We have shown previously that noroviruses are destroyed on copper alloy surfaces. In this new study, human coronavirus 229E was rapidly inactivated on a range of copper alloys (within a few minutes for simulated fingertip contamination) and Cu/Zn brasses were very effective at lower copper concentration. Exposure to copper destroyed the viral genomes and irreversibly affected virus morphology, including disintegration of envelope and dispersal of surface spikes. Cu(I) and Cu(II) moieties were responsible for the inactivation, which was enhanced by reactive oxygen species generation on alloy surfaces, resulting in even faster inactivation than was seen with nonenveloped viruses on copper. Consequently, copper alloy surfaces could be employed in communal areas and at any mass gatherings to help reduce transmission of respiratory viruses from contaminated surfaces and protect the public health.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS coronavirus: diagnostics, epidemiology and transmission</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 22</span></i><p style="text-align: justify; text-justify: inter-word;">The first known cases of Middle East respiratory syndrome (MERS), associated with infection by a novel coronavirus (CoV), occurred in 2012 in Jordan but were reported retrospectively. The case first to be publicly reported was from Jeddah, in the Kingdom of Saudi Arabia (KSA). Since then, MERS-CoV sequences have been found in a bat and in many dromedary camels (DC). MERS-CoV is enzootic in DC across the Arabian Peninsula and in parts of Africa, causing mild upper respiratory tract illness in its camel reservoir and sporadic, but relatively rare human infections. Precisely how virus transmits to humans remains unknown but close and lengthy exposure appears to be a requirement. The KSA is the focal point of MERS, with the majority of human cases. In humans, MERS is mostly known as a lower respiratory tract (LRT) disease involving fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of those infected. However, MERS-CoV has also been detected in mild and influenza-like illnesses and in those with no signs or symptoms. Older males most obviously suffer severe disease and MERS patients often have comorbidities. Compared to severe acute respiratory syndrome (SARS), another sometimes- fatal zoonotic coronavirus disease that has since disappeared, MERS progresses more rapidly to respiratory failure and acute kidney injury (it also has an affinity for growth in kidney cells under laboratory conditions), is more frequently reported in patients with underlying disease and is more often fatal. Most human cases of MERS have been linked to lapses in infection prevention and control (IPC) in healthcare settings, with approximately 20 % of all virus detections reported among healthcare workers (HCWs) and higher exposures in those with occupations that bring them into close contact with camels. Sero-surveys have found widespread evidence of past infection in adult camels and limited past exposure among humans. Sensitive, validated reverse transcriptase real-time polymerase chain reaction (RT-rtPCR)-based diagnostics have been available almost from the start of the emergence of MERS. While the basic virology of MERS-CoV has advanced over the past three years, understanding of the interplay between camel, environment, and human remains limited. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1186/s12985-015-0439-5) contains supplementary material, which is available to authorized users.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Lack of Middle East Respiratory Syndrome Coronavirus Transmission in Rabbits</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) transmission from dromedaries to humans has resulted in major outbreaks in the Middle East. Although some other livestock animal species have been shown to be susceptible to MERS-CoV, it is not fully understood why the spread of the virus in these animal species has not been observed in the field. In this study, we used rabbits to further characterize the transmission potential of MERS-CoV. In line with the presence of MERS-CoV receptor in the rabbit nasal epithelium, high levels of viral RNA were shed from the nose following virus inoculation. However, unlike MERS-CoV-infected dromedaries, these rabbits did not develop clinical manifestations including nasal discharge and did shed only limited amounts of infectious virus from the nose. Consistently, no transmission by contact or airborne routes was observed in rabbits. Our data indicate that despite relatively high viral RNA levels produced, low levels of infectious virus are excreted in the upper respiratory tract of rabbits as compared to dromedary camels, thus resulting in a lack of viral transmission.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential Intermediate Hosts for Coronavirus Transmission: No Evidence of Clade 2c Coronaviruses in Domestic Livestock from Ghana</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 10</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV), nearly a decade ago with worldwide distribution, was believed to be of zoonotic origin from bats with dromedary camels as intermediate hosts. There is a likelihood of other domestic livestock serving as intermediate hosts for this virus. The presence of coronaviruses, closely related to MERS-CoV in Ghanaian bats, presented the opportunity to test the hypothesis of transmissibility of this virus through domestic livestock species. The possible interactions between livestock and bats in 31 household farms were accessed by observation and interviews with farmers. Rectal swabs and serum from cattle, sheep, goats, donkeys, and swine from commercial and household farms were tested for MERS-CoV and a Nycteris sp. bat coronavirus, previously detected in Ghana. A pan-PCR assay to detect clade 2c viruses and recombinant immunofluorescence assay to detect anti-spike IgG antibodies against the target viruses were used. Likely contact between livestock and bats was determined for 13 farms (41.9%) that reported confining their livestock and also observing bats in their homes. Livestock were left unconfined on eight farms (25.8%) that also observed bats roosting in trees close to their homes. No viral RNA or antibodies against the two coronaviruses were detected in any of the livestock species tested. Cattle, sheep, goats, donkeys, and swine are not likely hosts of clade 2c coronaviruses.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization and Complete Genome Sequence of a Novel Coronavirus, Coronavirus HKU1, from Patients with Pneumonia</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Despite extensive laboratory investigations in patients with respiratory tract infections, no microbiological cause can be identified in a significant proportion of patients. In the past 3 years, several novel respiratory viruses, including human metapneumovirus, severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV), and human coronavirus NL63, were discovered. Here we report the discovery of another novel coronavirus, coronavirus HKU1 (CoV-HKU1), from a 71-year-old man with pneumonia who had just returned from Shenzhen, China. Quantitative reverse transcription-PCR showed that the amount of CoV-HKU1 RNA was 8.5 to 9.6 × 10(6) copies per ml in his nasopharyngeal aspirates (NPAs) during the first week of the illness and dropped progressively to undetectable levels in subsequent weeks. He developed increasing serum levels of specific antibodies against the recombinant nucleocapsid protein of CoV-HKU1, with immunoglobulin M (IgM) titers of 1:20, 1:40, and 1:80 and IgG titers of <1:1,000, 1:2,000, and 1:8,000 in the first, second and fourth weeks of the illness, respectively. Isolation of the virus by using various cell lines, mixed neuron-glia culture, and intracerebral inoculation of suckling mice was unsuccessful. The complete genome sequence of CoV-HKU1 is a 29,926-nucleotide, polyadenylated RNA, with G+C content of 32%, the lowest among all known coronaviruses with available genome sequence. Phylogenetic analysis reveals that CoV-HKU1 is a new group 2 coronavirus. Screening of 400 NPAs, negative for SARS-CoV, from patients with respiratory illness during the SARS period identified the presence of CoV-HKU1 RNA in an additional specimen, with a viral load of 1.13 × 10(6) copies per ml, from a 35-year-old woman with pneumonia. Our data support the existence of a novel group 2 coronavirus associated with pneumonia in humans.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus isolates SK and SD from multiple sclerosis patients are serologically related to murine coronaviruses A59 and JHM and human coronavirus OC43, but not to human coronavirus 229E.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two coronaviruses (SK and SD), isolated from fresh autopsy brain tissue from two multiple sclerosis patients, were compared with known human and murine coronaviruses. In plaque neutralization assays, antisera prepared against multiple sclerosis isolates SK and SD demonstrated significant cross-reactivity to each other and to murine coronavirus A59, weak cross-reactivity to murine coronavirus JHM, but no cross-reactivity to the human coronavirus 229E. Antiserum to SK or SD failed to inhibit hemagglutination of chicken erythrocytes by the human coronavirus OC43. However, OC43 antiserum neutralized both SD and SK. Specific coronavirus polypeptides were identified and compared by immunoprecipitation and polyacrylamide gel electrophoresis. Infected and mock-infected 17Cl-1 cells were pretreated with actinomycin D and labeled with [35S]methionine. Polypeptides in Nonidet P-40 cytoplasmic extracts were immunoprecipitated with homologous and heterologous antisera. Identical polypeptides were precipitated from A59-, SD-, or SK-infected cell extracts by SD, SK, OC43, or A59 antisera. The polypeptides of human virus 229E were antigenically distinct, with the exception of weak recognition of a polypeptide of 50,000 molecular weight. We conclude that the two multiple sclerosis virus isolates SK and SD are closely related serologically to the murine coronavirus A59 and the human coronavirus OC43.</p>
<p align="right"><i>score: 28</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular characterization of a new species in the genus Alphacoronavirus associated with mink epizootic catarrhal gastroenteritis</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">A coronavirus (CoV) previously shown to be associated with catarrhal gastroenteritis in mink (Mustela vison) was identified by electron microscopy in mink faeces from two fur farms in Wisconsin and Minnesota in 1998. A pan-coronavirus and a genus-specific RT-PCR assay were used initially to demonstrate that the newly discovered mink CoVs (MCoVs) were members of the genus Alphacoronavirus. Subsequently, using a random RT-PCR approach, full-genomic sequences were generated that further confirmed that, phylogenetically, the MCoVs belonged to the genus Alphacoronavirus, with closest relatedness to the recently identified but only partially sequenced (fragments of the polymerase, and full-length spike, 3c, envelope, nucleoprotein, membrane, 3x and 7b genes) ferret enteric coronavirus (FRECV) and ferret systemic coronavirus (FRSCV). The molecular data presented in this study provide the first genetic evidence for a new coronavirus associated with epizootic catarrhal gastroenteritis outbreaks in mink and demonstrate that MCoVs possess high genomic variability and relatively low overall nucleotide sequence identities (91.7 %) between contemporary strains. Additionally, the new MCoVs appeared to be phylogenetically distant from human (229E and NL63) and other alphacoronaviruses and did not belong to the species Alphacoronavirus 1. It is proposed that, together with the partially sequenced FRECV and FRSCV, they comprise a new species within the genus Alphacoronavirus.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus (MERS-CoV): A review</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As a novel coronavirus first reported by Saudi Arabia in 2012, the Middle East respiratory syndrome coronavirus (MERS-CoV) is responsible for an acute human respiratory syndrome. The virus, of 2C beta-CoV lineage, expresses the dipeptidyl peptidase 4 (DPP4) receptor and is densely endemic in dromedary camels of East Africa and the Arabian Peninsula. MERS-CoV is zoonotic but human-to-human transmission is also possible. Surveillance and phylogenetic researches indicate MERS-CoV to be closely associated with bats’ coronaviruses, suggesting bats as reservoirs, although unconfirmed. With no vaccine currently available for MERS-CoV nor approved prophylactics, its global spread to over 25 countries with high fatalities highlights its role as ongoing public health threat. An articulated action plan ought to be taken, preferably from a One Health perspective, for appropriately advanced countermeasures against MERS-CoV.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Animal models for SARS and MERS coronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome coronavirus (MERS-CoV), two strains of animal coronaviruses that crossed the species barrier to infect and cause severe respiratory infections in humans within the last 12 years, have taught us that coronaviruses represent a global threat that does not recognize international borders. We can expect to see other novel coronaviruses emerge in the future. An ideal animal model should reflect the clinical signs, viral replication and pathology seen in humans. In this review, we present factors to consider in establishing an animal model for the study of novel coronaviruses and compare the different animal models that have been employed to study SARS-CoV and MERS-CoV.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Long-term SARS Coronavirus Excretion from Patient Cohort, China</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Oct</span></i><p style="text-align: justify; text-justify: inter-word;">This study investigated the long-term excretion of severe acute respiratory syndrome–associated coronavirus in sputum and stool specimens from 56 infected patients. The median (range) duration of virus excretion in sputa and stools was 21 (14–52) and 27 (16–126) days, respectively. Coexisting illness or conditions were associated with longer viral excretion in stools.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Objectives Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. Methods Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. Results The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. Conclusion SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. METHODS: Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. RESULTS: The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. CONCLUSION: SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Host Determinants of MERS-CoV Transmission and Pathogenesis</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Mar 19</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic pathogen that causes respiratory infection in humans, ranging from asymptomatic to severe pneumonia. In dromedary camels, the virus only causes a mild infection but it spreads efficiently between animals. Differences in the behavior of the virus observed between individuals, as well as between humans and dromedary camels, highlight the role of host factors in MERS-CoV pathogenesis and transmission. One of these host factors, the MERS-CoV receptor dipeptidyl peptidase-4 (DPP4), may be a critical determinant because it is variably expressed in MERS-CoV-susceptible species as well as in humans. This could partially explain inter- and intraspecies differences in the tropism, pathogenesis, and transmissibility of MERS-CoV. In this review, we explore the role of DPP4 and other host factors in MERS-CoV transmission and pathogenesis—such as sialic acids, host proteases, and interferons. Further characterization of these host determinants may potentially offer novel insights to develop intervention strategies to tackle ongoing outbreaks.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of Group 1 Coronaviruses in Bats in North America</h3>
<i class="tooltip">2007<span class="tooltiptext">2007 Sep</span></i><p style="text-align: justify; text-justify: inter-word;">The epidemic of severe acute respiratory syndrome (SARS) was caused by a newly emerged coronavirus (SARS-CoV). Bats of several species in southern People’s Republic of China harbor SARS-like CoVs and may be reservoir hosts for them. To determine whether bats in North America also harbor coronaviruses, we used reverse transcription–PCR to detect coronavirus RNA in bats. We found coronavirus RNA in 6 of 28 fecal specimens from bats of 2 of 7 species tested. The prevalence of viral RNA shedding was high: 17% in Eptesicus fuscus and 50% in Myotis occultus. Sequence analysis of a 440-bp amplicon in gene 1b showed that these Rocky Mountain bat coronaviruses formed 3 clusters in phylogenetic group 1 that were distinct from group 1 coronaviruses of Asian bats. Because of the potential for bat coronaviruses to cause disease in humans and animals, further surveillance and characterization of bat coronaviruses in North America are needed.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>The emergence of the Middle East Respiratory Syndrome coronavirus (MERS-CoV)</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">On September 20, 2012, a Saudi Arabian physician reported the isolation of a novel coronavirus from a patient with pneumonia on ProMED-mail. Within a few days the same virus was detected in a Qatari patient receiving intensive care in a London hospital, a situation reminiscent of the role air travel played in the spread of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) in 2002. SARS-CoV originated in China’s Guangdong Province and affected more than 8000 patients in 26 countries before it was contained six months later. Over a year after the emergence of this novel coronavirus—Middle East Respiratory Syndrome coronavirus (MERS-CoV)—it has caused 178 laboratory confirmed cases and 76 deaths The emergence of a second highly pathogenic coronavirus within a decade highlights the importance of a coordinated global response incorporating reservoir surveillance, high-containment capacity with fundamental and applied research programs, and dependable communication pathways to ensure outbreak containment. Here we review the current state of knowledge on the epidemiology, ecology, molecular biology, clinical features and intervention strategies of the novel coronavirus, MERS-CoV.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses: Important Emerging Human Pathogens</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The identification of Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012 reaffirmed the importance of understanding how coronaviruses emerge, infect, and cause disease. By comparing what is known about severe acute respiratory syndrome coronavirus (SARS-CoV) to what has recently been found for MERS-CoV, researchers are discovering similarities and differences that may be important for pathogenesis. Here we discuss what is known about each virus and what gaps remain in our understanding, especially concerning MERS-CoV.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviridae and SARS-associated Coronavirus Strain HSR1</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Mar</span></i><p style="text-align: justify; text-justify: inter-word;">During the recent severe acute respiratory (SARS) outbreak, the etiologic agent was identified as a new coronavirus (CoV). We have isolated a SARS-associated CoV (SARS-CoV) strain by injecting Vero cells with a sputum specimen from an Italian patient affected by a severe pneumonia; the patient traveled from Vietnam to Italy in March 2003. Ultrastructural analysis of infected Vero cells showed the virions within cell vesicles and around the cell membrane. The full-length viral genome sequence was similar to those derived from the Hong-Kong Hotel M isolate. By using both real-time reverse transcription–polymerase chain reaction TaqMan assay and an infectivity plaque assay, we determined that approximately 360 viral genomes were required to generate a PFU. In addition, heparin (100 μg/mL) inhibited infection of Vero cells by 50%. Overall, the molecular and biologic characteristics of the strain HSR1 provide evidence that SARS-CoV forms a fourth genetic coronavirus group with distinct genomic and biologic features.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to COVID-19: A previously unknown SARS-CoV-2 virus of pandemic potential infecting humans – Call for a One Health approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses continue to pose a threat to human health. The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in December 2019 which causes coronavirus disease-2019 (COVID-19), an acute respiratory disease marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. This recent ongoing emergence of a previously unknown coronavirus in China leads to huge impacts on humans globally. Here, we discuss the outbreak in a one health context, highlighting the need for the implementation of one health measures and practices to improve human health and reduce the emergence of pandemic viruses.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>High Prevalence of Middle East Respiratory Coronavirus in Young Dromedary Camels in Jordan</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Prevalence of Middle East respiratory syndrome coronavirus (MERS-CoV) was determined in 45 dromedary camels from two geographically separated herds in Jordan. Virus shedding was only detected in swabs obtained from the respiratory tract and primarily observed in camels younger than 3 years. MERS-CoV seroprevalence increased with age of camels. Bovine and sheep sera were seronegative. Phylogenetic analysis of partial S2 clustered the Jordanian MERS-CoV strains with contemporary MERS-CoV strains associated with nosocomial outbreaks.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequence of Middle East Respiratory Syndrome Coronavirus KOR/KNIH/002_05_2015, Isolated in South Korea</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Aug 13</span></i><p style="text-align: justify; text-justify: inter-word;">The full genome sequence of a Middle East respiratory syndrome coronavirus (MERS-CoV) was identified from cultured and isolated in Vero cells. The viral genome sequence has high similarity to 53 human MERS-CoVs, ranging from 99.5% to 99.8% at the nucleotide level.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution of mouse hepatitis virus: detection and characterization of spike deletion variants during persistent infection.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">High-frequency RNA recombination has been proposed as an important mechanism for generating viral deletion variants of murine coronavirus. Indeed, a number of variants with deletions in the spike glycoprotein have been isolated from persistently infected animals. However, the significance of generating and potentially accumulating deletion variants in the persisting viral RNA population is unclear. To study this issue, we evaluated the evolution of spike variants by examining the population of spike RNA sequences detected in the brains and spinal cords of mice inoculated with coronavirus and sacrificed at 4, 42, or 100 days postinoculation. We focused on the S1 hypervariable region since previous investigators had shown that this region is subject to recombination and deletion. RNA isolated from the brains or spinal cords of infected mice was rescued by reverse transcription-PCR, and the amplified products were cloned and used in differential colony hybridizations to identify individual isolates with deletions. We found that 11 of 20 persistently infected mice harbored spike deletion variants (SDVs), indicating that deletions are common but not required for persistent infection. To determine if a specific type of SDV accumulated during persistence, we sequenced 106 of the deletion isolates. We identified 23 distinct patterns of SDVs, including 5 double-deletion variants. Furthermore, we found that each mouse harbored distinct variants in its central nervous system (CNS), suggesting that SDVs are generated during viral replication in the CNS. Interestingly, mice with the most severe and persisting neurological disease harbored the most prevalent and diverse quasispecies of SDVs. Overall, these findings illustrate the complexity of the population of persisting viral RNAs which may contribute to chronic disease.</p>
<p align="right"><i>score: 27</i></p>

</div><br /><br /><br /><div class="card"><h3>Replicative Capacity of MERS Coronavirus in Livestock Cell Lines</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Replicative capacity of Middle East respiratory syndrome coronavirus (MERS-CoV) was assessed in cell lines derived from livestock and peridomestic small mammals on the Arabian Peninsula. Only cell lines originating from goats and camels showed efficient replication of MERS-CoV. These results provide direction in the search for the intermediate host of MERS-CoV.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS and MERS: recent insights into emerging coronaviruses</h3>
<i class="tooltip">2016<span class="tooltiptext">2016</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012 marked the second introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. The continuing introductions of MERS-CoV from dromedary camels, the subsequent travel-related viral spread, the unprecedented nosocomial outbreaks and the high case-fatality rates highlight the need for prophylactic and therapeutic measures. Scientific advancements since the 2002-2003 severe acute respiratory syndrome coronavirus (SARS-CoV) pandemic allowed for rapid progress in our understanding of the epidemiology and pathogenesis of MERS-CoV and the development of therapeutics. In this Review, we detail our present understanding of the transmission and pathogenesis of SARS-CoV and MERS-CoV, and discuss the current state of development of measures to combat emerging coronaviruses.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Unanswered questions about the Middle East respiratory syndrome coronavirus (MERS-CoV)</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Jun 11</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The Middle East respiratory syndrome coronavirus (MERS-CoV) represents a current threat to the Arabian Peninsula, and potential pandemic disease. As of June 3, 2014, MERS CoV has reportedly infected 688 people and killed 282. We briefly summarize the state of the outbreak, and highlight unanswered questions and various explanations for the observed epidemiology. FINDINGS: The continuing but infrequent cases of MERS-CoV reported over the past two years have been puzzling and difficult to explain. The epidemiology of MERS-CoV, with many sporadic cases and a few hospital outbreaks, yet no sustained epidemic, suggests a low reproductive number. Furthermore, a clear source of infection to humans remains unknown. Also puzzling is the fact that MERS-CoV has been present in Saudi Arabia over several mass gatherings, including the 2012 and 2013 Hajj and Umrah pilgrimages, which predispose to epidemics, without an epidemic arising. CONCLUSIONS: The observed epidemiology of MERS-CoV is quite distinct and does not clearly fit either a sporadic or epidemic pattern. Possible explanations of the unusual features of the epidemiology of MERS-CoV include sporadic ongoing infections from a non-human source; human to human transmission with a large proportion of undetected cases; or a combination of both. The virus has been identified in camels; however the mode of transmission of the virus to humans remains unknown, and many cases have no history of animal contact. In order to gain a better understanding of the epidemiology of MERS CoV, further investigation is warranted.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun 3</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans. Though not confirmed yet, multiple surveillance and phylogenetic studies suggest a bat origin. The disease is heavily endemic in dromedary camel populations of East Africa and the Middle East. It is unclear as to when the virus was introduced to dromedary camels, but data from studies that investigated stored dromedary camel sera and geographical distribution of involved dromedary camel populations suggested that the virus was present in dromedary camels several decades ago. Though bats and alpacas can serve as potential reservoirs for MERS-CoV, dromedary camels seem to be the only animal host responsible for the spill over human infections.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-associated Coronavirus Transmission, United States</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">To better assess the risk for transmission of the severe acute respiratory syndrome–associated coronavirus (SARS-CoV), we obtained serial specimens and clinical and exposure data from seven confirmed U.S. SARS patients and their 10 household contacts. SARS-CoV was detected in a day-14 sputum specimen from one case-patient and in five stool specimens from two case-patients. In one case-patient, SARS-CoV persisted in stool for at least 26 days after symptom onset. The highest amounts of virus were in the day-14 sputum sample and a day-14 stool sample. Residual respiratory symptoms were still present in recovered SARS case-patients 2 months after illness onset. Possible transmission of SARS-CoV occurred in one household contact, but this person had also traveled to a SARS-affected area. The data suggest that SARS-CoV is not always transmitted efficiently. Laboratory diagnosis of SARS-CoV infection is difficult; thus, sputum and stool specimens should be included in the diagnostic work-up for SARS-CoV infection.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV Antibodies in Humans, Africa, 2013–2014</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">Dromedaries in Africa and elsewhere carry the Middle East respiratory syndrome coronavirus (MERS-CoV). To search for evidence of autochthonous MERS-CoV infection in humans, we tested archived serum from livestock handlers in Kenya for MERS-CoV antibodies. Serologic evidence of infection was confirmed for 2 persons sampled in 2013 and 2014.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV Antibodies in Humans, Africa, 2013–2014</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jun</span></i><p style="text-align: justify; text-justify: inter-word;">Dromedaries in Africa and elsewhere carry the Middle East respiratory syndrome coronavirus (MERS-CoV). To search for evidence of autochthonous MERS-CoV infection in humans, we tested archived serum from livestock handlers in Kenya for MERS-CoV antibodies. Serologic evidence of infection was confirmed for 2 persons sampled in 2013 and 2014.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviruses: emerging and re-emerging pathogens in humans and animals</h3>
<i class="tooltip">2015<span class="tooltiptext">2015 Dec 22</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome coronavirus (SARS-CoV) and recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV) epidemics have proven the ability of coronaviruses to cross species barrier and emerge rapidly in humans. Other coronaviruses such as porcine epidemic diarrhea virus (PEDV) are also known to cause major disease epidemics in animals wiith huge economic loss. This special issue in Virology Journal aims to highlight the advances and key discoveries in the animal origin, viral evolution, epidemiology, diagnostics and pathogenesis of the emerging and re-emerging coronaviruses in both humans and animals.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Role of the Spike Glycoprotein of Human Middle East Respiratory Syndrome Coronavirus (MERS-CoV) in Virus Entry and Syncytia Formation</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Oct 3</span></i><p style="text-align: justify; text-justify: inter-word;">Little is known about the biology of the emerging human group c betacoronavirus, Middle East Respiratory Syndrome coronavirus (MERS-CoV). Because coronavirus spike glycoproteins (S) mediate virus entry, affect viral host range, and elicit neutralizing antibodies, analyzing the functions of MERS-CoV S protein is a high research priority. MERS-CoV S on lentivirus pseudovirions mediated entry into a variety of cell types including embryo cells from New World Eptesicus fuscus bats. Surprisingly, a polyclonal antibody to the S protein of MHV, a group a murine betacoronavirus, cross-reacted in immunoblots with the S2 domain of group c MERS-CoV spike protein. MERS pseudovirions released from 293T cells contained only uncleaved S, and pseudovirus entry was blocked by lysosomotropic reagents NH(4)Cl and bafilomycin and inhibitors of cathepsin L. However, when MERS pseudovirions with uncleaved S protein were adsorbed at 4°C to Vero E6 cells, brief trypsin treatment at neutral pH triggered virus entry at the plasma membrane and syncytia formation. When 293T cells producing MERS pseudotypes co-expressed serine proteases TMPRSS-2 or -4, large syncytia formed at neutral pH, and the pseudovirions produced were non-infectious and deficient in S protein. These experiments show that if S protein on MERS pseudovirions is uncleaved, then viruses enter by endocytosis in a cathepsin L-dependent manner, but if MERS-CoV S is cleaved, either during virus maturation by serine proteases or on pseudovirions by trypsin in extracellular fluids, then viruses enter at the plasma membrane at neutral pH and cause massive syncytia formation even in cells that express little or no MERS-CoV receptor. Thus, whether MERS-CoV enters cells within endosomes or at the plasma membrane depends upon the host cell type and tissue, and is determined by the location of host proteases that cleave the viral spike glycoprotein and activate membrane fusion.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultrastructural Characterization of SARS Coronavirus</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) was first described during a 2002–2003 global outbreak of severe pneumonia associated with human deaths and person-to-person disease transmission. The etiologic agent was initially identified as a coronavirus by thin-section electron microscopic examination of a virus isolate. Virions were spherical, 78 nm in mean diameter, and composed of a helical nucleocapsid within an envelope with surface projections. Herein, we show that infection with the SARS-associated coronavirus resulted in distinct ultrastructural features: double-membrane vesicles, nucleocapsid inclusions, and large granular areas of cytoplasm. These three structures and the coronavirus particles were shown to be positive for viral proteins and RNA by using ultrastructural immunogold and in situ hybridization assays. In addition, ultrastructural examination of a bronchiolar lavage specimen from a SARS patient showed numerous coronavirus-infected cells with features similar to those in infected culture cells. Electron microscopic studies were critical in identifying the etiologic agent of the SARS outbreak and in guiding subsequent laboratory and epidemiologic investigations.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus: epidemiology and disease control measures</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Nov 3</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) infection in 2012 resulted in an increased concern of the spread of the infection globally. MERS-CoV infection had previously caused multiple health-care-associated outbreaks and resulted in transmission of the virus within families. Community onset MERS-CoV cases continue to occur. Dromedary camels are currently the most likely animal to be linked to human MERS-CoV cases. Serologic tests showed significant infection in adult camels compared to juvenile camels. The control of MERS-CoV infection relies on prompt identification of cases within health care facilities, with institutions applying appropriate infection control measures. In addition, determining the exact route of transmission from camels to humans would further add to the control measures of MERS-CoV infection.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of SARS Coronavirus in Patients with Suspected SARS</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Cases of severe acute respiratory syndrome (SARS) were investigated for SARS coronavirus (SARS-CoV) through RNA tests, serologic response, and viral culture. Of 537 specimens from patients in whom SARS was clinically diagnosed, 332 (60%) had SARS-CoV RNA in one or more clinical specimens, compared with 1 (0.3%) of 332 samples from controls. Of 417 patients with clinical SARS from whom paired serum samples were available, 92% had an antibody response. Rates of viral RNA positivity increased progressively and peaked at day 11 after onset of illness. Although viral RNA remained detectable in respiratory secretions and stool and urine specimens for >30 days in some patients, virus could not be cultured after week 3 of illness. Nasopharyngeal aspirates, throat swabs, or sputum samples were the most useful clinical specimens in the first 5 days of illness, but later in the illness viral RNA could be detected more readily in stool specimens.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative properties of feline coronaviruses in vitro.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two feline coronaviruses were characterized to determine their biological properties in vitro and their antigenic relatedness to a previously recognized feline infectious peritonitis virus and canine coronavirus. The viruses, designated WSU 79-1146 and WSU 79-1683, were shown to have comparable growth curves with the prototype feline infectious peritonitis virus. Treatment of the feline infectious peritonitis virus strains with 0.25% trypsin indicated that they were relatively resistant to proteolytic inactivation when compared with the feline enteric coronavirus strain. This observation may serve as a useful in vitro marker to distinguish closely related members of the feline coronavirus group. Plaque assay results indicated that the feline infectious peritonitis virus strains produced large homogeneous plaques in comparison to the feline enteric coronavirus strain and canine coronavirus, which showed a heterogenous plaque size distribution. No naturally temperature sensitive mutants were detected in either of the feline coronavirus populations. Both of the viruses were antigenically related to feline infectious peritonitis virus and to a lesser extent to canine coronavirus by virus neutralization.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Dromedary camels in northern Mali have high seropositivity to MERS-CoV</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Mar 10</span></i><p style="text-align: justify; text-justify: inter-word;">A high percentage (up to 90%) of dromedary camels in the Middle East as well as eastern and central Africa have antibodies to Middle East respiratory syndrome coronavirus (MERS-CoV). Here we report comparably high positivity of MERS-CoV antibodies in dromedary camels from northern Mali. This extends the range of MERS-CoV further west in Africa than reported to date and cautions that MERS-CoV should be considered in cases of severe respiratory disease in the region.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV: Understanding the Latest Human Coronavirus Threat</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Feb 24</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses cause both upper and lower respiratory tract infections in humans. In 2012, a sixth human coronavirus (hCoV) was isolated from a patient presenting with severe respiratory illness. The 60-year-old man died as a result of renal and respiratory failure after admission to a hospital in Jeddah, Saudi Arabia. The aetiological agent was eventually identified as a coronavirus and designated Middle East respiratory syndrome coronavirus (MERS-CoV). MERS-CoV has now been reported in more than 27 countries across the Middle East, Europe, North Africa and Asia. As of July 2017, 2040 MERS-CoV laboratory confirmed cases, resulting in 712 deaths, were reported globally, with a majority of these cases from the Arabian Peninsula. This review summarises the current understanding of MERS-CoV, with special reference to the (i) genome structure; (ii) clinical features; (iii) diagnosis of infection; and (iv) treatment and vaccine development.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete Genome Sequence of Middle East Respiratory Syndrome Coronavirus Isolated from a Dromedary Camel in Egypt</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Apr 28</span></i><p style="text-align: justify; text-justify: inter-word;">We generated the near-full genome sequence of Middle East respiratory syndrome coronavirus (MERS-CoV) from a collected nasal sample of dromedary camel in Egypt. The newly characterized Egyptian strain has high similarity to the previously characterized Egyptian virus and both of viruses fell into a cluster distinct from other MERS-CoVs.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Plaque assay for human coronavirus NL63 using human colon carcinoma cells</h3>
<i class="tooltip">2008<span class="tooltiptext">2008 Nov 12</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: Coronaviruses cause a broad range of diseases in animals and humans. Human coronavirus (hCoV) NL63 is associated with up to 10% of common colds. Viral plaque assays enable the characterization of virus infectivity and allow for purifying virus stock solutions. They are essential for drug screening. Hitherto used cell cultures for hCoV-NL63 show low levels of virus replication and weak and diffuse cytopathogenic effects. It has not yet been possible to establish practicable plaque assays for this important human pathogen. RESULTS: 12 different cell cultures were tested for susceptibility to hCoV-NL63 infection. Human colon carcinoma cells (CaCo-2) replicated virus more than 100 fold more efficiently than commonly used African green monkey kidney cells (LLC-MK2). CaCo-2 cells showed cytopathogenic effects 4 days post infection. Avicel, agarose and carboxymethyl-cellulose overlays proved suitable for plaque assays. Best results were achieved with Avicel, which produced large and clear plaques from the 4(th )day of infection. The utility of plaque assays with agrose overlay was demonstrated for purifying virus, thereby increasing viral infectivity by 1 log 10 PFU/mL. CONCLUSION: CaCo-2 cells support hCoV-NL63 better than LLC-MK2 cells and enable cytopathogenic plaque assays. Avicel overlay is favourable for plaque quantification, and agarose overlay is preferred for plaque purification. HCoV-NL63 virus stock of increased infectivity will be beneficial in antiviral screening, animal modelling of disease, and other experimental tasks.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>MIDDLE EAST RESPIRATORY SYNDROME VACCINES</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East Respiratory Syndrome-coronavirus (MERS-CoV) has infected over 1600 individuals with nearly 600 deaths since it was first identified in human populations in 2012. No anti-viral therapies or vaccines for treatment and prophylaxis are available. Here, we discuss approaches to developing MERS vaccines, including a summary of previous efforts to develop vaccines useful against human and non-human coronaviruses. A striking feature of MERS is the important role that camels have in transmission. Camel vaccination may be a novel approach to preventing the human infection.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Dromedary camels and the transmission of Middle East Respiratory Syndrome Coronavirus (MERS-CoV)</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an existential threat to global public health. The virus has been repeatedly detected in dromedary camels (Camelus dromedarius). Adult animals in many countries in the Middle East as well as in North and East Africa showed high (>90%) sero-prevalence to the virus. MERS-CoV isolated from dromedaries is genetically and phenotypically similar to viruses from humans. We summarise current understanding of the ecology of MERS-CoV in animals and transmission at the animal-human interface. We review aspects of husbandry, animal movements and trade and the use and consumption of camel dairy and meat products in the Middle East that may be relevant to the epidemiology of MERS. We also highlight the gaps in understanding the transmission of this virus in animals and from animals to humans.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Detectable 2019-nCoV viral RNA in blood is a strong indicator for the further clinical severity</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus (2019-nCoV) infection caused pneumonia. we retrospectively analyzed the virus presence in the pharyngeal swab, blood, and the anal swab detected by real-time PCR in the clinical lab. Unexpectedly, the 2109-nCoV RNA was readily detected in the blood (6 of 57 patients) and the anal swabs (11 of 28 patients). Importantly, all of the 6 patients with detectable viral RNA in the blood cohort progressed to severe symptom stage, indicating a strong correlation of serum viral RNA with the disease severity (p-value = 0.0001). Meanwhile, 8 of the 11 patients with annal swab virus-positive was in severe clinical stage. However, the concentration of viral RNA in the anal swab (Ct value = 24 + 39) was higher than in the blood (Ct value = 34 + 39) from patient 2, suggesting that the virus might replicate in the digestive tract. Altogether, our results confirmed the presence of virus RNA in extra-pulmonary sites.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 17</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world’s ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 17</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world’s ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibodies against MERS Coronavirus in Dromedary Camels, United Arab Emirates, 2003 and 2013</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Apr</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) has caused an ongoing outbreak of severe acute respiratory tract infection in humans in the Arabian Peninsula since 2012. Dromedary camels have been implicated as possible viral reservoirs. We used serologic assays to analyze 651 dromedary camel serum samples from the United Arab Emirates; 151 of 651 samples were obtained in 2003, well before onset of the current epidemic, and 500 serum samples were obtained in 2013. Recombinant spike protein–specific immunofluorescence and virus neutralization tests enabled clear discrimination between MERS-CoV and bovine CoV infections. Most (632/651, 97.1%) camels had antibodies against MERS-CoV. This result included all 151 serum samples obtained in 2003. Most (389/651, 59.8%) serum samples had MERS-CoV–neutralizing antibody titers >1,280. Dromedary camels from the United Arab Emirates were infected at high rates with MERS-CoV or a closely related, probably conspecific, virus long before the first human MERS cases.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A52 MERS coronaviruses from camels in Africa exhibit region-dependent genetic diversity</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Aug 22</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) causes a zoonotic respiratory disease of global public health concern, and dromedary camels are the only proven source of this zoonotic infection. Although MERS-CoV infection is ubiquitous in dromedaries across Africa and the Arabian Peninsula, the continuous appearance of zoonotic MERS cases in humans is confined to the Arabian Peninsula. MERS-CoV from Africa has hitherto been poorly studied. Here, we report the genetic and phenotypic characterization of MERS-CoV from dromedaries in African countries. Phylogenetically, viruses from dromedaries in Africa formed a monophyletic clade, which we have provisionally designated as virus clade C. Molecular dating analyses of MERS-CoV, including clade C viruses, suggests that the ancestral MERS-CoV in dromedaries could have spread to the two continents within a short timeframe. Camel MERS-CoVs from west and north African countries form a subclade (C1) that shares genetic signatures of a major deletion in the accessory gene ORF4b. Compared with human and camel MERS-CoV from Saudi Arabia, virus isolates from Burkina Faso (BF785) and Nigeria (Nig1657) had lower virus replication competence in Calu-3 cells and in ex vivo cultures of human bronchus and lung, and BF785 replicated to lower titer in lungs of human DPP4-transduced mice. However, it is still inconclusive whether ORF4b deletions may lead to the reduced replication competence of BF785 and Nig1657. Genetic and phenotypic differences in West African viruses may be relevant to the zoonotic potential of MERS-CoV.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Sero-epidemiology of MERS coronavirus in Saudi Arabia (1993) and Australia (2014) and characterization of assay specificity</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Pseudoparticle virus neutralization (ppNT) and a conventional microneutralization (MN) assays are specific for detecting antibodies to MERS coronavirus (MERS-CoV) when used in sero-epidemiological studies in animals. Genetically diverse MERS-CoV appear antigenically similar in MN tests. We confirm that MERS-CoV has been circulating in dromedaries in Saudi Arabia in 1993. Preliminary data suggests that feral Australian dromedaries may be free of MERS-CoV but larger confirmatory studies are needed.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Infection in Dromedary Camels in Saudi Arabia</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Feb 25</span></i><p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome (MERS) is proposed to be a zoonotic disease; however, the reservoir and mechanism for transmission of the causative agent, the MERS coronavirus, are unknown. Dromedary camels have been implicated through reports that some victims have been exposed to camels, camels in areas where the disease has emerged have antibodies to the virus, and viral sequences have been recovered from camels in association with outbreaks of the disease among humans. Nonetheless, whether camels mediate transmission to humans is unresolved. Here we provide evidence from a geographic and temporal survey of camels in the Kingdom of Saudi Arabia that MERS coronaviruses have been circulating in camels since at least 1992, are distributed countrywide, and can be phylogenetically classified into clades that correlate with outbreaks of the disease among humans. We found no evidence of infection in domestic sheep or domestic goats.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>A Mouse Model for Betacoronavirus Subgroup 2c Using a Bat Coronavirus Strain HKU5 Variant</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Mar 25</span></i><p style="text-align: justify; text-justify: inter-word;">Cross-species transmission of zoonotic coronaviruses (CoVs) can result in pandemic disease outbreaks. Middle East respiratory syndrome CoV (MERS-CoV), identified in 2012, has caused 182 cases to date, with ~43% mortality, and no small animal model has been reported. MERS-CoV and Pipistrellus bat coronavirus (BtCoV) strain HKU5 of Betacoronavirus (β-CoV) subgroup 2c share >65% identity at the amino acid level in several regions, including nonstructural protein 5 (nsp5) and the nucleocapsid (N) protein, which are significant drug and vaccine targets. BtCoV HKU5 has been described in silico but has not been shown to replicate in culture, thus hampering drug and vaccine studies against subgroup 2c β-CoVs. We report the synthetic reconstruction and testing of BtCoV HKU5 containing the severe acute respiratory syndrome (SARS)-CoV spike (S) glycoprotein ectodomain (BtCoV HKU5-SE). This virus replicates efficiently in cell culture and in young and aged mice, where the virus targets airway and alveolar epithelial cells. Unlike some subgroup 2b SARS-CoV vaccines that elicit a strong eosinophilia following challenge, we demonstrate that BtCoV HKU5 and MERS-CoV N-expressing Venezuelan equine encephalitis virus replicon particle (VRP) vaccines do not cause extensive eosinophilia following BtCoV HKU5-SE challenge. Passage of BtCoV HKU5-SE in young mice resulted in enhanced virulence, causing 20% weight loss, diffuse alveolar damage, and hyaline membrane formation in aged mice. Passaged virus was characterized by mutations in the nsp13, nsp14, open reading frame 5 (ORF5) and M genes. Finally, we identified an inhibitor active against the nsp5 proteases of subgroup 2c β-CoVs. Synthetic-genome platforms capable of reconstituting emerging zoonotic viral pathogens or their phylogenetic relatives provide new strategies for identifying broad-based therapeutics, evaluating vaccine outcomes, and studying viral pathogenesis.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe Acute Respiratory Syndrome Coronavirus as an Agent of Emerging and Reemerging Infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Before the emergence of severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) in 2003, only 12 other animal or human coronaviruses were known. The discovery of this virus was soon followed by the discovery of the civet and bat SARS-CoV and the human coronaviruses NL63 and HKU1. Surveillance of coronaviruses in many animal species has increased the number on the list of coronaviruses to at least 36. The explosive nature of the first SARS epidemic, the high mortality, its transient reemergence a year later, and economic disruptions led to a rush on research of the epidemiological, clinical, pathological, immunological, virological, and other basic scientific aspects of the virus and the disease. This research resulted in over 4,000 publications, only some of the most representative works of which could be reviewed in this article. The marked increase in the understanding of the virus and the disease within such a short time has allowed the development of diagnostic tests, animal models, antivirals, vaccines, and epidemiological and infection control measures, which could prove to be useful in randomized control trials if SARS should return. The findings that horseshoe bats are the natural reservoir for SARS-CoV-like virus and that civets are the amplification host highlight the importance of wildlife and biosecurity in farms and wet markets, which can serve as the source and amplification centers for emerging infections.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Guinea Fowl Coronavirus Diversity Has Phenotypic Consequences for Glycan and Tissue Binding</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 May 1</span></i><p style="text-align: justify; text-justify: inter-word;">Guinea fowl coronavirus (GfCoV) causes fulminating enteritis that can result in a daily death rate of 20% in guinea fowl flocks. Here, we studied GfCoV diversity and evaluated its phenotypic consequences. Over the period of 2014 to 2016, affected guinea fowl flocks were sampled in France, and avian coronavirus presence was confirmed by PCR on intestinal content and immunohistochemistry of intestinal tissue. Sequencing revealed 89% amino acid identity between the viral attachment protein S1 of GfCoV/2014 and that of the previously identified GfCoV/2011. To study the receptor interactions as a determinant for tropism and pathogenicity, recombinant S1 proteins were produced and analyzed by glycan and tissue arrays. Glycan array analysis revealed that, in addition to the previously elucidated biantennary di-N-acetyllactosamine (diLacNAc) receptor, viral attachment S1 proteins from GfCoV/2014 and GfCoV/2011 can bind to glycans capped with alpha-2,6-linked sialic acids. Interestingly, recombinant GfCoV/2014 S1 has an increased affinity for these glycans compared to that of GfCoV/2011 S1, which was in agreement with the increased avidity of GfCoV/2014 S1 for gastrointestinal tract tissues. Enzymatic removal of receptors from tissues before application of spike proteins confirmed the specificity of S1 tissue binding. Overall, we demonstrate that diversity in GfCoV S1 proteins results in differences in glycan and tissue binding properties. IMPORTANCE Avian coronaviruses cause major global problems in the poultry industry. As causative agents of huge economic losses, the detection and understanding of the molecular determinants of viral tropism are of ultimate importance. Here, we set out to study those parameters and obtained in-depth insight into the virus-host interactions of guinea fowl coronavirus (GfCoV). Our data indicate that diversity in GfCoV viral attachment proteins results in differences in degrees of affinity for glycan receptors, as well as altered avidity for intestinal tract tissues, which might have consequences for GfCoV tissue tropism and pathogenesis in guinea fowls.</p>
<p align="right"><i>score: 26</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Advanced Reverse Transcription-PCR Assays and an Alternative PCR Target Region for Detection of Severe Acute Respiratory Syndrome-Associated Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">First-generation reverse transcription-PCR (RT-PCR) assays for severe acute respiratory syndrome-associated coronavirus (SARS-CoV) gave false-negative results in a considerable fraction of patients. In the present study, we evaluated two second-generation, replicase (R) gene-based, real-time RT-PCR test kits—the RealArt HPA coronavirus LC kit (Artus, Hamburg, Germany) and the LightCycler SARS-CoV quantification kit (Roche, Penzberg, Germany)—and a real-time RT-PCR assay for the nucleocapsid (N) gene. Detecting the N-gene RNA might be advantageous due to its high abundance in cells. The kits achieved sensitivities of 70.8% (Artus) and 67.1% (Roche) in 66 specimens from patients with confirmed SARS (samples primarily from the upper and lower respiratory tract and stool). The sensitivity of the N-gene assay was 74.2%. The differences in all of the sensitivities were not statistically significant (P = 0.680 [analysis of variance]). Culture cells initially contained five times more N- than R-gene RNA, but the respective levels converged during 4 days of virus replication. In clinical samples the median concentrations of R- and N-gene RNA, respectively, were 1.2 × 10(6) and 2.8 × 10(6) copies/ml (sputum and endotracheal aspirates), 4.3 × 10(4) and 5.5 × 10(4) copies/ml (stool), and 5.5 × 10(2) and 5.2 × 10(2) copies/sample (throat swabs and saliva). Differences between the samples types were significant but not between the types of target RNA. All (n = 12) samples from the lower respiratory tract tested positive in all tests. In conclusion, the novel assays are more sensitive than the first-generation tests, but they still do not allow a comprehensive ruling out of SARS. Methods for the routine sampling of sputum without infection risk are needed to improve SARS RT-PCR.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome: A new global threat</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of Middle East respiratory syndrome (MERS) is reported from Saudi Arabia and the Republic of Korea. It is a respiratory disease caused by coronavirus. Camels are considered as a source for MERS transmission in humans, although the exact source is unknown. Human-to-human transmission is reported in the community with droplet and contact spread being the possible modes. Most patients without any underlying diseases remain asymptomatic or develop mild clinical disease, but some patients require critical care for mechanical ventilation, dialysis and other organ support. MERS is a disease with pandemic potential and awareness, and surveillance can prevent such further outbreaks.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and Characterization of a Novel Bat Coronavirus Closely Related to the Direct Progenitor of Severe Acute Respiratory Syndrome Coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We report the isolation and characterization of a novel bat coronavirus which is much closer to the severe acute respiratory syndrome coronavirus (SARS-CoV) in genomic sequence than others previously reported, particularly in its S gene. Cell entry and susceptibility studies indicated that this virus can use ACE2 as a receptor and infect animal and human cell lines. Our results provide further evidence of the bat origin of the SARS-CoV and highlight the likelihood of future bat coronavirus emergence in humans.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution and variation of 2019-novel coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-30</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.  MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.  ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1{middle dot}05 x 10-2 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 x 10-4 to 2.72 x 10-2) for N, 5.34 x 10-3 (5.10 x 10-4, 1.28 x 10-2) for S, 1.69 x 10-3 (3.94 x 10-4, 3.60 x 10-3) for P, 1.65 x 10-3 (4.47 x 10-4, 3.24 x 10-3) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.  ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation, propagation, genome analysis and epidemiology of HKU1 betacoronaviruses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">From 1 January 2009 to 31 May 2013, 15 287 respiratory specimens submitted to the Clinical Virology Laboratory at the Children’s Hospital Colorado were tested for human coronavirus RNA by reverse transcription-PCR. Human coronaviruses HKU1, OC43, 229E and NL63 co-circulated during each of the respiratory seasons but with significant year-to-year variability, and cumulatively accounted for 7.4–15.6 % of all samples tested during the months of peak activity. A total of 79 (0.5 % prevalence) specimens were positive for human betacoronavirus HKU1 RNA. Genotypes HKU1 A and B were both isolated from clinical specimens and propagated on primary human tracheal–bronchial epithelial cells cultured at the air–liquid interface and were neutralized in vitro by human intravenous immunoglobulin and by polyclonal rabbit antibodies to the spike glycoprotein of HKU1. Phylogenetic analysis of the deduced amino acid sequences of seven full-length genomes of Colorado HKU1 viruses and the spike glycoproteins from four additional HKU1 viruses from Colorado and three from Brazil demonstrated remarkable conservation of these sequences with genotypes circulating in Hong Kong and France. Within genotype A, all but one of the Colorado HKU1 sequences formed a unique subclade defined by three amino acid substitutions (W197F, F613Y and S752F) in the spike glycoprotein and exhibited a unique signature in the acidic tandem repeat in the N-terminal region of the nsp3 subdomain. Elucidating the function of and mechanisms responsible for the formation of these varying tandem repeats will increase our understanding of the replication process and pathogenicity of HKU1 and potentially of other coronaviruses.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of SARS-associated Coronavirus in Throat Wash and Saliva in Early Diagnosis</h3>
<i class="tooltip">2004<span class="tooltiptext">2004 Jul</span></i><p style="text-align: justify; text-justify: inter-word;">The severe acute respiratory syndrome–associated coronavirus (SARS-CoV) is thought to be transmitted primarily through dispersal of droplets, but little is known about the load of SARS-CoV in oral droplets. We examined oral specimens, including throat wash and saliva, and found large amounts of SARS-CoV RNA in both throat wash (9.58 x 10(2) to 5.93 x 10(6) copies/mL) and saliva (7.08 x 10(3) to 6.38 x 10(8) copies/mL) from all specimens of 17 consecutive probable SARS case-patients, supporting the possibility of transmission through oral droplets. Immunofluorescence study showed replication of SARS-CoV in the cells derived from throat wash, demonstrating the possibility of developing a convenient antigen detection assay. This finding, with the high detection rate a median of 4 days after disease onset and before the development of lung lesions in four patients, suggests that throat wash and saliva should be included in sample collection guidelines for SARS diagnosis.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can utilize GRP78 for attachment onto host cells</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus tropism is predominantly determined by the interaction between coronavirus spikes and the host receptors. In this regard, coronaviruses have evolved a complicated receptor-recognition system through their spike proteins. Spikes from highly related coronaviruses can recognize distinct receptors, whereas spikes of distant coronaviruses can employ the same cell-surface molecule for entry. Moreover, coronavirus spikes can recognize a broad range of cell-surface molecules in addition to the receptors and thereby can augment coronavirus attachment or entry. The receptor of Middle East respiratory syndrome coronavirus (MERS-CoV) is dipeptidyl peptidase 4 (DPP4). In this study, we identified membrane-associated 78-kDa glucose-regulated protein (GRP78) as an additional binding target of the MERS-CoV spike. Further analyses indicated that GRP78 could not independently render nonpermissive cells susceptible to MERS-CoV infection but could facilitate MERS-CoV entry into permissive cells by augmenting virus attachment. More importantly, by exploring potential interactions between GRP78 and spikes of other coronaviruses, we discovered that the highly conserved human GRP78 could interact with the spike protein of bat coronavirus HKU9 (bCoV-HKU9) and facilitate its attachment to the host cell surface. Taken together, our study has identified GRP78 as a host factor that can interact with the spike proteins of two Betacoronaviruses, the lineage C MERS-CoV and the lineage D bCoV-HKU9. The capacity of GRP78 to facilitate surface attachment of both a human coronavirus and a phylogenetically related bat coronavirus exemplifies the need for continuous surveillance of the evolution of animal coronaviruses to monitor their potential for human adaptations.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Betacoronavirus in Dromedaries of the Middle East, 2013</h3>
<i class="tooltip">2014<span class="tooltiptext">2014 Apr</span></i><p style="text-align: justify; text-justify: inter-word;">In 2013, a novel betacoronavirus was identified in fecal samples from dromedaries in Dubai, United Arab Emirates. Antibodies against the recombinant nucleocapsid protein of the virus, which we named dromedary camel coronavirus (DcCoV) UAE-HKU23, were detected in 52% of 59 dromedary serum samples tested. In an analysis of 3 complete DcCoV UAE-HKU23 genomes, we identified the virus as a betacoronavirus in lineage A1. The DcCoV UAE-HKU23 genome has G+C contents; a general preference for G/C in the third position of codons; a cleavage site for spike protein; and a membrane protein of similar length to that of other betacoronavirus A1 members, to which DcCoV UAE-HKU23 is phylogenetically closely related. Along with this coronavirus, viruses of at least 8 other families have been found to infect camels. Because camels have a close association with humans, continuous surveillance should be conducted to understand the potential for virus emergence in camels and for virus transmission to humans.</p>
<p align="right"><i>score: 25</i></p>

</div><br /><br /><br /></body>