
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: role of the private sector to help quickly migrate SARS-CoV-2 assays onto devices</h2><br /></div><br /><br /><br /><div class="card"><h3>Coronavirus and the race to distribute reliable diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">International teams worked at speed to make tests for the virus available in record time. The medical community is rallying to develop a set of rapid and reliable molecular diagnostic tests for the new human coronavirus that appeared in China — now dubbed sudden acute respiratory syndrome coronavirus-2 (SARS-CoV-2).</p>
<p align="right"><i>score: 84</i></p>

</div><br /><br /><br /><div class="card"><h3>Four ways researchers are responding to the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts. How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Labs scramble to spot hidden coronavirus infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The seeming precision of the global tallies of cases and deaths caused by the novel coronavirus now spreading from Wuhan, China belies an alarming fact. ... (A study group of the International Committee on Taxonomy of Viruses christened the novel virus severe acute respiratory syndrome coronavirus 2, or SARS-CoV-2, the same day). But many news stories have reported shortages of diagnostics in Hubei.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a quantitative RT-PCR assay for the detection of the emerging coronavirus SARS-CoV-2 using a high throughput system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ability to quickly confirm or clear suspected cases is crucial during global outbreak scenarios, especially when clinical manifestations are difficult to distinguish from other respiratory infections such as influenza, molecular diagnostics is key for detection of the emerging virus. A variety of suitable assays were made available early on during the course of the outbreak, notably by Corman et al. and others [4,5]. However, their implementation in the diagnostics laboratory usually relies on manual PCR setups requiring a high degree of human interaction for execution and interpretation, thus limiting their capacity to be scaled up for handling large numbers of samples. In this study we report the analytical evaluation of a laboratory-developed test for the detection of SARS-CoV-2 using the open channel (utility channel) of the cobas 6800 system.%U https://eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.9.2000152</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Journey of a Thai Taxi Driver and Novel Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On January 20, 2020, a 51-year-old male taxi driver had fever, cough, and myalgia and went to a local pharmacy to get unspecified over-the-counter medications. At the time, he was not aware of the emergence of SARS-CoV-2 or the illness it causes (Covid-19). As the symptoms persisted, he decided to visit a private primary care clinic in Bangkok on January 23. The body temperature was 36.8°C (98°F). The clinic physician ordered a throat swab for influenza A and B; the swab was negative for both strains. Additional medications were prescribed for treatment of the patient’s symptoms.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid establishment of laboratory diagnostics for the novel coronavirus SARS-CoV-2 in Bavaria, Germany, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In connection with the ongoing outbreak of a novel coronavirus in the province Hubei and surrounding areas in China, it was expected that Europe would also be confronted with the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), as infections in travellers in several Asian countries outside of China were confirmed shortly after the announcement of the outbreak in Wuhan [1-4]. Therefore, it was necessary to rapidly implement adequately quick and sensitive diagnostic assays for outbreak management of SARS-CoV-2 in public health laboratories. As soon as the World Health Organization (WHO) published the first protocols for real-time RT-PCR assays, the Bavarian Food and Health Authority started to implement them. We ordered control material and oligonucleotides (see details below) in week 4 and ran our first SARS-CoV-2 assays on 27 January (week 5). On the same day, the first German case of coronavirus disease 2019 (COVID-19) was diagnosed in Bavaria [1]. In the following days, health authorities implemented comprehensive measures to prevent further transmission of SARS-CoV-2, including testing of contact persons. We initially used the protocol based on the E gene and RdRp gene developed by the German Consiliary Laboratory for Coronaviruses hosted at the Charité in Berlin [5]. Real-time RT-PCR was initially performed with the QuantiTect Virus +Rox Vial kit (QIAGEN, Hilden, Germany) on the Bio-Rad CFX96 Touch Real-Time PCR Detection System (Bio-Rad, Feldkirchen, Germany). The kit was chosen for its frequent and successful use in our laboratory with other assays and its immediate availability. Primers and probes were used as described [5] and provided by TIB Molbiol (Berlin, Germany). Control material was ordered from the European Virus Archive (EVAg) and consisted of synthetic Wuhan coronavirus 2019 E gene control (reference number 026N-03866) and SARS-CoV Frankfurt 1 RNA (reference number 004N-02005) [6]. In addition, the control of LightMix Modular Wuhan CoV RdRP-gene (TibMolbiol, Berlin, Germany) was used for the SARS-CoV-2 specific assay. Respiratory samples (nasopharyngeal swabs or sputum) were obtained from patients and contact persons. Sputum samples were diluted in 2 mL phosphate buffered saline (PBS). RNA was extracted using the QIAamp Bio Robot kit (QIAGEN) on a Hamilton Microlab Star (Hamilton, Bonaduz, Switzerland). As at 1 March, 87,024 cases and 2,979 associated deaths have been reported worldwide [1]. The vast majority of the deaths (96%) have been reported in China [1]. Despite the high number of cases reported globally, estimates of the severity pyramid of disease and case fatality rate remain very uncertain; one large study conducted in China estimated that the majority (81%) of the cases were mild (i.e. non-pneumonia or mild pneumonia), 14% were severe (e.g. with dyspnoea) and 5% were in a critical condition (i.e. respiratory failure, septic shock and/or multiple organ dysfunction/failure) [2]. The case fatality ratio was 2.3% [2]. Despite extraordinary containment measures implemented in China, including the enforced lockdown of several cities and closures of schools, the virus has spread throughout the country and internationally [2]. It is too early to predict with any certainty the epidemiological developments over the coming weeks, but the possibility of widespread community transmission becoming established throughout the EU/EEA is becoming increasingly likely.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Biotech start-ups hit by coronavirus work stoppages</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">s the outbreak of the new coronavirus, now officially named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), continues to mushroom, the related work stoppages in China have put a spotlight on Western biotech companies’ dependence on Chinese contract research organizations (CROs). The situation has prompted some start-ups to rethink their contingency plans—particularly when it comes to chemistry services. Biotech firms had anticipated delays related to the Lunar New Year holidays, and so far outbreak-related travel restrictions have extended that work gap by only a week. But as the virus spreads—according to Chinese authorities, as of Feb. 13, more than 60,100 people had been infected and nearly 1,400 had died, most of them in China—some biotech firms that rely on China are becoming nervous about how long their projects could be on hold. Much attention is on WuXi AppTec, the largest Chinese CRO.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>A Locally Transmitted Case of SARS-CoV-2 Infection in Taiwan</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On January 25, 2020, a 52-year-old woman with a history of type 2 diabetes presented with fever to an emergency department in central Taiwan. She was admitted to the hospital because of suspicion of pneumonia associated with SARS-CoV-2 infection. She had lived in Wuhan from October 21, 2019, to January 20, 2020. She returned to Taiwan from Wuhan on January 20 on an airplane. On the same day, a throat swab was obtained from another passenger on that flight; that passenger was confirmed to have the first known imported case of SARS-CoV-2 infection in Taiwan when the swab was found to be positive for the virus on January 21.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus  broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</jats:p></p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Sensitive one-step isothermal detection of pathogen-derived RNAs</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreaks of Ebola, Zika, MERS, and SARS-CoV-2 (2019-nCoV) require fast, simple, and sensitive onsite nucleic acid diagnostics that can be developed rapidly to prevent the spread of diseases. We have developed a SENsitive Splint-based one-step isothermal RNA detection (SENSR) method for rapid and straightforward onsite detection of pathogen RNAs with high sensitivity and specificity. SENSR consists of two simple enzymatic reactions: a ligation reaction by SplintR ligase and subsequent transcription by T7 RNA polymerase. The resulting transcript forms an RNA aptamer that induces fluorescence. Here, we demonstrate that SENSR is an effective and highly sensitive method for the detection of the current epidemic pathogen, <jats:italic>severe acute respiratory syndrome-related coronavirus 2</jats:italic> (SARS-CoV-2). We also show that the platform can be extended to the detection of five other pathogens. Overall, SENSR is a molecular diagnostic method that can be developed rapidly for onsite uses requiring high sensitivity, specificity, and short assaying times.</jats:p></p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Real-time dissemination of scientific information to fight a public health emergency of international concern</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rapidly sharing scientific information is an effective way to reduce public panic about COVID-19, and doing so is the key to providing real-time guidance to epidemiologists working to contain the outbreak, clinicians managing patients, and modelers helping to understand future developments and the possible effectiveness of various interventions. This issue has rapidly reviewed and published articles describing COVID-19, including the drug treatment options for SARS-CoV-2, its clinical characteristics, and therapies involving a combination of Chinese and Western medicine, the efficacy of chloroquine phosphate in the treatment of COVID-19 associated pneumonia according to clinical studies, and reflections on the system of reserve medical supplies for public health emergencies. As an academic journal, we will continue to quickly and transparently share data with frontline healthcare workers who need to know the epidemiological and clinical features of COVID-19.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>What we know so far: COVID-19 current clinical knowledge and research</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, health authorities in Wuhan, China, identified a cluster of pneumonia cases of unknown aetiology linked to the city's South China Seafood Market. Subsequent investigations revealed a novel coronavirus, SARS-CoV-2, as the causative agent now at the heart of a major outbreak. The rising case numbers have been accompanied by unprecedented public health action, including the wholesale isolation of Wuhan. Alongside this has been a robust scientific response, including early publication of the pathogen genome, and rapid development of highly specific diagnostics. This article will review the new knowledge of SARS-CoV-2 COVID-19 acute respiratory disease, and summarise its clinical features.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>The United States badly bungled coronavirus testing—but things may soon improve | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? The World Health Organization (WHO) has shipped testing kits to 57 countries. China had five commercial tests on the market 1 month ago and can now do up to 1.6 million tests a week; South Korea has tested 65,000 people so far. The U. S. Centers for Disease Control and Prevention (CDC), in contrast, has done only 459 tests since the epidemic began. The rollout of a CDC-designed test kit to state and local labs has become a fiasco because it contained a faulty reagent. Labs around the country eager to test more suspected cases—and test them faster—have been unable to do so. No commercial or state labs have the approval to use their own tests. In what is already an infamous snafu, CDC initially refused a request to test a patient in Northern California who turned out to be the first probable COVID19 case without known links to an infected person.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 in the Shadows of MERS-CoV in the Kingdom of Saudi Arabia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) has plagued the Middle East since it was first reported in 2012. Recently, at the end of December 2019, a cluster of pneumonia cases were reported from Wuhan city, Hubei Province, China, linked to a wet seafood market with a new coronavirus identified as the etiologic agent currently named SARS-CoV-2. Most cases are in Mainland China with international spread to 25 countries. The novelty of the virus, the rapid national and international spread, and the lack of therapeutic and preventative strategies have led the WHO International Health Regulation emergency committee to declare the disease as Public Health Emergency of International Concern (PHEIC) on January 30, 2020. As it relates to countries with the ongoing MERS-CoV community cases and hospital acquired infections, there will be a huge challenge for HCWs to deal with both coronaviruses, especially with the lack of standardized and approved point of care testing. This challenge will now be faced by the whole global health community dealing with COVID-19 since both coronaviruses have similar presentation. Those patients should now be tested for both MERS-CoV and SARS-CoV-2 simultaneously, and with the continuing wide international spread of SARS-CoV-2, the travel history to China in the last 14 days will be of less significance</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">hina continues to fight the outbreak of a novel coronavirus, called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which has infected tens of thousands and killed more than 2,100 people since the end of 2019. The outbreak’s hot spot is in the central province of Hubei, in particular, its capital city Wuhan, where there have been more than 62,000 infections and 2,029 deaths as of last week, according to the World Health Organization (WHO). There, doctors and scientists are desperately struggling to find reliable ways to diagnose infected patients to help treat them and control the spread of the virus. The struggle has involved two diagnostic options—computed tomography (CT) scans of patients’ lungs and a nucleic acid lab test—each with advantages and disadvantages. With “a suddenly rampant new virus, it is normal to have multiple test approaches,” says a virologist at the Chinese Center for Disease Control and Prevention</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the scale of COVID-19 Epidemic in the United States: Simulations Based on Air Traffic directly from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Introduction: Coronavirus Disease 2019 (COVID-19) infection has been characterized by rapid spread and unusually large case clusters. It is important to have an estimate of the current state of COVID-19 epidemic in the U.S. to develop informed public health strategies.  Methods:  We estimated the potential scale of the COVID-19 epidemic (as of 03/01/2020) in the U.S. from cases imported directly from Wuhan area. We used simulations based on transmission dynamics parameters estimated from previous studies and air traffic data from Wuhan to the U.S and deliberately built our model based on conservative assumptions. Detection and quarantine of individual COVID-19 cases in the U.S before 03/01/2020 were also taken into account. We. A SEIR model was used to simulate the growth of the number of infected individuals in Wuhan area and in the U.S.  Results: With the most likely model, we estimated that there would be 9,484 infected cases (90%CI 2,054-24,241) as of 03/01/2020 if no intervention procedure had been taken to reduce the transmissibility in unidentified cases. Assuming current preventive procedures have reduced 25% of the transmissibility in unidentified cases, the number of infected cases would be 1,043 (90%CI 107-2,474). Conclusion: Our research indicates that, as of 03/01/2020., it is likely that there are already thousands of individuals in the US infected with SARS-CoV-2. Our model is dynamic and is available to the research community to further evaluate as the  situation becomes clearer.</jats:p></p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>The response of Milan's Emergency Medical System to the COVID-19 outbreak in Italy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The number of people infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus causing coronavirus disease 2019 (COVID-19), is dramatically increasing worldwide.The first person-to-person transmission in Italy was reported on Feb 21, 2020, and led to an infection chain that represents the largest COVID-19 outbreak outside Asia to date. Here we document the response of the Emergency Medical System (EMS) of the metropolitan area of Milan, Italy, to the COVID-19 outbreak.On Jan 30, 2020, WHO declared the COVID-19 outbreak a public health emergency of international concern.2 Since then, the Italian Government has implemented extraordinary measures to restrict viral spread, including interruptions of air traffic from China, organised repatriation flights and quarantines for Italian travellers in China, and strict controls at international airports' arrival terminals. Local medical authorities adopted specific WHO recommendations to identify and isolate suspected cases of COVID-19.Such recommendations were addressed to patients presenting with respiratory symptoms and who had travelled to an endemic area in the previous 14 days or who had worked in the health-care sector, having been in close contact with patients with severe respiratory disease with unknown aetiology. Suspected cases were transferred to preselected hospital facilities where the SARS-CoV-2 test was available and infectious disease units were ready for isolation of confirmed cases.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19—New Insights on a Rapidly Changing Epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since first reported in Wuhan, China, in late December 2019, the outbreak of the novel coronavirus now known as SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) has spread globally. As of February 27, 2020, more than 82 000 cases of coronavirus disease 2019 (COVID-19) (the disease caused by SARS-CoV-2) and 2800 deaths have been reported, of which approximately 95% of cases and 97% of deaths are in China. Cases have now been reported in 49 other countries. A particularly large outbreak occurred among the passengers and crew of the Diamond Princess cruise ship, where more than 700 infections are reported.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Outbreak investigation for COVID-19 in northern Vietnam</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two Vietnamese adults returned to their home province of Vinh Phuc in northern Vietnam on Jan 17, 2020, from Wuhan, China, where they had been living since Nov 15, 2019, for a business trip. They presented with mild respiratory symptoms to their local health facilities at 4 days and 8 days, respectively, after arrival in Vinh Phuc. Both individuals were initially placed into respiratory isolation in hospital. Case 1 tested positive for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative organism of coronavirus disease 2019 (COVID-19), on Jan 30, 2020, and remained in isolation until recovery. Case 2 was discharged from isolation in hospital after having one negative test result on Jan 28 (11 days after returning from Wuhan). Following discharge, the patient attended a family social function. 2 days later, she was readmitted after a second nasal swab for SARS-CoV-2 taken during her time in hospital was reported as positive. Screening of 79 individuals who had been in contact with these two patients (namely, family members in the same household and anyone who had been within 2 m of them) was initiated on Jan 31. Six individuals from the same work team, who had also travelled from Wuhan on Jan 17, were isolated, and four of them tested positive for SARS-CoV-2 (cases 3, 4, and 8 in Vinh Phuc, and one case from another province). Five secondary cases were diagnosed within the social network of case 2. These included three household members (cases 6, 7, and 11) and two people who had attended the social function (cases 5 and 9; appendix p 1). Four of these individuals reported mild respiratory symptoms; the remaining patient was asymptomatic (case 7) at the time of diagnosis.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Labs scramble to produce new coronavirus diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The seeming precision of the global tallies of cases and deaths caused by the novel coronavirus now spreading from Wuhan, China, belies an alarming fact. The world is in the dark about the epidemic's real scale and speed, because existing tests have limited powers—and testing is far too spotty. “We are underestimating how common this infection is,” cautions Jeremy Farrar, head of the Wellcome Trust. Within days of Chinese researchers releasing the sequence of the virus on 11 January, scientists developed tests capable of detecting genetic sequences that distinguish the new agent from other coronaviruses circulating in humans. By 28 January, China's National Medical Products Administration had approved diagnostic test kits from five companies. It was an astonishing pace for the response to a pathogen never seen before—and yet it was only a beginning</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>The outbreak of SARS-CoV-2 pneumonia calls for viral vaccines</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The outbreak of 2019-novel coronavirus disease (COVID-19) that is caused by SARS-CoV-2 has spread rapidly in China, and has developed to be a Public Health Emergency of International Concern. However, no specific antiviral treatments or vaccines are available yet. This work aims to share strategies and candidate antigens to develop safe and effective vaccines against SARS-CoV-2. An outbreak of 2019-novel coronavirus (SARS-CoV-2) that causes atypical pneumonia (COVID-19) has raged in China since mid-December 2019 and has spread to 26 countries (February 20, 2020). The epidemic was identified by the first four cases confirmed on December 29, 2019 and was traced to the Huanan Seafood Wholesale Market, Wuhan city, Hubei Province, China1. A total of 75,465 cases with SARS-CoV-2 infections have been confirmed up to date (February 20, 2020), and 2,236 people have died in China2. COVID-19 spreads rapidly by human-to-human transmission with a median incubation period of 3.0 days (range, 0 to 24.0), and the time from symptom onset to developing pneumonia is 4.0 days (range, 2.0 to 7.0)3. Respiratory droplets and direct contact are conventional transmission routes for SARS-CoV-2, and fecal-to-oral transmission might also have a role3. Fever, dry cough, and fatigue are common symptoms at onset of COVID-194. Most patients have lymphopenia and bilateral ground-glass opacity changes on chest CT scans4,5. No specific antiviral treatments or vaccines are available because it is a new emerging viral disease. Development of SARS-CoV-2-based vaccines is urgently required. The entire virus particle-based preparation of vaccines, including inactivated and attenuated virus vaccines is advisable, because it is based on previous studies about the prevention and control of seasonal influenza vaccines6. The first SARS-CoV-2 (Wuhan-Hu-1) was successfully sequenced and its genomic sequence submitted to GenBank on January 5, 2020 (Accession no. MN908947.3)7. Subsequently large-scale culture of SARS-CoV-2 was quickly performed, and an inactivated virus vaccine could be prepared through the employment of established physical and chemical methods such as UV light, formaldehyde, and β-propiolactone8. The development of attenuated-virus vaccines is also possible by carefully screening the serially propagated SARS-CoV-2 with reduced pathogenesis such as induced minimal lung injury, diminished limited neutrophil influx, and increased anti-inflammatory cytokine expressions compared with the wild-type virus9. Both inactivated and attenuated virus vaccines have their own disadvantages and side effects (Table 1). Alternatively, new vaccine designs based on the putative protective antigen/peptides derived from SARS-CoV-2 should be considered</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Update: Public Health Response to the Coronavirus Disease 2019 Outbreak - United States, February 24, 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of coronavirus disease 2019 (COVID-19) caused by the 2019 novel coronavirus (SARS-CoV-2) began in Wuhan, Hubei Province, China in December 2019, and has spread throughout China and to 31 other countries and territories, including the United States (1). As of February 23, 2020, there were 76,936 reported cases in mainland China and 1,875 cases in locations outside mainland China (1). There have been 2,462 associated deaths worldwide; no deaths have been reported in the United States. Fourteen cases have been diagnosed in the United States, and an additional 39 cases have occurred among repatriated persons from high-risk settings, for a current total of 53 cases within the United States. This report summarizes the aggressive measures (2,3) that CDC, state and local health departments, multiple other federal agencies, and other partners are implementing to slow and try to contain transmission of COVID-19 in the United States. These measures require the identification of cases and contacts of persons with COVID-19 in the United States and the recommended assessment, monitoring, and care of travelers arriving from areas with substantial COVID-19 transmission. Although these measures might not prevent widespread transmission of the virus in the United States, they are being implemented to 1) slow the spread of illness; 2) provide time to better prepare state and local health departments, health care systems, businesses, educational organizations, and the general public in the event that widespread transmission occurs; and 3) better characterize COVID-19 to guide public health recommendations and the development and deployment of medical countermeasures, including diagnostics, therapeutics, and vaccines. U.S. public health authorities are monitoring the situation closely, and CDC is coordinating efforts with the World Health Organization (WHO) and other global partners. Interim guidance is available at https://www.cdc.gov/coronavirus/index.html. As more is learned about this novel virus and this outbreak, CDC will rapidly incorporate new knowledge into guidance for action by CDC, state and local health departments, health care providers, and communities.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Has China faced only a herald wave of SARS-CoV-2?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The attack rate of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) calculated by mathematical models, from estimates of the basic reproduction number, R0, of 2–3, suggests that 50–60% of the population should eventually be infected because the population seems to be entirely naive to the new virus.1 The observed attack rate on board the Diamond Princess cruise ship remained slightly below 20% (705 of 3711 passengers and crew members became infected).1 It is of upmost importance to know whether the SARS-CoV-2 outbreak in China is subsiding, as local authorities and the entire international community might wish. With 80 026 COVID-19 cases officially reported from China as of March 2, 2020,2 the proportion of the population affected remains far from 50%, or even 20%, of China's 1·4 billion people. Has China just experienced a herald wave, to use terminology borrowed from those who study tsunamis, and is the big wave still to come? Serosurveys can help answer these questions precisely.3 To serosurvey the outbreak would involve testing sera of blood samples from the most representative sample of the population at the epicentre of the epidemic, Wuhan. Serology analysis with neutralising antibodies from the 1000 people could allow for the rate of SARS-CoV-2 infections to be estimated with good accuracy. This rate could be extrapolated to the city's entire population and thus inform more precisely whether the provisional attack rate during this period was a few cases per thousand or perhaps affected 1–2% of the population, 20%, or more. Serosurveys should be seen as polls before elections; they can be repeated several times,3 week after week, to monitor the epidemic precisely. There is no reason to wait for the end of the epidemic before doing serosurveys. The results would be tremendously informative to China, first and foremost, and to the entire international community, on the risk of big secondary epidemic waves</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Sanofi, BARDA team for coronavirus vaccine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Sanofi Pasteur is working with the US Biomedical Advanced Research and Development Authority (BARDA) to develop a vaccine against the novel coronavirus, known as severe acute respiratory syndrome-CoV-2, which as of Feb. 20 had infected more than 75,000 people and killed more than 2,100. Sanofi will build on work on a vaccine against SARS, another coronavirus that circulated in the early 2000s. In December 2019, BARDA awarded the firm a $226 million contract to expand manufacturing capacity for use in a potential flu pandemic. Sanofi says the new vaccine will use the same technology.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosis of Acute Respiratory Syndrome Coronavirus 2 Infection by Detection of Nucleocapsid Protein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND Nucleic acid test and antibody assay have been employed in the diagnosis for SARS-CoV-2 infection, but the use of viral antigen for diagnosis has not been successfully developed.  Theoretically, viral antigen is the specific marker of the virus and precedes antibody appearance within the infected population. There is a clear need of detection of viral antigen for rapid and early diagnosis. METHODS We included a cohort of 239 participants with suspected SARS-CoV-2 infection from 7 centers for the study. We measured nucleocapsid protein in nasopharyngeal swab samples in parallel with the nucleic acid test. Nucleic acid test was taken as the reference standard, and statistical evaluation was taken in blind. We detected nucleocapsid protein in 20 urine samples in another center, employing nasopharyngeal swab nucleic acid test as reference standard. RESULTS We developed a fluorescence immunochromatographic assay for detecting nucleocapsid protein of SARS-CoV-2 in nasopharyngeal swab sample and urine within 10 minutes.  100% of nucleocapsid protein positive and negative participants accord with nucleic acid test for same samples. Further, earliest participant after 3 days of fever can be identified by the method.  In an additional preliminary study, we detected nucleocapsid protein in urine in 73.6% of diagnosed COVID-19 patients.  CONCLUSIONS Those findings indicate that nucleocapsid protein assay is an accurate, rapid, early and simple method for diagnosis of COVID-19. Appearance of nucleocapsid protein in urine coincides our finding of the SARS-CoV-2 invading kidney and might be of diagnostic value.</jats:p></p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>An outbreak of <scp>COVID</scp> ‐19 caused by a new coronavirus: what we know so far</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of a novel coronavirus, now formally named severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) and causing coronavirus disease 2019 (COVID‐19), emerged in the city of Wuhan in Hubei province in central China in December 2019. The first cases were noted as a cluster of patients with pneumonia who were all linked to a live animal market, and testing found the presence of a previously unknown coronavirus. Coronaviruses are a group of viruses that affect both animals and humans, and several (OC43, 229E, HKU1 and NL63) are a cause of the common cold.1, 2 However, two coronaviruses have previously caused significant outbreaks associated with more severe disease: the SARS coronavirus in 2002–2003 and the Middle East respiratory syndrome coronavirus that emerged in 2012.1, 2 Chinese authorities and researchers should be commended for their rapid sharing of viral sequences which enabled laboratories worldwide to develop diagnostic tests within weeks of discovery of the pathogen.3 An Australian laboratory subsequently isolated the virus from a clinical sample (the first to do so outside of China), and rapidly shared this virus with relevant global agencies, further aiding diagnostic, therapeutic and vaccine development efforts. Information on the new virus and its impact is being updated constantly. We know that SARS‐CoV‐2 can cause severe disease, although active surveillance of contacts is required to define the milder end of the disease spectrum and to estimate the true hospitalisation and case fatality ratio. The cases reported to date suggest that most are older adults; it is currently unclear whether comorbidities reflect the age group affected or whether they are risk factors for severe disease.4, 5 Early studies using data before the institution of public health interventions in China suggest that SARS‐CoV‐2 is as transmissible as SARS coronavirus and probably more transmissible than influenza viruses.6, 7 The timing of infectiousness relative to symptom onset is a particularly important parameter with implications for public health control. While reports suggest that asymptomatic infection and transmission may result from minimally symptomatic cases, the contribution of this to transmission is not yet known.8 Careful analysis of early data suggests that the mean incubation period is 6 days, with a range of up to 14 days.9 Reports of large outbreaks, particularly associated with hospitals and closed communities, raise the possibility of “superspreading” events, a feature of previous coronavirus outbreaks.10 The importance of infection control is also reinforced by a report that 41% of cases in Wuhan were acquired nosocomially (including 40 health care workers and 17 patients).4 As at 26 February 2020, there were 23 confirmed cases of COVID‐19 in Australia, across five jurisdictions. Although reported case numbers in China are slowing, large outbreaks on a cruise ship in Japan, and in South Korea, Iran and Italy are concerning and highlight our interconnected world. However, it is likely that this will change in the days or weeks ahead. Although absolute case numbers are small in Australia, the public health, political and societal ramifications have already been considerable, ranging from travel restrictions on non‐Australians coming from China, the use of offshore and remote quarantine facilities, and disturbing reports of racism against members of our Asian community (https://insig​htplus.mja.com.au/2020/5/coron​avirus-no-place-for-racism-xenop​hobia/).</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Labs scramble to spot hidden coronavirus infections | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The seeming precision of the global tallies of cases and deaths caused by the novel coronavirus now spreading from Wuhan, China belies an alarming fact. The world is in the dark about the epidemic’s real scale and speed, because existing tests have limited powers—and testing is far too spotty. “We are underestimating how common this infection is,” cautions Jeremy Farrar, head of the Wellcome Trust. Within days of Chinese researchers releasing the sequence of the virus on 11 January, scientists developed tests capable of detecting genetic sequences that distinguish the new agent from other coronaviruses circulating in humans. By 28 January, China’s National Medical Products Administration had approved diagnostic test kits from five companies. It was an astonishing pace for the response to a pathogen never seen before—and yet it was only a beginning.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease. AVAILABILITY: https://www.genomedetective.com/app/typingtool/cov. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country.  Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival &amp; traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% &amp; 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using 'Feidian', Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.</jats:p></p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenges to the system of reserve medical supplies for public health emergencies: reflections on the outbreak of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On December 31, 2019, the Wuhan Municipal Health Commission announced an outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), China is now at a critical period in the control of the epidemic. The Chinese Government has been taking a series of rapid, comprehensive, and effective prevention and control measures. As the pandemic has developed, a fact has become apparent: there is a serious dearth of emergency medical supplies, and especially an extreme shortage of personal protective equipment such as masks and medical protective clothing. This is one of the major factors affecting the progress of epidemic prevention and control. Although China has made great efforts to strengthen the ability to quickly respond to public health emergencies since the SARS outbreak in 2003 and it has clarified requirements for emergency supplies through legislation, the emergency reserve supplies program has not been effectively implemented, and there are also deficiencies in the types, quantity, and availability of emergency medical supplies. A sound system of emergency reserve supplies is crucial to the management of public health emergencies. Based on international experiences with pandemic control, the world should emphasize improving the system of emergency reserve medical supplies in the process of establishing and improving public health emergency response systems, and it should promote the establishment of international cooperative programs to jointly deal with public health emergencies of international concern in the future.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 23</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. AIM: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. METHODS: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. RESULTS: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive – Global (EVAg), a European Union infrastructure project. CONCLUSION: The present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Jan 23</span></i><p style="text-align: justify; text-justify: inter-word;">BACKGROUND: The ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. AIM: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. METHODS: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. RESULTS: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive – Global (EVAg), a European Union infrastructure project. CONCLUSION: The present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>The deadly coronaviruses: The 2003 SARS pandemic and the 2020 novel coronavirus epidemic in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The 2019-nCoV is officially called SARS-CoV-2 and is the cause of the disease named COVID-19. This viral epidemic in China has led to the deaths of over 1800 people, mostly elderly or those with an underlying chronic disease or immunosuppressed state. This is the third serious Coronavirus outbreak in less than 20 years, following SARS in 2002–2003 and MERS in 2012. While human strains of Coronavirus are associated with about 15% of cases of the common cold, the SARS-CoV-2 may present with varying degrees of severity, from flu-like symptoms to death. It is currently believed that this deadly Coronavirus strain originated from wild animals at the Huanan market in Wuhan, a city in Hubei province. Bats, snakes and pangolins have been cited as potential carriers based on the sequence homology of CoV isolated from these animals and the viral nucleic acids of the virus isolated from SARS-CoV-2 infected patients. Extreme quarantine measures, including sealing off large cities, closing borders and confining people to their homes, were instituted in January 2020 to prevent spread of the virus, but by that time much of the damage had been done, as human-human transmission became evident. While these quarantine measures are necessary and have prevented a historical disaster along the lines of the Spanish flu, earlier recognition and earlier implementation of quarantine measures may have been even more effective. Lessons learned from SARS resulted in faster determination of the nucleic acid sequence and a more robust quarantine strategy. However, it is clear that finding an effective antiviral and developing a vaccine are still significant challenges. The costs of the epidemic are not limited to medical aspects, as the virus has led to significant sociological, psychological and economic effects globally. Unfortunately, emergence of SARS-CoV-2 has led to numerous reports of Asians being subjected to racist behavior and hate crimes across the world.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Economic Impacts of Wuhan 2019-nCoV on China and the World</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Uncertainties over the Wuhan 2019 Novel Coronavirus (2019-nCoV), which has killed 1,017 people and sickened more than 43,100 as of Feb 11,(1) has interrupted global trade and supply chains, depressed asset prices, and forced multinational businesses to make hard decisions with limited information. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Active Monitoring of Persons Exposed to Patients with Confirmed COVID-19 — United States, January–February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, an outbreak of coronavirus disease 2019 (COVID-19), caused by the virus SARS-CoV-2, began in Wuhan, China (1). The disease spread widely in China, and, as of February 26, 2020, COVID-19 cases had been identified in 36 other countries and territories, including the United States. Person-to-person transmission has been widely documented, and a limited number of countries have reported sustained person-to-person spread.* On January 20, state and local health departments in the United States, in collaboration with teams deployed from CDC, began identifying and monitoring all persons considered to have had close contact† with patients with confirmed COVID-19 (2). The aims of these efforts were to ensure rapid evaluation and care of patients, limit further transmission, and better understand risk factors for transmission.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19, Australia: Epidemiology Report 3 (Reporting week ending 19:00 AEDT 15 February 2020)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">This is the third epidemiological report for coronavirus disease 2019 (COVID-19), previously known as novel coronavirus (2019-nCoV), from the virus now known as SARS-CoV-2, reported in Australia as at 19:00 Australian Eastern Daylight Time [AEDT] 15 February 2020. It includes data on the COVID-19 Australian cases, the international situation and current information on the severity, transmission and spread.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Update on Influenza Diagnostics: Lessons from the Novel H1N1 Influenza A Pandemic</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Summary: The menu of diagnostic tools that can be utilized to establish a diagnosis of influenza is extensive and includes classic virology techniques as well as new and emerging methods. This review of how the various existing diagnostic methods have been utilized, first in the context of a rapidly evolving outbreak of novel influenza virus and then during the different subsequent phases and waves of the pandemic, demonstrates the unique roles, advantages, and limitations of each of these methods. Rapid antigen tests were used extensively throughout the pandemic. Recognition of the low negative predictive values of these tests is important. Private laboratories with preexisting expertise, infrastructure, and resources for rapid development, validation, and implementation of laboratory-developed assays played an unprecedented role in helping to meet the diagnostic demands during the pandemic. FDA-cleared assays remain an important element of the diagnostic armamentarium during a pandemic, and a process must be developed with the FDA to allow manufacturers to modify these assays for detection of novel strains in a timely fashion. The need and role for subtyping of influenza viruses and antiviral susceptibility testing will likely depend on qualitative (circulating subtypes and their resistance patterns) and quantitative (relative prevalence) characterization of influenza viruses circulating during future epidemics and pandemics.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Top stories: Bungled coronavirus testing, disarming ‘atomic bomb’ cells, and jet stream blocking</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? One answer is that a test kit designed by the U.S. Centers for Disease Control and Prevention (CDC) and rolled out to state and local labs late last month contained a faulty reagent. The problems have led many to doubt the accuracy of confirmed tallies of the disease. But the situation may soon improve, as the Food and Drug Administration comes up with a workaround for the faulty CDC kit.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the cross-species evolution of 2019 novel coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Recent study reported in this journal that the threats of continuous evolution and dissemination of 2019 human coronaviruses.1 Since its emergence in December 2019, a “seventh” member of the family of 2019 human coronavirus named “SARS-CoV-2” was responsible for an outbreak of coronavirus disease (COVID-19) in Wuhan, China.2 As of February 23, 2020, China had reported more than 77,042 confirmed cases of SARS-CoV-2, with 2,445 fatalities and counting (http://www.nhc.gov.cn). Strikingly, SARS-CoV-2 had been transmitted rapidly in more than 29 countries to date (https://www.who.int), including Asia, Europe, North America, Africa, and Oceania, posing serious concerns about its pandemic potential. Despite of droplet and contact transmissions of SARS-CoV-2, recent studies demonstrated that SARS-CoV-2 might be transmitted via aerosol and fecal–oral routes 3 (Figure 1), which needs to be paid attention in particular.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">BackgroundThe ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur.AimWe aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available.MethodsHere we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology.ResultsThe workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is made available through European Virus Archive - Global (EVAg), a European Union infrastructure project.ConclusionThe present study demonstrates the enormous response capacity achieved through coordination of academic and public laboratories in national and European research networks.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Isolation from the First Patient with SARS-CoV-2 in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 18</span></i><p style="text-align: justify; text-justify: inter-word;">Novel coronavirus (SARS-CoV-2) is found to cause a large outbreak started from Wuhan since December 2019 in China and SARS-CoV-2 infections have been reported with epidemiological linkage to China in 25 countries until now. We isolated SARS-CoV-2 from the oropharyngeal sample obtained from the patient with the first laboratory-confirmed SARS-CoV-2 infection in Korea. Cytopathic effects of SARS-CoV-2 in the Vero cell cultures were confluent 3 days after the first blind passage of the sample. Coronavirus was confirmed with spherical particle having a fringe reminiscent of crown on transmission electron microscopy. Phylogenetic analyses of whole genome sequences showed that it clustered with other SARS-CoV-2 reported from Wuhan.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>2019 novel coronavirus disease (COVID-19) in Taiwan: Reports of two cases from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We reported two cases with community-acquired pneumonia caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) who returned from Wuhan, China in January, 2020. The reported cases highlight non-specific clinical presentations of 2019 novel coronavirus disease (COVID-19) as well as the importance of rapid laboratory-based diagnosis.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis on the epidemic factors for the Corona Virus Disease</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, corona virus disease 2019 (COVID-19) , an emerging infection disease occurred in Wuhan, has spread in the mainland China. The epidemic factors on the basis of knowledge of SARS-CoV-2 were discussed in this paper. This puts a lot of pressure on clinical resources and care. SARS-CoV-2 is a novel corona virus, the onset of COVID-19 is slow, and the pathogenesis of SARS-CoV-2 remains unclear and may lead to multiple organ damage. These put a lot of pressure on clinical resources and care. Source of infection including the patients, asymptomatic carrier and patients in the incubation period are contagious. It is difficult to control source of infection. Routes of SARS-CoV-2 transmission are diversified and the main routes of transmission for COVID-19 are droplet transmission and close contact transmission. All population have susceptibility to SARS-CoV-2. Social factors such population movements and aggregation accelerated the spread of SARS-CoV-2. The Chinese government's adopted measures are positive and effective, and are accepted by the expert group from the World Health Organization. However, it will be a long-term hard work in the future to seriously summarize and think deeply to achieve public health security in China.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus latest: Scientists clash over virus name</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Some researchers in China are unhappy with the designated name for the new coronavirus, SARS-CoV-2. They worry that the use of ‘SARS-CoV’ will confuse the public and impede efforts to control the pathogen’s spread.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background The outbreak of the recently emerged novel corona virus disease 2019 (COVID-19) poses a challenge for public health laboratories. We aimed to evaluate the diagnostic value of serological assay for SARS-CoV-2.  Methods A newly-developed ELISA assay for IgM and IgG antibodies against N protein of SARS-CoV-2 were used to screen the serums of 238 admitted hospital patients with confirmed or suspected SARS-CoV-2 infection from February 6 to February 14, 2020. SARS-CoV-2 RNA was detected by real time RT-PCR on pharyngeal swab specimens. Findings Of the 238 patients, 194 (81.5%) were detected to be antibody (IgM and/or IgG) positive, which was significantly higher than the positive rate of viral RNA (64.3%). There was no difference in the positive rate of antibody between the confirmed patients (83.0%, 127/153) and the suspected patients (78.8%, 67/85) whose nucleic acid tests were negative. After the patients were defined to the different stages of disease based on the day when the test samples were collected, the analysis results showed that the antibody positive rates were very low in the first five days after initial onset of symptoms, and then rapidly increased as the disease progressed. After 10 days, the antibody positive rates jumped to above 80% from less than 50%. On the contrary, the positive rates of viral RNA kept above 60% in the first 11 days after initial onset of symptoms, and then rapidly decreased. In addition, half of the suspected patients with symptoms for 6-10 days were detected to be antibody positive. Interpretation The suspected patients were most likely infected by SARS-CoV-2. Before the 11th day after initial onset of symptoms, nucleic acid test is important for confirmation of viral infection. The combination of serological assay can greatly improve the diagnostic efficacy. After that, the diagnosis for viral infection should be majorly dependent on serological assay. Keywords. SARS-CoV-2; diagnosis; serological assay; nucleic acid test</jats:p></p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-05</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Virus Isolation from the First Patient with SARS-CoV-2 in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Novel coronavirus (SARS-CoV-2) is found to cause a large outbreak started from Wuhan since December 2019 in China and SARS-CoV-2 infections have been reported with epidemiological linkage to China in 25 countries until now. We isolated SARS-CoV-2 from the oropharyngeal sample obtained from the patient with the first laboratory-confirmed SARS- CoV-2 infection in Korea. Cytopathic effects of SARS-CoV-2 in the Vero cell cultures were confluent 3 days after the first blind passage of the sample. Coronavirus was confirmed with spherical particle having a fringe reminiscent of crown on transmission electron microscopy.Phylogenetic analyses of whole genome sequences showed that it clustered with other SARS- CoV-2 reported from Wuhan.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: combining antiviral and anti-inflammatory treatments</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Both coronavirus disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS) are characterised by an overexuberant inflammatory response and, for SARS, viral load is not correlated with the worsening of symptoms. In our previous Correspondence to The Lancet, we described how BenevolentAI's proprietary artificial intelligence (AI)-derived knowledge graph, queried by a suite of algorithms, enabled identification of a target and a potential therapeutic against SARS coronavirus 2 (SARS-CoV-2; the causative organism in COVID-19). We identified a group of approved drugs that could inhibit clathrin-mediated endocytosis and thereby inhibit viral infection of cells (appendix). The drug targets are members of the numb-associated kinase (NAK) family—including AAK1 and GAK—the inhibition of which has been shown to reduce viral infection in vitro. Baricitinib was identified as a NAK inhibitor, with a particularly high affinity for AAK1, a pivotal regulator of clathrin-mediated endocytosis. We suggested that this drug could be of use in countering SARS-CoV-2 infections, subject to appropriate clinical testing.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Emergence of a novel human coronavirus threatening human health</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In late December 2019, a cluster of patients with ‘atypical pneumonia’ of unknown etiology was reported in Wuhan, China. A novel human coronavirus, now provisionally called ‘SARS-CoV-2’, was identified as the cause of this disease, now named ‘COVID-19’.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Innovation and challenges in funding rapid research responses to emerging infectious diseases: Lessons learned from the outbreak of severe acute respiratory syndrome</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Although the local public health response to the severe acute respiratory syndrome outbreak in Canada was critical to the diagnosis, management and treatment of patients, such a rapid research response required a national effort to engage the research and stakeholder communities. The Canadian research effort, coordinated through the Institute of Infection and Immunity of the Canadian Institutes of Health Research and the Michael Smith Foundation for Health Research, has provided insight into the mechanisms required to ensure the rapid development of strategical initiatives in response to emerging infectious diseases. It has also provided a rational basis to set up a national network to be engaged if needed in the future.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Does SARS-CoV-2 has a longer incubation period than SARS and MERS?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of a novel coronavirus (SARS-CoV-2) since Dec 2019 in Wuhan, the major transportation hub in central China, became an emergency of major international concern. While several etiological studies have begun to reveal the specific biological features of this virus, the epidemic characteristics need to be elucidated. Notably, a long incubation time was reported to be associated with SARS-CoV-2 infection, leading to adjustments in screening and control policies. To avoid the risk of virus spread, all potentially exposed subjects are required to be isolated for 14 days, which is the longest predicted incubation time. However, based on our analysis of a larger dataset available so far, we find there is no observable difference between the incubation time for SARS-CoV-2, SARS and MERS, highlighting the need for larger and well annotated datasets. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and corona virus disease-2019 (COVID-19): the epidemic and the challenges</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">ABSTRACT Emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019, has caused a large global outbreak and a major public health issue. As of February 11, 2020, data from the WHO has shown that more than 43,000 confirmed cases have been identified in 28 countries/regions, with more than 99% of the cases being detected in China. On January 30, 2020, WHO has declared COVID-19 as the sixth public health emergency of international concern. The SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24-3.58. Among the patients with pneumonia caused by the SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground glass opacity was the most common finding from computerized tomography images of the chest. Although the one case of SARS-CoV-2 pneumonia in the United States responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of the SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Lessons for managing high-consequence infections from first COVID-19 cases in the UK</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">These first UK cases of COVID-19 raise important points about the management of cases of HCID in England. The decision to test for SARS-CoV-2 is based on a clinical and epidemiological case definition, and testing is only approved if this is met. When tested, neither of these people clearly met the current case definition, and had criteria been strictly applied, testing might not have been done. A decision to test was made because of high clinical suspicion and in response to latest available information about the distribution of infection. It is important that testing is appropriately targeted, and this is best done by applying clear case definitions. However, with any newly emerging infection, case definitions must evolve rapidly as information accrues. There should also be room for flexibility on the basis of discussion with clinical and public health experts</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Lessons from SARS and MERS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Within weeks of the first cases, a series of papers were released detailing the epidemiology of the disease (now termed COVID‐19) [1-3]. By early January 2020 the virus was identified and the sequence determined. The virus (termed SARS‐CoV‐2) shares 88% sequence identity to two coronaviruses found in bats, bat‐SLCoVZC45 and bat‐SL‐CoVZXC21, 79% identity with the Severe Acute Respiratory Syndrome (SARS) coronavirus and 50% identity with Middle Eastern Respiratory Syndrome (MERS) coronavirus [4]. From the first cohort of patients, 8 complete genomes were 99.9% identical in sequence. Given that the typical RNA coronavirus evolves at a rate of 104 nucleotide substitutions per year, this suggests a recent single source emergence in early December or late November 2019 [4]. SARS‐CoV‐2 is thought to be transmitted via contaminated hands, surfaces and aerosolised droplets; extensive human‐to‐human transmission is evident, with clusters of infected families and medical staff [5]. The number of confirmed cases has increased rapidly, at a rate that far outstripped the rate of rise of cases of SARS in 2002/3, raising serious global health concerns. By the 21st January, COVID‐19 cases were widespread across mainland China, soon spreading beyond the Chinese borders.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus puts drug chemical industry on alert</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Major drug companies have issued statements in recent days assuring the public that their inventories are adequate in the face of supply chain threats stemming from the novel coronavirus. Suppliers of active pharmaceutical ingredients (APIs) are also assuring customers that they are prepared for temporary interruption in the supply of key ingredients from firms in China, where the outbreak originated. API makers in Europe and the US warn, however, that supply disruptions could result from a protracted delay in restarting production at plants closed in recent weeks by the Chinese government or from prolonged transportation restrictions. James Bruno, president of the consulting firm Chemical and Pharmaceutical Solutions, says travel restrictions are already interrupting business with Chinese suppliers. “First of all, nobody is going to be able to get to China,” he says, “so all the audits are going to be canceled.” Bruno adds that the restrictions will prolong plant closures</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Study on SARS-COV-2 transmission and the effects of control measures in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.</jats:p></p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe SARS-CoV-2 infections: practical considerations and management strategy for intensivists</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Etiological agent and epidemiology The new agent causing this pneumonia, a coronavirus (SARS-CoV-2), was identified and sequenced [3] and diagnostic tests were developed [4]. On January 30, 2020, the World Health Organization issued a worldwide public health alert on the emergence of a new epidemic viral disease. On February 3, 2020, 17,391 confirmed cases (153 cases outside of China) have been reported (https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/). The overall mortality rate of affected patients is difficult to assess at this time, because of the lack of a reliable denominator. Severe forms represent 14% of the reported cases, and the overall mortality is around 2% of the confirmed cases. To date, 153 cases have been reported in 23 countries outside China (overall, 24 cases in Europe), most of them being imported cases: tourists coming from China, or China-originating persons returning to their country of residence after traveling to visit family in Wuhan or other Chinese regions. In Europe, at least three cases in Germany and one case in France involved patients with no history of travel to China. The German case occurred after exposure to an asymptomatic contact coming from China [5].AU - Lucet, Jean-Christophe</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Better Tests, Better Care: Improved Diagnostics for Infectious Diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In this IDSA policy paper, we review the current diagnostic landscape, including unmet needs and emerging technologies, and assess the challenges to the development and clinical integration of improved tests. To fulfill the promise of emerging diagnostics, IDSA presents recommendations that address a host of identified barriers. Achieving these goals will require the engagement and coordination of a number of stakeholders, including Congress, funding and regulatory bodies, public health agencies, the diagnostics industry, healthcare systems, professional societies, and individual clinicians.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>From SARS to COVID-19: A previously unknown SARS-CoV-2 virus of pandemic potential infecting humans – Call for a One Health approach</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Human coronaviruses continue to pose a threat to human health. The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in December 2019 which causes coronavirus disease-2019 (COVID-19), an acute respiratory disease marked the third introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. This recent ongoing emergence of a previously unknown coronavirus in China leads to huge impacts on humans globally. Here, we discuss the outbreak in a one health context, highlighting the need for the implementation of one health measures and practices to improve human health and reduce the emergence of pandemic viruses.</p>
<p align="right"><i>score: 37</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenges and countermeasures for organ donation during the SARS-CoV-2 epidemic: the experience of Sichuan Provincial People’s Hospital</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Up to 09:40 February 17, 2020, there have been 70,640 confirmed SARS-CoV-2 (aka 2019-nCoV) cases in China. Sichuan Provincial People’s Hospital acts as an organ transplant center in west of China, approximately 200 solid organ transplants performed each year and can perform heart, lung, liver, kidney, small bowel, stem cell transplantation, so it is necessary to establish a hospital-specific protocol to deal with the SARS-CoV-2 infection for the donor and recipient.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>The Novel Coronavirus Outbreak: What We Know and What We Don&#x2019;t</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Phylogenetic analyses reveal that SARS-CoV-2 is closely related to a group of SARS-like coronaviruses. However, it remains unclear where the virus comes from and how it was transmitted to humans in the first place. Unlike with other zoonotic agents such as hantavirus and arenavirus, thus far we haven’t found a SARS virus in animals that is the same as that in humans. Fortunately, SARS virus has not appeared in humans since 2004. In contrast, this new virus seems to have stronger transmission capabilities among people. Compared to the primary virus in humans, we still know less about whether, what, and how the virus has changed and the effect of the changes for their epidemics in humans. Control and prevention of the disease is especially difficult in China and elsewhere if there are infected individuals with no clinical signs.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;">Severe Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 - a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.</jats:p></p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>The transmission and diagnosis of 2019 novel coronavirus infection disease (COVID-19): A Chinese perspective</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">2019 novel coronavirus (SARS-CoV-2), which originated in Wuhan, China, has attracted the world's attention over the last month. The Chinese government has taken emergency measures to control the outbreak and has undertaken initial steps in the diagnosis and treatment of 2019 novel coronavirus infection disease (COVID-19). However, SARS-CoV-2 possesses powerful pathogenicity as well as transmissibility and still holds many mysteries that are yet to be solved, such as whether the virus can be transmitted by asymptomatic patients or by mothers to their infants. Our research presents selected available cases of COVID-19 in China to better understand the transmission and diagnosis regarding this infectious disease. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world. With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>New frontiers in applied veterinary point‐of‐capture diagnostics: Toward early detection and control of zoonotic influenza</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Nov 23</span></i><p style="text-align: justify; text-justify: inter-word;">Among the chief limitations in achieving early detection and control of animal‐origin influenza of pandemic potential in high‐risk livestock populations is the existing lag time between sample collection and diagnostic result. Advances in molecular diagnostics are permitting deployment of affordable, rapid, highly sensitive, and specific point‐of‐capture assays, providing opportunities for targeted surveillance driving containment strategies with potentially compelling returns on investment. Interrupting disease transmission at source holds promise of disrupting cycles of animal‐origin influenza incursion to endemicity and limiting impact on animal production, food security, and public health. Adoption of new point‐of‐capture diagnostics should be undertaken in the context of promoting robust veterinary services systems and parallel support for operationalizing pre‐authorized plans and communication strategies that will ensure that the full potential of these new platforms is realized.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has spread globally with sustained human-to-human transmission outside China.To report the initial experience in Singapore with the epidemiologic investigation of this outbreak, clinical features, and management.Descriptive case series of the first 18 patients diagnosed with polymerase chain reaction (PCR)–confirmed SARS-CoV-2 infection at 4 hospitals in Singapore from January 23 to February 3, 2020; final follow-up date was February 25, 2020.Confirmed SARS-CoV-2 infection.Clinical, laboratory, and radiologic data were collected, including PCR cycle threshold values from nasopharyngeal swabs and viral shedding in blood, urine, and stool. Clinical course was summarized, including requirement for supplemental oxygen and intensive care and use of empirical treatment with lopinavir-ritonavir.Among the 18 hospitalized patients with PCR-confirmed SARS-CoV-2 infection (median age, 47 years; 9 [50%] women), clinical presentation was an upper respiratory tract infection in 12 (67%), and viral shedding from the nasopharynx was prolonged for 7 days or longer among 15 (83%). Six individuals (33%) required supplemental oxygen; of these, 2 required intensive care. There were no deaths. Virus was detectable in the stool (4/8 [50%]) and blood (1/12 [8%]) by PCR but not in urine. Five individuals requiring supplemental oxygen were treated with lopinavir-ritonavir. For 3 of the 5 patients, fever resolved and supplemental oxygen requirement was reduced within 3 days, whereas 2 deteriorated with progressive respiratory failure. Four of the 5 patients treated with lopinavir-ritonavir developed nausea, vomiting, and/or diarrhea, and 3 developed abnormal liver function test results.Among the first 18 patients diagnosed with SARS-CoV-2 infection in Singapore, clinical presentation was frequently a mild respiratory tract infection. Some patients required supplemental oxygen and had variable clinical outcomes following treatment with an antiretroviral agent.</p>
<p align="right"><i>score: 36</i></p>

</div><br /><br /><br /><div class="card"><h3>ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control.  Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk.   Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 32 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days.   Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.</jats:p></p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>News at a glance</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Just weeks after a novel coronavirus emerged in Wuhan, China, the outbreak continued to expand in gravity and scope. This week, the cumulative death toll surpassed 1000, compared with the more than 800 killed by another coronavirus in the 2002–03 outbreaks of severe acute respiratory syndrome (SARS). Authorities assigned official names to the virus, SARS-CoV-2, and the resulting disease, COVID-19. Li Wenliang, a 34-year-old doctor who sounded an early alarm in Wuhan about the disease only to be summoned by local police for doing so, succumbed to the virus last week.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world.With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Journals, Academics, and Pandemics</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 25</span></i><p style="text-align: justify; text-justify: inter-word;">In the wake of the SARS epidemic and the H1N1 pandemic, the PLoS Medicine editors ask whether journal publishing is an efficient enough mechanism for information sharing.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The novel coronavirus (SARS-CoV-2) infections in China: prevention, control and challenges</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, an outbreak of novel coronavirus (SARS-CoV-2) that began from Wuhan, Hubei Province, has rapidly spread to 34 provincial-level divisions of China [1] and 25 countries around the world [2]. Up to February 15, 2020, 68,586 cases in China and 526 cases in other countries have been identified as having COVID-19 (eFigure 1). The estimated mortality rate is 3.1% in Wuhan, 2.8% in Hubei Province, 0.6% in other provinces of China, 2.4% in China and 0.4% in other countries, respectively. Faced with such a grim situation, the Chinese government has taken a series of unprecedented rigorous measures. First, all the 31 provincial-level divisions in China mainland have launched the highest level of responding mechanism for major public health emergency. Second, Wuhan and other 12 cities in Hubei Province have consecutively shut down all outbound transportation channels and suspended public transportation. Third, the State Council announced that both the Spring Festival holiday and winter vacation will be extended. Fourth, two emergency makeshift hospitals (Huoshenshan and Leishenshan hospitals) with a total capacity of 2600 beds were built and more than ten cabin hospitals were renovated with more than 10,000 beds in Wuhan. Fifth, more than 30,000 members from military and public hospitals have successively headed out to Wuhan and other cities in Hubei Province.ER -</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVES: Following reports of patients with unexplained pneumonia at the end of December 2019 in Wuhan, China, the causative agent was identified as coronavirus (SARS-CoV-2), and the 2019 novel coronavirus disease was named COVID-19 by the World Health Organization. Putative patients with COVID-19 have been identified in South Korea, and attempts have been made to isolate the pathogen from these patients. METHODS: Upper and lower respiratory tract secretion samples from putative patients with COVID-19 were inoculated onto cells to isolate the virus. Full genome sequencing and electron microscopy were used to identify the virus. RESULTS: The virus replicated in Vero cells and cytopathic effects were observed. Full genome sequencing showed that the virus genome exhibited sequence homology of more than 99.9% with SARS-CoV-2 which was isolated from patients from other countries, for instance China. Sequence homology of SARS-CoV-2 with SARS-CoV, and MERS-CoV was 77.5% and 50%, respectively. Coronavirus-specific morphology was observed by electron microscopy in virus-infected Vero cells. CONCLUSION: SARS-CoV-2 was isolated from putative patients with unexplained pneumonia and intermittent coughing and fever. The isolated virus was named BetaCoV/Korea/KCDC03/2020.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>The SARS-CoV-2 Vaccine Pipeline: an Overview</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19).</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Managing severe acute respiratory syndrome (SARS) intellectual property rights: the possible role of patent pooling.</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Patent applications that incorporate the genomic sequence of the severe acute respiratory syndrome (SARS) coronavirus, have been filed by a number of organizations. This is likely to result in a fragmentation of intellectual property (IP) rights which in turn may adversely affect the development of products, such as vaccines, to combat SARS. Placing these patent rights into a patent pool to be licensed on a non-exclusive basis may circumvent these difficulties and set a key precedent for the use of this form of mechanism in other areas of health care, leading to benefits to public health.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Update: Public Health Response to the Coronavirus Disease 2019 Outbreak — United States, February 24, 2020 | MMWR</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Fourteen cases have been diagnosed in the United States, in addition to 39 cases among repatriated persons from high-risk settings, for a current total of 53 cases within the United States. U.S. government agencies and public health partners are implementing aggressive measures to slow and try to contain transmission of COVID-19 in the United States.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovering drugs to treat coronavirus disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 virus emerged in December 2019 and then spread rapidly worldwide, particularly to China, Japan, and South Korea. Scientists are endeavoring to find antivirals specific to the virus. Several drugs such as chloroquine, arbidol, remdesivir, and favipiravir are currently undergoing clinical studies to test their efficacy and safety in the treatment of coronavirus disease 2019 (COVID-19) in China; some promising results have been achieved thus far. This article summarizes agents with potential efficacy against SARS-CoV-2.</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects and challenges of Open Science - Academic movements related to the Novel coronavirus (2019-nCoV) and COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">オープンサイエンスの効果はさまざまに論じられてき た。たとえば経済協力開発機構（OECD）の報告書 1) では， 科学研究の効率化，研究の透明性や質の向上，技術革新の 加速，経済への波及効果，地球規模の課題への取り組み， 共同研究の推進などが挙げられている。 折しも 2019 年 12 月以降，新型コロナウイルス（2019- nCoV）の感染が中国の武漢市から世界中に拡大して「地 球規模の課題」となっている。2020 年 1 月 30 日には世 界保健機関（WHO）が国際的な緊急事態であると宣言し た 2)。これに対して各国・地域の研究者，助成機関，学術 出版社といったステークホルダーが最新の研究成果の公開 と再利用を進めて対策にあたるという，まさに「オープン サイエンス」というべき動きが起きている。そこで本稿は 新型コロナウイルスに関する学術界の動向を概観した上 で，今回の事例におけるオープンサイエンスの効果と課題 について考察したい。なお，本稿の記述は 2020 年 2 月 6 日時点の情報に基づいている（情報が古い箇所もあろうか と思いますが，ご容赦を）。</p>
<p align="right"><i>score: 35</i></p>

</div><br /><br /><br /><div class="card"><h3>Psychological responses, behavioral changes and public perceptions during the early phase of the COVID-19 outbreak in China: a population based cross-sectional survey</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-20</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To investigate psychological and behavioral responses to the threat of SARS-CoV-2 infections and their associations with public perceptions in China  Design: Cross sectional population-based telephone survey via random digital dialing between 1 and 10 February, 2020 Setting: Wuhan (the epicentre and quarantined city), and Shanghai (a typical major city with close transportation link with Wuhan) Participants: Random sample of 510 residents in Wuhan and 501 residents in Shanghai aged above 18 Main outcome measures: Anxiety (measured by the 7-item generalized anxiety disorder [GAD-7] scale), recommended and avoidance behaviors (engaged in all six behaviors such as increasing surface cleaning and reducing going out). Results: The prevalence rates of moderate or severe anxiety (score ≥10 on GAD-7) were 32.7% (n=167) among Wuhan participants and 20.4% (n=102) among Shanghai participants. 78.6% (n=401) of Wuhan participants and 63.9% (n=320) of Shanghai participants had carried out all six precautionary behaviors. For both measures, Wuhan participants were more responsive to the outbreak (p&lt;0.001). Controlling for personal characteristics, logistic regression results suggested that risks of moderate or severe anxiety were positively associated with perceived susceptibility (odds ratio 1.6, 95% confidence interval 1.3-1.8) and severity of the disease (1.6, 1.4-1.9) and confusion about information reliability (1.6, 1.5-1.9). Having confidence in taking measures to protect oneself against the disease was associated with a lower risk (0.6, 0.5-0.7). The strongest predictor of behavioral change was perceived severity (1.2, 1.1-1.4), followed by confusion about information reliability (1.1, 1.0-1.3).   Conclusions: Psychological and behavioral responses to COVID-19 have been dramatic during the rising phase of the outbreak. Our results support efforts for timely dissemination of accurate and reliable information to address the high anxiety level.</jats:p></p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Clinical Application of A Rapid IgM-IgG Combined Antibody Test for SARS-CoV-2 Infection Diagnosis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of the novel coronavirus disease (COVID-19) quickly spread all over China and to more than 20 other countries. Although the virus (SARS-Cov-2) nucleic acid RT-PCR test has become the standard method for diagnosis of SARS-CoV-2 infection, these real-time PCR test kits have many limitations. In addition, high false negative rates were reported. There is an urgent need for an accurate and rapid test method to quickly identify large number of infected patients and asymptomatic carriers to prevent virus transmission and assure timely treatment of patients. We have developed a rapid and simple point-of-care lateral flow immunoassay which can detect IgM and IgG antibodies simultaneously against SARS-CoV-2 virus in human blood within 15 minutes which can detect patients at different infection stages. With this test kit, we carried out clinical studies to validate its clinical efficacy uses. The clinical detection sensitivity and specificity of this test were measured using blood samples collected from 397 PCR confirmed COVID-19 patients and 128 negative patients at 8 different clinical sites. The overall testing sensitivity was 88.66% and specificity was 90.63%. In addition, we evaluated clinical diagnosis results obtained from different types of venous and fingerstick blood samples. The results indicated great detection consistency among samples from fingerstick blood, serum and plasma of venous blood. The IgM-IgG combined assay has better utility and sensitivity compared with a single IgM or IgG test. It can be used for the rapid screening of SARS-CoV-2 carriers, symptomatic or asymptomatic, in hospitals, clinics, and test laboratories. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome: obstacles and prospects for vaccine development</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of Middle East respiratory syndrome (MERS) highlights the need to engineer new methods for expediting vaccine development against emerging diseases. However, several obstacles prevent pursuit of a licensable MERS vaccine. First, the lack of a suitable animal model for MERS complicates in vivo testing of candidate vaccines. Second, due to the low number of MERS cases, pharmaceutical companies have little incentive to pursue MERS vaccine production as the costs of clinical trials are high. In addition, the timeline from bench research to approved vaccine use is 10 years or longer. Using novel methods and cost-saving strategies, genetically engineered vaccines can be produced quickly and cost-effectively. Along with progress in MERS animal model development, these obstacles can be circumvented or at least mitigated.</p>
<p align="right"><i>score: 34</i></p>

</div><br /><br /><br /></body>