
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: separation of SARS-CoV-2 assay development issues from instruments</h2><br /></div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a quantitative RT-PCR assay for the detection of the emerging coronavirus SARS-CoV-2 using a high throughput system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ability to quickly confirm or clear suspected cases is crucial during global outbreak scenarios, especially when clinical manifestations are difficult to distinguish from other respiratory infections such as influenza, molecular diagnostics is key for detection of the emerging virus. A variety of suitable assays were made available early on during the course of the outbreak, notably by Corman et al. and others [4,5]. However, their implementation in the diagnostics laboratory usually relies on manual PCR setups requiring a high degree of human interaction for execution and interpretation, thus limiting their capacity to be scaled up for handling large numbers of samples. In this study we report the analytical evaluation of a laboratory-developed test for the detection of SARS-CoV-2 using the open channel (utility channel) of the cobas 6800 system.%U https://eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.9.2000152</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Clinical Application of A Rapid IgM-IgG Combined Antibody Test for SARS-CoV-2 Infection Diagnosis</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The outbreak of the novel coronavirus disease (COVID-19) quickly spread all over China and to more than 20 other countries. Although the virus (SARS-Cov-2) nucleic acid RT-PCR test has become the standard method for diagnosis of SARS-CoV-2 infection, these real-time PCR test kits have many limitations. In addition, high false negative rates were reported. There is an urgent need for an accurate and rapid test method to quickly identify large number of infected patients and asymptomatic carriers to prevent virus transmission and assure timely treatment of patients. We have developed a rapid and simple point-of-care lateral flow immunoassay which can detect IgM and IgG antibodies simultaneously against SARS-CoV-2 virus in human blood within 15 minutes which can detect patients at different infection stages. With this test kit, we carried out clinical studies to validate its clinical efficacy uses. The clinical detection sensitivity and specificity of this test were measured using blood samples collected from 397 PCR confirmed COVID-19 patients and 128 negative patients at 8 different clinical sites. The overall testing sensitivity was 88.66% and specificity was 90.63%. In addition, we evaluated clinical diagnosis results obtained from different types of venous and fingerstick blood samples. The results indicated great detection consistency among samples from fingerstick blood, serum and plasma of venous blood. The IgM-IgG combined assay has better utility and sensitivity compared with a single IgM or IgG test. It can be used for the rapid screening of SARS-CoV-2 carriers, symptomatic or asymptomatic, in hospitals, clinics, and test laboratories. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Corona Virus Disease 2019 (COVID-19) is a recently emerged life-threatening disease caused by SARS-CoV-2. Real- time fluorescent PCR (RT-PCR) is the clinical standard for SARS-CoV-2 nucleic acid detection. To detect SARS-CoV-2 early and control the disease spreading on time, a faster and more convenient method for SARS-CoV-2 nucleic acid detecting, RT-LAMP method (reverse transcription loop-mediated isothermal amplification) was developed. RNA reverse transcription and nucleic acid amplification were performed in one step at 63 ℃ isothermal conditions, and the results can be obtained within 30 minutes. ORF1ab gene, E gene and N gene were detected at the same time. ORF1ab gene was very specific and N gene was very sensitivity, so they can guarantee both sensitivity and specificity for SARS-CoV-2. The sensitivity of RT-LAMP assay is similar to RT-PCR, and specificity was 99% as detecting 208 clinical specimens. The RT-LAMP assay reported here has the advantages of rapid amplification, simple operation, and easy detection, which is useful for the rapid and reliable clinical diagnosis of SARS-CoV-2.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus and the race to distribute reliable diagnostics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">International teams worked at speed to make tests for the virus available in record time. The medical community is rallying to develop a set of rapid and reliable molecular diagnostic tests for the new human coronavirus that appeared in China — now dubbed sudden acute respiratory syndrome coronavirus-2 (SARS-CoV-2).</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosis of Acute Respiratory Syndrome Coronavirus 2 Infection by Detection of Nucleocapsid Protein</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND Nucleic acid test and antibody assay have been employed in the diagnosis for SARS-CoV-2 infection, but the use of viral antigen for diagnosis has not been successfully developed.  Theoretically, viral antigen is the specific marker of the virus and precedes antibody appearance within the infected population. There is a clear need of detection of viral antigen for rapid and early diagnosis. METHODS We included a cohort of 239 participants with suspected SARS-CoV-2 infection from 7 centers for the study. We measured nucleocapsid protein in nasopharyngeal swab samples in parallel with the nucleic acid test. Nucleic acid test was taken as the reference standard, and statistical evaluation was taken in blind. We detected nucleocapsid protein in 20 urine samples in another center, employing nasopharyngeal swab nucleic acid test as reference standard. RESULTS We developed a fluorescence immunochromatographic assay for detecting nucleocapsid protein of SARS-CoV-2 in nasopharyngeal swab sample and urine within 10 minutes.  100% of nucleocapsid protein positive and negative participants accord with nucleic acid test for same samples. Further, earliest participant after 3 days of fever can be identified by the method.  In an additional preliminary study, we detected nucleocapsid protein in urine in 73.6% of diagnosed COVID-19 patients.  CONCLUSIONS Those findings indicate that nucleocapsid protein assay is an accurate, rapid, early and simple method for diagnosis of COVID-19. Appearance of nucleocapsid protein in urine coincides our finding of the SARS-CoV-2 invading kidney and might be of diagnostic value.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Analytical sensibility and specificity of two RT-qPCR protocols for SARS-CoV-2 detection performed in an automated workflow</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The World Health Organization declared that COVID-19 outbreak constituted a Public Health Emergency of International Concern and the development of reliable laboratory diagnosis of SARS-CoV-2 became mandatory to identify, isolate and provide optimized care for patients early. RT-qPCR testing of respiratory secretions is routinely used to detect causative viruses in acute respiratory infection. RT-qPCR in-house protocols to detect the SARS-CoV-2 have been described. Validations of these protocols are considered a key knowledge gap for COVID-19, especially if executed in a high throughput format. Here, we investigate the analytical sensitivity and specificity of two interim RT-qPCR protocols for the qualitative detection of SARS-CoV-2 executed in a fully automated platform. Under our conditions, the N1 and RdRP (modified) showed the highest analytical sensitivity for their RNA targets. E assay, in its original concentration, was considered a tertiary confirmatory assay. Taken together, N1, RdRP (optimized) and E presented appropriated analytical sensibility and specificity in our automated RT-qPCR workflow for COVID-19 virus, E being at least 4-fold less sensitive than the others. This study highlights the importance of local validation of in-house assays before its availability to the population. The use of the synthetic RT-qPCR target to investigate novel assays diagnostic parameters in automated workflows is a quick, simple effective way to be prepared for upcoming threats. The proposed assay detected the first SARS-CoV-2 infection in Brazilian Central-West.</jats:p></p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid establishment of laboratory diagnostics for the novel coronavirus SARS-CoV-2 in Bavaria, Germany, February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In connection with the ongoing outbreak of a novel coronavirus in the province Hubei and surrounding areas in China, it was expected that Europe would also be confronted with the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), as infections in travellers in several Asian countries outside of China were confirmed shortly after the announcement of the outbreak in Wuhan [1-4]. Therefore, it was necessary to rapidly implement adequately quick and sensitive diagnostic assays for outbreak management of SARS-CoV-2 in public health laboratories. As soon as the World Health Organization (WHO) published the first protocols for real-time RT-PCR assays, the Bavarian Food and Health Authority started to implement them. We ordered control material and oligonucleotides (see details below) in week 4 and ran our first SARS-CoV-2 assays on 27 January (week 5). On the same day, the first German case of coronavirus disease 2019 (COVID-19) was diagnosed in Bavaria [1]. In the following days, health authorities implemented comprehensive measures to prevent further transmission of SARS-CoV-2, including testing of contact persons. We initially used the protocol based on the E gene and RdRp gene developed by the German Consiliary Laboratory for Coronaviruses hosted at the Charité in Berlin [5]. Real-time RT-PCR was initially performed with the QuantiTect Virus +Rox Vial kit (QIAGEN, Hilden, Germany) on the Bio-Rad CFX96 Touch Real-Time PCR Detection System (Bio-Rad, Feldkirchen, Germany). The kit was chosen for its frequent and successful use in our laboratory with other assays and its immediate availability. Primers and probes were used as described [5] and provided by TIB Molbiol (Berlin, Germany). Control material was ordered from the European Virus Archive (EVAg) and consisted of synthetic Wuhan coronavirus 2019 E gene control (reference number 026N-03866) and SARS-CoV Frankfurt 1 RNA (reference number 004N-02005) [6]. In addition, the control of LightMix Modular Wuhan CoV RdRP-gene (TibMolbiol, Berlin, Germany) was used for the SARS-CoV-2 specific assay. Respiratory samples (nasopharyngeal swabs or sputum) were obtained from patients and contact persons. Sputum samples were diluted in 2 mL phosphate buffered saline (PBS). RNA was extracted using the QIAamp Bio Robot kit (QIAGEN) on a Hamilton Microlab Star (Hamilton, Bonaduz, Switzerland). As at 1 March, 87,024 cases and 2,979 associated deaths have been reported worldwide [1]. The vast majority of the deaths (96%) have been reported in China [1]. Despite the high number of cases reported globally, estimates of the severity pyramid of disease and case fatality rate remain very uncertain; one large study conducted in China estimated that the majority (81%) of the cases were mild (i.e. non-pneumonia or mild pneumonia), 14% were severe (e.g. with dyspnoea) and 5% were in a critical condition (i.e. respiratory failure, septic shock and/or multiple organ dysfunction/failure) [2]. The case fatality ratio was 2.3% [2]. Despite extraordinary containment measures implemented in China, including the enforced lockdown of several cities and closures of schools, the virus has spread throughout the country and internationally [2]. It is too early to predict with any certainty the epidemiological developments over the coming weeks, but the possibility of widespread community transmission becoming established throughout the EU/EEA is becoming increasingly likely.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan, China, from Jan to Feb 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Background There’s an outbreak of a novel coronavirus (SARS-CoV-2) infection since December 2019, first in China, and currently with more than 80 thousand confirmed infection globally in 29 countries till March 2, 2020. Identification, isolation and caring for patients early are essential to limit human-to-human transmission including reducing secondary infections among close contacts and health care workers, preventing transmission amplification events. The RT-PCR detection of viral nucleic acid test (NAT) was one of the most quickly established laboratory diagnosis method in a novel viral pandemic, just as in this COVID-19 outbreak. Methods 4880 cases that had respiratory infection symptoms or close contact with COVID-19 patients in hospital in Wuhan, China, were tested for SARS-CoV-2 infection by use of quantitative RT-PCR (qRT-PCR) on samples from the respiratory tract. Positive rates were calculated in groups divided by genders or ages. Results The positive rate was about 38% for the total 4880 specimens. Male and older population had a significant higher positive rates. However, 57% was positive among the specimens from the Fever Clinics. Binary logistic regression analysis showed that age, not gender, was the risk factor for SARS-CoV-2 infection in fever clinics. Conclusions Therefore, we concluded that viral NAT played an important role in identifying SARS-CoV-2 infection.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple laboratory parameter facilitates early identification of COVID-19 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.</jats:p></p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Labs scramble to spot hidden coronavirus infections</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The seeming precision of the global tallies of cases and deaths caused by the novel coronavirus now spreading from Wuhan, China belies an alarming fact. ... (A study group of the International Committee on Taxonomy of Viruses christened the novel virus severe acute respiratory syndrome coronavirus 2, or SARS-CoV-2, the same day). But many news stories have reported shortages of diagnostics in Hubei.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">1The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.</jats:p></p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnosing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">hina continues to fight the outbreak of a novel coronavirus, called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which has infected tens of thousands and killed more than 2,100 people since the end of 2019. The outbreak’s hot spot is in the central province of Hubei, in particular, its capital city Wuhan, where there have been more than 62,000 infections and 2,029 deaths as of last week, according to the World Health Organization (WHO). There, doctors and scientists are desperately struggling to find reliable ways to diagnose infected patients to help treat them and control the spread of the virus. The struggle has involved two diagnostic options—computed tomography (CT) scans of patients’ lungs and a nucleic acid lab test—each with advantages and disadvantages. With “a suddenly rampant new virus, it is normal to have multiple test approaches,” says a virologist at the Chinese Center for Disease Control and Prevention</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>A Peptide-based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Corona Virus Disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>A respiratory illness has been spreading rapidly in China, since its outbreak in Wuhan city, Hubei province in December 2019. The illness was caused by a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Clinical manifestations related to SARS-CoV-2 infection ranged from no symptom to fatal pneumonia. World Health Organization (WHO) named the diseases associated with SARS-CoV-2 infection as COVID-19. Real time RT-PCR is the only laboratory test available till now to confirm the infection. However, the accuracy of real time RT-PCR depends on many factors, including sampling location and of methods, quality of RNA extraction and training of operators etc. Variations in these factors might significantly lower the sensitivity of the detection. We developed a peptide-based luminescent immunoassay to detect IgG and IgM. Cut-off value of this assay was determined by the detection of 200 healthy sera and 167 sera from patients infected with other pathogens than SARS-CoV-2. To evaluate the performance of this assay, we detected IgG and IgM in the 276 sera from confirmed patients. The positive rate of IgG and IgM were 71.4% (197/276) and 57.2% (158/276) respectively. By combining with real time RT-PCR detection, this assay might help to enhance the accuracy of diagnosis of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus  broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</jats:p></p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;">Coronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the  2019-nCoV_N2, N3 of USA and the  ORF1ab of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>A proposal of an alternative primer for the ARTIC Network's multiplex PCR to improve coverage of SARS-CoV-2 genome sequencing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">A group of biologists, ARTIC Network, has proposed a multiplexed PCR primer set for whole genome analysis of the novel corona virus, SARS-CoV-2, soon after the epidemics of this pathogen was revealed. The primer set seems to have been adapted already by many researchers worldwide and contributed for the high-quality and prompt genome epidemiology of this potential pandemic virus. We have also seen the great performance of their primer set and protocol; the primer set was able to amplify all desired PCR products with fairy small amplification bias from clinical samples with relatively high viral load. However, we observed acute drop of reads derived from two particular PCR products, 18 and 76, out of the 98 designated products as sample's viral load decreases. We suspected the reason for this low coverage issue was due to dimer formation between primers used to amplify those two PCR products. Here, we propose replacing just one of those primers, nCoV-2019_76_RIGHT(-), to a newly designed primer. The result of the replacement of primer showed improvement in coverage at both regions targeted by the products, 18 and 76. We expect this simple modification will extend the limit for whole SARS-CoV-2 genome analysis to samples with lower viral load and enhance genomic epidemiology of this pathogen.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Duration of quarantine in hospitalized patients with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection: a question needing an answer</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019 a new form of pneumonia was observed in the Chinese province of Hubei.[1] The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was subsequently identified as responsible of this condition, defined coronavirus disease (COVID-19).[2] The virus has now spread outside Chinese borders with 82,297 cases and 2,804 deaths worldwide at the 26th of February.[3] After infection, symptoms appear after an incubation time of 3-5 days, with 80% of those infected developing a mild disease, 15% a severe disease and 5% will require support in intensive care unit (ICU).[4] Overall, the estimated case-fatality rate is comprised between 0.4% and 2.9% and the basic reproduction number is approximately 3.28.[4,5] SARS-CoV-2 is a new pathogen for humankind and any type of immune protection exist, thus everybody can be potentially infected. Moreover, no primary prophylaxis measures (vaccination) nor effective treatments are available. If the above represented percentages are applied to the worldwide populations, it appears clear why any measure should be considered to avoid a further diffusion of the virus and prevent the saturation and collapse of health systems and the most catastrophic pandemic since 1919 Spanish flu. Isolation of those affected and the use of personal protective equipment (PPE) are the mainstay to block transmission of this pathogen, which is presumed through respiratory droplets. A 14 days quarantine is applied to subjects coming from endemic areas or who had contact with confirmed cases. It is assumed that, if in this period the subject does not develop any sign or symptoms compatible with COVID-19, he is not infected and thus the quarantine can be removed, and the subject returned to the community. Domiciliary quarantine of 14 days since a positive test is applied also for patients with a diagnosed mild disease who did not need medical support. These rules are effective in controlling infections in the community, but several doubts arise when it is necessary to transpose them in the hospital setting. Hospitals are indeed a delicate place in epidemics: they collect fragile persons who can be exposed to the virus and are subsequently readmitted to the community thus spreading the infection. Indeed, the ongoing outbreak in Northern Italy has been linked to a single infected patient who accessed to a community hospital where he transmitted the virus to several other patients and health-care operators.[6] Moreover, the isolation of patients in the hospital setting impose a significant burden in terms of PPE used by the health-care operators, space dedicated and time employed in their management. Even more complex is the situation of patients in ICU, where viral spreading is facilitated by endotracheal tubes and manoeuvres performed on the respiratory tract. Therefore, a clear definition of the infectiousness timing and intensity of viral spreading is mandatory to alleviate the burden on the health-care system. Unfortunately, the data available on the topic are scarce and composed only of measurements of viral shedding, without an assessment of the infectivity. Kim et al.[7] assessed the viral load kinetics of SARS-CoV-2 in upper and lower respiratory tract materials in the first two confirmed patients in Korea. They employed real-time reverse transcriptase polymerase chain reaction (rRT-PCR) to detect SARS-CoV-2 and converted cycle threshold (CT) values of rRT-PCR into RNA copy number. The detection limit of rRT-PCR was 2,690 copies/mL. Overall, viral load above detection limit was detected until 14 and 25 days after symptoms onset and for 13 and 11 days after the first detection, respectively.[7] Of note, both patients received treatment with lopinavir/ritonavir. Instead, Zou and colleagues analysed viral load in repeated nasal and throat swabs obtained from the 17 symptomatic patients.[8] They also employed rRT-PCR and considered a CT of 40 as detection limit. Higher viral loads were observed in nasal </p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 sensitive to type I interferon pretreatment.</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B coronaviruses share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel coronavirus is much more sensitive to type I interferon pretreatment. We subsequently examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN response between SARS-CoV and SARS-CoV-2 that could help inform disease progression, treatment options, and animal model development.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Biotech start-ups hit by coronavirus work stoppages</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">s the outbreak of the new coronavirus, now officially named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), continues to mushroom, the related work stoppages in China have put a spotlight on Western biotech companies’ dependence on Chinese contract research organizations (CROs). The situation has prompted some start-ups to rethink their contingency plans—particularly when it comes to chemistry services. Biotech firms had anticipated delays related to the Lunar New Year holidays, and so far outbreak-related travel restrictions have extended that work gap by only a week. But as the virus spreads—according to Chinese authorities, as of Feb. 13, more than 60,100 people had been infected and nearly 1,400 had died, most of them in China—some biotech firms that rely on China are becoming nervous about how long their projects could be on hold. Much attention is on WuXi AppTec, the largest Chinese CRO.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Outbreak investigation for COVID-19 in northern Vietnam</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Two Vietnamese adults returned to their home province of Vinh Phuc in northern Vietnam on Jan 17, 2020, from Wuhan, China, where they had been living since Nov 15, 2019, for a business trip. They presented with mild respiratory symptoms to their local health facilities at 4 days and 8 days, respectively, after arrival in Vinh Phuc. Both individuals were initially placed into respiratory isolation in hospital. Case 1 tested positive for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative organism of coronavirus disease 2019 (COVID-19), on Jan 30, 2020, and remained in isolation until recovery. Case 2 was discharged from isolation in hospital after having one negative test result on Jan 28 (11 days after returning from Wuhan). Following discharge, the patient attended a family social function. 2 days later, she was readmitted after a second nasal swab for SARS-CoV-2 taken during her time in hospital was reported as positive. Screening of 79 individuals who had been in contact with these two patients (namely, family members in the same household and anyone who had been within 2 m of them) was initiated on Jan 31. Six individuals from the same work team, who had also travelled from Wuhan on Jan 17, were isolated, and four of them tested positive for SARS-CoV-2 (cases 3, 4, and 8 in Vinh Phuc, and one case from another province). Five secondary cases were diagnosed within the social network of case 2. These included three household members (cases 6, 7, and 11) and two people who had attended the social function (cases 5 and 9; appendix p 1). Four of these individuals reported mild respiratory symptoms; the remaining patient was asymptomatic (case 7) at the time of diagnosis.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>What we know so far: COVID-19 current clinical knowledge and research</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, health authorities in Wuhan, China, identified a cluster of pneumonia cases of unknown aetiology linked to the city's South China Seafood Market. Subsequent investigations revealed a novel coronavirus, SARS-CoV-2, as the causative agent now at the heart of a major outbreak. The rising case numbers have been accompanied by unprecedented public health action, including the wholesale isolation of Wuhan. Alongside this has been a robust scientific response, including early publication of the pathogen genome, and rapid development of highly specific diagnostics. This article will review the new knowledge of SARS-CoV-2 COVID-19 acute respiratory disease, and summarise its clinical features.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SARS-CoV-2 has caused tens of thousands of infections and more than one thousand deaths. There are currently no registered therapies for treating coronavirus infections. Because of time consuming process of new drug development, drug repositioning may be the only solution to the epidemic of sudden infectious diseases. We systematically analyzed all the proteins encoded by SARS-CoV-2 genes, compared them with proteins from other coronaviruses, predicted their structures, and built 19 structures that could be done by homology modeling. By performing target-based virtual ligand screening, a total of 21 targets (including two human targets) were screened against compound libraries including ZINC drug database and our own database of natural products. Structure and screening results of important targets such as 3-chymotrypsin-like protease (3CLpro), Spike, RNA-dependent RNA polymerase (RdRp), and papain like protease (PLpro) were discussed in detail. In addition, a database of 78 commonly used anti-viral drugs including those currently on the market and undergoing clinical trials for SARS-CoV-2 was constructed. Possible targets of these compounds and potential drugs acting on a certain target were predicted. This study will provide new lead compounds and targets for further in vitro and in vivo studies of SARS-CoV-2, new insights for those drugs currently ongoing clinical studies, and also possible new strategies for drug repositioning to treat SARS-CoV-2 infections.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 20</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world. With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenges and countermeasures for organ donation during the SARS-CoV-2 epidemic: the experience of Sichuan Provincial People’s Hospital</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Up to 09:40 February 17, 2020, there have been 70,640 confirmed SARS-CoV-2 (aka 2019-nCoV) cases in China. Sichuan Provincial People’s Hospital acts as an organ transplant center in west of China, approximately 200 solid organ transplants performed each year and can perform heart, lung, liver, kidney, small bowel, stem cell transplantation, so it is necessary to establish a hospital-specific protocol to deal with the SARS-CoV-2 infection for the donor and recipient.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 31st December 2019, the World Health Organization was informed of a cluster of cases of pneumonia of unknown etiology in Wuhan, China. Subsequent investigations identified a novel coronavirus, now named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), from the affected patients. Highly sensitive and specific laboratory diagnostics are important for controlling the rapidly evolving SARS-CoV-2-associated Coronavirus Disease 2019 (COVID-19) epidemic. In this study, we developed and compared the performance of three novel real-time RT-PCR assays targeting the RNA-dependent RNA polymerase (RdRp)/helicase (Hel), spike (S), and nucleocapsid (N) genes of SARS-CoV-2 with that of the reported RdRp-P2 assay which is used in >30 European laboratories. Among the three novel assays, the COVID-19-RdRp/Hel assay had the lowest limit of detection in vitro (1.8 TCID50/ml with genomic RNA and 11.2 RNA copies/reaction with in vitro RNA transcripts). Among 273 specimens from 15 patients with laboratory-confirmed COVID-19 in Hong Kong, 77 (28.2%) were positive by both the COVID-19-RdRp/Hel and RdRp-P2 assays. The COVID-19-RdRp/Hel assay was positive for an additional 42 RdRd-P2-negative specimens [119/273 (43.6%) vs 77/273 (28.2%), P<0.001], including 29/120 (24.2%) respiratory tract specimens and 13/153 (8.5%) non-respiratory tract specimens. The mean viral load of these specimens was 3.21*104 RNA copies/ml (range, 2.21*102 to 4.71*105 RNA copies/ml). The COVID-19-RdRp/Hel assay did not cross-react with other human-pathogenic coronaviruses and respiratory pathogens in cell culture and clinical specimens, whereas the RdRp-P2 assay cross-reacted with SARS-CoV in cell culture. The highly sensitive and specific COVID-19-RdRp/Hel assay may help to improve the laboratory diagnosis of COVID-19.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As of February 2020, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) outbreak started in China in December 2019 has been spreading in many countries in the world.With the numbers of confirmed cases are increasing, information on the epidemiologic investigation and clinical manifestation have been accumulated. However, data on viral load kinetics in confirmed cases are lacking. Here, we present the viral load kinetics of the first two confirmed patients with mild to moderate illnesses in Korea in whom distinct viral load kinetics are shown. This report suggests that viral load kinetics of SARS-CoV-2 may be different from that of previously reported other coronavirus infections such as SARS-CoV.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA sequencing and early evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The rapid sharing of sequence information as seen throughout the current SARS-CoV-2 epidemic, represents an inflection point for genomic epidemiology. Here we describe aspects of coronavirus evolutionary genetics revealed from these data, and provide the first direct RNA sequence of SARS-CoV-2, detailing coronaviral subgenome-length mRNA architecture.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>The course of clinical diagnosis and treatment of a case infected with coronavirus disease 2019</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract A pneumonia outbreak caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was first identified in Wuhan, present a major threat to public health since December 2019. There are more than 50,000 confirmed cases and 1300 dead cases worldwide for the past month or more, because of the occurrence of a highly contagious performance. Patients had clinical manifestations of fever, cough, shortness of breath, diarrhea, vomiting and so on. We herein report a case of SARS-CoV-2, describe the epidemic history, clinical diagnosis and the changes of clinical parameters during the combination therapy. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>A descriptive study of the impact of diseases control and prevention on the epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai, lessons learned for metropolis epidemics prevention</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective: To describe and evaluate the impact of diseases control and prevention on epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai.  Design: A retrospective descriptive study  Setting: China  Participants: Epidemiology information was collected from publicly accessible database. 265 patients admitted to Shanghai Public Health Center with confirmed COVID-19 were enrolled for clinical features analysis.  Main outcome measure: Prevention and control measures taken by Shanghai government,  epidemiological, demographic, clinical, laboratory and radiology data were collected. Weibull distribution, Chi-square test, Fisher's exact test, t test or Mann-Whitney U test were used in statistical analysis.   Results: COVID-19 transmission rate within Shanghai had reduced over 99% than previous speculated, and the exponential growth has been stopped so far. Epidemic was characterized by the first stage mainly composed of imported cases and the second stage where &gt;50% of cases were local. The incubation period was 6.4 (95% CI 5.3 to 7.6) days and the mean onset-admission interval was 5.5 days (95% CI, 5.1 to 5.9). Median time for COVID-19 progressed to severe diseases were 8.5 days (IQR: 4.8-11.0 days). By February 11th, proportion of patients being mild, moderate, severe and critically ill were 1.9%(5/265), 89.8%(238/265), 3.8%(10/265), 4.5%(12/265), respectively; 47 people in our cohort were discharged, and 1 patient died.  Conclusion: Strict controlling of the transmission rate at the early stage of an epidemic in metropolis can quickly prohibit the spread of the diseases. Controlling local clusters is the key to prevent outbreaks from imported cases. Most COVID-19 severe cases progressed within 14 days of disease onset. Multiple systemic laboratory abnormalities had been observed before significant respiratory dysfunction.   Keyword: COVID-19, SARS-CoV-2, epidemics dynamics, diseases control, clinical features</jats:p></p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We report the kinetics of the immune response in relation to clinical and virological features of a patient with mild-to-moderate coronavirus disease-19 (COVID-19) requiring hospitalisation. Increased antibody-secreting cells, follicular T-helper cells, activated CD4+ and CD8+ T-cells and IgM/IgG SARS-CoV-2-binding antibodies were detected in blood, prior to symptomatic recovery. These immunological changes persisted for at least 7 days following full resolution of symptoms, indicating substantial anti-viral immunity in this non-severe COVID-19.</jats:p></p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Recommendations on the pediatric flexible bronchoscopy during the outbreak of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infection in China (Trial Edition)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An outbreak of pneumonia caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)infection has spread to children.Due to its strong infectivity, people are generally susceptible, including children.The main source of infection is the SARS-CoV-2 infection patients, asymptomatic infection may also become the source of infection. Bronchoscopy is a high-risk operation since during the procedure, with children & prime's coughing and an open airway, a huge number of droplets and secretions are produced, which will contaminate the desktop, equipment and air, and even infect medical staff, other children and caregivers in close contact.For this reason, experts were specially organized to write recommendations on the diagnosis and treatment of pediatric flexible bronchoscopy during the epidemic period of SARS-CoV-2 infection (trial version), establish indications and prevention and control plans of pediatric bronchoscopy during the epidemic period, and provide a basis for medical staff engaged in pediatric bronchoscopy.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>A Locally Transmitted Case of SARS-CoV-2 Infection in Taiwan</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On January 25, 2020, a 52-year-old woman with a history of type 2 diabetes presented with fever to an emergency department in central Taiwan. She was admitted to the hospital because of suspicion of pneumonia associated with SARS-CoV-2 infection. She had lived in Wuhan from October 21, 2019, to January 20, 2020. She returned to Taiwan from Wuhan on January 20 on an airplane. On the same day, a throat swab was obtained from another passenger on that flight; that passenger was confirmed to have the first known imported case of SARS-CoV-2 infection in Taiwan when the swab was found to be positive for the virus on January 21.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-01</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited.  METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG).  CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be greatly relieved.</jats:p></p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;">Severe Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 - a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>On the origin and continuing evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.</jats:p></p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiological and clinical features of COVID-19 patients with and without pneumonia in Beijing, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background:SARS-CoV-2-caused coronavirus disease (COVID-19) is posinga large casualty. The features of COVID-19patients withand without pneumonia,SARS-CoV-2 transmissibility in asymptomatic carriers, and factors predicting disease progression remain unknown.  Methods: We collected information on clinical characteristics, exposure history, andlaboratory examinations of all laboratory-confirmed COVID-19 patients admitted to PLA General Hospital. Cox regression analysis was applied to identify prognostic factors. The last follow-up was February 18, 2020. Results:We characterized 55 consecutive COVID-19 patients. The mean incubation was 8.42(95% confidence interval [CI], 6.55-10.29) days. The mean SARS-CoV-2-positive duration from first positive test to clearance was 9.71(95%CI, 8.21-11.22) days. COVID-19 course was approximately 2 weeks. Asymptomatic carriers might transmit SARS-CoV-2. Compared with patients without pneumonia, those with pneumonia were 15 years older and had a higher rate of hypertension, higher frequencies of having a fever and cough, and higher levels of interleukin-6 (14.61 vs. 8.06pg/mL, P=0.040), B lymphocyte proportion (13.0% vs.10.0%, P=0.024), low account (&lt;190/μL) of CD8+ T cells (33.3% vs. 0, P=0.019). Multivariate Cox regression analysis indicated that circulating interleukin-6 andlactate independently predicted COVID-19 progression, with a hazard ratio (95%CI) of 1.052 (1.000-1.107) and 1.082 (1.013-1.155), respectively. During disease course,T lymphocytes were generally lower,neutrophils higher, in pneumonia patients than in pneumonia-free patients. CD8+ lymphocytes did not increase at the 20th days after illness onset. Conclusion: The epidemiological features areimportant for COVID-19 prophylaxis. Circulating interleukin-6 and lactateare independent prognostic factors. CD8+ T cell exhaustion might be critical in the development of COVID-19.</jats:p></p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Disease 2019: Coronaviruses and Blood Safety</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">With the outbreak of unknown pneumonia in Wuhan, China in December 2019, a new coronavirus, Severe Acute Respiratory Syndrome Corona Virus-2 (SARS-CoV-2) aroused the attention of the entire world. The current outbreak of infections with SARS-CoV-2 is termed Corona Virus Disease- 2019 (COVID-19). The World Health Organization (WHO) declared COVID-19 in China as a Public Health Emergency of International Concern (PHEIC). Two other corona virus infections-- SARS in 2002–2003 and Middle East Respiratory Syndrome (MERS) in 2012-- both caused severe respiratory syndrome in humans. All three of these emerging infectious diseases leading to a global spread are caused by beta-coronaviruses. Although coronaviruses usually infect the upper or lower respiratory tract, viral shedding in plasma or serum is common. Therefore, there is still a theoretical risk of transmission of coronaviruses through the transfusion of labile blood products. Because more and more asymptomatic infections are being found among COVID-19 cases, considerations of blood safety and coronaviruses have arisen especially in endemic areas. In this review, we detail current evidence and understanding of the transmission of SARS-CoV, MERS-CoV and SARS-CoV-2 through blood products as of February 10, 2020 and also discuss pathogen inactivation methods on coronaviruses.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical manifestations and outcome of SARS-CoV-2 infection during pregnancy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Tang and colleagues, in this Journal, drew readers attention to emerging COVID19.1 We focused on the pregnant COVID19 patients. Given the maternal physiologic and immune function changes in pregnancy,2 pregnant individuals might face greater risk of getting infected by SARS-CoV-2 and might have more complicated clinical events. We described epidemiological, clinical characteristics, pregnancy and perinatal outcomes of all hospitalized pregnant patients diagnosed with COVID-19 in China. We identified all hospitalized pregnant patients with laboratory-confirmed SARS-CoV-2 infection between December 8, 2019, and February 25, 2020 officially reported by the central government, in areas outside Wuhan, China. Information including age, geographic location, epidemiological history, prenatal course, maternal and newborn hospital course, discharge data and outcome were obtained by Centers for Disease Control and Prevention and Local Health Commission. When necessary, we attempted to contact local hospital or patients by telephone to supply missing information. This investigation was part of an emergency public health outbreak investigation and therefore not subject to institutional review board. There were a total of 13 Chinese patients with SARS-CoV-2 admitted to hospitals outside of Wuhan (Table 1). There were 3 patients from Zhejiang, 3 from other cities of Hubei and 1 each from Fujian, Shanxi, Beijing, Guangdong, Jiangxi, Heilongjiang and Anhui. The maternal age ranged between 22 to 36 years. Two women were less than 28 weeks of gestation and the other 11 patients were in their third trimesters at presentation. None of the patients had underlying medical disease.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 and the cardiovascular system</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects host cells through ACE2 receptors, leading to coronavirus disease (COVID-19)-related pneumonia, while also causing acute myocardial injury and chronic damage to the cardiovascular system. Therefore, particular attention should be given to cardiovascular protection during treatment for COVID-19.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-04</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract  Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control.  Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk.   Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 32 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days.   Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.</jats:p></p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Four ways researchers are responding to the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts. How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Angiotensin-converting enzyme 2 (ACE2) is the cellular receptor for SARS coronavirus (SARS-CoV) and the new coronavirus (SARS-CoV-2) that is causing the serious epidemic COVID-19. Here we present cryo-EM structures of full-length human ACE2, in the presence of a neutral amino acid transporter B(0)AT1, with or without the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) of SARS-CoV-2, both at an overall resolution of 2.9 A, with a local resolution of 3.5 A at the ACE2-RBD interface. The ACE2-B(0)AT1 complex is assembled as a dimer of heterodimers, with the Collectrin-like domain (CLD) of ACE2 mediating homo-dimerization. The RBD is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. These findings provide important insights to the molecular basis for coronavirus recognition and infection.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP).  We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS.  We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene.  Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.</jats:p></p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus disease (COVID-19) and neonate: What neonatologist need to know</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About 3 neonates and more than 230 children cases are reported. The disease condition of mainly children was mild. There is currently no evidence that SARS-CoV-2can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID-19) is based on adult experience. Thus far, no deaths have been reported in the paediatric age group. This review describes the current understanding of COVID-19 infection in newborns and children. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>2019 novel coronavirus disease (COVID-19) in Taiwan: Reports of two cases from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We reported two cases with community-acquired pneumonia caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) who returned from Wuhan, China in January, 2020. The reported cases highlight non-specific clinical presentations of 2019 novel coronavirus disease (COVID-19) as well as the importance of rapid laboratory-based diagnosis.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>The transmission and diagnosis of 2019 novel coronavirus infection disease (COVID-19): A Chinese perspective</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">2019 novel coronavirus (SARS-CoV-2), which originated in Wuhan, China, has attracted the world's attention over the last month. The Chinese government has taken emergency measures to control the outbreak and has undertaken initial steps in the diagnosis and treatment of 2019 novel coronavirus infection disease (COVID-19). However, SARS-CoV-2 possesses powerful pathogenicity as well as transmissibility and still holds many mysteries that are yet to be solved, such as whether the virus can be transmitted by asymptomatic patients or by mothers to their infants. Our research presents selected available cases of COVID-19 in China to better understand the transmission and diagnosis regarding this infectious disease. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel Immunoglobulin Domain Proteins Provide Insights into Evolution and Pathogenesis Mechanisms of SARS-Related Coronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) is the causative agent of an emergent severe respiratory disease (COVID-19) in humans that is threatening to result in a global health crisis. By using genomic, sequence, structural and evolutionary analysis, we show that Alpha- and Beta-CoVs possess several novel families of immunoglobulin (Ig) domain proteins, including ORF8 and ORF7a from SARS-related coronaviruses and two protein groups from certain Alpha-CoVs. Among them, ORF8 is distinguished in being rapidly evolving, possessing a unique insert and a hypervariable position among SARS-CoV-2 genomes in its predicted ligand-binding groove. We also uncover many Ig proteins from several metazoan viruses which are distinct in sequence and structure but share an architecture comparable to that of CoV Ig domain proteins. Hence, we propose that deployment of Ig domain proteins is a widely-used strategy by viruses, and SARS-CoV-2 ORF8 is a potential pathogenicity factor which evolves rapidly to counter the immune response and facilitate the transmission between hosts.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 in the Shadows of MERS-CoV in the Kingdom of Saudi Arabia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) has plagued the Middle East since it was first reported in 2012. Recently, at the end of December 2019, a cluster of pneumonia cases were reported from Wuhan city, Hubei Province, China, linked to a wet seafood market with a new coronavirus identified as the etiologic agent currently named SARS-CoV-2. Most cases are in Mainland China with international spread to 25 countries. The novelty of the virus, the rapid national and international spread, and the lack of therapeutic and preventative strategies have led the WHO International Health Regulation emergency committee to declare the disease as Public Health Emergency of International Concern (PHEIC) on January 30, 2020. As it relates to countries with the ongoing MERS-CoV community cases and hospital acquired infections, there will be a huge challenge for HCWs to deal with both coronaviruses, especially with the lack of standardized and approved point of care testing. This challenge will now be faced by the whole global health community dealing with COVID-19 since both coronaviruses have similar presentation. Those patients should now be tested for both MERS-CoV and SARS-CoV-2 simultaneously, and with the continuing wide international spread of SARS-CoV-2, the travel history to China in the last 14 days will be of less significance</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Managing neonates with respiratory failure due to SARS-CoV-2</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">In their Comment in The Lancet Child & Adolescent Health, Jianhui Wang and colleagues1 suggested a plan to handle neonates with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections and outbreaks in neonatal intensive care units (NICUs). This is a timely reflection, given the public health problem represented by this infection and the need to anticipate any critical care issue, irrespective of patients' ages. However, the plan is incomplete or unsuitable in many points. We do not know anything about neonatal SARS-CoV-2 infections, and we must reasonably follow data from adult critical care. First, testing all NICU-admitted neonates for SARS-CoV-2 represents a wrongful use of resources. Neonatal respiratory failure can result from a wide range of causes, and testing everybody when other causes are reasonably suspected will divert laboratory resources from adult critical care. Tests should be done for infants from families infected by SARS-CoV-2 or exposed to other infected people, irrespective of their symptoms.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-13</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country.  Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival &amp; traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% &amp; 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.</jats:p></p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Aerodynamic Characteristics and RNA Concentration of SARS-CoV-2 Aerosol in Wuhan Hospitals during COVID-19 Outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">Background: The ongoing outbreak of COVID-19 has spread rapidly and sparked global concern. While the transmission of SARS-CoV-2 through human respiratory droplets and contact with infected persons is clear, the aerosol transmission of SARS-CoV-2 has been little studied. Methods: Thirty-five aerosol samples of three different types (total suspended particle, size segregated and deposition aerosol) were collected in Patient Areas (PAA) and Medical Staff Areas (MSA) of Renmin Hospital of Wuhan University (Renmin) and Wuchang Fangcang Field Hospital (Fangcang), and Public Areas (PUA) in Wuhan, China during COVID-19 outbreak. A robust droplet digital polymerase chain reaction (ddPCR) method was employed to quantitate the viral SARS-CoV-2 RNA genome and determine aerosol RNA concentration. Results: The ICU, CCU and general patient rooms inside Renmin, patient hall inside Fangcang had undetectable or low airborne SARS-CoV-2 concentration but deposition samples inside ICU and air sample in Fangcang patient toilet tested positive. The airborne SARS-CoV-2 in Fangcang MSA had bimodal distribution with higher concentration than those in Renmin during the outbreak but turned negative after patients number reduced and rigorous sanitization implemented. PUA had undetectable airborne SARS-CoV-2 concentration but obviously increased with accumulating crowd flow. Conclusions: Room ventilation, open space, proper use and disinfection of toilet can effectively limit aerosol transmission of SARS-CoV-2. Gathering of crowds with asymptomatic carriers is a potential source of airborne SARS-CoV-2. The virus aerosol deposition on protective apparel or floor surface and their subsequent resuspension is a potential transmission pathway and effective sanitization is critical in minimizing aerosol transmission of SARS-CoV-2.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>The response of Milan's Emergency Medical System to the COVID-19 outbreak in Italy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The number of people infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus causing coronavirus disease 2019 (COVID-19), is dramatically increasing worldwide.The first person-to-person transmission in Italy was reported on Feb 21, 2020, and led to an infection chain that represents the largest COVID-19 outbreak outside Asia to date. Here we document the response of the Emergency Medical System (EMS) of the metropolitan area of Milan, Italy, to the COVID-19 outbreak.On Jan 30, 2020, WHO declared the COVID-19 outbreak a public health emergency of international concern.2 Since then, the Italian Government has implemented extraordinary measures to restrict viral spread, including interruptions of air traffic from China, organised repatriation flights and quarantines for Italian travellers in China, and strict controls at international airports' arrival terminals. Local medical authorities adopted specific WHO recommendations to identify and isolate suspected cases of COVID-19.Such recommendations were addressed to patients presenting with respiratory symptoms and who had travelled to an endemic area in the previous 14 days or who had worked in the health-care sector, having been in close contact with patients with severe respiratory disease with unknown aetiology. Suspected cases were transferred to preselected hospital facilities where the SARS-CoV-2 test was available and infectious disease units were ready for isolation of confirmed cases.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the scale of COVID-19 Epidemic in the United States: Simulations Based on Air Traffic directly from Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Introduction: Coronavirus Disease 2019 (COVID-19) infection has been characterized by rapid spread and unusually large case clusters. It is important to have an estimate of the current state of COVID-19 epidemic in the U.S. to develop informed public health strategies.  Methods:  We estimated the potential scale of the COVID-19 epidemic (as of 03/01/2020) in the U.S. from cases imported directly from Wuhan area. We used simulations based on transmission dynamics parameters estimated from previous studies and air traffic data from Wuhan to the U.S and deliberately built our model based on conservative assumptions. Detection and quarantine of individual COVID-19 cases in the U.S before 03/01/2020 were also taken into account. We. A SEIR model was used to simulate the growth of the number of infected individuals in Wuhan area and in the U.S.  Results: With the most likely model, we estimated that there would be 9,484 infected cases (90%CI 2,054-24,241) as of 03/01/2020 if no intervention procedure had been taken to reduce the transmissibility in unidentified cases. Assuming current preventive procedures have reduced 25% of the transmissibility in unidentified cases, the number of infected cases would be 1,043 (90%CI 107-2,474). Conclusion: Our research indicates that, as of 03/01/2020., it is likely that there are already thousands of individuals in the US infected with SARS-CoV-2. Our model is dynamic and is available to the research community to further evaluate as the  situation becomes clearer.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Recurrence of positive SARS-CoV-2 RNA in COVID-19: A case report</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of COVID-19 that began in Wuhan, China, has constituted a Public Health Emergency of International Concern, with cases confirmed in multiple countries. Currently patients are the main source of infection. We report a confirmed case of COVID-19 whose oropharyngeal swab test of SARS-CoV-2 RNA turned positive in convalescence. This case highlights the importance of dynamic surveillance of SARS-CoV-2 RNA for infectivity assessment.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Positive result of Sars-Cov-2 in sputum from a cured patient with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">ince December 2019, an outbreak of the novel coronavirus (SARS-CoV-2) infection has spread rapidly in Wuhan, China [1]1. Over a month since the outbreak, more than 13,000 patients with COVID-19 have be cured and discharged from hospital until now. For the clinical cure criteria in China, twice successive negative results of Sars-Cov-2 nucleic acid detection are the important index, in addition to normal body temperature for 3 days as well as obvious improvement in respiratory symptoms and CT scan [2]. In the present work, we reported that Sars-Cov-2 nucleic acid was still detectable in sputum obtained by nebulization from a cured patient. On January 22, a 49-year-old man presented himself with fever for 4 days to a clinic. Throat swab detection was positive for SARS-CoV-2 nucleic acid by real-time RT-PCR. Subsequently, the patient was diagnosed with COVID-19 according to the diagnostic criteria [2] as follows (1): the positive result of SARS-CoV-2 nucleic acid detection (2); a history of short stay in Wuhan within 14 days; and (3) symptoms of fever, and multiple patchy areas of ground-glass opacity on CT scan. After the active treatment, the patient recovered from fever and other respiratory symptoms on February 4. On February 9 and February 10, the SARS-CoV-2 nucleic acid detection was successively negative in his throat swab samples. The CT scan result showed that the inflammation was significantly decreased in both lungs. Both the results of SARS-CoV-2 nucleic acid detection and CT scans indicated a recovery trend, and the patient was ready for discharge. On February 13, the throat swab and sputum by nebulization were collected before the patient was discharged. Notably, SARS-CoV-2 nucleic acid was still detected in sputum from the patient although negative result of throat swab detection.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical outcomes of 402 patients with COVID-2019 from a single center in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The SARS-CoV-2 outbreak is causing widespread infections and significant mortality. Previous studies describing clinical characteristics of the disease contained small cohorts from individual centers or larger series consisting of mixed cases from different hospitals. We report analyses of mortality and disease severity among 402 patients from a single hospital. The cohort included 297 patients with confirmed and 105 with suspected diagnosis. The latter group met the criteria for clinical diagnosis but nucleic acid tests results were initially interpreted as suspicious. Data were compared between genders and among different age groups. The overall case fatality is 5.2%. However, patients 70 years of age or older suffered a significantly higher mortality (17.8%), associated with more patients having severe or critical illness (57.5%). Patients 50 years of age or older had a mortality of 8.0%, and those younger than 50 years, 1.2%. Male patients had a mortality of 7.6% versus 2.9% in females.</jats:p></p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of novel coronavirus’s spike protein solved</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">n a piece of research accomplished with astonishing speed, scientists from the University of Texas at Austin and the National Institutes of Health have released a cryo-electron microscopy (cryo-EM) structure of part of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the novel coronavirus that has infected tens of thousands of people and killed more than 2,000 since the end of December (Science 2020, DOI: 10.1126/science.abb2507). The part of the virus imaged, called the spike protein, helps the virus attach to and infect human cells, and the determination of its structure comes just weeks after the virus’s genome sequence was published. The breakthrough is a huge step toward developing a vaccine against the virus as well as treatments for COVID-19, the disease that it causes, the researchers say. UT Austin’s Jason McLellan and his colleagues, who have spent many years studying other coronaviruses, had already figured out how to use select</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Has China faced only a herald wave of SARS-CoV-2?</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The attack rate of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) calculated by mathematical models, from estimates of the basic reproduction number, R0, of 2–3, suggests that 50–60% of the population should eventually be infected because the population seems to be entirely naive to the new virus.1 The observed attack rate on board the Diamond Princess cruise ship remained slightly below 20% (705 of 3711 passengers and crew members became infected).1 It is of upmost importance to know whether the SARS-CoV-2 outbreak in China is subsiding, as local authorities and the entire international community might wish. With 80 026 COVID-19 cases officially reported from China as of March 2, 2020,2 the proportion of the population affected remains far from 50%, or even 20%, of China's 1·4 billion people. Has China just experienced a herald wave, to use terminology borrowed from those who study tsunamis, and is the big wave still to come? Serosurveys can help answer these questions precisely.3 To serosurvey the outbreak would involve testing sera of blood samples from the most representative sample of the population at the epicentre of the epidemic, Wuhan. Serology analysis with neutralising antibodies from the 1000 people could allow for the rate of SARS-CoV-2 infections to be estimated with good accuracy. This rate could be extrapolated to the city's entire population and thus inform more precisely whether the provisional attack rate during this period was a few cases per thousand or perhaps affected 1–2% of the population, 20%, or more. Serosurveys should be seen as polls before elections; they can be repeated several times,3 week after week, to monitor the epidemic precisely. There is no reason to wait for the end of the epidemic before doing serosurveys. The results would be tremendously informative to China, first and foremost, and to the entire international community, on the risk of big secondary epidemic waves</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19.  Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding SARS-CoV-2-Mediated Inflammatory Responses: From Mechanisms to Potential Therapeutic Tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Currently there is no effective antiviral therapy for SARS-CoV-2 infection, which frequently leads to fatal inflammatory responses and acute lung injury. Here, we discuss the various mechanisms of SARS-CoV-mediated inflammation. We also assume that SARS-CoV-2 likely shares similar inflammatory responses. Potential therapeutic tools to reduce SARS-CoV-2-induced inflammatory responses include various methods to block FcR activation. In the absence of a proven clinical FcR blocker, the use of intravenous immunoglobulin to block FcR activation may be a viable option for the urgent treatment of pulmonary inflammation to prevent severe lung injury. Such treatment may also be combined with systemic anti-inflammatory drugs or corticosteroids. However, these strategies, as proposed here, remain to be clinically tested for effectiveness.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Intensive care during the coronavirus epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As a response to the epidemic, the local government had appointed several designated hospitals for patients with SARS-CoV-2 infection. Despite a common coping strategy for mass casualty (earthquake and blast injury) in China, SARI epidemic has proposed a new challenge for healthcare workers, especially intensivists. About 15–20% of suspected and confirmed patients with SARS-CoV-2 infection in fever clinics developed severe hypoxemia (since the second week of disease course), and required some form of ventilatory support such as high-flow nasal cannula, and non-invasive and invasive mechanical ventilation. In addition, other complications might occur, including, but not limited to, shock, acute kidney injury, gastrointestinal bleeding, and rhabdomyolysis. No antiviral agents have been proven to be effective against the coronavirus. Therefore, management of critically ill patients with SARS-CoV-2 infection still remains supportive rather than definitive, indicating remarkable workload for intensive care physicians and nurses. This surge of critically ill patients in designated hospitals as well as fever clinics represents urgent demands for intensive care with regards to space, supplies, and staff (Table 1) [5,6,7,8]. Response to these demands requires cooperation between the medical rescue team, infection control specialists, local health authorities, and center for disease control and prevention [9].ER -</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Sensitive one-step isothermal detection of pathogen-derived RNAs</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-09</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreaks of Ebola, Zika, MERS, and SARS-CoV-2 (2019-nCoV) require fast, simple, and sensitive onsite nucleic acid diagnostics that can be developed rapidly to prevent the spread of diseases. We have developed a SENsitive Splint-based one-step isothermal RNA detection (SENSR) method for rapid and straightforward onsite detection of pathogen RNAs with high sensitivity and specificity. SENSR consists of two simple enzymatic reactions: a ligation reaction by SplintR ligase and subsequent transcription by T7 RNA polymerase. The resulting transcript forms an RNA aptamer that induces fluorescence. Here, we demonstrate that SENSR is an effective and highly sensitive method for the detection of the current epidemic pathogen, <jats:italic>severe acute respiratory syndrome-related coronavirus 2</jats:italic> (SARS-CoV-2). We also show that the platform can be extended to the detection of five other pathogens. Overall, SENSR is a molecular diagnostic method that can be developed rapidly for onsite uses requiring high sensitivity, specificity, and short assaying times.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Although the SARS-CoV-2 viral load detection of respiratory specimen has been widely used for novel coronavirus disease (COVID-19) diagnosis, it is undeniable that serum SARS-CoV-2 nucleic acid (RNAaemia) could be detected  in  a fraction of the COVID-19 patients. However, it is not clear that if the incidence of RNAaemia could be correlated with the occurrence of cytokine storm or with the specific class of patients.  Methods: This study enrolled 48 patients with COVID-19 admitted to the General Hospital of Central Theater Command, PLA, a designated hospital in Wuhan, China. The patients were divided into three groups according to the Diagnosis and Treatment of New Coronavirus Pneumonia (version 6) published by the National Health Commission of China. The clinical and laboratory data were collected. The serum viral load detection and serum IL-6 levels were determined. Except for routine statistical analysis, Generalized Linear Models (GLMs) analysis was used to establish a patient status prediction model based on real-time RT-PCR Ct value. Findings: The Result showed that cases with RNAaemia were exclusively confirmed in critically ill patients group and appeared to reflect the illness severity. Further more, the inflammatory cytokine IL-6 levels were significantly elevated in critically ill patients, which is almost 10-folds higher than those in other patients. More importantly, the extremely high IL-6 level was closely correlated with the incidence of RNAaemia (R=0.902) and the vital signs of COVID-19 patients (R= -0.682).  Interpretation:    Serum SARS-CoV-2  viral load (RNAaemia) is strongly associated with cytokine storm and can be used to  predict the  poor prognosis of COVID-19 patients. Moreover, our results strongly suggest that cytokine IL-6 should be considered as a therapeutic target in critically ill patients with excessive inflammatory response.</jats:p></p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has spread globally with sustained human-to-human transmission outside China.To report the initial experience in Singapore with the epidemiologic investigation of this outbreak, clinical features, and management.Descriptive case series of the first 18 patients diagnosed with polymerase chain reaction (PCR)–confirmed SARS-CoV-2 infection at 4 hospitals in Singapore from January 23 to February 3, 2020; final follow-up date was February 25, 2020.Confirmed SARS-CoV-2 infection.Clinical, laboratory, and radiologic data were collected, including PCR cycle threshold values from nasopharyngeal swabs and viral shedding in blood, urine, and stool. Clinical course was summarized, including requirement for supplemental oxygen and intensive care and use of empirical treatment with lopinavir-ritonavir.Among the 18 hospitalized patients with PCR-confirmed SARS-CoV-2 infection (median age, 47 years; 9 [50%] women), clinical presentation was an upper respiratory tract infection in 12 (67%), and viral shedding from the nasopharynx was prolonged for 7 days or longer among 15 (83%). Six individuals (33%) required supplemental oxygen; of these, 2 required intensive care. There were no deaths. Virus was detectable in the stool (4/8 [50%]) and blood (1/12 [8%]) by PCR but not in urine. Five individuals requiring supplemental oxygen were treated with lopinavir-ritonavir. For 3 of the 5 patients, fever resolved and supplemental oxygen requirement was reduced within 3 days, whereas 2 deteriorated with progressive respiratory failure. Four of the 5 patients treated with lopinavir-ritonavir developed nausea, vomiting, and/or diarrhea, and 3 developed abnormal liver function test results.Among the first 18 patients diagnosed with SARS-CoV-2 infection in Singapore, clinical presentation was frequently a mild respiratory tract infection. Some patients required supplemental oxygen and had variable clinical outcomes following treatment with an antiretroviral agent.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19—New Insights on a Rapidly Changing Epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Since first reported in Wuhan, China, in late December 2019, the outbreak of the novel coronavirus now known as SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) has spread globally. As of February 27, 2020, more than 82 000 cases of coronavirus disease 2019 (COVID-19) (the disease caused by SARS-CoV-2) and 2800 deaths have been reported, of which approximately 95% of cases and 97% of deaths are in China. Cases have now been reported in 49 other countries. A particularly large outbreak occurred among the passengers and crew of the Diamond Princess cruise ship, where more than 700 infections are reported.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Characteristics of SARS-CoV-2 Pneumonia Compared to Controls in Chinese Han Population</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in Chinese Han population is limited. Our objective is to describe the epidemiological and clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in the Chinese Han population. Methods Retrospective, multi-center case series of the 69 consecutive hospitalized patients with confirmed SARS-CoV-2 pneumonia, from February 7 to February 28, 2020; final date of follow-up was February 29, 2020. Results The study population included 69 hospitalized patients with confirmed SARS-CoV-2 pneumonia without comorbidities and 14,117 normal controls. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. Compared with mild patients (n = 44), severe or critical patients (n = 25) were significantly older (median age, 67 years [IQR, 58-79] vs. 49 years [IQR, 36-60]; P &lt; 0.01). Fever was present in 98.6% of the patients. The second most common symptom was cough (62.3%), fatigue (58.0%), sputum (39.1%), and headache (33.3%). The median incubation period was 4 days (IQR, 2 to 7). Leukocyte count was 74.1% of normal controls and lymphocyte count was 45.9% of normal controls. The phenomenon of lymphocyte depletion (PLD) observed in severe or critical cases in 100%. Levels of lactate dehydrogenase, D-dimer, procalcitonin, and interleukin-6 were showed significant differences between mild and severe or critical cases. Chest computed tomographic scans showed bilateral patchy patterns (49.3%), local patchy shadowing (29.0%), and ground glass opacity (21.7%). 7.3% patients were diagnosed ARDS, 7.3% patients were diagnosed acute cardiac injury (troponin I &gt;28 pg/mL) and 4.4% patients were diagnosed fungal infections or shock. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Conclusions In this multicenter case series of 69 patients without comorbidities, the full spectrum of asymptomatic, mild, severe, and critical cases is described. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Among the 25 patients with severe or critical disease, 12.0% patients were underwent non-invasive mechanical ventilation, 8.0% patients underwent invasive mechanical ventilation, and 4.0% patients died.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using 'Feidian', Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in Chinese Han population is limited. Our objective is to describe the clinical characteristics of SARS-CoV-2 pneumonia compared to normal controls in the Chinese Han population. In this case series of 752 patients, the full spectrum of cases is described. Fever was present in 86-90% of the patients. The second most common symptom was cough (49.1-51.0%), fatigue (25.2-27.1%), sputum (20.0-23.1%), and headache (9.8-11.1%). the mortality rate is 4.6% in Wuhan, 1.9% in Beijing, and 0.9% in Shanghai. Our findings showed that the levels of lymphocytes were 0.8（IQR, 0.6-1.1）109/L in Wuhan, 1.0（IQR, 0.7-1.4）109/L in Beijing, and 1.1 (IQR, 0.8-1.5) 109/L in Shanghai before admission to hospitals, respectively, indicating that cellular immune function might relate to the mortality. Based on the reference ranges of normal Chinese Han population and the data of the critically ill patients we have observed, it is recommended that reference ranges of people at high risk of COVID-19 infection are CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500 cells/mm3, and CD8+ lymphocytes below 300 cells/mm3.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-27</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2.  Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-08</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: In coronavirus disease 2019 (COVID-19), current case definitions presume mainly lower respiratory tract infection. However, cases seen outside the epicenter of the epidemic may differ in their overall clinical appearance due to more sensitive case finding. Methods: We studied viral load courses by RT-PCR in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases. Infectious virus was detected by cell culture. Active replication was demonstrated by analysis of viral subgenomic replicative intermediates. Serology including neutralization testing was done to characterize immune response. Results: Seven cases had upper respiratory tract disease. Lower respiratory tract symptoms seen in two cases were limited. Clinical sensitivity of RT-PCR on swabs taken on days 1-5 of symptoms was 100%, with no differences comparing swab and sputum samples taken simultaneously. Average viral load was 6.76x10E5 copies per swab during the first 5 days. Live virus isolates were obtained from swabs during the first week of illness. Proof of active viral replication in upper respiratory tract tissues was obtained by detection of subgenomic viral RNA. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after about one week. Conclusions: The present study shows that COVID-19 can often present as a common cold-like illness. SARS-CoV-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool. These findings suggest adjustments of current case definitions and re-evaluation of the prospects of outbreak containment.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Study on SARS-COV-2 transmission and the effects of control measures in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-18</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Lessons for managing high-consequence infections from first COVID-19 cases in the UK</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">These first UK cases of COVID-19 raise important points about the management of cases of HCID in England. The decision to test for SARS-CoV-2 is based on a clinical and epidemiological case definition, and testing is only approved if this is met. When tested, neither of these people clearly met the current case definition, and had criteria been strictly applied, testing might not have been done. A decision to test was made because of high clinical suspicion and in response to latest available information about the distribution of infection. It is important that testing is appropriately targeted, and this is best done by applying clear case definitions. However, with any newly emerging infection, case definitions must evolve rapidly as information accrues. There should also be room for flexibility on the basis of discussion with clinical and public health experts</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomic variations of SARS-CoV-2 suggest multiple outbreak sources of transmission</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>We examined 169 genomes of SARS-CoV-2 and found that they can be classified into two major genotypes, Type I and Type II. Type I can be further divided into Type IA and IB. Our phylogenetic analysis showed that the Type IA resembles the ancestral SARS-CoV-2 most. Type II was likely evolved from Type I and predominant in the infections. Our results suggest that Type II SARS-CoV-2 was the source of the outbreak in the Wuhan Huanan market and it was likely originated from a super-spreader. The outbreak caused by the Type I virus should have occurred somewhere else, because the patients had no direct link to the market. Furthermore, by analyzing three genomic sites that distinguish Type I and Type II strains, we found that synonymous changes at two of the three sites confer higher protein translational efficiencies in Type II strains than in Type I strains, which might explain why Type II strains are predominant, implying that Type II is more contagious (transmissible) than Type I. These findings could be valuable for the current epidemic prevention and control.</jats:p></p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>Machine learning-based CT radiomics model for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection: A multicenter study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-03</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Abstract Objectives To develop and test machine learning-based CT radiomics models for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection. Design Cross-sectional Setting Multicenter Participants A total of 52 patients with laboratory-confirmed SARS-CoV-2 infection and their initial CT images were enrolled from 5 designated hospitals in Ankang, Lishui, Zhenjiang, Lanzhou, and Linxia between January 23, 2020 and February 8, 2020. As of February 20, patients remained in hospital or with non-findings in CT were excluded. Therefore, 31 patients with 72 lesion segments were included in the final analysis.  Intervention CT radiomics models based on logistic regression (LR) and random forest (RF) were developed on features extracted from pneumonia lesions in training and inter-validation datasets. The predictive performance was further evaluated in test dataset on lung lobe- and patients-level. Main outcomes Short-term hospital stay (≤10 days) and long-term hospital stay (&gt;10 days). Results The CT radiomics models based on 6 second-order features were effective in discriminating short- and long-term hospital stay in patients with pneumonia associated with SARS-CoV-2 infection, with areas under the curves of 0.97 (95%CI 0.83-1.0) and 0.92 (95%CI 0.67-1.0) by LR and RF, respectively, in the test dataset. The LR model showed a sensitivity and specificity of 1.0 and 0.89, and the RF model showed similar performance with sensitivity and specificity of 0.75 and 1.0 in test dataset.  Conclusions  The machine learning-based CT radiomics models showed feasibility and accuracy for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease. AVAILABILITY: https://www.genomedetective.com/app/typingtool/cov. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Public health might be endangered by possible prolonged discharge of SARS-CoV-2 in stool</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The published data, which showed the COVID-19 patients with low digestive?manifestation, might be misleading. Case with negative URT test showed positive in?rectal scarab which challenge the isolation protocol.?As fomite transmission caused clusters of infection of SARS, adequate disinfection?operations should be adopted in SARS-CoV-2 outbreak.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The recent emergence of the novel, pathogenic SARS-coronavirus 2 (SARS-CoV-2) in China and its rapid national and international spread pose a global health emergency. Cell entry of coronaviruses depends on binding of the viral spike (S) proteins to cellular receptors and on S protein priming by host cell proteases. Unravelling which cellular factors are used by SARS-CoV-2 for entry might provide insights into viral transmission and reveal therapeutic targets. Here, we demonstrate that SARS-CoV-2 uses the SARS-CoV receptor ACE2 for entry and the serine protease TMPRSS2 for S protein priming. A TMPRSS2 inhibitor approved for clinical use blocked entry and might constitute a treatment option. Finally, we show that the sera from convalescent SARS patients cross-neutralized SARS-2-S-driven entry. Our results reveal important commonalities between SARS-CoV-2 and SARS-CoV infection and identify a potential target for antiviral intervention.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>The within-host viral kinetics of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into the cross-species evolution of 2019 novel coronavirus</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Recent study reported in this journal that the threats of continuous evolution and dissemination of 2019 human coronaviruses.1 Since its emergence in December 2019, a “seventh” member of the family of 2019 human coronavirus named “SARS-CoV-2” was responsible for an outbreak of coronavirus disease (COVID-19) in Wuhan, China.2 As of February 23, 2020, China had reported more than 77,042 confirmed cases of SARS-CoV-2, with 2,445 fatalities and counting (http://www.nhc.gov.cn). Strikingly, SARS-CoV-2 had been transmitted rapidly in more than 29 countries to date (https://www.who.int), including Asia, Europe, North America, Africa, and Oceania, posing serious concerns about its pandemic potential. Despite of droplet and contact transmissions of SARS-CoV-2, recent studies demonstrated that SARS-CoV-2 might be transmitted via aerosol and fecal–oral routes 3 (Figure 1), which needs to be paid attention in particular.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Severe SARS-CoV-2 infections: practical considerations and management strategy for intensivists</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Etiological agent and epidemiology The new agent causing this pneumonia, a coronavirus (SARS-CoV-2), was identified and sequenced [3] and diagnostic tests were developed [4]. On January 30, 2020, the World Health Organization issued a worldwide public health alert on the emergence of a new epidemic viral disease. On February 3, 2020, 17,391 confirmed cases (153 cases outside of China) have been reported (https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/). The overall mortality rate of affected patients is difficult to assess at this time, because of the lack of a reliable denominator. Severe forms represent 14% of the reported cases, and the overall mortality is around 2% of the confirmed cases. To date, 153 cases have been reported in 23 countries outside China (overall, 24 cases in Europe), most of them being imported cases: tourists coming from China, or China-originating persons returning to their country of residence after traveling to visit family in Wuhan or other Chinese regions. In Europe, at least three cases in Germany and one case in France involved patients with no history of travel to China. The German case occurred after exposure to an asymptomatic contact coming from China [5].AU - Lucet, Jean-Christophe</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Hypokalemia and Clinical Implications in Patients with Coronavirus Disease 2019 (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-29</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>BACKGROUND: SARS-CoV-2 has caused a series of COVID-19 globally. SARS-CoV-2 binds angiotensin I converting enzyme 2 (ACE2) of renin-angiotensin system (RAS) and causes prevalent hypokalemia METHODS: The patients with COVID-19 were classified into severe hypokalemia, hypokalemia, and normokalemia group. The study aimed to determine the relationship between hypokalemia and clinical features, the underlying causes and clinical implications of hypokalemia.  RESULTS: By Feb 15, 2020, 175 patients with COVID-19 (92 women and 83 men; median age, 46 [IQR, 34-54] years) were admitted to hospital in Wenzhou, China, consisting 39 severe hypokalemia-, 69 hypokalemia-, and 67 normokalemia patients.  Gastrointestinal symptoms were not associated with hypokalemia among 108 hypokalemia patients (P&gt;0.05).  Body temperature, CK, CK-MB, LDH, and CRP were significantly associated with the severity of hypokalemia (P&lt;0.01). 93% of severe and critically ill patients had hypokalemia which was most common among elevated CK, CK-MB, LDH, and CRP. Urine K+ loss was the primary cause of hypokalemia.  severe hypokalemia patients was given 3 g/day, adding up to an average of 34 (SD=4) g potassium during hospital stay. The exciting finding was that patients responded well to K+ supplements when they were inclined to recovery.  CONCLUSIONS: Hypokalemia is prevailing in patients with COVID-19. The correction of hypokalemia is challenging because of continuous renal K+ loss resulting from the degradation of ACE2. The end of urine K+ loss indicates a good prognosis and may be a reliable, in-time, and sensitive biomarker directly reflecting the end of adverse effect on RAS system.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Covert COVID-19 and false-positive dengue serology in Singapore</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Dengue and coronavirus disease 2019 (COVID-19) are difficult to distinguish because they have shared clinical and laboratory features.1 , 2 We describe two patients in Singapore with false-positive results from rapid serological testing for dengue, who were later confirmed to have severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, the causative virus of COVID-19. The first case is a 57-year-old man with no relevant past medical, travel, or contact history, who presented to a regional hospital on Feb 9, 2020, with 3 days of fever and cough. He had thrombocytopenia (platelet count 140 × 109/mL) and a normal chest radiograph. He was discharged after a negative rapid test for dengue NS1, IgM, and IgG (SD Bioline Dengue Duo Kit; Abbott, South Korea). He returned to a public primary health-care clinic with persistent fever, worsening thrombocytopenia (89 × 109/mL), and new onset lymphopenia (0·43 × 109/mL). A repeat dengue rapid test was positive for dengue IgM and IgG (Dengue Combo; Wells Bio, South Korea). He was referred to hospital for dengue with worsening cough and dyspnoea. A chest radiograph led to testing for SARS-CoV-2 by RT-PCR (in-house laboratory-developed test detecting the N and ORF1ab genes) from a nasopharyngeal swab, which returned positive. The original seropositive sample and additional urine and blood samples tested negative for dengue, chikungunya, and Zika viruses by RT-PCR,3 , 4 , 5 and a repeat dengue rapid test (SD Bioline) was also negative. Thus, the initial dengue seroconversion result was deemed a false positive.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Active Monitoring of Persons Exposed to Patients with Confirmed COVID-19 — United States, January–February 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, an outbreak of coronavirus disease 2019 (COVID-19), caused by the virus SARS-CoV-2, began in Wuhan, China (1). The disease spread widely in China, and, as of February 26, 2020, COVID-19 cases had been identified in 36 other countries and territories, including the United States. Person-to-person transmission has been widely documented, and a limited number of countries have reported sustained person-to-person spread.* On January 20, state and local health departments in the United States, in collaboration with teams deployed from CDC, began identifying and monitoring all persons considered to have had close contact† with patients with confirmed COVID-19 (2). The aims of these efforts were to ensure rapid evaluation and care of patients, limit further transmission, and better understand risk factors for transmission.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: combining antiviral and anti-inflammatory treatments</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Both coronavirus disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS) are characterised by an overexuberant inflammatory response and, for SARS, viral load is not correlated with the worsening of symptoms. In our previous Correspondence to The Lancet, we described how BenevolentAI's proprietary artificial intelligence (AI)-derived knowledge graph, queried by a suite of algorithms, enabled identification of a target and a potential therapeutic against SARS coronavirus 2 (SARS-CoV-2; the causative organism in COVID-19). We identified a group of approved drugs that could inhibit clathrin-mediated endocytosis and thereby inhibit viral infection of cells (appendix). The drug targets are members of the numb-associated kinase (NAK) family—including AAK1 and GAK—the inhibition of which has been shown to reduce viral infection in vitro. Baricitinib was identified as a NAK inhibitor, with a particularly high affinity for AAK1, a pivotal regulator of clathrin-mediated endocytosis. We suggested that this drug could be of use in countering SARS-CoV-2 infections, subject to appropriate clinical testing.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Guidelines for pregnant women with suspected SARS-CoV-2 infection</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Coronaviruses responsible for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) can cause severe adverse pregnancy outcomes, such as miscarriage, premature delivery, intrauterine growth restriction, and maternal death.1 , 2 Vertical transmission of the virus responsible for 2019 novel coronavirus disease (COVID-19), severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has not yet been detected, whereas perinatal transmission has been suspected in one case.3 Consequences of infection with SARS-CoV-2 for pregnancies are uncertain, with no evidence so far of severe outcomes for mothers and infants; however, the possibility should be considered.4 The recent experience with Zika virus suggests that when a new pathogen emerges, the health-care community should be prepared for the worst-case scenario.5 Therefore, recommendations for management of pregnant women at risk of SARS-CoV-2 infection are urgently needed. To this end, we propose a detailed management algorithm for health-care providers (appendix).</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Escalating infection control response to the rapidly evolving epidemiology of the Coronavirus disease 2019 (COVID-19) due to SARS-CoV-2 in Hong Kong</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Background:To describe the infection control preparedness for Coronavirus Disease (COVID-19) due to SARS-CoV-2 [previously known as 2019-novel coronavirus] in the first 42 days after announcement of a cluster of pneumonia in China, on 31 December 2019 (day 1) in Hong Kong.Methods:A bundle approach of active and enhanced laboratory surveillance, early airborne infection isolation, rapid molecular diagnostic testing, and contact tracing for healthcare workers (HCWs) with unprotected exposure in the hospitals was implemented. Epidemiological characteristics of confirmed cases, environmental and air samples were collected and analyzed.Results:From day 1 to day 42, forty-two (3.3%) of 1275 patients fulfilling active (n=29) and enhanced laboratory surveillance (n=13) confirmed to have SARS-CoV-2 infection. The number of locally acquired case significantly increased from 1 (7.7%) of 13 [day 22 to day 32] to 27 (93.1%) of 29 confirmed case [day 33 to day 42] (p&lt;0.001). Twenty-eight patients (66.6%) came from 8 family clusters. Eleven (2.7%) of 413 HCWs caring these confirmed cases were found to have unprotected exposure requiring quarantine for 14 days. None of them was infected and nosocomial transmission of SARS-CoV-2 was not observed. Environmental surveillance performed in a patient with viral load of 3.3x106 copies/ml (pooled nasopharyngeal/ throat swab) and 5.9x106 copies/ml (saliva) respectively. SARS-CoV-2 revealed in 1 (7.7%) of 13 environmental samples, but not in 8 air samples collected at a distance of 10 cm from patient's chin with or without wearing a surgical mask.Conclusion:Appropriate hospital infection control measures could prevent nosocomial transmission of SARS-CoV-2.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /></body>