
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: physical science of the coronavirus</h2><br /></div><br /><br /><br /><div class="card"><h3>The Coronavirus Nucleocapsid Is a Multifunctional Protein</h3>
<p style="text-align: justify; text-justify: inter-word;">Abstract: The coronavirus nucleocapsid (N) is a structural protein that forms complexes with genomic RNA, interacts with the viral membrane protein during virion assembly and plays a critical role in enhancing the efficiency of virus transcription and assembly. Recent studies have confirmed that N is a multifunctional protein. The aim of this review is to highlight the properties and functions of the N protein, with specific reference to (i) the topology; (ii) the intracellular localization and (iii) the functions of the protein.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</h3>
<p style="text-align: justify; text-justify: inter-word;">the coronavirus, responsible for host cell attachment and mediating host cell membrane and viral membrane fusion during infection, is key to the viral life cycle and a major target for antiviral drugs and vaccines. In this study, we report the structures of different conformational states of the SARS-CoV S glycoprotein during virus entry. Specifically, we found that the S glycoprotein retains the prefusion trimer structure after trypsin cleavage and low-pH treatment. Additionally, binding with host cell receptor ACE2 promotes the release of S1 subunits from the S trimer and triggers the pre-to postfusion conformational transition. Our results provide new insights for understanding the mechanisms involved in coronavirus S glycoprotein-mediated virus entry.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of the Role of Hexamer AGUAAA and Poly(A) Tail in Coronavirus Polyadenylation</h3>
<p style="text-align: justify; text-justify: inter-word;">Similar to eukaryotic mRNA, the positive-strand coronavirus genome of~30 kilobases is 5'capped and 3'-polyadenylated. It has been demonstrated that the length of the coronaviral poly(A) tail is not static but regulated during infection; however, little is known regarding the factors involved in coronaviral polyadenylation and its regulation. Here, we show that during infection, the level of coronavirus poly(A) tail lengthening depends on the initial length upon infection and that the minimum length to initiate lengthening may lie between 5 and 9 nucleotides. By mutagenesis analysis, it was found that (i) the hexamer AGUAAA and poly(A) tail are two important elements responsible for synthesis of the coronavirus poly(A) tail and may function in concert to accomplish polyadenylation and (ii) the function of the hexamer AGUAAA in coronaviral polyadenylation is position dependent. Based on these findings, we propose a process for how the coronaviral poly(A) tail is synthesized and undergoes variation. Our results provide the first genetic evidence to gain insight into coronaviral polyadenylation.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV NSP12 polymerase bound to NSP7 and NSP8 co-factors</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS-and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (NSP) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV NSP12 polymerase bound to its essential co-factors, NSP7 and NSP8, using single particle cryo-electron microscopy. NSP12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is unexpectedly bound by two NSP8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into NSP12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>A Single Polar Residue and Distinct Membrane Topologies Impact the Function of the Infectious Bronchitis Coronavirus E Protein</h3>
<p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein with a single predicted hydrophobic domain (HD), and has a poorly defined role in infection. The E protein is thought to promote virion assembly, which occurs in the Golgi region of infected cells. It has also been implicated in the release of infectious particles after budding. The E protein has ion channel activity in vitro, although a role for channel activity in infection has not been established. Furthermore, the membrane topology of the E protein is of considerable debate, and the protein may adopt more than one topology during infection. We previously showed that the HD of the infectious bronchitis virus (IBV) E protein is required for the efficient release of infectious virus, an activity that correlated with disruption of the secretory pathway. Here we report that a single residue within the hydrophobic domain, Thr16, is required for secretory pathway disruption. Substitutions of other residues for Thr16 were not tolerated. Mutations of Thr16 did not impact virus assembly as judged by virus-like particle production, suggesting that alteration of secretory pathway and assembly are independent activities. We also examined how the membrane topology of IBV E affected its function by generating mutant versions that adopted either a transmembrane or membrane hairpin topology. We found that a transmembrane topology was required for disrupting the secretory pathway, but was less efficient for virus-like particle production. The hairpin version of E was unable to disrupt the secretory pathway or produce particles. The findings reported here identify properties of the E protein that are important for its function, and provide insight into how the E protein may perform multiple roles during infection.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and Inhibition of the SARS Coronavirus Envelope Protein Ion Channel</h3>
<p style="text-align: justify; text-justify: inter-word;">The envelope (E) protein from coronaviruses is a small polypeptide that contains at least one a-helical transmembrane domain. Absence, or inactivation, of E protein results in attenuated viruses, due to alterations in either virion morphology or tropism. Apart from its morphogenetic properties, protein E has been reported to have membrane permeabilizing activity. Further, the drug hexamethylene amiloride (HMA), but not amiloride, inhibited in vitro ion channel activity of some synthetic coronavirus E proteins, and also viral replication. We have previously shown for the coronavirus species responsible for severe acute respiratory syndrome (SARS-CoV) that the transmembrane domain of E protein (ETM) forms pentameric a-helical bundles that are likely responsible for the observed channel activity. Herein, using solution NMR in dodecylphosphatidylcholine micelles and energy minimization, we have obtained a model of this channel which features regular a-helices that form a pentameric left-handed parallel bundle. The drug HMA was found to bind inside the lumen of the channel, at both the C-terminal and the N-terminal openings, and, in contrast to amiloride, induced additional chemical shifts in ETM. Full length SARS-CoV E displayed channel activity when transiently expressed in human embryonic kidney 293 (HEK-293) cells in a whole-cell patch clamp set-up. This activity was significantly reduced by hexamethylene amiloride (HMA), but not by amiloride. The channel structure presented herein provides a possible rationale for inhibition, and a platform for future structure-based drug design of this potential pharmacological target.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent history is punctuated by the emergence of highly pathogenic coronaviruses such as SARS-and MERS-CoV into human circulation. Upon infecting host cells, coronaviruses assemble a multi-subunit RNA-synthesis complex of viral non-structural proteins (nsp) responsible for the replication and transcription of the viral genome. Here, we present the 3.1 Å resolution structure of the SARS-CoV nsp12 polymerase bound to its essential co-factors, nsp7 and nsp8, using single particle cryo-electron microscopy. nsp12 possesses an architecture common to all viral polymerases as well as a large N-terminal extension containing a kinase-like fold and is bound by two nsp8 co-factors. This structure illuminates the assembly of the coronavirus core RNA-synthesis machinery, provides key insights into nsp12 polymerase catalysis and fidelity and acts as a template for the design of novel antiviral therapeutics.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>The 39-Terminal 55 Nucleotides of Bovine Coronavirus Defective Interfering RNA Harbor Cis-Acting Elements Required for Both Negative-and Positive-Strand RNA Synthesis</h3>
<p style="text-align: justify; text-justify: inter-word;">The synthesis of the negative-strand [(2)-strand] complement of the ,30 kilobase, positive-strand [(+)-strand] coronaviral genome is a necessary early step for genome replication. The identification of cis-acting elements required for (2)-strand RNA synthesis in coronaviruses, however, has been hampered due to insufficiencies in the techniques used to detect the (2 )-strand RNA species. Here, we employed a method of head-to-tail ligation and real-time quantitative reverse transcription polymerase chain reaction (qRT-PCR) to detect and quantitate the synthesis of bovine coronavirus (BCoV) defective interfering (DI) RNA (2) strands. Furthermore, using the aforementioned techniques along with Northern blot assay, we specifically defined the cis-acting RNA elements within the 39-terminal 55 nucleotides (nts) which function in the synthesis of (2)-or (+)-strand BCoV DI RNA. The major findings are as follows: (i) nts from -5 to -39 within the 39-terminal 55 nts are the cis-acting elements responsible for (2)-strand BCoV DI RNA synthesis, (ii) nts from 23 to 234 within the 39-terminal 55 nts are cis-acting elements required for (+)-strand BCoV DI RNA synthesis, and (iii) the nucleotide species at the 39-most position (21) is important, but not critical, for both (2)-and (+)-strand BCoV DI RNA synthesis. These results demonstrate that the 39-terminal 55 nts in BCoV DI RNA harbor cis-acting RNA elements required for both (2)-and (+)-strand DI RNA synthesis and extend our knowledge on the mechanisms of coronavirus replication. The method of head-to-tail ligation and qRT-PCR employed in the study may also be applied to identify other cis-acting elements required for (2)-strand RNA synthesis in coronaviruses.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Biogenesis and Dynamics of the Coronavirus Replicative Structures</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-strand RNA viruses that are important infectious agents of both animals and humans. A common feature among positive-strand RNA viruses is their assembly of replication-transcription complexes in association with cytoplasmic membranes. Upon infection, coronaviruses extensively rearrange cellular membranes into organelle-like replicative structures that consist of double-membrane vesicles and convoluted membranes to which the nonstructural proteins involved in RNA synthesis localize. Double-stranded RNA, presumably functioning as replicative intermediate during viral RNA synthesis, has been detected at the double-membrane vesicle interior. Recent studies have provided new insights into the assembly and functioning of the coronavirus replicative structures. This review will summarize the current knowledge on the biogenesis of the replicative structures, the membrane anchoring of the replication-transcription complexes, and the location of viral RNA synthesis, with particular focus on the dynamics of the coronavirus replicative structures and individual replication-associated proteins.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultrastructural Characterization of SARS Coronavirus EMERGENCE OF SARS</h3>
<p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) was first described during a 2002-2003 global outbreak of severe pneumonia associated with human deaths and person-toperson disease transmission. The etiologic agent was initially identified as a coronavirus by thin-section electron microscopic examination of a virus isolate. Virions were spherical, 78 nm in mean diameter, and composed of a helical nucleocapsid within an envelope with surface projections. We show that infection with the SARS-associated coronavirus resulted in distinct ultrastructural features: double-membrane vesicles, nucleocapsid inclusions, and large granular areas of cytoplasm. These three structures and the coronavirus particles were shown to be positive for viral proteins and RNA by using ultrastructural immunogold and in situ hybridization assays. In addition, ultrastructural examination of a bronchiolar lavage specimen from a SARS patient showed numerous coronavirus-infected cells with features similar to those in infected culture cells. Electron microscopic studies were critical in identifying the etiologic agent of the SARS outbreak and in guiding subsequent laboratory and epidemiologic investigations.</p>
<p align="right"><i>score: 122</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus E Protein: Assembly and Beyond</h3>
<p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein that has an important role in the assembly of virions. Recent studies have indicated that the E protein has functions during infection beyond assembly, including in virus egress and in the host stress response. Additionally, the E protein has ion channel activity, interacts with host proteins, and may have multiple membrane topologies. The goal of this review is to highlight the properties and functions of the E protein, and speculate on how they may be related.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Acquisition of Cell-Cell Fusion Activity by Amino Acid Substitutions in Spike Protein Determines the Infectivity of a Coronavirus in Cultured Cells</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronavirus host and cell specificities are determined by specific interactions between the viral spike (S) protein and host cell receptor(s). Avian coronavirus infectious bronchitis (IBV) has been adapted to embryonated chicken eggs, primary chicken kidney (CK) cells, monkey kidney cell line Vero, and other human and animal cells. Here we report that acquisition of the cell-cell fusion activity by amino acid mutations in the S protein determines the infectivity of IBV in cultured cells. Expression of S protein derived from Vero-and CK-adapted strains showed efficient induction of membrane fusion. However, expression of S protein cloned from the third passage of IBV in chicken embryo (EP3) did not show apparent syncytia formation. By construction of chimeric S constructs and site-directed mutagenesis, a point mutation (L857-F) at amino acid position 857 in the heptad repeat 1 region of S protein was shown to be responsible for its acquisition of the cell-cell fusion activity. Furthermore, a G405-D point mutation in the S1 domain, which was acquired during further propagation of Vero-adapted IBV in Vero cells, could enhance the cell-cell fusion activity of the protein. Re-introduction of L857 back to the S gene of Vero-adapted IBV allowed recovery of variants that contain the introduced L857. However, compensatory mutations in S1 and some distant regions of S2 were required for restoration of the cell-cell fusion activity of S protein carrying L857 and for the infectivity of the recovered variants in cultured cells. This study demonstrates that acquisition of the cell-cell fusion activity in S protein determines the selection and/or adaptation of a coronavirus from chicken embryo to cultured cells of human and animal origins.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Viral Transmission Behavior via Protein Intrinsic Disorder Prediction: Coronaviruses</h3>
<p style="text-align: justify; text-justify: inter-word;">Besides being a common threat to farm animals and poultry, coronavirus (CoV) was responsible for the human severe acute respiratory syndrome (SARS) epidemic in 2002-4. However, many aspects of CoV behavior, including modes of its transmission, are yet to be fully understood. We show that the amount and the peculiarities of distribution of the protein intrinsic disorder in the viral shell can be used for the efficient analysis of the behavior and transmission modes of CoV. The proposed model allows categorization of the various CoVs by the peculiarities of disorder distribution in their membrane (M) and nucleocapsid (N). This categorization enables quick identification of viruses with similar behaviors in transmission, regardless of genetic proximity. Based on this analysis, an empirical model for predicting the viral transmission behavior is developed. This model is able to explain some behavioral aspects of important coronaviruses that previously were not fully understood. The new predictor can be a useful tool for better epidemiological, clinical, and structural understanding of behavior of both newly emerging viruses and viruses that have been known for a long time. A potentially new vaccine strategy could involve searches for viral strains that are characterized by the evolutionary misfit between the peculiarities of the disorder distribution in their shells and their behavior.</p>
<p align="right"><i>score: 118</i></p>

</div><br /><br /><br /><div class="card"><h3>Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells</h3>
<p style="text-align: justify; text-justify: inter-word;">The N-terminal domain of the coronavirus nucleocapsid (N) protein adopts a fold resembling a right hand with a flexible, positively charged b-hairpin and a hydrophobic palm. This domain was shown to interact with the genomic RNA for coronavirus infectious bronchitis virus (IBV) and severe acute respiratory syndrome coronavirus (SARS-CoV). Based on its 3D structure, we used site-directed mutagenesis to identify residues essential for the RNA-binding activity of the IBV N protein and viral infectivity. Alanine substitution of either Arg-76 or Tyr-94 in the N-terminal domain of IBV N protein led to a significant decrease in its RNA-binding activity and a total loss of the infectivity of the viral RNA to Vero cells. In contrast, mutation of amino acid Gln-74 to an alanine, which does not affect the binding activity of the N-terminal domain, showed minimal, if any, detrimental effect on the infectivity of IBV. This study thus identifies residues critical for RNA binding on the nucleocapsid surface, and presents biochemical and genetic evidence that directly links the RNA binding capacity of the coronavirus N protein to the viral infectivity in cultured cells. This information would be useful in development of preventive and treatment approaches against coronavirus infection.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>The Porcine Deltacoronavirus Replication Organelle Comprises Double-Membrane Vesicles and Zippered Endoplasmic Reticulum with Double-Membrane Spherules</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine deltacoronavirus (PDCoV) was first identified in Hong Kong in 2012 from samples taken from pigs in 2009. PDCoV was subsequently identified in the USA in 2014 in pigs with a history of severe diarrhea. The virus has now been detected in pigs in several countries around the world. Following the development of tissue culture adapted strains of PDCoV, it is now possible to address questions regarding virus-host cell interactions for this genera of coronavirus. Here, we presented a detailed study of PDCoV-induced replication organelles. All positive-strand RNA viruses induce the rearrangement of cellular membranes during virus replication to support viral RNA synthesis, forming the replication organelle. Replication organelles for the Alpha-, Beta-, and Gammacoronavirus genera have been characterized. All coronavirus genera induced the formation of double-membrane vesicles (DMVs). In addition, Alphaand Betacoronaviruses induce the formation of convoluted membranes, while Gammacoronaviruses induce the formation of zippered endoplasmic reticulum (ER) with tethered double-membrane spherules. However, the structures induced by Deltacoronaviruses, particularly the presence of convoluted membranes or double-membrane spherules, are unknown. Initially, the dynamics of PDCoV strain OH-FD22 replication were assessed with the onset of viral RNA synthesis, protein synthesis, and progeny particle release determined. Subsequently, virus-induced membrane rearrangements were identified in infected cells by electron microscopy. As has been observed for all other coronaviruses studied to date, PDCoV replication was found to induce the formation of double-membrane vesicles. Significantly, however, PDCoV replication was also found to induce the formation of regions of zippered endoplasmic reticulum, small associated tethered vesicles, and double-membrane spherules. These structures strongly resemble the replication organelle induced by avian Gammacoronavirus infectious bronchitis virus.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Modeling Evaluation of the Binding Effect of Ritonavir, Lopinavir and Darunavir to Severe Acute Respiratory Syndrome Coronavirus 2 Proteases</h3>
<p style="text-align: justify; text-justify: inter-word;">Three anti-HIV drugs, ritonavir, lopinavir and darunavir, might have therapeutic effect on coronavirus disease 2019 . In this study, the structure models of two severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteases, coronavirus endopeptidase C30 (CEP_C30) and papain like viral protease (PLVP), were built by homology modeling. Ritonavir, lopinavir and darunavir were then docked to the models, respectively, followed by energy minimization of the protease-drug complexes. In the simulations, ritonavir can bind to CEP_C30 most suitably, and induce significant conformation changes of CEP_C30; lopinavir can also bind to CEP_C30 suitably, and induce significant conformation changes of CEP_C30; darunavir can bind to PLVP suitably with slight conformation changes of PLVP. It is suggested that the therapeutic effect of ritonavir and lopinavir on COVID-19 may be mainly due to their inhibitory effect on CEP_C30, while ritonavir may have stronger efficacy; the inhibitory effect of darunavir on SARS-CoV-2 and its potential therapeutic effect may be mainly due to its inhibitory effect on PLVP.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>First two months of the 2019 Coronavirus Disease (COVID-19) epidemic in China: real- time surveillance and evaluation with a second derivative model</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. Methods: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. Results: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. Conclusions: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Characterization of an Immunodominant Epitope in the Endodomain of the Coronavirus Membrane Protein</h3>
<p style="text-align: justify; text-justify: inter-word;">The coronavirus membrane (M) protein acts as a dominant immunogen and is a major player in virus assembly. In this study, we prepared two monoclonal antibodies (mAbs; 1C3 and 4C7) directed against the transmissible gastroenteritis virus (TGEV) M protein. The 1C3 and 4C7 mAbs both reacted with the native TGEV M protein in western blotting and immunofluorescence (IFA) assays. Two linear epitopes, 243YSTEART249 (1C3) and 243YSTEARTDNLSEQEKLLHMV262 (4C7), were identified in the endodomain of the TGEV M protein. The 1C3 mAb can be used for the detection of the TGEV M protein in different assays. An IFA method for the detection of TGEV M protein was optimized using mAb 1C3. Furthermore, the ability of the epitope identified in this study to stimulate antibody production was also evaluated. An immunodominant epitope in the TGEV membrane protein endodomain was identified. The results of this study have implications for further research on TGEV replication.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Infectious Bronchitis Virus Nonstructural Protein 4 Alone Induces Membrane Pairing</h3>
<p style="text-align: justify; text-justify: inter-word;">Positive-strand RNA viruses, such as coronaviruses, induce cellular membrane rearrangements during replication to form replication organelles allowing for efficient viral RNA synthesis. Infectious bronchitis virus (IBV), a pathogenic avian Gammacoronavirus of significant importance to the global poultry industry, has been shown to induce the formation of double membrane vesicles (DMVs), zippered endoplasmic reticulum (zER) and tethered vesicles, known as spherules. These membrane rearrangements are virally induced; however, it remains unclear which viral proteins are responsible. In this study, membrane rearrangements induced when expressing viral non-structural proteins (nsps) from two different strains of IBV were compared. Three non-structural transmembrane proteins, nsp3, nsp4, and nsp6, were expressed in cells singularly or in combination and the effects on cellular membranes investigated using electron microscopy and electron tomography. In contrast to previously studied coronaviruses, IBV nsp4 alone is necessary and sufficient to induce membrane pairing; however, expression of the transmembrane proteins together was not sufficient to fully recapitulate DMVs. This indicates that although nsp4 is able to singularly induce membrane pairing, further viral or host factors are required in order to fully assemble IBV replicative structures. This study highlights further differences in the mechanism of membrane rearrangements between members of the coronavirus family.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>Dynamically-Driven Inactivation of the Catalytic Machinery of the SARS 3C-Like Protease by the N214A Mutation on the Extra Domain</h3>
<p style="text-align: justify; text-justify: inter-word;">Despite utilizing the same chymotrypsin fold to host the catalytic machinery, coronavirus 3C-like proteases (3CLpro) noticeably differ from picornavirus 3C proteases in acquiring an extra helical domain in evolution. Previously, the extra domain was demonstrated to regulate the catalysis of the SARS-CoV 3CLpro by controlling its dimerization. Here, we studied N214A, another mutant with only a doubled dissociation constant but significantly abolished activity. Unexpectedly, N214A still adopts the dimeric structure almost identical to that of the wild-type (WT) enzyme. Thus, we conducted 30-ns molecular dynamics (MD) simulations for N214A, WT, and R298A which we previously characterized to be a monomer with the collapsed catalytic machinery. Remarkably, three proteases display distinctive dynamical behaviors. While in WT, the catalytic machinery stably retains in the activated state; in R298A it remains largely collapsed in the inactivated state, thus implying that two states are not only structurally very distinguishable but also dynamically well separated. Surprisingly, in N214A the catalytic dyad becomes dynamically unstable and many residues constituting the catalytic machinery jump to sample the conformations highly resembling those of R298A. Therefore, the N214A mutation appears to trigger the dramatic change of the enzyme dynamics in the context of the dimeric form which ultimately inactivates the catalytic machinery. The present MD simulations represent the longest reported so far for the SARS-CoV 3CLpro, unveiling that its catalysis is critically dependent on the dynamics, which can be amazingly modulated by the extra domain. Consequently, mediating the dynamics may offer a potential avenue to inhibit the SARS-CoV 3CLpro.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /></body>