
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

.tooltip {
  position: relative;
  display: inline-block;
}

/* Tooltip text */
.tooltip .tooltiptext {
  visibility: hidden;
  width: 120px;
  background-color: black;
  color: #fff;
  text-align: center;
  padding: 5px 0;
  border-radius: 6px;
 
  /* Position the tooltip text - see examples below! */
  position: absolute;
  z-index: 1;
}

/* Show the tooltip text when you mouse over the tooltip container */
.tooltip:hover .tooltiptext {
  visibility: visible;
}

.tooltip .tooltiptext {
  width: 120px;
  bottom: 120%;
  left: 50%;
  margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
}

.tooltip .tooltiptext::after {
  content: " ";
  position: absolute;
  top: 100%; /* At the bottom of the tooltip */
  left: 50%;
  margin-left: -5px;
  border-width: 5px;
  border-style: solid;
  border-color: black transparent transparent transparent;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: using CRISPR to improve response times to COVID-19 and future diseases</h2><br /></div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.</jats:p></p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-23</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third zoonotic CoV has raised great public health concern. The demand for rapid and accurate diagnosis of this novel pathogen brought significant clinical and technological challenges. Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR (RT-PCR) are the most widely used molecular diagnostics. Methods: 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was analyzed and used for the design and development of an isothermal, CRISPR-based diagnostic for this novel virus. The diagnostic performance of CRISPR-nCoV was assessed and compared across three technology platforms (mNGS, RT-PCR and CRISPR). Results: 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral RNA was observed in clinical specimens. CRISPR-nCoV demonstrated a near single-copy sensitivity and great clinical sensitivity with a shorter turn-around time than RT-PCR. Conclusion: CRISPR-nCoV presents as a promising diagnostic option for the emerging pathogen.</jats:p></p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>The United States badly bungled coronavirus testing—but things may soon improve | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Speed is critical in the response to COVID-19. So why has the United States been so slow in its attempt to develop reliable diagnostic tests and use them widely? The World Health Organization (WHO) has shipped testing kits to 57 countries. China had five commercial tests on the market 1 month ago and can now do up to 1.6 million tests a week; South Korea has tested 65,000 people so far. The U. S. Centers for Disease Control and Prevention (CDC), in contrast, has done only 459 tests since the epidemic began. The rollout of a CDC-designed test kit to state and local labs has become a fiasco because it contained a faulty reagent. Labs around the country eager to test more suspected cases—and test them faster—have been unable to do so. No commercial or state labs have the approval to use their own tests. In what is already an infamous snafu, CDC initially refused a request to test a patient in Northern California who turned out to be the first probable COVID19 case without known links to an infected person.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Straining the System: Novel Coronavirus (COVID-19) and Preparedness for Concomitant Disasters</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Just a few weeks before the first confirmed case of novel coronavirus (COVID-19) was reported in the United States, the US Centers for Disease Control and Prevention (CDC) issued a bold promise to the nation: the agency will use its scientific expertise to bring a new level of preparedness in the United States and global health security against current and growing threats, finally eliminate certain diseases, and bring an end to the devastation of epidemics.(1) The current outbreak of COVID-19 reminds us how urgent this promise is and just how critical it is to continue to sustain and strengthen our nation's public health infrastructure. The unprecedented pace of the public health response to COVID-19 has only been possible because of prior investments in public health preparedness. To accelerate our pace and meet the challenges of current and future health threats, we must advance our world-class data and analytics capabilities; maintain and expand our state-of-the-art public health laboratory capacity; continue building a workforce of trusted, expert, public health professionals; sustain our capacity to rapidly respond to outbreaks at their source; and assure a strong global and domestic preparedness capacity. (Am J Public Health. Published online ahead of print February 13, 2020: e1-e2. doi:10.2105/AJPH.2020.305618).</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Emerging technologies for the detection of viral infections</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viruses represent one of the major environmental agents that cause human illness and disease. However, the ability to diagnose viral infections is limited by detection capability and scope. Here we describe several emerging technologies that provide rapid and/or high-quality viral diagnostic information. Two technologies, novel CRISPR-based diagnostics and a portable DNA sequencing instrument, are uniquely suited to increase the number of viral agents analyzed, even in point of care settings. We also discuss a phage-based method for generating comprehensive viral profiles of previous exposure/infection and a fluid-phase immunoassay that yields highly quantitative viral antibody analyses. Future applications of these approaches will accelerate on-site clinical diagnosis of viral infections and provide insights into the role viruses play in complex diseases.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Cas12a-Based On-Site and Rapid Nucleic Acid Detection of African Swine Fever</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Dec 10</span></i><p style="text-align: justify; text-justify: inter-word;">The mortality rate of hemorrhagic African swine fever (ASF), which targets domestic pigs and wild boars is caused by African swine fever virus (ASFV), can reach 100%. Since the first confirmed ASF outbreak in China on 3 August 2018, 156 ASF outbreaks were detected in 32 provinces. About 1,170,000 pigs were culled in order to halt further spread. There is no effective treatment or vaccine for it and the present molecular diagnosis technologies have trade-offs in sensitivity, specificity, cost and speed, and none of them cater perfectly to ASF control. Thus, a technology that overcomes the need for laboratory facilities, is relatively low cost, and rapidly and sensitively detects ASFV would be highly valuable. Here, we describe an RAA-Cas12a-based system that combines recombinase aided amplification (RAA) and CRISPR/Cas12a for ASFV detection. The fluorescence intensity readout of this system detected ASFV p72 gene levels as low as 10 aM. For on-site ASFV detection, lateral-flow strip readout was introduced for the first time in the RAA-Cas12a based system (named CORDS, Cas12a-based On-site and Rapid Detection System). We used CORDS to detect target DNA highly specifically using the lateral-flow strip readout and the assay displayed no cross-reactivity to other 13 swine viruses including classical swine fever (CSF). CORDS could identify the ASFV DNA target at femtomolar sensitivity in an hour at 37°C, and only requires an incubator. For ease of use, the reagents of CORDS were lyophilized to three tubes and remained the same sensitivity when stored at 4°C for at least 7 days. Thus, CORDS provide a rapid, sensitive and easily operable method for ASFV on-site detection. Lyophilized CORDS can withstand long-term transportation and storage, and is ready for field-based applications.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Time to use the p-word? Coronavirus enters dangerous new phase</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As outbreaks surge worldwide, scientists fear that COVID-19 might soon become pandemic. As outbreaks surge worldwide, scientists fear that COVID-19 might soon become pandemic.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-mediated activation of endogenous BST-2/tetherin expression inhibits wild-type HIV-1 production</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 28</span></i><p style="text-align: justify; text-justify: inter-word;">The CRISPR technology not only can knock out target genes by using the RNA-guided Cas9 nuclease but also can activate their expression when a nuclease-deficient Cas9 (dCas9) is employed. Using the latter function, we here show the effect of the CRISPR-mediated pinpoint activation of endogenous expression of BST-2 (also known as tetherin), a virus restriction factor with a broad antiviral spectrum. Single-guide RNA (sgRNA) sequences targeting the BST-2 promoter were selected by promoter assays. Potential sgRNAs and dCas9 fused to the VP64 transactivation domain, along with an accessory transcriptional activator complex, were introduced into cells by lentiviral transduction. Increased expression of BST-2 mRNA in transduced cells was confirmed by real-time RT-PCR. Cells in which BST-2 expression was highly enhanced showed the effective inhibition of HIV-1 production and replication even in the presence of the viral antagonist Vpu against BST-2. These findings confirm that the physiological stoichiometry between host restriction factors and viral antagonists may determine the outcome of the battle with viruses.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-mediated activation of endogenous BST-2/tetherin expression inhibits wild-type HIV-1 production</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Feb 28</span></i><p style="text-align: justify; text-justify: inter-word;">The CRISPR technology not only can knock out target genes by using the RNA-guided Cas9 nuclease but also can activate their expression when a nuclease-deficient Cas9 (dCas9) is employed. Using the latter function, we here show the effect of the CRISPR-mediated pinpoint activation of endogenous expression of BST-2 (also known as tetherin), a virus restriction factor with a broad antiviral spectrum. Single-guide RNA (sgRNA) sequences targeting the BST-2 promoter were selected by promoter assays. Potential sgRNAs and dCas9 fused to the VP64 transactivation domain, along with an accessory transcriptional activator complex, were introduced into cells by lentiviral transduction. Increased expression of BST-2 mRNA in transduced cells was confirmed by real-time RT-PCR. Cells in which BST-2 expression was highly enhanced showed the effective inhibition of HIV-1 production and replication even in the presence of the viral antagonist Vpu against BST-2. These findings confirm that the physiological stoichiometry between host restriction factors and viral antagonists may determine the outcome of the battle with viruses.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>The race to unravel the United States’ biggest coronavirus outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rohit Shankar left the virology laboratory at 2 a.m. on Wednesday, and was back at the lab bench by 7 a.m. the same day. “It’s okay,” he says, “I had a doughnut and a coffee.” Shankar, a medical scientist, and his colleagues at the University of Washington in Seattle are poised to exponentially drive up the number of confirmed cases of the coronavirus disease COVID-19 around the city, in western Washington state. That’s because this week, they began analysing a mountain of nose and throat swabs collected from hospitals in the region. Already, the researchers are seeing clear signs that the virus has infected vastly more people than have been formally detected. Scientists fear coronavirus spread in countries least able to contain it Washington state has become the United States’ ground zero for COVID-19, which has now spread to more than 90 countries worldwide in what seems to be a new and dangerous phase of the outbreak. Washington has declared a state emergency, and ten people there have died from the disease. But the number of confirmed cases in Washington — 70 — is still an underestimate resulting from a lack of testing, researchers agree. A genomic analysis posted online on 29 February suggested that hundreds of people in western Washington might already be infected. Academic scientists have mostly been prevented from measuring the extent of the US outbreak because of federal rules restricting the number of labs qualified to run diagnostic tests. But that is changing now, and helps account for why the state's caseload jumped from 10 to 70 this week. Dozens of virologists and genomicists have now kicked into high gear in Seattle, dropping or adapting projects to devote resources to the outbreak. Researchers are working around the clock to find out how many people have the disease in the area. Others are analysing genomes to reveal how the virus is transmitted or developing new therapies. The scientists are racing to help Washington avoid the fate of Hubei province in China, where more than 2,900 people have died of COVID-19 so far. The coronavirus emerged in the province’s city Wuhan in December, and the initial response from officials was slow. “We are past the point of containment,” says Helen Chu, an infectious-disease specialist at the University of Washington School of Medicine (UW Medicine) in Seattle. “So now we need to keep the people who are vulnerable from getting sick.”</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Making vaccines “on demand”: A potential solution for emerging pathogens and biodefense?</h3>
<i class="tooltip">2013<span class="tooltiptext">2013 Sep 1</span></i><p style="text-align: justify; text-justify: inter-word;">The integrated US Public Health Emergency Medical Countermeasures Enterprise (PHEMCE) has made great strides in strategic preparedness and response capabilities. There have been numerous advances in planning, biothreat countermeasure development, licensure, manufacturing, stockpiling and deployment. Increased biodefense surveillance capability has dramatically improved, while new tools and increased awareness have fostered rapid identification of new potential public health pathogens. Unfortunately, structural delays in vaccine design, development, manufacture, clinical testing and licensure processes remain significant obstacles to an effective national biodefense rapid response capability. This is particularly true for the very real threat of “novel pathogens” such as the avian-origin influenzas H7N9 and H5N1, and new coronaviruses such as hCoV-EMC. Conventional approaches to vaccine development, production, clinical testing and licensure are incompatible with the prompt deployment needed for an effective public health response. An alternative approach, proposed here, is to apply computational vaccine design tools and rapid production technologies that now make it possible to engineer vaccines for novel emerging pathogen and WMD biowarfare agent countermeasures in record time. These new tools have the potential to significantly reduce the time needed to design string-of-epitope vaccines for previously unknown pathogens. The design process—from genome to gene sequence, ready to insert in a DNA plasmid—can now be accomplished in less than 24 h. While these vaccines are by no means “standard,” the need for innovation in the vaccine design and production process is great. Should such vaccines be developed, their 60-d start-to-finish timeline would represent a 2-fold faster response than the current standard.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevent and predict</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the COVID-19 outbreak continues, the next pandemic could be prevented by ending the wildlife trade and reinvesting in the monitoring of potential zoonoses.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>[The importance of strengthening the ability of fundamental disease prevention and control system from the perspective of the epidemic situation of COVID-19]</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 has been in epidemic for nearly two months. The prevention and control measures have achieved remarkable results. From the response and disposal process of this epidemic, it is exposed that fundamental disease prevention and control system are insufficient in human resources and ability of laboratory testing. It is suggested that the disease control institutions should strengthen the construction of these aspects in future.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Responding to Covid-19 — A Once-in-a-Century Pandemic?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In any crisis, leaders have two equally important responsibilities: solve the immediate problem and keep it from happening again. The Covid-19 pandemic is a case in point. We need to save lives now while also improving the way we respond to outbreaks in general. The first point is more pressing, but the second has crucial long-term consequences. The long-term challenge — improving our ability to respond to outbreaks — isn’t new. Global health experts have been saying for years that another pandemic whose speed and severity rivaled those of the 1918 influenza epidemic was a matter not of if but of when.1 The Bill and Melinda Gates Foundation has committed substantial resources in recent years to helping the world prepare for such a scenario. Now we also face an immediate crisis. In the past week, Covid-19 has started behaving a lot like the once-in-a-century pathogen we’ve been worried about. I hope it’s not that bad, but we should assume it will be until we know otherwise. There are two reasons that Covid-19 is such a threat. First, it can kill healthy adults in addition to elderly people with existing health problems. The data so far suggest that the virus has a case fatality risk around 1%; this rate would make it many times more severe than typical seasonal influenza, putting it somewhere between the 1957 influenza pandemic (0.6%) and the 1918 influenza pandemic (2%).2</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic diversity and evolution of SARS-CoV-2</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Generation of antibodies against COVID-19 virus for development of diagnostic tools</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.</jats:p></p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Déjà Vu or Jamais Vu? How the Severe Acute Respiratory Syndrome Experience Influenced a Singapore Radiology Department's Response to the Coronavirus Disease (COVID-19) Epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">OBJECTIVE. This article shares the ground operational perspective of how a tertiary hospital radiology department in Singapore is responding to the coronavirus disease (COVID-19) epidemic. This same department was also deeply impacted by the severe acute respiratory syndrome (SARS) outbreak in 2003. CONCLUSION. Though similar to SARS, the COVID-19 outbreak has several differences. We share how lessons from 2003 are applied and modified in our ongoing operational response to this evolving novel pathogen.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>A CRISPR toolbox to study virus–host interactions</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Viruses depend on their hosts to complete their replication cycles; they exploit cellular receptors for entry and hijack cellular functions to replicate their genome, assemble progeny virions and spread. Recently, genome-scale CRISPR–Cas screens have been used to identify host factors that are required for virus replication, including the replication of clinically relevant viruses such as Zika virus, West Nile virus, dengue virus and hepatitis C virus. In this Review, we discuss the technical aspects of genome-scale knockout screens using CRISPR–Cas technology, and we compare these screens with alternative genetic screening technologies. The relative ease of use and reproducibility of CRISPR–Cas make it a powerful tool for probing virus–host interactions and for identifying new antiviral targets.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19's graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control.  Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation.  Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.</jats:p></p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-22</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Synthetic recombinant bat SARS-like coronavirus is infectious in cultured cells and in mice</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Defining prospective pathways by which zoonoses evolve and emerge as human pathogens is critical for anticipating and controlling both natural and deliberate pandemics. However, predicting tenable pathways of animal-to-human movement has been hindered by challenges in identifying reservoir species, cultivating zoonotic organisms in culture, and isolating full-length genomes for cloning and genetic studies. The ability to design and recover pathogens reconstituted from synthesized cDNAs has the potential to overcome these obstacles by allowing studies of replication and pathogenesis without identification of reservoir species or cultivation of primary isolates. Here, we report the design, synthesis, and recovery of the largest synthetic replicating life form, a 29.7-kb bat severe acute respiratory syndrome (SARS)-like coronavirus (Bat-SCoV), a likely progenitor to the SARS-CoV epidemic. To test a possible route of emergence from the noncultivable Bat-SCoV to human SARS-CoV, we designed a consensus Bat-SCoV genome and replaced the Bat-SCoV Spike receptor-binding domain (RBD) with the SARS-CoV RBD (Bat-SRBD). Bat-SRBD was infectious in cell culture and in mice and was efficiently neutralized by antibodies specific for both bat and human CoV Spike proteins. Rational design, synthesis, and recovery of hypothetical recombinant viruses can be used to investigate mechanisms of transspecies movement of zoonoses and has great potential to aid in rapid public health responses to known or predicted emerging microbial threats.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Singapore claims first use of antibody test to track coronavirus infections | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In what appears to be a first, disease trackers in Singapore have used an experimental antibody test for COVID-19 to confirm that a suspected patient was infected with the coronavirus. The patient was one of two people who together formed a missing link between two clusters of cases that each occurred in a Singaporean church. Researchers around the world are racing to develop antibody tests, also called serological tests, that can confirm whether someone was infected even after their immune system has cleared the virus that causes COVID-19. The group that developed the test, at Duke-NUS Medical School in Singapore, is among the front-runners, although its assay has to be validated before it is taken into production and deployed widely.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-07</span></i><p style="text-align: justify; text-justify: inter-word;">The etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.  Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Genomics and High-Consequence Infectious Diseases: A Scoping Review of Emerging Science and Potential Ethical Issues</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Host genomic research on high-consequence infectious diseases is a growing area, but the ethical, legal, and social implications of such findings related to potential applications of the research have not yet been identified. While there is a robust ethical debate about the ethical, legal, and social implications of research during an emergency, there has been less consideration of issues facing research conducted outside of the scope of emergency response. Addressing the implications of research at an early stage (anticipatory ethics) helps define the issue space, facilitates preparedness, and promotes ethically and socially responsible practices. To lay the groundwork for more comprehensive anticipatory ethics work, this article provides a preliminary assessment of the state of the field with a scoping review of host genomic research on a subset of high-consequence infectious diseases of relevance to high-level isolation units, focusing on its ethically relevant features and identifying several ethical, legal, and social implications raised by the literature. We discuss the challenges of genomic studies of low-frequency, high-risk events and applications of the science, including identifying targets to guide the development of new therapeutics, improving vaccine development, finding biomarkers to predict disease outcome, and guiding decisions about repurposing existing drugs and genetic screening. Some ethical, legal, and social implications identified in the literature included the rise of systems biology and paradigm shifts in medical countermeasure development; controversies over repurposing of existing drugs; genetic privacy and discrimination; and benefit-sharing and global inequity as part of the broader ecosystem surrounding high-level isolation units. Future anticipatory ethics work should forecast the science and its applications; identify a more comprehensive list of ethical, legal, and social implications; and facilitate evaluation by multiple stakeholders to inform the integration of ethical concerns into high-level isolation unit policy and practice.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>The network structure and eco-evolutionary dynamics of CRISPR-induced immune diversification</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-12</span></i><p style="text-align: justify; text-justify: inter-word;">As a heritable sequence-specific adaptive immune system, CRISPR-Cas is a powerful force shaping strain diversity in host-virus systems. While the diversity of CRISPR alleles has been explored, the associated structure and dynamics of host-virus interactions has not. We develop theory on the role of CRISPR immunity in mediating the interplay between host-virus interaction structure and eco-evolutionary dynamics in a computational model and three natural systems. We show that the structures of networks describing who infects whom and who is protected from whom are modular and weighted-nested, respectively. The dynamic interplay between these networks influences transitions between dynamical regimes of virus diversification and host control, leading to eventual virus extinction. The three empirical systems exhibit weighted nestedness, a pattern our theory shows is indicative of viral control by hosts. Previously missing from studies of microbial host-pathogen systems, the protection network plays a key role in the coevolutionary dynamics.</p>
<p align="right"><i>score: 47</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostic System for Rapid and Sensitive Differential Detection of Pathogens</h3>
<i class="tooltip">2005<span class="tooltiptext">2005 Feb</span></i><p style="text-align: justify; text-justify: inter-word;">Naturally emerging and deliberately released pathogens demand new detection strategies to allow early recognition and containment. We describe a diagnostic system for rapid, sensitive, multiplex discrimination of microbial gene sequences and report its application for detecting 22 respiratory pathogens in clinical samples.</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>In bid to rapidly expand coronavirus testing, U.S. agency abruptly changes rules | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">The Food and Drug Administration (FDA) today recommended a dramatic shift in how it implements regulations that control whether laboratories can use diagnostic kits created in-house to test for infections of coronavirus-2019 (COVID-19). “We issued a policy this morning that allows us to have a lot of flexibility around the development of diagnostic tests,” said FDA Commissioner Stephen Hahn at a White House briefing with President Donald Trump this afternoon. “We expect this policy to have a significant impact.” The change could greatly expand the number of laboratories able to do coronavirus testing. The U.S. government has come under severe criticism for not providing nearly enough tests needed to understand the extent of spread in the population. A test kit produced and distributed by the U.S. Centers for Disease Control and Prevention (CDC) was shelved after state and local lab trying it out discovered that it contained a faulty reagent. As a result, many labs that have the capability to test themselves have not been allowed to do so. The new recommendations focus on “high-complexity testing laboratories” that are certified under federal rules known as Clinical Laboratory Improvement Amendments. This group of facilities includes many hospital labs, like the one that epidemiologist Michael Mina works at Brigham and Women’s Hospital in Boston. “Essentially it’s opening up a clear and concise avenue for labs like the one at Brigham and Women’s,” says Mina. “It’s what I’ve been advocating for a month now.”</p>
<p align="right"><i>score: 46</i></p>

</div><br /><br /><br /><div class="card"><h3>World Health Organization declares Global Emergency: A review of the 2019 Novel Coronavirus (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">An unprecedented outbreak of pneumonia of unknown aetiology in Wuhan City, Hubei province in China emerged in December of 2019. A novel coronavirus was identified as the causative agent and was subsequently termed COVID-19 by the World Health Organization (WHO). Considered a relative of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), COVID-19 is a betacoronavirus that affects the lower respiratory tract and manifests as pneumonia in humans. Despite rigorous global containment and quarantine efforts, the incidence of COVID-19 continues to rise, with 50,580 laboratory-confirmed cases and 1,526 deaths worldwide. In response to this global outbreak, we summarise the current state of knowledge surrounding COVID-19.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Dynamics Simulations Indicate the COVID-19 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-02</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed molecular dynamics simulations of the main protease (Mpro). We compared and contrasted the Mpro for COVID-19 with a highly similar SARS protein. In spite of a high level of sequence similarity, the active sites in both proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the pockets time-dependence indicates its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicates that the virus mutability will pose a further challenge to the rational design of small-molecule inhibitors.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-24</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus  broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</jats:p></p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Management and clinical thinking of Coronavirus Disease 2019</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, the 2019 novel coronavirus pneumonia (NCP, officially named Coronavirus Disease 2019(COVID-19) by the World Health Organization) broke out in Wuhan, Hubei, and it quickly spread to the whole country and abroad. The situation was at stake. The sudden and serious COVID-19 epidemic has brought us a lot of urgent problems. How to effectively control the spread of COVID-19? When does the population infection rate rise to its peak? What will eventually be the number of infected patients? How to make early diagnosis? What effective antiviral drugs are available? How to effectively treat with existing drugs? Can it successfully improve the survival rate of critically patients? In response to the above questions, we put forward corresponding suggestions and reflections from the perspective of the infectious clinician.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-10</span></i><p style="text-align: justify; text-justify: inter-word;">The novel coronavirus disease first identified in 2019 in Wuhan, China (COVID-19) has become a serious global public health concern. One current issue is the ability to adequately screen for the virus causing COVID-2 (SARS-CoV-2). Here we demonstrate the feasibility of shotgun proteomics as a SARS-CoV-2 screening method, through the detection of viral peptides in proteolytically digested body fluids. Using in silico methods, we generated trypsin-based shotgun proteomics methods optimized for LCMS systems from 5 commercial instrument vendors (Thermo, SCIEX, Waters, Shimadzu, and Agilent). First, we generated protein FASTA files and their protein digest maps. Second, the FASTA files were used to generate spectral libraries based on experimental data. Third, transition lists were derived from the spectral libraries using the vendor neutral and open Skyline software environment. Finally, we identified 17 post-translational modifications using linear motif modeling.</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>Response to COVID-19 in Taiwan: Big Data Analytics, New Technology, and Proactive Testing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Taiwan is 81 miles off the coast of mainland China and was expected to have the second highest number of cases of coronavirus disease 2019 (COVID-19) due to its proximity to and number of flights between China.1 The country has 23 million citizens of which 850 000 reside in and 404 000 work in China.2,3 In 2019, 2.71 million visitors from the mainland traveled to Taiwan.4 As such, Taiwan has been on constant alert and ready to act on epidemics arising from China ever since the severe acute respiratory syndrome (SARS) epidemic in 2003. Given the continual spread of COVID-19 around the world, understanding the action items that were implemented quickly in Taiwan and assessing the effectiveness of these actions in preventing a large-scale epidemic may be instructive for other countries. COVID-19 occurred just before the Lunar New Year during which time millions of Chinese and Taiwanese were expected to travel for the holidays. Taiwan quickly mobilized and instituted specific approaches for case identification, containment, and resource allocation to protect the public health. Taiwan leveraged its national health insurance database and integrated it with its immigration and customs database to begin the creation of big data for analytics; it generated real-time alerts during a clinical visit based on travel history and clinical symptoms to aid case identification. It also used new technology, including QR code scanning and online reporting of travel history and health symptoms to classify travelers’ infectious risks based on flight origin and travel history in the past 14 days. Persons with low risk (no travel to level 3 alert areas) were sent a health declaration border pass via SMS (short message service) messaging to their phones for faster immigration clearance; those with higher risk (recent travel to level 3 alert areas) were quarantined at home and tracked through their mobile phone to ensure that they remained at home during the incubation period. Moreover, Taiwan enhanced COVID-19 case finding by proactively seeking out patients with severe respiratory symptoms (based on information from the National Health Insurance [NHI] database) who had tested negative for influenza and retested them for COVID-19; 1 was found of 113 cases. The toll-free number 1922 served as a hotline for citizens to report suspicious symptoms or cases in themselves or others; as the disease progressed, this hotline has reached full capacity, so each major city was asked to create its own hotline as an alternative. It is not known how often this hotline has been used. The government addressed the issue of disease stigma and compassion for those affected by providing food, frequent health checks, and encouragement for those under quarantine. This rapid response included hundreds of action items (eTable in the Supplement).</p>
<p align="right"><i>score: 45</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19 R0: Magic number or conundrum?</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">There is an increasing concern about COVID-19 worldwide. This is a new emerging infectious disease caused by a novel coronavirus (SARS-CoV-2), which recently broke out from the Chinese city of Wuhan and has quickly spread in China, with sporadic cases in each continent [...].</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in ‘lockdown’ directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation’s economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and advances how smart city networks should work towards enhancing standardization protocols for increased data sharing in the event of outbreaks or disasters, leading to better global understanding and management of the same.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Deep learning-based model for detecting 2019 novel coronavirus pneumonia on high-resolution computed tomography: a prospective study</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-26</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Computed tomography (CT) is the preferred imaging method for diagnosing 2019 novel coronavirus (COVID19) pneumonia. Our research aimed to construct a system based on deep learning for detecting COVID-19 pneumonia on high resolution CT, relieve working pressure of radiologists and contribute to the control of the epidemic. Methods: For model development and validation, 46,096 anonymous images from 106 admitted patients, including 51 patients of laboratory confirmed COVID-19 pneumonia and 55 control patients of other diseases in Renmin Hospital of Wuhan University (Wuhan, Hubei province, China) were retrospectively collected and processed. Twenty-seven consecutive patients undergoing CT scans in Feb, 5, 2020 in Renmin Hospital of Wuhan University were prospectively collected to evaluate and compare the efficiency of radiologists against 2019-CoV pneumonia with that of the model. Findings: The model achieved a per-patient sensitivity of 100%, specificity of 93.55%, accuracy of 95.24%, PPV of 84.62%, and NPV of 100%; a per-image sensitivity of 94.34%, specificity of 99.16%, accuracy of 98.85%, PPV of 88.37%, and NPV of 99.61% in retrospective dataset. For 27 prospective patients, the model achieved a comparable performance to that of expert radiologist. With the assistance of the model, the reading time of radiologists was greatly decreased by 65%. Conclusion: The deep learning model showed a comparable performance with expert radiologist, and greatly improve the efficiency of radiologists in clinical practice. It holds great potential to relieve the pressure of frontline radiologists, improve early diagnosis, isolation and treatment, and thus contribute to the control of the epidemic.</jats:p></p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>Measures for diagnosing and treating infections by a novel coronavirus responsible for a pneumonia outbreak originating in Wuhan, China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">On 10 January 2020, a new coronavirus causing a pneumonia outbreak in Wuhan City in central China was denoted as 2019-nCoV by the World Health Organization (WHO). As of 24 January 2020, there were 887 confirmed cases of 2019-nCoV infection, including 26 deaths, reported in China and other countries. Therefore, combating this new virus and stopping the epidemic is a matter of urgency. Here, we focus on advances in research and development of fast diagnosis methods, as well as potential prophylactics and therapeutics to prevent or treat 2019-nCoV infection.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020</h3>
<i class="tooltip">2020<span class="tooltiptext">2020 Feb 13</span></i><p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Programmable low-cost DNA-based platform for viral RNA detection</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-01-16</span></i><p style="text-align: justify; text-justify: inter-word;">Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with [~]24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Daily briefing: The potential for repurposing existing drugs to fight COVID-19 coronavirus</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Time is of the essence — here’s what we’ve already got. Plus, biology’s cryo-electron microscopy boom and why Scotland is bringing back bogs. Time is of the essence — here’s what we’ve already got. Plus, biology’s cryo-electron microscopy boom and why Scotland is bringing back bogs.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR/Cas9 knockout of USP18 enhances type I IFN responsiveness and restricts HIV-1 infection in macrophages</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The IFN-stimulated gene ubiquitin specific proteinase 18 (USP18) encodes a protein that negatively regulates T1 IFN signaling via stearic inhibition of JAK1 recruitment to the IFNα receptor 2 subunit (IFNAR2). Here we demonstrate that USP18 expression is induced by HIV-1 in a T1 IFN dependent manner. Experimental depletion of USP18 by CRISPR/Cas9 gene editing results in a significant restriction of HIV-1 replication in an induced pluripotent stem cell (iPSC)-derived macrophage model. In the absence of USP18, macrophages have increased responsiveness to stimulation with T1 IFNs with prolonged phosphorylation of STAT1 and STAT2 and increased expression of interferon-stimulated genes that are key for antiviral responses. Interestingly, HIV-1 requires some signaling through the T1 IFN receptor to replicate efficiently because a neutralizing antibody that inhibits T1 IFN activity reduces HIV-1 replication rate in monocyte-derived macrophages. USP18 induction by HIV-1 tunes the IFN response to optimal levels allowing for efficient transcription from the HIV-1 LTR promoter while minimizing the T1 IFN-induced antiviral response that would otherwise restrict viral replication and spread. Finally, iPSC and CRISPR/Cas9 gene targeting offer a powerful tool to study host factors that regulate innate immune responses.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>The Lab Without Walls: A Deployable Approach to Tropical Infectious Diseases</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The Laboratory Without Walls is a modular field application of molecular biology that provides clinical laboratory support in resource-limited, remote locations. The current repertoire arose from early attempts to deliver clinical pathology and public health investigative services in remote parts of tropical Australia, to address the shortcomings of conventional methods when faced with emerging infectious diseases. Advances in equipment platforms and reagent chemistry have enabling rapid progress, but also ensure the Laboratory Without Walls is subject to continual improvement. Although new molecular biology methods may lead to more easily deployable clinical laboratory capability, logistic and technical governance issues continue to act as important constraints on wider implementation.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Risk Assessment of Novel Coronavirus COVID-19 Outbreaks Outside China Characteristics of and Public Health Responses to the Coronavirus Disease 2019 Outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">We developed a computational tool to assess the risks of novel coronavirus outbreaks outside of China. We estimate the dependence of the risk of a major outbreak in a country from imported cases on key parameters such as: (i) the evolution of the cumulative number of cases in mainland China outside the closed areas; (ii) the connectivity of the destination country with China, including baseline travel frequencies, the effect of travel restrictions, and the efficacy of entry screening at destination; and (iii) the efficacy of control measures in the destination country (expressed by the local reproduction number R loc ). We found that in countries with low connectivity to China but with relatively high R loc , the most beneficial control measure to reduce the risk of outbreaks is a further reduction in their importation number either by entry screening or travel restrictions. Countries with high connectivity but low R loc benefit the most from policies that further reduce R loc . Countries in the middle should consider a combination of such policies. Risk assessments were illustrated for selected groups of countries from America, Asia, and Europe. We investigated how their risks depend on those parameters, and how the risk is increasing in time as the number of cases in China is growing. In December 2019, cases of unidentified pneumonia with a history of exposure in the Huanan Seafood Market were reported in Wuhan, Hubei Province. A novel coronavirus, SARS-CoV-2, was identified to be accountable for this disease. Human-to-human transmission is confirmed, and this disease (named COVID-19 by World Health Organization (WHO)) spread rapidly around the country and the world. As of 18 February 2020, the number of confirmed cases had reached 75,199 with 2009 fatalities. The COVID-19 resulted in a much lower case-fatality rate (about 2.67%) among the confirmed cases, compared with Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS). Among the symptom composition of the 45 fatality cases collected from the released official reports, the top four are fever, cough, short of breath, and chest tightness/pain. The major comorbidities of the fatality cases include hypertension, diabetes, coronary heart disease, cerebral infarction, and chronic bronchitis. The source of the virus and the pathogenesis of this disease are still unconfirmed. No specific therapeutic drug has been found. The Chinese Government has initiated a level-1 public health response to prevent the spread of the disease. Meanwhile, it is also crucial to speed up the development of vaccines and drugs for treatment, which will enable us to defeat COVID-19 as soon as possible.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Innovation in observation: a vision for early outbreak detection</h3>
<i class="tooltip">2010<span class="tooltiptext">2010 May 20</span></i><p style="text-align: justify; text-justify: inter-word;">The emergence of new infections and resurgence of old ones—health threats stemming from environmental contamination or purposeful acts of bioterrorism—call for a worldwide effort in improving early outbreak detection, with the goal of ameliorating current and future risks. In some cases, the problem of outbreak detection is logistically straightforward and mathematically easy: a single case of a disease of great concern can constitute an outbreak. However, for the vast majority of maladies, a simple analytical solution does not exist. Furthermore, each step in developing reliable, sensitive, effective surveillance systems demonstrates enormous complexities in the transmission, manifestation, detection, and control of emerging health threats. In this communication, we explore potential future innovations in early outbreak detection systems that can overcome the pitfalls of current surveillance. We believe that modern advances in assembling data, techniques for collating and processing information, and technology that enables integrated analysis will facilitate a new paradigm in outbreak definition and detection. We anticipate that moving forward in this direction will provide the highly desired sensitivity and specificity in early detection required to meet the emerging challenges of global disease surveillance.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Modulation of metabolic functions through Cas13d-mediated gene knockdown in liver</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-19</span></i><p style="text-align: justify; text-justify: inter-word;">RNA knockdown in vivo carries significant potential for disease modelings and therapies. Despite the emerging approaches of CRISPR/Cas9-mediated permanent knock out of targeted genes, strategies targeting RNA for disruption are advantageous in the treatment of acquired metabolic disorders when permanent modification of the genome DNA is not appropriate, and RNA virus infection diseases when pathogenic DNA is not available (such as SARS-Cov-2 and MERS infections). Recently, Cas13d, a family of RNA-targeting CRISPR effectors, has been shown to accomplish robust down-regulation of cellular RNAs in mammalian cells in vitro. Among the various Cas13d subtypes, CasRx (RfxCas13d) showed the most potent RNA knockdown efficiency in HEK293T cells. However, the RNA-targeting activity of Cas13d still needs to be verified in vivo. In this study, the CasRx system was demonstrated to efficiently and functionally knock down genes related to metabolism functions, including Pten, Pcsk9 and lncLstr, in mouse hepatocytes. CasRx-mediated simultaneous knockdown of multiple genes was also achieved by sgRNA arrays, providing a useful strategy to modulate complex metabolism networks. Moreover, the AAV (adeno-associated virus)-mediated delivery of CasRx and Pcsk9 sgRNAs into mouse liver successfully decreased serum PCSK9, resulting in significant reduction of serum cholesterol levels. Importantly, CasRx-mediated knockdown of Pcsk9 is reversible and Pcsk9 could be repeatedly down-regulated, providing an effective strategy to reversibly modulate metabolic genes. The present work supplies a successful proof-of-concept trial that suggests efficient and regulatory knockdown of target metabolic genes for a designed metabolism modulation in the liver.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">As the world is witnessing the epidemic of COVID-19, a disease caused by a novel coronavirus, SARS-CoV-2, emerging genetics and clinical evidences suggest a similar path to those of SARS and MERS. The rapid genomic sequencing and open access data, together with advanced vaccine technology, are expected to give us more knowledge on the pathogen itself, including the host immune response as well as the plan for therapeutic vaccines in the near future. This review aims to provide a comparative view among SARS-CoV, MERS-CoV and the newly epidemic SARS-CoV-2, in the hope to gain a better understanding of the host-pathogen interaction, host immune responses, and the pathogen immune evasion strategies. This predictive view may help in designing an immune intervention or preventive vaccine for COVID-19 in the near future.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Unique epidemiological and clinical features of the emerging 2019 novel coronavirus pneumonia (COVID-19) implicate special control measures</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract By Feb 27th, 2020, the outbreak of COVID-19 caused 82623 confirmed cases and 2858 deaths globally, more than Severe Acute Respiratory Syndrome (SARS) (8273 cases, 775 deaths) and Middle East Respiratory Syndrome (MERS) (1139 cases, 431 deaths) caused in 2003 and 2013 respectively. COVID-19 has spread to 46 countries internationally. Total fatality rate of COVID-19 is estimated at 3.46% by far based on published data from Chinese Center for Disease Control and Prevention (China CDC). Average incubation period of COVID-19 is around 6.4 days, ranges from 0-24 days. The basic reproductive number (R0) of COVID-19 ranges from 2-3.5 at the early phase regardless of different prediction models, which is higher than SARS and MERS. A study from China CDC showed majority of patients (80.9%) were considered asymptomatic or mild pneumonia but released large amounts of viruses at the early phase of infection, which posed enormous challenges for containing the spread of COVID-19. Nosocomial transmission was another severe problem. 3019 health workers were infected by Feb 12, 2020, which accounted for 3.83% of total number of infections, and extremely burdened the health system, especially in Wuhan. Limited epidemiological and clinical data suggest that the disease spectrum of COVID-19 may differ from SARS or MERS. We summarize latest literatures on genetic, epidemiological, and clinical features of COVID-19 in comparison to SARS and MERS and emphasize special measures on diagnosis and potential interventions. This review will improve our understanding of the unique features of COVID-19 and enhance our control measures in the future. This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>US Emergency Legal Responses to Novel Coronavirus: Balancing Public Health and Civil Liberties</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">With increasing numbers of cases of coronavirus disease 2019 (COVID-19) globally and in the United States, Health and Human Services (HHS) Secretary Alex Azar declared a national public health emergency on January 31. The emergency declaration of the HHS authorizes additional resources, enhanced federal powers, interjurisdictional coordination, and waivers of specific regulations. State and local public health emergency declarations are also likely. During crises, government has a special responsibility to thoughtfully balance public health protections and civil liberties.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus outbreak: the role of companies in preparedness and responses</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">As in previous health crises, the coronavirus disease 2019 (COVID-19) outbreak has raised questions about preparedness and emergency responses in many countries. In this crisis, what role can companies play? Public and private companies must continue to produce or provide their services, but with consideration of the health context. Many companies are involved with the COVID-19 outbreak because they are established in or work with China (client or supplier), and most have already activated their business continuity planning or equivalent. During an infectious disease outbreak like COVID-19, most large companies around the world have a major part to play, especially in terms of preparedness and emergency response. Indeed, companies should be integrated into the governmental health contingency plan developed in many countries, and by WHO and the International Labor Organization. Helped by their occupational practitioners, healthcare advisers, and safety professionals, companies that have a financial capacity and responsibilities (including governmental, federal, or state administrations) will thus have to prepare their business continuity planning for when cases of infected patients occur in the company. They also must be prepared for the potential psychosocial and psychological effects of outbreaks. All health professionals should be involved in the development and implementation of recommendations for companies and their environments</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-25</span></i><p style="text-align: justify; text-justify: inter-word;">The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Harnessing DNA Synthesis to Develop Rapid Responses to Emerging and Pandemic Pathogens</h3>
<i class="tooltip">2011<span class="tooltiptext">2011 Mar 16</span></i><p style="text-align: justify; text-justify: inter-word;">Given the interconnected nature of our world today, emerging pathogens and pandemic outbreaks are an ever-growing threat to the health and economic stability of the global community. This is evident by the recent 2009 Influenza A (H1N1) pandemic, the SARS outbreak, as well as the ever-present threat of global bioterrorism. Fortunately, the biomedical community has been able to rapidly generate sequence data so these pathogens can be readily identified. To date, however, the utilization of this sequence data to rapidly produce relevant experimental results or actionable treatments is lagging in spite of obtained sequence data. Thus, a pathogenic threat that has emerged and/or developed into a pandemic can be rapidly identified; however, translating this identification into a targeted therapeutic or treatment that is rapidly available has not yet materialized. This commentary suggests that the growing technology of DNA synthesis should be fully implemented as a means to rapidly generate in vivo data and possibly actionable therapeutics soon after sequence data becomes available.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple Platform for the Rapid Development of Antimicrobials</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 14</span></i><p style="text-align: justify; text-justify: inter-word;">Recent infectious outbreaks highlight the need for platform technologies that can be quickly deployed to develop therapeutics needed to contain the outbreak. We present a simple concept for rapid development of new antimicrobials. The goal was to produce in as little as one week thousands of doses of an intervention for a new pathogen. We tested the feasibility of a system based on antimicrobial synbodies. The system involves creating an array of 100 peptides that have been selected for broad capability to bind and/or kill viruses and bacteria. The peptides are pre-screened for low cell toxicity prior to large scale synthesis. Any pathogen is then assayed on the chip to find peptides that bind or kill it. Peptides are combined in pairs as synbodies and further screened for activity and toxicity. The lead synbody can be quickly produced in large scale, with completion of the entire process in one week.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple Platform for the Rapid Development of Antimicrobials</h3>
<i class="tooltip">2017<span class="tooltiptext">2017 Dec 14</span></i><p style="text-align: justify; text-justify: inter-word;">Recent infectious outbreaks highlight the need for platform technologies that can be quickly deployed to develop therapeutics needed to contain the outbreak. We present a simple concept for rapid development of new antimicrobials. The goal was to produce in as little as one week thousands of doses of an intervention for a new pathogen. We tested the feasibility of a system based on antimicrobial synbodies. The system involves creating an array of 100 peptides that have been selected for broad capability to bind and/or kill viruses and bacteria. The peptides are pre-screened for low cell toxicity prior to large scale synthesis. Any pathogen is then assayed on the chip to find peptides that bind or kill it. Peptides are combined in pairs as synbodies and further screened for activity and toxicity. The lead synbody can be quickly produced in large scale, with completion of the entire process in one week.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>The response of Milan's Emergency Medical System to the COVID-19 outbreak in Italy</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The number of people infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus causing coronavirus disease 2019 (COVID-19), is dramatically increasing worldwide.The first person-to-person transmission in Italy was reported on Feb 21, 2020, and led to an infection chain that represents the largest COVID-19 outbreak outside Asia to date. Here we document the response of the Emergency Medical System (EMS) of the metropolitan area of Milan, Italy, to the COVID-19 outbreak.On Jan 30, 2020, WHO declared the COVID-19 outbreak a public health emergency of international concern.2 Since then, the Italian Government has implemented extraordinary measures to restrict viral spread, including interruptions of air traffic from China, organised repatriation flights and quarantines for Italian travellers in China, and strict controls at international airports' arrival terminals. Local medical authorities adopted specific WHO recommendations to identify and isolate suspected cases of COVID-19.Such recommendations were addressed to patients presenting with respiratory symptoms and who had travelled to an endemic area in the previous 14 days or who had worked in the health-care sector, having been in close contact with patients with severe respiratory disease with unknown aetiology. Suspected cases were transferred to preselected hospital facilities where the SARS-CoV-2 test was available and infectious disease units were ready for isolation of confirmed cases.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Effectiveness for the Response to COVID-19: The MERS Outbreak Containment Procedures</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In the current issue of Osong Public Health and Research Perspectives, 3 studies are presented dealing with COVID-19. A study by Kim et al, analyzed the genetic information of the COVID-19 virus isolated from a patient in South Korea. The virus was identified by real-time reverse transcriptase (RT) PCR followed by Next Generation Sequencing of the full-length of the genome [5]. In an interim report by the COVID-19 National Emergency Response Center of the KCDC, epidemiological and clinical characteristics of COVID-19 in 28 cases in South Korea were reported. There were 53.9% of cases who were males, with 16 cases in patients who had traveled abroad, of which 11 cases (39.3%) were from Wuhan, China, and 12 of the remaining cases were believed to have been infected in South Korea. The incubation period was 4.8 days. Most secondary infected cases were from family members outside the family home (66.7%) and within the family home (75%) [6].</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>From the frontlines of COVID-19 – How prepared are we as obstetricians: a commentary</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The World Health Organization (WHO) has declared the outbreak of novel coronavirus (2019-nCoV) ? now known as Coronavirus Disease (COVID-19)1 - as a global health emergency. Singapore currently stands as the country with the highest number of reported cases of COVID-19 outside of China2, excluding patients on a cruise ship offshore of Japan.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Targeted next-generation sequencing for the detection of ciprofloxacin resistance markers using molecular inversion probes</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 May 13</span></i><p style="text-align: justify; text-justify: inter-word;">Antibiotic resistance (AR) is an epidemic of increasing magnitude requiring rapid identification and profiling for appropriate and timely therapeutic measures and containment strategies. In this context, ciprofloxacin is part of the first-line of countermeasures against numerous high consequence bacteria. Significant resistance can occur via single nucleotide polymorphisms (SNP) and deletions within ciprofloxacin targeted genes. Ideally, use of ciprofloxacin would be prefaced with AR determination to avoid overuse or misuse of the antibiotic. Here, we describe the development and evaluation of a panel of 44 single-stranded molecular inversion probes (MIPs) coupled to next-generation sequencing (NGS) for the detection of genetic variants known to confer ciprofloxacin resistance in Bacillus anthracis, Yersinia pestis, and Francisella tularensis. Sequencing results demonstrate MIPs capture and amplify targeted regions of interest at significant levels of coverage. Depending on the genetic variant, limits of detection (LOD) for high-throughput pooled sequencing ranged from approximately 300–1800 input genome copies. LODs increased 10-fold in the presence of contaminating human genome DNA. In addition, we show that MIPs can be used as an enrichment step with high resolution melt (HRM) real-time PCR which is a sensitive assay with a rapid time-to-answer. Overall, this technology is a multiplexable upfront enrichment applicable with multiple downstream molecular assays for the detection of targeted genetic regions.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Targeted next-generation sequencing for the detection of ciprofloxacin resistance markers using molecular inversion probes</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 May 13</span></i><p style="text-align: justify; text-justify: inter-word;">Antibiotic resistance (AR) is an epidemic of increasing magnitude requiring rapid identification and profiling for appropriate and timely therapeutic measures and containment strategies. In this context, ciprofloxacin is part of the first-line of countermeasures against numerous high consequence bacteria. Significant resistance can occur via single nucleotide polymorphisms (SNP) and deletions within ciprofloxacin targeted genes. Ideally, use of ciprofloxacin would be prefaced with AR determination to avoid overuse or misuse of the antibiotic. Here, we describe the development and evaluation of a panel of 44 single-stranded molecular inversion probes (MIPs) coupled to next-generation sequencing (NGS) for the detection of genetic variants known to confer ciprofloxacin resistance in Bacillus anthracis, Yersinia pestis, and Francisella tularensis. Sequencing results demonstrate MIPs capture and amplify targeted regions of interest at significant levels of coverage. Depending on the genetic variant, limits of detection (LOD) for high-throughput pooled sequencing ranged from approximately 300–1800 input genome copies. LODs increased 10-fold in the presence of contaminating human genome DNA. In addition, we show that MIPs can be used as an enrichment step with high resolution melt (HRM) real-time PCR which is a sensitive assay with a rapid time-to-answer. Overall, this technology is a multiplexable upfront enrichment applicable with multiple downstream molecular assays for the detection of targeted genetic regions.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination of RT-qPCR Testing and Clinical Features For Diagnosis of COVID-19 facilitates management of SARS-CoV-2 Outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, a cluster of acute respiratory illness occurred in Wuhan, Hubei Province, China. This disease is now officially known as 2019 novel coronavirus disease (COVID-19) from WHO, novel coronavirus pneumonia This article is protected by copyright. All rights reserved.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Characteristics of and Public Health Responses to the Coronavirus Disease 2019 Outbreak in China</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December 2019, cases of unidentified pneumonia with a history of exposure in the Huanan Seafood Market were reported in Wuhan, Hubei Province. A novel coronavirus, SARS-CoV-2, was identified to be accountable for this disease. Human-to-human transmission is confirmed, and this disease (named COVID-19 by World Health Organization (WHO)) spread rapidly around the country and the world. As of 18 February 2020, the number of confirmed cases had reached 75,199 with 2009 fatalities. The COVID-19 resulted in a much lower case-fatality rate (about 2.67%) among the confirmed cases, compared with Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS). Among the symptom composition of the 45 fatality cases collected from the released official reports, the top four are fever, cough, short of breath, and chest tightness/pain. The major comorbidities of the fatality cases include hypertension, diabetes, coronary heart disease, cerebral infarction, and chronic bronchitis. The source of the virus and the pathogenesis of this disease are still unconfirmed. No specific therapeutic drug has been found. The Chinese Government has initiated a level-1 public health response to prevent the spread of the disease. Meanwhile, it is also crucial to speed up the development of vaccines and drugs for treatment, which will enable us to defeat COVID-19 as soon as possible.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Four ways researchers are responding to the COVID-19 outbreak</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts. How did researchers react so quickly to the SARS-CoV-2 epidemic? Nature Medicine has asked some key experts.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Q&A: The novel coronavirus outbreak causing COVID-19</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">What is COVID-19, and what do we know so far about its clinical presentation? The virus responsible for COVID-19, SARS-CoV-2, is in the species SARS-like corona viruses. At 125 nm, it is slightly larger than influenza, SARS and MERS viruses. It is almost certainly a descendant from a bat corona virus of which there are many. The closest is a virus that originated from the Rhinolophus bat which is > 96% homologous with the current SARS-CoV-2 virus. It is only 79% homologous with the original SARS CoV [1]. The near identical gene sequences of 90 analysed cases from outside of China suggests it has likely emerged after a solitary species jump in early December 2019 from an unknown (likely mammalian) intermediate host [2]. Pangolins are an endangered ant-eating mammal from which scientists in Guangzhou have shown a coronavirus with 99% homology, with a receptor binding domain identical to that of SARS-CoV-2. However, this has not been confirmed, and, in addition, the pangolin's rarity means this may not be the only mammal involved.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>The 2019-nCoV epidemic control strategies and future challenges of building healthy smart cities</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">reported and confirmed on 31 December 2019, in Wuhan, Hubei Province, China, which is one of China’s largest cities and a major domestic transport hub (located in the central part of China, as shown in Figure 1).1 The epidemic is attracting worldwide concern due to its rapid spread and transmission rate between humans. On 30 January 2020, the International Health Regulations, Emergency Committee of the World Health Organization (WHO) declared the outbreak – a ‘public health emergency of international concern’.2 On 8 February 2020 (24:00 GMT?+?8), there were 37,198 confirmed infections in China (including 811 deaths with a death ratio of 2.1%; and 6188 patients were confirmed in serious condition and 28,942 suspected cases).3 The COVID 19 infections were also reported in 26 other countries on 7 February 2020, including Canada, the USA, Australia, India, Sri-Lanka, Cambodia, Thailand, Vietnam, Malaysia, Singapore, Taiwan, the Republic of Korea, Sri Lanka, Japan, Philippines, Nepal, United Arab Emirates, Russia, Italy, Germany, Sweden, Finland, Belgium, Spain, France and the UK.4</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Can an Infectious Disease Genomics Project Predict and Prevent the Next Pandemic?</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Oct 26</span></i><p style="text-align: justify; text-justify: inter-word;">Infectious diseases need a globally coordinated genomic-based movement linking sequencing efforts to development of response tools to mitigate the impact of existing and emerging threats.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Hospital Emergency Management Plan During the COVID-19 Epidemic</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Abstract The confirmed and suspected cases of the 2019 novel coronavirus disease (COVID-19) have increased not only in Wuhan, Hubei Province but also China and the world. Enormous demand for handling the COVID-19 outbreak challenged both the healthcare personnel and medical supply system. In West China Hospital, Emergency Department (ED) undertook the mission of clinical reception, primary diagnosis, and interim treatment for the suspected cases of COVID-19.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Mystery deepens over animal source of coronavirus Podcast Extra: ‘There is lots of anxiety’: a scientist’s view from South Korea Coronavirus latest: Brazil reports first case in South America</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Pangolins are a prime suspect, but a slew of genetic analyses has yet to find conclusive proof. Pangolins are a prime suspect, but a slew of genetic analyses has yet to find conclusive proof. Nick Howe speaks to chemist Bartosz Grzybowski about his experience with the coronavirus outbreak in South Korea. Nick Howe speaks to chemist Bartosz Grzybowski about his experience with the coronavirus outbreak in South Korea. A case of COVID-19 has been confirmed in Brazil — the first in South America. On 26 February, Brazil’s minister of health, Luiz Henrique Mandetta, confirmed that a man who traveled to northern Italy between 9 and 21 February has the disease. Italy’s outbreak has escalated to 324 cases and 12 deaths, according to a virus tracker maintained by Johns Hopkins University in Baltimore, Maryland.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>“The disruption is enormous.” Coronavirus epidemic snarls science worldwide | Science | AAAS</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Normal daily life has come to a virtual standstill in large parts of China as a result of the epidemic of COVID-19—and so has science. Universities across the country remain closed; access to labs is restricted, projects have been mothballed, field work interrupted, and travel severely curtailed. But scientists elsewhere in the world are noticing an impact as well, as collaborations with China are on pause and scientific meetings for the next five months have been canceled or postponed. The damage to science pales compared to the human suffering; the total number of cases has risen to 71,429, the World Health Organization (WHO) reported today, almost 99% of them in China, and there have been 1775 deaths. Still, for individual researchers the losses can be serious—and stressful. “Basically, everything has completely stopped,” says John Speakman, who runs an animal behavior lab at the Chinese Academy of Sciences (CAS) in Beijing that has effectively been shut since the Lunar New Year on 25 January. “The disruption is enormous. The stress on the staff is really high.” But Speakman says he understands why the Chinese government took the measures. “It’s annoying, but I completely support what they have done,” he says.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>A Polymorphism within the Internal Fusion Loop of the Ebola Virus Glycoprotein Modulates Host Cell Entry</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">The large scale of the Ebola virus disease (EVD) outbreak in West Africa in 2013-2016 raised the question whether the host cell interactions of the responsible Ebola virus (EBOV) strain differed from those of other ebolaviruses. We previously reported that the glycoprotein (GP) of the virus circulating in West Africa in 2014 (EBOV2014) exhibited reduced ability to mediate entry into two nonhuman primate (NHP)-derived cell lines relative to the GP of EBOV1976. Here, we investigated the molecular determinants underlying the differential entry efficiency. We found that EBOV2014-GP-driven entry into diverse NHP-derived cell lines, as well as human monocyte-derived macrophages and dendritic cells, was reduced compared to EBOV1976-GP, although entry into most human- and all bat-derived cell lines tested was comparable. Moreover, EBOV2014 replication in NHP but not human cells was diminished relative to EBOV1976, suggesting that reduced cell entry translated into reduced viral spread. Mutagenic analysis of EBOV2014-GP and EBOV1976-GP revealed that an amino acid polymorphism in the receptor-binding domain, A82V, modulated entry efficiency in a cell line-independent manner and did not account for the reduced EBOV2014-GP-driven entry into NHP cells. In contrast, polymorphism T544I, located in the internal fusion loop in the GP2 subunit, was found to be responsible for the entry phenotype. These results suggest that position 544 is an important determinant of EBOV infectivity for both NHP and certain human target cells. IMPORTANCE The Ebola virus disease outbreak in West Africa in 2013 entailed more than 10,000 deaths. The scale of the outbreak and its dramatic impact on human health raised the question whether the responsible virus was particularly adept at infecting human cells. Our study shows that an amino acid exchange, A82V, that was acquired during the epidemic and that was not observed in previously circulating viruses, increases viral entry into diverse target cells. In contrast, the epidemic virus showed a reduced ability to enter cells of nonhuman primates compared to the virus circulating in 1976, and a single amino acid exchange in the internal fusion loop of the viral glycoprotein was found to account for this phenotype.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>PLANT-Dx: A Molecular Diagnostic for Point of Use Detection of Plant Pathogens</h3>
<i class="tooltip">2018<span class="tooltiptext">2018-12-17</span></i><p style="text-align: justify; text-justify: inter-word;">Synthetic biology based diagnostic technologies have improved upon gold standard diagnostic methodologies by decreasing cost, increasing accuracy, and enhancing portability. However there has been little effort in adapting these technologies towards applications related to point-of-use monitoring of plant and crop health. Here, we take a step towards this vision by developing an approach that couples isothermal amplification of specific plant pathogen genomic sequences with customizable synthetic RNA regulators that are designed to trigger the production of a colorimetric output in cell-free gene expression reactions. We demonstrate our system can sense viral derived sequences with high-sensitivity and specificity, and can be utilized to directly detect viruses from infected plant material. Furthermore, we demonstrate that the entire system can operate using only body heat and naked-eye visual analysis of outputs. We anticipate these strategies to be important components of user-friendly and deployable diagnostic systems that can be configured to detect a range of important plant pathogens.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>A CRISPR/Cas9 and Cre/Lox system-based express vaccine development strategy against re-emerging Pseudorabies virus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 18</span></i><p style="text-align: justify; text-justify: inter-word;">Virus evolves rapidly to escape vaccine-induced immunity, posing a desperate demand for efficient vaccine development biotechnologies. Here we present an express vaccine development strategy based on CRISPR/Cas9 and Cre/Lox system against re-emerging Pseudorabies virus, which caused the recent devastating swine pseudorabies outbreak in China. By CRISPR/Cas9 system, the virulent genes of the newly isolated strain were simultaneously substituted by marker genes, which were subsequently excised using Cre/Lox system for vaccine safety concern. Notably, single cell FACS technology was applied to further promote virus purification efficiency. The combination of these state-of-art technologies greatly accelerated vaccine development. Finally, vaccination and challenge experiments proved this vaccine candidate’s protective efficacy in pigs and the promise to control current pseudorabies outbreak. This is, to our knowledge, the first successful vaccine development based on gene edit technologies, demonstrating these technologies leap from laboratory to industry. It may pave the way for future express antiviral vaccine development.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>A CRISPR/Cas9 and Cre/Lox system-based express vaccine development strategy against re-emerging Pseudorabies virus</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Jan 18</span></i><p style="text-align: justify; text-justify: inter-word;">Virus evolves rapidly to escape vaccine-induced immunity, posing a desperate demand for efficient vaccine development biotechnologies. Here we present an express vaccine development strategy based on CRISPR/Cas9 and Cre/Lox system against re-emerging Pseudorabies virus, which caused the recent devastating swine pseudorabies outbreak in China. By CRISPR/Cas9 system, the virulent genes of the newly isolated strain were simultaneously substituted by marker genes, which were subsequently excised using Cre/Lox system for vaccine safety concern. Notably, single cell FACS technology was applied to further promote virus purification efficiency. The combination of these state-of-art technologies greatly accelerated vaccine development. Finally, vaccination and challenge experiments proved this vaccine candidate’s protective efficacy in pigs and the promise to control current pseudorabies outbreak. This is, to our knowledge, the first successful vaccine development based on gene edit technologies, demonstrating these technologies leap from laboratory to industry. It may pave the way for future express antiviral vaccine development.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Human genomics and preparedness for infectious threats</h3>
<i class="tooltip">2009<span class="tooltiptext">2009 Dec 29</span></i><p style="text-align: justify; text-justify: inter-word;">Public health preparedness requires effective surveillance of and rapid response to infectious disease outbreaks. Inclusion of research activities within the outbreak setting provides important opportunities to maximize limited resources, to enhance gains in scientific knowledge, and ultimately to increase levels of preparedness. With rapid advances in laboratory technologies, banking and analysis of human genomic specimens can be conducted as part of public health investigations, enabling valuable research well into the future.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Industry and Government Perspective in Influenza Control</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">We have had recent reminders of the threats posed by naturally occurring and bioengineered pandemic respiratory infections. It is estimated that if a pandemic infection were to arise anywhere in the world, such an infection would become widespread within 3 months and would have its maximum effect within 6 months. At present, the fastest that a vaccine effective against a new combination of antigens can be developed, purified, and produced is 9–12 months, not counting time for mass production. The current rate at which the production of influenza vaccines can be accelerated is limited by the fact that production is carried out in eggs. Therefore, there is urgent need for cell-based vaccine technologies. These are under way in several centers, yet attainment of a safe product remains several years away. Furthermore, there is need for public and private investment in manufacturing surge capacity and/or dedicated National Institutes of Health facilities to enable accelerated production. We must support efforts to shorten development time by developing and approving subunit antigens and immunogens that anticipate the most virulent viral mutations. Surveillance sites and their electronic interconnections must be expanded. Another component still lacking is funding for laboratories with high throughput screening and strong informatics capabilities to enable the fingerprinting and cataloguing of all known specimens of influenza and other pathogenic organisms for rapid identification of emerging or bioengineered pathogens. In all these efforts, we look to the federal government and to the biomedical research community in both public and private sectors.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Daily briefing: When a tiny lab accident almost leads to amputation</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Three weeks ago, on the basis of genetic analyses, pangolins became the prime suspect as the animal source of the coronavirus that causes COVID-19. Further analysis of those data — alongside three other genome studies of pangolin coronaviruses — shows that although the scaly anteater is still a contender, the link is far from conclusive. Whatever the source, scientists emphasize that animals should not be vilified for their role in the outbreak. “The problem is not the animals, it’s that we get in contact with them,” says animal-behaviour researcher Sara Platto. (Nature | 4 min read)</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating the Efficacy of Traffic Blockage and Quarantine for the Epidemic Caused by 2019-nCoV (COVID-19)</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-02-17</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Background: Since the 2019-nCoV (COVID-19) outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The quarantine and traffic blockage can alleviate the risk of the epidemic and the infections, henceforth evaluating the efficacy of such actions is essential to inform policy makers and raise the public awareness of the importance of self-isolation and quarantine.  Method: We collected confirmed case data and the migration data, and introduced the quarantine factor and traffic blockage factor to the Flow-SEIR model. By varying the quarantine factor and traffic blockage factor, we simulated the change of the peak number and arrival time of infections, then the efficacy of these two intervation measures can be analyzed in our simulation. In our study,  the self-protection at home is also included in quarantine.  Results: In the simulated results, the quarantine and traffic blockage are effective for epidemic control. For Hubei province, the current quarantine factor is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine, and the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who had flowed out from Hubei. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, and the current traffic blockage factor is estimated to be 0.26. With the quarantine and traffic blockage factor increasing, the number of infections decrease dramatically. We also simulated the start dates of quarantine and traffic blockage at four time points, the simulated results show that the early of warning is also effective for epidemic containing. However, provincial level traffic blockage can only alleviate 21.06% - 22.38% of the peak number of infections. In general, the quarantine is much more effective than the traffic blockage control.   Conclusion: Both of quarantine and traffic blockage are effective ways to control the spread of COVID-19. However, the eff  icacy of quarantine is found to be much stronger than that of traffic blockage. Considering traffic blockage may also cause huge losses of economy, we propose to gradually deregulate the traffic blockage, and improve quarantine instead. Also, there might be a large number of asymptomatic carriers of COVID-19, the quarantine should be continued for a long time until the epidemic is totally under control.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimated effectiveness of symptom and risk screening to prevent the spread of COVID-19</h3>
<i class="tooltip">(no date)<span class="tooltiptext">(no date)</span></i><p style="text-align: justify; text-justify: inter-word;">Traveller screening is being used to limit further spread of COVID-19 following its recent emergence, and symptom screening has become a ubiquitous tool in the global response. Previously, we developed a mathematical model to understand factors governing the effectiveness of traveller screening to prevent spread of emerging pathogens (Gostic et al., 2015). Here, we estimate the impact of different screening programs given current knowledge of key COVID-19 life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss more than half of infected people. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. Our work underscores the need for measures to limit transmission by individuals who become ill after being missed by a screening program. These findings can support evidence-based policy to combat the spread of COVID-19, and prospective planning to mitigate future emerging pathogens.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>COVID-19: Real-time dissemination of scientific information to fight a public health emergency of international concern</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">Rapidly sharing scientific information is an effective way to reduce public panic about COVID-19, and doing so is the key to providing real-time guidance to epidemiologists working to contain the outbreak, clinicians managing patients, and modelers helping to understand future developments and the possible effectiveness of various interventions. This issue has rapidly reviewed and published articles describing COVID-19, including the drug treatment options for SARS-CoV-2, its clinical characteristics, and therapies involving a combination of Chinese and Western medicine, the efficacy of chloroquine phosphate in the treatment of COVID-19 associated pneumonia according to clinical studies, and reflections on the system of reserve medical supplies for public health emergencies. As an academic journal, we will continue to quickly and transparently share data with frontline healthcare workers who need to know the epidemiological and clinical features of COVID-19.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Global Capacity for Emerging Infectious Disease Detection, 1996–2014</h3>
<i class="tooltip">2016<span class="tooltiptext">2016 Oct</span></i><p style="text-align: justify; text-justify: inter-word;">The speed with which disease outbreaks are recognized is critical for establishing effective control efforts. We evaluate global improvements in the timeliness of outbreak discovery and communication during 2010–2014 as a follow-up to a 2010 report. For all outbreaks reported by the World Health Organization’s Disease Outbreak News, we estimate the number of days from first symptoms until outbreak discovery and until first public communication. We report median discovery and communication delays overall, by region, and by Human Development Index (HDI) quartile. We use Cox proportional hazards regression to assess changes in these 2 outcomes over time, along with Loess curves for visualization. Improvement since 1996 was greatest in the Eastern Mediterranean and Western Pacific regions and in countries in the middle HDI quartiles. However, little progress has occurred since 2010. Further improvements in surveillance will likely require additional international collaboration with a focus on regions of low or unstable HDI.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-06</span></i><p style="text-align: justify; text-justify: inter-word;"><jats:p>Corona Virus Disease 2019 (COVID-19) has spread rapidly to more than 70 countries and regions overseas and over 80000 cases have been infected, resulting in more than three thousand deaths. Rapid diagnosis of patients remains a bottleneck in containing the progress of the epidemic. We used automated chemiluminescent immunoassay to detect serum IgM and IgG antibodies to 2019-nCoV of 736 subjects. COVID-19 patients were becoming reactive(positive) for specific antibodies from 7-12 days after the onset of morbidity. Specific IgM and IgG increased with the progression of the disease. The areas under the ROC curves of IgM and IgG were 0.988 and 1.000, respectively. Specific antibody detection has good sensitivity and specificity. Detection of specific antibodies in patients with fever can be a good distinction between COVID-19 and other diseases, so as to be a complement to nucleic acid diagnosis to early diagnosis of suspected cases.</jats:p></p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>PLANT-Dx: A Molecular Diagnostic for Point-of-Use Detection of Plant Pathogens</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 19</span></i><p style="text-align: justify; text-justify: inter-word;">[Image: see text] Synthetic biology based diagnostic technologies have improved upon gold standard diagnostic methodologies by decreasing cost, increasing accuracy, and enhancing portability. However, there has been little effort in adapting these technologies toward applications related to point-of-use monitoring of plant and crop health. Here, we take a step toward this vision by developing an approach that couples isothermal amplification of specific plant pathogen genomic sequences with customizable synthetic RNA regulators that are designed to trigger the production of a colorimetric output in cell-free gene expression reactions. We demonstrate our system can sense viral derived sequences with high sensitivity and specificity, and can be utilized to directly detect viruses from infected plant material. Furthermore, we demonstrate that the entire system can operate using only body heat and naked-eye visual analysis of outputs. We anticipate these strategies to be important components of user-friendly and deployable diagnostic systems that can be configured to detect a range of important plant pathogens.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>PLANT-Dx: A Molecular Diagnostic for Point-of-Use Detection of Plant Pathogens</h3>
<i class="tooltip">2019<span class="tooltiptext">2019 Apr 19</span></i><p style="text-align: justify; text-justify: inter-word;">[Image: see text] Synthetic biology based diagnostic technologies have improved upon gold standard diagnostic methodologies by decreasing cost, increasing accuracy, and enhancing portability. However, there has been little effort in adapting these technologies toward applications related to point-of-use monitoring of plant and crop health. Here, we take a step toward this vision by developing an approach that couples isothermal amplification of specific plant pathogen genomic sequences with customizable synthetic RNA regulators that are designed to trigger the production of a colorimetric output in cell-free gene expression reactions. We demonstrate our system can sense viral derived sequences with high sensitivity and specificity, and can be utilized to directly detect viruses from infected plant material. Furthermore, we demonstrate that the entire system can operate using only body heat and naked-eye visual analysis of outputs. We anticipate these strategies to be important components of user-friendly and deployable diagnostic systems that can be configured to detect a range of important plant pathogens.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>The Efficacy of the Interferon Alpha/Beta Response versus Arboviruses Is Temperature Dependent</h3>
<i class="tooltip">2018<span class="tooltiptext">2018 Apr 24</span></i><p style="text-align: justify; text-justify: inter-word;">Interferon alpha/beta (IFN-α/β) is a critical mediator of protection against most viruses, with host survival frequently impossible in its absence. Many studies have investigated the pathways involved in the induction of IFN-α/β after virus infection and the resultant upregulation of antiviral IFN-stimulated genes (ISGs) through IFN-α/β receptor complex signaling. However, other than examining the effects of genetic deletion of induction or effector pathway components, little is known regarding the functionality of these responses in intact hosts and whether host genetic or environmental factors might influence their potency. Here, we demonstrate that the IFN-α/β response against multiple arthropod-vectored viruses, which replicate over a wide temperature range, is extremely sensitive to fluctuations in temperature, exhibiting reduced antiviral efficacy at subnormal cellular temperatures and increased efficacy at supranormal temperatures. The effect involves both IFN-α/β and ISG upregulation pathways with a major aspect of altered potency reflecting highly temperature-dependent transcription of IFN response genes that leads to altered IFN-α/β and ISG protein levels. Discordantly, signaling steps prior to transcription that were examined showed the opposite effect from gene transcription, with potentiation at low temperature and inhibition at high temperature. Finally, we demonstrate that by lowering the temperature of mice, chikungunya arbovirus replication and disease are exacerbated in an IFN-α/β-dependent manner. This finding raises the potential for use of hyperthermia as a therapeutic modality for viral infections and in other contexts such as antitumor therapy. The increased IFN-α/β efficacy at high temperatures may also reflect an innate immune-relevant aspect of the febrile response.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>A precision medicine approach to managing Wuhan Coronavirus pneumonia</h3>
<i class="tooltip">2020<span class="tooltiptext">2020</span></i><p style="text-align: justify; text-justify: inter-word;">In December, 2019, several patients with pneumonia of an unknown cause were detected in Wuhan, China. On January 7, 2020, the causal organism was identified as a new coronavirus, later named as the “2019 novel coronavirus” (2019-nCoV). Genome sequencing found the genetic sequence of 2019-nCoV homologous to that of Severe Acute Respiratory Syndrome-Associated Coronavirus (SARS-CoV). As of January 29, 2020, the virus had been diagnosed in more than 7,000 patients in China and 77 outside this country. It is reported that both symptomatic and asymptomatic patients with 2019-nCov can play a role in disease transmission via airborne and contact. This finding has caused a great concern about the prevention of illness spread. The clinical features of the infection are not specific and are often indistinguishable from those of other respiratory infections, making it difficult to diagnose. Given that the virus has a strong ability to spread between individuals, it is of top priority to identify potential or suspected patients as soon as possible. Or the virus may cause a serious pandemic. Therefore, a precision medicine approach to managing this disease is urgently needed for detecting and controlling the spread of the virus. In this article, we present such an approach to managing 2019-nCoV-related pneumonia based on the unique traits of the virus recently revealed, and on our experience with coronaviruses at West China Hospital in Chengdu, China.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</h3>
<i class="tooltip">2020<span class="tooltiptext">2020-03-05</span></i><p style="text-align: justify; text-justify: inter-word;">Since December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</p>
<p align="right"><i>score: 38</i></p>

</div><br /><br /><br /></body>