
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: tracking the evolution of a virus</h2><br /></div><br /><br /><br /><div class="card"><h3>Ultra-deep mutant spectrum profiling: improving sequencing accuracy using overlapping read pairs</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: We demonstrate that overlapping read pairs (ORP) --generated by combining short fragment sequencing libraries and longer sequencing reads --significantly reduce sequencing error rates and improve rare variant detection accuracy. Using this sequencing protocol and an error model optimized for variant detection, we are able to capture a large number of genetic mutations present within a viral population at ultra-low frequency levels (<0.05%). Conclusions: Our rare variant detection strategies have important implications beyond viral evolution and can be applied to any basic and clinical research area that requires the identification of rare mutations.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Diversity in Heilongjiang Province, Northeast China</h3>
<p style="text-align: justify; text-justify: inter-word;">To trace the evolution of canine coronavirus (CCoV), 201 stool samples from diarrheic dogs in northeast China were subjected to reverse transcription-polymerase chain reactions (RT-PCRs) targeting the partial M and S genes of CCoV, followed by an epidemiological analysis. M gene RT-PCRs showed that 28.36% (57/201) of the samples were positive for CCoV; of the 57 positive samples, CCoV-I and CCoV-II accounted for 15.79% (9/57) and 84.21% (48/57), respectively. A sequence comparison of the partial M gene revealed nucleotide homologies of 88.4%-100% among the 57 CCoV strains, and 88.7%-96.2% identity between the 57 CCoV strains and the Chinese reference strain HF3. The CCoV-I and CCoV-II strains exhibited genetic diversity when compared with reference strains from China and other countries. The 57 CCoV strains exhibited high co-infection rates with canine kobuvirus (CaKV) (33.33%) and canine parvovirus-2 (CPV-2) (31.58%). The CCoV prevalence in diarrheic dogs differed significantly with immunization status, regions, seasons, and ages. Moreover, 28 S genes were amplified from the 57 CCoV-positive samples, including 26 CCoV-IIa strains, one CCoV-IIb strain, and one CCoV-I strain. A sequence comparison of the partial S gene revealed 86.3%-100% nucleotide identity among the 26 CCoV-IIa strains, and 89.6%-92.2% identity between the 26 CCoV-IIa strains and the Chinese reference strain V1. The 26 CCoV-IIa strains showed genetic diversity when compared with reference strains from China and other countries. Our data provide evidence that CCoV-I, CCoV-IIa, and CCoV-IIb strains co-circulate in the diarrhoetic dogs in northeast China, high co-infection rates with CaKV and CPV-2 were observed, and the CCoV-II strains exhibited high prevalence and genetic diversity.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Analysis of Avian Coronavirus Infectious Bronchitis Virus in Yellow Chickens in Southern China over the Past Decade: Revealing the Changes of Genetic Diversity, Dominant Genotypes, and Selection Pressure</h3>
<p style="text-align: justify; text-justify: inter-word;">The high mutation rates of infectious bronchitis virus (IBV) pose economic threats to the poultry industry. In order to track the genetic evolutionary of IBV isolates circulating in yellow chickens, we continued to conduct the genetic analyses of the structural genes S1, E, M, and N from 64 IBV isolates in southern China during 2009-2017. The results showed that the dominant genotypes based on the four genes had changed when compared with those during 1985-2008. Based on the S1 gene phylogenetic tree, LX4-type (GI-19) was the most dominant genotype, which was different from that during 1985-2008. The second most dominant genotype was LDT3-A-type, but this genotype disappeared after 2012. New-type 1 (GVI-1) isolates showed increasing tendency and there were four aa (QKEP) located in the hypervariable region (HVR) III and one aa (S) insertion in all the New-type 1 isolates. Both the analyses of amino acid entropy and molecular evolutionary rate revealed that the variations from large to small were S1, E, M, and N. Purifying selection was detected in the S1, E, M, and N gene proteins, which was different from the positive selection during 1985-2008. Six isolates were confirmed to be recombinants, possibly generated from a vaccine virus of the 4/91-type or LDT3-A-type and a circulating virus. The estimated times for the most recent common ancestors based on the S1, E, M, and N genes were the years of 1744, 1893, 1940, and 1945, respectively. Bayesian skyline analysis revealed a sharp decrease in genetic diversity of all the four structural genes after 2010 and since late 2015, the viral population rapidly rose. In conclusion, the IBVs circulating in southern China over the past decade have experienced a remarkable change in genetic diversity, dominant genotypes, and selection pressure, indicating the importance of permanent monitoring of circulating strains and the urgency for developing new vaccines to counteract the emerging LX4-type and New-type IBVs.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Characterization of the ORF3 and S1 Genes of Porcine Epidemic Diarrhea Virus Non S-INDEL Strains in Seven Regions of China</h3>
<p style="text-align: justify; text-justify: inter-word;">In an effort to trace the evolution of porcine epidemic diarrhea virus (PEDV), S1 and ORF3 genes of viruses identified in 41 pig farms from seven regions (North, Northeast, Northwest, Central, East, South West, and South, respectively) of China in 2015 were sequenced and analyzed. Sequence analysis revealed that the 41 ORF3 genes and 29 S1 genes identified in our study exhibited nucleotide homologies of 98.2%-100% and 96.6%-100%, respectively; these two genes exhibited low nucleotide sequence similarities with classical CV777 strain and early Chinese strain LZC. Phylogenetic analysis indicated that the identified PEDV strains belonged to global non S-INDEL strains, and exhibited genetic diversity; S1 gene of the HLJ2015/DP1-1 strain harbored an unique deletion of 12 nucleotides (A 1130 CAACTCCA CTG 1141 ); while the Chinese PEDV S-INDEL reference strains included two types of the "CV777" S-INDEL as well as the "US" S-INDEL, and all co-circulated with Chinese non S-INDEL strains. Of 29 identified S1 genes, the SS2 epitope (Y 748 SNIGVCK 755 ) was highly conserved, while the SS6 epitope (L 764 QDGQVKI 771 ) and pAPN receptor-binding region (aa 490-615) exhibited amino substitutions. Nine possible recombination events were identified between the 29 identifed S1 genes and the 3 S1 reference genes from early Chinese PEDV strains. The complete S genes of selected Chinese PEDV field strains (2011)(2012)(2013)(2014)(2015) showed 5.18%-6.07% nucleotide divergence, which is far higher than the divergence observed in early Chinese PEDV strains (3.1%) (P<0.05). Our data provide evidence that PEDV non S-INDEL strains with genetic diversities and potential recombination circulate in seven regions of China in 2015; Chinese PEDV S-INDEL strains exhibit genetic diversity and co-circulate with non S-INDEL strains.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Rhinovirus Genome Variation during Chronic Upper and Lower Respiratory Tract Infections</h3>
<p style="text-align: justify; text-justify: inter-word;">Routine screening of lung transplant recipients and hospital patients for respiratory virus infections allowed to identify human rhinovirus (HRV) in the upper and lower respiratory tracts, including immunocompromised hosts chronically infected with the same strain over weeks or months. Phylogenetic analysis of 144 HRV-positive samples showed no apparent correlation between a given viral genotype or species and their ability to invade the lower respiratory tract or lead to protracted infection. By contrast, protracted infections were found almost exclusively in immunocompromised patients, thus suggesting that host factors rather than the virus genotype modulate disease outcome, in particular the immune response. Complete genome sequencing of five chronic cases to study rhinovirus genome adaptation showed that the calculated mutation frequency was in the range observed during acute human infections. Analysis of mutation hot spot regions between specimens collected at different times or in different body sites revealed that non-synonymous changes were mostly concentrated in the viral capsid genes VP1, VP2 and VP3, independent of the HRV type. In an immunosuppressed lung transplant recipient infected with the same HRV strain for more than two years, both classical and ultra-deep sequencing of samples collected at different time points in the upper and lower respiratory tracts showed that these virus populations were phylogenetically indistinguishable over the course of infection, except for the last month. Specific signatures were found in the last two lower respiratory tract populations, including changes in the 59UTR polypyrimidine tract and the VP2 immunogenic site 2. These results highlight for the first time the ability of a given rhinovirus to evolve in the course of a natural infection in immunocompromised patients and complement data obtained from previous experimental inoculation studies in immunocompetent volunteers.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Genomics Microarray-based identification of antigenic variants of foot-and-mouth disease virus: a bioinformatics quality assessment</h3>
<p style="text-align: justify; text-justify: inter-word;">The evolution of viral quasispecies can influence viral pathogenesis and the response to antiviral treatments. Mutant clouds in infected organisms represent the first stage in the genetic and antigenic diversification of RNA viruses, such as foot and mouth disease virus (FMDV), an important animal pathogen. Antigenic variants of FMDV have been classically diagnosed by immunological or RT-PCR-based methods. DNA microarrays are becoming increasingly useful for the analysis of gene expression and single nucleotide polymorphisms (SNPs). Recently, a FMDV microarray was described to detect simultaneously the seven FMDV serotypes. These results encourage the development of new oligonucleotide microarrays to probe the fine genetic and antigenic composition of FMDV for diagnosis, vaccine design, and to gain insight into the molecular epidemiology of this pathogen.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Co-Circulation of the Rare CPV-2c with Unique Gln370Arg Substitution, New CPV-2b with Unique Thr440Ala Substitution, and New CPV-2a with High Prevalence and Variation in Heilongjiang Province, Northeast China</h3>
<p style="text-align: justify; text-justify: inter-word;">To trace evolution of canine parvovirus-2 (CPV-2), a total of 201 stool samples were collected from dogs with diarrhea in Heilongjiang province of northeast China from May 2014 to April 2015. The presence of CPV-2 in the samples was determined by PCR amplification of the VP2 gene (568 bp) of CPV-2. The results revealed that 95 samples (47.26%) were positive for CPV-2, and they showed 98.8%-100% nucleotide identity and 97.6%-100% amino acid identity. Of 95 CPV-2-positive samples, types new2a (Ser297Ala), new2b (Ser297Ala), and 2c accounted for 64.21%, 21.05%, and 14.74%, respectively. The positive rate of CPV-2 and the distribution of the new2a, new2b and 2c types exhibited differences among regions, seasons, and ages. Immunized dogs accounted for 48.42% of 95 CPV-2-positive samples. Coinfections with canine coronavirus, canine kobuvirus, and canine bocavirus were identified. Phylogenetic analysis revealed that the identified new2a, new2b, and CPV-2c strains in our study exhibited a close relationship with most of the CPV-2 strains from China; type new2a strains exhibited high variability, forming three subgroups; type new2b and CPV-2c strains formed one group with reference strains from China. Of 95 CPV-2 strains, Tyr324Ile and Thr440Ala substitutions accounted for 100% and 64.21%, respectively; all type new2b strains exhibited the Thr440Ala substitution, while the unique Gln370Arg substitution was found in all type 2c strains. Recombination analysis using entire VP2 gene indicated possible recombination events between the identified CPV-2 strains and reference strains from China. Our data revealed the co-circulation of new CPV-2a, new CPV-2b, and rare CPV-2c, as well as potential recombination events among Chinese CPV-2 strains.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultra-deep mutant spectrum profiling: improving sequencing accuracy using overlapping read pairs</h3>
<p style="text-align: justify; text-justify: inter-word;">Backgound: High throughput sequencing is beginning to make a transformative impact in the area of viral evolution. Deep sequencing has the potential to reveal the mutant spectrum within a viral sample at high resolution, thus enabling the close examination of viral mutational dynamics both within-and between-hosts. The challenge however, is to accurately model the errors in the sequencing data and differentiate real viral mutations, particularly those that exist at low frequencies, from sequencing errors.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Characteristics of Traveler with Middle East Respiratory Syndrome, China, 2015</h3>
<p style="text-align: justify; text-justify: inter-word;">Tracking the spread of RHDV2 in Australia, in competition with existing field strains, highlights the value of Australia's rabbits and their diseases as a model system for emerging infectious diseases. The point releases of both myxoma virus and RHDV into large naive host populations represent a grand experiment in disease emergence and evolution (10), which provides a unique opportunity to study the virulence evolution of emerging pathogens as well as their complex interactions with each other. It is notable that since the release of RHDV in Australia in 1995, strains of 1 viral lineage dominate the viral population nationwide despite hundreds of deliberate rereleases of the original virus strain for local rabbit control, which strongly suggests it has a major selective advantage (7). That RHDV2 appeared in a wild rabbit is therefore remarkable, particularly because Australian field strains were spreading simultaneously in the same area. Comparing the epidemiology of this strain in Australia to the epidemiology of its well-documented spread in Europe will provide valuable insights into RHDV epidemiology relevant to both continents.</p>
<p align="right"><i>score: 121</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment</h3>
<p style="text-align: justify; text-justify: inter-word;">Preparedness for a possible global pandemic caused by viruses such as the highly pathogenic influenza A subtype H5N1 has become a global priority. In particular, it is critical to monitor the appearance of any new emerging subtypes. Comparative phyloinformatics can be used to monitor, analyze, and possibly predict the evolution of viruses. However, in order to utilize the full functionality of available analysis packages for largescale phyloinformatics studies, a team of computer scientists, biostatisticians and virologists is needed -a requirement which cannot be fulfilled in many cases. Furthermore, the time complexities of many algorithms involved leads to prohibitive runtimes on sequential computer platforms. This has so far hindered the use of comparative phyloinformatics as a commonly applied tool in this area.</p>
<p align="right"><i>score: 120</i></p>

</div><br /><br /><br /><div class="card"><h3>The global spread of Middle East respiratory syndrome: an analysis fusing traditional epidemiological tracing and molecular phylodynamics</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusion: Together, these analyses can help us identify factors associated with viral evolution and spread as well as establish efficacy of infection control measures. The results are especially pertinent to countries without current MERS-CoV endemic, since their unfamiliarity makes them particularly susceptible to uncontrollable spread of a virus that may be imported by travelers.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Infectious Diseases A simple and rapid approach for screening of SARS-coronavirus genotypes: an evaluation study</h3>
<p style="text-align: justify; text-justify: inter-word;">The Severe Acute Respiratory Syndrome (SARS) was a newly emerged infectious disease which caused a global epidemic in [2002][2003]. Sequence analysis of SARS-coronavirus isolates revealed that specific genotypes predominated at different periods of the epidemic. This information can be used as a footprint for tracing the epidemiology of infections and monitor viral evolution. However, direct sequencing analysis of a large number of clinical samples is cumbersome and time consuming. We present here a simple and rapid assay for the screening of SARScoronavirus genotypes based on the use of fluorogenic oligonucleotide probes for allelic discrimination.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolutionary and genetic analysis of the VP2 gene of canine parvovirus</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Canine parvovirus (CPV) type 2 emerged in 1978 in the USA and quickly spread among dog populations all over the world with high morbidity. Although CPV is a DNA virus, its genomic substitution rate is similar to some RNA viruses. Therefore, it is important to trace the evolution of CPV to monitor the appearance of mutations that might affect vaccine effectiveness. Results: Our analysis shows that the VP2 genes of CPV isolated from 1979 to 2016 are divided into six groups: GI, GII, GIII, GIV, GV, and GVI. Amino acid mutation analysis revealed several undiscovered important mutation sites: F267Y, Y324I, and T440A. Of note, the evolutionary rate of the CPV VP2 gene from Asia and Europe decreased. Codon usage analysis showed that the VP2 gene of CPV exhibits high bias with an ENC ranging from 34.93 to 36.7. Furthermore, we demonstrate that natural selection plays a major role compared to mutation pressure driving CPV evolution. Conclusions: There are few studies on the codon usage of CPV. Here, we comprehensively studied the genetic evolution, codon usage pattern, and evolutionary characterization of the VP2 gene of CPV. The novel findings revealing the evolutionary process of CPV will greatly serve future CPV research.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Respiratory Syncytial Virus whole- genome sequencing identifies convergent evolution of sequence duplication in the C-terminus of the G gene OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">circulate are likely mechanisms contributing to the evasion of herd immunity. Surveillance and largescale whole-genome sequencing of RSV is currently limited but would help identify its evolutionary dynamics and sites of selective immune evasion. In this study, we performed complete-genome nextgeneration sequencing of 92 RSV isolates from infants in central Tennessee during the 2012-2014 RSV seasons. We identified multiple co-circulating clades of RSV from both the A and B groups. Each clade is defined by signature N-and O-linked glycosylation patterns. Analyses of specific RSV genes revealed high rates of positive selection in the attachment (G) gene. We identified RSV-A viruses in circulation with and without a recently reported 72-nucleotide G gene sequence duplication. Furthermore, we show evidence of convergent evolution of G gene sequence duplication and fixation over time, which suggests a potential fitness advantage of RSV with the G sequence duplication.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>An unusually high substitution rate in transplant-associated BK polyomavirus in vivo is further concentrated in HLA-C-bound viral peptides Author summary</h3>
<p style="text-align: justify; text-justify: inter-word;">Infection with human BK polyomavirus, a small double-stranded DNA virus, potentially results in severe complications in immunocompromised patients. Here, we describe the in vivo variability and evolution of the BK polyomavirus by deep sequencing. Our data reveal the highest genomic evolutionary rate described in double-stranded DNA viruses, i.e., 10 −3 -10 −5 substitutions per nucleotide site per year. High mutation rates in viruses allow their escape from immune surveillance and adaptation to new hosts. By combining mutational landscapes across viral genomes with in silico prediction of viral peptides, we demonstrate the presence of significantly more coding substitutions within predicted cognate HLA-C-bound viral peptides than outside. This finding suggests a role for HLA-C in antiviral immunity, perhaps through the action of killer cell immunoglobulin-like receptors. The present study provides a comprehensive view of viral evolution and immune escape in a DNA virus.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen's evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies. IMPORTANCE Surveillance programs have become key assets in determining the emergence or prevalence of pathogens circulating in human and animal populations. Genomic surveillance, in particular, provides comprehensive information on the history of isolates and potential molecular markers for infectivity and pathogenicity. Current techniques for evaluating genomic surveillance are inaccurate, ignoring the pathogen's evolutionary rate and biodiversity, as well as the timing of sequence collection. Using sequence data, we propose the q2 coefficient as a quantitative measure of surveillance completeness that combines elements of time and evolution without defining arbitrary criteria for clades or species. Through several case studies of influenza A, dengue, and West Nile viruses, we employed the q2 coefficient to identify sampling deficiencies in different host species and locations, as well as examine the effects of different public health policies through historical records of the q2 coefficient. These results can guide public health agencies to focus resource allocation and virus collection to bolster specific problems in surveillance.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Kingdom of Saudi Arabia e ; Saudi Aramco Medical Services Organization, Saudi Aramco, Dhahran, Kingdom of Saudi Arabia f ; Prince Sultan Military Medical City, Riyadh, Kingdom of Saudi Arabia g</h3>
<p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first documented in the Kingdom of Saudi Arabia (KSA) in 2012 and, to date, has been identified in 180 cases with 43% mortality. In this study, we have determined the MERS-CoV evolutionary rate, documented genetic variants of the virus and their distribution throughout the Arabian peninsula, and identified the genome positions under positive selection, important features for monitoring adaptation of MERS-CoV to human transmission and for identifying the source of infections. Respiratory samples from confirmed KSA MERS cases from May to September 2013 were subjected to whole-genome deep sequencing, and 32 complete or partial sequences (20 were >99% complete, 7 were 50 to 94% complete, and 5 were 27 to 50% complete) were obtained, bringing the total available MERS-CoV genomic sequences to 65. An evolutionary rate of 1.12 ؋ 10 ؊3 substitutions per site per year (95% credible interval [95% CI], 8.76 ؋ 10 ؊4 ; 1.37 ؋ 10 ؊3 ) was estimated, bringing the time to most recent common ancestor to March 2012 (95% CI, December 2011; June 2012). Only one MERS-CoV codon, spike 1020, located in a domain required for cell entry, is under strong positive selection. Four KSA MERS-CoV phylogenetic clades were found, with 3 clades apparently no longer contributing to current cases. The size of the population infected with MERS-CoV showed a gradual increase to June 2013, followed by a decline, possibly due to increased surveillance and infection control measures combined with a basic reproduction number (R 0 ) for the virus that is less than 1. IMPORTANCE MERS-CoV adaptation toward higher rates of sustained human-to-human transmission appears not to have occurred yet. While MERS-CoV transmission currently appears weak, careful monitoring of changes in MERS-CoV genomes and of the MERS epidemic should be maintained. The observation of phylogenetically related MERS-CoV in geographically diverse locations must be taken into account in efforts to identify the animal source and transmission of the virus.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>Short-Sighted Virus Evolution and a Germline Hypothesis for Chronic Viral Infections</h3>
<p style="text-align: justify; text-justify: inter-word;">With extremely short generation times and high mutability, many viruses can rapidly evolve and adapt to changing environments.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</h3>
<p style="text-align: justify; text-justify: inter-word;">Sequence analyses of RNA virus genomes remain challenging due to the exceptional genetic plasticity of these viruses. Because of high mutation and recombination rates, genome replication by viral RNA-dependent RNA polymerases leads to populations of closely related viruses, so-called 'quasispecies'. Standard (short-read) sequencing technologies are ill-suited to reconstruct large numbers of full-length haplotypes of (i) RNA virus genomes and (ii) subgenome-length (sg) RNAs comprised of noncontiguous genome regions. Here, we used a full-length, direct RNA sequencing (DRS) approach based on nanopores to characterize viral RNAs produced in cells infected with a human coronavirus. Using DRS, we were able to map the longest (∼26 kb) contiguous read to the viral reference genome. By combining Illumina and nanopore sequencing, we reconstructed a highly accurate consensus sequence of the human coronavirus (HCoV) 229E genome (27.3 kb). Furthermore, using long reads that did not require an assembly step, we were able to identify, in infected cells, diverse and novel HCoV-229E sg RNAs that remain to be characterized. Also, the DRS approach, which circumvents reverse transcription and amplification of RNA, allowed us to detect methylation sites in viral RNAs. Our work paves the way for haplotype-based analyses of viral quasispecies by demonstrating the feasibility of intra-sample haplotype separation. Even though several technical challenges remain to be addressed to exploit the potential of the nanopore technology fully, our work illustrates that direct RNA sequencing may significantly advance genomic studies of complex virus populations, including predictions on long-range interactions in individual full-length viral RNA haplotypes.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Local Evolutionary Patterns of Human Respiratory Syncytial Virus Derived from Whole-Genome Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">RSV molecular epidemiology, evolution, and transmission requires detailed sequence data and can inform new strategies for virus control and vaccine development. We have generated 27 complete or nearly complete genomes of RSV from hospitalized children attending a rural coastal district hospital in Kilifi, Kenya, over a 10-year period using a novel full-genome deep-sequencing process. Phylogenetic analysis of the new genomes demonstrated the existence and cocirculation of multiple genotypes in both RSV A and B groups in Kilifi. Comparison of local versus global strains demonstrated that most RSV A variants observed locally in Kilifi were also seen in other parts of the world, while the Kilifi RSV B genomes encoded a high degree of variation that was not observed in other parts of the world. The nucleotide substitution rates for the individual open reading frames (ORFs) were highest in the regions encoding the attachment (G) glycoprotein and the NS2 protein. The analysis of RSV full genomes, compared to subgenomic regions, provided more precise estimates of the RSV sequence changes and revealed important patterns of RSV genomic variation and global movement. The novel sequencing method and the new RSV genomic sequences reported here expand our knowledge base for large-scale RSV epidemiological and transmission studies.</p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /></body>