
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: methods for coordinating data-gathering with standardized nomenclature</h2><br /></div><br /><br /><br /><div class="card"><h3>Collecting Data To Assess SARS Interventions</h3>
<p style="text-align: justify; text-justify: inter-word;">A minimum set of data on intervention effectiveness should be collected in a uniform manner from each identified SARS case-patient at each location. Without such standardization, datasets from different locales may not be sufficiently comparable, thereby limiting the ability to scientifically evaluate both the effect of SARS and the interventions to control and prevent its spread.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Journal of Biomedical Discovery and Collaboration Biological information specialists for biological informatics</h3>
<p style="text-align: justify; text-justify: inter-word;">Data management and integration are complicated and ongoing problems that will require commitment of resources and expertise from the various biological science communities. Primary components of successful cross-scale integration are smooth information management and migration from one context to another. We call for a broadening of the definition of bioinformatics and bioinformatics training to span biological disciplines and biological scales. Training programs are needed that educate a new kind of informatics professional, Biological Information Specialists, to work in collaboration with various discipline-specific research personnel. Biological Information Specialists are an extension of the informationist movement that began within library and information science (LIS) over 30 years ago as a professional position to fill a gap in clinical medicine. These professionals will help advance science by improving access to scientific information and by freeing scientists who are not interested in data management to concentrate on their science.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Nonproliferative and Proliferative Lesions of the Gastrointestinal Tract, Pancreas and Salivary Glands of the Rat and Mouse</h3>
<p style="text-align: justify; text-justify: inter-word;">The INHAND (International Harmonization of Nomenclature and Diagnostic Criteria for Lesions in Rats and Mice) project is a joint initiative of the Societies of Toxicologic Pathology from Europe (ESTP), Great Britain (BSTP), Japan (JSTP), and North America (STP) to develop an internationally accepted nomenclature and diagnostic criteria for nonproliferative and proliferative lesions in laboratory animals. The purpose of this publication is to provide a standardized nomenclature and diagnostic criteria for classifying lesions in the digestive system including the salivary glands and the exocrine pancreas of laboratory rats and mice. Most lesions are illustrated by color photomicrographs. The standardized nomenclature, the diagnostic criteria, and the photomicrographs are also available electronically on the Internet (http://www.goreni.org/). Sources of material included histopathology databases from government, academia, and industrial laboratories throughout the world. Content includes spontaneous and age related lesions as well as lesions induced by exposure to test items. Relevant infectious and parasitic lesions are included as well. A widely accepted and utilized international harmonization of nomenclature and diagnostic criteria for the digestive system will decrease misunderstandings among regulatory and scientific research organizations in different countries and provide a common language to increase and enrich international exchanges of information among toxicologists and pathologists.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>SNAD: sequence name annotation-based designer</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: A growing diversity of biological data is tagged with unique identifiers (UIDs) associated with polynucleotides and proteins to ensure efficient computer-mediated data storage, maintenance, and processing. These identifiers, which are not informative for most people, are often substituted by biologically meaningful names in various presentations to facilitate utilization and dissemination of sequence-based knowledge. This substitution is commonly done manually that may be a tedious exercise prone to mistakes and omissions.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Microbiology The Microbial Rosetta Stone Database: A compilation of global and emerging infectious microorganisms and bioterrorist threat agents</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Thousands of different microorganisms affect the health, safety, and economic stability of populations. Many different medical and governmental organizations have created lists of the pathogenic microorganisms relevant to their missions; however, the nomenclature for biological agents on these lists and pathogens described in the literature is inexact. This ambiguity can be a significant block to effective communication among the diverse communities that must deal with epidemics or bioterrorist attacks.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Immunome Research An ontology for immune epitopes: application to the design of a broad scope database of immune reactivities</h3>
<p style="text-align: justify; text-justify: inter-word;">The IEDB's ontology is the first ontology specifically designed to capture both intrinsic chemical and biochemical information relating to immune epitopes with information relating to the interaction of these structures with molecules derived from the host immune system. We anticipate that the development of this type of ontology and associated databases will facilitate rigorous description of data related to immune epitopes, and might ultimately lead to completely new methods for describing and modeling immune responses.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<p style="text-align: justify; text-justify: inter-word;">This disparity in technologies and data collection methods results in incomplete surveillance.</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Exploratory Analysis of Methods for Automated Classification of Laboratory Test Orders into Syndromic Groups in Veterinary Medicine</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Recent focus on earlier detection of pathogen introduction in human and animal populations has led to the development of surveillance systems based on automated monitoring of health data. Real-or near real-time monitoring of pre-diagnostic data requires automated classification of records into syndromes-syndromic surveillance-using algorithms that incorporate medical knowledge in a reliable and efficient way, while remaining comprehensible to end users.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>SNAD: sequence name annotation-based designer</h3>
<p style="text-align: justify; text-justify: inter-word;">A tool for controllable annotation-based conversion of sequence UIDs into biologically meaningful names and acronyms has been developed and placed into service, fostering links between quality of sequence annotation, and efficiency of communication and knowledge dissemination among researchers.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenges with using names to link digital biodiversity information</h3>
<p style="text-align: justify; text-justify: inter-word;">The need for a names-based cyber-infrastructure for digital biology is based on the argument that scientific names serve as a standardized metadata system that has been used consistently and near universally for 250 years. As we move towards data-centric biology, name-strings can be called on to discover, index, manage, and analyze accessible digital biodiversity information from multiple sources. Known impediments to the use of scientific names as metadata include synonyms, homonyms, mis-spellings, and the use of other strings as identifiers. We here compare the name-strings in GenBank, Catalogue of Life (CoL), and the Dryad Digital Repository (DRYAD) to assess the effectiveness of the current names-management toolkit developed by Global Names to achieve interoperability among distributed data sources. New tools that have been used here include Parser (to break name-strings into component parts and to promote the use of canonical versions of the names), a modified TaxaMatch fuzzy-matcher (to help manage typographical, transliteration, and OCR errors), and Cross-Mapper (to make comparisons among data sets). The data sources include scientific names at multiple ranks; vernacular (common) names; acronyms; strain identifiers and other surrogates including idiosyncratic abbreviations and concatenations. About 40% of the name-strings in GenBank are scientific names representing about 400,000 species or infraspecies and their synonyms. ‡ § | ¶,# © Patterson D et al. This is an open access article distributed under the terms of the Creative Commons Attribution License (CC BY 4.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Veterinary Research Linkages between animal and human health sentinel data</h3>
<p style="text-align: justify; text-justify: inter-word;">We found limited use of animal data to quantitatively predict human risk and listed the methods from the literature that performed analytic linkage. The lack of analytic linkage in the literature might not be solely related to technological barriers including access to electronic database, statistical software packages, and Geographical Information System (GIS). Rather, the problem might be from a lack of understanding by researchers of the importance of animal data as a 'sentinel' for human health. Researchers performing zoonotic surveillance should be aware of the value of animal-sentinel approaches for predicting human risk and consider analytic methods for linking animal and human data. Qualitative work needs to be done in order to examine researchers' decisions in linkage strategies between animal and human data.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Sharing public health data and information across borders: lessons from Southeast Asia</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: There has been a great intensification in the circulation of data, information, and expertise across borders in Southeast Asia. However, findings from this study document ways in which the movement of data and information from production sites to other places can be challenging due to different standards and practices, language barriers, different national structures and rules that govern the circulation of health information inside and outside countries, imbalances in capacities and power, and sustainability of financing arrangements.</p>
<p align="right"><i>score: 55</i></p>

</div><br /><br /><br /><div class="card"><h3>Collecting Data To Assess SARS Interventions</h3>
<p style="text-align: justify; text-justify: inter-word;">With cases of severe acute respiratory syndrome (SARS) occurring across geographic regions, data collection on the effectiveness of intervention strategies should be standardized to facilitate analysis. We propose a minimum dataset to capture data needed to examine the basic reproduction rate, case status and criteria, symptoms, and outcomes of SARS. F irst detected in China, confirmed and probable cases of severe acute respiratory syndrome (SARS) have now appeared in at least 30 countries in five continents. SARS is the first new severe infectious disease to occur in the 21st century, and little is known about its epidemiologic features (1). To assess the effect of SARS on public health and outcomes, data are needed about who becomes ill, how they contracted their illness, and the sequelae.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Globalization and Health Antibiotic resistance as a global threat: Evidence from China, Kuwait and the United States</h3>
<p style="text-align: justify; text-justify: inter-word;">We study the recent patterns of antibiotic resistance in three geographically separated, and culturally and economically distinct countries -China, Kuwait and the United States -to gauge the range and depth of this global health threat, and its potential for growth as globalization expands. Our primary measures are the prevalence of resistance of specific bacteria to specific antibiotics. We also propose and illustrate methods for aggregating specific "bug-drug" data. We use these aggregate measures to summarize the resistance pattern for each country and to study the extent of correlation between countries' patterns of drug resistance.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated vocabulary discovery for geo-parsing online epidemic intelligence</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Automated surveillance of the Internet provides a timely and sensitive method for alerting on global emerging infectious disease threats. HealthMap is part of a new generation of online systems designed to monitor and visualize, on a real-time basis, disease outbreak alerts as reported by online news media and public health sources. HealthMap is of specific interest for national and international public health organizations and international travelers. A particular task that makes such a surveillance useful is the automated discovery of the geographic references contained in the retrieved outbreak alerts. This task is sometimes referred to as "geo-parsing". A typical approach to geo-parsing would demand an expensive training corpus of alerts manually tagged by a human.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>SHORT REPORT Use of linked electronic health records to assess mortality and length of stay associated with pandemic influenza A(H1N1)pdm09 at a UK teaching hospital</h3>
<p style="text-align: justify; text-justify: inter-word;">Effective use of data linkage is becoming an increasingly important focus in the new healthcare system in England. We linked data from the results of a multiplex PCR assay for respiratory viruses for a population of 230 inpatients at a UK teaching hospital with their patient administrative system records in order to compare the mortality and length of stay of patients who tested positive for influenza A(H1N1)pdm09 with those positive for another influenza A virus. The results indicated a reduced risk of death among influenza A(H1N1)pdm09 patients compared to other influenza A strains, with an adjusted risk ratio of 0·25 (95% confidence interval 0·08-0·75, P = 0·01), while no significant differences were found between the lengths of stay in the hospital for these two groups. Further development of such methods to link hospital data in a routine fashion could provide a rapid means of gaining epidemiological insights into emerging infectious diseases.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Advances in Visualization Tools for Phylogenomic and Phylodynamic Studies of Viral Diseases</h3>
<p style="text-align: justify; text-justify: inter-word;">Genomic and epidemiological monitoring have become an integral part of our response to emerging and ongoing epidemics of viral infectious diseases. Advances in high-throughput sequencing, including portable genomic sequencing at reduced costs and turnaround time, are paralleled by continuing developments in methodology to infer evolutionary histories (dynamics/patterns) and to identify factors driving viral spread in space and time. The traditionally static nature of visualizing phylogenetic trees that represent these evolutionary relationships/processes has also evolved, albeit perhaps at a slower rate. Advanced visualization tools with increased resolution assist in drawing conclusions from phylogenetic estimates and may even have potential to better inform public health and treatment decisions, but the design (and choice of what analyses are shown) is hindered by the complexity of information embedded within current phylogenetic models and the integration of available meta-data. In this review, we discuss visualization challenges for the interpretation and exploration of reconstructed histories of viral epidemics that arose from increasing volumes of sequence data and the wealth of additional data layers that can be integrated. We focus on solutions that address joint temporal and spatial visualization but also consider what the future may bring in terms of visualization and how this may become of value for the coming era of real-time digital pathogen surveillance, where actionable results and adequate intervention strategies need to be obtained within days.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /></body>