
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: can livestock be infected by the virus</h2><br /></div><br /><br /><br /><div class="card"><h3>Inoculation of Goats, Sheep, and Horses with MERS-CoV Does Not Result in Productive Viral Shedding</h3>
<p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first recognized in 2012 and can cause severe disease in infected humans. Dromedary camels are the reservoir for the virus, although, other than nasal discharge, these animals do not display any overt clinical disease. Data from in vitro experiments suggest that other livestock such as sheep, goats, and horses might also contribute to viral transmission, although field data has not identified any seropositive animals. In order to understand if these animals could be infected, we challenged young goats and horses and adult sheep with MERS-CoV by intranasal inoculation. Minimal or no virus shedding was detected in all of the animals. During the four weeks following inoculation, neutralizing antibodies were detected in the young goats, but not in sheep or horses.</p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Tropical Medicine and Infectious Disease Potential Intermediate Hosts for Coronavirus Transmission: No Evidence of Clade 2c Coronaviruses in Domestic Livestock from Ghana</h3>
<p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV), nearly a decade ago with worldwide distribution, was believed to be of zoonotic origin from bats with dromedary camels as intermediate hosts. There is a likelihood of other domestic livestock serving as intermediate hosts for this virus. The presence of coronaviruses, closely related to MERS-CoV in Ghanaian bats, presented the opportunity to test the hypothesis of transmissibility of this virus through domestic livestock species. The possible interactions between livestock and bats in 31 household farms were accessed by observation and interviews with farmers. Rectal swabs and serum from cattle, sheep, goats, donkeys, and swine from commercial and household farms were tested for MERS-CoV and a Nycteris sp. bat coronavirus, previously detected in Ghana. A pan-PCR assay to detect clade 2c viruses and recombinant immunofluorescence assay to detect anti-spike IgG antibodies against the target viruses were used. Likely contact between livestock and bats was determined for 13 farms (41.9%) that reported confining their livestock and also observing bats in their homes. Livestock were left unconfined on eight farms (25.8%) that also observed bats roosting in trees close to their homes. No viral RNA or antibodies against the two coronaviruses were detected in any of the livestock species tested. Cattle, sheep, goats, donkeys, and swine are not likely hosts of clade 2c coronaviruses.</p>
<p align="right"><i>score: 123</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus infection in non-camelid domestic mammals</h3>
<p style="text-align: justify; text-justify: inter-word;">Dromedary camels are natural host of the Middle East respiratory syndrome coronavirus (MERS-CoV). However, there are limited studies of MERS-CoV infection of other domestic mammals exposed to infected dromedaries. We expanded our surveillance among camels in Egypt, Tunisia, and Senegal to include other domestic mammalian species in contact with infected camels. A total of 820 sera and 823 nasal swabs from cattle, sheep, goats, donkeys, buffaloes, mules, and horses were collected. Swabs were tested using RT-PCR and virus RNA-positive samples were genetically sequenced and phylogenetically analysed. Sera were screened using virus microneutralization tests and positive sera (where available) were confirmed using plaque reduction neutralization tests (PRNT). We detected 90% PRNT confirmed MERS-CoV antibody in 35 (55.6%) of 63 sera from sheep collected from Senegal, two sheep (1.8%) of 114 in Tunisia and a goat (0.9%) of 107 in Egypt, with titres ranging from 1:80 to â‰¥1:320. We detected MERS-CoV RNA in swabs from three sheep (1.2%) of 254 and five goats (4.1%) of 121 from Egypt and Senegal, as well as one cow (1.9%) of 53 and three donkeys (7.1%) of 42 from Egypt. Partial sequences of the RT-PCR amplicons confirmed specificity of the results. This study showed that domestic livestock in contact with MERS-CoV infected camels may be at risk of infection. We recommend expanding current MERS-CoV surveillance in animals to include other livestock in close contact with dromedary camels. The segregation of camels from other livestock in farms and live animal markets may need to be considered.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63</h3>
<p style="text-align: justify; text-justify: inter-word;">Known human coronaviruses are believed to have originated in animals and made use of intermediate hosts for transmission to humans. The intermediate hosts of most of the human coronaviruses are known, but not for HCoV-NL63. This study aims to assess the possible role of some major domestic livestock species as intermediate hosts of HCoV-NL63. We developed a testing algorithm for high throughput screening of livestock sera with ELISA and confirmation with recombinant immunofluorescence assay testing for antibodies against HCoV-NL63 in livestock. Optimization of the ELISA showed a capability of the assay to significantly distinguish HCoV-NL63 from HCoV-229E (U = 27.50, p < 0.001) and HCoV-OC43 (U = 55.50, p < 0.001) in coronavirus-characterized sera. Evaluation of the assay with collected human samples showed no significant difference in mean optical density values of immunofluorescence-classified HCoV-NL63-positive and HCoV-NL63-negative samples (F (1, 215) = 0.437, p = 0.509). All the top 5% (n = 8) most reactive human samples tested by ELISA were HCoV-NL63 positive by immunofluorescence testing. In comparison, only a proportion (84%, n = 42) of the top 25% were positive by immunofluorescence testing, indicating an increased probability of the highly ELISA reactive samples testing positive by the immunofluorescence assay. None of the top 5% most ELISA reactive livestock samples were positive for HCoV-NL63-related viruses by immunofluorescence confirmation. Ghanaian domestic livestock are not likely intermediate hosts of HCoV-NL63-related coronaviruses.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>First detection and genetic characterization of canine Kobuvirus in domestic dogs in Thailand</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Canine Kobuvirus (CaKoV) has been detected both in healthy and diarrheic dogs and in asymptomatic wild carnivores. In this study, we conducted a survey of CaKoV at small animal hospitals in Bangkok and vicinity of Thailand during September 2016 to September 2018. Results: Three hundred and seven rectal swab samples were collected from healthy dogs (n = 55) and dogs with gastroenteritis symptoms (n = 252). Of 307 swab samples tested by using one-step RT-PCR specific to 3D gene, we found CaKoV positivity at 17.59% (54/307). CaKoVs could be detected in both sick (19.44%) and healthy (9.09%) animals. In relation to age group, CaKoV could be frequently detected in younger dogs (25.45%). Our result showed no seasonal pattern of CaKoV infection in domestic dogs. In this study, we characterized CaKoVs by whole genome sequencing (n = 4) or 3D and VP1 gene sequencing (n = 8). Genetic and phylogenetic analyses showed that whole genomes of Thai CaKoVs were closely related to Chinese CaKoVs with highest 99.5% amino acid identity suggesting possible origin of CaKoVs in Thailand. Conclusions: In conclusion, this study was the first to report the detection and genetic characteristics of CaKoVs in domestic dogs in Thailand. CaKoVs could be detected in both sick and healthy dogs. The virus is frequently detected in younger dogs. Thai CaKoVs were genetically closely related and grouped with Chinese CaKoVs. Our result raises the concerns to vet practitioners that diarrhea in dogs due to canine Kobuvirus infection should not be ignored.</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and phylogenetic analysis of canine kobuviruses in diarrhoetic dogs in northeast China</h3>
<p style="text-align: justify; text-justify: inter-word;">Canine kobuviruses (CaKVs) are newly recognized picornaviruses that have been recently detected in dogs in the U.S.A., Italy, U.K., the Republic of Korea and Tanzania. To trace the evolution of CaKV strains, a total of 201 fecal samples from rectal swabs of diarrheic dogs, which were obtained from May 2014 to April 2015 in northeast China, were detected by reverse transcription-PCR targeting a partial (504 bp) fragment of the 3D gene. Furthermore, a phylogenetic analysis of the CaKV strains identified in northeast China was conducted based on the partial 3D gene sequence. The results indicated that 36 fecal samples (17.91%, 36/201) were positive for CaKV, in which the co-infection rates of canine coronavirus, canine parvovirus-2 and canine bocavirus were 58.33%, 41.67%, and 11.11%, respectively. Sequence comparison of the partial 3D gene revealed nucleotide homologies of 94.4-100%, 95.6-98.6%, 94.3-97.6%, 94.4-96.3% and 93.3-95.1% within the 36 Chinese CaKV strains, and between the 36 Chinese CaKV strains and four CaKV reference strains from South Korea, Italy, U.S.A. and Tanzania, respectively. A phylogenetic tree revealed that the 36 Chinese CaKV strains formed one specific CaKV lineage with CaKVs that have recently been identified in other countries. The 36 Chinese CaKV strains were closely related to CaKV reference strains from Asia and Europe, but differed genetically from CaKV reference strains from North America and Africa. This study provides evidence that CaKVs circulate in diarrhoetic dogs in China and that they exhibit substantial genetic diversity and high co-infection rates with other enteric viruses.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Running title: The transcriptional and translational landscape of equine torovirus 1</h3>
<p style="text-align: justify; text-justify: inter-word;">sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the 14 prototype species equine torovirus (EToV) we performed parallel RNA sequencing 15 (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels 16 of the known torovirus proteins and transcripts, chimaeric sequences produced via 17 discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and 18 changes in host transcription and translation as a result of EToV infection. RNA 19 sequencing confirmed that EToV utilises a unique combination of discontinuous and 20 non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we 21 identified transcripts arising from both mechanisms that would result in sgRNAs 22 encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes 23 efficiently translate two novel CUG-initiated ORFs, located within the so-called 5' 24 UTR. We have termed the resulting proteins U1 and U2. Comparative genomic 25 analysis confirmed that these ORFs are conserved across all available torovirus 26 . CC-BY 4.0</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>A seroepidemiologic study of Reston ebolavirus in swine in the Philippines</h3>
<p style="text-align: justify; text-justify: inter-word;">The high prevalence of REBOV infection in swine in the affected farms in 2008 suggests that swine is susceptible for REBOV infection. The multiple serological assays used in the study are thought to be useful for future surveillance of REOBV infection in swine in the Philippines.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Accumulation of Virulent Rift Valley Fever Virus in Mice from an Attenuated Virus Carrying a Single Nucleotide Substitution in the M RNA</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Rift Valley fever virus (RVFV), a member of the genus Phlebovirus within the family Bunyaviridae, is a negativestranded RNA virus with a tripartite genome. RVFV is transmitted by mosquitoes and causes fever and severe hemorrhagic illness among humans, while in livestock it causes fever and high abortion rates.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Evidence of infectivity of airborne porcine epidemic diarrhea virus and detection of airborne viral RNA at long distances from infected herds</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine epidemic diarrhea virus (PEDV) spread rapidly after being diagnosed in the USA in April 2013. In this study we assessed whether PEDV could become airborne and if so, whether the virus was infectious. Air samples were collected both from a room containing experimentally infected pigs and at various distances from the outside of swine farms experiencing acute PEDV outbreaks. Results indicated presence of infectious PEDV in the air from experimentally infected pigs and genetic material of PEDV was detected up to 10 miles downwind from naturally infected farms. Airborne transmission should be considered as a potential route for PEDV dissemination.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>viruses Phleboviruses and the Type I Interferon Response</h3>
<p style="text-align: justify; text-justify: inter-word;">Phleboviruses are arboviruses that are taxonomically divided into dipteran-and tick-borne viruses. Dipteran-borne phleboviruses are generally found in eponymous Phlebotomus sandflies [22, 31] , with RVFV representing an outlier that is associated with Aedes and Culex mosquitoes, and more promiscuous in its vector range. The specific vector species are thought to be predominantly responsible for the maintenance of the viruses by vertical (transovarial) transmission, the geographic distribution of the virus and the spatial and temporal occurrence of the specific disease. Given the increasing spread of competent vector species, concerns have been raised about the potential introduction of RVFV into new areas with both susceptible vectors and hosts, and potential consequences for the human population and massive economic loss caused among affected livestock [32, 33] .</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular epidemiology and phylogenetic analysis of diverse bovine astroviruses associated with diarrhea in cattle and water buffalo calves in China</h3>
<p style="text-align: justify; text-justify: inter-word;">Astroviruses are the principal causative agents of gastroenteritis in humans and have been associated with diarrhea in other mammals as well as birds. However, astroviral infection of animals had been poorly studied. In the present study, 211 rectal swabs collected from cattle and water buffalo calves with mild to severe diarrhea were tested for bovine astrovirus (BAstV) by RT-PCR. Results: 92/211 (43.6%) samples were positive for BAstV, at a rate of 46.10% (71/154) in cattle and 36.84% (21/57) in water buffalo. Phylogenetic analysis based on the partial and full-length of 25 ORF2 amino acid sequences obtained in this study classified the Guangxi BAstVs isolates into five subgroups under the genus of Mamastrovirus, genotype MAstV33, which suggested that the water buffalo was a new host of this genogroup that previously included only cattle and roe deer. Despite the origin of the host, the Guangxi BAstV isolates were closely related to the BAstV Hong Kong isolates (B18/HK and B76-2/HK), but highly divergent from the BAstV NeuroS1 isolate previously associated with neurologic disease in cattle in the U.S.A. Nucleotide sequence-based characterization of the ORF1b/ORF2 junction and corresponding overlapping regions showed distinctive properties, which may be common to BAstVs. Our results suggested that cattle and water buffalo are prone to infection of closely related astroviruses, which probably evolved from the same ancestor. The current study described astroviruses in water buffalo for the first time and is thus far among the largest epidemiological investigations of BAstV infection in cattle conducted in China.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>NOTE Development of a fluorescent loop-mediated isothermal amplification assay for rapid and simple diagnosis of bovine leukemia virus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Serological tests that detect persistent anti-BLV antibodies, such as agar-gel immunodiffusion and enzyme linked immunosorbent assays (ELISAs) are routinely used worldwide for the effective screening of BLV-infected cattle [9, 11, 16] . However, these tests are insufficient to correctly diagnose the early stages of infection [5, 9, 16] . Therefore, supplemental use of a rapid and sensitive assay for the detection of BLV-genomic DNA is required to reinforce accurate diagnosis.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Lack of Middle East Respiratory Syndrome Coronavirus Transmission in Rabbits</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) transmission from dromedaries to humans has resulted in major outbreaks in the Middle East. Although some other livestock animal species have been shown to be susceptible to MERS-CoV, it is not fully understood why the spread of the virus in these animal species has not been observed in the field. In this study, we used rabbits to further characterize the transmission potential of MERS-CoV. In line with the presence of MERS-CoV receptor in the rabbit nasal epithelium, high levels of viral RNA were shed from the nose following virus inoculation. However, unlike MERS-CoV-infected dromedaries, these rabbits did not develop clinical manifestations including nasal discharge and did shed only limited amounts of infectious virus from the nose. Consistently, no transmission by contact or airborne routes was observed in rabbits. Our data indicate that despite relatively high viral RNA levels produced, low levels of infectious virus are excreted in the upper respiratory tract of rabbits as compared to dromedary camels, thus resulting in a lack of viral transmission.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Tilapia Lake Virus Does Not Hemagglutinate Avian and Piscine Erythrocytes and NH 4 Cl Does Not Inhibit Viral Replication In Vitro</h3>
<p style="text-align: justify; text-justify: inter-word;">Tilapia lake virus (TiLV) is a negative-sense single-stranded RNA (-ssRNA) icosahedral virus classified to be the only member in the family Amnoonviridae. Although TiLV segment-1 shares homology with the influenza C virus PB1 and has four conserved motifs similar to influenza A, B, and C polymerases, it is unknown whether there are other properties shared between TiLV and orthomyxovirus. In the present study, we wanted to determine whether TiLV agglutinated avian and piscine erythrocytes, and whether its replication was inhibited by lysosomotropic agents, such as ammonium chloride (NH 4 Cl), as seen for orthomyxoviruses. Our findings showed that influenza virus strain A/Puerto Rico/8 (PR8) was able to hemagglutinate turkey (Meleagris gallopavo), Atlantic salmon (Salmo salar L), and Nile tilapia (Oreochromis niloticus) red blood cells (RBCs), while infectious salmon anemia virus (ISAV) only agglutinated Atlantic salmon, but not turkey or tilapia, RBCs. In contrast to PR8 and ISAV, TiLV did not agglutinate turkey, Atlantic salmon, or tilapia RBCs. qRT-PCR analysis showed that 30 mM NH 4 Cl, a basic lysosomotropic agent, neither inhibited nor enhanced TiLV replication in E-11 cells. There was no difference in viral quantities in the infected cells with or without NH 4 Cl treatment during virus adsorption or at 1, 2, and 3 h post-infection. Given that hemagglutinin proteins that bind RBCs also serve as ligands that bind host cells during virus entry leading to endocytosis in orthomyxoviruses, the data presented here suggest that TiLV may use mechanisms that are different from orthomyxoviruses for entry and replication in host cells. Therefore, future studies should seek to elucidate the mechanisms used by TiLV for entry into host cells and to determine its mode of replication in infected cells.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>In Vivo Characterisation of Five Strains of Bovine Viral Diarrhoea Virus 1 (Subgenotype 1c)</h3>
<p style="text-align: justify; text-justify: inter-word;">Bovine viral diarrhoea virus 1 (BVDV-1) is strongly associated with several important diseases of cattle, such as bovine respiratory disease, diarrhoea and haemoragic lesions. To date many subgenotypes have been reported for BVDV-1, currently ranging from subgenotype 1a to subgenotype 1u. While BVDV-1 has a world-wide distribution, the subgenotypes have a more restricted geographical distribution. As an example, BVDV-1 subgenotypes 1a and 1b are frequently detected in North America and Europe, while the subgenotype 1c is rarely detected. In contrast, BVDV-1 subgenotype 1c is by far the most commonly reported in Australia. Despite this, uneven distribution of the biological importance of the subgenotypes remains unclear. The aim of this study was to characterise the in vivo properties of five strains of BVDV-1 subgenotype 1c in cattle infection studies. No overt respiratory signs were reported in any of the infected cattle regardless of strain. Consistent with other subgenotypes, transient pyrexia and leukopenia were commonly identified, while thrombocytopenia was not. The quantity of virus detected in the nasal secretions of transiently infected animals suggested the likelihood of horizontal transmission was very low. Further studies are required to fully understand the variability and importance of the BVDV-1 subgenotype 1c.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Metagenomic Next-Generation Sequencing Reveal Presence of a Novel Ungulate Bocaparvovirus in Alpacas</h3>
<p style="text-align: justify; text-justify: inter-word;">Viruses belonging to the genus Bocaparvovirus (BoV) are a genetically diverse group of DNA viruses known to cause respiratory, enteric, and neurological diseases in animals, including humans. An intestinal sample from an alpaca (Vicugna pacos) herd with reoccurring diarrhea and respiratory disease was submitted for next-generation sequencing, revealing the presence of a BoV strain. The alpaca BoV strain (AlBoV) had a 58.58% whole genome nucleotide percent identity to a camel BoV from Dubai, belonging to a tentative ungulate BoV 8 species (UBoV8). Recombination events were lacking with other UBoV strains. The AlBoV genome was comprised of the NS1, NP1, and VP1 proteins. The NS1 protein had the highest amino acid percent identity range (57.89-67.85%) to the members of UBoV8, which was below the 85% cut-off set by the International Committee on Taxonomy of Viruses. The low NS1 amino acid identity suggests that AlBoV is a tentative new species. The whole genome, NS1, NP1, and VP1 phylogenetic trees illustrated distinct branching of AlBoV, sharing a common ancestor with UBoV8. Walker loop and Phospholipase A2 (PLA2) motifs that are vital for virus infectivity were identified in NS1 and VP1 proteins, respectively. Our study reports a novel BoV strain in an alpaca intestinal sample and highlights the need for additional BoV research.</p>
<p align="right"><i>score: 92</i></p>

</div><br /><br /><br /><div class="card"><h3>Stability of Porcine Epidemic Diarrhea Virus on Fomite Materials at Different Temperatures</h3>
<p style="text-align: justify; text-justify: inter-word;">Indirect transmission of porcine epidemic diarrhea virus (PEDV) ensues when susceptible animals contact PEDV-contaminated fomite materials. Although the survival of PEDV under various pHs and temperatures has been studied, virus stability on different fomite surfaces under varying temperature conditions has not been explored. Hence, we evaluated the survival of PEDV on inanimate objects routinely used on swine farms such as styrofoam, rubber, plastic, coveralls, and other equipment. The titer of infectious PEDV at 4 â€¢ C decreased by only 1 to 2 log during the first 5 days, and the virus was recoverable for up to 15 days on Styrofoam, aluminum, Tyvek Â® coverall, cloth, and plastic. However, viral titers decreased precipitously when stored at room temperature; no virus was detectable after one day on all materials tested. A more sensitive immunoplaque assay was able to detect virus from Styrofoam, metal, and plastic at 20 days post application, representing a 3-log loss of input virus on fomite materials. Recovery of infectious PEDV from Tyvek Â® coverall and rubber was above detection limit at 20 days. Our findings indicate that the type of fomite material and temperatures impact PEDV stability, which is important in understanding the nuances of indirect transmission and epidemiology of PEDV.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>viruses Animals as Reservoir for Human Norovirus</h3>
<p style="text-align: justify; text-justify: inter-word;">Norovirus is the most common cause of non-bacterial gastroenteritis and is a burden worldwide. The increasing norovirus diversity is currently categorized into at least 10 genogroups which are further classified into more than 40 genotypes. In addition to humans, norovirus can infect a broad range of hosts including livestock, pets, and wild animals, e.g., marine mammals and bats. Little is known about norovirus infections in most non-human hosts, but the close genetic relatedness between some animal and human noroviruses coupled with lack of understanding where newly appearing human norovirus genotypes and variants are emerging from has led to the hypothesis that norovirus may not be host restricted and might be able to jump the species barrier. We have systematically reviewed the literature to describe the diversity, prevalence, and geographic distribution of noroviruses found in animals, and the pathology associated with infection. We further discuss the evidence that exists for or against interspecies transmission including surveillance data and data from in vitro and in vivo experiments.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Phylogenetic Analysis of Astrovirus and Kobuvirus in Korean Dogs</h3>
<p style="text-align: justify; text-justify: inter-word;">Astroviruses and kobuviruses are frequently found in mammalian feces, including that of humans. The present study examined fecal samples from 91 Korean dogs suffering from diarrhea. Canine astroviruses (CAstVs) and canine kobuviruses (CKoVs) were identified in 2 (2.1%) and 46 (50.6%) dogs, respectively. Nucleotide sequence analysis coupled with phylogenetic analysis using the neighbor-joining method showed that CAstVs clustered into four genetically diverse groups. Two Korean CAstVs belonged to group 2 alongside strains isolated in Italy and France. Twelve of the Korean CKoVs belonged to a single clade, along with strain UK003 identified in the UK and six CKoVs identified in the USA. Thus, the results suggest that the Korean strain of CAstV is closely related to strains isolated in Europe. Surely, CKoV in South Korea could identify the circulation among dogs population.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /></body>