
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: new platforms and technology to improve response times to future diseases</h2><br /></div><br /><br /><br /><div class="card"><h3>A Simple Platform for the Rapid Development of Antimicrobials</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent infectious outbreaks highlight the need for platform technologies that can be quickly deployed to develop therapeutics needed to contain the outbreak. We present a simple concept for rapid development of new antimicrobials. The goal was to produce in as little as one week thousands of doses of an intervention for a new pathogen. We tested the feasibility of a system based on antimicrobial synbodies. The system involves creating an array of 100 peptides that have been selected for broad capability to bind and/or kill viruses and bacteria. The peptides are pre-screened for low cell toxicity prior to large scale synthesis. Any pathogen is then assayed on the chip to find peptides that bind or kill it. Peptides are combined in pairs as synbodies and further screened for activity and toxicity. The lead synbody can be quickly produced in large scale, with completion of the entire process in one week.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a Broadly Accessible Venezuelan Equine Encephalitis Virus Replicon Particle Vaccine Platform</h3>
<p style="text-align: justify; text-justify: inter-word;">Zoonotic viruses circulate as swarms in animal reservoirs and can emerge into human populations, causing epidemics that adversely affect public health. Portable, safe, and effective vaccine platforms are needed in the context of these outbreak and emergence situations. In this work, we report the generation and characterization of an alphavirus replicon vaccine platform based on a non-select agent, attenuated Venezuelan equine encephalitis (VEE) virus vaccine, strain 3526 (VRP 3526). Using both noroviruses and coronaviruses as model systems, we demonstrate the utility of the VRP 3526 platform in the generation of recombinant proteins, production of virus-like particles, and in vivo efficacy as a vaccine against emergent viruses. Importantly, packaging under biosafety level 2 (BSL2) conditions distinguishes VRP 3526 from previously reported alphavirus platforms and makes this approach accessible to the majority of laboratories around the world. In addition, improved outcomes in the vulnerable aged models as well as against heterologous challenge suggest improved efficacy compared to that of previously attenuated VRP approaches. Taking these results together, the VRP 3526 platform represents a safe and highly portable system that can be rapidly deployed under BSL2 conditions for generation of candidate vaccines against emerging microbial pathogens. IMPORTANCE While VEE virus replicon particles provide a robust, established platform for antigen expression and vaccination, its utility has been limited by the requirement for high-containment-level facilities for production and packaging. In this work, we utilize an attenuated vaccine strain capable of use at lower biocontainment level but retaining the capacity of the wild-type replicon particle. Importantly, the new replicon platform provides equal protection for aged mice and following heterologous challenge, which distinguishes it from other attenuated replicon platforms. Together, the new system represents a highly portable, safe system for use in the context of disease emergence.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Recombinant vesicular stomatitis virus vector vaccines for WHO blueprint priority pathogens</h3>
<p style="text-align: justify; text-justify: inter-word;">The devastating Ebola virus (EBOV) outbreak in West Africa in 2013-2016 has flagged the need for the timely development of vaccines for high-threat pathogens. To be better prepared for new epidemics, the WHO has compiled a list of priority pathogens that are likely to cause future outbreaks and for which R&D efforts are, therefore, paramount (R&D Blueprint: https://www.who.int/blueprint/priority-diseases/en/). To this end, the detailed characterization of vaccine platforms is needed. The vesicular stomatitis virus (VSV) has been established as a robust vaccine vector backbone for infectious diseases for well over a decade. The recent clinical trials testing the vaccine candidate VSV-EBOV against EBOV disease now have added a substantial amount of clinical data and suggest VSV to be an ideal vaccine vector candidate for outbreak pathogens. In this review, we discuss insights gained from the clinical VSV-EBOV vaccine trials as well as from animal studies investigating vaccine candidates for Blueprint pathogens.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Programmable low-cost DNA-based platform for viral RNA detection</h3>
<p style="text-align: justify; text-justify: inter-word;">Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal preamplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ~24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>A Toolkit for Rapid Modular Construction of Biological Circuits in Mammalian Cells</h3>
<p style="text-align: justify; text-justify: inter-word;">As a proof of concept, we built a hAAVS1 landing pad for single integration of genetic circuits, created and delivered libraries of protein variants and sgRNAs and rapidly compared multi-species Cas9-based genetic circuits. Finally, we used Ebola virus (EBOV) as a test case to address an emerging need to streamline the generation of BSL2 reagents during outbreaks of novel BSL4 viruses. Using the MTK, we rapidly built a non-infectious multicistronic ribonucleoprotein expression system for EBOV in a mammalian host. This system recapitulates the transcription and replication sub-lifecycle of the virus in a mammalian host, and can therefore be used to accelerate the study of viral biology and discovery of antiviral molecules.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Screening for Entry Inhibitors of Highly Pathogenic Viruses under Low-Level Biocontainment</h3>
<p style="text-align: justify; text-justify: inter-word;">Emerging viruses including Nipah, Hendra, Lujo, and Junin viruses have enormous potential to spread rapidly. Nipah virus, after emerging as a zoonosis, has also evolved the capacity for human-to-human transmission. Most of the diseases caused by these pathogens are untreatable and require high biocontainment conditions. Universal methods for rapidly identifying and screening candidate antivirals are urgently needed. We have developed a modular antiviral platform strategy that relies on simple bioinformatic and genetic information about each pathogen. Central to this platform is the use of envelope glycoprotein cDNAs to establish multi-cycle replication systems under BSL2 conditions for viral pathogens that normally require BSL3 and BSL4 facilities. We generated monoclonal antibodies against Nipah G by cDNA immunization in rats, and we showed that these antibodies neutralize both Nipah and Hendra live viruses. We then used these effective Henipavirus inhibitors to validate our screening strategy. Our proposed strategy should contribute to the response capability for emerging infectious diseases, providing a way to initiate antiviral development immediately upon identifying novel viruses.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Make Data Sharing Routine to Prepare for Public Health Emergencies</h3>
<p style="text-align: justify; text-justify: inter-word;">â€¢ Ongoing research that does not directly relate to an emergency now may be critical for the next unpredictable outbreak.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>New Vaccine Technologies to Combat Outbreak Situations</h3>
<p style="text-align: justify; text-justify: inter-word;">Ever since the development of the first vaccine more than 200 years ago, vaccinations have greatly decreased the burden of infectious diseases worldwide, famously leading to the eradication of small pox and allowing the restriction of diseases such as polio, tetanus, diphtheria, and measles. A multitude of research efforts focuses on the improvement of established and the discovery of new vaccines such as the HPV (human papilloma virus) vaccine in 2006. However, radical changes in the density, age distribution and traveling habits of the population worldwide as well as the changing climate favor the emergence of old and new pathogens that bear the risk of becoming pandemic threats. In recent years, the rapid spread of severe infections such as HIV, SARS, Ebola, and Zika have highlighted the dire need for global preparedness for pandemics, which necessitates the extremely rapid development and comprehensive distribution of vaccines against potentially previously unknown pathogens. What is more, the emergence of antibiotic resistant bacteria calls for new approaches to prevent infections. Given these changes, established methods for the identification of new vaccine candidates are no longer sufficient to ensure global protection. Hence, new vaccine technologies able to achieve rapid development as well as large scale production are of pivotal importance. This review will discuss viral vector and nucleic acid-based vaccines (DNA and mRNA vaccines) as new approaches that might be able to tackle these challenges to global health.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Genome Sequencing of RNA Viruses</h3>
<p style="text-align: justify; text-justify: inter-word;">We developed a system for rapid determination of viral RNA sequences whereby genomic sequence is obtained from cultured virus isolates without subcloning into plasmid vectors. This method affords new opportunities to address the challenges of unknown or untypeable emerging viruses.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Wide Identification of SARS-CoV Susceptibility Loci Using the Collaborative Cross</h3>
<p style="text-align: justify; text-justify: inter-word;">New systems genetics approaches are needed to rapidly identify host genes and genetic networks that regulate complex disease outcomes. Using genetically diverse animals from incipient lines of the Collaborative Cross mouse panel, we demonstrate a greatly expanded range of phenotypes relative to classical mouse models of SARS-CoV infection including lung pathology, weight loss and viral titer. Genetic mapping revealed several loci contributing to differential disease responses, including an 8.5Mb locus associated with vascular cuffing on chromosome 3 that contained 23 genes and 13 noncoding RNAs. Integrating phenotypic and genetic data narrowed this region to a single gene, Trim55, an E3 ubiquitin ligase with a role in muscle fiber maintenance. Lung pathology and transcriptomic data from mice genetically deficient in Trim55 were used to validate its role in SARS-CoV-induced vascular cuffing and inflammation. These data establish the Collaborative Cross platform as a powerful genetic resource for uncovering genetic contributions of complex traits in microbial disease severity, inflammation and virus replication in models of outbred populations.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines 1 2</h3>
<p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to 8 prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved 9 coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current 10 and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a 11 conserved 2'O methyltransferase (MTase), could provide a broad attenuation platform against 12 future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was 13 evaluated for protection from heterologous challenge, efficacy in the aging host, and potential 14 for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and 15 potential for reversion makes targeting NSP16 alone an untenable approach. However, 16 combining a 2'O MTase mutation with a previously described CoV fidelity mutant produced a 17 vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, 18 and no evidence for reversion. Together, the results indicate that targeting the CoV 2'O MTase 19 in parallel with other conserved attenuating mutations may provide a platform strategy for 20 rapidly generating live-attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct nucleic acid analysis of mosquitoes for high fidelity species identification and 1 detection of Wolbachia using a cellphone 2 3</h3>
<p style="text-align: justify; text-justify: inter-word;">portable, nucleic acid diagnostic platforms are needed that rapidly produce accurate results without 101 requiring complex procedures, instruments, and laboratory infrastructure. In this regard, isothermal 102 nucleic acid amplification assays such as loop-mediated isothermal amplification (LAMP) have 103 begun to be employed because they do not require complex thermocycling instruments [29][30][31]. 104</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Access to</h3>
<p style="text-align: justify; text-justify: inter-word;">Given the interconnected nature of our world today, emerging pathogens and pandemic outbreaks are an ever-growing threat to the health and economic stability of the global community. This is evident by the recent 2009 Influenza A (H1N1) pandemic, the SARS outbreak, as well as the ever-present threat of global bioterrorism. Fortunately, the biomedical community has been able to rapidly generate sequence data so these pathogens can be readily identified. To date, however, the utilization of this sequence data to rapidly produce relevant experimental results or actionable treatments is lagging in spite of obtained sequence data. Thus, a pathogenic threat that has emerged and/or developed into a pandemic can be rapidly identified; however, translating this identification into a targeted therapeutic or treatment that is rapidly available has not yet materialized. This commentary suggests that the growing technology of DNA synthesis should be fully implemented as a means to rapidly generate in vivo data and possibly actionable therapeutics soon after sequence data becomes available.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Antibody Derived Peptides for Detection of Ebola Virus Glycoprotein</h3>
<p style="text-align: justify; text-justify: inter-word;">Current Ebola virus (EBOV) detection methods are costly and impractical for epidemic scenarios. Different immune-based assays have been reported for the detection and quantification of Ebola virus (EBOV) proteins. In particular, several monoclonal antibodies (mAbs) have been described that bind the capsid glycoprotein (GP) of EBOV GP. However, the currently available platforms for the design and production of full-length mAbs are cumbersome and costly. The use of antibody fragments, rather than full-length antibodies, might represent a cost-effective alternative for the development of diagnostic and possibly even therapeutic alternatives for EBOV.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</h3>
<p style="text-align: justify; text-justify: inter-word;">We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Clinical-Stage Human Monoclonal Antibodies That Treat Advanced Ebola Virus Disease in Nonhuman Primates</h3>
<p style="text-align: justify; text-justify: inter-word;">Background. For most classes of drugs, rapid development of therapeutics to treat emerging infections is challenged by the timelines needed to identify compounds with the desired efficacy, safety, and pharmacokinetic profiles. Fully human monoclonal antibodies (mAbs) provide an attractive method to overcome many of these hurdles to rapidly produce therapeutics for emerging diseases.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Recent years have seen unprecedented investment in research and development for countermeasures for high-threat pathogens, including specific and ambitious objectives for development of diagnostics, therapeutics, and vaccines. The inadequate availability of biological reference materials for these pathogens poses a genuine obstacle in pursuit of these objectives, and the lack of a comprehensive and equitable framework for developing reference materials is a weakness. We outline the need for internationally standardized biological materials for high-threat pathogens as a core element of global health security. We also outline the key components of a framework for addressing this deficiency.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>An approach to and web-based tool for infectious disease outbreak intervention analysis OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Despite substantial public health improvements in the last century, infectious diseases remain one of the leading causes of both morbidity and mortality 1,2 . When confronted with an infectious disease outbreak, public health officials typically work to control the outbreak by performing assessments, analyzing surveillance data, identifying resources and interacting with subject mater experts 2-4 . Control measures are then implemented based on the cumulative information collected. These approaches rely heavily on good surveillance systems, access to experts, and good intuition about which control measures to use. As such, they are largely subjective, time consuming, and the infrastructure required is often not present in high disease burden areas.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Point-of-View Making vaccines "on demand" A potential solution for emerging pathogens and biodefense?</h3>
<p style="text-align: justify; text-justify: inter-word;">Emergency Medical Countermeasures Enterprise (PHEMCE) has made great strides in strategic preparedness and response capabilities. There have been numerous advances in planning, biothreat countermeasure development, licensure, manufacturing, stockpiling and deployment. Increased biodefense surveillance capability has dramatically improved, while new tools and increased awareness have fostered rapid identification of new potential public health pathogens. Unfortunately, structural delays in vaccine design, development, manufacture, clinical testing and licensure processes remain significant obstacles to an effective national biodefense rapid response capability. This is particularly true for the very real threat of "novel pathogens" such as the avian-origin influenzas H7N9 and H5N1, and new coronaviruses such as hCoV-EMC. Conventional approaches to vaccine development, production, clinical testing and licensure are incompatible with the prompt deployment needed for an effective public health response. An alternative approach, proposed here, is to apply computational vaccine design tools and rapid production technologies that now make it possible to engineer vaccines for novel emerging pathogen and WMD biowarfare agent countermeasures in record time. These new tools have the potential to significantly reduce the time needed to design string-of-epitope vaccines for previously unknown</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>viruses Rapid Viral Diagnosis of Orthopoxviruses by Electron Microscopy: Optional or a Must?</h3>
<p style="text-align: justify; text-justify: inter-word;">With concerns over the possible re-emergence of poxvirus infections, either through the spread of existing animal viruses to susceptible humans [1] [2] [3] [4] [5] , survival and escape of old material [6-8], molecular manipulation and synthetic biology [9] or even bioterrorism [10-12], there will be a need to provide accurate and quick diagnosis. This paper presents the possibilities and advantages of DEM and how DEM can help with organised future preparedness.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /></body>