
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: enhancing capacity for sequencing with advanced analytics for unknown pathogens</h2><br /></div><br /><br /><br /><div class="card"><h3>Pathosphere.org: pathogen detection and characterization through a web-based, open source informatics platform</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: The detection of pathogens in complex sample backgrounds has been revolutionized by wide access to next-generation sequencing (NGS) platforms. However, analytical methods to support NGS platforms are not as uniformly available. Pathosphere (found at Pathosphere.org) is a cloud -based open -sourced community tool that allows for communication, collaboration and sharing of NGS analytical tools and data amongst scientists working in academia, industry and government. The architecture allows for users to upload data and run available bioinformatics pipelines without the need for onsite processing hardware or technical support.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>Amplicon based MinION sequencing of SARS-CoV-2 and metagenomic characterisation of nasopharyngeal swabs from patients with COVID-19</h3>
<p style="text-align: justify; text-justify: inter-word;">COVID-19 is a complex disease phenotype where the underlying microbiome could influence morbidity and mortality. Amplicon and metagenomic MinION based sequencing was used to rapidly (within 8 hours) identify SARS-CoV-2 and assess the microbiome in nasopharyngeal swabs obtained from patients with COVID-19 by the ISARIC 4C consortium.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>A Universal Next-Generation Sequencing Protocol To Generate Noninfectious Barcoded cDNA Libraries from High- Containment RNA Viruses</h3>
<p style="text-align: justify; text-justify: inter-word;">Several biosafety level 3 and/or 4 (BSL-3/4) pathogens are highconsequence, single-stranded RNA viruses, and their genomes, when introduced into permissive cells, are infectious. Moreover, many of these viruses are select agents (SAs), and their genomes are also considered SAs. For this reason, cDNAs and/or their derivatives must be tested to ensure the absence of infectious virus and/or viral RNA before transfer out of the BSL-3/4 and/or SA laboratory. This tremendously limits the capacity to conduct viral genomic research, particularly the application of next-generation sequencing (NGS). Here, we present a sequence-independent method to rapidly amplify viral genomic RNA while simultaneously abolishing both viral and genomic RNA infectivity across multiple single-stranded positive-sense RNA (ssRNAÏ©) virus families. The process generates barcoded DNA amplicons that range in length from 300 to 1,000 bp, which cannot be used to rescue a virus and are stable to transport at room temperature. Our barcoding approach allows for up to 288 barcoded samples to be pooled into a single library and run across various NGS platforms without potential reconstitution of the viral genome. Our data demonstrate that this approach provides full-length genomic sequence information not only from high-titer virion preparations but it can also recover specific viral sequence from samples with limited starting material in the background of cellular RNA, and it can be used to identify pathogens from unknown samples. In summary, we describe a rapid, universal standard operating procedure that generates highquality NGS libraries free of infectious virus and infectious viral RNA.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia's first case of COVID-</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Citation: Targeted NGS Platforms for Genetic Screening and Gene Discovery in Primary Immunodeficiencies</h3>
<p style="text-align: justify; text-justify: inter-word;">Objective: To evaluate the strength and weakness of targeted NGS sequencing using custom-made Ion Torrent and Haloplex (Agilent) panels for diagnostics and research purposes.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">It is time to shake up public health surveillance. New technologies for sequencing, aided by friction-free approaches to data sharing, could have an impact on public health efforts.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>A porcine enterovirus G associated with enteric disease contains a novel papain-like cysteine protease</h3>
<p style="text-align: justify; text-justify: inter-word;">Identification of unknown pathogens in pigs displaying enteric illness is difficult due to the large diversity of bacterial and viral species found within faecal samples. Current methods often require bacterial or viral isolation, or testing only a limited number of known species using quantitative PCR analysis. Herein, faeces from two 25-day-old piglets with diarrhoea from Texas, USA, were analysed by metagenomic next-generation sequencing to rapidly identify possible pathogens. Our analysis included a bioinformatics pipeline of rapid short-read classification and de novo genome assembly which resulted in the identification of a porcine enterovirus G (EV-G), a complete genome with substantial nucleotide differences (>30 %) among current sequences, and a novel non-structural protein similar in sequence to the Torovirus papain-like cysteine protease (PL pro ). This discovery led to the identification and circulation of an EV-G with a novel PL pro in the USA that has not been previously reported.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Etiology of fever in Ugandan children: identification of microbial pathogens using 1 metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic 2 analysis 3 4</h3>
<p style="text-align: justify; text-justify: inter-word;">To address these challenges, we developed IDseq, a cloud-based open-source 90 bioinformatics platform and service for detection of microbial pathogens from metagenomic 91 next-generation sequencing (mNGS) data. IDseq requires minimal computational hardware and 92 is designed to enhance accessibility and build informatics capacity in resource limited regions. 93</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Challenges in the analysis of viral metagenomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Genome sequencing technologies continue to develop with remarkable pace, yet analytical approaches for reconstructing and classifying viral genomes from mixed samples remain limited in their performance and usability. Existing solutions generally target expert users and often have unclear scope, making it challenging to critically evaluate their performance. There is a growing need for intuitive analytical tooling for researchers lacking specialist computing expertise and that is applicable in diverse experimental circumstances. Notable technical challenges have impeded progress; for example, fragments of viral genomes are typically orders of magnitude less abundant than those of host, bacteria, and/or other organisms in clinical and environmental metagenomes; observed viral genomes often deviate considerably from reference genomes demanding use of exhaustive alignment approaches; high intrapopulation viral diversity can lead to ambiguous sequence reconstruction; and finally, the relatively few documented viral reference genomes compared to the estimated number of distinct viral taxa renders classification problematic. Various software tools have been developed to accommodate the unique challenges and use cases associated with characterizing viral sequences; however, the quality of these tools varies, and their use often necessitates computing expertise or access to powerful computers, thus limiting their usefulness to many researchers. In this review, we consider the general and application-specific challenges posed by viral sequencing and analysis, outline the landscape of available tools and methodologies, and propose ways of overcoming the current barriers to effective analysis.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>The Utility of Data Transformation for Alignment, De Novo Assembly and Classification of Short Read Virus Sequences</h3>
<p style="text-align: justify; text-justify: inter-word;">Advances in DNA sequencing technology are facilitating genomic analyses of unprecedented scope and scale, widening the gap between our abilities to generate and fully exploit biological sequence data. Comparable analytical challenges are encountered in other data-intensive fields involving sequential data, such as signal processing, in which dimensionality reduction (i.e., compression) methods are routinely used to lessen the computational burden of analyses. In this work, we explored the application of dimensionality reduction methods to numerically represent high-throughput sequence data for three important biological applications of virus sequence data: reference-based mapping, short sequence classification and de novo assembly. Leveraging highly compressed sequence transformations to accelerate sequence comparison, our approach yielded comparable accuracy to existing approaches, further demonstrating its suitability for sequences originating from diverse virus populations. We assessed the application of our methodology using both synthetic and real viral pathogen sequences. Our results show that the use of highly compressed sequence approximations can provide accurate results, with analytical performance retained and even enhanced through appropriate dimensionality reduction of sequence data.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Public health preparedness requires effective surveillance of and rapid response to infectious disease outbreaks. Inclusion of research activities within the outbreak setting provides important opportunities to maximize limited resources, to enhance gains in scientific knowledge, and ultimately to increase levels of preparedness. With rapid advances in laboratory technologies, banking and analysis of human genomic specimens can be conducted as part of public health investigations, enabling valuable research well into the future.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>VIP: an integrated pipeline for metagenomics of virus identification and discovery OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">A key feature of latest NGS platforms is their speed. It takes minimum turnaround times about 8 hours for sequencing 13 . Thus, it is critical that subsequent computational handling of the large amount of sequence</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification and characterization of unrecognized viruses in stool samples of non-polio acute flaccid paralysis children by simplified VIDISCA</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: The use of sequence independent methods combined with next generation sequencing for identification purposes in clinical samples appears promising and exciting results have been achieved to understand unexplained infections. One sequence independent method, Virus Discovery based on cDNA Amplified Fragment Length Polymorphism (VIDISCA) is capable of identifying viruses that would have remained unidentified in standard diagnostics or cell cultures. Methods: VIDISCA is normally combined with next generation sequencing, however, we set up a simplified VIDISCA which can be used in case next generation sequencing is not possible. Stool samples of 10 patients with unexplained acute flaccid paralysis showing cytopathic effect in rhabdomyosarcoma cells and/or mouse cells were used to test the efficiency of this method. To further characterize the viruses, VIDISCA-positive samples were amplified and sequenced with gene specific primers.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>A Universal Next-Generation Sequencing Protocol To Generate Noninfectious Barcoded cDNA Libraries from High- Containment RNA Viruses</h3>
<p style="text-align: justify; text-justify: inter-word;">This report establishes and validates a standard operating procedure (SOP) for select agents (SAs) and other biosafety level 3 and/or 4 (BSL-3/4) RNA viruses to rapidly generate noninfectious, barcoded cDNA amenable for nextgeneration sequencing (NGS). This eliminates the burden of testing all processed samples derived from high-consequence pathogens prior to transfer from highcontainment laboratories to lower-containment facilities for sequencing. Our established protocol can be scaled up for high-throughput sequencing of hundreds of samples simultaneously, which can dramatically reduce the cost and effort required for NGS library construction. NGS data from this SOP can provide complete genome coverage from viral stocks and can also detect virus-specific reads from limited start-TB, Shabman RS. 2016. A universal next-generation sequencing protocol to generate noninfectious barcoded cDNA libraries from high-containment RNA viruses. mSystems 1(3):e00039-15.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Etiology of fever in Ugandan children: identification of microbial pathogens using 1 metagenomic next-generation sequencing and IDseq, a platform for unbiased metagenomic 2 analysis 3 4</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Febrile illness is a major burden in African children, and non-malarial causes of 45 fever are uncertain. We built and employed IDseq, a cloud-based, open access, bioinformatics 46 platform and service to identify microbes from metagenomic next-generation sequencing of 47 tissue samples. In this pilot study, we evaluated blood, nasopharyngeal, and stool specimens 48 from 94 children (aged 2-54 months) with febrile illness admitted to Tororo District Hospital, 49</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Human virome in nasopharynx and tracheal secretion samples</h3>
<p style="text-align: justify; text-justify: inter-word;">BACKGROUND In Brazil the implementation of the Sentinel Surveillance System of Influenza began in 2000. Central public health laboratories use reverse transcription-quantitative polymerase chain reaction (RT-qPCR) for diagnosis of respiratory viruses, but this protocol identifies only specific targets, resulted in inconclusive diagnosis for many samples. Thus, highthroughput sequencing (HTS) would be complementary method in the identification of pathogens in inconclusive samples for RT-qPCR or other specific detection protocols.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Diagnostics of rare disorders: whole-exome sequencing deciphering locus heterogeneity in telomere biology disorders</h3>
<p style="text-align: justify; text-justify: inter-word;">Methods: Clinical sequencing was performed on a cohort of patients presenting with variable immune phenotypes lacking molecular diagnoses. Hypothesis-free whole-exome sequencing (WES) was selected in the absence of compelling diagnostic hints in patients with variable immunological and haematological conditions.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>deep sequencing in infectious diseases: immune and pathogen repertoires for the improvement of patient outcomes The inaugural workshop "Deep Sequencing in Infectious Diseases: Immune and Pathogen Repertoires for the Improvement of Patient Outcomes" was held in Singapore</h3>
<p style="text-align: justify; text-justify: inter-word;">The aim of the workshop was to discuss the latest trends in using high-throughput sequencing, bioinformatics, and allied technologies to analyze immune and pathogen repertoires and their interplay within the host, bringing together key international players in the field and Singapore-based researchers and clinicianscientists. The focus was in particular on the application of these technologies for the improvement of patient diagnosis, prognosis and treatment, and for other broad public health outcomes. The presentations by scientists and clinicians showed the potential of deep sequencing technology to capture the coevolution of adaptive immunity and pathogens. For clinical applications, some key challenges remain, such as the long turnaround time and relatively high cost of deep sequencing for pathogen identification and characterization and the lack of international standardization in immune repertoire analysis.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Field pathogenomics reveals the emergence of a diverse wheat yellow rust population</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Emerging and re-emerging pathogens imperil public health and global food security. Responding to these threats requires improved surveillance and diagnostic systems. Despite their potential, genomic tools have not been readily applied to emerging or re-emerging plant pathogens such as the wheat yellow (stripe) rust pathogen Puccinia striiformis f. sp. tritici (PST). This is due largely to the obligate parasitic nature of PST, as culturing PST isolates for DNA extraction remains slow and tedious.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>SIANN: Strain Identification by Alignment to Near Neighbors</h3>
<p style="text-align: justify; text-justify: inter-word;">Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species-and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /></body>