
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: seasonality of transmission</h2><br /></div><br /><br /><br /><div class="card"><h3>A spatial model of CoVID-19 transmission in England and Wales: early spread and peak timing</h3>
<p style="text-align: justify; text-justify: inter-word;">We provide initial estimates of the potential course of CoVID-19 in England and Wales in the absence of control measures. These results can be refined with improved estimates of epidemiological parameters, and permit investigation of control measures and cost effectiveness analyses. Seasonal changes in transmission rate could shift the timing of the peak into winter months, which will have important implications for health-care capacity planning.</p>
<p align="right"><i>score: 215</i></p>

</div><br /><br /><br /><div class="card"><h3>Potential impact of seasonal forcing on a SARS-CoV-2 pandemic</h3>
<p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures, and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.</p>
<p align="right"><i>score: 211</i></p>

</div><br /><br /><br /><div class="card"><h3>Global Influenza Programme, World Health Organization</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions The transmission of A(H1N1)pdm09 exhibited similarities with seasonal influenza transmission in that activity varied between temperate and tropical-subtropical countries and by time of year. Our findings suggest the potential utility of accounting for these factors during future pandemic planning. Keywords 2009 influenza A (H1N1), pandemic influenza, seasonality. Please cite this paper as: Storms et al. (2013) Worldwide transmission and seasonal variation of pandemic influenza A(H1N1)2009 virus activity during the</p>
<p align="right"><i>score: 204</i></p>

</div><br /><br /><br /><div class="card"><h3>Smallpox and Season: Reanalysis of Historical Data</h3>
<p style="text-align: justify; text-justify: inter-word;">Seasonal variation in smallpox transmission is one of the most pressing ecological questions and is relevant to bioterrorism preparedness. The present study reanalyzed 7 historical datasets which recorded monthly cases or deaths. In addition to time series analyses of reported data, an estimation and spectral analysis of the effective reproduction number at calendar time t, R(t), were made. Meteorological variables were extracted from a report in India from 1890-1921 and compared with smallpox mortality as well as R(t). Annual cycles of smallpox transmission were clearly shown not only in monthly reports but also in the estimates of R(t). Even short-term epidemic data clearly exhibited an annual peak every January. Both mortality and R(t) revealed significant negative association (P < .01) and correlation (P < .01), respectively, with humidity. These findings suggest that smallpox transmission greatly varies with season and is most likely enhanced by dry weather.</p>
<p align="right"><i>score: 201</i></p>

</div><br /><br /><br /><div class="card"><h3>Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period</h3>
<p style="text-align: justify; text-justify: inter-word;">There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.</p>
<p align="right"><i>score: 177</i></p>

</div><br /><br /><br /><div class="card"><h3>Pathogen seasonality and links with weather in England and Wales: a big data time series analysis</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Many infectious diseases of public health importance display annual seasonal patterns in their incidence. We aimed to systematically document the seasonality of several human infectious disease pathogens in England and Wales, highlighting those organisms that appear weather-sensitive and therefore may be influenced by climate change in the future.</p>
<p align="right"><i>score: 173</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying the seasonal drivers of transmission for Lassa fever in Nigeria</h3>
<p style="text-align: justify; text-justify: inter-word;">Subject Areas: computational biology, ecology, environmental science, health and disease and epidemiology Lassa fever (LF) is a zoonotic disease that is widespread in West Africa and involves animal-to-human and human-to-human transmission. Animal-tohuman transmission occurs upon exposure to rodent excreta and secretions, i.e. urine and saliva, and human-to-human transmission occurs via the bodily fluids of an infected person. To elucidate the seasonal drivers of LF epidemics, we employed a mathematical model to analyse the datasets of human infection, rodent population dynamics and climatological variations and capture the underlying transmission dynamics. The surveillance-based incidence data of human cases in Nigeria were explored, and moreover, a mathematical model was used for describing the transmission dynamics of LF in rodent populations. While quantifying the case fatality risk and the rate of exposure of humans to animals, we explicitly estimated the corresponding contact rate of humans with infected rodents, accounting for the seasonal population dynamics of rodents. Our findings reveal that seasonal migratory dynamics of rodents play a key role in regulating the cyclical pattern of LF epidemics. The estimated timing of high exposure of humans to animals coincides with the time shortly after the start of the dry season and can be associated with the breeding season of rodents in Nigeria.</p>
<p align="right"><i>score: 166</i></p>

</div><br /><br /><br /><div class="card"><h3>A spatial model of CoVID-19 transmission in England and Wales: early spread and peak timing</h3>
<p style="text-align: justify; text-justify: inter-word;">We predict that a CoVID-19 outbreak will peak 126 to 147 days (~4 months) after the start of person-to-person transmission in England and Wales in the absence of controls, assuming biological parameters remain unchanged. Therefore, if person-to-person transmission persists from February, we predict the epidemic peak would occur in June. The starting location has minimal impact on peak timing, and model stochasticity varies peak timing by 10 days. Incorporating realistic parameter uncertainty leads to estimates of peak time ranging from 78 days to 241 days after person-to-person transmission has been established. Seasonal changes in transmission rate substantially impact the timing and size of the epidemic peak, as well as the total attack rate.</p>
<p align="right"><i>score: 165</i></p>

</div><br /><br /><br /><div class="card"><h3>we enrolled children with pneumonia who were admitted into wards of the De- partments of Respiratory Medicine, Infectious Diseases</h3>
<p style="text-align: justify; text-justify: inter-word;">we enrolled 4,225 hospitalized children with pneumonia in a study to determine the seasonality of respiratory syncytial virus (RSV) infection in Beijing, China. We defined season as the period during which >10% of total PCRs performed each week were RSV positive. We identified 8 distinctive RSV seasons. On average, the season onset occurred at week 41 (mid-October) and lasted 33 weeks, through week 20 of the next year (mid-May); 97% of all RSV-positive cases occurred during the season. RSV seasons occurred 3-5 weeks earlier and lasted ≈6 weeks longer in RSV subgroup A-dominant years than in RSV subgroup B-dominant years. Our analysis indicates that monitoring such RSV subgroup shifts might provide better estimates for the onset of RSV transmission. PCR-based tests could be a flexible or complementary way of determining RSV seasonality in locations where RSV surveillance is less well-established, such as local hospitals throughout China.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>Seasonal and interannual risks of dengue introduction from South-East Asia into China, 2005-2015</h3>
<p style="text-align: justify; text-justify: inter-word;">Due to worldwide increased human mobility, air-transportation data and mathematical models have been widely used to measure risks of global dispersal of pathogens. However, the seasonal and interannual risks of pathogens importation and onward transmission from endemic countries have rarely been quantified and validated. We constructed a modelling framework, integrating air travel, epidemiological, demographical, entomological and meteorological data, to measure the seasonal probability of dengue introduction from endemic countries. This framework has been applied retrospectively to elucidate spatiotemporal patterns and increasing seasonal risk of dengue importation from South-East Asia into China via air travel in multiple populations, Chinese travelers and local residents, over a decade of 2005-15. We found that the volume of airline travelers from South-East Asia into China has quadrupled from 2005 to 2015 with Chinese travelers increased rapidly. Following the growth of air traffic, the probability of dengue importation from South-East Asia into China has increased dramatically from 2005 to 2015. This study also revealed seasonal asymmetries of PLOS Neglected Tropical Diseases | https://doi.</p>
<p align="right"><i>score: 157</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemic in Southern Spain (Andalusia) in 2007 Using Epidemic Trees</h3>
<p style="text-align: justify; text-justify: inter-word;">Andalusia (Southern Spain) is considered one of the main routes of introduction of bluetongue virus (BTV) into Europe, evidenced by a devastating epidemic caused by BTV-1 in 2007. Understanding the pattern and the drivers of BTV-1 spread in Andalusia is critical for effective detection and control of future epidemics. A long-standing metric for quantifying the behaviour of infectious diseases is the case-reproduction ratio (R t ), defined as the average number of secondary cases arising from a single infected case at time t (for t>0). Here we apply a method using epidemic trees to estimate the between-herd case reproduction ratio directly from epidemic data allowing the spatial and temporal variability in transmission to be described. We then relate this variability to predictors describing the hosts, vectors and the environment to better understand why the epidemic spread more quickly in some regions or periods. The R t value for the BTV-1 epidemic in Andalusia peaked in July at 4.6, at the start of the epidemic, then decreased to 2.2 by August, dropped below 1 by September (0.8), and by October it had decreased to 0.02. BTV spread was the consequence of both local transmission within established disease foci and BTV expansion to distant new areas (i.e. new foci), which resulted in a high variability in BTV transmission, not only among different areas, but particularly through time, which suggests that general control measures applied at broad spatial scales are unlikely to be effective. This high variability through time was probably due to the impact of temperature on BTV transmission, as evidenced by a reduction in the value of R t by 0.0041 for every unit increase (day) in the extrinsic incubation period (EIP), which is itself directly dependent on temperature. Moreover, within the range of values at which BTV-1 transmission occurred in Andalusia (20.6°C to 29.5°C) there was a positive correlation between temperature and R t values, although the PLOS ONE | relationship was not linear, probably as a result of the complex relationship between temperature and the different parameters affecting BTV transmission. R t values for BTV-1 in Andalusia fell below the threshold of 1 when temperatures dropped below 21°C, a much higher threshold than that reported in other BTV outbreaks, such as the BTV-8 epidemic in Northern Europe. This divergence may be explained by differences in the adaptation to temperature of the main vectors of the BTV-1 epidemic in Andalusia (Culicoides imicola) compared those of the BTV-8 epidemic in Northern Europe (Culicoides obsoletus). Importantly, we found that BTV transmission (R t value) increased significantly in areas with higher densities of sheep. Our analysis also established that control of BTV-1 in Andalusia was complicated by the simultaneous establishment of several distant foci at the start of the epidemic, which may have been caused by several independent introductions of infected vectors from the North of Africa. We discuss the implications of these findings for BTV surveillance and control in this region of Europe.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission patterns and evolution of respiratory syncytial virus in a community outbreak identified by genomic analysis</h3>
<p style="text-align: justify; text-justify: inter-word;">Detailed information on the source, spread and evolution of respiratory syncytial virus (RSV) during seasonal community outbreaks remains sparse. Molecular analyses of attachment (G) gene sequences from hospitalized cases suggest that multiple genotypes and variants co-circulate during epidemics and that RSV persistence over successive seasons is characterized by replacement and multiple new introductions of variants. No studies have defined the patterns of introduction, spread and evolution of RSV at the local community and household level. We present a whole genome sequence analysis of 131 RSV group A viruses collected during 6-month household-based RSV infection surveillance in Coastal Kenya, 2010 within an area of 12 km 2 . RSV infections were identified by regular symptom-independent screening of all household members twice weekly. Phylogenetic analysis revealed that the RSV A viruses in nine households were closely related to genotype GA2 and fell within a single branch of the global phylogeny. Genomic analysis allowed the detection of householdspecific variation in seven households. For comparison, using only G gene analysis, household-specific variation was found only in one of the nine households. Nucleotide changes were observed both intra-host (viruses identified from same individual in follow-up sampling) and inter-host (viruses identified from different household members) and these coupled with sampling dates enabled a partial reconstruction of the within household transmission chains. The genomic evolutionary rate for the household dataset was estimated as 2.307 Â 10 À 3 (95% highest posterior density: 0.935-4.165Â 10 À 3 ) substitutions/site/year. We conclude that (i) at the household level, most RSV infections arise from the introduction of a single virus variant followed by accumulation of household specific variation and (ii) analysis of complete virus genomes is crucial to better understand viral transmission in the community. A key question arising is whether prevention of RSV introduction or spread within the household by vaccinating key transmitting household members would lead to a reduced onward community-wide transmission.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Infectious Diseases Clinical and epidemiological predictors of transmission in Severe Acute Respiratory Syndrome (SARS)</h3>
<p style="text-align: justify; text-justify: inter-word;">Clinical and epidemiological factors can help us to explain why transmission was observed in some instances but not in others.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020</h3>
<p style="text-align: justify; text-justify: inter-word;">The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.</p>
<p align="right"><i>score: 153</i></p>

</div><br /><br /><br /><div class="card"><h3>Identifying Meteorological Drivers for the Seasonal Variations of Influenza Infections in a Subtropical City -Hong Kong</h3>
<p style="text-align: justify; text-justify: inter-word;">Compared with temperate areas, the understanding of seasonal variations of influenza infections is lacking in subtropical and tropical regions. Insufficient information about viral activity increases the difficulty of forecasting the disease burden and thus hampers official preparation efforts. Here we identified potential meteorological factors that drove the seasonal variations in influenza infections in a subtropical city, Hong Kong. We fitted the meteorological data and influenza mortality data from 2002 to 2009 in a Susceptible-Infected-Recovered model. From the results, air temperature was a common significant driver of seasonal patterns and cold temperature was associated with an increase in transmission intensity for most of the influenza epidemics. Except 2004, the fitted models with significant meteorological factors could account for more than 10% of the variance in additional to the null model. Rainfall was also found to be a significant driver of seasonal influenza, although results were less robust. The identified meteorological indicators could alert officials to take appropriate control measures for influenza epidemics, such as enhancing vaccination activities before cold seasons. Further studies are required to fully justify the associations.</p>
<p align="right"><i>score: 151</i></p>

</div><br /><br /><br /><div class="card"><h3>Determining the dynamics of influenza transmission by age</h3>
<p style="text-align: justify; text-justify: inter-word;">We show substantial heterogeneity in transmission by age group during the Influenza A H1N1 pandemic in South Africa. This information can greatly assist in targeting interventions and implementing social distancing measures.</p>
<p align="right"><i>score: 150</i></p>

</div><br /><br /><br /><div class="card"><h3>Ecological niche modeling to determine potential niche of Vaccinia virus: a case only study</h3>
<p style="text-align: justify; text-justify: inter-word;">The final ENM predicted all areas where Brazilian outbreaks occurred, one out of five of the Colombian outbreak regions and identified new regions within Brazil that are suitable for transmission based on bioclimatic factors. Further, the most important factors in determining transmission suitability are precipitation of the wettest quarter, annual precipitation, mean temperature of the coldest quarter and mean diurnal range.</p>
<p align="right"><i>score: 149</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Potential of Chikungunya Virus and Control Measures: The Case of Italy</h3>
<p style="text-align: justify; text-justify: inter-word;">During summer 2007 Italy has experienced an epidemic caused by Chikungunya virus -the first large outbreak documented in a temperate climate country -with approximately 161 laboratory confirmed cases concentrated in two bordering villages in North-Eastern Italy comprising 3,968 inhabitants. The seroprevalence was recently estimated to be 10.2%. In this work we provide estimates of the transmission potential of the virus and we assess the efficacy of the measures undertaken by public health authorities to control the epidemic spread. To such aim, we developed a model describing the temporal dynamics of the competent vector, known as Aedes albopictus, explicitly depending on climatic factors, coupled to an epidemic transmission model describing the spread of the epidemic in both humans and mosquitoes. The cumulative number of notified cases predicted by the model was 185 on average (95% CI 117-278), in good agreement with observed data. The probability of observing a major outbreak after the introduction of an infective human case was estimated to be in the range of 32%-76%. We found that the basic reproduction number was in the range of 1.8-6 but it could have been even larger, depending on the density of mosquitoes, which in turn depends on seasonal meteorological effects, besides other local abiotic factors. These results confirm the increasing risk of tropical vector-borne diseases in temperate climate countries, as a consequence of globalization. However, our results show that an epidemic can be controlled by performing a timely intervention, even if the transmission potential of Chikungunya virus is sensibly high.</p>
<p align="right"><i>score: 148</i></p>

</div><br /><br /><br /><div class="card"><h3>Pandemic (H1N1) 2009 Influenza Community Transmission Was Established in One Australian State When the Virus Was First Identified in North America</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: Epidemic transmission occurred earlier in Victoria and later in WA. Only 5% of the first 100 Victorian cases were not locally acquired and three of these were brothers in one family. By contrast, 53% of the first 102 cases in WA were associated with importation from Victoria. Using plausible model input data, estimation of the effective reproductive number for the Victorian epidemic required us to invoke an earlier date for commencement of transmission to explain the observed data. This was not required in modelling the epidemic in WA.</p>
<p align="right"><i>score: 146</i></p>

</div><br /><br /><br /><div class="card"><h3>Network Analysis of Global Influenza Spread</h3>
<p style="text-align: justify; text-justify: inter-word;">Although vaccines pose the best means of preventing influenza infection, strain selection and optimal implementation remain difficult due to antigenic drift and a lack of understanding global spread. Detecting viral movement by sequence analysis is complicated by skewed geographic and seasonal distributions in viral isolates. We propose a probabilistic method that accounts for sampling bias through spatiotemporal clustering and modeling regional and seasonal transmission as a binomial process. Analysis of H3N2 not only confirmed East-Southeast Asia as a source of new seasonal variants, but also increased the resolution of observed transmission to a country level. H1N1 data revealed similar viral spread from the tropics. Network analysis suggested China and Hong Kong as the origins of new seasonal H3N2 strains and the United States as a region where increased vaccination would maximally disrupt global spread of the virus. These techniques provide a promising methodology for the analysis of any seasonal virus, as well as for the continued surveillance of influenza.</p>
<p align="right"><i>score: 145</i></p>

</div><br /><br /><br /></body>