
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: charge distribution of the coronavirus</h2><br /></div><br /><br /><br /><div class="card"><h3>Amino acid residues critical for RNA-binding in the N-terminal domain of the nucleocapsid protein are essential determinants for the infectivity of coronavirus in cultured cells</h3>
<p style="text-align: justify; text-justify: inter-word;">The N-terminal domain of the coronavirus nucleocapsid (N) protein adopts a fold resembling a right hand with a flexible, positively charged b-hairpin and a hydrophobic palm. This domain was shown to interact with the genomic RNA for coronavirus infectious bronchitis virus (IBV) and severe acute respiratory syndrome coronavirus (SARS-CoV). Based on its 3D structure, we used site-directed mutagenesis to identify residues essential for the RNA-binding activity of the IBV N protein and viral infectivity. Alanine substitution of either Arg-76 or Tyr-94 in the N-terminal domain of IBV N protein led to a significant decrease in its RNA-binding activity and a total loss of the infectivity of the viral RNA to Vero cells. In contrast, mutation of amino acid Gln-74 to an alanine, which does not affect the binding activity of the N-terminal domain, showed minimal, if any, detrimental effect on the infectivity of IBV. This study thus identifies residues critical for RNA binding on the nucleocapsid surface, and presents biochemical and genetic evidence that directly links the RNA binding capacity of the coronavirus N protein to the viral infectivity in cultured cells. This information would be useful in development of preventive and treatment approaches against coronavirus infection.</p>
<p align="right"><i>score: 154</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus E Protein: Assembly and Beyond</h3>
<p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein that has an important role in the assembly of virions. Recent studies have indicated that the E protein has functions during infection beyond assembly, including in virus egress and in the host stress response. Additionally, the E protein has ion channel activity, interacts with host proteins, and may have multiple membrane topologies. The goal of this review is to highlight the properties and functions of the E protein, and speculate on how they may be related.</p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Structure and Inhibition of the SARS Coronavirus Envelope Protein Ion Channel</h3>
<p style="text-align: justify; text-justify: inter-word;">The envelope (E) protein from coronaviruses is a small polypeptide that contains at least one a-helical transmembrane domain. Absence, or inactivation, of E protein results in attenuated viruses, due to alterations in either virion morphology or tropism. Apart from its morphogenetic properties, protein E has been reported to have membrane permeabilizing activity. Further, the drug hexamethylene amiloride (HMA), but not amiloride, inhibited in vitro ion channel activity of some synthetic coronavirus E proteins, and also viral replication. We have previously shown for the coronavirus species responsible for severe acute respiratory syndrome (SARS-CoV) that the transmembrane domain of E protein (ETM) forms pentameric a-helical bundles that are likely responsible for the observed channel activity. Herein, using solution NMR in dodecylphosphatidylcholine micelles and energy minimization, we have obtained a model of this channel which features regular a-helices that form a pentameric left-handed parallel bundle. The drug HMA was found to bind inside the lumen of the channel, at both the C-terminal and the N-terminal openings, and, in contrast to amiloride, induced additional chemical shifts in ETM. Full length SARS-CoV E displayed channel activity when transiently expressed in human embryonic kidney 293 (HEK-293) cells in a whole-cell patch clamp set-up. This activity was significantly reduced by hexamethylene amiloride (HMA), but not by amiloride. The channel structure presented herein provides a possible rationale for inhibition, and a platform for future structure-based drug design of this potential pharmacological target.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Transient Oligomerization of the SARS-CoV N Protein -Implication for Virus Ribonucleoprotein Packaging</h3>
<p style="text-align: justify; text-justify: inter-word;">The nucleocapsid (N) phosphoprotein of the severe acute respiratory syndrome coronavirus (SARS-CoV) packages the viral genome into a helical ribonucleocapsid and plays a fundamental role during viral self-assembly. The N protein consists of two structural domains interspersed between intrinsically disordered regions and dimerizes through the C-terminal structural domain (CTD). A key activity of the protein is the ability to oligomerize during capsid formation by utilizing the dimer as a building block, but the structural and mechanistic bases of this activity are not well understood. By disulfide trapping technique we measured the amount of transient oligomers of N protein mutants with strategically located cysteine residues and showed that CTD acts as a primary transient oligomerization domain in solution. The data is consistent with the helical oligomer packing model of N protein observed in crystal. A systematic study of the oligomerization behavior revealed that altering the intermolecular electrostatic repulsion through changes in solution salt concentration or phosphorylation-mimicking mutations affects oligomerization propensity. We propose a biophysical mechanism where electrostatic repulsion acts as a switch to regulate N protein oligomerization.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>The Coronavirus Nucleocapsid Is a Multifunctional Protein</h3>
<p style="text-align: justify; text-justify: inter-word;">Abstract: The coronavirus nucleocapsid (N) is a structural protein that forms complexes with genomic RNA, interacts with the viral membrane protein during virion assembly and plays a critical role in enhancing the efficiency of virus transcription and assembly. Recent studies have confirmed that N is a multifunctional protein. The aim of this review is to highlight the properties and functions of the N protein, with specific reference to (i) the topology; (ii) the intracellular localization and (iii) the functions of the protein.</p>
<p align="right"><i>score: 131</i></p>

</div><br /><br /><br /><div class="card"><h3>Understanding Viral Transmission Behavior via Protein Intrinsic Disorder Prediction: Coronaviruses</h3>
<p style="text-align: justify; text-justify: inter-word;">Besides being a common threat to farm animals and poultry, coronavirus (CoV) was responsible for the human severe acute respiratory syndrome (SARS) epidemic in 2002-4. However, many aspects of CoV behavior, including modes of its transmission, are yet to be fully understood. We show that the amount and the peculiarities of distribution of the protein intrinsic disorder in the viral shell can be used for the efficient analysis of the behavior and transmission modes of CoV. The proposed model allows categorization of the various CoVs by the peculiarities of disorder distribution in their membrane (M) and nucleocapsid (N). This categorization enables quick identification of viruses with similar behaviors in transmission, regardless of genetic proximity. Based on this analysis, an empirical model for predicting the viral transmission behavior is developed. This model is able to explain some behavioral aspects of important coronaviruses that previously were not fully understood. The new predictor can be a useful tool for better epidemiological, clinical, and structural understanding of behavior of both newly emerging viruses and viruses that have been known for a long time. A potentially new vaccine strategy could involve searches for viral strains that are characterized by the evolutionary misfit between the peculiarities of the disorder distribution in their shells and their behavior.</p>
<p align="right"><i>score: 129</i></p>

</div><br /><br /><br /><div class="card"><h3>Mesodynamics in the SARS nucleocapsid measured by NMR field cycling</h3>
<p style="text-align: justify; text-justify: inter-word;">Protein motions on all timescales faster than molecular tumbling are encoded in the spectral density. The dissection of complex protein dynamics is typically performed using relaxation rates determined at high and ultrahigh field. Here we expand this range of the spectral density to low fields through field cycling using the nucleocapsid protein of the SARS coronavirus as a model system. The field-cycling approach enables site-specific measurements of R 1 at low fields with the sensitivity and resolution of a high-field magnet. These data, together with high-field relaxation and heteronuclear NOE, provide evidence for correlated rigid-body motions of the entire b-hairpin, and corresponding motions of adjacent loops with a time constant of 0.8 ns (mesodynamics). MD simulations substantiate these findings and provide direct verification of the time scale and collective nature of these motions.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</h3>
<p style="text-align: justify; text-justify: inter-word;">the coronavirus, responsible for host cell attachment and mediating host cell membrane and viral membrane fusion during infection, is key to the viral life cycle and a major target for antiviral drugs and vaccines. In this study, we report the structures of different conformational states of the SARS-CoV S glycoprotein during virus entry. Specifically, we found that the S glycoprotein retains the prefusion trimer structure after trypsin cleavage and low-pH treatment. Additionally, binding with host cell receptor ACE2 promotes the release of S1 subunits from the S trimer and triggers the pre-to postfusion conformational transition. Our results provide new insights for understanding the mechanisms involved in coronavirus S glycoprotein-mediated virus entry.</p>
<p align="right"><i>score: 127</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain 1 reveals potential unique drug targeting sites</h3>
<p style="text-align: justify; text-justify: inter-word;">CoV-2 nucleocapsid protein is yet to be clear. Herein, we have determined the 2.7 Å crystal 28 structure of the N-terminal RNA binding domain of SARS-CoV-2 nucleocapsid protein. 29 Although overall structure is similar with other reported coronavirus nucleocapsid protein N-30 terminal domain, the surface electrostatic potential characteristics between them are distinct. 31 Further comparison with mild virus type HCoV-OC43 equivalent domain demonstrates a unique 32 potential RNA binding pocket alongside the b-sheet core. Complemented by in vitro binding 33 studies, our data provide several atomic resolution features of SARS-CoV-2 nucleocapsid 34 protein N-terminal domain, guiding the design of novel antiviral agents specific targeting to 35 SARS-CoV-2. 36</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Crystal structure of the post-fusion core of the Human coronavirus 229E spike protein at 1.86 Å resolution</h3>
<p style="text-align: justify; text-justify: inter-word;">Human coronavirus 229E (HCoV-229E) usually causes mild upper respiratory infections in heathy adults, but may lead to severe complications or mortality in individuals with weakened immune systems. Virus entry of HCoV-229E is mediated by its spike (S) protein, where the S1 domain facilitates attachment to host cells and the S2 domain is involved in subsequent fusion of the virus and host membranes. During the fusion process, two heptad repeats, HR1 and HR2, in the S2 domain assemble into a six-helix membrane-fusion structure termed the fusion core. Here, the complete fusion-core structure of HCoV-229E has been determined at 1.86 Å resolution, representing the most complete postfusion conformation thus far among published human alphacoronavirus (-HCoV) fusion-core structures. The overall structure of the HCoV-229E fusion core is similar to those of SARS, MERS and HCoV-NL63, but the packing of its 3HR1 core differs from those of SARS and MERS in that it contains more noncanonical 'x' and 'da' layers. Side-by-side electrostatic surface comparisons reveal that the electrostatic surface potentials are opposite in -HCoVs and -HCoVs at certain positions and that the HCoV-229E surface also appears to be the most hydrophobic among the various HCoVs. In addition to the highly conserved hydrophobic interactions between HR1 and HR2, some polar and electrostatic interactions are also well preserved across different HCoVs. This study adds to the structural profiling of HCoVs to aid in the structure-based design of pan-coronavirus small molecules or peptides to inhibit viral fusion.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Vulnerabilities in coronavirus glycan shields despite extensive glycosylation</h3>
<p style="text-align: justify; text-justify: inter-word;">Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.</p>
<p align="right"><i>score: 118</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus nucleocapsid proteins assemble constitutively in high molecular oligomers OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoV) are enveloped viruses and rely on their nucleocapsid N protein to incorporate the positive-stranded genomic RNA into the virions. CoV N proteins form oligomers but the mechanism and relevance underlying their multimerization remain to be fully understood. Using in vitro pulldown experiments and density glycerol gradients, we found that at least 3 regions distributed over its entire length mediate the self-interaction of mouse hepatitis virus (MHV) and severe acute respiratory syndrome coronavirus (SARS-CoV) N protein. The fact that these regions can bind reciprocally between themselves provides a possible molecular basis for N protein oligomerization. Interestingly, cytoplasmic N molecules of MHV-infected cells constitutively assemble into oligomers through a process that does not require binding to genomic RNA. Based on our data, we propose a model where constitutive N protein oligomerization allows the optimal loading of the genomic viral RNA into a ribonucleoprotein complex via the presentation of multiple viral RNA binding motifs.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</h3>
<p style="text-align: justify; text-justify: inter-word;">CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins</h3>
<p style="text-align: justify; text-justify: inter-word;">As cell-invading molecular machinery, coronavirus spike proteins pose an evolutionary conundrum due to their high divergence. In this study, we determined the cryo-EM structure of avian infectious bronchitis coronavirus (IBV) spike protein from the γ-genus. The trimeric IBV spike ectodomain contains three receptor-binding S1 heads and a trimeric membranefusion S2 stalk. While IBV S2 is structurally similar to those from the other genera, IBV S1 possesses structural features that are unique to different other genera, thereby bridging these diverse spikes into an evolutionary spectrum. Specifically, among different genera, the two domains of S1, the N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), diverge from simpler tertiary structures and quaternary packing to more complex ones, leading to different functions of the spikes in receptor usage and membrane fusion. Based on the above structural and functional comparisons, we propose that the evolutionary spectrum of coronavirus spikes follows the order of α-, δ-, γ-, and β-genus. This study has provided insight into the evolutionary relationships among coronavirus spikes and deepened our understanding of their structural and functional diversity.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>A Single Polar Residue and Distinct Membrane Topologies Impact the Function of the Infectious Bronchitis Coronavirus E Protein</h3>
<p style="text-align: justify; text-justify: inter-word;">The coronavirus E protein is a small membrane protein with a single predicted hydrophobic domain (HD), and has a poorly defined role in infection. The E protein is thought to promote virion assembly, which occurs in the Golgi region of infected cells. It has also been implicated in the release of infectious particles after budding. The E protein has ion channel activity in vitro, although a role for channel activity in infection has not been established. Furthermore, the membrane topology of the E protein is of considerable debate, and the protein may adopt more than one topology during infection. We previously showed that the HD of the infectious bronchitis virus (IBV) E protein is required for the efficient release of infectious virus, an activity that correlated with disruption of the secretory pathway. Here we report that a single residue within the hydrophobic domain, Thr16, is required for secretory pathway disruption. Substitutions of other residues for Thr16 were not tolerated. Mutations of Thr16 did not impact virus assembly as judged by virus-like particle production, suggesting that alteration of secretory pathway and assembly are independent activities. We also examined how the membrane topology of IBV E affected its function by generating mutant versions that adopted either a transmembrane or membrane hairpin topology. We found that a transmembrane topology was required for disrupting the secretory pathway, but was less efficient for virus-like particle production. The hairpin version of E was unable to disrupt the secretory pathway or produce particles. The findings reported here identify properties of the E protein that are important for its function, and provide insight into how the E protein may perform multiple roles during infection.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>The human coronavirus HCoV-229E S-protein structure and receptor binding</h3>
<p style="text-align: justify; text-justify: inter-word;">The coronavirus S-protein mediates receptor binding and fusion of the viral and host cell membranes. In HCoV-229E, its receptor binding domain (RBD) shows extensive sequence variation but how S-protein function is maintained is not understood. Reported are the X-ray crystal structures of Class III-V RBDs in complex with human aminopeptidase N (hAPN), as well as the electron cryomicroscopy structure of the 229E S-protein. The structures show that common core interactions define the specificity for hAPN and that the peripheral RBD sequence variation is accommodated by loop plasticity. The results provide insight into immune evasion and the crossspecies transmission of 229E and related coronaviruses. We also find that the 229E S-protein can expose a portion of its helical core to solvent. This is undoubtedly facilitated by hydrophilic subunit interfaces that we show are conserved among coronaviruses. These interfaces likely play a role in the S-protein conformational changes associated with membrane fusion.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Ca 2+ ions promote fusion of Middle East Respiratory Syndrome coronavirus with host cells and increase infectivity</h3>
<p style="text-align: justify; text-justify: inter-word;">Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca 2+ ) plays an important role for fusogenic activity via a Ca 2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca 2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudoparticles (PPs) on host cells without and with Ca 2+ . Our results demonstrate that intracellular Ca 2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca 2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca 2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca 2+ , as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca 2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca 2+ ions interacting with the FP has implications on the fusion dynamics of the virus.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins</h3>
<p style="text-align: justify; text-justify: inter-word;">Funding: This work was supported by National Institute of Allergy and Infectious Diseases grants R01AI089728 and R01AI110700 (to FL) and University of Minnesota funding (to WZ). The funders had no role in study design, data collection to different other genera; hence it brings the spikes from different coronavirus genera into one evolutionary spectrum, with itself sitting in the middle of this spectrum. The order of the evolutionary spectrum is α-, δ-, γ-, and β-genus. Importantly, through comparisons between IBV spike and other coronavirus spikes, this study illustrates how coronavirus spikes have achieved structural and functional diversity through evolution to guide viral entry into host cells.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins</h3>
<p style="text-align: justify; text-justify: inter-word;">Because of their structural and functional diversity, coronavirus spike proteins represent a model system for studying viral evolution. Their evolutionary relationships and history pose major puzzles in virology. A critical missing piece in understanding the evolution of coronavirus spikes is the unavailability of the tertiary structure of γ-coronavirus spikes.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</h3>
<p style="text-align: justify; text-justify: inter-word;">The spike (S) glycoprotein of the coronavirus is a class I viral fusion protein located on the outer envelope of the virion that plays a critical role in viral infection by recognizing host cell receptors and mediating fusion of the viral and cellular membranes [4] . The coronavirus S glycoprotein is synthesized as a precursor protein consisting of~1,300 amino acids that is then cleaved into an amino (N)-terminal S1 subunit (~700 amino acids) and a carboxyl (C)-terminal S2 subunit (~600 amino acids). Three S1/S2 heterodimers assemble to form a trimer spike protruding from the viral envelope. The S1 subunit contains a receptor-binding domain (RBD), while the S2 subunit contains a hydrophobic fusion peptide and two heptad repeat regions. Triggered by receptor binding, proteolytic processing and/or acidic pH in the cellular compartments, the class I viral fusion protein undergoes a transition from a metastable prefusion state to a stable postfusion state during infection, in which the receptor-binding subunit is cleaved, and the fusion subunit undergoes large-scale conformational rearrangements to expose the hydrophobic fusion peptide, induce the formation of a six-helix bundle, and bring the viral and cellular membranes close for fusion [5] . Coronavirus S glycoprotein features two distinct protease cleavage sites. The S1/S2 cleavage site of the SARS-CoV S glycoprotein is located after residue 667 of the precursor protein, whereas the S2' cleavage site of the SARS-CoV S glycoprotein is on the S2 subunit and is 130 amino acids from the N terminus of the S2 subunit [6] [7] [8] . The S1/S2 cleavage site is located in a flexible loop of residues 660-675 that is completely exposed in the prefusion S1-S2 trimer spike. The S2' cleavage site of the SARS-CoV S glycoprotein is highly conserved among coronaviruses and is completely buried in the prefusion SARS-CoV S glycoprotein [6] [7] [8] . Cleavage of the S2' site by host cell proteases is required for successful infection by 9] . However, the mechanisms involved in exposure and cleavage of the S2' cleavage site are not well understood. Structural biology studies, especially recent cryo-electron microscopy (cryo-EM) studies, have advanced our understanding of the role of the coronavirus S glycoprotein in virus entry. S glycoprotein structures in the prefusion state have been reported for members from the Alphacoronavirus genus (HCoV-NL63), Betacoronavirus genus (mouse hepatitis virus (MHV), HKU1, SARS-CoV and MERS-CoV), Gammacoronavirus genus (IBV), and Deltacoronavirus genus (PdCoV) [7, [10] [11] [12] [13] [14] [15] [16] [17] . Prefusion Structure of the SARS coronavirus spike glycoprotein and ACE2 complex PLOS Pathogens | https://doi.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /></body>