
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: animal hosts and any evidence of continued spill-over to humans</h2><br /></div><br /><br /><br /><div class="card"><h3>One health -an ecological and evolutionary framework for tackling Neglected Zoonotic Diseases</h3>
<p style="text-align: justify; text-justify: inter-word;">Understanding the complex population biology and transmission ecology of multihost parasites has been declared as one of the major challenges of biomedical sciences for the 21st century and the Neglected Zoonotic Diseases (NZDs) are perhaps the most neglected of all the Neglected Tropical Diseases (NTDs). Here we consider how multihost parasite transmission and evolutionary dynamics may affect the success of human and animal disease control programmes, particularly neglected diseases of the developing world. We review the different types of zoonotic interactions that occur, both ecological and evolutionary, their potential relevance for current human control activities, and make suggestions for the development of an empirical evidence base and theoretical framework to better understand and predict the outcome of such interactions. In particular, we consider whether preventive chemotherapy, the current mainstay of NTD control, can be successful without a One Health approach. Transmission within and between animal reservoirs and humans can have important ecological and evolutionary consequences, driving the evolution and establishment of drug resistance, as well as providing selective pressures for spill-over, host switching, hybridizations and introgressions between animal and human parasites. Our aim here is to highlight the importance of both elucidating disease ecology, including identifying key hosts and tailoring control effort accordingly, and understanding parasite evolution, such as precisely how infectious agents may respond and adapt to anthropogenic change. Both elements are essential if we are to alleviate disease risks from NZDs in humans, domestic animals and wildlife.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>What Have We Learned About Middle East Respiratory Syndrome Coronavirus Emergence in Humans? A Systematic Literature Review</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Middle East respiratory syndrome coronavirus (MERS-CoV) was first identified in humans in 2012. A systematic literature review was conducted to synthesize current knowledge and identify critical knowledge gaps. Materials and Methods: We conducted a systematic review on MERS-CoV using PRISMA guidelines. We identified 407 relevant, peer-reviewed publications and selected 208 of these based on their contributions to four key areas: virology; clinical characteristics, outcomes, therapeutic and preventive options; epidemiology and transmission; and animal interface and the search for natural hosts of MERS-CoV. Results: Dipeptidyl peptidase 4 (DPP4/CD26) was identified as the human receptor for MERS-CoV, and a variety of molecular and serological assays developed. Dromedary camels remain the only documented zoonotic source of human infection, but MERS-like CoVs have been detected in bat species globally, as well as in dromedary camels throughout the Middle East and Africa. However, despite evidence of camel-to-human MERS-CoV transmission and cases apparently related to camel contact, the source of many primary cases remains unknown. There have been sustained health care-associated human outbreaks in Saudi Arabia and South Korea, the latter originating from one traveler returning from the Middle East. Transmission mechanisms are poorly understood; for health care, this may include environmental contamination. Various potential therapeutics have been identified, but not yet evaluated in human clinical trials. At least one candidate vaccine has progressed to Phase I trials.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>A qualitative study of zoonotic risk factors among rural communities in southern China</h3>
<p style="text-align: justify; text-justify: inter-word;">This study explored the relationship among zoonotic risk and human behaviour, environment and policies in rural communities in southern China. It identifies key behavioural risk factors that can be targeted for development of tailored risk-mitigation strategies to reduce the threat of novel zoonoses.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Evolution and emergence of novel human infections</h3>
<p style="text-align: justify; text-justify: inter-word;">Some zoonotic pathogens cause sporadic infection in humans but rarely propagate further, while others have succeeded in overcoming the species barrier and becoming established in the human population. Adaptation, driven by selection pressure in human hosts, can play a significant role in allowing pathogens to cross this species barrier. Here we use a simple mathematical model to study potential epidemiological markers of adaptation. We ask: under what circumstances could ongoing adaptation be signalled by large clusters of human infection? If a pathogen has caused hundreds of cases but with little transmission, does this indicate that the species barrier cannot be crossed? Finally, how can case reports be monitored to detect an imminent emergence event? We distinguish evolutionary scenarios under which adaptation is likely to be signalled by large clusters of infection and under which emergence is likely to occur without any prior warning. Moreover, we show that a lack of transmission never rules out adaptability, regardless of how many zoonoses have occurred. Indeed, after the first 100 zoonotic cases, continuing sporadic zoonotic infections without onward, human-to-human transmission offer little extra information on pathogen adaptability. Finally, we present a simple method for monitoring outbreaks for signs of emergence and discuss public health implications.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Full Title: A systematic review of MERS-CoV (Middle East Respiratory Syndrome Coronavirus) 2 seroprevalence and viral RNA prevalence in dromedary camels: implications for animal vaccination</h3>
<p style="text-align: justify; text-justify: inter-word;">Human infection with Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is driven by 28 recurring dromedary-to-human spill-over events, leading decision-makers to consider dromedary 29 vaccination. Dromedary vaccine candidates in the development pipeline are showing hopeful results, 30 but gaps in our understanding of the epidemiology of MERS-CoV in dromedaries must be addressed 31 to design and evaluate potential vaccination strategies. We systematically reviewed the published 32 literature reporting seroprevalence and/or prevalence of active MERS-CoV infection in dromedary 33 populations from both cross-sectional and longitudinal studies, including 60 studies in our qualitative 34 syntheses. MERS-CoV seroprevalence increased with age up to 80-100% in adult dromedaries 35 supporting geographically wide spread endemicity of MERS-CoV in dromedaries in both the Arabian 36 Peninsula and countries exporting dromedaries from Africa. The high prevalence of active infection 37 measured in juveniles and at sites where dromedary populations mix should guide further 38 investigation -particularly of dromedary movement -and inform vaccination strategy design. 39 40 41 42 43 44 45 46 47 48 49 50 . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/574103 doi: bioRxiv preprint 65 studies (7). 66 However, vaccine strategy evaluation is currently precluded by gaps in the understanding of the 67 epidemiology of MERS-CoV in dromedaries. The dromedary population is highly heterogeneous and 68 spans a wide geographic area stretching from West Africa through to the Middle East and parts of 69 Asia. Knowing when and where dromedaries would need to be targeted, and the likely impact of 70 vaccination, is necessary before further consideration of dromedary vaccination in the wider 71 socioeconomic and cultural context. 72 Here, we systematically reviewed published studies that measured MERS-CoV antibody 73 seroprevalence in dromedaries and/or prevalence of viral RNA in dromedaries. Assuming assay 74 specificity and long-term presence of antibodies after infection, seroprevalence can be used to 75 estimate what proportion of a dromedary population has ever been infected with MERS-CoV. Broken 76 down by age class, this can tell us when most animals encounter the infection for the first time. 77 Additionally, although whole-virus isolation and culture is necessary to confirm whether the shedding . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/574103 doi: bioRxiv preprint 101 When available, we took the results of neutralisation-based testing over methods that determined 102 seropositivity based on antibody screening tests alone. Neutralising antibody tests are more specific 103 and are the WHO (World Health Organisation) recommended method for confirming MERS-CoV 104 seropositivity (9).</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>viruses Drivers of MERS-CoV Emergence in Qatar</h3>
<p style="text-align: justify; text-justify: inter-word;">Abstract: MERS-CoV (Middle East respiratory syndrome corona virus) antibodies were detected in camels since 1983, but the first human case was only detected in 2012. This study sought to identify and quantify possible drivers for the MERS-CoV emergence and spillover to humans. A list of potential human, animal and environmental drivers for disease emergence were identified from literature. Trends in possible drivers were analyzed from national and international databases, and through structured interviews with experts in Qatar. The discovery and exploitation of oil and gas led to a 5-fold increase in Qatar GDP coupled with a 7-fold population growth in the past 30 years. The lifestyle gradually transformed from Bedouin life to urban sedentary life, along with a sharp increase in obesity and other comorbidities. Owing to substantial governmental support, camel husbandry and competitions flourished, exacerbating the already rapidly occurring desertification that forced banning of free grazing in 2005. Consequently, camels were housed in compact barns alongside their workers. The transition in husbandry leading to high density camel farming along with increased exposure to humans, combined with the increase of camel movement for the racing and breeding industry, have led to a convergence of factors driving spillover of MERS-CoV from camels to humans. 13 of 15 to provide "hard evidence" of causality. Nevertheless, this is the first comprehensive quantitative overview of possible drivers of MERS-CoV in Qatar.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">The ongoing outbreak of viral pneumonia in China and beyond is associated with a novel 28 coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated 29 with a seafood market in Wuhan, China, where the sale of wild animals may be the source of 30 zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of 31 any intermediate host facilitating transfer to humans is unknown. Here, we report the 32 identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in 33 anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin 34 associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, 35 including one very closely related to 2019-nCoV in the receptor-binding domain. The 36 discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV 37 suggests that pangolins should be considered as possible intermediate hosts for this novel 38 human virus and should be removed from wet markets to prevent zoonotic transmission.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying transmission of emerging zoonoses: 3 Using mathematical models to maximize the value of surveillance data 4 5 6 7</h3>
<p style="text-align: justify; text-justify: inter-word;">Although collecting exact spatial coordinates has become more practical in contemporary disease 111 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/677021 doi: bioRxiv preprint 6 surveillance, privacy and confidentiality concerns can arise in both human and agricultural 112 contexts when data contains high-resolution spatial information [19, 20,[22][23][24][25], leading to data 113 being reported in a non-localized manner. Methods that can use this inexact spatial information 114 are especially needed for zoonotic diseases, where any additional information about the 115 proximity of human cases to one another can improve the power to distinguish between human-116 to-human transmission and zoonotic spillover. 117</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Evidence of Middle East Respiratory Syndrome Coronavirus (MERS-Cov) and Widespread Seroprevalence among Camels in Kenya</h3>
<p style="text-align: justify; text-justify: inter-word;">We describe the first genome isolation of Middle East respiratory syndrome coronavirus (MERS-CoV) in Kenya. This fatal zoonotic pathogen was first described in the Kingdom of Saudi Arabia in 2012. Epidemiological and molecular evidence revealed zoonotic transmission from camels to humans and between humans. Currently, MERS-CoV is classified by the WHO as having high pandemic potential requiring greater surveillance. Previous studies of MERS-CoV in Kenya mainly focused on site-specific and archived camel and human serum samples for antibodies. We conducted active nationwide cross-sectional surveillance of camels and humans in Kenya, targeting both nasal swabs and plasma samples from 1,163 camels and 486 humans collected from January 2016 to June 2018. A total of 792 camel plasma samples were positive by ELISA. Seroprevalence increased with age, and the highest prevalence was observed in adult camels (82.37%, 95% confidence interval (CI) 79.50-84.91). More female camels were significantly seropositive (74.28%, 95% CI 71.14-77.19) than male camels (P \ 0.001) (53.74%, 95% CI 48.48-58.90). Only 11 camel nasal swabs were positive for MERS-CoV by reverse transcription-quantitative PCR. Phylogenetic analysis of whole genome sequences showed that Kenyan MERS-CoV clustered within sub-clade C2, which is associated with the African clade, but did not contain signature deletions of orf4b in African viruses. None of the human plasma screened contained neutralizing antibodies against MERS-CoV. This study confirms the geographically widespread occurrence of MERS-CoV in Kenyan camels. Further one-health surveillance approaches in camels, wildlife, and human populations are needed.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV spillover at the camel-human interface</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while casebased and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Halton et al. A systematic review of community-based interventions for emerging zoonotic infectious diseases in Southeast Asia A systematic review of community-based interventions for emerging zoonotic infectious diseases in Southeast Asia Halton et al. A systematic review of community-based interventions for emerging zoonotic infectious diseases in Southeast Asia Halton et al. A systematic review of community-based interventions for emerging zoonotic infectious diseases in Southeast Asia Implications for practice Halton et al. A systematic review of community-based interventions for emerging zoonotic infectious diseases in Southeast Asia Figure 1: Framework for evaluating the impact of context and behavioural mechanisms on intervention outcomes Review objective/questions</h3>
<p style="text-align: justify; text-justify: inter-word;">The objective of this review is to identify the effectiveness of surveillance systems and communitybased interventions in identifying and responding to emerging and re-emerging zoonotic infections in SE Asia.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>A case-crossover analysis of the impact of weather on primary cases of Middle East respiratory syndrome</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Middle East respiratory syndrome coronavirus (MERS-CoV) is endemic in dromedary camels in the Arabian Peninsula, and zoonotic transmission to people is a sporadic event. In the absence of epidemiological data on the reservoir species, patterns of zoonotic transmission have largely been approximated from primary human cases. This study aimed to identify meteorological factors that may increase the risk of primary MERS infections in humans.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>A Generic Quantitative Risk Assessment Framework for the Entry of Bat-Borne Zoonotic Viruses into the European Union a11111 under ANTIGONE (Global Onset of Novel Epidemics) with project number 278976. Additional matched funding was provided by the UK Department for Environment, Food and Rural Affairs (Defra; https://www.gov.uk/government/ organisations/department-for-environment-food</h3>
<p style="text-align: justify; text-justify: inter-word;">Africa. Here we describe a model designed to provide initial quantitative predictions of the risk of entry of such viruses to European Union (EU) Member States (MSs) through four routes: human travel, legal trade (e.g. fruit and animal products), live animal movements and illegal importation of bushmeat. The model utilises available datasets to assess the movement via these routes between individual countries of the world and EU MSs. These data are combined with virus specific data to assess the relative risk of entry between EU MSs. As a case study, the model was parameterised for NiV. Scenario analyses showed that the selection of exporting countries with NiV and potentially contaminated trade products were essential to the accuracy of all model outputs. Uncertainty analyses of other model parameters identified that the model expected number of years to an introduction event within the EU was highly susceptible to the prevalence of NiV in bats. The relative rankings of the MSs and routes, however, were more robust. The UK, the Netherlands and Germany were consistently the most likely points of entry and the ranking of most MSs varied by no more than three places (maximum variation five places). Legal trade was consistently the most likely route of entry, only falling below human travel when the estimate of the prevalence of NiV in bats was particularly low. Any model-based calculation is dependent on the data available to feed into the model and there are distinct gaps in our knowledge, particularly in regard to various pathogen/virus as well as host/bat characteristics. However, the strengths of this model lie in the provision of relative comparisons of risk among routes and MSs. The potential for expansion of the model to include other routes and viruses and the possibility of rapid parameterisation demonstrates its potential for use in an outbreak situation.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Inhibition of novel b coronavirus replication by a combination of interferon-a2b and ribavirin</h3>
<p style="text-align: justify; text-justify: inter-word;">A novel b coronavirus (nCoV), has been identified as the etiological agent of 17 confirmed cases (11 deaths) of a severe respiratory illness with occasional renal failure from patients in Saudi Arabia 1 , Qatar 2 , Jordan and the United Kingdom 3,4 . Complete genome sequencing 1,5 determined that this new virus is closely related to two Asian bat betacoronaviruses (HKU4 and HKU5) 6 in lineage C. This makes nCoV the first lineage C betacoronavirus known to infect humans 5 . While human-to-human transmission is assumed to be less extensive as compared to SARS-CoV, three of the cases in Saudi Arabia were within one family and several healthcare workers who cared for two of the cases in Jordan have been classified as probable cases 5 . Moreover, in the most recent cluster of cases 7 , two of the three cases did not have a history of travel to the Middle East, but are suspected to have resulted from human-to-human transmission within the UK from a family member with a travel history to Saudi Arabia and Pakistan. This would suggest that human-to-human transmission can occur in close contact settings. Presumably, the sporadic nature of the apparently unlinked index cases in three different, albeit geographically proximal countries, and the close relationship to Asian bat coronaviruses, suggests that the source of nCoV is zoonotic 8 .</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral factors in influenza pandemic risk assessment</h3>
<p style="text-align: justify; text-justify: inter-word;">The threat of an influenza A virus pandemic stems from continual virus spillovers from reservoir species, a tiny fraction of which spark sustained transmission in humans. To date, no pandemic emergence of a new influenza strain has been preceded by detection of a closely related precursor in an animal or human. Nonetheless, influenza surveillance efforts are expanding, prompting a need for tools to assess the pandemic risk posed by a detected virus. The goal would be to use genetic sequence and/or biological assays of viral traits to identify those non-human influenza viruses with the greatest risk of evolving into pandemic threats, and/or to understand drivers of such evolution, to prioritize pandemic prevention or response measures. We describe such efforts, identify progress and ongoing challenges, and discuss three specific traits of influenza viruses (hemagglutinin receptor binding specificity, hemagglutinin pH of activation, and polymerase complex efficiency) that contribute to pandemic risk.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Neglected Zoonotic Diseases-The Long and Winding Road to Advocacy</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Years of advocacy for the neglected tropical diseases (NTDs) have focused the world's attention on these diseases of the poor, resulting most recently in the 2012 ''London Declaration'' and the recent World Health Assembly Resolution WHA66.12 on NTDs in May 2013. Control of the endemic neglected zoonotic diseases (NZDs) would benefit from a similar campaign, which needs the support of a global community.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Epidemiology and Infection Large-scale Lassa fever outbreaks in Nigeria: quantifying the association between disease reproduction number and local rainfall</h3>
<p style="text-align: justify; text-justify: inter-word;">Lassa fever (LF) is increasingly recognised as an important rodent-borne viral haemorrhagic fever presenting a severe public health threat to sub-Saharan West Africa. In 2017-18, LF caused an unprecedented epidemic in Nigeria and the situation was worsening in 2018-19. This work aims to study the epidemiological features of epidemics in different Nigerian regions and quantify the association between reproduction number (R) and state rainfall. We quantify the infectivity of LF by the reproduction numbers estimated from four different growth models: the Richards, three-parameter logistic, Gompertz and Weibull growth models. LF surveillance data are used to fit the growth models and estimate the Rs and epidemic turning points (τ) in different regions at different time periods. Cochran's Q test is further applied to test the spatial heterogeneity of the LF epidemics. A linear random-effect regression model is adopted to quantify the association between R and state rainfall with various lag terms. Our estimated Rs for 2017-18 (1.33 with 95% CI 1.29-1.37) was significantly higher than those for 2016-17 (1.23 with 95% CI: (1.22, 1.24)) and 2018-19 (ranged from 1.08 to 1.36). We report spatial heterogeneity in the Rs for epidemics in different Nigerian regions. We find that a oneunit (mm) increase in average monthly rainfall over the past 7 months could cause a 0.62% (95% CI 0.20%-1.05%)) rise in R. There is significant spatial heterogeneity in the LF epidemics in different Nigerian regions. We report clear evidence of rainfall impacts on LF epidemics in Nigeria and quantify the impact.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Evolution of MERS Coronavirus: Dromedaries as a Recent Intermediate Host or Long-Time Animal Reservoir?</h3>
<p style="text-align: justify; text-justify: inter-word;">While dromedary camels are the immediate animal source of MERS coronavirus (MERS-CoV) infection, the evolutionary origin of MERS-CoV remains obscure. We analyzed 219 camel and human MERS-CoV genome sequences available in GenBank. Phylogenetic analysis showed that 5 and 214 strains belong to clade A and B, respectively, with clade A further divided into lineage A1 (3 human strains) and lineage A2 (2 camel strains), and clade B divided into B1 to B6 (each containing both human and camel strains). Recombination analysis showed potential recombination events in five strains from dromedaries in Saudi Arabia, with recombination between lineage B5 and B3 in four strains, and between lineage B3 and B4 in one strain. The spike protein showed the highest number of amino acid substitutions, especially between A2 and other lineages, and contained positively selected codons. Notably, codon 1020 was positively selected among B and B5 strains, and can distinguish between clade A (Q1020) and B (R1020/H1020) strains, suggesting that this residue may play a role in the evolution of S protein during divergence of different lineages. The time of the most recent common ancestor of all MERS-CoV was dated to approximately 2010. The implications on the role of camels in the evolution of MERS-CoV are discussed.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Complex Epidemiology of a Zoonotic Disease in a Culturally Diverse Region: Phylogeography of Rabies Virus in the Middle East</h3>
<p style="text-align: justify; text-justify: inter-word;">Despite being one of the oldest recognised infectious diseases, rabies continues to cause thousands of preventable human deaths per year. As a zoonotic disease, control of infection in the reservoir has been proven the most efficient route to reduction of human cases. In some regions, the epidemiology is well understood, with either dogs or wildlife known to be the primary reservoir and with little or no movement from, or into other regions. This is not the case in the Middle East, where rabies is underreported in animals and humans, there is little laboratory confirmation of infection, and the extent of rabies spread from country to country is not known. Previous studies have demonstrated transboundary movement of rabies but have been limited by a low number of available samples from some countries, and the direction of spread has been difficult to estimate. Here we use rabies virus partial genome sequences of 183 viruses from over 20 countries, combined with geographical and temporal information, to reconstruct the evolution of rabies viruses circulating in the Middle East. The results reveal an apparent barrier to spread between some regions but regular movement between others. These analyses will support policy on rabies control by indicating the relative importance of local control and animal movement restrictions when allocating resources.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>A camel-derived MERS-CoV with a variant spike protein cleavage site and distinct fusion activation properties</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) continues to circulate in both humans and camels, and the origin and evolution of the virus remain unclear. Here we characterize the spike protein of a camel-derived MERS-CoV (NRCE-HKU205) identified in 2013, early in the MERS outbreak. NRCE-HKU205 spike protein has a variant cleavage motif with regard to the S2′ fusion activation site-notably, a novel substitution of isoleucine for the otherwise invariant serine at the critical P1′ cleavage site position. The substitutions resulted in a loss of furin-mediated cleavage, as shown by fluorogenic peptide cleavage and western blot assays. Cell-cell fusion and pseudotyped virus infectivity assays demonstrated that the S2′ substitutions decreased spike-mediated fusion and viral entry. However, cathepsin and trypsin-like protease activation were retained, albeit with much reduced efficiency compared with the prototypical EMC/2012 human strain. We show that NRCE-HKU205 has more limited fusion activation properties possibly resulting in more restricted viral tropism and may represent an intermediate in the complex pattern of MERS-CoV ecology and evolution.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /></body>