
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: natural history of the virus and shedding of it from an infected person</h2><br /></div><br /><br /><br /><div class="card"><h3>viruses Extracellular Vesicles and Ebola Virus: A New Mechanism of Immune Evasion</h3>
<p style="text-align: justify; text-justify: inter-word;">In those who are fortunate enough to survive acute Ebola virus disease (EVD), the virus becomes undetectable in the blood, depicting a typical phenotype of viral clearance. However, either entire virus or viral components may remain detectable in patients relatively long after recovery, particularly in immune-privileged sites [17] [18] [19] [20] [21] [22] [23] [24] . Notably, recent studies have shown that over 1 in 4 male survivors from the 2016 outbreak contained virus within their semen up to 7-9 months past the disappearance of EVD symptoms. Furthermore, virus has still been detectable up to 16-18 months later [17] . This type of "clinical latency" or persistent infection has significance for public health in the following ways: (1) viral reservoirs within immune privileged sites have implications for reemergence and persistence of the virus within the infected individual, potentially allowing for resurgence of infection, morbidity, and mortality [25]; (2) these reservoirs may allow for the transmission of the virus to additional individuals long after initial infection, hindering efforts to control the spread of outbreaks by public health infrastructures [25]; and (3) significant political, socio-economic, and social stigma considerations must be taken into account for policies to support the normal life and travel of EVD survivors [26] [27] [28] [29] . Spread of the virus by this route is of particular concern, as several cases of sexual transmission from male to female have already been documented. One of the first such cases involved transmission from a man 6 months past his recovery to a woman in Liberia [30, 31] . To date, three additional "flare-ups" of EBOV by sexual contact have been determined, together accounting for approximately half of the eight mini-outbreaks caused by persistently infected survivors documented since 2016 [32] [33] [34] [35] . The most concerning of these cases occurred over 480 days (nearly 17 months) past the initial recovery of the patient [35] . To further complicate the matter, there have been multiple reports addressing potential asymptomatic or undiagnosed portions of the population. Previous studies have put the number of seropositive but undiagnosed/asymptomatic individuals anywhere between 0%-47% [36] [37] [38] [39] [40] [41] [42] [43] [44] . This of course implies that such individuals may represent an invisible source of viral transmission and future outbreaks. Exemplifying this is a case involving an asymptomatic mother who likely transmitted EBOV to her 9-month-old infant (as determined by sequencing and phylogenetic analysis) through breast milk, ultimately resulting in the fatality of the child [19] . Interestingly, the asymptomatic father's semen was also EBOV-positive, but was distantly related to the strains found in the mother and child. Currently, the mechanism of this viral persistence is not well characterized, although new studies have hinted as several possibilities, which will be discussed later.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Deep Sequencing and Phylogenetic Analysis of Novel Human Betacoronavirus</h3>
<p style="text-align: justify; text-justify: inter-word;">We describe a strategy for rapidly designing the primers necessary for reverse transcription and cDNA amplification of such diverse RNA viruses and report the full-genome determination of the novel CoV directly from patient sputum using next-generation short-read sequencing. Full genomes from 2 epidemiologically unlinked novel CoV infections separated in time by >2 months were analyzed to gain clues to 2 major questions: what are the precursors of the virus, and how long has the virus been circulating in its current form?</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>SARS-associated Coronavirus Transmission, United States</h3>
<p style="text-align: justify; text-justify: inter-word;">To better assess the risk for transmission of the severe acute respiratory syndrome-associated coronavirus (SARS-CoV), we obtained serial specimens and clinical and exposure data from seven confirmed U.S. SARS patients and their 10 household contacts. SARS-CoV was detected in a day-14 sputum specimen from one casepatient and in five stool specimens from two case-patients. In one case-patient, SARS-CoV persisted in stool for at least 26 days after symptom onset. The highest amounts of virus were in the day-14 sputum sample and a day-14 stool sample. Residual respiratory symptoms were still present in recovered SARS case-patients 2 months after illness onset. Possible transmission of SARS-CoV occurred in one household contact, but this person had also traveled to a SARS-affected area. The data suggest that SARS-CoV is not always transmitted efficiently. Routine collection and testing of stool and sputum specimens of probable SARS case-patients may help the early detection of SARS-CoV infection.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Novel, Divergent Simian Hemorrhagic Fever Viruses in a Wild Ugandan Red Colobus Monkey Discovered Using Direct Pyrosequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions/Significance: These results represent the first documentation of SHFV in a wild primate and confirm the unusual 39 genetic architecture of SHFV relative to the other arteriviruses. They also demonstrate a degree of evolutionary divergence within SHFV that is roughly equivalent to the degree of divergence between other arterivirus species. The presence of two such highly divergent SHFV variants co-infecting a single individual represents a degree of within-host viral diversity that exceeds what has previously been reported for any arterivirus. These results expand our knowledge of the natural history and diversity of the arteriviruses and underscore the importance of wild primates as reservoirs for novel pathogens.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Medicine Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<p style="text-align: justify; text-justify: inter-word;">The geographic spread of 2019 novel coronavirus infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</h3>
<p style="text-align: justify; text-justify: inter-word;">The geographic spread of 2019 novel coronavirus infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Engineering a Replication-Competent, Propagation-Defective Middle East Respiratory Syndrome Coronavirus as a Vaccine Candidate</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging coronavirus infecting humans that is associated with acute pneumonia, occasional renal failure, and a high mortality rate and is considered a threat to public health. The construction of a full-length infectious cDNA clone of the MERS-CoV genome in a bacterial artificial chromosome is reported here, providing a reverse genetics system to study the molecular biology of the virus and to develop attenuated viruses as vaccine candidates. Following transfection with the cDNA clone, infectious virus was rescued in both Vero A66 and Huh-7 cells. Recombinant MERS-CoVs (rMERS-CoVs) lacking the accessory genes 3, 4a, 4b, and 5 were successfully rescued from cDNA clones with these genes deleted. The mutant viruses presented growth kinetics similar to those of the wild-type virus, indicating that accessory genes were not essential for MERS-CoV replication in cell cultures. In contrast, an engineered mutant virus lacking the structural E protein (rMERS-CoV-⌬E) was not successfully rescued, since viral infectivity was lost at early passages. Interestingly, the rMERS-CoV-⌬E genome replicated after cDNA clone was transfected into cells.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Severe acute respiratory syndrome (SARS) is caused by a novel coronavirus (SARS-CoV). In a longitudinal cross-sectional study, we determined the prevalence of virus in bodily excretions and time of seroconversion in discharged patients with SARS. Conjunctival, throat, stool, and urine specimens were collected weekly from 64 patients and tested for SARS-CoV RNA by real-time polymerase chain reaction; serum samples were collected weekly and tested for SARS-CoV antibody with indirect enzyme immunoassay and immunofluorescence assay. In total, 126 conjunctival, 124 throat swab, 116 stool, and 124 urine specimens were analyzed. Five patients had positive stool samples, collected in weeks 5-9. Two patients seroconverted in weeks 7 and 8; the others were seropositive at the first serum sample collection. In this study, 5 (7.8%) of 64 patients continued to shed viral RNA in stool samples only, for up to week 8 of illness. Most seroconversions occurred by week 6 of illness.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Inhibition of Hepatitis E Virus Spread by the Natural Compound Silvestrol</h3>
<p style="text-align: justify; text-justify: inter-word;">HEV is classified as a (+)-strand ssRNA virus and a member of the Hepeviridae family. Four major human-pathogenic genotypes have been identified, with genotype 1 and 2 being restricted to humans, whereas genotype 3 and 4 are able to infect both human and swine. All genotypes contain a~7.4 kb genome [11] , which is composed of three open reading frames (ORF): ORF1 encodes a non-structural polyprotein (pORF1), which is mainly responsible for efficient virus replication, ORF2 encodes the capsid-forming core protein (pORF2), and ORF3 is the coding region for a protein of unknown function (pORF3) [12] . HEV pORF1 is the only polyprotein that is found in the virus' proteome, which is composed of four different subdomains: a methyltransferase domain, a papain-like cysteine protease domain, an RNA helicase domain, and an RNA dependent RNA polymerase domain, as indicated by homology analyses [13] . There is evidence that this polyprotein is further processed and cleaved into smaller proteins by a cysteine protease, with each fragment showing the proposed catalytic activities [14] . The viral RNA genome further contains a 5 -capped, short 5 -UTR (~26 base pairs) [15] . This indicates a dependency on cap-recognizing proteins and a possible regulation that is based on the untranslated region, since the genome itself serves as a template for protein biosynthesis. Moreover, 3 -poly adenylation is found in the virus' genome. Additional studies on the HEV genome showed that not only the full genome itself serves as the sole template for viral protein synthesis, but also a bicistronic, subgenomic RNA coding for pORF2, and pORF3 [16, 17] . HEV egress, after capsid assembly, is then managed via the exosomal pathway [18], with pORF3 being an important interaction partner of tumor susceptibility gene 101 (TSG101) [19] , possibly tethering the capsid to the endosomal sorting complexes that are required for transport (ESCRT) that mediate the entry into the multivesicular bodies (MVBs). Therefore, viral particles are found as quasi-enveloped particles (surrounded by an exosomal membrane) in both cell culture supernatant and patient serum, while being excreted (via feces) as naked capsid viral particles [18, 20, 21] . Just recently, a form of pORF2 has been described to be secreted as homodimers, in addition to the population found in assembled capsids [22] .</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Influence of age, severity of infection, and co-infection on the duration of respiratory syncytial virus (RSV) shedding</h3>
<p style="text-align: justify; text-justify: inter-word;">RSV is the most important viral cause of pneumonia and bronchiolitis in children worldwide and has been associated with significant disease burden. With the renewed interest in RSV vaccines, we provide realistic estimates on duration, and influencing factors on RSV shedding which are required to better understand the impact of vaccination on the virus transmission dynamics. The data arise from a prospective study of 47 households (493 individuals) in rural Kenya, followed through a 6-month period of an RSV seasonal outbreak. Deep nasopharyngeal swabs were collected twice each week from all household members, irrespective of symptoms, and tested for RSV by multiplex PCR. The RSV G gene was sequenced. A total of 205 RSV infection episodes were detected in 179 individuals from 40 different households. The infection data were interval censored and assuming a random event time between observations, the average duration of virus shedding was 11·2 (95% confidence interval 10·1-12·3) days. The shedding durations were longer than previous estimates (3·9-7·4 days) based on immunofluorescence antigen detection or viral culture, and were shown to be strongly associated with age, severity of infection, and revealed potential interaction with other respiratory viruses. These findings are key to our understanding of the spread of this important virus and are relevant in the design of control programmes.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of the potential incidence of COVID-19 and effectiveness of contention measures in Spain: a data-driven approach</h3>
<p style="text-align: justify; text-justify: inter-word;">Our society is currently experiencing an unprecedented challenge, managing and containing an outbreak of a new coronavirus disease known as COVID-19. While China -where the outbreak started -seems to have been able to contain the growth of the epidemic, different outbreaks are nowadays being detected in multiple countries. Much is currently unknown about the natural history of the disease, such as a possible asymptomatic spreading or the role of age in both the susceptibility and mortality of the disease. Nonetheless, authorities have to take action and implement contention measures, even if not everything is known. To facilitate this task, we have studied the effect of different containment strategies that can be put into effect. Our work specifically refers to the situation in Spain as of February 28th, 2020, where a few dozens of cases have been detected. We implemented an SEIR-metapopulation model that allows tracing explicitly the spatial spread of the disease through data-driven stochastic simulations. Our results are in line with the most recent recommendations from the World Health Organization, namely, that the best strategy is the early detection and isolation of individuals with symptoms, followed by interventions and public recommendations aimed at reducing the transmissibility of the disease, which although not efficacious for disease eradication, would produce as a second order effect a delay of several days in the raise of the number of infected cases.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>The Role of Severe Acute Respiratory Syndrome (SARS)-Coronavirus Accessory Proteins in Virus Pathogenesis</h3>
<p style="text-align: justify; text-justify: inter-word;">A respiratory disease caused by a novel coronavirus, termed the severe acute respiratory syndrome coronavirus (SARS-CoV), was first reported in China in late 2002. The subsequent efficient human-to-human transmission of this virus eventually affected more than 30 countries worldwide, resulting in a mortality rate of ~10% of infected individuals. The spread of the virus was ultimately controlled by isolation of infected individuals and there has been no infections reported since April 2004. However, the natural reservoir of the virus was never identified and it is not known if this virus will re-emerge and, therefore, research on this virus continues. The SARS-CoV genome is about 30 kb in length and is predicted to contain 14 functional open reading frames (ORFs). The genome encodes for proteins that are homologous to known coronavirus proteins, such as the replicase proteins (ORFs 1a and 1b) and the four major structural proteins: nucleocapsid (N), spike (S), membrane (M) and envelope (E). SARS-CoV also encodes for eight unique proteins, called accessory proteins, with no known homologues. This review will summarize the current knowledge on SARS-CoV accessory proteins and will include: (i) expression and processing; (ii) the effects on cellular processes; and (iii) functional studies.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Human polyomavirus type six in respiratory samples from hospitalized children with respiratory tract infections in Beijing, China</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: HPyV6 is a novel human polyomavirus (HPyV), and neither its natural history nor its prevalence in human disease is well known. Therefore, the epidemiology and phylogenetic status of HPyV6 must be systematically characterized. Methods: The VP1 gene of HPyV6 was detected with an established TaqMan real-time PCR from nasopharyngeal aspirate specimens collected from hospitalized children with respiratory tract infections. The HPyV6-positive specimens were screened for other common respiratory viruses with real-time PCR assays.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Reply to "Studies on Influenza Virus Transmission between Ferrets: the Public Health Risks Revisited"</h3>
<p style="text-align: justify; text-justify: inter-word;">As in our article and Fouchier's letter, here we proceed through the calculation, starting with probability of laboratory-acquired infections and the conditional probability of sparking a pandemic given such an infection and concluding with the consequences thereof. We then discuss some more general considerations.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>First Complete Genome Sequence of Human Coronavirus HKU1 from a Nonill Bat Guano Miner in Thailand</h3>
<p style="text-align: justify; text-justify: inter-word;">Human coronavirus HKU1 (HCoV-HKU1) was first detected in a patient with viral pneumonia from Hong Kong in 2004. Here, we report the first complete genome sequence of HCoV-HKU1 from Thailand, obtained from a nonill person who worked in a bat cave. Phylogenetic tree analysis revealed it as a group B HCoV-HKU1.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronaviridae and SARS-associated Coronavirus Strain HSR1</h3>
<p style="text-align: justify; text-justify: inter-word;">During the recent severe acute respiratory syndrome (SARS) outbreak, the etiologic agent was identified as a new coronavirus (CoV). We have isolated a SARS-associated CoV (SARS-CoV) strain by injecting Vero cells with a sputum specimen from an Italian patient affected by a severe pneumonia; the patient traveled from Vietnam to Italy in March 2003. Ultrastructural analysis of infected Vero cells showed the virions within cell vesicles and around the cell membrane. The full-length viral genome sequence was similar to those derived from the Hong-Kong Hotel M isolate. By using both real-time reverse transcription-polymerase chain reaction TaqMan assay and an infectivity plaque assay, we determined that approximately 360 viral genomes were required to generate a PFU. In addition, heparin (100 mg/mL) inhibited infection of Vero cells by 50%. Overall, the molecular and biologic characteristics of the strain HSR1 provide evidence that SARS-CoV forms a fourth genetic coronavirus group with distinct genomic and biologic features.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Infection Dynamics and Antibody Responses among Clinically Diverse Patients, Saudi Arabia</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) shedding and antibody responses are not fully understood, particularly in relation to underlying medical conditions, clinical manifestations, and mortality. We enrolled MERS-CoVpositive patients at a hospital in Saudi Arabia and periodically collected specimens from multiple sites for real-time reverse transcription PCR and serologic testing. We conducted interviews and chart abstractions to collect clinical, epidemiologic, and laboratory information. We found that diabetes mellitus among survivors was associated with prolonged MERS-CoV RNA detection in the respiratory tract. Among case-patients who died, development of robust neutralizing serum antibody responses during the second and third weeks of illness was not sufficient for patient recovery or virus clearance. Fever and cough among mildly ill patients typically aligned with RNA detection in the upper respiratory tract; RNA levels peaked during the first week of illness. These findings should be considered in the development of infection control policies, vaccines, and antibody therapeutics.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Sequence of Human Betacoronavirus 2c Jordan-N3/2012 after Serial Passage in Mammalian Cells</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is the etiologic agent of a highly lethal pneumonia. Here, we report the full-genome sequence of the Jordan-N3/2012 strain after serial passage in two distinct mammalian cell lines. The genome exhibits noteworthy stability, which may inform the development of vaccines and therapeutics used to treat infection with this virus.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Merkel Cell Polyomavirus DNA in Respiratory Specimens from Children and Adults</h3>
<p style="text-align: justify; text-justify: inter-word;">Merkel cell polyomavirus (MCPyV) DNA was detected in 7 (1.3%) of 526 respiratory tract samples from patients in Australia with upper or lower respiratory tract symptoms. Partial T antigen and major capsid protein sequences of MCPyV identifi ed in respiratory secretions showed high homology (99%-100%) to those found in Merkel cell carcinoma.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Children who possibly transmitted the virus to their asymptomatic contacts had significantly higher viral load than children who probably did not transmit (P < .0001).</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /></body>