
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: latency issues and when there is sufficient viral load to detect the pathogen</h2><br /></div><br /><br /><br /><div class="card"><h3>Detection of multiple respiratory pathogens during primary respiratory infection: nasal swab versus nasopharyngeal aspirate using real-time polymerase chain reaction</h3>
<p style="text-align: justify; text-justify: inter-word;">In this study, we present the multiple detection of respiratory viruses in infants during primary respiratory illness, investigate the sensitivity of nasal swabs and nasopharyngeal aspirates, and assess whether patient characteristics and viral load played a role in the sensitivity. Healthy infants were included at signs of first respiratory tract infection. Paired nasopharyngeal aspirates and nasal swabs were collected. Real-time polymerase chain reaction (PCR) was carried out for 11 respiratory pathogens. Paired nasopharyngeal aspirates and nasal swabs were collected in 98 infants. Rhinovirus (n=67) and respiratory syncytial virus (n=39) were the most frequently detected. Co-infection occurred in 48% (n=45) of the infants. The sensitivity of the nasal swab was lower than the nasopharyngeal aspirate, in particular, for respiratory syncytial virus (51% vs. 100%) and rhinovirus (75% vs. 97%). The sensitivity of the nasal swab was strongly determined by the cycle threshold (CT)</p>
<p align="right"><i>score: 57</i></p>

</div><br /><br /><br /><div class="card"><h3>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Among the 87 laboratory-confirmed cases, there were still 19 cases with RT-PCR results testing negative in the first assay. In the false negatives can be caused by poor sample quality, such as respiratory tract samples collected from the oropharynx; collection that is too early or late in the progression of the disease, in the early stages, the number of viruses in the body is not enough to be . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers</h3>
<p style="text-align: justify; text-justify: inter-word;">We estimate that the global ability to detect imported cases is 38% ( 95% HPDI 22% -64%) of Singapore's capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 -4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore, which leads to 1.8 (95% CI 0.5 -3.4) undetected cases per detected case. The ability to detect imported cases among high surveillance countries is 40% (95% HPDI 22% -67%), among intermediate surveillance countries it is 37% (95% HPDI 18% -68%), and among low surveillance countries it is 11% (95% HPDI 0% -42%). : medRxiv preprint 7</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>An Analysis on the Detection of Biological Contaminants Aboard Aircraft</h3>
<p style="text-align: justify; text-justify: inter-word;">The spread of infectious disease via commercial airliner travel is a significant and realistic threat. To shed some light on the feasibility of detecting airborne pathogens, a sensor integration study has been conducted and computational investigations of contaminant transport in an aircraft cabin have been performed. Our study took into consideration sensor sensitivity as well as the time-to-answer, size, weight and the power of best available commercial off-the-shelf (COTS) devices. We conducted computational fluid dynamics simulations to investigate three types of scenarios: (1) nominal breathing (up to 20 breaths per minute) and coughing (20 times per hour); (2) nominal breathing and sneezing (4 times per hour); and (3) nominal breathing only. Each scenario was implemented with one or seven infectious passengers expelling air and sneezes or coughs at the stated frequencies. Scenario 2 was implemented with two additional cases in which one infectious passenger expelled 20 and 50 sneezes per hour, respectively. All computations were based on 90 minutes of sampling using specifications from a COTS aerosol collector and biosensor. Only biosensors that could provide an answer in under 20 minutes without any manual preparation steps were included. The principal finding was that the steady-state bacteria concentrations in aircraft would be high enough to be detected in the case where seven infectious passengers are exhaling under scenarios 1 and 2 and where one infectious passenger is actively exhaling in scenario 2. Breathing alone failed to generate sufficient bacterial particles for detection, and none of the scenarios generated sufficient viral particles for detection to be feasible. These results suggest that more sensitive sensors than the COTS devices currently available and/or sampling of individual passengers would be needed for the detection of bacteria and viruses in aircraft.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating Sensitivity of Laboratory Testing for Influenza in Canada through Modelling</h3>
<p style="text-align: justify; text-justify: inter-word;">The estimated sensitivity of influenza tests reported to this national laboratory surveillance system is considerably less than reported test characteristics for most laboratory tests. A number of factors may explain this difference, including sample quality and specimen procurement issues as well as test characteristics. Improved diagnosis would permit better estimation of the burden of influenza.</p>
<p align="right"><i>score: 48</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Metagenomics on Cerebrospinal Fluid</h3>
<p style="text-align: justify; text-justify: inter-word;">Identifying the causative pathogen in central nervous system (CNS) infections is crucial for patient management and prognosis. Many viruses can cause CNS infections, yet screening for each individually is costly and time-consuming. Most metagenomic assays can theoretically detect all pathogens, but often fail to detect viruses because of their small genome and low viral load. Viral metagenomics overcomes this by enrichment of the viral genomic content in a sample. VIDISCA-NGS is one of the available workflows for viral metagenomics, which requires only a small input volume and allows multiplexing of multiple samples per run. The performance of VIDISCA-NGS was tested on 45 cerebrospinal fluid (CSF) samples from patients with suspected CNS infections in which a virus was identified and quantified by polymerase chain reaction. Eighteen were positive for an RNA virus, and 34 for a herpesvirus. VIDISCA-NGS detected all RNA viruses with a viral load >2 Ã— 10 4 RNA copies/mL (n = 6) and 8 of 12 of the remaining low load samples. Only one herpesvirus was identified by VIDISCA-NGS, however, when withholding a DNase treatment, 11 of 18 samples with a herpesvirus load >10 4 DNA copies/mL were detected. Our results indicate that VIDISCA-NGS has the capacity to detect low load RNA viruses in CSF. Herpesvirus DNA in clinical samples is probably non-encapsidated and therefore difficult to detect by VIDISCA-NGS.</p>
<p align="right"><i>score: 44</i></p>

</div><br /><br /><br /><div class="card"><h3>LOMA: A fast method to generate efficient tagged-random primers despite amplification bias of random PCR on pathogens</h3>
<p style="text-align: justify; text-justify: inter-word;">In this paper, we study how the efficiency of random PCR amplification affects hybridization signals. We describe a model that predicts the amplification efficiency of a given random primer on a target viral genome. The prediction allows us to filter false-negative probes of the genome that lie in regions of poor random PCR amplification and improves the accuracy of pathogen detection. Subsequently, we propose LOMA, an algorithm to generate random primers that have good amplification efficiency. Wet-lab validation showed that the generated random primers improve the amplification efficiency significantly.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">The diagnosis of respiratory virus infections has evolved substantially in recent years, with the emergence of new pathogens and the development of novel detection methods. While recent advances have improved the sensitivity and turn-around time of diagnostic tests for respiratory viruses, they have also raised important issues such as cost, and the clinical significance of detecting multiple viruses in a single specimen by molecular methods. This article reviews recent advances in specimen collection and detection methods for diagnosis of respiratory virus infections, and discusses the performance characteristics and limitations of these methods.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>The Impact of Host-Based Early Warning on Disease Outbreaks</h3>
<p style="text-align: justify; text-justify: inter-word;">The detection of communicable pathogens responsible for major outbreaks relies on health care professionals' recognition of symptoms manifesting in infectious individuals. Early warning of such communicable diseases before the onset of symptoms could improve both patient care and public health responses. However, the potential impact of such a host-based early warning system on containing the spread of an outbreak and in steering public health response is unknown.</p>
<p align="right"><i>score: 43</i></p>

</div><br /><br /><br /><div class="card"><h3>Study and interest of cellular load in respiratory samples for the optimization of molecular virological diagnosis in clinical practice</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Respiratory viral diagnosis of upper respiratory tract infections has largely developed through multiplex molecular techniques. Although the sensitivity of different types of upper respiratory tract samples seems to be correlated to the number of sampled cells, this link remains largely unexplored. Methods: Our study included 800 upper respiratory tract specimens of which 400 negative and 400 positive for viral detection in multiplex PCR. All samples were selected and matched for age in these 2 groups. For the positive group, samples were selected for the detected viral species. Results: Among the factors influencing the cellularity were the type of sample (p < 0.0001); patient age (p < 0.001); viral positive or negative nature of the sample (p = 0.002); and, for the positive samples, the number of viral targets detected (0.004 < p < 0.049) and viral species. Conclusion: The cellular load of upper respiratory samples is multifactorial and occurs for many in the sensitivity of molecular detection. However it was not possible to determine a minimum cellularity threshold allowing molecular viral detection. The differences according to the type of virus remain to be studied on a larger scale.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<p style="text-align: justify; text-justify: inter-word;">This disparity in technologies and data collection methods results in incomplete surveillance.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating Incidence Curves of Several Infections Using Symptom Surveillance Data</h3>
<p style="text-align: justify; text-justify: inter-word;">We introduce a method for estimating incidence curves of several co-circulating infectious pathogens, where each infection has its own probabilities of particular symptom profiles. Our deconvolution method utilizes weekly surveillance data on symptoms from a defined population as well as additional data on symptoms from a sample of virologically confirmed infectious episodes. We illustrate this method by numerical simulations and by using data from a survey conducted on the University of Michigan campus. Last, we describe the data needs to make such estimates accurate.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Clinical Medicine Communicating the Risk of Death from Novel Coronavirus Disease (COVID-19)</h3>
<p style="text-align: justify; text-justify: inter-word;">To understand the severity of infection for a given disease, it is common epidemiological practice to estimate the case fatality risk, defined as the risk of death among cases. However, there are three technical obstacles that should be addressed to appropriately measure this risk. First, division of the cumulative number of deaths by that of cases tends to underestimate the actual risk because deaths that will occur have not yet observed, and so the delay in time from illness onset to death must be addressed. Second, the observed dataset of reported cases represents only a proportion of all infected individuals and there can be a substantial number of asymptomatic and mildly infected individuals who are never diagnosed. Third, ascertainment bias and risk of death among all those infected would be smaller when estimated using shorter virus detection windows and less sensitive diagnostic laboratory tests. In the ongoing COVID-19 epidemic, health authorities must cope with the uncertainty in the risk of death from COVID-19, and high-risk individuals should be identified using approaches that can address the abovementioned three problems. Although COVID-19 involves mostly mild infections among the majority of the general population, the risk of death among young adults is higher than that of seasonal influenza, and elderly with underlying comorbidities require additional care.</p>
<p align="right"><i>score: 42</i></p>

</div><br /><br /><br /><div class="card"><h3>Effectiveness of traveller screening for emerging pathogens is shaped by epidemiology and natural history of infection</h3>
<p style="text-align: justify; text-justify: inter-word;">During outbreaks of high-consequence pathogens, airport screening programs have been deployed to curtail geographic spread of infection. The effectiveness of screening depends on several factors, including pathogen natural history and epidemiology, human behavior, and characteristics of the source epidemic. We developed a mathematical model to understand how these factors combine to influence screening outcomes. We analyzed screening programs for six emerging pathogens in the early and late stages of an epidemic. We show that the effectiveness of different screening tools depends strongly on pathogen natural history and epidemiological features, as well as human factors in implementation and compliance. For pathogens with longer incubation periods, exposure risk detection dominates in growing epidemics, while fever becomes a better target in stable or declining epidemics. For pathogens with short incubation, fever screening drives detection in any epidemic stage. However, even in the most optimistic scenario arrival screening will miss the majority of cases.</p>
<p align="right"><i>score: 41</i></p>

</div><br /><br /><br /><div class="card"><h3>Metagenomic Nanopore Sequencing of Influenza Virus Direct from Clinical Respiratory Samples</h3>
<p style="text-align: justify; text-justify: inter-word;">Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential for a diagnostic test for influenza virus which also provides insights on transmission, evolution, and drug resistance and simultaneously detects other viruses. We therefore set out to apply the Oxford Nanopore Technologies sequencing method to metagenomic sequencing of respiratory samples. We generated influenza virus reads down to a limit of detection of 10 2 to 10 3 genome copies/ml in pooled samples, observing a strong relationship between the viral titer and the proportion of influenza virus reads (P Ï­ 4.7 Ï« 10 Ïª5 ). Applying our methods to clinical throat swabs, we generated influenza virus reads for 27/27 samples with mid-to-high viral titers (cycle threshold [C T ] values, Ï½30) and 6/13 samples with low viral titers (C T values, 30 to 40). No false-positive reads were generated from 10 influenza virus-negative samples. Thus, Nanopore sequencing operated with 83% sensitivity (95% confidence interval [CI], 67 to 93%) and 100% specificity (95% CI, 69 to 100%) compared to the current diagnostic standard. Coverage of full-length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza virus titers, we were able to reconstruct Ï¾99% complete sequences for all eight gene segments. We also detected a human coronavirus coinfection in one clinical sample. While further optimization is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza virus and other respiratory viruses.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Optimizing Viral Discovery in Bats</h3>
<p style="text-align: justify; text-justify: inter-word;">Viral discovery studies in bats have increased dramatically over the past decade, yet a rigorous synthesis of the published data is lacking. We extract and analyze data from 93 studies published between 2007-2013 to examine factors that increase success of viral discovery in bats, and specific trends and patterns of infection across host taxa and viral families. Over the study period, 248 novel viruses from 24 viral families have been described. Using generalized linear models, at a study level we show the number of host species and viral families tested best explained number of viruses detected. We demonstrate that prevalence varies significantly across viral family, specimen type, and host taxonomy, and calculate mean PCR prevalence by viral family and specimen type across all studies. Using a logistic model, we additionally identify factors most likely to increase viral detection at an individual level for the entire dataset and by viral families with sufficient sample sizes. Our analysis highlights major taxonomic gaps in recent bat viral discovery efforts and identifies ways to improve future viral pathogen detection through the design of more efficient and targeted sample collection and screening approaches.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Buying Time-The Immune System Determinants of the Incubation Period to Respiratory Viruses</h3>
<p style="text-align: justify; text-justify: inter-word;">Respiratory viruses cause disease in humans characterized by an abrupt onset of symptoms. Studies in humans and animal models have shown that symptoms are not immediate and appear days or even weeks after infection. Since the initial symptoms are a manifestation of virus recognition by elements of the innate immune response, early virus replication must go largely undetected. The interval between infection and the emergence of symptoms is called the incubation period and is widely used as a clinical score. While incubation periods have been described for many virus infections the underlying mechanism for this asymptomatic phase has not been comprehensively documented. Here we review studies of the interaction between human pathogenic respiratory RNA viruses and the host with a particular emphasis on the mechanisms used by viruses to inhibit immunity. We discuss the concept of the "stealth phase", defined as the time between infection and the earliest detectable inflammatory response. We propose that the "stealth phase" phenomenon is primarily responsible for the suppression of symptoms during the incubation period and results from viral antagonism that inhibits major pathways of the innate immune system allowing an extended time of unhindered virus replication.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Medical Informatics and Decision Making Automated real time constant-specificity surveillance for disease outbreaks</h3>
<p style="text-align: justify; text-justify: inter-word;">Modeling the variance of visit patterns enables real-time detection with known, constant specificity at all times. With constant specificity, public health practitioners can better interpret the alarms and better evaluate the cost-effectiveness of surveillance systems.</p>
<p align="right"><i>score: 40</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers</h3>
<p style="text-align: justify; text-justify: inter-word;">We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% -64%) of Singapore's capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 -4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% -67%), among intermediate surveillance locations it is 37% (95% HPDI 18% -68%), and among low surveillance locations it is 11% (95% HPDI 0% -42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 39</i></p>

</div><br /><br /><br /></body>