
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: surveillance of mixed wildlife-livestock farms for viruses</h2><br /></div><br /><br /><br /><div class="card"><h3>viruses Virus Metagenomics in Farm Animals: A Systematic Review</h3>
<p style="text-align: justify; text-justify: inter-word;">Advances in Next-Generation Sequencing (NGS) technologies and rapid development of bioinformatics and computational tools offer new opportunities for EID surveillance in quality and in scale. Particularly, metagenomic NGS (mNGS) allows unbiased detection of all microbes and viruses in a sample, showing potential for timely detection of rare or novel infectious etiologies, as well as for surveillance of foodborne and waterborne viruses [20] [21] [22][23][24][25][26]. However, the use of mNGS as a potential surveillance tool requires a deeper understanding of what is "normal" diversity in humans [27], as well as wildlife [28,29] and farm animals [30]. Characterizing species-specific metagenomes could potentially be used to provide a surveillance baseline for early detection and for tracking of movements of pathogens across different hosts, and has been promoted by projects like the global virome project [31]. However, for such applications, detailed background is needed regarding coverage, representativeness, and biases in the study designs. Here, we conduct a systematic review to identify available literature that performed viral mNGS in common farm animals including cattle, small ruminants (goats and sheep), poultry, and pigs. We reviewed the data and metadata availability and quality of these studies. We further summarized reported viromes of common farm animals in order to translate these efforts as background virus diversity profiles of common farm animals to guide preparedness of diseases at the livestock-human interface.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Metagenomic Survey for Viruses in Western Arctic Caribou, Alaska, through Iterative Assembly of Taxonomic Units</h3>
<p style="text-align: justify; text-justify: inter-word;">Pathogen surveillance in animals does not provide a sufficient level of vigilance because it is generally confined to surveillance of pathogens with known economic impact in domestic animals and practically nonexistent in wildlife species. As most (re-)emerging viral infections originate from animal sources, it is important to obtain insight into viral pathogens present in the wildlife reservoir from a public health perspective. When monitoring living, free-ranging wildlife for viruses, sample collection can be challenging and availability of nucleic acids isolated from samples is often limited. The development of viral metagenomics platforms allows a more comprehensive inventory of viruses present in wildlife. We report a metagenomic viral survey of the Western Arctic herd of barren ground caribou (Rangifer tarandus granti) in Alaska, USA. The presence of mammalian viruses in eye and nose swabs of 39 free-ranging caribou was investigated by random amplification combined with a metagenomic analysis approach that applied exhaustive iterative assembly of sequencing results to define taxonomic units of each metagenome. Through homology search methods we identified the presence of several mammalian viruses, including different papillomaviruses, a novel parvovirus, polyomavirus, and a virus that potentially represents a member of a novel genus in the family Coronaviridae.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">A novel bat-origin coronavirus emerged in Wuhan, China in December 2019 and continues to spread across China and the world. At the time of writing, a massive global response has been implemented to control the disease as it spreads from person to person. Yet the high-risk human-wildlife interactions and interfaces that led to the emergence of SARS-CoV and of 2019-nCoV continue to exist in emerging disease hotspots globally. To prevent the next epidemic and pandemic related to these interfaces, we call for research and investment in three areas: 1) surveillance among wildlife to identify the high-risk pathogens they carry; 2) surveillance among people who have contact with wildlife to identify early spillover events; and 3) improvement of market biosecurity regarding the wildlife trade. As the emergence of a novel virus anywhere can impact the furthest reaches of our connected world, international collaboration among scientists is essential to address these risks and prevent the next pandemic.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular Sciences Viral Metagenomics on Blood-Feeding Arthropods as a Tool for Human Disease Surveillance</h3>
<p style="text-align: justify; text-justify: inter-word;">Surveillance and monitoring of viral pathogens circulating in humans and wildlife, together with the identification of emerging infectious diseases (EIDs), are critical for the prediction of future disease outbreaks and epidemics at an early stage. It is advisable to sample a broad range of vertebrates and invertebrates at different temporospatial levels on a regular basis to detect possible candidate viruses at their natural source. However, virus surveillance systems can be expensive, costly in terms of finances and resources and inadequate for sampling sufficient numbers of different host species over space and time. Recent publications have presented the concept of a new virus surveillance system, coining the terms "flying biological syringes", "xenosurveillance" and "vector-enabled metagenomics". According to these novel and promising surveillance approaches, viral metagenomics on engorged mosquitoes might reflect the viral diversity of numerous mammals, birds and humans, combined in the mosquitoes' blood meal during feeding on the host. In this review article, we summarize the literature on vector-enabled metagenomics (VEM) techniques and its application in disease surveillance in humans. Furthermore, we highlight the combination of VEM and "invertebrate-derived DNA" (iDNA) analysis to identify the host DNA within the mosquito midgut.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of potentially novel paramyxovirus and coronavirus viral RNA in bats and rats in the Mekong Delta region of southern Viet Nam</h3>
<p style="text-align: justify; text-justify: inter-word;">Bats and rodents are being increasingly recognized as reservoirs of emerging zoonotic viruses. Various studies have investigated bat viruses in tropical regions, but to date there are no data regarding viruses with zoonotic potential that circulate in bat and rat populations in Viet Nam. To address this paucity of data, we sampled three bat farms and three wet markets trading in rat meat in the Mekong Delta region of southern Viet Nam. Faecal and urine samples were screened for the presence of RNA from paramyxoviruses, coronaviruses and filoviruses. Paramyxovirus RNA was detected in 4 of 248 (1%) and 11 of 222 (4.9%) bat faecal and urine samples, respectively. Coronavirus RNA was detected in 55 of 248 (22%) of bat faecal samples; filovirus RNA was not detected in any of the bat samples. Further, coronavirus RNA was detected in 12 of 270 (4.4%) of rat faecal samples; all samples tested negative for paramyxovirus.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>Tropical Medicine and Infectious Disease Potential Intermediate Hosts for Coronavirus Transmission: No Evidence of Clade 2c Coronaviruses in Domestic Livestock from Ghana</h3>
<p style="text-align: justify; text-justify: inter-word;">The emergence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV), nearly a decade ago with worldwide distribution, was believed to be of zoonotic origin from bats with dromedary camels as intermediate hosts. There is a likelihood of other domestic livestock serving as intermediate hosts for this virus. The presence of coronaviruses, closely related to MERS-CoV in Ghanaian bats, presented the opportunity to test the hypothesis of transmissibility of this virus through domestic livestock species. The possible interactions between livestock and bats in 31 household farms were accessed by observation and interviews with farmers. Rectal swabs and serum from cattle, sheep, goats, donkeys, and swine from commercial and household farms were tested for MERS-CoV and a Nycteris sp. bat coronavirus, previously detected in Ghana. A pan-PCR assay to detect clade 2c viruses and recombinant immunofluorescence assay to detect anti-spike IgG antibodies against the target viruses were used. Likely contact between livestock and bats was determined for 13 farms (41.9%) that reported confining their livestock and also observing bats in their homes. Livestock were left unconfined on eight farms (25.8%) that also observed bats roosting in trees close to their homes. No viral RNA or antibodies against the two coronaviruses were detected in any of the livestock species tested. Cattle, sheep, goats, donkeys, and swine are not likely hosts of clade 2c coronaviruses.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East respiratory syndrome coronavirus infection in non-camelid domestic mammals</h3>
<p style="text-align: justify; text-justify: inter-word;">We carried out 17 sampling visits to mixed farms or herds where camels and other livestock were raised in Senegal and Egypt and a livestock market in Tunisia.</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>viruses Virus Metagenomics in Farm Animals: A Systematic Review</h3>
<p style="text-align: justify; text-justify: inter-word;">Viruses 2020, 12, 107 2 of 22 flocks and wild birds, suggesting that pre-existing biosecurity measurements could not keep up with the rate of livestock intensification [16] . In 2007-2010, a large-scale Q fever outbreak was reported in the Netherlands, affecting more than 3500 human cases and resulting in a huge economic loss [17, 18] . A steep increase in the number of goat farms most likely was the driver for the increased prevalence of Coxiella burnetii infections, with animal abortion waves that had gone unnoticed. The policy of voluntary reporting abortion outbreaks to the Animal Health Service hindered the timely detection of the circulation of Q fever, and therefore early interventions. These examples indicate that zoonotic risks in the livestock industry should be carefully managed and adapted to livestock intensification. The One Health approach has been coined for advocating collaboration between multiple stakeholders including veterinarians, clinicians, epidemiologists, virologists, microbiologists, ecologists, and policy makers to prevent and control EIDs through the human-animal-environment interface [19] . Surveillance of livestock and the surrounding environment is a hallmark of early detection but is currently targeted to known risks.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral metagenomic analysis of feces of wild small carnivores</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: A number of sequences of possible novel viruses or virus variants were detected, including a theilovirus, phleboviruses, an amdovirus, a kobuvirus and picobirnaviruses. Conclusions: Using random PCR in combination with next generation sequencing, sequences of various novel viruses or virus variants were detected in fecal samples collected from Spanish carnivores. Detected novel viruses highlight the viral diversity that is present in fecal material of wild carnivores.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Metagenomics on Animals as a Tool for the Detection of Zoonoses Prior to Human Infection?</h3>
<p style="text-align: justify; text-justify: inter-word;">Many human viral infections have a zoonotic, i.e., wild or domestic animal, origin. Several zoonotic viruses are transmitted to humans directly via contact with an animal or indirectly via exposure to the urine or feces of infected animals or the bite of a bloodsucking arthropod. If a virus is able to adapt and replicate in its new human host, human-to-human transmissions may occur, possibly resulting in an epidemic, such as the A/H1N1 flu pandemic in 2009. Thus, predicting emerging zoonotic infections is an important challenge for public health officials in the coming decades. The recent development of viral metagenomics, i.e., the characterization of the complete viral diversity isolated from an organism or an environment using high-throughput sequencing technologies, is promising for the surveillance of such diseases and can be accomplished by analyzing the viromes of selected animals and arthropods that are closely in contact with humans. In this review, we summarize our current knowledge of viral diversity within such animals (in particular blood-feeding arthropods, wildlife and domestic animals) using metagenomics and present its possible future application for the surveillance of zoonotic and arboviral diseases.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Sample descriptors linked to metagenomic sequencing data from human and animal enteric samples from Vietnam</h3>
<p style="text-align: justify; text-justify: inter-word;">there is still limited information on the diversity of viruses co-circulating in humans and animals. Here, we report data obtained from a large field collection of enteric samples taken from humans, pigs, rodents and other mammal hosts in Vietnam between 2012 and 2016. Each of 2100 stool or rectal swab samples was subjected to virally-enriched agnostic metagenomic sequencing; the short read sequence data are accessible from the European Nucleotide Archive (ENA). We link the sequence data to metadata on host type and demography and geographic location, distinguishing hospital patients, members of a cohort identified as a high risk of zoonotic infections (e.g. abattoir workers, rat traders) and animals. these data are suitable for further studies of virus diversity and virus discovery in humans and animals from Vietnam and to identify viruses found in multiple hosts that are potentially zoonotic.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of potentially novel paramyxovirus and coronavirus viral RNA in bats and rats in the Mekong Delta region of southern Viet Nam</h3>
<p style="text-align: justify; text-justify: inter-word;">Phylogenetic analysis revealed that the bat paramyxoviruses and bat and rat coronaviruses were related to viruses circulating in bat and rodent populations globally, but showed no cross-species mixing of viruses between bat and rat populations within Viet Nam. Our study shows that potentially novel variants of paramyxoviruses and coronaviruses commonly circulate in bat and rat populations in Viet Nam. Further characterization of the viruses and additional human and animal surveillance is required to evaluate the likelihood of viral spillover and to assess whether these viruses pose a risk to human health.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Detection of potentially novel paramyxovirus and coronavirus viral RNA in bats and rats in the Mekong Delta region of southern Viet Nam</h3>
<p style="text-align: justify; text-justify: inter-word;">bats, coronavirus, paramyxovirus, rats, Viet Nam, zoonotic viruses *VIZIONS consortium group members are present in Appendix 1.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Zoonoses with a wildlife reservoir represent a major public health problem, affecting all continents. Hundreds of pathogens and many different transmission modes are involved, and many factors influence the epidemiology of the various zoonoses. The importance and recognition of wildlife as a reservoir of zoonoses are increasing. Costeffective prevention and control of these zoonoses necessitate an interdisciplinary and holistic approach and international cooperation. Surveillance, laboratory capability, research, training and education, and communication are key elements.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Spillover and pandemic properties of zoonotic viruses with high host plasticity OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Most human infectious diseases, especially recently emerging pathogens, originate from animals, and ongoing disease transmission from animals to people presents a significant global health burden. Recognition of the epidemiologic circumstances involved in zoonotic spillover, amplification, and spread of diseases is essential for prioritizing surveillance and predicting future disease emergence risk. We examine the animal hosts and transmission mechanisms involved in spillover of zoonotic viruses to date, and discover that viruses with high host plasticity (i.e. taxonomically and ecologically diverse host range) were more likely to amplify viral spillover by secondary human-to-human transmission and have broader geographic spread. Viruses transmitted to humans during practices that facilitate mixing of diverse animal species had significantly higher host plasticity. Our findings suggest that animal-to-human spillover of new viruses that are capable of infecting diverse host species signal emerging disease events with higher pandemic potential in that these viruses are more likely to amplify by human-to-human transmission with spread on a global scale.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>Single Stranded DNA Viruses Associated with Capybara Faeces Sampled in Brazil</h3>
<p style="text-align: justify; text-justify: inter-word;">Capybaras (Hydrochoerus hydrochaeris), the world's largest rodents, are distributed throughout South America. These wild herbivores are commonly found near water bodies and are well adapted to rural and urban areas. There is limited information on the viruses circulating through capybaras. This study aimed to expand the knowledge on the viral diversity associated with capybaras by sampling their faeces. Using a viral metagenomics approach, we identified diverse single-stranded DNA viruses in the capybara faeces sampled in the Distrito Federal, Brazil. A total of 148 complete genomes of viruses in the Microviridae family were identified. In addition, 14 genomoviruses (family Genomoviridae), a novel cyclovirus (family Circoviridae), and a smacovirus (family Smacoviridae) were identified. Also, 37 diverse viruses that cannot be assigned to known families and more broadly referred to as unclassified circular replication associated protein encoding single-stranded (CRESS) DNA viruses were identified. This study provides a snapshot of the viral diversity associated with capybaras that may be infectious to these animals or associated with their microbiota or diet.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>viruses Virus Metagenomics in Farm Animals: A Systematic Review</h3>
<p style="text-align: justify; text-justify: inter-word;">A majority of emerging infectious diseases are of zoonotic origin. Metagenomic Next-Generation Sequencing (mNGS) has been employed to identify uncommon and novel infectious etiologies and characterize virus diversity in human, animal, and environmental samples. Here, we systematically reviewed studies that performed viral mNGS in common livestock (cattle, small ruminants, poultry, and pigs). We identified 2481 records and 120 records were ultimately included after a first and second screening. Pigs were the most frequently studied livestock and the virus diversity found in samples from poultry was the highest. Known animal viruses, zoonotic viruses, and novel viruses were reported in available literature, demonstrating the capacity of mNGS to identify both known and novel viruses. However, the coverage of metagenomic studies was patchy, with few data on the virome of small ruminants and respiratory virome of studied livestock. Essential metadata such as age of livestock and farm types were rarely mentioned in available literature, and only 10.8% of the datasets were publicly available. Developing a deeper understanding of livestock virome is crucial for detection of potential zoonotic and animal pathogens and One Health preparedness. Metagenomic studies can provide this background but only when combined with essential metadata and following the "FAIR" (Findable, Accessible, Interoperable, and Reusable) data principles.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral infection detection using metagenomics technology in six poultry farms of eastern China</h3>
<p style="text-align: justify; text-justify: inter-word;">With rapidly increasing animal pathogen surveillance requirements, new technologies are needed for a comprehensive understanding of the roles of pathogens in the occurrence and development of animal diseases. We applied metagenomic technology to avian virus surveillance to study the main viruses infecting six poultry farms in two provinces in eastern China. Cloacal/throat double swabs were collected from 60 birds at each farm according to a random sampling method. The results showed that the method could simultaneously detect major viruses infecting farms, including avian influenza virus, infectious bronchitis virus, Newcastle disease virus, rotavirus G, duck hepatitis B virus, and avian leukemia virus subgroup J in several farms. The test results were consistent with the results from traditional polymerase chain reaction (PCR) or reverse transcription-PCR analyses. Five H9N2 and one H3N8 avian influenza viruses were detected at the farms and were identified as low pathogenic avian influenza viruses according to HA cleavage sites analysis. One detected Newcastle disease virus was classified as Class II genotype I and avirulent type according to F0 cleavage sites analysis. Three avian infectious bronchitis viruses were identified as 4/ 91, CK/CH/LSC/99I and TC07-2 genotypes by phylogenetic analysis of S1 genes. The viral infection surveillance method using metagenomics technology enables the monitoring of multiple viral infections, which allows the detection of main infectious viruses.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Bringing together emerging and endemic zoonoses surveillance: shared challenges and a common solution</h3>
<p style="text-align: justify; text-justify: inter-word;">Early detection of disease outbreaks in human and animal populations is crucial to the effective surveillance of emerging infectious diseases. However, there are marked geographical disparities in capacity for early detection of outbreaks, which limit the effectiveness of global surveillance strategies. Linking surveillance approaches for emerging and neglected endemic zoonoses, with a renewed focus on existing disease problems in developing countries, has the potential to overcome several limitations and to achieve additional health benefits. Poor reporting is a major constraint to the surveillance of both emerging and endemic zoonoses, and several important barriers to reporting can be identified: (i) a lack of tangible benefits when reports are made; (ii) a lack of capacity to enforce regulations; (iii) poor communication among communities, institutions and sectors; and (iv) complexities of the international regulatory environment. Redirecting surveillance efforts to focus on endemic zoonoses in developing countries offers a pragmatic approach that overcomes some of these barriers and provides support in regions where surveillance capacity is currently weakest. In addition, this approach addresses immediate health and development problems, and provides an equitable and sustainable mechanism for building the culture of surveillance and the core capacities that are needed for all zoonotic pathogens, including emerging disease threats.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of Alpha and Beta Coronavirus in Wildlife Species in France: Bats, Rodents, Rabbits, and Hedgehogs</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronaviruses are closely monitored in the context of emerging diseases and, as illustrated with Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome-coronavirus (MERS-CoV), are known to cross the species barrier and eventually to move from wildlife to humans. Knowledge of the diversity of coronaviruses in wildlife is therefore essential to better understand and prevent emergence events. This study explored the presence of coronaviruses in four wild mammal orders in France: Bats, rodents, lagomorphs, and hedgehogs. Betacoronavirus and Alphacoronavirus genera were identified. The results obtained suggest the circulation of potentially evolving virus strains, with the potential to cross the species barrier.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /></body>