
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: understanding whether there is more than one strain of the virus in circulation</h2><br /></div><br /><br /><br /><div class="card"><h3>Occurrence and phylogenetic analysis of bovine respiratory syncytial virus in outbreaks of respiratory disease in Norway</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: BRSV infection was detected either serologically and/or virologically in 18 (86%) of 21 outbreaks and in most cases as a single viral agent. When serology indicated that bovine coronavirus and/or bovine parainfluenza virus 3 were present, the number of BRSV positive animals in the herd was always higher, supporting the view of BRSV as the main pathogen. Sequencing of the G gene of BRSV positive samples showed that the current circulating Norwegian BRSVs belong to genetic subgroup II, along with other North European isolates. One isolate from an outbreak in Norway in 1976 was also investigated. This strain formed a separate branch in subgroup II, clearly different from the current Scandinavian sequences. The currently circulating BRSV could be divided into two different strains that were present in the same geographical area at the same time. The sequence variations between the two strains were in an antigenic important part of the G protein.</p>
<p align="right"><i>score: 91</i></p>

</div><br /><br /><br /><div class="card"><h3>A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes severe acute respiratory illness with a case fatality rate (CFR) of 35,5%. The highest number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this disease. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its epidemiological cycle, prevention and containment measures can be derived from mathematical models of disease epidemiology. We constructed 2-strain models to predict past outbreaks in the interval 2012-2016 and derive key epidemiological information for Macca, Madina and Riyadh. We approached variability in infection through three different disease incidence functions capturing social behavior in response to an epidemic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015-2016 season and accurately predicted both the number of cases at the peak of seasonal incidence and the overall shape of the epidemic cycle. The evolution in the basic reproduction number (R 0 ) warns that MERS-CoV may easily take an epidemic form. The best model correctly captures this feature, indicating a high epidemic risk (1�R 0 �2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be aware that measures for strict containment are urgently needed before new epidemics take off in the region.</p>
<p align="right"><i>score: 87</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological Evidence for Multiple Strains of Canine Norovirus in the UK Dog Population</h3>
<p style="text-align: justify; text-justify: inter-word;">Noroviruses are associated with intestinal disease in humans, cows, pigs, mice, and, more recently, dogs. In 2007, the first canine norovirus (CNV) was identified and characterized in Italy. Subsequent studies have identified CNV in stools of dogs from Portugal, Greece, and the United States. To investigate the prevalence of CNV in the UK dog population, 228 canine stool samples were screened for CNV by qPCR, and 396 serum samples were screened for anti-CNV antibodies. qPCR of RNA extracted from canine stool samples did not reveal any CNV-positive samples, based on samples collected from diarrhoeic and control dogs in 2012-2013. CNV virus-like particles to three different CNV strains were produced using recombinant baculoviruses and a seroprevalence screen undertaken. Anti-CNV antibodies were identified at significant levels in canine serum; 38.1% of samples collected between 1999-2001 and 60.1% of samples collected in 2012-2013 were seropositive. The increase in seroprevalence over time (p,0.001) suggests that the CNV strains screened for are becoming more widespread. Variation in seroprevalence to different CNV strains was also identified. Two-thirds of the dogs were seropositive to a single strain, whereas the remaining third were seropositive to two or three of the strains analysed. This study has provided the first evidence that CNV is present in the UK, with seroprevalence identified to multiple circulating strains. This warrants further study and increased awareness of this recently discovered canine virus.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic characterization of S1 gene of infectious bronchitis virus isolated from commercial poultry flocks in West Java, Indonesia</h3>
<p style="text-align: justify; text-justify: inter-word;">Our result indicates that at least two IBV strains were circulating among poultry in West Java, Indonesia, which is IBV close to vaccine strain 4/91 and 233A. The present study provides updates on the circulating IBV in commercial poultry flocks in West Java, Indonesia, and might use as guidance on selecting a proper IB vaccine strain to improve IB vaccination efficacy in certain region.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>NOTE Repeated avian infectious bronchitis virus infections within a single chicken farm</h3>
<p style="text-align: justify; text-justify: inter-word;">Genotyping of avian infectious bronchitis virus (IBV) was performed on trachea and kidney samples of six chickens obtained from a single farm in Japan. Using two primer sets targeting the spike (S) protein gene, the S1 and S2 regions of DNA fragments were amplified. Sequences of amplified S1 fragments extracted from both organs were identical among the six chickens, showing a JP-I genotype. Sequences of amplified S2 fragments differed between trachea and kidney samples. The kidney profile showed a group IV genotype, whereas the trachea profile showed an unclassified group. This result showed that two different IBVs infected the six chickens. The first IBV infection induced poor protective immunity in this farm, permitting a second IBV infection to occur.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Local Evolutionary Patterns of Human Respiratory Syncytial Virus Derived from Whole-Genome Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">RSV molecular epidemiology, evolution, and transmission requires detailed sequence data and can inform new strategies for virus control and vaccine development. We have generated 27 complete or nearly complete genomes of RSV from hospitalized children attending a rural coastal district hospital in Kilifi, Kenya, over a 10-year period using a novel full-genome deep-sequencing process. Phylogenetic analysis of the new genomes demonstrated the existence and cocirculation of multiple genotypes in both RSV A and B groups in Kilifi. Comparison of local versus global strains demonstrated that most RSV A variants observed locally in Kilifi were also seen in other parts of the world, while the Kilifi RSV B genomes encoded a high degree of variation that was not observed in other parts of the world. The nucleotide substitution rates for the individual open reading frames (ORFs) were highest in the regions encoding the attachment (G) glycoprotein and the NS2 protein. The analysis of RSV full genomes, compared to subgenomic regions, provided more precise estimates of the RSV sequence changes and revealed important patterns of RSV genomic variation and global movement. The novel sequencing method and the new RSV genomic sequences reported here expand our knowledge base for large-scale RSV epidemiological and transmission studies.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Respiratory Syncytial Virus whole- genome sequencing identifies convergent evolution of sequence duplication in the C-terminus of the G gene OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">RSV has an epidemic seasonality similar to the influenza viruses, with increased cases during the winter in temperate climates and during the monsoon season in tropical and sub-tropical climates 1, 9, 13, 14 . RSV can be classified into two antigenic groups (A and B), each containing several distinct subgroups based on antigenic and genomic sequence differences, especially in the G glycoprotein 15-17 . Studies suggest group A viruses cause more severe disease and transmit more readily than group B viruses in infants 9 . These two groups tend to alternate in prevalence between RSV seasons and also show evidence of multiple co-circulating intra-group viral genotypes, or clades, during any given season 9, 13, 16, 18, 19 , resulting in a diverse set of circulating viruses that can adapt to herd immunity. It is unclear if this represents a gradual evolution of viral genomes or stochastic differences in infection rates by co-circulating strains.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Human Coronavirus NL63 Molecular Epidemiology and Evolutionary Patterns in Rural Coastal Kenya</h3>
<p style="text-align: justify; text-justify: inter-word;">Results. HCoV-NL63 was detected in 1.3% (75/5573) of child pneumonia admissions. Two HCoV-NL63 genotypes circulated in Kilifi between 2008 and 2014. Full genome sequences formed a monophyletic clade closely related to contemporary HCoV-NL63 from other global locations. An unexpected pattern of repeat infections was observed with some individuals showing higher viral titers during their second infection. Similar patterns for 2 other endemic coronaviruses, HCoV-229E and HCoV-OC43, were observed. Repeat infections by HCoV-NL63 were not accompanied by detectable genotype switching.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Phylogenetic investigation of enteric bovine coronavirus in Ireland reveals partitioning between European and global strains</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Bovine coronavirus is a primary cause of neonatal calf diarrhea worldwide, and is also associated with acute diarrhea in adult cattle during the winter season. There are no reports on molecular characterization of bovine coronavirus in Ireland, and little data exists apart from serological studies. Findings: In this study, 11 neonatal (mean age 9 days) calf BCoV strains from the south of Ireland were collected over a one year period and characterized using molecular methods. The spike gene which encodes a protein involved in viral entry, infectivity and immune response shows the most variability amongst the isolates and was subsequently selected for in depth analysis. Phylogenetic analysis of the spike gene revealed that the Irish strains clustered with novel BCoV strains from Europe in a unique clade, possibly indicating lineage partitioning. Direct analysis of alignments identified amino acid changes in the spike protein unique to the Irish clade. Conclusion: Thus, monitoring of bovine coronavirus in Ireland is important as the current isolates in circulation in the south of Ireland may be diverging from the available vaccine strain, which may have implications regarding future BCoV vaccine efficacy.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Influenza-like illness outbreaks in nursing homes in Corsica, France, 2014-2015: epidemiological and molecular characterization</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: To study the molecular epidemiology of the influenza outbreaks in nursing homes (NHs) to determine whether multiple influenza strains were involved.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine epidemic diarrhea virus (PEDV), which emerged in the United States in 2013, has spread throughout North America. Limited availability of PEDV complete genomes worldwide has impeded our understanding of PEDV introduction into the United States. To determine the relationship between the North American strains and global emerging and historic PEDV strains, we sequenced and analyzed complete genomes of 74 strains from North America; the strains clustered into 2 distinct clades. Compared with the initially reported virulent US PEDV strains, 7 (9.7%) strains from 4 states contained insertions and deletions in the spike gene (S INDELs). These S INDEL strains share 99.8%-100% nt identity with each other and 96.2%-96.7% nt identity with the initial US strains. Furthermore, the S IN-DEL strains form a distinct cluster within North American clade II, sharing 98.6%-100% nt identity overall. In the United States, the S INDEL and original PEDV strains are cocirculating and could have been introduced simultaneously.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of the spike, ORF3, and nucleocapsid genes of porcine epidemic diarrhea virus circulating on Thai swine farms, 2011-2016 Distributed under Creative Commons CC-BY 4.0</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine epidemic diarrhea virus (PEDV) outbreaks on pig farms have caused significant economic loss in the swine industry since it was first reported in Thailand a decade ago. Anecdotal evidence suggests that PEDV is now endemic in this region, therefore genome information of circulating PEDV is important for molecular surveillance and evaluation of potential benefits of field vaccination. Here, we characterized PEDV infection on commercial Thai swine farms by screening 769 samples of feces and small intestinal contents from pigs with diarrhea between 2011 and 2016. Using reverse-transcription polymerase chain reaction targeting the spike (S) gene, 153 PEDV-positive samples were further subjected to analysis of the open reading frame 3 and nucleocapsid (N) genes. Comparison of 95 samples in which nucleotide sequencing was successfully obtained for all three genes revealed evolutionary diversity among the Thai PEDV strains. Phylogenetic analyses suggest that although some Thai strains changed little from years past, others resembled more closely to the recent strains reported in China. Interestingly, eight Thai PEDV strains possessed amino acid deletions in the N protein. The PEDV sequence divergence may be responsible for driving periodic outbreaks and continued persistence of PEDV on commercial swine farms. Our findings provide important insight into regional PEDV strains in circulation, which may assist future inclusions of suitable strains for future PEDV vaccines.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>A cluster of adenovirus type B55 infection in a neurosurgical inpatient department of a general hospital in Guangdong, China Background: Human adenovirus type 55 is a re-emerging human respiratory pathogen</h3>
<p style="text-align: justify; text-justify: inter-word;">All these genes showed 100% homology and fell into the HAdV55 [P14H11F14] cluster, indicating that HAdV55 was the single viral strain for the outbreak. While not conclusive, the epidemic investigation revealed that the outbreak was introduced by nurses from sources outside the hospital. It was likely that a transmission from staff to inpatients had occurred.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Lineage specific antigenic differences in porcine torovirus hemagglutinin-esterase (PToV-HE) protein</h3>
<p style="text-align: justify; text-justify: inter-word;">Hemagglutinin-esterases (HE) are viral envelope proteins present in some members from the toro-, corona-and orthomyxovirus families, all related with enteric and/or respiratory tract infections. HE proteins mediate reversible binding to sialic acid receptor determinants, very abundant glycan residues in the enteric and respiratory tracts. The role of the HE protein during the torovirus infection cycle remains unknown, although it is believed to be important in the natural infection process. The phylogenetic analysis of HE coding sequences from porcine torovirus (PToV) field strains revealed the existence of two distinct HE lineages. In a previous study, PToV virus strains with HE proteins from the two lineages were found coexisting in a pig herd, and they were even obtained from the same animal at two consecutive sampling time points. In this work, we report antigenic differences between the two HE lineages, and discuss the possible implications that the coexistence of viruses belonging to both lineages might have on the spread and sustainment of PToV infection in the farms.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Full-Genome Deep Sequencing and Phylogenetic Analysis of Novel Human Betacoronavirus</h3>
<p style="text-align: justify; text-justify: inter-word;">We describe a strategy for rapidly designing the primers necessary for reverse transcription and cDNA amplification of such diverse RNA viruses and report the full-genome determination of the novel CoV directly from patient sputum using next-generation short-read sequencing. Full genomes from 2 epidemiologically unlinked novel CoV infections separated in time by >2 months were analyzed to gain clues to 2 major questions: what are the precursors of the virus, and how long has the virus been circulating in its current form?</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and Genetic Diversity Analysis of Human Coronavirus OC43 among Adult Patients with Acute Respiratory Infections in Beijing</h3>
<p style="text-align: justify; text-justify: inter-word;">To determine the prevalence, epidemiology and genetic diversity of human coronavirus OC43 (HCoV-OC43) among adult patients with acute respiratory infections (ARI) in Beijing,five hundred and fifty-nine nasopharyngeal swab samples were collected from adult patients with ARI in Beijing. The prevalence of HCoV-OC43 infection among these patients was assessed using two different OneStep reverse transcription polymerase chain reaction (RT-PCR) assays. The epidemiological profiles of the patients with HCoV-OC43 infection were described. Partial S and N genes of HCoV-OC43 circulating strains were sequenced followed by phylogenetic analysis and amino acid alignment. Our results showed that the prevalence of HCoV-OC43 infection was 12.52% (95% CI: 9.78-15.26%), and the epidemic peak occurred in autumn. Fifty partial S and 40 partial N fragments were obtained from these patients. Phylogenetic analysis based on neighbour-joining method showed that at least three distinct clusters (A, B, C/D) of HCoV-OC43 strains were circulating among adult patients with ARI in Beijing. In addition, some novel unique clusters (UNT) of HCoV-OC43 were found in the S-and N-based phylogenetic trees. Furthermore, consensus amino acids substitutes for each cluster were also found after alignment of partial S or N sequence coding region in this study. In conclusion, we herein describe the prevalence of HCoV-OC43 among adult patients and provide substantial evidence for the genetic diversity of HCoV-OC43 circulating in Beijing.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Respiratory Syncytial Virus whole- genome sequencing identifies convergent evolution of sequence duplication in the C-terminus of the G gene OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">circulate are likely mechanisms contributing to the evasion of herd immunity. Surveillance and largescale whole-genome sequencing of RSV is currently limited but would help identify its evolutionary dynamics and sites of selective immune evasion. In this study, we performed complete-genome nextgeneration sequencing of 92 RSV isolates from infants in central Tennessee during the 2012-2014 RSV seasons. We identified multiple co-circulating clades of RSV from both the A and B groups. Each clade is defined by signature N-and O-linked glycosylation patterns. Analyses of specific RSV genes revealed high rates of positive selection in the attachment (G) gene. We identified RSV-A viruses in circulation with and without a recently reported 72-nucleotide G gene sequence duplication. Furthermore, we show evidence of convergent evolution of G gene sequence duplication and fixation over time, which suggests a potential fitness advantage of RSV with the G sequence duplication.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Effects of memory on the shapes of simple outbreak trees OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Understanding outbreaks of an infectious disease is important for understanding how a pathogen spreads, and in determining the best steps to take to control it. Recently, the advent of next-generation sequencing has permitted the use of genomic data to assist in understanding outbreaks. Even small amounts of genetic variation within an outbreak can potentially be detected with whole-genome sequencing, and used to aid in reconstructing who infected whom 1-8 . Genomic data are typically analysed by inferring phylogenetic trees (phylogenies), namely trees in which the tips correspond to pathogen isolates from infected cases, and the internal nodes correspond to inferred ancestors. Phylogenies are usually rooted, binary trees. An ongoing challenge in epidemiology is to make the best use of genomic data, usually with the help of inference and analysis of phylogenetic trees that carry information on parameters including the basic reproduction number (R 0 ) 9,10 .</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Molecular characterization of type 1 porcine reproductive and respiratory syndrome viruses (PRRSV) isolated in the Netherlands from 2014 to 2016</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine reproductive and respiratory syndrome virus (PRRSV) is the causative agent of a devastating pig disease present all over the world. The remarkable genetic variation of PRRSV, makes epidemiological and molecular analysis of circulating viruses highly important to review current diagnostic tools and vaccine efficacy. Monitoring PRRS viruses supports modern herd management by explaining the source of found viruses, either internally or externally from the herd. No epidemiological or molecular study has been published on circulating PRRS-viruses in the Netherlands, since the early nineties. Therefore, the objective of this study is to investigate circulating PRRS-viruses in the Netherlands in 2014, 2015 and 2016 on a molecular level by sequencing ORF2, ORF3, ORF4, ORF5, ORF6 and ORF7. The results demonstrate that the 74 PRRSV strains belong to PRRSV-1, but the diversity among strains is high, based on nucleotide identity, individual ORF length and phylogenetic trees of individual ORFs. Furthermore, the data presented here show that the phylogenetic topology of some viruses is ORF dependent and suggests recombination. The identity of the strain of interest might be misinterpreted and wrong conclusions may be drawn in a diagnostic and epidemiological perspective, when only ORF5 is analyzed, as performed in many routine sequencing procedures.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>Etiology of respiratory tract infections in the community and clinic in Ilorin, Nigeria</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: Children were selected via convenience sampling from communities and a tertiary care center (n = 91) during spring 2017 in Ilorin, Nigeria. Nasal swabs were collected and tested using polymerase chain reaction. The majority (79.1%) of subjects were under 6 years old, of whom 46 were infected (63.9%). A total of 33 of the 91 subjects had one or more respiratory tract virus; there were 10 cases of triple infection and 5 of quadruple. Parainfluenza virus 4, respiratory syncytial virus B and enterovirus were the most common viruses in the clinical sample; present in 93.8% (15/16) of clinical subjects, and 6.7% (5/75) of community subjects (significant difference, p < 0.001). Coronavirus OC43 was the most common virus detected in community members (13.3%, 10/75). A different strain, Coronavirus OC 229 E/NL63 was detected among subjects from the clinic (2/16) and not detected in the community. This pilot study provides evidence that data from the community can potentially represent different information than that sourced clinically, suggesting the need for community surveillance to enhance public health efforts and scientific understanding of respiratory infections.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /></body>