
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: tracking variations of the virus over time</h2><br /></div><br /><br /><br /><div class="card"><h3>Local Evolutionary Patterns of Human Respiratory Syncytial Virus Derived from Whole-Genome Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">RSV molecular epidemiology, evolution, and transmission requires detailed sequence data and can inform new strategies for virus control and vaccine development. We have generated 27 complete or nearly complete genomes of RSV from hospitalized children attending a rural coastal district hospital in Kilifi, Kenya, over a 10-year period using a novel full-genome deep-sequencing process. Phylogenetic analysis of the new genomes demonstrated the existence and cocirculation of multiple genotypes in both RSV A and B groups in Kilifi. Comparison of local versus global strains demonstrated that most RSV A variants observed locally in Kilifi were also seen in other parts of the world, while the Kilifi RSV B genomes encoded a high degree of variation that was not observed in other parts of the world. The nucleotide substitution rates for the individual open reading frames (ORFs) were highest in the regions encoding the attachment (G) glycoprotein and the NS2 protein. The analysis of RSV full genomes, compared to subgenomic regions, provided more precise estimates of the RSV sequence changes and revealed important patterns of RSV genomic variation and global movement. The novel sequencing method and the new RSV genomic sequences reported here expand our knowledge base for large-scale RSV epidemiological and transmission studies.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Enterovirus D68 in Hospitalized Children: Sequence Variation, Viral Loads and Clinical Outcomes</h3>
<p style="text-align: justify; text-justify: inter-word;">An outbreak of enterovirus D68 (EV-D68) caused severe respiratory illness in 2014. The disease spectrum of EV-D68 infections in children with underlying medical conditions other than asthma, the role of EV-D68 loads on clinical illness, and the variation of EV-D68 strains within the same institution over time have not been described. We sought to define the association between EV-D68 loads and sequence variation, and the clinical characteristic in hospitalized children at our institution from 2011 to 2014.</p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Identification of an infectious bronchitis coronavirus strain exhibiting a classical genotype but altered antigenicity, pathogenicity, and innate immunity profile OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Avian coronavirus infectious bronchitis virus (IBV) is a member of Coronaviridae family, and the virus is responsible for infectious bronchitis, an acute, highly contagious chicken viral disease. Respiratory signs such as coughing, sneezing, and nasal discharge are the primary symptoms associated with IBV infections, and some of the strains also display kidney and oviduct tropisms, causing urogenital symptoms 1-3 . The disease poses economic threat to the poultry industry worldwide. More than 50 serotypes of IBV have been documented since the first isolation of virus in Massachusetts (Mass), US in the 1930 s 2 . The most commonly detected genotype of IBV in the US is Ark-DPI, accounting for 23% to 65% of total isolation per year 4 . While most strains circulating in the US cause respiratory symptoms, nephropathogenic QX-like strains are dominant in Europe 2 . In the Far East, nephropathogenic LX4, LDL as well as proventricular-type QX-like strains have been reported frequently 2 . Presently, vaccination is adopted to manage the disease, and Mass type vaccine H120 is the most widely used vaccine formulation 2,5 . Due to frequent strain variations, however, traditional Mass type vaccines can be ineffective against infections by different IBV variants 2,5,6 . Effective disease management thus demands better elucidation of strain variations.</p>
<p align="right"><i>score: 90</i></p>

</div><br /><br /><br /><div class="card"><h3>Bioinformatics analysis of SARS coronavirus genome polymorphism</h3>
<p style="text-align: justify; text-justify: inter-word;">The comparative study of SARS-CoV isolates provides essential information for genome polymorphism, indication of strain differences and variants evolution. It may help with the development of effective treatment.</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Kingdom of Saudi Arabia e ; Saudi Aramco Medical Services Organization, Saudi Aramco, Dhahran, Kingdom of Saudi Arabia f ; Prince Sultan Military Medical City, Riyadh, Kingdom of Saudi Arabia g</h3>
<p style="text-align: justify; text-justify: inter-word;">The Middle East respiratory syndrome coronavirus (MERS-CoV) was first documented in the Kingdom of Saudi Arabia (KSA) in 2012 and, to date, has been identified in 180 cases with 43% mortality. In this study, we have determined the MERS-CoV evolutionary rate, documented genetic variants of the virus and their distribution throughout the Arabian peninsula, and identified the genome positions under positive selection, important features for monitoring adaptation of MERS-CoV to human transmission and for identifying the source of infections. Respiratory samples from confirmed KSA MERS cases from May to September 2013 were subjected to whole-genome deep sequencing, and 32 complete or partial sequences (20 were >99% complete, 7 were 50 to 94% complete, and 5 were 27 to 50% complete) were obtained, bringing the total available MERS-CoV genomic sequences to 65. An evolutionary rate of 1.12 ؋ 10 ؊3 substitutions per site per year (95% credible interval [95% CI], 8.76 ؋ 10 ؊4 ; 1.37 ؋ 10 ؊3 ) was estimated, bringing the time to most recent common ancestor to March 2012 (95% CI, December 2011; June 2012). Only one MERS-CoV codon, spike 1020, located in a domain required for cell entry, is under strong positive selection. Four KSA MERS-CoV phylogenetic clades were found, with 3 clades apparently no longer contributing to current cases. The size of the population infected with MERS-CoV showed a gradual increase to June 2013, followed by a decline, possibly due to increased surveillance and infection control measures combined with a basic reproduction number (R 0 ) for the virus that is less than 1. IMPORTANCE MERS-CoV adaptation toward higher rates of sustained human-to-human transmission appears not to have occurred yet. While MERS-CoV transmission currently appears weak, careful monitoring of changes in MERS-CoV genomes and of the MERS epidemic should be maintained. The observation of phylogenetically related MERS-CoV in geographically diverse locations must be taken into account in efforts to identify the animal source and transmission of the virus.</p>
<p align="right"><i>score: 89</i></p>

</div><br /><br /><br /><div class="card"><h3>Analysis of intrapatient heterogeneity uncovers the microevolution of Middle East respiratory syndrome coronavirus | RESEARCH REPORT</h3>
<p style="text-align: justify; text-justify: inter-word;">Genome sequence analysis of Middle East respiratory syndrome coronavirus (MERS-CoV) variants from patient specimens has revealed the evolutionary dynamics and mechanisms of pathogenesis of the virus. However, most studies have analyzed the consensus sequences of MERS-CoVs, precluding an investigation of intrapatient heterogeneity. Here, we analyzed non-consensus sequences to characterize intrapatient heterogeneity in cases associated with the 2015 outbreak of MERS in South Korea. Deepsequencing analysis of MERS-CoV genomes performed on specimens from eight patients revealed significant intrapatient variation; therefore, sequence heterogeneity was further analyzed using targeted deep sequencing. A total of 35 specimens from 24 patients (including a super-spreader) were sequenced to detect and analyze variants displaying intrapatient heterogeneity. Based on the analysis of non-consensus sequences, we demonstrated the intrapatient heterogeneity of MERS-CoVs, with the highest level in the super-spreader specimen. The heterogeneity could be transmitted in a close association with variation in the consensus sequences, suggesting the occurrence of multiple MERS-CoV infections. Analysis of intrapatient heterogeneity revealed a relationship between D510G and I529T mutations in the receptor-binding domain (RBD) of the viral spike glycoprotein. These two mutations have been reported to reduce the affinity of the RBD for human CD26. Notably, although the frequency of both D510G and I529T varied greatly among specimens, the combined frequency of the single mutants was consistently high (87.7% ± 1.9% on average). Concurrently, the frequency of occurrence of the wild type at the two positions was only 6.5% ± 1.7% on average, supporting the hypothesis that selection pressure exerted by the host immune response played a critical role in shaping genetic variants and their interaction in human MERS-CoVs during the outbreak.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Genetic Analysis of Avian Coronavirus Infectious Bronchitis Virus in Yellow Chickens in Southern China over the Past Decade: Revealing the Changes of Genetic Diversity, Dominant Genotypes, and Selection Pressure</h3>
<p style="text-align: justify; text-justify: inter-word;">The high mutation rates of infectious bronchitis virus (IBV) pose economic threats to the poultry industry. In order to track the genetic evolutionary of IBV isolates circulating in yellow chickens, we continued to conduct the genetic analyses of the structural genes S1, E, M, and N from 64 IBV isolates in southern China during 2009-2017. The results showed that the dominant genotypes based on the four genes had changed when compared with those during 1985-2008. Based on the S1 gene phylogenetic tree, LX4-type (GI-19) was the most dominant genotype, which was different from that during 1985-2008. The second most dominant genotype was LDT3-A-type, but this genotype disappeared after 2012. New-type 1 (GVI-1) isolates showed increasing tendency and there were four aa (QKEP) located in the hypervariable region (HVR) III and one aa (S) insertion in all the New-type 1 isolates. Both the analyses of amino acid entropy and molecular evolutionary rate revealed that the variations from large to small were S1, E, M, and N. Purifying selection was detected in the S1, E, M, and N gene proteins, which was different from the positive selection during 1985-2008. Six isolates were confirmed to be recombinants, possibly generated from a vaccine virus of the 4/91-type or LDT3-A-type and a circulating virus. The estimated times for the most recent common ancestors based on the S1, E, M, and N genes were the years of 1744, 1893, 1940, and 1945, respectively. Bayesian skyline analysis revealed a sharp decrease in genetic diversity of all the four structural genes after 2010 and since late 2015, the viral population rapidly rose. In conclusion, the IBVs circulating in southern China over the past decade have experienced a remarkable change in genetic diversity, dominant genotypes, and selection pressure, indicating the importance of permanent monitoring of circulating strains and the urgency for developing new vaccines to counteract the emerging LX4-type and New-type IBVs.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission patterns and evolution of respiratory syncytial virus in a community outbreak identified by genomic analysis</h3>
<p style="text-align: justify; text-justify: inter-word;">Detailed information on the source, spread and evolution of respiratory syncytial virus (RSV) during seasonal community outbreaks remains sparse. Molecular analyses of attachment (G) gene sequences from hospitalized cases suggest that multiple genotypes and variants co-circulate during epidemics and that RSV persistence over successive seasons is characterized by replacement and multiple new introductions of variants. No studies have defined the patterns of introduction, spread and evolution of RSV at the local community and household level. We present a whole genome sequence analysis of 131 RSV group A viruses collected during 6-month household-based RSV infection surveillance in Coastal Kenya, 2010 within an area of 12 km 2 . RSV infections were identified by regular symptom-independent screening of all household members twice weekly. Phylogenetic analysis revealed that the RSV A viruses in nine households were closely related to genotype GA2 and fell within a single branch of the global phylogeny. Genomic analysis allowed the detection of householdspecific variation in seven households. For comparison, using only G gene analysis, household-specific variation was found only in one of the nine households. Nucleotide changes were observed both intra-host (viruses identified from same individual in follow-up sampling) and inter-host (viruses identified from different household members) and these coupled with sampling dates enabled a partial reconstruction of the within household transmission chains. The genomic evolutionary rate for the household dataset was estimated as 2.307 Â 10 À 3 (95% highest posterior density: 0.935-4.165Â 10 À 3 ) substitutions/site/year. We conclude that (i) at the household level, most RSV infections arise from the introduction of a single virus variant followed by accumulation of household specific variation and (ii) analysis of complete virus genomes is crucial to better understand viral transmission in the community. A key question arising is whether prevention of RSV introduction or spread within the household by vaccinating key transmitting household members would lead to a reduced onward community-wide transmission.</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Laboratory Investigation and Phylogenetic Analysis of an Imported Middle East Respiratory Syndrome Coronavirus Case in Greece</h3>
<p style="text-align: justify; text-justify: inter-word;">Rapid and reliable laboratory diagnosis of persons suspected of Middle East respiratory syndrome coronavirus (MERS-CoV) infection is important for timely implementation of infection control practices and disease management. In addition, monitoring molecular changes in the virus can help elucidate chains of transmission and identify mutations that might influence virus transmission efficiency. This was illustrated by a recent laboratory investigation we conducted on an imported MERS-CoV case in Greece. Two oropharyngeal swab specimens were collected on the 1 st and 2 nd day of patient hospitalization and tested using two real-time RT-PCR (rRT-PCR) assays targeting the UpE and Orf-1a regions of the MERS-CoV genome and RT-PCR and partial sequencing of RNA-dependent RNA polymerase and nucleocapsid genes. Serum specimens were also collected and serological test were performed. Results from the first swab sample were inconclusive while the second swab was strongly positive for MERS-CoV RNA by rRT-PCR and confirmed positive by RT-PCR and partial gene sequencing. Positive serologic test results further confirmed MERS-CoV infection. Full-length nucleocapsid and spike gene coding sequences were later obtained from the positive swab sample. Phylogenetic analysis revealed that the virus was closely related to recent human-derived MERS-CoV strains obtained in Jeddah and Makkah, Saudi Arabia, in April 2014 and dromedary camels in Saudi Arabia and Qatar. These findings were consistent with the patient's history. We also identified a unique amino acid substitution in the spike receptor binding domain that may have implications for receptor binding efficiency. Our initial inconclusive rRT-PCR results highlight the importance of collecting multiple specimens from suspect MERS-CoV cases and particularly specimens from the lower respiratory tract.</p>
<p align="right"><i>score: 83</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.</p>
<p align="right"><i>score: 81</i></p>

</div><br /><br /><br /><div class="card"><h3>Title: 2 GI-16 lineage (624/I or Q1), there and Back Again: the 3 history of one of the major threat for poultry farming of our 4 era. 5 6 Short title: 7 Phylodynamic analysis of IBV GI-16 lineage</h3>
<p style="text-align: justify; text-justify: inter-word;">75 Until then, the two genotypes had been named (and considered) independent: the 624/I was first 76 reported in Italy in 1993 [8], and thereafter in Slovenia [9], Poland [10], and Russia [11]. Moreover, doi: bioRxiv preprint 4 77 retrospective studies demonstrated its wide circulation in Italy since the early 1960s [12], where it 78 remained the most prevalent genotype for more than a decade. However, from the 1990s, its 79 detection and reporting became sporadic or absent. At the same time, in 1996, a new virus (later 80 renamed Q1) was identified in animals affected by proventriculitis [13] in China and thereafter in 81 other Asian [14,15], Middle Eastern [16,17], South American [18,19] and European countries 82 [7,20-22]. Remarkably, not recognizing the relationship between these strains, a new spreading 83 from Asia was supposed, similarly to what proven for the GI-19 lineage (QX genotype) [23]. The 84 insight provided by recent genetic studies [6,7] raised new questions about the history, 85 epidemiology and spreading of the GI-16 lineage around the world. The knowledge of these 86 processes would be of interest not only for the particular case under investigation, but could also 87 provide useful information on the general intra-and inter-continents viral dispersion patterns, aiding 88 the development and implementation of adequate control measures. 89 Unfortunately, the paucity of sequences available before Q1 detection in China prevented this 90 investigation for a long time. In the present study, a high number of retrospective GI-16 sequences 91 were generated from Italian samples collected since 1963 and analyzed together with the worldwide 92 available ones using a phylodynamic approach to reconstruct the history of this genotype and infer 93 its spreading pattern around the world. 94 95 Material and Methods 96 Italian strain detection and sequencing 97 IBV samples collected between 1963-2017 were included in the study. Italian sequences were 98 obtained during the routine diagnostic activity of the Istituto Zooprofilattico Sperimentale della 99 Lombardia e dell'Emilia Romagna. Samples of trachea, kidneys, lungs, cecal tonsils, or tracheal</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid, Unbiased PRRSV Strain Detection Using MinION Direct RNA Sequencing and Bioinformatics Tools</h3>
<p style="text-align: justify; text-justify: inter-word;">Prompt detection of pathogens during an outbreak is essential for efficient disease control. Real-time quantitative reverse transcription polymerase chain reaction (RT-qPCR) testing, which has the advantages of short turnaround time and high sensitivity [27, 28] , is currently the primary molecular diagnostic method for PRRSV detection and is performed routinely in diagnostic laboratories. The current PCR methods can quickly detect the presence of PRRSV in general [29] , but cannot identify specific strains unless targeted primers are designed which requires prior knowledge about the strains of interest [26, 30] . Nucleic acid sequencing technologies have been incorporated as a new diagnostic tool in recent years to provide necessary support, such as strain information, for PCR in clinical sample diagnostics [31] [32] [33] . Sequencing approaches have been applied successfully to various fields in virology so far, such as the discovery of novel viruses/strains, whole viral genome sequencing, quasispecies detection, and epidemiological investigations [34, 35] . The fact that sequencing is robust and does not need prior knowledge of the pathogens/strains under detection is particularly important for rapid responses to highly variable pathogens, such as PRRSV [36] . The routine use of viral genome sequencing and genomic surveillance will not only serve as a powerful tool for PRRSV detection [37] , but also provides researchers with a better understanding of PRRSV epidemiology and how the virus transmits, spreads, and evolves, thus facilitating effective prevention and control measures [38] .</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>The global spread of Middle East respiratory syndrome: an analysis fusing traditional epidemiological tracing and molecular phylodynamics</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusion: Together, these analyses can help us identify factors associated with viral evolution and spread as well as establish efficacy of infection control measures. The results are especially pertinent to countries without current MERS-CoV endemic, since their unfamiliarity makes them particularly susceptible to uncontrollable spread of a virus that may be imported by travelers.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Respiratory Syncytial Virus whole- genome sequencing identifies convergent evolution of sequence duplication in the C-terminus of the G gene OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">circulate are likely mechanisms contributing to the evasion of herd immunity. Surveillance and largescale whole-genome sequencing of RSV is currently limited but would help identify its evolutionary dynamics and sites of selective immune evasion. In this study, we performed complete-genome nextgeneration sequencing of 92 RSV isolates from infants in central Tennessee during the 2012-2014 RSV seasons. We identified multiple co-circulating clades of RSV from both the A and B groups. Each clade is defined by signature N-and O-linked glycosylation patterns. Analyses of specific RSV genes revealed high rates of positive selection in the attachment (G) gene. We identified RSV-A viruses in circulation with and without a recently reported 72-nucleotide G gene sequence duplication. Furthermore, we show evidence of convergent evolution of G gene sequence duplication and fixation over time, which suggests a potential fitness advantage of RSV with the G sequence duplication.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Rhinovirus Genome Variation during Chronic Upper and Lower Respiratory Tract Infections</h3>
<p style="text-align: justify; text-justify: inter-word;">Routine screening of lung transplant recipients and hospital patients for respiratory virus infections allowed to identify human rhinovirus (HRV) in the upper and lower respiratory tracts, including immunocompromised hosts chronically infected with the same strain over weeks or months. Phylogenetic analysis of 144 HRV-positive samples showed no apparent correlation between a given viral genotype or species and their ability to invade the lower respiratory tract or lead to protracted infection. By contrast, protracted infections were found almost exclusively in immunocompromised patients, thus suggesting that host factors rather than the virus genotype modulate disease outcome, in particular the immune response. Complete genome sequencing of five chronic cases to study rhinovirus genome adaptation showed that the calculated mutation frequency was in the range observed during acute human infections. Analysis of mutation hot spot regions between specimens collected at different times or in different body sites revealed that non-synonymous changes were mostly concentrated in the viral capsid genes VP1, VP2 and VP3, independent of the HRV type. In an immunosuppressed lung transplant recipient infected with the same HRV strain for more than two years, both classical and ultra-deep sequencing of samples collected at different time points in the upper and lower respiratory tracts showed that these virus populations were phylogenetically indistinguishable over the course of infection, except for the last month. Specific signatures were found in the last two lower respiratory tract populations, including changes in the 59UTR polypyrimidine tract and the VP2 immunogenic site 2. These results highlight for the first time the ability of a given rhinovirus to evolve in the course of a natural infection in immunocompromised patients and complement data obtained from previous experimental inoculation studies in immunocompetent volunteers.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Temporal Dynamics of Co-circulating Lineages of Porcine Reproductive and Respiratory Syndrome Virus</h3>
<p style="text-align: justify; text-justify: inter-word;">Porcine Reproductive and Respiratory Syndrome Virus (PRRSV) is the most important endemic pathogen in the U.S. swine industry. Despite control efforts involving improved biosecurity and different vaccination protocols, the virus continues to circulate and evolve. One of the foremost challenges in its control is high levels of genetic and antigenic diversity. Here, we quantify the co-circulation, emergence and sequential turnover of multiple PRRSV lineages in a single swine-producing region in the United States over a span of 9 years (2009)(2010)(2011)(2012)(2013)(2014)(2015)(2016)(2017). By classifying over 4,000 PRRSV sequences (open-reading frame 5) into phylogenetic lineages and sub-lineages, we document the ongoing diversification and temporal dynamics of the PRRSV population, including the rapid emergence of a novel sub-lineage that appeared to be absent globally pre-2008. In addition, lineage 9 was the most prevalent lineage from 2009 to 2010, but its occurrence fell to 0.5% of all sequences identified per year after 2014, coinciding with the emergence or re-emergence of lineage 1 as the dominant lineage. The sequential dominance of different lineages, as well as three different sublineages within lineage 1, is consistent with the immune-mediated selection hypothesis for the sequential turnover in the dominant lineage. As host populations build immunity through natural infection or vaccination toward the most common variant, this dominant (sub-) lineage may be replaced by an emerging variant to which the population is more susceptible. An analysis of patterns of non-synonymous and synonymous mutations revealed evidence of positive selection on immunologically important regions of the genome, further supporting the potential that immune-mediated selection shapes the evolutionary and epidemiological dynamics for this virus. This has important implications for patterns of emergence and re-emergence of genetic variants of PRRSV that have negative impacts on the swine industry. Constant surveillance on PRRSV occurrence is crucial to a better understanding of the epidemiological and evolutionary dynamics of co-circulating viral lineages. Further studies utilizing whole genome sequencing and exploring the extent of cross-immunity between heterologous PRRS viruses could shed further light on PRRSV immunological response and aid in developing strategies that might be able to diminish disease impact.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen's evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies. IMPORTANCE Surveillance programs have become key assets in determining the emergence or prevalence of pathogens circulating in human and animal populations. Genomic surveillance, in particular, provides comprehensive information on the history of isolates and potential molecular markers for infectivity and pathogenicity. Current techniques for evaluating genomic surveillance are inaccurate, ignoring the pathogen's evolutionary rate and biodiversity, as well as the timing of sequence collection. Using sequence data, we propose the q2 coefficient as a quantitative measure of surveillance completeness that combines elements of time and evolution without defining arbitrary criteria for clades or species. Through several case studies of influenza A, dengue, and West Nile viruses, we employed the q2 coefficient to identify sampling deficiencies in different host species and locations, as well as examine the effects of different public health policies through historical records of the q2 coefficient. These results can guide public health agencies to focus resource allocation and virus collection to bolster specific problems in surveillance.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence and genetic diversity analysis of human coronaviruses among cross-border children</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: More than a decade after the outbreak of human coronaviruses (HCoVs) SARS in Guangdong province and Hong Kong SAR of China in 2002, there is still no reoccurrence, but the evolution and recombination of the coronaviruses in this region are still unknown. Therefore, surveillance on the prevalence and the virus variation of HCoVs circulation in this region is conducted. Methods: A total of 3298 nasopharyngeal swabs samples were collected from cross-border children (<6 years, crossing border between Southern China and Hong Kong SAR) showing symptoms of respiratory tract infection, such as fever (body temperature > 37.5°C), from 2014 May to 2015 Dec. Viral nucleic acids were analyzed and sequenced to study the prevalence and genetic diversity of the four human coronaviruses. The statistical significance of the data was evaluated with Fisher chi-square test. Results: 78 (2.37%; 95%CI 1.8-2.8%) out of 3298 nasopharyngeal swabs specimens were found to be positive for OC43 (36;1.09%), HKU1 (34; 1.03%), NL63 (6; 0.18%) and 229E (2;0.01%). None of SARS or MERS was detected. The HCoVs predominant circulating season was in transition of winter to spring, especially January and February and NL63 detected only in summer and fall. Complex population with an abundant genetic diversity of coronaviruses was circulating and they shared homology with the published strains (99-100%). Besides, phylogenetic evolutionary analysis indicated that OC43 coronaviruses were clustered into three clades (B,D,E), HKU1 clustered into two clades(A,B) and NL63 clustered into two clades(A,B). Moreover, several novel mutations including nucleotides substitution and the insertion of spike of the glycoprotein on the viral surface were discovered.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Network Analysis of Global Influenza Spread</h3>
<p style="text-align: justify; text-justify: inter-word;">Although vaccines pose the best means of preventing influenza infection, strain selection and optimal implementation remain difficult due to antigenic drift and a lack of understanding global spread. Detecting viral movement by sequence analysis is complicated by skewed geographic and seasonal distributions in viral isolates. We propose a probabilistic method that accounts for sampling bias through spatiotemporal clustering and modeling regional and seasonal transmission as a binomial process. Analysis of H3N2 not only confirmed East-Southeast Asia as a source of new seasonal variants, but also increased the resolution of observed transmission to a country level. H1N1 data revealed similar viral spread from the tropics. Network analysis suggested China and Hong Kong as the origins of new seasonal H3N2 strains and the United States as a region where increased vaccination would maximally disrupt global spread of the virus. These techniques provide a promising methodology for the analysis of any seasonal virus, as well as for the continued surveillance of influenza.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /></body>