
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: rapid sequencing for a target regions of genome</h2><br /></div><br /><br /><br /><div class="card"><h3>Rapid Genome Sequencing of RNA Viruses</h3>
<p style="text-align: justify; text-justify: inter-word;">We developed a system for rapid determination of viral RNA sequences whereby genomic sequence is obtained from cultured virus isolates without subcloning into plasmid vectors. This method affords new opportunities to address the challenges of unknown or untypeable emerging viruses.</p>
<p align="right"><i>score: 135</i></p>

</div><br /><br /><br /><div class="card"><h3>Complete viral RNA genome sequencing of ultra-low copy samples by sequence-independent amplification</h3>
<p style="text-align: justify; text-justify: inter-word;">RNA viruses are the causative agents for AIDS, influenza, SARS, and other serious health threats. Development of rapid and broadly applicable methods for complete viral genome sequencing is highly desirable to fully understand all aspects of these infectious agents as well as for surveillance of viral pandemic threats and emerging pathogens. However, traditional viral detection methods rely on prior sequence or antigen knowledge. In this study, we describe sequenceindependent amplification for samples containing ultra-low amounts of viral RNA coupled with Illumina sequencing and de novo assembly optimized for viral genomes. With 5 million reads, we capture 96 to 100% of the viral protein coding region of HIV, respiratory syncytial and West Nile viral samples from as little as 100 copies of viral RNA. The methods presented here are scalable to large numbers of samples and capable of generating full or near full length viral genomes from clone and clinical samples with low amounts of viral RNA, without prior sequence information and in the presence of substantial host contamination.</p>
<p align="right"><i>score: 132</i></p>

</div><br /><br /><br /><div class="card"><h3>Ultra-deep mutant spectrum profiling: improving sequencing accuracy using overlapping read pairs</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: We demonstrate that overlapping read pairs (ORP) --generated by combining short fragment sequencing libraries and longer sequencing reads --significantly reduce sequencing error rates and improve rare variant detection accuracy. Using this sequencing protocol and an error model optimized for variant detection, we are able to capture a large number of genetic mutations present within a viral population at ultra-low frequency levels (<0.05%). Conclusions: Our rare variant detection strategies have important implications beyond viral evolution and can be applied to any basic and clinical research area that requires the identification of rare mutations.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Citation: Targeted NGS Platforms for Genetic Screening and Gene Discovery in Primary Immunodeficiencies</h3>
<p style="text-align: justify; text-justify: inter-word;">Objective: To evaluate the strength and weakness of targeted NGS sequencing using custom-made Ion Torrent and Haloplex (Agilent) panels for diagnostics and research purposes.</p>
<p align="right"><i>score: 126</i></p>

</div><br /><br /><br /><div class="card"><h3>An analysis of the feasibility of short read sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Several methods for ultra high-throughput DNA sequencing are currently under investigation. Many of these methods yield very short blocks of sequence information (reads). Here we report on an analysis showing the level of genome sequencing possible as a function of read length. It is shown that resequencing and de novo sequencing of the majority of a bacterial genome is possible with read lengths of 20-30 nt, and that reads of 50 nt can provide reconstructed contigs (a contiguous fragment of sequence data) of 1000 nt and greater that cover 80% of human chromosome 1.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time, MinION-based, amplicon sequencing for lineage typing of infectious bronchitis virus from upper respiratory samples</h3>
<p style="text-align: justify; text-justify: inter-word;">Recently, third generation sequencing technology has been used for detection of viral nucleotides and sequencing ultra-long DNA molecules. 34 The MinION Nanopore sequencer, a new DNA sequencing technology that allows for rapid, in-house, real-time technique detection and differentiation of IBV lineages, may be cost effective and useful in the field. 19 Ampliconbased sequencing has also been used to amplify specific regions of Newcastle disease 3,13 , infectious laryngotracheitis virus 39 Zika 34 , Ebola 35 and avian influenza viruses 46 , by simple RT-PCR and then sequencing on the MinION device. Real-time data analysis, the lack of significant start-up cost investment or maintenance expenses, simultaneous and sequential multiplexing unique to MinION, and the ability to sequence long DNA molecules so that primers are in conserved regions while the product contains the variable region are the features that make this technology highly feasible to be used for disease diagnostics.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia's first case of COVID-</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated degenerate PCR primer design for high-throughput sequencing improves efficiency of viral sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: Augmenting our previously developed and published JCVI Primer Design Pipeline, we achieved similarly high sequencing success rates with only minor software modifications. The recommended methodology for the construction of the consensus sequence that encapsulates the allelic variation of the targeted population and is a key step prior to designing degenerate primers is also formally described.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Opportunities Revealed for Antimicrobial Stewardship and Clinical Practice with Implementation of a Rapid Respiratory Multiplex Assay</h3>
<p style="text-align: justify; text-justify: inter-word;">Few studies assess the utility of rapid multiplex molecular respiratory panels in adult patients. Previous multiplex PCR assays took hours to days from order time to result. We analyze the clinical impact of switching to a molecular assay with a 3-h test-turnaround-time (TAT). We performed a retrospective review of adult patients who presented to our emergency departments with respiratory symptoms and had a respiratory viral panel (xTAG RVP; RVP) or respiratory pathogen panel (ePlex RP; RPP) within 48 h of presentation. The average TATs for the RVP and RPP were 27.9 and 3.0 h, respectively (P Ͻ 0.0001). In RVP-positive and RPP-positive patients, 68.9 and 44.5% of those with normal chest imaging received antibiotics (P ϭ 0.013), while 95.4 and 89.6% of those with abnormal imaging received antibiotics, respectively (P ϭ 0.187). There was no difference in antibiotic duration in RVPpositive and RPP-positive patients with abnormal chest imaging (6.2 and 6.0 days, respectively; P ϭ 0.923) and normal chest imaging (4.5 and 4.3 days, respectively; P ϭ 0.922). Fewer patients were admitted in the RPP-positive compared to the RVPpositive group (76.9 and 88.6%, respectively; P ϭ 0.013), while the proportion of admissions were similar among RPP-negative and RVP-negative patients (85.3 and 87.1%, P ϭ 0.726). Switching to a multiplex respiratory panel with a clinically actionable TAT is associated with reduced hospital admissions and, in admitted adults without focal radiographic findings, reduced antibiotic initiation. Opportunities to further mitigate inappropriate antibiotic use may be realized by combining rapid multiplex PCR with provider education, clinical decision-care algorithms, and active antibiotic stewardship.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Global Surveillance of Emerging Influenza Virus Genotypes by Mass Spectrometry</h3>
<p style="text-align: justify; text-justify: inter-word;">Background. Effective influenza surveillance requires new methods capable of rapid and inexpensive genomic analysis of evolving viral species for pandemic preparedness, to understand the evolution of circulating viral species, and for vaccine strain selection. We have developed one such approach based on previously described broad-range reverse transcription PCR/ electrospray ionization mass spectrometry (RT-PCR/ESI-MS) technology. Methods and Principal Findings. Analysis of base compositions of RT-PCR amplicons from influenza core gene segments (PB1, PB2, PA, M, NS, NP) are used to provide subspecies identification and infer influenza virus H and N subtypes. Using this approach, we detected and correctly identified 92 mammalian and avian influenza isolates, representing 30 different H and N types, including 29 avian H5N1 isolates. Further, direct analysis of 656 human clinical respiratory specimens collected over a seven-year period (1999)(2000)(2001)(2002)(2003)(2004)(2005)(2006) showed correct identification of the viral species and subtypes with .97% sensitivity and specificity. Base composition derived clusters inferred from this analysis showed 100% concordance to previously established clades. Ongoing surveillance of samples from the recent influenza virus seasons (2005)(2006) showed evidence for emergence and establishment of new genotypes of circulating H3N2 strains worldwide. Mixed viral quasispecies were found in approximately 1% of these recent samples providing a view into viral evolution. Conclusion/Significance. Thus, rapid RT-PCR/ESI-MS analysis can be used to simultaneously identify all species of influenza viruses with clade-level resolution, identify mixed viral populations and monitor global spread and emergence of novel viral genotypes. This high-throughput method promises to become an integral component of influenza surveillance.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Local Evolutionary Patterns of Human Respiratory Syncytial Virus Derived from Whole-Genome Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">RSV molecular epidemiology, evolution, and transmission requires detailed sequence data and can inform new strategies for virus control and vaccine development. We have generated 27 complete or nearly complete genomes of RSV from hospitalized children attending a rural coastal district hospital in Kilifi, Kenya, over a 10-year period using a novel full-genome deep-sequencing process. Phylogenetic analysis of the new genomes demonstrated the existence and cocirculation of multiple genotypes in both RSV A and B groups in Kilifi. Comparison of local versus global strains demonstrated that most RSV A variants observed locally in Kilifi were also seen in other parts of the world, while the Kilifi RSV B genomes encoded a high degree of variation that was not observed in other parts of the world. The nucleotide substitution rates for the individual open reading frames (ORFs) were highest in the regions encoding the attachment (G) glycoprotein and the NS2 protein. The analysis of RSV full genomes, compared to subgenomic regions, provided more precise estimates of the RSV sequence changes and revealed important patterns of RSV genomic variation and global movement. The novel sequencing method and the new RSV genomic sequences reported here expand our knowledge base for large-scale RSV epidemiological and transmission studies.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Genomics Viral genome sequencing by random priming methods</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Most emerging health threats are of zoonotic origin. For the overwhelming majority, their causative agents are RNA viruses which include but are not limited to HIV, Influenza, SARS, Ebola, Dengue, and Hantavirus. Of increasing importance therefore is a better understanding of global viral diversity to enable better surveillance and prediction of pandemic threats; this will require rapid and flexible methods for complete viral genome sequencing.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time, MinION-based, amplicon sequencing for lineage typing of infectious bronchitis virus from upper respiratory samples</h3>
<p style="text-align: justify; text-justify: inter-word;">Accurate identification of IBV genotypes from samples, including detection of multiple isolates from a single sample, is crucial to respiratory disease diagnosis, selection of appropriate IBV vaccines, and epidemiologic studies. Therefore, the aim of the current study is to create a single, amplicon-based protocol to sequence IBV-S1 gene and develop a sequence analysis workflow to identify IBV isolates from clinical swab samples. This method provides a useful assay for IBV and a model for the development of future amplicon sequencing (AmpSeq) based assays.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>VIGOR, an annotation program for small viral genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: The decrease in cost for sequencing and improvement in technologies has made it easier and more common for the re-sequencing of large genomes as well as parallel sequencing of small genomes. It is possible to completely sequence a small genome within days and this increases the number of publicly available genomes. Among the types of genomes being rapidly sequenced are those of microbial and viral genomes responsible for infectious diseases. However, accurate gene prediction is a challenge that persists for decoding a newly sequenced genome. Therefore, accurate and efficient gene prediction programs are highly desired for rapid and cost effective surveillance of RNA viruses through full genome sequencing. Results: We have developed VIGOR (Viral Genome ORF Reader), a web application tool for gene prediction in influenza virus, rotavirus, rhinovirus and coronavirus subtypes. VIGOR detects protein coding regions based on sequence similarity searches and can accurately detect genome specific features such as frame shifts, overlapping genes, embedded genes, and can predict mature peptides within the context of a single polypeptide open reading frame. Genotyping capability for influenza and rotavirus is built into the program. We compared VIGOR to previously described gene prediction programs, ZCURVE_V, GeneMarkS and FLAN. The specificity and sensitivity of VIGOR are greater than 99% for the RNA viral genomes tested. Conclusions: VIGOR is a user friendly web-based genome annotation program for five different viral agents, influenza, rotavirus, rhinovirus, coronavirus and SARS coronavirus. This is the first gene prediction program for rotavirus and rhinovirus for public access. VIGOR is able to accurately predict protein coding genes for the above five viral types and has the capability to assign function to the predicted open reading frames and genotype influenza virus. The prediction software was designed for performing high throughput annotation and closure validation in a post-sequencing production pipeline.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid detection of respiratory organisms with the FilmArray respiratory panel in a large children's hospital in China</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: FilmArray RP allows the rapid simultaneous detection of a wide number of respiratory organisms, with limited hands-on time, in Chinese pediatric patients with RTIs.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid, Unbiased PRRSV Strain Detection Using MinION Direct RNA Sequencing and Bioinformatics Tools</h3>
<p style="text-align: justify; text-justify: inter-word;">Prompt detection and effective control of porcine reproductive and respiratory syndrome virus (PRRSV) during outbreaks is important given its immense adverse impact on the swine industry. However, the diagnostic process can be challenging due to the high genetic diversity and high mutation rate of PRRSV. A diagnostic method that can provide more detailed genetic information about pathogens is urgently needed. In this study, we evaluated the ability of Oxford Nanopore MinION direct RNA sequencing to generate a PRRSV whole genome sequence and detect and discriminate virus at the strain-level. A nearly full length PRRSV genome was successfully generated from raw sequence reads, achieving an accuracy of 96% after consensus genome generation. Direct RNA sequencing reliably detected the PRRSV strain present with an accuracy of 99.9% using as few as 5 raw sequencing reads and successfully differentiated multiple co-infecting strains present in a sample. In addition, PRRSV strain information was obtained from clinical samples containing 10 4 to 10 6 viral copies or more within 6 hours of sequencing. Overall, direct viral RNA sequencing followed by bioinformatic analysis proves to be a promising approach for identification of the viral strain or strains involved in clinical infections, allowing for more precise prevention and control strategies during PRRSV outbreaks.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>ANDES: Statistical tools for the ANalyses of DEep Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">As new sequencing technologies evolve, deep sequencing will become increasingly cost-efficient and the inter and intra-sample comparisons of largely homogeneous sequences will become more common. We have provided a software package and demonstrated its application on various empirically-derived datasets. Investigators may download the software from Sourceforge at https://sourceforge.net/projects/andestools.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Distributed under Creative Commons CC-BY 4.0 Respiratory syncytial virus, human metapneumovirus, and influenza virus infection in Bangkok, 2016-2017</h3>
<p style="text-align: justify; text-justify: inter-word;">Children and adults residing in densely populated urban centers around the world are at risk of seasonal influenza-like illness caused by respiratory viruses such as influenza virus, human metapneumovirus (hMPV), and respiratory syncytial virus (RSV). In a large metropolitan of Thailand's capital city Bangkok, most respiratory infections are rarely confirmed by molecular diagnostics. We therefore examined the frequency of RSV, hMPV, and influenza virus in 8,842 patients who presented influenza-like illness and sought medical care at a large hospital in Bangkok between 2016 and 2017. Using a multiplex real-time reverse-transcription polymerase chain reaction (RT-PCR), 30.5% (2,699/8,842) of nasopharyngeal (NP) swab samples tested positive for one or more of these viruses. Influenza virus comprised 17.3% (1,528/8,842), of which the majority were influenza A/H3N2. Such infection was most prevalent among adults and the elderly. RSV was identified in 11.4% (1,011/8,842) and were mostly ON1 and BA9 genotypes. Of the hMPV-positive samples (3.6%, 318/8,842), genotypes A2, B1, and B2 were detected. A small number of individuals experienced co-infections (1.8%, 155/8,842), most commonly between RSV and influenza A/H3N2. RSV and hMPV co-infections were also found, but mainly in young children. Viral respiratory tract infection peaked locally in the rainy season (June to September). These findings support the utility of rapid nucleic acid testing of RSV, hMPV, and influenza virus in patients with ILI. Agapov E, Sumino KC, Gaudreault-Keener M, Storch GA, Holtzman MJ. 2006. Genetic variability of human metapneumovirus infection: evidence of a shift in viral genotype without a change in illness.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated degenerate PCR primer design for high-throughput sequencing improves efficiency of viral sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: In a high-throughput environment, to PCR amplify and sequence a large set of viral isolates from populations that are potentially heterogeneous and continuously evolving, the use of degenerate PCR primers is an important strategy. Degenerate primers allow for the PCR amplification of a wider range of viral isolates with only one set of pre-mixed primers, thus increasing amplification success rates and minimizing the necessity for genome finishing activities. To successfully select a large set of degenerate PCR primers necessary to tile across an entire viral genome and maximize their success, this process is best performed computationally. Results: We have developed a fully automated degenerate PCR primer design system that plays a key role in the J. Craig Venter Institute's (JCVI) high-throughput viral sequencing pipeline. A consensus viral genome, or a set of consensus segment sequences in the case of a segmented virus, is specified using IUPAC ambiguity codes in the consensus template sequence to represent the allelic diversity of the target population. PCR primer pairs are then selected computationally to produce a minimal amplicon set capable of tiling across the full length of the specified target region. As part of the tiling process, primer pairs are computationally screened to meet the criteria for successful PCR with one of two described amplification protocols. The actual sequencing success rates for designed primers for measles virus, mumps virus, human parainfluenza virus 1 and 3, human respiratory syncytial virus A and B and human metapneumovirus are described, where >90% of designed primer pairs were able to consistently successfully amplify >75% of the isolates.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Real-time, MinION-based, amplicon sequencing for lineage typing of infectious bronchitis virus from upper respiratory samples</h3>
<p style="text-align: justify; text-justify: inter-word;">The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/634600 doi: bioRxiv preprint IBV types. As such, the development of a pan-IBV sequencing method to rapidly determine the genotype(s) in samples would aid in lineage typing and tracking of IBV genotypes circulating in poultry flocks.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /></body>