
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: a mechanism for coordinating the rapid dissemination of genome tracking information</h2><br /></div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<p style="text-align: justify; text-justify: inter-word;">Comprehensive and uniform diagnostic test data will aid in the identification of potential outbreaks. A combination of broad respiratory pathogen testing and an internal electronic surveillance system enabled the rapid dissemination of data across the largest health care system 80 in New York, the North Shore-LIJ Health System (now Northwell Health) during the influenza A H1N1-2009 pandemic in the New York City area. Pathogen-specific molecular testing permitted rapid a) notification to state epidemiologists; b) tracking of the virus so that health care resources could be managed effectively; and c) evaluation of influenza diagnostics [27, 28] .</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Standards for Sequencing Viral Genomes in the Era of High- Throughput Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Thanks to high-throughput sequencing technologies, genome sequencing has become a common component in nearly all aspects of viral research; thus, we are experiencing an explosion in both the number of available genome sequences and the number of institutions producing such data. However, there are currently no common standards used to convey the quality, and therefore utility, of these various genome sequences. Here, we propose five "standard" categories that encompass all stages of viral genome finishing, and we define them using simple criteria that are agnostic to the technology used for sequencing. We also provide genome finishing recommendations for various downstream applications, keeping in mind the cost-benefit tradeoffs associated with different levels of finishing. Our goal is to define a common vocabulary that will allow comparison of genome quality across different research groups, sequencing platforms, and assembly techniques.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>Make Data Sharing Routine to Prepare for Public Health Emergencies</h3>
<p style="text-align: justify; text-justify: inter-word;">In February 2016, Wellcome Trust organized a pledge among leading scientific organizations and health agencies encouraging researchers to release data relevant to the Zika outbreak as rapidly and widely as possible [1] . This initiative echoed a September 2015 World Health Organization (WHO) consultation that assessed data sharing during the recent West Africa Ebola outbreak and called on researchers to make data publicly available during public health emergencies [2]. These statements were necessary because the traditional way of communicating research results-publication in peer-reviewed journals, often months or years after data collection-is too slow during an emergency.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>SNAD: sequence name annotation-based designer</h3>
<p style="text-align: justify; text-justify: inter-word;">A tool for controllable annotation-based conversion of sequence UIDs into biologically meaningful names and acronyms has been developed and placed into service, fostering links between quality of sequence annotation, and efficiency of communication and knowledge dissemination among researchers.</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /><div class="card"><h3>2 Division of Global Health Protection</h3>
<p style="text-align: justify; text-justify: inter-word;">The metagenomic surveillance approach for monitoring circulating pathogens in sewage was able to detect putative pathogen and resistance loads in an urban informal settlement. Thus, valuable if generated in real time to serve as a comprehensive infectious disease agent surveillance system with the potential to guide disease prevention and treatment. The approach may lead to a paradigm shift in conducting real-time global genomics-based surveillance in settings with limited access to health care.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Make Data Sharing Routine to Prepare for Public Health Emergencies</h3>
<p style="text-align: justify; text-justify: inter-word;">• The recent outbreaks caused by Ebola and Zika viruses highlighted the importance of medical and public health research in accelerating outbreak control and prompted calls for researchers to share data rapidly and widely during public health emergencies.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated analysis of phylogenetic clusters Automated analysis of phylogenetic clusters on behalf of the UK HIV Drug Resistance Database</h3>
<p style="text-align: justify; text-justify: inter-word;">The Cluster Picker and Cluster Matcher can rapidly process phylogenetic trees containing tens of thousands of sequences. Together these tools will facilitate comparisons of pathogen transmission dynamics between studies and countries.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</h3>
<p style="text-align: justify; text-justify: inter-word;">Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the "coronavirus" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.</p>
<p align="right"><i>score: 54</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated vocabulary discovery for geo-parsing online epidemic intelligence</h3>
<p style="text-align: justify; text-justify: inter-word;">The results of this analysis provide a framework for future automated global surveillance efforts that reduce manual input and improve timeliness of reporting.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Public Health Distributed data processing for public health surveillance</h3>
<p style="text-align: justify; text-justify: inter-word;">For well-defined surveillance requirements, the model can be successfully deployed with very low risk of inadvertent disclosure of PHI -a feature that may make participation in surveillance systems more feasible for organizations and more appealing to the individuals whose PHI they hold. It is possible to design and implement distributed systems to support non-routine public health needs if required.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>SEED Servers: High-Performance Access to the SEED Genomes, Annotations, and Metabolic Models</h3>
<p style="text-align: justify; text-justify: inter-word;">The remarkable advance in sequencing technology and the rising interest in medical and environmental microbiology, biotechnology, and synthetic biology resulted in a deluge of published microbial genomes. Yet, genome annotation, comparison, and modeling remain a major bottleneck to the translation of sequence information into biological knowledge, hence computational analysis tools are continuously being developed for rapid genome annotation and interpretation. Among the earliest, most comprehensive resources for prokaryotic genome analysis, the SEED project, initiated in 2003 as an integration of genomic data and analysis tools, now contains .5,000 complete genomes, a constantly updated set of curated annotations embodied in a large and growing collection of encoded subsystems, a derived set of protein families, and hundreds of genome-scale metabolic models. Until recently, however, maintaining current copies of the SEED code and data at remote locations has been a pressing issue. To allow high-performance remote access to the SEED database, we developed the SEED Servers (http://www.theseed.org/servers): four network-based servers intended to expose the data in the underlying relational database, support basic annotation services, offer programmatic access to the capabilities of the RAST annotation server, and provide access to a growing collection of metabolic models that support flux balance analysis. The SEED servers offer open access to regularly updated data, the ability to annotate prokaryotic genomes, the ability to create metabolic reconstructions and detailed models of metabolism, and access to hundreds of existing metabolic models. This work offers and supports a framework upon which other groups can build independent research efforts. Large integrations of genomic data represent one of the major intellectual resources driving research in biology, and programmatic access to the SEED data will provide significant utility to a broad collection of potential users.</p>
<p align="right"><i>score: 53</i></p>

</div><br /><br /><br /><div class="card"><h3>GenomeFingerprinter: The Genome Fingerprint and the Universal Genome Fingerprint Analysis for Systematic Comparative Genomics</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: First, we create a method, GenomeFingerprinter, to unambiguously produce a set of three-dimensional coordinates from a sequence, followed by one three-dimensional plot and six two-dimensional trajectory projections, to illustrate the genome fingerprint of a given genome sequence. Second, we develop a set of concepts and tools, and thereby establish a method called the universal genome fingerprint analysis (UGFA). Particularly, we define the total genetic component configuration (TGCC) (including chromosome, plasmid, and phage) for describing a strain as a systematic unit, the universal genome fingerprint map (UGFM) of TGCC for differentiating strains as a universal system, and the systematic comparative genomics (SCG) for comparing a set of genomes crossing genetic components and biological categories. Third, we construct a method of quantitative analysis to compare two genomes by using the outcome dataset of genome fingerprint analysis. Specifically, we define the geometric center and its geometric mean for a given genome fingerprint map, followed by the Euclidean distance, the differentiate rate, and the weighted differentiate rate to quantitatively describe the difference between two genomes of comparison. Moreover, we demonstrate the applications through case studies on various genome sequences, giving tremendous insights into the critical issues in microbial genomics and taxonomy.</p>
<p align="right"><i>score: 52</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome3D: A viewer-model framework for integrating and visualizing multi-scale epigenomic information within a three-dimensional genome</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: New technologies are enabling the measurement of many types of genomic and epigenomic information at scales ranging from the atomic to nuclear. Much of this new data is increasingly structural in nature, and is often difficult to coordinate with other data sets. There is a legitimate need for integrating and visualizing these disparate data sets to reveal structural relationships not apparent when looking at these data in isolation. Results: We have applied object-oriented technology to develop a downloadable visualization tool, Genome3D, for integrating and displaying epigenomic data within a prescribed three-dimensional physical model of the human genome. In order to integrate and visualize large volume of data, novel statistical and mathematical approaches have been developed to reduce the size of the data. To our knowledge, this is the first such tool developed that can visualize human genome in three-dimension. We describe here the major features of Genome3D and discuss our multi-scale data framework using a representative basic physical model. We then demonstrate many of the issues and benefits of multi-resolution data integration. Conclusions: Genome3D is a software visualization tool that explores a wide range of structural genomic and epigenetic data. Data from various sources of differing scales can be integrated within a hierarchical framework that is easily adapted to new developments concerning the structure of the physical genome. In addition, our tool has a simple annotation mechanism to incorporate non-structural information. Genome3D is unique is its ability to manipulate large amounts of multi-resolution data from diverse sources to uncover complex and new structural relationships within the genome.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">A rigorous approach to facilitate and guarantee the correctness of the genetic testing management in human genome information systems Abstract Background: Recent medical and biological technology advances have stimulated the development of new testing systems that have been providing huge, varied amounts of molecular and clinical data. Growing data volumes pose significant challenges for information processing systems in research centers. Additionally, the routines of genomics laboratory are typically characterized by high parallelism in testing and constant procedure changes. Results: This paper describes a formal approach to address this challenge through the implementation of a genetic testing management system applied to human genome laboratory. We introduced the Human Genome Research Center Information System (CEGH) in Brazil, a system that is able to support constant changes in human genome testing and can provide patients updated results based on the most recent and validated genetic knowledge. Our approach uses a common repository for process planning to ensure reusability, specification, instantiation, monitoring, and execution of processes, which are defined using a relational database and rigorous control flow specifications based on process algebra (ACP). The main difference between our approach and related works is that we were able to join two important aspects: 1) process scalability achieved through relational database implementation, and 2) correctness of processes using process algebra. Furthermore, the software allows end users to define genetic testing without requiring any knowledge about business process notation or process algebra.</p>
<p align="right"><i>score: 51</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Preprints: An underutilized mechanism to accelerate outbreak science</h3>
<p style="text-align: justify; text-justify: inter-word;">• Preprints-manuscripts posted openly online prior to peer review-offer an opportunity to accelerate the dissemination of scientific findings to support responses to infectious disease outbreaks.</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</h3>
<p style="text-align: justify; text-justify: inter-word;">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia's first case of COVID-</p>
<p align="right"><i>score: 50</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficient oligonucleotide probe selection for pan-genomic tiling arrays</h3>
<p style="text-align: justify; text-justify: inter-word;">PanArray is an oligonucleotide probe selection algorithm for tiling multiple genome sequences using a minimal number of probes. It is capable of fully tiling all genomes of a species on a single microarray chip. These unique pan-genome tiling arrays provide maximum flexibility for the analysis of both known and uncharacterized strains.</p>
<p align="right"><i>score: 49</i></p>

</div><br /><br /><br /></body>