
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: design and execution of targeted surveillance experiments</h2><br /></div><br /><br /><br /><div class="card"><h3>Use of Highly Pathogenic Avian Influenza A(H5N1) Gain-Of-Function Studies for Molecular-Based Surveillance and Pandemic Preparedness</h3>
<p style="text-align: justify; text-justify: inter-word;">. Use of highly pathogenic avian influenza A(H5N1) gain-of-function studies for molecular-based surveillance and pandemic preparedness. mBio 5(6):e02431-14.</p>
<p align="right"><i>score: 133</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of Bat Coronaviruses through Surveillance and Probe Capture-Based Next-Generation Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronaviruses (CoVs) of bat origin have caused two pandemics in this century. Severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV both originated from bats, and it is highly likely that bat coronaviruses will cause future outbreaks. Active surveillance is both urgent and essential to predict and mitigate the emergence of these viruses in humans. Next-generation sequencing (NGS) is currently the preferred methodology for virus discovery to ensure unbiased sequencing of bat CoVs, considering their high genetic diversity. However, unbiased NGS is an expensive methodology and is prone to missing low-abundance CoV sequences due to the high background level of nonviral sequences present in surveillance field samples. Here, we employ a capture-based NGS approach using baits targeting most of the CoV species. Using this technology, we effectively reduced sequencing costs by increasing the sensitivity of detection. We discovered nine full genomes of bat CoVs in this study and revealed great genetic diversity for eight of them. IMPORTANCE Active surveillance is both urgent and essential to predict and mitigate the emergence of bat-origin CoV in humans and livestock. However, great genetic diversity increases the chance of homologous recombination among CoVs. Performing targeted PCR, a common practice for many surveillance studies, would not reflect this diversity. NGS, on the other hand, is an expensive methodology and is prone to missing low-abundance CoV sequences. Here, we employ a capture-based NGS approach using baits targeting all CoVs. Our work demonstrates that targeted, cost-effective, large-scale, genome-level surveillance of bat CoVs is now highly feasible. 2020. Discovery of bat coronaviruses through surveillance and probe capture-based next-generation sequencing. mSphere 5:e00807-19. https://doi .org/10.1128/mSphere.</p>
<p align="right"><i>score: 116</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral gain-of-function experiments uncover residues under diversifying selection in nature</h3>
<p style="text-align: justify; text-justify: inter-word;">Our results provide evidence for widespread host-shift evolution in nature and a proof of concept for integrating experiments with genomic epidemiology.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Genomic Characterization of a Novel Bat Sapovirus with Unusual Genomic Features and Phylogenetic Position</h3>
<p style="text-align: justify; text-justify: inter-word;">Sapovirus is a genus of caliciviruses that are known to cause enteric disease in humans and animals. There is considerable genetic diversity among the sapoviruses, which are classified into different genogroups based on phylogenetic analysis of the full-length capsid protein sequence. While several mammalian species, including humans, pigs, minks, and dogs, have been identified as animal hosts for sapoviruses, there were no reports of sapoviruses in bats in spite of their biological diversity. In this report, we present the results of a targeted surveillance study in different bat species in Hong Kong. Five of the 321 specimens from the bat species, Hipposideros pomona, were found to be positive for sapoviruses by RT-PCR. Complete or nearly full-length genome sequences of approximately 7.7 kb in length were obtained for three strains, which showed similar organization of the genome compared to other sapoviruses. Interestingly, they possess many genomic features atypical of most sapoviruses, like high G+C content and minimal CpG suppression. Phylogenetic analysis of the viral proteins suggested that the bat sapovirus descended from an ancestral sapovirus lineage and is most closely related to the porcine sapoviruses. Codon usage analysis showed that the bat sapovirus genome has greater codon usage bias relative to other sapovirus genomes. In summary, we report the discovery and genomic characterization of the first bat calicivirus, which appears to have evolved under different conditions after early divergence from other sapovirus lineages.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated degenerate PCR primer design for high-throughput sequencing improves efficiency of viral sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions: Augmenting our previously developed and published JCVI Primer Design Pipeline, we achieved similarly high sequencing success rates with only minor software modifications. The recommended methodology for the construction of the consensus sequence that encapsulates the allelic variation of the targeted population and is a key step prior to designing degenerate primers is also formally described.</p>
<p align="right"><i>score: 113</i></p>

</div><br /><br /><br /><div class="card"><h3>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</h3>
<p style="text-align: justify; text-justify: inter-word;">The emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/ .</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>Vorpal: A novel RNA virus feature- 1 extraction algorithm demonstrated 2 through interpretable genotype-to- 3 phenotype linear models 4 5</h3>
<p style="text-align: justify; text-justify: inter-word;">words" methods with the complications presented by the intrinsic variance of RNA virus 24 space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects 25 an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the 26 construction of interpretable linear models with clustered, representative 'degenerate' K-27 mers as the input vector and, through regularization, sparse predictors of binary 28 phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying 29 nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA 30 virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, 31 hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-32 host vs. non-human host in Influenza A. The capacity of this approach for in silico 33 identification of hypotheses which can be validated by direct experimentation, as well as 34 identification of genomic targets for preemptive biosurveillance of emerging viruses, is 35 discussed. The code is available for download at https://github.com/mriglobal/vorpal. 36 37 Feature Extraction Algorithm Overview 38</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct nucleic acid analysis of mosquitoes for high fidelity species identification and 1 detection of Wolbachia using a cellphone 2 3</h3>
<p style="text-align: justify; text-justify: inter-word;">Our easy to use and easy to interpret assays should facilitate widespread field mosquito 63 surveillance with minimal instrumentation and high accuracy. 64 65 66 All rights reserved. No reuse allowed without permission.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>Data Descriptor: Epidemiological characteristics of human-infective RNA viruses Background and Summary</h3>
<p style="text-align: justify; text-justify: inter-word;">Design Type(s) data integration objective • epidemiological study</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Assessing the use of mobile phone data to describe recurrent mobility patterns in spatial epidemic models</h3>
<p style="text-align: justify; text-justify: inter-word;">Biology (whole organism) Subject Areas: computational biology/health and disease and epidemiology/computer modelling and simulation</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral factors in influenza pandemic risk assessment</h3>
<p style="text-align: justify; text-justify: inter-word;">The threat of an influenza A virus pandemic stems from continual virus spillovers from reservoir species, a tiny fraction of which spark sustained transmission in humans. To date, no pandemic emergence of a new influenza strain has been preceded by detection of a closely related precursor in an animal or human. Nonetheless, influenza surveillance efforts are expanding, prompting a need for tools to assess the pandemic risk posed by a detected virus. The goal would be to use genetic sequence and/or biological assays of viral traits to identify those non-human influenza viruses with the greatest risk of evolving into pandemic threats, and/or to understand drivers of such evolution, to prioritize pandemic prevention or response measures. We describe such efforts, identify progress and ongoing challenges, and discuss three specific traits of influenza viruses (hemagglutinin receptor binding specificity, hemagglutinin pH of activation, and polymerase complex efficiency) that contribute to pandemic risk.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<p style="text-align: justify; text-justify: inter-word;">Citation Chan JM, Rabadan R. 2013. Quantifying pathogen surveillance using temporal genomic data. mBio 4(1):e00524-12.</p>
<p align="right"><i>score: 100</i></p>

</div><br /><br /><br /><div class="card"><h3>Adaptive Management and the Value of Information: Learning Via Intervention in Epidemiology</h3>
<p style="text-align: justify; text-justify: inter-word;">Optimal intervention for disease outbreaks is often impeded by severe scientific uncertainty. Adaptive management (AM), long-used in natural resource management, is a structured decision-making approach to solving dynamic problems that accounts for the value of resolving uncertainty via real-time evaluation of alternative models. We propose an AM approach to design and evaluate intervention strategies in epidemiology, using real-time surveillance to resolve model uncertainty as management proceeds, with foot-and-mouth disease (FMD) culling and measles vaccination as case studies. We use simulations of alternative intervention strategies under competing models to quantify the effect of model uncertainty on decision making, in terms of the value of information, and quantify the benefit of adaptive versus static intervention strategies. Culling decisions during the 2001 UK FMD outbreak were contentious due to uncertainty about the spatial scale of transmission. The expected benefit of resolving this uncertainty prior to a new outbreak on a UK-like landscape would be £45-£60 million relative to the strategy that minimizes livestock losses averaged over alternate transmission models. AM during the outbreak would be expected to recover up to £20.1 million of this expected benefit. AM would also recommend a more conservative initial approach (culling of infected premises and dangerous contact farms) than would a fixed strategy (which would additionally require culling of contiguous premises). For optimal targeting of measles vaccination, based on an outbreak in Malawi in 2010, AM allows better distribution of resources across the affected region; its utility depends on uncertainty about both the at-risk population and logistical capacity. When daily vaccination rates are highly constrained, the optimal initial strategy is to conduct a small, quick campaign; a reduction in expected burden of approximately 10,000 cases could result if campaign targets can be updated on the basis of the true susceptible population. Formal incorporation of a policy to update future management actions in response to information gained in the course of an outbreak can change the optimal initial response and result in significant cost savings. AM provides a framework for using multiple models to facilitate public-health decision making and an objective basis for updating management actions in response to improved scientific understanding.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Combining field epidemiological information and genetic data to comprehensively reconstruct the invasion history and the microevolution of the sudden oak death agent Phytophthora ramorum (Stramenopila: Oomycetes) in California</h3>
<p style="text-align: justify; text-justify: inter-word;">Keywords Emergent forest disease Á Genetic epidemiology Á Microsatellite Á Coalescent analysis Á Network analysis Peter J. P. Croucher and Silvia Mascheretti contributed equally to this paper.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid Identification of Emerging Pathogens: Coronavirus Rapid Identification of Emerging Pathogens</h3>
<p style="text-align: justify; text-justify: inter-word;">We describe a new approach for infectious disease surveillance that facilitates rapid identification of known and emerging pathogens. The process uses broad-range polymerase chain reaction (PCR) to amplify nucleic acid targets from large groupings of organisms, electrospray ionization mass spectrometry for accurate mass measurements of PCR products, and base composition signature analysis to identify organisms in a sample. We demonstrate this principle by using 14 isolates of 9 diverse Coronavirus spp., including the severe acute respiratory syndrome-associated coronavirus (SARS-CoV). We show that this method could identify and distinguish between SARS and other known CoV, including the human CoV 229E and OC43, individually and in a mixture of all 3 human viruses. The sensitivity of detection, measured by using titered SARS-CoV spiked into human serum, was ≈1 PFU/mL. This approach, applicable to the surveillance of bacterial, viral, fungal, or protozoal pathogens, is capable of automated analysis of >900 PCR reactions per day.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantifying Pathogen Surveillance Using Temporal Genomic Data</h3>
<p style="text-align: justify; text-justify: inter-word;">With the advent of deep sequencing, genomic surveillance has become a popular method for detection of infectious disease, supplementing information gathered by classic clinical or serological techniques to identify host-determinant markers and trace the origin of transmission. However, two main factors complicate genomic surveillance. First, pathogens exhibiting high genetic diversity demand higher levels of scrutiny to obtain an accurate representation of the entire population. Second, current systems of detection are nonuniform, with significant gaps in certain geographic locations and animal reservoirs. Despite past unforeseen pandemics like the 2009 swine-origin H1N1 influenza virus, there is no standardized way of evaluating surveillance. A more complete surveillance system should capture a greater proportion of pathogen diversity. Here we present a novel quantitative method of assessing the completeness of genomic surveillance that incorporates the time of sequence collection, as well as the pathogen's evolutionary rate. We propose the q2 coefficient, which measures the proportion of sequenced isolates whose closest neighbor in the past is within a genetic distance equivalent to 2 years of evolution, roughly the median time of changing strain selection for influenza A vaccines. Easily interpretable and significantly faster than other methods, the q2 coefficient requires no full phylogenetic characterization or use of arbitrary clade definitions. Application of the q2 coefficient to influenza A virus confirmed poor sampling of swine and avian populations and identified regions with deficient surveillance. We demonstrate that the q2 coefficient can not only be applied to other pathogens, including dengue and West Nile viruses, but also used to describe surveillance dynamics, particularly the effects of different public health policies. IMPORTANCE Surveillance programs have become key assets in determining the emergence or prevalence of pathogens circulating in human and animal populations. Genomic surveillance, in particular, provides comprehensive information on the history of isolates and potential molecular markers for infectivity and pathogenicity. Current techniques for evaluating genomic surveillance are inaccurate, ignoring the pathogen's evolutionary rate and biodiversity, as well as the timing of sequence collection. Using sequence data, we propose the q2 coefficient as a quantitative measure of surveillance completeness that combines elements of time and evolution without defining arbitrary criteria for clades or species. Through several case studies of influenza A, dengue, and West Nile viruses, we employed the q2 coefficient to identify sampling deficiencies in different host species and locations, as well as examine the effects of different public health policies through historical records of the q2 coefficient. These results can guide public health agencies to focus resource allocation and virus collection to bolster specific problems in surveillance.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences, and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and identify touching points and temporal pathways that are candidates for deeper investigation by geneticists and epidemiologists.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Public Health Distributed data processing for public health surveillance</h3>
<p style="text-align: justify; text-justify: inter-word;">For well-defined surveillance requirements, the model can be successfully deployed with very low risk of inadvertent disclosure of PHI -a feature that may make participation in surveillance systems more feasible for organizations and more appealing to the individuals whose PHI they hold. It is possible to design and implement distributed systems to support non-routine public health needs if required.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery of DNA Viruses in Wild-Caught Mosquitoes Using Small RNA High throughput Sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Mosquito-borne infectious diseases pose a severe threat to public health in many areas of the world. Current methods for pathogen detection and surveillance are usually dependent on prior knowledge of the etiologic agents involved. Hence, efficient approaches are required for screening wild mosquito populations to detect known and unknown pathogens.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment</h3>
<p style="text-align: justify; text-justify: inter-word;">Preparedness for a possible global pandemic caused by viruses such as the highly pathogenic influenza A subtype H5N1 has become a global priority. In particular, it is critical to monitor the appearance of any new emerging subtypes. Comparative phyloinformatics can be used to monitor, analyze, and possibly predict the evolution of viruses. However, in order to utilize the full functionality of available analysis packages for largescale phyloinformatics studies, a team of computer scientists, biostatisticians and virologists is needed -a requirement which cannot be fulfilled in many cases. Furthermore, the time complexities of many algorithms involved leads to prohibitive runtimes on sequential computer platforms. This has so far hindered the use of comparative phyloinformatics as a commonly applied tool in this area.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /></body>