
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: real-time tracking of whole genomes</h2><br /></div><br /><br /><br /><div class="card"><h3>Private collection: high correlation of sample collection and patient admission date in clinical microbiological testing complicates sharing of phylodynamic metadata</h3>
<p style="text-align: justify; text-justify: inter-word;">Infectious pathogens are known for their rapid evolutionary rates with new mutations arising over days to weeks. The ability to rapidly recover whole genome sequences and analyze the spread and evolution of pathogens using genetic information and pathogen collection dates has lead to interest in real-time tracking of infectious transmission and outbreaks. However, the level of temporal resolution afforded by these analyses may conflict with definitions of what constitutes protected health information (PHI) and privacy requirements for de-identification for publication and public sharing of research data and metadata. In the United States, dates and locations associated with patient care that provide greater resolution than year or the first three digits of the zip code are generally considered patient identifiers. Admission and discharge dates are specifically named as identifiers in Department of Health and Human Services guidance. To understand the degree to which one can impute admission dates from specimen collection dates, we examined sample collection dates and patient admission dates associated with more than 270,000 unique microbiological results from the University of Washington Laboratory Medicine Department between 2010 and 2017. Across all positive microbiological tests, the sample collection date exactly matched the patient admission date in 68.8% of tests. Collection dates and admission dates were identical from emergency department and outpatient testing 86.7% and 96.5% of the time, respectively, with >99% of tests collected within 1 day from the patient admission date. Samples from female patients were significantly more likely to be collected closer to admission date that those from male patients. We show that PHI-associated dates such as admission date can confidently be imputed from deposited collection date. We suggest that publicly depositing microbiological collection dates at greater resolution than the year may not meet routine Safe Harbor-based requirements for patient de-identification. We recommend the use of Expert Determination to determine PHI for a given study and/or direct patient consent if clinical laboratories or phylodynamic practitioners desire to make these data available.</p>
<p align="right"><i>score: 162</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for SARS-CoV-2. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines to stop the COVID-19 disease.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>Title: Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes Short title: Automated tool for phylogenetic and mutational analysis of coronaviruses genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.</p>
<p align="right"><i>score: 151</i></p>

</div><br /><br /><br /><div class="card"><h3>Spatiotemporal dynamics of HSV genome nuclear entry and compaction state transitions using bioorthogonal chemistry and super-resolution microscopy</h3>
<p style="text-align: justify; text-justify: inter-word;">Virtually all DNA virus classes as well as many RNA viruses must deposit their genomes within the nucleus for transcription, genome replication and subsequent capsid assembly. While infecting capsids have been studied by various methods and biochemical approaches have been used to investigate the bulk genome population characteristics, quantitative spatiotemporal information of the infecting genome itself at the single particle level has been lacking. This is required for any complete understanding of many critical aspects of virus infection and virus pathogenesis. Using novel techniques in bioorthogonal chemistry to produce normal non-recombinant viruses with readily traceable genomes, we provide the first direct quantitative spatiotemporal analysis of HSV genome transport and presentation to the cellular environment. Using these techniques which discriminate encapsidated from uncoated genomes and input from replicated DNA, our work provides a comprehensive analysis, using direct measures for genome detection not dependant on surrogate outputs. The results reveal completely novel aspects of early genome localisation and organisation not previously appreciated or amenable to study. Furthermore the work also provides a roadmap for similar studies in other systems and for future analysis of many aspects in different fields of the biology of infecting virus genomes early during cell infection.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Spatiotemporal dynamics of HSV genome nuclear entry and compaction state transitions using bioorthogonal chemistry and super-resolution microscopy</h3>
<p style="text-align: justify; text-justify: inter-word;">We investigated the spatiotemporal dynamics of HSV genome transport during the initiation of infection using viruses containing bioorthogonal traceable precursors incorporated into their genomes (HSV EdC ). In vitro assays revealed a structural alteration in the capsid induced upon HSV EdC binding to solid supports that allowed coupling to external capture agents and demonstrated that the vast majority of individual virions contained bioorthogonally-tagged genomes. Using HSV EdC in vivo we reveal novel aspects of the kinetics, localisation, mechanistic entry requirements and morphological transitions of infecting genomes. Uncoating and nuclear import was observed within 30 min, with genomes in a defined compaction state (ca. 3-fold volume increase from capsids). Free cytosolic uncoated genomes were infrequent (7-10% of the total uncoated genomes), likely a consequence of subpopulations of cells receiving high particle numbers. Uncoated nuclear genomes underwent temporal transitions in condensation state and while ICP4 efficiently associated with condensed foci of initial infecting genomes, this relationship switched away from residual longer lived condensed foci to increasingly decondensed genomes as infection progressed. Inhibition of transcription had no effect on nuclear entry but in the absence of transcription, genomes persisted as tightly condensed foci. Ongoing transcription, in the absence of protein synthesis, revealed a distinct spatial clustering of genomes, which we have termed genome congregation, not seen with non-transcribing genomes. Genomes expanded to more decondensed forms in the absence of DNA replication indicating additional transitional steps. During full progression of infection, genomes decondensed further, with a diffuse low intensity signal dissipated within replication compartments, but frequently with tight foci remaining peripherally, representing unreplicated genomes or condensed parental strands of replicated DNA. Uncoating and nuclear entry was independent of proteasome function and resistant to inhibitors of nuclear export. Together with additional data our results reveal new insight into the spatiotemporal dynamics of HSV genome uncoating, transport and organisation.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Arapan-S: a fast and highly accurate whole-genome assembly software for viruses and small genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Findings: In this paper, we present Arapan-S, a whole-genome assembly program dedicated to handling small genomes. It provides only one contig (along with the reverse complement of this contig) in many cases. Although genomes consist of a number of segments, the implemented algorithm can detect all the segments, as we demonstrate for Influenza Virus A. The Arapan-S program is based on the de Bruijn graph. We have implemented a very sophisticated and fast method to reconstruct the original sequence and neglect erroneous k-mers. The method explores the graph by using neither the shortest nor the longest path, but rather a specific and reliable path based on the coverage level or k-mers' lengths. Arapan-S uses short reads, and it was tested on raw data downloaded from the NCBI Trace Archive.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>A bioinformatic filter for improved base-call accuracy and polymorphism detection using the Affymetrix GeneChip Õ whole-genome resequencing platform</h3>
<p style="text-align: justify; text-justify: inter-word;">DNA resequencing arrays enable rapid acquisition of high-quality sequence data. This technology represents a promising platform for rapid highresolution genotyping of microorganisms. Traditional array-based resequencing methods have relied on the use of specific PCR-amplified fragments from the query samples as hybridization targets. While this specificity in the target DNA population reduces the potential for artifacts caused by cross-hybridization, the subsampling of the query genome limits the sequence coverage that can be obtained and therefore reduces the technique's resolution as a genotyping method. We have developed and validated an Affymetrix Inc. GeneChip array-based, whole-genome resequencing platform for Francisella tularensis, the causative agent of tularemia. A set of bioinformatic filters that targeted systematic base-calling errors caused by cross-hybridization between the wholegenome sample and the array probes and by deletions in the sample DNA relative to the chip reference sequence were developed. Our approach eliminated 91% of the false-positive singlenucleotide polymorphism calls identified in the SCHU S4 query sample, at the cost of 10.7% of the true positives, yielding a total base-calling accuracy of 99.992%.</p>
<p align="right"><i>score: 112</i></p>

</div><br /><br /><br /><div class="card"><h3>An analysis of the feasibility of short read sequencing</h3>
<p style="text-align: justify; text-justify: inter-word;">Several methods for ultra high-throughput DNA sequencing are currently under investigation. Many of these methods yield very short blocks of sequence information (reads). Here we report on an analysis showing the level of genome sequencing possible as a function of read length. It is shown that resequencing and de novo sequencing of the majority of a bacterial genome is possible with read lengths of 20-30 nt, and that reads of 50 nt can provide reconstructed contigs (a contiguous fragment of sequence data) of 1000 nt and greater that cover 80% of human chromosome 1.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Transmission Pathways of Foot-and-Mouth Disease Virus in the United Kingdom</h3>
<p style="text-align: justify; text-justify: inter-word;">Foot-and-mouth disease (FMD) virus causes an acute vesicular disease of domesticated and wild ruminants and pigs. Identifying sources of FMD outbreaks is often confounded by incomplete epidemiological evidence and the numerous routes by which virus can spread (movements of infected animals or their products, contaminated persons, objects, and aerosols). Here, we show that the outbreaks of FMD in the United Kingdom in August 2007 were caused by a derivative of FMDV O 1 BFS 1860, a virus strain handled at two FMD laboratories located on a single site at Pirbright in Surrey. Genetic analysis of complete viral genomes generated in real-time reveals a probable chain of transmission events, predicting undisclosed infected premises, and connecting the second cluster of outbreaks in September to those in August. Complete genome sequence analysis of FMD viruses conducted in real-time have identified the initial and intermediate sources of these outbreaks and demonstrate the value of such techniques in providing information useful to contemporary disease control programmes.</p>
<p align="right"><i>score: 107</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficient oligonucleotide probe selection for pan-genomic tiling arrays</h3>
<p style="text-align: justify; text-justify: inter-word;">Results: This paper presents a new probe selection algorithm (PanArray) that can tile multiple whole genomes using a minimal number of probes. Unlike arrays built on clustered gene families, PanArray uses an unbiased, probe-centric approach that does not rely on annotations, gene clustering, or multi-alignments. Instead, probes are evenly tiled across all sequences of the pangenome at a consistent level of coverage. To minimize the required number of probes, probes conserved across multiple strains in the pan-genome are selected first, and additional probes are used only where necessary to span polymorphic regions of the genome. The viability of the algorithm is demonstrated by array designs for seven different bacterial pan-genomes and, in particular, the design of a 385,000 probe array that fully tiles the genomes of 20 different Listeria monocytogenes strains with overlapping probes at greater than twofold coverage.</p>
<p align="right"><i>score: 106</i></p>

</div><br /><br /><br /><div class="card"><h3>2 Division of Global Health Protection</h3>
<p style="text-align: justify; text-justify: inter-word;">The metagenomic surveillance approach for monitoring circulating pathogens in sewage was able to detect putative pathogen and resistance loads in an urban informal settlement. Thus, valuable if generated in real time to serve as a comprehensive infectious disease agent surveillance system with the potential to guide disease prevention and treatment. The approach may lead to a paradigm shift in conducting real-time global genomics-based surveillance in settings with limited access to health care.</p>
<p align="right"><i>score: 105</i></p>

</div><br /><br /><br /><div class="card"><h3>Rapid, Unbiased PRRSV Strain Detection Using MinION Direct RNA Sequencing and Bioinformatics Tools</h3>
<p style="text-align: justify; text-justify: inter-word;">Prompt detection and effective control of porcine reproductive and respiratory syndrome virus (PRRSV) during outbreaks is important given its immense adverse impact on the swine industry. However, the diagnostic process can be challenging due to the high genetic diversity and high mutation rate of PRRSV. A diagnostic method that can provide more detailed genetic information about pathogens is urgently needed. In this study, we evaluated the ability of Oxford Nanopore MinION direct RNA sequencing to generate a PRRSV whole genome sequence and detect and discriminate virus at the strain-level. A nearly full length PRRSV genome was successfully generated from raw sequence reads, achieving an accuracy of 96% after consensus genome generation. Direct RNA sequencing reliably detected the PRRSV strain present with an accuracy of 99.9% using as few as 5 raw sequencing reads and successfully differentiated multiple co-infecting strains present in a sample. In addition, PRRSV strain information was obtained from clinical samples containing 10 4 to 10 6 viral copies or more within 6 hours of sequencing. Overall, direct viral RNA sequencing followed by bioinformatic analysis proves to be a promising approach for identification of the viral strain or strains involved in clinical infections, allowing for more precise prevention and control strategies during PRRSV outbreaks.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>Genome3D: A viewer-model framework for integrating and visualizing multi-scale epigenomic information within a three-dimensional genome</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: New technologies are enabling the measurement of many types of genomic and epigenomic information at scales ranging from the atomic to nuclear. Much of this new data is increasingly structural in nature, and is often difficult to coordinate with other data sets. There is a legitimate need for integrating and visualizing these disparate data sets to reveal structural relationships not apparent when looking at these data in isolation. Results: We have applied object-oriented technology to develop a downloadable visualization tool, Genome3D, for integrating and displaying epigenomic data within a prescribed three-dimensional physical model of the human genome. In order to integrate and visualize large volume of data, novel statistical and mathematical approaches have been developed to reduce the size of the data. To our knowledge, this is the first such tool developed that can visualize human genome in three-dimension. We describe here the major features of Genome3D and discuss our multi-scale data framework using a representative basic physical model. We then demonstrate many of the issues and benefits of multi-resolution data integration. Conclusions: Genome3D is a software visualization tool that explores a wide range of structural genomic and epigenetic data. Data from various sources of differing scales can be integrated within a hierarchical framework that is easily adapted to new developments concerning the structure of the physical genome. In addition, our tool has a simple annotation mechanism to incorporate non-structural information. Genome3D is unique is its ability to manipulate large amounts of multi-resolution data from diverse sources to uncover complex and new structural relationships within the genome.</p>
<p align="right"><i>score: 104</i></p>

</div><br /><br /><br /><div class="card"><h3>The Utility of Data Transformation for Alignment, De Novo Assembly and Classification of Short Read Virus Sequences</h3>
<p style="text-align: justify; text-justify: inter-word;">Advances in DNA sequencing technology are facilitating genomic analyses of unprecedented scope and scale, widening the gap between our abilities to generate and fully exploit biological sequence data. Comparable analytical challenges are encountered in other data-intensive fields involving sequential data, such as signal processing, in which dimensionality reduction (i.e., compression) methods are routinely used to lessen the computational burden of analyses. In this work, we explored the application of dimensionality reduction methods to numerically represent high-throughput sequence data for three important biological applications of virus sequence data: reference-based mapping, short sequence classification and de novo assembly. Leveraging highly compressed sequence transformations to accelerate sequence comparison, our approach yielded comparable accuracy to existing approaches, further demonstrating its suitability for sequences originating from diverse virus populations. We assessed the application of our methodology using both synthetic and real viral pathogen sequences. Our results show that the use of highly compressed sequence approximations can provide accurate results, with analytical performance retained and even enhanced through appropriate dimensionality reduction of sequence data.</p>
<p align="right"><i>score: 99</i></p>

</div><br /><br /><br /><div class="card"><h3>Capturing diverse microbial sequence with comprehensive and scalable probe design</h3>
<p style="text-align: justify; text-justify: inter-word;">Metagenomic sequencing has the potential to transform microbial detection and characterization, but new tools are needed to improve its sensitivity. We developed CATCH (Compact Aggregation of Targets for Comprehensive Hybridization), a computational method to enhance nucleic acid capture for enrichment of diverse microbial taxa. CATCH designs compact probe sets that achieve full coverage of known sequence diversity and that scale well with this diversity. To illustrate applications of CATCH, we focused on capturing viral genomes. We designed, synthesized, and validated multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriched unique viral content on average 18× and allowed us to assemble genomes that we could not otherwise recover, while accurately preserving within-sample diversity. We used this approach to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of viral infections in samples with unknown content. Together, this work demonstrates a path toward more sensitive, cost-effective metagenomic sequencing.</p>
<p align="right"><i>score: 98</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</h3>
<p style="text-align: justify; text-justify: inter-word;">Sequence analyses of RNA virus genomes remain challenging due to the exceptional genetic plasticity of these viruses. Because of high mutation and recombination rates, genome replication by viral RNA-dependent RNA polymerases leads to populations of closely related viruses, so-called 'quasispecies'. Standard (short-read) sequencing technologies are ill-suited to reconstruct large numbers of full-length haplotypes of (i) RNA virus genomes and (ii) subgenome-length (sg) RNAs comprised of noncontiguous genome regions. Here, we used a full-length, direct RNA sequencing (DRS) approach based on nanopores to characterize viral RNAs produced in cells infected with a human coronavirus. Using DRS, we were able to map the longest (∼26 kb) contiguous read to the viral reference genome. By combining Illumina and nanopore sequencing, we reconstructed a highly accurate consensus sequence of the human coronavirus (HCoV) 229E genome (27.3 kb). Furthermore, using long reads that did not require an assembly step, we were able to identify, in infected cells, diverse and novel HCoV-229E sg RNAs that remain to be characterized. Also, the DRS approach, which circumvents reverse transcription and amplification of RNA, allowed us to detect methylation sites in viral RNAs. Our work paves the way for haplotype-based analyses of viral quasispecies by demonstrating the feasibility of intra-sample haplotype separation. Even though several technical challenges remain to be addressed to exploit the potential of the nanopore technology fully, our work illustrates that direct RNA sequencing may significantly advance genomic studies of complex virus populations, including predictions on long-range interactions in individual full-length viral RNA haplotypes.</p>
<p align="right"><i>score: 97</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Genomics An evaluation of Comparative Genome Sequencing (CGS) by comparing two previously-sequenced bacterial genomes</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: With the development of new technology, it has recently become practical to resequence the genome of a bacterium after experimental manipulation. It is critical though to know the accuracy of the technique used, and to establish confidence that all of the mutations were detected.</p>
<p align="right"><i>score: 96</i></p>

</div><br /><br /><br /><div class="card"><h3>Moving pathogen genomics out of the lab and into the clinic: what will it take?</h3>
<p style="text-align: justify; text-justify: inter-word;">Pathogen genomic analysis is a potentially transformative new approach to the clinical and public-health management of infectious diseases. Health systems investing in this technology will need to build infrastructure and develop policies that ensure genomic information can be generated, shared and acted upon in a timely manner.</p>
<p align="right"><i>score: 95</i></p>

</div><br /><br /><br /><div class="card"><h3>GenomeFingerprinter: The Genome Fingerprint and the Universal Genome Fingerprint Analysis for Systematic Comparative Genomics</h3>
<p style="text-align: justify; text-justify: inter-word;">We have created a method, GenomeFingerprinter, for rapidly computing, geometrically visualizing, intuitively comparing a set of genomes at genome fingerprint level, and hence established a method called the universal genome fingerprint analysis, as well as developed a method of quantitative analysis of the outcome dataset. These have set up the methodology of systematic comparative genomics based on the genome fingerprint analysis.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /><div class="card"><h3>Viral Phylogenomics Using an Alignment-Free Method: A Three- Step Approach to Determine Optimal Length of k-mer OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">A major challenge in using the FFP method for comparing whole genomes is determining the optimal k-mer length. In previous studies of dsDNA eukaryotic viruses 15,16,21 , the optimal feature length was based on cumulative relative entropy (CRE) and relative sequence divergence (RSD). For each individual genome and a value of k, the CRE (determined by a comparison of the observed FFP and the expected FFP from a second-order Markov model) captures how much information from the whole-genome sequence is encoded in the FFP. In other words, CRE indicates the power of the FFP to reconstruct the whole-genome sequence. Smaller CRE values, which result from longer k-mers, are indicative of the ability to better identify individual genomes. For a whole genome, the RSD for a value of k is a measure of the relatedness of the genome sequence (in terms of FFP) to a random sequence of the same length. According to Wu et al. 32 , the optimal value of k is the value when both CRE and RSD decrease to < 10% of their maximum values as k is increased.</p>
<p align="right"><i>score: 94</i></p>

</div><br /><br /><br /></body>