
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: using existing surveillance platforms to increase the capacity of existing diagnostic platforms</h2><br /></div><br /><br /><br /><div class="card"><h3>Expanding severe acute respiratory infection (SARI) surveillance beyond influenza: The process and data from 1 year of implementation in Vietnam</h3>
<p style="text-align: justify; text-justify: inter-word;">The Government of Vietnam has demonstrated that expanding respiratory viral surveillance by strengthening and building upon an influenza platform is feasible, efficient, and practical.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Expanding severe acute respiratory infection (SARI) surveillance beyond influenza: The process and data from 1 year of implementation in Vietnam</h3>
<p style="text-align: justify; text-justify: inter-word;">The process of expanding the suite of pathogens for routine testing by real-time reverse transcriptase-polymerase chain reaction (rRT-PCR) included laboratory trainings, procurement/distribution of reagents, and strengthening and aligning SARI surveillance epidemiology practices at sentinel sites and regional institutes (RI).</p>
<p align="right"><i>score: 103</i></p>

</div><br /><br /><br /><div class="card"><h3>Bringing together emerging and endemic zoonoses surveillance: shared challenges and a common solution</h3>
<p style="text-align: justify; text-justify: inter-word;">Early detection of disease outbreaks in human and animal populations is crucial to the effective surveillance of emerging infectious diseases. However, there are marked geographical disparities in capacity for early detection of outbreaks, which limit the effectiveness of global surveillance strategies. Linking surveillance approaches for emerging and neglected endemic zoonoses, with a renewed focus on existing disease problems in developing countries, has the potential to overcome several limitations and to achieve additional health benefits. Poor reporting is a major constraint to the surveillance of both emerging and endemic zoonoses, and several important barriers to reporting can be identified: (i) a lack of tangible benefits when reports are made; (ii) a lack of capacity to enforce regulations; (iii) poor communication among communities, institutions and sectors; and (iv) complexities of the international regulatory environment. Redirecting surveillance efforts to focus on endemic zoonoses in developing countries offers a pragmatic approach that overcomes some of these barriers and provides support in regions where surveillance capacity is currently weakest. In addition, this approach addresses immediate health and development problems, and provides an equitable and sustainable mechanism for building the culture of surveillance and the core capacities that are needed for all zoonotic pathogens, including emerging disease threats.</p>
<p align="right"><i>score: 93</i></p>

</div><br /><br /><br /><div class="card"><h3>Expanding severe acute respiratory infection (SARI) surveillance beyond influenza: The process and data from 1 year of implementation in Vietnam</h3>
<p style="text-align: justify; text-justify: inter-word;">GHS; USAID; CDC; WHO Background: In 2016, as a component of the Global Health Security Agenda, the Vietnam Ministry of Health expanded its existing influenza sentinel surveillance for severe acute respiratory infections (SARI) to include testing for 7 additional viral respiratory pathogens. This article describes the steps taken to implement expanded SARI surveillance in Vietnam and reports data from 1 year of expanded surveillance.</p>
<p align="right"><i>score: 88</i></p>

</div><br /><br /><br /><div class="card"><h3>Prevalence of non-influenza respiratory viruses in acute respiratory infection cases in Mexico</h3>
<p style="text-align: justify; text-justify: inter-word;">Acute respiratory infections are the leading cause of morbidity and mortality worldwide. Although a viral aetiological agent is estimated to be involved in up to 80% of cases, the majority of these agents have never been specifically identified. Since 2009, diagnostic and surveillance efforts for influenza virus have been applied worldwide. However, insufficient epidemiological information is available for the many other respiratory viruses that can cause Acute respiratory infections.</p>
<p align="right"><i>score: 86</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct nucleic acid analysis of mosquitoes for high fidelity species identification and 1 detection of Wolbachia using a cellphone 2 3</h3>
<p style="text-align: justify; text-justify: inter-word;">Yellow fever viruses and spreads through mosquito populations. For effective evaluation of this 48 approach, regular surveillance of Wolbachia infections in Ae. aegypti is required, but current 49 diagnostic tools are not well suited to support these critical surveillance needs. To fill this need 50</p>
<p align="right"><i>score: 85</i></p>

</div><br /><br /><br /><div class="card"><h3>Survey of Clinical Laboratory Practices for 2015 Middle East Respiratory Syndrome Coronavirus Outbreak in the Republic of Korea</h3>
<p style="text-align: justify; text-justify: inter-word;">Clinical laboratories were able to quickly expand their diagnostic capacity in response to the 2015 MERS-CoV outbreak. Our results show that clinical laboratories play an important role in the maintenance and enhancement of laboratory response in preparation for future emerging infections.</p>
<p align="right"><i>score: 82</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">Cite this article: Jephcott FL, Wood JLN, Cunningham AA. 2017 Facility-based surveillance for emerging infectious diseases; diagnostic practices in rural West African hospital settings: observations from Ghana.</p>
<p align="right"><i>score: 80</i></p>

</div><br /><br /><br /><div class="card"><h3>Building laboratory capacity to detect and characterize pathogens of public and global health security concern in Kenya</h3>
<p style="text-align: justify; text-justify: inter-word;">Since 1979, multiple CDC Kenya programs have supported the development of diagnostic expertise and laboratory capacity in Kenya. In 2004, CDC's Global Disease Detection (GDD) program within the Division of Global Health Protection in Kenya (DGHP-Kenya) initiated close collaboration with Kenya Medical Research Institute (KEMRI) and developed a laboratory partnership called the Diagnostic and Laboratory Systems Program (DLSP). DLSP built onto previous efforts by malaria, human immunodeficiency virus (HIV) and tuberculosis (TB) programs and supported the expansion of the diagnostic expertise and capacity in KEMRI and the Ministry of Health. First, DLSP developed laboratory capacity for surveillance of diarrheal, respiratory, zoonotic and febrile illnesses to understand the etiology burden of these common illnesses and support evidenced-based decisions on vaccine introductions and recommendations in Kenya. Second, we have evaluated and implemented new diagnostic technologies such as TaqMan Array Cards (TAC) to detect emerging or reemerging pathogens and have recently added a next generation sequencer (NGS). Third, DLSP provided rapid laboratory diagnostic support for outbreak investigation to Kenya and regional countries. Fourth, DLSP has been assisting the Kenya National Public Health laboratory-National Influenza Center and microbiology reference laboratory to obtain World Health Organization (WHO) certification and ISO15189 accreditation respectively. Fifth, we have supported biosafety and biosecurity curriculum development to help Kenyan laboratories safely and appropriately manage infectious pathogens. These achievements, highlight how in collaboration with existing CDC programs working on HIV, tuberculosis and malaria, the Global Health Security Agenda can have significantly improve public health in Kenya and the region. Moreover, Kenya provides an example as to how laboratory science can help countries detect and control of infectious disease outbreaks and other public health threats more rapidly, thus enhancing global health security.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Etiology of respiratory tract infections in the community and clinic in Ilorin, Nigeria</h3>
<p style="text-align: justify; text-justify: inter-word;">Objective: Recognizing increasing interest in community disease surveillance globally, the goal of this study was to investigate whether respiratory viruses circulating in the community may be represented through clinical (hospital) surveillance in Nigeria.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<p style="text-align: justify; text-justify: inter-word;">This disparity in technologies and data collection methods results in incomplete surveillance.</p>
<p align="right"><i>score: 78</i></p>

</div><br /><br /><br /><div class="card"><h3>Efficient and unbiased metagenomic recovery of RNA virus genomes from human plasma samples OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Single-stranded RNA viruses exhibit exceptional genetic diversity due to low fidelity replication mechanisms 1, 2 . As a group, they constitute the major source of emerging infections in humans such as Ebola, chikungunya, Zika, West Nile virus and Middle East respiratory syndrome-related coronavirus 3-7 . With new human RNA viruses being discovered each year 8 , the serological and nucleic acid amplification techniques that have dominated virus diagnostics for many years are becoming increasingly unable to respond to the ever-expanding range of pathogens.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Ebola virus disease outbreak in Korea: use of a mathematical model and stochastic simulation to estimate risk</h3>
<p style="text-align: justify; text-justify: inter-word;">Timely and rigorous diagnosis is important to reduce the spreading scale of infection when a new communicable disease is inflowed into Korea. Moreover, it is imperative to strengthen the local surveillance system and diagnostic protocols to avoid missing cases of secondary infection.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Feasibility study for the use of self-collected nasal swabs to identify pathogens among participants of a population- based surveillance system for acute respiratory infections (GrippeWeb-Plus)-Germany, 2016</h3>
<p style="text-align: justify; text-justify: inter-word;">We recruited 103 GrippeWeb participants (73 adults and 30 children) and provided them with a kit, instructions and a questionnaire for each sample. In the first half of 2016, participants took an anterior nasal swab and sent it to the Robert Koch Institute whenever an acute respiratory illness occurred. Reporting of illnesses through the GrippeWeb platform continued as usual. We analysed swabs for the presence of human c-myc-DNA and 22 viral and bacterial pathogens. After the study, we sent participants an evaluation questionnaire. We analysed timeliness, completeness, acceptability and validity.</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Discovery and Characterization of Novel Bat Coronavirus Lineages from Kazakhstan</h3>
<p style="text-align: justify; text-justify: inter-word;">Coronaviruses are positive-stranded RNA viruses that infect a variety of hosts, resulting in a range of symptoms from gastrointestinal illness to respiratory distress. Bats are reservoirs for a high diversity of coronaviruses, and focused surveillance detected several strains genetically similar to MERS-coronavirus, SARS-coronavirus, and the human coronaviruses 229E and NL63. The bat fauna of central Asia, which link China to eastern Europe, are relatively less studied than other regions of the world. Kazakhstan is the world's ninth largest country; however, little is understood about the prevalence and diversity of bat-borne viruses. In this study, bat guano was collected from bat caves in three different sites of southern Kazakhstan that tested positive for coronaviruses. Our phylogenetic reconstruction indicates these are novel bat coronaviruses that belong to the genus Alphacoronavirus. In addition, two distinct lineages of Kazakhstan bat coronaviruses were detected. Both lineages are closely related to bat coronaviruses from China, France, Spain, and South Africa, suggesting that co-circulation of coronaviruses is common in multiple bat species with overlapping geographical distributions. Our study highlights the need for collaborative efforts in understudied countries to increase integrated surveillance capabilities toward better monitoring and detection of infectious diseases.</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Dogs (Canis familiaris) as Sentinels for Human Infectious Disease and Application to Canadian Populations: A Systematic Review</h3>
<p style="text-align: justify; text-justify: inter-word;">In a world where climate change, vector expansion, human activity, and pathogen dispersal do not respect boundaries, the human-animal-pathogen interface has become less defined. Consequently, a One Health approach to disease surveillance and control has generated much interest across several disciplines. This systematic review evaluates current global research on the use of domestic dogs as sentinels for human infectious disease, and critically appraises how this may be applied within Canada. Results highlighted a bias in research from highand middle-income-economy countries, with 35% of the studies describing data from the Latin America/Caribbean region, 25% from North America, and 11% from the European/Central Asia region. Bacteria were the most studied type of infectious agent, followed by protozoa, viruses, helminths, and fungi. Only six out of 142 studies described disease in Canada: four researched a variety of pathogens within Indigenous communities, one researched Borrelia burgdorferi in British Columbia, and one researched arboviruses in Quebec. Results from this review suggest that dogs could provide excellent sentinels for certain infectious-disease pathogens in Canada, yet are currently overlooked. Further research into the use of dog-sentinel surveillance is specifically recommended for California serogroup viruses, Chikungunya virus, West Nile virus, Lyme borreliosis, Rickettsia spp., Ehrlichia spp., and Dirofilaria immitis.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>BMC Veterinary Research Molecular characterisation of a bovine-like rotavirus detected from a giraffe</h3>
<p style="text-align: justify; text-justify: inter-word;">This study demonstrates the need for continuous surveillance of RV strains in various animal populations, which will facilitate the identification of rotavirus hosts not previously reported. Furthermore, extending typical epidemiology studies to a broader host range will contribute to the timely identification of new emerging strain types.</p>
<p align="right"><i>score: 74</i></p>

</div><br /><br /><br /><div class="card"><h3>Field pathogenomics reveals the emergence of a diverse wheat yellow rust population</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Emerging and re-emerging pathogens imperil public health and global food security. Responding to these threats requires improved surveillance and diagnostic systems. Despite their potential, genomic tools have not been readily applied to emerging or re-emerging plant pathogens such as the wheat yellow (stripe) rust pathogen Puccinia striiformis f. sp. tritici (PST). This is due largely to the obligate parasitic nature of PST, as culturing PST isolates for DNA extraction remains slow and tedious.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Ensuring HIV Data Availability, Transparency and Integrity in the MENA Region Comment on "Improving the Quality and Quantity of HIV Data in the Middle East and North Africa: Key Challenges and Ways Forward" Commentary</h3>
<p style="text-align: justify; text-justify: inter-word;">In this commentary, we elaborate on the main points that Karamouzian and colleagues have made about HIV data scarcity in Middle Eastern and North African (MENA) countries. Without accessible and reliable data, no epidemic can be managed effectively or efficiently. Clearly, increased investments are needed to bolster capabilities to capture and interpret HIV surveillance data. We believe that this enhanced capacity can be achieved, in part, by leveraging and repurposing existing data platforms, technologies and patient cohorts. An immediate modest investment that capitalizes on available infrastructure can generate data on the HIV burden and spread that can be persuasive for MENA policy-makers to intensify efforts to track and contain the growing HIV epidemic in this region. A focus on key populations will yield the most valuable data, including among men who have sex with men (MSM), transgender women and men, persons who inject drugs (PWIDs), female partners of high risk men and female sex workers.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</h3>
<p style="text-align: justify; text-justify: inter-word;">Comprehensive and uniform diagnostic test data will aid in the identification of potential outbreaks. A combination of broad respiratory pathogen testing and an internal electronic surveillance system enabled the rapid dissemination of data across the largest health care system 80 in New York, the North Shore-LIJ Health System (now Northwell Health) during the influenza A H1N1-2009 pandemic in the New York City area. Pathogen-specific molecular testing permitted rapid a) notification to state epidemiologists; b) tracking of the virus so that health care resources could be managed effectively; and c) evaluation of influenza diagnostics [27, 28] .</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /></body>