
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: universal coronavirus vaccine</h2><br /></div><br /><br /><br /><div class="card"><h3>A highly conserved WDYPKCDRA epitope in the RNA directed RNA polymerase of human coronaviruses can be used as epitope-based universal vaccine design</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Coronaviruses are the diverse group of RNA virus. From 1960, six strains of human coronaviruses have emerged that includes SARS-CoV and the recent infection by deadly MERS-CoV which is now going to cause another outbreak. Prevention of these viruses is urgent and a universal vaccine for all strain could be a promising solution in this circumstance. In this study we aimed to design an epitope based vaccine against all strain of human coronavirus. Results: Multiple sequence alignment (MSA) approach was employed among spike (S), membrane (M), enveloped (E) and nucleocapsid (N) protein and replicase polyprotein 1ab to identify which one is highly conserve in all coronaviruses strains. Next, we use various in silico tools to predict consensus immunogenic and conserved peptide. We found that conserved region is present only in the RNA directed RNA polymerase protein. In this protein we identified one epitope WDYPKCDRA is highly immunogenic and 100% conserved among all available human coronavirus strains. Conclusions: Here we suggest in vivo study of our identified novel peptide antigen in RNA directed RNA polymerase protein for universal vaccinewhich may be the way to prevent all human coronavirus disease.</p>
<p align="right"><i>score: 241</i></p>

</div><br /><br /><br /><div class="card"><h3>Evaluation of a recombination-resistant coronavirus as a broadly applicable, rapidly implementable vaccine platform</h3>
<p style="text-align: justify; text-justify: inter-word;">Emerging and re-emerging zoonotic viral diseases are major threats to global health, economic stability, and national security. Vaccines are key for reducing coronaviral disease burden; however, the utility of live-attenuated vaccines is limited by risks of reversion or repair. Because of their history of emergence events due to their prevalence in zoonotic pools, designing live-attenuated coronavirus vaccines that can be rapidly and broadly implemented is essential for outbreak preparedness. Here, we show that coronaviruses with completely rewired transcription regulatory networks (TRNs) are effective vaccines against SARS-CoV. The TRN-rewired viruses are attenuated and protect against lethal SARS-CoV challenge. While a 3-nt rewired TRN reverts via second-site mutation upon serial passage, a 7-nt rewired TRN is more stable, suggesting that a more extensively rewired TRN might be essential for avoiding growth selection. In summary, rewiring the TRN is a feasible strategy for limiting reversion in an effective live-attenuated coronavirus vaccine candidate that is potentially portable across the Nidovirales order.</p>
<p align="right"><i>score: 215</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines 1 2</h3>
<p style="text-align: justify; text-justify: inter-word;">With an ongoing threat posed by circulating zoonotic strains, new strategies are required to 8 prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved 9 coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current 10 and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a 11 conserved 2'O methyltransferase (MTase), could provide a broad attenuation platform against 12 future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was 13 evaluated for protection from heterologous challenge, efficacy in the aging host, and potential 14 for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and 15 potential for reversion makes targeting NSP16 alone an untenable approach. However, 16 combining a 2'O MTase mutation with a previously described CoV fidelity mutant produced a 17 vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, 18 and no evidence for reversion. Together, the results indicate that targeting the CoV 2'O MTase 19 in parallel with other conserved attenuating mutations may provide a platform strategy for 20 rapidly generating live-attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 196</i></p>

</div><br /><br /><br /><div class="card"><h3>Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development</h3>
<p style="text-align: justify; text-justify: inter-word;">Novel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARSlike bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.</p>
<p align="right"><i>score: 192</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines 1 2</h3>
<p style="text-align: justify; text-justify: inter-word;">Targeting both conserved coronavirus activities produced a stable, attenuated virus capable of 80 protection from heterologous challenge, efficacy in aged mice, and absence of reversion in 81 immunocompromised models. Together, the results indicate that targeting coronavirus NSP16 82 may be a critical component of a future live attenuated coronavirus vaccine approach. 83 84 . CC-BY-NC-ND 4.0 International license is made available under a</p>
<p align="right"><i>score: 189</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus Non-Structural Protein 1 Is a Major Pathogenicity Factor: Implications for the Rational Design of Coronavirus Vaccines</h3>
<p style="text-align: justify; text-justify: inter-word;">Attenuated viral vaccines can be generated by targeting essential pathogenicity factors. We report here the rational design of an attenuated recombinant coronavirus vaccine based on a deletion in the coding sequence of the nonstructural protein 1 (nsp1). In cell culture, nsp1 of mouse hepatitis virus (MHV), like its SARS-coronavirus homolog, strongly reduced cellular gene expression. The effect of nsp1 on MHV replication in vitro and in vivo was analyzed using a recombinant MHV encoding a deletion in the nsp1-coding sequence. The recombinant MHV nsp1 mutant grew normally in tissue culture, but was severely attenuated in vivo. Replication and spread of the nsp1 mutant virus was restored almost to wild-type levels in type I interferon (IFN) receptor-deficient mice, indicating that nsp1 interferes efficiently with the type I IFN system. Importantly, replication of nsp1 mutant virus in professional antigen-presenting cells such as conventional dendritic cells and macrophages, and induction of type I IFN in plasmacytoid dendritic cells, was not impaired. Furthermore, even low doses of nsp1 mutant MHV elicited potent cytotoxic T cell responses and protected mice against homologous and heterologous virus challenge. Taken together, the presented attenuation strategy provides a paradigm for the development of highly efficient coronavirus vaccines.</p>
<p align="right"><i>score: 183</i></p>

</div><br /><br /><br /><div class="card"><h3>vaccines Harnessing the Power of T Cells: The Promising Hope for a Universal Influenza Vaccine</h3>
<p style="text-align: justify; text-justify: inter-word;">Next-generation vaccines that utilize T cells could potentially overcome the limitations of current influenza vaccines that rely on antibodies to provide narrow subtype-specific protection and are prone to antigenic mismatch with circulating strains. Evidence from animal models shows that T cells can provide heterosubtypic protection and are crucial for immune control of influenza virus infections. This has provided hope for the design of a universal vaccine able to prime against diverse influenza virus strains and subtypes. However, multiple hurdles exist for the realisation of a universal T cell vaccine. Overall primary concerns are: extrapolating human clinical studies, seeding durable effective T cell resident memory (Trm), population human leucocyte antigen (HLA) coverage, and the potential for T cell-mediated immune escape. Further comprehensive human clinical data is needed during natural infection to validate the protective role T cells play during infection in the absence of antibodies. Furthermore, fundamental questions still exist regarding the site, longevity and duration, quantity, and phenotype of T cells needed for optimal protection. Standardised experimental methods, and eventually simplified commercial assays, to assess peripheral influenza-specific T cell responses are needed for larger-scale clinical studies of T cells as a correlate of protection against influenza infection. The design and implementation of a T cell-inducing vaccine will require a consensus on the level of protection acceptable in the community, which may not provide sterilizing immunity but could protect the individual from severe disease, reduce the length of infection, and potentially reduce transmission in the community. Therefore, increasing the standard of care potentially offered by T cell vaccines should be considered in the context of pandemic preparedness and zoonotic infections, and in combination with improved antibody vaccine targeting methods. Current pandemic vaccine preparedness measures and ongoing clinical trials under-utilise T cell-inducing vaccines, reflecting the myriad questions that remain about how, when, where, and which T cells are needed to fight influenza virus infection. This review aims to bring together basic fundamentals of T cell biology with human clinical data, which need to be considered for the implementation of a universal vaccine against influenza that harnesses the power of T cells.</p>
<p align="right"><i>score: 181</i></p>

</div><br /><br /><br /><div class="card"><h3>Combination attenuation offers strategy for live-attenuated coronavirus vaccines 1 2</h3>
<p style="text-align: justify; text-justify: inter-word;">. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 31 attenuation as a major component of future live-attenuated coronavirus vaccines.</p>
<p align="right"><i>score: 172</i></p>

</div><br /><br /><br /><div class="card"><h3>MERS-CoV NSP16 necessary for IFN resistance and viral pathogenesis</h3>
<p style="text-align: justify; text-justify: inter-word;">16 All rights reserved. No reuse allowed without permission. Significance 1 Coronavirus emergence in both human and livestock represents a significant threat to global 2 public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and 3 swine delta coronavirus in the 21 st century. These studies describe an approach that effectively 4 targets the highly conserved 2'O methyl-transferase activity of coronaviruses for attenuation. 5 With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable 6 target for a live attenuated vaccine platform as well as therapeutic development for both current 7 and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-8 coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad 9 applicability of targeting viral 2'O MTase function across coronaviruses.</p>
<p align="right"><i>score: 169</i></p>

</div><br /><br /><br /><div class="card"><h3>a section of the journal Frontiers in Immunology Polyanhydride Nanovaccine Induces Robust Pulmonary B and T Cell Immunity and Confers Protection Against Homologous and Heterologous Influenza A Virus Infections</h3>
<p style="text-align: justify; text-justify: inter-word;">Influenza A virus (IAV) is a major cause of respiratory illness. Given the disease severity, associated economic costs, and recent appearance of novel IAV strains, there is a renewed interest in developing novel and efficacious "universal" IAV vaccination strategies. Recent studies have highlighted that immunizations capable of generating local (i.e., nasal mucosa and lung) tissue-resident memory T and B cells in addition to systemic immunity offer the greatest protection against future IAV encounters. Current IAV vaccines are designed to largely stimulate IAV-specific antibodies, but do not generate the lung-resident memory T and B cells induced during IAV infections. Herein, we report on an intranasally administered biocompatible polyanhydride nanoparticle-based IAV vaccine (IAV-nanovax) capable of providing protection against subsequent homologous and heterologous IAV infections in both inbred and outbred populations. Our findings also demonstrate that vaccination with IAV-nanovax promotes the induction of germinal center B cells within the lungs, both systemic and lung local IAV-specific antibodies, and IAV-specific lung-resident memory CD4 and CD8 T cells. Altogether our findings show that an intranasally administered nanovaccine can induce immunity within the lungs, similar to what occurs during IAV infections, and thus could prove useful as a strategy for providing "universal" protection against IAV.</p>
<p align="right"><i>score: 168</i></p>

</div><br /><br /><br /><div class="card"><h3>Vulnerabilities in coronavirus glycan shields despite extensive glycosylation</h3>
<p style="text-align: justify; text-justify: inter-word;">Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.</p>
<p align="right"><i>score: 162</i></p>

</div><br /><br /><br /><div class="card"><h3>An H5N1-based matrix protein 2 ectodomain tetrameric peptide vaccine provides cross-protection against lethal infection with H7N9 influenza virus</h3>
<p style="text-align: justify; text-justify: inter-word;">In March 2013, a patient infected with a novel avian influenza A H7N9 virus was reported in China. Since then, there have been 458 confirmed infection cases and 177 deaths. The virus contains several human-adapted markers, indicating that H7N9 has pandemic potential. The outbreak of this new influenza virus highlighted the need for the development of universal influenza vaccines. Previously, we demonstrated that a tetrameric peptide vaccine based on the matrix protein 2 ectodomain (M2e) of the H5N1 virus (H5N1-M2e) could protect mice from lethal infection with different clades of H5N1 and 2009 pandemic H1N1 influenza viruses. In this study, we investigated the cross-protection of H5N1-M2e against lethal infection with the new H7N9 virus. Although five amino acid differences existed at positions 13, 14, 18, 20, and 21 between M2e of H5N1 and H7N9, H5N1-M2e vaccination with either Freund's adjuvant or the Sigma adjuvant system (SAS) induced a high level of anti-M2e antibody, which cross-reacted with H7N9-M2e peptide. A mouse-adapted H7N9 strain, A/Anhui/01/2013m, was used for lethal challenge in animal experiments. H5N1-M2e vaccination provided potent cross-protection against lethal challenge of the H7N9 virus. Reduced viral replication and histopathological damage of mouse lungs were also observed in the vaccinated mice. Our results suggest that the tetrameric H5N1-M2e peptide vaccine could protect against different subtypes of influenza virus infections. Therefore, this vaccine may be an ideal candidate for developing a universal vaccine to prevent the reemergence of avian influenza A H7N9 virus and the emergence of potential novel reassortants of influenza virus.</p>
<p align="right"><i>score: 160</i></p>

</div><br /><br /><br /><div class="card"><h3>Dendritic Cell-Specific Antigen Delivery by Coronavirus Vaccine Vectors Induces Long-Lasting Protective Antiviral and Antitumor Immunity</h3>
<p style="text-align: justify; text-justify: inter-word;">Citation Cervantes-Barragan, L., R. Züst, R. Maier, S. Sierro, J. Janda, et al. 2010. Dendritic cell-specific antigen delivery by coronavirus vaccine vectors induces long-lasting protective antiviral and antitumor immunity. mBio 1(4):</p>
<p align="right"><i>score: 159</i></p>

</div><br /><br /><br /><div class="card"><h3>Introduction of neutralizing immunogenicity index to the rational design of MERS coronavirus subunit vaccines</h3>
<p style="text-align: justify; text-justify: inter-word;">Viral subunit vaccines often contain immunodominant non-neutralizing epitopes that divert host immune responses. These epitopes should be eliminated in vaccine design, but there is no reliable method for evaluating an epitope's capacity to elicit neutralizing immune responses. Here we introduce a new concept 'neutralizing immunogenicity index' (NII) to evaluate an epitope's neutralizing immunogenicity. To determine the NII, we mask the epitope with a glycan probe and then assess the epitope's contribution to the vaccine's overall neutralizing immunogenicity. As proof-of-concept, we measure the NII for different epitopes on an immunogen comprised of the receptor-binding domain from MERS coronavirus (MERS-CoV). Further, we design a variant form of this vaccine by masking an epitope that has a negative NII score. This engineered vaccine demonstrates significantly enhanced efficacy in protecting transgenic mice from lethal MERS-CoV challenge. Our study may guide the rational design of highly effective subunit vaccines to combat MERS-CoV and other life-threatening viruses.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>Towards a coronavirus-based HIV multigene vaccine</h3>
<p style="text-align: justify; text-justify: inter-word;">Human immunodeficiency virus (HIV) infection represents one of the major health threats in the developing world. The costly treatment of infected individuals with multiple highly efficient anti-HIV drugs is only affordable in industrialized countries. Thus, an efficient vaccination strategy is required to prevent the further spread of the infection. The molecular biology of coronaviruses and particular features of the human coronavirus 229E (HCoV 229E) indicate that HCoV 229E-based vaccine vectors can become a new class of highly efficient vaccines. First, the receptor of HCoV 229E, human aminopeptidase N (hAPN or CD13) is expressed mainly on human dendritic cells (DCs) and macrophages indicating that targeting of HCoV 229E-based vectors to professional antigen presenting cells can be achieved by receptor-mediated transduction. Second, HCoV 229E structural genes can be replaced by multiple transcriptional units encoding various antigens. These virus-like particles (VLPs) containing HCoV 229E-based vector RNA have the ability to transduce human DCs and to mediate heterologous gene expression in these cells. Finally, coronavirus infections are associated with mainly respiratory and enteric diseases, and natural transmission of coronaviruses occurs via mucosal surfaces. In humans, HCoV 229E causes common cold by infecting the upper respiratory tract. HCoV 229E infections are mainly encountered in children and re-infection occurs frequently in adults. It is thus most likely that pre-existing immunity against HCoV 229E will not significantly impact on the vaccination efficiency if HCoV 229E-based vectors are used in humans.</p>
<p align="right"><i>score: 158</i></p>

</div><br /><br /><br /><div class="card"><h3>Computational Approaches and Challenges to Developing Universal Influenza Vaccines</h3>
<p style="text-align: justify; text-justify: inter-word;">The traditional design of effective vaccines for rapidly-evolving pathogens, such as influenza A virus, has failed to provide broad spectrum and long-lasting protection. With low cost whole genome sequencing technology and powerful computing capabilities, novel computational approaches have demonstrated the potential to facilitate the design of a universal influenza vaccine. However, few studies have integrated computational optimization in the design and discovery of new vaccines. Understanding the potential of computational vaccine design is necessary before these approaches can be implemented on a broad scale. This review summarizes some promising computational approaches under current development, including computationally optimized broadly reactive antigens with consensus sequences, phylogenetic model-based ancestral sequence reconstruction, and immunomics to compute conserved cross-reactive T-cell epitopes. Interactions between virus-host-environment determine the evolvability of the influenza population. We propose that with the development of novel technologies that allow the integration of data sources such as protein structural modeling, host antibody repertoire analysis and advanced phylodynamic modeling, computational approaches will be crucial for the development of a long-lasting universal influenza vaccine. Taken together, computational approaches are powerful and promising tools for the development of a universal influenza vaccine with durable and broad protection.</p>
<p align="right"><i>score: 157</i></p>

</div><br /><br /><br /><div class="card"><h3>Vaccination to Conserved Influenza Antigens in Mice Using a Novel Simian Adenovirus Vector, PanAd3, Derived from the Bonobo Pan paniscus</h3>
<p style="text-align: justify; text-justify: inter-word;">Among approximately 1000 adenoviruses from chimpanzees and bonobos studied recently, the Pan Adenovirus type 3 (PanAd3, isolated from a bonobo, Pan paniscus) has one of the best profiles for a vaccine vector, combining potent transgene immunogenicity with minimal pre-existing immunity in the human population. In this study, we inserted into a replication defective PanAd3 a transgene expressing a fusion protein of conserved influenza antigens nucleoprotein (NP) and matrix 1 (M1). We then studied antibody and T cell responses as well as protection from challenge infection in a mouse model. A single intranasal administration of PanAd3-NPM1 vaccine induced strong antibody and T cell responses, and protected against high dose lethal influenza virus challenge. Thus PanAd3 is a promising candidate vector for vaccines, including universal influenza vaccines.</p>
<p align="right"><i>score: 157</i></p>

</div><br /><br /><br /><div class="card"><h3>Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</h3>
<p style="text-align: justify; text-justify: inter-word;">The tremendous pandemic potential of coronaviruses was demonstrated twice in the last decades by two global outbreaks of deadly pneumonia. Entry of coronaviruses into cells is mediated by the transmembrane spike glycoprotein S, which forms a trimer carrying receptor-binding and membrane fusion functions 1 . S also contains the principal antigenic determinants and is the target of neutralizing antibodies. Here we present the structure of a murine coronavirus S trimer ectodomain determined at 4.0 Å resolution by single particle cryo-electron microscopy. It reveals the metastable pre-fusion architecture of S and highlights key interactions stabilizing it. The structure shares a common core with paramyxovirus F proteins 2,3 , implicating mechanistic similarities and an evolutionary connection between these viral fusion proteins. The accessibility of the highly conserved fusion peptide at the periphery of the trimer indicates potential vaccinology strategies to elicit broadly neutralizing antibodies against coronaviruses. Finally, comparison with crystal structures of human coronavirus S domains allows rationalization of the molecular basis for species specificity based on the use of spatially contiguous but distinct domains.</p>
<p align="right"><i>score: 155</i></p>

</div><br /><br /><br /><div class="card"><h3>antibody-Dependent cell-Mediated cytotoxicity epitopes on the hemagglutinin head region of Pandemic h1n1 influenza Virus Play Detrimental roles in h1n1-infected Mice</h3>
<p style="text-align: justify; text-justify: inter-word;">Engaging the antibody-dependent cell-mediated cytotoxicity (ADCC) for killing of virus-infected cells and secretion of antiviral cytokines and chemokines was incorporated as one of the important features in the design of universal influenza vaccines. However, investigation of the ADCC epitopes on the highly immunogenic influenza hemagglutinin (HA) head region has been rarely reported. In this study, we determined the ADCC and antiviral activities of two putative ADCC epitopes, designated E1 and E2, on the HA head of a pandemic H1N1 influenza virus in vitro and in a lethal mouse model. Our data demonstrated that sera from the E1-vaccinated mice could induce high ADCC activities. Importantly, the induction of ADCC response modestly decreased viral load in the lungs of H1N1-infected mice. However, the elevated ADCC significantly increased mouse alveolar damage and mortality than that of the PBS-vaccinated group (P < 0.0001). The phenotype was potentially due to an exaggerated inflammatory cell infiltration triggered by ADCC, as an upregulated release of cytotoxic granules (perforin) was observed in the lung tissue of E1-vaccinated mice after H1N1 influenza virus challenge. Overall, our data suggested that ADCC elicited by certain domains of HA head region might have a detrimental rather than protective effect during influenza virus infection. Thus, future design of universal influenza vaccine shall strike a balance between the induction of protective immunity and potential side effects of ADCC.</p>
<p align="right"><i>score: 155</i></p>

</div><br /><br /><br /><div class="card"><h3>Coronavirus S protein-induced fusion is blocked prior to hemifusion by Abl kinase inhibitors</h3>
<p style="text-align: justify; text-justify: inter-word;">infection. This will provide insight into possible therapeutic targets to treat infections by current as well as newly emerging coronaviruses.</p>
<p align="right"><i>score: 152</i></p>

</div><br /><br /><br /></body>