
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: separation of assay development issues from instruments</h2><br /></div><br /><br /><br /><div class="card"><h3>A Low-Cost Palmtop High- Speed Capillary Electrophoresis Bioanalyzer with Laser Induced Fluorescence Detection</h3>
<p style="text-align: justify; text-justify: inter-word;">In this work, we developed a miniaturized palmtop high-speed capillary electrophoresis (CE) system integrating whole modules, including picoliter-scale sample injection, short capillary-based fast CE, high-voltage power supply, orthogonal laser induced fluorescence (LIF) detection, battery, system control, on-line data acquisition, processing, storage, and display modules. A strategy of minimalist miniaturization combining minimal system design and low-cost system construction was adopted to achieve the instrument miniaturization with extremely low cost, which is differing from the current microfabrication strategy used in most reported miniaturized CE systems. With such a strategy, the total size of the bioanalyzer was minimized to 90 × 75 × 77 mm (length × width × height) and the instrument cost was reduced to ca. $500, which demonstrated the smallest and lowest-cost CE instrument with LIF detection in so far reported systems. The present bioanalyzer also exhibited comparable analytical performances to previously-reported high-speed CE systems. A limit of detection of 1.02 nM sodium fluorescein was obtained. Fast separations were achieved for multiple types of samples as amino acids, amino acid enantiomers, DNA fragments, and proteins with high efficiency. We applied this instrument in colorectal cancer diagnosis for detecting KRAS mutation status by polymerase chain reaction-restriction fragment length polymorphism (PCR-RFLP) method.</p>
<p align="right"><i>score: 101</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple, Inexpensive Device for Nucleic Acid Amplification without Electricity-Toward Instrument- Free Molecular Diagnostics in Low-Resource Settings</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Molecular assays targeted to nucleic acid (NA) markers are becoming increasingly important to medical diagnostics. However, these are typically confined to wealthy, developed countries; or, to the national reference laboratories of developing-world countries. There are many infectious diseases that are endemic in low-resource settings (LRS) where the lack of simple, instrument-free, NA diagnostic tests is a critical barrier to timely treatment. One of the primary barriers to the practicality and availability of NA assays in LRS has been the complexity and power requirements of polymerase chain reaction (PCR) instrumentation (another is sample preparation).</p>
<p align="right"><i>score: 77</i></p>

</div><br /><br /><br /><div class="card"><h3>Research Article</h3>
<p style="text-align: justify; text-justify: inter-word;">Testing for the presence of genetically modified material in seed samples is of critical importance for all stakeholders in the agricultural industry, including growers, seed manufacturers, and regulatory bodies. While rapid antibody-based testing for the transgenic protein has fulfilled this need in the past, the introduction of new variants of a given transgene demands new diagnostic regimen that allows distinguishing different traits at the nucleic acid level. Although such molecular tests can be performed by PCR in the laboratory, their requirement for expensive equipment and sophisticated operation have prevented its uptake in pointof-use applications. A recently developed isothermal DNA amplification technique, recombinase polymerase amplification (RPA), combines simple sample preparation and amplification work-flow procedures with the use of minimal detection equipment in real time. Here, we report the development of a highly sensitive and specific RPA-based detection system for Genuity Roundup Ready 2 Yield (RR2Y) material in soybean (Glycine max) seed samples and present the results of studies applying the method in both laboratory and field-type settings.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Improved production of human type II procollagen in the yeast Pichia pastoris in shake flasks by a wireless-controlled fed-batch system</h3>
<p style="text-align: justify; text-justify: inter-word;">The recommended standard protocol for methanol addition in shake flasks using pulse feeding is nonoptimal and leads to repeated long phases of methanol starvation. The problem can be solved by applying the fed-batch technology. The presented wireless feeding unit, together with an on-line monitoring system offers a flexible, simple, and low-cost solution for initial optimization of the production in shake flasks which can be performed in parallel. By this way the fed-batch strategy can be applied from the early screening steps also in laboratories which do not have access to highcost and complicated bioreactor systems.</p>
<p align="right"><i>score: 73</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of a quantitative loop- mediated isothermal amplification assay for the field detection of Erysiphe necator</h3>
<p style="text-align: justify; text-justify: inter-word;">Plant pathogen detection systems have been useful tools to monitor inoculum presence and initiate management schedules. More recently, a loop-mediated isothermal amplification (LAMP) assay was successfully designed for field use in the grape powdery mildew pathosystem; however, false negatives or false positives were prevalent in grower-conducted assays due to the difficulty in perceiving the magnesium pyrophosphate precipitate at low DNA concentrations. A quantitative LAMP (qLAMP) assay using a fluorescence resonance energy transfer-based probe was assessed by grape growers in the Willamette Valley of Oregon. Custom impaction spore samplers were placed at a research vineyard and six commercial vineyard locations, and were tested bi-weekly by the lab and by growers. Growerconducted qLAMP assays used a beta-version of the Smart-DART handheld LAMP reaction devices (Diagenetix, Inc., Honolulu, HI, USA), connected to Android 4.4 enabled, Bluetooth-capable Nexus 7 tablets for output. Quantification by a quantitative PCR assay was assumed correct to compare the lab and grower qLAMP assay quantification. Growers were able to conduct and interpret qLAMP results; however, the Erysiphe necator inoculum quantification was unreliable using the beta-Smart-DART devices. The qLAMP assay developed was sensitive to one spore in early testing of the assay, but decreased to >20 spores by the end of the trial. The qLAMP assay is not likely a suitable management tool for grape powdery mildew due to losses in sensitivity and decreasing costs and portability for other, more reliable molecular tools.</p>
<p align="right"><i>score: 72</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiplex logic processing isothermal diagnostic assays for an evolving virus</h3>
<p style="text-align: justify; text-justify: inter-word;">While point-of-care (POC) diagnostic assays can be performed at or near the site of sample acquisition, they have for the most part been considered to be relatively simplistic tests that provide relatively little information to a clinician or public health worker. Familiar examples include electrochemical sensors for glucose, 1 or rapid immunoassays for metabolites such as human chorionic gonadotropin (the canonical pregnancy test), 2 and pathogens, either directly (influenza viruses) or via immune responses (antibodies against HIV-1/2). 3 The range of conditions and pathogens that could be tested for could likely be greatly expanded by developing POC diagnostics for nucleic acids 3 that would have greater sensitivity and accuracy. However, the current gold standard for molecular diagnostics, the quantitative polymerase chain reaction (qPCR), requires significant technical expertise and expensive and cumbersome equipment. Even portable instruments, such the Cepheid GeneXpert Omni, cost several thousand dollars and rely on expensive qPCR cartridge consumables for individual tests.</p>
<p align="right"><i>score: 71</i></p>

</div><br /><br /><br /><div class="card"><h3>Single Fluorescence Channel- based Multiplex Detection of Avian Influenza Virus by Quantitative PCR with Intercalating Dye OPEN</h3>
<p style="text-align: justify; text-justify: inter-word;">Since its invention in 1985 the polymerase chain reaction (PCR) has become a well-established method for amplification and detection of segments of double-stranded DNA. Incorporation of fluorogenic probe or DNA intercalating dyes (such as SYBR Green) into the PCR mixture allowed real-time reaction monitoring and extraction of quantitative information (qPCR). Probes with different excitation spectra enable multiplex qPCR of several DNA segments using multi-channel optical detection systems. Here we show multiplex qPCR using an economical EvaGreen-based system with single optical channel detection. Previously reported non quantitative multiplex realtime PCR techniques based on intercalating dyes were conducted once the PCR is completed by performing melting curve analysis (MCA). The technique presented in this paper is both qualitative and quantitative as it provides information about the presence of multiple DNA strands as well as the number of starting copies in the tested sample. Besides important internal control, multiplex qPCR also allows detecting concentrations of more than one DNA strand within the same sample. Detection of the avian influenza virus H7N9 by PCR is a well established method. Multiplex qPCR greatly enhances its specificity as it is capable of distinguishing both haemagglutinin (HA) and neuraminidase (NA) genes as well as their ratio.</p>
<p align="right"><i>score: 69</i></p>

</div><br /><br /><br /><div class="card"><h3>Label-Free Electrochemical Diagnosis of Viral Antigens with Genetically Engineered Fusion Protein</h3>
<p style="text-align: justify; text-justify: inter-word;">Sensors 2012, 12 10098 protein can directly bind onto the gold substrate with the strong binding affinity between the GBP and the gold surface, while the recognition site orients toward the sample for target binding at the same time. Furthermore, this one-step immobilization strategy greatly simplifies a fabrication process without any chemical modification as well as maintaining activity of biological recognition elements. This system allows specific immobilization of proteins and sensitive detection of targets, which were verified by surface plasmon resonance analysis and successfully applied to electrochemical cyclic voltammetry and impedance spectroscopy upto 0.14 ng/mL HBsAg.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Sequence specific visual detection of LAMP reactions by addition of cationic polymers</h3>
<p style="text-align: justify; text-justify: inter-word;">The presence or absence of minute amount of nucleic acid templates could be detected in a simple manner through visual assessment for the color of the LAMP amplicon-PEI complex precipitate. We conclude that this detection method may facilitate development of small and simple g-POCT device.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Electrospray ionisation-cleavable tandem nucleic acid mass tag-peptide nucleic acid conjugates: synthesis and applications to quantitative genomic analysis using electrospray ionisation-MS/MS</h3>
<p style="text-align: justify; text-justify: inter-word;">The synthesis and characterization of isotopomer tandem nucleic acid mass tag-peptide nucleic acid (TNT-PNA) conjugates is described along with their use as electrospray ionisation-cleavable (ESI-Cleavable) hybridization probes for the detection and quantification of target DNA sequences by electrospray ionisation tandem mass spectrometry (ESI-MS/MS). ESI-cleavable peptide TNT isotopomers were introduced into PNA oligonucleotide sequences in a total synthesis approach. These conjugates were evaluated as hybridization probes for the detection and quantification of immobilized synthetic target DNAs using ESI-MS/MS. In these experiments, the PNA portion of the conjugate acts as a hybridization probe, whereas the peptide TNT is released in a collision-based process during the ionization of the probe conjugate in the electrospray ion source. The cleaved TNT acts as a uniquely resolvable marker to identify and quantify a unique target DNA sequence. The method should be applicable to a wide variety of assays requiring highly multiplexed, quantitative DNA/RNA analysis, including gene expression monitoring, genetic profiling and the detection of pathogens.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>A Simple, Inexpensive Device for Nucleic Acid Amplification without Electricity-Toward Instrument- Free Molecular Diagnostics in Low-Resource Settings</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions/Significance: We demonstrate that an electricity-free heater based on exothermic chemical reactions and engineered phase change materials can successfully incubate isothermal NA amplification assays, and that the results of those assays are not significantly different from ones incubated in parallel in commercially available PCR instruments. These results clearly suggest the potential of the non-instrumented nucleic acid amplification (NINA) heater for molecular diagnostics in LRS. When combined with other innovations in development that eliminate power requirements for sample preparation, cold reagent storage, and readout, the NINA heater will comprise part of a kit that should enable electricity-free NA testing for many important analytes.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Picoliter Well Array Chip-Based Digital Recombinase Polymerase Amplification for Absolute Quantification of Nucleic Acids</h3>
<p style="text-align: justify; text-justify: inter-word;">Absolute, precise quantification methods expand the scope of nucleic acids research and have many practical applications. Digital polymerase chain reaction (dPCR) is a powerful method for nucleic acid detection and absolute quantification. However, it requires thermal cycling and accurate temperature control, which are difficult in resource-limited conditions. Accordingly, isothermal methods, such as recombinase polymerase amplification (RPA), are more attractive. We developed a picoliter well array (PWA) chip with 27,000 consistently sized picoliter reactions (314 pL) for isothermal DNA quantification using digital RPA (dRPA) at 39°C. Sample loading using a scraping liquid blade was simple, fast, and required small reagent volumes (i.e., <20 μL). Passivating the chip surface using a methoxy-PEG-silane agent effectively eliminated cross-contamination during dRPA. Our creative optical design enabled wide-field fluorescence imaging in situ and both end-point and real-time analyses of picoliter wells in a 6-cm 2 area. It was not necessary to use scan shooting and stitch serial small images together. Using this method, we quantified serial dilutions of a Listeria monocytogenes gDNA stock solution from 9 × 10 -1 to 4 × 10 -3 copies per well with an average error of less than 11% (N = 15). Overall dRPA-on-chip processing required less than 30 min, which was a 4-fold decrease compared to dPCR, requiring approximately 2 h. dRPA on the PWA chip provides a simple and highly sensitive method to quantify nucleic acids without thermal cycling or precise micropump/microvalve control. It has applications in fast field analysis and critical clinical diagnostics under resource-limited settings.</p>
<p align="right"><i>score: 65</i></p>

</div><br /><br /><br /><div class="card"><h3>Advances in Directly Amplifying Nucleic Acids from Complex Samples</h3>
<p style="text-align: justify; text-justify: inter-word;">Advances in nucleic acid amplification technologies have revolutionized diagnostics for systemic, inherited, and infectious diseases. Current assays and platforms, however, often require lengthy experimental procedures and multiple instruments to remove contaminants and inhibitors from clinically-relevant, complex samples. This requirement of sample preparation has been a bottleneck for using nucleic acid amplification tests (NAATs) at the point of care (POC), though advances in "lab-on-chip" platforms that integrate sample preparation and NAATs have made great strides in this space. Alternatively, direct NAATs-techniques that minimize or even bypass sample preparation-present promising strategies for developing POC diagnostic tools for analyzing real-world samples. In this review, we discuss the current status of direct NAATs. Specifically, we surveyed potential testing systems published from 1989 to 2017, and analyzed their performances in terms of robustness, sensitivity, clinical relevance, and suitability for POC diagnostics. We introduce bubble plots to facilitate our analysis, as bubble plots enable effective visualization of the performances of these direct NAATs. Through our review, we hope to initiate an in-depth examination of direct NAATs and their potential for realizing POC diagnostics, and ultimately transformative technologies that can further enhance healthcare.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Test Paper for Colorimetric Inspection of Fatty Acids and Edible Oils</h3>
<p style="text-align: justify; text-justify: inter-word;">Recently, test paper-based colorimetric sensors have been widely applicated in various fields such as disease diagnosis [23-26], environmental monitoring [27,28] and food safety inspection [29, 30] due to their simplicity, disposability, economical efficiency and minimal samples requirement [31] . More importantly, they are easier to use in in situ inspections. According to the principle of paper-based colorimetric sensing, some specific reagents are necessary to develop color intensity and/or hue correlates with types and concentration of analytes [32] . Herein, considering the Ag + mediated color variation of TMB and selective binding of Ag + with olefinic bonds, we propose a test paper for FAs inspection (Scheme 1). By mixing Ag + with FAs in a solution, a part of Ag + could form Ag + -FAs complexes and the remnant part is free Ag + ions. The free Ag + could further oxidize TMB and develop blue colors. The proportion of free Ag + is dependent on the concentration and type of FAs. Therefore, by taking advantage of the complexation difference of Ag + to olefinic bonds of FAs, we successfully identified five types of model FAs with subtle structural differences. Using the approach described herein, the structural variation of FAs with heat treatment can be successfully monitored. Furthermore, the quality changes of mixture FAs in edible hot pot oils with heat treatment were successfully monitored by the test paper presented herein.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Controlled fluorescence quenching by antibody-conjugated graphene oxide to measure tau protein</h3>
<p style="text-align: justify; text-justify: inter-word;">In contrast with traditional enzyme-linked immunosorbent assay (ELISA), our method does not require enzyme-linked secondary antibodies for protein recognition nor does it require an enzyme substrate for optical signal generation. This requires fewer reagents and has less systematic error than the antigenantibody recognition steps in ELISA. Our method has a tau protein detection limit of 0.14 pmol ml −1 in buffer. This approach could be developed into a promising biosensor for the detection of tau protein and may be useful in the clinical diagnosis of tau-induced neurodegeneration syndromes.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>The IRIDICA BAC BSI Assay: Rapid, Sensitive and Culture-Independent Identification of Bacteria and Candida in Blood</h3>
<p style="text-align: justify; text-justify: inter-word;">Bloodstream infection (BSI) and sepsis are rising in incidence throughout the developed world. The spread of multi-drug resistant organisms presents increasing challenges to treatment. Surviving BSI is dependent on rapid and accurate identification of causal organisms, and timely application of appropriate antibiotics. Current culture-based methods used to detect and identify agents of BSI are often too slow to impact early therapy and may fail to detect relevant organisms in many positive cases. Existing methods for direct molecular detection of microbial DNA in blood are limited in either sensitivity (likely the result of small sample volumes) or in breadth of coverage, often because the PCR primers and probes used target only a few specific pathogens. There is a clear unmet need for a sensitive molecular assay capable of identifying the diverse bacteria and yeast associated with BSI directly from uncultured whole blood samples. We have developed a method of extracting DNA from larger volumes of whole blood (5 ml per sample), amplifying multiple widely conserved bacterial and fungal genes using a mismatch-and background-tolerant PCR chemistry, and identifying hundreds of diverse organisms from the amplified fragments on the basis of species-specific genetic signatures using electrospray ionization mass spectrometry (PCR/ESI-MS). We describe the analytical characteristics of the IRIDICA BAC BSI Assay and compare its preclinical performance to current standard-of-care methods in a collection of prospectively collected blood specimens from patients with symptoms of sepsis. The assay generated matching results in 80% of culture-positive cases (86% when common contaminants were excluded from the analysis), and twice the total number of positive detections. The described method is capable of providing organism identifications directly from uncultured blood in less than 8 hours. Disclaimer: The IRIDICA BAC BSI Assay is not available in the United States.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Multiplex logic processing isothermal diagnostic assays for an evolving virus</h3>
<p style="text-align: justify; text-justify: inter-word;">The ease-of-use with minimal instrumentation, broad programmability, and built-in fail-safe reliability make our smart molecular diagnostic attractive for POC use.</p>
<p align="right"><i>score: 61</i></p>

</div><br /><br /><br /><div class="card"><h3>Quantitative and Chemical Fingerprint Analysis for the Quality Evaluation of Isatis indigotica based on Ultra-Performance Liquid Chromatography with Photodiode Array Detector Combined with Chemometric Methods</h3>
<p style="text-align: justify; text-justify: inter-word;">A simple and reliable method of ultra-performance liquid chromatography with photodiode array detector (UPLC-PDA) was developed to control the quality of Radix Isatidis (dried root of Isatis indigotica) for chemical fingerprint analysis and quantitative analysis of eight bioactive constituents, including R,S-goitrin, progoitrin, epiprogoitrin, gluconapin, adenosine, uridine, guanosine, and hypoxanthine. In quantitative analysis, the eight components showed good regression (R > 0.9997) within test ranges, and the recovery method ranged from 99.5% to 103.0%. The UPLC fingerprints of the Radix Isatidis samples were compared by performing chemometric procedures, including similarity analysis, hierarchical clustering analysis, and principal component analysis. The chemometric procedures classified Radix Isatidis and its finished products such that all samples could be successfully grouped according to crude herbs, prepared slices, and adulterant Baphicacanthis cusiae Rhizoma et Radix. The combination of quantitative and chromatographic fingerprint analysis can be used for the quality assessment of Radix Isatidis and its finished products.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Test Paper for Colorimetric Inspection of Fatty Acids and Edible Oils</h3>
<p style="text-align: justify; text-justify: inter-word;">Fatty acids (FAs) are of interest to the areas of food science and medicine because they are important dietary sources of fuel for animals and play important roles in many biological processes. The health effects of FAs are different due to the diversity of olefinic bonds in the alkyl chains including number, position and configuration. However, the discrimination of FAs is difficult from a chemical sensing perspective due to the lack of diversity in terms of functional groups. Until now, only a few chemosensors have been developed for selective sensing of FAs based on their overall shape, however they are still limited in discrimination of FAs with subtle structural differences, moreover, they cannot be used for rapid and in situ inspections. Herein, for the first time, we designed a test paper for in situ colorimetric inspection for FAs based on the combination of the highly selective binding of Ag + to olefinic bonds and Ag + mediated color variation of 3,3 ,5,5 ,-tetramethylbenzidine. As a result, the sensor exhibited high sensitivity and good selectivity for five FAs with subtle structural differences. Furthermore, our method described herein was successfully applied to monitor the structural variations of FAs and quality changes in mixture edible hot pot oils with heat treatment in time course. Hence, the test paper presented herein holds great potential in the inspection of fats and edible oils in food industries. Therefore, the approaches to determine FAs profile in food industries are important in order to correct nutrition labeling and control of labeling authenticity, which is of great significance to the implementation of the plan of REPLACE.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Sequence specific visual detection of LAMP reactions by addition of cationic polymers</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Development of a practical gene point-of-care testing device (g-POCT device) requires innovative detection methods for demonstrating the results of the gene amplification reaction without the use of expensive equipment. We have studied a new method for the sequence-specific visual detection of minute amounts of nucleic acids using precipitation reaction by addition of cationic polymers to amplicons of Loop mediated isothermal Amplification (LAMP).</p>
<p align="right"><i>score: 59</i></p>

</div><br /><br /><br /></body>