
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: experimental infections to test host range for this pathogen</h2><br /></div><br /><br /><br /><div class="card"><h3>Bacterial Pathogen Emergence Requires More than Direct Contact with a Novel Passerine Host</h3>
<p style="text-align: justify; text-justify: inter-word;">While direct contact may sometimes be sufficient to allow a pathogen to jump into a new host species, in other cases, fortuitously adaptive mutations that arise in the original donor host are also necessary. Viruses have been the focus of most host shift studies, so less is known about the importance of ecological versus evolutionary processes to successful bacterial host shifts. Here we tested whether direct contact with the novel host was sufficient to enable the mid-1990s jump of the bacterium Mycoplasma gallisepticum from domestic poultry to house finches (Haemorhous mexicanus). We experimentally inoculated house finches with two genetically distinct M. gallisepticum strains obtained either from poultry (Rlow) or from house finches (HF1995) during an epizootic outbreak. All 15 house finches inoculated with HF1995 became infected, whereas Rlow successfully infected 12 of 15 (80%) inoculated house finches. Comparisons among infected birds showed that, relative to HF1995, Rlow achieved substantially lower bacterial loads in the host respiratory mucosa and was cleared faster. Furthermore, Rlow-infected finches were less likely to develop clinical symptoms than HF1995-infected birds and, when they did, displayed milder conjunctivitis. The lower infection success of Rlow relative to HF1995 was not, however, due to a heightened host antibody response to Rlow. Taken together, our results indicate that contact between infected poultry and house finches was not, by itself, sufficient to explain the jump of M. gallisepticum to house finches. Instead, mutations arising in the original poultry host would have been necessary for successful pathogen emergence in the novel finch host.</p>
<p align="right"><i>score: 108</i></p>

</div><br /><br /><br /><div class="card"><h3>Isolation and Identification of Feline Herpesvirus Type 1 from a South China Tiger in China</h3>
<p style="text-align: justify; text-justify: inter-word;">Our finding extends the host range of FHV-1 and has implications for FHV-1 infection and South China tiger conservation.</p>
<p align="right"><i>score: 79</i></p>

</div><br /><br /><br /><div class="card"><h3>Foodborne Transmission of Deformed Wing Virus to Ants (Myrmica rubra)</h3>
<p style="text-align: justify; text-justify: inter-word;">Here, we empirically test for the first time whether ants, Myrmica rubra, which are common in Europe [46] and an invasive species in North America [47], can be alternative hosts of DWV and if foodborne transmission is feasible. Monitoring of viral loads after consumption of virus-infected food will shed light on the fate of the viruses. If negative-sense strands of the RNA viruses can be found, this would suggest virus replication [48, 49] .</p>
<p align="right"><i>score: 76</i></p>

</div><br /><br /><br /><div class="card"><h3>Changes in temperature alter the potential outcomes of virus host shifts</h3>
<p style="text-align: justify; text-justify: inter-word;">Host shifts-where a pathogen jumps between different host species-are an important source of emerging infectious disease. With on-going climate change there is an increasing need to understand the effect changes in temperature may have on emerging infectious disease. We investigated whether species' susceptibilities change with temperature and ask if susceptibility is greatest at different temperatures in different species. We infected 45 species of Drosophilidae with an RNA virus and measured how viral load changes with temperature. We found the host phylogeny explained a large proportion of the variation in viral load at each temperature, with strong phylogenetic correlations between viral loads across temperature. The variance in viral load increased with temperature, while the mean viral load did not. This suggests that as temperature increases the most susceptible species become more susceptible, and the least susceptible less so. We found no significant relationship between a species' susceptibility across temperatures, and proxies for thermal optima (critical thermal maximum and minimum or basal metabolic rate). These results suggest that whilst the rank order of species susceptibilities may remain the same with changes in temperature, some species may become more susceptible to a novel pathogen, and others less so.</p>
<p align="right"><i>score: 75</i></p>

</div><br /><br /><br /><div class="card"><h3>Development of Onchocerca volvulus in humanized NSG mice and detection of parasite biomarkers in urine and serum</h3>
<p style="text-align: justify; text-justify: inter-word;">The study of Onchocerca volvulus has been limited by its host range, with only humans and non-human primates shown to be susceptible to the full life cycle infection. Small animal models that support the development of adult parasites have not been identified.</p>
<p align="right"><i>score: 70</i></p>

</div><br /><br /><br /><div class="card"><h3>A Single Residue in Ebola Virus Receptor NPC1 Influences Cellular Host Range in Reptiles crossmark</h3>
<p style="text-align: justify; text-justify: inter-word;">Filoviruses are the causative agents of an increasing number of disease outbreaks in human populations, including the current unprecedented Ebola virus disease (EVD) outbreak in western Africa. One obstacle to controlling these epidemics is our poor understanding of the host range of filoviruses and their natural reservoirs. Here, we investigated the role of the intracellular filovirus receptor, Niemann-Pick C1 (NPC1) as a molecular determinant of Ebola virus (EBOV) host range at the cellular level. Whereas human cells can be infected by EBOV, a cell line derived from a Russell's viper (Daboia russellii) (VH-2) is resistant to infection in an NPC1-dependent manner. We found that VH-2 cells are resistant to EBOV infection because the Russell's viper NPC1 ortholog bound poorly to the EBOV spike glycoprotein (GP). Analysis of panels of viper-human NPC1 chimeras and point mutants allowed us to identify a single amino acid residue in NPC1, at position 503, that bidirectionally influenced both its binding to EBOV GP and its viral receptor activity in cells. Significantly, this single residue change perturbed neither NPC1's endosomal localization nor its housekeeping role in cellular cholesterol trafficking. Together with other recent work, these findings identify sequences in NPC1 that are important for viral receptor activity by virtue of their direct interaction with EBOV GP and suggest that they may influence filovirus host range in nature. Broader surveys of NPC1 orthologs from vertebrates may delineate additional sequence polymorphisms in this gene that control susceptibility to filovirus infection.</p>
<p align="right"><i>score: 68</i></p>

</div><br /><br /><br /><div class="card"><h3>Contrasting Patterns in Mammal-Bacteria Coevolution: Bartonella and Leptospira in Bats and Rodents</h3>
<p style="text-align: justify; text-justify: inter-word;">Conclusions/Significance: In both bat and rodent hosts, Bartonella exhibits significant coevolution with minimal host switching, while Leptospira in bats lacks evolutionary congruence with its host and has high number of host switches. Reasons underlying these variable coevolutionary patterns in host range are likely due to differences in disease-specific transmission and host ecology. Understanding the coevolutionary patterns and frequency of host-switching events between bacterial pathogens and their hosts will allow better prediction of spillover between mammal reservoirs, and ultimately to humans.</p>
<p align="right"><i>score: 67</i></p>

</div><br /><br /><br /><div class="card"><h3>Middle East Respiratory Syndrome Coronavirus Intra-Host Populations Are Characterized by Numerous High Frequency Variants</h3>
<p style="text-align: justify; text-justify: inter-word;">Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging human pathogen related to SARS virus. In vitro studies indicate this virus may have a broad host range suggesting an increased pandemic potential. Genetic and epidemiological evidence indicate camels serve as a reservoir for MERS virus but the mechanism of cross species transmission is unclear and many questions remain regarding the susceptibility of humans to infection. Deep sequencing data was obtained from the nasal samples of three camels that had been experimentally infected with a human MERS-CoV isolate. A majority of the genome was covered and average coverage was greater than 12,000x depth. Although only 5 mutations were detected in the consensus sequences, 473 intrahost single nucleotide variants were identified. Many of these variants were present at high frequencies and could potentially influence viral phenotype and the sensitivity of detection assays that target these regions for primer or probe binding.</p>
<p align="right"><i>score: 66</i></p>

</div><br /><br /><br /><div class="card"><h3>Systemic Spread and Propagation of a Plant-Pathogenic Virus in European Honeybees, Apis mellifera</h3>
<p style="text-align: justify; text-justify: inter-word;">Emerging and reemerging diseases that result from pathogen host shifts are a threat to the health of humans and their domesticates. RNA viruses have extremely high mutation rates and thus represent a significant source of these infectious diseases. In the present study, we showed that a plant-pathogenic RNA virus, tobacco ringspot virus (TRSV), could replicate and produce virions in honeybees, Apis mellifera, resulting in infections that were found throughout the entire body. Additionally, we showed that TRSV-infected individuals were continually present in some monitored colonies. While intracellular life cycle, species-level genetic variation, and pathogenesis of the virus in honeybee hosts remain to be determined, the increasing prevalence of TRSV in conjunction with other bee viruses from spring toward winter in infected colonies was associated with gradual decline of host populations and winter colony collapse, suggesting the negative impact of the virus on colony survival. Furthermore, we showed that TRSV was also found in ectoparasitic Varroa mites that feed on bee hemolymph, but in those instances the virus was restricted to the gastric cecum of Varroa mites, suggesting that Varroa mites may facilitate the spread of TRSV in bees but do not experience systemic invasion. Finally, our phylogenetic analysis revealed that TRSV isolates from bees, bee pollen, and Varroa mites clustered together, forming a monophyletic clade.</p>
<p align="right"><i>score: 64</i></p>

</div><br /><br /><br /><div class="card"><h3>Contrasting Patterns in Mammal-Bacteria Coevolution: Bartonella and Leptospira in Bats and Rodents</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Emerging bacterial zoonoses in bats and rodents remain relatively understudied. We conduct the first comparative host-pathogen coevolutionary analyses of bacterial pathogens in these hosts, using Bartonella spp. and Leptospira spp. as a model.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Combining field epidemiological information and genetic data to comprehensively reconstruct the invasion history and the microevolution of the sudden oak death agent Phytophthora ramorum (Stramenopila: Oomycetes) in California</h3>
<p style="text-align: justify; text-justify: inter-word;">Understanding the migration patterns of invasive organisms is of paramount importance to predict and prevent their further spread. Previous attempts at reconstructing the entire history of the sudden oak death (SOD) epidemic in California were limited by: (1) incomplete sampling; (2) the inability to include infestations caused by a single genotype of the pathogen; (3) collapsing of non-spatially contiguous yet genetically similar samples into large metasamples that confounded the coalescent analyses. Here, we employ an intensive sampling coverage of 832 isolates of Phytopthora ramorum (the causative agent of SOD) from 60 California forests, genotyped at nine microsatellite loci, to reconstruct its invasion. By using age of infestation as a constraint on coalescent analyses, by dividing genetically indistinguishable meta-populations into highly-resolved sets of spatially contiguous populations, and by using Bruvo genetic distances for most analyses, we reconstruct the entire history of the epidemic and convincingly show infected nursery plants are the original source for the entire California epidemic. Results indicate that multiple human-mediated introductions occurred in most counties and that further disease sources were represented by large wild infestations. The study also identifies minor introductions, some of them relatively recent, linked to infected ornamental plants. Finally, using archival isolates collected soon after the discovery of the pathogen in California, we corroborate that the epidemic is likely to have resulted form 3 to 4 core founder individuals evolved from a single genotype. This is probably the most complete reconstruction ever completed for an invasion by an exotic forest pathogen, and the approach here described may be useful for the reconstruction of invasions by any clonally reproducing organism with a relatively limited natural dispersal range.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>A Serological Survey of Infectious Disease in Yellowstone National Park's Canid Community</h3>
<p style="text-align: justify; text-justify: inter-word;">Methodology/Principal Findings: We analyzed sympatric wolf, coyote (Canis latrans), and red fox (Vulpes vulpes) serologic data from YNP, spanning 1991-2007, to identify long-term patterns of pathogen exposure, identify associated risk factors, and examine evidence for disease-induced mortality among wolves for which there were survival data. We found high, constant exposure to canine parvovirus (wolf seroprevalence: 100%; coyote: 94%), canine adenovirus-1 (wolf pups [0.5-0.9 yr]: 91%, adults [$1 yr]: 96%; coyote juveniles [0.5-1.5 yrs]: 18%, adults [$1.6 yrs]: 83%), and canine herpesvirus (wolf: 87%; coyote juveniles: 23%, young adults [1.6-4.9 yrs]: 51%, old adults [$5 yrs]: 87%) suggesting that these pathogens were enzootic within YNP wolves and coyotes. An average of 50% of wolves exhibited exposure to the protozoan parasite, Neospora caninum, although individuals' odds of exposure tended to increase with age and was temporally variable. Wolf, coyote, and fox exposure to canine distemper virus (CDV) was temporally variable, with evidence for distinct multi-host outbreaks in 1999 and 2005, and perhaps a smaller, isolated outbreak among wolves in the interior of YNP in 2002. The years of high wolf-pup mortality in 1999 and 2005 in the northern region of the park were correlated with peaks in CDV seroprevalence, suggesting that CDV contributed to the observed mortality.</p>
<p align="right"><i>score: 63</i></p>

</div><br /><br /><br /><div class="card"><h3>Serological evidence and experimental infection of cynomolgus macaques with pteropine orthoreovirus reveal monkeys as potential hosts for transmission to humans</h3>
<p style="text-align: justify; text-justify: inter-word;">Pteropine orthoreoviruses (PRV) are emerging bat-borne viruses with proven zoonotic transmission. We recently demonstrated human exposure to PRV in Singapore, which together with previous reports from Malaysia and Vietnam suggest that human infection of PRV may occur periodically in the region. This raises the question whether bats are the only sources of human infection. In this study, we screened 517 cynomolgus macaques caught in Singapore for evidence of exposure to PRV3M (also known as Melaka virus), which was first isolated from human patients in Melaka, Malaysia. We found that 67 serum samples were PRV3M positive by ELISA and 34 were also positive by virus neutralization assay. To investigate whether monkeys could act as hosts for PRV transmission, we experimentally infected cynomolgus macaques with PRV3M and housed these animals with uninfected monkeys. Although no clinical signs of infection were observed in infected animals, viral RNA was detected in nasal and rectal swabs and all infected macaques seroconverted. Additionally, one of the uninfected animals seroconverted, implying active shedding and transmission of PRV3M. We provide evidence that PRV exposure in the macaque population in Singapore occurs at a relatively high prevalence and this study suggests that cynomolgus macaques may be an intermediate or reservoir host for PRVs.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Direct nucleic acid analysis of mosquitoes for high fidelity species identification and 1 detection of Wolbachia using a cellphone 2 3</h3>
<p style="text-align: justify; text-justify: inter-word;">Mosquitoes spread many human pathogens and novel approaches are required to reduce the 45 burden of mosquito-borne disease. One promising approach is transferring Wolbachia into 46</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>CEACAM1 recognition by bacterial pathogens is species-specific</h3>
<p style="text-align: justify; text-justify: inter-word;">Our results establish the selective interaction of several human-restricted bacterial pathogens with human CEACAM1 and suggest that co-evolution of microbial adhesins with their corresponding receptors on mammalian cells contributes to the limited host range of these highly adapted infectious agents.</p>
<p align="right"><i>score: 62</i></p>

</div><br /><br /><br /><div class="card"><h3>Host range, host specificity and hypothesized host shift events among viruses of lower vertebrates</h3>
<p style="text-align: justify; text-justify: inter-word;">The successful replication of a viral agent in a host is a complex process that often leads to a species specificity of the virus and can make interspecies transmission difficult. Despite this difficulty, natural host switch seems to have been frequent among viruses of lower vertebrates, especially fish viruses, since there are several viruses known to be able to infect a wide range of species. In the present review we will focus on well documented reports of broad host range, variations in host specificity, and host shift events hypothesized for viruses within the genera Ranavirus, Novirhabdovirus, Betanodavirus, Isavirus, and some herpesvirus.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Origin and Evolution of the Kiwifruit Canker Pandemic</h3>
<p style="text-align: justify; text-justify: inter-word;">Recurring epidemics of kiwifruit (Actinidia spp.) bleeding canker disease are caused by Pseudomonas syringae pv. actinidiae (Psa). In order to strengthen understanding of population structure, phylogeography, and evolutionary dynamics, we isolated Pseudomonas from cultivated and wild kiwifruit across six provinces in China. Based on the analysis of 80 sequenced Psa genomes, we show that China is the origin of the pandemic lineage but that strain diversity in China is confined to just a single clade. In contrast, Korea and Japan harbor strains from multiple clades. Distinct independent transmission events marked introduction of the pandemic lineage into New Zealand, Chile, Europe, Korea, and Japan. Despite high similarity within the core genome and minimal impact of within-clade recombination, we observed extensive variation even within the single clade from which the global pandemic arose.</p>
<p align="right"><i>score: 60</i></p>

</div><br /><br /><br /><div class="card"><h3>Assessment of Virally Vectored Autoimmunity as a Biocontrol Strategy for Cane Toads</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: The cane toad, Bufo (Chaunus) marinus, is one of the most notorious vertebrate pests introduced into Australia over the last 200 years and, so far, efforts to identify a naturally occurring B. marinus-specific pathogen for use as a biological control agent have been unsuccessful. We explored an alternative approach that entailed genetically modifying a pathogen with broad host specificity so that it no longer caused disease, but carried a gene to disrupt the cane toad life cycle in a species specific manner.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Experimental Cross-Species Infection of Common Marmosets by Titi Monkey Adenovirus</h3>
<p style="text-align: justify; text-justify: inter-word;">Adenoviruses are DNA viruses that infect a number of vertebrate hosts and are associated with both sporadic and epidemic disease in humans. We previously identified a novel adenovirus, titi monkey adenovirus (TMAdV), as the cause of a fulminant pneumonia outbreak in a colony of titi monkeys (Callicebus cupreus) at a national primate center in 2009. Serological evidence of infection by TMAdV was also found in a human researcher at the facility and household family member, raising concerns for potential cross-species transmission of the virus. Here we present experimental evidence of cross-species TMAdV infection in common marmosets (Callithrix jacchus). Nasal inoculation of a cell cultured-adapted TMAdV strain into three marmosets produced an acute, mild respiratory illness characterized by low-grade fever, reduced activity, anorexia, and sneezing. An increase in virus-specific neutralization antibody titers accompanied the development of clinical signs. Although serially collected nasal swabs were positive for TMAdV for at least 8 days, all 3 infected marmosets spontaneously recovered by day 12 post-inoculation, and persistence of the virus in tissues could not be established. Thus, the pathogenesis of experimental inoculation of TMAdV in common marmosets resembled the mild, self-limiting respiratory infection typically seen in immunocompetent human hosts rather than the rapidly progressive, fatal pneumonia observed in 19 of 23 titi monkeys during the prior 2009 outbreak. These findings further establish the potential for adenovirus crossspecies transmission and provide the basis for development of a monkey model useful for assessing the zoonotic potential of adenoviruses.</p>
<p align="right"><i>score: 58</i></p>

</div><br /><br /><br /><div class="card"><h3>Phylogenetic Insight into Zika and Emerging Viruses for a Perspective on Potential Hosts</h3>
<p style="text-align: justify; text-justify: inter-word;">For example, examining flaviviruses closely related to ZIKV, the phylogenetic framework indicates New World monkeys are the most likely candidates to be potential reservoirs for ZIKV. Secondarily, mammals that are in close proximity to humans should be considered because of the increased opportunity for pathogen exchange.</p>
<p align="right"><i>score: 56</i></p>

</div><br /><br /><br /></body>