
<head>
<style>
.card {
  /* Add shadows to create the "card" effect */
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;

}

.card-query {
    background: #EEEEFF;
    box-shadow: 0 0px 8px 0 rgba(0,0,0,0.2);
}

.button {
  background-color: #EEEEEE;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
}

</style>
</head>
<body style="margin-left: 10%; margin-right: 10%; padding-top: 70px;">
<div class="card card-query"><h2>Query: basic reproductive number of the virus</h2><br /></div><br /><br /><br /><div class="card"><h3>Estimation of basic reproduction number of the Middle East respiratory syndrome coronavirus (MERS-CoV) during the outbreak in South Korea, 2015</h3>
<p style="text-align: justify; text-justify: inter-word;">: We fit the model to the epidemic data of the South Korea outbreak minimizing the sum of the squared errors to identify model parameters. Also we derive the basic reproductive number as the terms of the parameters of the SIR model. Then we determine the basic reproduction number of the MERS-CoV in South Korea in 2015 as 8.0977. It is worth comparing with the basic reproductive number of the 2014 Ebola outbreak in West Africa including Guinea, Sierra Leone, and Liberia, which had values of 1.5-2.5. Conclusions: There was no intervention to control the infection in the early phase of the outbreak, thus the data used here provide the best conditions to evaluate the epidemic characteristics of MERS, such as the basic reproduction number. An evaluation of basic reproduction number using epidemic data could be problematic if there are stochastic fluctuations in the early phase of the outbreak, or if the report is not accurate and there is bias in the data. Such problems are not relevant to this study because the data used here were precisely reported and verified by Korea Hospital Association.</p>
<p align="right"><i>score: 140</i></p>

</div><br /><br /><br /><div class="card"><h3>Theoretical conditions for the coexistence of viral strains with differences in phenotypic traits: a bifurcation analysis</h3>
<p style="text-align: justify; text-justify: inter-word;">We investigate the dynamics of a wild-type viral strain which generates mutant strains differing in phenotypic properties for infectivity, virulence and mutation rates. We study, by means of a mathematical model and bifurcation analysis, conditions under which the wild-type and mutant viruses, which compete for the same host cells, can coexist. The coexistence conditions are formulated in terms of the basic reproductive numbers of the strains, a maximum value of the mutation rate and the virulence of the pathogens. The analysis reveals that parameter space can be divided into five regions, each with distinct dynamics, that are organized around degenerate Bogdanov-Takens and zero-Hopf bifurcations, the latter of which gives rise to a curve of transcritical bifurcations of periodic orbits. These results provide new insights into the conditions by which viral populations may contain multiple coexisting strains in a stable manner. environmental changes such as new host species, the presence of antiviral drugs, new transmission routes or to new vectors [1]. This tremendous evolvability, coupled with densely populated animal and plant susceptible hosts (in many cases lacking genetic variability for resistance to infection), are the reasons for the persistence and emergence of new viral diseases or the re-emergence of new strains with novel properties of already known diseases. The continuous emergence of new mutants leads to an overlap in existence of wild-type (hereafter referred to as wt) and mutant genotypes of the same virus within individual infected hosts [2, 3] . This cloud of mutants is usually known as a viral quasi-species [1] and it constitutes the target of selection, instead of the individual viral genomes. This reservoir of coexisting genetic variants may lead to the emergence of new genotypes with different host ranges, pathologies and epidemiological properties that may result in outbreaks [4, 5] .</p>
<p align="right"><i>score: 139</i></p>

</div><br /><br /><br /><div class="card"><h3>(no title)</h3>
<p style="text-align: justify; text-justify: inter-word;">severe acute respiratory syndrome (SARS) outbreak was based on rapid diagnosis coupled with effective patient isolation. We used uncertainty and sensitivity analysis of the basic reproductive number R 0 to assess the role that model parameters play in outbreak control. The transmission rate and isolation effectiveness have the largest fractional effect on R 0 . We estimated the distribution of the reproductive number R 0 under perfect isolation conditions. The distribution lies in the interquartile range 0.19-1.08, with a median of 0.49. Even though the median of R 0 is <1, we found that 25% of our R 0 distribution lies at R 0 > 1, even with perfect isolation. This implies the need to simultaneously apply more than one method of control.</p>
<p align="right"><i>score: 138</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of basic reproduction number of the Middle East respiratory syndrome coronavirus (MERS-CoV) during the outbreak in South Korea, 2015</h3>
<p style="text-align: justify; text-justify: inter-word;">The basic reproduction number of an epidemic is defined as the average number of secondary cases that an infected subject produces over its infectious period in a susceptible and uninfected population. To estimate the basic reproduction number of the MERS-CoV, we employ data from the 2015 South Korea MERS outbreak and the susceptible-infected-removed (SIR) model, a mathematical model that uses a set of ordinary differential equations (ODEs).</p>
<p align="right"><i>score: 137</i></p>

</div><br /><br /><br /><div class="card"><h3>The within-host viral kinetics of SARS-CoV-2</h3>
<p style="text-align: justify; text-justify: inter-word;">In this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</p>
<p align="right"><i>score: 136</i></p>

</div><br /><br /><br /><div class="card"><h3>How Can Viral Dynamics Models Inform Endpoint Measures in Clinical Trials of Therapies for Acute Viral Infections?</h3>
<p style="text-align: justify; text-justify: inter-word;">Acute viral infections pose many practical challenges for the accurate assessment of the impact of novel therapies on viral growth and decay. Using the example of influenza A, we illustrate how the measurement of infection-related quantities that determine the dynamics of viral load within the human host, can inform investigators on the course and severity of infection and the efficacy of a novel treatment. We estimated the values of key infectionrelated quantities that determine the course of natural infection from viral load data, using Markov Chain Monte Carlo methods. The data were placebo group viral load measurements collected during volunteer challenge studies, conducted by Roche, as part of the oseltamivir trials. We calculated the values of the quantities for each patient and the correlations between the quantities, symptom severity and body temperature. The greatest variation among individuals occurred in the viral load peak and area under the viral load curve. Total symptom severity correlated positively with the basic reproductive number. The most sensitive endpoint for therapeutic trials with the goal to cure patients is the duration of infection. We suggest laboratory experiments to obtain more precise estimates of virological quantities that can supplement clinical endpoint measurements.</p>
<p align="right"><i>score: 128</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimation of basic reproduction number of the Middle East respiratory syndrome coronavirus (MERS-CoV) during the outbreak in South Korea, 2015</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: In South Korea, an outbreak of Middle East respiratory syndrome (MERS) occurred in 2015. It was the second largest MERS outbreak. As a result of the outbreak in South Korea, 186 infections were reported, and 36 patients died. At least 16,693 people were isolated with suspicious symptoms. This paper estimates the basic reproduction number of the MERS coronavirus (CoV), using data on the spread of MERS in South Korea.</p>
<p align="right"><i>score: 125</i></p>

</div><br /><br /><br /><div class="card"><h3>The Impact of Prior Information on Estimates of Disease Transmissibility Using Bayesian Tools</h3>
<p style="text-align: justify; text-justify: inter-word;">The basic reproductive number (R₀) and the distribution of the serial interval (SI) are often used to quantify transmission during an infectious disease outbreak. In this paper, we present estimates of R₀ and SI from the 2003 SARS outbreak in Hong Kong and Singapore, and the 2009 pandemic influenza A(H1N1) outbreak in South Africa using methods that expand upon an existing Bayesian framework. This expanded framework allows for the incorporation of additional information, such as contact tracing or household data, through prior distributions. The results for the R₀ and the SI from the influenza outbreak in South Africa were similar regardless of the prior information (R 0 = 1.36-1.46,m = 2.0-2.7,m = mean of the SI). The estimates of R₀ and μ for the SARS outbreak ranged from 2.0-4.4 and 7.4-11.3, respectively, and were shown to vary depending on the use of contact tracing data. The impact of the contact tracing data was likely due to the small number of SARS cases relative to the size of the contact tracing sample.</p>
<p align="right"><i>score: 124</i></p>

</div><br /><br /><br /><div class="card"><h3>Improving the estimation of the death rate of infected cells from time course data during the acute phase of virus infections: application to acute HIV-1 infection in a humanized mouse model</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: Mathematical modeling of virus dynamics has provided quantitative insights into viral infections such as influenza, the simian immunodeficiency virus/ human immunodeficiency virus, hepatitis B, and hepatitis C. Through modeling, we can estimate the half-life of infected cells, the exponential growth rate, and the basic reproduction number (R 0 ). To calculate R 0 from virus load data, the death rate of productively infected cells is required. This can be readily estimated from treatment data collected during the chronic phase, but is difficult to determine from acute infection data. Here, we propose two new models that can reliably estimate the average life span of infected cells from acute-phase data, and apply both methods to experimental data from humanized mice infected with HIV-1. Methods: Both new models, called as the reduced quasi-steady state (RQS) model and the piece-wise regression (PWR) model, are derived by simplification of a standard model for the acute-phase dynamics of target cells, viruses and infected cells. By having only a limited number of parameters, both models allow us to reliably estimate the death rate of productively infected cells. Simulated datasets with plausible parameter values are generated with the standard model to compare the performance of the new models with that of the major previous model (i.e., the simple exponential model). Finally, we fit models to time course data from HIV-1 infected humanized mice to estimate the several important parameters characterizing their acute infection. Results and conclusions: The new models provided much better estimates than the previous model because they more precisely capture the de novo infection process. Both models describe the acute phase of HIV-1 infected humanized mice reasonably well, and we estimated an average death rate of infected cells of 0.61 and 0.61, an average exponential growth rate of 0.69 and 0.76, and an average basic reproduction number of 2.30 and 2.38 in the RQS model and the PWR model, respectively. These estimates are fairly close to those obtained in humans.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak</h3>
<p style="text-align: justify; text-justify: inter-word;">A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number R 0 -the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of R 0 vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of R 0 across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate r, the mean generation intervalḠ, and the generation-interval dispersion κ. We then apply our framework to early estimates of R 0 for the SARS-CoV-2 outbreak. We show that many early R 0 estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of R 0 , including the shape of the generation-interval distribution, in efforts to estimate R 0 at the outset of an epidemic.</p>
<p align="right"><i>score: 119</i></p>

</div><br /><br /><br /><div class="card"><h3>Beyond R 0 : the importance of contact tracing when predicting epidemics</h3>
<p style="text-align: justify; text-justify: inter-word;">The basic reproductive number -R 0 -is one of the most common and most commonly misapplied numbers in public health. Nevertheless, estimating R 0 for every transmissible pathogen, emerging or endemic, remains a priority for epidemiologists the world over. Although often used to compare outbreaks and forecast pandemic risk, this single number belies the complexity that two different pathogens can exhibit, even when they have the same R 0 . Here, we show how predicting outbreak size requires both an estimate of R 0 and an estimate of the heterogeneity in the number of secondary infections. To facilitate rapid determination of outbreak risk, we propose a reformulation of a classic result from random network theory that relies on contact tracing data to simultaneously determine the first moment (R 0 ) and the higher moments (representing the heterogeneity) in the distribution of secondary infections. Further, we show how this framework is robust in the face of the typically limited amount of data for emerging pathogens. Lastly, we demonstrate that without data on the heterogeneity in secondary infections for emerging pathogens like 2019-nCoV, the uncertainty in outbreak size ranges dramatically, in the case of 2019-nCoV from 5-40% of susceptible individuals. Taken together, our work highlights the critical need for contact tracing during emerging infectious disease outbreaks and the need to look beyond R 0 when predicting epidemic size.</p>
<p align="right"><i>score: 118</i></p>

</div><br /><br /><br /><div class="card"><h3>Robustness of the reproductive number estimates in vector-borne disease systems Supplementary material Individual Based Model</h3>
<p style="text-align: justify; text-justify: inter-word;">Initially, a disease-free stochastic individual based model for the hosts and vectors of the virus was constructed with no spatial component, and time t discretized into days. Daily, individual humans and vectors were aged, and terminated if their age exceeded their life expectancy. Only the adult stage of a vector's life cycle was considered, since only adult vectors generally acquire and transmit the vector-borne disease. We assumed age-dependent mortalities for both the hosts and the vectors. For both the host and vector, this is governed a Weibull distribution.</p>
<p align="right"><i>score: 117</i></p>

</div><br /><br /><br /><div class="card"><h3>Contact Heterogeneity, Rather Than Transmission Efficiency, Limits the Emergence and Spread of Canine Influenza Virus</h3>
<p style="text-align: justify; text-justify: inter-word;">Host-range shifts in influenza virus are a major risk factor for pandemics. A key question in the study of emerging zoonoses is how the evolution of transmission efficiency interacts with heterogeneity in contact patterns in the new host species, as this interplay influences disease dynamics and prospects for control. Here we use a synergistic mixture of models and data to tease apart the evolutionary and demographic processes controlling a host-range shift in equine H3N8-derived canine influenza virus (CIV). CIV has experienced 15 years of continuous transfer among dogs in the United States, but maintains a patchy distribution, characterized by sporadic short-lived outbreaks coupled with endemic hotspots in large animal shelters. We show that CIV has a high reproductive potential in these facilities (mean R 0 = 3.9) and that these hotspots act as refugia from the sparsely connected majority of the dog population. Intriguingly, CIV has evolved a transmission efficiency that closely matches the minimum required to persist in these refugia, leaving it poised on the extinction/invasion threshold of the host contact network. Corresponding phylogenetic analyses show strong geographic clustering in three US regions, and that the effective reproductive number of the virus (R e ) in the general dog population is close to 1.0. Our results highlight the critical role of host contact structure in CIV dynamics, and show how host contact networks could shape the evolution of pathogen transmission efficiency. Importantly, efficient control measures could eradicate the virus, in turn minimizing the risk of future sustained transmission among companion dogs that could represent a potential new axis to the humananimal interface for influenza.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Using Routine Surveillance Data to Estimate the Epidemic Potential of Emerging Zoonoses: Application to the Emergence of US Swine Origin Influenza A H3N2v Virus</h3>
<p style="text-align: justify; text-justify: inter-word;">Our approach is based on a parsimonious mathematical model of disease transmission and only requires data collected through routine surveillance and standard case investigations. We apply it to assess the transmissibility of swine-origin influenza A H3N2v-M virus in the US, Nipah virus in Malaysia and Bangladesh, and also present a non-zoonotic example (cholera in the Dominican Republic). Estimation is based on two simple summary statistics, the proportion infected by the natural reservoir among detected cases (G) and among the subset of the first detected cases in each cluster (F). If detection of a case does not affect detection of other cases from the same cluster, we find that R can be estimated by 12G; otherwise R can be estimated by 12F when the case detection rate is low. In more general cases, bounds on R can still be derived.</p>
<p align="right"><i>score: 115</i></p>

</div><br /><br /><br /><div class="card"><h3>Estimating human-to-human transmissibility of hepatitis A virus in an outbreak at an elementary school in China, 2011</h3>
<p style="text-align: justify; text-justify: inter-word;">Hepatitis A is caused by hepatitis A virus and occurs worldwide. Estimating the transmissibility, which is usually characterized by the basic reproductive number R 0 , the mean number of secondary infectious cases generated by a single primary infectious case introduced into a totally susceptible population, provides crucial information for the effort required to stop infection spreading. Hepatitis A virus is usually transmitted indirectly through contaminated food and environment. An outbreak from March to June 2011 was reported to have occurred at an elementary school of 698 pupils in China and it was found that the outbreak was due to direct transmission between school children. Based on the symptom onset date and the social contact network of the children, in this study we estimate the serial interval (i.e. the gap in symptom onset between an infectee and its infector) and use different statistical methods to estimate R 0 . Combining with the positivity of IgG antibodies tests, we develop a compartmental transmission dynamics model which includes both asymptomatic and symptomatic infections to estimate the overall R 0 . Our analysis suggests a serial interval of mean = 23.9 days and standard deviation = 20.9 days. The different statistical methods suggest estimates for R 0 in the outbreak varying from 2.1 to 2.8, and the estimates from the transmission dynamics model are consistent with this range. Our estimates are in agreement with that from one study in England but are higher than that from one study in the United States. Our transmission dynamics model suggests that the proportion of symptomatic infections is about 9%, implying that there were about 344 asymptomatic infections along with the 32 observed symptomatic cases. Furthermore, it is shown that the inclusion of asymptomatic infection in the epidemic process increases the estimate of R 0 but does not do so greatly provided that the proportion of symptomatic infections is constant over the outbreak and there is no difference in transmissibility between symptomatic and asymptomatic infections.</p>
<p align="right"><i>score: 114</i></p>

</div><br /><br /><br /><div class="card"><h3>Thompson and Duintjer Tebbens The Journal of Infectious Diseases ®</h3>
<p style="text-align: justify; text-justify: inter-word;">Results. Relatively isolated subpopulations benefit little from high surrounding population immunity to transmission and will sustain transmission as long as they do not attain high vaccination coverage. Failing to reach such subpopulations with high coverage represents the root cause of polio eradication delays. Achieving and maintaining eradication requires addressing the weakest links, which includes immunizing populations in insecure areas and/or with disrupted or poor-performing health systems and managing the risks of individuals with primary immunodeficiencies who can excrete vaccine-derived poliovirus long-term.</p>
<p align="right"><i>score: 111</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</h3>
<p style="text-align: justify; text-justify: inter-word;">Backgrounds: An ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</h3>
<p style="text-align: justify; text-justify: inter-word;">The early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Determining the dynamics of influenza transmission by age</h3>
<p style="text-align: justify; text-justify: inter-word;">Background: It is widely accepted that influenza transmission dynamics vary by age; however methods to quantify the reproductive number by age group are limited. We introduce a simple method to estimate the reproductive number by modifying the method originally proposed by Wallinga and Teunis and using existing information on contact patterns between age groups. We additionally perform a sensitivity analysis to determine the potential impact of differential healthcare seeking patterns by age. We illustrate this method using data from the 2009 H1N1 Influenza pandemic in Gauteng Province, South Africa.</p>
<p align="right"><i>score: 110</i></p>

</div><br /><br /><br /><div class="card"><h3>Assessing the In Vitro Fitness of an Oseltamivir-Resistant Seasonal A/H1N1 Influenza Strain Using a Mathematical Model</h3>
<p style="text-align: justify; text-justify: inter-word;">In 2007, the A/Brisbane/59/2007 (H1N1) seasonal influenza virus strain acquired the oseltamivir-resistance mutation H275Y in its neuraminidase (NA) gene. Although previous studies had demonstrated that this mutation impaired the replication capacity of the influenza virus in vitro and in vivo, the A/Brisbane/59/2007 H275Y oseltamivir-resistant mutant completely out-competed the wild-type (WT) strain and was, in the 2008-2009 influenza season, the primary A/H1N1 circulating strain. Using a combination of plaque and viral yield assays, and a simple mathematical model, approximate values were extracted for two basic viral kinetics parameters of the in vitro infection. In the ST6GalI-MDCK cell line, the latent infection period (i.e., the time for a newly infected cell to start releasing virions) was found to be 1-3 h for the WT strain and more than 7 h for the H275Y mutant. The infecting time (i.e., the time for a single infectious cell to cause the infection of another one) was between 30 and 80 min for the WT, and less than 5 min for the H275Y mutant. Single-cycle viral yield experiments have provided qualitative confirmation of these findings. These results, though preliminary, suggest that the increased fitness success of the A/Brisbane/59/2007 H275Y mutant may be due to increased infectivity compensating for an impaired or delayed viral release, and are consistent with recent evidence for the mechanistic origins of fitness reduction and recovery in NA expression. The method applied here can reconcile seemingly contradictory results from the plaque and yield assays as two complementary views of replication kinetics, with both required to fully capture a strain's fitness.</p>
<p align="right"><i>score: 109</i></p>

</div><br /><br /><br /></body>